{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "provenance": [],
      "authorship_tag": "ABX9TyP07xNMR0oBwjOKkana5XZd",
      "include_colab_link": true
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    },
    "language_info": {
      "name": "python"
    }
  },
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "view-in-github",
        "colab_type": "text"
      },
      "source": [
        "<a href=\"https://colab.research.google.com/github/RochaGerd/Chemistry_with_Python/blob/main/8th_EQC_Suporte.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "|    |    |    |\n",
        "|:----:|:----:|:----:|\n",
        "|<img src = \"https://github.com/RochaGerd/Chemistry_with_Python/blob/main/figures/Imagem_UFPB.gif?raw=true\" width = \"150\"> | <img src = \"https://github.com/RochaGerd/Chemistry_with_Python/blob/main/figures/Imagem_DQ_UFPB.png?raw=true\" width = \"250\"> | <img src = \"https://github.com/RochaGerd/Chemistry_with_Python/blob/main/figures/LQQC_2024.png?raw=true\" width = \"300\">|"
      ],
      "metadata": {
        "id": "HcusHx6dYc_v"
      }
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "5R90HG8BEeUO"
      },
      "source": [
        "#**M√âTODOS QU√ÇNTICOS SEMIEMP√çRICOS: TEORIA E PR√ÅTICA**\n",
        "##***Modelando sistemas biol√≥gicos com m√©todos Semiemp√≠ricos de qu√≠mica qu√¢ntica***\n",
        "\n",
        "##Autores: Prof. Gerd Bruno Rocha, Dr. Igor Barden Grillo e M.Sc. Luiz Eduado Gomes da Cruz\n",
        "###(gbr@academico.ufpb.br e https://www.quantum-chem.pro.br/)\n",
        "\n",
        "####Links:\n",
        "\n",
        "1. <https://github.com/RochaGerd/Chemistry_with_Python>\n",
        "2. <https://pense-python.caravela.club/introducao.html>\n",
        "3. <https://github.com/pythonchembook>\n",
        "\n",
        "***Vers√£o: 1.0 (04-DEZ-24)***\n",
        "\n",
        "----------------------\n",
        "\n",
        "### **Minicurso pr√°tico**\n",
        "#### *Dura√ß√£o Total: 4 horas, divididos em dois dias*\n",
        "\n",
        "Requisitos: Ter familiaridade com Python e conceitos de qu√≠mica. Possuir conta no Google (**prefer√™ncia**) ou Replit.\n",
        "\n",
        "Ambiente de programa√ß√£o: **Plataforma Google Colab (<https://colab.research.google.com/>)** ou Replit (<https://replit.com/>).\n",
        "\n",
        "\n",
        "***Material bibliogr√°fico sobre uso de Python em Qu√≠mica:***\n",
        "\n",
        "1. Aprendendo Qu√≠mica com Python, Rodrigo Queiroz e Gerd Rocha, 2021, Amazon Book. <https://github.com/pythonchembook>\n",
        "\n",
        "  Link para compra: <https://www.amazon.com/Aprendendo-Qu%C3%ADmica-com-python-Portuguese/dp/B09LGSG6TY>\n",
        "\n",
        "2. J. Mueller. Come√ßando a Programar em Python Para Leigos. Alta Books, 2020.\n",
        "3. A. Downey. Pense em Python: Pense como um cientista da computa√ß√£o. Novatec, 2016.\n",
        "4. <https://penseallen.github.io/PensePython2e/>\n",
        "5. <https://python-guide-pt-br.readthedocs.io/pt_BR/latest/>\n",
        "6. <https://aprendendo-computacao-com-python.readthedocs.io/en/latest/index.html>\n",
        "7. Websites, tutoriais, artigos (visite o portal do Journal of chemical education) e outros mais.\n",
        "\n",
        "---------------------------\n",
        "\n",
        "\n"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "TiDeGJfU1aFL"
      },
      "source": [
        "\n",
        "###**Conte√∫do do Minicurso te√≥rico/pr√°tico:**\n",
        "\n",
        "####**(Dia 10/12/24): Teoria semiemp√≠rica, apresenta√ß√£o do programa MOPAC, C√°lculos termodin√¢micos de biomol√©culas e C√°lculos de estrutura eletr√¥nica de biomol√©culas**\n",
        "\n",
        "####**Considera√ß√µes finais**\n",
        "####**Formul√°rio de *feedback* do minicurso**\n",
        "\n",
        "-------------\n",
        "\n",
        "**Tutoriais e Documenta√ß√£o**\n",
        "   - [Google Colaboratory Documentation](<https://colab.research.google.com/notebooks/intro.ipynb>)\n",
        "   "
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "cy85UL_CgaHz"
      },
      "source": [
        "---------------\n",
        "##**Objetivos**\n",
        "\n",
        "1. Permitir a populariza√ß√£o da programa√ß√£o para qu√≠micos(a)s\n",
        "2. Divulgar aspectos introdut√≥rios dos m√©todos de modelagem/simula√ß√£o molecular como estrat√©gia de aprendizado de qu√≠mica\n",
        "3. Mostrar a potencialidade da modelagem/simula√ß√£o molecular para pesquisa em qu√≠mica e √°reas afins\n",
        "4. Apresentar uma platafoma online que permite realizar modelagem/simula√ß√£o molecular *open source*."
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "7RthalL70Dyl"
      },
      "source": [
        "-----------\n",
        "## **P√∫blico alvo**\n",
        "\n",
        "1. Alunos de gradua√ß√£o e de p√≥s-gradua√ß√£o sem experi√™ncia com modelagem/simula√ß√£o molecular\n",
        "2. Alunos de gradua√ß√£o e de p√≥s-gradua√ß√£o com pouca base de programa√ß√£o em python\n",
        "3. Pesquisadores interessados na tem√°tica\n",
        "\n",
        "------------"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 1,
      "metadata": {
        "id": "I8OzKLPWnB6t",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "2be0343c-0615-4e1d-fb38-660813e14739"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Running on colab\n",
            "‚ú®üç∞‚ú® Everything looks OK!\n"
          ]
        }
      ],
      "source": [
        "#@title <font color=\"purple\"> **Instalando Conda/Mamba (OBS: come√ßar por esse comando \\*)** </font>\n",
        "# Execute this cell to install mamba in the Colab environment\n",
        "\n",
        "if 'google.colab' in str(get_ipython()):\n",
        "  print('Running on colab')\n",
        "  !pip install -q condacolab\n",
        "  import condacolab\n",
        "  condacolab.install_mambaforge()\n",
        "else:\n",
        "  print('Not running on colab.')\n",
        "  print('Make sure you create and activate a new conda environment!')"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 2,
      "metadata": {
        "id": "oEvqiAnPnCCR",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "50792676-c538-446a-b3f7-b65fff6b9479"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "conda 23.11.0\n"
          ]
        }
      ],
      "source": [
        "!conda --version"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "C_IkuWmpnCF0"
      },
      "outputs": [],
      "source": [
        "#!which conda"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "modS6m1qnCJD"
      },
      "outputs": [],
      "source": [
        "# Instalando o AMBERTools\n",
        "\n",
        "#!conda install -c conda-forge ambertools"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "PKE4K-AqojzL"
      },
      "outputs": [],
      "source": [
        "#!which tleap"
      ]
    },
    {
      "cell_type": "code",
      "source": [],
      "metadata": {
        "id": "3RzD6F3nPy23"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "#@title **Instalando/Importando libs importantes**\n",
        "# install the 3rd party chemistry packages\n",
        "!pip install -q rdkit\n",
        "!pip install -q py3Dmol # --> https://pypi.org/project/py3Dmol/\n",
        "!pip install -q pythreejs\n",
        "##!pip install -q git+https://github.com/funkymunkycool/Cube-Toolz.git\n",
        "\n",
        "# Instalando o NGL Viwer\n",
        "!pip install ipywidgets==7.7.2 ## nglview\n",
        "\n",
        "# import everything\n",
        "import warnings\n",
        "warnings.filterwarnings(\"ignore\")\n",
        "import pathlib\n",
        "import sys, os\n",
        "import numpy as np\n",
        "import pandas as pd\n",
        "import seaborn as sns\n",
        "from termcolor import cprint\n",
        "import py3Dmol as p3d\n",
        "#import cube_tools\n",
        "from google.colab import output\n",
        "output.enable_custom_widget_manager()\n",
        "# This import is required to handle files\n",
        "from google.colab import files\n",
        "# This imports are required to use NGL viewer as well as the capabilities of pyCapsid to generate structures.\n",
        "#import nglview as ngl\n",
        "#from nglview.adaptor import FileStructure\n",
        "import matplotlib\n",
        "from matplotlib import pyplot as plt\n",
        "from matplotlib.offsetbox import OffsetImage, AnnotationBbox\n",
        "import ipywidgets as widgets\n",
        "%matplotlib inline\n",
        "import copy\n",
        "from IPython.display import Markdown, display, clear_output"
      ],
      "metadata": {
        "id": "CMaXcLsyhp7r",
        "cellView": "form",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "5cc83465-a5d9-4600-d2d0-3c28b3c1431d"
      },
      "execution_count": 28,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Requirement already satisfied: ipywidgets==7.7.2 in /usr/local/lib/python3.10/site-packages (7.7.2)\n",
            "Requirement already satisfied: ipykernel>=4.5.1 in /usr/local/lib/python3.10/site-packages (from ipywidgets==7.7.2) (6.29.5)\n",
            "Requirement already satisfied: ipython-genutils~=0.2.0 in /usr/local/lib/python3.10/site-packages (from ipywidgets==7.7.2) (0.2.0)\n",
            "Requirement already satisfied: traitlets>=4.3.1 in /usr/local/lib/python3.10/site-packages (from ipywidgets==7.7.2) (5.14.3)\n",
            "Requirement already satisfied: widgetsnbextension~=3.6.0 in /usr/local/lib/python3.10/site-packages (from ipywidgets==7.7.2) (3.6.10)\n",
            "Requirement already satisfied: ipython>=4.0.0 in /usr/local/lib/python3.10/site-packages (from ipywidgets==7.7.2) (8.30.0)\n",
            "Requirement already satisfied: jupyterlab-widgets<3,>=1.0.0 in /usr/local/lib/python3.10/site-packages (from ipywidgets==7.7.2) (1.1.11)\n",
            "Requirement already satisfied: comm>=0.1.1 in /usr/local/lib/python3.10/site-packages (from ipykernel>=4.5.1->ipywidgets==7.7.2) (0.2.2)\n",
            "Requirement already satisfied: debugpy>=1.6.5 in /usr/local/lib/python3.10/site-packages (from ipykernel>=4.5.1->ipywidgets==7.7.2) (1.8.9)\n",
            "Requirement already satisfied: jupyter-client>=6.1.12 in /usr/local/lib/python3.10/site-packages (from ipykernel>=4.5.1->ipywidgets==7.7.2) (8.6.3)\n",
            "Requirement already satisfied: jupyter-core!=5.0.*,>=4.12 in /usr/local/lib/python3.10/site-packages (from ipykernel>=4.5.1->ipywidgets==7.7.2) (5.7.2)\n",
            "Requirement already satisfied: matplotlib-inline>=0.1 in /usr/local/lib/python3.10/site-packages (from ipykernel>=4.5.1->ipywidgets==7.7.2) (0.1.7)\n",
            "Requirement already satisfied: nest-asyncio in /usr/local/lib/python3.10/site-packages (from ipykernel>=4.5.1->ipywidgets==7.7.2) (1.6.0)\n",
            "Requirement already satisfied: packaging in /usr/local/lib/python3.10/site-packages (from ipykernel>=4.5.1->ipywidgets==7.7.2) (23.2)\n",
            "Requirement already satisfied: psutil in /usr/local/lib/python3.10/site-packages (from ipykernel>=4.5.1->ipywidgets==7.7.2) (6.1.0)\n",
            "Requirement already satisfied: pyzmq>=24 in /usr/local/lib/python3.10/site-packages (from ipykernel>=4.5.1->ipywidgets==7.7.2) (26.2.0)\n",
            "Requirement already satisfied: tornado>=6.1 in /usr/local/lib/python3.10/site-packages (from ipykernel>=4.5.1->ipywidgets==7.7.2) (6.4.2)\n",
            "Requirement already satisfied: decorator in /usr/local/lib/python3.10/site-packages (from ipython>=4.0.0->ipywidgets==7.7.2) (5.1.1)\n",
            "Requirement already satisfied: exceptiongroup in /usr/local/lib/python3.10/site-packages (from ipython>=4.0.0->ipywidgets==7.7.2) (1.2.2)\n",
            "Requirement already satisfied: jedi>=0.16 in /usr/local/lib/python3.10/site-packages (from ipython>=4.0.0->ipywidgets==7.7.2) (0.19.2)\n",
            "Requirement already satisfied: pexpect>4.3 in /usr/local/lib/python3.10/site-packages (from ipython>=4.0.0->ipywidgets==7.7.2) (4.9.0)\n",
            "Requirement already satisfied: prompt_toolkit<3.1.0,>=3.0.41 in /usr/local/lib/python3.10/site-packages (from ipython>=4.0.0->ipywidgets==7.7.2) (3.0.48)\n",
            "Requirement already satisfied: pygments>=2.4.0 in /usr/local/lib/python3.10/site-packages (from ipython>=4.0.0->ipywidgets==7.7.2) (2.18.0)\n",
            "Requirement already satisfied: stack_data in /usr/local/lib/python3.10/site-packages (from ipython>=4.0.0->ipywidgets==7.7.2) (0.6.3)\n",
            "Requirement already satisfied: typing_extensions>=4.6 in /usr/local/lib/python3.10/site-packages (from ipython>=4.0.0->ipywidgets==7.7.2) (4.12.2)\n",
            "Requirement already satisfied: notebook>=4.4.1 in /usr/local/lib/python3.10/site-packages (from widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (7.3.1)\n",
            "Requirement already satisfied: parso<0.9.0,>=0.8.4 in /usr/local/lib/python3.10/site-packages (from jedi>=0.16->ipython>=4.0.0->ipywidgets==7.7.2) (0.8.4)\n",
            "Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.10/site-packages (from jupyter-client>=6.1.12->ipykernel>=4.5.1->ipywidgets==7.7.2) (2.9.0.post0)\n",
            "Requirement already satisfied: platformdirs>=2.5 in /usr/local/lib/python3.10/site-packages (from jupyter-core!=5.0.*,>=4.12->ipykernel>=4.5.1->ipywidgets==7.7.2) (4.1.0)\n",
            "Requirement already satisfied: jupyter-server<3,>=2.4.0 in /usr/local/lib/python3.10/site-packages (from notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (2.14.2)\n",
            "Requirement already satisfied: jupyterlab-server<3,>=2.27.1 in /usr/local/lib/python3.10/site-packages (from notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (2.27.3)\n",
            "Requirement already satisfied: jupyterlab<4.4,>=4.3.2 in /usr/local/lib/python3.10/site-packages (from notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (4.3.3)\n",
            "Requirement already satisfied: notebook-shim<0.3,>=0.2 in /usr/local/lib/python3.10/site-packages (from notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (0.2.4)\n",
            "Requirement already satisfied: ptyprocess>=0.5 in /usr/local/lib/python3.10/site-packages (from pexpect>4.3->ipython>=4.0.0->ipywidgets==7.7.2) (0.7.0)\n",
            "Requirement already satisfied: wcwidth in /usr/local/lib/python3.10/site-packages (from prompt_toolkit<3.1.0,>=3.0.41->ipython>=4.0.0->ipywidgets==7.7.2) (0.2.13)\n",
            "Requirement already satisfied: executing>=1.2.0 in /usr/local/lib/python3.10/site-packages (from stack_data->ipython>=4.0.0->ipywidgets==7.7.2) (2.1.0)\n",
            "Requirement already satisfied: asttokens>=2.1.0 in /usr/local/lib/python3.10/site-packages (from stack_data->ipython>=4.0.0->ipywidgets==7.7.2) (3.0.0)\n",
            "Requirement already satisfied: pure-eval in /usr/local/lib/python3.10/site-packages (from stack_data->ipython>=4.0.0->ipywidgets==7.7.2) (0.2.3)\n",
            "Requirement already satisfied: anyio>=3.1.0 in /usr/local/lib/python3.10/site-packages (from jupyter-server<3,>=2.4.0->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (4.7.0)\n",
            "Requirement already satisfied: argon2-cffi>=21.1 in /usr/local/lib/python3.10/site-packages (from jupyter-server<3,>=2.4.0->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (23.1.0)\n",
            "Requirement already satisfied: jinja2>=3.0.3 in /usr/local/lib/python3.10/site-packages (from jupyter-server<3,>=2.4.0->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (3.1.4)\n",
            "Requirement already satisfied: jupyter-events>=0.9.0 in /usr/local/lib/python3.10/site-packages (from jupyter-server<3,>=2.4.0->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (0.10.0)\n",
            "Requirement already satisfied: jupyter-server-terminals>=0.4.4 in /usr/local/lib/python3.10/site-packages (from jupyter-server<3,>=2.4.0->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (0.5.3)\n",
            "Requirement already satisfied: nbconvert>=6.4.4 in /usr/local/lib/python3.10/site-packages (from jupyter-server<3,>=2.4.0->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (7.16.4)\n",
            "Requirement already satisfied: nbformat>=5.3.0 in /usr/local/lib/python3.10/site-packages (from jupyter-server<3,>=2.4.0->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (5.10.4)\n",
            "Requirement already satisfied: overrides>=5.0 in /usr/local/lib/python3.10/site-packages (from jupyter-server<3,>=2.4.0->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (7.7.0)\n",
            "Requirement already satisfied: prometheus-client>=0.9 in /usr/local/lib/python3.10/site-packages (from jupyter-server<3,>=2.4.0->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (0.21.1)\n",
            "Requirement already satisfied: send2trash>=1.8.2 in /usr/local/lib/python3.10/site-packages (from jupyter-server<3,>=2.4.0->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (1.8.3)\n",
            "Requirement already satisfied: terminado>=0.8.3 in /usr/local/lib/python3.10/site-packages (from jupyter-server<3,>=2.4.0->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (0.18.1)\n",
            "Requirement already satisfied: websocket-client>=1.7 in /usr/local/lib/python3.10/site-packages (from jupyter-server<3,>=2.4.0->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (1.8.0)\n",
            "Requirement already satisfied: async-lru>=1.0.0 in /usr/local/lib/python3.10/site-packages (from jupyterlab<4.4,>=4.3.2->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (2.0.4)\n",
            "Requirement already satisfied: httpx>=0.25.0 in /usr/local/lib/python3.10/site-packages (from jupyterlab<4.4,>=4.3.2->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (0.28.1)\n",
            "Requirement already satisfied: jupyter-lsp>=2.0.0 in /usr/local/lib/python3.10/site-packages (from jupyterlab<4.4,>=4.3.2->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (2.2.5)\n",
            "Requirement already satisfied: setuptools>=40.8.0 in /usr/local/lib/python3.10/site-packages (from jupyterlab<4.4,>=4.3.2->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (68.2.2)\n",
            "Requirement already satisfied: tomli>=1.2.2 in /usr/local/lib/python3.10/site-packages (from jupyterlab<4.4,>=4.3.2->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (2.2.1)\n",
            "Requirement already satisfied: babel>=2.10 in /usr/local/lib/python3.10/site-packages (from jupyterlab-server<3,>=2.27.1->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (2.16.0)\n",
            "Requirement already satisfied: json5>=0.9.0 in /usr/local/lib/python3.10/site-packages (from jupyterlab-server<3,>=2.27.1->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (0.10.0)\n",
            "Requirement already satisfied: jsonschema>=4.18.0 in /usr/local/lib/python3.10/site-packages (from jupyterlab-server<3,>=2.27.1->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (4.23.0)\n",
            "Requirement already satisfied: requests>=2.31 in /usr/local/lib/python3.10/site-packages (from jupyterlab-server<3,>=2.27.1->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (2.31.0)\n",
            "Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.10/site-packages (from python-dateutil>=2.8.2->jupyter-client>=6.1.12->ipykernel>=4.5.1->ipywidgets==7.7.2) (1.17.0)\n",
            "Requirement already satisfied: idna>=2.8 in /usr/local/lib/python3.10/site-packages (from anyio>=3.1.0->jupyter-server<3,>=2.4.0->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (3.6)\n",
            "Requirement already satisfied: sniffio>=1.1 in /usr/local/lib/python3.10/site-packages (from anyio>=3.1.0->jupyter-server<3,>=2.4.0->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (1.3.1)\n",
            "Requirement already satisfied: argon2-cffi-bindings in /usr/local/lib/python3.10/site-packages (from argon2-cffi>=21.1->jupyter-server<3,>=2.4.0->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (21.2.0)\n",
            "Requirement already satisfied: certifi in /usr/local/lib/python3.10/site-packages (from httpx>=0.25.0->jupyterlab<4.4,>=4.3.2->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (2024.8.30)\n",
            "Requirement already satisfied: httpcore==1.* in /usr/local/lib/python3.10/site-packages (from httpx>=0.25.0->jupyterlab<4.4,>=4.3.2->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (1.0.7)\n",
            "Requirement already satisfied: h11<0.15,>=0.13 in /usr/local/lib/python3.10/site-packages (from httpcore==1.*->httpx>=0.25.0->jupyterlab<4.4,>=4.3.2->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (0.14.0)\n",
            "Requirement already satisfied: MarkupSafe>=2.0 in /usr/local/lib/python3.10/site-packages (from jinja2>=3.0.3->jupyter-server<3,>=2.4.0->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (3.0.2)\n",
            "Requirement already satisfied: attrs>=22.2.0 in /usr/local/lib/python3.10/site-packages (from jsonschema>=4.18.0->jupyterlab-server<3,>=2.27.1->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (24.2.0)\n",
            "Requirement already satisfied: jsonschema-specifications>=2023.03.6 in /usr/local/lib/python3.10/site-packages (from jsonschema>=4.18.0->jupyterlab-server<3,>=2.27.1->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (2024.10.1)\n",
            "Requirement already satisfied: referencing>=0.28.4 in /usr/local/lib/python3.10/site-packages (from jsonschema>=4.18.0->jupyterlab-server<3,>=2.27.1->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (0.35.1)\n",
            "Requirement already satisfied: rpds-py>=0.7.1 in /usr/local/lib/python3.10/site-packages (from jsonschema>=4.18.0->jupyterlab-server<3,>=2.27.1->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (0.22.3)\n",
            "Requirement already satisfied: python-json-logger>=2.0.4 in /usr/local/lib/python3.10/site-packages (from jupyter-events>=0.9.0->jupyter-server<3,>=2.4.0->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (2.0.7)\n",
            "Requirement already satisfied: pyyaml>=5.3 in /usr/local/lib/python3.10/site-packages (from jupyter-events>=0.9.0->jupyter-server<3,>=2.4.0->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (6.0.2)\n",
            "Requirement already satisfied: rfc3339-validator in /usr/local/lib/python3.10/site-packages (from jupyter-events>=0.9.0->jupyter-server<3,>=2.4.0->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (0.1.4)\n",
            "Requirement already satisfied: rfc3986-validator>=0.1.1 in /usr/local/lib/python3.10/site-packages (from jupyter-events>=0.9.0->jupyter-server<3,>=2.4.0->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (0.1.1)\n",
            "Requirement already satisfied: beautifulsoup4 in /usr/local/lib/python3.10/site-packages (from nbconvert>=6.4.4->jupyter-server<3,>=2.4.0->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (4.12.3)\n",
            "Requirement already satisfied: bleach!=5.0.0 in /usr/local/lib/python3.10/site-packages (from nbconvert>=6.4.4->jupyter-server<3,>=2.4.0->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (6.2.0)\n",
            "Requirement already satisfied: defusedxml in /usr/local/lib/python3.10/site-packages (from nbconvert>=6.4.4->jupyter-server<3,>=2.4.0->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (0.7.1)\n",
            "Requirement already satisfied: jupyterlab-pygments in /usr/local/lib/python3.10/site-packages (from nbconvert>=6.4.4->jupyter-server<3,>=2.4.0->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (0.3.0)\n",
            "Requirement already satisfied: mistune<4,>=2.0.3 in /usr/local/lib/python3.10/site-packages (from nbconvert>=6.4.4->jupyter-server<3,>=2.4.0->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (3.0.2)\n",
            "Requirement already satisfied: nbclient>=0.5.0 in /usr/local/lib/python3.10/site-packages (from nbconvert>=6.4.4->jupyter-server<3,>=2.4.0->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (0.10.1)\n",
            "Requirement already satisfied: pandocfilters>=1.4.1 in /usr/local/lib/python3.10/site-packages (from nbconvert>=6.4.4->jupyter-server<3,>=2.4.0->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (1.5.1)\n",
            "Requirement already satisfied: tinycss2 in /usr/local/lib/python3.10/site-packages (from nbconvert>=6.4.4->jupyter-server<3,>=2.4.0->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (1.4.0)\n",
            "Requirement already satisfied: fastjsonschema>=2.15 in /usr/local/lib/python3.10/site-packages (from nbformat>=5.3.0->jupyter-server<3,>=2.4.0->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (2.21.1)\n",
            "Requirement already satisfied: charset-normalizer<4,>=2 in /usr/local/lib/python3.10/site-packages (from requests>=2.31->jupyterlab-server<3,>=2.27.1->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (3.3.2)\n",
            "Requirement already satisfied: urllib3<3,>=1.21.1 in /usr/local/lib/python3.10/site-packages (from requests>=2.31->jupyterlab-server<3,>=2.27.1->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (2.1.0)\n",
            "Requirement already satisfied: webencodings in /usr/local/lib/python3.10/site-packages (from bleach!=5.0.0->nbconvert>=6.4.4->jupyter-server<3,>=2.4.0->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (0.5.1)\n",
            "Requirement already satisfied: fqdn in /usr/local/lib/python3.10/site-packages (from jsonschema[format-nongpl]>=4.18.0->jupyter-events>=0.9.0->jupyter-server<3,>=2.4.0->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (1.5.1)\n",
            "Requirement already satisfied: isoduration in /usr/local/lib/python3.10/site-packages (from jsonschema[format-nongpl]>=4.18.0->jupyter-events>=0.9.0->jupyter-server<3,>=2.4.0->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (20.11.0)\n",
            "Requirement already satisfied: jsonpointer>1.13 in /usr/local/lib/python3.10/site-packages (from jsonschema[format-nongpl]>=4.18.0->jupyter-events>=0.9.0->jupyter-server<3,>=2.4.0->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (2.4)\n",
            "Requirement already satisfied: uri-template in /usr/local/lib/python3.10/site-packages (from jsonschema[format-nongpl]>=4.18.0->jupyter-events>=0.9.0->jupyter-server<3,>=2.4.0->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (1.3.0)\n",
            "Requirement already satisfied: webcolors>=24.6.0 in /usr/local/lib/python3.10/site-packages (from jsonschema[format-nongpl]>=4.18.0->jupyter-events>=0.9.0->jupyter-server<3,>=2.4.0->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (24.11.1)\n",
            "Requirement already satisfied: cffi>=1.0.1 in /usr/local/lib/python3.10/site-packages (from argon2-cffi-bindings->argon2-cffi>=21.1->jupyter-server<3,>=2.4.0->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (1.16.0)\n",
            "Requirement already satisfied: soupsieve>1.2 in /usr/local/lib/python3.10/site-packages (from beautifulsoup4->nbconvert>=6.4.4->jupyter-server<3,>=2.4.0->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (2.6)\n",
            "Requirement already satisfied: pycparser in /usr/local/lib/python3.10/site-packages (from cffi>=1.0.1->argon2-cffi-bindings->argon2-cffi>=21.1->jupyter-server<3,>=2.4.0->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (2.21)\n",
            "Requirement already satisfied: arrow>=0.15.0 in /usr/local/lib/python3.10/site-packages (from isoduration->jsonschema[format-nongpl]>=4.18.0->jupyter-events>=0.9.0->jupyter-server<3,>=2.4.0->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (1.3.0)\n",
            "Requirement already satisfied: types-python-dateutil>=2.8.10 in /usr/local/lib/python3.10/site-packages (from arrow>=0.15.0->isoduration->jsonschema[format-nongpl]>=4.18.0->jupyter-events>=0.9.0->jupyter-server<3,>=2.4.0->notebook>=4.4.1->widgetsnbextension~=3.6.0->ipywidgets==7.7.2) (2.9.0.20241206)\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "# import py3Dmol as p3d\n",
        "print('Vers√£o do py3DMol --> ', p3d.__version__)"
      ],
      "metadata": {
        "id": "TySN5AQgmxRf",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "047b3e69-38e3-457a-b90b-586883f9794d"
      },
      "execution_count": 29,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Vers√£o do py3DMol -->  2.4.2\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "# import nglview as ngl\n",
        "#print('Vers√£o do nglview --> ', ngl.__version__)"
      ],
      "metadata": {
        "id": "cKjNbcp-kCXu"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "source": [
        "-----------"
      ],
      "metadata": {
        "id": "3uS8vVG6w9ZV"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "# Instalando RDKit\n",
        "!pip install --upgrade rdkit"
      ],
      "metadata": {
        "id": "WrJpL1HoE5R3",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "fdff9c45-ac84-46f1-f40c-36ec7fae95ba"
      },
      "execution_count": 3,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Requirement already satisfied: rdkit in /usr/local/lib/python3.10/site-packages (2024.3.6)\n",
            "Requirement already satisfied: numpy in /usr/local/lib/python3.10/site-packages (from rdkit) (2.2.0)\n",
            "Requirement already satisfied: Pillow in /usr/local/lib/python3.10/site-packages (from rdkit) (11.0.0)\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "# Importando os m√≥dulos do RDKit\n",
        "from rdkit import Chem\n",
        "from rdkit.Chem.Draw import IPythonConsole\n",
        "from rdkit.Chem import Draw, AllChem\n",
        "from rdkit.Chem import rdDistGeom, rdCoordGen\n",
        "from rdkit.Chem import rdMolAlign\n",
        "IPythonConsole.ipython_useSVG=True  # <-- Colocar Falso se voc√™ quiser PNGs ao inv√©s de SVGs\n",
        "%config InlineBackend.figure_format = 'svg'\n",
        "from rdkit import rdBase\n",
        "print(\"Vers√£o da RDKit:\", rdBase.rdkitVersion)\n",
        "print(\"Vers√£o da RDKit base:\", rdBase.boostVersion)"
      ],
      "metadata": {
        "id": "hFETv0RXg-Mg",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "e7eca607-7e53-4107-9ed8-778efd31600f"
      },
      "execution_count": 4,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Vers√£o da RDKit: 2024.03.6\n",
            "Vers√£o da RDKit base: 1_85\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "# Gerar estrutura de linhas a partir do c√≥digo SMILES\n",
        "mol = Chem.MolFromSmiles('c1ccccc1')\n",
        "mol"
      ],
      "metadata": {
        "id": "oQQGao1Pg-QJ",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 171
        },
        "outputId": "d08a78b6-e771-46ea-8be6-cf65ee97581a"
      },
      "execution_count": 5,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "<rdkit.Chem.rdchem.Mol at 0x7c62bf39a340>"
            ],
            "text/html": [
              "<?xml version='1.0' encoding='iso-8859-1'?>\n",
              "<svg version='1.1' baseProfile='full'\n",
              "              xmlns='http://www.w3.org/2000/svg'\n",
              "                      xmlns:rdkit='http://www.rdkit.org/xml'\n",
              "                      xmlns:xlink='http://www.w3.org/1999/xlink'\n",
              "                  xml:space='preserve'\n",
              "width='450px' height='150px' viewBox='0 0 450 150'>\n",
              "<!-- END OF HEADER -->\n",
              "<rect style='opacity:1.0;fill:#FFFFFF;stroke:none' width='450.0' height='150.0' x='0.0' y='0.0'> </rect>\n",
              "<path class='bond-0 atom-0 atom-1' d='M 302.9,75.0 L 264.0,142.5' style='fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:2.0px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1' />\n",
              "<path class='bond-0 atom-0 atom-1' d='M 289.4,75.0 L 257.2,130.8' style='fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:2.0px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1' />\n",
              "<path class='bond-1 atom-1 atom-2' d='M 264.0,142.5 L 186.0,142.5' style='fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:2.0px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1' />\n",
              "<path class='bond-2 atom-2 atom-3' d='M 186.0,142.5 L 147.1,75.0' style='fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:2.0px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1' />\n",
              "<path class='bond-2 atom-2 atom-3' d='M 192.8,130.8 L 160.6,75.0' style='fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:2.0px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1' />\n",
              "<path class='bond-3 atom-3 atom-4' d='M 147.1,75.0 L 186.0,7.5' style='fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:2.0px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1' />\n",
              "<path class='bond-4 atom-4 atom-5' d='M 186.0,7.5 L 264.0,7.5' style='fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:2.0px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1' />\n",
              "<path class='bond-4 atom-4 atom-5' d='M 192.8,19.2 L 257.2,19.2' style='fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:2.0px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1' />\n",
              "<path class='bond-5 atom-5 atom-0' d='M 264.0,7.5 L 302.9,75.0' style='fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:2.0px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1' />\n",
              "<path d='M 301.0,78.4 L 302.9,75.0 L 301.0,71.6' style='fill:none;stroke:#000000;stroke-width:2.0px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1;' />\n",
              "<path d='M 265.9,139.1 L 264.0,142.5 L 260.1,142.5' style='fill:none;stroke:#000000;stroke-width:2.0px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1;' />\n",
              "<path d='M 189.9,142.5 L 186.0,142.5 L 184.1,139.1' style='fill:none;stroke:#000000;stroke-width:2.0px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1;' />\n",
              "<path d='M 149.0,78.4 L 147.1,75.0 L 149.0,71.6' style='fill:none;stroke:#000000;stroke-width:2.0px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1;' />\n",
              "<path d='M 184.1,10.9 L 186.0,7.5 L 189.9,7.5' style='fill:none;stroke:#000000;stroke-width:2.0px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1;' />\n",
              "<path d='M 260.1,7.5 L 264.0,7.5 L 265.9,10.9' style='fill:none;stroke:#000000;stroke-width:2.0px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1;' />\n",
              "</svg>\n"
            ],
            "image/svg+xml": "<?xml version='1.0' encoding='iso-8859-1'?>\n<svg version='1.1' baseProfile='full'\n              xmlns='http://www.w3.org/2000/svg'\n                      xmlns:rdkit='http://www.rdkit.org/xml'\n                      xmlns:xlink='http://www.w3.org/1999/xlink'\n                  xml:space='preserve'\nwidth='450px' height='150px' viewBox='0 0 450 150'>\n<!-- END OF HEADER -->\n<rect style='opacity:1.0;fill:#FFFFFF;stroke:none' width='450.0' height='150.0' x='0.0' y='0.0'> </rect>\n<path class='bond-0 atom-0 atom-1' d='M 302.9,75.0 L 264.0,142.5' style='fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:2.0px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1' />\n<path class='bond-0 atom-0 atom-1' d='M 289.4,75.0 L 257.2,130.8' style='fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:2.0px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1' />\n<path class='bond-1 atom-1 atom-2' d='M 264.0,142.5 L 186.0,142.5' style='fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:2.0px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1' />\n<path class='bond-2 atom-2 atom-3' d='M 186.0,142.5 L 147.1,75.0' style='fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:2.0px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1' />\n<path class='bond-2 atom-2 atom-3' d='M 192.8,130.8 L 160.6,75.0' style='fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:2.0px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1' />\n<path class='bond-3 atom-3 atom-4' d='M 147.1,75.0 L 186.0,7.5' style='fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:2.0px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1' />\n<path class='bond-4 atom-4 atom-5' d='M 186.0,7.5 L 264.0,7.5' style='fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:2.0px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1' />\n<path class='bond-4 atom-4 atom-5' d='M 192.8,19.2 L 257.2,19.2' style='fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:2.0px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1' />\n<path class='bond-5 atom-5 atom-0' d='M 264.0,7.5 L 302.9,75.0' style='fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:2.0px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1' />\n<path d='M 301.0,78.4 L 302.9,75.0 L 301.0,71.6' style='fill:none;stroke:#000000;stroke-width:2.0px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1;' />\n<path d='M 265.9,139.1 L 264.0,142.5 L 260.1,142.5' style='fill:none;stroke:#000000;stroke-width:2.0px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1;' />\n<path d='M 189.9,142.5 L 186.0,142.5 L 184.1,139.1' style='fill:none;stroke:#000000;stroke-width:2.0px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1;' />\n<path d='M 149.0,78.4 L 147.1,75.0 L 149.0,71.6' style='fill:none;stroke:#000000;stroke-width:2.0px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1;' />\n<path d='M 184.1,10.9 L 186.0,7.5 L 189.9,7.5' style='fill:none;stroke:#000000;stroke-width:2.0px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1;' />\n<path d='M 260.1,7.5 L 264.0,7.5 L 265.9,10.9' style='fill:none;stroke:#000000;stroke-width:2.0px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1;' />\n</svg>\n",
            "image/png": "iVBORw0KGgoAAAANSUhEUgAAAcIAAACWCAIAAADCEh9HAAAABmJLR0QA/wD/AP+gvaeTAAAXJElEQVR4nO3da1RU9f4G8C84KMcABRU4XlAMAR0CEswLlMcUSwWPArpaIljmBdEUEE0uSiroOZjmKZPsrHVM1GUq6jIxSytTvB9CEkMZQAUvXIIh7g4zzPxf7P7TqaWbSWD/ZvZ+Pq+/L57F0offZs98f2Y6nY4AAOBZmbMOAABg2lCjAAAdghoFAOgQ1CgAQIegRgEAOkTGOgCYjJycnMTERK1WS0QymWz06NGsE3Wyq1evajQaIjI3N09NTfX19WWdCEyDGT7wBIY4c+bMa6+9Jp1/LWZmZqdOnXrttddYBwETgBoFgzg4OFRVVVlbW3t6ehKRhYXF+PHjWYfqZOfOnVOr1UR048aNhoYGe3v7yspK1qHABOChHtqXlpZWVVVlZmZ24cIFrkbFraCgwMPDo6qqauPGjWvXrmUdB4wdTqPQDo1GM2jQoIqKiqCgoC+++IJ1HIEEBQVlZWU5ODjcv3/fwsKCdRwwanhTD+346KOPKioqBg8evH//ftZZhHPgwIGhQ4dWVlZ+/PHHrLOAscNpFPgolcphw4YplcqsrKxp06axjiOorKysoKCg3r17KxSKfv36sY4DxgunUeCTkJCgVConTZoktQ4losDAwClTpvzyyy/r1q1jnQWMGk6j8FQ//fSTt7c3EeXl5cnlctZxGLh9+7anp6dWq/3hhx+8vLxYxwEjhdMoPFVMTIxGo1m2bJk0O5SI3N3do6Ki2traYmJiWGcB44XTKDxZZmbmrFmz7OzsFApFnz59WMdhpra21tXVtbq6OjMzMyQkhHUcMEY4jcITqFSq+Ph4IkpNTZVyhxKRra3thg0biGjVqlWPHz9mHQeMEU6j8AQpKSlr166Vy+V5eXkyGd93NG7cuDFv3jzBgnWFPXv28H+noK2tzcfH58cff0xJSUlMTBQsGJgK1Cj80cOHD93d3RsbG0+fPh0QEMA/fPny5XHjxgkTrItcunRp7Nix/DNnz5599dVXe/bsefv27UGDBgkTDEwFahT+KCIiYu/evSEhIZmZme0ONzc3FxYWCpCq67i5ufXs2bPdsZCQkKNHj0ZEROzZs0eAVGBCUKPwO1euXBk3blz37t1v3rzp4uLCOo4RuXv37ogRI1Qq1fnz5/39/VnHASOCV0zwG51OFx0drdPp4uLi0KF/4OzsHBsby/2IuKWrABycRuE3GRkZ8+bNc3R0LCwstLGxYR3H6DQ2Nrq5uT169CgjIyM8PJx1HDAWOI3CrxobG7kPOaWlpaFDn8jKymrz5s1EtHr16vr6etZxwFigRuFXmzdvfvTokY+PT1hYGOssxis8PHz06NEVFRVpaWmss4CxwEM9EOH9yZ+Bt3DwBziNAhFRXFzc48ePw8PD0aHtGjNmzNy5c1Uq1Zo1a1hnAaOA0yj8+tlyKyurwsLC/v37s45jAioqKtzc3Orr6w35hgKIHk6jUqdfXxQfH48ONZCjo+Pq1avp/5dgsY4DjOE0KnXp6elRUVHOzs4FBQWWlpas45gMlUrl4eFRXFycnp4eGRnJOg6whBqVNGyB6wjsEgQOHuolbf369dXV1RMmTECHPoPQ0NCAgAClUpmSksI6C7CE06h04YaMjsM9K0A4jUpZbGysWq1euHAhOvSZyeXyt99+W6PRREdHs84CzOA0KlG4PbizSPkOauDgNCpFra2tcXFxRJScnIwO7SA7O7ukpCQiio6OVqlUrOMAA6hRKfrwww8LCwvd3d2XLl3KOosYvPPOO3K5vLi4eMeOHayzAAN4qJecqqoqV1fXurq6L7/8csqUKazjiMQ333wTEBBgbW2tUCgcHR1ZxwFB4TQqOUlJSXV1ddOmTUOHdqJJkyZNnTq1oaEhOTmZdRYQGk6j0pKXl+fr62tubp6fn+/m5sY6jqgUFxd7eHio1eqrV6/6+vqyjgPCwWlUWqKjo9va2pYvX44O7XQuLi5Lly7VarXcRSys44BwcBqVkIMHD77xxhv29vaFhYW9e/dmHUeE6uvr3dzcKioqDh48OHv2bNZxQCA4jUpFS0sLtx8zJSUFHdpFbGxs1q9fT0RxcXHNzc2s44BAUKNSsWXLlnv37nl7e8+fP591FjFbsGCBj4/P/fv3t27dyjoLCAQP9ZLw8OFDNze3pqam77//fvz48azjiNzFixdffvllS0vL27dvOzk5sY4DXQ6nUUlYvXp1U1PT7Nmz0aEC8PPzCw0NbWlpSUhIYJ0FhIDTqPhdvnzZz8/P0tKyoKBgyJAhrONIwv37993d3VtaWnBFoBTgNCpy+s/fxMXFoUMFM2jQoNjYWJ1Ot2LFCq1WyzoOdC2cRkVu9+7d8+fPHzBgQGFh4XPPPcc6joQ0NzcPHz68rKxs9+7db775Jus40IVQo2LW0NDg5uZWXl6+b9++sLAw1nEkZ9++feHh4Q4ODgqFwsbGhnUc6Cp4qBez1NTU8vLysWPHzpkzh3UWKQoLC/P396+srNy8eTPrLNCFcBoVrTt37owYMUKtVl++fPmll15iHUeicnNzR40aJZPJbt68OWzYMNZxoEvgNCpaK1euVKlUERER6FCGRo4cGR4e3tra+u6777LOAl0Fp1Fx+u677yZOnGhlZVVYWNi/f3/WcSStsrLS1dW1vr7+66+/njx5Mus40PlwGhWhtrY27oa1xMREdChzDg4O3DaDmJgYjUbDOg50PtSoCKWnp+fn5w8dOhTXVRqJlStXDhs2rKCgYNeuXayzQOfDQ73Y1NbWurq6VldXHzt2bMaMGazjwK+OHTsWHBxsa2tbVFTUp08f1nGgM+E0KjbJycnV1dWvvvoqOtSozJw5c/LkybW1tRs2bGCdBToZTqOicuvWLS8vL61Wm5ub6+npyToO/E5BQYGXlxcR5ebmvvDCC6zjQKfBaVRUYmJi1Gp1ZGQkOtQIjRgxYtGiRRqNJiYmhnUW6Ew4jYrH8ePHZ8yYYWtrq1Ao+vbtyzoOPIFSqXR1da2pqTl+/Pj06dNZx4HOgdOoSLS2tq5evZqI1q9fjw41WnZ2duvWrSOi2NhYlUrFOg50DtSoSGzfvl2hUAwfPjwyMpJ1FuATFRXl4eFRUlLy4Ycfss4CnQMP9WJQVVXl6upaV1d36tSp119/nXUcaMe33347adIka2vrwsLCv/71r6zjQEfhNCoG8fHxdXV106dPR4eahIkTJwYFBTU0NKxdu5Z1FugEOI2avOvXr/v6+spksvz8fFdXV9ZxwCAlJSVyuVytVl+5cmXUqFGs40CH4DRq8rhrKqKjo9GhJuT5559fvny5VqtdsWIFjjKmDqdR03bgwIE5c+bY29srFIpevXqxjgN/gv5uggMHDrzxxhus48Czw2nUhLW0tMTHxxPRpk2b0KEmx9ramvti6KpVq5qamljHgWeHGjVh//znP0tLS1988cW33nqLdRZ4FvPnzx81atSDBw+2bNnCOgs8OzzUm6pr1675+flpNJpz58698sorrOPAM7p06ZK/v7+5ufmFCxfGjBnDOg48C9SoqRo8eHBZWdmQIUPu3r3LOgt0iLOz871795ycnEpLS1lngWeBh3qTtGvXrrKyMiLav38/6yzQUYcOHSKisrKyjz76iHUWeBY4jZok7vzy8ssvnz9/nn+yvr4+Pz/fz89PmGDwBxcvXnzhhRfavaT+lVdeyc7OxrOFiZKxDgBd6MGDB76+vmq1WqFQYOO68GpqaqZPn25hYZGTkzNw4EDWcaCr4KHeJCUkJBBRdnb2xYsXecYGDhzo7e2tVCqTk5OFiga/WbdunVKp9PDw4O/Qa9euZWdnE1FcXJxQ0aAz4aHeVBn4ikm/D//69evYuC4kbte9Tqdr9yc/ZMiQ0tLSwYMH37t3T6h00JlwGjVVhw8flslk9+7d4//z6PDhwxcvXqy/chkEw12nHBUVxd+hly5dKisr69at2+effy5YNuhkOjBZ3KP6iy++qNFoeMaUSiW3yPnYsWOCZZO4o0ePEpGtrW11dTXPWFtbm6+vLxG99957gmWDTocaNWHNzc2DBw8mon//+9/8k9wnaYYOHdrS0iJMNilTqVTDhg0joh07dvBPfvrpp0Q0cODAxsZGYbJBV0CNmrYDBw4Qkb29/S+//MIzptFouEfLf/zjH4Jlk6zNmzcT0YgRI9RqNc9YfX29o6MjEX3++eeCZYOugBo1edw3QVetWsU/9u233xKRtbX1o0ePhAkmTRUVFdynRL/++mv+Se69/Lhx47RarTDZoIugRk1ebm6uubl59+7dCwsL+Sf//ve/E9Fbb70lTDBpevPNN4loxowZ/GPFxcU9evQwNze/du2aMMGg66BGxeDtt98moqCgIP6xkpIS7r/u1atXhQkmNT/88AP3K02hUPBPBgYGEtGCBQuECQZdCjUqBpWVldy+0VOnTvFPvvvuu0Q0duxYPEh2Oq1W6+/vT0Rr1qzhnzxz5gz+wCImqFGRSEtLI6Lhw4e3trbyjNXX13NXUe7fv1+wbBKxb98+InJwcKirq+MZU6vVHh4eRLRlyxbBskGXQo2KhEql4u5i+te//sU/+Z///IeIBgwYgA/ZdKLm5mYnJyci2r17N//k9u3biej5559//PixINGgy6FGxeP48ePcR75//vlnnrG2traXXnqJiNatWydYNtHjrkoeOXJkW1sbz1hNTQ23I+aLL74QLBt0NdSoqHD31EdFRfGPXbp0yczM7C9/+cvdu3cFySVyZWVlPXv2NDMzO3/+PP/kkiVLiGjixInCBANhoEZFpaCgwMLColu3bj/++CP/5Jw5c4ho9uzZwgQTt1mzZhFRWFgY/9jNmzdlMplMJsvPzxcmGAgDNSo2y5cvJ6IJEybwjz148OC5554jou+//16YYGJ14cIF7mhfWlrKPxkQEEBE3MX0ICaoUbHRLyI5evQo/+T69euJyNvbm3+zCfDQ7xbZsGED/+SRI0eIyM7Ojn9ZCZgi1KgI7dixw5BFJM3NzUOGDCGiTz/9VLBsIrNr1y4iGjRoUFNTE8+YflnJxx9/LFg2EAxqVIQ0Go2npycRbdq0iX+S23Fpb29fW1srTDYxqaur43aLHDx4kH8yNTXVkGUlYKJQo+LELSKxsrJ6+PAh/+T48eOJaOXKlcIEE5PY2Fgi8vPz4/9KmOHLSsBEoUZFa+bMmUQ0b948/rHr169369bNwsLi9u3bguQSiaKiIm5BwX//+1/+yYiICCIKDg4WJhgIDzUqWiUlJZaWlmZmZu0uIlmwYAERBQYGChNMHKZNm0ZECxcu5B/LyckxcFkJmC7UqJitWbOGiMaMGcP/1KnfbPLll18Kls2k6XeLlJeX84zpl5XEx8cLlg2EhxoVs4aGhv79+xPR3r17+Se3bNlCRO7u7vybTUCn06nVarlcTkTvv/8+/2RGRoYhy0rA1KFGRW737t3cIpKGhgaeMZVK5ebmRkTbt28XLJuJ+uCDD4jIxcWFf7dIU1MTt6zks88+EywbMIEaFTmtVsstIklKSuKfPHHihCGbTSROv1skKyuLfzIxMZGIfHx8+JeVgAigRsXv8uXLZmZmlpaW7S4i4TabLFmyRJBcJikyMpKIJk2axD+mX1aSnZ0tTDBgCDUqCWFhYUQ0a9Ys/jH9ZpO8vDxhgpkWw3eLhIaGEtHcuXOFCQZsoUYlQb+I5OzZs/yTK1asMGSziTRxu0Wio6P5x7Kzs83MzHr27NnushIQB9SoVGzYsIGIvLy8+BeR6DebHDlyRLBsJiEzM9OQ3SJtbW0+Pj5EtHHjRsGyAVuoUaloaWnhFpHs2rWLf3Lnzp1E5OzszL/ZRFIeP37s4uJCROnp6fyTn3zyiSHLSkBMUKMScvDgQSLq168f/yIS/WaT1NRUwbIZuZSUFCKSy+X8u0X0y0oOHTokWDZgDjUqLdwiktjYWP6x7777zsDNJlJQXl7O7RY5ffo0/2RMTIwhy0pAZFCj0mL4IpLg4GAiioiIECaYMQsPDyeikJAQ/rGioqLu3bubm5vn5OQIEwyMBGpUchYtWkREU6dO5R+7c+cOt9nkypUrwgQzTtxukR49ehQVFfFPTpkyhYgWL14sTDAwHqhRyamqqurdu7chi0ji4+MN2WwiYlqt1s/Pj4gSEhL4J0+fPk1ENjY2/MtKQJRQo1L0/vvvc98KV6lUPGP6zSYZGRmCZTMqe/bsMWS3SGtrK7eRYOvWrYJlA+OBGpUi/X/7bdu28U9+9tlnROTo6CjBHUX63yJ79uzhn9y6dashy0pArFCjEpWVlWXIQ6hWqx09ejQRJSYmCpbNSCQkJBiyW0T/R5KTJ08Klg2MCmpUugx8JcJtNjHkHYuY6N+wtbtbhHtlFxAQIEwwMEKoUem6deuWhYWFIR/QmTt3LhGFhoYKE8wYhISEEFF4eDj/WF5eXrdu3WQy2c2bN4UJBkYINSpp0dHRROTv78//Lv7BgwdWVlaGfP5cHLhvHxiyW+Rvf/sbEcXExAgTDIwTalTSamtr+/XrR0SHDx/mn9y4caMh34YUAY1G4+XlRUQpKSn8k4cOHeKWldTU1AiTDYwTalTq0tPTDVml0dLS4uzsTESffPKJYNmY4H4gTk5O7f5AuFUvov+BQLtQo1Jn+OHr8OHDoj986Y/nmZmZ/JPSOZ5Du1Cj8CfWDIv+T4F/9o/F7a7BBilAjYJOZ/CLaf0tGqJ8Mc19dMGQO1Qk+NEF4IEaBZ1OpystLTXwCrbFixcbcqebKeI+SBsZGck/pr8i8M6dO8IEAyOHGoVfGXghcE1NjZ2dnfi+tMN9rat3795VVVU8Y4ZfWA3SgRqFXzU1NTk5ORnyFfJt27aJ7Cvk+iUDH3zwAf8kt2RgwIABDQ0NwmQD44cahd9kZGQYstBIrVbL5XIxLTTiVl65u7u3trbyjOmXlezdu1ewbGD8UKPwG2mu16ysrMQCVugI1Cj8DrfsvXv37gqFgn9y6tSpRLRo0SJhgnWdhQsXEtG0adP4x0pKSnAdADwRahT+KCIigoiCg4P5x4qKinr06GHqVw8ZfjnVzJkziWjevHmC5AJTYk4Av5eWlmZjY3P06NEzZ87wjLm4uERFRfXr16+yslKwbJ3u0aNHffv2XbZsGfeK6WnOnj177NgxKyurTZs2CZYNTIWZTqdjnQGMTmpqalJSklwuz8vLk8lkTxtraGjQarW9evUSMlunq6urMzc3t7a2ftpAW1vbyJEjb9y4kZqayu1yBvhfqFF4ApVK5eHhUVxcvHPnziVLlrCOw9jOnTuXLl3q7OxcUFBgaWnJOg4YHdQoPNmRI0dCQ0Pt7OwUCkWfPn1Yx2GmtrbW1dW1urr6yJEjwcHBrOOAMcLfRuHJQkJCAgIClEolt8pIst57773q6uoJEyagQ+FpcBqFp/rpp5+8vb2J6Pr16x4eHqzjMHDr1i0vLy+tVpubm+vp6ck6DhgpnEbhqeRy+YIFCzQaTUxMDOssbMTGxqrV6kWLFqFDgQdOo8BHqVS6urrW1NScOHEiMDCQdRxBnThxYvr06ba2tgqFom/fvqzjgPHCaRT42NnZJSUlEdHSpUsbGhpYxxFOY2PjihUriCg5ORkdCvxwGoV2aDQaJyen8vLywMDAEydOsI4jkKCgoKysLAcHh/v371tYWLCOA0YNp1Foh0wm467WOHny5I0bN1jHEUJBQcHJkyeJaNmyZehQaBdOo2AQBweHqqqqXr16jRo1iohkMtno0aNZh+pkV69e1Wg0RHTt2rX6+np7e3uT/p4rCAY1CgY5ffr066+/Lp1/LWZmZl999dXkyZNZBwETgBoFQ+Xk5MTGxmq1WiKysLAYP34860Sd7Ny5c2q1mojMzc23bdvm6+vLOhGYBtQoAECH4BUTAECHoEYBADoENQoA0CGoUQCADkGNAgB0yP8BmlH/cQkHLUIAAACDelRYdHJka2l0UEtMIHJka2l0IDIwMjQuMDMuNgAAeJx7v2/tPQYgEABiRgYIYIPiBkY2hwwgzcxMGoObgTGDiZEpgYk5g4mZJYGFNYOJlSHBiRFsLCsLMxOjeBCSdSC7DtivXqWlAuE62D90W7Yfyt6PYB/YX1pSp4okbo+kHswWAwCbYR1TefQwpgAAAMh6VFh0TU9MIHJka2l0IDIwMjQuMDMuNgAAeJyNkdsKwjAMhu/7FP8LWNLDpr3c1iEi60Cn7+C974+JUruJjCUNJOEjpyqIXOL58cRXbFQKoJUXQsDdEZEaIA7a/nhK6KamzZluvKXpipqVRJdkM41Dzhh0MLoiEZCmXydzljnS+zeIndE2BHKHP6BjcLeJ9EJu6V0tSq5UrOdDrnB9iosjfM7SjimWs3g2W7b3bK7s6MXKIqJVGZcD1GUoz2bmveedJM5fx756AWxqZD4REQGpAAAASnpUWHRTTUlMRVMgcmRraXQgMjAyNC4wMy42AAB4nEs2TAYBQ4UaDV0DPXNTHV1DPSNLSwMTHWsgy1THAEiDxeHCqDyoGlStmjUAVb8Sgogw7QgAAAAASUVORK5CYII=\n"
          },
          "metadata": {},
          "execution_count": 5
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "#### **3.2.1 - O C√≥digo SMILES para Modelagem Molecular**\n",
        "\n",
        "O **Simplified Molecular Input Line Entry System (SMILES)** √© uma nota√ß√£o que permite representar estruturas qu√≠micas de forma textual, facilitando o armazenamento e a manipula√ß√£o de informa√ß√µes moleculares em programas de computador. Desenvolvido por Arthur Weininger e David Weininger nos anos 1980, o SMILES usa caracteres simples para descrever √°tomos, liga√ß√µes, e a estrutura geral de uma mol√©cula.\n",
        "\n",
        "##### Caracter√≠sticas Principais:\n",
        "\n",
        "- **√Åtomos**: Representados por seus s√≠mbolos qu√≠micos (ex: C, N, O).\n",
        "- **Liga√ß√µes**: Liga√ß√µes simples s√£o impl√≠citas, enquanto duplas (=), triplas (#), e arom√°ticas (:) s√£o explicitamente indicadas.\n",
        "- **Ramifica√ß√µes**: Indicadas por par√™nteses.\n",
        "- **Ciclos**: Numerados ap√≥s o √°tomo onde o ciclo come√ßa e fecha.\n",
        "\n",
        "##### Exemplo:\n",
        "\n",
        "- **Etileno**: `C=C`\n",
        "- **Benzeno**: `c1ccccc1`\n",
        "\n",
        "O SMILES √© amplamente utilizado em software de modelagem molecular e quimioinform√°tica para tarefas como a busca de mol√©culas, a predi√ß√£o de propriedades qu√≠micas, e a visualiza√ß√£o de estruturas. A nota√ß√£o SMILES pode ser convertida de volta para estruturas qu√≠micas completas por diversos programas de qu√≠mica computacional, tornando-se uma ferramenta vers√°til e poderosa na pesquisa qu√≠mica.\n",
        "\n",
        "##### **Aplica√ß√µes:**\n",
        "\n",
        "- **Armazenamento de dados qu√≠micos**\n",
        "- **Busca e recupera√ß√£o de informa√ß√µes moleculares**\n",
        "- **Modelagem e simula√ß√£o de propriedades moleculares**\n",
        "\n",
        "A simplicidade e efici√™ncia do SMILES continuam a fazer dele um padr√£o amplamente adotado na qu√≠mica computacional, quimioinform√°tica e na bioinform√°tica.\n",
        "\n",
        "##### **Materiais interessantes**\n",
        "\n",
        "- <https://en.wikipedia.org/wiki/Simplified_molecular-input_line-entry_system>\n",
        "- <https://www.cheminfo.org/Chemistry/Cheminformatics/FormatConverter/index.html>\n",
        "- <https://pubchem.ncbi.nlm.nih.gov/>"
      ],
      "metadata": {
        "id": "5PnPYoTdbej1"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "# Adiconando hidrog√™nios a mol√©cula mol\n",
        "mol_H = Chem.AddHs(mol)\n",
        "# Contando o n√∫mero de √°tomos do objeto \"mol_H\"\n",
        "print(f'O n√∫mero de √°tomos da mol√©cula de benzeno com os hidrog√™nios √© {mol_H.GetNumAtoms()}')"
      ],
      "metadata": {
        "id": "x7q_R_adu0in",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "915771a4-f1ad-452d-aa3f-a311a4d00bf3"
      },
      "execution_count": 6,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "O n√∫mero de √°tomos da mol√©cula de benzeno com os hidrog√™nios √© 12\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [],
      "metadata": {
        "id": "ejWJoiCDu09L"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "def show_prot(filename):\n",
        "  p = p3d.view(width=400, height=300)\n",
        "  p.addModel(open(filename, 'r').read(),'pdb')\n",
        "  p.setStyle({'cartoon':{'color':'spectrum'},'stick':{'radius':0.15}})\n",
        "  #p.setStyle({'cartoon': {'color':'spectrum'}})\n",
        "  p.center()\n",
        "  p.zoomTo()\n",
        "  return p"
      ],
      "metadata": {
        "id": "5eMErYL-yqUj"
      },
      "execution_count": 30,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "# baixe um arquivo pdb do site do pdb\n",
        "!wget https://files.rcsb.org/download/1UBQ.pdb"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "VJRt_2Dh0Ynk",
        "outputId": "ff9e62d6-f04f-4d73-8531-e0e130eea95f"
      },
      "execution_count": 32,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "--2024-12-10 14:45:43--  https://files.rcsb.org/download/1UBQ.pdb\n",
            "Resolving files.rcsb.org (files.rcsb.org)... 132.249.213.147\n",
            "Connecting to files.rcsb.org (files.rcsb.org)|132.249.213.147|:443... connected.\n",
            "HTTP request sent, awaiting response... 200 OK\n",
            "Length: unspecified [application/octet-stream]\n",
            "Saving to: ‚Äò1UBQ.pdb‚Äô\n",
            "\n",
            "1UBQ.pdb                [ <=>                ]  76.73K  --.-KB/s    in 0.1s    \n",
            "\n",
            "2024-12-10 14:45:43 (769 KB/s) - ‚Äò1UBQ.pdb‚Äô saved [78570]\n",
            "\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "# show_prot('SEU_ARQUIVO_PDB.pdb')  # copie um arquivo pdb para seu diret√≥rio e teste essa fun√ß√£o\n",
        "show_prot('1UBQ.pdb')"
      ],
      "metadata": {
        "id": "wkCRY30YleiQ",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 334
        },
        "outputId": "803262f2-8de4-4a83-a06c-a341bf259d6d"
      },
      "execution_count": 33,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_17338419469247565\"  style=\"position: relative; width: 400px; height: 300px;\">\n        <p id=\"3dmolwarning_17338419469247565\" style=\"background-color:#ffcccc;color:black\">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    //this is to ignore the existence of requirejs amd\n    var savedexports, savedmodule;\n    if (typeof exports !== 'undefined') savedexports = exports;\n    else exports = {}\n    if (typeof module !== 'undefined') savedmodule = module;\n    else module = {}\n\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n        exports = savedexports;\n        module = savedmodule;\n        resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.4.2/3Dmol-min.js');\n}\n\nvar viewer_17338419469247565 = null;\nvar warn = document.getElementById(\"3dmolwarning_17338419469247565\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_17338419469247565 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_17338419469247565\"),{backgroundColor:\"white\"});\nviewer_17338419469247565.zoomTo();\n\tviewer_17338419469247565.addModel(\"HEADER    CHROMOSOMAL PROTEIN                     02-JAN-87   1UBQ              \\nTITLE     STRUCTURE OF UBIQUITIN REFINED AT 1.8 ANGSTROMS RESOLUTION            \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: UBIQUITIN;                                                 \\nCOMPND   3 CHAIN: A;                                                            \\nCOMPND   4 ENGINEERED: YES                                                      \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 ORGANISM_SCIENTIFIC: HOMO SAPIENS;                                   \\nSOURCE   3 ORGANISM_COMMON: HUMAN;                                              \\nSOURCE   4 ORGANISM_TAXID: 9606                                                 \\nKEYWDS    CHROMOSOMAL PROTEIN                                                   \\nEXPDTA    X-RAY DIFFRACTION                                                     \\nAUTHOR    S.VIJAY-KUMAR,C.E.BUGG,W.J.COOK                                       \\nREVDAT   6   14-FEB-24 1UBQ    1       REMARK                                   \\nREVDAT   5   09-MAR-11 1UBQ    1       REMARK                                   \\nREVDAT   4   24-FEB-09 1UBQ    1       VERSN                                    \\nREVDAT   3   01-APR-03 1UBQ    1       JRNL                                     \\nREVDAT   2   16-JUL-87 1UBQ    1       JRNL   REMARK                            \\nREVDAT   1   16-APR-87 1UBQ    0                                                \\nJRNL        AUTH   S.VIJAY-KUMAR,C.E.BUGG,W.J.COOK                              \\nJRNL        TITL   STRUCTURE OF UBIQUITIN REFINED AT 1.8 A RESOLUTION.          \\nJRNL        REF    J.MOL.BIOL.                   V. 194   531 1987              \\nJRNL        REFN                   ISSN 0022-2836                               \\nJRNL        PMID   3041007                                                      \\nJRNL        DOI    10.1016/0022-2836(87)90679-6                                 \\nREMARK   1                                                                      \\nREMARK   1 REFERENCE 1                                                          \\nREMARK   1  AUTH   S.VIJAY-KUMAR,C.E.BUGG,K.D.WILKINSON,R.D.VIERSTRA,           \\nREMARK   1  AUTH 2 P.M.HATFIELD,W.J.COOK                                        \\nREMARK   1  TITL   COMPARISON OF THE THREE-DIMENSIONAL STRUCTURES OF HUMAN,     \\nREMARK   1  TITL 2 YEAST, AND OAT UBIQUITIN                                     \\nREMARK   1  REF    J.BIOL.CHEM.                  V. 262  6396 1987              \\nREMARK   1  REFN                   ISSN 0021-9258                               \\nREMARK   1 REFERENCE 2                                                          \\nREMARK   1  AUTH   S.VIJAY-KUMAR,C.E.BUGG,K.D.WILKINSON,W.J.COOK                \\nREMARK   1  TITL   THREE-DIMENSIONAL STRUCTURE OF UBIQUITIN AT 2.8 ANGSTROMS    \\nREMARK   1  TITL 2 RESOLUTION                                                   \\nREMARK   1  REF    PROC.NATL.ACAD.SCI.USA        V.  82  3582 1985              \\nREMARK   1  REFN                   ISSN 0027-8424                               \\nREMARK   1 REFERENCE 3                                                          \\nREMARK   1  AUTH   W.J.COOK,F.L.SUDDATH,C.E.BUGG,G.GOLDSTEIN                    \\nREMARK   1  TITL   CRYSTALLIZATION AND PRELIMINARY X-RAY INVESTIGATION OF       \\nREMARK   1  TITL 2 UBIQUITIN, A NON-HISTONE CHROMOSOMAL PROTEIN                 \\nREMARK   1  REF    J.MOL.BIOL.                   V. 130   353 1979              \\nREMARK   1  REFN                   ISSN 0022-2836                               \\nREMARK   1 REFERENCE 4                                                          \\nREMARK   1  AUTH   D.H.SCHLESINGER,G.GOLDSTEIN                                  \\nREMARK   1  TITL   MOLECULAR CONSERVATION OF 74 AMINO ACID SEQUENCE OF          \\nREMARK   1  TITL 2 UBIQUITIN BETWEEN CATTLE AND MAN                             \\nREMARK   1  REF    NATURE                        V. 255   423 1975              \\nREMARK   1  REFN                   ISSN 0028-0836                               \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION.    1.80 ANGSTROMS.                                       \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : PROLSQ                                               \\nREMARK   3   AUTHORS     : KONNERT,HENDRICKSON                                  \\nREMARK   3                                                                      \\nREMARK   3  DATA USED IN REFINEMENT.                                            \\nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 1.80                           \\nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : NULL                           \\nREMARK   3   DATA CUTOFF            (SIGMA(F)) : NULL                           \\nREMARK   3   COMPLETENESS FOR RANGE        (%) : NULL                           \\nREMARK   3   NUMBER OF REFLECTIONS             : NULL                           \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \\nREMARK   3   CROSS-VALIDATION METHOD          : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SELECTION  : NULL                            \\nREMARK   3   R VALUE     (WORKING + TEST SET) : 0.176                           \\nREMARK   3   R VALUE            (WORKING SET) : NULL                            \\nREMARK   3   FREE R VALUE                     : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : NULL                            \\nREMARK   3   FREE R VALUE TEST SET COUNT      : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  FIT/AGREEMENT OF MODEL WITH ALL DATA.                               \\nREMARK   3   R VALUE   (WORKING + TEST SET, NO CUTOFF) : NULL                   \\nREMARK   3   R VALUE          (WORKING SET, NO CUTOFF) : NULL                   \\nREMARK   3   FREE R VALUE                  (NO CUTOFF) : NULL                   \\nREMARK   3   FREE R VALUE TEST SET SIZE (%, NO CUTOFF) : NULL                   \\nREMARK   3   FREE R VALUE TEST SET COUNT   (NO CUTOFF) : NULL                   \\nREMARK   3   TOTAL NUMBER OF REFLECTIONS   (NO CUTOFF) : NULL                   \\nREMARK   3                                                                      \\nREMARK   3  NUMBER OF NON-HYDROGEN ATOMS USED IN REFINEMENT.                    \\nREMARK   3   PROTEIN ATOMS            : 602                                     \\nREMARK   3   NUCLEIC ACID ATOMS       : 0                                       \\nREMARK   3   HETEROGEN ATOMS          : 0                                       \\nREMARK   3   SOLVENT ATOMS            : 58                                      \\nREMARK   3                                                                      \\nREMARK   3  B VALUES.                                                           \\nREMARK   3   FROM WILSON PLOT           (A**2) : NULL                           \\nREMARK   3   MEAN B VALUE      (OVERALL, A**2) : NULL                           \\nREMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \\nREMARK   3    B11 (A**2) : NULL                                                 \\nREMARK   3    B22 (A**2) : NULL                                                 \\nREMARK   3    B33 (A**2) : NULL                                                 \\nREMARK   3    B12 (A**2) : NULL                                                 \\nREMARK   3    B13 (A**2) : NULL                                                 \\nREMARK   3    B23 (A**2) : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  ESTIMATED COORDINATE ERROR.                                         \\nREMARK   3   ESD FROM LUZZATI PLOT        (A) : NULL                            \\nREMARK   3   ESD FROM SIGMAA              (A) : NULL                            \\nREMARK   3   LOW RESOLUTION CUTOFF        (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  RMS DEVIATIONS FROM IDEAL VALUES.                                   \\nREMARK   3   DISTANCE RESTRAINTS.                    RMS    SIGMA               \\nREMARK   3    BOND LENGTH                     (A) : 0.016 ; NULL                \\nREMARK   3    ANGLE DISTANCE                  (A) : NULL  ; NULL                \\nREMARK   3    INTRAPLANAR 1-4 DISTANCE        (A) : NULL  ; NULL                \\nREMARK   3    H-BOND OR METAL COORDINATION    (A) : NULL  ; NULL                \\nREMARK   3                                                                      \\nREMARK   3   PLANE RESTRAINT                  (A) : NULL  ; NULL                \\nREMARK   3   CHIRAL-CENTER RESTRAINT       (A**3) : NULL  ; NULL                \\nREMARK   3                                                                      \\nREMARK   3   NON-BONDED CONTACT RESTRAINTS.                                     \\nREMARK   3    SINGLE TORSION                  (A) : NULL  ; NULL                \\nREMARK   3    MULTIPLE TORSION                (A) : NULL  ; NULL                \\nREMARK   3    H-BOND (X...Y)                  (A) : NULL  ; NULL                \\nREMARK   3    H-BOND (X-H...Y)                (A) : NULL  ; NULL                \\nREMARK   3                                                                      \\nREMARK   3   CONFORMATIONAL TORSION ANGLE RESTRAINTS.                           \\nREMARK   3    SPECIFIED                 (DEGREES) : NULL  ; NULL                \\nREMARK   3    PLANAR                    (DEGREES) : NULL  ; NULL                \\nREMARK   3    STAGGERED                 (DEGREES) : NULL  ; NULL                \\nREMARK   3    TRANSVERSE                (DEGREES) : NULL  ; NULL                \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL FACTOR RESTRAINTS.    RMS    SIGMA                \\nREMARK   3   MAIN-CHAIN BOND               (A**2) : NULL  ; NULL                \\nREMARK   3   MAIN-CHAIN ANGLE              (A**2) : NULL  ; NULL                \\nREMARK   3   SIDE-CHAIN BOND               (A**2) : NULL  ; NULL                \\nREMARK   3   SIDE-CHAIN ANGLE              (A**2) : NULL  ; NULL                \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS: NULL                                      \\nREMARK   4                                                                      \\nREMARK   4 1UBQ COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY BNL.                                \\nREMARK 100 THE DEPOSITION ID IS D_1000176905.                                   \\nREMARK 200                                                                      \\nREMARK 200 EXPERIMENTAL DETAILS                                                 \\nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \\nREMARK 200  DATE OF DATA COLLECTION        : NULL                               \\nREMARK 200  TEMPERATURE           (KELVIN) : NULL                               \\nREMARK 200  PH                             : NULL                               \\nREMARK 200  NUMBER OF CRYSTALS USED        : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  SYNCHROTRON              (Y/N) : NULL                               \\nREMARK 200  RADIATION SOURCE               : NULL                               \\nREMARK 200  BEAMLINE                       : NULL                               \\nREMARK 200  X-RAY GENERATOR MODEL          : NULL                               \\nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : NULL                               \\nREMARK 200  WAVELENGTH OR RANGE        (A) : NULL                               \\nREMARK 200  MONOCHROMATOR                  : NULL                               \\nREMARK 200  OPTICS                         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  DETECTOR TYPE                  : NULL                               \\nREMARK 200  DETECTOR MANUFACTURER          : NULL                               \\nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : NULL                               \\nREMARK 200  DATA SCALING SOFTWARE          : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : NULL                               \\nREMARK 200  RESOLUTION RANGE HIGH      (A) : NULL                               \\nREMARK 200  RESOLUTION RANGE LOW       (A) : NULL                               \\nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 OVERALL.                                                             \\nREMARK 200  COMPLETENESS FOR RANGE     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY                : NULL                               \\nREMARK 200  R MERGE                    (I) : NULL                               \\nREMARK 200  R SYM                      (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : NULL                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : NULL                     \\nREMARK 200  COMPLETENESS FOR SHELL     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY IN SHELL       : NULL                               \\nREMARK 200  R MERGE FOR SHELL          (I) : NULL                               \\nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR SHELL         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 DIFFRACTION PROTOCOL: NULL                                           \\nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: NULL                         \\nREMARK 200 SOFTWARE USED: NULL                                                  \\nREMARK 200 STARTING MODEL: NULL                                                 \\nREMARK 200                                                                      \\nREMARK 200 REMARK: NULL                                                         \\nREMARK 280                                                                      \\nREMARK 280 CRYSTAL                                                              \\nREMARK 280 SOLVENT CONTENT, VS   (%): 32.94                                     \\nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): 1.83                     \\nREMARK 280                                                                      \\nREMARK 280 CRYSTALLIZATION CONDITIONS: NULL                                     \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \\nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: P 21 21 21                       \\nREMARK 290                                                                      \\nREMARK 290      SYMOP   SYMMETRY                                                \\nREMARK 290     NNNMMM   OPERATOR                                                \\nREMARK 290       1555   X,Y,Z                                                   \\nREMARK 290       2555   -X+1/2,-Y,Z+1/2                                         \\nREMARK 290       3555   -X,Y+1/2,-Z+1/2                                         \\nREMARK 290       4555   X+1/2,-Y+1/2,-Z                                         \\nREMARK 290                                                                      \\nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \\nREMARK 290           MMM -> TRANSLATION VECTOR                                  \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \\nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \\nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \\nREMARK 290 RELATED MOLECULES.                                                   \\nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   2 -1.000000  0.000000  0.000000       25.42000            \\nREMARK 290   SMTRY2   2  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   2  0.000000  0.000000  1.000000       14.47500            \\nREMARK 290   SMTRY1   3 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   3  0.000000  1.000000  0.000000       21.38500            \\nREMARK 290   SMTRY3   3  0.000000  0.000000 -1.000000       14.47500            \\nREMARK 290   SMTRY1   4  1.000000  0.000000  0.000000       25.42000            \\nREMARK 290   SMTRY2   4  0.000000 -1.000000  0.000000       21.38500            \\nREMARK 290   SMTRY3   4  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290                                                                      \\nREMARK 290 REMARK: NULL                                                         \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1                                                       \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: MONOMERIC                         \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A                                     \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: CLOSE CONTACTS                                             \\nREMARK 500                                                                      \\nREMARK 500 THE FOLLOWING ATOMS THAT ARE RELATED BY CRYSTALLOGRAPHIC             \\nREMARK 500 SYMMETRY ARE IN CLOSE CONTACT.  AN ATOM LOCATED WITHIN 0.15          \\nREMARK 500 ANGSTROMS OF A SYMMETRY RELATED ATOM IS ASSUMED TO BE ON A           \\nREMARK 500 SPECIAL POSITION AND IS, THEREFORE, LISTED IN REMARK 375             \\nREMARK 500 INSTEAD OF REMARK 500.  ATOMS WITH NON-BLANK ALTERNATE               \\nREMARK 500 LOCATION INDICATORS ARE NOT INCLUDED IN THE CALCULATIONS.            \\nREMARK 500                                                                      \\nREMARK 500 DISTANCE CUTOFF:                                                     \\nREMARK 500 2.2 ANGSTROMS FOR CONTACTS NOT INVOLVING HYDROGEN ATOMS              \\nREMARK 500 1.6 ANGSTROMS FOR CONTACTS INVOLVING HYDROGEN ATOMS                  \\nREMARK 500                                                                      \\nREMARK 500  ATM1  RES C  SSEQI   ATM2  RES C  SSEQI  SSYMOP   DISTANCE          \\nREMARK 500   OE2  GLU A    16     NH1  ARG A    72     1554     2.02            \\nREMARK 500   NZ   LYS A    48     OXT  GLY A    76     4467     2.16            \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: COVALENT BOND ANGLES                                       \\nREMARK 500                                                                      \\nREMARK 500 THE STEREOCHEMICAL PARAMETERS OF THE FOLLOWING RESIDUES              \\nREMARK 500 HAVE VALUES WHICH DEVIATE FROM EXPECTED VALUES BY MORE               \\nREMARK 500 THAN 6*RMSD (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN               \\nREMARK 500 IDENTIFIER; SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                 \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT: (10X,I3,1X,A3,1X,A1,I4,A1,3(1X,A4,2X),12X,F5.1)              \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES PROTEIN: ENGH AND HUBER, 1999                        \\nREMARK 500 EXPECTED VALUES NUCLEIC ACID: CLOWNEY ET AL 1996                     \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI ATM1   ATM2   ATM3                                     \\nREMARK 500    LEU A  15   CA  -  CB  -  CG  ANGL. DEV. =  14.0 DEGREES          \\nREMARK 500    ARG A  54   CD  -  NE  -  CZ  ANGL. DEV. =  12.4 DEGREES          \\nREMARK 500    ARG A  54   NE  -  CZ  -  NH1 ANGL. DEV. =   5.5 DEGREES          \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nDBREF  1UBQ A    1    76  UNP    P62988   UBIQ_HUMAN       1     76             \\nSEQRES   1 A   76  MET GLN ILE PHE VAL LYS THR LEU THR GLY LYS THR ILE          \\nSEQRES   2 A   76  THR LEU GLU VAL GLU PRO SER ASP THR ILE GLU ASN VAL          \\nSEQRES   3 A   76  LYS ALA LYS ILE GLN ASP LYS GLU GLY ILE PRO PRO ASP          \\nSEQRES   4 A   76  GLN GLN ARG LEU ILE PHE ALA GLY LYS GLN LEU GLU ASP          \\nSEQRES   5 A   76  GLY ARG THR LEU SER ASP TYR ASN ILE GLN LYS GLU SER          \\nSEQRES   6 A   76  THR LEU HIS LEU VAL LEU ARG LEU ARG GLY GLY                  \\nFORMUL   2  HOH   *58(H2 O)                                                     \\nHELIX    1  H1 ILE A   23  GLU A   34  1                                  12    \\nHELIX    2  H2 LEU A   56  TYR A   59  5                                   4    \\nSHEET    1 BET 5 GLY A  10  VAL A  17  0                                        \\nSHEET    2 BET 5 MET A   1  THR A   7 -1                                        \\nSHEET    3 BET 5 GLU A  64  ARG A  72  1                                        \\nSHEET    4 BET 5 GLN A  40  PHE A  45 -1                                        \\nSHEET    5 BET 5 LYS A  48  LEU A  50 -1                                        \\nCRYST1   50.840   42.770   28.950  90.00  90.00  90.00 P 21 21 21    4          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      0.019670  0.000000  0.000000        0.00000                         \\nSCALE2      0.000000  0.023381  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  0.034542        0.00000                         \\nATOM      1  N   MET A   1      27.340  24.430   2.614  1.00  9.67           N  \\nATOM      2  CA  MET A   1      26.266  25.413   2.842  1.00 10.38           C  \\nATOM      3  C   MET A   1      26.913  26.639   3.531  1.00  9.62           C  \\nATOM      4  O   MET A   1      27.886  26.463   4.263  1.00  9.62           O  \\nATOM      5  CB  MET A   1      25.112  24.880   3.649  1.00 13.77           C  \\nATOM      6  CG  MET A   1      25.353  24.860   5.134  1.00 16.29           C  \\nATOM      7  SD  MET A   1      23.930  23.959   5.904  1.00 17.17           S  \\nATOM      8  CE  MET A   1      24.447  23.984   7.620  1.00 16.11           C  \\nATOM      9  N   GLN A   2      26.335  27.770   3.258  1.00  9.27           N  \\nATOM     10  CA  GLN A   2      26.850  29.021   3.898  1.00  9.07           C  \\nATOM     11  C   GLN A   2      26.100  29.253   5.202  1.00  8.72           C  \\nATOM     12  O   GLN A   2      24.865  29.024   5.330  1.00  8.22           O  \\nATOM     13  CB  GLN A   2      26.733  30.148   2.905  1.00 14.46           C  \\nATOM     14  CG  GLN A   2      26.882  31.546   3.409  1.00 17.01           C  \\nATOM     15  CD  GLN A   2      26.786  32.562   2.270  1.00 20.10           C  \\nATOM     16  OE1 GLN A   2      27.783  33.160   1.870  1.00 21.89           O  \\nATOM     17  NE2 GLN A   2      25.562  32.733   1.806  1.00 19.49           N  \\nATOM     18  N   ILE A   3      26.849  29.656   6.217  1.00  5.87           N  \\nATOM     19  CA  ILE A   3      26.235  30.058   7.497  1.00  5.07           C  \\nATOM     20  C   ILE A   3      26.882  31.428   7.862  1.00  4.01           C  \\nATOM     21  O   ILE A   3      27.906  31.711   7.264  1.00  4.61           O  \\nATOM     22  CB  ILE A   3      26.344  29.050   8.645  1.00  6.55           C  \\nATOM     23  CG1 ILE A   3      27.810  28.748   8.999  1.00  4.72           C  \\nATOM     24  CG2 ILE A   3      25.491  27.771   8.287  1.00  5.58           C  \\nATOM     25  CD1 ILE A   3      27.967  28.087  10.417  1.00 10.83           C  \\nATOM     26  N   PHE A   4      26.214  32.097   8.771  1.00  4.55           N  \\nATOM     27  CA  PHE A   4      26.772  33.436   9.197  1.00  4.68           C  \\nATOM     28  C   PHE A   4      27.151  33.362  10.650  1.00  5.30           C  \\nATOM     29  O   PHE A   4      26.350  32.778  11.395  1.00  5.58           O  \\nATOM     30  CB  PHE A   4      25.695  34.498   8.946  1.00  4.83           C  \\nATOM     31  CG  PHE A   4      25.288  34.609   7.499  1.00  7.97           C  \\nATOM     32  CD1 PHE A   4      24.147  33.966   7.038  1.00  6.69           C  \\nATOM     33  CD2 PHE A   4      26.136  35.346   6.640  1.00  8.34           C  \\nATOM     34  CE1 PHE A   4      23.812  34.031   5.677  1.00  9.10           C  \\nATOM     35  CE2 PHE A   4      25.810  35.392   5.267  1.00 10.61           C  \\nATOM     36  CZ  PHE A   4      24.620  34.778   4.853  1.00  8.90           C  \\nATOM     37  N   VAL A   5      28.260  33.943  11.096  1.00  4.44           N  \\nATOM     38  CA  VAL A   5      28.605  33.965  12.503  1.00  3.87           C  \\nATOM     39  C   VAL A   5      28.638  35.461  12.900  1.00  4.93           C  \\nATOM     40  O   VAL A   5      29.522  36.103  12.320  1.00  6.84           O  \\nATOM     41  CB  VAL A   5      29.963  33.317  12.814  1.00  2.99           C  \\nATOM     42  CG1 VAL A   5      30.211  33.394  14.304  1.00  5.28           C  \\nATOM     43  CG2 VAL A   5      29.957  31.838  12.352  1.00  9.13           C  \\nATOM     44  N   LYS A   6      27.751  35.867  13.740  1.00  6.04           N  \\nATOM     45  CA  LYS A   6      27.691  37.315  14.143  1.00  6.12           C  \\nATOM     46  C   LYS A   6      28.469  37.475  15.420  1.00  6.57           C  \\nATOM     47  O   LYS A   6      28.213  36.753  16.411  1.00  5.76           O  \\nATOM     48  CB  LYS A   6      26.219  37.684  14.307  1.00  7.45           C  \\nATOM     49  CG  LYS A   6      25.884  39.139  14.615  1.00 11.12           C  \\nATOM     50  CD  LYS A   6      24.348  39.296  14.642  1.00 14.54           C  \\nATOM     51  CE  LYS A   6      23.865  40.723  14.749  1.00 18.84           C  \\nATOM     52  NZ  LYS A   6      22.375  40.720  14.907  1.00 20.55           N  \\nATOM     53  N   THR A   7      29.426  38.430  15.446  1.00  7.41           N  \\nATOM     54  CA  THR A   7      30.225  38.643  16.662  1.00  7.48           C  \\nATOM     55  C   THR A   7      29.664  39.839  17.434  1.00  8.75           C  \\nATOM     56  O   THR A   7      28.850  40.565  16.859  1.00  8.58           O  \\nATOM     57  CB  THR A   7      31.744  38.879  16.299  1.00  9.61           C  \\nATOM     58  OG1 THR A   7      31.737  40.257  15.824  1.00 11.78           O  \\nATOM     59  CG2 THR A   7      32.260  37.969  15.171  1.00  9.17           C  \\nATOM     60  N   LEU A   8      30.132  40.069  18.642  1.00  9.84           N  \\nATOM     61  CA  LEU A   8      29.607  41.180  19.467  1.00 14.15           C  \\nATOM     62  C   LEU A   8      30.075  42.538  18.984  1.00 17.37           C  \\nATOM     63  O   LEU A   8      29.586  43.570  19.483  1.00 17.01           O  \\nATOM     64  CB  LEU A   8      29.919  40.890  20.938  1.00 16.63           C  \\nATOM     65  CG  LEU A   8      29.183  39.722  21.581  1.00 18.88           C  \\nATOM     66  CD1 LEU A   8      29.308  39.750  23.095  1.00 19.31           C  \\nATOM     67  CD2 LEU A   8      27.700  39.721  21.228  1.00 18.59           C  \\nATOM     68  N   THR A   9      30.991  42.571  17.998  1.00 18.33           N  \\nATOM     69  CA  THR A   9      31.422  43.940  17.553  1.00 19.24           C  \\nATOM     70  C   THR A   9      30.755  44.351  16.277  1.00 19.48           C  \\nATOM     71  O   THR A   9      31.207  45.268  15.566  1.00 23.14           O  \\nATOM     72  CB  THR A   9      32.979  43.918  17.445  1.00 18.97           C  \\nATOM     73  OG1 THR A   9      33.174  43.067  16.265  1.00 20.24           O  \\nATOM     74  CG2 THR A   9      33.657  43.319  18.672  1.00 19.70           C  \\nATOM     75  N   GLY A  10      29.721  43.673  15.885  1.00 19.43           N  \\nATOM     76  CA  GLY A  10      28.978  43.960  14.678  1.00 18.74           C  \\nATOM     77  C   GLY A  10      29.604  43.507  13.393  1.00 17.62           C  \\nATOM     78  O   GLY A  10      29.219  43.981  12.301  1.00 19.74           O  \\nATOM     79  N   LYS A  11      30.563  42.623  13.495  1.00 13.56           N  \\nATOM     80  CA  LYS A  11      31.191  42.012  12.331  1.00 11.91           C  \\nATOM     81  C   LYS A  11      30.459  40.666  12.130  1.00 10.18           C  \\nATOM     82  O   LYS A  11      30.253  39.991  13.133  1.00  9.10           O  \\nATOM     83  CB  LYS A  11      32.672  41.717  12.505  1.00 13.43           C  \\nATOM     84  CG  LYS A  11      33.280  41.086  11.227  1.00 16.69           C  \\nATOM     85  CD  LYS A  11      34.762  40.799  11.470  1.00 17.92           C  \\nATOM     86  CE  LYS A  11      35.614  40.847  10.240  1.00 20.81           C  \\nATOM     87  NZ  LYS A  11      35.100  40.073   9.101  1.00 21.93           N  \\nATOM     88  N   THR A  12      30.163  40.338  10.886  1.00  9.63           N  \\nATOM     89  CA  THR A  12      29.542  39.020  10.653  1.00  9.85           C  \\nATOM     90  C   THR A  12      30.494  38.261   9.729  1.00 11.66           C  \\nATOM     91  O   THR A  12      30.849  38.850   8.706  1.00 12.33           O  \\nATOM     92  CB  THR A  12      28.113  39.049  10.015  1.00 10.85           C  \\nATOM     93  OG1 THR A  12      27.280  39.722  10.996  1.00 10.91           O  \\nATOM     94  CG2 THR A  12      27.588  37.635   9.715  1.00  9.63           C  \\nATOM     95  N   ILE A  13      30.795  37.015  10.095  1.00 10.42           N  \\nATOM     96  CA  ILE A  13      31.720  36.289   9.176  1.00 11.84           C  \\nATOM     97  C   ILE A  13      30.955  35.211   8.459  1.00 10.55           C  \\nATOM     98  O   ILE A  13      30.025  34.618   9.040  1.00 11.92           O  \\nATOM     99  CB  ILE A  13      32.995  35.883   9.934  1.00 14.86           C  \\nATOM    100  CG1 ILE A  13      33.306  34.381   9.840  1.00 14.87           C  \\nATOM    101  CG2 ILE A  13      33.109  36.381  11.435  1.00 17.08           C  \\nATOM    102  CD1 ILE A  13      34.535  34.028  10.720  1.00 16.46           C  \\nATOM    103  N   THR A  14      31.244  34.986   7.197  1.00  9.39           N  \\nATOM    104  CA  THR A  14      30.505  33.884   6.512  1.00  9.63           C  \\nATOM    105  C   THR A  14      31.409  32.680   6.446  1.00 11.20           C  \\nATOM    106  O   THR A  14      32.619  32.812   6.125  1.00 11.63           O  \\nATOM    107  CB  THR A  14      30.091  34.393   5.078  1.00 10.38           C  \\nATOM    108  OG1 THR A  14      31.440  34.513   4.487  1.00 16.30           O  \\nATOM    109  CG2 THR A  14      29.420  35.756   5.119  1.00 11.66           C  \\nATOM    110  N   LEU A  15      30.884  31.485   6.666  1.00  8.29           N  \\nATOM    111  CA  LEU A  15      31.677  30.275   6.639  1.00  9.03           C  \\nATOM    112  C   LEU A  15      31.022  29.288   5.665  1.00  8.59           C  \\nATOM    113  O   LEU A  15      29.809  29.395   5.545  1.00  7.79           O  \\nATOM    114  CB  LEU A  15      31.562  29.686   8.045  1.00 11.08           C  \\nATOM    115  CG  LEU A  15      32.631  29.444   9.060  1.00 15.79           C  \\nATOM    116  CD1 LEU A  15      33.814  30.390   9.030  1.00 15.88           C  \\nATOM    117  CD2 LEU A  15      31.945  29.449  10.436  1.00 15.27           C  \\nATOM    118  N   GLU A  16      31.834  28.412   5.125  1.00 11.04           N  \\nATOM    119  CA  GLU A  16      31.220  27.341   4.275  1.00 11.50           C  \\nATOM    120  C   GLU A  16      31.440  26.079   5.080  1.00 10.13           C  \\nATOM    121  O   GLU A  16      32.576  25.802   5.461  1.00  9.83           O  \\nATOM    122  CB  GLU A  16      31.827  27.262   2.894  1.00 17.22           C  \\nATOM    123  CG  GLU A  16      31.363  28.410   1.962  1.00 23.33           C  \\nATOM    124  CD  GLU A  16      31.671  28.291   0.498  1.00 26.99           C  \\nATOM    125  OE1 GLU A  16      30.869  28.621  -0.366  1.00 28.86           O  \\nATOM    126  OE2 GLU A  16      32.835  27.861   0.278  1.00 28.90           O  \\nATOM    127  N   VAL A  17      30.310  25.458   5.384  1.00  8.99           N  \\nATOM    128  CA  VAL A  17      30.288  24.245   6.193  1.00  8.85           C  \\nATOM    129  C   VAL A  17      29.279  23.227   5.641  1.00  8.04           C  \\nATOM    130  O   VAL A  17      28.478  23.522   4.725  1.00  8.99           O  \\nATOM    131  CB  VAL A  17      29.903  24.590   7.665  1.00  9.78           C  \\nATOM    132  CG1 VAL A  17      30.862  25.496   8.389  1.00 12.05           C  \\nATOM    133  CG2 VAL A  17      28.476  25.135   7.705  1.00 10.54           C  \\nATOM    134  N   GLU A  18      29.380  22.057   6.232  1.00  7.29           N  \\nATOM    135  CA  GLU A  18      28.468  20.940   5.980  1.00  7.08           C  \\nATOM    136  C   GLU A  18      27.819  20.609   7.316  1.00  6.45           C  \\nATOM    137  O   GLU A  18      28.449  20.674   8.360  1.00  5.28           O  \\nATOM    138  CB  GLU A  18      29.213  19.697   5.506  1.00 10.28           C  \\nATOM    139  CG  GLU A  18      29.728  19.755   4.060  1.00 12.65           C  \\nATOM    140  CD  GLU A  18      28.754  20.061   2.978  1.00 14.15           C  \\nATOM    141  OE1 GLU A  18      27.546  19.992   2.985  1.00 14.33           O  \\nATOM    142  OE2 GLU A  18      29.336  20.423   1.904  1.00 18.17           O  \\nATOM    143  N   PRO A  19      26.559  20.220   7.288  1.00  7.24           N  \\nATOM    144  CA  PRO A  19      25.829  19.825   8.494  1.00  7.07           C  \\nATOM    145  C   PRO A  19      26.541  18.732   9.251  1.00  6.65           C  \\nATOM    146  O   PRO A  19      26.333  18.536  10.457  1.00  6.37           O  \\nATOM    147  CB  PRO A  19      24.469  19.332   7.952  1.00  7.61           C  \\nATOM    148  CG  PRO A  19      24.299  20.134   6.704  1.00  8.16           C  \\nATOM    149  CD  PRO A  19      25.714  20.108   6.073  1.00  7.49           C  \\nATOM    150  N   SER A  20      27.361  17.959   8.559  1.00  6.80           N  \\nATOM    151  CA  SER A  20      28.054  16.835   9.210  1.00  6.28           C  \\nATOM    152  C   SER A  20      29.258  17.318   9.984  1.00  8.45           C  \\nATOM    153  O   SER A  20      29.930  16.477  10.606  1.00  7.26           O  \\nATOM    154  CB  SER A  20      28.523  15.820   8.182  1.00  8.57           C  \\nATOM    155  OG  SER A  20      28.946  16.445   6.967  1.00 11.13           O  \\nATOM    156  N   ASP A  21      29.599  18.599   9.828  1.00  7.50           N  \\nATOM    157  CA  ASP A  21      30.796  19.083  10.566  1.00  7.70           C  \\nATOM    158  C   ASP A  21      30.491  19.162  12.040  1.00  7.08           C  \\nATOM    159  O   ASP A  21      29.367  19.523  12.441  1.00  8.11           O  \\nATOM    160  CB  ASP A  21      31.155  20.515  10.048  1.00 11.00           C  \\nATOM    161  CG  ASP A  21      31.923  20.436   8.755  1.00 15.32           C  \\nATOM    162  OD1 ASP A  21      32.493  19.374   8.456  1.00 18.03           O  \\nATOM    163  OD2 ASP A  21      31.838  21.402   7.968  1.00 14.36           O  \\nATOM    164  N   THR A  22      31.510  18.936  12.852  1.00  5.37           N  \\nATOM    165  CA  THR A  22      31.398  19.064  14.286  1.00  6.01           C  \\nATOM    166  C   THR A  22      31.593  20.553  14.655  1.00  8.01           C  \\nATOM    167  O   THR A  22      32.159  21.311  13.861  1.00  8.11           O  \\nATOM    168  CB  THR A  22      32.492  18.193  14.995  1.00  8.92           C  \\nATOM    169  OG1 THR A  22      33.778  18.739  14.516  1.00 10.22           O  \\nATOM    170  CG2 THR A  22      32.352  16.700  14.630  1.00  9.65           C  \\nATOM    171  N   ILE A  23      31.113  20.863  15.860  1.00  8.32           N  \\nATOM    172  CA  ILE A  23      31.288  22.201  16.417  1.00  9.92           C  \\nATOM    173  C   ILE A  23      32.776  22.519  16.577  1.00 10.01           C  \\nATOM    174  O   ILE A  23      33.233  23.659  16.384  1.00  8.71           O  \\nATOM    175  CB  ILE A  23      30.520  22.300  17.764  1.00 10.78           C  \\nATOM    176  CG1 ILE A  23      29.006  22.043  17.442  1.00 11.38           C  \\nATOM    177  CG2 ILE A  23      30.832  23.699  18.358  1.00 10.90           C  \\nATOM    178  CD1 ILE A  23      28.407  22.948  16.366  1.00 12.30           C  \\nATOM    179  N   GLU A  24      33.548  21.526  16.950  1.00  9.54           N  \\nATOM    180  CA  GLU A  24      35.031  21.722  17.069  1.00 11.81           C  \\nATOM    181  C   GLU A  24      35.615  22.190  15.759  1.00 11.14           C  \\nATOM    182  O   GLU A  24      36.532  23.046  15.724  1.00 10.62           O  \\nATOM    183  CB  GLU A  24      35.667  20.383  17.447  1.00 19.24           C  \\nATOM    184  CG  GLU A  24      37.128  20.293  17.872  1.00 27.76           C  \\nATOM    185  CD  GLU A  24      37.561  18.851  18.082  1.00 32.92           C  \\nATOM    186  OE1 GLU A  24      37.758  18.024  17.195  1.00 34.80           O  \\nATOM    187  OE2 GLU A  24      37.628  18.599  19.313  1.00 36.51           O  \\nATOM    188  N   ASN A  25      35.139  21.624  14.662  1.00  9.43           N  \\nATOM    189  CA  ASN A  25      35.590  21.945  13.302  1.00 10.96           C  \\nATOM    190  C   ASN A  25      35.238  23.382  12.920  1.00  9.68           C  \\nATOM    191  O   ASN A  25      36.066  24.109  12.333  1.00  9.33           O  \\nATOM    192  CB  ASN A  25      35.064  20.957  12.255  1.00 16.78           C  \\nATOM    193  CG  ASN A  25      35.541  21.418  10.871  1.00 22.31           C  \\nATOM    194  OD1 ASN A  25      36.772  21.623  10.676  1.00 25.66           O  \\nATOM    195  ND2 ASN A  25      34.628  21.595   9.920  1.00 24.70           N  \\nATOM    196  N   VAL A  26      34.007  23.745  13.250  1.00  6.52           N  \\nATOM    197  CA  VAL A  26      33.533  25.097  12.978  1.00  5.53           C  \\nATOM    198  C   VAL A  26      34.441  26.099  13.684  1.00  4.42           C  \\nATOM    199  O   VAL A  26      34.883  27.090  13.093  1.00  3.40           O  \\nATOM    200  CB  VAL A  26      32.060  25.257  13.364  1.00  3.86           C  \\nATOM    201  CG1 VAL A  26      31.684  26.749  13.342  1.00  7.25           C  \\nATOM    202  CG2 VAL A  26      31.152  24.421  12.477  1.00  8.12           C  \\nATOM    203  N   LYS A  27      34.734  25.822  14.949  1.00  2.64           N  \\nATOM    204  CA  LYS A  27      35.596  26.715  15.736  1.00  4.14           C  \\nATOM    205  C   LYS A  27      36.975  26.826  15.107  1.00  5.58           C  \\nATOM    206  O   LYS A  27      37.579  27.926  15.159  1.00  4.11           O  \\nATOM    207  CB  LYS A  27      35.715  26.203  17.172  1.00  3.97           C  \\nATOM    208  CG  LYS A  27      34.343  26.445  17.898  1.00  7.45           C  \\nATOM    209  CD  LYS A  27      34.509  26.077  19.360  1.00  9.02           C  \\nATOM    210  CE  LYS A  27      33.206  26.311  20.122  1.00 12.90           C  \\nATOM    211  NZ  LYS A  27      33.455  25.910  21.546  1.00 15.47           N  \\nATOM    212  N   ALA A  28      37.499  25.743  14.571  1.00  6.61           N  \\nATOM    213  CA  ALA A  28      38.794  25.761  13.880  1.00  7.74           C  \\nATOM    214  C   ALA A  28      38.728  26.591  12.611  1.00  9.17           C  \\nATOM    215  O   ALA A  28      39.704  27.346  12.277  1.00 11.45           O  \\nATOM    216  CB  ALA A  28      39.285  24.336  13.566  1.00  7.68           C  \\nATOM    217  N   LYS A  29      37.633  26.543  11.867  1.00  8.96           N  \\nATOM    218  CA  LYS A  29      37.471  27.391  10.668  1.00  7.90           C  \\nATOM    219  C   LYS A  29      37.441  28.882  11.052  1.00  6.92           C  \\nATOM    220  O   LYS A  29      38.020  29.772  10.382  1.00  6.87           O  \\nATOM    221  CB  LYS A  29      36.193  27.058   9.911  1.00 10.28           C  \\nATOM    222  CG  LYS A  29      36.153  25.620   9.409  1.00 14.94           C  \\nATOM    223  CD  LYS A  29      34.758  25.280   8.900  1.00 19.69           C  \\nATOM    224  CE  LYS A  29      34.793  24.264   7.767  1.00 22.63           C  \\nATOM    225  NZ  LYS A  29      34.914  24.944   6.441  1.00 24.98           N  \\nATOM    226  N   ILE A  30      36.811  29.170  12.192  1.00  4.57           N  \\nATOM    227  CA  ILE A  30      36.731  30.570  12.645  1.00  5.58           C  \\nATOM    228  C   ILE A  30      38.148  30.981  13.069  1.00  7.26           C  \\nATOM    229  O   ILE A  30      38.544  32.150  12.856  1.00  9.46           O  \\nATOM    230  CB  ILE A  30      35.708  30.776  13.806  1.00  5.36           C  \\nATOM    231  CG1 ILE A  30      34.228  30.630  13.319  1.00  2.94           C  \\nATOM    232  CG2 ILE A  30      35.874  32.138  14.512  1.00  2.78           C  \\nATOM    233  CD1 ILE A  30      33.284  30.504  14.552  1.00  2.00           C  \\nATOM    234  N   GLN A  31      38.883  30.110  13.713  1.00  7.06           N  \\nATOM    235  CA  GLN A  31      40.269  30.508  14.115  1.00  8.67           C  \\nATOM    236  C   GLN A  31      41.092  30.808  12.851  1.00 10.90           C  \\nATOM    237  O   GLN A  31      41.828  31.808  12.681  1.00  9.63           O  \\nATOM    238  CB  GLN A  31      40.996  29.399  14.865  1.00  9.12           C  \\nATOM    239  CG  GLN A  31      42.445  29.848  15.182  1.00 10.76           C  \\nATOM    240  CD  GLN A  31      43.090  28.828  16.095  1.00 13.78           C  \\nATOM    241  OE1 GLN A  31      42.770  27.655  15.906  1.00 14.48           O  \\nATOM    242  NE2 GLN A  31      43.898  29.252  17.050  1.00 14.76           N  \\nATOM    243  N   ASP A  32      41.001  29.878  11.931  1.00 10.93           N  \\nATOM    244  CA  ASP A  32      41.718  30.022  10.643  1.00 14.01           C  \\nATOM    245  C   ASP A  32      41.399  31.338   9.967  1.00 14.04           C  \\nATOM    246  O   ASP A  32      42.260  32.036   9.381  1.00 13.39           O  \\nATOM    247  CB  ASP A  32      41.398  28.780   9.810  1.00 18.01           C  \\nATOM    248  CG  ASP A  32      42.626  28.557   8.928  1.00 24.33           C  \\nATOM    249  OD1 ASP A  32      43.666  28.262   9.539  1.00 26.29           O  \\nATOM    250  OD2 ASP A  32      42.430  28.812   7.728  1.00 25.17           O  \\nATOM    251  N   LYS A  33      40.117  31.750   9.988  1.00 14.22           N  \\nATOM    252  CA  LYS A  33      39.808  32.994   9.233  1.00 14.00           C  \\nATOM    253  C   LYS A  33      39.837  34.271   9.995  1.00 12.37           C  \\nATOM    254  O   LYS A  33      40.164  35.323   9.345  1.00 12.17           O  \\nATOM    255  CB  LYS A  33      38.615  32.801   8.320  1.00 18.62           C  \\nATOM    256  CG  LYS A  33      37.220  32.822   8.827  1.00 24.00           C  \\nATOM    257  CD  LYS A  33      36.351  33.613   7.838  1.00 27.61           C  \\nATOM    258  CE  LYS A  33      36.322  32.944   6.477  1.00 27.64           C  \\nATOM    259  NZ  LYS A  33      35.768  33.945   5.489  1.00 30.06           N  \\nATOM    260  N   GLU A  34      39.655  34.335  11.285  1.00 10.11           N  \\nATOM    261  CA  GLU A  34      39.676  35.547  12.072  1.00 10.07           C  \\nATOM    262  C   GLU A  34      40.675  35.527  13.200  1.00  9.32           C  \\nATOM    263  O   GLU A  34      40.814  36.528  13.911  1.00 11.61           O  \\nATOM    264  CB  GLU A  34      38.290  35.814  12.698  1.00 14.77           C  \\nATOM    265  CG  GLU A  34      37.156  35.985  11.688  1.00 18.75           C  \\nATOM    266  CD  GLU A  34      37.192  37.361  11.033  1.00 22.28           C  \\nATOM    267  OE1 GLU A  34      37.519  38.360  11.645  1.00 21.95           O  \\nATOM    268  OE2 GLU A  34      36.861  37.320   9.822  1.00 25.19           O  \\nATOM    269  N   GLY A  35      41.317  34.393  13.432  1.00  7.22           N  \\nATOM    270  CA  GLY A  35      42.345  34.269  14.431  1.00  6.29           C  \\nATOM    271  C   GLY A  35      41.949  34.076  15.842  1.00  6.93           C  \\nATOM    272  O   GLY A  35      42.829  34.000  16.739  1.00  7.41           O  \\nATOM    273  N   ILE A  36      40.642  33.916  16.112  1.00  5.86           N  \\nATOM    274  CA  ILE A  36      40.226  33.716  17.509  1.00  6.07           C  \\nATOM    275  C   ILE A  36      40.449  32.278  17.945  1.00  6.36           C  \\nATOM    276  O   ILE A  36      39.936  31.336  17.315  1.00  6.18           O  \\nATOM    277  CB  ILE A  36      38.693  34.106  17.595  1.00  7.47           C  \\nATOM    278  CG1 ILE A  36      38.471  35.546  17.045  1.00  8.52           C  \\nATOM    279  CG2 ILE A  36      38.146  33.932  19.027  1.00  7.36           C  \\nATOM    280  CD1 ILE A  36      36.958  35.746  16.680  1.00  9.49           C  \\nATOM    281  N   PRO A  37      41.189  32.085  19.031  1.00  8.65           N  \\nATOM    282  CA  PRO A  37      41.461  30.751  19.594  1.00  9.18           C  \\nATOM    283  C   PRO A  37      40.168  30.026  19.918  1.00  9.85           C  \\nATOM    284  O   PRO A  37      39.264  30.662  20.521  1.00  8.51           O  \\nATOM    285  CB  PRO A  37      42.195  31.142  20.913  1.00 11.42           C  \\nATOM    286  CG  PRO A  37      42.904  32.414  20.553  1.00  9.27           C  \\nATOM    287  CD  PRO A  37      41.822  33.188  19.813  1.00  8.33           C  \\nATOM    288  N   PRO A  38      40.059  28.758  19.607  1.00  8.71           N  \\nATOM    289  CA  PRO A  38      38.817  28.020  19.889  1.00  9.08           C  \\nATOM    290  C   PRO A  38      38.421  28.048  21.341  1.00  9.28           C  \\nATOM    291  O   PRO A  38      37.213  28.036  21.704  1.00  6.50           O  \\nATOM    292  CB  PRO A  38      39.090  26.629  19.325  1.00 10.31           C  \\nATOM    293  CG  PRO A  38      40.082  26.904  18.198  1.00 10.81           C  \\nATOM    294  CD  PRO A  38      41.035  27.909  18.879  1.00 12.00           C  \\nATOM    295  N   ASP A  39      39.374  28.090  22.240  1.00 11.20           N  \\nATOM    296  CA  ASP A  39      39.063  28.063  23.695  1.00 14.96           C  \\nATOM    297  C   ASP A  39      38.365  29.335  24.159  1.00 13.99           C  \\nATOM    298  O   ASP A  39      37.684  29.390  25.221  1.00 13.75           O  \\nATOM    299  CB  ASP A  39      40.340  27.692  24.468  1.00 24.16           C  \\nATOM    300  CG  ASP A  39      40.559  28.585  25.675  1.00 31.06           C  \\nATOM    301  OD1 ASP A  39      40.716  29.809  25.456  1.00 35.55           O  \\nATOM    302  OD2 ASP A  39      40.549  28.090  26.840  1.00 34.22           O  \\nATOM    303  N   GLN A  40      38.419  30.373  23.341  1.00 11.60           N  \\nATOM    304  CA  GLN A  40      37.738  31.637  23.712  1.00 10.76           C  \\nATOM    305  C   GLN A  40      36.334  31.742  23.087  1.00  8.01           C  \\nATOM    306  O   GLN A  40      35.574  32.618  23.483  1.00  8.96           O  \\nATOM    307  CB  GLN A  40      38.528  32.854  23.182  1.00 11.14           C  \\nATOM    308  CG  GLN A  40      39.919  32.854  23.840  1.00 14.85           C  \\nATOM    309  CD  GLN A  40      40.760  34.036  23.394  1.00 16.11           C  \\nATOM    310  OE1 GLN A  40      41.975  34.008  23.624  1.00 20.52           O  \\nATOM    311  NE2 GLN A  40      40.140  35.007  22.775  1.00 18.16           N  \\nATOM    312  N   GLN A  41      36.000  30.860  22.172  1.00  6.52           N  \\nATOM    313  CA  GLN A  41      34.738  30.875  21.473  1.00  3.87           C  \\nATOM    314  C   GLN A  41      33.589  30.189  22.181  1.00  4.79           C  \\nATOM    315  O   GLN A  41      33.580  29.009  22.499  1.00  6.34           O  \\nATOM    316  CB  GLN A  41      34.876  30.237  20.066  1.00  4.20           C  \\nATOM    317  CG  GLN A  41      36.012  30.860  19.221  1.00  3.20           C  \\nATOM    318  CD  GLN A  41      36.083  30.194  17.875  1.00  4.89           C  \\nATOM    319  OE1 GLN A  41      35.048  29.702  17.393  1.00  5.21           O  \\nATOM    320  NE2 GLN A  41      37.228  30.126  17.233  1.00  7.13           N  \\nATOM    321  N   ARG A  42      32.478  30.917  22.269  1.00  5.73           N  \\nATOM    322  CA  ARG A  42      31.200  30.329  22.780  1.00  6.97           C  \\nATOM    323  C   ARG A  42      30.210  30.509  21.650  1.00  7.15           C  \\nATOM    324  O   ARG A  42      29.978  31.726  21.269  1.00  7.33           O  \\nATOM    325  CB  ARG A  42      30.847  30.931  24.118  1.00 13.23           C  \\nATOM    326  CG  ARG A  42      29.412  30.796  24.598  1.00 21.27           C  \\nATOM    327  CD  ARG A  42      29.271  31.314  26.016  1.00 26.14           C  \\nATOM    328  NE  ARG A  42      27.875  31.317  26.443  1.00 32.26           N  \\nATOM    329  CZ  ARG A  42      27.132  32.423  26.574  1.00 34.32           C  \\nATOM    330  NH1 ARG A  42      27.630  33.656  26.461  1.00 35.30           N  \\nATOM    331  NH2 ARG A  42      25.810  32.299  26.732  1.00 36.39           N  \\nATOM    332  N   LEU A  43      29.694  29.436  21.054  1.00  4.65           N  \\nATOM    333  CA  LEU A  43      28.762  29.573  19.906  1.00  3.51           C  \\nATOM    334  C   LEU A  43      27.331  29.317  20.364  1.00  5.56           C  \\nATOM    335  O   LEU A  43      27.101  28.346  21.097  1.00  4.19           O  \\nATOM    336  CB  LEU A  43      29.151  28.655  18.755  1.00  3.74           C  \\nATOM    337  CG  LEU A  43      30.416  28.912  17.980  1.00  6.32           C  \\nATOM    338  CD1 LEU A  43      30.738  27.693  17.122  1.00  9.55           C  \\nATOM    339  CD2 LEU A  43      30.205  30.168  17.129  1.00  6.41           C  \\nATOM    340  N   ILE A  44      26.436  30.232  20.004  1.00  4.58           N  \\nATOM    341  CA  ILE A  44      25.034  30.170  20.401  1.00  5.55           C  \\nATOM    342  C   ILE A  44      24.101  30.149  19.196  1.00  5.46           C  \\nATOM    343  O   ILE A  44      24.196  30.948  18.287  1.00  6.04           O  \\nATOM    344  CB  ILE A  44      24.639  31.426  21.286  1.00  6.80           C  \\nATOM    345  CG1 ILE A  44      25.646  31.670  22.421  1.00 10.31           C  \\nATOM    346  CG2 ILE A  44      23.181  31.309  21.824  1.00  7.39           C  \\nATOM    347  CD1 ILE A  44      25.778  30.436  23.356  1.00 13.90           C  \\nATOM    348  N   PHE A  45      23.141  29.187  19.241  1.00  6.75           N  \\nATOM    349  CA  PHE A  45      22.126  29.062  18.183  1.00  4.70           C  \\nATOM    350  C   PHE A  45      20.835  28.629  18.904  1.00  6.34           C  \\nATOM    351  O   PHE A  45      20.821  27.734  19.749  1.00  5.45           O  \\nATOM    352  CB  PHE A  45      22.494  28.057  17.109  1.00  5.51           C  \\nATOM    353  CG  PHE A  45      21.447  27.869  16.026  1.00  5.98           C  \\nATOM    354  CD1 PHE A  45      21.325  28.813  15.005  1.00  6.86           C  \\nATOM    355  CD2 PHE A  45      20.638  26.735  16.053  1.00  5.87           C  \\nATOM    356  CE1 PHE A  45      20.369  28.648  14.001  1.00  6.68           C  \\nATOM    357  CE2 PHE A  45      19.677  26.539  15.051  1.00  6.64           C  \\nATOM    358  CZ  PHE A  45      19.593  27.465  14.021  1.00  6.84           C  \\nATOM    359  N   ALA A  46      19.810  29.378  18.578  1.00  6.53           N  \\nATOM    360  CA  ALA A  46      18.443  29.143  19.083  1.00  7.15           C  \\nATOM    361  C   ALA A  46      18.453  28.941  20.591  1.00  9.00           C  \\nATOM    362  O   ALA A  46      17.860  27.994  21.128  1.00 11.15           O  \\nATOM    363  CB  ALA A  46      17.864  27.977  18.346  1.00  8.99           C  \\nATOM    364  N   GLY A  47      19.172  29.808  21.243  1.00  9.35           N  \\nATOM    365  CA  GLY A  47      19.399  29.894  22.655  1.00 11.68           C  \\nATOM    366  C   GLY A  47      20.083  28.729  23.321  1.00 11.14           C  \\nATOM    367  O   GLY A  47      19.991  28.584  24.561  1.00 13.93           O  \\nATOM    368  N   LYS A  48      20.801  27.931  22.578  1.00 10.47           N  \\nATOM    369  CA  LYS A  48      21.550  26.796  23.133  1.00  8.82           C  \\nATOM    370  C   LYS A  48      23.046  27.087  22.913  1.00  7.68           C  \\nATOM    371  O   LYS A  48      23.383  27.627  21.870  1.00  6.47           O  \\nATOM    372  CB  LYS A  48      21.242  25.519  22.391  1.00  9.74           C  \\nATOM    373  CG  LYS A  48      19.762  25.077  22.455  1.00 14.14           C  \\nATOM    374  CD  LYS A  48      19.634  23.885  21.531  1.00 16.32           C  \\nATOM    375  CE  LYS A  48      18.791  24.221  20.313  1.00 20.04           C  \\nATOM    376  NZ  LYS A  48      17.440  24.655  20.827  1.00 23.92           N  \\nATOM    377  N   GLN A  49      23.880  26.727  23.851  1.00  8.89           N  \\nATOM    378  CA  GLN A  49      25.349  26.872  23.643  1.00  7.18           C  \\nATOM    379  C   GLN A  49      25.743  25.586  22.922  1.00  8.23           C  \\nATOM    380  O   GLN A  49      25.325  24.489  23.378  1.00  9.70           O  \\nATOM    381  CB  GLN A  49      26.070  27.025  24.960  1.00 11.67           C  \\nATOM    382  CG  GLN A  49      27.553  27.356  24.695  1.00 15.82           C  \\nATOM    383  CD  GLN A  49      28.262  27.576  26.020  1.00 20.21           C  \\nATOM    384  OE1 GLN A  49      29.189  26.840  26.335  1.00 23.23           O  \\nATOM    385  NE2 GLN A  49      27.777  28.585  26.739  1.00 20.67           N  \\nATOM    386  N   LEU A  50      26.465  25.689  21.833  1.00  6.51           N  \\nATOM    387  CA  LEU A  50      26.826  24.521  21.012  1.00  7.41           C  \\nATOM    388  C   LEU A  50      27.994  23.781  21.643  1.00  8.27           C  \\nATOM    389  O   LEU A  50      28.904  24.444  22.098  1.00  8.34           O  \\nATOM    390  CB  LEU A  50      27.043  24.992  19.571  1.00  7.13           C  \\nATOM    391  CG  LEU A  50      25.931  25.844  18.959  1.00  7.53           C  \\nATOM    392  CD1 LEU A  50      26.203  26.083  17.471  1.00  8.14           C  \\nATOM    393  CD2 LEU A  50      24.577  25.190  19.079  1.00  9.11           C  \\nATOM    394  N   GLU A  51      27.942  22.448  21.648  1.00  9.43           N  \\nATOM    395  CA  GLU A  51      29.015  21.657  22.288  1.00 11.90           C  \\nATOM    396  C   GLU A  51      29.942  21.106  21.240  1.00 11.49           C  \\nATOM    397  O   GLU A  51      29.470  20.677  20.190  1.00  9.88           O  \\nATOM    398  CB  GLU A  51      28.348  20.540  23.066  1.00 16.56           C  \\nATOM    399  CG  GLU A  51      29.247  19.456  23.705  1.00 26.06           C  \\nATOM    400  CD  GLU A  51      28.722  19.047  25.066  1.00 29.86           C  \\nATOM    401  OE1 GLU A  51      29.139  18.132  25.746  1.00 32.13           O  \\nATOM    402  OE2 GLU A  51      27.777  19.842  25.367  1.00 33.44           O  \\nATOM    403  N   ASP A  52      31.233  21.090  21.459  1.00 12.71           N  \\nATOM    404  CA  ASP A  52      32.262  20.670  20.514  1.00 16.56           C  \\nATOM    405  C   ASP A  52      32.128  19.364  19.750  1.00 15.83           C  \\nATOM    406  O   ASP A  52      32.546  19.317  18.558  1.00 17.21           O  \\nATOM    407  CB  ASP A  52      33.638  20.716  21.242  1.00 21.05           C  \\nATOM    408  CG  ASP A  52      34.174  22.129  21.354  1.00 25.12           C  \\nATOM    409  OD1 ASP A  52      35.252  22.322  21.958  1.00 28.37           O  \\nATOM    410  OD2 ASP A  52      33.544  23.086  20.883  1.00 25.82           O  \\nATOM    411  N   GLY A  53      31.697  18.311  20.406  1.00 15.00           N  \\nATOM    412  CA  GLY A  53      31.568  16.962  19.825  1.00 11.77           C  \\nATOM    413  C   GLY A  53      30.320  16.698  19.051  1.00 11.10           C  \\nATOM    414  O   GLY A  53      30.198  15.657  18.366  1.00 11.25           O  \\nATOM    415  N   ARG A  54      29.340  17.594  19.076  1.00  8.53           N  \\nATOM    416  CA  ARG A  54      28.108  17.439  18.276  1.00  9.05           C  \\nATOM    417  C   ARG A  54      28.375  17.999  16.887  1.00  8.96           C  \\nATOM    418  O   ARG A  54      29.326  18.786  16.690  1.00 11.60           O  \\nATOM    419  CB  ARG A  54      26.926  18.191  18.892  1.00  7.97           C  \\nATOM    420  CG  ARG A  54      26.621  17.799  20.352  1.00  9.62           C  \\nATOM    421  CD  ARG A  54      26.010  16.370  20.280  1.00 12.20           C  \\nATOM    422  NE  ARG A  54      26.975  15.521  20.942  1.00 18.23           N  \\nATOM    423  CZ  ARG A  54      27.603  14.423  20.655  1.00 22.08           C  \\nATOM    424  NH1 ARG A  54      27.479  13.733  19.537  1.00 23.38           N  \\nATOM    425  NH2 ARG A  54      28.519  13.967  21.550  1.00 25.50           N  \\nATOM    426  N   THR A  55      27.510  17.689  15.954  1.00  9.05           N  \\nATOM    427  CA  THR A  55      27.574  18.192  14.563  1.00  9.03           C  \\nATOM    428  C   THR A  55      26.482  19.280  14.432  1.00  8.15           C  \\nATOM    429  O   THR A  55      25.609  19.388  15.287  1.00  5.91           O  \\nATOM    430  CB  THR A  55      27.299  17.055  13.533  1.00 11.15           C  \\nATOM    431  OG1 THR A  55      25.925  16.611  13.913  1.00 11.95           O  \\nATOM    432  CG2 THR A  55      28.236  15.864  13.558  1.00 11.71           C  \\nATOM    433  N   LEU A  56      26.585  20.063  13.378  1.00  6.91           N  \\nATOM    434  CA  LEU A  56      25.594  21.109  13.072  1.00  8.29           C  \\nATOM    435  C   LEU A  56      24.241  20.436  12.857  1.00  8.05           C  \\nATOM    436  O   LEU A  56      23.264  20.951  13.329  1.00 10.17           O  \\nATOM    437  CB  LEU A  56      26.084  21.888  11.833  1.00  6.60           C  \\nATOM    438  CG  LEU A  56      27.426  22.616  11.902  1.00  7.73           C  \\nATOM    439  CD1 LEU A  56      27.718  23.341  10.578  1.00  9.85           C  \\nATOM    440  CD2 LEU A  56      27.380  23.721  12.955  1.00  8.64           C  \\nATOM    441  N   SER A  57      24.240  19.233  12.246  1.00  8.92           N  \\nATOM    442  CA  SER A  57      22.924  18.583  12.025  1.00  9.00           C  \\nATOM    443  C   SER A  57      22.229  18.244  13.325  1.00  9.44           C  \\nATOM    444  O   SER A  57      20.963  18.253  13.395  1.00 10.91           O  \\nATOM    445  CB  SER A  57      23.059  17.326  11.154  1.00 10.32           C  \\nATOM    446  OG  SER A  57      23.914  16.395  11.755  1.00 13.59           O  \\nATOM    447  N   ASP A  58      22.997  17.978  14.366  1.00  9.11           N  \\nATOM    448  CA  ASP A  58      22.418  17.638  15.693  1.00  7.91           C  \\nATOM    449  C   ASP A  58      21.460  18.737  16.163  1.00  9.12           C  \\nATOM    450  O   ASP A  58      20.497  18.506  16.900  1.00  8.61           O  \\nATOM    451  CB  ASP A  58      23.461  17.331  16.741  1.00  8.41           C  \\nATOM    452  CG  ASP A  58      24.184  16.016  16.619  1.00 11.50           C  \\nATOM    453  OD1 ASP A  58      25.303  15.894  17.152  1.00 10.05           O  \\nATOM    454  OD2 ASP A  58      23.572  15.107  15.975  1.00 11.70           O  \\nATOM    455  N   TYR A  59      21.846  19.954  15.905  1.00  7.97           N  \\nATOM    456  CA  TYR A  59      21.079  21.149  16.251  1.00  8.45           C  \\nATOM    457  C   TYR A  59      20.142  21.590  15.149  1.00 10.98           C  \\nATOM    458  O   TYR A  59      19.499  22.645  15.321  1.00 12.95           O  \\nATOM    459  CB  TYR A  59      22.085  22.254  16.581  1.00  7.94           C  \\nATOM    460  CG  TYR A  59      22.945  21.951  17.785  1.00  6.91           C  \\nATOM    461  CD1 TYR A  59      24.272  21.544  17.644  1.00  4.59           C  \\nATOM    462  CD2 TYR A  59      22.437  22.157  19.065  1.00  6.98           C  \\nATOM    463  CE1 TYR A  59      25.052  21.285  18.776  1.00  5.39           C  \\nATOM    464  CE2 TYR A  59      23.204  21.907  20.192  1.00  6.52           C  \\nATOM    465  CZ  TYR A  59      24.517  21.470  20.030  1.00  6.76           C  \\nATOM    466  OH  TYR A  59      25.248  21.302  21.191  1.00  7.63           O  \\nATOM    467  N   ASN A  60      19.993  20.884  14.049  1.00 12.38           N  \\nATOM    468  CA  ASN A  60      19.065  21.352  12.999  1.00 13.94           C  \\nATOM    469  C   ASN A  60      19.442  22.745  12.510  1.00 14.16           C  \\nATOM    470  O   ASN A  60      18.571  23.610  12.289  1.00 14.26           O  \\nATOM    471  CB  ASN A  60      17.586  21.282  13.461  1.00 19.23           C  \\nATOM    472  CG  ASN A  60      16.576  21.258  12.315  1.00 22.65           C  \\nATOM    473  OD1 ASN A  60      15.440  21.819  12.378  1.00 25.45           O  \\nATOM    474  ND2 ASN A  60      16.924  20.586  11.216  1.00 24.09           N  \\nATOM    475  N   ILE A  61      20.717  22.964  12.260  1.00 11.08           N  \\nATOM    476  CA  ILE A  61      21.184  24.263  11.690  1.00 11.78           C  \\nATOM    477  C   ILE A  61      21.110  24.111  10.173  1.00 13.74           C  \\nATOM    478  O   ILE A  61      21.841  23.198   9.686  1.00 14.60           O  \\nATOM    479  CB  ILE A  61      22.650  24.516  12.172  1.00 11.80           C  \\nATOM    480  CG1 ILE A  61      22.662  24.819  13.699  1.00 11.56           C  \\nATOM    481  CG2 ILE A  61      23.376  25.645  11.409  1.00 13.29           C  \\nATOM    482  CD1 ILE A  61      24.123  24.981  14.195  1.00 11.42           C  \\nATOM    483  N   GLN A  62      20.291  24.875   9.507  1.00 13.97           N  \\nATOM    484  CA  GLN A  62      20.081  24.773   8.033  1.00 15.52           C  \\nATOM    485  C   GLN A  62      20.822  25.914   7.332  1.00 13.94           C  \\nATOM    486  O   GLN A  62      21.323  26.830   8.008  1.00 12.15           O  \\nATOM    487  CB  GLN A  62      18.599  24.736   7.727  1.00 19.53           C  \\nATOM    488  CG  GLN A  62      17.819  23.434   7.900  1.00 26.38           C  \\nATOM    489  CD  GLN A  62      16.509  23.529   7.116  1.00 30.61           C  \\nATOM    490  OE1 GLN A  62      15.446  22.980   7.433  1.00 33.23           O  \\nATOM    491  NE2 GLN A  62      16.539  24.293   6.009  1.00 32.71           N  \\nATOM    492  N   LYS A  63      20.924  25.862   6.006  1.00 11.73           N  \\nATOM    493  CA  LYS A  63      21.656  26.847   5.240  1.00 11.97           C  \\nATOM    494  C   LYS A  63      21.127  28.240   5.574  1.00 10.41           C  \\nATOM    495  O   LYS A  63      19.958  28.465   5.842  1.00  9.59           O  \\nATOM    496  CB  LYS A  63      21.631  26.642   3.731  1.00 13.73           C  \\nATOM    497  CG  LYS A  63      20.210  26.423   3.175  1.00 16.98           C  \\nATOM    498  CD  LYS A  63      20.268  26.589   1.656  1.00 20.19           C  \\nATOM    499  CE  LYS A  63      19.202  25.857   0.891  1.00 23.42           C  \\nATOM    500  NZ  LYS A  63      17.884  26.544   1.075  1.00 25.97           N  \\nATOM    501  N   GLU A  64      22.099  29.163   5.605  1.00 10.04           N  \\nATOM    502  CA  GLU A  64      21.907  30.563   5.881  1.00 10.94           C  \\nATOM    503  C   GLU A  64      21.466  30.953   7.261  1.00  9.74           C  \\nATOM    504  O   GLU A  64      21.066  32.112   7.533  1.00  9.42           O  \\nATOM    505  CB  GLU A  64      21.023  31.223   4.784  1.00 18.31           C  \\nATOM    506  CG  GLU A  64      21.861  31.342   3.474  1.00 24.16           C  \\nATOM    507  CD  GLU A  64      21.156  30.726   2.311  1.00 29.00           C  \\nATOM    508  OE1 GLU A  64      19.942  30.793   2.170  1.00 31.72           O  \\nATOM    509  OE2 GLU A  64      21.954  30.152   1.535  1.00 32.61           O  \\nATOM    510  N   SER A  65      21.674  30.034   8.191  1.00  6.85           N  \\nATOM    511  CA  SER A  65      21.419  30.253   9.620  1.00  6.90           C  \\nATOM    512  C   SER A  65      22.504  31.228  10.136  1.00  4.72           C  \\nATOM    513  O   SER A  65      23.579  31.321   9.554  1.00  3.91           O  \\nATOM    514  CB  SER A  65      21.637  28.923  10.353  1.00  7.28           C  \\nATOM    515  OG  SER A  65      20.544  28.047  10.059  1.00 10.56           O  \\nATOM    516  N   THR A  66      22.241  31.873  11.241  1.00  4.48           N  \\nATOM    517  CA  THR A  66      23.212  32.762  11.891  1.00  3.80           C  \\nATOM    518  C   THR A  66      23.509  32.224  13.290  1.00  4.60           C  \\nATOM    519  O   THR A  66      22.544  31.942  14.034  1.00  5.33           O  \\nATOM    520  CB  THR A  66      22.699  34.267  11.985  1.00  2.85           C  \\nATOM    521  OG1 THR A  66      22.495  34.690  10.589  1.00  2.15           O  \\nATOM    522  CG2 THR A  66      23.727  35.131  12.722  1.00  3.40           C  \\nATOM    523  N   LEU A  67      24.790  32.021  13.618  1.00  4.17           N  \\nATOM    524  CA  LEU A  67      25.149  31.609  14.980  1.00  3.85           C  \\nATOM    525  C   LEU A  67      25.698  32.876  15.669  1.00  3.80           C  \\nATOM    526  O   LEU A  67      26.158  33.730  14.894  1.00  5.54           O  \\nATOM    527  CB  LEU A  67      26.310  30.594  14.967  1.00  7.18           C  \\nATOM    528  CG  LEU A  67      26.290  29.480  13.960  1.00  9.67           C  \\nATOM    529  CD1 LEU A  67      27.393  28.442  14.229  1.00  8.12           C  \\nATOM    530  CD2 LEU A  67      24.942  28.807  13.952  1.00 11.66           C  \\nATOM    531  N   HIS A  68      25.621  32.945  16.950  1.00  2.94           N  \\nATOM    532  CA  HIS A  68      26.179  34.127  17.650  1.00  4.17           C  \\nATOM    533  C   HIS A  68      27.475  33.651  18.304  1.00  5.32           C  \\nATOM    534  O   HIS A  68      27.507  32.587  18.958  1.00  7.70           O  \\nATOM    535  CB  HIS A  68      25.214  34.565  18.780  1.00  5.57           C  \\nATOM    536  CG  HIS A  68      23.978  35.121  18.126  1.00  9.95           C  \\nATOM    537  ND1 HIS A  68      23.853  36.432  17.781  1.00 13.74           N  \\nATOM    538  CD2 HIS A  68      22.824  34.514  17.782  1.00 12.79           C  \\nATOM    539  CE1 HIS A  68      22.674  36.627  17.200  1.00 14.75           C  \\nATOM    540  NE2 HIS A  68      22.045  35.455  17.173  1.00 16.30           N  \\nATOM    541  N   LEU A  69      28.525  34.447  18.189  1.00  5.29           N  \\nATOM    542  CA  LEU A  69      29.801  34.145  18.829  1.00  3.97           C  \\nATOM    543  C   LEU A  69      30.052  35.042  20.004  1.00  5.07           C  \\nATOM    544  O   LEU A  69      30.105  36.305  19.788  1.00  4.34           O  \\nATOM    545  CB  LEU A  69      30.925  34.304  17.753  1.00  6.08           C  \\nATOM    546  CG  LEU A  69      32.345  34.183  18.358  1.00  7.37           C  \\nATOM    547  CD1 LEU A  69      32.555  32.783  18.870  1.00  6.87           C  \\nATOM    548  CD2 LEU A  69      33.361  34.491  17.245  1.00  9.96           C  \\nATOM    549  N   VAL A  70      30.124  34.533  21.191  1.00  4.29           N  \\nATOM    550  CA  VAL A  70      30.479  35.369  22.374  1.00  6.26           C  \\nATOM    551  C   VAL A  70      31.901  34.910  22.728  1.00  9.22           C  \\nATOM    552  O   VAL A  70      32.190  33.696  22.635  1.00  9.36           O  \\nATOM    553  CB  VAL A  70      29.472  35.181  23.498  1.00  8.69           C  \\nATOM    554  CG1 VAL A  70      29.821  35.957  24.765  1.00  9.76           C  \\nATOM    555  CG2 VAL A  70      28.049  35.454  23.071  1.00  8.54           C  \\nATOM    556  N   LEU A  71      32.763  35.831  23.090  1.00 12.71           N  \\nATOM    557  CA  LEU A  71      34.145  35.472  23.481  1.00 16.06           C  \\nATOM    558  C   LEU A  71      34.239  35.353  24.979  1.00 18.09           C  \\nATOM    559  O   LEU A  71      33.707  36.197  25.728  1.00 19.26           O  \\nATOM    560  CB  LEU A  71      35.114  36.564  22.907  1.00 17.10           C  \\nATOM    561  CG  LEU A  71      35.926  35.979  21.737  1.00 19.37           C  \\nATOM    562  CD1 LEU A  71      35.003  35.084  20.920  1.00 17.51           C  \\nATOM    563  CD2 LEU A  71      36.533  37.087  20.917  1.00 19.57           C  \\nATOM    564  N   ARG A  72      34.930  34.384  25.451  1.00 21.47           N  \\nATOM    565  CA  ARG A  72      35.161  34.174  26.896  1.00 25.83           C  \\nATOM    566  C   ARG A  72      36.671  34.296  27.089  1.00 27.74           C  \\nATOM    567  O   ARG A  72      37.305  33.233  26.795  1.00 30.65           O  \\nATOM    568  CB  ARG A  72      34.717  32.760  27.286  1.00 28.49           C  \\nATOM    569  CG  ARG A  72      35.752  32.054  28.160  1.00 31.79           C  \\nATOM    570  CD  ARG A  72      35.612  30.577  28.044  1.00 34.05           C  \\nATOM    571  NE  ARG A  72      35.040  30.252  26.730  1.00 35.08           N  \\nATOM    572  CZ  ARG A  72      34.338  29.103  26.650  1.00 34.67           C  \\nATOM    573  NH1 ARG A  72      34.110  28.437  27.768  1.00 35.02           N  \\nATOM    574  NH2 ARG A  72      34.014  28.657  25.457  1.00 34.97           N  \\nATOM    575  N   LEU A  73      37.197  35.397  27.513  0.45 28.93           N  \\nATOM    576  CA  LEU A  73      38.668  35.502  27.680  0.45 30.76           C  \\nATOM    577  C   LEU A  73      39.076  34.931  29.031  0.45 32.18           C  \\nATOM    578  O   LEU A  73      38.297  34.946  29.996  0.45 32.31           O  \\nATOM    579  CB  LEU A  73      39.080  36.941  27.406  0.45 30.53           C  \\nATOM    580  CG  LEU A  73      39.502  37.340  26.002  0.45 30.16           C  \\nATOM    581  CD1 LEU A  73      38.684  36.647  24.923  0.45 29.57           C  \\nATOM    582  CD2 LEU A  73      39.337  38.854  25.862  0.45 29.11           C  \\nATOM    583  N   ARG A  74      40.294  34.412  29.045  0.45 33.82           N  \\nATOM    584  CA  ARG A  74      40.873  33.802  30.253  0.45 35.33           C  \\nATOM    585  C   ARG A  74      41.765  34.829  30.944  0.45 36.22           C  \\nATOM    586  O   ARG A  74      42.945  34.994  30.583  0.45 36.70           O  \\nATOM    587  CB  ARG A  74      41.651  32.529  29.923  0.45 36.91           C  \\nATOM    588  CG  ARG A  74      41.608  31.444  30.989  0.45 38.62           C  \\nATOM    589  CD  ARG A  74      41.896  30.080  30.456  0.45 39.75           C  \\nATOM    590  NE  ARG A  74      43.311  29.735  30.563  0.45 41.13           N  \\nATOM    591  CZ  ARG A  74      44.174  29.905  29.554  0.45 41.91           C  \\nATOM    592  NH1 ARG A  74      43.754  30.312  28.356  0.45 42.75           N  \\nATOM    593  NH2 ARG A  74      45.477  29.726  29.763  0.45 41.93           N  \\nATOM    594  N   GLY A  75      41.165  35.531  31.898  0.25 36.31           N  \\nATOM    595  CA  GLY A  75      41.845  36.550  32.686  0.25 36.07           C  \\nATOM    596  C   GLY A  75      41.251  37.941  32.588  0.25 36.16           C  \\nATOM    597  O   GLY A  75      41.102  38.523  31.500  0.25 36.26           O  \\nATOM    598  N   GLY A  76      40.946  38.472  33.757  0.25 36.05           N  \\nATOM    599  CA  GLY A  76      40.373  39.813  33.944  0.25 36.19           C  \\nATOM    600  C   GLY A  76      40.031  39.992  35.432  0.25 36.20           C  \\nATOM    601  O   GLY A  76      38.933  40.525  35.687  0.25 36.13           O  \\nATOM    602  OXT GLY A  76      40.862  39.575  36.251  0.25 36.27           O  \\nTER     603      GLY A  76                                                      \\nHETATM  604  O   HOH A  77      45.747  30.081  19.708  1.00 12.43           O  \\nHETATM  605  O   HOH A  78      19.168  31.868  17.050  1.00 12.65           O  \\nHETATM  606  O   HOH A  79      32.010  38.387  19.636  1.00 12.83           O  \\nHETATM  607  O   HOH A  80      42.084  27.361  21.953  1.00 22.27           O  \\nHETATM  608  O   HOH A  81      21.314  20.644   8.719  1.00 18.33           O  \\nHETATM  609  O   HOH A  82      31.965  38.637   3.699  1.00 31.69           O  \\nHETATM  610  O   HOH A  83      27.707  15.908   4.653  1.00 20.30           O  \\nHETATM  611  O   HOH A  84      19.969  32.720  14.769  1.00 10.14           O  \\nHETATM  612  O   HOH A  85      29.847  13.577  10.864  1.00 29.65           O  \\nHETATM  613  O   HOH A  86      23.893  27.864   1.501  1.00 23.48           O  \\nHETATM  614  O   HOH A  87      19.638  23.312   4.775  1.00 18.40           O  \\nHETATM  615  O   HOH A  88      34.628  29.369   4.779  1.00 26.17           O  \\nHETATM  616  O   HOH A  89      42.240  24.744  25.707  1.00 31.34           O  \\nHETATM  617  O   HOH A  90      30.290  42.500   8.820  1.00 16.49           O  \\nHETATM  618  O   HOH A  91      24.512  39.162  10.841  1.00 13.14           O  \\nHETATM  619  O   HOH A  92      26.557  43.450  19.940  1.00 19.38           O  \\nHETATM  620  O   HOH A  93      42.535  22.385  13.872  1.00 29.35           O  \\nHETATM  621  O   HOH A  94      42.440  26.381  12.686  1.00 29.46           O  \\nHETATM  622  O   HOH A  95      22.651  14.457  13.085  1.00 22.07           O  \\nHETATM  623  O   HOH A  96      35.325  26.551  23.202  1.00 15.20           O  \\nHETATM  624  O   HOH A  97      23.629  20.940   3.146  1.00 15.45           O  \\nHETATM  625  O   HOH A  98      25.928  21.774   2.325  1.00 13.70           O  \\nHETATM  626  O   HOH A  99      33.388  21.973   5.659  1.00 24.89           O  \\nHETATM  627  O   HOH A 100      18.326  23.911  17.697  1.00 24.10           O  \\nHETATM  628  O   HOH A 101      18.160  27.072  10.662  1.00 20.76           O  \\nHETATM  629  O   HOH A 102      34.746  17.167  18.219  1.00 32.86           O  \\nHETATM  630  O   HOH A 103      19.801  32.364  20.210  1.00 21.09           O  \\nHETATM  631  O   HOH A 104      30.285  26.829  22.191  1.00  8.56           O  \\nHETATM  632  O   HOH A 105      44.612  32.306  16.961  1.00  7.69           O  \\nHETATM  633  O   HOH A 106      16.287  25.999  13.142  0.78 28.90           O  \\nHETATM  634  O   HOH A 107      27.101  42.135  15.494  0.51 23.36           O  \\nHETATM  635  O   HOH A 108      37.209  23.795  21.367  0.74 27.88           O  \\nHETATM  636  O   HOH A 109      19.582  32.034  -0.685  0.49 22.24           O  \\nHETATM  637  O   HOH A 110      28.824  25.094   0.886  0.77 36.99           O  \\nHETATM  638  O   HOH A 111      25.146  19.162  25.323  0.87 36.70           O  \\nHETATM  639  O   HOH A 112      20.747  37.769  14.674  0.85 29.64           O  \\nHETATM  640  O   HOH A 113      16.035  17.841   8.765  0.61 23.89           O  \\nHETATM  641  O   HOH A 114      35.712  46.814  12.926  0.48 27.11           O  \\nHETATM  642  O   HOH A 115      15.570  27.475   7.482  0.51 24.18           O  \\nHETATM  643  O   HOH A 116      33.447  21.075   2.918  0.59 26.03           O  \\nHETATM  644  O   HOH A 117      41.116  39.021  13.061  0.63 22.39           O  \\nHETATM  645  O   HOH A 118      32.346  13.689  18.912  0.48 24.09           O  \\nHETATM  646  O   HOH A 119      31.197  13.048   7.920  0.71 29.54           O  \\nHETATM  647  O   HOH A 120      42.853  39.375  29.308  0.64 46.90           O  \\nHETATM  648  O   HOH A 121      39.646  23.959   9.699  0.41 18.25           O  \\nHETATM  649  O   HOH A 122      34.405  45.181  13.420  0.87 26.13           O  \\nHETATM  650  O   HOH A 123      26.517  24.300  27.592  0.41 21.02           O  \\nHETATM  651  O   HOH A 124      40.740  38.734   9.602  0.45 16.60           O  \\nHETATM  652  O   HOH A 125      31.494  18.276  23.170  0.67 26.53           O  \\nHETATM  653  O   HOH A 126      37.752  30.947   1.059  0.87 32.52           O  \\nHETATM  654  O   HOH A 127      31.771  16.941   7.511  0.64 15.94           O  \\nHETATM  655  O   HOH A 128      41.628  24.537  10.145  0.57 22.53           O  \\nHETATM  656  O   HOH A 129      28.988  22.175  -1.744  0.56 29.32           O  \\nHETATM  657  O   HOH A 130      14.882  16.539  10.692  0.53 24.82           O  \\nHETATM  658  O   HOH A 131      32.589  40.385   7.523  0.36 26.01           O  \\nHETATM  659  O   HOH A 132      38.363  30.369   5.579  0.49 35.45           O  \\nHETATM  660  O   HOH A 133      27.841  46.062  17.589  0.81 32.15           O  \\nHETATM  661  O   HOH A 134      37.667  43.421  17.000  0.50 33.32           O  \\nMASTER      260    0    0    2    5    0    0    6  660    1    0    6          \\nEND                                                                             \\n\",\"pdb\");\n\tviewer_17338419469247565.setStyle({\"cartoon\": {\"color\": \"spectrum\"}, \"stick\": {\"radius\": 0.15}});\n\tviewer_17338419469247565.center();\n\tviewer_17338419469247565.zoomTo();\nviewer_17338419469247565.render();\n});\n</script>",
            "text/html": [
              "<div id=\"3dmolviewer_17338419469247565\"  style=\"position: relative; width: 400px; height: 300px;\">\n",
              "        <p id=\"3dmolwarning_17338419469247565\" style=\"background-color:#ffcccc;color:black\">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br></p>\n",
              "        </div>\n",
              "<script>\n",
              "\n",
              "var loadScriptAsync = function(uri){\n",
              "  return new Promise((resolve, reject) => {\n",
              "    //this is to ignore the existence of requirejs amd\n",
              "    var savedexports, savedmodule;\n",
              "    if (typeof exports !== 'undefined') savedexports = exports;\n",
              "    else exports = {}\n",
              "    if (typeof module !== 'undefined') savedmodule = module;\n",
              "    else module = {}\n",
              "\n",
              "    var tag = document.createElement('script');\n",
              "    tag.src = uri;\n",
              "    tag.async = true;\n",
              "    tag.onload = () => {\n",
              "        exports = savedexports;\n",
              "        module = savedmodule;\n",
              "        resolve();\n",
              "    };\n",
              "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
              "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
              "});\n",
              "};\n",
              "\n",
              "if(typeof $3Dmolpromise === 'undefined') {\n",
              "$3Dmolpromise = null;\n",
              "  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.4.2/3Dmol-min.js');\n",
              "}\n",
              "\n",
              "var viewer_17338419469247565 = null;\n",
              "var warn = document.getElementById(\"3dmolwarning_17338419469247565\");\n",
              "if(warn) {\n",
              "    warn.parentNode.removeChild(warn);\n",
              "}\n",
              "$3Dmolpromise.then(function() {\n",
              "viewer_17338419469247565 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_17338419469247565\"),{backgroundColor:\"white\"});\n",
              "viewer_17338419469247565.zoomTo();\n",
              "\tviewer_17338419469247565.addModel(\"HEADER    CHROMOSOMAL PROTEIN                     02-JAN-87   1UBQ              \\nTITLE     STRUCTURE OF UBIQUITIN REFINED AT 1.8 ANGSTROMS RESOLUTION            \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: UBIQUITIN;                                                 \\nCOMPND   3 CHAIN: A;                                                            \\nCOMPND   4 ENGINEERED: YES                                                      \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 ORGANISM_SCIENTIFIC: HOMO SAPIENS;                                   \\nSOURCE   3 ORGANISM_COMMON: HUMAN;                                              \\nSOURCE   4 ORGANISM_TAXID: 9606                                                 \\nKEYWDS    CHROMOSOMAL PROTEIN                                                   \\nEXPDTA    X-RAY DIFFRACTION                                                     \\nAUTHOR    S.VIJAY-KUMAR,C.E.BUGG,W.J.COOK                                       \\nREVDAT   6   14-FEB-24 1UBQ    1       REMARK                                   \\nREVDAT   5   09-MAR-11 1UBQ    1       REMARK                                   \\nREVDAT   4   24-FEB-09 1UBQ    1       VERSN                                    \\nREVDAT   3   01-APR-03 1UBQ    1       JRNL                                     \\nREVDAT   2   16-JUL-87 1UBQ    1       JRNL   REMARK                            \\nREVDAT   1   16-APR-87 1UBQ    0                                                \\nJRNL        AUTH   S.VIJAY-KUMAR,C.E.BUGG,W.J.COOK                              \\nJRNL        TITL   STRUCTURE OF UBIQUITIN REFINED AT 1.8 A RESOLUTION.          \\nJRNL        REF    J.MOL.BIOL.                   V. 194   531 1987              \\nJRNL        REFN                   ISSN 0022-2836                               \\nJRNL        PMID   3041007                                                      \\nJRNL        DOI    10.1016/0022-2836(87)90679-6                                 \\nREMARK   1                                                                      \\nREMARK   1 REFERENCE 1                                                          \\nREMARK   1  AUTH   S.VIJAY-KUMAR,C.E.BUGG,K.D.WILKINSON,R.D.VIERSTRA,           \\nREMARK   1  AUTH 2 P.M.HATFIELD,W.J.COOK                                        \\nREMARK   1  TITL   COMPARISON OF THE THREE-DIMENSIONAL STRUCTURES OF HUMAN,     \\nREMARK   1  TITL 2 YEAST, AND OAT UBIQUITIN                                     \\nREMARK   1  REF    J.BIOL.CHEM.                  V. 262  6396 1987              \\nREMARK   1  REFN                   ISSN 0021-9258                               \\nREMARK   1 REFERENCE 2                                                          \\nREMARK   1  AUTH   S.VIJAY-KUMAR,C.E.BUGG,K.D.WILKINSON,W.J.COOK                \\nREMARK   1  TITL   THREE-DIMENSIONAL STRUCTURE OF UBIQUITIN AT 2.8 ANGSTROMS    \\nREMARK   1  TITL 2 RESOLUTION                                                   \\nREMARK   1  REF    PROC.NATL.ACAD.SCI.USA        V.  82  3582 1985              \\nREMARK   1  REFN                   ISSN 0027-8424                               \\nREMARK   1 REFERENCE 3                                                          \\nREMARK   1  AUTH   W.J.COOK,F.L.SUDDATH,C.E.BUGG,G.GOLDSTEIN                    \\nREMARK   1  TITL   CRYSTALLIZATION AND PRELIMINARY X-RAY INVESTIGATION OF       \\nREMARK   1  TITL 2 UBIQUITIN, A NON-HISTONE CHROMOSOMAL PROTEIN                 \\nREMARK   1  REF    J.MOL.BIOL.                   V. 130   353 1979              \\nREMARK   1  REFN                   ISSN 0022-2836                               \\nREMARK   1 REFERENCE 4                                                          \\nREMARK   1  AUTH   D.H.SCHLESINGER,G.GOLDSTEIN                                  \\nREMARK   1  TITL   MOLECULAR CONSERVATION OF 74 AMINO ACID SEQUENCE OF          \\nREMARK   1  TITL 2 UBIQUITIN BETWEEN CATTLE AND MAN                             \\nREMARK   1  REF    NATURE                        V. 255   423 1975              \\nREMARK   1  REFN                   ISSN 0028-0836                               \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION.    1.80 ANGSTROMS.                                       \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : PROLSQ                                               \\nREMARK   3   AUTHORS     : KONNERT,HENDRICKSON                                  \\nREMARK   3                                                                      \\nREMARK   3  DATA USED IN REFINEMENT.                                            \\nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 1.80                           \\nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : NULL                           \\nREMARK   3   DATA CUTOFF            (SIGMA(F)) : NULL                           \\nREMARK   3   COMPLETENESS FOR RANGE        (%) : NULL                           \\nREMARK   3   NUMBER OF REFLECTIONS             : NULL                           \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \\nREMARK   3   CROSS-VALIDATION METHOD          : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SELECTION  : NULL                            \\nREMARK   3   R VALUE     (WORKING + TEST SET) : 0.176                           \\nREMARK   3   R VALUE            (WORKING SET) : NULL                            \\nREMARK   3   FREE R VALUE                     : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : NULL                            \\nREMARK   3   FREE R VALUE TEST SET COUNT      : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  FIT/AGREEMENT OF MODEL WITH ALL DATA.                               \\nREMARK   3   R VALUE   (WORKING + TEST SET, NO CUTOFF) : NULL                   \\nREMARK   3   R VALUE          (WORKING SET, NO CUTOFF) : NULL                   \\nREMARK   3   FREE R VALUE                  (NO CUTOFF) : NULL                   \\nREMARK   3   FREE R VALUE TEST SET SIZE (%, NO CUTOFF) : NULL                   \\nREMARK   3   FREE R VALUE TEST SET COUNT   (NO CUTOFF) : NULL                   \\nREMARK   3   TOTAL NUMBER OF REFLECTIONS   (NO CUTOFF) : NULL                   \\nREMARK   3                                                                      \\nREMARK   3  NUMBER OF NON-HYDROGEN ATOMS USED IN REFINEMENT.                    \\nREMARK   3   PROTEIN ATOMS            : 602                                     \\nREMARK   3   NUCLEIC ACID ATOMS       : 0                                       \\nREMARK   3   HETEROGEN ATOMS          : 0                                       \\nREMARK   3   SOLVENT ATOMS            : 58                                      \\nREMARK   3                                                                      \\nREMARK   3  B VALUES.                                                           \\nREMARK   3   FROM WILSON PLOT           (A**2) : NULL                           \\nREMARK   3   MEAN B VALUE      (OVERALL, A**2) : NULL                           \\nREMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \\nREMARK   3    B11 (A**2) : NULL                                                 \\nREMARK   3    B22 (A**2) : NULL                                                 \\nREMARK   3    B33 (A**2) : NULL                                                 \\nREMARK   3    B12 (A**2) : NULL                                                 \\nREMARK   3    B13 (A**2) : NULL                                                 \\nREMARK   3    B23 (A**2) : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  ESTIMATED COORDINATE ERROR.                                         \\nREMARK   3   ESD FROM LUZZATI PLOT        (A) : NULL                            \\nREMARK   3   ESD FROM SIGMAA              (A) : NULL                            \\nREMARK   3   LOW RESOLUTION CUTOFF        (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  RMS DEVIATIONS FROM IDEAL VALUES.                                   \\nREMARK   3   DISTANCE RESTRAINTS.                    RMS    SIGMA               \\nREMARK   3    BOND LENGTH                     (A) : 0.016 ; NULL                \\nREMARK   3    ANGLE DISTANCE                  (A) : NULL  ; NULL                \\nREMARK   3    INTRAPLANAR 1-4 DISTANCE        (A) : NULL  ; NULL                \\nREMARK   3    H-BOND OR METAL COORDINATION    (A) : NULL  ; NULL                \\nREMARK   3                                                                      \\nREMARK   3   PLANE RESTRAINT                  (A) : NULL  ; NULL                \\nREMARK   3   CHIRAL-CENTER RESTRAINT       (A**3) : NULL  ; NULL                \\nREMARK   3                                                                      \\nREMARK   3   NON-BONDED CONTACT RESTRAINTS.                                     \\nREMARK   3    SINGLE TORSION                  (A) : NULL  ; NULL                \\nREMARK   3    MULTIPLE TORSION                (A) : NULL  ; NULL                \\nREMARK   3    H-BOND (X...Y)                  (A) : NULL  ; NULL                \\nREMARK   3    H-BOND (X-H...Y)                (A) : NULL  ; NULL                \\nREMARK   3                                                                      \\nREMARK   3   CONFORMATIONAL TORSION ANGLE RESTRAINTS.                           \\nREMARK   3    SPECIFIED                 (DEGREES) : NULL  ; NULL                \\nREMARK   3    PLANAR                    (DEGREES) : NULL  ; NULL                \\nREMARK   3    STAGGERED                 (DEGREES) : NULL  ; NULL                \\nREMARK   3    TRANSVERSE                (DEGREES) : NULL  ; NULL                \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL FACTOR RESTRAINTS.    RMS    SIGMA                \\nREMARK   3   MAIN-CHAIN BOND               (A**2) : NULL  ; NULL                \\nREMARK   3   MAIN-CHAIN ANGLE              (A**2) : NULL  ; NULL                \\nREMARK   3   SIDE-CHAIN BOND               (A**2) : NULL  ; NULL                \\nREMARK   3   SIDE-CHAIN ANGLE              (A**2) : NULL  ; NULL                \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS: NULL                                      \\nREMARK   4                                                                      \\nREMARK   4 1UBQ COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY BNL.                                \\nREMARK 100 THE DEPOSITION ID IS D_1000176905.                                   \\nREMARK 200                                                                      \\nREMARK 200 EXPERIMENTAL DETAILS                                                 \\nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \\nREMARK 200  DATE OF DATA COLLECTION        : NULL                               \\nREMARK 200  TEMPERATURE           (KELVIN) : NULL                               \\nREMARK 200  PH                             : NULL                               \\nREMARK 200  NUMBER OF CRYSTALS USED        : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  SYNCHROTRON              (Y/N) : NULL                               \\nREMARK 200  RADIATION SOURCE               : NULL                               \\nREMARK 200  BEAMLINE                       : NULL                               \\nREMARK 200  X-RAY GENERATOR MODEL          : NULL                               \\nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : NULL                               \\nREMARK 200  WAVELENGTH OR RANGE        (A) : NULL                               \\nREMARK 200  MONOCHROMATOR                  : NULL                               \\nREMARK 200  OPTICS                         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  DETECTOR TYPE                  : NULL                               \\nREMARK 200  DETECTOR MANUFACTURER          : NULL                               \\nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : NULL                               \\nREMARK 200  DATA SCALING SOFTWARE          : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : NULL                               \\nREMARK 200  RESOLUTION RANGE HIGH      (A) : NULL                               \\nREMARK 200  RESOLUTION RANGE LOW       (A) : NULL                               \\nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 OVERALL.                                                             \\nREMARK 200  COMPLETENESS FOR RANGE     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY                : NULL                               \\nREMARK 200  R MERGE                    (I) : NULL                               \\nREMARK 200  R SYM                      (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : NULL                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : NULL                     \\nREMARK 200  COMPLETENESS FOR SHELL     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY IN SHELL       : NULL                               \\nREMARK 200  R MERGE FOR SHELL          (I) : NULL                               \\nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR SHELL         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 DIFFRACTION PROTOCOL: NULL                                           \\nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: NULL                         \\nREMARK 200 SOFTWARE USED: NULL                                                  \\nREMARK 200 STARTING MODEL: NULL                                                 \\nREMARK 200                                                                      \\nREMARK 200 REMARK: NULL                                                         \\nREMARK 280                                                                      \\nREMARK 280 CRYSTAL                                                              \\nREMARK 280 SOLVENT CONTENT, VS   (%): 32.94                                     \\nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): 1.83                     \\nREMARK 280                                                                      \\nREMARK 280 CRYSTALLIZATION CONDITIONS: NULL                                     \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \\nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: P 21 21 21                       \\nREMARK 290                                                                      \\nREMARK 290      SYMOP   SYMMETRY                                                \\nREMARK 290     NNNMMM   OPERATOR                                                \\nREMARK 290       1555   X,Y,Z                                                   \\nREMARK 290       2555   -X+1/2,-Y,Z+1/2                                         \\nREMARK 290       3555   -X,Y+1/2,-Z+1/2                                         \\nREMARK 290       4555   X+1/2,-Y+1/2,-Z                                         \\nREMARK 290                                                                      \\nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \\nREMARK 290           MMM -> TRANSLATION VECTOR                                  \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \\nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \\nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \\nREMARK 290 RELATED MOLECULES.                                                   \\nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   2 -1.000000  0.000000  0.000000       25.42000            \\nREMARK 290   SMTRY2   2  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   2  0.000000  0.000000  1.000000       14.47500            \\nREMARK 290   SMTRY1   3 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   3  0.000000  1.000000  0.000000       21.38500            \\nREMARK 290   SMTRY3   3  0.000000  0.000000 -1.000000       14.47500            \\nREMARK 290   SMTRY1   4  1.000000  0.000000  0.000000       25.42000            \\nREMARK 290   SMTRY2   4  0.000000 -1.000000  0.000000       21.38500            \\nREMARK 290   SMTRY3   4  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290                                                                      \\nREMARK 290 REMARK: NULL                                                         \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1                                                       \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: MONOMERIC                         \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A                                     \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: CLOSE CONTACTS                                             \\nREMARK 500                                                                      \\nREMARK 500 THE FOLLOWING ATOMS THAT ARE RELATED BY CRYSTALLOGRAPHIC             \\nREMARK 500 SYMMETRY ARE IN CLOSE CONTACT.  AN ATOM LOCATED WITHIN 0.15          \\nREMARK 500 ANGSTROMS OF A SYMMETRY RELATED ATOM IS ASSUMED TO BE ON A           \\nREMARK 500 SPECIAL POSITION AND IS, THEREFORE, LISTED IN REMARK 375             \\nREMARK 500 INSTEAD OF REMARK 500.  ATOMS WITH NON-BLANK ALTERNATE               \\nREMARK 500 LOCATION INDICATORS ARE NOT INCLUDED IN THE CALCULATIONS.            \\nREMARK 500                                                                      \\nREMARK 500 DISTANCE CUTOFF:                                                     \\nREMARK 500 2.2 ANGSTROMS FOR CONTACTS NOT INVOLVING HYDROGEN ATOMS              \\nREMARK 500 1.6 ANGSTROMS FOR CONTACTS INVOLVING HYDROGEN ATOMS                  \\nREMARK 500                                                                      \\nREMARK 500  ATM1  RES C  SSEQI   ATM2  RES C  SSEQI  SSYMOP   DISTANCE          \\nREMARK 500   OE2  GLU A    16     NH1  ARG A    72     1554     2.02            \\nREMARK 500   NZ   LYS A    48     OXT  GLY A    76     4467     2.16            \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: COVALENT BOND ANGLES                                       \\nREMARK 500                                                                      \\nREMARK 500 THE STEREOCHEMICAL PARAMETERS OF THE FOLLOWING RESIDUES              \\nREMARK 500 HAVE VALUES WHICH DEVIATE FROM EXPECTED VALUES BY MORE               \\nREMARK 500 THAN 6*RMSD (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN               \\nREMARK 500 IDENTIFIER; SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                 \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT: (10X,I3,1X,A3,1X,A1,I4,A1,3(1X,A4,2X),12X,F5.1)              \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES PROTEIN: ENGH AND HUBER, 1999                        \\nREMARK 500 EXPECTED VALUES NUCLEIC ACID: CLOWNEY ET AL 1996                     \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI ATM1   ATM2   ATM3                                     \\nREMARK 500    LEU A  15   CA  -  CB  -  CG  ANGL. DEV. =  14.0 DEGREES          \\nREMARK 500    ARG A  54   CD  -  NE  -  CZ  ANGL. DEV. =  12.4 DEGREES          \\nREMARK 500    ARG A  54   NE  -  CZ  -  NH1 ANGL. DEV. =   5.5 DEGREES          \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nDBREF  1UBQ A    1    76  UNP    P62988   UBIQ_HUMAN       1     76             \\nSEQRES   1 A   76  MET GLN ILE PHE VAL LYS THR LEU THR GLY LYS THR ILE          \\nSEQRES   2 A   76  THR LEU GLU VAL GLU PRO SER ASP THR ILE GLU ASN VAL          \\nSEQRES   3 A   76  LYS ALA LYS ILE GLN ASP LYS GLU GLY ILE PRO PRO ASP          \\nSEQRES   4 A   76  GLN GLN ARG LEU ILE PHE ALA GLY LYS GLN LEU GLU ASP          \\nSEQRES   5 A   76  GLY ARG THR LEU SER ASP TYR ASN ILE GLN LYS GLU SER          \\nSEQRES   6 A   76  THR LEU HIS LEU VAL LEU ARG LEU ARG GLY GLY                  \\nFORMUL   2  HOH   *58(H2 O)                                                     \\nHELIX    1  H1 ILE A   23  GLU A   34  1                                  12    \\nHELIX    2  H2 LEU A   56  TYR A   59  5                                   4    \\nSHEET    1 BET 5 GLY A  10  VAL A  17  0                                        \\nSHEET    2 BET 5 MET A   1  THR A   7 -1                                        \\nSHEET    3 BET 5 GLU A  64  ARG A  72  1                                        \\nSHEET    4 BET 5 GLN A  40  PHE A  45 -1                                        \\nSHEET    5 BET 5 LYS A  48  LEU A  50 -1                                        \\nCRYST1   50.840   42.770   28.950  90.00  90.00  90.00 P 21 21 21    4          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      0.019670  0.000000  0.000000        0.00000                         \\nSCALE2      0.000000  0.023381  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  0.034542        0.00000                         \\nATOM      1  N   MET A   1      27.340  24.430   2.614  1.00  9.67           N  \\nATOM      2  CA  MET A   1      26.266  25.413   2.842  1.00 10.38           C  \\nATOM      3  C   MET A   1      26.913  26.639   3.531  1.00  9.62           C  \\nATOM      4  O   MET A   1      27.886  26.463   4.263  1.00  9.62           O  \\nATOM      5  CB  MET A   1      25.112  24.880   3.649  1.00 13.77           C  \\nATOM      6  CG  MET A   1      25.353  24.860   5.134  1.00 16.29           C  \\nATOM      7  SD  MET A   1      23.930  23.959   5.904  1.00 17.17           S  \\nATOM      8  CE  MET A   1      24.447  23.984   7.620  1.00 16.11           C  \\nATOM      9  N   GLN A   2      26.335  27.770   3.258  1.00  9.27           N  \\nATOM     10  CA  GLN A   2      26.850  29.021   3.898  1.00  9.07           C  \\nATOM     11  C   GLN A   2      26.100  29.253   5.202  1.00  8.72           C  \\nATOM     12  O   GLN A   2      24.865  29.024   5.330  1.00  8.22           O  \\nATOM     13  CB  GLN A   2      26.733  30.148   2.905  1.00 14.46           C  \\nATOM     14  CG  GLN A   2      26.882  31.546   3.409  1.00 17.01           C  \\nATOM     15  CD  GLN A   2      26.786  32.562   2.270  1.00 20.10           C  \\nATOM     16  OE1 GLN A   2      27.783  33.160   1.870  1.00 21.89           O  \\nATOM     17  NE2 GLN A   2      25.562  32.733   1.806  1.00 19.49           N  \\nATOM     18  N   ILE A   3      26.849  29.656   6.217  1.00  5.87           N  \\nATOM     19  CA  ILE A   3      26.235  30.058   7.497  1.00  5.07           C  \\nATOM     20  C   ILE A   3      26.882  31.428   7.862  1.00  4.01           C  \\nATOM     21  O   ILE A   3      27.906  31.711   7.264  1.00  4.61           O  \\nATOM     22  CB  ILE A   3      26.344  29.050   8.645  1.00  6.55           C  \\nATOM     23  CG1 ILE A   3      27.810  28.748   8.999  1.00  4.72           C  \\nATOM     24  CG2 ILE A   3      25.491  27.771   8.287  1.00  5.58           C  \\nATOM     25  CD1 ILE A   3      27.967  28.087  10.417  1.00 10.83           C  \\nATOM     26  N   PHE A   4      26.214  32.097   8.771  1.00  4.55           N  \\nATOM     27  CA  PHE A   4      26.772  33.436   9.197  1.00  4.68           C  \\nATOM     28  C   PHE A   4      27.151  33.362  10.650  1.00  5.30           C  \\nATOM     29  O   PHE A   4      26.350  32.778  11.395  1.00  5.58           O  \\nATOM     30  CB  PHE A   4      25.695  34.498   8.946  1.00  4.83           C  \\nATOM     31  CG  PHE A   4      25.288  34.609   7.499  1.00  7.97           C  \\nATOM     32  CD1 PHE A   4      24.147  33.966   7.038  1.00  6.69           C  \\nATOM     33  CD2 PHE A   4      26.136  35.346   6.640  1.00  8.34           C  \\nATOM     34  CE1 PHE A   4      23.812  34.031   5.677  1.00  9.10           C  \\nATOM     35  CE2 PHE A   4      25.810  35.392   5.267  1.00 10.61           C  \\nATOM     36  CZ  PHE A   4      24.620  34.778   4.853  1.00  8.90           C  \\nATOM     37  N   VAL A   5      28.260  33.943  11.096  1.00  4.44           N  \\nATOM     38  CA  VAL A   5      28.605  33.965  12.503  1.00  3.87           C  \\nATOM     39  C   VAL A   5      28.638  35.461  12.900  1.00  4.93           C  \\nATOM     40  O   VAL A   5      29.522  36.103  12.320  1.00  6.84           O  \\nATOM     41  CB  VAL A   5      29.963  33.317  12.814  1.00  2.99           C  \\nATOM     42  CG1 VAL A   5      30.211  33.394  14.304  1.00  5.28           C  \\nATOM     43  CG2 VAL A   5      29.957  31.838  12.352  1.00  9.13           C  \\nATOM     44  N   LYS A   6      27.751  35.867  13.740  1.00  6.04           N  \\nATOM     45  CA  LYS A   6      27.691  37.315  14.143  1.00  6.12           C  \\nATOM     46  C   LYS A   6      28.469  37.475  15.420  1.00  6.57           C  \\nATOM     47  O   LYS A   6      28.213  36.753  16.411  1.00  5.76           O  \\nATOM     48  CB  LYS A   6      26.219  37.684  14.307  1.00  7.45           C  \\nATOM     49  CG  LYS A   6      25.884  39.139  14.615  1.00 11.12           C  \\nATOM     50  CD  LYS A   6      24.348  39.296  14.642  1.00 14.54           C  \\nATOM     51  CE  LYS A   6      23.865  40.723  14.749  1.00 18.84           C  \\nATOM     52  NZ  LYS A   6      22.375  40.720  14.907  1.00 20.55           N  \\nATOM     53  N   THR A   7      29.426  38.430  15.446  1.00  7.41           N  \\nATOM     54  CA  THR A   7      30.225  38.643  16.662  1.00  7.48           C  \\nATOM     55  C   THR A   7      29.664  39.839  17.434  1.00  8.75           C  \\nATOM     56  O   THR A   7      28.850  40.565  16.859  1.00  8.58           O  \\nATOM     57  CB  THR A   7      31.744  38.879  16.299  1.00  9.61           C  \\nATOM     58  OG1 THR A   7      31.737  40.257  15.824  1.00 11.78           O  \\nATOM     59  CG2 THR A   7      32.260  37.969  15.171  1.00  9.17           C  \\nATOM     60  N   LEU A   8      30.132  40.069  18.642  1.00  9.84           N  \\nATOM     61  CA  LEU A   8      29.607  41.180  19.467  1.00 14.15           C  \\nATOM     62  C   LEU A   8      30.075  42.538  18.984  1.00 17.37           C  \\nATOM     63  O   LEU A   8      29.586  43.570  19.483  1.00 17.01           O  \\nATOM     64  CB  LEU A   8      29.919  40.890  20.938  1.00 16.63           C  \\nATOM     65  CG  LEU A   8      29.183  39.722  21.581  1.00 18.88           C  \\nATOM     66  CD1 LEU A   8      29.308  39.750  23.095  1.00 19.31           C  \\nATOM     67  CD2 LEU A   8      27.700  39.721  21.228  1.00 18.59           C  \\nATOM     68  N   THR A   9      30.991  42.571  17.998  1.00 18.33           N  \\nATOM     69  CA  THR A   9      31.422  43.940  17.553  1.00 19.24           C  \\nATOM     70  C   THR A   9      30.755  44.351  16.277  1.00 19.48           C  \\nATOM     71  O   THR A   9      31.207  45.268  15.566  1.00 23.14           O  \\nATOM     72  CB  THR A   9      32.979  43.918  17.445  1.00 18.97           C  \\nATOM     73  OG1 THR A   9      33.174  43.067  16.265  1.00 20.24           O  \\nATOM     74  CG2 THR A   9      33.657  43.319  18.672  1.00 19.70           C  \\nATOM     75  N   GLY A  10      29.721  43.673  15.885  1.00 19.43           N  \\nATOM     76  CA  GLY A  10      28.978  43.960  14.678  1.00 18.74           C  \\nATOM     77  C   GLY A  10      29.604  43.507  13.393  1.00 17.62           C  \\nATOM     78  O   GLY A  10      29.219  43.981  12.301  1.00 19.74           O  \\nATOM     79  N   LYS A  11      30.563  42.623  13.495  1.00 13.56           N  \\nATOM     80  CA  LYS A  11      31.191  42.012  12.331  1.00 11.91           C  \\nATOM     81  C   LYS A  11      30.459  40.666  12.130  1.00 10.18           C  \\nATOM     82  O   LYS A  11      30.253  39.991  13.133  1.00  9.10           O  \\nATOM     83  CB  LYS A  11      32.672  41.717  12.505  1.00 13.43           C  \\nATOM     84  CG  LYS A  11      33.280  41.086  11.227  1.00 16.69           C  \\nATOM     85  CD  LYS A  11      34.762  40.799  11.470  1.00 17.92           C  \\nATOM     86  CE  LYS A  11      35.614  40.847  10.240  1.00 20.81           C  \\nATOM     87  NZ  LYS A  11      35.100  40.073   9.101  1.00 21.93           N  \\nATOM     88  N   THR A  12      30.163  40.338  10.886  1.00  9.63           N  \\nATOM     89  CA  THR A  12      29.542  39.020  10.653  1.00  9.85           C  \\nATOM     90  C   THR A  12      30.494  38.261   9.729  1.00 11.66           C  \\nATOM     91  O   THR A  12      30.849  38.850   8.706  1.00 12.33           O  \\nATOM     92  CB  THR A  12      28.113  39.049  10.015  1.00 10.85           C  \\nATOM     93  OG1 THR A  12      27.280  39.722  10.996  1.00 10.91           O  \\nATOM     94  CG2 THR A  12      27.588  37.635   9.715  1.00  9.63           C  \\nATOM     95  N   ILE A  13      30.795  37.015  10.095  1.00 10.42           N  \\nATOM     96  CA  ILE A  13      31.720  36.289   9.176  1.00 11.84           C  \\nATOM     97  C   ILE A  13      30.955  35.211   8.459  1.00 10.55           C  \\nATOM     98  O   ILE A  13      30.025  34.618   9.040  1.00 11.92           O  \\nATOM     99  CB  ILE A  13      32.995  35.883   9.934  1.00 14.86           C  \\nATOM    100  CG1 ILE A  13      33.306  34.381   9.840  1.00 14.87           C  \\nATOM    101  CG2 ILE A  13      33.109  36.381  11.435  1.00 17.08           C  \\nATOM    102  CD1 ILE A  13      34.535  34.028  10.720  1.00 16.46           C  \\nATOM    103  N   THR A  14      31.244  34.986   7.197  1.00  9.39           N  \\nATOM    104  CA  THR A  14      30.505  33.884   6.512  1.00  9.63           C  \\nATOM    105  C   THR A  14      31.409  32.680   6.446  1.00 11.20           C  \\nATOM    106  O   THR A  14      32.619  32.812   6.125  1.00 11.63           O  \\nATOM    107  CB  THR A  14      30.091  34.393   5.078  1.00 10.38           C  \\nATOM    108  OG1 THR A  14      31.440  34.513   4.487  1.00 16.30           O  \\nATOM    109  CG2 THR A  14      29.420  35.756   5.119  1.00 11.66           C  \\nATOM    110  N   LEU A  15      30.884  31.485   6.666  1.00  8.29           N  \\nATOM    111  CA  LEU A  15      31.677  30.275   6.639  1.00  9.03           C  \\nATOM    112  C   LEU A  15      31.022  29.288   5.665  1.00  8.59           C  \\nATOM    113  O   LEU A  15      29.809  29.395   5.545  1.00  7.79           O  \\nATOM    114  CB  LEU A  15      31.562  29.686   8.045  1.00 11.08           C  \\nATOM    115  CG  LEU A  15      32.631  29.444   9.060  1.00 15.79           C  \\nATOM    116  CD1 LEU A  15      33.814  30.390   9.030  1.00 15.88           C  \\nATOM    117  CD2 LEU A  15      31.945  29.449  10.436  1.00 15.27           C  \\nATOM    118  N   GLU A  16      31.834  28.412   5.125  1.00 11.04           N  \\nATOM    119  CA  GLU A  16      31.220  27.341   4.275  1.00 11.50           C  \\nATOM    120  C   GLU A  16      31.440  26.079   5.080  1.00 10.13           C  \\nATOM    121  O   GLU A  16      32.576  25.802   5.461  1.00  9.83           O  \\nATOM    122  CB  GLU A  16      31.827  27.262   2.894  1.00 17.22           C  \\nATOM    123  CG  GLU A  16      31.363  28.410   1.962  1.00 23.33           C  \\nATOM    124  CD  GLU A  16      31.671  28.291   0.498  1.00 26.99           C  \\nATOM    125  OE1 GLU A  16      30.869  28.621  -0.366  1.00 28.86           O  \\nATOM    126  OE2 GLU A  16      32.835  27.861   0.278  1.00 28.90           O  \\nATOM    127  N   VAL A  17      30.310  25.458   5.384  1.00  8.99           N  \\nATOM    128  CA  VAL A  17      30.288  24.245   6.193  1.00  8.85           C  \\nATOM    129  C   VAL A  17      29.279  23.227   5.641  1.00  8.04           C  \\nATOM    130  O   VAL A  17      28.478  23.522   4.725  1.00  8.99           O  \\nATOM    131  CB  VAL A  17      29.903  24.590   7.665  1.00  9.78           C  \\nATOM    132  CG1 VAL A  17      30.862  25.496   8.389  1.00 12.05           C  \\nATOM    133  CG2 VAL A  17      28.476  25.135   7.705  1.00 10.54           C  \\nATOM    134  N   GLU A  18      29.380  22.057   6.232  1.00  7.29           N  \\nATOM    135  CA  GLU A  18      28.468  20.940   5.980  1.00  7.08           C  \\nATOM    136  C   GLU A  18      27.819  20.609   7.316  1.00  6.45           C  \\nATOM    137  O   GLU A  18      28.449  20.674   8.360  1.00  5.28           O  \\nATOM    138  CB  GLU A  18      29.213  19.697   5.506  1.00 10.28           C  \\nATOM    139  CG  GLU A  18      29.728  19.755   4.060  1.00 12.65           C  \\nATOM    140  CD  GLU A  18      28.754  20.061   2.978  1.00 14.15           C  \\nATOM    141  OE1 GLU A  18      27.546  19.992   2.985  1.00 14.33           O  \\nATOM    142  OE2 GLU A  18      29.336  20.423   1.904  1.00 18.17           O  \\nATOM    143  N   PRO A  19      26.559  20.220   7.288  1.00  7.24           N  \\nATOM    144  CA  PRO A  19      25.829  19.825   8.494  1.00  7.07           C  \\nATOM    145  C   PRO A  19      26.541  18.732   9.251  1.00  6.65           C  \\nATOM    146  O   PRO A  19      26.333  18.536  10.457  1.00  6.37           O  \\nATOM    147  CB  PRO A  19      24.469  19.332   7.952  1.00  7.61           C  \\nATOM    148  CG  PRO A  19      24.299  20.134   6.704  1.00  8.16           C  \\nATOM    149  CD  PRO A  19      25.714  20.108   6.073  1.00  7.49           C  \\nATOM    150  N   SER A  20      27.361  17.959   8.559  1.00  6.80           N  \\nATOM    151  CA  SER A  20      28.054  16.835   9.210  1.00  6.28           C  \\nATOM    152  C   SER A  20      29.258  17.318   9.984  1.00  8.45           C  \\nATOM    153  O   SER A  20      29.930  16.477  10.606  1.00  7.26           O  \\nATOM    154  CB  SER A  20      28.523  15.820   8.182  1.00  8.57           C  \\nATOM    155  OG  SER A  20      28.946  16.445   6.967  1.00 11.13           O  \\nATOM    156  N   ASP A  21      29.599  18.599   9.828  1.00  7.50           N  \\nATOM    157  CA  ASP A  21      30.796  19.083  10.566  1.00  7.70           C  \\nATOM    158  C   ASP A  21      30.491  19.162  12.040  1.00  7.08           C  \\nATOM    159  O   ASP A  21      29.367  19.523  12.441  1.00  8.11           O  \\nATOM    160  CB  ASP A  21      31.155  20.515  10.048  1.00 11.00           C  \\nATOM    161  CG  ASP A  21      31.923  20.436   8.755  1.00 15.32           C  \\nATOM    162  OD1 ASP A  21      32.493  19.374   8.456  1.00 18.03           O  \\nATOM    163  OD2 ASP A  21      31.838  21.402   7.968  1.00 14.36           O  \\nATOM    164  N   THR A  22      31.510  18.936  12.852  1.00  5.37           N  \\nATOM    165  CA  THR A  22      31.398  19.064  14.286  1.00  6.01           C  \\nATOM    166  C   THR A  22      31.593  20.553  14.655  1.00  8.01           C  \\nATOM    167  O   THR A  22      32.159  21.311  13.861  1.00  8.11           O  \\nATOM    168  CB  THR A  22      32.492  18.193  14.995  1.00  8.92           C  \\nATOM    169  OG1 THR A  22      33.778  18.739  14.516  1.00 10.22           O  \\nATOM    170  CG2 THR A  22      32.352  16.700  14.630  1.00  9.65           C  \\nATOM    171  N   ILE A  23      31.113  20.863  15.860  1.00  8.32           N  \\nATOM    172  CA  ILE A  23      31.288  22.201  16.417  1.00  9.92           C  \\nATOM    173  C   ILE A  23      32.776  22.519  16.577  1.00 10.01           C  \\nATOM    174  O   ILE A  23      33.233  23.659  16.384  1.00  8.71           O  \\nATOM    175  CB  ILE A  23      30.520  22.300  17.764  1.00 10.78           C  \\nATOM    176  CG1 ILE A  23      29.006  22.043  17.442  1.00 11.38           C  \\nATOM    177  CG2 ILE A  23      30.832  23.699  18.358  1.00 10.90           C  \\nATOM    178  CD1 ILE A  23      28.407  22.948  16.366  1.00 12.30           C  \\nATOM    179  N   GLU A  24      33.548  21.526  16.950  1.00  9.54           N  \\nATOM    180  CA  GLU A  24      35.031  21.722  17.069  1.00 11.81           C  \\nATOM    181  C   GLU A  24      35.615  22.190  15.759  1.00 11.14           C  \\nATOM    182  O   GLU A  24      36.532  23.046  15.724  1.00 10.62           O  \\nATOM    183  CB  GLU A  24      35.667  20.383  17.447  1.00 19.24           C  \\nATOM    184  CG  GLU A  24      37.128  20.293  17.872  1.00 27.76           C  \\nATOM    185  CD  GLU A  24      37.561  18.851  18.082  1.00 32.92           C  \\nATOM    186  OE1 GLU A  24      37.758  18.024  17.195  1.00 34.80           O  \\nATOM    187  OE2 GLU A  24      37.628  18.599  19.313  1.00 36.51           O  \\nATOM    188  N   ASN A  25      35.139  21.624  14.662  1.00  9.43           N  \\nATOM    189  CA  ASN A  25      35.590  21.945  13.302  1.00 10.96           C  \\nATOM    190  C   ASN A  25      35.238  23.382  12.920  1.00  9.68           C  \\nATOM    191  O   ASN A  25      36.066  24.109  12.333  1.00  9.33           O  \\nATOM    192  CB  ASN A  25      35.064  20.957  12.255  1.00 16.78           C  \\nATOM    193  CG  ASN A  25      35.541  21.418  10.871  1.00 22.31           C  \\nATOM    194  OD1 ASN A  25      36.772  21.623  10.676  1.00 25.66           O  \\nATOM    195  ND2 ASN A  25      34.628  21.595   9.920  1.00 24.70           N  \\nATOM    196  N   VAL A  26      34.007  23.745  13.250  1.00  6.52           N  \\nATOM    197  CA  VAL A  26      33.533  25.097  12.978  1.00  5.53           C  \\nATOM    198  C   VAL A  26      34.441  26.099  13.684  1.00  4.42           C  \\nATOM    199  O   VAL A  26      34.883  27.090  13.093  1.00  3.40           O  \\nATOM    200  CB  VAL A  26      32.060  25.257  13.364  1.00  3.86           C  \\nATOM    201  CG1 VAL A  26      31.684  26.749  13.342  1.00  7.25           C  \\nATOM    202  CG2 VAL A  26      31.152  24.421  12.477  1.00  8.12           C  \\nATOM    203  N   LYS A  27      34.734  25.822  14.949  1.00  2.64           N  \\nATOM    204  CA  LYS A  27      35.596  26.715  15.736  1.00  4.14           C  \\nATOM    205  C   LYS A  27      36.975  26.826  15.107  1.00  5.58           C  \\nATOM    206  O   LYS A  27      37.579  27.926  15.159  1.00  4.11           O  \\nATOM    207  CB  LYS A  27      35.715  26.203  17.172  1.00  3.97           C  \\nATOM    208  CG  LYS A  27      34.343  26.445  17.898  1.00  7.45           C  \\nATOM    209  CD  LYS A  27      34.509  26.077  19.360  1.00  9.02           C  \\nATOM    210  CE  LYS A  27      33.206  26.311  20.122  1.00 12.90           C  \\nATOM    211  NZ  LYS A  27      33.455  25.910  21.546  1.00 15.47           N  \\nATOM    212  N   ALA A  28      37.499  25.743  14.571  1.00  6.61           N  \\nATOM    213  CA  ALA A  28      38.794  25.761  13.880  1.00  7.74           C  \\nATOM    214  C   ALA A  28      38.728  26.591  12.611  1.00  9.17           C  \\nATOM    215  O   ALA A  28      39.704  27.346  12.277  1.00 11.45           O  \\nATOM    216  CB  ALA A  28      39.285  24.336  13.566  1.00  7.68           C  \\nATOM    217  N   LYS A  29      37.633  26.543  11.867  1.00  8.96           N  \\nATOM    218  CA  LYS A  29      37.471  27.391  10.668  1.00  7.90           C  \\nATOM    219  C   LYS A  29      37.441  28.882  11.052  1.00  6.92           C  \\nATOM    220  O   LYS A  29      38.020  29.772  10.382  1.00  6.87           O  \\nATOM    221  CB  LYS A  29      36.193  27.058   9.911  1.00 10.28           C  \\nATOM    222  CG  LYS A  29      36.153  25.620   9.409  1.00 14.94           C  \\nATOM    223  CD  LYS A  29      34.758  25.280   8.900  1.00 19.69           C  \\nATOM    224  CE  LYS A  29      34.793  24.264   7.767  1.00 22.63           C  \\nATOM    225  NZ  LYS A  29      34.914  24.944   6.441  1.00 24.98           N  \\nATOM    226  N   ILE A  30      36.811  29.170  12.192  1.00  4.57           N  \\nATOM    227  CA  ILE A  30      36.731  30.570  12.645  1.00  5.58           C  \\nATOM    228  C   ILE A  30      38.148  30.981  13.069  1.00  7.26           C  \\nATOM    229  O   ILE A  30      38.544  32.150  12.856  1.00  9.46           O  \\nATOM    230  CB  ILE A  30      35.708  30.776  13.806  1.00  5.36           C  \\nATOM    231  CG1 ILE A  30      34.228  30.630  13.319  1.00  2.94           C  \\nATOM    232  CG2 ILE A  30      35.874  32.138  14.512  1.00  2.78           C  \\nATOM    233  CD1 ILE A  30      33.284  30.504  14.552  1.00  2.00           C  \\nATOM    234  N   GLN A  31      38.883  30.110  13.713  1.00  7.06           N  \\nATOM    235  CA  GLN A  31      40.269  30.508  14.115  1.00  8.67           C  \\nATOM    236  C   GLN A  31      41.092  30.808  12.851  1.00 10.90           C  \\nATOM    237  O   GLN A  31      41.828  31.808  12.681  1.00  9.63           O  \\nATOM    238  CB  GLN A  31      40.996  29.399  14.865  1.00  9.12           C  \\nATOM    239  CG  GLN A  31      42.445  29.848  15.182  1.00 10.76           C  \\nATOM    240  CD  GLN A  31      43.090  28.828  16.095  1.00 13.78           C  \\nATOM    241  OE1 GLN A  31      42.770  27.655  15.906  1.00 14.48           O  \\nATOM    242  NE2 GLN A  31      43.898  29.252  17.050  1.00 14.76           N  \\nATOM    243  N   ASP A  32      41.001  29.878  11.931  1.00 10.93           N  \\nATOM    244  CA  ASP A  32      41.718  30.022  10.643  1.00 14.01           C  \\nATOM    245  C   ASP A  32      41.399  31.338   9.967  1.00 14.04           C  \\nATOM    246  O   ASP A  32      42.260  32.036   9.381  1.00 13.39           O  \\nATOM    247  CB  ASP A  32      41.398  28.780   9.810  1.00 18.01           C  \\nATOM    248  CG  ASP A  32      42.626  28.557   8.928  1.00 24.33           C  \\nATOM    249  OD1 ASP A  32      43.666  28.262   9.539  1.00 26.29           O  \\nATOM    250  OD2 ASP A  32      42.430  28.812   7.728  1.00 25.17           O  \\nATOM    251  N   LYS A  33      40.117  31.750   9.988  1.00 14.22           N  \\nATOM    252  CA  LYS A  33      39.808  32.994   9.233  1.00 14.00           C  \\nATOM    253  C   LYS A  33      39.837  34.271   9.995  1.00 12.37           C  \\nATOM    254  O   LYS A  33      40.164  35.323   9.345  1.00 12.17           O  \\nATOM    255  CB  LYS A  33      38.615  32.801   8.320  1.00 18.62           C  \\nATOM    256  CG  LYS A  33      37.220  32.822   8.827  1.00 24.00           C  \\nATOM    257  CD  LYS A  33      36.351  33.613   7.838  1.00 27.61           C  \\nATOM    258  CE  LYS A  33      36.322  32.944   6.477  1.00 27.64           C  \\nATOM    259  NZ  LYS A  33      35.768  33.945   5.489  1.00 30.06           N  \\nATOM    260  N   GLU A  34      39.655  34.335  11.285  1.00 10.11           N  \\nATOM    261  CA  GLU A  34      39.676  35.547  12.072  1.00 10.07           C  \\nATOM    262  C   GLU A  34      40.675  35.527  13.200  1.00  9.32           C  \\nATOM    263  O   GLU A  34      40.814  36.528  13.911  1.00 11.61           O  \\nATOM    264  CB  GLU A  34      38.290  35.814  12.698  1.00 14.77           C  \\nATOM    265  CG  GLU A  34      37.156  35.985  11.688  1.00 18.75           C  \\nATOM    266  CD  GLU A  34      37.192  37.361  11.033  1.00 22.28           C  \\nATOM    267  OE1 GLU A  34      37.519  38.360  11.645  1.00 21.95           O  \\nATOM    268  OE2 GLU A  34      36.861  37.320   9.822  1.00 25.19           O  \\nATOM    269  N   GLY A  35      41.317  34.393  13.432  1.00  7.22           N  \\nATOM    270  CA  GLY A  35      42.345  34.269  14.431  1.00  6.29           C  \\nATOM    271  C   GLY A  35      41.949  34.076  15.842  1.00  6.93           C  \\nATOM    272  O   GLY A  35      42.829  34.000  16.739  1.00  7.41           O  \\nATOM    273  N   ILE A  36      40.642  33.916  16.112  1.00  5.86           N  \\nATOM    274  CA  ILE A  36      40.226  33.716  17.509  1.00  6.07           C  \\nATOM    275  C   ILE A  36      40.449  32.278  17.945  1.00  6.36           C  \\nATOM    276  O   ILE A  36      39.936  31.336  17.315  1.00  6.18           O  \\nATOM    277  CB  ILE A  36      38.693  34.106  17.595  1.00  7.47           C  \\nATOM    278  CG1 ILE A  36      38.471  35.546  17.045  1.00  8.52           C  \\nATOM    279  CG2 ILE A  36      38.146  33.932  19.027  1.00  7.36           C  \\nATOM    280  CD1 ILE A  36      36.958  35.746  16.680  1.00  9.49           C  \\nATOM    281  N   PRO A  37      41.189  32.085  19.031  1.00  8.65           N  \\nATOM    282  CA  PRO A  37      41.461  30.751  19.594  1.00  9.18           C  \\nATOM    283  C   PRO A  37      40.168  30.026  19.918  1.00  9.85           C  \\nATOM    284  O   PRO A  37      39.264  30.662  20.521  1.00  8.51           O  \\nATOM    285  CB  PRO A  37      42.195  31.142  20.913  1.00 11.42           C  \\nATOM    286  CG  PRO A  37      42.904  32.414  20.553  1.00  9.27           C  \\nATOM    287  CD  PRO A  37      41.822  33.188  19.813  1.00  8.33           C  \\nATOM    288  N   PRO A  38      40.059  28.758  19.607  1.00  8.71           N  \\nATOM    289  CA  PRO A  38      38.817  28.020  19.889  1.00  9.08           C  \\nATOM    290  C   PRO A  38      38.421  28.048  21.341  1.00  9.28           C  \\nATOM    291  O   PRO A  38      37.213  28.036  21.704  1.00  6.50           O  \\nATOM    292  CB  PRO A  38      39.090  26.629  19.325  1.00 10.31           C  \\nATOM    293  CG  PRO A  38      40.082  26.904  18.198  1.00 10.81           C  \\nATOM    294  CD  PRO A  38      41.035  27.909  18.879  1.00 12.00           C  \\nATOM    295  N   ASP A  39      39.374  28.090  22.240  1.00 11.20           N  \\nATOM    296  CA  ASP A  39      39.063  28.063  23.695  1.00 14.96           C  \\nATOM    297  C   ASP A  39      38.365  29.335  24.159  1.00 13.99           C  \\nATOM    298  O   ASP A  39      37.684  29.390  25.221  1.00 13.75           O  \\nATOM    299  CB  ASP A  39      40.340  27.692  24.468  1.00 24.16           C  \\nATOM    300  CG  ASP A  39      40.559  28.585  25.675  1.00 31.06           C  \\nATOM    301  OD1 ASP A  39      40.716  29.809  25.456  1.00 35.55           O  \\nATOM    302  OD2 ASP A  39      40.549  28.090  26.840  1.00 34.22           O  \\nATOM    303  N   GLN A  40      38.419  30.373  23.341  1.00 11.60           N  \\nATOM    304  CA  GLN A  40      37.738  31.637  23.712  1.00 10.76           C  \\nATOM    305  C   GLN A  40      36.334  31.742  23.087  1.00  8.01           C  \\nATOM    306  O   GLN A  40      35.574  32.618  23.483  1.00  8.96           O  \\nATOM    307  CB  GLN A  40      38.528  32.854  23.182  1.00 11.14           C  \\nATOM    308  CG  GLN A  40      39.919  32.854  23.840  1.00 14.85           C  \\nATOM    309  CD  GLN A  40      40.760  34.036  23.394  1.00 16.11           C  \\nATOM    310  OE1 GLN A  40      41.975  34.008  23.624  1.00 20.52           O  \\nATOM    311  NE2 GLN A  40      40.140  35.007  22.775  1.00 18.16           N  \\nATOM    312  N   GLN A  41      36.000  30.860  22.172  1.00  6.52           N  \\nATOM    313  CA  GLN A  41      34.738  30.875  21.473  1.00  3.87           C  \\nATOM    314  C   GLN A  41      33.589  30.189  22.181  1.00  4.79           C  \\nATOM    315  O   GLN A  41      33.580  29.009  22.499  1.00  6.34           O  \\nATOM    316  CB  GLN A  41      34.876  30.237  20.066  1.00  4.20           C  \\nATOM    317  CG  GLN A  41      36.012  30.860  19.221  1.00  3.20           C  \\nATOM    318  CD  GLN A  41      36.083  30.194  17.875  1.00  4.89           C  \\nATOM    319  OE1 GLN A  41      35.048  29.702  17.393  1.00  5.21           O  \\nATOM    320  NE2 GLN A  41      37.228  30.126  17.233  1.00  7.13           N  \\nATOM    321  N   ARG A  42      32.478  30.917  22.269  1.00  5.73           N  \\nATOM    322  CA  ARG A  42      31.200  30.329  22.780  1.00  6.97           C  \\nATOM    323  C   ARG A  42      30.210  30.509  21.650  1.00  7.15           C  \\nATOM    324  O   ARG A  42      29.978  31.726  21.269  1.00  7.33           O  \\nATOM    325  CB  ARG A  42      30.847  30.931  24.118  1.00 13.23           C  \\nATOM    326  CG  ARG A  42      29.412  30.796  24.598  1.00 21.27           C  \\nATOM    327  CD  ARG A  42      29.271  31.314  26.016  1.00 26.14           C  \\nATOM    328  NE  ARG A  42      27.875  31.317  26.443  1.00 32.26           N  \\nATOM    329  CZ  ARG A  42      27.132  32.423  26.574  1.00 34.32           C  \\nATOM    330  NH1 ARG A  42      27.630  33.656  26.461  1.00 35.30           N  \\nATOM    331  NH2 ARG A  42      25.810  32.299  26.732  1.00 36.39           N  \\nATOM    332  N   LEU A  43      29.694  29.436  21.054  1.00  4.65           N  \\nATOM    333  CA  LEU A  43      28.762  29.573  19.906  1.00  3.51           C  \\nATOM    334  C   LEU A  43      27.331  29.317  20.364  1.00  5.56           C  \\nATOM    335  O   LEU A  43      27.101  28.346  21.097  1.00  4.19           O  \\nATOM    336  CB  LEU A  43      29.151  28.655  18.755  1.00  3.74           C  \\nATOM    337  CG  LEU A  43      30.416  28.912  17.980  1.00  6.32           C  \\nATOM    338  CD1 LEU A  43      30.738  27.693  17.122  1.00  9.55           C  \\nATOM    339  CD2 LEU A  43      30.205  30.168  17.129  1.00  6.41           C  \\nATOM    340  N   ILE A  44      26.436  30.232  20.004  1.00  4.58           N  \\nATOM    341  CA  ILE A  44      25.034  30.170  20.401  1.00  5.55           C  \\nATOM    342  C   ILE A  44      24.101  30.149  19.196  1.00  5.46           C  \\nATOM    343  O   ILE A  44      24.196  30.948  18.287  1.00  6.04           O  \\nATOM    344  CB  ILE A  44      24.639  31.426  21.286  1.00  6.80           C  \\nATOM    345  CG1 ILE A  44      25.646  31.670  22.421  1.00 10.31           C  \\nATOM    346  CG2 ILE A  44      23.181  31.309  21.824  1.00  7.39           C  \\nATOM    347  CD1 ILE A  44      25.778  30.436  23.356  1.00 13.90           C  \\nATOM    348  N   PHE A  45      23.141  29.187  19.241  1.00  6.75           N  \\nATOM    349  CA  PHE A  45      22.126  29.062  18.183  1.00  4.70           C  \\nATOM    350  C   PHE A  45      20.835  28.629  18.904  1.00  6.34           C  \\nATOM    351  O   PHE A  45      20.821  27.734  19.749  1.00  5.45           O  \\nATOM    352  CB  PHE A  45      22.494  28.057  17.109  1.00  5.51           C  \\nATOM    353  CG  PHE A  45      21.447  27.869  16.026  1.00  5.98           C  \\nATOM    354  CD1 PHE A  45      21.325  28.813  15.005  1.00  6.86           C  \\nATOM    355  CD2 PHE A  45      20.638  26.735  16.053  1.00  5.87           C  \\nATOM    356  CE1 PHE A  45      20.369  28.648  14.001  1.00  6.68           C  \\nATOM    357  CE2 PHE A  45      19.677  26.539  15.051  1.00  6.64           C  \\nATOM    358  CZ  PHE A  45      19.593  27.465  14.021  1.00  6.84           C  \\nATOM    359  N   ALA A  46      19.810  29.378  18.578  1.00  6.53           N  \\nATOM    360  CA  ALA A  46      18.443  29.143  19.083  1.00  7.15           C  \\nATOM    361  C   ALA A  46      18.453  28.941  20.591  1.00  9.00           C  \\nATOM    362  O   ALA A  46      17.860  27.994  21.128  1.00 11.15           O  \\nATOM    363  CB  ALA A  46      17.864  27.977  18.346  1.00  8.99           C  \\nATOM    364  N   GLY A  47      19.172  29.808  21.243  1.00  9.35           N  \\nATOM    365  CA  GLY A  47      19.399  29.894  22.655  1.00 11.68           C  \\nATOM    366  C   GLY A  47      20.083  28.729  23.321  1.00 11.14           C  \\nATOM    367  O   GLY A  47      19.991  28.584  24.561  1.00 13.93           O  \\nATOM    368  N   LYS A  48      20.801  27.931  22.578  1.00 10.47           N  \\nATOM    369  CA  LYS A  48      21.550  26.796  23.133  1.00  8.82           C  \\nATOM    370  C   LYS A  48      23.046  27.087  22.913  1.00  7.68           C  \\nATOM    371  O   LYS A  48      23.383  27.627  21.870  1.00  6.47           O  \\nATOM    372  CB  LYS A  48      21.242  25.519  22.391  1.00  9.74           C  \\nATOM    373  CG  LYS A  48      19.762  25.077  22.455  1.00 14.14           C  \\nATOM    374  CD  LYS A  48      19.634  23.885  21.531  1.00 16.32           C  \\nATOM    375  CE  LYS A  48      18.791  24.221  20.313  1.00 20.04           C  \\nATOM    376  NZ  LYS A  48      17.440  24.655  20.827  1.00 23.92           N  \\nATOM    377  N   GLN A  49      23.880  26.727  23.851  1.00  8.89           N  \\nATOM    378  CA  GLN A  49      25.349  26.872  23.643  1.00  7.18           C  \\nATOM    379  C   GLN A  49      25.743  25.586  22.922  1.00  8.23           C  \\nATOM    380  O   GLN A  49      25.325  24.489  23.378  1.00  9.70           O  \\nATOM    381  CB  GLN A  49      26.070  27.025  24.960  1.00 11.67           C  \\nATOM    382  CG  GLN A  49      27.553  27.356  24.695  1.00 15.82           C  \\nATOM    383  CD  GLN A  49      28.262  27.576  26.020  1.00 20.21           C  \\nATOM    384  OE1 GLN A  49      29.189  26.840  26.335  1.00 23.23           O  \\nATOM    385  NE2 GLN A  49      27.777  28.585  26.739  1.00 20.67           N  \\nATOM    386  N   LEU A  50      26.465  25.689  21.833  1.00  6.51           N  \\nATOM    387  CA  LEU A  50      26.826  24.521  21.012  1.00  7.41           C  \\nATOM    388  C   LEU A  50      27.994  23.781  21.643  1.00  8.27           C  \\nATOM    389  O   LEU A  50      28.904  24.444  22.098  1.00  8.34           O  \\nATOM    390  CB  LEU A  50      27.043  24.992  19.571  1.00  7.13           C  \\nATOM    391  CG  LEU A  50      25.931  25.844  18.959  1.00  7.53           C  \\nATOM    392  CD1 LEU A  50      26.203  26.083  17.471  1.00  8.14           C  \\nATOM    393  CD2 LEU A  50      24.577  25.190  19.079  1.00  9.11           C  \\nATOM    394  N   GLU A  51      27.942  22.448  21.648  1.00  9.43           N  \\nATOM    395  CA  GLU A  51      29.015  21.657  22.288  1.00 11.90           C  \\nATOM    396  C   GLU A  51      29.942  21.106  21.240  1.00 11.49           C  \\nATOM    397  O   GLU A  51      29.470  20.677  20.190  1.00  9.88           O  \\nATOM    398  CB  GLU A  51      28.348  20.540  23.066  1.00 16.56           C  \\nATOM    399  CG  GLU A  51      29.247  19.456  23.705  1.00 26.06           C  \\nATOM    400  CD  GLU A  51      28.722  19.047  25.066  1.00 29.86           C  \\nATOM    401  OE1 GLU A  51      29.139  18.132  25.746  1.00 32.13           O  \\nATOM    402  OE2 GLU A  51      27.777  19.842  25.367  1.00 33.44           O  \\nATOM    403  N   ASP A  52      31.233  21.090  21.459  1.00 12.71           N  \\nATOM    404  CA  ASP A  52      32.262  20.670  20.514  1.00 16.56           C  \\nATOM    405  C   ASP A  52      32.128  19.364  19.750  1.00 15.83           C  \\nATOM    406  O   ASP A  52      32.546  19.317  18.558  1.00 17.21           O  \\nATOM    407  CB  ASP A  52      33.638  20.716  21.242  1.00 21.05           C  \\nATOM    408  CG  ASP A  52      34.174  22.129  21.354  1.00 25.12           C  \\nATOM    409  OD1 ASP A  52      35.252  22.322  21.958  1.00 28.37           O  \\nATOM    410  OD2 ASP A  52      33.544  23.086  20.883  1.00 25.82           O  \\nATOM    411  N   GLY A  53      31.697  18.311  20.406  1.00 15.00           N  \\nATOM    412  CA  GLY A  53      31.568  16.962  19.825  1.00 11.77           C  \\nATOM    413  C   GLY A  53      30.320  16.698  19.051  1.00 11.10           C  \\nATOM    414  O   GLY A  53      30.198  15.657  18.366  1.00 11.25           O  \\nATOM    415  N   ARG A  54      29.340  17.594  19.076  1.00  8.53           N  \\nATOM    416  CA  ARG A  54      28.108  17.439  18.276  1.00  9.05           C  \\nATOM    417  C   ARG A  54      28.375  17.999  16.887  1.00  8.96           C  \\nATOM    418  O   ARG A  54      29.326  18.786  16.690  1.00 11.60           O  \\nATOM    419  CB  ARG A  54      26.926  18.191  18.892  1.00  7.97           C  \\nATOM    420  CG  ARG A  54      26.621  17.799  20.352  1.00  9.62           C  \\nATOM    421  CD  ARG A  54      26.010  16.370  20.280  1.00 12.20           C  \\nATOM    422  NE  ARG A  54      26.975  15.521  20.942  1.00 18.23           N  \\nATOM    423  CZ  ARG A  54      27.603  14.423  20.655  1.00 22.08           C  \\nATOM    424  NH1 ARG A  54      27.479  13.733  19.537  1.00 23.38           N  \\nATOM    425  NH2 ARG A  54      28.519  13.967  21.550  1.00 25.50           N  \\nATOM    426  N   THR A  55      27.510  17.689  15.954  1.00  9.05           N  \\nATOM    427  CA  THR A  55      27.574  18.192  14.563  1.00  9.03           C  \\nATOM    428  C   THR A  55      26.482  19.280  14.432  1.00  8.15           C  \\nATOM    429  O   THR A  55      25.609  19.388  15.287  1.00  5.91           O  \\nATOM    430  CB  THR A  55      27.299  17.055  13.533  1.00 11.15           C  \\nATOM    431  OG1 THR A  55      25.925  16.611  13.913  1.00 11.95           O  \\nATOM    432  CG2 THR A  55      28.236  15.864  13.558  1.00 11.71           C  \\nATOM    433  N   LEU A  56      26.585  20.063  13.378  1.00  6.91           N  \\nATOM    434  CA  LEU A  56      25.594  21.109  13.072  1.00  8.29           C  \\nATOM    435  C   LEU A  56      24.241  20.436  12.857  1.00  8.05           C  \\nATOM    436  O   LEU A  56      23.264  20.951  13.329  1.00 10.17           O  \\nATOM    437  CB  LEU A  56      26.084  21.888  11.833  1.00  6.60           C  \\nATOM    438  CG  LEU A  56      27.426  22.616  11.902  1.00  7.73           C  \\nATOM    439  CD1 LEU A  56      27.718  23.341  10.578  1.00  9.85           C  \\nATOM    440  CD2 LEU A  56      27.380  23.721  12.955  1.00  8.64           C  \\nATOM    441  N   SER A  57      24.240  19.233  12.246  1.00  8.92           N  \\nATOM    442  CA  SER A  57      22.924  18.583  12.025  1.00  9.00           C  \\nATOM    443  C   SER A  57      22.229  18.244  13.325  1.00  9.44           C  \\nATOM    444  O   SER A  57      20.963  18.253  13.395  1.00 10.91           O  \\nATOM    445  CB  SER A  57      23.059  17.326  11.154  1.00 10.32           C  \\nATOM    446  OG  SER A  57      23.914  16.395  11.755  1.00 13.59           O  \\nATOM    447  N   ASP A  58      22.997  17.978  14.366  1.00  9.11           N  \\nATOM    448  CA  ASP A  58      22.418  17.638  15.693  1.00  7.91           C  \\nATOM    449  C   ASP A  58      21.460  18.737  16.163  1.00  9.12           C  \\nATOM    450  O   ASP A  58      20.497  18.506  16.900  1.00  8.61           O  \\nATOM    451  CB  ASP A  58      23.461  17.331  16.741  1.00  8.41           C  \\nATOM    452  CG  ASP A  58      24.184  16.016  16.619  1.00 11.50           C  \\nATOM    453  OD1 ASP A  58      25.303  15.894  17.152  1.00 10.05           O  \\nATOM    454  OD2 ASP A  58      23.572  15.107  15.975  1.00 11.70           O  \\nATOM    455  N   TYR A  59      21.846  19.954  15.905  1.00  7.97           N  \\nATOM    456  CA  TYR A  59      21.079  21.149  16.251  1.00  8.45           C  \\nATOM    457  C   TYR A  59      20.142  21.590  15.149  1.00 10.98           C  \\nATOM    458  O   TYR A  59      19.499  22.645  15.321  1.00 12.95           O  \\nATOM    459  CB  TYR A  59      22.085  22.254  16.581  1.00  7.94           C  \\nATOM    460  CG  TYR A  59      22.945  21.951  17.785  1.00  6.91           C  \\nATOM    461  CD1 TYR A  59      24.272  21.544  17.644  1.00  4.59           C  \\nATOM    462  CD2 TYR A  59      22.437  22.157  19.065  1.00  6.98           C  \\nATOM    463  CE1 TYR A  59      25.052  21.285  18.776  1.00  5.39           C  \\nATOM    464  CE2 TYR A  59      23.204  21.907  20.192  1.00  6.52           C  \\nATOM    465  CZ  TYR A  59      24.517  21.470  20.030  1.00  6.76           C  \\nATOM    466  OH  TYR A  59      25.248  21.302  21.191  1.00  7.63           O  \\nATOM    467  N   ASN A  60      19.993  20.884  14.049  1.00 12.38           N  \\nATOM    468  CA  ASN A  60      19.065  21.352  12.999  1.00 13.94           C  \\nATOM    469  C   ASN A  60      19.442  22.745  12.510  1.00 14.16           C  \\nATOM    470  O   ASN A  60      18.571  23.610  12.289  1.00 14.26           O  \\nATOM    471  CB  ASN A  60      17.586  21.282  13.461  1.00 19.23           C  \\nATOM    472  CG  ASN A  60      16.576  21.258  12.315  1.00 22.65           C  \\nATOM    473  OD1 ASN A  60      15.440  21.819  12.378  1.00 25.45           O  \\nATOM    474  ND2 ASN A  60      16.924  20.586  11.216  1.00 24.09           N  \\nATOM    475  N   ILE A  61      20.717  22.964  12.260  1.00 11.08           N  \\nATOM    476  CA  ILE A  61      21.184  24.263  11.690  1.00 11.78           C  \\nATOM    477  C   ILE A  61      21.110  24.111  10.173  1.00 13.74           C  \\nATOM    478  O   ILE A  61      21.841  23.198   9.686  1.00 14.60           O  \\nATOM    479  CB  ILE A  61      22.650  24.516  12.172  1.00 11.80           C  \\nATOM    480  CG1 ILE A  61      22.662  24.819  13.699  1.00 11.56           C  \\nATOM    481  CG2 ILE A  61      23.376  25.645  11.409  1.00 13.29           C  \\nATOM    482  CD1 ILE A  61      24.123  24.981  14.195  1.00 11.42           C  \\nATOM    483  N   GLN A  62      20.291  24.875   9.507  1.00 13.97           N  \\nATOM    484  CA  GLN A  62      20.081  24.773   8.033  1.00 15.52           C  \\nATOM    485  C   GLN A  62      20.822  25.914   7.332  1.00 13.94           C  \\nATOM    486  O   GLN A  62      21.323  26.830   8.008  1.00 12.15           O  \\nATOM    487  CB  GLN A  62      18.599  24.736   7.727  1.00 19.53           C  \\nATOM    488  CG  GLN A  62      17.819  23.434   7.900  1.00 26.38           C  \\nATOM    489  CD  GLN A  62      16.509  23.529   7.116  1.00 30.61           C  \\nATOM    490  OE1 GLN A  62      15.446  22.980   7.433  1.00 33.23           O  \\nATOM    491  NE2 GLN A  62      16.539  24.293   6.009  1.00 32.71           N  \\nATOM    492  N   LYS A  63      20.924  25.862   6.006  1.00 11.73           N  \\nATOM    493  CA  LYS A  63      21.656  26.847   5.240  1.00 11.97           C  \\nATOM    494  C   LYS A  63      21.127  28.240   5.574  1.00 10.41           C  \\nATOM    495  O   LYS A  63      19.958  28.465   5.842  1.00  9.59           O  \\nATOM    496  CB  LYS A  63      21.631  26.642   3.731  1.00 13.73           C  \\nATOM    497  CG  LYS A  63      20.210  26.423   3.175  1.00 16.98           C  \\nATOM    498  CD  LYS A  63      20.268  26.589   1.656  1.00 20.19           C  \\nATOM    499  CE  LYS A  63      19.202  25.857   0.891  1.00 23.42           C  \\nATOM    500  NZ  LYS A  63      17.884  26.544   1.075  1.00 25.97           N  \\nATOM    501  N   GLU A  64      22.099  29.163   5.605  1.00 10.04           N  \\nATOM    502  CA  GLU A  64      21.907  30.563   5.881  1.00 10.94           C  \\nATOM    503  C   GLU A  64      21.466  30.953   7.261  1.00  9.74           C  \\nATOM    504  O   GLU A  64      21.066  32.112   7.533  1.00  9.42           O  \\nATOM    505  CB  GLU A  64      21.023  31.223   4.784  1.00 18.31           C  \\nATOM    506  CG  GLU A  64      21.861  31.342   3.474  1.00 24.16           C  \\nATOM    507  CD  GLU A  64      21.156  30.726   2.311  1.00 29.00           C  \\nATOM    508  OE1 GLU A  64      19.942  30.793   2.170  1.00 31.72           O  \\nATOM    509  OE2 GLU A  64      21.954  30.152   1.535  1.00 32.61           O  \\nATOM    510  N   SER A  65      21.674  30.034   8.191  1.00  6.85           N  \\nATOM    511  CA  SER A  65      21.419  30.253   9.620  1.00  6.90           C  \\nATOM    512  C   SER A  65      22.504  31.228  10.136  1.00  4.72           C  \\nATOM    513  O   SER A  65      23.579  31.321   9.554  1.00  3.91           O  \\nATOM    514  CB  SER A  65      21.637  28.923  10.353  1.00  7.28           C  \\nATOM    515  OG  SER A  65      20.544  28.047  10.059  1.00 10.56           O  \\nATOM    516  N   THR A  66      22.241  31.873  11.241  1.00  4.48           N  \\nATOM    517  CA  THR A  66      23.212  32.762  11.891  1.00  3.80           C  \\nATOM    518  C   THR A  66      23.509  32.224  13.290  1.00  4.60           C  \\nATOM    519  O   THR A  66      22.544  31.942  14.034  1.00  5.33           O  \\nATOM    520  CB  THR A  66      22.699  34.267  11.985  1.00  2.85           C  \\nATOM    521  OG1 THR A  66      22.495  34.690  10.589  1.00  2.15           O  \\nATOM    522  CG2 THR A  66      23.727  35.131  12.722  1.00  3.40           C  \\nATOM    523  N   LEU A  67      24.790  32.021  13.618  1.00  4.17           N  \\nATOM    524  CA  LEU A  67      25.149  31.609  14.980  1.00  3.85           C  \\nATOM    525  C   LEU A  67      25.698  32.876  15.669  1.00  3.80           C  \\nATOM    526  O   LEU A  67      26.158  33.730  14.894  1.00  5.54           O  \\nATOM    527  CB  LEU A  67      26.310  30.594  14.967  1.00  7.18           C  \\nATOM    528  CG  LEU A  67      26.290  29.480  13.960  1.00  9.67           C  \\nATOM    529  CD1 LEU A  67      27.393  28.442  14.229  1.00  8.12           C  \\nATOM    530  CD2 LEU A  67      24.942  28.807  13.952  1.00 11.66           C  \\nATOM    531  N   HIS A  68      25.621  32.945  16.950  1.00  2.94           N  \\nATOM    532  CA  HIS A  68      26.179  34.127  17.650  1.00  4.17           C  \\nATOM    533  C   HIS A  68      27.475  33.651  18.304  1.00  5.32           C  \\nATOM    534  O   HIS A  68      27.507  32.587  18.958  1.00  7.70           O  \\nATOM    535  CB  HIS A  68      25.214  34.565  18.780  1.00  5.57           C  \\nATOM    536  CG  HIS A  68      23.978  35.121  18.126  1.00  9.95           C  \\nATOM    537  ND1 HIS A  68      23.853  36.432  17.781  1.00 13.74           N  \\nATOM    538  CD2 HIS A  68      22.824  34.514  17.782  1.00 12.79           C  \\nATOM    539  CE1 HIS A  68      22.674  36.627  17.200  1.00 14.75           C  \\nATOM    540  NE2 HIS A  68      22.045  35.455  17.173  1.00 16.30           N  \\nATOM    541  N   LEU A  69      28.525  34.447  18.189  1.00  5.29           N  \\nATOM    542  CA  LEU A  69      29.801  34.145  18.829  1.00  3.97           C  \\nATOM    543  C   LEU A  69      30.052  35.042  20.004  1.00  5.07           C  \\nATOM    544  O   LEU A  69      30.105  36.305  19.788  1.00  4.34           O  \\nATOM    545  CB  LEU A  69      30.925  34.304  17.753  1.00  6.08           C  \\nATOM    546  CG  LEU A  69      32.345  34.183  18.358  1.00  7.37           C  \\nATOM    547  CD1 LEU A  69      32.555  32.783  18.870  1.00  6.87           C  \\nATOM    548  CD2 LEU A  69      33.361  34.491  17.245  1.00  9.96           C  \\nATOM    549  N   VAL A  70      30.124  34.533  21.191  1.00  4.29           N  \\nATOM    550  CA  VAL A  70      30.479  35.369  22.374  1.00  6.26           C  \\nATOM    551  C   VAL A  70      31.901  34.910  22.728  1.00  9.22           C  \\nATOM    552  O   VAL A  70      32.190  33.696  22.635  1.00  9.36           O  \\nATOM    553  CB  VAL A  70      29.472  35.181  23.498  1.00  8.69           C  \\nATOM    554  CG1 VAL A  70      29.821  35.957  24.765  1.00  9.76           C  \\nATOM    555  CG2 VAL A  70      28.049  35.454  23.071  1.00  8.54           C  \\nATOM    556  N   LEU A  71      32.763  35.831  23.090  1.00 12.71           N  \\nATOM    557  CA  LEU A  71      34.145  35.472  23.481  1.00 16.06           C  \\nATOM    558  C   LEU A  71      34.239  35.353  24.979  1.00 18.09           C  \\nATOM    559  O   LEU A  71      33.707  36.197  25.728  1.00 19.26           O  \\nATOM    560  CB  LEU A  71      35.114  36.564  22.907  1.00 17.10           C  \\nATOM    561  CG  LEU A  71      35.926  35.979  21.737  1.00 19.37           C  \\nATOM    562  CD1 LEU A  71      35.003  35.084  20.920  1.00 17.51           C  \\nATOM    563  CD2 LEU A  71      36.533  37.087  20.917  1.00 19.57           C  \\nATOM    564  N   ARG A  72      34.930  34.384  25.451  1.00 21.47           N  \\nATOM    565  CA  ARG A  72      35.161  34.174  26.896  1.00 25.83           C  \\nATOM    566  C   ARG A  72      36.671  34.296  27.089  1.00 27.74           C  \\nATOM    567  O   ARG A  72      37.305  33.233  26.795  1.00 30.65           O  \\nATOM    568  CB  ARG A  72      34.717  32.760  27.286  1.00 28.49           C  \\nATOM    569  CG  ARG A  72      35.752  32.054  28.160  1.00 31.79           C  \\nATOM    570  CD  ARG A  72      35.612  30.577  28.044  1.00 34.05           C  \\nATOM    571  NE  ARG A  72      35.040  30.252  26.730  1.00 35.08           N  \\nATOM    572  CZ  ARG A  72      34.338  29.103  26.650  1.00 34.67           C  \\nATOM    573  NH1 ARG A  72      34.110  28.437  27.768  1.00 35.02           N  \\nATOM    574  NH2 ARG A  72      34.014  28.657  25.457  1.00 34.97           N  \\nATOM    575  N   LEU A  73      37.197  35.397  27.513  0.45 28.93           N  \\nATOM    576  CA  LEU A  73      38.668  35.502  27.680  0.45 30.76           C  \\nATOM    577  C   LEU A  73      39.076  34.931  29.031  0.45 32.18           C  \\nATOM    578  O   LEU A  73      38.297  34.946  29.996  0.45 32.31           O  \\nATOM    579  CB  LEU A  73      39.080  36.941  27.406  0.45 30.53           C  \\nATOM    580  CG  LEU A  73      39.502  37.340  26.002  0.45 30.16           C  \\nATOM    581  CD1 LEU A  73      38.684  36.647  24.923  0.45 29.57           C  \\nATOM    582  CD2 LEU A  73      39.337  38.854  25.862  0.45 29.11           C  \\nATOM    583  N   ARG A  74      40.294  34.412  29.045  0.45 33.82           N  \\nATOM    584  CA  ARG A  74      40.873  33.802  30.253  0.45 35.33           C  \\nATOM    585  C   ARG A  74      41.765  34.829  30.944  0.45 36.22           C  \\nATOM    586  O   ARG A  74      42.945  34.994  30.583  0.45 36.70           O  \\nATOM    587  CB  ARG A  74      41.651  32.529  29.923  0.45 36.91           C  \\nATOM    588  CG  ARG A  74      41.608  31.444  30.989  0.45 38.62           C  \\nATOM    589  CD  ARG A  74      41.896  30.080  30.456  0.45 39.75           C  \\nATOM    590  NE  ARG A  74      43.311  29.735  30.563  0.45 41.13           N  \\nATOM    591  CZ  ARG A  74      44.174  29.905  29.554  0.45 41.91           C  \\nATOM    592  NH1 ARG A  74      43.754  30.312  28.356  0.45 42.75           N  \\nATOM    593  NH2 ARG A  74      45.477  29.726  29.763  0.45 41.93           N  \\nATOM    594  N   GLY A  75      41.165  35.531  31.898  0.25 36.31           N  \\nATOM    595  CA  GLY A  75      41.845  36.550  32.686  0.25 36.07           C  \\nATOM    596  C   GLY A  75      41.251  37.941  32.588  0.25 36.16           C  \\nATOM    597  O   GLY A  75      41.102  38.523  31.500  0.25 36.26           O  \\nATOM    598  N   GLY A  76      40.946  38.472  33.757  0.25 36.05           N  \\nATOM    599  CA  GLY A  76      40.373  39.813  33.944  0.25 36.19           C  \\nATOM    600  C   GLY A  76      40.031  39.992  35.432  0.25 36.20           C  \\nATOM    601  O   GLY A  76      38.933  40.525  35.687  0.25 36.13           O  \\nATOM    602  OXT GLY A  76      40.862  39.575  36.251  0.25 36.27           O  \\nTER     603      GLY A  76                                                      \\nHETATM  604  O   HOH A  77      45.747  30.081  19.708  1.00 12.43           O  \\nHETATM  605  O   HOH A  78      19.168  31.868  17.050  1.00 12.65           O  \\nHETATM  606  O   HOH A  79      32.010  38.387  19.636  1.00 12.83           O  \\nHETATM  607  O   HOH A  80      42.084  27.361  21.953  1.00 22.27           O  \\nHETATM  608  O   HOH A  81      21.314  20.644   8.719  1.00 18.33           O  \\nHETATM  609  O   HOH A  82      31.965  38.637   3.699  1.00 31.69           O  \\nHETATM  610  O   HOH A  83      27.707  15.908   4.653  1.00 20.30           O  \\nHETATM  611  O   HOH A  84      19.969  32.720  14.769  1.00 10.14           O  \\nHETATM  612  O   HOH A  85      29.847  13.577  10.864  1.00 29.65           O  \\nHETATM  613  O   HOH A  86      23.893  27.864   1.501  1.00 23.48           O  \\nHETATM  614  O   HOH A  87      19.638  23.312   4.775  1.00 18.40           O  \\nHETATM  615  O   HOH A  88      34.628  29.369   4.779  1.00 26.17           O  \\nHETATM  616  O   HOH A  89      42.240  24.744  25.707  1.00 31.34           O  \\nHETATM  617  O   HOH A  90      30.290  42.500   8.820  1.00 16.49           O  \\nHETATM  618  O   HOH A  91      24.512  39.162  10.841  1.00 13.14           O  \\nHETATM  619  O   HOH A  92      26.557  43.450  19.940  1.00 19.38           O  \\nHETATM  620  O   HOH A  93      42.535  22.385  13.872  1.00 29.35           O  \\nHETATM  621  O   HOH A  94      42.440  26.381  12.686  1.00 29.46           O  \\nHETATM  622  O   HOH A  95      22.651  14.457  13.085  1.00 22.07           O  \\nHETATM  623  O   HOH A  96      35.325  26.551  23.202  1.00 15.20           O  \\nHETATM  624  O   HOH A  97      23.629  20.940   3.146  1.00 15.45           O  \\nHETATM  625  O   HOH A  98      25.928  21.774   2.325  1.00 13.70           O  \\nHETATM  626  O   HOH A  99      33.388  21.973   5.659  1.00 24.89           O  \\nHETATM  627  O   HOH A 100      18.326  23.911  17.697  1.00 24.10           O  \\nHETATM  628  O   HOH A 101      18.160  27.072  10.662  1.00 20.76           O  \\nHETATM  629  O   HOH A 102      34.746  17.167  18.219  1.00 32.86           O  \\nHETATM  630  O   HOH A 103      19.801  32.364  20.210  1.00 21.09           O  \\nHETATM  631  O   HOH A 104      30.285  26.829  22.191  1.00  8.56           O  \\nHETATM  632  O   HOH A 105      44.612  32.306  16.961  1.00  7.69           O  \\nHETATM  633  O   HOH A 106      16.287  25.999  13.142  0.78 28.90           O  \\nHETATM  634  O   HOH A 107      27.101  42.135  15.494  0.51 23.36           O  \\nHETATM  635  O   HOH A 108      37.209  23.795  21.367  0.74 27.88           O  \\nHETATM  636  O   HOH A 109      19.582  32.034  -0.685  0.49 22.24           O  \\nHETATM  637  O   HOH A 110      28.824  25.094   0.886  0.77 36.99           O  \\nHETATM  638  O   HOH A 111      25.146  19.162  25.323  0.87 36.70           O  \\nHETATM  639  O   HOH A 112      20.747  37.769  14.674  0.85 29.64           O  \\nHETATM  640  O   HOH A 113      16.035  17.841   8.765  0.61 23.89           O  \\nHETATM  641  O   HOH A 114      35.712  46.814  12.926  0.48 27.11           O  \\nHETATM  642  O   HOH A 115      15.570  27.475   7.482  0.51 24.18           O  \\nHETATM  643  O   HOH A 116      33.447  21.075   2.918  0.59 26.03           O  \\nHETATM  644  O   HOH A 117      41.116  39.021  13.061  0.63 22.39           O  \\nHETATM  645  O   HOH A 118      32.346  13.689  18.912  0.48 24.09           O  \\nHETATM  646  O   HOH A 119      31.197  13.048   7.920  0.71 29.54           O  \\nHETATM  647  O   HOH A 120      42.853  39.375  29.308  0.64 46.90           O  \\nHETATM  648  O   HOH A 121      39.646  23.959   9.699  0.41 18.25           O  \\nHETATM  649  O   HOH A 122      34.405  45.181  13.420  0.87 26.13           O  \\nHETATM  650  O   HOH A 123      26.517  24.300  27.592  0.41 21.02           O  \\nHETATM  651  O   HOH A 124      40.740  38.734   9.602  0.45 16.60           O  \\nHETATM  652  O   HOH A 125      31.494  18.276  23.170  0.67 26.53           O  \\nHETATM  653  O   HOH A 126      37.752  30.947   1.059  0.87 32.52           O  \\nHETATM  654  O   HOH A 127      31.771  16.941   7.511  0.64 15.94           O  \\nHETATM  655  O   HOH A 128      41.628  24.537  10.145  0.57 22.53           O  \\nHETATM  656  O   HOH A 129      28.988  22.175  -1.744  0.56 29.32           O  \\nHETATM  657  O   HOH A 130      14.882  16.539  10.692  0.53 24.82           O  \\nHETATM  658  O   HOH A 131      32.589  40.385   7.523  0.36 26.01           O  \\nHETATM  659  O   HOH A 132      38.363  30.369   5.579  0.49 35.45           O  \\nHETATM  660  O   HOH A 133      27.841  46.062  17.589  0.81 32.15           O  \\nHETATM  661  O   HOH A 134      37.667  43.421  17.000  0.50 33.32           O  \\nMASTER      260    0    0    2    5    0    0    6  660    1    0    6          \\nEND                                                                             \\n\",\"pdb\");\n",
              "\tviewer_17338419469247565.setStyle({\"cartoon\": {\"color\": \"spectrum\"}, \"stick\": {\"radius\": 0.15}});\n",
              "\tviewer_17338419469247565.center();\n",
              "\tviewer_17338419469247565.zoomTo();\n",
              "viewer_17338419469247565.render();\n",
              "});\n",
              "</script>"
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "<py3Dmol.view at 0x7eb65585a5c0>"
            ]
          },
          "metadata": {},
          "execution_count": 33
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "bvujVC4mn2G7"
      },
      "source": [
        "---------------------\n",
        "\n",
        "#<font color=\"green\"> **Instalando e usando pacotes de modelagem molecular (*)** </font>\n",
        "\n",
        "A partir desse ponto vamos instalar e usar v√°rios pacotes de modelagem e simula√ß√£o molecular como uma esp√©cie de preparativo para pesquisadores da √°rea.\n",
        "\n",
        "As instala√ß√µes ser√£o feitas tanto a partir das distribui√ß√µes como bibliotecas quanto atrav√©s do *download* do c√≥digo e sua compila√ß√£o.\n",
        "\n",
        "N√£o vamos explorar muito os recursos de cada programa. A sugest√£o √© gerar um novo *notebook* para cada ferramenta.\n",
        "\n",
        "A lista dos softwares que vamos cobrir √©:\n",
        "\n",
        "* MOPAC 2016\n",
        "* PRIMoRDiA\n",
        "-------------------------"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "6UevOB5Kn2G7"
      },
      "source": [
        "### <font color=\"purple\"> **MOPAC** </font>\n",
        "\n",
        "Vamos rodar o MOPAC de duas formas diferentes.\n",
        "\n",
        "1. Forma direta em linha de comando\n",
        "2. Forma indireta atrav√©s da biblioteca em python (ASE).\n",
        "\n",
        "Links:\n",
        "\n",
        "* [MOPAC](http://openmopac.net/)\n",
        "* [MOPAC github](https://github.com/openmopac/)\n",
        "* [MOPAC manual](http://openmopac.net/manual/index.html)\n",
        "* [ASE](https://wiki.fysik.dtu.dk/ase/)\n",
        "\n",
        "--------------------"
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "##### **Executando o MOPAC por linha de comando (1a forma)**"
      ],
      "metadata": {
        "id": "XWmQ9MZuv5Dw"
      }
    },
    {
      "cell_type": "code",
      "execution_count": 8,
      "metadata": {
        "id": "EiHOWm0Rn2G7",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "efb15ae5-00c9-46f0-be34-eab930cd126d"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "--2024-12-10 13:30:43--  https://github.com/openmopac/mopac/releases/download/v23.0.3/mopac-23.0.3-linux.tar.gz\n",
            "Resolving github.com (github.com)... 140.82.114.4\n",
            "Connecting to github.com (github.com)|140.82.114.4|:443... connected.\n",
            "HTTP request sent, awaiting response... 302 Found\n",
            "Location: https://objects.githubusercontent.com/github-production-release-asset-2e65be/177640376/42a11379-8aba-4f17-9c99-6007f421a88f?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=releaseassetproduction%2F20241210%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20241210T133043Z&X-Amz-Expires=300&X-Amz-Signature=c6eaaeb1df1a1a395167ecb4c0d4d329e2d3e3d611233e11e2bf1d387a3274e8&X-Amz-SignedHeaders=host&response-content-disposition=attachment%3B%20filename%3Dmopac-23.0.3-linux.tar.gz&response-content-type=application%2Foctet-stream [following]\n",
            "--2024-12-10 13:30:43--  https://objects.githubusercontent.com/github-production-release-asset-2e65be/177640376/42a11379-8aba-4f17-9c99-6007f421a88f?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=releaseassetproduction%2F20241210%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20241210T133043Z&X-Amz-Expires=300&X-Amz-Signature=c6eaaeb1df1a1a395167ecb4c0d4d329e2d3e3d611233e11e2bf1d387a3274e8&X-Amz-SignedHeaders=host&response-content-disposition=attachment%3B%20filename%3Dmopac-23.0.3-linux.tar.gz&response-content-type=application%2Foctet-stream\n",
            "Resolving objects.githubusercontent.com (objects.githubusercontent.com)... 185.199.108.133, 185.199.109.133, 185.199.110.133, ...\n",
            "Connecting to objects.githubusercontent.com (objects.githubusercontent.com)|185.199.108.133|:443... connected.\n",
            "HTTP request sent, awaiting response... 200 OK\n",
            "Length: 8718878 (8.3M) [application/octet-stream]\n",
            "Saving to: ‚Äòmopac-23.0.3-linux.tar.gz‚Äô\n",
            "\n",
            "mopac-23.0.3-linux. 100%[===================>]   8.31M  38.8MB/s    in 0.2s    \n",
            "\n",
            "2024-12-10 13:30:44 (38.8 MB/s) - ‚Äòmopac-23.0.3-linux.tar.gz‚Äô saved [8718878/8718878]\n",
            "\n"
          ]
        }
      ],
      "source": [
        "# Esse link √© para vers√£o mais recente em 07-DEZ-24.\n",
        "# Atualizar para vers√£o mais nova caso esteja acompanhando esse tutorial em data futura.\n",
        "!wget https://github.com/openmopac/mopac/releases/download/v23.0.3/mopac-23.0.3-linux.tar.gz"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 9,
      "metadata": {
        "id": "7u1uX3qxn2G8"
      },
      "outputs": [],
      "source": [
        "!tar xfz mopac-23.0.3-linux.tar.gz"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 10,
      "metadata": {
        "id": "AZFZIGF8n2G8",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "a2392c22-acc8-4c3c-9f4c-d07e1d51c256"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "-rwxr-xr-x 1 root root   32008 Dec  3 18:54 /content/mopac-23.0.3-linux/bin/mopac\n",
            "-rwxr-xr-x 1 root root 1051200 Dec  3 18:53 /content/mopac-23.0.3-linux/bin/mopac-makpol\n",
            "-rwxr-xr-x 1 root root   32008 Dec  3 18:54 /content/mopac-23.0.3-linux/bin/mopac-param\n"
          ]
        }
      ],
      "source": [
        "!ls -la /content/mopac-23.0.3-linux/bin/*"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 11,
      "metadata": {
        "id": "rYLx6AoIsyAG"
      },
      "outputs": [],
      "source": [
        "# Para colocar o diret√≥rio do MOPAC no PATH do sistema\n",
        "import os\n",
        "os.environ['PATH'] += ':/content/mopac-23.0.3-linux/bin'"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 12,
      "metadata": {
        "id": "Ns4IQrGgn2G7",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "26308014-3a60-41ee-e926-95cf3c016cdd"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Writing acrolein.mop\n"
          ]
        }
      ],
      "source": [
        "%%file acrolein.mop\n",
        "GNORM=0.01 RM1 AUX ALLVECS LARGE EPS=74.8 BFGS\n",
        "1SCF calculation for acrolein with RM1\n",
        "Testing MOPAC on Google Colab\n",
        "C   0.811 1  0.465 1  0.000  1\n",
        "C  -0.667 1  0.603 1 -0.000  1\n",
        "C  -1.468 1 -0.469 1  0.000  1\n",
        "H  -1.067 1  1.615 1 -0.000  1\n",
        "H  -2.551 1 -0.382 1  0.000  1\n",
        "H  -1.041 1 -1.468 1  0.000  1\n",
        "H   1.374 1  1.423 1  0.000  1\n",
        "O   1.404 1 -0.598 1 -0.000  1\n",
        "0"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 13,
      "metadata": {
        "id": "Qs_kdgY00sqg",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "4df1c404-bba8-4f9d-f2f9-5c30f974f44c"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "nohup: appending output to 'nohup.out'\n"
          ]
        }
      ],
      "source": [
        "!nohup mopac acrolein.mop &"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 14,
      "metadata": {
        "id": "JMRHXKhI0stc",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "d111945d-9ff7-4123-a0c5-fe18917a7395"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\n",
            "\n",
            "                     SUMMARY OF  RM1 CALCULATION\n",
            "\n",
            "                                                       MOPAC v23.0.3 Linux\n",
            "                                                       Tue Dec 10 13:30:59 2024\n",
            "\n",
            "           Empirical Formula: C3 H4 O  =     8 atoms\n",
            "\n",
            " GNORM=0.01 RM1 AUX ALLVECS LARGE EPS=74.8 BFGS\n",
            " 1SCF calculation for acrolein with RM1\n",
            " Testing MOPAC on Google Colab\n",
            "\n",
            "\n",
            "     THE LINE MINIMIZATION FAILED TWICE IN A ROW.   TAKE CARE!\n",
            "     SCF FIELD WAS ACHIEVED                                   \n",
            "\n",
            "          HEAT OF FORMATION       =        -20.82148 KCAL/MOL =     -87.11707 KJ/MOL\n",
            "          DIELECTRIC ENERGY       =         -0.23880 EV\n",
            "          GRADIENT NORM           =          0.22230          =       0.07859 PER ATOM\n",
            "          DIPOLE                  =          3.16110 DEBYE   POINT GROUP:  Cs  \n",
            "          NO. OF FILLED LEVELS    =         11\n",
            "          IONIZATION POTENTIAL    =         10.679241 EV\n",
            "          HOMO LUMO ENERGIES (EV) =        -10.679  0.152\n",
            "          MOLECULAR WEIGHT        =         56.0640\n",
            "          COSMO AREA              =         95.82 SQUARE ANGSTROMS\n",
            "          COSMO VOLUME            =         79.08 CUBIC ANGSTROMS\n",
            "\n",
            "          MOLECULAR DIMENSIONS (Angstroms)\n",
            "\n",
            "            Atom       Atom       Distance\n",
            "            H     7    H     5     4.32503\n",
            "            O     8    H     4     3.03935\n",
            "            H     6    H     4     0.00907\n",
            "          SCF CALCULATIONS        =        119\n",
            "          WALL-CLOCK TIME         =      0.441 SECONDS\n",
            "          COMPUTATION TIME        =      0.432 SECONDS\n",
            "\n",
            "\n",
            "          FINAL GEOMETRY OBTAINED\n",
            " GNORM=0.01 RM1 AUX ALLVECS LARGE EPS=74.8 BFGS\n",
            " 1SCF calculation for acrolein with RM1\n",
            " Testing MOPAC on Google Colab\n",
            "  C     0.80798865 +1   0.47865615 +1  -0.00214848 +1\n",
            "  C    -0.65845762 +1   0.59637051 +1  -0.00677529 +1\n",
            "  C    -1.45850022 +1  -0.46590774 +1  -0.00257306 +1\n",
            "  H    -1.05470550 +1   1.61767630 +1  -0.01353363 +1\n",
            "  H    -2.54344906 +1  -0.39174176 +1  -0.00555594 +1\n",
            "  H    -1.10445568 +1  -1.49467630 +1   0.00435744 +1\n",
            "  H     1.38178506 +1   1.42439925 +1  -0.00001907 +1\n",
            "  O     1.37392963 +1  -0.60751893 +1  -0.00047030 +1\n",
            " \n"
          ]
        }
      ],
      "source": [
        "# Para verificar o resultado do c√°lculo realizado\n",
        "!cat < acrolein.arc"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "4f5ELu3In2G8"
      },
      "outputs": [],
      "source": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "JNBRd97in2G9"
      },
      "source": [
        "##### **Executando o MOPAC atrav√©s da ASE (2a forma)**"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 1,
      "metadata": {
        "id": "zICex8oZn2G9",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "80b7aec0-0556-473b-f979-9d93bb09d4a0"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Requirement already satisfied: ase in /usr/local/lib/python3.10/site-packages (3.23.0)\n",
            "Requirement already satisfied: numpy>=1.18.5 in /usr/local/lib/python3.10/site-packages (from ase) (2.2.0)\n",
            "Requirement already satisfied: scipy>=1.6.0 in /usr/local/lib/python3.10/site-packages (from ase) (1.14.1)\n",
            "Requirement already satisfied: matplotlib>=3.3.4 in /usr/local/lib/python3.10/site-packages (from ase) (3.9.3)\n",
            "Requirement already satisfied: contourpy>=1.0.1 in /usr/local/lib/python3.10/site-packages (from matplotlib>=3.3.4->ase) (1.3.1)\n",
            "Requirement already satisfied: cycler>=0.10 in /usr/local/lib/python3.10/site-packages (from matplotlib>=3.3.4->ase) (0.12.1)\n",
            "Requirement already satisfied: fonttools>=4.22.0 in /usr/local/lib/python3.10/site-packages (from matplotlib>=3.3.4->ase) (4.55.2)\n",
            "Requirement already satisfied: kiwisolver>=1.3.1 in /usr/local/lib/python3.10/site-packages (from matplotlib>=3.3.4->ase) (1.4.7)\n",
            "Requirement already satisfied: packaging>=20.0 in /usr/local/lib/python3.10/site-packages (from matplotlib>=3.3.4->ase) (23.2)\n",
            "Requirement already satisfied: pillow>=8 in /usr/local/lib/python3.10/site-packages (from matplotlib>=3.3.4->ase) (11.0.0)\n",
            "Requirement already satisfied: pyparsing>=2.3.1 in /usr/local/lib/python3.10/site-packages (from matplotlib>=3.3.4->ase) (3.2.0)\n",
            "Requirement already satisfied: python-dateutil>=2.7 in /usr/local/lib/python3.10/site-packages (from matplotlib>=3.3.4->ase) (2.9.0.post0)\n",
            "Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.10/site-packages (from python-dateutil>=2.7->matplotlib>=3.3.4->ase) (1.17.0)\n"
          ]
        }
      ],
      "source": [
        "!pip install ase\n",
        "# Se pedir para reiniciar, rode mais uma vez. No caso, voc√™ deve rodar novamente\n",
        "# a c√©lula que define a vari√°vel PATH para o MOPAC"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 2,
      "metadata": {
        "id": "aS-5oDvcXZ-E"
      },
      "outputs": [],
      "source": [
        "# Para colocar o diret√≥rio do MOPAC no PATH\n",
        "import os\n",
        "os.environ['PATH'] += ':/content/mopac-23.0.3-linux/bin'"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 3,
      "metadata": {
        "id": "wYRJRn1sn2G9"
      },
      "outputs": [],
      "source": [
        "# https://wiki.fysik.dtu.dk/ase/ase/calculators/mopac.html\n",
        "from ase.build import molecule\n",
        "from ase.calculators.mopac import MOPAC\n",
        "\n",
        "atoms = molecule('H2O')\n",
        "atoms.calc = MOPAC(label='H2O')\n",
        "atoms.get_potential_energy()\n",
        "eigs = atoms.calc.get_eigenvalues()\n",
        "# somos = atoms.calc.get_somo_levels()  # Ativar no caso da sua mol√©cula ter multiplicidade diferente de 1\n",
        "homo, lumo = atoms.calc.get_homo_lumo_levels()"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 4,
      "metadata": {
        "id": "z_NBN_Ozn2G9",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "9065eb6d-ce22-47f9-c0f2-b6a967cd9b5e"
      },
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "array([-29.39049, -17.61355, -14.25684, -12.03346,   3.97073,   6.21044])"
            ]
          },
          "metadata": {},
          "execution_count": 4
        }
      ],
      "source": [
        "eigs"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 6,
      "metadata": {
        "id": "SaleKUFHn2G9",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "76979a96-e362-4a02-c7b8-fc29284974f6"
      },
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "0.9954438198079898"
            ]
          },
          "metadata": {},
          "execution_count": 6
        }
      ],
      "source": [
        "atoms = molecule('C6H6')\n",
        "atoms.calc = MOPAC(label='C6H6', task='GRADIENTS')\n",
        "atoms.get_potential_energy()"
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "OBS: O interesse aqui n√£o √© explorarmos muito a lib [ASE](https://wiki.fysik.dtu.dk/ase/ase/calculators/mopac.html). Para mais informa√ß√µes, acessar o portal da lib e tamb√©m o minicurso apresentado no 8a Encontro de Qu√≠mica da UFPB, link: [8oEQ_UFPB](https://github.com/RochaGerd/Chemistry_with_Python/blob/main/8o_Enconto_Qu%C3%ADmica_UFPB_2024_Parte_01.ipynb)."
      ],
      "metadata": {
        "id": "Rgt3gkiUtRKp"
      }
    },
    {
      "cell_type": "code",
      "execution_count": 7,
      "metadata": {
        "id": "Il3I3t9Fn2G9",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "2320f3d0-1f86-4373-f172-2b0876ccb8c1"
      },
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "array([-9.82352,  0.23591])"
            ]
          },
          "metadata": {},
          "execution_count": 7
        }
      ],
      "source": [
        "atoms = MOPAC.read_atoms('C6H6')  # ler o input C6H6.mop salvo previamente no diret√≥rio\n",
        "atoms.calc.get_homo_lumo_levels()"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 8,
      "metadata": {
        "id": "r4A5N5Awn2G9",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "51b705b8-9906-4770-8229-6c94084c52b5"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Heat of formation (kcal/mol) =  22.95548\n"
          ]
        }
      ],
      "source": [
        "print('Heat of formation (kcal/mol) = ',\n",
        "      round(atoms.calc.get_final_heat_of_formation()*23.060548,5))  # em kcal/mol"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "FBULvwB7n2G-"
      },
      "outputs": [],
      "source": [
        "atoms.get_all_distances()"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "cGF_1jQNn2G-"
      },
      "outputs": [],
      "source": [
        "atoms.get_dipole_moment()"
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "-------------------\n",
        "## **Considera√ß√µes sobre o c√°lculo de biomol√©culas com o MOPAC**\n",
        "\n",
        "O programa MOPAC permite o c√°lculo de sistemas biomoleculares (sistemas com muitos √°tomos) por meio de algoritmos especiais de escalonamento linear. O mais eficiente que est√° implementado no programa MOPAC √© o [MOZYME](https://onlinelibrary.wiley.com/doi/abs/10.1002/%28SICI%291097-461X%281996%2958%3A2%3C133%3A%3AAID-QUA2%3E3.0.CO%3B2-Z).\n",
        "\n",
        "O algoritmo MOZYME √© ativado pela keyword *[MOZYME](http://openmopac.net/manual/index.html)* no input do c√°lculo.\n",
        "\n",
        "Mas, antes de realizar qualquer c√°lculo qu√¢ntico de biomol√©culas, √© interessante dar uma conferida no portal do MOPAC sobre a modelagem de biomol√©culas, pois l√° voc√™ pode encontrar v√°rios direcionamentos, ferramentas e dicas interessantes sobre como preparar e executar esses c√°lculos.\n",
        "\n",
        "Acesse o link a seguir:\n",
        "\n",
        "- <http://openmopac.net/manual/proteins.html>\n",
        "\n",
        "--------\n",
        "#### **Adicionando √°tomos de hidrog√™nio a uma estrutura que vem do PDB**\n",
        "\n",
        "- Proteina: [1UBQ](https://files.rcsb.org/download/1UBQ.pdb)\n",
        "- Proteina: [1NAX](https://files.rcsb.org/download/1NAX.pdb)\n",
        "- <http://openmopac.net/manual/Preparing_a_starting_data_set.html>"
      ],
      "metadata": {
        "id": "fHnowW2HxEJZ"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "!rm *.out *.arc *.mop *.html *.log"
      ],
      "metadata": {
        "id": "qfL9mxuo2cTL",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "c0e9e026-abf1-474e-c3c7-e94d6a9b1a9d"
      },
      "execution_count": 9,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "rm: cannot remove '*.html': No such file or directory\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!wget https://files.rcsb.org/download/1NAX.pdb"
      ],
      "metadata": {
        "id": "TxDsnPW-zw5f",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "2b38fc83-8e42-459b-b9e4-55b075e7e990"
      },
      "execution_count": 10,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "--2024-12-10 13:35:04--  https://files.rcsb.org/download/1NAX.pdb\n",
            "Resolving files.rcsb.org (files.rcsb.org)... 128.6.159.157\n",
            "Connecting to files.rcsb.org (files.rcsb.org)|128.6.159.157|:443... connected.\n",
            "HTTP request sent, awaiting response... 200 OK\n",
            "Length: unspecified [application/octet-stream]\n",
            "Saving to: ‚Äò1NAX.pdb‚Äô\n",
            "\n",
            "1NAX.pdb                [ <=>                ] 194.04K  --.-KB/s    in 0.1s    \n",
            "\n",
            "2024-12-10 13:35:04 (1.66 MB/s) - ‚Äò1NAX.pdb‚Äô saved [198693]\n",
            "\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!cat < 1NAX.pdb\n",
        "#!tail -n 200 1UBQ.pdb"
      ],
      "metadata": {
        "id": "tR-z6EOR0-9P",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "76c6f1fb-3b99-405b-ac6c-42c9bbb839ba"
      },
      "execution_count": 11,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "HEADER    MEMBRANE PROTEIN                        29-NOV-02   1NAX              \n",
            "TITLE     THYROID RECEPTOR BETA1 IN COMPLEX WITH A BETA-SELECTIVE LIGAND        \n",
            "COMPND    MOL_ID: 1;                                                            \n",
            "COMPND   2 MOLECULE: THYROID HORMONE RECEPTOR BETA-1;                           \n",
            "COMPND   3 CHAIN: A;                                                            \n",
            "COMPND   4 FRAGMENT: LIGAND BINDING DOMAIN (RESIDUES 209-460);                  \n",
            "COMPND   5 ENGINEERED: YES                                                      \n",
            "SOURCE    MOL_ID: 1;                                                            \n",
            "SOURCE   2 ORGANISM_SCIENTIFIC: HOMO SAPIENS;                                   \n",
            "SOURCE   3 ORGANISM_COMMON: HUMAN;                                              \n",
            "SOURCE   4 ORGANISM_TAXID: 9606;                                                \n",
            "SOURCE   5 GENE: THRB OR NR1A2 OR ERBA2 OR THR1;                                \n",
            "SOURCE   6 EXPRESSION_SYSTEM: ESCHERICHIA COLI BL21(DE3);                       \n",
            "SOURCE   7 EXPRESSION_SYSTEM_TAXID: 469008;                                     \n",
            "SOURCE   8 EXPRESSION_SYSTEM_STRAIN: BL21 (DE3);                                \n",
            "SOURCE   9 EXPRESSION_SYSTEM_VECTOR_TYPE: PLASMID;                              \n",
            "SOURCE  10 EXPRESSION_SYSTEM_PLASMID: PET28A                                    \n",
            "KEYWDS    NUCLEAR RECEPTOR, THYROID RECEPTOR, COMPLEX, MEMBRANE PROTEIN         \n",
            "EXPDTA    X-RAY DIFFRACTION                                                     \n",
            "AUTHOR    L.YE,Y.L.LI,K.MELLSTROM,C.MELLIN,L.G.BLADH,K.KOEHLER,N.GARG,          \n",
            "AUTHOR   2 A.M.GARCIA COLLAZO,C.LITTEN,B.HUSMAN,K.PERSSON,J.LJUNGGREN,G.GROVER, \n",
            "AUTHOR   3 P.G.SLEPH,R.GEORGE,J.MALM                                            \n",
            "REVDAT   5   14-FEB-24 1NAX    1       REMARK                                   \n",
            "REVDAT   4   15-FEB-12 1NAX    1       HETATM VERSN                             \n",
            "REVDAT   3   13-JUL-11 1NAX    1       VERSN                                    \n",
            "REVDAT   2   10-FEB-09 1NAX    1       VERSN  HET                               \n",
            "REVDAT   1   17-JUN-03 1NAX    0                                                \n",
            "JRNL        AUTH   L.YE,Y.L.LI,K.MELLSTROM,C.MELLIN,L.G.BLADH,K.KOEHLER,N.GARG, \n",
            "JRNL        AUTH 2 A.M.GARCIA COLLAZO,C.LITTEN,B.HUSMAN,K.PERSSON,J.LJUNGGREN,  \n",
            "JRNL        AUTH 3 G.GROVER,P.G.SLEPH,R.GEORGE,J.MALM                           \n",
            "JRNL        TITL   THYROID RECEPTOR LIGANDS. 1. AGONIST LIGANDS SELECTIVE FOR   \n",
            "JRNL        TITL 2 THE THYROID RECEPTOR BETA1.                                  \n",
            "JRNL        REF    J.MED.CHEM.                   V.  46  1580 2003              \n",
            "JRNL        REFN                   ISSN 0022-2623                               \n",
            "JRNL        PMID   12699376                                                     \n",
            "JRNL        DOI    10.1021/JM021080F                                            \n",
            "REMARK   2                                                                      \n",
            "REMARK   2 RESOLUTION.    2.70 ANGSTROMS.                                       \n",
            "REMARK   3                                                                      \n",
            "REMARK   3 REFINEMENT.                                                          \n",
            "REMARK   3   PROGRAM     : CNX 2000.1                                           \n",
            "REMARK   3   AUTHORS     : BRUNGER,ADAMS,CLORE,DELANO,GROS,GROSSE-              \n",
            "REMARK   3               : KUNSTLEVE,JIANG,KUSZEWSKI,NILGES,PANNU,              \n",
            "REMARK   3               : READ,RICE,SIMONSON,WARREN,MOLECULAR                  \n",
            "REMARK   3               : SIMULATIONS (BADGER,BERARD,KUMAR,SZALMA,             \n",
            "REMARK   3               : YIP,DZAKULA)                                         \n",
            "REMARK   3                                                                      \n",
            "REMARK   3  DATA USED IN REFINEMENT.                                            \n",
            "REMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 2.70                           \n",
            "REMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : 38.26                          \n",
            "REMARK   3   DATA CUTOFF            (SIGMA(F)) : 0.000                          \n",
            "REMARK   3   DATA CUTOFF HIGH         (ABS(F)) : NULL                           \n",
            "REMARK   3   DATA CUTOFF LOW          (ABS(F)) : NULL                           \n",
            "REMARK   3   COMPLETENESS (WORKING+TEST)   (%) : 96.8                           \n",
            "REMARK   3   NUMBER OF REFLECTIONS             : 7643                           \n",
            "REMARK   3                                                                      \n",
            "REMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \n",
            "REMARK   3   CROSS-VALIDATION METHOD          : THROUGHOUT                      \n",
            "REMARK   3   FREE R VALUE TEST SET SELECTION  : RANDOM                          \n",
            "REMARK   3   R VALUE     (WORKING + TEST SET) : 0.196                           \n",
            "REMARK   3   R VALUE            (WORKING SET) : 0.196                           \n",
            "REMARK   3   FREE R VALUE                     : 0.237                           \n",
            "REMARK   3   FREE R VALUE TEST SET SIZE   (%) : 5.100                           \n",
            "REMARK   3   FREE R VALUE TEST SET COUNT      : 390                             \n",
            "REMARK   3   ESTIMATED ERROR OF FREE R VALUE  : 0.012                           \n",
            "REMARK   3                                                                      \n",
            "REMARK   3  FIT/AGREEMENT OF MODEL WITH ALL DATA.                               \n",
            "REMARK   3   R VALUE     (WORKING + TEST SET, NO CUTOFF) : NULL                 \n",
            "REMARK   3   R VALUE            (WORKING SET, NO CUTOFF) : NULL                 \n",
            "REMARK   3   FREE R VALUE                    (NO CUTOFF) : NULL                 \n",
            "REMARK   3   FREE R VALUE TEST SET SIZE   (%, NO CUTOFF) : NULL                 \n",
            "REMARK   3   FREE R VALUE TEST SET COUNT     (NO CUTOFF) : NULL                 \n",
            "REMARK   3   ESTIMATED ERROR OF FREE R VALUE (NO CUTOFF) : NULL                 \n",
            "REMARK   3   TOTAL NUMBER OF REFLECTIONS     (NO CUTOFF) : 7643                 \n",
            "REMARK   3                                                                      \n",
            "REMARK   3  FIT IN THE HIGHEST RESOLUTION BIN.                                  \n",
            "REMARK   3   TOTAL NUMBER OF BINS USED           : 6                            \n",
            "REMARK   3   BIN RESOLUTION RANGE HIGH       (A) : 2.70                         \n",
            "REMARK   3   BIN RESOLUTION RANGE LOW        (A) : 2.87                         \n",
            "REMARK   3   BIN COMPLETENESS (WORKING+TEST) (%) : 99.00                        \n",
            "REMARK   3   REFLECTIONS IN BIN    (WORKING SET) : 1210                         \n",
            "REMARK   3   BIN R VALUE           (WORKING SET) : 0.2210                       \n",
            "REMARK   3   BIN FREE R VALUE                    : 0.3010                       \n",
            "REMARK   3   BIN FREE R VALUE TEST SET SIZE  (%) : 4.90                         \n",
            "REMARK   3   BIN FREE R VALUE TEST SET COUNT     : 62                           \n",
            "REMARK   3   ESTIMATED ERROR OF BIN FREE R VALUE : 0.038                        \n",
            "REMARK   3                                                                      \n",
            "REMARK   3  NUMBER OF NON-HYDROGEN ATOMS USED IN REFINEMENT.                    \n",
            "REMARK   3   PROTEIN ATOMS            : 1981                                    \n",
            "REMARK   3   NUCLEIC ACID ATOMS       : 0                                       \n",
            "REMARK   3   HETEROGEN ATOMS          : 23                                      \n",
            "REMARK   3   SOLVENT ATOMS            : 33                                      \n",
            "REMARK   3                                                                      \n",
            "REMARK   3  B VALUES.                                                           \n",
            "REMARK   3   FROM WILSON PLOT           (A**2) : 37.90                          \n",
            "REMARK   3   MEAN B VALUE      (OVERALL, A**2) : 34.70                          \n",
            "REMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \n",
            "REMARK   3    B11 (A**2) : 1.54000                                              \n",
            "REMARK   3    B22 (A**2) : 6.98000                                              \n",
            "REMARK   3    B33 (A**2) : -8.52000                                             \n",
            "REMARK   3    B12 (A**2) : 0.00000                                              \n",
            "REMARK   3    B13 (A**2) : 0.00000                                              \n",
            "REMARK   3    B23 (A**2) : 0.00000                                              \n",
            "REMARK   3                                                                      \n",
            "REMARK   3  ESTIMATED COORDINATE ERROR.                                         \n",
            "REMARK   3   ESD FROM LUZZATI PLOT        (A) : NULL                            \n",
            "REMARK   3   ESD FROM SIGMAA              (A) : NULL                            \n",
            "REMARK   3   LOW RESOLUTION CUTOFF        (A) : NULL                            \n",
            "REMARK   3                                                                      \n",
            "REMARK   3  CROSS-VALIDATED ESTIMATED COORDINATE ERROR.                         \n",
            "REMARK   3   ESD FROM C-V LUZZATI PLOT    (A) : 0.37                            \n",
            "REMARK   3   ESD FROM C-V SIGMAA          (A) : 0.31                            \n",
            "REMARK   3                                                                      \n",
            "REMARK   3  RMS DEVIATIONS FROM IDEAL VALUES.                                   \n",
            "REMARK   3   BOND LENGTHS                 (A) : 0.007                           \n",
            "REMARK   3   BOND ANGLES            (DEGREES) : 1.100                           \n",
            "REMARK   3   DIHEDRAL ANGLES        (DEGREES) : 19.80                           \n",
            "REMARK   3   IMPROPER ANGLES        (DEGREES) : 0.850                           \n",
            "REMARK   3                                                                      \n",
            "REMARK   3  ISOTROPIC THERMAL MODEL : RESTRAINED                                \n",
            "REMARK   3                                                                      \n",
            "REMARK   3  ISOTROPIC THERMAL FACTOR RESTRAINTS.    RMS    SIGMA                \n",
            "REMARK   3   MAIN-CHAIN BOND              (A**2) : 1.250 ; 1.000                \n",
            "REMARK   3   MAIN-CHAIN ANGLE             (A**2) : 2.080 ; 1.200                \n",
            "REMARK   3   SIDE-CHAIN BOND              (A**2) : 1.970 ; 1.200                \n",
            "REMARK   3   SIDE-CHAIN ANGLE             (A**2) : 2.930 ; 1.500                \n",
            "REMARK   3                                                                      \n",
            "REMARK   3  BULK SOLVENT MODELING.                                              \n",
            "REMARK   3   METHOD USED : FLAT MODEL                                           \n",
            "REMARK   3   KSOL        : 0.38                                                 \n",
            "REMARK   3   BSOL        : 34.02                                                \n",
            "REMARK   3                                                                      \n",
            "REMARK   3  NCS MODEL : NULL                                                    \n",
            "REMARK   3                                                                      \n",
            "REMARK   3  NCS RESTRAINTS.                         RMS   SIGMA/WEIGHT          \n",
            "REMARK   3   GROUP  1  POSITIONAL            (A) : NULL  ; NULL                 \n",
            "REMARK   3   GROUP  1  B-FACTOR           (A**2) : NULL  ; NULL                 \n",
            "REMARK   3                                                                      \n",
            "REMARK   3  PARAMETER FILE  1  : PROTEIN_REP.PARAM                              \n",
            "REMARK   3  PARAMETER FILE  2  : WATER.PARAM                                    \n",
            "REMARK   3  PARAMETER FILE  3  : INH.PAR                                        \n",
            "REMARK   3  PARAMETER FILE  4  : ION.PARAM                                      \n",
            "REMARK   3  PARAMETER FILE  5  : NULL                                           \n",
            "REMARK   3  TOPOLOGY FILE  1   : PROTEIN.TOP                                    \n",
            "REMARK   3  TOPOLOGY FILE  2   : WATER.TOP                                      \n",
            "REMARK   3  TOPOLOGY FILE  3   : INH.PAR                                        \n",
            "REMARK   3  TOPOLOGY FILE  4   : ION.TOP                                        \n",
            "REMARK   3  TOPOLOGY FILE  5   : NULL                                           \n",
            "REMARK   3                                                                      \n",
            "REMARK   3  OTHER REFINEMENT REMARKS: NULL                                      \n",
            "REMARK   4                                                                      \n",
            "REMARK   4 1NAX COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \n",
            "REMARK 100                                                                      \n",
            "REMARK 100 THIS ENTRY HAS BEEN PROCESSED BY RCSB ON 12-DEC-02.                  \n",
            "REMARK 100 THE DEPOSITION ID IS D_1000017725.                                   \n",
            "REMARK 200                                                                      \n",
            "REMARK 200 EXPERIMENTAL DETAILS                                                 \n",
            "REMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \n",
            "REMARK 200  DATE OF DATA COLLECTION        : 20-SEP-02                          \n",
            "REMARK 200  TEMPERATURE           (KELVIN) : 100                                \n",
            "REMARK 200  PH                             : 8                                  \n",
            "REMARK 200  NUMBER OF CRYSTALS USED        : 1                                  \n",
            "REMARK 200                                                                      \n",
            "REMARK 200  SYNCHROTRON              (Y/N) : N                                  \n",
            "REMARK 200  RADIATION SOURCE               : ROTATING ANODE                     \n",
            "REMARK 200  BEAMLINE                       : NULL                               \n",
            "REMARK 200  X-RAY GENERATOR MODEL          : RIGAKU RU300                       \n",
            "REMARK 200  MONOCHROMATIC OR LAUE    (M/L) : M                                  \n",
            "REMARK 200  WAVELENGTH OR RANGE        (A) : 1.5418                             \n",
            "REMARK 200  MONOCHROMATOR                  : NULL                               \n",
            "REMARK 200  OPTICS                         : OSMIC MIRRORS                      \n",
            "REMARK 200                                                                      \n",
            "REMARK 200  DETECTOR TYPE                  : AREA DETECTOR                      \n",
            "REMARK 200  DETECTOR MANUFACTURER          : MARRESEARCH                        \n",
            "REMARK 200  INTENSITY-INTEGRATION SOFTWARE : MOSFLM                             \n",
            "REMARK 200  DATA SCALING SOFTWARE          : SCALA, CCP4 (SCALA)                \n",
            "REMARK 200                                                                      \n",
            "REMARK 200  NUMBER OF UNIQUE REFLECTIONS   : NULL                               \n",
            "REMARK 200  RESOLUTION RANGE HIGH      (A) : 2.500                              \n",
            "REMARK 200  RESOLUTION RANGE LOW       (A) : 50.000                             \n",
            "REMARK 200  REJECTION CRITERIA  (SIGMA(I)) : -3.000                             \n",
            "REMARK 200                                                                      \n",
            "REMARK 200 OVERALL.                                                             \n",
            "REMARK 200  COMPLETENESS FOR RANGE     (%) : 97.7                               \n",
            "REMARK 200  DATA REDUNDANCY                : NULL                               \n",
            "REMARK 200  R MERGE                    (I) : NULL                               \n",
            "REMARK 200  R SYM                      (I) : 0.08700                            \n",
            "REMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : 7.9000                             \n",
            "REMARK 200                                                                      \n",
            "REMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \n",
            "REMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : 2.70                     \n",
            "REMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : 2.85                     \n",
            "REMARK 200  COMPLETENESS FOR SHELL     (%) : 99.2                               \n",
            "REMARK 200  DATA REDUNDANCY IN SHELL       : NULL                               \n",
            "REMARK 200  R MERGE FOR SHELL          (I) : NULL                               \n",
            "REMARK 200  R SYM FOR SHELL            (I) : 0.29000                            \n",
            "REMARK 200  <I/SIGMA(I)> FOR SHELL         : 2.600                              \n",
            "REMARK 200                                                                      \n",
            "REMARK 200 DIFFRACTION PROTOCOL: SINGLE WAVELENGTH                              \n",
            "REMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: FOURIER SYNTHESIS            \n",
            "REMARK 200 SOFTWARE USED: CNX 2000.1                                            \n",
            "REMARK 200 STARTING MODEL: NULL                                                 \n",
            "REMARK 200                                                                      \n",
            "REMARK 200 REMARK: NULL                                                         \n",
            "REMARK 280                                                                      \n",
            "REMARK 280 CRYSTAL                                                              \n",
            "REMARK 280 SOLVENT CONTENT, VS   (%): 49.19                                     \n",
            "REMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): 2.42                     \n",
            "REMARK 280                                                                      \n",
            "REMARK 280 CRYSTALLIZATION CONDITIONS: AMMONIUM SULPHATE, TRIS, NDSB-201, PH    \n",
            "REMARK 280  8, VAPOR DIFFUSION, HANGING DROP, TEMPERATURE 298K                  \n",
            "REMARK 290                                                                      \n",
            "REMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \n",
            "REMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: C 2 2 21                         \n",
            "REMARK 290                                                                      \n",
            "REMARK 290      SYMOP   SYMMETRY                                                \n",
            "REMARK 290     NNNMMM   OPERATOR                                                \n",
            "REMARK 290       1555   X,Y,Z                                                   \n",
            "REMARK 290       2555   -X,-Y,Z+1/2                                             \n",
            "REMARK 290       3555   -X,Y,-Z+1/2                                             \n",
            "REMARK 290       4555   X,-Y,-Z                                                 \n",
            "REMARK 290       5555   X+1/2,Y+1/2,Z                                           \n",
            "REMARK 290       6555   -X+1/2,-Y+1/2,Z+1/2                                     \n",
            "REMARK 290       7555   -X+1/2,Y+1/2,-Z+1/2                                     \n",
            "REMARK 290       8555   X+1/2,-Y+1/2,-Z                                         \n",
            "REMARK 290                                                                      \n",
            "REMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \n",
            "REMARK 290           MMM -> TRANSLATION VECTOR                                  \n",
            "REMARK 290                                                                      \n",
            "REMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \n",
            "REMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \n",
            "REMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \n",
            "REMARK 290 RELATED MOLECULES.                                                   \n",
            "REMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \n",
            "REMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \n",
            "REMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \n",
            "REMARK 290   SMTRY1   2 -1.000000  0.000000  0.000000        0.00000            \n",
            "REMARK 290   SMTRY2   2  0.000000 -1.000000  0.000000        0.00000            \n",
            "REMARK 290   SMTRY3   2  0.000000  0.000000  1.000000       33.44800            \n",
            "REMARK 290   SMTRY1   3 -1.000000  0.000000  0.000000        0.00000            \n",
            "REMARK 290   SMTRY2   3  0.000000  1.000000  0.000000        0.00000            \n",
            "REMARK 290   SMTRY3   3  0.000000  0.000000 -1.000000       33.44800            \n",
            "REMARK 290   SMTRY1   4  1.000000  0.000000  0.000000        0.00000            \n",
            "REMARK 290   SMTRY2   4  0.000000 -1.000000  0.000000        0.00000            \n",
            "REMARK 290   SMTRY3   4  0.000000  0.000000 -1.000000        0.00000            \n",
            "REMARK 290   SMTRY1   5  1.000000  0.000000  0.000000       38.26000            \n",
            "REMARK 290   SMTRY2   5  0.000000  1.000000  0.000000       53.87150            \n",
            "REMARK 290   SMTRY3   5  0.000000  0.000000  1.000000        0.00000            \n",
            "REMARK 290   SMTRY1   6 -1.000000  0.000000  0.000000       38.26000            \n",
            "REMARK 290   SMTRY2   6  0.000000 -1.000000  0.000000       53.87150            \n",
            "REMARK 290   SMTRY3   6  0.000000  0.000000  1.000000       33.44800            \n",
            "REMARK 290   SMTRY1   7 -1.000000  0.000000  0.000000       38.26000            \n",
            "REMARK 290   SMTRY2   7  0.000000  1.000000  0.000000       53.87150            \n",
            "REMARK 290   SMTRY3   7  0.000000  0.000000 -1.000000       33.44800            \n",
            "REMARK 290   SMTRY1   8  1.000000  0.000000  0.000000       38.26000            \n",
            "REMARK 290   SMTRY2   8  0.000000 -1.000000  0.000000       53.87150            \n",
            "REMARK 290   SMTRY3   8  0.000000  0.000000 -1.000000        0.00000            \n",
            "REMARK 290                                                                      \n",
            "REMARK 290 REMARK: NULL                                                         \n",
            "REMARK 300                                                                      \n",
            "REMARK 300 BIOMOLECULE: 1, 2                                                    \n",
            "REMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \n",
            "REMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \n",
            "REMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \n",
            "REMARK 300 BURIED SURFACE AREA.                                                 \n",
            "REMARK 350                                                                      \n",
            "REMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \n",
            "REMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \n",
            "REMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \n",
            "REMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \n",
            "REMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \n",
            "REMARK 350                                                                      \n",
            "REMARK 350 BIOMOLECULE: 1                                                       \n",
            "REMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: MONOMERIC                         \n",
            "REMARK 350 SOFTWARE DETERMINED QUATERNARY STRUCTURE: MONOMERIC                  \n",
            "REMARK 350 SOFTWARE USED: PQS                                                   \n",
            "REMARK 350 APPLY THE FOLLOWING TO CHAINS: A                                     \n",
            "REMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \n",
            "REMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \n",
            "REMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \n",
            "REMARK 350                                                                      \n",
            "REMARK 350 BIOMOLECULE: 2                                                       \n",
            "REMARK 350 SOFTWARE DETERMINED QUATERNARY STRUCTURE: DIMERIC                    \n",
            "REMARK 350 SOFTWARE USED: PISA                                                  \n",
            "REMARK 350 TOTAL BURIED SURFACE AREA: 3620 ANGSTROM**2                          \n",
            "REMARK 350 SURFACE AREA OF THE COMPLEX: 22310 ANGSTROM**2                       \n",
            "REMARK 350 CHANGE IN SOLVENT FREE ENERGY: -34.0 KCAL/MOL                        \n",
            "REMARK 350 APPLY THE FOLLOWING TO CHAINS: A                                     \n",
            "REMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \n",
            "REMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \n",
            "REMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \n",
            "REMARK 350   BIOMT1   2 -1.000000  0.000000  0.000000        0.00000            \n",
            "REMARK 350   BIOMT2   2  0.000000  1.000000  0.000000        0.00000            \n",
            "REMARK 350   BIOMT3   2  0.000000  0.000000 -1.000000       33.44800            \n",
            "REMARK 465                                                                      \n",
            "REMARK 465 MISSING RESIDUES                                                     \n",
            "REMARK 465 THE FOLLOWING RESIDUES WERE NOT LOCATED IN THE                       \n",
            "REMARK 465 EXPERIMENT. (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN               \n",
            "REMARK 465 IDENTIFIER; SSSEQ=SEQUENCE NUMBER; I=INSERTION CODE.)                \n",
            "REMARK 465                                                                      \n",
            "REMARK 465   M RES C SSSEQI                                                     \n",
            "REMARK 465     GLY A   209                                                      \n",
            "REMARK 465     HIS A   210                                                      \n",
            "REMARK 500                                                                      \n",
            "REMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \n",
            "REMARK 500 SUBTOPIC: TORSION ANGLES                                             \n",
            "REMARK 500                                                                      \n",
            "REMARK 500 TORSION ANGLES OUTSIDE THE EXPECTED RAMACHANDRAN REGIONS:            \n",
            "REMARK 500 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \n",
            "REMARK 500 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                             \n",
            "REMARK 500                                                                      \n",
            "REMARK 500 STANDARD TABLE:                                                      \n",
            "REMARK 500 FORMAT:(10X,I3,1X,A3,1X,A1,I4,A1,4X,F7.2,3X,F7.2)                    \n",
            "REMARK 500                                                                      \n",
            "REMARK 500 EXPECTED VALUES: GJ KLEYWEGT AND TA JONES (1996). PHI/PSI-           \n",
            "REMARK 500 CHOLOGY: RAMACHANDRAN REVISITED. STRUCTURE 4, 1395 - 1400            \n",
            "REMARK 500                                                                      \n",
            "REMARK 500  M RES CSSEQI        PSI       PHI                                   \n",
            "REMARK 500    ALA A 234       50.36     37.35                                   \n",
            "REMARK 500    GLU A 324      -70.51    -62.66                                   \n",
            "REMARK 500    ASN A 331       19.93     54.16                                   \n",
            "REMARK 500    GLU A 333      -11.60   -152.17                                   \n",
            "REMARK 500    HIS A 413       57.29    -90.51                                   \n",
            "REMARK 500                                                                      \n",
            "REMARK 500 REMARK: NULL                                                         \n",
            "REMARK 800                                                                      \n",
            "REMARK 800 SITE                                                                 \n",
            "REMARK 800 SITE_IDENTIFIER: AC1                                                 \n",
            "REMARK 800 EVIDENCE_CODE: SOFTWARE                                              \n",
            "REMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE IH5 A 501                 \n",
            "REMARK 900                                                                      \n",
            "REMARK 900 RELATED ENTRIES                                                      \n",
            "REMARK 900 RELATED ID: 1NAV   RELATED DB: PDB                                   \n",
            "REMARK 900 THYROID RECEPTOR ALPHA IN COMPLEX WITH AN AGONIST SELECTIVE FOR      \n",
            "REMARK 900 THYROID RECEPTOR BETA1                                               \n",
            "DBREF  1NAX A  209   460  UNP    P10828   THB1_HUMAN     209    460             \n",
            "SEQRES   1 A  252  GLY HIS LYS PRO GLU PRO THR ASP GLU GLU TRP GLU LEU          \n",
            "SEQRES   2 A  252  ILE LYS THR VAL THR GLU ALA HIS VAL ALA THR ASN ALA          \n",
            "SEQRES   3 A  252  GLN GLY SER HIS TRP LYS GLN LYS ARG LYS PHE LEU PRO          \n",
            "SEQRES   4 A  252  GLU ASP ILE GLY GLN ALA PRO ILE VAL ASN ALA PRO GLU          \n",
            "SEQRES   5 A  252  GLY GLY LYS VAL ASP LEU GLU ALA PHE SER HIS PHE THR          \n",
            "SEQRES   6 A  252  LYS ILE ILE THR PRO ALA ILE THR ARG VAL VAL ASP PHE          \n",
            "SEQRES   7 A  252  ALA LYS LYS LEU PRO MET PHE CYS GLU LEU PRO CYS GLU          \n",
            "SEQRES   8 A  252  ASP GLN ILE ILE LEU LEU LYS GLY CYS CYS MET GLU ILE          \n",
            "SEQRES   9 A  252  MET SER LEU ARG ALA ALA VAL ARG TYR ASP PRO GLU SER          \n",
            "SEQRES  10 A  252  GLU THR LEU THR LEU ASN GLY GLU MET ALA VAL THR ARG          \n",
            "SEQRES  11 A  252  GLY GLN LEU LYS ASN GLY GLY LEU GLY VAL VAL SER ASP          \n",
            "SEQRES  12 A  252  ALA ILE PHE ASP LEU GLY MET SER LEU SER SER PHE ASN          \n",
            "SEQRES  13 A  252  LEU ASP ASP THR GLU VAL ALA LEU LEU GLN ALA VAL LEU          \n",
            "SEQRES  14 A  252  LEU MET SER SER ASP ARG PRO GLY LEU ALA CYS VAL GLU          \n",
            "SEQRES  15 A  252  ARG ILE GLU LYS TYR GLN ASP SER PHE LEU LEU ALA PHE          \n",
            "SEQRES  16 A  252  GLU HIS TYR ILE ASN TYR ARG LYS HIS HIS VAL THR HIS          \n",
            "SEQRES  17 A  252  PHE TRP PRO LYS LEU LEU MET LYS VAL THR ASP LEU ARG          \n",
            "SEQRES  18 A  252  MET ILE GLY ALA CYS HIS ALA SER ARG PHE LEU HIS MET          \n",
            "SEQRES  19 A  252  LYS VAL GLU CYS PRO THR GLU LEU PHE PRO PRO LEU PHE          \n",
            "SEQRES  20 A  252  LEU GLU VAL PHE GLU                                          \n",
            "HET    IH5  A 501      23                                                       \n",
            "HETNAM     IH5 {3,5-DICHLORO-4-[4-HYDROXY-3-(PROPAN-2-YL)                       \n",
            "HETNAM   2 IH5  PHENOXY]PHENYL}ACETIC ACID                                      \n",
            "HETSYN     IH5 3,5-DICHLORO-4-[(4-HYDROXY-3-ISOPROPYLPHENOXY)                   \n",
            "HETSYN   2 IH5  PHENYL]ACETIC ACID                                              \n",
            "FORMUL   2  IH5    C17 H16 CL2 O4                                               \n",
            "FORMUL   3  HOH   *33(H2 O)                                                     \n",
            "HELIX    1   1 THR A  215  THR A  232  1                                  18    \n",
            "HELIX    2   2 HIS A  238  ARG A  243  1                                   6    \n",
            "HELIX    3   3 ASP A  265  LYS A  288  1                                  24    \n",
            "HELIX    4   4 LEU A  290  GLU A  295  1                                   6    \n",
            "HELIX    5   5 PRO A  297  VAL A  319  1                                  23    \n",
            "HELIX    6   6 THR A  337  GLY A  344  1                                   8    \n",
            "HELIX    7   7 GLY A  347  SER A  361  1                                  15    \n",
            "HELIX    8   8 ASP A  366  MET A  379  1                                  14    \n",
            "HELIX    9   9 CYS A  388  LYS A  411  1                                  24    \n",
            "HELIX   10  10 HIS A  416  CYS A  446  1                                  31    \n",
            "HELIX   11  11 PRO A  447  PHE A  451  5                                   5    \n",
            "HELIX   12  12 PRO A  452  GLU A  460  1                                   9    \n",
            "SHEET    1   A 4 LYS A 244  PHE A 245  0                                        \n",
            "SHEET    2   A 4 MET A 334  VAL A 336  1  O  ALA A 335   N  LYS A 244           \n",
            "SHEET    3   A 4 THR A 327  LEU A 330 -1  N  LEU A 330   O  MET A 334           \n",
            "SHEET    4   A 4 TYR A 321  ASP A 322 -1  N  ASP A 322   O  THR A 327           \n",
            "SITE     1 AC1 15 ILE A 275  ILE A 276  ALA A 279  ARG A 282                    \n",
            "SITE     2 AC1 15 MET A 313  LEU A 330  ASN A 331  GLY A 344                    \n",
            "SITE     3 AC1 15 LEU A 346  ILE A 353  HIS A 435  MET A 442                    \n",
            "SITE     4 AC1 15 PHE A 455  HOH A 606  HOH A 607                               \n",
            "CRYST1   76.520  107.743   66.896  90.00  90.00  90.00 C 2 2 21      8          \n",
            "ORIGX1      1.000000  0.000000  0.000000        0.00000                         \n",
            "ORIGX2      0.000000  1.000000  0.000000        0.00000                         \n",
            "ORIGX3      0.000000  0.000000  1.000000        0.00000                         \n",
            "SCALE1      0.013068  0.000000  0.000000        0.00000                         \n",
            "SCALE2      0.000000  0.009281  0.000000        0.00000                         \n",
            "SCALE3      0.000000  0.000000  0.014949        0.00000                         \n",
            "ATOM      1  N   LYS A 211      23.762  -3.506   9.387  1.00 54.42           N  \n",
            "ATOM      2  CA  LYS A 211      23.345  -2.328  10.202  1.00 52.79           C  \n",
            "ATOM      3  C   LYS A 211      24.494  -1.837  11.070  1.00 51.27           C  \n",
            "ATOM      4  O   LYS A 211      24.771  -2.400  12.129  1.00 50.89           O  \n",
            "ATOM      5  CB  LYS A 211      22.153  -2.690  11.086  1.00 53.79           C  \n",
            "ATOM      6  CG  LYS A 211      20.884  -2.996  10.315  1.00 55.40           C  \n",
            "ATOM      7  CD  LYS A 211      19.698  -3.131  11.256  1.00 59.27           C  \n",
            "ATOM      8  CE  LYS A 211      18.388  -3.267  10.493  1.00 60.54           C  \n",
            "ATOM      9  NZ  LYS A 211      18.357  -4.488   9.634  1.00 61.48           N  \n",
            "ATOM     10  N   PRO A 212      25.169  -0.763  10.634  1.00 50.17           N  \n",
            "ATOM     11  CA  PRO A 212      26.301  -0.177  11.356  1.00 49.35           C  \n",
            "ATOM     12  C   PRO A 212      25.986   0.277  12.776  1.00 50.11           C  \n",
            "ATOM     13  O   PRO A 212      24.950   0.896  13.030  1.00 50.85           O  \n",
            "ATOM     14  CB  PRO A 212      26.709   0.991  10.464  1.00 48.11           C  \n",
            "ATOM     15  CG  PRO A 212      25.408   1.404   9.853  1.00 48.65           C  \n",
            "ATOM     16  CD  PRO A 212      24.792   0.081   9.486  1.00 49.36           C  \n",
            "ATOM     17  N   GLU A 213      26.881  -0.043  13.704  1.00 49.60           N  \n",
            "ATOM     18  CA  GLU A 213      26.718   0.370  15.087  1.00 49.47           C  \n",
            "ATOM     19  C   GLU A 213      27.501   1.670  15.263  1.00 50.19           C  \n",
            "ATOM     20  O   GLU A 213      28.323   2.025  14.419  1.00 50.70           O  \n",
            "ATOM     21  CB  GLU A 213      27.229  -0.720  16.029  1.00 49.63           C  \n",
            "ATOM     22  CG  GLU A 213      26.197  -1.801  16.306  1.00 50.33           C  \n",
            "ATOM     23  CD  GLU A 213      26.719  -2.919  17.199  1.00 51.08           C  \n",
            "ATOM     24  OE1 GLU A 213      27.627  -2.664  18.021  1.00 50.61           O  \n",
            "ATOM     25  OE2 GLU A 213      26.203  -4.053  17.091  1.00 51.29           O  \n",
            "ATOM     26  N   PRO A 214      27.268   2.399  16.364  1.00 50.48           N  \n",
            "ATOM     27  CA  PRO A 214      27.990   3.660  16.566  1.00 49.84           C  \n",
            "ATOM     28  C   PRO A 214      29.523   3.604  16.517  1.00 49.50           C  \n",
            "ATOM     29  O   PRO A 214      30.151   2.674  17.028  1.00 49.58           O  \n",
            "ATOM     30  CB  PRO A 214      27.453   4.148  17.913  1.00 48.91           C  \n",
            "ATOM     31  CG  PRO A 214      27.110   2.887  18.618  1.00 48.57           C  \n",
            "ATOM     32  CD  PRO A 214      26.458   2.064  17.549  1.00 49.76           C  \n",
            "ATOM     33  N   THR A 215      30.111   4.615  15.884  1.00 48.25           N  \n",
            "ATOM     34  CA  THR A 215      31.559   4.728  15.767  1.00 46.92           C  \n",
            "ATOM     35  C   THR A 215      32.137   5.192  17.107  1.00 46.93           C  \n",
            "ATOM     36  O   THR A 215      31.421   5.296  18.095  1.00 46.33           O  \n",
            "ATOM     37  CB  THR A 215      31.944   5.755  14.693  1.00 45.80           C  \n",
            "ATOM     38  OG1 THR A 215      31.542   7.062  15.119  1.00 46.75           O  \n",
            "ATOM     39  CG2 THR A 215      31.258   5.431  13.378  1.00 44.74           C  \n",
            "ATOM     40  N   ASP A 216      33.433   5.465  17.145  1.00 47.36           N  \n",
            "ATOM     41  CA  ASP A 216      34.048   5.923  18.382  1.00 49.18           C  \n",
            "ATOM     42  C   ASP A 216      33.617   7.353  18.672  1.00 48.77           C  \n",
            "ATOM     43  O   ASP A 216      33.319   7.705  19.814  1.00 47.94           O  \n",
            "ATOM     44  CB  ASP A 216      35.575   5.871  18.285  1.00 50.88           C  \n",
            "ATOM     45  CG  ASP A 216      36.105   4.459  18.206  1.00 54.05           C  \n",
            "ATOM     46  OD1 ASP A 216      35.717   3.633  19.063  1.00 54.25           O  \n",
            "ATOM     47  OD2 ASP A 216      36.915   4.178  17.294  1.00 55.27           O  \n",
            "ATOM     48  N   GLU A 217      33.593   8.174  17.628  1.00 47.80           N  \n",
            "ATOM     49  CA  GLU A 217      33.211   9.565  17.768  1.00 46.88           C  \n",
            "ATOM     50  C   GLU A 217      31.746   9.659  18.194  1.00 45.48           C  \n",
            "ATOM     51  O   GLU A 217      31.384  10.495  19.024  1.00 46.07           O  \n",
            "ATOM     52  CB  GLU A 217      33.441  10.307  16.448  1.00 48.59           C  \n",
            "ATOM     53  CG  GLU A 217      34.102  11.668  16.624  1.00 52.57           C  \n",
            "ATOM     54  CD  GLU A 217      34.416  12.358  15.306  1.00 55.06           C  \n",
            "ATOM     55  OE1 GLU A 217      35.168  11.777  14.494  1.00 55.40           O  \n",
            "ATOM     56  OE2 GLU A 217      33.916  13.486  15.087  1.00 56.59           O  \n",
            "ATOM     57  N   GLU A 218      30.906   8.792  17.636  1.00 43.23           N  \n",
            "ATOM     58  CA  GLU A 218      29.490   8.786  17.976  1.00 40.39           C  \n",
            "ATOM     59  C   GLU A 218      29.251   8.337  19.416  1.00 39.86           C  \n",
            "ATOM     60  O   GLU A 218      28.323   8.810  20.066  1.00 38.85           O  \n",
            "ATOM     61  CB  GLU A 218      28.721   7.884  17.015  1.00 39.82           C  \n",
            "ATOM     62  CG  GLU A 218      28.690   8.398  15.587  1.00 41.02           C  \n",
            "ATOM     63  CD  GLU A 218      27.895   7.494  14.660  1.00 42.54           C  \n",
            "ATOM     64  OE1 GLU A 218      28.179   6.277  14.642  1.00 43.88           O  \n",
            "ATOM     65  OE2 GLU A 218      26.995   7.997  13.947  1.00 41.26           O  \n",
            "ATOM     66  N   TRP A 219      30.091   7.430  19.911  1.00 40.14           N  \n",
            "ATOM     67  CA  TRP A 219      29.972   6.931  21.280  1.00 40.70           C  \n",
            "ATOM     68  C   TRP A 219      30.270   8.038  22.272  1.00 41.77           C  \n",
            "ATOM     69  O   TRP A 219      29.678   8.094  23.352  1.00 42.48           O  \n",
            "ATOM     70  CB  TRP A 219      30.934   5.768  21.529  1.00 40.32           C  \n",
            "ATOM     71  CG  TRP A 219      30.348   4.411  21.266  1.00 39.71           C  \n",
            "ATOM     72  CD1 TRP A 219      30.899   3.412  20.515  1.00 38.36           C  \n",
            "ATOM     73  CD2 TRP A 219      29.112   3.894  21.775  1.00 39.52           C  \n",
            "ATOM     74  NE1 TRP A 219      30.084   2.307  20.523  1.00 38.79           N  \n",
            "ATOM     75  CE2 TRP A 219      28.982   2.574  21.290  1.00 39.44           C  \n",
            "ATOM     76  CE3 TRP A 219      28.103   4.417  22.597  1.00 40.47           C  \n",
            "ATOM     77  CZ2 TRP A 219      27.880   1.767  21.599  1.00 39.29           C  \n",
            "ATOM     78  CZ3 TRP A 219      27.005   3.610  22.907  1.00 39.07           C  \n",
            "ATOM     79  CH2 TRP A 219      26.907   2.300  22.407  1.00 39.20           C  \n",
            "ATOM     80  N   GLU A 220      31.197   8.916  21.904  1.00 42.71           N  \n",
            "ATOM     81  CA  GLU A 220      31.561  10.034  22.762  1.00 43.10           C  \n",
            "ATOM     82  C   GLU A 220      30.423  11.046  22.804  1.00 41.61           C  \n",
            "ATOM     83  O   GLU A 220      30.094  11.576  23.865  1.00 40.46           O  \n",
            "ATOM     84  CB  GLU A 220      32.853  10.680  22.260  1.00 46.22           C  \n",
            "ATOM     85  CG  GLU A 220      34.066   9.789  22.463  1.00 52.50           C  \n",
            "ATOM     86  CD  GLU A 220      34.121   9.211  23.879  1.00 57.92           C  \n",
            "ATOM     87  OE1 GLU A 220      34.157  10.009  24.845  1.00 59.39           O  \n",
            "ATOM     88  OE2 GLU A 220      34.119   7.963  24.030  1.00 58.95           O  \n",
            "ATOM     89  N   LEU A 221      29.825  11.307  21.644  1.00 40.04           N  \n",
            "ATOM     90  CA  LEU A 221      28.699  12.225  21.557  1.00 38.47           C  \n",
            "ATOM     91  C   LEU A 221      27.582  11.626  22.409  1.00 38.63           C  \n",
            "ATOM     92  O   LEU A 221      26.991  12.303  23.254  1.00 38.63           O  \n",
            "ATOM     93  CB  LEU A 221      28.222  12.362  20.104  1.00 35.90           C  \n",
            "ATOM     94  CG  LEU A 221      26.991  13.250  19.831  1.00 36.75           C  \n",
            "ATOM     95  CD1 LEU A 221      27.266  14.691  20.272  1.00 32.37           C  \n",
            "ATOM     96  CD2 LEU A 221      26.633  13.211  18.341  1.00 31.62           C  \n",
            "ATOM     97  N   ILE A 222      27.315  10.342  22.185  1.00 38.03           N  \n",
            "ATOM     98  CA  ILE A 222      26.275   9.628  22.912  1.00 36.61           C  \n",
            "ATOM     99  C   ILE A 222      26.451   9.756  24.413  1.00 36.89           C  \n",
            "ATOM    100  O   ILE A 222      25.477   9.938  25.139  1.00 37.95           O  \n",
            "ATOM    101  CB  ILE A 222      26.243   8.118  22.512  1.00 36.81           C  \n",
            "ATOM    102  CG1 ILE A 222      25.615   7.969  21.116  1.00 34.33           C  \n",
            "ATOM    103  CG2 ILE A 222      25.464   7.298  23.545  1.00 34.92           C  \n",
            "ATOM    104  CD1 ILE A 222      25.529   6.541  20.615  1.00 30.11           C  \n",
            "ATOM    105  N   LYS A 223      27.688   9.670  24.885  1.00 37.25           N  \n",
            "ATOM    106  CA  LYS A 223      27.937   9.774  26.317  1.00 37.06           C  \n",
            "ATOM    107  C   LYS A 223      27.670  11.200  26.780  1.00 35.80           C  \n",
            "ATOM    108  O   LYS A 223      27.263  11.436  27.917  1.00 35.04           O  \n",
            "ATOM    109  CB  LYS A 223      29.378   9.373  26.641  1.00 40.39           C  \n",
            "ATOM    110  CG  LYS A 223      29.635   9.144  28.130  1.00 45.22           C  \n",
            "ATOM    111  CD  LYS A 223      31.021   8.544  28.392  1.00 48.69           C  \n",
            "ATOM    112  CE  LYS A 223      32.096   9.610  28.549  1.00 50.73           C  \n",
            "ATOM    113  NZ  LYS A 223      31.945  10.341  29.842  1.00 53.35           N  \n",
            "ATOM    114  N   THR A 224      27.888  12.148  25.879  1.00 35.25           N  \n",
            "ATOM    115  CA  THR A 224      27.672  13.554  26.178  1.00 35.39           C  \n",
            "ATOM    116  C   THR A 224      26.193  13.920  26.358  1.00 34.23           C  \n",
            "ATOM    117  O   THR A 224      25.854  14.588  27.331  1.00 33.71           O  \n",
            "ATOM    118  CB  THR A 224      28.280  14.446  25.080  1.00 36.77           C  \n",
            "ATOM    119  OG1 THR A 224      29.700  14.243  25.035  1.00 38.16           O  \n",
            "ATOM    120  CG2 THR A 224      27.985  15.911  25.363  1.00 36.69           C  \n",
            "ATOM    121  N   VAL A 225      25.324  13.498  25.433  1.00 32.00           N  \n",
            "ATOM    122  CA  VAL A 225      23.891  13.797  25.548  1.00 30.72           C  \n",
            "ATOM    123  C   VAL A 225      23.234  13.039  26.689  1.00 30.47           C  \n",
            "ATOM    124  O   VAL A 225      22.460  13.612  27.456  1.00 30.52           O  \n",
            "ATOM    125  CB  VAL A 225      23.076  13.451  24.264  1.00 29.72           C  \n",
            "ATOM    126  CG1 VAL A 225      23.260  14.513  23.230  1.00 30.41           C  \n",
            "ATOM    127  CG2 VAL A 225      23.481  12.093  23.725  1.00 30.06           C  \n",
            "ATOM    128  N   THR A 226      23.535  11.747  26.785  1.00 29.94           N  \n",
            "ATOM    129  CA  THR A 226      22.965  10.906  27.826  1.00 29.42           C  \n",
            "ATOM    130  C   THR A 226      23.224  11.512  29.199  1.00 30.99           C  \n",
            "ATOM    131  O   THR A 226      22.298  11.698  29.985  1.00 32.09           O  \n",
            "ATOM    132  CB  THR A 226      23.559   9.482  27.793  1.00 28.79           C  \n",
            "ATOM    133  OG1 THR A 226      23.510   8.962  26.456  1.00 25.28           O  \n",
            "ATOM    134  CG2 THR A 226      22.760   8.563  28.714  1.00 25.66           C  \n",
            "ATOM    135  N   GLU A 227      24.483  11.824  29.489  1.00 32.57           N  \n",
            "ATOM    136  CA  GLU A 227      24.828  12.409  30.778  1.00 34.22           C  \n",
            "ATOM    137  C   GLU A 227      24.163  13.768  30.964  1.00 33.31           C  \n",
            "ATOM    138  O   GLU A 227      23.755  14.119  32.078  1.00 32.65           O  \n",
            "ATOM    139  CB  GLU A 227      26.347  12.548  30.919  1.00 36.42           C  \n",
            "ATOM    140  CG  GLU A 227      27.080  11.217  30.986  1.00 42.03           C  \n",
            "ATOM    141  CD  GLU A 227      28.564  11.378  31.251  1.00 44.98           C  \n",
            "ATOM    142  OE1 GLU A 227      29.214  12.176  30.537  1.00 49.26           O  \n",
            "ATOM    143  OE2 GLU A 227      29.085  10.705  32.165  1.00 44.94           O  \n",
            "ATOM    144  N   ALA A 228      24.057  14.521  29.870  1.00 31.09           N  \n",
            "ATOM    145  CA  ALA A 228      23.434  15.840  29.895  1.00 31.64           C  \n",
            "ATOM    146  C   ALA A 228      21.974  15.691  30.304  1.00 31.22           C  \n",
            "ATOM    147  O   ALA A 228      21.465  16.427  31.157  1.00 30.89           O  \n",
            "ATOM    148  CB  ALA A 228      23.524  16.495  28.512  1.00 32.43           C  \n",
            "ATOM    149  N   HIS A 229      21.307  14.730  29.678  1.00 29.01           N  \n",
            "ATOM    150  CA  HIS A 229      19.917  14.458  29.962  1.00 27.99           C  \n",
            "ATOM    151  C   HIS A 229      19.743  14.021  31.405  1.00 29.46           C  \n",
            "ATOM    152  O   HIS A 229      18.989  14.630  32.156  1.00 31.80           O  \n",
            "ATOM    153  CB  HIS A 229      19.400  13.358  29.038  1.00 25.27           C  \n",
            "ATOM    154  CG  HIS A 229      18.023  12.881  29.393  1.00 22.38           C  \n",
            "ATOM    155  ND1 HIS A 229      16.967  13.739  29.568  1.00 21.26           N  \n",
            "ATOM    156  CD2 HIS A 229      17.546  11.635  29.609  1.00 21.19           C  \n",
            "ATOM    157  CE1 HIS A 229      15.884  13.042  29.878  1.00 21.31           C  \n",
            "ATOM    158  NE2 HIS A 229      16.210  11.769  29.908  1.00 21.06           N  \n",
            "ATOM    159  N   VAL A 230      20.446  12.958  31.785  1.00 29.57           N  \n",
            "ATOM    160  CA  VAL A 230      20.356  12.406  33.132  1.00 29.43           C  \n",
            "ATOM    161  C   VAL A 230      20.576  13.447  34.226  1.00 30.92           C  \n",
            "ATOM    162  O   VAL A 230      19.842  13.486  35.214  1.00 32.36           O  \n",
            "ATOM    163  CB  VAL A 230      21.366  11.255  33.308  1.00 27.11           C  \n",
            "ATOM    164  CG1 VAL A 230      21.357  10.766  34.730  1.00 25.77           C  \n",
            "ATOM    165  CG2 VAL A 230      21.017  10.125  32.371  1.00 25.00           C  \n",
            "ATOM    166  N   ALA A 231      21.584  14.291  34.041  1.00 31.03           N  \n",
            "ATOM    167  CA  ALA A 231      21.907  15.337  35.004  1.00 31.31           C  \n",
            "ATOM    168  C   ALA A 231      20.801  16.381  35.137  1.00 32.03           C  \n",
            "ATOM    169  O   ALA A 231      20.726  17.080  36.148  1.00 32.38           O  \n",
            "ATOM    170  CB  ALA A 231      23.200  16.030  34.596  1.00 30.12           C  \n",
            "ATOM    171  N   THR A 232      19.947  16.480  34.122  1.00 31.20           N  \n",
            "ATOM    172  CA  THR A 232      18.879  17.465  34.122  1.00 30.29           C  \n",
            "ATOM    173  C   THR A 232      17.494  16.856  34.083  1.00 31.85           C  \n",
            "ATOM    174  O   THR A 232      16.522  17.547  33.782  1.00 31.95           O  \n",
            "ATOM    175  CB  THR A 232      19.012  18.420  32.913  1.00 30.69           C  \n",
            "ATOM    176  OG1 THR A 232      18.996  17.664  31.691  1.00 28.26           O  \n",
            "ATOM    177  CG2 THR A 232      20.304  19.228  33.009  1.00 27.88           C  \n",
            "ATOM    178  N   ASN A 233      17.382  15.573  34.395  1.00 34.30           N  \n",
            "ATOM    179  CA  ASN A 233      16.070  14.945  34.349  1.00 38.02           C  \n",
            "ATOM    180  C   ASN A 233      15.439  14.633  35.701  1.00 41.54           C  \n",
            "ATOM    181  O   ASN A 233      14.630  13.713  35.808  1.00 43.82           O  \n",
            "ATOM    182  CB  ASN A 233      16.124  13.669  33.527  1.00 37.44           C  \n",
            "ATOM    183  CG  ASN A 233      14.746  13.156  33.171  1.00 37.75           C  \n",
            "ATOM    184  OD1 ASN A 233      14.549  11.958  33.001  1.00 40.46           O  \n",
            "ATOM    185  ND2 ASN A 233      13.788  14.063  33.043  1.00 35.78           N  \n",
            "ATOM    186  N   ALA A 234      15.801  15.394  36.727  1.00 44.95           N  \n",
            "ATOM    187  CA  ALA A 234      15.243  15.205  38.063  1.00 47.92           C  \n",
            "ATOM    188  C   ALA A 234      15.014  13.738  38.422  1.00 49.99           C  \n",
            "ATOM    189  O   ALA A 234      13.933  13.366  38.867  1.00 50.27           O  \n",
            "ATOM    190  CB  ALA A 234      13.928  15.979  38.180  1.00 46.79           C  \n",
            "ATOM    191  N   GLN A 235      16.029  12.903  38.234  1.00 53.22           N  \n",
            "ATOM    192  CA  GLN A 235      15.884  11.487  38.544  1.00 56.05           C  \n",
            "ATOM    193  C   GLN A 235      16.972  10.972  39.482  1.00 58.40           C  \n",
            "ATOM    194  O   GLN A 235      17.553   9.907  39.255  1.00 57.94           O  \n",
            "ATOM    195  CB  GLN A 235      15.852  10.658  37.249  1.00 57.05           C  \n",
            "ATOM    196  CG  GLN A 235      17.102  10.748  36.369  1.00 58.94           C  \n",
            "ATOM    197  CD  GLN A 235      16.928  10.044  35.019  1.00 61.05           C  \n",
            "ATOM    198  OE1 GLN A 235      16.613   8.852  34.957  1.00 59.20           O  \n",
            "ATOM    199  NE2 GLN A 235      17.134  10.788  33.931  1.00 62.17           N  \n",
            "ATOM    200  N   GLY A 236      17.251  11.738  40.536  1.00 60.38           N  \n",
            "ATOM    201  CA  GLY A 236      18.252  11.318  41.500  1.00 62.58           C  \n",
            "ATOM    202  C   GLY A 236      17.767  10.003  42.072  1.00 64.64           C  \n",
            "ATOM    203  O   GLY A 236      18.548   9.082  42.317  1.00 64.52           O  \n",
            "ATOM    204  N   SER A 237      16.453   9.935  42.278  1.00 66.24           N  \n",
            "ATOM    205  CA  SER A 237      15.778   8.746  42.789  1.00 67.52           C  \n",
            "ATOM    206  C   SER A 237      14.649   8.439  41.798  1.00 67.86           C  \n",
            "ATOM    207  O   SER A 237      14.073   9.356  41.209  1.00 67.64           O  \n",
            "ATOM    208  CB  SER A 237      15.195   9.007  44.184  1.00 67.68           C  \n",
            "ATOM    209  OG  SER A 237      14.077   9.880  44.130  1.00 67.94           O  \n",
            "ATOM    210  N   HIS A 238      14.340   7.160  41.608  1.00 67.91           N  \n",
            "ATOM    211  CA  HIS A 238      13.287   6.776  40.679  1.00 67.49           C  \n",
            "ATOM    212  C   HIS A 238      11.994   7.523  40.998  1.00 66.39           C  \n",
            "ATOM    213  O   HIS A 238      11.572   7.596  42.154  1.00 67.11           O  \n",
            "ATOM    214  CB  HIS A 238      13.046   5.265  40.731  1.00 69.69           C  \n",
            "ATOM    215  CG  HIS A 238      12.263   4.743  39.565  1.00 72.56           C  \n",
            "ATOM    216  ND1 HIS A 238      12.728   4.805  38.268  1.00 73.50           N  \n",
            "ATOM    217  CD2 HIS A 238      11.035   4.176  39.497  1.00 73.31           C  \n",
            "ATOM    218  CE1 HIS A 238      11.819   4.301  37.451  1.00 73.38           C  \n",
            "ATOM    219  NE2 HIS A 238      10.783   3.912  38.171  1.00 73.39           N  \n",
            "ATOM    220  N   TRP A 239      11.375   8.074  39.959  1.00 64.34           N  \n",
            "ATOM    221  CA  TRP A 239      10.140   8.844  40.080  1.00 61.77           C  \n",
            "ATOM    222  C   TRP A 239       8.950   8.066  40.636  1.00 61.46           C  \n",
            "ATOM    223  O   TRP A 239       8.165   8.608  41.412  1.00 61.28           O  \n",
            "ATOM    224  CB  TRP A 239       9.752   9.425  38.715  1.00 59.56           C  \n",
            "ATOM    225  CG  TRP A 239       9.571   8.380  37.651  1.00 54.82           C  \n",
            "ATOM    226  CD1 TRP A 239      10.559   7.731  36.964  1.00 54.10           C  \n",
            "ATOM    227  CD2 TRP A 239       8.330   7.837  37.180  1.00 52.32           C  \n",
            "ATOM    228  NE1 TRP A 239      10.009   6.817  36.096  1.00 53.67           N  \n",
            "ATOM    229  CE2 TRP A 239       8.644   6.860  36.209  1.00 52.28           C  \n",
            "ATOM    230  CE3 TRP A 239       6.984   8.077  37.486  1.00 50.98           C  \n",
            "ATOM    231  CZ2 TRP A 239       7.660   6.125  35.540  1.00 50.84           C  \n",
            "ATOM    232  CZ3 TRP A 239       6.005   7.344  36.819  1.00 49.92           C  \n",
            "ATOM    233  CH2 TRP A 239       6.351   6.380  35.858  1.00 49.83           C  \n",
            "ATOM    234  N   LYS A 240       8.808   6.809  40.222  1.00 61.21           N  \n",
            "ATOM    235  CA  LYS A 240       7.701   5.967  40.673  1.00 60.51           C  \n",
            "ATOM    236  C   LYS A 240       7.594   5.939  42.197  1.00 59.48           C  \n",
            "ATOM    237  O   LYS A 240       6.561   5.555  42.749  1.00 59.54           O  \n",
            "ATOM    238  CB  LYS A 240       7.880   4.536  40.152  1.00 61.42           C  \n",
            "ATOM    239  CG  LYS A 240       7.755   4.384  38.640  1.00 63.05           C  \n",
            "ATOM    240  CD  LYS A 240       6.334   4.026  38.223  1.00 65.03           C  \n",
            "ATOM    241  CE  LYS A 240       5.951   2.624  38.685  1.00 65.80           C  \n",
            "ATOM    242  NZ  LYS A 240       4.525   2.295  38.385  1.00 67.62           N  \n",
            "ATOM    243  N   GLN A 241       8.658   6.360  42.872  1.00 57.86           N  \n",
            "ATOM    244  CA  GLN A 241       8.685   6.356  44.327  1.00 57.24           C  \n",
            "ATOM    245  C   GLN A 241       8.562   7.729  44.983  1.00 56.27           C  \n",
            "ATOM    246  O   GLN A 241       7.949   7.863  46.042  1.00 55.73           O  \n",
            "ATOM    247  CB  GLN A 241       9.968   5.688  44.810  1.00 57.41           C  \n",
            "ATOM    248  CG  GLN A 241      10.131   4.262  44.347  1.00 57.26           C  \n",
            "ATOM    249  CD  GLN A 241      11.394   3.630  44.887  1.00 58.24           C  \n",
            "ATOM    250  OE1 GLN A 241      11.632   2.438  44.703  1.00 59.52           O  \n",
            "ATOM    251  NE2 GLN A 241      12.214   4.429  45.557  1.00 57.10           N  \n",
            "ATOM    252  N   LYS A 242       9.148   8.746  44.365  1.00 55.61           N  \n",
            "ATOM    253  CA  LYS A 242       9.097  10.091  44.927  1.00 55.64           C  \n",
            "ATOM    254  C   LYS A 242       7.823  10.860  44.560  1.00 54.76           C  \n",
            "ATOM    255  O   LYS A 242       7.497  11.878  45.178  1.00 55.19           O  \n",
            "ATOM    256  CB  LYS A 242      10.329  10.895  44.484  1.00 55.84           C  \n",
            "ATOM    257  CG  LYS A 242      10.453  12.248  45.173  0.50 56.38           C  \n",
            "ATOM    258  CD  LYS A 242      11.679  13.012  44.708  0.50 57.12           C  \n",
            "ATOM    259  CE  LYS A 242      11.791  14.349  45.427  0.50 57.39           C  \n",
            "ATOM    260  NZ  LYS A 242      13.021  15.102  45.043  0.50 57.68           N  \n",
            "ATOM    261  N   ARG A 243       7.093  10.365  43.569  1.00 52.24           N  \n",
            "ATOM    262  CA  ARG A 243       5.879  11.036  43.127  1.00 51.03           C  \n",
            "ATOM    263  C   ARG A 243       4.717  11.055  44.126  1.00 50.35           C  \n",
            "ATOM    264  O   ARG A 243       4.327  10.021  44.678  1.00 51.30           O  \n",
            "ATOM    265  CB  ARG A 243       5.384  10.409  41.823  1.00 49.56           C  \n",
            "ATOM    266  CG  ARG A 243       5.009   8.958  41.987  1.00 47.20           C  \n",
            "ATOM    267  CD  ARG A 243       4.280   8.420  40.795  1.00 44.77           C  \n",
            "ATOM    268  NE  ARG A 243       4.035   6.991  40.942  1.00 43.65           N  \n",
            "ATOM    269  CZ  ARG A 243       3.287   6.277  40.110  1.00 44.87           C  \n",
            "ATOM    270  NH1 ARG A 243       2.708   6.866  39.072  1.00 46.88           N  \n",
            "ATOM    271  NH2 ARG A 243       3.124   4.978  40.308  1.00 43.98           N  \n",
            "ATOM    272  N   LYS A 244       4.175  12.249  44.356  1.00 48.36           N  \n",
            "ATOM    273  CA  LYS A 244       3.011  12.426  45.216  1.00 46.09           C  \n",
            "ATOM    274  C   LYS A 244       1.866  12.623  44.235  1.00 43.54           C  \n",
            "ATOM    275  O   LYS A 244       2.045  13.258  43.193  1.00 42.66           O  \n",
            "ATOM    276  CB  LYS A 244       3.150  13.672  46.095  1.00 47.82           C  \n",
            "ATOM    277  CG  LYS A 244       3.682  13.398  47.498  1.00 51.73           C  \n",
            "ATOM    278  CD  LYS A 244       5.093  12.808  47.463  1.00 54.87           C  \n",
            "ATOM    279  CE  LYS A 244       5.558  12.366  48.847  1.00 56.00           C  \n",
            "ATOM    280  NZ  LYS A 244       5.481  13.472  49.854  1.00 58.66           N  \n",
            "ATOM    281  N   PHE A 245       0.699  12.069  44.535  1.00 41.24           N  \n",
            "ATOM    282  CA  PHE A 245      -0.427  12.240  43.628  1.00 38.89           C  \n",
            "ATOM    283  C   PHE A 245      -1.163  13.536  43.927  1.00 37.91           C  \n",
            "ATOM    284  O   PHE A 245      -1.416  13.863  45.087  1.00 38.17           O  \n",
            "ATOM    285  CB  PHE A 245      -1.392  11.056  43.719  1.00 36.31           C  \n",
            "ATOM    286  CG  PHE A 245      -0.828   9.781  43.178  1.00 36.77           C  \n",
            "ATOM    287  CD1 PHE A 245      -0.002   8.979  43.961  1.00 36.44           C  \n",
            "ATOM    288  CD2 PHE A 245      -1.078   9.404  41.863  1.00 37.20           C  \n",
            "ATOM    289  CE1 PHE A 245       0.570   7.816  43.440  1.00 37.81           C  \n",
            "ATOM    290  CE2 PHE A 245      -0.511   8.244  41.327  1.00 37.47           C  \n",
            "ATOM    291  CZ  PHE A 245       0.315   7.448  42.116  1.00 37.86           C  \n",
            "ATOM    292  N   LEU A 246      -1.462  14.291  42.876  1.00 36.49           N  \n",
            "ATOM    293  CA  LEU A 246      -2.192  15.541  43.027  1.00 36.08           C  \n",
            "ATOM    294  C   LEU A 246      -3.589  15.119  43.473  1.00 34.66           C  \n",
            "ATOM    295  O   LEU A 246      -4.239  14.323  42.799  1.00 35.39           O  \n",
            "ATOM    296  CB  LEU A 246      -2.263  16.282  41.686  1.00 35.45           C  \n",
            "ATOM    297  CG  LEU A 246      -2.807  17.713  41.687  1.00 37.09           C  \n",
            "ATOM    298  CD1 LEU A 246      -1.939  18.606  42.574  1.00 35.65           C  \n",
            "ATOM    299  CD2 LEU A 246      -2.832  18.239  40.261  1.00 37.18           C  \n",
            "ATOM    300  N   PRO A 247      -4.064  15.630  44.624  1.00 34.52           N  \n",
            "ATOM    301  CA  PRO A 247      -5.400  15.264  45.115  1.00 33.31           C  \n",
            "ATOM    302  C   PRO A 247      -6.465  15.243  44.014  1.00 32.60           C  \n",
            "ATOM    303  O   PRO A 247      -6.533  16.140  43.183  1.00 32.40           O  \n",
            "ATOM    304  CB  PRO A 247      -5.669  16.318  46.184  1.00 32.08           C  \n",
            "ATOM    305  CG  PRO A 247      -4.301  16.546  46.748  1.00 32.42           C  \n",
            "ATOM    306  CD  PRO A 247      -3.437  16.632  45.509  1.00 32.83           C  \n",
            "ATOM    307  N   GLU A 248      -7.286  14.203  44.016  1.00 33.13           N  \n",
            "ATOM    308  CA  GLU A 248      -8.337  14.041  43.017  1.00 35.93           C  \n",
            "ATOM    309  C   GLU A 248      -9.298  15.237  42.967  1.00 36.51           C  \n",
            "ATOM    310  O   GLU A 248      -9.863  15.557  41.915  1.00 36.95           O  \n",
            "ATOM    311  CB  GLU A 248      -9.118  12.762  43.307  1.00 37.35           C  \n",
            "ATOM    312  CG  GLU A 248     -10.142  12.414  42.255  1.00 42.12           C  \n",
            "ATOM    313  CD  GLU A 248     -10.950  11.171  42.614  1.00 46.11           C  \n",
            "ATOM    314  OE1 GLU A 248     -11.668  10.662  41.724  1.00 47.24           O  \n",
            "ATOM    315  OE2 GLU A 248     -10.875  10.708  43.780  1.00 46.52           O  \n",
            "ATOM    316  N   ASP A 249      -9.477  15.888  44.112  1.00 35.03           N  \n",
            "ATOM    317  CA  ASP A 249     -10.348  17.046  44.220  1.00 33.99           C  \n",
            "ATOM    318  C   ASP A 249      -9.950  18.139  43.227  1.00 33.98           C  \n",
            "ATOM    319  O   ASP A 249     -10.808  18.758  42.592  1.00 33.91           O  \n",
            "ATOM    320  CB  ASP A 249     -10.284  17.613  45.634  1.00 32.01           C  \n",
            "ATOM    321  CG  ASP A 249     -11.259  18.740  45.846  1.00 30.95           C  \n",
            "ATOM    322  OD1 ASP A 249     -12.475  18.473  45.880  1.00 31.58           O  \n",
            "ATOM    323  OD2 ASP A 249     -10.814  19.895  45.971  1.00 32.49           O  \n",
            "ATOM    324  N   ILE A 250      -8.647  18.379  43.112  1.00 33.12           N  \n",
            "ATOM    325  CA  ILE A 250      -8.118  19.392  42.206  1.00 32.52           C  \n",
            "ATOM    326  C   ILE A 250      -8.372  19.007  40.754  1.00 32.82           C  \n",
            "ATOM    327  O   ILE A 250      -7.910  17.970  40.289  1.00 32.06           O  \n",
            "ATOM    328  CB  ILE A 250      -6.600  19.585  42.422  1.00 32.96           C  \n",
            "ATOM    329  CG1 ILE A 250      -6.345  20.051  43.855  1.00 31.37           C  \n",
            "ATOM    330  CG2 ILE A 250      -6.043  20.608  41.428  1.00 31.86           C  \n",
            "ATOM    331  CD1 ILE A 250      -4.905  20.343  44.143  1.00 34.50           C  \n",
            "ATOM    332  N   GLY A 251      -9.115  19.854  40.046  1.00 33.97           N  \n",
            "ATOM    333  CA  GLY A 251      -9.428  19.588  38.655  1.00 33.51           C  \n",
            "ATOM    334  C   GLY A 251     -10.826  19.012  38.500  1.00 33.29           C  \n",
            "ATOM    335  O   GLY A 251     -11.323  18.849  37.387  1.00 35.66           O  \n",
            "ATOM    336  N   GLN A 252     -11.467  18.705  39.617  1.00 30.88           N  \n",
            "ATOM    337  CA  GLN A 252     -12.812  18.143  39.591  1.00 30.96           C  \n",
            "ATOM    338  C   GLN A 252     -13.875  19.222  39.416  1.00 30.21           C  \n",
            "ATOM    339  O   GLN A 252     -13.949  20.160  40.201  1.00 30.26           O  \n",
            "ATOM    340  CB  GLN A 252     -13.077  17.389  40.889  1.00 30.38           C  \n",
            "ATOM    341  CG  GLN A 252     -14.253  16.468  40.826  1.00 30.58           C  \n",
            "ATOM    342  CD  GLN A 252     -14.390  15.665  42.085  1.00 34.97           C  \n",
            "ATOM    343  OE1 GLN A 252     -13.394  15.194  42.639  1.00 37.72           O  \n",
            "ATOM    344  NE2 GLN A 252     -15.623  15.489  42.549  1.00 34.05           N  \n",
            "ATOM    345  N   ALA A 253     -14.701  19.079  38.389  1.00 30.13           N  \n",
            "ATOM    346  CA  ALA A 253     -15.765  20.041  38.127  1.00 30.16           C  \n",
            "ATOM    347  C   ALA A 253     -16.946  19.793  39.065  1.00 30.43           C  \n",
            "ATOM    348  O   ALA A 253     -17.471  18.684  39.135  1.00 29.62           O  \n",
            "ATOM    349  CB  ALA A 253     -16.221  19.935  36.677  1.00 29.89           C  \n",
            "ATOM    350  N   PRO A 254     -17.368  20.826  39.812  1.00 31.53           N  \n",
            "ATOM    351  CA  PRO A 254     -18.498  20.636  40.725  1.00 32.07           C  \n",
            "ATOM    352  C   PRO A 254     -19.810  20.401  39.982  1.00 31.69           C  \n",
            "ATOM    353  O   PRO A 254     -20.596  19.538  40.358  1.00 34.05           O  \n",
            "ATOM    354  CB  PRO A 254     -18.502  21.929  41.549  1.00 32.98           C  \n",
            "ATOM    355  CG  PRO A 254     -17.901  22.946  40.608  1.00 31.54           C  \n",
            "ATOM    356  CD  PRO A 254     -16.781  22.172  39.963  1.00 32.02           C  \n",
            "ATOM    357  N   ILE A 255     -20.045  21.171  38.927  1.00 30.25           N  \n",
            "ATOM    358  CA  ILE A 255     -21.266  21.027  38.154  1.00 28.99           C  \n",
            "ATOM    359  C   ILE A 255     -20.953  20.288  36.870  1.00 31.03           C  \n",
            "ATOM    360  O   ILE A 255     -20.333  20.822  35.946  1.00 31.05           O  \n",
            "ATOM    361  CB  ILE A 255     -21.885  22.382  37.825  1.00 27.27           C  \n",
            "ATOM    362  CG1 ILE A 255     -22.077  23.198  39.115  1.00 27.40           C  \n",
            "ATOM    363  CG2 ILE A 255     -23.187  22.175  37.095  1.00 24.48           C  \n",
            "ATOM    364  CD1 ILE A 255     -22.894  22.505  40.206  1.00 26.98           C  \n",
            "ATOM    365  N   VAL A 256     -21.392  19.037  36.846  1.00 32.05           N  \n",
            "ATOM    366  CA  VAL A 256     -21.183  18.120  35.742  1.00 31.66           C  \n",
            "ATOM    367  C   VAL A 256     -21.789  18.537  34.401  1.00 32.60           C  \n",
            "ATOM    368  O   VAL A 256     -21.203  18.288  33.346  1.00 31.27           O  \n",
            "ATOM    369  CB  VAL A 256     -21.723  16.728  36.143  1.00 29.72           C  \n",
            "ATOM    370  CG1 VAL A 256     -21.777  15.804  34.951  1.00 31.31           C  \n",
            "ATOM    371  CG2 VAL A 256     -20.843  16.145  37.219  1.00 26.70           C  \n",
            "ATOM    372  N   ASN A 257     -22.951  19.178  34.438  1.00 33.68           N  \n",
            "ATOM    373  CA  ASN A 257     -23.626  19.569  33.206  1.00 33.08           C  \n",
            "ATOM    374  C   ASN A 257     -23.318  20.959  32.668  1.00 32.21           C  \n",
            "ATOM    375  O   ASN A 257     -24.022  21.447  31.787  1.00 31.46           O  \n",
            "ATOM    376  CB  ASN A 257     -25.136  19.402  33.385  1.00 33.59           C  \n",
            "ATOM    377  CG  ASN A 257     -25.545  17.946  33.506  1.00 35.08           C  \n",
            "ATOM    378  OD1 ASN A 257     -25.441  17.179  32.546  1.00 36.30           O  \n",
            "ATOM    379  ND2 ASN A 257     -26.006  17.554  34.690  1.00 33.70           N  \n",
            "ATOM    380  N   ALA A 258     -22.274  21.594  33.187  1.00 32.03           N  \n",
            "ATOM    381  CA  ALA A 258     -21.901  22.924  32.718  1.00 34.14           C  \n",
            "ATOM    382  C   ALA A 258     -20.845  22.766  31.636  1.00 35.40           C  \n",
            "ATOM    383  O   ALA A 258     -19.851  22.069  31.826  1.00 37.40           O  \n",
            "ATOM    384  CB  ALA A 258     -21.350  23.771  33.866  1.00 34.21           C  \n",
            "ATOM    385  N   PRO A 259     -21.054  23.406  30.478  1.00 35.66           N  \n",
            "ATOM    386  CA  PRO A 259     -20.121  23.341  29.355  1.00 35.34           C  \n",
            "ATOM    387  C   PRO A 259     -18.676  23.601  29.754  1.00 36.05           C  \n",
            "ATOM    388  O   PRO A 259     -18.371  24.574  30.436  1.00 35.89           O  \n",
            "ATOM    389  CB  PRO A 259     -20.656  24.408  28.410  1.00 34.30           C  \n",
            "ATOM    390  CG  PRO A 259     -22.123  24.297  28.616  1.00 35.68           C  \n",
            "ATOM    391  CD  PRO A 259     -22.242  24.202  30.120  1.00 35.99           C  \n",
            "ATOM    392  N   GLU A 260     -17.788  22.722  29.306  1.00 38.49           N  \n",
            "ATOM    393  CA  GLU A 260     -16.364  22.843  29.587  1.00 39.48           C  \n",
            "ATOM    394  C   GLU A 260     -15.667  23.815  28.624  1.00 37.66           C  \n",
            "ATOM    395  O   GLU A 260     -14.806  24.595  29.037  1.00 37.02           O  \n",
            "ATOM    396  CB  GLU A 260     -15.711  21.465  29.504  1.00 42.50           C  \n",
            "ATOM    397  CG  GLU A 260     -16.399  20.427  30.380  1.00 46.41           C  \n",
            "ATOM    398  CD  GLU A 260     -15.522  19.225  30.650  1.00 48.72           C  \n",
            "ATOM    399  OE1 GLU A 260     -15.304  18.418  29.719  1.00 49.39           O  \n",
            "ATOM    400  OE2 GLU A 260     -15.041  19.100  31.799  1.00 51.71           O  \n",
            "ATOM    401  N   GLY A 261     -16.055  23.771  27.351  1.00 35.67           N  \n",
            "ATOM    402  CA  GLY A 261     -15.464  24.646  26.356  1.00 34.85           C  \n",
            "ATOM    403  C   GLY A 261     -13.953  24.628  26.448  1.00 35.82           C  \n",
            "ATOM    404  O   GLY A 261     -13.348  23.557  26.545  1.00 36.20           O  \n",
            "ATOM    405  N   GLY A 262     -13.340  25.809  26.419  1.00 35.52           N  \n",
            "ATOM    406  CA  GLY A 262     -11.894  25.891  26.533  1.00 34.80           C  \n",
            "ATOM    407  C   GLY A 262     -11.457  26.527  27.846  1.00 35.00           C  \n",
            "ATOM    408  O   GLY A 262     -10.337  27.028  27.957  1.00 34.24           O  \n",
            "ATOM    409  N   LYS A 263     -12.345  26.496  28.841  1.00 35.15           N  \n",
            "ATOM    410  CA  LYS A 263     -12.093  27.074  30.164  1.00 34.52           C  \n",
            "ATOM    411  C   LYS A 263     -10.909  26.444  30.892  1.00 32.66           C  \n",
            "ATOM    412  O   LYS A 263     -10.412  25.396  30.493  1.00 34.69           O  \n",
            "ATOM    413  CB  LYS A 263     -13.356  26.954  31.009  1.00 35.56           C  \n",
            "ATOM    414  CG  LYS A 263     -14.554  27.633  30.367  1.00 37.48           C  \n",
            "ATOM    415  CD  LYS A 263     -15.871  27.172  30.974  1.00 37.41           C  \n",
            "ATOM    416  CE  LYS A 263     -17.044  27.875  30.305  1.00 36.40           C  \n",
            "ATOM    417  NZ  LYS A 263     -18.351  27.324  30.738  1.00 37.26           N  \n",
            "ATOM    418  N   VAL A 264     -10.459  27.085  31.963  1.00 30.98           N  \n",
            "ATOM    419  CA  VAL A 264      -9.318  26.581  32.726  1.00 28.96           C  \n",
            "ATOM    420  C   VAL A 264      -9.587  26.533  34.233  1.00 28.58           C  \n",
            "ATOM    421  O   VAL A 264     -10.133  27.468  34.794  1.00 30.17           O  \n",
            "ATOM    422  CB  VAL A 264      -8.065  27.469  32.489  1.00 29.03           C  \n",
            "ATOM    423  CG1 VAL A 264      -6.869  26.901  33.240  1.00 28.58           C  \n",
            "ATOM    424  CG2 VAL A 264      -7.760  27.577  30.989  1.00 28.14           C  \n",
            "ATOM    425  N   ASP A 265      -9.213  25.434  34.879  1.00 29.80           N  \n",
            "ATOM    426  CA  ASP A 265      -9.377  25.289  36.324  1.00 30.36           C  \n",
            "ATOM    427  C   ASP A 265      -8.261  26.118  36.948  1.00 30.00           C  \n",
            "ATOM    428  O   ASP A 265      -7.098  25.719  36.904  1.00 30.02           O  \n",
            "ATOM    429  CB  ASP A 265      -9.207  23.824  36.748  1.00 31.90           C  \n",
            "ATOM    430  CG  ASP A 265      -9.573  23.586  38.213  1.00 35.93           C  \n",
            "ATOM    431  OD1 ASP A 265      -9.070  24.303  39.106  1.00 36.43           O  \n",
            "ATOM    432  OD2 ASP A 265     -10.368  22.665  38.475  1.00 40.52           O  \n",
            "ATOM    433  N   LEU A 266      -8.602  27.261  37.532  1.00 29.60           N  \n",
            "ATOM    434  CA  LEU A 266      -7.583  28.123  38.127  1.00 30.39           C  \n",
            "ATOM    435  C   LEU A 266      -6.741  27.455  39.212  1.00 29.01           C  \n",
            "ATOM    436  O   LEU A 266      -5.531  27.677  39.296  1.00 25.54           O  \n",
            "ATOM    437  CB  LEU A 266      -8.226  29.395  38.686  1.00 31.69           C  \n",
            "ATOM    438  CG  LEU A 266      -8.895  30.290  37.648  1.00 34.82           C  \n",
            "ATOM    439  CD1 LEU A 266      -9.257  31.624  38.293  1.00 35.00           C  \n",
            "ATOM    440  CD2 LEU A 266      -7.955  30.495  36.461  1.00 37.08           C  \n",
            "ATOM    441  N   GLU A 267      -7.388  26.646  40.044  1.00 30.49           N  \n",
            "ATOM    442  CA  GLU A 267      -6.691  25.955  41.119  1.00 31.84           C  \n",
            "ATOM    443  C   GLU A 267      -5.655  25.002  40.539  1.00 31.55           C  \n",
            "ATOM    444  O   GLU A 267      -4.489  25.017  40.947  1.00 32.15           O  \n",
            "ATOM    445  CB  GLU A 267      -7.671  25.171  41.985  1.00 32.74           C  \n",
            "ATOM    446  CG  GLU A 267      -7.050  24.643  43.252  1.00 33.21           C  \n",
            "ATOM    447  CD  GLU A 267      -7.902  23.587  43.920  1.00 36.31           C  \n",
            "ATOM    448  OE1 GLU A 267      -7.775  23.424  45.159  1.00 39.42           O  \n",
            "ATOM    449  OE2 GLU A 267      -8.686  22.911  43.211  1.00 35.32           O  \n",
            "ATOM    450  N   ALA A 268      -6.078  24.182  39.581  1.00 29.47           N  \n",
            "ATOM    451  CA  ALA A 268      -5.161  23.236  38.952  1.00 29.62           C  \n",
            "ATOM    452  C   ALA A 268      -4.044  24.002  38.251  1.00 29.01           C  \n",
            "ATOM    453  O   ALA A 268      -2.862  23.710  38.441  1.00 29.12           O  \n",
            "ATOM    454  CB  ALA A 268      -5.908  22.353  37.951  1.00 28.50           C  \n",
            "ATOM    455  N   PHE A 269      -4.431  24.987  37.445  1.00 28.40           N  \n",
            "ATOM    456  CA  PHE A 269      -3.476  25.805  36.713  1.00 27.72           C  \n",
            "ATOM    457  C   PHE A 269      -2.432  26.330  37.686  1.00 29.17           C  \n",
            "ATOM    458  O   PHE A 269      -1.241  26.386  37.376  1.00 28.48           O  \n",
            "ATOM    459  CB  PHE A 269      -4.198  26.977  36.042  1.00 25.40           C  \n",
            "ATOM    460  CG  PHE A 269      -3.285  27.928  35.311  1.00 24.55           C  \n",
            "ATOM    461  CD1 PHE A 269      -2.789  27.613  34.048  1.00 25.81           C  \n",
            "ATOM    462  CD2 PHE A 269      -2.931  29.150  35.881  1.00 25.53           C  \n",
            "ATOM    463  CE1 PHE A 269      -1.951  28.500  33.353  1.00 25.21           C  \n",
            "ATOM    464  CE2 PHE A 269      -2.093  30.046  35.203  1.00 26.55           C  \n",
            "ATOM    465  CZ  PHE A 269      -1.602  29.717  33.929  1.00 26.08           C  \n",
            "ATOM    466  N   SER A 270      -2.894  26.702  38.875  1.00 30.98           N  \n",
            "ATOM    467  CA  SER A 270      -2.015  27.249  39.902  1.00 32.14           C  \n",
            "ATOM    468  C   SER A 270      -0.883  26.291  40.311  1.00 32.18           C  \n",
            "ATOM    469  O   SER A 270       0.281  26.697  40.414  1.00 30.59           O  \n",
            "ATOM    470  CB  SER A 270      -2.839  27.636  41.122  1.00 29.89           C  \n",
            "ATOM    471  OG  SER A 270      -2.055  28.421  41.994  1.00 31.64           O  \n",
            "ATOM    472  N   HIS A 271      -1.221  25.027  40.549  1.00 31.45           N  \n",
            "ATOM    473  CA  HIS A 271      -0.206  24.055  40.928  1.00 32.45           C  \n",
            "ATOM    474  C   HIS A 271       0.763  23.785  39.783  1.00 32.07           C  \n",
            "ATOM    475  O   HIS A 271       1.963  23.625  40.007  1.00 34.49           O  \n",
            "ATOM    476  CB  HIS A 271      -0.844  22.739  41.365  1.00 32.46           C  \n",
            "ATOM    477  CG  HIS A 271      -1.591  22.827  42.660  1.00 32.82           C  \n",
            "ATOM    478  ND1 HIS A 271      -2.778  23.510  42.791  1.00 32.39           N  \n",
            "ATOM    479  CD2 HIS A 271      -1.326  22.292  43.876  1.00 31.16           C  \n",
            "ATOM    480  CE1 HIS A 271      -3.219  23.390  44.032  1.00 30.93           C  \n",
            "ATOM    481  NE2 HIS A 271      -2.357  22.656  44.708  1.00 32.01           N  \n",
            "ATOM    482  N   PHE A 272       0.241  23.729  38.562  1.00 30.53           N  \n",
            "ATOM    483  CA  PHE A 272       1.066  23.486  37.382  1.00 29.44           C  \n",
            "ATOM    484  C   PHE A 272       2.112  24.575  37.156  1.00 28.64           C  \n",
            "ATOM    485  O   PHE A 272       3.262  24.273  36.874  1.00 28.63           O  \n",
            "ATOM    486  CB  PHE A 272       0.188  23.364  36.131  1.00 29.22           C  \n",
            "ATOM    487  CG  PHE A 272      -0.658  22.121  36.097  1.00 27.66           C  \n",
            "ATOM    488  CD1 PHE A 272      -1.982  22.178  35.664  1.00 26.55           C  \n",
            "ATOM    489  CD2 PHE A 272      -0.137  20.898  36.493  1.00 25.82           C  \n",
            "ATOM    490  CE1 PHE A 272      -2.771  21.031  35.632  1.00 25.94           C  \n",
            "ATOM    491  CE2 PHE A 272      -0.921  19.743  36.465  1.00 26.78           C  \n",
            "ATOM    492  CZ  PHE A 272      -2.237  19.813  36.033  1.00 26.95           C  \n",
            "ATOM    493  N   THR A 273       1.719  25.840  37.261  1.00 29.76           N  \n",
            "ATOM    494  CA  THR A 273       2.678  26.922  37.060  1.00 31.25           C  \n",
            "ATOM    495  C   THR A 273       3.717  26.958  38.174  1.00 33.45           C  \n",
            "ATOM    496  O   THR A 273       4.787  27.543  38.001  1.00 36.06           O  \n",
            "ATOM    497  CB  THR A 273       2.012  28.333  36.978  1.00 29.41           C  \n",
            "ATOM    498  OG1 THR A 273       1.206  28.567  38.137  1.00 28.89           O  \n",
            "ATOM    499  CG2 THR A 273       1.171  28.456  35.734  1.00 28.08           C  \n",
            "ATOM    500  N   LYS A 274       3.421  26.337  39.313  1.00 33.71           N  \n",
            "ATOM    501  CA  LYS A 274       4.384  26.345  40.399  1.00 34.08           C  \n",
            "ATOM    502  C   LYS A 274       5.521  25.362  40.175  1.00 34.32           C  \n",
            "ATOM    503  O   LYS A 274       6.555  25.462  40.832  1.00 36.50           O  \n",
            "ATOM    504  CB  LYS A 274       3.699  26.098  41.742  1.00 35.66           C  \n",
            "ATOM    505  CG  LYS A 274       2.788  27.257  42.134  1.00 41.72           C  \n",
            "ATOM    506  CD  LYS A 274       2.302  27.187  43.571  1.00 44.74           C  \n",
            "ATOM    507  CE  LYS A 274       1.395  28.377  43.894  1.00 45.48           C  \n",
            "ATOM    508  NZ  LYS A 274       1.071  28.449  45.354  1.00 44.84           N  \n",
            "ATOM    509  N   ILE A 275       5.350  24.426  39.245  1.00 32.68           N  \n",
            "ATOM    510  CA  ILE A 275       6.411  23.466  38.945  1.00 31.38           C  \n",
            "ATOM    511  C   ILE A 275       6.878  23.557  37.495  1.00 30.52           C  \n",
            "ATOM    512  O   ILE A 275       7.779  22.836  37.082  1.00 31.45           O  \n",
            "ATOM    513  CB  ILE A 275       5.980  22.010  39.196  1.00 31.82           C  \n",
            "ATOM    514  CG1 ILE A 275       4.898  21.604  38.202  1.00 32.53           C  \n",
            "ATOM    515  CG2 ILE A 275       5.486  21.853  40.612  1.00 33.55           C  \n",
            "ATOM    516  CD1 ILE A 275       4.440  20.176  38.368  1.00 35.58           C  \n",
            "ATOM    517  N   ILE A 276       6.279  24.448  36.720  1.00 29.10           N  \n",
            "ATOM    518  CA  ILE A 276       6.662  24.570  35.326  1.00 28.87           C  \n",
            "ATOM    519  C   ILE A 276       7.990  25.283  35.139  1.00 28.25           C  \n",
            "ATOM    520  O   ILE A 276       8.722  25.020  34.183  1.00 28.57           O  \n",
            "ATOM    521  CB  ILE A 276       5.593  25.311  34.508  1.00 28.87           C  \n",
            "ATOM    522  CG1 ILE A 276       5.932  25.194  33.022  1.00 27.60           C  \n",
            "ATOM    523  CG2 ILE A 276       5.531  26.775  34.933  1.00 26.66           C  \n",
            "ATOM    524  CD1 ILE A 276       4.748  25.136  32.127  1.00 28.34           C  \n",
            "ATOM    525  N   THR A 277       8.299  26.191  36.049  1.00 27.17           N  \n",
            "ATOM    526  CA  THR A 277       9.540  26.931  35.960  1.00 27.95           C  \n",
            "ATOM    527  C   THR A 277      10.774  26.045  36.151  1.00 27.66           C  \n",
            "ATOM    528  O   THR A 277      11.772  26.212  35.460  1.00 28.40           O  \n",
            "ATOM    529  CB  THR A 277       9.543  28.083  36.966  1.00 28.31           C  \n",
            "ATOM    530  OG1 THR A 277       9.025  27.623  38.215  1.00 33.90           O  \n",
            "ATOM    531  CG2 THR A 277       8.675  29.211  36.481  1.00 29.24           C  \n",
            "ATOM    532  N   PRO A 278      10.733  25.099  37.103  1.00 28.71           N  \n",
            "ATOM    533  CA  PRO A 278      11.934  24.269  37.245  1.00 27.39           C  \n",
            "ATOM    534  C   PRO A 278      12.041  23.295  36.073  1.00 27.78           C  \n",
            "ATOM    535  O   PRO A 278      13.132  22.835  35.724  1.00 28.26           O  \n",
            "ATOM    536  CB  PRO A 278      11.702  23.560  38.575  1.00 27.35           C  \n",
            "ATOM    537  CG  PRO A 278      10.869  24.547  39.342  1.00 26.21           C  \n",
            "ATOM    538  CD  PRO A 278       9.874  24.966  38.294  1.00 27.48           C  \n",
            "ATOM    539  N   ALA A 279      10.896  22.989  35.467  1.00 26.09           N  \n",
            "ATOM    540  CA  ALA A 279      10.853  22.085  34.327  1.00 25.30           C  \n",
            "ATOM    541  C   ALA A 279      11.461  22.784  33.120  1.00 24.81           C  \n",
            "ATOM    542  O   ALA A 279      12.213  22.179  32.357  1.00 26.13           O  \n",
            "ATOM    543  CB  ALA A 279       9.418  21.673  34.028  1.00 23.63           C  \n",
            "ATOM    544  N   ILE A 280      11.142  24.060  32.958  1.00 23.44           N  \n",
            "ATOM    545  CA  ILE A 280      11.668  24.833  31.845  1.00 23.63           C  \n",
            "ATOM    546  C   ILE A 280      13.179  25.020  31.976  1.00 25.67           C  \n",
            "ATOM    547  O   ILE A 280      13.909  24.967  30.979  1.00 27.91           O  \n",
            "ATOM    548  CB  ILE A 280      10.977  26.222  31.751  1.00 22.64           C  \n",
            "ATOM    549  CG1 ILE A 280       9.513  26.050  31.318  1.00 20.09           C  \n",
            "ATOM    550  CG2 ILE A 280      11.724  27.116  30.768  1.00 21.00           C  \n",
            "ATOM    551  CD1 ILE A 280       8.695  27.325  31.349  1.00 18.84           C  \n",
            "ATOM    552  N   THR A 281      13.647  25.226  33.204  1.00 26.15           N  \n",
            "ATOM    553  CA  THR A 281      15.072  25.429  33.462  1.00 25.32           C  \n",
            "ATOM    554  C   THR A 281      15.908  24.199  33.141  1.00 25.52           C  \n",
            "ATOM    555  O   THR A 281      17.024  24.318  32.633  1.00 25.97           O  \n",
            "ATOM    556  CB  THR A 281      15.338  25.804  34.933  1.00 24.00           C  \n",
            "ATOM    557  OG1 THR A 281      14.798  27.095  35.203  1.00 23.00           O  \n",
            "ATOM    558  CG2 THR A 281      16.825  25.817  35.221  1.00 21.64           C  \n",
            "ATOM    559  N   ARG A 282      15.387  23.018  33.438  1.00 24.85           N  \n",
            "ATOM    560  CA  ARG A 282      16.160  21.824  33.163  1.00 26.51           C  \n",
            "ATOM    561  C   ARG A 282      16.321  21.661  31.663  1.00 27.86           C  \n",
            "ATOM    562  O   ARG A 282      17.356  21.171  31.199  1.00 30.16           O  \n",
            "ATOM    563  CB  ARG A 282      15.503  20.595  33.784  1.00 28.56           C  \n",
            "ATOM    564  CG  ARG A 282      15.304  20.733  35.287  1.00 30.13           C  \n",
            "ATOM    565  CD  ARG A 282      15.336  19.394  35.985  1.00 34.83           C  \n",
            "ATOM    566  NE  ARG A 282      14.026  19.014  36.489  1.00 41.69           N  \n",
            "ATOM    567  CZ  ARG A 282      12.985  18.707  35.719  1.00 45.79           C  \n",
            "ATOM    568  NH1 ARG A 282      13.095  18.729  34.396  1.00 45.81           N  \n",
            "ATOM    569  NH2 ARG A 282      11.825  18.385  36.275  1.00 46.37           N  \n",
            "ATOM    570  N   VAL A 283      15.317  22.083  30.893  1.00 25.97           N  \n",
            "ATOM    571  CA  VAL A 283      15.433  21.984  29.438  1.00 25.38           C  \n",
            "ATOM    572  C   VAL A 283      16.563  22.909  28.977  1.00 25.10           C  \n",
            "ATOM    573  O   VAL A 283      17.334  22.557  28.083  1.00 25.59           O  \n",
            "ATOM    574  CB  VAL A 283      14.124  22.372  28.704  1.00 23.89           C  \n",
            "ATOM    575  CG1 VAL A 283      14.353  22.343  27.201  1.00 19.60           C  \n",
            "ATOM    576  CG2 VAL A 283      13.004  21.396  29.074  1.00 24.33           C  \n",
            "ATOM    577  N   VAL A 284      16.663  24.084  29.593  1.00 23.00           N  \n",
            "ATOM    578  CA  VAL A 284      17.723  25.022  29.259  1.00 23.11           C  \n",
            "ATOM    579  C   VAL A 284      19.073  24.417  29.669  1.00 24.91           C  \n",
            "ATOM    580  O   VAL A 284      20.017  24.404  28.877  1.00 24.22           O  \n",
            "ATOM    581  CB  VAL A 284      17.537  26.387  29.991  1.00 21.98           C  \n",
            "ATOM    582  CG1 VAL A 284      18.726  27.297  29.723  1.00 21.28           C  \n",
            "ATOM    583  CG2 VAL A 284      16.257  27.060  29.530  1.00 21.26           C  \n",
            "ATOM    584  N   ASP A 285      19.155  23.906  30.899  1.00 25.69           N  \n",
            "ATOM    585  CA  ASP A 285      20.397  23.313  31.402  1.00 27.35           C  \n",
            "ATOM    586  C   ASP A 285      20.854  22.122  30.565  1.00 26.84           C  \n",
            "ATOM    587  O   ASP A 285      22.046  21.891  30.397  1.00 26.30           O  \n",
            "ATOM    588  CB  ASP A 285      20.241  22.878  32.866  1.00 28.03           C  \n",
            "ATOM    589  CG  ASP A 285      20.013  24.049  33.798  1.00 32.31           C  \n",
            "ATOM    590  OD1 ASP A 285      20.557  25.143  33.528  1.00 34.31           O  \n",
            "ATOM    591  OD2 ASP A 285      19.299  23.878  34.811  1.00 35.30           O  \n",
            "ATOM    592  N   PHE A 286      19.896  21.358  30.055  1.00 27.41           N  \n",
            "ATOM    593  CA  PHE A 286      20.207  20.206  29.226  1.00 26.84           C  \n",
            "ATOM    594  C   PHE A 286      20.960  20.676  27.986  1.00 25.52           C  \n",
            "ATOM    595  O   PHE A 286      22.010  20.136  27.635  1.00 26.17           O  \n",
            "ATOM    596  CB  PHE A 286      18.922  19.498  28.789  1.00 24.22           C  \n",
            "ATOM    597  CG  PHE A 286      19.132  18.503  27.686  1.00 22.48           C  \n",
            "ATOM    598  CD1 PHE A 286      19.864  17.346  27.905  1.00 22.01           C  \n",
            "ATOM    599  CD2 PHE A 286      18.611  18.734  26.416  1.00 23.99           C  \n",
            "ATOM    600  CE1 PHE A 286      20.074  16.427  26.872  1.00 23.57           C  \n",
            "ATOM    601  CE2 PHE A 286      18.816  17.819  25.373  1.00 22.59           C  \n",
            "ATOM    602  CZ  PHE A 286      19.548  16.662  25.603  1.00 21.49           C  \n",
            "ATOM    603  N   ALA A 287      20.397  21.682  27.330  1.00 23.94           N  \n",
            "ATOM    604  CA  ALA A 287      20.964  22.249  26.116  1.00 23.68           C  \n",
            "ATOM    605  C   ALA A 287      22.330  22.851  26.385  1.00 23.86           C  \n",
            "ATOM    606  O   ALA A 287      23.267  22.682  25.605  1.00 22.98           O  \n",
            "ATOM    607  CB  ALA A 287      20.028  23.319  25.566  1.00 21.65           C  \n",
            "ATOM    608  N   LYS A 288      22.429  23.562  27.500  1.00 24.70           N  \n",
            "ATOM    609  CA  LYS A 288      23.664  24.214  27.869  1.00 26.44           C  \n",
            "ATOM    610  C   LYS A 288      24.758  23.237  28.284  1.00 28.33           C  \n",
            "ATOM    611  O   LYS A 288      25.832  23.667  28.703  1.00 29.57           O  \n",
            "ATOM    612  CB  LYS A 288      23.406  25.231  28.980  1.00 26.37           C  \n",
            "ATOM    613  CG  LYS A 288      22.414  26.317  28.588  1.00 28.29           C  \n",
            "ATOM    614  CD  LYS A 288      22.922  27.175  27.450  1.00 29.62           C  \n",
            "ATOM    615  CE  LYS A 288      23.758  28.325  27.959  1.00 33.19           C  \n",
            "ATOM    616  NZ  LYS A 288      24.397  29.083  26.836  1.00 33.60           N  \n",
            "ATOM    617  N   LYS A 289      24.494  21.937  28.158  1.00 27.66           N  \n",
            "ATOM    618  CA  LYS A 289      25.480  20.917  28.505  1.00 28.50           C  \n",
            "ATOM    619  C   LYS A 289      26.025  20.278  27.243  1.00 30.66           C  \n",
            "ATOM    620  O   LYS A 289      26.874  19.380  27.292  1.00 31.20           O  \n",
            "ATOM    621  CB  LYS A 289      24.863  19.834  29.381  1.00 27.09           C  \n",
            "ATOM    622  CG  LYS A 289      24.740  20.217  30.834  1.00 25.63           C  \n",
            "ATOM    623  CD  LYS A 289      24.128  19.089  31.642  1.00 23.90           C  \n",
            "ATOM    624  CE  LYS A 289      23.893  19.511  33.084  1.00 23.34           C  \n",
            "ATOM    625  NZ  LYS A 289      25.159  19.788  33.821  1.00 22.81           N  \n",
            "ATOM    626  N   LEU A 290      25.520  20.743  26.109  1.00 31.64           N  \n",
            "ATOM    627  CA  LEU A 290      25.949  20.232  24.817  1.00 32.58           C  \n",
            "ATOM    628  C   LEU A 290      26.817  21.294  24.139  1.00 33.15           C  \n",
            "ATOM    629  O   LEU A 290      26.344  22.398  23.837  1.00 30.56           O  \n",
            "ATOM    630  CB  LEU A 290      24.722  19.903  23.954  1.00 31.32           C  \n",
            "ATOM    631  CG  LEU A 290      23.696  18.969  24.609  1.00 30.18           C  \n",
            "ATOM    632  CD1 LEU A 290      22.506  18.761  23.689  1.00 26.63           C  \n",
            "ATOM    633  CD2 LEU A 290      24.367  17.647  24.951  1.00 30.01           C  \n",
            "ATOM    634  N   PRO A 291      28.111  20.980  23.920  1.00 34.54           N  \n",
            "ATOM    635  CA  PRO A 291      29.079  21.883  23.279  1.00 35.00           C  \n",
            "ATOM    636  C   PRO A 291      28.606  22.394  21.920  1.00 34.24           C  \n",
            "ATOM    637  O   PRO A 291      28.846  23.550  21.558  1.00 35.56           O  \n",
            "ATOM    638  CB  PRO A 291      30.333  21.022  23.182  1.00 34.80           C  \n",
            "ATOM    639  CG  PRO A 291      30.247  20.201  24.437  1.00 35.09           C  \n",
            "ATOM    640  CD  PRO A 291      28.790  19.768  24.416  1.00 34.58           C  \n",
            "ATOM    641  N   MET A 292      27.930  21.532  21.173  1.00 32.33           N  \n",
            "ATOM    642  CA  MET A 292      27.412  21.923  19.874  1.00 34.06           C  \n",
            "ATOM    643  C   MET A 292      26.450  23.096  20.000  1.00 33.16           C  \n",
            "ATOM    644  O   MET A 292      26.481  24.016  19.193  1.00 34.43           O  \n",
            "ATOM    645  CB  MET A 292      26.696  20.746  19.210  1.00 36.67           C  \n",
            "ATOM    646  CG  MET A 292      27.599  19.848  18.404  1.00 39.46           C  \n",
            "ATOM    647  SD  MET A 292      26.791  18.307  17.942  1.00 47.05           S  \n",
            "ATOM    648  CE  MET A 292      28.203  17.172  17.910  1.00 45.01           C  \n",
            "ATOM    649  N   PHE A 293      25.601  23.056  21.022  1.00 33.02           N  \n",
            "ATOM    650  CA  PHE A 293      24.607  24.094  21.258  1.00 31.76           C  \n",
            "ATOM    651  C   PHE A 293      25.202  25.403  21.765  1.00 33.13           C  \n",
            "ATOM    652  O   PHE A 293      24.859  26.481  21.280  1.00 32.59           O  \n",
            "ATOM    653  CB  PHE A 293      23.577  23.598  22.274  1.00 29.83           C  \n",
            "ATOM    654  CG  PHE A 293      22.437  24.562  22.520  1.00 28.93           C  \n",
            "ATOM    655  CD1 PHE A 293      21.339  24.597  21.660  1.00 26.83           C  \n",
            "ATOM    656  CD2 PHE A 293      22.454  25.414  23.623  1.00 26.53           C  \n",
            "ATOM    657  CE1 PHE A 293      20.274  25.453  21.896  1.00 27.32           C  \n",
            "ATOM    658  CE2 PHE A 293      21.394  26.276  23.870  1.00 27.91           C  \n",
            "ATOM    659  CZ  PHE A 293      20.298  26.298  23.003  1.00 28.34           C  \n",
            "ATOM    660  N   CYS A 294      26.092  25.291  22.744  1.00 34.38           N  \n",
            "ATOM    661  CA  CYS A 294      26.719  26.442  23.370  1.00 35.77           C  \n",
            "ATOM    662  C   CYS A 294      27.530  27.320  22.420  1.00 36.91           C  \n",
            "ATOM    663  O   CYS A 294      27.748  28.498  22.696  1.00 37.04           O  \n",
            "ATOM    664  CB  CYS A 294      27.592  25.965  24.530  1.00 36.82           C  \n",
            "ATOM    665  SG  CYS A 294      26.694  24.973  25.732  1.00 38.20           S  \n",
            "ATOM    666  N   GLU A 295      27.969  26.752  21.303  1.00 37.09           N  \n",
            "ATOM    667  CA  GLU A 295      28.745  27.500  20.329  1.00 37.47           C  \n",
            "ATOM    668  C   GLU A 295      27.859  28.082  19.225  1.00 37.28           C  \n",
            "ATOM    669  O   GLU A 295      28.339  28.462  18.156  1.00 37.08           O  \n",
            "ATOM    670  CB  GLU A 295      29.848  26.604  19.754  1.00 39.39           C  \n",
            "ATOM    671  CG  GLU A 295      30.962  26.330  20.782  1.00 46.73           C  \n",
            "ATOM    672  CD  GLU A 295      31.945  25.257  20.354  1.00 49.14           C  \n",
            "ATOM    673  OE1 GLU A 295      32.449  25.343  19.216  1.00 49.71           O  \n",
            "ATOM    674  OE2 GLU A 295      32.218  24.336  21.160  1.00 50.51           O  \n",
            "ATOM    675  N   LEU A 296      26.556  28.139  19.488  1.00 35.69           N  \n",
            "ATOM    676  CA  LEU A 296      25.605  28.717  18.546  1.00 34.22           C  \n",
            "ATOM    677  C   LEU A 296      25.359  30.131  19.046  1.00 33.48           C  \n",
            "ATOM    678  O   LEU A 296      25.417  30.381  20.246  1.00 32.88           O  \n",
            "ATOM    679  CB  LEU A 296      24.276  27.941  18.539  1.00 33.96           C  \n",
            "ATOM    680  CG  LEU A 296      24.205  26.588  17.810  1.00 32.98           C  \n",
            "ATOM    681  CD1 LEU A 296      22.873  25.919  18.092  1.00 28.56           C  \n",
            "ATOM    682  CD2 LEU A 296      24.394  26.791  16.310  1.00 29.45           C  \n",
            "ATOM    683  N   PRO A 297      25.089  31.078  18.136  1.00 34.08           N  \n",
            "ATOM    684  CA  PRO A 297      24.838  32.464  18.553  1.00 33.70           C  \n",
            "ATOM    685  C   PRO A 297      23.737  32.463  19.603  1.00 33.25           C  \n",
            "ATOM    686  O   PRO A 297      22.880  31.576  19.598  1.00 32.13           O  \n",
            "ATOM    687  CB  PRO A 297      24.383  33.140  17.263  1.00 33.44           C  \n",
            "ATOM    688  CG  PRO A 297      25.064  32.328  16.196  1.00 34.21           C  \n",
            "ATOM    689  CD  PRO A 297      24.896  30.920  16.684  1.00 33.90           C  \n",
            "ATOM    690  N   CYS A 298      23.762  33.447  20.496  1.00 32.69           N  \n",
            "ATOM    691  CA  CYS A 298      22.757  33.542  21.543  1.00 32.74           C  \n",
            "ATOM    692  C   CYS A 298      21.352  33.743  20.945  1.00 33.26           C  \n",
            "ATOM    693  O   CYS A 298      20.349  33.330  21.534  1.00 32.69           O  \n",
            "ATOM    694  CB  CYS A 298      23.110  34.689  22.491  1.00 32.37           C  \n",
            "ATOM    695  SG  CYS A 298      21.897  34.994  23.793  1.00 36.51           S  \n",
            "ATOM    696  N   GLU A 299      21.289  34.365  19.771  1.00 32.13           N  \n",
            "ATOM    697  CA  GLU A 299      20.020  34.607  19.101  1.00 33.21           C  \n",
            "ATOM    698  C   GLU A 299      19.436  33.280  18.622  1.00 32.31           C  \n",
            "ATOM    699  O   GLU A 299      18.214  33.086  18.609  1.00 33.38           O  \n",
            "ATOM    700  CB  GLU A 299      20.208  35.542  17.901  1.00 36.91           C  \n",
            "ATOM    701  CG  GLU A 299      20.670  36.957  18.237  1.00 42.04           C  \n",
            "ATOM    702  CD  GLU A 299      22.075  37.002  18.823  1.00 46.50           C  \n",
            "ATOM    703  OE1 GLU A 299      23.000  36.401  18.221  1.00 48.35           O  \n",
            "ATOM    704  OE2 GLU A 299      22.255  37.644  19.883  1.00 46.72           O  \n",
            "ATOM    705  N   ASP A 300      20.305  32.368  18.213  1.00 29.37           N  \n",
            "ATOM    706  CA  ASP A 300      19.832  31.072  17.764  1.00 28.86           C  \n",
            "ATOM    707  C   ASP A 300      19.493  30.185  18.952  1.00 27.57           C  \n",
            "ATOM    708  O   ASP A 300      18.571  29.385  18.891  1.00 28.94           O  \n",
            "ATOM    709  CB  ASP A 300      20.877  30.383  16.891  1.00 29.07           C  \n",
            "ATOM    710  CG  ASP A 300      20.892  30.916  15.486  1.00 30.52           C  \n",
            "ATOM    711  OD1 ASP A 300      19.976  31.686  15.133  1.00 33.27           O  \n",
            "ATOM    712  OD2 ASP A 300      21.809  30.559  14.725  1.00 31.78           O  \n",
            "ATOM    713  N   GLN A 301      20.240  30.330  20.037  1.00 26.03           N  \n",
            "ATOM    714  CA  GLN A 301      19.987  29.528  21.220  1.00 25.51           C  \n",
            "ATOM    715  C   GLN A 301      18.586  29.822  21.753  1.00 24.42           C  \n",
            "ATOM    716  O   GLN A 301      17.879  28.916  22.186  1.00 24.18           O  \n",
            "ATOM    717  CB  GLN A 301      21.051  29.816  22.284  1.00 25.56           C  \n",
            "ATOM    718  CG  GLN A 301      22.456  29.418  21.863  1.00 25.88           C  \n",
            "ATOM    719  CD  GLN A 301      23.445  29.480  23.012  1.00 26.82           C  \n",
            "ATOM    720  OE1 GLN A 301      23.134  29.076  24.124  1.00 28.67           O  \n",
            "ATOM    721  NE2 GLN A 301      24.650  29.968  22.742  1.00 28.50           N  \n",
            "ATOM    722  N   ILE A 302      18.184  31.089  21.702  1.00 23.63           N  \n",
            "ATOM    723  CA  ILE A 302      16.861  31.498  22.166  1.00 22.47           C  \n",
            "ATOM    724  C   ILE A 302      15.763  30.871  21.287  1.00 23.18           C  \n",
            "ATOM    725  O   ILE A 302      14.879  30.154  21.785  1.00 22.93           O  \n",
            "ATOM    726  CB  ILE A 302      16.720  33.043  22.140  1.00 21.29           C  \n",
            "ATOM    727  CG1 ILE A 302      17.691  33.677  23.143  1.00 21.37           C  \n",
            "ATOM    728  CG2 ILE A 302      15.301  33.448  22.467  1.00 19.41           C  \n",
            "ATOM    729  CD1 ILE A 302      17.718  35.184  23.106  1.00 13.82           C  \n",
            "ATOM    730  N   ILE A 303      15.834  31.134  19.982  1.00 21.71           N  \n",
            "ATOM    731  CA  ILE A 303      14.863  30.605  19.031  1.00 20.51           C  \n",
            "ATOM    732  C   ILE A 303      14.682  29.086  19.183  1.00 21.86           C  \n",
            "ATOM    733  O   ILE A 303      13.552  28.591  19.212  1.00 23.14           O  \n",
            "ATOM    734  CB  ILE A 303      15.282  30.962  17.577  1.00 19.09           C  \n",
            "ATOM    735  CG1 ILE A 303      15.120  32.477  17.352  1.00 15.27           C  \n",
            "ATOM    736  CG2 ILE A 303      14.452  30.171  16.565  1.00 17.02           C  \n",
            "ATOM    737  CD1 ILE A 303      15.708  32.993  16.039  1.00  7.55           C  \n",
            "ATOM    738  N   LEU A 304      15.790  28.358  19.307  1.00 19.91           N  \n",
            "ATOM    739  CA  LEU A 304      15.752  26.910  19.461  1.00 19.41           C  \n",
            "ATOM    740  C   LEU A 304      15.108  26.450  20.763  1.00 19.52           C  \n",
            "ATOM    741  O   LEU A 304      14.415  25.434  20.788  1.00 18.27           O  \n",
            "ATOM    742  CB  LEU A 304      17.170  26.320  19.383  1.00 19.44           C  \n",
            "ATOM    743  CG  LEU A 304      17.911  26.399  18.043  1.00 22.11           C  \n",
            "ATOM    744  CD1 LEU A 304      19.279  25.755  18.170  1.00 20.75           C  \n",
            "ATOM    745  CD2 LEU A 304      17.106  25.710  16.953  1.00 22.66           C  \n",
            "ATOM    746  N   LEU A 305      15.353  27.182  21.845  1.00 20.00           N  \n",
            "ATOM    747  CA  LEU A 305      14.812  26.817  23.153  1.00 19.74           C  \n",
            "ATOM    748  C   LEU A 305      13.311  27.050  23.252  1.00 20.81           C  \n",
            "ATOM    749  O   LEU A 305      12.578  26.204  23.762  1.00 19.59           O  \n",
            "ATOM    750  CB  LEU A 305      15.552  27.582  24.257  1.00 18.71           C  \n",
            "ATOM    751  CG  LEU A 305      16.944  27.007  24.534  1.00 18.63           C  \n",
            "ATOM    752  CD1 LEU A 305      17.721  27.864  25.529  1.00 15.53           C  \n",
            "ATOM    753  CD2 LEU A 305      16.769  25.572  25.037  1.00 16.85           C  \n",
            "ATOM    754  N   LYS A 306      12.856  28.196  22.757  1.00 22.44           N  \n",
            "ATOM    755  CA  LYS A 306      11.435  28.506  22.770  1.00 24.04           C  \n",
            "ATOM    756  C   LYS A 306      10.695  27.518  21.867  1.00 23.62           C  \n",
            "ATOM    757  O   LYS A 306       9.517  27.230  22.076  1.00 24.04           O  \n",
            "ATOM    758  CB  LYS A 306      11.212  29.940  22.287  1.00 25.22           C  \n",
            "ATOM    759  CG  LYS A 306      11.943  30.972  23.130  1.00 29.57           C  \n",
            "ATOM    760  CD  LYS A 306      11.736  32.390  22.618  1.00 32.05           C  \n",
            "ATOM    761  CE  LYS A 306      10.301  32.847  22.801  1.00 34.52           C  \n",
            "ATOM    762  NZ  LYS A 306      10.095  34.205  22.237  1.00 35.56           N  \n",
            "ATOM    763  N   GLY A 307      11.410  26.983  20.881  1.00 22.83           N  \n",
            "ATOM    764  CA  GLY A 307      10.821  26.044  19.951  1.00 21.83           C  \n",
            "ATOM    765  C   GLY A 307      10.681  24.604  20.416  1.00 23.92           C  \n",
            "ATOM    766  O   GLY A 307       9.671  23.967  20.123  1.00 26.78           O  \n",
            "ATOM    767  N   CYS A 308      11.661  24.083  21.149  1.00 22.64           N  \n",
            "ATOM    768  CA  CYS A 308      11.613  22.693  21.591  1.00 20.79           C  \n",
            "ATOM    769  C   CYS A 308      11.274  22.480  23.055  1.00 21.72           C  \n",
            "ATOM    770  O   CYS A 308      11.002  21.355  23.460  1.00 22.61           O  \n",
            "ATOM    771  CB  CYS A 308      12.949  22.010  21.322  1.00 20.26           C  \n",
            "ATOM    772  SG  CYS A 308      14.223  22.457  22.511  1.00 20.59           S  \n",
            "ATOM    773  N   CYS A 309      11.290  23.541  23.850  1.00 21.51           N  \n",
            "ATOM    774  CA  CYS A 309      11.002  23.417  25.277  1.00 21.77           C  \n",
            "ATOM    775  C   CYS A 309       9.850  22.465  25.590  1.00 22.27           C  \n",
            "ATOM    776  O   CYS A 309      10.021  21.492  26.332  1.00 22.50           O  \n",
            "ATOM    777  CB  CYS A 309      10.703  24.789  25.867  1.00 23.15           C  \n",
            "ATOM    778  SG  CYS A 309      10.636  24.803  27.653  1.00 24.34           S  \n",
            "ATOM    779  N   MET A 310       8.684  22.740  25.016  1.00 22.44           N  \n",
            "ATOM    780  CA  MET A 310       7.510  21.908  25.250  1.00 23.09           C  \n",
            "ATOM    781  C   MET A 310       7.566  20.546  24.578  1.00 23.57           C  \n",
            "ATOM    782  O   MET A 310       6.892  19.618  25.012  1.00 22.55           O  \n",
            "ATOM    783  CB  MET A 310       6.246  22.628  24.793  1.00 23.30           C  \n",
            "ATOM    784  CG  MET A 310       4.980  21.850  25.067  1.00 25.40           C  \n",
            "ATOM    785  SD  MET A 310       4.669  21.548  26.834  1.00 28.95           S  \n",
            "ATOM    786  CE  MET A 310       3.050  20.854  26.730  1.00 26.79           C  \n",
            "ATOM    787  N   GLU A 311       8.354  20.418  23.515  1.00 24.42           N  \n",
            "ATOM    788  CA  GLU A 311       8.457  19.136  22.829  1.00 24.31           C  \n",
            "ATOM    789  C   GLU A 311       9.258  18.183  23.710  1.00 22.82           C  \n",
            "ATOM    790  O   GLU A 311       8.932  17.003  23.828  1.00 23.79           O  \n",
            "ATOM    791  CB  GLU A 311       9.132  19.322  21.472  1.00 26.25           C  \n",
            "ATOM    792  CG  GLU A 311       8.675  20.586  20.773  1.00 28.91           C  \n",
            "ATOM    793  CD  GLU A 311       9.082  20.635  19.329  1.00 29.55           C  \n",
            "ATOM    794  OE1 GLU A 311      10.268  20.378  19.036  1.00 30.64           O  \n",
            "ATOM    795  OE2 GLU A 311       8.210  20.940  18.488  1.00 30.91           O  \n",
            "ATOM    796  N   ILE A 312      10.293  18.713  24.349  1.00 21.68           N  \n",
            "ATOM    797  CA  ILE A 312      11.136  17.926  25.237  1.00 20.51           C  \n",
            "ATOM    798  C   ILE A 312      10.438  17.664  26.564  1.00 21.92           C  \n",
            "ATOM    799  O   ILE A 312      10.520  16.558  27.105  1.00 23.53           O  \n",
            "ATOM    800  CB  ILE A 312      12.487  18.632  25.484  1.00 18.73           C  \n",
            "ATOM    801  CG1 ILE A 312      13.247  18.743  24.155  1.00 16.76           C  \n",
            "ATOM    802  CG2 ILE A 312      13.301  17.862  26.511  1.00 15.60           C  \n",
            "ATOM    803  CD1 ILE A 312      14.484  19.599  24.206  1.00 17.94           C  \n",
            "ATOM    804  N   MET A 313       9.741  18.667  27.092  1.00 23.62           N  \n",
            "ATOM    805  CA  MET A 313       9.031  18.482  28.353  1.00 24.22           C  \n",
            "ATOM    806  C   MET A 313       7.905  17.460  28.195  1.00 24.89           C  \n",
            "ATOM    807  O   MET A 313       7.683  16.637  29.078  1.00 26.81           O  \n",
            "ATOM    808  CB  MET A 313       8.447  19.800  28.856  1.00 24.53           C  \n",
            "ATOM    809  CG  MET A 313       9.475  20.873  29.164  1.00 25.13           C  \n",
            "ATOM    810  SD  MET A 313       8.776  22.210  30.148  1.00 26.26           S  \n",
            "ATOM    811  CE  MET A 313       7.745  22.994  28.934  1.00 25.53           C  \n",
            "ATOM    812  N   SER A 314       7.195  17.508  27.074  1.00 23.77           N  \n",
            "ATOM    813  CA  SER A 314       6.106  16.566  26.839  1.00 24.41           C  \n",
            "ATOM    814  C   SER A 314       6.663  15.153  26.713  1.00 24.61           C  \n",
            "ATOM    815  O   SER A 314       6.056  14.194  27.184  1.00 25.37           O  \n",
            "ATOM    816  CB  SER A 314       5.335  16.939  25.568  1.00 26.16           C  \n",
            "ATOM    817  OG  SER A 314       4.739  18.222  25.678  1.00 26.87           O  \n",
            "ATOM    818  N   LEU A 315       7.817  15.026  26.064  1.00 24.78           N  \n",
            "ATOM    819  CA  LEU A 315       8.465  13.723  25.912  1.00 24.54           C  \n",
            "ATOM    820  C   LEU A 315       8.857  13.215  27.301  1.00 25.65           C  \n",
            "ATOM    821  O   LEU A 315       8.613  12.053  27.645  1.00 25.44           O  \n",
            "ATOM    822  CB  LEU A 315       9.721  13.842  25.039  1.00 23.20           C  \n",
            "ATOM    823  CG  LEU A 315      10.750  12.709  25.145  1.00 21.16           C  \n",
            "ATOM    824  CD1 LEU A 315      10.105  11.392  24.751  1.00 20.30           C  \n",
            "ATOM    825  CD2 LEU A 315      11.947  13.012  24.258  1.00 18.20           C  \n",
            "ATOM    826  N   ARG A 316       9.465  14.100  28.093  1.00 26.06           N  \n",
            "ATOM    827  CA  ARG A 316       9.885  13.764  29.448  1.00 24.58           C  \n",
            "ATOM    828  C   ARG A 316       8.708  13.337  30.312  1.00 23.89           C  \n",
            "ATOM    829  O   ARG A 316       8.862  12.498  31.192  1.00 26.17           O  \n",
            "ATOM    830  CB  ARG A 316      10.607  14.945  30.093  1.00 23.20           C  \n",
            "ATOM    831  CG  ARG A 316      11.977  15.181  29.518  1.00 23.17           C  \n",
            "ATOM    832  CD  ARG A 316      12.737  16.234  30.283  1.00 24.24           C  \n",
            "ATOM    833  NE  ARG A 316      14.173  16.095  30.069  1.00 26.55           N  \n",
            "ATOM    834  CZ  ARG A 316      15.100  16.781  30.729  1.00 25.76           C  \n",
            "ATOM    835  NH1 ARG A 316      16.384  16.577  30.471  1.00 26.51           N  \n",
            "ATOM    836  NH2 ARG A 316      14.744  17.670  31.642  1.00 25.60           N  \n",
            "ATOM    837  N   ALA A 317       7.532  13.902  30.070  1.00 22.77           N  \n",
            "ATOM    838  CA  ALA A 317       6.370  13.511  30.854  1.00 24.11           C  \n",
            "ATOM    839  C   ALA A 317       5.822  12.178  30.339  1.00 25.96           C  \n",
            "ATOM    840  O   ALA A 317       5.467  11.294  31.126  1.00 25.93           O  \n",
            "ATOM    841  CB  ALA A 317       5.300  14.580  30.779  1.00 23.24           C  \n",
            "ATOM    842  N   ALA A 318       5.773  12.046  29.013  1.00 26.56           N  \n",
            "ATOM    843  CA  ALA A 318       5.276  10.848  28.336  1.00 27.52           C  \n",
            "ATOM    844  C   ALA A 318       5.992   9.550  28.719  1.00 28.64           C  \n",
            "ATOM    845  O   ALA A 318       5.357   8.505  28.856  1.00 28.49           O  \n",
            "ATOM    846  CB  ALA A 318       5.341  11.049  26.811  1.00 28.32           C  \n",
            "ATOM    847  N   VAL A 319       7.314   9.607  28.874  1.00 31.56           N  \n",
            "ATOM    848  CA  VAL A 319       8.083   8.412  29.243  1.00 32.10           C  \n",
            "ATOM    849  C   VAL A 319       7.838   8.025  30.703  1.00 32.77           C  \n",
            "ATOM    850  O   VAL A 319       8.159   6.907  31.119  1.00 33.48           O  \n",
            "ATOM    851  CB  VAL A 319       9.602   8.617  29.037  1.00 29.98           C  \n",
            "ATOM    852  CG1 VAL A 319       9.899   8.842  27.564  1.00 32.04           C  \n",
            "ATOM    853  CG2 VAL A 319      10.081   9.787  29.858  1.00 30.58           C  \n",
            "ATOM    854  N   ARG A 320       7.277   8.956  31.474  1.00 32.06           N  \n",
            "ATOM    855  CA  ARG A 320       6.966   8.699  32.870  1.00 31.62           C  \n",
            "ATOM    856  C   ARG A 320       5.479   8.447  33.060  1.00 31.47           C  \n",
            "ATOM    857  O   ARG A 320       4.897   8.828  34.074  1.00 31.62           O  \n",
            "ATOM    858  CB  ARG A 320       7.420   9.853  33.769  1.00 31.72           C  \n",
            "ATOM    859  CG  ARG A 320       8.913   9.898  33.961  1.00 31.80           C  \n",
            "ATOM    860  CD  ARG A 320       9.310  10.478  35.307  1.00 35.93           C  \n",
            "ATOM    861  NE  ARG A 320       9.960  11.767  35.122  1.00 39.08           N  \n",
            "ATOM    862  CZ  ARG A 320      11.214  12.048  35.458  1.00 41.03           C  \n",
            "ATOM    863  NH1 ARG A 320      12.002  11.136  36.020  1.00 40.21           N  \n",
            "ATOM    864  NH2 ARG A 320      11.674  13.260  35.205  1.00 43.12           N  \n",
            "ATOM    865  N   TYR A 321       4.868   7.799  32.072  1.00 31.18           N  \n",
            "ATOM    866  CA  TYR A 321       3.452   7.457  32.146  1.00 31.14           C  \n",
            "ATOM    867  C   TYR A 321       3.313   6.139  32.905  1.00 32.08           C  \n",
            "ATOM    868  O   TYR A 321       4.061   5.194  32.661  1.00 32.50           O  \n",
            "ATOM    869  CB  TYR A 321       2.852   7.301  30.741  1.00 28.39           C  \n",
            "ATOM    870  CG  TYR A 321       1.417   6.819  30.747  1.00 27.08           C  \n",
            "ATOM    871  CD1 TYR A 321       0.393   7.621  31.250  1.00 27.33           C  \n",
            "ATOM    872  CD2 TYR A 321       1.085   5.545  30.289  1.00 28.01           C  \n",
            "ATOM    873  CE1 TYR A 321      -0.924   7.170  31.309  1.00 25.76           C  \n",
            "ATOM    874  CE2 TYR A 321      -0.234   5.078  30.339  1.00 27.62           C  \n",
            "ATOM    875  CZ  TYR A 321      -1.236   5.901  30.855  1.00 27.94           C  \n",
            "ATOM    876  OH  TYR A 321      -2.541   5.455  30.950  1.00 28.74           O  \n",
            "ATOM    877  N   ASP A 322       2.361   6.082  33.829  1.00 32.64           N  \n",
            "ATOM    878  CA  ASP A 322       2.127   4.878  34.616  1.00 33.51           C  \n",
            "ATOM    879  C   ASP A 322       0.753   4.306  34.267  1.00 35.21           C  \n",
            "ATOM    880  O   ASP A 322      -0.269   4.872  34.642  1.00 36.87           O  \n",
            "ATOM    881  CB  ASP A 322       2.201   5.228  36.094  1.00 33.30           C  \n",
            "ATOM    882  CG  ASP A 322       1.837   4.076  36.984  1.00 37.07           C  \n",
            "ATOM    883  OD1 ASP A 322       2.550   3.050  36.960  1.00 38.24           O  \n",
            "ATOM    884  OD2 ASP A 322       0.832   4.201  37.717  1.00 39.85           O  \n",
            "ATOM    885  N   PRO A 323       0.712   3.175  33.538  1.00 35.91           N  \n",
            "ATOM    886  CA  PRO A 323      -0.557   2.545  33.141  1.00 36.45           C  \n",
            "ATOM    887  C   PRO A 323      -1.437   2.154  34.331  1.00 36.48           C  \n",
            "ATOM    888  O   PRO A 323      -2.656   2.057  34.203  1.00 36.34           O  \n",
            "ATOM    889  CB  PRO A 323      -0.108   1.317  32.343  1.00 36.87           C  \n",
            "ATOM    890  CG  PRO A 323       1.265   1.694  31.859  1.00 36.51           C  \n",
            "ATOM    891  CD  PRO A 323       1.856   2.383  33.057  1.00 35.15           C  \n",
            "ATOM    892  N   GLU A 324      -0.804   1.932  35.481  1.00 37.11           N  \n",
            "ATOM    893  CA  GLU A 324      -1.505   1.544  36.701  1.00 37.58           C  \n",
            "ATOM    894  C   GLU A 324      -2.477   2.637  37.152  1.00 36.55           C  \n",
            "ATOM    895  O   GLU A 324      -3.691   2.475  37.058  1.00 37.27           O  \n",
            "ATOM    896  CB  GLU A 324      -0.483   1.247  37.806  1.00 38.70           C  \n",
            "ATOM    897  CG  GLU A 324      -1.055   0.594  39.063  1.00 42.32           C  \n",
            "ATOM    898  CD  GLU A 324      -1.800  -0.706  38.773  1.00 44.89           C  \n",
            "ATOM    899  OE1 GLU A 324      -2.987  -0.645  38.381  1.00 45.13           O  \n",
            "ATOM    900  OE2 GLU A 324      -1.195  -1.790  38.927  1.00 46.90           O  \n",
            "ATOM    901  N   SER A 325      -1.934   3.750  37.631  1.00 35.96           N  \n",
            "ATOM    902  CA  SER A 325      -2.736   4.876  38.092  1.00 34.26           C  \n",
            "ATOM    903  C   SER A 325      -3.220   5.728  36.925  1.00 34.24           C  \n",
            "ATOM    904  O   SER A 325      -4.108   6.564  37.083  1.00 34.57           O  \n",
            "ATOM    905  CB  SER A 325      -1.909   5.743  39.039  1.00 34.24           C  \n",
            "ATOM    906  OG  SER A 325      -0.746   6.239  38.391  1.00 32.47           O  \n",
            "ATOM    907  N   GLU A 326      -2.628   5.508  35.757  1.00 33.98           N  \n",
            "ATOM    908  CA  GLU A 326      -2.972   6.258  34.557  1.00 34.78           C  \n",
            "ATOM    909  C   GLU A 326      -2.676   7.731  34.743  1.00 33.62           C  \n",
            "ATOM    910  O   GLU A 326      -3.488   8.584  34.400  1.00 34.34           O  \n",
            "ATOM    911  CB  GLU A 326      -4.445   6.054  34.201  1.00 35.78           C  \n",
            "ATOM    912  CG  GLU A 326      -4.698   4.707  33.546  1.00 40.30           C  \n",
            "ATOM    913  CD  GLU A 326      -6.164   4.408  33.341  1.00 41.43           C  \n",
            "ATOM    914  OE1 GLU A 326      -6.879   5.259  32.769  1.00 44.26           O  \n",
            "ATOM    915  OE2 GLU A 326      -6.600   3.312  33.746  1.00 42.26           O  \n",
            "ATOM    916  N   THR A 327      -1.497   8.022  35.278  1.00 32.36           N  \n",
            "ATOM    917  CA  THR A 327      -1.089   9.398  35.524  1.00 31.45           C  \n",
            "ATOM    918  C   THR A 327       0.278   9.674  34.923  1.00 31.26           C  \n",
            "ATOM    919  O   THR A 327       1.025   8.748  34.611  1.00 32.23           O  \n",
            "ATOM    920  CB  THR A 327      -1.014   9.705  37.046  1.00 31.70           C  \n",
            "ATOM    921  OG1 THR A 327       0.069   8.977  37.634  1.00 30.54           O  \n",
            "ATOM    922  CG2 THR A 327      -2.306   9.300  37.738  1.00 30.98           C  \n",
            "ATOM    923  N   LEU A 328       0.588  10.952  34.731  1.00 30.18           N  \n",
            "ATOM    924  CA  LEU A 328       1.890  11.353  34.214  1.00 28.59           C  \n",
            "ATOM    925  C   LEU A 328       2.547  12.033  35.401  1.00 29.48           C  \n",
            "ATOM    926  O   LEU A 328       1.893  12.775  36.135  1.00 26.76           O  \n",
            "ATOM    927  CB  LEU A 328       1.777  12.361  33.054  1.00 26.80           C  \n",
            "ATOM    928  CG  LEU A 328       1.371  11.928  31.633  1.00 26.28           C  \n",
            "ATOM    929  CD1 LEU A 328       1.371  13.140  30.698  1.00 19.56           C  \n",
            "ATOM    930  CD2 LEU A 328       2.338  10.872  31.113  1.00 25.05           C  \n",
            "ATOM    931  N   THR A 329       3.827  11.753  35.614  1.00 30.81           N  \n",
            "ATOM    932  CA  THR A 329       4.552  12.376  36.707  1.00 32.26           C  \n",
            "ATOM    933  C   THR A 329       5.330  13.559  36.148  1.00 32.38           C  \n",
            "ATOM    934  O   THR A 329       6.255  13.380  35.353  1.00 31.40           O  \n",
            "ATOM    935  CB  THR A 329       5.514  11.389  37.391  1.00 32.21           C  \n",
            "ATOM    936  OG1 THR A 329       4.760  10.466  38.184  1.00 32.64           O  \n",
            "ATOM    937  CG2 THR A 329       6.483  12.129  38.290  1.00 34.80           C  \n",
            "ATOM    938  N   LEU A 330       4.933  14.761  36.574  1.00 33.00           N  \n",
            "ATOM    939  CA  LEU A 330       5.548  16.013  36.130  1.00 33.02           C  \n",
            "ATOM    940  C   LEU A 330       6.660  16.508  37.047  1.00 33.15           C  \n",
            "ATOM    941  O   LEU A 330       6.566  16.389  38.262  1.00 32.86           O  \n",
            "ATOM    942  CB  LEU A 330       4.482  17.104  36.000  1.00 30.93           C  \n",
            "ATOM    943  CG  LEU A 330       3.285  16.787  35.104  1.00 28.92           C  \n",
            "ATOM    944  CD1 LEU A 330       2.409  18.019  35.023  1.00 29.23           C  \n",
            "ATOM    945  CD2 LEU A 330       3.743  16.360  33.719  1.00 27.56           C  \n",
            "ATOM    946  N   ASN A 331       7.700  17.080  36.447  1.00 34.88           N  \n",
            "ATOM    947  CA  ASN A 331       8.847  17.590  37.175  1.00 36.43           C  \n",
            "ATOM    948  C   ASN A 331       9.403  16.475  38.061  1.00 36.02           C  \n",
            "ATOM    949  O   ASN A 331      10.109  16.730  39.026  1.00 35.55           O  \n",
            "ATOM    950  CB  ASN A 331       8.465  18.814  38.022  1.00 38.49           C  \n",
            "ATOM    951  CG  ASN A 331       9.687  19.571  38.556  1.00 41.04           C  \n",
            "ATOM    952  OD1 ASN A 331      10.531  20.033  37.799  1.00 41.40           O  \n",
            "ATOM    953  ND2 ASN A 331       9.757  19.720  39.889  1.00 41.31           N  \n",
            "ATOM    954  N   GLY A 332       9.061  15.235  37.722  1.00 37.39           N  \n",
            "ATOM    955  CA  GLY A 332       9.521  14.090  38.485  1.00 38.97           C  \n",
            "ATOM    956  C   GLY A 332       8.943  13.912  39.879  1.00 40.42           C  \n",
            "ATOM    957  O   GLY A 332       9.481  13.139  40.677  1.00 40.93           O  \n",
            "ATOM    958  N   GLU A 333       7.836  14.590  40.175  1.00 41.86           N  \n",
            "ATOM    959  CA  GLU A 333       7.233  14.488  41.502  1.00 41.54           C  \n",
            "ATOM    960  C   GLU A 333       5.730  14.707  41.624  1.00 38.75           C  \n",
            "ATOM    961  O   GLU A 333       5.156  14.416  42.668  1.00 39.84           O  \n",
            "ATOM    962  CB  GLU A 333       7.968  15.421  42.467  1.00 44.34           C  \n",
            "ATOM    963  CG  GLU A 333       8.574  16.643  41.807  1.00 51.17           C  \n",
            "ATOM    964  CD  GLU A 333       7.613  17.812  41.736  1.00 55.12           C  \n",
            "ATOM    965  OE1 GLU A 333       7.110  18.225  42.801  1.00 57.55           O  \n",
            "ATOM    966  OE2 GLU A 333       7.368  18.325  40.621  1.00 56.84           O  \n",
            "ATOM    967  N   MET A 334       5.082  15.205  40.579  1.00 36.18           N  \n",
            "ATOM    968  CA  MET A 334       3.636  15.422  40.643  1.00 31.95           C  \n",
            "ATOM    969  C   MET A 334       2.900  14.524  39.648  1.00 30.15           C  \n",
            "ATOM    970  O   MET A 334       2.992  14.729  38.441  1.00 29.41           O  \n",
            "ATOM    971  CB  MET A 334       3.305  16.885  40.343  1.00 32.66           C  \n",
            "ATOM    972  CG  MET A 334       1.870  17.302  40.663  1.00 32.48           C  \n",
            "ATOM    973  SD  MET A 334       1.411  18.823  39.808  1.00 32.63           S  \n",
            "ATOM    974  CE  MET A 334       2.063  20.063  40.925  1.00 33.08           C  \n",
            "ATOM    975  N   ALA A 335       2.178  13.528  40.153  1.00 27.95           N  \n",
            "ATOM    976  CA  ALA A 335       1.429  12.629  39.288  1.00 27.67           C  \n",
            "ATOM    977  C   ALA A 335       0.028  13.194  39.079  1.00 28.29           C  \n",
            "ATOM    978  O   ALA A 335      -0.737  13.381  40.022  1.00 29.09           O  \n",
            "ATOM    979  CB  ALA A 335       1.356  11.225  39.895  1.00 27.20           C  \n",
            "ATOM    980  N   VAL A 336      -0.294  13.458  37.822  1.00 28.69           N  \n",
            "ATOM    981  CA  VAL A 336      -1.572  14.024  37.447  1.00 27.68           C  \n",
            "ATOM    982  C   VAL A 336      -2.328  13.100  36.485  1.00 28.95           C  \n",
            "ATOM    983  O   VAL A 336      -1.726  12.279  35.792  1.00 30.16           O  \n",
            "ATOM    984  CB  VAL A 336      -1.348  15.401  36.774  1.00 27.84           C  \n",
            "ATOM    985  CG1 VAL A 336      -0.583  16.307  37.718  1.00 26.71           C  \n",
            "ATOM    986  CG2 VAL A 336      -0.565  15.236  35.459  1.00 24.90           C  \n",
            "ATOM    987  N   THR A 337      -3.649  13.221  36.452  1.00 27.73           N  \n",
            "ATOM    988  CA  THR A 337      -4.432  12.402  35.542  1.00 27.55           C  \n",
            "ATOM    989  C   THR A 337      -4.607  13.171  34.246  1.00 26.53           C  \n",
            "ATOM    990  O   THR A 337      -4.142  14.300  34.123  1.00 28.20           O  \n",
            "ATOM    991  CB  THR A 337      -5.814  12.056  36.123  1.00 26.55           C  \n",
            "ATOM    992  OG1 THR A 337      -6.484  13.256  36.519  1.00 29.20           O  \n",
            "ATOM    993  CG2 THR A 337      -5.667  11.153  37.314  1.00 24.42           C  \n",
            "ATOM    994  N   ARG A 338      -5.262  12.544  33.279  1.00 25.80           N  \n",
            "ATOM    995  CA  ARG A 338      -5.499  13.151  31.976  1.00 23.70           C  \n",
            "ATOM    996  C   ARG A 338      -6.426  14.363  32.128  1.00 23.22           C  \n",
            "ATOM    997  O   ARG A 338      -6.160  15.427  31.578  1.00 22.03           O  \n",
            "ATOM    998  CB  ARG A 338      -6.113  12.098  31.041  1.00 21.89           C  \n",
            "ATOM    999  CG  ARG A 338      -6.315  12.520  29.600  1.00 22.79           C  \n",
            "ATOM   1000  CD  ARG A 338      -7.119  11.462  28.823  1.00 22.19           C  \n",
            "ATOM   1001  NE  ARG A 338      -6.364  10.240  28.528  1.00 22.16           N  \n",
            "ATOM   1002  CZ  ARG A 338      -5.804   9.974  27.349  1.00 22.65           C  \n",
            "ATOM   1003  NH1 ARG A 338      -5.131   8.841  27.164  1.00 19.53           N  \n",
            "ATOM   1004  NH2 ARG A 338      -5.925  10.840  26.348  1.00 21.61           N  \n",
            "ATOM   1005  N   GLY A 339      -7.501  14.201  32.897  1.00 22.81           N  \n",
            "ATOM   1006  CA  GLY A 339      -8.442  15.288  33.100  1.00 21.26           C  \n",
            "ATOM   1007  C   GLY A 339      -7.904  16.481  33.875  1.00 23.43           C  \n",
            "ATOM   1008  O   GLY A 339      -8.315  17.622  33.618  1.00 22.46           O  \n",
            "ATOM   1009  N   GLN A 340      -7.006  16.225  34.831  1.00 22.80           N  \n",
            "ATOM   1010  CA  GLN A 340      -6.414  17.284  35.644  1.00 23.68           C  \n",
            "ATOM   1011  C   GLN A 340      -5.552  18.204  34.795  1.00 25.03           C  \n",
            "ATOM   1012  O   GLN A 340      -5.685  19.431  34.853  1.00 26.02           O  \n",
            "ATOM   1013  CB  GLN A 340      -5.555  16.693  36.762  1.00 23.57           C  \n",
            "ATOM   1014  CG  GLN A 340      -6.340  16.031  37.882  1.00 22.80           C  \n",
            "ATOM   1015  CD  GLN A 340      -5.442  15.411  38.936  1.00 22.75           C  \n",
            "ATOM   1016  OE1 GLN A 340      -4.551  14.625  38.623  1.00 23.88           O  \n",
            "ATOM   1017  NE2 GLN A 340      -5.680  15.753  40.195  1.00 23.70           N  \n",
            "ATOM   1018  N   LEU A 341      -4.668  17.600  34.007  1.00 24.03           N  \n",
            "ATOM   1019  CA  LEU A 341      -3.773  18.351  33.142  1.00 25.51           C  \n",
            "ATOM   1020  C   LEU A 341      -4.545  19.005  31.985  1.00 26.30           C  \n",
            "ATOM   1021  O   LEU A 341      -4.152  20.060  31.472  1.00 24.75           O  \n",
            "ATOM   1022  CB  LEU A 341      -2.674  17.417  32.614  1.00 24.02           C  \n",
            "ATOM   1023  CG  LEU A 341      -1.574  18.051  31.769  1.00 24.70           C  \n",
            "ATOM   1024  CD1 LEU A 341      -0.858  19.127  32.572  1.00 25.35           C  \n",
            "ATOM   1025  CD2 LEU A 341      -0.603  16.980  31.325  1.00 26.37           C  \n",
            "ATOM   1026  N   LYS A 342      -5.647  18.371  31.591  1.00 27.67           N  \n",
            "ATOM   1027  CA  LYS A 342      -6.510  18.868  30.522  1.00 29.56           C  \n",
            "ATOM   1028  C   LYS A 342      -7.283  20.106  30.951  1.00 30.53           C  \n",
            "ATOM   1029  O   LYS A 342      -7.305  21.111  30.238  1.00 31.32           O  \n",
            "ATOM   1030  CB  LYS A 342      -7.502  17.791  30.101  1.00 31.06           C  \n",
            "ATOM   1031  CG  LYS A 342      -8.483  18.252  29.051  1.00 32.90           C  \n",
            "ATOM   1032  CD  LYS A 342      -9.042  17.078  28.282  1.00 35.00           C  \n",
            "ATOM   1033  CE  LYS A 342      -9.742  17.555  27.025  1.00 37.38           C  \n",
            "ATOM   1034  NZ  LYS A 342     -10.173  16.415  26.182  1.00 38.96           N  \n",
            "ATOM   1035  N   ASN A 343      -7.920  20.021  32.117  1.00 31.12           N  \n",
            "ATOM   1036  CA  ASN A 343      -8.694  21.131  32.663  1.00 31.14           C  \n",
            "ATOM   1037  C   ASN A 343      -7.787  22.214  33.211  1.00 30.59           C  \n",
            "ATOM   1038  O   ASN A 343      -8.134  23.396  33.179  1.00 30.61           O  \n",
            "ATOM   1039  CB  ASN A 343      -9.600  20.653  33.791  1.00 32.33           C  \n",
            "ATOM   1040  CG  ASN A 343     -10.496  19.525  33.371  1.00 34.59           C  \n",
            "ATOM   1041  OD1 ASN A 343     -10.967  19.483  32.234  1.00 35.21           O  \n",
            "ATOM   1042  ND2 ASN A 343     -10.756  18.604  34.292  1.00 36.52           N  \n",
            "ATOM   1043  N   GLY A 344      -6.627  21.796  33.715  1.00 28.96           N  \n",
            "ATOM   1044  CA  GLY A 344      -5.677  22.729  34.288  1.00 26.43           C  \n",
            "ATOM   1045  C   GLY A 344      -5.004  23.669  33.310  1.00 26.23           C  \n",
            "ATOM   1046  O   GLY A 344      -4.184  24.479  33.718  1.00 26.42           O  \n",
            "ATOM   1047  N   GLY A 345      -5.321  23.556  32.023  1.00 26.66           N  \n",
            "ATOM   1048  CA  GLY A 345      -4.720  24.452  31.055  1.00 25.83           C  \n",
            "ATOM   1049  C   GLY A 345      -4.381  23.914  29.674  1.00 26.68           C  \n",
            "ATOM   1050  O   GLY A 345      -4.511  24.629  28.681  1.00 27.32           O  \n",
            "ATOM   1051  N   LEU A 346      -3.955  22.663  29.582  1.00 26.03           N  \n",
            "ATOM   1052  CA  LEU A 346      -3.566  22.125  28.289  1.00 24.83           C  \n",
            "ATOM   1053  C   LEU A 346      -4.679  21.687  27.342  1.00 26.41           C  \n",
            "ATOM   1054  O   LEU A 346      -4.460  21.611  26.125  1.00 25.35           O  \n",
            "ATOM   1055  CB  LEU A 346      -2.576  20.977  28.489  1.00 23.76           C  \n",
            "ATOM   1056  CG  LEU A 346      -1.216  21.386  29.057  1.00 20.67           C  \n",
            "ATOM   1057  CD1 LEU A 346      -0.306  20.171  29.177  1.00 19.15           C  \n",
            "ATOM   1058  CD2 LEU A 346      -0.602  22.444  28.161  1.00 18.85           C  \n",
            "ATOM   1059  N   GLY A 347      -5.866  21.407  27.880  1.00 25.58           N  \n",
            "ATOM   1060  CA  GLY A 347      -6.948  20.969  27.024  1.00 24.26           C  \n",
            "ATOM   1061  C   GLY A 347      -6.558  19.718  26.242  1.00 25.76           C  \n",
            "ATOM   1062  O   GLY A 347      -6.017  18.771  26.820  1.00 24.99           O  \n",
            "ATOM   1063  N   VAL A 348      -6.818  19.709  24.934  1.00 24.47           N  \n",
            "ATOM   1064  CA  VAL A 348      -6.502  18.553  24.098  1.00 25.12           C  \n",
            "ATOM   1065  C   VAL A 348      -5.030  18.139  24.098  1.00 24.89           C  \n",
            "ATOM   1066  O   VAL A 348      -4.710  16.984  23.823  1.00 25.88           O  \n",
            "ATOM   1067  CB  VAL A 348      -6.944  18.766  22.608  1.00 26.25           C  \n",
            "ATOM   1068  CG1 VAL A 348      -8.449  18.633  22.482  1.00 25.82           C  \n",
            "ATOM   1069  CG2 VAL A 348      -6.480  20.127  22.101  1.00 23.08           C  \n",
            "ATOM   1070  N   VAL A 349      -4.138  19.079  24.394  1.00 25.60           N  \n",
            "ATOM   1071  CA  VAL A 349      -2.702  18.799  24.420  1.00 25.51           C  \n",
            "ATOM   1072  C   VAL A 349      -2.430  17.748  25.497  1.00 25.48           C  \n",
            "ATOM   1073  O   VAL A 349      -1.525  16.924  25.371  1.00 25.35           O  \n",
            "ATOM   1074  CB  VAL A 349      -1.900  20.100  24.714  1.00 26.69           C  \n",
            "ATOM   1075  CG1 VAL A 349      -0.406  19.852  24.594  1.00 27.42           C  \n",
            "ATOM   1076  CG2 VAL A 349      -2.329  21.198  23.742  1.00 27.49           C  \n",
            "ATOM   1077  N   SER A 350      -3.224  17.782  26.560  1.00 25.56           N  \n",
            "ATOM   1078  CA  SER A 350      -3.088  16.810  27.628  1.00 26.09           C  \n",
            "ATOM   1079  C   SER A 350      -3.316  15.429  27.047  1.00 25.08           C  \n",
            "ATOM   1080  O   SER A 350      -2.549  14.510  27.314  1.00 27.18           O  \n",
            "ATOM   1081  CB  SER A 350      -4.115  17.062  28.728  1.00 27.70           C  \n",
            "ATOM   1082  OG  SER A 350      -4.076  16.024  29.691  1.00 27.12           O  \n",
            "ATOM   1083  N   ASP A 351      -4.377  15.290  26.253  1.00 24.59           N  \n",
            "ATOM   1084  CA  ASP A 351      -4.713  14.014  25.620  1.00 25.64           C  \n",
            "ATOM   1085  C   ASP A 351      -3.617  13.529  24.684  1.00 24.97           C  \n",
            "ATOM   1086  O   ASP A 351      -3.350  12.341  24.604  1.00 24.87           O  \n",
            "ATOM   1087  CB  ASP A 351      -6.021  14.125  24.837  1.00 27.84           C  \n",
            "ATOM   1088  CG  ASP A 351      -7.240  14.134  25.733  1.00 31.38           C  \n",
            "ATOM   1089  OD1 ASP A 351      -7.375  13.219  26.578  1.00 33.05           O  \n",
            "ATOM   1090  OD2 ASP A 351      -8.071  15.054  25.583  1.00 33.51           O  \n",
            "ATOM   1091  N   ALA A 352      -2.997  14.460  23.970  1.00 25.24           N  \n",
            "ATOM   1092  CA  ALA A 352      -1.924  14.141  23.051  1.00 23.80           C  \n",
            "ATOM   1093  C   ALA A 352      -0.762  13.505  23.806  1.00 25.00           C  \n",
            "ATOM   1094  O   ALA A 352      -0.246  12.456  23.397  1.00 27.07           O  \n",
            "ATOM   1095  CB  ALA A 352      -1.458  15.393  22.348  1.00 22.23           C  \n",
            "ATOM   1096  N   ILE A 353      -0.357  14.134  24.907  1.00 23.11           N  \n",
            "ATOM   1097  CA  ILE A 353       0.754  13.631  25.705  1.00 23.32           C  \n",
            "ATOM   1098  C   ILE A 353       0.452  12.286  26.369  1.00 24.30           C  \n",
            "ATOM   1099  O   ILE A 353       1.334  11.428  26.464  1.00 26.00           O  \n",
            "ATOM   1100  CB  ILE A 353       1.171  14.645  26.797  1.00 23.78           C  \n",
            "ATOM   1101  CG1 ILE A 353       1.505  15.998  26.170  1.00 23.27           C  \n",
            "ATOM   1102  CG2 ILE A 353       2.393  14.132  27.526  1.00 23.50           C  \n",
            "ATOM   1103  CD1 ILE A 353       1.596  17.118  27.162  1.00 21.49           C  \n",
            "ATOM   1104  N   PHE A 354      -0.774  12.090  26.841  1.00 23.08           N  \n",
            "ATOM   1105  CA  PHE A 354      -1.105  10.813  27.460  1.00 25.47           C  \n",
            "ATOM   1106  C   PHE A 354      -1.132   9.707  26.408  1.00 25.71           C  \n",
            "ATOM   1107  O   PHE A 354      -0.659   8.597  26.652  1.00 28.41           O  \n",
            "ATOM   1108  CB  PHE A 354      -2.456  10.870  28.194  1.00 25.62           C  \n",
            "ATOM   1109  CG  PHE A 354      -2.358  11.393  29.605  1.00 24.25           C  \n",
            "ATOM   1110  CD1 PHE A 354      -2.337  12.766  29.862  1.00 22.66           C  \n",
            "ATOM   1111  CD2 PHE A 354      -2.243  10.510  30.677  1.00 24.76           C  \n",
            "ATOM   1112  CE1 PHE A 354      -2.198  13.247  31.156  1.00 21.71           C  \n",
            "ATOM   1113  CE2 PHE A 354      -2.101  10.986  31.989  1.00 23.61           C  \n",
            "ATOM   1114  CZ  PHE A 354      -2.079  12.354  32.226  1.00 22.72           C  \n",
            "ATOM   1115  N   ASP A 355      -1.693  10.007  25.242  1.00 25.35           N  \n",
            "ATOM   1116  CA  ASP A 355      -1.747   9.035  24.157  1.00 26.25           C  \n",
            "ATOM   1117  C   ASP A 355      -0.316   8.625  23.781  1.00 27.24           C  \n",
            "ATOM   1118  O   ASP A 355      -0.038   7.452  23.531  1.00 27.96           O  \n",
            "ATOM   1119  CB  ASP A 355      -2.457   9.627  22.934  1.00 25.43           C  \n",
            "ATOM   1120  CG  ASP A 355      -3.966   9.725  23.115  1.00 26.85           C  \n",
            "ATOM   1121  OD1 ASP A 355      -4.505   9.112  24.063  1.00 27.90           O  \n",
            "ATOM   1122  OD2 ASP A 355      -4.624  10.403  22.299  1.00 28.66           O  \n",
            "ATOM   1123  N   LEU A 356       0.588   9.599  23.752  1.00 26.96           N  \n",
            "ATOM   1124  CA  LEU A 356       1.984   9.348  23.419  1.00 27.49           C  \n",
            "ATOM   1125  C   LEU A 356       2.625   8.451  24.482  1.00 30.01           C  \n",
            "ATOM   1126  O   LEU A 356       3.370   7.511  24.159  1.00 30.11           O  \n",
            "ATOM   1127  CB  LEU A 356       2.748  10.681  23.335  1.00 25.31           C  \n",
            "ATOM   1128  CG  LEU A 356       4.229  10.630  22.937  1.00 23.36           C  \n",
            "ATOM   1129  CD1 LEU A 356       4.342  10.140  21.505  1.00 19.98           C  \n",
            "ATOM   1130  CD2 LEU A 356       4.874  12.008  23.099  1.00 18.32           C  \n",
            "ATOM   1131  N   GLY A 357       2.337   8.763  25.746  1.00 30.43           N  \n",
            "ATOM   1132  CA  GLY A 357       2.876   7.998  26.854  1.00 32.27           C  \n",
            "ATOM   1133  C   GLY A 357       2.386   6.567  26.827  1.00 33.76           C  \n",
            "ATOM   1134  O   GLY A 357       3.094   5.642  27.226  1.00 35.53           O  \n",
            "ATOM   1135  N   MET A 358       1.162   6.381  26.360  1.00 34.09           N  \n",
            "ATOM   1136  CA  MET A 358       0.601   5.048  26.264  1.00 36.47           C  \n",
            "ATOM   1137  C   MET A 358       1.343   4.225  25.211  1.00 36.94           C  \n",
            "ATOM   1138  O   MET A 358       1.660   3.060  25.445  1.00 37.82           O  \n",
            "ATOM   1139  CB  MET A 358      -0.888   5.117  25.906  1.00 34.77           C  \n",
            "ATOM   1140  CG  MET A 358      -1.762   5.672  27.014  1.00 36.56           C  \n",
            "ATOM   1141  SD  MET A 358      -3.508   5.732  26.596  1.00 36.07           S  \n",
            "ATOM   1142  CE  MET A 358      -3.839   3.994  26.378  1.00 32.15           C  \n",
            "ATOM   1143  N   SER A 359       1.636   4.832  24.063  1.00 37.09           N  \n",
            "ATOM   1144  CA  SER A 359       2.310   4.107  22.995  1.00 37.92           C  \n",
            "ATOM   1145  C   SER A 359       3.824   4.005  23.138  1.00 39.95           C  \n",
            "ATOM   1146  O   SER A 359       4.460   3.224  22.424  1.00 40.05           O  \n",
            "ATOM   1147  CB  SER A 359       1.956   4.703  21.629  1.00 36.21           C  \n",
            "ATOM   1148  OG  SER A 359       2.580   5.946  21.421  1.00 38.52           O  \n",
            "ATOM   1149  N   LEU A 360       4.407   4.772  24.058  1.00 40.90           N  \n",
            "ATOM   1150  CA  LEU A 360       5.852   4.705  24.266  1.00 42.22           C  \n",
            "ATOM   1151  C   LEU A 360       6.275   3.548  25.181  1.00 43.83           C  \n",
            "ATOM   1152  O   LEU A 360       7.427   3.106  25.134  1.00 43.87           O  \n",
            "ATOM   1153  CB  LEU A 360       6.376   6.026  24.834  1.00 41.61           C  \n",
            "ATOM   1154  CG  LEU A 360       6.530   7.199  23.859  1.00 41.62           C  \n",
            "ATOM   1155  CD1 LEU A 360       6.952   8.437  24.624  1.00 40.79           C  \n",
            "ATOM   1156  CD2 LEU A 360       7.556   6.858  22.792  1.00 36.97           C  \n",
            "ATOM   1157  N   SER A 361       5.343   3.057  25.999  1.00 45.33           N  \n",
            "ATOM   1158  CA  SER A 361       5.611   1.954  26.925  1.00 47.16           C  \n",
            "ATOM   1159  C   SER A 361       6.280   0.730  26.298  1.00 49.25           C  \n",
            "ATOM   1160  O   SER A 361       7.146   0.104  26.912  1.00 50.09           O  \n",
            "ATOM   1161  CB  SER A 361       4.319   1.506  27.604  1.00 47.07           C  \n",
            "ATOM   1162  OG  SER A 361       3.873   2.467  28.545  1.00 48.84           O  \n",
            "ATOM   1163  N   SER A 362       5.878   0.378  25.082  1.00 49.99           N  \n",
            "ATOM   1164  CA  SER A 362       6.467  -0.777  24.419  1.00 49.50           C  \n",
            "ATOM   1165  C   SER A 362       7.821  -0.461  23.776  1.00 48.92           C  \n",
            "ATOM   1166  O   SER A 362       8.601  -1.372  23.478  1.00 50.15           O  \n",
            "ATOM   1167  CB  SER A 362       5.508  -1.320  23.359  1.00 49.72           C  \n",
            "ATOM   1168  OG  SER A 362       5.173  -0.311  22.423  1.00 51.09           O  \n",
            "ATOM   1169  N   PHE A 363       8.111   0.820  23.562  1.00 46.64           N  \n",
            "ATOM   1170  CA  PHE A 363       9.381   1.186  22.944  1.00 44.15           C  \n",
            "ATOM   1171  C   PHE A 363      10.568   0.934  23.878  1.00 44.16           C  \n",
            "ATOM   1172  O   PHE A 363      11.702   0.765  23.422  1.00 43.15           O  \n",
            "ATOM   1173  CB  PHE A 363       9.364   2.655  22.485  1.00 41.81           C  \n",
            "ATOM   1174  CG  PHE A 363       8.624   2.887  21.188  1.00 39.44           C  \n",
            "ATOM   1175  CD1 PHE A 363       7.243   2.742  21.120  1.00 39.17           C  \n",
            "ATOM   1176  CD2 PHE A 363       9.315   3.238  20.031  1.00 38.70           C  \n",
            "ATOM   1177  CE1 PHE A 363       6.555   2.940  19.922  1.00 39.02           C  \n",
            "ATOM   1178  CE2 PHE A 363       8.639   3.441  18.823  1.00 39.05           C  \n",
            "ATOM   1179  CZ  PHE A 363       7.255   3.291  18.769  1.00 39.09           C  \n",
            "ATOM   1180  N   ASN A 364      10.308   0.897  25.183  1.00 44.12           N  \n",
            "ATOM   1181  CA  ASN A 364      11.372   0.659  26.158  1.00 44.65           C  \n",
            "ATOM   1182  C   ASN A 364      12.577   1.550  25.861  1.00 43.10           C  \n",
            "ATOM   1183  O   ASN A 364      13.674   1.059  25.601  1.00 44.65           O  \n",
            "ATOM   1184  CB  ASN A 364      11.807  -0.813  26.115  1.00 47.17           C  \n",
            "ATOM   1185  CG  ASN A 364      12.778  -1.165  27.229  1.00 49.90           C  \n",
            "ATOM   1186  OD1 ASN A 364      12.487  -0.942  28.406  1.00 52.33           O  \n",
            "ATOM   1187  ND2 ASN A 364      13.934  -1.718  26.867  1.00 50.06           N  \n",
            "ATOM   1188  N   LEU A 365      12.370   2.860  25.901  1.00 39.84           N  \n",
            "ATOM   1189  CA  LEU A 365      13.433   3.812  25.610  1.00 34.70           C  \n",
            "ATOM   1190  C   LEU A 365      14.349   4.059  26.802  1.00 32.31           C  \n",
            "ATOM   1191  O   LEU A 365      13.884   4.349  27.898  1.00 33.67           O  \n",
            "ATOM   1192  CB  LEU A 365      12.812   5.133  25.156  1.00 34.40           C  \n",
            "ATOM   1193  CG  LEU A 365      11.873   5.088  23.944  1.00 33.26           C  \n",
            "ATOM   1194  CD1 LEU A 365      11.291   6.483  23.712  1.00 30.99           C  \n",
            "ATOM   1195  CD2 LEU A 365      12.625   4.591  22.699  1.00 31.51           C  \n",
            "ATOM   1196  N   ASP A 366      15.656   3.951  26.603  1.00 29.52           N  \n",
            "ATOM   1197  CA  ASP A 366      16.567   4.192  27.711  1.00 27.93           C  \n",
            "ATOM   1198  C   ASP A 366      16.981   5.670  27.774  1.00 27.17           C  \n",
            "ATOM   1199  O   ASP A 366      16.603   6.464  26.912  1.00 26.45           O  \n",
            "ATOM   1200  CB  ASP A 366      17.790   3.259  27.627  1.00 27.38           C  \n",
            "ATOM   1201  CG  ASP A 366      18.648   3.499  26.403  1.00 29.51           C  \n",
            "ATOM   1202  OD1 ASP A 366      18.392   4.459  25.655  1.00 30.19           O  \n",
            "ATOM   1203  OD2 ASP A 366      19.598   2.721  26.194  1.00 30.97           O  \n",
            "ATOM   1204  N   ASP A 367      17.749   6.042  28.791  1.00 26.81           N  \n",
            "ATOM   1205  CA  ASP A 367      18.157   7.432  28.947  1.00 28.39           C  \n",
            "ATOM   1206  C   ASP A 367      18.881   7.970  27.722  1.00 29.10           C  \n",
            "ATOM   1207  O   ASP A 367      18.881   9.180  27.474  1.00 29.01           O  \n",
            "ATOM   1208  CB  ASP A 367      19.041   7.613  30.190  1.00 28.90           C  \n",
            "ATOM   1209  CG  ASP A 367      18.362   7.145  31.468  1.00 31.34           C  \n",
            "ATOM   1210  OD1 ASP A 367      17.199   7.524  31.719  1.00 32.40           O  \n",
            "ATOM   1211  OD2 ASP A 367      18.999   6.400  32.234  1.00 34.15           O  \n",
            "ATOM   1212  N   THR A 368      19.498   7.076  26.956  1.00 28.27           N  \n",
            "ATOM   1213  CA  THR A 368      20.213   7.487  25.755  1.00 27.51           C  \n",
            "ATOM   1214  C   THR A 368      19.228   7.787  24.631  1.00 28.00           C  \n",
            "ATOM   1215  O   THR A 368      19.254   8.864  24.041  1.00 29.29           O  \n",
            "ATOM   1216  CB  THR A 368      21.192   6.397  25.289  1.00 27.44           C  \n",
            "ATOM   1217  OG1 THR A 368      22.213   6.215  26.280  1.00 27.29           O  \n",
            "ATOM   1218  CG2 THR A 368      21.825   6.785  23.961  1.00 24.04           C  \n",
            "ATOM   1219  N   GLU A 369      18.348   6.836  24.344  1.00 28.98           N  \n",
            "ATOM   1220  CA  GLU A 369      17.364   7.020  23.288  1.00 29.53           C  \n",
            "ATOM   1221  C   GLU A 369      16.500   8.243  23.546  1.00 29.34           C  \n",
            "ATOM   1222  O   GLU A 369      16.194   8.993  22.611  1.00 29.58           O  \n",
            "ATOM   1223  CB  GLU A 369      16.516   5.761  23.140  1.00 29.64           C  \n",
            "ATOM   1224  CG  GLU A 369      17.371   4.587  22.722  1.00 30.12           C  \n",
            "ATOM   1225  CD  GLU A 369      16.718   3.248  22.944  1.00 28.83           C  \n",
            "ATOM   1226  OE1 GLU A 369      16.080   3.049  23.995  1.00 28.60           O  \n",
            "ATOM   1227  OE2 GLU A 369      16.873   2.382  22.068  1.00 30.41           O  \n",
            "ATOM   1228  N   VAL A 370      16.120   8.454  24.806  1.00 28.47           N  \n",
            "ATOM   1229  CA  VAL A 370      15.316   9.622  25.165  1.00 26.68           C  \n",
            "ATOM   1230  C   VAL A 370      16.181  10.882  25.032  1.00 26.77           C  \n",
            "ATOM   1231  O   VAL A 370      15.717  11.908  24.530  1.00 27.09           O  \n",
            "ATOM   1232  CB  VAL A 370      14.765   9.517  26.610  1.00 24.18           C  \n",
            "ATOM   1233  CG1 VAL A 370      14.243  10.866  27.080  1.00 23.24           C  \n",
            "ATOM   1234  CG2 VAL A 370      13.652   8.505  26.656  1.00 21.40           C  \n",
            "ATOM   1235  N   ALA A 371      17.439  10.792  25.461  1.00 26.23           N  \n",
            "ATOM   1236  CA  ALA A 371      18.354  11.923  25.374  1.00 26.16           C  \n",
            "ATOM   1237  C   ALA A 371      18.619  12.320  23.921  1.00 27.60           C  \n",
            "ATOM   1238  O   ALA A 371      18.718  13.504  23.609  1.00 28.05           O  \n",
            "ATOM   1239  CB  ALA A 371      19.664  11.597  26.072  1.00 25.61           C  \n",
            "ATOM   1240  N   LEU A 372      18.733  11.335  23.032  1.00 28.84           N  \n",
            "ATOM   1241  CA  LEU A 372      18.976  11.623  21.619  1.00 27.91           C  \n",
            "ATOM   1242  C   LEU A 372      17.737  12.218  20.976  1.00 28.56           C  \n",
            "ATOM   1243  O   LEU A 372      17.842  13.057  20.085  1.00 30.82           O  \n",
            "ATOM   1244  CB  LEU A 372      19.393  10.360  20.873  1.00 28.87           C  \n",
            "ATOM   1245  CG  LEU A 372      20.787   9.816  21.205  1.00 29.14           C  \n",
            "ATOM   1246  CD1 LEU A 372      20.977   8.493  20.491  1.00 30.60           C  \n",
            "ATOM   1247  CD2 LEU A 372      21.863  10.808  20.785  1.00 26.15           C  \n",
            "ATOM   1248  N   LEU A 373      16.562  11.784  21.426  1.00 28.30           N  \n",
            "ATOM   1249  CA  LEU A 373      15.307  12.316  20.905  1.00 27.87           C  \n",
            "ATOM   1250  C   LEU A 373      15.244  13.815  21.213  1.00 28.21           C  \n",
            "ATOM   1251  O   LEU A 373      14.792  14.611  20.389  1.00 26.67           O  \n",
            "ATOM   1252  CB  LEU A 373      14.109  11.607  21.551  1.00 26.81           C  \n",
            "ATOM   1253  CG  LEU A 373      13.644  10.266  20.976  1.00 25.09           C  \n",
            "ATOM   1254  CD1 LEU A 373      12.594   9.655  21.868  1.00 25.88           C  \n",
            "ATOM   1255  CD2 LEU A 373      13.092  10.473  19.595  1.00 24.39           C  \n",
            "ATOM   1256  N   GLN A 374      15.702  14.194  22.406  1.00 27.13           N  \n",
            "ATOM   1257  CA  GLN A 374      15.702  15.599  22.814  1.00 24.91           C  \n",
            "ATOM   1258  C   GLN A 374      16.635  16.450  21.944  1.00 23.49           C  \n",
            "ATOM   1259  O   GLN A 374      16.276  17.558  21.540  1.00 21.46           O  \n",
            "ATOM   1260  CB  GLN A 374      16.072  15.703  24.297  1.00 23.47           C  \n",
            "ATOM   1261  CG  GLN A 374      15.005  15.102  25.214  1.00 23.18           C  \n",
            "ATOM   1262  CD  GLN A 374      15.399  15.083  26.686  1.00 25.19           C  \n",
            "ATOM   1263  OE1 GLN A 374      14.601  14.695  27.541  1.00 23.10           O  \n",
            "ATOM   1264  NE2 GLN A 374      16.633  15.497  26.988  1.00 23.13           N  \n",
            "ATOM   1265  N   ALA A 375      17.819  15.912  21.649  1.00 22.76           N  \n",
            "ATOM   1266  CA  ALA A 375      18.820  16.576  20.810  1.00 22.93           C  \n",
            "ATOM   1267  C   ALA A 375      18.272  16.813  19.396  1.00 22.51           C  \n",
            "ATOM   1268  O   ALA A 375      18.483  17.868  18.799  1.00 21.70           O  \n",
            "ATOM   1269  CB  ALA A 375      20.093  15.726  20.745  1.00 20.21           C  \n",
            "ATOM   1270  N   VAL A 376      17.568  15.820  18.866  1.00 22.16           N  \n",
            "ATOM   1271  CA  VAL A 376      16.972  15.932  17.543  1.00 21.72           C  \n",
            "ATOM   1272  C   VAL A 376      15.937  17.061  17.552  1.00 24.34           C  \n",
            "ATOM   1273  O   VAL A 376      15.906  17.888  16.641  1.00 26.21           O  \n",
            "ATOM   1274  CB  VAL A 376      16.296  14.606  17.138  1.00 19.57           C  \n",
            "ATOM   1275  CG1 VAL A 376      15.497  14.782  15.860  1.00 20.62           C  \n",
            "ATOM   1276  CG2 VAL A 376      17.346  13.543  16.944  1.00 17.47           C  \n",
            "ATOM   1277  N   LEU A 377      15.098  17.094  18.589  1.00 25.10           N  \n",
            "ATOM   1278  CA  LEU A 377      14.083  18.129  18.726  1.00 25.02           C  \n",
            "ATOM   1279  C   LEU A 377      14.726  19.503  18.884  1.00 24.94           C  \n",
            "ATOM   1280  O   LEU A 377      14.281  20.480  18.283  1.00 25.18           O  \n",
            "ATOM   1281  CB  LEU A 377      13.187  17.835  19.935  1.00 25.39           C  \n",
            "ATOM   1282  CG  LEU A 377      12.351  16.551  19.853  1.00 27.50           C  \n",
            "ATOM   1283  CD1 LEU A 377      11.708  16.257  21.208  1.00 24.49           C  \n",
            "ATOM   1284  CD2 LEU A 377      11.297  16.692  18.740  1.00 27.23           C  \n",
            "ATOM   1285  N   LEU A 378      15.778  19.570  19.694  1.00 24.74           N  \n",
            "ATOM   1286  CA  LEU A 378      16.490  20.821  19.942  1.00 24.93           C  \n",
            "ATOM   1287  C   LEU A 378      17.106  21.415  18.679  1.00 26.18           C  \n",
            "ATOM   1288  O   LEU A 378      16.917  22.602  18.390  1.00 27.02           O  \n",
            "ATOM   1289  CB  LEU A 378      17.603  20.606  20.978  1.00 22.51           C  \n",
            "ATOM   1290  CG  LEU A 378      18.508  21.809  21.290  1.00 18.78           C  \n",
            "ATOM   1291  CD1 LEU A 378      17.772  22.795  22.165  1.00 15.37           C  \n",
            "ATOM   1292  CD2 LEU A 378      19.780  21.340  21.987  1.00 16.69           C  \n",
            "ATOM   1293  N   MET A 379      17.843  20.588  17.936  1.00 26.52           N  \n",
            "ATOM   1294  CA  MET A 379      18.508  21.032  16.713  1.00 27.18           C  \n",
            "ATOM   1295  C   MET A 379      17.628  21.029  15.464  1.00 27.67           C  \n",
            "ATOM   1296  O   MET A 379      17.989  20.432  14.446  1.00 28.30           O  \n",
            "ATOM   1297  CB  MET A 379      19.753  20.176  16.455  1.00 28.47           C  \n",
            "ATOM   1298  CG  MET A 379      20.800  20.263  17.552  1.00 30.41           C  \n",
            "ATOM   1299  SD  MET A 379      21.351  21.947  17.894  1.00 30.13           S  \n",
            "ATOM   1300  CE  MET A 379      22.849  21.642  18.777  1.00 30.09           C  \n",
            "ATOM   1301  N   SER A 380      16.478  21.699  15.540  1.00 27.23           N  \n",
            "ATOM   1302  CA  SER A 380      15.570  21.784  14.401  1.00 26.26           C  \n",
            "ATOM   1303  C   SER A 380      15.818  23.096  13.657  1.00 26.07           C  \n",
            "ATOM   1304  O   SER A 380      15.492  24.159  14.168  1.00 24.63           O  \n",
            "ATOM   1305  CB  SER A 380      14.129  21.713  14.886  1.00 26.18           C  \n",
            "ATOM   1306  OG  SER A 380      13.927  20.525  15.618  1.00 28.11           O  \n",
            "ATOM   1307  N   SER A 381      16.387  23.018  12.455  1.00 27.56           N  \n",
            "ATOM   1308  CA  SER A 381      16.709  24.221  11.673  1.00 30.93           C  \n",
            "ATOM   1309  C   SER A 381      15.542  24.911  10.966  1.00 31.03           C  \n",
            "ATOM   1310  O   SER A 381      15.713  25.962  10.346  1.00 33.56           O  \n",
            "ATOM   1311  CB  SER A 381      17.824  23.917  10.650  1.00 31.10           C  \n",
            "ATOM   1312  OG  SER A 381      17.538  22.770   9.864  1.00 31.15           O  \n",
            "ATOM   1313  N   ASP A 382      14.356  24.334  11.076  1.00 30.22           N  \n",
            "ATOM   1314  CA  ASP A 382      13.176  24.904  10.456  1.00 29.77           C  \n",
            "ATOM   1315  C   ASP A 382      12.546  26.023  11.293  1.00 29.65           C  \n",
            "ATOM   1316  O   ASP A 382      11.582  26.659  10.853  1.00 30.76           O  \n",
            "ATOM   1317  CB  ASP A 382      12.149  23.801  10.199  1.00 31.45           C  \n",
            "ATOM   1318  CG  ASP A 382      11.961  22.894  11.403  1.00 37.24           C  \n",
            "ATOM   1319  OD1 ASP A 382      10.800  22.654  11.813  1.00 40.26           O  \n",
            "ATOM   1320  OD2 ASP A 382      12.978  22.412  11.947  1.00 39.21           O  \n",
            "ATOM   1321  N   ARG A 383      13.070  26.265  12.494  1.00 28.61           N  \n",
            "ATOM   1322  CA  ARG A 383      12.522  27.326  13.345  1.00 28.06           C  \n",
            "ATOM   1323  C   ARG A 383      12.719  28.684  12.648  1.00 28.95           C  \n",
            "ATOM   1324  O   ARG A 383      13.733  28.915  11.994  1.00 30.31           O  \n",
            "ATOM   1325  CB  ARG A 383      13.224  27.360  14.705  1.00 26.16           C  \n",
            "ATOM   1326  CG  ARG A 383      13.433  26.020  15.382  1.00 24.74           C  \n",
            "ATOM   1327  CD  ARG A 383      12.166  25.412  15.934  1.00 24.01           C  \n",
            "ATOM   1328  NE  ARG A 383      12.476  24.272  16.797  1.00 24.35           N  \n",
            "ATOM   1329  CZ  ARG A 383      11.582  23.388  17.230  1.00 25.87           C  \n",
            "ATOM   1330  NH1 ARG A 383      10.304  23.506  16.883  1.00 24.63           N  \n",
            "ATOM   1331  NH2 ARG A 383      11.972  22.379  17.997  1.00 24.47           N  \n",
            "ATOM   1332  N   PRO A 384      11.739  29.590  12.767  1.00 29.64           N  \n",
            "ATOM   1333  CA  PRO A 384      11.798  30.923  12.155  1.00 28.86           C  \n",
            "ATOM   1334  C   PRO A 384      12.865  31.839  12.744  1.00 30.09           C  \n",
            "ATOM   1335  O   PRO A 384      13.049  31.878  13.963  1.00 32.21           O  \n",
            "ATOM   1336  CB  PRO A 384      10.401  31.485  12.416  1.00 27.40           C  \n",
            "ATOM   1337  CG  PRO A 384       9.545  30.263  12.441  1.00 29.10           C  \n",
            "ATOM   1338  CD  PRO A 384      10.377  29.309  13.257  1.00 30.13           C  \n",
            "ATOM   1339  N   GLY A 385      13.551  32.577  11.874  1.00 29.48           N  \n",
            "ATOM   1340  CA  GLY A 385      14.558  33.523  12.314  1.00 28.27           C  \n",
            "ATOM   1341  C   GLY A 385      15.951  33.018  12.624  1.00 29.69           C  \n",
            "ATOM   1342  O   GLY A 385      16.836  33.823  12.935  1.00 30.28           O  \n",
            "ATOM   1343  N   LEU A 386      16.162  31.707  12.556  1.00 29.53           N  \n",
            "ATOM   1344  CA  LEU A 386      17.483  31.149  12.844  1.00 30.33           C  \n",
            "ATOM   1345  C   LEU A 386      18.522  31.680  11.875  1.00 30.89           C  \n",
            "ATOM   1346  O   LEU A 386      18.243  31.874  10.697  1.00 33.03           O  \n",
            "ATOM   1347  CB  LEU A 386      17.464  29.623  12.766  1.00 28.37           C  \n",
            "ATOM   1348  CG  LEU A 386      16.803  28.904  13.942  1.00 29.62           C  \n",
            "ATOM   1349  CD1 LEU A 386      16.731  27.405  13.642  1.00 28.37           C  \n",
            "ATOM   1350  CD2 LEU A 386      17.596  29.181  15.229  1.00 24.36           C  \n",
            "ATOM   1351  N   ALA A 387      19.729  31.908  12.375  1.00 31.90           N  \n",
            "ATOM   1352  CA  ALA A 387      20.808  32.411  11.539  1.00 30.12           C  \n",
            "ATOM   1353  C   ALA A 387      21.681  31.278  11.000  1.00 30.46           C  \n",
            "ATOM   1354  O   ALA A 387      21.966  31.226   9.809  1.00 30.72           O  \n",
            "ATOM   1355  CB  ALA A 387      21.651  33.381  12.324  1.00 27.37           C  \n",
            "ATOM   1356  N   CYS A 388      22.082  30.363  11.879  1.00 30.50           N  \n",
            "ATOM   1357  CA  CYS A 388      22.942  29.251  11.496  1.00 30.44           C  \n",
            "ATOM   1358  C   CYS A 388      22.150  28.001  11.153  1.00 31.57           C  \n",
            "ATOM   1359  O   CYS A 388      22.289  26.966  11.804  1.00 32.10           O  \n",
            "ATOM   1360  CB  CYS A 388      23.928  28.946  12.625  1.00 30.88           C  \n",
            "ATOM   1361  SG  CYS A 388      24.815  30.412  13.253  1.00 36.06           S  \n",
            "ATOM   1362  N   VAL A 389      21.325  28.103  10.119  1.00 32.80           N  \n",
            "ATOM   1363  CA  VAL A 389      20.498  26.988   9.670  1.00 34.44           C  \n",
            "ATOM   1364  C   VAL A 389      21.339  25.775   9.263  1.00 35.65           C  \n",
            "ATOM   1365  O   VAL A 389      21.021  24.637   9.610  1.00 36.10           O  \n",
            "ATOM   1366  CB  VAL A 389      19.610  27.407   8.464  1.00 34.00           C  \n",
            "ATOM   1367  CG1 VAL A 389      18.739  26.242   8.021  1.00 31.98           C  \n",
            "ATOM   1368  CG2 VAL A 389      18.746  28.616   8.844  1.00 32.94           C  \n",
            "ATOM   1369  N   GLU A 390      22.412  26.021   8.525  1.00 36.64           N  \n",
            "ATOM   1370  CA  GLU A 390      23.271  24.939   8.074  1.00 38.08           C  \n",
            "ATOM   1371  C   GLU A 390      23.976  24.243   9.238  1.00 36.90           C  \n",
            "ATOM   1372  O   GLU A 390      23.925  23.021   9.365  1.00 35.64           O  \n",
            "ATOM   1373  CB  GLU A 390      24.306  25.481   7.098  1.00 42.25           C  \n",
            "ATOM   1374  CG  GLU A 390      25.194  24.409   6.500  1.00 48.32           C  \n",
            "ATOM   1375  CD  GLU A 390      26.428  24.994   5.842  1.00 53.20           C  \n",
            "ATOM   1376  OE1 GLU A 390      27.324  25.456   6.593  1.00 52.00           O  \n",
            "ATOM   1377  OE2 GLU A 390      26.486  25.000   4.584  1.00 53.94           O  \n",
            "ATOM   1378  N   ARG A 391      24.645  25.029  10.075  1.00 35.75           N  \n",
            "ATOM   1379  CA  ARG A 391      25.355  24.499  11.227  1.00 35.93           C  \n",
            "ATOM   1380  C   ARG A 391      24.434  23.691  12.167  1.00 35.17           C  \n",
            "ATOM   1381  O   ARG A 391      24.866  22.710  12.771  1.00 34.39           O  \n",
            "ATOM   1382  CB  ARG A 391      26.021  25.649  11.990  1.00 38.20           C  \n",
            "ATOM   1383  CG  ARG A 391      26.821  25.204  13.207  1.00 44.56           C  \n",
            "ATOM   1384  CD  ARG A 391      28.171  25.904  13.266  1.00 49.20           C  \n",
            "ATOM   1385  NE  ARG A 391      28.085  27.345  13.520  1.00 53.50           N  \n",
            "ATOM   1386  CZ  ARG A 391      28.066  27.900  14.732  1.00 54.86           C  \n",
            "ATOM   1387  NH1 ARG A 391      28.121  27.138  15.820  1.00 54.95           N  \n",
            "ATOM   1388  NH2 ARG A 391      28.021  29.224  14.858  1.00 54.20           N  \n",
            "ATOM   1389  N   ILE A 392      23.169  24.100  12.278  1.00 32.45           N  \n",
            "ATOM   1390  CA  ILE A 392      22.206  23.415  13.135  1.00 31.64           C  \n",
            "ATOM   1391  C   ILE A 392      21.747  22.103  12.497  1.00 32.47           C  \n",
            "ATOM   1392  O   ILE A 392      21.631  21.070  13.167  1.00 30.64           O  \n",
            "ATOM   1393  CB  ILE A 392      20.970  24.323  13.404  1.00 31.42           C  \n",
            "ATOM   1394  CG1 ILE A 392      21.382  25.491  14.296  1.00 29.78           C  \n",
            "ATOM   1395  CG2 ILE A 392      19.829  23.522  14.057  1.00 29.38           C  \n",
            "ATOM   1396  CD1 ILE A 392      20.367  26.608  14.358  1.00 27.30           C  \n",
            "ATOM   1397  N   GLU A 393      21.488  22.161  11.196  1.00 33.72           N  \n",
            "ATOM   1398  CA  GLU A 393      21.045  20.998  10.445  1.00 35.66           C  \n",
            "ATOM   1399  C   GLU A 393      22.141  19.932  10.476  1.00 36.45           C  \n",
            "ATOM   1400  O   GLU A 393      21.855  18.738  10.476  1.00 37.34           O  \n",
            "ATOM   1401  CB  GLU A 393      20.714  21.415   9.003  1.00 36.35           C  \n",
            "ATOM   1402  CG  GLU A 393      19.820  20.442   8.244  1.00 38.26           C  \n",
            "ATOM   1403  CD  GLU A 393      19.066  21.099   7.085  1.00 40.56           C  \n",
            "ATOM   1404  OE1 GLU A 393      18.329  22.080   7.325  1.00 37.41           O  \n",
            "ATOM   1405  OE2 GLU A 393      19.202  20.626   5.933  1.00 42.55           O  \n",
            "ATOM   1406  N   LYS A 394      23.396  20.371  10.521  1.00 37.31           N  \n",
            "ATOM   1407  CA  LYS A 394      24.525  19.454  10.562  1.00 37.87           C  \n",
            "ATOM   1408  C   LYS A 394      24.568  18.733  11.898  1.00 37.32           C  \n",
            "ATOM   1409  O   LYS A 394      24.757  17.519  11.946  1.00 37.67           O  \n",
            "ATOM   1410  CB  LYS A 394      25.846  20.201  10.364  1.00 40.15           C  \n",
            "ATOM   1411  CG  LYS A 394      27.039  19.265  10.277  1.00 43.24           C  \n",
            "ATOM   1412  CD  LYS A 394      28.380  19.970  10.454  1.00 47.02           C  \n",
            "ATOM   1413  CE  LYS A 394      29.523  18.963  10.283  1.00 48.92           C  \n",
            "ATOM   1414  NZ  LYS A 394      30.883  19.570  10.360  1.00 50.70           N  \n",
            "ATOM   1415  N   TYR A 395      24.406  19.485  12.985  1.00 37.14           N  \n",
            "ATOM   1416  CA  TYR A 395      24.426  18.897  14.323  1.00 36.03           C  \n",
            "ATOM   1417  C   TYR A 395      23.290  17.884  14.447  1.00 35.21           C  \n",
            "ATOM   1418  O   TYR A 395      23.449  16.820  15.040  1.00 33.23           O  \n",
            "ATOM   1419  CB  TYR A 395      24.269  19.979  15.399  1.00 35.68           C  \n",
            "ATOM   1420  CG  TYR A 395      25.403  20.989  15.479  1.00 37.36           C  \n",
            "ATOM   1421  CD1 TYR A 395      26.720  20.628  15.173  1.00 37.10           C  \n",
            "ATOM   1422  CD2 TYR A 395      25.170  22.293  15.936  1.00 38.20           C  \n",
            "ATOM   1423  CE1 TYR A 395      27.776  21.539  15.324  1.00 36.49           C  \n",
            "ATOM   1424  CE2 TYR A 395      26.218  23.206  16.094  1.00 38.59           C  \n",
            "ATOM   1425  CZ  TYR A 395      27.519  22.823  15.788  1.00 39.21           C  \n",
            "ATOM   1426  OH  TYR A 395      28.552  23.723  15.968  1.00 40.37           O  \n",
            "ATOM   1427  N   GLN A 396      22.143  18.219  13.871  1.00 35.76           N  \n",
            "ATOM   1428  CA  GLN A 396      21.003  17.325  13.917  1.00 36.51           C  \n",
            "ATOM   1429  C   GLN A 396      21.301  16.034  13.164  1.00 36.32           C  \n",
            "ATOM   1430  O   GLN A 396      20.871  14.960  13.583  1.00 37.98           O  \n",
            "ATOM   1431  CB  GLN A 396      19.775  17.988  13.313  1.00 37.07           C  \n",
            "ATOM   1432  CG  GLN A 396      18.500  17.215  13.569  1.00 38.57           C  \n",
            "ATOM   1433  CD  GLN A 396      17.317  17.824  12.862  1.00 40.08           C  \n",
            "ATOM   1434  OE1 GLN A 396      16.191  17.778  13.359  1.00 41.08           O  \n",
            "ATOM   1435  NE2 GLN A 396      17.561  18.395  11.690  1.00 39.22           N  \n",
            "ATOM   1436  N   ASP A 397      22.025  16.134  12.051  1.00 35.25           N  \n",
            "ATOM   1437  CA  ASP A 397      22.365  14.941  11.283  1.00 34.77           C  \n",
            "ATOM   1438  C   ASP A 397      23.302  14.061  12.096  1.00 33.08           C  \n",
            "ATOM   1439  O   ASP A 397      23.180  12.838  12.100  1.00 31.41           O  \n",
            "ATOM   1440  CB  ASP A 397      23.022  15.310   9.950  1.00 35.20           C  \n",
            "ATOM   1441  CG  ASP A 397      22.046  15.926   8.970  1.00 38.02           C  \n",
            "ATOM   1442  OD1 ASP A 397      20.856  15.535   8.980  1.00 37.56           O  \n",
            "ATOM   1443  OD2 ASP A 397      22.472  16.794   8.175  1.00 39.63           O  \n",
            "ATOM   1444  N   SER A 398      24.230  14.699  12.796  1.00 32.72           N  \n",
            "ATOM   1445  CA  SER A 398      25.182  13.989  13.632  1.00 34.35           C  \n",
            "ATOM   1446  C   SER A 398      24.450  13.190  14.710  1.00 34.89           C  \n",
            "ATOM   1447  O   SER A 398      24.834  12.062  15.013  1.00 35.53           O  \n",
            "ATOM   1448  CB  SER A 398      26.150  14.978  14.284  1.00 34.25           C  \n",
            "ATOM   1449  OG  SER A 398      26.755  15.816  13.315  1.00 35.65           O  \n",
            "ATOM   1450  N   PHE A 399      23.394  13.771  15.282  1.00 35.54           N  \n",
            "ATOM   1451  CA  PHE A 399      22.618  13.094  16.325  1.00 36.66           C  \n",
            "ATOM   1452  C   PHE A 399      21.784  11.960  15.744  1.00 37.02           C  \n",
            "ATOM   1453  O   PHE A 399      21.784  10.843  16.270  1.00 37.51           O  \n",
            "ATOM   1454  CB  PHE A 399      21.689  14.081  17.050  1.00 36.77           C  \n",
            "ATOM   1455  CG  PHE A 399      22.400  15.010  18.004  1.00 36.70           C  \n",
            "ATOM   1456  CD1 PHE A 399      22.338  16.390  17.823  1.00 35.99           C  \n",
            "ATOM   1457  CD2 PHE A 399      23.123  14.506  19.088  1.00 36.02           C  \n",
            "ATOM   1458  CE1 PHE A 399      22.985  17.259  18.706  1.00 37.04           C  \n",
            "ATOM   1459  CE2 PHE A 399      23.777  15.366  19.983  1.00 34.91           C  \n",
            "ATOM   1460  CZ  PHE A 399      23.709  16.745  19.791  1.00 36.03           C  \n",
            "ATOM   1461  N   LEU A 400      21.064  12.255  14.665  1.00 37.35           N  \n",
            "ATOM   1462  CA  LEU A 400      20.222  11.263  14.008  1.00 36.90           C  \n",
            "ATOM   1463  C   LEU A 400      21.051  10.032  13.664  1.00 36.58           C  \n",
            "ATOM   1464  O   LEU A 400      20.623   8.896  13.870  1.00 36.36           O  \n",
            "ATOM   1465  CB  LEU A 400      19.622  11.857  12.734  1.00 35.74           C  \n",
            "ATOM   1466  CG  LEU A 400      18.437  12.806  12.928  1.00 36.30           C  \n",
            "ATOM   1467  CD1 LEU A 400      18.213  13.654  11.689  1.00 33.72           C  \n",
            "ATOM   1468  CD2 LEU A 400      17.203  11.991  13.246  1.00 35.94           C  \n",
            "ATOM   1469  N   LEU A 401      22.255  10.278  13.164  1.00 35.77           N  \n",
            "ATOM   1470  CA  LEU A 401      23.157   9.213  12.768  1.00 35.97           C  \n",
            "ATOM   1471  C   LEU A 401      23.584   8.363  13.953  1.00 35.70           C  \n",
            "ATOM   1472  O   LEU A 401      23.552   7.136  13.895  1.00 37.04           O  \n",
            "ATOM   1473  CB  LEU A 401      24.380   9.810  12.071  1.00 35.41           C  \n",
            "ATOM   1474  CG  LEU A 401      25.079   8.861  11.097  1.00 35.59           C  \n",
            "ATOM   1475  CD1 LEU A 401      24.120   8.454   9.997  1.00 32.66           C  \n",
            "ATOM   1476  CD2 LEU A 401      26.300   9.539  10.520  1.00 36.90           C  \n",
            "ATOM   1477  N   ALA A 402      23.988   9.019  15.030  1.00 36.57           N  \n",
            "ATOM   1478  CA  ALA A 402      24.398   8.315  16.238  1.00 36.23           C  \n",
            "ATOM   1479  C   ALA A 402      23.193   7.583  16.832  1.00 36.49           C  \n",
            "ATOM   1480  O   ALA A 402      23.327   6.496  17.399  1.00 36.28           O  \n",
            "ATOM   1481  CB  ALA A 402      24.956   9.306  17.248  1.00 33.46           C  \n",
            "ATOM   1482  N   PHE A 403      22.014   8.184  16.682  1.00 36.46           N  \n",
            "ATOM   1483  CA  PHE A 403      20.789   7.610  17.221  1.00 36.44           C  \n",
            "ATOM   1484  C   PHE A 403      20.488   6.285  16.537  1.00 36.97           C  \n",
            "ATOM   1485  O   PHE A 403      20.299   5.265  17.198  1.00 36.94           O  \n",
            "ATOM   1486  CB  PHE A 403      19.625   8.585  17.020  1.00 35.59           C  \n",
            "ATOM   1487  CG  PHE A 403      18.458   8.363  17.957  1.00 33.27           C  \n",
            "ATOM   1488  CD1 PHE A 403      18.402   7.254  18.792  1.00 32.93           C  \n",
            "ATOM   1489  CD2 PHE A 403      17.402   9.268  17.985  1.00 32.03           C  \n",
            "ATOM   1490  CE1 PHE A 403      17.304   7.051  19.640  1.00 33.66           C  \n",
            "ATOM   1491  CE2 PHE A 403      16.307   9.073  18.827  1.00 32.40           C  \n",
            "ATOM   1492  CZ  PHE A 403      16.257   7.962  19.655  1.00 31.35           C  \n",
            "ATOM   1493  N   GLU A 404      20.456   6.311  15.209  1.00 37.27           N  \n",
            "ATOM   1494  CA  GLU A 404      20.177   5.117  14.421  1.00 37.50           C  \n",
            "ATOM   1495  C   GLU A 404      21.198   3.998  14.693  1.00 36.71           C  \n",
            "ATOM   1496  O   GLU A 404      20.827   2.830  14.853  1.00 35.15           O  \n",
            "ATOM   1497  CB  GLU A 404      20.170   5.479  12.936  1.00 37.95           C  \n",
            "ATOM   1498  CG  GLU A 404      19.729   4.352  12.033  1.00 42.50           C  \n",
            "ATOM   1499  CD  GLU A 404      19.781   4.729  10.565  1.00 44.07           C  \n",
            "ATOM   1500  OE1 GLU A 404      20.230   5.856  10.258  1.00 45.11           O  \n",
            "ATOM   1501  OE2 GLU A 404      19.372   3.898   9.722  1.00 43.90           O  \n",
            "ATOM   1502  N   HIS A 405      22.479   4.357  14.745  1.00 35.82           N  \n",
            "ATOM   1503  CA  HIS A 405      23.533   3.383  15.017  1.00 35.89           C  \n",
            "ATOM   1504  C   HIS A 405      23.348   2.776  16.410  1.00 35.50           C  \n",
            "ATOM   1505  O   HIS A 405      23.540   1.577  16.604  1.00 35.75           O  \n",
            "ATOM   1506  CB  HIS A 405      24.907   4.047  14.898  1.00 36.19           C  \n",
            "ATOM   1507  CG  HIS A 405      25.265   4.424  13.495  1.00 37.75           C  \n",
            "ATOM   1508  ND1 HIS A 405      26.366   5.195  13.187  1.00 39.21           N  \n",
            "ATOM   1509  CD2 HIS A 405      24.662   4.145  12.318  1.00 37.96           C  \n",
            "ATOM   1510  CE1 HIS A 405      26.420   5.377  11.881  1.00 39.66           C  \n",
            "ATOM   1511  NE2 HIS A 405      25.398   4.751  11.328  1.00 39.05           N  \n",
            "ATOM   1512  N   TYR A 406      22.968   3.607  17.375  1.00 33.21           N  \n",
            "ATOM   1513  CA  TYR A 406      22.735   3.124  18.718  1.00 32.29           C  \n",
            "ATOM   1514  C   TYR A 406      21.543   2.160  18.681  1.00 34.41           C  \n",
            "ATOM   1515  O   TYR A 406      21.489   1.193  19.438  1.00 34.91           O  \n",
            "ATOM   1516  CB  TYR A 406      22.423   4.293  19.651  1.00 30.77           C  \n",
            "ATOM   1517  CG  TYR A 406      22.297   3.887  21.103  1.00 27.70           C  \n",
            "ATOM   1518  CD1 TYR A 406      23.430   3.566  21.860  1.00 25.89           C  \n",
            "ATOM   1519  CD2 TYR A 406      21.043   3.766  21.707  1.00 25.59           C  \n",
            "ATOM   1520  CE1 TYR A 406      23.312   3.137  23.180  1.00 22.85           C  \n",
            "ATOM   1521  CE2 TYR A 406      20.917   3.335  23.018  1.00 23.73           C  \n",
            "ATOM   1522  CZ  TYR A 406      22.053   3.021  23.749  1.00 23.99           C  \n",
            "ATOM   1523  OH  TYR A 406      21.922   2.590  25.047  1.00 23.70           O  \n",
            "ATOM   1524  N   ILE A 407      20.589   2.436  17.796  1.00 36.24           N  \n",
            "ATOM   1525  CA  ILE A 407      19.397   1.602  17.647  1.00 37.68           C  \n",
            "ATOM   1526  C   ILE A 407      19.793   0.215  17.166  1.00 38.95           C  \n",
            "ATOM   1527  O   ILE A 407      19.213  -0.785  17.595  1.00 39.31           O  \n",
            "ATOM   1528  CB  ILE A 407      18.383   2.225  16.634  1.00 37.85           C  \n",
            "ATOM   1529  CG1 ILE A 407      17.699   3.440  17.263  1.00 38.01           C  \n",
            "ATOM   1530  CG2 ILE A 407      17.344   1.197  16.210  1.00 36.37           C  \n",
            "ATOM   1531  CD1 ILE A 407      16.994   3.133  18.576  1.00 36.01           C  \n",
            "ATOM   1532  N   ASN A 408      20.778   0.159  16.272  1.00 39.42           N  \n",
            "ATOM   1533  CA  ASN A 408      21.251  -1.119  15.754  1.00 39.60           C  \n",
            "ATOM   1534  C   ASN A 408      21.973  -1.883  16.863  1.00 40.57           C  \n",
            "ATOM   1535  O   ASN A 408      21.855  -3.102  16.954  1.00 41.24           O  \n",
            "ATOM   1536  CB  ASN A 408      22.188  -0.905  14.560  1.00 38.60           C  \n",
            "ATOM   1537  CG  ASN A 408      21.527  -0.138  13.429  1.00 37.57           C  \n",
            "ATOM   1538  OD1 ASN A 408      20.317  -0.219  13.233  1.00 37.40           O  \n",
            "ATOM   1539  ND2 ASN A 408      22.324   0.596  12.667  1.00 38.01           N  \n",
            "ATOM   1540  N   TYR A 409      22.709  -1.156  17.703  1.00 41.36           N  \n",
            "ATOM   1541  CA  TYR A 409      23.444  -1.739  18.830  1.00 43.50           C  \n",
            "ATOM   1542  C   TYR A 409      22.494  -2.349  19.860  1.00 43.66           C  \n",
            "ATOM   1543  O   TYR A 409      22.768  -3.410  20.413  1.00 44.11           O  \n",
            "ATOM   1544  CB  TYR A 409      24.305  -0.662  19.503  1.00 45.26           C  \n",
            "ATOM   1545  CG  TYR A 409      24.729  -0.982  20.925  1.00 48.01           C  \n",
            "ATOM   1546  CD1 TYR A 409      25.736  -1.915  21.192  1.00 48.17           C  \n",
            "ATOM   1547  CD2 TYR A 409      24.120  -0.340  22.009  1.00 48.26           C  \n",
            "ATOM   1548  CE1 TYR A 409      26.127  -2.199  22.509  1.00 48.93           C  \n",
            "ATOM   1549  CE2 TYR A 409      24.500  -0.614  23.321  1.00 49.17           C  \n",
            "ATOM   1550  CZ  TYR A 409      25.502  -1.541  23.570  1.00 50.47           C  \n",
            "ATOM   1551  OH  TYR A 409      25.875  -1.791  24.880  1.00 50.37           O  \n",
            "ATOM   1552  N   ARG A 410      21.382  -1.665  20.110  1.00 44.60           N  \n",
            "ATOM   1553  CA  ARG A 410      20.372  -2.114  21.069  1.00 44.89           C  \n",
            "ATOM   1554  C   ARG A 410      19.644  -3.371  20.608  1.00 45.65           C  \n",
            "ATOM   1555  O   ARG A 410      19.272  -4.210  21.422  1.00 46.06           O  \n",
            "ATOM   1556  CB  ARG A 410      19.359  -0.991  21.319  1.00 42.18           C  \n",
            "ATOM   1557  CG  ARG A 410      19.922   0.156  22.152  1.00 42.51           C  \n",
            "ATOM   1558  CD  ARG A 410      19.618  -0.032  23.634  1.00 39.62           C  \n",
            "ATOM   1559  NE  ARG A 410      18.237   0.345  23.932  1.00 38.79           N  \n",
            "ATOM   1560  CZ  ARG A 410      17.591   0.047  25.055  1.00 36.97           C  \n",
            "ATOM   1561  NH1 ARG A 410      16.339   0.445  25.221  1.00 36.13           N  \n",
            "ATOM   1562  NH2 ARG A 410      18.181  -0.661  26.004  1.00 35.20           N  \n",
            "ATOM   1563  N   LYS A 411      19.447  -3.488  19.298  1.00 47.33           N  \n",
            "ATOM   1564  CA  LYS A 411      18.760  -4.629  18.703  1.00 48.43           C  \n",
            "ATOM   1565  C   LYS A 411      17.368  -4.892  19.284  1.00 49.12           C  \n",
            "ATOM   1566  O   LYS A 411      17.106  -5.961  19.835  1.00 48.85           O  \n",
            "ATOM   1567  CB  LYS A 411      19.616  -5.894  18.826  0.50 48.64           C  \n",
            "ATOM   1568  CG  LYS A 411      20.901  -5.856  18.012  0.50 49.40           C  \n",
            "ATOM   1569  CD  LYS A 411      21.612  -7.197  18.032  0.50 49.07           C  \n",
            "ATOM   1570  CE  LYS A 411      22.893  -7.159  17.213  0.50 49.43           C  \n",
            "ATOM   1571  NZ  LYS A 411      23.885  -6.198  17.770  0.50 49.07           N  \n",
            "ATOM   1572  N   HIS A 412      16.476  -3.916  19.149  1.00 50.13           N  \n",
            "ATOM   1573  CA  HIS A 412      15.104  -4.052  19.641  1.00 51.29           C  \n",
            "ATOM   1574  C   HIS A 412      14.364  -5.147  18.865  1.00 51.89           C  \n",
            "ATOM   1575  O   HIS A 412      14.631  -5.365  17.681  1.00 52.84           O  \n",
            "ATOM   1576  CB  HIS A 412      14.366  -2.720  19.482  1.00 50.06           C  \n",
            "ATOM   1577  CG  HIS A 412      14.797  -1.669  20.458  1.00 50.46           C  \n",
            "ATOM   1578  ND1 HIS A 412      14.311  -1.608  21.744  1.00 51.45           N  \n",
            "ATOM   1579  CD2 HIS A 412      15.672  -0.645  20.333  1.00 51.05           C  \n",
            "ATOM   1580  CE1 HIS A 412      14.866  -0.584  22.374  1.00 50.01           C  \n",
            "ATOM   1581  NE2 HIS A 412      15.693   0.015  21.541  1.00 49.96           N  \n",
            "ATOM   1582  N   HIS A 413      13.438  -5.836  19.524  1.00 52.23           N  \n",
            "ATOM   1583  CA  HIS A 413      12.685  -6.891  18.853  1.00 53.69           C  \n",
            "ATOM   1584  C   HIS A 413      11.398  -6.358  18.210  1.00 53.61           C  \n",
            "ATOM   1585  O   HIS A 413      10.291  -6.818  18.507  1.00 53.28           O  \n",
            "ATOM   1586  CB  HIS A 413      12.338  -8.018  19.833  1.00 55.94           C  \n",
            "ATOM   1587  CG  HIS A 413      11.773  -9.235  19.169  1.00 58.37           C  \n",
            "ATOM   1588  ND1 HIS A 413      12.558 -10.160  18.511  1.00 59.99           N  \n",
            "ATOM   1589  CD2 HIS A 413      10.494  -9.651  19.006  1.00 58.93           C  \n",
            "ATOM   1590  CE1 HIS A 413      11.790 -11.087  17.973  1.00 59.81           C  \n",
            "ATOM   1591  NE2 HIS A 413      10.530 -10.801  18.258  1.00 59.99           N  \n",
            "ATOM   1592  N   VAL A 414      11.544  -5.380  17.326  1.00 52.56           N  \n",
            "ATOM   1593  CA  VAL A 414      10.391  -4.807  16.645  1.00 52.51           C  \n",
            "ATOM   1594  C   VAL A 414      10.839  -4.306  15.282  1.00 51.68           C  \n",
            "ATOM   1595  O   VAL A 414      11.790  -3.530  15.178  1.00 51.81           O  \n",
            "ATOM   1596  CB  VAL A 414       9.754  -3.655  17.482  1.00 52.39           C  \n",
            "ATOM   1597  CG1 VAL A 414      10.820  -2.703  17.946  1.00 52.73           C  \n",
            "ATOM   1598  CG2 VAL A 414       8.712  -2.918  16.662  1.00 53.34           C  \n",
            "ATOM   1599  N   THR A 415      10.169  -4.774  14.235  1.00 51.18           N  \n",
            "ATOM   1600  CA  THR A 415      10.530  -4.372  12.884  1.00 51.52           C  \n",
            "ATOM   1601  C   THR A 415      10.254  -2.896  12.635  1.00 50.08           C  \n",
            "ATOM   1602  O   THR A 415       9.223  -2.358  13.051  1.00 47.82           O  \n",
            "ATOM   1603  CB  THR A 415       9.810  -5.236  11.816  1.00 52.42           C  \n",
            "ATOM   1604  OG1 THR A 415       8.412  -5.331  12.121  1.00 53.61           O  \n",
            "ATOM   1605  CG2 THR A 415      10.427  -6.632  11.770  1.00 52.53           C  \n",
            "ATOM   1606  N   HIS A 416      11.207  -2.252  11.964  1.00 49.10           N  \n",
            "ATOM   1607  CA  HIS A 416      11.127  -0.830  11.650  1.00 48.00           C  \n",
            "ATOM   1608  C   HIS A 416      10.989   0.005  12.930  1.00 46.57           C  \n",
            "ATOM   1609  O   HIS A 416      10.095   0.850  13.046  1.00 46.12           O  \n",
            "ATOM   1610  CB  HIS A 416       9.947  -0.556  10.707  1.00 48.97           C  \n",
            "ATOM   1611  CG  HIS A 416       9.983  -1.361   9.443  1.00 48.91           C  \n",
            "ATOM   1612  ND1 HIS A 416      11.135  -1.543   8.707  1.00 47.72           N  \n",
            "ATOM   1613  CD2 HIS A 416       9.006  -2.023   8.780  1.00 49.30           C  \n",
            "ATOM   1614  CE1 HIS A 416      10.868  -2.284   7.648  1.00 46.77           C  \n",
            "ATOM   1615  NE2 HIS A 416       9.583  -2.588   7.668  1.00 49.93           N  \n",
            "ATOM   1616  N   PHE A 417      11.879  -0.243  13.888  1.00 43.15           N  \n",
            "ATOM   1617  CA  PHE A 417      11.858   0.487  15.145  1.00 39.88           C  \n",
            "ATOM   1618  C   PHE A 417      12.245   1.937  14.905  1.00 38.55           C  \n",
            "ATOM   1619  O   PHE A 417      11.606   2.845  15.424  1.00 38.00           O  \n",
            "ATOM   1620  CB  PHE A 417      12.831  -0.134  16.142  1.00 39.91           C  \n",
            "ATOM   1621  CG  PHE A 417      12.802   0.509  17.499  1.00 38.15           C  \n",
            "ATOM   1622  CD1 PHE A 417      11.682   0.395  18.312  1.00 38.09           C  \n",
            "ATOM   1623  CD2 PHE A 417      13.881   1.258  17.952  1.00 37.93           C  \n",
            "ATOM   1624  CE1 PHE A 417      11.642   1.003  19.559  1.00 36.61           C  \n",
            "ATOM   1625  CE2 PHE A 417      13.849   1.871  19.202  1.00 35.97           C  \n",
            "ATOM   1626  CZ  PHE A 417      12.727   1.749  20.001  1.00 35.96           C  \n",
            "ATOM   1627  N   TRP A 418      13.297   2.147  14.118  1.00 36.42           N  \n",
            "ATOM   1628  CA  TRP A 418      13.770   3.493  13.815  1.00 35.85           C  \n",
            "ATOM   1629  C   TRP A 418      12.662   4.329  13.168  1.00 37.47           C  \n",
            "ATOM   1630  O   TRP A 418      12.374   5.444  13.611  1.00 37.02           O  \n",
            "ATOM   1631  CB  TRP A 418      14.989   3.421  12.894  1.00 32.59           C  \n",
            "ATOM   1632  CG  TRP A 418      15.556   4.747  12.527  1.00 31.72           C  \n",
            "ATOM   1633  CD1 TRP A 418      15.795   5.213  11.268  1.00 31.16           C  \n",
            "ATOM   1634  CD2 TRP A 418      15.974   5.781  13.424  1.00 32.80           C  \n",
            "ATOM   1635  NE1 TRP A 418      16.337   6.472  11.320  1.00 32.11           N  \n",
            "ATOM   1636  CE2 TRP A 418      16.457   6.848  12.633  1.00 33.19           C  \n",
            "ATOM   1637  CE3 TRP A 418      15.990   5.910  14.819  1.00 33.38           C  \n",
            "ATOM   1638  CZ2 TRP A 418      16.951   8.034  13.193  1.00 31.23           C  \n",
            "ATOM   1639  CZ3 TRP A 418      16.482   7.090  15.375  1.00 33.00           C  \n",
            "ATOM   1640  CH2 TRP A 418      16.956   8.134  14.559  1.00 32.25           C  \n",
            "ATOM   1641  N   PRO A 419      12.036   3.805  12.099  1.00 38.16           N  \n",
            "ATOM   1642  CA  PRO A 419      10.961   4.524  11.414  1.00 38.40           C  \n",
            "ATOM   1643  C   PRO A 419       9.776   4.797  12.334  1.00 38.30           C  \n",
            "ATOM   1644  O   PRO A 419       9.080   5.795  12.176  1.00 38.85           O  \n",
            "ATOM   1645  CB  PRO A 419      10.603   3.584  10.265  1.00 36.63           C  \n",
            "ATOM   1646  CG  PRO A 419      11.919   2.976   9.937  1.00 36.82           C  \n",
            "ATOM   1647  CD  PRO A 419      12.442   2.630  11.307  1.00 37.94           C  \n",
            "ATOM   1648  N   LYS A 420       9.545   3.905  13.290  1.00 38.98           N  \n",
            "ATOM   1649  CA  LYS A 420       8.443   4.082  14.226  1.00 39.17           C  \n",
            "ATOM   1650  C   LYS A 420       8.804   5.141  15.254  1.00 38.64           C  \n",
            "ATOM   1651  O   LYS A 420       7.941   5.865  15.729  1.00 41.19           O  \n",
            "ATOM   1652  CB  LYS A 420       8.104   2.761  14.911  1.00 39.43           C  \n",
            "ATOM   1653  CG  LYS A 420       7.469   1.760  13.966  1.00 41.66           C  \n",
            "ATOM   1654  CD  LYS A 420       7.153   0.458  14.660  1.00 43.56           C  \n",
            "ATOM   1655  CE  LYS A 420       6.776  -0.621  13.660  1.00 44.99           C  \n",
            "ATOM   1656  NZ  LYS A 420       6.725  -1.958  14.327  1.00 46.33           N  \n",
            "ATOM   1657  N   LEU A 421      10.085   5.235  15.586  1.00 37.97           N  \n",
            "ATOM   1658  CA  LEU A 421      10.557   6.237  16.533  1.00 36.49           C  \n",
            "ATOM   1659  C   LEU A 421      10.409   7.627  15.918  1.00 36.19           C  \n",
            "ATOM   1660  O   LEU A 421       9.927   8.556  16.570  1.00 37.06           O  \n",
            "ATOM   1661  CB  LEU A 421      12.027   6.000  16.884  1.00 36.04           C  \n",
            "ATOM   1662  CG  LEU A 421      12.355   5.353  18.230  1.00 36.23           C  \n",
            "ATOM   1663  CD1 LEU A 421      13.861   5.203  18.352  1.00 35.58           C  \n",
            "ATOM   1664  CD2 LEU A 421      11.812   6.207  19.369  1.00 35.49           C  \n",
            "ATOM   1665  N   LEU A 422      10.820   7.762  14.658  1.00 34.10           N  \n",
            "ATOM   1666  CA  LEU A 422      10.734   9.041  13.968  1.00 31.94           C  \n",
            "ATOM   1667  C   LEU A 422       9.294   9.566  13.923  1.00 31.93           C  \n",
            "ATOM   1668  O   LEU A 422       9.066  10.779  13.923  1.00 31.97           O  \n",
            "ATOM   1669  CB  LEU A 422      11.310   8.918  12.557  1.00 30.61           C  \n",
            "ATOM   1670  CG  LEU A 422      12.800   8.549  12.447  1.00 31.07           C  \n",
            "ATOM   1671  CD1 LEU A 422      13.249   8.541  10.978  1.00 25.14           C  \n",
            "ATOM   1672  CD2 LEU A 422      13.626   9.541  13.252  1.00 28.70           C  \n",
            "ATOM   1673  N   MET A 423       8.320   8.662  13.903  1.00 29.58           N  \n",
            "ATOM   1674  CA  MET A 423       6.931   9.101  13.887  1.00 28.19           C  \n",
            "ATOM   1675  C   MET A 423       6.582   9.782  15.209  1.00 27.50           C  \n",
            "ATOM   1676  O   MET A 423       5.758  10.701  15.242  1.00 24.04           O  \n",
            "ATOM   1677  CB  MET A 423       5.983   7.924  13.648  1.00 27.43           C  \n",
            "ATOM   1678  CG  MET A 423       6.011   7.374  12.241  1.00 28.89           C  \n",
            "ATOM   1679  SD  MET A 423       4.647   6.227  11.944  1.00 30.93           S  \n",
            "ATOM   1680  CE  MET A 423       5.290   4.731  12.721  1.00 32.91           C  \n",
            "ATOM   1681  N   LYS A 424       7.212   9.321  16.292  1.00 27.23           N  \n",
            "ATOM   1682  CA  LYS A 424       6.985   9.886  17.621  1.00 27.60           C  \n",
            "ATOM   1683  C   LYS A 424       7.597  11.274  17.690  1.00 26.68           C  \n",
            "ATOM   1684  O   LYS A 424       7.101  12.143  18.413  1.00 27.19           O  \n",
            "ATOM   1685  CB  LYS A 424       7.604   8.999  18.709  1.00 27.31           C  \n",
            "ATOM   1686  CG  LYS A 424       7.073   7.584  18.698  1.00 30.78           C  \n",
            "ATOM   1687  CD  LYS A 424       5.571   7.583  18.833  1.00 32.96           C  \n",
            "ATOM   1688  CE  LYS A 424       4.969   6.290  18.332  1.00 34.76           C  \n",
            "ATOM   1689  NZ  LYS A 424       3.477   6.379  18.290  1.00 35.06           N  \n",
            "ATOM   1690  N   VAL A 425       8.681  11.472  16.942  1.00 24.69           N  \n",
            "ATOM   1691  CA  VAL A 425       9.349  12.764  16.908  1.00 24.85           C  \n",
            "ATOM   1692  C   VAL A 425       8.399  13.752  16.236  1.00 24.58           C  \n",
            "ATOM   1693  O   VAL A 425       8.313  14.909  16.647  1.00 23.16           O  \n",
            "ATOM   1694  CB  VAL A 425      10.694  12.717  16.116  1.00 25.58           C  \n",
            "ATOM   1695  CG1 VAL A 425      11.308  14.116  16.062  1.00 23.41           C  \n",
            "ATOM   1696  CG2 VAL A 425      11.669  11.732  16.778  1.00 21.94           C  \n",
            "ATOM   1697  N   THR A 426       7.678  13.283  15.216  1.00 24.03           N  \n",
            "ATOM   1698  CA  THR A 426       6.704  14.117  14.512  1.00 23.62           C  \n",
            "ATOM   1699  C   THR A 426       5.524  14.449  15.436  1.00 26.11           C  \n",
            "ATOM   1700  O   THR A 426       4.929  15.523  15.332  1.00 27.78           O  \n",
            "ATOM   1701  CB  THR A 426       6.170  13.414  13.228  1.00 24.36           C  \n",
            "ATOM   1702  OG1 THR A 426       7.196  13.383  12.230  1.00 22.32           O  \n",
            "ATOM   1703  CG2 THR A 426       4.962  14.156  12.653  1.00 25.55           C  \n",
            "ATOM   1704  N   ASP A 427       5.190  13.530  16.339  1.00 26.08           N  \n",
            "ATOM   1705  CA  ASP A 427       4.094  13.739  17.273  1.00 26.18           C  \n",
            "ATOM   1706  C   ASP A 427       4.418  14.881  18.223  1.00 27.86           C  \n",
            "ATOM   1707  O   ASP A 427       3.566  15.735  18.498  1.00 29.36           O  \n",
            "ATOM   1708  CB  ASP A 427       3.824  12.473  18.096  1.00 28.05           C  \n",
            "ATOM   1709  CG  ASP A 427       3.178  11.370  17.283  1.00 28.33           C  \n",
            "ATOM   1710  OD1 ASP A 427       2.527  11.696  16.271  1.00 29.62           O  \n",
            "ATOM   1711  OD2 ASP A 427       3.302  10.183  17.661  1.00 27.58           O  \n",
            "ATOM   1712  N   LEU A 428       5.651  14.888  18.726  1.00 27.07           N  \n",
            "ATOM   1713  CA  LEU A 428       6.108  15.911  19.660  1.00 25.53           C  \n",
            "ATOM   1714  C   LEU A 428       6.163  17.296  19.027  1.00 26.34           C  \n",
            "ATOM   1715  O   LEU A 428       5.900  18.299  19.692  1.00 27.27           O  \n",
            "ATOM   1716  CB  LEU A 428       7.474  15.517  20.218  1.00 23.78           C  \n",
            "ATOM   1717  CG  LEU A 428       7.392  14.280  21.118  1.00 23.89           C  \n",
            "ATOM   1718  CD1 LEU A 428       8.749  13.591  21.242  1.00 19.96           C  \n",
            "ATOM   1719  CD2 LEU A 428       6.848  14.708  22.478  1.00 22.56           C  \n",
            "ATOM   1720  N   ARG A 429       6.501  17.358  17.744  1.00 26.51           N  \n",
            "ATOM   1721  CA  ARG A 429       6.548  18.638  17.051  1.00 27.55           C  \n",
            "ATOM   1722  C   ARG A 429       5.121  19.158  16.874  1.00 26.50           C  \n",
            "ATOM   1723  O   ARG A 429       4.887  20.361  16.902  1.00 25.85           O  \n",
            "ATOM   1724  CB  ARG A 429       7.232  18.496  15.689  1.00 30.49           C  \n",
            "ATOM   1725  CG  ARG A 429       8.728  18.266  15.774  1.00 33.80           C  \n",
            "ATOM   1726  CD  ARG A 429       9.316  18.018  14.398  1.00 39.72           C  \n",
            "ATOM   1727  NE  ARG A 429      10.726  17.618  14.441  1.00 45.66           N  \n",
            "ATOM   1728  CZ  ARG A 429      11.704  18.349  14.976  1.00 47.14           C  \n",
            "ATOM   1729  NH1 ARG A 429      11.430  19.531  15.522  1.00 48.84           N  \n",
            "ATOM   1730  NH2 ARG A 429      12.955  17.900  14.962  1.00 45.14           N  \n",
            "ATOM   1731  N   MET A 430       4.171  18.243  16.699  1.00 26.02           N  \n",
            "ATOM   1732  CA  MET A 430       2.770  18.619  16.553  1.00 24.92           C  \n",
            "ATOM   1733  C   MET A 430       2.273  19.140  17.892  1.00 24.45           C  \n",
            "ATOM   1734  O   MET A 430       1.585  20.157  17.948  1.00 23.52           O  \n",
            "ATOM   1735  CB  MET A 430       1.931  17.417  16.109  1.00 25.43           C  \n",
            "ATOM   1736  CG  MET A 430       2.322  16.871  14.741  1.00 26.43           C  \n",
            "ATOM   1737  SD  MET A 430       2.436  18.189  13.488  1.00 25.43           S  \n",
            "ATOM   1738  CE  MET A 430       0.802  18.347  13.074  1.00 27.99           C  \n",
            "ATOM   1739  N   ILE A 431       2.635  18.437  18.965  1.00 23.06           N  \n",
            "ATOM   1740  CA  ILE A 431       2.260  18.829  20.321  1.00 23.18           C  \n",
            "ATOM   1741  C   ILE A 431       2.845  20.203  20.688  1.00 23.64           C  \n",
            "ATOM   1742  O   ILE A 431       2.115  21.104  21.091  1.00 23.63           O  \n",
            "ATOM   1743  CB  ILE A 431       2.735  17.780  21.371  1.00 23.92           C  \n",
            "ATOM   1744  CG1 ILE A 431       1.954  16.476  21.211  1.00 22.99           C  \n",
            "ATOM   1745  CG2 ILE A 431       2.549  18.324  22.783  1.00 24.57           C  \n",
            "ATOM   1746  CD1 ILE A 431       2.336  15.410  22.217  1.00 21.79           C  \n",
            "ATOM   1747  N   GLY A 432       4.157  20.364  20.554  1.00 22.78           N  \n",
            "ATOM   1748  CA  GLY A 432       4.764  21.640  20.876  1.00 23.52           C  \n",
            "ATOM   1749  C   GLY A 432       4.081  22.806  20.179  1.00 24.65           C  \n",
            "ATOM   1750  O   GLY A 432       3.863  23.854  20.783  1.00 27.21           O  \n",
            "ATOM   1751  N   ALA A 433       3.736  22.638  18.907  1.00 23.81           N  \n",
            "ATOM   1752  CA  ALA A 433       3.077  23.704  18.155  1.00 24.51           C  \n",
            "ATOM   1753  C   ALA A 433       1.652  23.944  18.655  1.00 25.54           C  \n",
            "ATOM   1754  O   ALA A 433       1.158  25.074  18.670  1.00 25.28           O  \n",
            "ATOM   1755  CB  ALA A 433       3.046  23.359  16.679  1.00 21.85           C  \n",
            "ATOM   1756  N   CYS A 434       0.985  22.872  19.053  1.00 26.15           N  \n",
            "ATOM   1757  CA  CYS A 434      -0.375  22.995  19.529  1.00 26.57           C  \n",
            "ATOM   1758  C   CYS A 434      -0.375  23.725  20.858  1.00 26.89           C  \n",
            "ATOM   1759  O   CYS A 434      -1.318  24.439  21.176  1.00 25.99           O  \n",
            "ATOM   1760  CB  CYS A 434      -1.012  21.617  19.679  1.00 26.57           C  \n",
            "ATOM   1761  SG  CYS A 434      -2.785  21.714  19.962  1.00 30.76           S  \n",
            "ATOM   1762  N   HIS A 435       0.700  23.544  21.622  1.00 27.80           N  \n",
            "ATOM   1763  CA  HIS A 435       0.863  24.182  22.926  1.00 25.65           C  \n",
            "ATOM   1764  C   HIS A 435       0.995  25.690  22.779  1.00 25.90           C  \n",
            "ATOM   1765  O   HIS A 435       0.559  26.450  23.634  1.00 25.25           O  \n",
            "ATOM   1766  CB  HIS A 435       2.094  23.618  23.632  1.00 23.95           C  \n",
            "ATOM   1767  CG  HIS A 435       2.467  24.363  24.877  1.00 25.00           C  \n",
            "ATOM   1768  ND1 HIS A 435       3.338  25.428  24.873  1.00 23.36           N  \n",
            "ATOM   1769  CD2 HIS A 435       2.026  24.237  26.153  1.00 23.16           C  \n",
            "ATOM   1770  CE1 HIS A 435       3.415  25.934  26.095  1.00 24.31           C  \n",
            "ATOM   1771  NE2 HIS A 435       2.629  25.229  26.885  1.00 21.54           N  \n",
            "ATOM   1772  N   ALA A 436       1.602  26.111  21.679  1.00 28.75           N  \n",
            "ATOM   1773  CA  ALA A 436       1.792  27.524  21.393  1.00 31.83           C  \n",
            "ATOM   1774  C   ALA A 436       0.449  28.244  21.314  1.00 34.36           C  \n",
            "ATOM   1775  O   ALA A 436       0.255  29.289  21.937  1.00 35.12           O  \n",
            "ATOM   1776  CB  ALA A 436       2.543  27.678  20.088  1.00 30.86           C  \n",
            "ATOM   1777  N   SER A 437      -0.482  27.681  20.551  1.00 37.08           N  \n",
            "ATOM   1778  CA  SER A 437      -1.795  28.288  20.416  1.00 39.53           C  \n",
            "ATOM   1779  C   SER A 437      -2.542  28.184  21.739  1.00 40.01           C  \n",
            "ATOM   1780  O   SER A 437      -3.392  29.015  22.049  1.00 42.88           O  \n",
            "ATOM   1781  CB  SER A 437      -2.596  27.589  19.315  1.00 40.57           C  \n",
            "ATOM   1782  OG  SER A 437      -2.994  26.290  19.713  1.00 43.47           O  \n",
            "ATOM   1783  N   ARG A 438      -2.207  27.163  22.518  1.00 39.30           N  \n",
            "ATOM   1784  CA  ARG A 438      -2.844  26.925  23.805  1.00 39.22           C  \n",
            "ATOM   1785  C   ARG A 438      -2.407  27.942  24.861  1.00 39.43           C  \n",
            "ATOM   1786  O   ARG A 438      -3.195  28.338  25.725  1.00 38.72           O  \n",
            "ATOM   1787  CB  ARG A 438      -2.525  25.508  24.285  1.00 39.33           C  \n",
            "ATOM   1788  CG  ARG A 438      -3.365  25.070  25.457  1.00 39.82           C  \n",
            "ATOM   1789  CD  ARG A 438      -4.828  25.078  25.085  1.00 39.33           C  \n",
            "ATOM   1790  NE  ARG A 438      -5.681  25.018  26.262  1.00 40.43           N  \n",
            "ATOM   1791  CZ  ARG A 438      -6.997  25.180  26.229  1.00 41.80           C  \n",
            "ATOM   1792  NH1 ARG A 438      -7.603  25.410  25.068  1.00 43.64           N  \n",
            "ATOM   1793  NH2 ARG A 438      -7.705  25.120  27.352  1.00 39.39           N  \n",
            "ATOM   1794  N   PHE A 439      -1.146  28.353  24.790  1.00 39.53           N  \n",
            "ATOM   1795  CA  PHE A 439      -0.601  29.337  25.718  1.00 38.89           C  \n",
            "ATOM   1796  C   PHE A 439      -1.411  30.621  25.604  1.00 38.88           C  \n",
            "ATOM   1797  O   PHE A 439      -1.826  31.196  26.605  1.00 37.96           O  \n",
            "ATOM   1798  CB  PHE A 439       0.863  29.630  25.371  1.00 38.18           C  \n",
            "ATOM   1799  CG  PHE A 439       1.509  30.643  26.271  1.00 38.95           C  \n",
            "ATOM   1800  CD1 PHE A 439       1.778  30.342  27.601  1.00 40.10           C  \n",
            "ATOM   1801  CD2 PHE A 439       1.853  31.899  25.792  1.00 38.92           C  \n",
            "ATOM   1802  CE1 PHE A 439       2.384  31.276  28.439  1.00 40.18           C  \n",
            "ATOM   1803  CE2 PHE A 439       2.458  32.841  26.623  1.00 39.01           C  \n",
            "ATOM   1804  CZ  PHE A 439       2.722  32.527  27.947  1.00 39.79           C  \n",
            "ATOM   1805  N   LEU A 440      -1.627  31.061  24.366  1.00 39.62           N  \n",
            "ATOM   1806  CA  LEU A 440      -2.375  32.281  24.087  1.00 39.49           C  \n",
            "ATOM   1807  C   LEU A 440      -3.740  32.277  24.770  1.00 39.77           C  \n",
            "ATOM   1808  O   LEU A 440      -4.183  33.293  25.308  1.00 39.01           O  \n",
            "ATOM   1809  CB  LEU A 440      -2.537  32.459  22.569  1.00 38.13           C  \n",
            "ATOM   1810  CG  LEU A 440      -1.210  32.628  21.805  1.00 39.09           C  \n",
            "ATOM   1811  CD1 LEU A 440      -1.441  32.645  20.302  1.00 34.80           C  \n",
            "ATOM   1812  CD2 LEU A 440      -0.523  33.910  22.260  1.00 38.69           C  \n",
            "ATOM   1813  N   HIS A 441      -4.395  31.123  24.755  1.00 41.33           N  \n",
            "ATOM   1814  CA  HIS A 441      -5.710  30.981  25.362  1.00 42.68           C  \n",
            "ATOM   1815  C   HIS A 441      -5.647  31.085  26.881  1.00 42.75           C  \n",
            "ATOM   1816  O   HIS A 441      -6.532  31.674  27.511  1.00 43.69           O  \n",
            "ATOM   1817  CB  HIS A 441      -6.321  29.641  24.965  1.00 46.08           C  \n",
            "ATOM   1818  CG  HIS A 441      -7.750  29.492  25.374  1.00 50.43           C  \n",
            "ATOM   1819  ND1 HIS A 441      -8.665  30.516  25.252  1.00 52.83           N  \n",
            "ATOM   1820  CD2 HIS A 441      -8.437  28.429  25.853  1.00 51.39           C  \n",
            "ATOM   1821  CE1 HIS A 441      -9.853  30.090  25.637  1.00 53.14           C  \n",
            "ATOM   1822  NE2 HIS A 441      -9.743  28.826  26.007  1.00 53.10           N  \n",
            "ATOM   1823  N   MET A 442      -4.605  30.494  27.463  1.00 41.02           N  \n",
            "ATOM   1824  CA  MET A 442      -4.406  30.531  28.906  1.00 38.95           C  \n",
            "ATOM   1825  C   MET A 442      -4.300  31.995  29.323  1.00 39.12           C  \n",
            "ATOM   1826  O   MET A 442      -4.810  32.395  30.366  1.00 38.99           O  \n",
            "ATOM   1827  CB  MET A 442      -3.118  29.797  29.289  1.00 38.00           C  \n",
            "ATOM   1828  CG  MET A 442      -3.126  28.280  29.105  1.00 35.09           C  \n",
            "ATOM   1829  SD  MET A 442      -1.521  27.564  29.594  1.00 34.37           S  \n",
            "ATOM   1830  CE  MET A 442      -1.321  26.238  28.385  1.00 32.30           C  \n",
            "ATOM   1831  N   LYS A 443      -3.630  32.788  28.494  1.00 39.93           N  \n",
            "ATOM   1832  CA  LYS A 443      -3.454  34.212  28.754  1.00 41.43           C  \n",
            "ATOM   1833  C   LYS A 443      -4.789  34.925  28.839  1.00 42.03           C  \n",
            "ATOM   1834  O   LYS A 443      -4.909  35.957  29.498  1.00 43.63           O  \n",
            "ATOM   1835  CB  LYS A 443      -2.634  34.859  27.646  1.00 42.23           C  \n",
            "ATOM   1836  CG  LYS A 443      -1.143  34.925  27.906  1.00 44.62           C  \n",
            "ATOM   1837  CD  LYS A 443      -0.495  35.653  26.744  1.00 46.93           C  \n",
            "ATOM   1838  CE  LYS A 443       0.951  35.985  26.997  1.00 47.30           C  \n",
            "ATOM   1839  NZ  LYS A 443       1.463  36.785  25.855  1.00 47.53           N  \n",
            "ATOM   1840  N   VAL A 444      -5.785  34.373  28.153  1.00 41.64           N  \n",
            "ATOM   1841  CA  VAL A 444      -7.124  34.940  28.131  1.00 40.01           C  \n",
            "ATOM   1842  C   VAL A 444      -7.974  34.435  29.292  1.00 41.14           C  \n",
            "ATOM   1843  O   VAL A 444      -8.665  35.206  29.950  1.00 41.59           O  \n",
            "ATOM   1844  CB  VAL A 444      -7.842  34.590  26.810  1.00 39.67           C  \n",
            "ATOM   1845  CG1 VAL A 444      -9.319  34.984  26.892  1.00 37.19           C  \n",
            "ATOM   1846  CG2 VAL A 444      -7.152  35.290  25.640  1.00 35.95           C  \n",
            "ATOM   1847  N   GLU A 445      -7.912  33.135  29.547  1.00 42.14           N  \n",
            "ATOM   1848  CA  GLU A 445      -8.697  32.529  30.614  1.00 41.98           C  \n",
            "ATOM   1849  C   GLU A 445      -8.125  32.667  32.026  1.00 41.20           C  \n",
            "ATOM   1850  O   GLU A 445      -8.866  32.551  33.003  1.00 39.74           O  \n",
            "ATOM   1851  CB  GLU A 445      -8.906  31.049  30.299  1.00 43.28           C  \n",
            "ATOM   1852  CG  GLU A 445      -9.752  30.800  29.076  1.00 45.77           C  \n",
            "ATOM   1853  CD  GLU A 445     -11.206  31.136  29.316  1.00 47.56           C  \n",
            "ATOM   1854  OE1 GLU A 445     -11.495  32.278  29.725  1.00 50.47           O  \n",
            "ATOM   1855  OE2 GLU A 445     -12.066  30.260  29.097  1.00 49.16           O  \n",
            "ATOM   1856  N   CYS A 446      -6.822  32.921  32.135  1.00 41.29           N  \n",
            "ATOM   1857  CA  CYS A 446      -6.173  33.024  33.444  1.00 42.11           C  \n",
            "ATOM   1858  C   CYS A 446      -5.544  34.381  33.764  1.00 43.02           C  \n",
            "ATOM   1859  O   CYS A 446      -5.044  35.079  32.876  1.00 43.77           O  \n",
            "ATOM   1860  CB  CYS A 446      -5.089  31.944  33.583  1.00 41.13           C  \n",
            "ATOM   1861  SG  CYS A 446      -5.615  30.256  33.224  1.00 35.25           S  \n",
            "ATOM   1862  N   PRO A 447      -5.544  34.757  35.056  1.00 43.51           N  \n",
            "ATOM   1863  CA  PRO A 447      -4.978  36.025  35.522  1.00 43.99           C  \n",
            "ATOM   1864  C   PRO A 447      -3.476  36.089  35.245  1.00 45.03           C  \n",
            "ATOM   1865  O   PRO A 447      -2.755  35.109  35.440  1.00 44.89           O  \n",
            "ATOM   1866  CB  PRO A 447      -5.299  36.010  37.010  1.00 43.67           C  \n",
            "ATOM   1867  CG  PRO A 447      -5.221  34.553  37.343  1.00 43.89           C  \n",
            "ATOM   1868  CD  PRO A 447      -5.975  33.928  36.197  1.00 43.15           C  \n",
            "ATOM   1869  N   THR A 448      -3.010  37.248  34.795  1.00 46.82           N  \n",
            "ATOM   1870  CA  THR A 448      -1.599  37.427  34.462  1.00 48.41           C  \n",
            "ATOM   1871  C   THR A 448      -0.653  37.250  35.637  1.00 48.59           C  \n",
            "ATOM   1872  O   THR A 448       0.529  36.984  35.442  1.00 48.50           O  \n",
            "ATOM   1873  CB  THR A 448      -1.352  38.808  33.821  1.00 48.81           C  \n",
            "ATOM   1874  OG1 THR A 448      -2.164  38.937  32.645  1.00 49.50           O  \n",
            "ATOM   1875  CG2 THR A 448       0.113  38.954  33.417  1.00 50.13           C  \n",
            "ATOM   1876  N   GLU A 449      -1.171  37.388  36.854  1.00 49.50           N  \n",
            "ATOM   1877  CA  GLU A 449      -0.351  37.237  38.052  1.00 50.35           C  \n",
            "ATOM   1878  C   GLU A 449       0.084  35.789  38.265  1.00 49.31           C  \n",
            "ATOM   1879  O   GLU A 449       1.126  35.523  38.868  1.00 49.69           O  \n",
            "ATOM   1880  CB  GLU A 449      -1.115  37.714  39.293  1.00 52.87           C  \n",
            "ATOM   1881  CG  GLU A 449      -1.636  39.134  39.197  1.00 58.30           C  \n",
            "ATOM   1882  CD  GLU A 449      -3.044  39.207  38.632  1.00 60.82           C  \n",
            "ATOM   1883  OE1 GLU A 449      -3.980  38.757  39.324  1.00 62.89           O  \n",
            "ATOM   1884  OE2 GLU A 449      -3.219  39.710  37.501  1.00 61.94           O  \n",
            "ATOM   1885  N   LEU A 450      -0.714  34.849  37.774  1.00 47.53           N  \n",
            "ATOM   1886  CA  LEU A 450      -0.386  33.443  37.939  1.00 45.06           C  \n",
            "ATOM   1887  C   LEU A 450       0.682  32.921  36.984  1.00 42.33           C  \n",
            "ATOM   1888  O   LEU A 450       1.100  31.771  37.108  1.00 43.63           O  \n",
            "ATOM   1889  CB  LEU A 450      -1.650  32.588  37.814  1.00 45.93           C  \n",
            "ATOM   1890  CG  LEU A 450      -2.588  32.569  39.022  1.00 45.40           C  \n",
            "ATOM   1891  CD1 LEU A 450      -3.737  31.605  38.769  1.00 46.86           C  \n",
            "ATOM   1892  CD2 LEU A 450      -1.813  32.129  40.251  1.00 46.56           C  \n",
            "ATOM   1893  N   PHE A 451       1.128  33.754  36.045  1.00 38.53           N  \n",
            "ATOM   1894  CA  PHE A 451       2.158  33.343  35.082  1.00 36.72           C  \n",
            "ATOM   1895  C   PHE A 451       3.561  33.739  35.514  1.00 34.18           C  \n",
            "ATOM   1896  O   PHE A 451       3.855  34.924  35.646  1.00 36.20           O  \n",
            "ATOM   1897  CB  PHE A 451       1.913  33.958  33.694  1.00 35.87           C  \n",
            "ATOM   1898  CG  PHE A 451       0.670  33.469  33.013  1.00 35.57           C  \n",
            "ATOM   1899  CD1 PHE A 451      -0.590  33.800  33.506  1.00 37.39           C  \n",
            "ATOM   1900  CD2 PHE A 451       0.758  32.683  31.872  1.00 35.18           C  \n",
            "ATOM   1901  CE1 PHE A 451      -1.746  33.346  32.868  1.00 36.94           C  \n",
            "ATOM   1902  CE2 PHE A 451      -0.386  32.226  31.229  1.00 34.56           C  \n",
            "ATOM   1903  CZ  PHE A 451      -1.640  32.560  31.726  1.00 34.59           C  \n",
            "ATOM   1904  N   PRO A 452       4.446  32.750  35.739  1.00 31.81           N  \n",
            "ATOM   1905  CA  PRO A 452       5.835  33.004  36.151  1.00 29.95           C  \n",
            "ATOM   1906  C   PRO A 452       6.632  33.669  35.018  1.00 28.77           C  \n",
            "ATOM   1907  O   PRO A 452       6.485  33.306  33.850  1.00 28.54           O  \n",
            "ATOM   1908  CB  PRO A 452       6.366  31.601  36.459  1.00 29.32           C  \n",
            "ATOM   1909  CG  PRO A 452       5.152  30.851  36.875  1.00 29.08           C  \n",
            "ATOM   1910  CD  PRO A 452       4.121  31.318  35.877  1.00 30.96           C  \n",
            "ATOM   1911  N   PRO A 453       7.497  34.640  35.347  1.00 27.59           N  \n",
            "ATOM   1912  CA  PRO A 453       8.277  35.300  34.294  1.00 26.99           C  \n",
            "ATOM   1913  C   PRO A 453       9.008  34.348  33.341  1.00 27.74           C  \n",
            "ATOM   1914  O   PRO A 453       9.176  34.659  32.159  1.00 27.56           O  \n",
            "ATOM   1915  CB  PRO A 453       9.239  36.195  35.083  1.00 24.10           C  \n",
            "ATOM   1916  CG  PRO A 453       9.352  35.509  36.399  1.00 24.18           C  \n",
            "ATOM   1917  CD  PRO A 453       7.928  35.102  36.676  1.00 25.54           C  \n",
            "ATOM   1918  N   LEU A 454       9.439  33.195  33.845  1.00 26.52           N  \n",
            "ATOM   1919  CA  LEU A 454      10.144  32.242  33.001  1.00 26.65           C  \n",
            "ATOM   1920  C   LEU A 454       9.190  31.634  31.976  1.00 26.69           C  \n",
            "ATOM   1921  O   LEU A 454       9.557  31.422  30.820  1.00 26.62           O  \n",
            "ATOM   1922  CB  LEU A 454      10.777  31.132  33.853  1.00 24.84           C  \n",
            "ATOM   1923  CG  LEU A 454      11.569  30.089  33.054  1.00 25.23           C  \n",
            "ATOM   1924  CD1 LEU A 454      12.543  30.775  32.089  1.00 21.36           C  \n",
            "ATOM   1925  CD2 LEU A 454      12.303  29.178  34.009  1.00 23.13           C  \n",
            "ATOM   1926  N   PHE A 455       7.964  31.366  32.418  1.00 26.69           N  \n",
            "ATOM   1927  CA  PHE A 455       6.921  30.785  31.582  1.00 25.95           C  \n",
            "ATOM   1928  C   PHE A 455       6.636  31.736  30.403  1.00 27.53           C  \n",
            "ATOM   1929  O   PHE A 455       6.641  31.333  29.238  1.00 26.27           O  \n",
            "ATOM   1930  CB  PHE A 455       5.673  30.562  32.456  1.00 25.50           C  \n",
            "ATOM   1931  CG  PHE A 455       4.611  29.692  31.829  1.00 25.22           C  \n",
            "ATOM   1932  CD1 PHE A 455       4.943  28.705  30.898  1.00 25.94           C  \n",
            "ATOM   1933  CD2 PHE A 455       3.276  29.837  32.203  1.00 26.82           C  \n",
            "ATOM   1934  CE1 PHE A 455       3.966  27.874  30.347  1.00 24.55           C  \n",
            "ATOM   1935  CE2 PHE A 455       2.280  29.008  31.658  1.00 27.61           C  \n",
            "ATOM   1936  CZ  PHE A 455       2.633  28.024  30.724  1.00 26.98           C  \n",
            "ATOM   1937  N   LEU A 456       6.413  33.007  30.714  1.00 28.69           N  \n",
            "ATOM   1938  CA  LEU A 456       6.139  34.007  29.694  1.00 28.26           C  \n",
            "ATOM   1939  C   LEU A 456       7.286  34.106  28.697  1.00 30.02           C  \n",
            "ATOM   1940  O   LEU A 456       7.074  34.077  27.482  1.00 30.43           O  \n",
            "ATOM   1941  CB  LEU A 456       5.908  35.370  30.359  1.00 26.94           C  \n",
            "ATOM   1942  CG  LEU A 456       4.617  35.483  31.189  1.00 26.23           C  \n",
            "ATOM   1943  CD1 LEU A 456       4.691  36.650  32.163  1.00 22.08           C  \n",
            "ATOM   1944  CD2 LEU A 456       3.427  35.622  30.241  1.00 23.13           C  \n",
            "ATOM   1945  N   GLU A 457       8.507  34.201  29.209  1.00 30.48           N  \n",
            "ATOM   1946  CA  GLU A 457       9.669  34.332  28.345  1.00 30.89           C  \n",
            "ATOM   1947  C   GLU A 457       9.803  33.227  27.307  1.00 32.60           C  \n",
            "ATOM   1948  O   GLU A 457       9.981  33.503  26.119  1.00 33.35           O  \n",
            "ATOM   1949  CB  GLU A 457      10.946  34.403  29.181  1.00 28.95           C  \n",
            "ATOM   1950  CG  GLU A 457      12.158  34.821  28.387  1.00 28.75           C  \n",
            "ATOM   1951  CD  GLU A 457      13.405  34.977  29.239  1.00 31.57           C  \n",
            "ATOM   1952  OE1 GLU A 457      14.416  35.494  28.713  1.00 32.25           O  \n",
            "ATOM   1953  OE2 GLU A 457      13.387  34.582  30.427  1.00 30.90           O  \n",
            "ATOM   1954  N   VAL A 458       9.710  31.978  27.747  1.00 33.45           N  \n",
            "ATOM   1955  CA  VAL A 458       9.865  30.848  26.836  1.00 35.04           C  \n",
            "ATOM   1956  C   VAL A 458       8.760  30.662  25.789  1.00 36.21           C  \n",
            "ATOM   1957  O   VAL A 458       9.041  30.245  24.663  1.00 36.87           O  \n",
            "ATOM   1958  CB  VAL A 458      10.044  29.524  27.634  1.00 34.49           C  \n",
            "ATOM   1959  CG1 VAL A 458      10.126  28.336  26.692  1.00 35.57           C  \n",
            "ATOM   1960  CG2 VAL A 458      11.312  29.591  28.461  1.00 33.37           C  \n",
            "ATOM   1961  N   PHE A 459       7.516  30.987  26.135  1.00 37.46           N  \n",
            "ATOM   1962  CA  PHE A 459       6.407  30.793  25.196  1.00 38.91           C  \n",
            "ATOM   1963  C   PHE A 459       5.731  32.056  24.665  1.00 41.52           C  \n",
            "ATOM   1964  O   PHE A 459       4.709  31.982  23.982  1.00 39.82           O  \n",
            "ATOM   1965  CB  PHE A 459       5.368  29.862  25.830  1.00 34.40           C  \n",
            "ATOM   1966  CG  PHE A 459       5.935  28.538  26.245  1.00 33.94           C  \n",
            "ATOM   1967  CD1 PHE A 459       5.915  28.141  27.579  1.00 33.32           C  \n",
            "ATOM   1968  CD2 PHE A 459       6.537  27.702  25.308  1.00 31.99           C  \n",
            "ATOM   1969  CE1 PHE A 459       6.489  26.931  27.976  1.00 31.47           C  \n",
            "ATOM   1970  CE2 PHE A 459       7.114  26.490  25.693  1.00 31.69           C  \n",
            "ATOM   1971  CZ  PHE A 459       7.091  26.104  27.028  1.00 30.51           C  \n",
            "ATOM   1972  N   GLU A 460       6.319  33.210  24.966  1.00 46.24           N  \n",
            "ATOM   1973  CA  GLU A 460       5.779  34.493  24.518  1.00 48.61           C  \n",
            "ATOM   1974  C   GLU A 460       6.079  34.686  23.039  1.00 48.84           C  \n",
            "ATOM   1975  O   GLU A 460       5.117  34.886  22.257  1.00 50.05           O  \n",
            "ATOM   1976  CB  GLU A 460       6.415  35.640  25.298  1.00 51.43           C  \n",
            "ATOM   1977  CG  GLU A 460       5.542  36.868  25.381  1.00 55.47           C  \n",
            "ATOM   1978  CD  GLU A 460       4.963  37.063  26.772  1.00 58.31           C  \n",
            "ATOM   1979  OE1 GLU A 460       5.696  37.558  27.662  1.00 57.66           O  \n",
            "ATOM   1980  OE2 GLU A 460       3.781  36.707  26.976  1.00 58.43           O  \n",
            "ATOM   1981  OXT GLU A 460       7.283  34.643  22.691  1.00 46.39           O  \n",
            "TER    1982      GLU A 460                                                      \n",
            "HETATM 1983  O3  IH5 A 501       8.258  16.321  33.839  1.00 42.43           O  \n",
            "HETATM 1984  C15 IH5 A 501       8.950  17.348  33.292  1.00 41.79           C  \n",
            "HETATM 1985  O4  IH5 A 501      10.093  17.690  33.630  1.00 44.06           O  \n",
            "HETATM 1986  C13 IH5 A 501       8.252  18.097  32.198  1.00 39.80           C  \n",
            "HETATM 1987  C01 IH5 A 501       6.856  18.745  32.328  1.00 38.19           C  \n",
            "HETATM 1988  C11 IH5 A 501       5.955  18.402  31.250  1.00 36.69           C  \n",
            "HETATM 1989  C09 IH5 A 501       4.638  18.923  31.254  1.00 35.89           C  \n",
            "HETATM 1990  C07 IH5 A 501       4.169  19.826  32.328  1.00 34.73           C  \n",
            "HETATM 1991  O2  IH5 A 501       2.887  20.267  32.225  1.00 34.64           O  \n",
            "HETATM 1992  C2  IH5 A 501       2.596  21.554  31.536  1.00 31.02           C  \n",
            "HETATM 1993  C12 IH5 A 501       3.316  22.000  30.314  1.00 26.35           C  \n",
            "HETATM 1994  C10 IH5 A 501       3.007  23.220  29.695  1.00 26.30           C  \n",
            "HETATM 1995  C8  IH5 A 501       1.986  24.059  30.211  1.00 26.28           C  \n",
            "HETATM 1996  O1  IH5 A 501       1.737  25.219  29.560  1.00 24.58           O  \n",
            "HETATM 1997  C6  IH5 A 501       1.217  23.699  31.426  1.00 28.45           C  \n",
            "HETATM 1998  C4  IH5 A 501       1.542  22.412  32.085  1.00 28.62           C  \n",
            "HETATM 1999  C14 IH5 A 501       0.100  24.681  31.931  1.00 27.46           C  \n",
            "HETATM 2000  C16 IH5 A 501       0.708  25.497  33.090  1.00 27.72           C  \n",
            "HETATM 2001  C18 IH5 A 501      -1.179  23.903  32.434  1.00 27.41           C  \n",
            "HETATM 2002  C05 IH5 A 501       5.054  20.165  33.368  1.00 34.66           C  \n",
            "HETATM 2003  C03 IH5 A 501       6.395  19.637  33.393  1.00 36.31           C  \n",
            "HETATM 2004 CL5  IH5 A 501       4.576  21.219  34.619  1.00 36.84          CL  \n",
            "HETATM 2005 CL6  IH5 A 501       3.595  18.466  29.959  1.00 36.86          CL  \n",
            "HETATM 2006  O   HOH A 601       7.351  11.237  10.609  1.00 18.05           O  \n",
            "HETATM 2007  O   HOH A 602      14.679  23.765  18.704  1.00 24.51           O  \n",
            "HETATM 2008  O   HOH A 603       9.781  21.565  14.110  1.00 27.37           O  \n",
            "HETATM 2009  O   HOH A 604      -7.727  23.978  30.264  1.00 32.95           O  \n",
            "HETATM 2010  O   HOH A 605      -6.550  12.266  45.913  1.00 40.88           O  \n",
            "HETATM 2011  O   HOH A 606       8.965  13.943  34.969  1.00 26.02           O  \n",
            "HETATM 2012  O   HOH A 607      12.046  19.291  31.925  1.00 25.35           O  \n",
            "HETATM 2013  O   HOH A 608      -5.859   9.748  33.412  1.00 31.76           O  \n",
            "HETATM 2014  O   HOH A 609       5.288  25.372  22.747  1.00 26.02           O  \n",
            "HETATM 2015  O   HOH A 610      -8.897  11.939  34.133  1.00 36.61           O  \n",
            "HETATM 2016  O   HOH A 611       9.777  32.144  36.552  1.00 34.80           O  \n",
            "HETATM 2017  O   HOH A 612      30.241  15.639  22.792  1.00 30.48           O  \n",
            "HETATM 2018  O   HOH A 613      -3.343   3.136  30.268  1.00 33.31           O  \n",
            "HETATM 2019  O   HOH A 614      -5.531   8.325  30.538  1.00 30.56           O  \n",
            "HETATM 2020  O   HOH A 615      11.054  29.194  18.176  1.00 37.72           O  \n",
            "HETATM 2021  O   HOH A 616      -0.083  12.045  20.753  1.00 26.52           O  \n",
            "HETATM 2022  O   HOH A 617      -6.575  39.862  38.301  1.00 31.27           O  \n",
            "HETATM 2023  O   HOH A 618       9.354   3.801  26.905  1.00 41.98           O  \n",
            "HETATM 2024  O   HOH A 619       0.287  10.752  47.292  1.00 35.47           O  \n",
            "HETATM 2025  O   HOH A 620     -11.445  30.005  35.002  1.00 32.11           O  \n",
            "HETATM 2026  O   HOH A 621      11.453  20.792  41.638  1.00 39.82           O  \n",
            "HETATM 2027  O   HOH A 622     -11.275  22.149  44.466  1.00 41.63           O  \n",
            "HETATM 2028  O   HOH A 623       6.845  23.098  17.755  1.00 43.51           O  \n",
            "HETATM 2029  O   HOH A 624       1.282  -2.066  38.500  1.00 33.69           O  \n",
            "HETATM 2030  O   HOH A 625       3.801  37.497  36.059  1.00 46.78           O  \n",
            "HETATM 2031  O   HOH A 626      30.527  25.013  23.662  1.00 50.87           O  \n",
            "HETATM 2032  O   HOH A 627       5.207   4.030  30.474  1.00 40.83           O  \n",
            "HETATM 2033  O   HOH A 628     -18.469  20.709  33.782  1.00 27.39           O  \n",
            "HETATM 2034  O   HOH A 629      15.031   9.271  30.872  1.00 42.34           O  \n",
            "HETATM 2035  O   HOH A 630      25.543  16.813   7.649  1.00 44.57           O  \n",
            "HETATM 2036  O   HOH A 631      26.530  25.970  29.496  1.00 35.80           O  \n",
            "HETATM 2037  O   HOH A 632      22.568  35.680  15.419  1.00 46.06           O  \n",
            "HETATM 2038  O   HOH A 633      27.145  22.288   7.490  1.00 39.66           O  \n",
            "CONECT 1983 1984                                                                \n",
            "CONECT 1984 1983 1985 1986                                                      \n",
            "CONECT 1985 1984                                                                \n",
            "CONECT 1986 1984 1987                                                           \n",
            "CONECT 1987 1986 1988 2003                                                      \n",
            "CONECT 1988 1987 1989                                                           \n",
            "CONECT 1989 1988 1990 2005                                                      \n",
            "CONECT 1990 1989 1991 2002                                                      \n",
            "CONECT 1991 1990 1992                                                           \n",
            "CONECT 1992 1991 1993 1998                                                      \n",
            "CONECT 1993 1992 1994                                                           \n",
            "CONECT 1994 1993 1995                                                           \n",
            "CONECT 1995 1994 1996 1997                                                      \n",
            "CONECT 1996 1995                                                                \n",
            "CONECT 1997 1995 1998 1999                                                      \n",
            "CONECT 1998 1992 1997                                                           \n",
            "CONECT 1999 1997 2000 2001                                                      \n",
            "CONECT 2000 1999                                                                \n",
            "CONECT 2001 1999                                                                \n",
            "CONECT 2002 1990 2003 2004                                                      \n",
            "CONECT 2003 1987 2002                                                           \n",
            "CONECT 2004 2002                                                                \n",
            "CONECT 2005 1989                                                                \n",
            "MASTER      299    0    1   12    4    0    4    6 2037    1   23   20          \n",
            "END                                                                             \n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "%%file 1nax_add_H.mop\n",
        "GEO_DAT=\"1NAX.pdb\" OUTPUT HTML ADD-H NOCOMMENTS SITE=(SALT)\n",
        "Adding hydrogen atoms to 1NAX"
      ],
      "metadata": {
        "id": "icfXjkUE0QvG",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "91ca68eb-49ed-4000-c32c-d5aff29e75aa"
      },
      "execution_count": 12,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Writing 1nax_add_H.mop\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!mopac 1nax_add_H.mop"
      ],
      "metadata": {
        "id": "kgDZ0-pm0RAa",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "3f29130f-ff0e-4498-e4c1-d2bc7d7b5e10"
      },
      "execution_count": 13,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "          Tue Dec 10 13:37:35 2024  Job: '1nax_add_H' started successfully\n",
            "\n",
            "\n",
            "          MOPAC Job: \"1nax_add_H.mop\" ended normally on Dec 10, 2024, at 13:37.\n",
            "\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!cat < 1nax_add_H.out"
      ],
      "metadata": {
        "id": "BvqXypnC0RG5",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "c868bbfc-c595-4e5f-b24c-804ea5e2acb3"
      },
      "execution_count": 14,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            " *******************************************************************************\n",
            " **                                                                           **\n",
            " **                              MOPAC v23.0.3                                **\n",
            " **                                                                           **\n",
            " *******************************************************************************\n",
            " **          Digital Object Identifier (DOI): 10.5281/zenodo.6511958          **\n",
            " **    Visit the DOI location for information on how to cite this program.    **\n",
            " *******************************************************************************\n",
            "\n",
            "                              PM7 CALCULATION RESULTS\n",
            "\n",
            " *******************************************************************************\n",
            " *  CALCULATION DONE:                                Tue Dec 10 13:37:35 2024  *\n",
            " *  START_RES  - STARTING RESIDUE NUMBERS DEFINED\n",
            " *  Keyword:     START_RES=(211A 501A-601) \n",
            " *  GEO_DAT    - DATA SET GEOMETRY IS IN FILE \"1NAX.pdb\"\n",
            " *  CHAINS     - PDB CHAIN LETTERS EXPLICITLY DEFINED\n",
            " *  Keyword:     CHAINS=(AA) \n",
            " *  ADD-H      - ADD HYDROGEN ATOMS TO SATISFY VALENCE\n",
            " *  SITE       - SET IONIZATION LEVELS OF IONIZABLE RESIDUES \n",
            " *  T=         - A TIME OF 172800.0 SECONDS REQUESTED\n",
            " *  DUMP=N     - RESTART FILE WRITTEN EVERY 7200.0 SECONDS REQUESTED\n",
            " *  OUTPUT     - MINIMIZE OUTPUT\n",
            " *  NOCOMMENTS - SUPPRESS PDB COMMENTS\n",
            " *  HTML       - WRITE HTML SCRIPT TO READ PDB FILE USING JSMOL\n",
            " *******************************************************************************\n",
            " START_RES=(211A 501A-601)  CHAINS=(AA) GEO_DAT=\"1NAX.pdb\" OUTPUT HTML ADD-H NOCOMMENTS SITE=(SALT)\n",
            " Adding hydrogen atoms to 1NAX\n",
            "\n",
            " \n",
            "\n",
            " General Reference for PM7:\n",
            " \"Optimization of Parameters for Semiempirical Methods VI: More Modifications to the \n",
            " NDDO Approximations and Re-optimization of Parameters\", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)\n",
            " https://link.springer.com/article/10.1007/s00894-012-1667-x\n",
            "\n",
            "           Empirical Formula: C1288 N330 O401 S16 Cl2  =  2037 atoms\n",
            "\n",
            "\n",
            "                RESIDUE SEQUENCE IN PROTEIN Chain: A\n",
            "\n",
            "             1     2     3     4     5     6     7     8     9    10\n",
            "      210   LYS   PRO   GLU   PRO   THR   ASP   GLU   GLU   TRP   GLU   \n",
            "      220   LEU   ILE   LYS   THR   VAL   THR   GLU   ALA   HIS   VAL   \n",
            "      230   ALA   THR   ASN   ALA   GLN   GLY   SER   HIS   TRP   LYS   \n",
            "      240   GLN   LYS   ARG   LYS   PHE   LEU   PRO   GLU   ASP   ILE   \n",
            "      250   GLY   GLN   ALA   PRO   ILE   VAL   ASN   ALA   PRO   GLU   \n",
            "      260   GLY   GLY   LYS   VAL   ASP   LEU   GLU   ALA   PHE   SER   \n",
            "      270   HIS   PHE   THR   LYS   ILE   ILE   THR   PRO   ALA   ILE   \n",
            "      280   THR   ARG   VAL   VAL   ASP   PHE   ALA   LYS   LYS   LEU   \n",
            "      290   PRO   MET   PHE   CYS   GLU   LEU   PRO   CYS   GLU   ASP   \n",
            "      300   GLN   ILE   ILE   LEU   LEU   LYS   GLY   CYS   CYS   MET   \n",
            "      310   GLU   ILE   MET   SER   LEU   ARG   ALA   ALA   VAL   ARG   \n",
            "      320   TYR   ASP   PRO   GLU   SER   GLU   THR   LEU   THR   LEU   \n",
            "      330   ASN   GLY   GLU   MET   ALA   VAL   THR   ARG   GLY   GLN   \n",
            "      340   LEU   LYS   ASN   GLY   GLY   LEU   GLY   VAL   VAL   SER   \n",
            "      350   ASP   ALA   ILE   PHE   ASP   LEU   GLY   MET   SER   LEU   \n",
            "      360   SER   SER   PHE   ASN   LEU   ASP   ASP   THR   GLU   VAL   \n",
            "      370   ALA   LEU   LEU   GLN   ALA   VAL   LEU   LEU   MET   SER   \n",
            "      380   SER   ASP   ARG   PRO   GLY   LEU   ALA   CYS   VAL   GLU   \n",
            "      390   ARG   ILE   GLU   LYS   TYR   GLN   ASP   SER   PHE   LEU   \n",
            "      400   LEU   ALA   PHE   GLU   HIS   TYR   ILE   ASN   TYR   ARG   \n",
            "      410   LYS   HIS   HIS   VAL   THR   HIS   PHE   TRP   PRO   LYS   \n",
            "      420   LEU   LEU   MET   LYS   VAL   THR   ASP   LEU   ARG   MET   \n",
            "      430   ILE   GLY   ALA   CYS   HIS   ALA   SER   ARG   PHE   LEU   \n",
            "      440   HIS   MET   LYS   VAL   GLU   CYS   PRO   THR   GLU   LEU   \n",
            "      450   PHE   PRO   PRO   LEU   PHE   LEU   GLU   VAL   PHE   GLU   \n",
            "\n",
            "                RESIDUE SEQUENCE IN PROTEIN Chain: A\n",
            "\n",
            "      210  KPEPTDEEWE LIKTVTEAHV ATNAQGSHWK QKRKFLPEDI GQAPIVNAPE\n",
            "      260  GGKVDLEAFS HFTKIITPAI TRVVDFAKKL PMFCELPCED QIILLKGCCM\n",
            "      310  EIMSLRAAVR YDPESETLTL NGEMAVTRGQ LKNGGLGVVS DAIFDLGMSL\n",
            "      360  SSFNLDDTEV ALLQAVLLMS SDRPGLACVE RIEKYQDSFL LAFEHYINYR\n",
            "      410  KHHVTHFWPK LLMKVTDLRM IGACHASRFL HMKVECPTEL FPPLFLEVFE\n",
            "\n",
            "          ADD-H: SYSTEM HAS BEEN HYDROGENATED\n",
            "\n",
            "\n",
            "                   Salt Bridges Found (Up to 4.0 Angstroms)\n",
            "\n",
            "     No.          Cationic site                            Anionic site                Dist. (Angstroms)\n",
            "\n",
            "      1  (ATOM    460  NZ  LYS A 240)  A240(+)    (ATOM   1775  OD1 ASP A 322)  A322(-)      2.55\n",
            "      2  (ATOM   3026  ND1 HIS A 405)  A405(+)    (ATOM    117  OE1 GLU A 218)  A218(-)      2.56\n",
            "      3  (ATOM   2073  NZ  LYS A 342)  A342(+)    (ATOM   2190  OD2 ASP A 351)  A351(-)      2.57\n",
            "      4  (ATOM   3683  NZ  LYS A 443)  A443(+)    (ATOM   3968  OE2 GLU A 460)  A460(-)      2.58\n",
            "      5  (ATOM   3165  NE2 HIS A 412)  A412(+)    (ATOM   2454  OE2 GLU A 369)  A369(-)      2.70\n",
            "      6  (ATOM   2669  NH2 ARG A 383)  A383(+)    (ATOM   1591  OE1 GLU A 311)  A311(-)      2.83\n",
            "      7  (ATOM   1525  NZ  LYS A 306)  A306(+)    (ATOM   3969  OXT GLU A 460)  A460(-)      2.88\n",
            "      8  (ATOM   2008  NH1 ARG A 338)  A338(+)    (ATOM   2251  OD1 ASP A 355)  A355(-)      3.18\n",
            "      9  (ATOM   3143  NZ  LYS A 411)  A411(+)    (ATOM     45  OE2 GLU A 213)  A213(-)      3.23\n",
            "     10  (ATOM    212  NZ  LYS A 223)  A223(+)    (ATOM    276  OE1 GLU A 227)  A227(-)      3.36\n",
            "     11  (ATOM   3119  NE  ARG A 410)  A410(+)    (ATOM   2413  OD2 ASP A 366)  A366(-)      3.55\n",
            "     12  (ATOM   3379  NZ  LYS A 424)  A424(+)    (ATOM   3429  OD2 ASP A 427)  A427(-)      3.86\n",
            "\n",
            "       Changes in Ionization caused by keyword SITE\n",
            "\n",
            "       Keyword: \"SITE=(SALT)\" =>  SITE=(A240(+),A322(-),A405(+),A218(-),A342(+),A351(-),A443(+),\n",
            "       A460(-),A412(+),A369(-),A383(+),A311(-),A306(+),A460(-),A338(+),A355(-),A411(+),A213(-),\n",
            "       A223(+),A227(-),A410(+),A366(-),A424(+),A427(-))\n",
            "\n",
            "      Hydrogen atoms added to     Hydrogen atoms deleted from\n",
            "\n",
            "  1       NZ  LYS A 223                 OE1 GLU A 213\n",
            "  2       NZ  LYS A 240                 OE2 GLU A 218\n",
            "  3       NZ  LYS A 306                 OE1 GLU A 227\n",
            "  4       NH2 ARG A 338                 OE2 GLU A 311\n",
            "  5       NZ  LYS A 342                 OD2 ASP A 322\n",
            "  6       NH2 ARG A 383                 OD1 ASP A 351\n",
            "  7       NE2 HIS A 405                 OD1 ASP A 355\n",
            "  8       NH2 ARG A 410                 OD2 ASP A 366\n",
            "  9       NZ  LYS A 411                 OE1 GLU A 369\n",
            " 10       NE2 HIS A 412                 OD2 ASP A 427\n",
            " 11       NZ  LYS A 424                 OE1 GLU A 460\n",
            " 12       NZ  LYS A 443                 OXT GLU A 460\n",
            "\n",
            "          Change in net ionization:  +0\n",
            "\n",
            "          CHARGES MODIFIED BY SITE COMMAND\n",
            "\n",
            "          HYDROGEN ATOMS ADDED OR DELETED\n",
            "\n",
            "                  UNUSUALLY SHORT HYDROGEN BOND DISTANCES\n",
            "\n",
            "  Hydrogen   R(H-A)  Atom A    R(H-B)  Atom B     R(A-B)      Loc-A      Loc-B\n",
            "    3031      1.49   O  116     1.09   N 3023      2.56     GLU 218  A  HIS 405  A\n",
            " \n",
            "\n",
            " ******************************************************************************\n",
            " *                                                                            *\n",
            " *     Error and normal termination messages reported in this calculation     *\n",
            " *                                                                            *\n",
            " * ADD-H: SYSTEM HAS BEEN HYDROGENATED                                        *\n",
            " * CHARGES MODIFIED BY SITE COMMAND                                           *\n",
            " * HYDROGEN ATOMS ADDED OR DELETED                                            *\n",
            " * JOB ENDED NORMALLY                                                         *\n",
            " *                                                                            *\n",
            " ******************************************************************************\n",
            "\n",
            "\n",
            "\n",
            " TOTAL JOB TIME:             3.74 SECONDS\n",
            "\n",
            " == MOPAC DONE ==\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "# Vejam que a geometria com os √°tomos de hidrog√™nio adicionados est√£o no\n",
        "# novo arquivo PDB gerado - add_H.pdb\n",
        "!cat < 1nax_add_H.pdb"
      ],
      "metadata": {
        "id": "JDrjDv740ROS",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "807260f5-94e0-4192-c560-533ed718efe9"
      },
      "execution_count": 15,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "HEADER  data-set: 1nax_add_H\n",
            "REMARK  MOPAC, Version: 23.0.3\n",
            "REMARK  Date: Dec 10 2024 13:37\n",
            "ATOM      1  N   LYS A 211      23.762  -3.506   9.387  1.0   0.00      PROT N \n",
            "ATOM      2  CA  LYS A 211      23.345  -2.328  10.202  1.0   0.00      PROT C \n",
            "ATOM      3  C   LYS A 211      24.494  -1.837  11.070  1.0   0.00      PROT C \n",
            "ATOM      4  O   LYS A 211      24.771  -2.400  12.129  1.0   0.00      PROT O \n",
            "ATOM      5  CB  LYS A 211      22.153  -2.690  11.086  1.0   0.00      PROT C \n",
            "ATOM      6  CG  LYS A 211      20.884  -2.996  10.315  1.0   0.00      PROT C \n",
            "ATOM      7  CD  LYS A 211      19.698  -3.131  11.256  1.0   0.00      PROT C \n",
            "ATOM      8  CE  LYS A 211      18.388  -3.267  10.493  1.0   0.00      PROT C \n",
            "ATOM      9  NZ  LYS A 211      18.357  -4.488   9.634  1.0   0.00      PROT N \n",
            "ATOM     10  H1  LYS A 211      23.748  -3.452   8.369  1.0   0.00      PROT H \n",
            "ATOM     11  H2  LYS A 211      24.705  -3.884   9.482  1.0   0.00      PROT H \n",
            "ATOM     12  HA  LYS A 211      23.065  -1.542   9.500  1.0   0.00      PROT H \n",
            "ATOM     13  HB3 LYS A 211      21.978  -1.845  11.752  1.0   0.00      PROT H \n",
            "ATOM     14  HB2 LYS A 211      22.441  -3.562  11.673  1.0   0.00      PROT H \n",
            "ATOM     15  HG2 LYS A 211      20.683  -2.189   9.611  1.0   0.00      PROT H \n",
            "ATOM     16  HG3 LYS A 211      21.007  -3.932   9.769  1.0   0.00      PROT H \n",
            "ATOM     17  HD2 LYS A 211      19.827  -4.017  11.878  1.0   0.00      PROT H \n",
            "ATOM     18  HD3 LYS A 211      19.635  -2.247  11.891  1.0   0.00      PROT H \n",
            "ATOM     19  HE2 LYS A 211      17.561  -3.330  11.200  1.0   0.00      PROT H \n",
            "ATOM     20  HE3 LYS A 211      18.250  -2.398   9.850  1.0   0.00      PROT H \n",
            "ATOM     21  HZ1 LYS A 211      17.592  -5.082   9.920  1.0   0.00      PROT H \n",
            "ATOM     22  HZ2 LYS A 211      18.231  -4.218   8.669  1.0   0.00      PROT H \n",
            "ATOM     23  N   PRO A 212      25.169  -0.763  10.634  1.0   0.00      PROT N \n",
            "ATOM     24  CA  PRO A 212      26.301  -0.177  11.356  1.0   0.00      PROT C \n",
            "ATOM     25  C   PRO A 212      25.986   0.277  12.776  1.0   0.00      PROT C \n",
            "ATOM     26  O   PRO A 212      24.950   0.896  13.030  1.0   0.00      PROT O \n",
            "ATOM     27  CB  PRO A 212      26.709   0.991  10.464  1.0   0.00      PROT C \n",
            "ATOM     28  CG  PRO A 212      25.408   1.404   9.853  1.0   0.00      PROT C \n",
            "ATOM     29  CD  PRO A 212      24.792   0.081   9.486  1.0   0.00      PROT C \n",
            "ATOM     30  HA  PRO A 212      27.066  -0.940  11.499  1.0   0.00      PROT H \n",
            "ATOM     31  HB3 PRO A 212      27.404   0.699   9.677  1.0   0.00      PROT H \n",
            "ATOM     32  HB2 PRO A 212      27.128   1.826  11.026  1.0   0.00      PROT H \n",
            "ATOM     33  HG2 PRO A 212      24.821   1.957  10.587  1.0   0.00      PROT H \n",
            "ATOM     34  HG3 PRO A 212      25.598   2.039   8.987  1.0   0.00      PROT H \n",
            "ATOM     35  HD2 PRO A 212      23.714   0.199   9.379  1.0   0.00      PROT H \n",
            "ATOM     36  HD3 PRO A 212      25.214  -0.267   8.543  1.0   0.00      PROT H \n",
            "ATOM     37  N   GLU A 213      26.881  -0.043  13.704  1.0   0.00      PROT N \n",
            "ATOM     38  CA  GLU A 213      26.718   0.370  15.087  1.0   0.00      PROT C \n",
            "ATOM     39  C   GLU A 213      27.501   1.670  15.263  1.0   0.00      PROT C \n",
            "ATOM     40  O   GLU A 213      28.323   2.025  14.419  1.0   0.00      PROT O \n",
            "ATOM     41  CB  GLU A 213      27.229  -0.720  16.029  1.0   0.00      PROT C \n",
            "ATOM     42  CG  GLU A 213      26.197  -1.801  16.306  1.0   0.00      PROT C \n",
            "ATOM     43  CD  GLU A 213      26.719  -2.919  17.199  1.0   0.00      PROT C \n",
            "ATOM     44  OE1 GLU A 213      27.627  -2.664  18.021  1.0   0.00      PROT O \n",
            "ATOM     45  OE2 GLU A 213      26.203  -4.053  17.091  1.0   0.00      PROT O \n",
            "ATOM     46  H   GLU A 213      27.696  -0.585  13.454  1.0   0.00      PROT H \n",
            "ATOM     47  HA  GLU A 213      25.656   0.518  15.280  1.0   0.00      PROT H \n",
            "ATOM     48  HB3 GLU A 213      27.513  -0.240  16.966  1.0   0.00      PROT H \n",
            "ATOM     49  HB2 GLU A 213      28.110  -1.168  15.569  1.0   0.00      PROT H \n",
            "ATOM     50  HG2 GLU A 213      25.333  -1.360  16.804  1.0   0.00      PROT H \n",
            "ATOM     51  HG3 GLU A 213      25.882  -2.253  15.365  1.0   0.00      PROT H \n",
            "ATOM     52  N   PRO A 214      27.268   2.399  16.364  1.0   0.00      PROT N \n",
            "ATOM     53  CA  PRO A 214      27.990   3.660  16.566  1.0   0.00      PROT C \n",
            "ATOM     54  C   PRO A 214      29.523   3.604  16.517  1.0   0.00      PROT C \n",
            "ATOM     55  O   PRO A 214      30.151   2.674  17.028  1.0   0.00      PROT O \n",
            "ATOM     56  CB  PRO A 214      27.453   4.148  17.913  1.0   0.00      PROT C \n",
            "ATOM     57  CG  PRO A 214      27.110   2.887  18.618  1.0   0.00      PROT C \n",
            "ATOM     58  CD  PRO A 214      26.458   2.064  17.549  1.0   0.00      PROT C \n",
            "ATOM     59  HA  PRO A 214      27.777   4.305  15.713  1.0   0.00      PROT H \n",
            "ATOM     60  HB3 PRO A 214      26.558   4.761  17.808  1.0   0.00      PROT H \n",
            "ATOM     61  HB2 PRO A 214      28.201   4.694  18.488  1.0   0.00      PROT H \n",
            "ATOM     62  HG2 PRO A 214      28.020   2.427  19.002  1.0   0.00      PROT H \n",
            "ATOM     63  HG3 PRO A 214      26.436   3.106  19.446  1.0   0.00      PROT H \n",
            "ATOM     64  HD2 PRO A 214      26.513   1.009  17.818  1.0   0.00      PROT H \n",
            "ATOM     65  HD3 PRO A 214      25.414   2.360  17.449  1.0   0.00      PROT H \n",
            "ATOM     66  N   THR A 215      30.111   4.615  15.884  1.0   0.00      PROT N \n",
            "ATOM     67  CA  THR A 215      31.559   4.728  15.767  1.0   0.00      PROT C \n",
            "ATOM     68  C   THR A 215      32.137   5.192  17.107  1.0   0.00      PROT C \n",
            "ATOM     69  O   THR A 215      31.421   5.296  18.095  1.0   0.00      PROT O \n",
            "ATOM     70  CB  THR A 215      31.944   5.755  14.693  1.0   0.00      PROT C \n",
            "ATOM     71  OG1 THR A 215      31.542   7.062  15.119  1.0   0.00      PROT O \n",
            "ATOM     72  CG2 THR A 215      31.258   5.431  13.378  1.0   0.00      PROT C \n",
            "ATOM     73  H   THR A 215      29.550   5.341  15.462  1.0   0.00      PROT H \n",
            "ATOM     74  HA  THR A 215      31.909   3.733  15.491  1.0   0.00      PROT H \n",
            "ATOM     75  HB  THR A 215      33.025   5.787  14.557  1.0   0.00      PROT H \n",
            "ATOM     76  HG1 THR A 215      31.896   7.438  15.964  1.0   0.00      PROT H \n",
            "ATOM     77 HG21 THR A 215      31.542   6.169  12.628  1.0   0.00      PROT H \n",
            "ATOM     78 HG22 THR A 215      31.562   4.439  13.044  1.0   0.00      PROT H \n",
            "ATOM     79 HG23 THR A 215      30.177   5.452  13.517  1.0   0.00      PROT H \n",
            "ATOM     80  N   ASP A 216      33.433   5.465  17.145  1.0   0.00      PROT N \n",
            "ATOM     81  CA  ASP A 216      34.048   5.923  18.382  1.0   0.00      PROT C \n",
            "ATOM     82  C   ASP A 216      33.617   7.353  18.672  1.0   0.00      PROT C \n",
            "ATOM     83  O   ASP A 216      33.319   7.705  19.814  1.0   0.00      PROT O \n",
            "ATOM     84  CB  ASP A 216      35.575   5.871  18.285  1.0   0.00      PROT C \n",
            "ATOM     85  CG  ASP A 216      36.105   4.459  18.206  1.0   0.00      PROT C \n",
            "ATOM     86  OD1 ASP A 216      35.717   3.633  19.063  1.0   0.00      PROT O \n",
            "ATOM     87  OD2 ASP A 216      36.915   4.178  17.294  1.0   0.00      PROT O \n",
            "ATOM     88  H   ASP A 216      33.997   5.355  16.314  1.0   0.00      PROT H \n",
            "ATOM     89  HA  ASP A 216      33.692   5.242  19.155  1.0   0.00      PROT H \n",
            "ATOM     90  HB3 ASP A 216      35.983   6.363  19.168  1.0   0.00      PROT H \n",
            "ATOM     91  HB2 ASP A 216      35.872   6.421  17.392  1.0   0.00      PROT H \n",
            "ATOM     92  HD1 ASP A 216      36.147   2.745  18.902  1.0   0.00      PROT H \n",
            "ATOM     93  N   GLU A 217      33.593   8.174  17.628  1.0   0.00      PROT N \n",
            "ATOM     94  CA  GLU A 217      33.211   9.565  17.768  1.0   0.00      PROT C \n",
            "ATOM     95  C   GLU A 217      31.746   9.659  18.194  1.0   0.00      PROT C \n",
            "ATOM     96  O   GLU A 217      31.384  10.495  19.024  1.0   0.00      PROT O \n",
            "ATOM     97  CB  GLU A 217      33.441  10.307  16.448  1.0   0.00      PROT C \n",
            "ATOM     98  CG  GLU A 217      34.102  11.668  16.624  1.0   0.00      PROT C \n",
            "ATOM     99  CD  GLU A 217      34.416  12.358  15.306  1.0   0.00      PROT C \n",
            "ATOM    100  OE1 GLU A 217      35.168  11.777  14.494  1.0   0.00      PROT O \n",
            "ATOM    101  OE2 GLU A 217      33.916  13.486  15.087  1.0   0.00      PROT O \n",
            "ATOM    102  H   GLU A 217      33.845   7.833  16.711  1.0   0.00      PROT H \n",
            "ATOM    103  HA  GLU A 217      33.852   9.982  18.545  1.0   0.00      PROT H \n",
            "ATOM    104  HB3 GLU A 217      32.469  10.434  15.971  1.0   0.00      PROT H \n",
            "ATOM    105  HB2 GLU A 217      34.072   9.675  15.824  1.0   0.00      PROT H \n",
            "ATOM    106  HG3 GLU A 217      35.025  11.515  17.183  1.0   0.00      PROT H \n",
            "ATOM    107  HG2 GLU A 217      33.423  12.290  17.207  1.0   0.00      PROT H \n",
            "ATOM    108  HE2 GLU A 217      34.210  13.823  14.192  1.0   0.00      PROT H \n",
            "ATOM    109  N   GLU A 218      30.906   8.792  17.636  1.0   0.00      PROT N \n",
            "ATOM    110  CA  GLU A 218      29.490   8.786  17.976  1.0   0.00      PROT C \n",
            "ATOM    111  C   GLU A 218      29.251   8.337  19.416  1.0   0.00      PROT C \n",
            "ATOM    112  O   GLU A 218      28.323   8.810  20.066  1.0   0.00      PROT O \n",
            "ATOM    113  CB  GLU A 218      28.721   7.884  17.015  1.0   0.00      PROT C \n",
            "ATOM    114  CG  GLU A 218      28.690   8.398  15.587  1.0   0.00      PROT C \n",
            "ATOM    115  CD  GLU A 218      27.895   7.494  14.660  1.0   0.00      PROT C \n",
            "ATOM    116  OE1 GLU A 218      28.179   6.277  14.642  1.0   0.00      PROT O \n",
            "ATOM    117  OE2 GLU A 218      26.995   7.997  13.947  1.0   0.00      PROT O \n",
            "ATOM    118  H   GLU A 218      31.247   8.121  16.962  1.0   0.00      PROT H \n",
            "ATOM    119  HA  GLU A 218      29.175   9.824  17.869  1.0   0.00      PROT H \n",
            "ATOM    120  HB3 GLU A 218      27.702   7.801  17.393  1.0   0.00      PROT H \n",
            "ATOM    121  HB2 GLU A 218      29.198   6.904  17.038  1.0   0.00      PROT H \n",
            "ATOM    122  HG2 GLU A 218      28.231   9.386  15.567  1.0   0.00      PROT H \n",
            "ATOM    123  HG3 GLU A 218      29.708   8.462  15.202  1.0   0.00      PROT H \n",
            "ATOM    124  N   TRP A 219      30.091   7.430  19.911  1.0   0.00      PROT N \n",
            "ATOM    125  CA  TRP A 219      29.972   6.931  21.280  1.0   0.00      PROT C \n",
            "ATOM    126  C   TRP A 219      30.270   8.038  22.272  1.0   0.00      PROT C \n",
            "ATOM    127  O   TRP A 219      29.678   8.094  23.352  1.0   0.00      PROT O \n",
            "ATOM    128  CB  TRP A 219      30.934   5.768  21.529  1.0   0.00      PROT C \n",
            "ATOM    129  CG  TRP A 219      30.348   4.411  21.266  1.0   0.00      PROT C \n",
            "ATOM    130  CD1 TRP A 219      30.899   3.412  20.515  1.0   0.00      PROT C \n",
            "ATOM    131  CD2 TRP A 219      29.112   3.894  21.775  1.0   0.00      PROT C \n",
            "ATOM    132  NE1 TRP A 219      30.084   2.307  20.523  1.0   0.00      PROT N \n",
            "ATOM    133  CE2 TRP A 219      28.982   2.574  21.290  1.0   0.00      PROT C \n",
            "ATOM    134  CE3 TRP A 219      28.103   4.417  22.597  1.0   0.00      PROT C \n",
            "ATOM    135  CZ2 TRP A 219      27.880   1.767  21.599  1.0   0.00      PROT C \n",
            "ATOM    136  CZ3 TRP A 219      27.005   3.610  22.907  1.0   0.00      PROT C \n",
            "ATOM    137  CH2 TRP A 219      26.907   2.300  22.407  1.0   0.00      PROT C \n",
            "ATOM    138  H   TRP A 219      30.838   7.066  19.336  1.0   0.00      PROT H \n",
            "ATOM    139  HA  TRP A 219      28.938   6.599  21.376  1.0   0.00      PROT H \n",
            "ATOM    140  HB3 TRP A 219      31.248   5.833  22.571  1.0   0.00      PROT H \n",
            "ATOM    141  HB2 TRP A 219      31.796   5.924  20.880  1.0   0.00      PROT H \n",
            "ATOM    142  HD1 TRP A 219      31.856   3.568  20.018  1.0   0.00      PROT H \n",
            "ATOM    143  HE1 TRP A 219      30.183   1.406  20.078  1.0   0.00      PROT H \n",
            "ATOM    144  HE3 TRP A 219      28.163   5.431  22.992  1.0   0.00      PROT H \n",
            "ATOM    145  HZ2 TRP A 219      27.778   0.749  21.222  1.0   0.00      PROT H \n",
            "ATOM    146  HZ3 TRP A 219      26.209   3.994  23.545  1.0   0.00      PROT H \n",
            "ATOM    147  HH2 TRP A 219      26.034   1.705  22.674  1.0   0.00      PROT H \n",
            "ATOM    148  N   GLU A 220      31.197   8.916  21.904  1.0   0.00      PROT N \n",
            "ATOM    149  CA  GLU A 220      31.561  10.034  22.762  1.0   0.00      PROT C \n",
            "ATOM    150  C   GLU A 220      30.423  11.046  22.804  1.0   0.00      PROT C \n",
            "ATOM    151  O   GLU A 220      30.094  11.576  23.865  1.0   0.00      PROT O \n",
            "ATOM    152  CB  GLU A 220      32.853  10.680  22.260  1.0   0.00      PROT C \n",
            "ATOM    153  CG  GLU A 220      34.066   9.789  22.463  1.0   0.00      PROT C \n",
            "ATOM    154  CD  GLU A 220      34.121   9.211  23.879  1.0   0.00      PROT C \n",
            "ATOM    155  OE1 GLU A 220      34.157  10.009  24.845  1.0   0.00      PROT O \n",
            "ATOM    156  OE2 GLU A 220      34.119   7.963  24.030  1.0   0.00      PROT O \n",
            "ATOM    157  H   GLU A 220      31.664   8.815  21.014  1.0   0.00      PROT H \n",
            "ATOM    158  HA  GLU A 220      31.720   9.625  23.760  1.0   0.00      PROT H \n",
            "ATOM    159  HB3 GLU A 220      32.994  11.615  22.803  1.0   0.00      PROT H \n",
            "ATOM    160  HB2 GLU A 220      32.731  10.892  21.198  1.0   0.00      PROT H \n",
            "ATOM    161  HG2 GLU A 220      34.029   8.959  21.757  1.0   0.00      PROT H \n",
            "ATOM    162  HG3 GLU A 220      34.974  10.368  22.296  1.0   0.00      PROT H \n",
            "ATOM    163  HE2 GLU A 220      34.150   7.736  25.003  1.0   0.00      PROT H \n",
            "ATOM    164  N   LEU A 221      29.825  11.307  21.644  1.0   0.00      PROT N \n",
            "ATOM    165  CA  LEU A 221      28.699  12.225  21.557  1.0   0.00      PROT C \n",
            "ATOM    166  C   LEU A 221      27.582  11.626  22.409  1.0   0.00      PROT C \n",
            "ATOM    167  O   LEU A 221      26.991  12.303  23.254  1.0   0.00      PROT O \n",
            "ATOM    168  CB  LEU A 221      28.222  12.362  20.104  1.0   0.00      PROT C \n",
            "ATOM    169  CG  LEU A 221      26.991  13.250  19.831  1.0   0.00      PROT C \n",
            "ATOM    170  CD1 LEU A 221      27.266  14.691  20.272  1.0   0.00      PROT C \n",
            "ATOM    171  CD2 LEU A 221      26.633  13.211  18.341  1.0   0.00      PROT C \n",
            "ATOM    172  H   LEU A 221      30.151  10.864  20.797  1.0   0.00      PROT H \n",
            "ATOM    173  HA  LEU A 221      29.029  13.198  21.920  1.0   0.00      PROT H \n",
            "ATOM    174  HB2 LEU A 221      29.021  12.789  19.498  1.0   0.00      PROT H \n",
            "ATOM    175  HB3 LEU A 221      27.956  11.379  19.715  1.0   0.00      PROT H \n",
            "ATOM    176  HG  LEU A 221      26.144  12.871  20.403  1.0   0.00      PROT H \n",
            "ATOM    177 HD11 LEU A 221      27.519  14.705  21.332  1.0   0.00      PROT H \n",
            "ATOM    178 HD12 LEU A 221      28.098  15.095  19.695  1.0   0.00      PROT H \n",
            "ATOM    179 HD13 LEU A 221      26.377  15.299  20.103  1.0   0.00      PROT H \n",
            "ATOM    180 HD21 LEU A 221      26.440  12.181  18.041  1.0   0.00      PROT H \n",
            "ATOM    181 HD22 LEU A 221      25.742  13.814  18.166  1.0   0.00      PROT H \n",
            "ATOM    182 HD23 LEU A 221      27.463  13.610  17.757  1.0   0.00      PROT H \n",
            "ATOM    183  N   ILE A 222      27.315  10.342  22.185  1.0   0.00      PROT N \n",
            "ATOM    184  CA  ILE A 222      26.275   9.628  22.912  1.0   0.00      PROT C \n",
            "ATOM    185  C   ILE A 222      26.451   9.756  24.413  1.0   0.00      PROT C \n",
            "ATOM    186  O   ILE A 222      25.477   9.938  25.139  1.0   0.00      PROT O \n",
            "ATOM    187  CB  ILE A 222      26.243   8.118  22.512  1.0   0.00      PROT C \n",
            "ATOM    188  CG1 ILE A 222      25.615   7.969  21.116  1.0   0.00      PROT C \n",
            "ATOM    189  CG2 ILE A 222      25.464   7.298  23.545  1.0   0.00      PROT C \n",
            "ATOM    190  CD1 ILE A 222      25.529   6.541  20.615  1.0   0.00      PROT C \n",
            "ATOM    191  H   ILE A 222      27.841   9.830  21.491  1.0   0.00      PROT H \n",
            "ATOM    192  HA  ILE A 222      25.360  10.144  22.623  1.0   0.00      PROT H \n",
            "ATOM    193  HB  ILE A 222      27.257   7.719  22.485  1.0   0.00      PROT H \n",
            "ATOM    194 HG13 ILE A 222      26.223   8.559  20.430  1.0   0.00      PROT H \n",
            "ATOM    195 HG12 ILE A 222      24.613   8.395  21.172  1.0   0.00      PROT H \n",
            "ATOM    196 HG21 ILE A 222      25.925   7.417  24.526  1.0   0.00      PROT H \n",
            "ATOM    197 HG22 ILE A 222      24.432   7.648  23.583  1.0   0.00      PROT H \n",
            "ATOM    198 HG23 ILE A 222      25.481   6.246  23.261  1.0   0.00      PROT H \n",
            "ATOM    199 HD11 ILE A 222      25.074   6.530  19.625  1.0   0.00      PROT H \n",
            "ATOM    200 HD12 ILE A 222      26.531   6.115  20.559  1.0   0.00      PROT H \n",
            "ATOM    201 HD13 ILE A 222      24.921   5.951  21.301  1.0   0.00      PROT H \n",
            "ATOM    202  N   LYS A 223      27.688   9.670  24.885  1.0   0.00      PROT N \n",
            "ATOM    203  CA  LYS A 223      27.937   9.774  26.317  1.0   0.00      PROT C \n",
            "ATOM    204  C   LYS A 223      27.670  11.200  26.780  1.0   0.00      PROT C \n",
            "ATOM    205  O   LYS A 223      27.263  11.436  27.917  1.0   0.00      PROT O \n",
            "ATOM    206  CB  LYS A 223      29.378   9.373  26.641  1.0   0.00      PROT C \n",
            "ATOM    207  CG  LYS A 223      29.635   9.144  28.130  1.0   0.00      PROT C \n",
            "ATOM    208  CD  LYS A 223      31.021   8.544  28.392  1.0   0.00      PROT C \n",
            "ATOM    209  CE  LYS A 223      32.096   9.610  28.549  1.0   0.00      PROT C \n",
            "ATOM    210  NZ  LYS A 223      31.945  10.341  29.842  1.0   0.00      PROT N \n",
            "ATOM    211  H   LYS A 223      28.462   9.532  24.251  1.0   0.00      PROT H \n",
            "ATOM    212  HA  LYS A 223      27.245   9.085  26.800  1.0   0.00      PROT H \n",
            "ATOM    213  HB3 LYS A 223      30.025  10.172  26.279  1.0   0.00      PROT H \n",
            "ATOM    214  HB2 LYS A 223      29.590   8.456  26.091  1.0   0.00      PROT H \n",
            "ATOM    215  HG2 LYS A 223      29.572  10.094  28.660  1.0   0.00      PROT H \n",
            "ATOM    216  HG3 LYS A 223      28.888   8.456  28.526  1.0   0.00      PROT H \n",
            "ATOM    217  HD3 LYS A 223      31.271   7.897  27.552  1.0   0.00      PROT H \n",
            "ATOM    218  HD2 LYS A 223      30.963   7.950  29.304  1.0   0.00      PROT H \n",
            "ATOM    219  HE2 LYS A 223      33.079   9.140  28.527  1.0   0.00      PROT H \n",
            "ATOM    220  HE3 LYS A 223      32.018  10.328  27.732  1.0   0.00      PROT H \n",
            "ATOM    221  HZ1 LYS A 223      32.782  10.223  30.395  1.0   0.00      PROT H \n",
            "ATOM    222  HZ2 LYS A 223      31.798  11.323  29.658  1.0   0.00      PROT H \n",
            "ATOM    223  HZ3 LYS A 223      31.208  10.224  30.651  1.0   0.00      PROT H \n",
            "ATOM    224  N   THR A 224      27.888  12.148  25.879  1.0   0.00      PROT N \n",
            "ATOM    225  CA  THR A 224      27.672  13.554  26.178  1.0   0.00      PROT C \n",
            "ATOM    226  C   THR A 224      26.193  13.920  26.358  1.0   0.00      PROT C \n",
            "ATOM    227  O   THR A 224      25.854  14.588  27.331  1.0   0.00      PROT O \n",
            "ATOM    228  CB  THR A 224      28.280  14.446  25.080  1.0   0.00      PROT C \n",
            "ATOM    229  OG1 THR A 224      29.700  14.243  25.035  1.0   0.00      PROT O \n",
            "ATOM    230  CG2 THR A 224      27.985  15.911  25.363  1.0   0.00      PROT C \n",
            "ATOM    231  H   THR A 224      28.213  11.901  24.955  1.0   0.00      PROT H \n",
            "ATOM    232  HA  THR A 224      28.186  13.670  27.132  1.0   0.00      PROT H \n",
            "ATOM    233  HB  THR A 224      27.899  14.173  24.096  1.0   0.00      PROT H \n",
            "ATOM    234  HG1 THR A 224      30.066  13.340  24.860  1.0   0.00      PROT H \n",
            "ATOM    235 HG21 THR A 224      28.422  16.528  24.578  1.0   0.00      PROT H \n",
            "ATOM    236 HG22 THR A 224      26.906  16.066  25.389  1.0   0.00      PROT H \n",
            "ATOM    237 HG23 THR A 224      28.416  16.189  26.325  1.0   0.00      PROT H \n",
            "ATOM    238  N   VAL A 225      25.324  13.498  25.433  1.0   0.00      PROT N \n",
            "ATOM    239  CA  VAL A 225      23.891  13.797  25.548  1.0   0.00      PROT C \n",
            "ATOM    240  C   VAL A 225      23.234  13.039  26.689  1.0   0.00      PROT C \n",
            "ATOM    241  O   VAL A 225      22.460  13.612  27.456  1.0   0.00      PROT O \n",
            "ATOM    242  CB  VAL A 225      23.076  13.451  24.264  1.0   0.00      PROT C \n",
            "ATOM    243  CG1 VAL A 225      23.260  14.513  23.230  1.0   0.00      PROT C \n",
            "ATOM    244  CG2 VAL A 225      23.481  12.093  23.725  1.0   0.00      PROT C \n",
            "ATOM    245  H   VAL A 225      25.650  12.965  24.640  1.0   0.00      PROT H \n",
            "ATOM    246  HA  VAL A 225      23.921  14.869  25.745  1.0   0.00      PROT H \n",
            "ATOM    247  HB  VAL A 225      22.009  13.413  24.484  1.0   0.00      PROT H \n",
            "ATOM    248 HG11 VAL A 225      22.962  15.477  23.642  1.0   0.00      PROT H \n",
            "ATOM    249 HG12 VAL A 225      24.308  14.553  22.933  1.0   0.00      PROT H \n",
            "ATOM    250 HG13 VAL A 225      22.644  14.284  22.360  1.0   0.00      PROT H \n",
            "ATOM    251 HG21 VAL A 225      23.337  11.338  24.498  1.0   0.00      PROT H \n",
            "ATOM    252 HG22 VAL A 225      22.867  11.848  22.859  1.0   0.00      PROT H \n",
            "ATOM    253 HG23 VAL A 225      24.530  12.117  23.431  1.0   0.00      PROT H \n",
            "ATOM    254  N   THR A 226      23.535  11.747  26.785  1.0   0.00      PROT N \n",
            "ATOM    255  CA  THR A 226      22.965  10.906  27.826  1.0   0.00      PROT C \n",
            "ATOM    256  C   THR A 226      23.224  11.512  29.199  1.0   0.00      PROT C \n",
            "ATOM    257  O   THR A 226      22.298  11.698  29.985  1.0   0.00      PROT O \n",
            "ATOM    258  CB  THR A 226      23.559   9.482  27.793  1.0   0.00      PROT C \n",
            "ATOM    259  OG1 THR A 226      23.510   8.962  26.456  1.0   0.00      PROT O \n",
            "ATOM    260  CG2 THR A 226      22.760   8.563  28.714  1.0   0.00      PROT C \n",
            "ATOM    261  H   THR A 226      24.174  11.327  26.125  1.0   0.00      PROT H \n",
            "ATOM    262  HA  THR A 226      21.898  10.889  27.602  1.0   0.00      PROT H \n",
            "ATOM    263  HB  THR A 226      24.612   9.493  28.073  1.0   0.00      PROT H \n",
            "ATOM    264  HG1 THR A 226      23.978   9.436  25.723  1.0   0.00      PROT H \n",
            "ATOM    265 HG21 THR A 226      23.186   7.560  28.685  1.0   0.00      PROT H \n",
            "ATOM    266 HG22 THR A 226      22.801   8.946  29.734  1.0   0.00      PROT H \n",
            "ATOM    267 HG23 THR A 226      21.723   8.527  28.381  1.0   0.00      PROT H \n",
            "ATOM    268  N   GLU A 227      24.483  11.824  29.489  1.0   0.00      PROT N \n",
            "ATOM    269  CA  GLU A 227      24.828  12.409  30.778  1.0   0.00      PROT C \n",
            "ATOM    270  C   GLU A 227      24.163  13.768  30.964  1.0   0.00      PROT C \n",
            "ATOM    271  O   GLU A 227      23.755  14.119  32.078  1.0   0.00      PROT O \n",
            "ATOM    272  CB  GLU A 227      26.347  12.548  30.919  1.0   0.00      PROT C \n",
            "ATOM    273  CG  GLU A 227      27.080  11.217  30.986  1.0   0.00      PROT C \n",
            "ATOM    274  CD  GLU A 227      28.564  11.378  31.251  1.0   0.00      PROT C \n",
            "ATOM    275  OE1 GLU A 227      29.214  12.176  30.537  1.0   0.00      PROT O \n",
            "ATOM    276  OE2 GLU A 227      29.085  10.705  32.165  1.0   0.00      PROT O \n",
            "ATOM    277  H   GLU A 227      25.214  11.656  28.812  1.0   0.00      PROT H \n",
            "ATOM    278  HA  GLU A 227      24.437  11.708  31.516  1.0   0.00      PROT H \n",
            "ATOM    279  HB3 GLU A 227      26.537  13.114  31.831  1.0   0.00      PROT H \n",
            "ATOM    280  HB2 GLU A 227      26.702  13.115  30.059  1.0   0.00      PROT H \n",
            "ATOM    281  HG3 GLU A 227      26.934  10.708  30.033  1.0   0.00      PROT H \n",
            "ATOM    282  HG2 GLU A 227      26.630  10.628  31.785  1.0   0.00      PROT H \n",
            "ATOM    283  N   ALA A 228      24.057  14.521  29.870  1.0   0.00      PROT N \n",
            "ATOM    284  CA  ALA A 228      23.434  15.840  29.895  1.0   0.00      PROT C \n",
            "ATOM    285  C   ALA A 228      21.974  15.691  30.304  1.0   0.00      PROT C \n",
            "ATOM    286  O   ALA A 228      21.465  16.427  31.157  1.0   0.00      PROT O \n",
            "ATOM    287  CB  ALA A 228      23.524  16.495  28.512  1.0   0.00      PROT C \n",
            "ATOM    288  H   ALA A 228      24.414  14.183  28.988  1.0   0.00      PROT H \n",
            "ATOM    289  HA  ALA A 228      23.981  16.435  30.626  1.0   0.00      PROT H \n",
            "ATOM    290  HB1 ALA A 228      24.571  16.598  28.226  1.0   0.00      PROT H \n",
            "ATOM    291  HB2 ALA A 228      23.008  15.873  27.781  1.0   0.00      PROT H \n",
            "ATOM    292  HB3 ALA A 228      23.058  17.480  28.545  1.0   0.00      PROT H \n",
            "ATOM    293  N   HIS A 229      21.307  14.730  29.678  1.0   0.00      PROT N \n",
            "ATOM    294  CA  HIS A 229      19.917  14.458  29.962  1.0   0.00      PROT C \n",
            "ATOM    295  C   HIS A 229      19.743  14.021  31.405  1.0   0.00      PROT C \n",
            "ATOM    296  O   HIS A 229      18.989  14.630  32.156  1.0   0.00      PROT O \n",
            "ATOM    297  CB  HIS A 229      19.400  13.358  29.038  1.0   0.00      PROT C \n",
            "ATOM    298  CG  HIS A 229      18.023  12.881  29.393  1.0   0.00      PROT C \n",
            "ATOM    299  ND1 HIS A 229      16.967  13.739  29.568  1.0   0.00      PROT N \n",
            "ATOM    300  CD2 HIS A 229      17.546  11.635  29.609  1.0   0.00      PROT C \n",
            "ATOM    301  CE1 HIS A 229      15.884  13.042  29.878  1.0   0.00      PROT C \n",
            "ATOM    302  NE2 HIS A 229      16.210  11.769  29.908  1.0   0.00      PROT N \n",
            "ATOM    303  H   HIS A 229      21.771  14.165  28.981  1.0   0.00      PROT H \n",
            "ATOM    304  HA  HIS A 229      19.397  15.400  29.787  1.0   0.00      PROT H \n",
            "ATOM    305  HB3 HIS A 229      20.100  12.524  29.098  1.0   0.00      PROT H \n",
            "ATOM    306  HB2 HIS A 229      19.396  13.757  28.024  1.0   0.00      PROT H \n",
            "ATOM    307  HD1 HIS A 229      17.015  14.823  29.467  1.0   0.00      PROT H \n",
            "ATOM    308  HD2 HIS A 229      18.211  10.775  29.527  1.0   0.00      PROT H \n",
            "ATOM    309  HE1 HIS A 229      14.938  13.554  30.055  1.0   0.00      PROT H \n",
            "ATOM    310  N   VAL A 230      20.446  12.958  31.785  1.0   0.00      PROT N \n",
            "ATOM    311  CA  VAL A 230      20.356  12.406  33.132  1.0   0.00      PROT C \n",
            "ATOM    312  C   VAL A 230      20.576  13.447  34.226  1.0   0.00      PROT C \n",
            "ATOM    313  O   VAL A 230      19.842  13.486  35.214  1.0   0.00      PROT O \n",
            "ATOM    314  CB  VAL A 230      21.366  11.255  33.308  1.0   0.00      PROT C \n",
            "ATOM    315  CG1 VAL A 230      21.357  10.766  34.730  1.0   0.00      PROT C \n",
            "ATOM    316  CG2 VAL A 230      21.017  10.125  32.371  1.0   0.00      PROT C \n",
            "ATOM    317  H   VAL A 230      21.069  12.504  31.132  1.0   0.00      PROT H \n",
            "ATOM    318  HA  VAL A 230      19.324  12.060  33.192  1.0   0.00      PROT H \n",
            "ATOM    319  HB  VAL A 230      22.375  11.600  33.080  1.0   0.00      PROT H \n",
            "ATOM    320 HG11 VAL A 230      21.610  11.589  35.399  1.0   0.00      PROT H \n",
            "ATOM    321 HG12 VAL A 230      20.365  10.388  34.978  1.0   0.00      PROT H \n",
            "ATOM    322 HG13 VAL A 230      22.089   9.967  34.844  1.0   0.00      PROT H \n",
            "ATOM    323 HG21 VAL A 230      21.026  10.489  31.344  1.0   0.00      PROT H \n",
            "ATOM    324 HG22 VAL A 230      21.748   9.324  32.479  1.0   0.00      PROT H \n",
            "ATOM    325 HG23 VAL A 230      20.024   9.746  32.613  1.0   0.00      PROT H \n",
            "ATOM    326  N   ALA A 231      21.584  14.291  34.041  1.0   0.00      PROT N \n",
            "ATOM    327  CA  ALA A 231      21.907  15.337  35.004  1.0   0.00      PROT C \n",
            "ATOM    328  C   ALA A 231      20.801  16.381  35.137  1.0   0.00      PROT C \n",
            "ATOM    329  O   ALA A 231      20.726  17.080  36.148  1.0   0.00      PROT O \n",
            "ATOM    330  CB  ALA A 231      23.200  16.030  34.596  1.0   0.00      PROT C \n",
            "ATOM    331  H   ALA A 231      22.156  14.220  33.212  1.0   0.00      PROT H \n",
            "ATOM    332  HA  ALA A 231      22.011  14.808  35.951  1.0   0.00      PROT H \n",
            "ATOM    333  HB1 ALA A 231      23.994  15.290  34.499  1.0   0.00      PROT H \n",
            "ATOM    334  HB2 ALA A 231      23.056  16.535  33.641  1.0   0.00      PROT H \n",
            "ATOM    335  HB3 ALA A 231      23.476  16.761  35.356  1.0   0.00      PROT H \n",
            "ATOM    336  N   THR A 232      19.947  16.480  34.122  1.0   0.00      PROT N \n",
            "ATOM    337  CA  THR A 232      18.879  17.465  34.122  1.0   0.00      PROT C \n",
            "ATOM    338  C   THR A 232      17.494  16.856  34.083  1.0   0.00      PROT C \n",
            "ATOM    339  O   THR A 232      16.522  17.547  33.782  1.0   0.00      PROT O \n",
            "ATOM    340  CB  THR A 232      19.012  18.420  32.913  1.0   0.00      PROT C \n",
            "ATOM    341  OG1 THR A 232      18.996  17.664  31.691  1.0   0.00      PROT O \n",
            "ATOM    342  CG2 THR A 232      20.304  19.228  33.009  1.0   0.00      PROT C \n",
            "ATOM    343  H   THR A 232      20.033  15.862  33.328  1.0   0.00      PROT H \n",
            "ATOM    344  HA  THR A 232      19.021  17.958  35.084  1.0   0.00      PROT H \n",
            "ATOM    345  HB  THR A 232      18.158  19.095  32.851  1.0   0.00      PROT H \n",
            "ATOM    346  HG1 THR A 232      18.222  17.085  31.477  1.0   0.00      PROT H \n",
            "ATOM    347 HG21 THR A 232      20.380  19.894  32.150  1.0   0.00      PROT H \n",
            "ATOM    348 HG22 THR A 232      20.297  19.817  33.926  1.0   0.00      PROT H \n",
            "ATOM    349 HG23 THR A 232      21.157  18.549  33.020  1.0   0.00      PROT H \n",
            "ATOM    350  N   ASN A 233      17.382  15.573  34.395  1.0   0.00      PROT N \n",
            "ATOM    351  CA  ASN A 233      16.070  14.945  34.349  1.0   0.00      PROT C \n",
            "ATOM    352  C   ASN A 233      15.439  14.633  35.701  1.0   0.00      PROT C \n",
            "ATOM    353  O   ASN A 233      14.630  13.713  35.808  1.0   0.00      PROT O \n",
            "ATOM    354  CB  ASN A 233      16.124  13.669  33.527  1.0   0.00      PROT C \n",
            "ATOM    355  CG  ASN A 233      14.746  13.156  33.171  1.0   0.00      PROT C \n",
            "ATOM    356  OD1 ASN A 233      14.549  11.958  33.001  1.0   0.00      PROT O \n",
            "ATOM    357  ND2 ASN A 233      13.788  14.063  33.043  1.0   0.00      PROT N \n",
            "ATOM    358  H   ASN A 233      18.201  15.044  34.660  1.0   0.00      PROT H \n",
            "ATOM    359  HA  ASN A 233      15.472  15.726  33.880  1.0   0.00      PROT H \n",
            "ATOM    360  HB3 ASN A 233      16.653  12.915  34.109  1.0   0.00      PROT H \n",
            "ATOM    361  HB2 ASN A 233      16.681  13.879  32.614  1.0   0.00      PROT H \n",
            "ATOM    362 HD21 ASN A 233      13.997  15.050  33.194  1.0   0.00      PROT H \n",
            "ATOM    363 HD22 ASN A 233      12.842  13.774  32.794  1.0   0.00      PROT H \n",
            "ATOM    364  N   ALA A 234      15.801  15.394  36.727  1.0   0.00      PROT N \n",
            "ATOM    365  CA  ALA A 234      15.243  15.205  38.063  1.0   0.00      PROT C \n",
            "ATOM    366  C   ALA A 234      15.014  13.738  38.422  1.0   0.00      PROT C \n",
            "ATOM    367  O   ALA A 234      13.933  13.366  38.867  1.0   0.00      PROT O \n",
            "ATOM    368  CB  ALA A 234      13.928  15.979  38.180  1.0   0.00      PROT C \n",
            "ATOM    369  H   ALA A 234      16.481  16.129  36.595  1.0   0.00      PROT H \n",
            "ATOM    370  HA  ALA A 234      16.010  15.597  38.732  1.0   0.00      PROT H \n",
            "ATOM    371  HB1 ALA A 234      14.098  17.025  37.923  1.0   0.00      PROT H \n",
            "ATOM    372  HB2 ALA A 234      13.193  15.552  37.498  1.0   0.00      PROT H \n",
            "ATOM    373  HB3 ALA A 234      13.557  15.912  39.203  1.0   0.00      PROT H \n",
            "ATOM    374  N   GLN A 235      16.029  12.903  38.234  1.0   0.00      PROT N \n",
            "ATOM    375  CA  GLN A 235      15.884  11.487  38.544  1.0   0.00      PROT C \n",
            "ATOM    376  C   GLN A 235      16.972  10.972  39.482  1.0   0.00      PROT C \n",
            "ATOM    377  O   GLN A 235      17.553   9.907  39.255  1.0   0.00      PROT O \n",
            "ATOM    378  CB  GLN A 235      15.852  10.658  37.249  1.0   0.00      PROT C \n",
            "ATOM    379  CG  GLN A 235      17.102  10.748  36.369  1.0   0.00      PROT C \n",
            "ATOM    380  CD  GLN A 235      16.928  10.044  35.019  1.0   0.00      PROT C \n",
            "ATOM    381  OE1 GLN A 235      16.613   8.852  34.957  1.0   0.00      PROT O \n",
            "ATOM    382  NE2 GLN A 235      17.134  10.788  33.931  1.0   0.00      PROT N \n",
            "ATOM    383  H   GLN A 235      16.907  13.249  37.874  1.0   0.00      PROT H \n",
            "ATOM    384  HA  GLN A 235      14.932  11.430  39.072  1.0   0.00      PROT H \n",
            "ATOM    385  HB3 GLN A 235      14.986  11.006  36.686  1.0   0.00      PROT H \n",
            "ATOM    386  HB2 GLN A 235      15.696   9.623  37.554  1.0   0.00      PROT H \n",
            "ATOM    387  HG2 GLN A 235      17.333  11.795  36.172  1.0   0.00      PROT H \n",
            "ATOM    388  HG3 GLN A 235      17.942  10.281  36.883  1.0   0.00      PROT H \n",
            "ATOM    389 HE21 GLN A 235      17.393  11.770  34.026  1.0   0.00      PROT H \n",
            "ATOM    390 HE22 GLN A 235      17.032  10.375  33.004  1.0   0.00      PROT H \n",
            "ATOM    391  N   GLY A 236      17.251  11.738  40.536  1.0   0.00      PROT N \n",
            "ATOM    392  CA  GLY A 236      18.252  11.318  41.500  1.0   0.00      PROT C \n",
            "ATOM    393  C   GLY A 236      17.767  10.003  42.072  1.0   0.00      PROT C \n",
            "ATOM    394  O   GLY A 236      18.548   9.082  42.317  1.0   0.00      PROT O \n",
            "ATOM    395  H   GLY A 236      16.765  12.614  40.666  1.0   0.00      PROT H \n",
            "ATOM    396  HA2 GLY A 236      18.352  12.048  42.303  1.0   0.00      PROT H \n",
            "ATOM    397  HA3 GLY A 236      19.219  11.171  41.020  1.0   0.00      PROT H \n",
            "ATOM    398  N   SER A 237      16.453   9.935  42.278  1.0   0.00      PROT N \n",
            "ATOM    399  CA  SER A 237      15.778   8.746  42.789  1.0   0.00      PROT C \n",
            "ATOM    400  C   SER A 237      14.649   8.439  41.798  1.0   0.00      PROT C \n",
            "ATOM    401  O   SER A 237      14.073   9.356  41.209  1.0   0.00      PROT O \n",
            "ATOM    402  CB  SER A 237      15.195   9.007  44.184  1.0   0.00      PROT C \n",
            "ATOM    403  OG  SER A 237      14.077   9.880  44.130  1.0   0.00      PROT O \n",
            "ATOM    404  H   SER A 237      15.869  10.734  42.078  1.0   0.00      PROT H \n",
            "ATOM    405  HA  SER A 237      16.509   7.941  42.864  1.0   0.00      PROT H \n",
            "ATOM    406  HB2 SER A 237      14.873   8.064  44.626  1.0   0.00      PROT H \n",
            "ATOM    407  HB3 SER A 237      15.957   9.463  44.817  1.0   0.00      PROT H \n",
            "ATOM    408  HG  SER A 237      13.605  10.132  44.963  1.0   0.00      PROT H \n",
            "ATOM    409  N   HIS A 238      14.340   7.160  41.608  1.0   0.00      PROT N \n",
            "ATOM    410  CA  HIS A 238      13.287   6.776  40.679  1.0   0.00      PROT C \n",
            "ATOM    411  C   HIS A 238      11.994   7.523  40.998  1.0   0.00      PROT C \n",
            "ATOM    412  O   HIS A 238      11.572   7.596  42.154  1.0   0.00      PROT O \n",
            "ATOM    413  CB  HIS A 238      13.046   5.265  40.731  1.0   0.00      PROT C \n",
            "ATOM    414  CG  HIS A 238      12.263   4.743  39.565  1.0   0.00      PROT C \n",
            "ATOM    415  ND1 HIS A 238      12.728   4.805  38.268  1.0   0.00      PROT N \n",
            "ATOM    416  CD2 HIS A 238      11.035   4.176  39.497  1.0   0.00      PROT C \n",
            "ATOM    417  CE1 HIS A 238      11.819   4.301  37.451  1.0   0.00      PROT C \n",
            "ATOM    418  NE2 HIS A 238      10.783   3.912  38.171  1.0   0.00      PROT N \n",
            "ATOM    419  H   HIS A 238      14.838   6.440  42.111  1.0   0.00      PROT H \n",
            "ATOM    420  HA  HIS A 238      13.652   7.064  39.693  1.0   0.00      PROT H \n",
            "ATOM    421  HB3 HIS A 238      12.508   5.053  41.655  1.0   0.00      PROT H \n",
            "ATOM    422  HB2 HIS A 238      14.022   4.780  40.760  1.0   0.00      PROT H \n",
            "ATOM    423  HD1 HIS A 238      13.695   5.208  37.968  1.0   0.00      PROT H \n",
            "ATOM    424  HD2 HIS A 238      10.458   4.015  40.407  1.0   0.00      PROT H \n",
            "ATOM    425  HE1 HIS A 238      12.009   4.267  36.378  1.0   0.00      PROT H \n",
            "ATOM    426  N   TRP A 239      11.375   8.074  39.959  1.0   0.00      PROT N \n",
            "ATOM    427  CA  TRP A 239      10.140   8.844  40.080  1.0   0.00      PROT C \n",
            "ATOM    428  C   TRP A 239       8.950   8.066  40.636  1.0   0.00      PROT C \n",
            "ATOM    429  O   TRP A 239       8.165   8.608  41.412  1.0   0.00      PROT O \n",
            "ATOM    430  CB  TRP A 239       9.752   9.425  38.715  1.0   0.00      PROT C \n",
            "ATOM    431  CG  TRP A 239       9.571   8.380  37.651  1.0   0.00      PROT C \n",
            "ATOM    432  CD1 TRP A 239      10.559   7.731  36.964  1.0   0.00      PROT C \n",
            "ATOM    433  CD2 TRP A 239       8.330   7.837  37.180  1.0   0.00      PROT C \n",
            "ATOM    434  NE1 TRP A 239      10.009   6.817  36.096  1.0   0.00      PROT N \n",
            "ATOM    435  CE2 TRP A 239       8.644   6.860  36.209  1.0   0.00      PROT C \n",
            "ATOM    436  CE3 TRP A 239       6.984   8.077  37.486  1.0   0.00      PROT C \n",
            "ATOM    437  CZ2 TRP A 239       7.660   6.125  35.540  1.0   0.00      PROT C \n",
            "ATOM    438  CZ3 TRP A 239       6.005   7.344  36.819  1.0   0.00      PROT C \n",
            "ATOM    439  CH2 TRP A 239       6.351   6.380  35.858  1.0   0.00      PROT C \n",
            "ATOM    440  H   TRP A 239      11.757   7.970  39.030  1.0   0.00      PROT H \n",
            "ATOM    441  HA  TRP A 239      10.410   9.605  40.812  1.0   0.00      PROT H \n",
            "ATOM    442  HB3 TRP A 239      10.542  10.114  38.417  1.0   0.00      PROT H \n",
            "ATOM    443  HB2 TRP A 239       8.820   9.975  38.846  1.0   0.00      PROT H \n",
            "ATOM    444  HD1 TRP A 239      11.604   7.976  37.151  1.0   0.00      PROT H \n",
            "ATOM    445  HE1 TRP A 239      10.442   6.177  35.446  1.0   0.00      PROT H \n",
            "ATOM    446  HE3 TRP A 239       6.696   8.820  38.230  1.0   0.00      PROT H \n",
            "ATOM    447  HZ2 TRP A 239       7.902   5.372  34.790  1.0   0.00      PROT H \n",
            "ATOM    448  HZ3 TRP A 239       4.951   7.515  37.040  1.0   0.00      PROT H \n",
            "ATOM    449  HH2 TRP A 239       5.552   5.829  35.361  1.0   0.00      PROT H \n",
            "ATOM    450  N   LYS A 240       8.808   6.809  40.222  1.0   0.00      PROT N \n",
            "ATOM    451  CA  LYS A 240       7.701   5.967  40.673  1.0   0.00      PROT C \n",
            "ATOM    452  C   LYS A 240       7.594   5.939  42.197  1.0   0.00      PROT C \n",
            "ATOM    453  O   LYS A 240       6.561   5.555  42.749  1.0   0.00      PROT O \n",
            "ATOM    454  CB  LYS A 240       7.880   4.536  40.152  1.0   0.00      PROT C \n",
            "ATOM    455  CG  LYS A 240       7.755   4.384  38.640  1.0   0.00      PROT C \n",
            "ATOM    456  CD  LYS A 240       6.334   4.026  38.223  1.0   0.00      PROT C \n",
            "ATOM    457  CE  LYS A 240       5.951   2.624  38.685  1.0   0.00      PROT C \n",
            "ATOM    458  NZ  LYS A 240       4.525   2.295  38.385  1.0   0.00      PROT N \n",
            "ATOM    459  H   LYS A 240       9.476   6.411  39.578  1.0   0.00      PROT H \n",
            "ATOM    460  HA  LYS A 240       6.809   6.434  40.255  1.0   0.00      PROT H \n",
            "ATOM    461  HB3 LYS A 240       7.121   3.927  40.642  1.0   0.00      PROT H \n",
            "ATOM    462  HB2 LYS A 240       8.870   4.208  40.470  1.0   0.00      PROT H \n",
            "ATOM    463  HG3 LYS A 240       8.437   3.598  38.318  1.0   0.00      PROT H \n",
            "ATOM    464  HG2 LYS A 240       8.043   5.327  38.176  1.0   0.00      PROT H \n",
            "ATOM    465  HD2 LYS A 240       5.636   4.737  38.665  1.0   0.00      PROT H \n",
            "ATOM    466  HD3 LYS A 240       6.253   4.065  37.137  1.0   0.00      PROT H \n",
            "ATOM    467  HE2 LYS A 240       6.579   1.891  38.179  1.0   0.00      PROT H \n",
            "ATOM    468  HE3 LYS A 240       6.095   2.544  39.762  1.0   0.00      PROT H \n",
            "ATOM    469  HZ1 LYS A 240       4.486   1.488  37.779  1.0   0.00      PROT H \n",
            "ATOM    470  HZ2 LYS A 240       4.038   2.094  39.246  1.0   0.00      PROT H \n",
            "ATOM    471  HZ3 LYS A 240       3.738   2.890  37.899  1.0   0.00      PROT H \n",
            "ATOM    472  N   GLN A 241       8.658   6.360  42.872  1.0   0.00      PROT N \n",
            "ATOM    473  CA  GLN A 241       8.685   6.356  44.327  1.0   0.00      PROT C \n",
            "ATOM    474  C   GLN A 241       8.562   7.729  44.983  1.0   0.00      PROT C \n",
            "ATOM    475  O   GLN A 241       7.949   7.863  46.042  1.0   0.00      PROT O \n",
            "ATOM    476  CB  GLN A 241       9.968   5.688  44.810  1.0   0.00      PROT C \n",
            "ATOM    477  CG  GLN A 241      10.131   4.262  44.347  1.0   0.00      PROT C \n",
            "ATOM    478  CD  GLN A 241      11.394   3.630  44.887  1.0   0.00      PROT C \n",
            "ATOM    479  OE1 GLN A 241      11.632   2.438  44.703  1.0   0.00      PROT O \n",
            "ATOM    480  NE2 GLN A 241      12.214   4.429  45.557  1.0   0.00      PROT N \n",
            "ATOM    481  H   GLN A 241       9.472   6.692  42.375  1.0   0.00      PROT H \n",
            "ATOM    482  HA  GLN A 241       7.786   5.793  44.578  1.0   0.00      PROT H \n",
            "ATOM    483  HB3 GLN A 241       9.954   5.717  45.900  1.0   0.00      PROT H \n",
            "ATOM    484  HB2 GLN A 241      10.800   6.287  44.441  1.0   0.00      PROT H \n",
            "ATOM    485  HG2 GLN A 241       9.282   3.670  44.689  1.0   0.00      PROT H \n",
            "ATOM    486  HG3 GLN A 241      10.177   4.237  43.258  1.0   0.00      PROT H \n",
            "ATOM    487 HE21 GLN A 241      11.976   5.412  45.685  1.0   0.00      PROT H \n",
            "ATOM    488 HE22 GLN A 241      13.083   4.060  45.944  1.0   0.00      PROT H \n",
            "ATOM    489  N   LYS A 242       9.148   8.746  44.365  1.0   0.00      PROT N \n",
            "ATOM    490  CA  LYS A 242       9.097  10.091  44.927  1.0   0.00      PROT C \n",
            "ATOM    491  C   LYS A 242       7.823  10.860  44.560  1.0   0.00      PROT C \n",
            "ATOM    492  O   LYS A 242       7.497  11.878  45.178  1.0   0.00      PROT O \n",
            "ATOM    493  CB  LYS A 242      10.329  10.895  44.484  1.0   0.00      PROT C \n",
            "ATOM    494  CG  LYS A 242      10.453  12.248  45.173  1.0   0.00      PROT C \n",
            "ATOM    495  CD  LYS A 242      11.679  13.012  44.708  1.0   0.00      PROT C \n",
            "ATOM    496  CE  LYS A 242      11.791  14.349  45.427  1.0   0.00      PROT C \n",
            "ATOM    497  NZ  LYS A 242      13.021  15.102  45.043  1.0   0.00      PROT N \n",
            "ATOM    498  H   LYS A 242       9.637   8.594  43.494  1.0   0.00      PROT H \n",
            "ATOM    499  HA  LYS A 242       9.090   9.912  46.002  1.0   0.00      PROT H \n",
            "ATOM    500  HB3 LYS A 242      10.252  11.042  43.407  1.0   0.00      PROT H \n",
            "ATOM    501  HB2 LYS A 242      11.209  10.292  44.707  1.0   0.00      PROT H \n",
            "ATOM    502  HG3 LYS A 242      10.517  12.078  46.248  1.0   0.00      PROT H \n",
            "ATOM    503  HG2 LYS A 242       9.557  12.827  44.949  1.0   0.00      PROT H \n",
            "ATOM    504  HD2 LYS A 242      12.574  12.426  44.917  1.0   0.00      PROT H \n",
            "ATOM    505  HD3 LYS A 242      11.608  13.196  43.636  1.0   0.00      PROT H \n",
            "ATOM    506  HE2 LYS A 242      10.928  14.967  45.181  1.0   0.00      PROT H \n",
            "ATOM    507  HE3 LYS A 242      11.822  14.181  46.504  1.0   0.00      PROT H \n",
            "ATOM    508  HZ1 LYS A 242      12.760  15.989  44.638  1.0   0.00      PROT H \n",
            "ATOM    509  HZ2 LYS A 242      13.588  15.261  45.863  1.0   0.00      PROT H \n",
            "ATOM    510  N   ARG A 243       7.093  10.365  43.569  1.0   0.00      PROT N \n",
            "ATOM    511  CA  ARG A 243       5.879  11.036  43.127  1.0   0.00      PROT C \n",
            "ATOM    512  C   ARG A 243       4.717  11.055  44.126  1.0   0.00      PROT C \n",
            "ATOM    513  O   ARG A 243       4.327  10.021  44.678  1.0   0.00      PROT O \n",
            "ATOM    514  CB  ARG A 243       5.384  10.409  41.823  1.0   0.00      PROT C \n",
            "ATOM    515  CG  ARG A 243       5.009   8.958  41.987  1.0   0.00      PROT C \n",
            "ATOM    516  CD  ARG A 243       4.280   8.420  40.795  1.0   0.00      PROT C \n",
            "ATOM    517  NE  ARG A 243       4.035   6.991  40.942  1.0   0.00      PROT N \n",
            "ATOM    518  CZ  ARG A 243       3.287   6.277  40.110  1.0   0.00      PROT C \n",
            "ATOM    519  NH1 ARG A 243       2.708   6.866  39.072  1.0   0.00      PROT N \n",
            "ATOM    520  NH2 ARG A 243       3.124   4.978  40.308  1.0   0.00      PROT N \n",
            "ATOM    521  H   ARG A 243       7.377   9.511  43.110  1.0   0.00      PROT H \n",
            "ATOM    522  HA  ARG A 243       6.234  12.061  43.017  1.0   0.00      PROT H \n",
            "ATOM    523  HB3 ARG A 243       6.181  10.497  41.084  1.0   0.00      PROT H \n",
            "ATOM    524  HB2 ARG A 243       4.514  10.974  41.488  1.0   0.00      PROT H \n",
            "ATOM    525  HG3 ARG A 243       4.376   8.871  42.870  1.0   0.00      PROT H \n",
            "ATOM    526  HG2 ARG A 243       5.926   8.388  42.140  1.0   0.00      PROT H \n",
            "ATOM    527  HD2 ARG A 243       3.323   8.932  40.692  1.0   0.00      PROT H \n",
            "ATOM    528  HD3 ARG A 243       4.878   8.584  39.899  1.0   0.00      PROT H \n",
            "ATOM    529  HE  ARG A 243       4.475   6.545  41.735  1.0   0.00      PROT H \n",
            "ATOM    530 HH12 ARG A 243       2.138   6.318  38.427  1.0   0.00      PROT H \n",
            "ATOM    531 HH11 ARG A 243       2.833   7.867  38.918  1.0   0.00      PROT H \n",
            "ATOM    532  HH2 ARG A 243       2.554   4.431  39.663  1.0   0.00      PROT H \n",
            "ATOM    533  N   LYS A 244       4.175  12.249  44.356  1.0   0.00      PROT N \n",
            "ATOM    534  CA  LYS A 244       3.011  12.426  45.216  1.0   0.00      PROT C \n",
            "ATOM    535  C   LYS A 244       1.866  12.623  44.235  1.0   0.00      PROT C \n",
            "ATOM    536  O   LYS A 244       2.045  13.258  43.193  1.0   0.00      PROT O \n",
            "ATOM    537  CB  LYS A 244       3.150  13.672  46.095  1.0   0.00      PROT C \n",
            "ATOM    538  CG  LYS A 244       3.682  13.398  47.498  1.0   0.00      PROT C \n",
            "ATOM    539  CD  LYS A 244       5.093  12.808  47.463  1.0   0.00      PROT C \n",
            "ATOM    540  CE  LYS A 244       5.558  12.366  48.847  1.0   0.00      PROT C \n",
            "ATOM    541  NZ  LYS A 244       5.481  13.472  49.854  1.0   0.00      PROT N \n",
            "ATOM    542  H   LYS A 244       4.570  13.074  43.928  1.0   0.00      PROT H \n",
            "ATOM    543  HA  LYS A 244       2.903  11.562  45.872  1.0   0.00      PROT H \n",
            "ATOM    544  HB3 LYS A 244       2.161  14.124  46.163  1.0   0.00      PROT H \n",
            "ATOM    545  HB2 LYS A 244       3.826  14.353  45.577  1.0   0.00      PROT H \n",
            "ATOM    546  HG2 LYS A 244       3.714  14.330  48.063  1.0   0.00      PROT H \n",
            "ATOM    547  HG3 LYS A 244       3.026  12.689  48.003  1.0   0.00      PROT H \n",
            "ATOM    548  HD3 LYS A 244       5.772  13.569  47.077  1.0   0.00      PROT H \n",
            "ATOM    549  HD2 LYS A 244       5.088  11.950  46.790  1.0   0.00      PROT H \n",
            "ATOM    550  HE2 LYS A 244       4.930  11.547  49.197  1.0   0.00      PROT H \n",
            "ATOM    551  HE3 LYS A 244       6.594  12.031  48.793  1.0   0.00      PROT H \n",
            "ATOM    552  HZ1 LYS A 244       4.860  13.203  50.603  1.0   0.00      PROT H \n",
            "ATOM    553  HZ2 LYS A 244       6.402  13.651  50.228  1.0   0.00      PROT H \n",
            "ATOM    554  N   PHE A 245       0.699  12.069  44.535  1.0   0.00      PROT N \n",
            "ATOM    555  CA  PHE A 245      -0.427  12.240  43.628  1.0   0.00      PROT C \n",
            "ATOM    556  C   PHE A 245      -1.163  13.536  43.927  1.0   0.00      PROT C \n",
            "ATOM    557  O   PHE A 245      -1.416  13.863  45.087  1.0   0.00      PROT O \n",
            "ATOM    558  CB  PHE A 245      -1.392  11.056  43.719  1.0   0.00      PROT C \n",
            "ATOM    559  CG  PHE A 245      -0.828   9.781  43.178  1.0   0.00      PROT C \n",
            "ATOM    560  CD1 PHE A 245      -0.002   8.979  43.961  1.0   0.00      PROT C \n",
            "ATOM    561  CD2 PHE A 245      -1.078   9.404  41.863  1.0   0.00      PROT C \n",
            "ATOM    562  CE1 PHE A 245       0.570   7.816  43.440  1.0   0.00      PROT C \n",
            "ATOM    563  CE2 PHE A 245      -0.511   8.244  41.327  1.0   0.00      PROT C \n",
            "ATOM    564  CZ  PHE A 245       0.315   7.448  42.116  1.0   0.00      PROT C \n",
            "ATOM    565  H   PHE A 245       0.598  11.534  45.386  1.0   0.00      PROT H \n",
            "ATOM    566  HA  PHE A 245       0.014  12.285  42.632  1.0   0.00      PROT H \n",
            "ATOM    567  HB3 PHE A 245      -2.288  11.323  43.159  1.0   0.00      PROT H \n",
            "ATOM    568  HB2 PHE A 245      -1.646  10.923  44.771  1.0   0.00      PROT H \n",
            "ATOM    569  HD1 PHE A 245       0.206   9.256  44.995  1.0   0.00      PROT H \n",
            "ATOM    570  HD2 PHE A 245      -1.724  10.016  41.234  1.0   0.00      PROT H \n",
            "ATOM    571  HE1 PHE A 245       1.214   7.198  44.066  1.0   0.00      PROT H \n",
            "ATOM    572  HE2 PHE A 245      -0.714   7.962  40.294  1.0   0.00      PROT H \n",
            "ATOM    573  HZ  PHE A 245       0.759   6.543  41.700  1.0   0.00      PROT H \n",
            "ATOM    574  N   LEU A 246      -1.462  14.291  42.876  1.0   0.00      PROT N \n",
            "ATOM    575  CA  LEU A 246      -2.192  15.541  43.027  1.0   0.00      PROT C \n",
            "ATOM    576  C   LEU A 246      -3.589  15.119  43.473  1.0   0.00      PROT C \n",
            "ATOM    577  O   LEU A 246      -4.239  14.323  42.799  1.0   0.00      PROT O \n",
            "ATOM    578  CB  LEU A 246      -2.263  16.282  41.686  1.0   0.00      PROT C \n",
            "ATOM    579  CG  LEU A 246      -2.807  17.713  41.687  1.0   0.00      PROT C \n",
            "ATOM    580  CD1 LEU A 246      -1.939  18.606  42.574  1.0   0.00      PROT C \n",
            "ATOM    581  CD2 LEU A 246      -2.832  18.239  40.261  1.0   0.00      PROT C \n",
            "ATOM    582  H   LEU A 246      -1.182  14.001  41.950  1.0   0.00      PROT H \n",
            "ATOM    583  HA  LEU A 246      -1.684  16.182  43.747  1.0   0.00      PROT H \n",
            "ATOM    584  HB3 LEU A 246      -2.887  15.656  41.048  1.0   0.00      PROT H \n",
            "ATOM    585  HB2 LEU A 246      -1.241  16.282  41.307  1.0   0.00      PROT H \n",
            "ATOM    586  HG  LEU A 246      -3.823  17.722  42.082  1.0   0.00      PROT H \n",
            "ATOM    587 HD11 LEU A 246      -1.934  18.213  43.591  1.0   0.00      PROT H \n",
            "ATOM    588 HD12 LEU A 246      -0.921  18.624  42.186  1.0   0.00      PROT H \n",
            "ATOM    589 HD13 LEU A 246      -2.344  19.618  42.578  1.0   0.00      PROT H \n",
            "ATOM    590 HD21 LEU A 246      -3.455  17.590  39.645  1.0   0.00      PROT H \n",
            "ATOM    591 HD22 LEU A 246      -3.241  19.249  40.253  1.0   0.00      PROT H \n",
            "ATOM    592 HD23 LEU A 246      -1.818  18.255  39.862  1.0   0.00      PROT H \n",
            "ATOM    593  N   PRO A 247      -4.064  15.630  44.624  1.0   0.00      PROT N \n",
            "ATOM    594  CA  PRO A 247      -5.400  15.264  45.115  1.0   0.00      PROT C \n",
            "ATOM    595  C   PRO A 247      -6.465  15.243  44.014  1.0   0.00      PROT C \n",
            "ATOM    596  O   PRO A 247      -6.533  16.140  43.183  1.0   0.00      PROT O \n",
            "ATOM    597  CB  PRO A 247      -5.669  16.318  46.184  1.0   0.00      PROT C \n",
            "ATOM    598  CG  PRO A 247      -4.301  16.546  46.748  1.0   0.00      PROT C \n",
            "ATOM    599  CD  PRO A 247      -3.437  16.632  45.509  1.0   0.00      PROT C \n",
            "ATOM    600  HA  PRO A 247      -5.401  14.243  45.497  1.0   0.00      PROT H \n",
            "ATOM    601  HB3 PRO A 247      -6.333  15.966  46.973  1.0   0.00      PROT H \n",
            "ATOM    602  HB2 PRO A 247      -6.054  17.252  45.775  1.0   0.00      PROT H \n",
            "ATOM    603  HG2 PRO A 247      -4.296  17.469  47.329  1.0   0.00      PROT H \n",
            "ATOM    604  HG3 PRO A 247      -4.030  15.710  47.392  1.0   0.00      PROT H \n",
            "ATOM    605  HD2 PRO A 247      -3.480  17.644  45.105  1.0   0.00      PROT H \n",
            "ATOM    606  HD3 PRO A 247      -2.407  16.387  45.767  1.0   0.00      PROT H \n",
            "ATOM    607  N   GLU A 248      -7.286  14.203  44.016  1.0   0.00      PROT N \n",
            "ATOM    608  CA  GLU A 248      -8.337  14.041  43.017  1.0   0.00      PROT C \n",
            "ATOM    609  C   GLU A 248      -9.298  15.237  42.967  1.0   0.00      PROT C \n",
            "ATOM    610  O   GLU A 248      -9.863  15.557  41.915  1.0   0.00      PROT O \n",
            "ATOM    611  CB  GLU A 248      -9.118  12.762  43.307  1.0   0.00      PROT C \n",
            "ATOM    612  CG  GLU A 248     -10.142  12.414  42.255  1.0   0.00      PROT C \n",
            "ATOM    613  CD  GLU A 248     -10.950  11.171  42.614  1.0   0.00      PROT C \n",
            "ATOM    614  OE1 GLU A 248     -11.668  10.662  41.724  1.0   0.00      PROT O \n",
            "ATOM    615  OE2 GLU A 248     -10.875  10.708  43.780  1.0   0.00      PROT O \n",
            "ATOM    616  H   GLU A 248      -7.195  13.490  44.725  1.0   0.00      PROT H \n",
            "ATOM    617  HA  GLU A 248      -7.807  13.980  42.067  1.0   0.00      PROT H \n",
            "ATOM    618  HB3 GLU A 248      -9.616  12.901  44.267  1.0   0.00      PROT H \n",
            "ATOM    619  HB2 GLU A 248      -8.391  11.954  43.388  1.0   0.00      PROT H \n",
            "ATOM    620  HG2 GLU A 248     -10.838  13.244  42.137  1.0   0.00      PROT H \n",
            "ATOM    621  HG3 GLU A 248      -9.639  12.225  41.307  1.0   0.00      PROT H \n",
            "ATOM    622  HE2 GLU A 248     -11.461   9.902  43.866  1.0   0.00      PROT H \n",
            "ATOM    623  N   ASP A 249      -9.477  15.888  44.112  1.0   0.00      PROT N \n",
            "ATOM    624  CA  ASP A 249     -10.348  17.046  44.220  1.0   0.00      PROT C \n",
            "ATOM    625  C   ASP A 249      -9.950  18.139  43.227  1.0   0.00      PROT C \n",
            "ATOM    626  O   ASP A 249     -10.808  18.758  42.592  1.0   0.00      PROT O \n",
            "ATOM    627  CB  ASP A 249     -10.284  17.613  45.634  1.0   0.00      PROT C \n",
            "ATOM    628  CG  ASP A 249     -11.259  18.740  45.846  1.0   0.00      PROT C \n",
            "ATOM    629  OD1 ASP A 249     -12.475  18.473  45.880  1.0   0.00      PROT O \n",
            "ATOM    630  OD2 ASP A 249     -10.814  19.895  45.971  1.0   0.00      PROT O \n",
            "ATOM    631  H   ASP A 249      -8.999  15.582  44.947  1.0   0.00      PROT H \n",
            "ATOM    632  HA  ASP A 249     -11.344  16.671  43.986  1.0   0.00      PROT H \n",
            "ATOM    633  HB3 ASP A 249      -9.270  17.973  45.806  1.0   0.00      PROT H \n",
            "ATOM    634  HB2 ASP A 249     -10.506  16.805  46.331  1.0   0.00      PROT H \n",
            "ATOM    635  HD1 ASP A 249     -13.005  19.310  46.019  1.0   0.00      PROT H \n",
            "ATOM    636  N   ILE A 250      -8.647  18.379  43.112  1.0   0.00      PROT N \n",
            "ATOM    637  CA  ILE A 250      -8.118  19.392  42.206  1.0   0.00      PROT C \n",
            "ATOM    638  C   ILE A 250      -8.372  19.007  40.754  1.0   0.00      PROT C \n",
            "ATOM    639  O   ILE A 250      -7.910  17.970  40.289  1.0   0.00      PROT O \n",
            "ATOM    640  CB  ILE A 250      -6.600  19.585  42.422  1.0   0.00      PROT C \n",
            "ATOM    641  CG1 ILE A 250      -6.345  20.051  43.855  1.0   0.00      PROT C \n",
            "ATOM    642  CG2 ILE A 250      -6.043  20.608  41.428  1.0   0.00      PROT C \n",
            "ATOM    643  CD1 ILE A 250      -4.905  20.343  44.143  1.0   0.00      PROT C \n",
            "ATOM    644  H   ILE A 250      -7.986  17.851  43.663  1.0   0.00      PROT H \n",
            "ATOM    645  HA  ILE A 250      -8.671  20.298  42.454  1.0   0.00      PROT H \n",
            "ATOM    646  HB  ILE A 250      -6.084  18.639  42.259  1.0   0.00      PROT H \n",
            "ATOM    647 HG13 ILE A 250      -6.696  19.263  44.521  1.0   0.00      PROT H \n",
            "ATOM    648 HG12 ILE A 250      -6.937  20.952  44.014  1.0   0.00      PROT H \n",
            "ATOM    649 HG21 ILE A 250      -6.231  20.265  40.410  1.0   0.00      PROT H \n",
            "ATOM    650 HG22 ILE A 250      -6.532  21.569  41.584  1.0   0.00      PROT H \n",
            "ATOM    651 HG23 ILE A 250      -4.970  20.718  41.582  1.0   0.00      PROT H \n",
            "ATOM    652 HD11 ILE A 250      -4.800  20.668  45.178  1.0   0.00      PROT H \n",
            "ATOM    653 HD12 ILE A 250      -4.313  19.442  43.984  1.0   0.00      PROT H \n",
            "ATOM    654 HD13 ILE A 250      -4.554  21.131  43.477  1.0   0.00      PROT H \n",
            "ATOM    655  N   GLY A 251      -9.115  19.854  40.046  1.0   0.00      PROT N \n",
            "ATOM    656  CA  GLY A 251      -9.428  19.588  38.655  1.0   0.00      PROT C \n",
            "ATOM    657  C   GLY A 251     -10.826  19.012  38.500  1.0   0.00      PROT C \n",
            "ATOM    658  O   GLY A 251     -11.323  18.849  37.387  1.0   0.00      PROT O \n",
            "ATOM    659  H   GLY A 251      -9.470  20.696  40.476  1.0   0.00      PROT H \n",
            "ATOM    660  HA2 GLY A 251      -8.698  18.878  38.266  1.0   0.00      PROT H \n",
            "ATOM    661  HA3 GLY A 251      -9.357  20.523  38.100  1.0   0.00      PROT H \n",
            "ATOM    662  N   GLN A 252     -11.467  18.705  39.617  1.0   0.00      PROT N \n",
            "ATOM    663  CA  GLN A 252     -12.812  18.143  39.591  1.0   0.00      PROT C \n",
            "ATOM    664  C   GLN A 252     -13.875  19.222  39.416  1.0   0.00      PROT C \n",
            "ATOM    665  O   GLN A 252     -13.949  20.160  40.201  1.0   0.00      PROT O \n",
            "ATOM    666  CB  GLN A 252     -13.077  17.389  40.889  1.0   0.00      PROT C \n",
            "ATOM    667  CG  GLN A 252     -14.253  16.468  40.826  1.0   0.00      PROT C \n",
            "ATOM    668  CD  GLN A 252     -14.390  15.665  42.085  1.0   0.00      PROT C \n",
            "ATOM    669  OE1 GLN A 252     -13.394  15.194  42.639  1.0   0.00      PROT O \n",
            "ATOM    670  NE2 GLN A 252     -15.623  15.489  42.549  1.0   0.00      PROT N \n",
            "ATOM    671  H   GLN A 252     -11.025  18.859  40.512  1.0   0.00      PROT H \n",
            "ATOM    672  HA  GLN A 252     -12.825  17.477  38.728  1.0   0.00      PROT H \n",
            "ATOM    673  HB3 GLN A 252     -13.241  18.135  41.667  1.0   0.00      PROT H \n",
            "ATOM    674  HB2 GLN A 252     -12.179  16.816  41.118  1.0   0.00      PROT H \n",
            "ATOM    675  HG2 GLN A 252     -15.164  17.051  40.688  1.0   0.00      PROT H \n",
            "ATOM    676  HG3 GLN A 252     -14.131  15.780  39.989  1.0   0.00      PROT H \n",
            "ATOM    677 HE21 GLN A 252     -16.417  15.898  42.056  1.0   0.00      PROT H \n",
            "ATOM    678 HE22 GLN A 252     -15.776  14.945  43.398  1.0   0.00      PROT H \n",
            "ATOM    679  N   ALA A 253     -14.701  19.079  38.389  1.0   0.00      PROT N \n",
            "ATOM    680  CA  ALA A 253     -15.765  20.041  38.127  1.0   0.00      PROT C \n",
            "ATOM    681  C   ALA A 253     -16.946  19.793  39.065  1.0   0.00      PROT C \n",
            "ATOM    682  O   ALA A 253     -17.471  18.684  39.135  1.0   0.00      PROT O \n",
            "ATOM    683  CB  ALA A 253     -16.221  19.935  36.677  1.0   0.00      PROT C \n",
            "ATOM    684  H   ALA A 253     -14.601  18.290  37.767  1.0   0.00      PROT H \n",
            "ATOM    685  HA  ALA A 253     -15.336  21.026  38.314  1.0   0.00      PROT H \n",
            "ATOM    686  HB1 ALA A 253     -15.373  20.113  36.016  1.0   0.00      PROT H \n",
            "ATOM    687  HB2 ALA A 253     -16.621  18.938  36.494  1.0   0.00      PROT H \n",
            "ATOM    688  HB3 ALA A 253     -16.995  20.678  36.484  1.0   0.00      PROT H \n",
            "ATOM    689  N   PRO A 254     -17.368  20.826  39.812  1.0   0.00      PROT N \n",
            "ATOM    690  CA  PRO A 254     -18.498  20.636  40.725  1.0   0.00      PROT C \n",
            "ATOM    691  C   PRO A 254     -19.810  20.401  39.982  1.0   0.00      PROT C \n",
            "ATOM    692  O   PRO A 254     -20.596  19.538  40.358  1.0   0.00      PROT O \n",
            "ATOM    693  CB  PRO A 254     -18.502  21.929  41.549  1.0   0.00      PROT C \n",
            "ATOM    694  CG  PRO A 254     -17.901  22.946  40.608  1.0   0.00      PROT C \n",
            "ATOM    695  CD  PRO A 254     -16.781  22.172  39.963  1.0   0.00      PROT C \n",
            "ATOM    696  HA  PRO A 254     -18.364  19.736  41.325  1.0   0.00      PROT H \n",
            "ATOM    697  HB3 PRO A 254     -17.885  21.853  42.444  1.0   0.00      PROT H \n",
            "ATOM    698  HB2 PRO A 254     -19.509  22.241  41.827  1.0   0.00      PROT H \n",
            "ATOM    699  HG3 PRO A 254     -17.492  23.813  41.127  1.0   0.00      PROT H \n",
            "ATOM    700  HG2 PRO A 254     -18.601  23.279  39.842  1.0   0.00      PROT H \n",
            "ATOM    701  HD2 PRO A 254     -16.525  22.631  39.008  1.0   0.00      PROT H \n",
            "ATOM    702  HD3 PRO A 254     -15.909  22.182  40.617  1.0   0.00      PROT H \n",
            "ATOM    703  N   ILE A 255     -20.045  21.171  38.927  1.0   0.00      PROT N \n",
            "ATOM    704  CA  ILE A 255     -21.266  21.027  38.154  1.0   0.00      PROT C \n",
            "ATOM    705  C   ILE A 255     -20.953  20.288  36.870  1.0   0.00      PROT C \n",
            "ATOM    706  O   ILE A 255     -20.333  20.822  35.946  1.0   0.00      PROT O \n",
            "ATOM    707  CB  ILE A 255     -21.885  22.382  37.825  1.0   0.00      PROT C \n",
            "ATOM    708  CG1 ILE A 255     -22.077  23.198  39.115  1.0   0.00      PROT C \n",
            "ATOM    709  CG2 ILE A 255     -23.187  22.175  37.095  1.0   0.00      PROT C \n",
            "ATOM    710  CD1 ILE A 255     -22.894  22.505  40.206  1.0   0.00      PROT C \n",
            "ATOM    711  H   ILE A 255     -19.370  21.870  38.651  1.0   0.00      PROT H \n",
            "ATOM    712  HA  ILE A 255     -21.951  20.458  38.783  1.0   0.00      PROT H \n",
            "ATOM    713  HB  ILE A 255     -21.229  22.941  37.158  1.0   0.00      PROT H \n",
            "ATOM    714 HG13 ILE A 255     -22.571  24.126  38.826  1.0   0.00      PROT H \n",
            "ATOM    715 HG12 ILE A 255     -21.079  23.420  39.494  1.0   0.00      PROT H \n",
            "ATOM    716 HG21 ILE A 255     -23.008  21.595  36.190  1.0   0.00      PROT H \n",
            "ATOM    717 HG22 ILE A 255     -23.883  21.638  37.739  1.0   0.00      PROT H \n",
            "ATOM    718 HG23 ILE A 255     -23.612  23.143  36.828  1.0   0.00      PROT H \n",
            "ATOM    719 HD11 ILE A 255     -22.972  23.161  41.073  1.0   0.00      PROT H \n",
            "ATOM    720 HD12 ILE A 255     -23.892  22.283  39.827  1.0   0.00      PROT H \n",
            "ATOM    721 HD13 ILE A 255     -22.400  21.577  40.495  1.0   0.00      PROT H \n",
            "ATOM    722  N   VAL A 256     -21.392  19.037  36.846  1.0   0.00      PROT N \n",
            "ATOM    723  CA  VAL A 256     -21.183  18.120  35.742  1.0   0.00      PROT C \n",
            "ATOM    724  C   VAL A 256     -21.789  18.537  34.401  1.0   0.00      PROT C \n",
            "ATOM    725  O   VAL A 256     -21.203  18.288  33.346  1.0   0.00      PROT O \n",
            "ATOM    726  CB  VAL A 256     -21.723  16.728  36.143  1.0   0.00      PROT C \n",
            "ATOM    727  CG1 VAL A 256     -21.777  15.804  34.951  1.0   0.00      PROT C \n",
            "ATOM    728  CG2 VAL A 256     -20.843  16.145  37.219  1.0   0.00      PROT C \n",
            "ATOM    729  H   VAL A 256     -21.909  18.668  37.631  1.0   0.00      PROT H \n",
            "ATOM    730  HA  VAL A 256     -20.102  18.166  35.611  1.0   0.00      PROT H \n",
            "ATOM    731  HB  VAL A 256     -22.739  16.814  36.528  1.0   0.00      PROT H \n",
            "ATOM    732 HG11 VAL A 256     -22.416  16.239  34.183  1.0   0.00      PROT H \n",
            "ATOM    733 HG12 VAL A 256     -20.772  15.666  34.552  1.0   0.00      PROT H \n",
            "ATOM    734 HG13 VAL A 256     -22.182  14.839  35.257  1.0   0.00      PROT H \n",
            "ATOM    735 HG21 VAL A 256     -20.814  16.823  38.072  1.0   0.00      PROT H \n",
            "ATOM    736 HG22 VAL A 256     -21.244  15.182  37.534  1.0   0.00      PROT H \n",
            "ATOM    737 HG23 VAL A 256     -19.834  16.009  36.829  1.0   0.00      PROT H \n",
            "ATOM    738  N   ASN A 257     -22.951  19.178  34.438  1.0   0.00      PROT N \n",
            "ATOM    739  CA  ASN A 257     -23.626  19.569  33.206  1.0   0.00      PROT C \n",
            "ATOM    740  C   ASN A 257     -23.318  20.959  32.668  1.0   0.00      PROT C \n",
            "ATOM    741  O   ASN A 257     -24.022  21.447  31.787  1.0   0.00      PROT O \n",
            "ATOM    742  CB  ASN A 257     -25.136  19.402  33.385  1.0   0.00      PROT C \n",
            "ATOM    743  CG  ASN A 257     -25.545  17.946  33.506  1.0   0.00      PROT C \n",
            "ATOM    744  OD1 ASN A 257     -25.441  17.179  32.546  1.0   0.00      PROT O \n",
            "ATOM    745  ND2 ASN A 257     -26.006  17.554  34.690  1.0   0.00      PROT N \n",
            "ATOM    746  H   ASN A 257     -23.377  19.400  35.326  1.0   0.00      PROT H \n",
            "ATOM    747  HA  ASN A 257     -23.190  18.880  32.483  1.0   0.00      PROT H \n",
            "ATOM    748  HB3 ASN A 257     -25.629  19.847  32.521  1.0   0.00      PROT H \n",
            "ATOM    749  HB2 ASN A 257     -25.430  19.938  34.287  1.0   0.00      PROT H \n",
            "ATOM    750 HD21 ASN A 257     -26.075  18.223  35.457  1.0   0.00      PROT H \n",
            "ATOM    751 HD22 ASN A 257     -26.290  16.585  34.831  1.0   0.00      PROT H \n",
            "ATOM    752  N   ALA A 258     -22.274  21.594  33.187  1.0   0.00      PROT N \n",
            "ATOM    753  CA  ALA A 258     -21.901  22.924  32.718  1.0   0.00      PROT C \n",
            "ATOM    754  C   ALA A 258     -20.845  22.766  31.636  1.0   0.00      PROT C \n",
            "ATOM    755  O   ALA A 258     -19.851  22.069  31.826  1.0   0.00      PROT O \n",
            "ATOM    756  CB  ALA A 258     -21.350  23.771  33.866  1.0   0.00      PROT C \n",
            "ATOM    757  H   ALA A 258     -21.728  21.157  33.915  1.0   0.00      PROT H \n",
            "ATOM    758  HA  ALA A 258     -22.804  23.393  32.327  1.0   0.00      PROT H \n",
            "ATOM    759  HB1 ALA A 258     -22.113  23.879  34.637  1.0   0.00      PROT H \n",
            "ATOM    760  HB2 ALA A 258     -20.473  23.282  34.290  1.0   0.00      PROT H \n",
            "ATOM    761  HB3 ALA A 258     -21.071  24.755  33.490  1.0   0.00      PROT H \n",
            "ATOM    762  N   PRO A 259     -21.054  23.406  30.478  1.0   0.00      PROT N \n",
            "ATOM    763  CA  PRO A 259     -20.121  23.341  29.355  1.0   0.00      PROT C \n",
            "ATOM    764  C   PRO A 259     -18.676  23.601  29.754  1.0   0.00      PROT C \n",
            "ATOM    765  O   PRO A 259     -18.371  24.574  30.436  1.0   0.00      PROT O \n",
            "ATOM    766  CB  PRO A 259     -20.656  24.408  28.410  1.0   0.00      PROT C \n",
            "ATOM    767  CG  PRO A 259     -22.123  24.297  28.616  1.0   0.00      PROT C \n",
            "ATOM    768  CD  PRO A 259     -22.242  24.202  30.120  1.0   0.00      PROT C \n",
            "ATOM    769  HA  PRO A 259     -20.104  22.334  28.938  1.0   0.00      PROT H \n",
            "ATOM    770  HB3 PRO A 259     -20.420  24.209  27.365  1.0   0.00      PROT H \n",
            "ATOM    771  HB2 PRO A 259     -20.327  25.414  28.672  1.0   0.00      PROT H \n",
            "ATOM    772  HG2 PRO A 259     -22.615  25.181  28.210  1.0   0.00      PROT H \n",
            "ATOM    773  HG3 PRO A 259     -22.495  23.408  28.107  1.0   0.00      PROT H \n",
            "ATOM    774  HD2 PRO A 259     -22.228  25.204  30.550  1.0   0.00      PROT H \n",
            "ATOM    775  HD3 PRO A 259     -23.178  23.708  30.380  1.0   0.00      PROT H \n",
            "ATOM    776  N   GLU A 260     -17.788  22.722  29.306  1.0   0.00      PROT N \n",
            "ATOM    777  CA  GLU A 260     -16.364  22.843  29.587  1.0   0.00      PROT C \n",
            "ATOM    778  C   GLU A 260     -15.667  23.815  28.624  1.0   0.00      PROT C \n",
            "ATOM    779  O   GLU A 260     -14.806  24.595  29.037  1.0   0.00      PROT O \n",
            "ATOM    780  CB  GLU A 260     -15.711  21.465  29.504  1.0   0.00      PROT C \n",
            "ATOM    781  CG  GLU A 260     -16.399  20.427  30.380  1.0   0.00      PROT C \n",
            "ATOM    782  CD  GLU A 260     -15.522  19.225  30.650  1.0   0.00      PROT C \n",
            "ATOM    783  OE1 GLU A 260     -15.304  18.418  29.719  1.0   0.00      PROT O \n",
            "ATOM    784  OE2 GLU A 260     -15.041  19.100  31.799  1.0   0.00      PROT O \n",
            "ATOM    785  H   GLU A 260     -18.092  21.936  28.750  1.0   0.00      PROT H \n",
            "ATOM    786  HA  GLU A 260     -16.308  23.245  30.599  1.0   0.00      PROT H \n",
            "ATOM    787  HB3 GLU A 260     -14.672  21.572  29.815  1.0   0.00      PROT H \n",
            "ATOM    788  HB2 GLU A 260     -15.747  21.144  28.463  1.0   0.00      PROT H \n",
            "ATOM    789  HG3 GLU A 260     -17.308  20.107  29.870  1.0   0.00      PROT H \n",
            "ATOM    790  HG2 GLU A 260     -16.661  20.905  31.324  1.0   0.00      PROT H \n",
            "ATOM    791  HE2 GLU A 260     -14.475  18.277  31.845  1.0   0.00      PROT H \n",
            "ATOM    792  N   GLY A 261     -16.055  23.771  27.351  1.0   0.00      PROT N \n",
            "ATOM    793  CA  GLY A 261     -15.464  24.646  26.356  1.0   0.00      PROT C \n",
            "ATOM    794  C   GLY A 261     -13.953  24.628  26.448  1.0   0.00      PROT C \n",
            "ATOM    795  O   GLY A 261     -13.348  23.557  26.545  1.0   0.00      PROT O \n",
            "ATOM    796  H   GLY A 261     -16.772  23.121  27.064  1.0   0.00      PROT H \n",
            "ATOM    797  HA2 GLY A 261     -15.826  25.660  26.523  1.0   0.00      PROT H \n",
            "ATOM    798  HA3 GLY A 261     -15.772  24.309  25.366  1.0   0.00      PROT H \n",
            "ATOM    799  N   GLY A 262     -13.340  25.809  26.419  1.0   0.00      PROT N \n",
            "ATOM    800  CA  GLY A 262     -11.894  25.891  26.533  1.0   0.00      PROT C \n",
            "ATOM    801  C   GLY A 262     -11.457  26.527  27.846  1.0   0.00      PROT C \n",
            "ATOM    802  O   GLY A 262     -10.337  27.028  27.957  1.0   0.00      PROT O \n",
            "ATOM    803  H   GLY A 262     -13.880  26.656  26.317  1.0   0.00      PROT H \n",
            "ATOM    804  HA2 GLY A 262     -11.521  26.486  25.699  1.0   0.00      PROT H \n",
            "ATOM    805  HA3 GLY A 262     -11.491  24.880  26.467  1.0   0.00      PROT H \n",
            "ATOM    806  N   LYS A 263     -12.345  26.496  28.841  1.0   0.00      PROT N \n",
            "ATOM    807  CA  LYS A 263     -12.093  27.074  30.164  1.0   0.00      PROT C \n",
            "ATOM    808  C   LYS A 263     -10.909  26.444  30.892  1.0   0.00      PROT C \n",
            "ATOM    809  O   LYS A 263     -10.412  25.396  30.493  1.0   0.00      PROT O \n",
            "ATOM    810  CB  LYS A 263     -13.356  26.954  31.009  1.0   0.00      PROT C \n",
            "ATOM    811  CG  LYS A 263     -14.554  27.633  30.367  1.0   0.00      PROT C \n",
            "ATOM    812  CD  LYS A 263     -15.871  27.172  30.974  1.0   0.00      PROT C \n",
            "ATOM    813  CE  LYS A 263     -17.044  27.875  30.305  1.0   0.00      PROT C \n",
            "ATOM    814  NZ  LYS A 263     -18.351  27.324  30.738  1.0   0.00      PROT N \n",
            "ATOM    815  H   LYS A 263     -13.244  26.058  28.702  1.0   0.00      PROT H \n",
            "ATOM    816  HA  LYS A 263     -11.834  28.114  29.968  1.0   0.00      PROT H \n",
            "ATOM    817  HB3 LYS A 263     -13.155  27.411  31.978  1.0   0.00      PROT H \n",
            "ATOM    818  HB2 LYS A 263     -13.569  25.894  31.145  1.0   0.00      PROT H \n",
            "ATOM    819  HG2 LYS A 263     -14.477  28.712  30.505  1.0   0.00      PROT H \n",
            "ATOM    820  HG3 LYS A 263     -14.574  27.403  29.302  1.0   0.00      PROT H \n",
            "ATOM    821  HD2 LYS A 263     -15.978  26.096  30.835  1.0   0.00      PROT H \n",
            "ATOM    822  HD3 LYS A 263     -15.881  27.404  32.039  1.0   0.00      PROT H \n",
            "ATOM    823  HE2 LYS A 263     -17.024  28.935  30.557  1.0   0.00      PROT H \n",
            "ATOM    824  HE3 LYS A 263     -16.969  27.757  29.224  1.0   0.00      PROT H \n",
            "ATOM    825  HZ1 LYS A 263     -18.898  28.055  31.171  1.0   0.00      PROT H \n",
            "ATOM    826  HZ2 LYS A 263     -18.847  26.963  29.935  1.0   0.00      PROT H \n",
            "ATOM    827  N   VAL A 264     -10.459  27.085  31.963  1.0   0.00      PROT N \n",
            "ATOM    828  CA  VAL A 264      -9.318  26.581  32.726  1.0   0.00      PROT C \n",
            "ATOM    829  C   VAL A 264      -9.587  26.533  34.233  1.0   0.00      PROT C \n",
            "ATOM    830  O   VAL A 264     -10.133  27.468  34.794  1.0   0.00      PROT O \n",
            "ATOM    831  CB  VAL A 264      -8.065  27.469  32.489  1.0   0.00      PROT C \n",
            "ATOM    832  CG1 VAL A 264      -6.869  26.901  33.240  1.0   0.00      PROT C \n",
            "ATOM    833  CG2 VAL A 264      -7.760  27.577  30.989  1.0   0.00      PROT C \n",
            "ATOM    834  H   VAL A 264     -10.906  27.939  32.266  1.0   0.00      PROT H \n",
            "ATOM    835  HA  VAL A 264      -9.200  25.567  32.344  1.0   0.00      PROT H \n",
            "ATOM    836  HB  VAL A 264      -8.250  28.473  32.870  1.0   0.00      PROT H \n",
            "ATOM    837 HG11 VAL A 264      -7.106  26.833  34.302  1.0   0.00      PROT H \n",
            "ATOM    838 HG12 VAL A 264      -6.636  25.908  32.855  1.0   0.00      PROT H \n",
            "ATOM    839 HG13 VAL A 264      -6.009  27.556  33.101  1.0   0.00      PROT H \n",
            "ATOM    840 HG21 VAL A 264      -8.627  27.985  30.470  1.0   0.00      PROT H \n",
            "ATOM    841 HG22 VAL A 264      -6.904  28.235  30.839  1.0   0.00      PROT H \n",
            "ATOM    842 HG23 VAL A 264      -7.532  26.588  30.592  1.0   0.00      PROT H \n",
            "ATOM    843  N   ASP A 265      -9.213  25.434  34.879  1.0   0.00      PROT N \n",
            "ATOM    844  CA  ASP A 265      -9.377  25.289  36.324  1.0   0.00      PROT C \n",
            "ATOM    845  C   ASP A 265      -8.261  26.118  36.948  1.0   0.00      PROT C \n",
            "ATOM    846  O   ASP A 265      -7.098  25.719  36.904  1.0   0.00      PROT O \n",
            "ATOM    847  CB  ASP A 265      -9.207  23.824  36.748  1.0   0.00      PROT C \n",
            "ATOM    848  CG  ASP A 265      -9.573  23.586  38.213  1.0   0.00      PROT C \n",
            "ATOM    849  OD1 ASP A 265      -9.070  24.303  39.106  1.0   0.00      PROT O \n",
            "ATOM    850  OD2 ASP A 265     -10.368  22.665  38.475  1.0   0.00      PROT O \n",
            "ATOM    851  H   ASP A 265      -8.801  24.665  34.370  1.0   0.00      PROT H \n",
            "ATOM    852  HA  ASP A 265     -10.376  25.627  36.598  1.0   0.00      PROT H \n",
            "ATOM    853  HB3 ASP A 265      -8.165  23.548  36.584  1.0   0.00      PROT H \n",
            "ATOM    854  HB2 ASP A 265      -9.848  23.216  36.110  1.0   0.00      PROT H \n",
            "ATOM    855  HD1 ASP A 265      -9.404  24.016  40.004  1.0   0.00      PROT H \n",
            "ATOM    856  N   LEU A 266      -8.602  27.261  37.532  1.0   0.00      PROT N \n",
            "ATOM    857  CA  LEU A 266      -7.583  28.123  38.127  1.0   0.00      PROT C \n",
            "ATOM    858  C   LEU A 266      -6.741  27.455  39.212  1.0   0.00      PROT C \n",
            "ATOM    859  O   LEU A 266      -5.531  27.677  39.296  1.0   0.00      PROT O \n",
            "ATOM    860  CB  LEU A 266      -8.226  29.395  38.686  1.0   0.00      PROT C \n",
            "ATOM    861  CG  LEU A 266      -8.895  30.290  37.648  1.0   0.00      PROT C \n",
            "ATOM    862  CD1 LEU A 266      -9.257  31.624  38.293  1.0   0.00      PROT C \n",
            "ATOM    863  CD2 LEU A 266      -7.955  30.495  36.461  1.0   0.00      PROT C \n",
            "ATOM    864  H   LEU A 266      -9.572  27.540  37.567  1.0   0.00      PROT H \n",
            "ATOM    865  HA  LEU A 266      -6.924  28.330  37.284  1.0   0.00      PROT H \n",
            "ATOM    866  HB3 LEU A 266      -7.434  29.950  39.188  1.0   0.00      PROT H \n",
            "ATOM    867  HB2 LEU A 266      -8.965  29.074  39.420  1.0   0.00      PROT H \n",
            "ATOM    868  HG  LEU A 266      -9.808  29.818  37.285  1.0   0.00      PROT H \n",
            "ATOM    869 HD11 LEU A 266      -9.927  31.450  39.135  1.0   0.00      PROT H \n",
            "ATOM    870 HD12 LEU A 266      -8.350  32.115  38.645  1.0   0.00      PROT H \n",
            "ATOM    871 HD13 LEU A 266      -9.753  32.260  37.559  1.0   0.00      PROT H \n",
            "ATOM    872 HD21 LEU A 266      -7.712  29.529  36.019  1.0   0.00      PROT H \n",
            "ATOM    873 HD22 LEU A 266      -8.443  31.123  35.716  1.0   0.00      PROT H \n",
            "ATOM    874 HD23 LEU A 266      -7.040  30.979  36.802  1.0   0.00      PROT H \n",
            "ATOM    875  N   GLU A 267      -7.388  26.646  40.044  1.0   0.00      PROT N \n",
            "ATOM    876  CA  GLU A 267      -6.691  25.955  41.119  1.0   0.00      PROT C \n",
            "ATOM    877  C   GLU A 267      -5.655  25.002  40.539  1.0   0.00      PROT C \n",
            "ATOM    878  O   GLU A 267      -4.489  25.017  40.947  1.0   0.00      PROT O \n",
            "ATOM    879  CB  GLU A 267      -7.671  25.171  41.985  1.0   0.00      PROT C \n",
            "ATOM    880  CG  GLU A 267      -7.050  24.643  43.252  1.0   0.00      PROT C \n",
            "ATOM    881  CD  GLU A 267      -7.902  23.587  43.920  1.0   0.00      PROT C \n",
            "ATOM    882  OE1 GLU A 267      -7.775  23.424  45.159  1.0   0.00      PROT O \n",
            "ATOM    883  OE2 GLU A 267      -8.686  22.911  43.211  1.0   0.00      PROT O \n",
            "ATOM    884  H   GLU A 267      -8.382  26.501  39.937  1.0   0.00      PROT H \n",
            "ATOM    885  HA  GLU A 267      -6.212  26.737  41.709  1.0   0.00      PROT H \n",
            "ATOM    886  HB3 GLU A 267      -8.045  24.338  41.389  1.0   0.00      PROT H \n",
            "ATOM    887  HB2 GLU A 267      -8.496  25.838  42.235  1.0   0.00      PROT H \n",
            "ATOM    888  HG2 GLU A 267      -6.082  24.197  43.024  1.0   0.00      PROT H \n",
            "ATOM    889  HG3 GLU A 267      -6.915  25.461  43.959  1.0   0.00      PROT H \n",
            "ATOM    890  HE1 GLU A 267      -8.381  22.693  45.472  1.0   0.00      PROT H \n",
            "ATOM    891  N   ALA A 268      -6.078  24.182  39.581  1.0   0.00      PROT N \n",
            "ATOM    892  CA  ALA A 268      -5.161  23.236  38.952  1.0   0.00      PROT C \n",
            "ATOM    893  C   ALA A 268      -4.044  24.002  38.251  1.0   0.00      PROT C \n",
            "ATOM    894  O   ALA A 268      -2.862  23.710  38.441  1.0   0.00      PROT O \n",
            "ATOM    895  CB  ALA A 268      -5.908  22.353  37.951  1.0   0.00      PROT C \n",
            "ATOM    896  H   ALA A 268      -7.042  24.210  39.282  1.0   0.00      PROT H \n",
            "ATOM    897  HA  ALA A 268      -4.753  22.618  39.752  1.0   0.00      PROT H \n",
            "ATOM    898  HB1 ALA A 268      -6.702  21.812  38.466  1.0   0.00      PROT H \n",
            "ATOM    899  HB2 ALA A 268      -6.341  22.977  37.169  1.0   0.00      PROT H \n",
            "ATOM    900  HB3 ALA A 268      -5.213  21.641  37.506  1.0   0.00      PROT H \n",
            "ATOM    901  N   PHE A 269      -4.431  24.987  37.445  1.0   0.00      PROT N \n",
            "ATOM    902  CA  PHE A 269      -3.476  25.805  36.713  1.0   0.00      PROT C \n",
            "ATOM    903  C   PHE A 269      -2.432  26.330  37.686  1.0   0.00      PROT C \n",
            "ATOM    904  O   PHE A 269      -1.241  26.386  37.376  1.0   0.00      PROT O \n",
            "ATOM    905  CB  PHE A 269      -4.198  26.977  36.042  1.0   0.00      PROT C \n",
            "ATOM    906  CG  PHE A 269      -3.285  27.928  35.311  1.0   0.00      PROT C \n",
            "ATOM    907  CD1 PHE A 269      -2.789  27.613  34.048  1.0   0.00      PROT C \n",
            "ATOM    908  CD2 PHE A 269      -2.931  29.150  35.881  1.0   0.00      PROT C \n",
            "ATOM    909  CE1 PHE A 269      -1.951  28.500  33.353  1.0   0.00      PROT C \n",
            "ATOM    910  CE2 PHE A 269      -2.093  30.046  35.203  1.0   0.00      PROT C \n",
            "ATOM    911  CZ  PHE A 269      -1.602  29.717  33.929  1.0   0.00      PROT C \n",
            "ATOM    912  H   PHE A 269      -5.414  25.186  37.327  1.0   0.00      PROT H \n",
            "ATOM    913  HA  PHE A 269      -3.015  25.165  35.960  1.0   0.00      PROT H \n",
            "ATOM    914  HB3 PHE A 269      -4.731  27.515  36.826  1.0   0.00      PROT H \n",
            "ATOM    915  HB2 PHE A 269      -4.916  26.551  35.341  1.0   0.00      PROT H \n",
            "ATOM    916  HD1 PHE A 269      -3.050  26.663  33.581  1.0   0.00      PROT H \n",
            "ATOM    917  HD2 PHE A 269      -3.307  29.420  36.868  1.0   0.00      PROT H \n",
            "ATOM    918  HE1 PHE A 269      -1.574  28.237  32.365  1.0   0.00      PROT H \n",
            "ATOM    919  HE2 PHE A 269      -1.824  30.996  35.665  1.0   0.00      PROT H \n",
            "ATOM    920  HZ  PHE A 269      -0.952  30.408  33.392  1.0   0.00      PROT H \n",
            "ATOM    921  N   SER A 270      -2.894  26.702  38.875  1.0   0.00      PROT N \n",
            "ATOM    922  CA  SER A 270      -2.015  27.249  39.902  1.0   0.00      PROT C \n",
            "ATOM    923  C   SER A 270      -0.883  26.291  40.311  1.0   0.00      PROT C \n",
            "ATOM    924  O   SER A 270       0.281  26.697  40.414  1.0   0.00      PROT O \n",
            "ATOM    925  CB  SER A 270      -2.839  27.636  41.122  1.0   0.00      PROT C \n",
            "ATOM    926  OG  SER A 270      -2.055  28.421  41.994  1.0   0.00      PROT O \n",
            "ATOM    927  H   SER A 270      -3.877  26.609  39.086  1.0   0.00      PROT H \n",
            "ATOM    928  HA  SER A 270      -1.568  28.124  39.431  1.0   0.00      PROT H \n",
            "ATOM    929  HB2 SER A 270      -3.711  28.208  40.805  1.0   0.00      PROT H \n",
            "ATOM    930  HB3 SER A 270      -3.165  26.735  41.641  1.0   0.00      PROT H \n",
            "ATOM    931  HG  SER A 270      -2.443  28.758  42.840  1.0   0.00      PROT H \n",
            "ATOM    932  N   HIS A 271      -1.221  25.027  40.549  1.0   0.00      PROT N \n",
            "ATOM    933  CA  HIS A 271      -0.206  24.055  40.928  1.0   0.00      PROT C \n",
            "ATOM    934  C   HIS A 271       0.763  23.785  39.783  1.0   0.00      PROT C \n",
            "ATOM    935  O   HIS A 271       1.963  23.625  40.007  1.0   0.00      PROT O \n",
            "ATOM    936  CB  HIS A 271      -0.844  22.739  41.365  1.0   0.00      PROT C \n",
            "ATOM    937  CG  HIS A 271      -1.591  22.827  42.660  1.0   0.00      PROT C \n",
            "ATOM    938  ND1 HIS A 271      -2.778  23.510  42.791  1.0   0.00      PROT N \n",
            "ATOM    939  CD2 HIS A 271      -1.326  22.292  43.876  1.0   0.00      PROT C \n",
            "ATOM    940  CE1 HIS A 271      -3.219  23.390  44.032  1.0   0.00      PROT C \n",
            "ATOM    941  NE2 HIS A 271      -2.357  22.656  44.708  1.0   0.00      PROT N \n",
            "ATOM    942  H   HIS A 271      -2.185  24.738  40.467  1.0   0.00      PROT H \n",
            "ATOM    943  HA  HIS A 271       0.318  24.525  41.760  1.0   0.00      PROT H \n",
            "ATOM    944  HB3 HIS A 271      -0.042  22.007  41.457  1.0   0.00      PROT H \n",
            "ATOM    945  HB2 HIS A 271      -1.526  22.432  40.572  1.0   0.00      PROT H \n",
            "ATOM    946  HD1 HIS A 271      -3.274  24.063  41.994  1.0   0.00      PROT H \n",
            "ATOM    947  HD2 HIS A 271      -0.422  21.701  44.026  1.0   0.00      PROT H \n",
            "ATOM    948  HE1 HIS A 271      -4.155  23.866  44.326  1.0   0.00      PROT H \n",
            "ATOM    949  N   PHE A 272       0.241  23.729  38.562  1.0   0.00      PROT N \n",
            "ATOM    950  CA  PHE A 272       1.066  23.486  37.382  1.0   0.00      PROT C \n",
            "ATOM    951  C   PHE A 272       2.112  24.575  37.156  1.0   0.00      PROT C \n",
            "ATOM    952  O   PHE A 272       3.262  24.273  36.874  1.0   0.00      PROT O \n",
            "ATOM    953  CB  PHE A 272       0.188  23.364  36.131  1.0   0.00      PROT C \n",
            "ATOM    954  CG  PHE A 272      -0.658  22.121  36.097  1.0   0.00      PROT C \n",
            "ATOM    955  CD1 PHE A 272      -1.982  22.178  35.664  1.0   0.00      PROT C \n",
            "ATOM    956  CD2 PHE A 272      -0.137  20.898  36.493  1.0   0.00      PROT C \n",
            "ATOM    957  CE1 PHE A 272      -2.771  21.031  35.632  1.0   0.00      PROT C \n",
            "ATOM    958  CE2 PHE A 272      -0.921  19.743  36.465  1.0   0.00      PROT C \n",
            "ATOM    959  CZ  PHE A 272      -2.237  19.813  36.033  1.0   0.00      PROT C \n",
            "ATOM    960  H   PHE A 272      -0.752  23.856  38.431  1.0   0.00      PROT H \n",
            "ATOM    961  HA  PHE A 272       1.574  22.550  37.612  1.0   0.00      PROT H \n",
            "ATOM    962  HB3 PHE A 272       0.854  23.374  35.268  1.0   0.00      PROT H \n",
            "ATOM    963  HB2 PHE A 272      -0.456  24.243  36.103  1.0   0.00      PROT H \n",
            "ATOM    964  HD1 PHE A 272      -2.398  23.135  35.348  1.0   0.00      PROT H \n",
            "ATOM    965  HD2 PHE A 272       0.897  20.830  36.832  1.0   0.00      PROT H \n",
            "ATOM    966  HE1 PHE A 272      -3.805  21.087  35.293  1.0   0.00      PROT H \n",
            "ATOM    967  HE2 PHE A 272      -0.501  18.788  36.781  1.0   0.00      PROT H \n",
            "ATOM    968  HZ  PHE A 272      -2.851  18.913  36.008  1.0   0.00      PROT H \n",
            "ATOM    969  N   THR A 273       1.719  25.840  37.261  1.0   0.00      PROT N \n",
            "ATOM    970  CA  THR A 273       2.678  26.922  37.060  1.0   0.00      PROT C \n",
            "ATOM    971  C   THR A 273       3.717  26.958  38.174  1.0   0.00      PROT C \n",
            "ATOM    972  O   THR A 273       4.787  27.543  38.001  1.0   0.00      PROT O \n",
            "ATOM    973  CB  THR A 273       2.012  28.333  36.978  1.0   0.00      PROT C \n",
            "ATOM    974  OG1 THR A 273       1.206  28.567  38.137  1.0   0.00      PROT O \n",
            "ATOM    975  CG2 THR A 273       1.171  28.456  35.734  1.0   0.00      PROT C \n",
            "ATOM    976  H   THR A 273       0.756  26.053  37.480  1.0   0.00      PROT H \n",
            "ATOM    977  HA  THR A 273       3.130  26.640  36.109  1.0   0.00      PROT H \n",
            "ATOM    978  HB  THR A 273       2.755  29.131  36.980  1.0   0.00      PROT H \n",
            "ATOM    979  HG1 THR A 273       1.611  28.523  39.039  1.0   0.00      PROT H \n",
            "ATOM    980 HG21 THR A 273       0.717  29.447  35.699  1.0   0.00      PROT H \n",
            "ATOM    981 HG22 THR A 273       1.799  28.312  34.855  1.0   0.00      PROT H \n",
            "ATOM    982 HG23 THR A 273       0.388  27.698  35.749  1.0   0.00      PROT H \n",
            "ATOM    983  N   LYS A 274       3.421  26.337  39.313  1.0   0.00      PROT N \n",
            "ATOM    984  CA  LYS A 274       4.384  26.345  40.399  1.0   0.00      PROT C \n",
            "ATOM    985  C   LYS A 274       5.521  25.362  40.175  1.0   0.00      PROT C \n",
            "ATOM    986  O   LYS A 274       6.555  25.462  40.832  1.0   0.00      PROT O \n",
            "ATOM    987  CB  LYS A 274       3.699  26.098  41.742  1.0   0.00      PROT C \n",
            "ATOM    988  CG  LYS A 274       2.788  27.257  42.134  1.0   0.00      PROT C \n",
            "ATOM    989  CD  LYS A 274       2.302  27.187  43.571  1.0   0.00      PROT C \n",
            "ATOM    990  CE  LYS A 274       1.395  28.377  43.894  1.0   0.00      PROT C \n",
            "ATOM    991  NZ  LYS A 274       1.071  28.449  45.354  1.0   0.00      PROT N \n",
            "ATOM    992  H   LYS A 274       2.533  25.866  39.416  1.0   0.00      PROT H \n",
            "ATOM    993  HA  LYS A 274       4.800  27.352  40.372  1.0   0.00      PROT H \n",
            "ATOM    994  HB3 LYS A 274       4.470  25.969  42.501  1.0   0.00      PROT H \n",
            "ATOM    995  HB2 LYS A 274       3.111  25.184  41.663  1.0   0.00      PROT H \n",
            "ATOM    996  HG3 LYS A 274       1.932  27.241  41.460  1.0   0.00      PROT H \n",
            "ATOM    997  HG2 LYS A 274       3.351  28.179  41.985  1.0   0.00      PROT H \n",
            "ATOM    998  HD2 LYS A 274       1.740  26.265  43.720  1.0   0.00      PROT H \n",
            "ATOM    999  HD3 LYS A 274       3.158  27.203  44.245  1.0   0.00      PROT H \n",
            "ATOM   1000  HE2 LYS A 274       1.895  29.303  43.608  1.0   0.00      PROT H \n",
            "ATOM   1001  HE3 LYS A 274       0.461  28.283  43.340  1.0   0.00      PROT H \n",
            "ATOM   1002  HZ1 LYS A 274       1.400  29.327  45.729  1.0   0.00      PROT H \n",
            "ATOM   1003  HZ2 LYS A 274       0.071  28.382  45.481  1.0   0.00      PROT H \n",
            "ATOM   1004  N   ILE A 275       5.350  24.426  39.245  1.0   0.00      PROT N \n",
            "ATOM   1005  CA  ILE A 275       6.411  23.466  38.945  1.0   0.00      PROT C \n",
            "ATOM   1006  C   ILE A 275       6.878  23.557  37.495  1.0   0.00      PROT C \n",
            "ATOM   1007  O   ILE A 275       7.779  22.836  37.082  1.0   0.00      PROT O \n",
            "ATOM   1008  CB  ILE A 275       5.980  22.010  39.196  1.0   0.00      PROT C \n",
            "ATOM   1009  CG1 ILE A 275       4.898  21.604  38.202  1.0   0.00      PROT C \n",
            "ATOM   1010  CG2 ILE A 275       5.486  21.853  40.612  1.0   0.00      PROT C \n",
            "ATOM   1011  CD1 ILE A 275       4.440  20.176  38.368  1.0   0.00      PROT C \n",
            "ATOM   1012  H   ILE A 275       4.479  24.373  38.736  1.0   0.00      PROT H \n",
            "ATOM   1013  HA  ILE A 275       7.197  23.776  39.634  1.0   0.00      PROT H \n",
            "ATOM   1014  HB  ILE A 275       6.834  21.344  39.069  1.0   0.00      PROT H \n",
            "ATOM   1015 HG13 ILE A 275       5.305  21.741  37.200  1.0   0.00      PROT H \n",
            "ATOM   1016 HG12 ILE A 275       4.052  22.276  38.346  1.0   0.00      PROT H \n",
            "ATOM   1017 HG21 ILE A 275       6.273  22.148  41.306  1.0   0.00      PROT H \n",
            "ATOM   1018 HG22 ILE A 275       4.612  22.486  40.764  1.0   0.00      PROT H \n",
            "ATOM   1019 HG23 ILE A 275       5.216  20.812  40.789  1.0   0.00      PROT H \n",
            "ATOM   1020 HD11 ILE A 275       3.670  19.953  37.630  1.0   0.00      PROT H \n",
            "ATOM   1021 HD12 ILE A 275       5.286  19.504  38.224  1.0   0.00      PROT H \n",
            "ATOM   1022 HD13 ILE A 275       4.033  20.039  39.370  1.0   0.00      PROT H \n",
            "ATOM   1023  N   ILE A 276       6.279  24.448  36.720  1.0   0.00      PROT N \n",
            "ATOM   1024  CA  ILE A 276       6.662  24.570  35.326  1.0   0.00      PROT C \n",
            "ATOM   1025  C   ILE A 276       7.990  25.283  35.139  1.0   0.00      PROT C \n",
            "ATOM   1026  O   ILE A 276       8.722  25.020  34.183  1.0   0.00      PROT O \n",
            "ATOM   1027  CB  ILE A 276       5.593  25.311  34.508  1.0   0.00      PROT C \n",
            "ATOM   1028  CG1 ILE A 276       5.932  25.194  33.022  1.0   0.00      PROT C \n",
            "ATOM   1029  CG2 ILE A 276       5.531  26.775  34.933  1.0   0.00      PROT C \n",
            "ATOM   1030  CD1 ILE A 276       4.748  25.136  32.127  1.0   0.00      PROT C \n",
            "ATOM   1031  H   ILE A 276       5.556  25.042  37.099  1.0   0.00      PROT H \n",
            "ATOM   1032  HA  ILE A 276       6.759  23.528  35.022  1.0   0.00      PROT H \n",
            "ATOM   1033  HB  ILE A 276       4.612  24.869  34.685  1.0   0.00      PROT H \n",
            "ATOM   1034 HG13 ILE A 276       6.524  24.287  32.901  1.0   0.00      PROT H \n",
            "ATOM   1035 HG12 ILE A 276       6.540  26.061  32.765  1.0   0.00      PROT H \n",
            "ATOM   1036 HG21 ILE A 276       5.288  26.835  35.994  1.0   0.00      PROT H \n",
            "ATOM   1037 HG22 ILE A 276       6.497  27.247  34.754  1.0   0.00      PROT H \n",
            "ATOM   1038 HG23 ILE A 276       4.763  27.289  34.355  1.0   0.00      PROT H \n",
            "ATOM   1039 HD11 ILE A 276       5.079  25.054  31.092  1.0   0.00      PROT H \n",
            "ATOM   1040 HD12 ILE A 276       4.140  24.269  32.384  1.0   0.00      PROT H \n",
            "ATOM   1041 HD13 ILE A 276       4.156  26.043  32.248  1.0   0.00      PROT H \n",
            "ATOM   1042  N   THR A 277       8.299  26.191  36.049  1.0   0.00      PROT N \n",
            "ATOM   1043  CA  THR A 277       9.540  26.931  35.960  1.0   0.00      PROT C \n",
            "ATOM   1044  C   THR A 277      10.774  26.045  36.151  1.0   0.00      PROT C \n",
            "ATOM   1045  O   THR A 277      11.772  26.212  35.460  1.0   0.00      PROT O \n",
            "ATOM   1046  CB  THR A 277       9.543  28.083  36.966  1.0   0.00      PROT C \n",
            "ATOM   1047  OG1 THR A 277       9.025  27.623  38.215  1.0   0.00      PROT O \n",
            "ATOM   1048  CG2 THR A 277       8.675  29.211  36.481  1.0   0.00      PROT C \n",
            "ATOM   1049  H   THR A 277       7.667  26.374  36.815  1.0   0.00      PROT H \n",
            "ATOM   1050  HA  THR A 277       9.543  27.308  34.937  1.0   0.00      PROT H \n",
            "ATOM   1051  HB  THR A 277      10.559  28.437  37.143  1.0   0.00      PROT H \n",
            "ATOM   1052  HG1 THR A 277       9.470  26.878  38.691  1.0   0.00      PROT H \n",
            "ATOM   1053 HG21 THR A 277       8.690  30.021  37.210  1.0   0.00      PROT H \n",
            "ATOM   1054 HG22 THR A 277       9.053  29.575  35.526  1.0   0.00      PROT H \n",
            "ATOM   1055 HG23 THR A 277       7.653  28.855  36.355  1.0   0.00      PROT H \n",
            "ATOM   1056  N   PRO A 278      10.733  25.099  37.103  1.0   0.00      PROT N \n",
            "ATOM   1057  CA  PRO A 278      11.934  24.269  37.245  1.0   0.00      PROT C \n",
            "ATOM   1058  C   PRO A 278      12.041  23.295  36.073  1.0   0.00      PROT C \n",
            "ATOM   1059  O   PRO A 278      13.132  22.835  35.724  1.0   0.00      PROT O \n",
            "ATOM   1060  CB  PRO A 278      11.702  23.560  38.575  1.0   0.00      PROT C \n",
            "ATOM   1061  CG  PRO A 278      10.869  24.547  39.342  1.0   0.00      PROT C \n",
            "ATOM   1062  CD  PRO A 278       9.874  24.966  38.294  1.0   0.00      PROT C \n",
            "ATOM   1063  HA  PRO A 278      12.851  24.859  37.234  1.0   0.00      PROT H \n",
            "ATOM   1064  HB3 PRO A 278      12.629  23.369  39.116  1.0   0.00      PROT H \n",
            "ATOM   1065  HB2 PRO A 278      11.149  22.628  38.462  1.0   0.00      PROT H \n",
            "ATOM   1066  HG2 PRO A 278      10.413  24.050  40.198  1.0   0.00      PROT H \n",
            "ATOM   1067  HG3 PRO A 278      11.502  25.363  39.690  1.0   0.00      PROT H \n",
            "ATOM   1068  HD2 PRO A 278       9.110  24.195  38.189  1.0   0.00      PROT H \n",
            "ATOM   1069  HD3 PRO A 278       9.405  25.904  38.592  1.0   0.00      PROT H \n",
            "ATOM   1070  N   ALA A 279      10.896  22.989  35.467  1.0   0.00      PROT N \n",
            "ATOM   1071  CA  ALA A 279      10.853  22.085  34.327  1.0   0.00      PROT C \n",
            "ATOM   1072  C   ALA A 279      11.461  22.784  33.120  1.0   0.00      PROT C \n",
            "ATOM   1073  O   ALA A 279      12.213  22.179  32.357  1.0   0.00      PROT O \n",
            "ATOM   1074  CB  ALA A 279       9.418  21.673  34.028  1.0   0.00      PROT C \n",
            "ATOM   1075  H   ALA A 279      10.028  23.387  35.796  1.0   0.00      PROT H \n",
            "ATOM   1076  HA  ALA A 279      11.434  21.203  34.598  1.0   0.00      PROT H \n",
            "ATOM   1077  HB1 ALA A 279       8.994  21.174  34.899  1.0   0.00      PROT H \n",
            "ATOM   1078  HB2 ALA A 279       8.827  22.558  33.793  1.0   0.00      PROT H \n",
            "ATOM   1079  HB3 ALA A 279       9.405  20.992  33.177  1.0   0.00      PROT H \n",
            "ATOM   1080  N   ILE A 280      11.142  24.060  32.958  1.0   0.00      PROT N \n",
            "ATOM   1081  CA  ILE A 280      11.668  24.833  31.845  1.0   0.00      PROT C \n",
            "ATOM   1082  C   ILE A 280      13.179  25.020  31.976  1.0   0.00      PROT C \n",
            "ATOM   1083  O   ILE A 280      13.909  24.967  30.979  1.0   0.00      PROT O \n",
            "ATOM   1084  CB  ILE A 280      10.977  26.222  31.751  1.0   0.00      PROT C \n",
            "ATOM   1085  CG1 ILE A 280       9.513  26.050  31.318  1.0   0.00      PROT C \n",
            "ATOM   1086  CG2 ILE A 280      11.724  27.116  30.768  1.0   0.00      PROT C \n",
            "ATOM   1087  CD1 ILE A 280       8.695  27.325  31.349  1.0   0.00      PROT C \n",
            "ATOM   1088  H   ILE A 280      10.523  24.513  33.615  1.0   0.00      PROT H \n",
            "ATOM   1089  HA  ILE A 280      11.444  24.222  30.970  1.0   0.00      PROT H \n",
            "ATOM   1090  HB  ILE A 280      10.996  26.713  32.724  1.0   0.00      PROT H \n",
            "ATOM   1091 HG13 ILE A 280       9.068  25.316  31.990  1.0   0.00      PROT H \n",
            "ATOM   1092 HG12 ILE A 280       9.527  25.652  30.303  1.0   0.00      PROT H \n",
            "ATOM   1093 HG21 ILE A 280      12.759  27.226  31.091  1.0   0.00      PROT H \n",
            "ATOM   1094 HG22 ILE A 280      11.699  26.665  29.776  1.0   0.00      PROT H \n",
            "ATOM   1095 HG23 ILE A 280      11.248  28.096  30.734  1.0   0.00      PROT H \n",
            "ATOM   1096 HD11 ILE A 280       7.675  27.110  31.029  1.0   0.00      PROT H \n",
            "ATOM   1097 HD12 ILE A 280       8.681  27.723  32.364  1.0   0.00      PROT H \n",
            "ATOM   1098 HD13 ILE A 280       9.140  28.059  30.677  1.0   0.00      PROT H \n",
            "ATOM   1099  N   THR A 281      13.647  25.226  33.204  1.0   0.00      PROT N \n",
            "ATOM   1100  CA  THR A 281      15.072  25.429  33.462  1.0   0.00      PROT C \n",
            "ATOM   1101  C   THR A 281      15.908  24.199  33.141  1.0   0.00      PROT C \n",
            "ATOM   1102  O   THR A 281      17.024  24.318  32.633  1.0   0.00      PROT O \n",
            "ATOM   1103  CB  THR A 281      15.338  25.804  34.933  1.0   0.00      PROT C \n",
            "ATOM   1104  OG1 THR A 281      14.798  27.095  35.203  1.0   0.00      PROT O \n",
            "ATOM   1105  CG2 THR A 281      16.825  25.817  35.221  1.0   0.00      PROT C \n",
            "ATOM   1106  H   THR A 281      13.013  25.245  33.990  1.0   0.00      PROT H \n",
            "ATOM   1107  HA  THR A 281      15.320  26.241  32.778  1.0   0.00      PROT H \n",
            "ATOM   1108  HB  THR A 281      14.833  25.115  35.610  1.0   0.00      PROT H \n",
            "ATOM   1109  HG1 THR A 281      13.832  27.266  35.068  1.0   0.00      PROT H \n",
            "ATOM   1110 HG21 THR A 281      16.991  26.084  36.265  1.0   0.00      PROT H \n",
            "ATOM   1111 HG22 THR A 281      17.241  24.828  35.029  1.0   0.00      PROT H \n",
            "ATOM   1112 HG23 THR A 281      17.313  26.548  34.577  1.0   0.00      PROT H \n",
            "ATOM   1113  N   ARG A 282      15.387  23.018  33.438  1.0   0.00      PROT N \n",
            "ATOM   1114  CA  ARG A 282      16.160  21.824  33.163  1.0   0.00      PROT C \n",
            "ATOM   1115  C   ARG A 282      16.321  21.661  31.663  1.0   0.00      PROT C \n",
            "ATOM   1116  O   ARG A 282      17.356  21.171  31.199  1.0   0.00      PROT O \n",
            "ATOM   1117  CB  ARG A 282      15.503  20.595  33.784  1.0   0.00      PROT C \n",
            "ATOM   1118  CG  ARG A 282      15.304  20.733  35.287  1.0   0.00      PROT C \n",
            "ATOM   1119  CD  ARG A 282      15.336  19.394  35.985  1.0   0.00      PROT C \n",
            "ATOM   1120  NE  ARG A 282      14.026  19.014  36.489  1.0   0.00      PROT N \n",
            "ATOM   1121  CZ  ARG A 282      12.985  18.707  35.719  1.0   0.00      PROT C \n",
            "ATOM   1122  NH1 ARG A 282      13.095  18.729  34.396  1.0   0.00      PROT N \n",
            "ATOM   1123  NH2 ARG A 282      11.825  18.385  36.275  1.0   0.00      PROT N \n",
            "ATOM   1124  H   ARG A 282      14.466  22.961  33.849  1.0   0.00      PROT H \n",
            "ATOM   1125  HA  ARG A 282      17.135  21.976  33.627  1.0   0.00      PROT H \n",
            "ATOM   1126  HB3 ARG A 282      16.141  19.735  33.579  1.0   0.00      PROT H \n",
            "ATOM   1127  HB2 ARG A 282      14.536  20.453  33.301  1.0   0.00      PROT H \n",
            "ATOM   1128  HG3 ARG A 282      14.340  21.211  35.461  1.0   0.00      PROT H \n",
            "ATOM   1129  HG2 ARG A 282      16.099  21.367  35.680  1.0   0.00      PROT H \n",
            "ATOM   1130  HD2 ARG A 282      15.668  18.627  35.285  1.0   0.00      PROT H \n",
            "ATOM   1131  HD3 ARG A 282      16.026  19.440  36.828  1.0   0.00      PROT H \n",
            "ATOM   1132  HE  ARG A 282      13.934  18.992  37.495  1.0   0.00      PROT H \n",
            "ATOM   1133 HH12 ARG A 282      12.290  18.499  33.813  1.0   0.00      PROT H \n",
            "ATOM   1134 HH11 ARG A 282      13.985  18.976  33.962  1.0   0.00      PROT H \n",
            "ATOM   1135  HH2 ARG A 282      11.024  18.156  35.687  1.0   0.00      PROT H \n",
            "ATOM   1136  N   VAL A 283      15.317  22.083  30.893  1.0   0.00      PROT N \n",
            "ATOM   1137  CA  VAL A 283      15.433  21.984  29.438  1.0   0.00      PROT C \n",
            "ATOM   1138  C   VAL A 283      16.563  22.909  28.977  1.0   0.00      PROT C \n",
            "ATOM   1139  O   VAL A 283      17.334  22.557  28.083  1.0   0.00      PROT O \n",
            "ATOM   1140  CB  VAL A 283      14.124  22.372  28.704  1.0   0.00      PROT C \n",
            "ATOM   1141  CG1 VAL A 283      14.353  22.343  27.201  1.0   0.00      PROT C \n",
            "ATOM   1142  CG2 VAL A 283      13.004  21.396  29.074  1.0   0.00      PROT C \n",
            "ATOM   1143  H   VAL A 283      14.484  22.468  31.315  1.0   0.00      PROT H \n",
            "ATOM   1144  HA  VAL A 283      15.652  20.934  29.245  1.0   0.00      PROT H \n",
            "ATOM   1145  HB  VAL A 283      13.822  23.379  28.992  1.0   0.00      PROT H \n",
            "ATOM   1146 HG11 VAL A 283      15.151  23.039  26.942  1.0   0.00      PROT H \n",
            "ATOM   1147 HG12 VAL A 283      14.636  21.336  26.896  1.0   0.00      PROT H \n",
            "ATOM   1148 HG13 VAL A 283      13.436  22.634  26.688  1.0   0.00      PROT H \n",
            "ATOM   1149 HG21 VAL A 283      12.843  21.420  30.152  1.0   0.00      PROT H \n",
            "ATOM   1150 HG22 VAL A 283      12.086  21.686  28.563  1.0   0.00      PROT H \n",
            "ATOM   1151 HG23 VAL A 283      13.286  20.388  28.771  1.0   0.00      PROT H \n",
            "ATOM   1152  N   VAL A 284      16.663  24.084  29.593  1.0   0.00      PROT N \n",
            "ATOM   1153  CA  VAL A 284      17.723  25.022  29.259  1.0   0.00      PROT C \n",
            "ATOM   1154  C   VAL A 284      19.073  24.417  29.669  1.0   0.00      PROT C \n",
            "ATOM   1155  O   VAL A 284      20.017  24.404  28.877  1.0   0.00      PROT O \n",
            "ATOM   1156  CB  VAL A 284      17.537  26.387  29.991  1.0   0.00      PROT C \n",
            "ATOM   1157  CG1 VAL A 284      18.726  27.297  29.723  1.0   0.00      PROT C \n",
            "ATOM   1158  CG2 VAL A 284      16.257  27.060  29.530  1.0   0.00      PROT C \n",
            "ATOM   1159  H   VAL A 284      15.995  24.338  30.307  1.0   0.00      PROT H \n",
            "ATOM   1160  HA  VAL A 284      17.647  25.152  28.179  1.0   0.00      PROT H \n",
            "ATOM   1161  HB  VAL A 284      17.467  26.226  31.067  1.0   0.00      PROT H \n",
            "ATOM   1162 HG11 VAL A 284      19.641  26.807  30.056  1.0   0.00      PROT H \n",
            "ATOM   1163 HG12 VAL A 284      18.793  27.503  28.655  1.0   0.00      PROT H \n",
            "ATOM   1164 HG13 VAL A 284      18.597  28.233  30.266  1.0   0.00      PROT H \n",
            "ATOM   1165 HG21 VAL A 284      15.410  26.402  29.725  1.0   0.00      PROT H \n",
            "ATOM   1166 HG22 VAL A 284      16.123  27.996  30.072  1.0   0.00      PROT H \n",
            "ATOM   1167 HG23 VAL A 284      16.319  27.265  28.461  1.0   0.00      PROT H \n",
            "ATOM   1168  N   ASP A 285      19.155  23.906  30.899  1.0   0.00      PROT N \n",
            "ATOM   1169  CA  ASP A 285      20.397  23.313  31.402  1.0   0.00      PROT C \n",
            "ATOM   1170  C   ASP A 285      20.854  22.122  30.565  1.0   0.00      PROT C \n",
            "ATOM   1171  O   ASP A 285      22.046  21.891  30.397  1.0   0.00      PROT O \n",
            "ATOM   1172  CB  ASP A 285      20.241  22.878  32.866  1.0   0.00      PROT C \n",
            "ATOM   1173  CG  ASP A 285      20.013  24.049  33.798  1.0   0.00      PROT C \n",
            "ATOM   1174  OD1 ASP A 285      20.557  25.143  33.528  1.0   0.00      PROT O \n",
            "ATOM   1175  OD2 ASP A 285      19.299  23.878  34.811  1.0   0.00      PROT O \n",
            "ATOM   1176  H   ASP A 285      18.349  23.922  31.507  1.0   0.00      PROT H \n",
            "ATOM   1177  HA  ASP A 285      21.128  24.116  31.306  1.0   0.00      PROT H \n",
            "ATOM   1178  HB3 ASP A 285      21.150  22.353  33.158  1.0   0.00      PROT H \n",
            "ATOM   1179  HB2 ASP A 285      19.392  22.196  32.925  1.0   0.00      PROT H \n",
            "ATOM   1180  HD1 ASP A 285      20.331  25.820  34.229  1.0   0.00      PROT H \n",
            "ATOM   1181  N   PHE A 286      19.896  21.358  30.055  1.0   0.00      PROT N \n",
            "ATOM   1182  CA  PHE A 286      20.207  20.206  29.226  1.0   0.00      PROT C \n",
            "ATOM   1183  C   PHE A 286      20.960  20.676  27.986  1.0   0.00      PROT C \n",
            "ATOM   1184  O   PHE A 286      22.010  20.136  27.635  1.0   0.00      PROT O \n",
            "ATOM   1185  CB  PHE A 286      18.922  19.498  28.789  1.0   0.00      PROT C \n",
            "ATOM   1186  CG  PHE A 286      19.132  18.503  27.686  1.0   0.00      PROT C \n",
            "ATOM   1187  CD1 PHE A 286      19.864  17.346  27.905  1.0   0.00      PROT C \n",
            "ATOM   1188  CD2 PHE A 286      18.611  18.734  26.416  1.0   0.00      PROT C \n",
            "ATOM   1189  CE1 PHE A 286      20.074  16.427  26.872  1.0   0.00      PROT C \n",
            "ATOM   1190  CE2 PHE A 286      18.816  17.819  25.373  1.0   0.00      PROT C \n",
            "ATOM   1191  CZ  PHE A 286      19.548  16.662  25.603  1.0   0.00      PROT C \n",
            "ATOM   1192  H   PHE A 286      18.927  21.572  30.240  1.0   0.00      PROT H \n",
            "ATOM   1193  HA  PHE A 286      20.818  19.544  29.840  1.0   0.00      PROT H \n",
            "ATOM   1194  HB3 PHE A 286      18.223  20.266  28.457  1.0   0.00      PROT H \n",
            "ATOM   1195  HB2 PHE A 286      18.516  18.990  29.664  1.0   0.00      PROT H \n",
            "ATOM   1196  HD1 PHE A 286      20.283  17.145  28.891  1.0   0.00      PROT H \n",
            "ATOM   1197  HD2 PHE A 286      18.033  19.637  26.220  1.0   0.00      PROT H \n",
            "ATOM   1198  HE1 PHE A 286      20.652  15.523  27.063  1.0   0.00      PROT H \n",
            "ATOM   1199  HE2 PHE A 286      18.402  18.014  24.384  1.0   0.00      PROT H \n",
            "ATOM   1200  HZ  PHE A 286      19.707  15.948  24.795  1.0   0.00      PROT H \n",
            "ATOM   1201  N   ALA A 287      20.397  21.682  27.330  1.0   0.00      PROT N \n",
            "ATOM   1202  CA  ALA A 287      20.964  22.249  26.116  1.0   0.00      PROT C \n",
            "ATOM   1203  C   ALA A 287      22.330  22.851  26.385  1.0   0.00      PROT C \n",
            "ATOM   1204  O   ALA A 287      23.267  22.682  25.605  1.0   0.00      PROT O \n",
            "ATOM   1205  CB  ALA A 287      20.028  23.319  25.566  1.0   0.00      PROT C \n",
            "ATOM   1206  H   ALA A 287      19.537  22.088  27.670  1.0   0.00      PROT H \n",
            "ATOM   1207  HA  ALA A 287      21.067  21.420  25.416  1.0   0.00      PROT H \n",
            "ATOM   1208  HB1 ALA A 287      19.048  22.881  25.375  1.0   0.00      PROT H \n",
            "ATOM   1209  HB2 ALA A 287      19.930  24.125  26.293  1.0   0.00      PROT H \n",
            "ATOM   1210  HB3 ALA A 287      20.436  23.716  24.637  1.0   0.00      PROT H \n",
            "ATOM   1211  N   LYS A 288      22.429  23.562  27.500  1.0   0.00      PROT N \n",
            "ATOM   1212  CA  LYS A 288      23.664  24.214  27.869  1.0   0.00      PROT C \n",
            "ATOM   1213  C   LYS A 288      24.758  23.237  28.284  1.0   0.00      PROT C \n",
            "ATOM   1214  O   LYS A 288      25.832  23.667  28.703  1.0   0.00      PROT O \n",
            "ATOM   1215  CB  LYS A 288      23.406  25.231  28.980  1.0   0.00      PROT C \n",
            "ATOM   1216  CG  LYS A 288      22.414  26.317  28.588  1.0   0.00      PROT C \n",
            "ATOM   1217  CD  LYS A 288      22.922  27.175  27.450  1.0   0.00      PROT C \n",
            "ATOM   1218  CE  LYS A 288      23.758  28.325  27.959  1.0   0.00      PROT C \n",
            "ATOM   1219  NZ  LYS A 288      24.397  29.083  26.836  1.0   0.00      PROT N \n",
            "ATOM   1220  H   LYS A 288      21.631  23.656  28.111  1.0   0.00      PROT H \n",
            "ATOM   1221  HA  LYS A 288      23.982  24.702  26.947  1.0   0.00      PROT H \n",
            "ATOM   1222  HB3 LYS A 288      24.363  25.687  29.234  1.0   0.00      PROT H \n",
            "ATOM   1223  HB2 LYS A 288      23.024  24.684  29.842  1.0   0.00      PROT H \n",
            "ATOM   1224  HG3 LYS A 288      22.235  26.942  29.463  1.0   0.00      PROT H \n",
            "ATOM   1225  HG2 LYS A 288      21.484  25.833  28.291  1.0   0.00      PROT H \n",
            "ATOM   1226  HD3 LYS A 288      22.065  27.564  26.901  1.0   0.00      PROT H \n",
            "ATOM   1227  HD2 LYS A 288      23.525  26.552  26.789  1.0   0.00      PROT H \n",
            "ATOM   1228  HE2 LYS A 288      23.126  29.012  28.522  1.0   0.00      PROT H \n",
            "ATOM   1229  HE3 LYS A 288      24.546  27.942  28.607  1.0   0.00      PROT H \n",
            "ATOM   1230  HZ1 LYS A 288      24.086  30.044  26.858  1.0   0.00      PROT H \n",
            "ATOM   1231  HZ2 LYS A 288      25.401  29.052  26.938  1.0   0.00      PROT H \n",
            "ATOM   1232  N   LYS A 289      24.494  21.937  28.158  1.0   0.00      PROT N \n",
            "ATOM   1233  CA  LYS A 289      25.480  20.917  28.505  1.0   0.00      PROT C \n",
            "ATOM   1234  C   LYS A 289      26.025  20.278  27.243  1.0   0.00      PROT C \n",
            "ATOM   1235  O   LYS A 289      26.874  19.380  27.292  1.0   0.00      PROT O \n",
            "ATOM   1236  CB  LYS A 289      24.863  19.834  29.381  1.0   0.00      PROT C \n",
            "ATOM   1237  CG  LYS A 289      24.740  20.217  30.834  1.0   0.00      PROT C \n",
            "ATOM   1238  CD  LYS A 289      24.128  19.089  31.642  1.0   0.00      PROT C \n",
            "ATOM   1239  CE  LYS A 289      23.893  19.511  33.084  1.0   0.00      PROT C \n",
            "ATOM   1240  NZ  LYS A 289      25.159  19.788  33.821  1.0   0.00      PROT N \n",
            "ATOM   1241  H   LYS A 289      23.593  21.636  27.815  1.0   0.00      PROT H \n",
            "ATOM   1242  HA  LYS A 289      26.267  21.439  29.050  1.0   0.00      PROT H \n",
            "ATOM   1243  HB3 LYS A 289      25.493  18.949  29.291  1.0   0.00      PROT H \n",
            "ATOM   1244  HB2 LYS A 289      23.874  19.618  28.977  1.0   0.00      PROT H \n",
            "ATOM   1245  HG2 LYS A 289      25.728  20.439  31.236  1.0   0.00      PROT H \n",
            "ATOM   1246  HG3 LYS A 289      24.104  21.098  30.925  1.0   0.00      PROT H \n",
            "ATOM   1247  HD2 LYS A 289      23.172  18.804  31.203  1.0   0.00      PROT H \n",
            "ATOM   1248  HD3 LYS A 289      24.800  18.231  31.636  1.0   0.00      PROT H \n",
            "ATOM   1249  HE2 LYS A 289      23.292  20.420  33.103  1.0   0.00      PROT H \n",
            "ATOM   1250  HE3 LYS A 289      23.367  18.717  33.614  1.0   0.00      PROT H \n",
            "ATOM   1251  HZ1 LYS A 289      25.159  20.746  34.141  1.0   0.00      PROT H \n",
            "ATOM   1252  HZ2 LYS A 289      25.229  19.167  34.615  1.0   0.00      PROT H \n",
            "ATOM   1253  N   LEU A 290      25.520  20.743  26.109  1.0   0.00      PROT N \n",
            "ATOM   1254  CA  LEU A 290      25.949  20.232  24.817  1.0   0.00      PROT C \n",
            "ATOM   1255  C   LEU A 290      26.817  21.294  24.139  1.0   0.00      PROT C \n",
            "ATOM   1256  O   LEU A 290      26.344  22.398  23.837  1.0   0.00      PROT O \n",
            "ATOM   1257  CB  LEU A 290      24.722  19.903  23.954  1.0   0.00      PROT C \n",
            "ATOM   1258  CG  LEU A 290      23.696  18.969  24.609  1.0   0.00      PROT C \n",
            "ATOM   1259  CD1 LEU A 290      22.506  18.761  23.689  1.0   0.00      PROT C \n",
            "ATOM   1260  CD2 LEU A 290      24.367  17.647  24.951  1.0   0.00      PROT C \n",
            "ATOM   1261  H   LEU A 290      24.819  21.470  26.129  1.0   0.00      PROT H \n",
            "ATOM   1262  HA  LEU A 290      26.524  19.324  24.999  1.0   0.00      PROT H \n",
            "ATOM   1263  HB3 LEU A 290      25.099  19.445  23.040  1.0   0.00      PROT H \n",
            "ATOM   1264  HB2 LEU A 290      24.247  20.854  23.714  1.0   0.00      PROT H \n",
            "ATOM   1265  HG  LEU A 290      23.318  19.414  25.529  1.0   0.00      PROT H \n",
            "ATOM   1266 HD11 LEU A 290      22.048  19.724  23.463  1.0   0.00      PROT H \n",
            "ATOM   1267 HD12 LEU A 290      22.840  18.292  22.764  1.0   0.00      PROT H \n",
            "ATOM   1268 HD13 LEU A 290      21.775  18.118  24.180  1.0   0.00      PROT H \n",
            "ATOM   1269 HD21 LEU A 290      25.215  17.830  25.611  1.0   0.00      PROT H \n",
            "ATOM   1270 HD22 LEU A 290      23.651  16.995  25.452  1.0   0.00      PROT H \n",
            "ATOM   1271 HD23 LEU A 290      24.716  17.169  24.036  1.0   0.00      PROT H \n",
            "ATOM   1272  N   PRO A 291      28.111  20.980  23.920  1.0   0.00      PROT N \n",
            "ATOM   1273  CA  PRO A 291      29.079  21.883  23.279  1.0   0.00      PROT C \n",
            "ATOM   1274  C   PRO A 291      28.606  22.394  21.920  1.0   0.00      PROT C \n",
            "ATOM   1275  O   PRO A 291      28.846  23.550  21.558  1.0   0.00      PROT O \n",
            "ATOM   1276  CB  PRO A 291      30.333  21.022  23.182  1.0   0.00      PROT C \n",
            "ATOM   1277  CG  PRO A 291      30.247  20.201  24.437  1.0   0.00      PROT C \n",
            "ATOM   1278  CD  PRO A 291      28.790  19.768  24.416  1.0   0.00      PROT C \n",
            "ATOM   1279  HA  PRO A 291      29.214  22.790  23.869  1.0   0.00      PROT H \n",
            "ATOM   1280  HB3 PRO A 291      31.253  21.606  23.192  1.0   0.00      PROT H \n",
            "ATOM   1281  HB2 PRO A 291      30.331  20.362  22.315  1.0   0.00      PROT H \n",
            "ATOM   1282  HG2 PRO A 291      30.946  19.367  24.374  1.0   0.00      PROT H \n",
            "ATOM   1283  HG3 PRO A 291      30.500  20.823  25.296  1.0   0.00      PROT H \n",
            "ATOM   1284  HD2 PRO A 291      28.675  18.913  23.750  1.0   0.00      PROT H \n",
            "ATOM   1285  HD3 PRO A 291      28.479  19.489  25.423  1.0   0.00      PROT H \n",
            "ATOM   1286  N   MET A 292      27.930  21.532  21.173  1.0   0.00      PROT N \n",
            "ATOM   1287  CA  MET A 292      27.412  21.923  19.874  1.0   0.00      PROT C \n",
            "ATOM   1288  C   MET A 292      26.450  23.096  20.000  1.0   0.00      PROT C \n",
            "ATOM   1289  O   MET A 292      26.481  24.016  19.193  1.0   0.00      PROT O \n",
            "ATOM   1290  CB  MET A 292      26.696  20.746  19.210  1.0   0.00      PROT C \n",
            "ATOM   1291  CG  MET A 292      27.599  19.848  18.404  1.0   0.00      PROT C \n",
            "ATOM   1292  SD  MET A 292      26.791  18.307  17.942  1.0   0.00      PROT S \n",
            "ATOM   1293  CE  MET A 292      28.203  17.172  17.910  1.0   0.00      PROT C \n",
            "ATOM   1294  H   MET A 292      27.771  20.592  21.506  1.0   0.00      PROT H \n",
            "ATOM   1295  HA  MET A 292      28.289  22.221  19.299  1.0   0.00      PROT H \n",
            "ATOM   1296  HB3 MET A 292      25.927  21.165  18.561  1.0   0.00      PROT H \n",
            "ATOM   1297  HB2 MET A 292      26.225  20.168  20.006  1.0   0.00      PROT H \n",
            "ATOM   1298  HG2 MET A 292      27.899  20.360  17.490  1.0   0.00      PROT H \n",
            "ATOM   1299  HG3 MET A 292      28.485  19.603  18.990  1.0   0.00      PROT H \n",
            "ATOM   1300  HE1 MET A 292      27.860  16.173  17.638  1.0   0.00      PROT H \n",
            "ATOM   1301  HE2 MET A 292      28.931  17.518  17.176  1.0   0.00      PROT H \n",
            "ATOM   1302  HE3 MET A 292      28.667  17.141  18.896  1.0   0.00      PROT H \n",
            "ATOM   1303  N   PHE A 293      25.601  23.056  21.022  1.0   0.00      PROT N \n",
            "ATOM   1304  CA  PHE A 293      24.607  24.094  21.258  1.0   0.00      PROT C \n",
            "ATOM   1305  C   PHE A 293      25.202  25.403  21.765  1.0   0.00      PROT C \n",
            "ATOM   1306  O   PHE A 293      24.859  26.481  21.280  1.0   0.00      PROT O \n",
            "ATOM   1307  CB  PHE A 293      23.577  23.598  22.274  1.0   0.00      PROT C \n",
            "ATOM   1308  CG  PHE A 293      22.437  24.562  22.520  1.0   0.00      PROT C \n",
            "ATOM   1309  CD1 PHE A 293      21.339  24.597  21.660  1.0   0.00      PROT C \n",
            "ATOM   1310  CD2 PHE A 293      22.454  25.414  23.623  1.0   0.00      PROT C \n",
            "ATOM   1311  CE1 PHE A 293      20.274  25.453  21.896  1.0   0.00      PROT C \n",
            "ATOM   1312  CE2 PHE A 293      21.394  26.276  23.870  1.0   0.00      PROT C \n",
            "ATOM   1313  CZ  PHE A 293      20.298  26.298  23.003  1.0   0.00      PROT C \n",
            "ATOM   1314  H   PHE A 293      25.630  22.284  21.672  1.0   0.00      PROT H \n",
            "ATOM   1315  HA  PHE A 293      24.178  24.267  20.271  1.0   0.00      PROT H \n",
            "ATOM   1316  HB3 PHE A 293      24.109  23.419  23.208  1.0   0.00      PROT H \n",
            "ATOM   1317  HB2 PHE A 293      23.181  22.656  21.895  1.0   0.00      PROT H \n",
            "ATOM   1318  HD1 PHE A 293      21.327  23.938  20.792  1.0   0.00      PROT H \n",
            "ATOM   1319  HD2 PHE A 293      23.312  25.399  24.295  1.0   0.00      PROT H \n",
            "ATOM   1320  HE1 PHE A 293      19.422  25.463  21.216  1.0   0.00      PROT H \n",
            "ATOM   1321  HE2 PHE A 293      21.418  26.934  24.739  1.0   0.00      PROT H \n",
            "ATOM   1322  HZ  PHE A 293      19.464  26.974  23.191  1.0   0.00      PROT H \n",
            "ATOM   1323  N   CYS A 294      26.092  25.291  22.744  1.0   0.00      PROT N \n",
            "ATOM   1324  CA  CYS A 294      26.719  26.442  23.370  1.0   0.00      PROT C \n",
            "ATOM   1325  C   CYS A 294      27.530  27.320  22.420  1.0   0.00      PROT C \n",
            "ATOM   1326  O   CYS A 294      27.748  28.498  22.696  1.0   0.00      PROT O \n",
            "ATOM   1327  CB  CYS A 294      27.592  25.965  24.530  1.0   0.00      PROT C \n",
            "ATOM   1328  SG  CYS A 294      26.694  24.973  25.732  1.0   0.00      PROT S \n",
            "ATOM   1329  H   CYS A 294      26.358  24.377  23.082  1.0   0.00      PROT H \n",
            "ATOM   1330  HA  CYS A 294      25.873  27.043  23.703  1.0   0.00      PROT H \n",
            "ATOM   1331  HB2 CYS A 294      28.408  25.354  24.143  1.0   0.00      PROT H \n",
            "ATOM   1332  HB3 CYS A 294      28.001  26.828  25.055  1.0   0.00      PROT H \n",
            "ATOM   1333  HG  CYS A 294      27.329  24.484  26.806  1.0   0.00      PROT H \n",
            "ATOM   1334  N   GLU A 295      27.969  26.752  21.303  1.0   0.00      PROT N \n",
            "ATOM   1335  CA  GLU A 295      28.745  27.500  20.329  1.0   0.00      PROT C \n",
            "ATOM   1336  C   GLU A 295      27.859  28.082  19.225  1.0   0.00      PROT C \n",
            "ATOM   1337  O   GLU A 295      28.339  28.462  18.156  1.0   0.00      PROT O \n",
            "ATOM   1338  CB  GLU A 295      29.848  26.604  19.754  1.0   0.00      PROT C \n",
            "ATOM   1339  CG  GLU A 295      30.962  26.330  20.782  1.0   0.00      PROT C \n",
            "ATOM   1340  CD  GLU A 295      31.945  25.257  20.354  1.0   0.00      PROT C \n",
            "ATOM   1341  OE1 GLU A 295      32.449  25.343  19.216  1.0   0.00      PROT O \n",
            "ATOM   1342  OE2 GLU A 295      32.218  24.336  21.160  1.0   0.00      PROT O \n",
            "ATOM   1343  H   GLU A 295      27.765  25.781  21.117  1.0   0.00      PROT H \n",
            "ATOM   1344  HA  GLU A 295      29.183  28.332  20.881  1.0   0.00      PROT H \n",
            "ATOM   1345  HB2 GLU A 295      29.420  25.648  19.453  1.0   0.00      PROT H \n",
            "ATOM   1346  HB3 GLU A 295      30.296  27.090  18.887  1.0   0.00      PROT H \n",
            "ATOM   1347  HG3 GLU A 295      30.477  26.029  21.710  1.0   0.00      PROT H \n",
            "ATOM   1348  HG2 GLU A 295      31.494  27.269  20.938  1.0   0.00      PROT H \n",
            "ATOM   1349  HE2 GLU A 295      32.881  23.709  20.751  1.0   0.00      PROT H \n",
            "ATOM   1350  N   LEU A 296      26.556  28.139  19.488  1.0   0.00      PROT N \n",
            "ATOM   1351  CA  LEU A 296      25.605  28.717  18.546  1.0   0.00      PROT C \n",
            "ATOM   1352  C   LEU A 296      25.359  30.131  19.046  1.0   0.00      PROT C \n",
            "ATOM   1353  O   LEU A 296      25.417  30.381  20.246  1.0   0.00      PROT O \n",
            "ATOM   1354  CB  LEU A 296      24.276  27.941  18.539  1.0   0.00      PROT C \n",
            "ATOM   1355  CG  LEU A 296      24.205  26.588  17.810  1.0   0.00      PROT C \n",
            "ATOM   1356  CD1 LEU A 296      22.873  25.919  18.092  1.0   0.00      PROT C \n",
            "ATOM   1357  CD2 LEU A 296      24.394  26.791  16.310  1.0   0.00      PROT C \n",
            "ATOM   1358  H   LEU A 296      26.201  27.776  20.361  1.0   0.00      PROT H \n",
            "ATOM   1359  HA  LEU A 296      26.039  28.675  17.547  1.0   0.00      PROT H \n",
            "ATOM   1360  HB3 LEU A 296      23.559  28.631  18.094  1.0   0.00      PROT H \n",
            "ATOM   1361  HB2 LEU A 296      24.038  27.799  19.593  1.0   0.00      PROT H \n",
            "ATOM   1362  HG  LEU A 296      24.997  25.932  18.171  1.0   0.00      PROT H \n",
            "ATOM   1363 HD11 LEU A 296      22.755  25.782  19.167  1.0   0.00      PROT H \n",
            "ATOM   1364 HD12 LEU A 296      22.065  26.546  17.715  1.0   0.00      PROT H \n",
            "ATOM   1365 HD13 LEU A 296      22.842  24.949  17.597  1.0   0.00      PROT H \n",
            "ATOM   1366 HD21 LEU A 296      25.355  27.272  16.127  1.0   0.00      PROT H \n",
            "ATOM   1367 HD22 LEU A 296      24.370  25.824  15.807  1.0   0.00      PROT H \n",
            "ATOM   1368 HD23 LEU A 296      23.593  27.422  15.925  1.0   0.00      PROT H \n",
            "ATOM   1369  N   PRO A 297      25.089  31.078  18.136  1.0   0.00      PROT N \n",
            "ATOM   1370  CA  PRO A 297      24.838  32.464  18.553  1.0   0.00      PROT C \n",
            "ATOM   1371  C   PRO A 297      23.737  32.463  19.603  1.0   0.00      PROT C \n",
            "ATOM   1372  O   PRO A 297      22.880  31.576  19.598  1.0   0.00      PROT O \n",
            "ATOM   1373  CB  PRO A 297      24.383  33.140  17.263  1.0   0.00      PROT C \n",
            "ATOM   1374  CG  PRO A 297      25.064  32.328  16.196  1.0   0.00      PROT C \n",
            "ATOM   1375  CD  PRO A 297      24.896  30.920  16.684  1.0   0.00      PROT C \n",
            "ATOM   1376  HA  PRO A 297      25.713  32.941  18.994  1.0   0.00      PROT H \n",
            "ATOM   1377  HB3 PRO A 297      24.711  34.177  17.195  1.0   0.00      PROT H \n",
            "ATOM   1378  HB2 PRO A 297      23.303  33.091  17.123  1.0   0.00      PROT H \n",
            "ATOM   1379  HG3 PRO A 297      26.126  32.558  16.109  1.0   0.00      PROT H \n",
            "ATOM   1380  HG2 PRO A 297      24.588  32.434  15.221  1.0   0.00      PROT H \n",
            "ATOM   1381  HD2 PRO A 297      23.901  30.561  16.421  1.0   0.00      PROT H \n",
            "ATOM   1382  HD3 PRO A 297      25.648  30.282  16.219  1.0   0.00      PROT H \n",
            "ATOM   1383  N   CYS A 298      23.762  33.447  20.496  1.0   0.00      PROT N \n",
            "ATOM   1384  CA  CYS A 298      22.757  33.542  21.543  1.0   0.00      PROT C \n",
            "ATOM   1385  C   CYS A 298      21.352  33.743  20.945  1.0   0.00      PROT C \n",
            "ATOM   1386  O   CYS A 298      20.349  33.330  21.534  1.0   0.00      PROT O \n",
            "ATOM   1387  CB  CYS A 298      23.110  34.689  22.491  1.0   0.00      PROT C \n",
            "ATOM   1388  SG  CYS A 298      21.897  34.994  23.793  1.0   0.00      PROT S \n",
            "ATOM   1389  H   CYS A 298      24.489  34.147  20.456  1.0   0.00      PROT H \n",
            "ATOM   1390  HA  CYS A 298      22.793  32.590  22.073  1.0   0.00      PROT H \n",
            "ATOM   1391  HB2 CYS A 298      24.056  34.472  22.987  1.0   0.00      PROT H \n",
            "ATOM   1392  HB3 CYS A 298      23.201  35.615  21.923  1.0   0.00      PROT H \n",
            "ATOM   1393  HG  CYS A 298      22.101  35.975  24.682  1.0   0.00      PROT H \n",
            "ATOM   1394  N   GLU A 299      21.289  34.365  19.771  1.0   0.00      PROT N \n",
            "ATOM   1395  CA  GLU A 299      20.020  34.607  19.101  1.0   0.00      PROT C \n",
            "ATOM   1396  C   GLU A 299      19.436  33.280  18.622  1.0   0.00      PROT C \n",
            "ATOM   1397  O   GLU A 299      18.214  33.086  18.609  1.0   0.00      PROT O \n",
            "ATOM   1398  CB  GLU A 299      20.208  35.542  17.901  1.0   0.00      PROT C \n",
            "ATOM   1399  CG  GLU A 299      20.670  36.957  18.237  1.0   0.00      PROT C \n",
            "ATOM   1400  CD  GLU A 299      22.075  37.002  18.823  1.0   0.00      PROT C \n",
            "ATOM   1401  OE1 GLU A 299      23.000  36.401  18.221  1.0   0.00      PROT O \n",
            "ATOM   1402  OE2 GLU A 299      22.255  37.644  19.883  1.0   0.00      PROT O \n",
            "ATOM   1403  H   GLU A 299      22.136  34.683  19.322  1.0   0.00      PROT H \n",
            "ATOM   1404  HA  GLU A 299      19.376  35.075  19.846  1.0   0.00      PROT H \n",
            "ATOM   1405  HB3 GLU A 299      19.245  35.585  17.391  1.0   0.00      PROT H \n",
            "ATOM   1406  HB2 GLU A 299      20.942  35.065  17.251  1.0   0.00      PROT H \n",
            "ATOM   1407  HG3 GLU A 299      19.963  37.374  18.955  1.0   0.00      PROT H \n",
            "ATOM   1408  HG2 GLU A 299      20.641  37.541  17.317  1.0   0.00      PROT H \n",
            "ATOM   1409  HE1 GLU A 299      23.862  36.520  18.713  1.0   0.00      PROT H \n",
            "ATOM   1410  N   ASP A 300      20.305  32.368  18.213  1.0   0.00      PROT N \n",
            "ATOM   1411  CA  ASP A 300      19.832  31.072  17.764  1.0   0.00      PROT C \n",
            "ATOM   1412  C   ASP A 300      19.493  30.185  18.952  1.0   0.00      PROT C \n",
            "ATOM   1413  O   ASP A 300      18.571  29.385  18.891  1.0   0.00      PROT O \n",
            "ATOM   1414  CB  ASP A 300      20.877  30.383  16.891  1.0   0.00      PROT C \n",
            "ATOM   1415  CG  ASP A 300      20.892  30.916  15.486  1.0   0.00      PROT C \n",
            "ATOM   1416  OD1 ASP A 300      19.976  31.686  15.133  1.0   0.00      PROT O \n",
            "ATOM   1417  OD2 ASP A 300      21.809  30.559  14.725  1.0   0.00      PROT O \n",
            "ATOM   1418  H   ASP A 300      21.293  32.576  18.214  1.0   0.00      PROT H \n",
            "ATOM   1419  HA  ASP A 300      18.935  31.285  17.183  1.0   0.00      PROT H \n",
            "ATOM   1420  HB3 ASP A 300      20.650  29.317  16.873  1.0   0.00      PROT H \n",
            "ATOM   1421  HB2 ASP A 300      21.854  30.539  17.349  1.0   0.00      PROT H \n",
            "ATOM   1422  HD1 ASP A 300      20.102  31.953  14.178  1.0   0.00      PROT H \n",
            "ATOM   1423  N   GLN A 301      20.240  30.330  20.037  1.0   0.00      PROT N \n",
            "ATOM   1424  CA  GLN A 301      19.987  29.528  21.220  1.0   0.00      PROT C \n",
            "ATOM   1425  C   GLN A 301      18.586  29.822  21.753  1.0   0.00      PROT C \n",
            "ATOM   1426  O   GLN A 301      17.879  28.916  22.186  1.0   0.00      PROT O \n",
            "ATOM   1427  CB  GLN A 301      21.051  29.816  22.284  1.0   0.00      PROT C \n",
            "ATOM   1428  CG  GLN A 301      22.456  29.418  21.863  1.0   0.00      PROT C \n",
            "ATOM   1429  CD  GLN A 301      23.445  29.480  23.012  1.0   0.00      PROT C \n",
            "ATOM   1430  OE1 GLN A 301      23.134  29.076  24.124  1.0   0.00      PROT O \n",
            "ATOM   1431  NE2 GLN A 301      24.650  29.968  22.742  1.0   0.00      PROT N \n",
            "ATOM   1432  H   GLN A 301      20.993  31.003  20.045  1.0   0.00      PROT H \n",
            "ATOM   1433  HA  GLN A 301      20.050  28.485  20.908  1.0   0.00      PROT H \n",
            "ATOM   1434  HB3 GLN A 301      20.771  29.265  23.182  1.0   0.00      PROT H \n",
            "ATOM   1435  HB2 GLN A 301      21.023  30.886  22.491  1.0   0.00      PROT H \n",
            "ATOM   1436  HG3 GLN A 301      22.775  30.097  21.072  1.0   0.00      PROT H \n",
            "ATOM   1437  HG2 GLN A 301      22.415  28.401  21.474  1.0   0.00      PROT H \n",
            "ATOM   1438 HE21 GLN A 301      24.869  30.295  21.801  1.0   0.00      PROT H \n",
            "ATOM   1439 HE22 GLN A 301      25.357  30.016  23.476  1.0   0.00      PROT H \n",
            "ATOM   1440  N   ILE A 302      18.184  31.089  21.702  1.0   0.00      PROT N \n",
            "ATOM   1441  CA  ILE A 302      16.861  31.498  22.166  1.0   0.00      PROT C \n",
            "ATOM   1442  C   ILE A 302      15.763  30.871  21.287  1.0   0.00      PROT C \n",
            "ATOM   1443  O   ILE A 302      14.879  30.154  21.785  1.0   0.00      PROT O \n",
            "ATOM   1444  CB  ILE A 302      16.720  33.043  22.140  1.0   0.00      PROT C \n",
            "ATOM   1445  CG1 ILE A 302      17.691  33.677  23.143  1.0   0.00      PROT C \n",
            "ATOM   1446  CG2 ILE A 302      15.301  33.448  22.467  1.0   0.00      PROT C \n",
            "ATOM   1447  CD1 ILE A 302      17.718  35.184  23.106  1.0   0.00      PROT C \n",
            "ATOM   1448  H   ILE A 302      18.800  31.800  21.334  1.0   0.00      PROT H \n",
            "ATOM   1449  HA  ILE A 302      16.798  31.119  23.186  1.0   0.00      PROT H \n",
            "ATOM   1450  HB  ILE A 302      16.946  33.416  21.141  1.0   0.00      PROT H \n",
            "ATOM   1451 HG13 ILE A 302      18.684  33.290  22.916  1.0   0.00      PROT H \n",
            "ATOM   1452 HG12 ILE A 302      17.389  33.343  24.136  1.0   0.00      PROT H \n",
            "ATOM   1453 HG21 ILE A 302      14.619  32.993  21.749  1.0   0.00      PROT H \n",
            "ATOM   1454 HG22 ILE A 302      15.049  33.110  23.472  1.0   0.00      PROT H \n",
            "ATOM   1455 HG23 ILE A 302      15.212  34.533  22.415  1.0   0.00      PROT H \n",
            "ATOM   1456 HD11 ILE A 302      18.429  35.553  23.845  1.0   0.00      PROT H \n",
            "ATOM   1457 HD12 ILE A 302      18.020  35.518  22.113  1.0   0.00      PROT H \n",
            "ATOM   1458 HD13 ILE A 302      16.725  35.571  23.333  1.0   0.00      PROT H \n",
            "ATOM   1459  N   ILE A 303      15.834  31.134  19.982  1.0   0.00      PROT N \n",
            "ATOM   1460  CA  ILE A 303      14.863  30.605  19.031  1.0   0.00      PROT C \n",
            "ATOM   1461  C   ILE A 303      14.682  29.086  19.183  1.0   0.00      PROT C \n",
            "ATOM   1462  O   ILE A 303      13.552  28.591  19.212  1.0   0.00      PROT O \n",
            "ATOM   1463  CB  ILE A 303      15.282  30.962  17.577  1.0   0.00      PROT C \n",
            "ATOM   1464  CG1 ILE A 303      15.120  32.477  17.352  1.0   0.00      PROT C \n",
            "ATOM   1465  CG2 ILE A 303      14.452  30.171  16.565  1.0   0.00      PROT C \n",
            "ATOM   1466  CD1 ILE A 303      15.708  32.993  16.039  1.0   0.00      PROT C \n",
            "ATOM   1467  H   ILE A 303      16.577  31.716  19.624  1.0   0.00      PROT H \n",
            "ATOM   1468  HA  ILE A 303      13.928  31.095  19.303  1.0   0.00      PROT H \n",
            "ATOM   1469  HB  ILE A 303      16.325  30.691  17.417  1.0   0.00      PROT H \n",
            "ATOM   1470 HG13 ILE A 303      15.609  32.974  18.189  1.0   0.00      PROT H \n",
            "ATOM   1471 HG12 ILE A 303      14.051  32.687  17.379  1.0   0.00      PROT H \n",
            "ATOM   1472 HG21 ILE A 303      14.582  29.104  16.743  1.0   0.00      PROT H \n",
            "ATOM   1473 HG22 ILE A 303      13.399  30.432  16.675  1.0   0.00      PROT H \n",
            "ATOM   1474 HG23 ILE A 303      14.783  30.414  15.555  1.0   0.00      PROT H \n",
            "ATOM   1475 HD11 ILE A 303      15.547  34.069  15.967  1.0   0.00      PROT H \n",
            "ATOM   1476 HD12 ILE A 303      16.777  32.783  16.012  1.0   0.00      PROT H \n",
            "ATOM   1477 HD13 ILE A 303      15.219  32.496  15.202  1.0   0.00      PROT H \n",
            "ATOM   1478  N   LEU A 304      15.790  28.358  19.307  1.0   0.00      PROT N \n",
            "ATOM   1479  CA  LEU A 304      15.752  26.910  19.461  1.0   0.00      PROT C \n",
            "ATOM   1480  C   LEU A 304      15.108  26.450  20.763  1.0   0.00      PROT C \n",
            "ATOM   1481  O   LEU A 304      14.415  25.434  20.788  1.0   0.00      PROT O \n",
            "ATOM   1482  CB  LEU A 304      17.170  26.320  19.383  1.0   0.00      PROT C \n",
            "ATOM   1483  CG  LEU A 304      17.911  26.399  18.043  1.0   0.00      PROT C \n",
            "ATOM   1484  CD1 LEU A 304      19.279  25.755  18.170  1.0   0.00      PROT C \n",
            "ATOM   1485  CD2 LEU A 304      17.106  25.710  16.953  1.0   0.00      PROT C \n",
            "ATOM   1486  H   LEU A 304      16.694  28.808  19.296  1.0   0.00      PROT H \n",
            "ATOM   1487  HA  LEU A 304      15.120  26.600  18.628  1.0   0.00      PROT H \n",
            "ATOM   1488  HB3 LEU A 304      17.058  25.276  19.675  1.0   0.00      PROT H \n",
            "ATOM   1489  HB2 LEU A 304      17.740  26.852  20.145  1.0   0.00      PROT H \n",
            "ATOM   1490  HG  LEU A 304      18.043  27.444  17.764  1.0   0.00      PROT H \n",
            "ATOM   1491 HD11 LEU A 304      19.846  26.257  18.954  1.0   0.00      PROT H \n",
            "ATOM   1492 HD12 LEU A 304      19.162  24.702  18.424  1.0   0.00      PROT H \n",
            "ATOM   1493 HD13 LEU A 304      19.812  25.843  17.223  1.0   0.00      PROT H \n",
            "ATOM   1494 HD21 LEU A 304      16.127  26.181  16.872  1.0   0.00      PROT H \n",
            "ATOM   1495 HD22 LEU A 304      17.632  25.798  16.002  1.0   0.00      PROT H \n",
            "ATOM   1496 HD23 LEU A 304      16.983  24.656  17.203  1.0   0.00      PROT H \n",
            "ATOM   1497  N   LEU A 305      15.353  27.182  21.845  1.0   0.00      PROT N \n",
            "ATOM   1498  CA  LEU A 305      14.812  26.817  23.153  1.0   0.00      PROT C \n",
            "ATOM   1499  C   LEU A 305      13.311  27.050  23.252  1.0   0.00      PROT C \n",
            "ATOM   1500  O   LEU A 305      12.578  26.204  23.762  1.0   0.00      PROT O \n",
            "ATOM   1501  CB  LEU A 305      15.552  27.582  24.257  1.0   0.00      PROT C \n",
            "ATOM   1502  CG  LEU A 305      16.944  27.007  24.534  1.0   0.00      PROT C \n",
            "ATOM   1503  CD1 LEU A 305      17.721  27.864  25.529  1.0   0.00      PROT C \n",
            "ATOM   1504  CD2 LEU A 305      16.769  25.572  25.037  1.0   0.00      PROT C \n",
            "ATOM   1505  H   LEU A 305      15.925  28.012  21.775  1.0   0.00      PROT H \n",
            "ATOM   1506  HA  LEU A 305      14.985  25.744  23.236  1.0   0.00      PROT H \n",
            "ATOM   1507  HB3 LEU A 305      14.950  27.529  25.164  1.0   0.00      PROT H \n",
            "ATOM   1508  HB2 LEU A 305      15.644  28.621  23.942  1.0   0.00      PROT H \n",
            "ATOM   1509  HG  LEU A 305      17.536  27.009  23.619  1.0   0.00      PROT H \n",
            "ATOM   1510 HD11 LEU A 305      17.824  28.875  25.135  1.0   0.00      PROT H \n",
            "ATOM   1511 HD12 LEU A 305      17.185  27.896  26.477  1.0   0.00      PROT H \n",
            "ATOM   1512 HD13 LEU A 305      18.710  27.433  25.685  1.0   0.00      PROT H \n",
            "ATOM   1513 HD21 LEU A 305      16.211  24.994  24.300  1.0   0.00      PROT H \n",
            "ATOM   1514 HD22 LEU A 305      17.748  25.118  25.189  1.0   0.00      PROT H \n",
            "ATOM   1515 HD23 LEU A 305      16.223  25.582  25.980  1.0   0.00      PROT H \n",
            "ATOM   1516  N   LYS A 306      12.856  28.196  22.757  1.0   0.00      PROT N \n",
            "ATOM   1517  CA  LYS A 306      11.435  28.506  22.770  1.0   0.00      PROT C \n",
            "ATOM   1518  C   LYS A 306      10.695  27.518  21.867  1.0   0.00      PROT C \n",
            "ATOM   1519  O   LYS A 306       9.517  27.230  22.076  1.0   0.00      PROT O \n",
            "ATOM   1520  CB  LYS A 306      11.212  29.940  22.287  1.0   0.00      PROT C \n",
            "ATOM   1521  CG  LYS A 306      11.943  30.972  23.130  1.0   0.00      PROT C \n",
            "ATOM   1522  CD  LYS A 306      11.736  32.390  22.618  1.0   0.00      PROT C \n",
            "ATOM   1523  CE  LYS A 306      10.301  32.847  22.801  1.0   0.00      PROT C \n",
            "ATOM   1524  NZ  LYS A 306      10.095  34.205  22.237  1.0   0.00      PROT N \n",
            "ATOM   1525  H   LYS A 306      13.500  28.867  22.364  1.0   0.00      PROT H \n",
            "ATOM   1526  HA  LYS A 306      11.097  28.406  23.802  1.0   0.00      PROT H \n",
            "ATOM   1527  HB3 LYS A 306      10.140  30.137  22.320  1.0   0.00      PROT H \n",
            "ATOM   1528  HB2 LYS A 306      11.561  30.000  21.256  1.0   0.00      PROT H \n",
            "ATOM   1529  HG2 LYS A 306      11.578  30.928  24.156  1.0   0.00      PROT H \n",
            "ATOM   1530  HG3 LYS A 306      13.012  30.761  23.116  1.0   0.00      PROT H \n",
            "ATOM   1531  HD3 LYS A 306      11.993  32.414  21.559  1.0   0.00      PROT H \n",
            "ATOM   1532  HD2 LYS A 306      12.402  33.055  23.168  1.0   0.00      PROT H \n",
            "ATOM   1533  HE2 LYS A 306       9.631  32.154  22.292  1.0   0.00      PROT H \n",
            "ATOM   1534  HE3 LYS A 306      10.060  32.871  23.864  1.0   0.00      PROT H \n",
            "ATOM   1535  HZ1 LYS A 306       9.392  34.166  21.513  1.0   0.00      PROT H \n",
            "ATOM   1536  HZ2 LYS A 306       9.790  34.831  22.969  1.0   0.00      PROT H \n",
            "ATOM   1537  HZ3 LYS A 306      10.800  34.883  21.735  1.0   0.00      PROT H \n",
            "ATOM   1538  N   GLY A 307      11.410  26.983  20.881  1.0   0.00      PROT N \n",
            "ATOM   1539  CA  GLY A 307      10.821  26.044  19.951  1.0   0.00      PROT C \n",
            "ATOM   1540  C   GLY A 307      10.681  24.604  20.416  1.0   0.00      PROT C \n",
            "ATOM   1541  O   GLY A 307       9.671  23.967  20.123  1.0   0.00      PROT O \n",
            "ATOM   1542  H   GLY A 307      12.383  27.230  20.771  1.0   0.00      PROT H \n",
            "ATOM   1543  HA2 GLY A 307      11.451  26.079  19.062  1.0   0.00      PROT H \n",
            "ATOM   1544  HA3 GLY A 307       9.836  26.447  19.714  1.0   0.00      PROT H \n",
            "ATOM   1545  N   CYS A 308      11.661  24.083  21.149  1.0   0.00      PROT N \n",
            "ATOM   1546  CA  CYS A 308      11.613  22.693  21.591  1.0   0.00      PROT C \n",
            "ATOM   1547  C   CYS A 308      11.274  22.480  23.055  1.0   0.00      PROT C \n",
            "ATOM   1548  O   CYS A 308      11.002  21.355  23.460  1.0   0.00      PROT O \n",
            "ATOM   1549  CB  CYS A 308      12.949  22.010  21.322  1.0   0.00      PROT C \n",
            "ATOM   1550  SG  CYS A 308      14.223  22.457  22.511  1.0   0.00      PROT S \n",
            "ATOM   1551  H   CYS A 308      12.455  24.651  21.408  1.0   0.00      PROT H \n",
            "ATOM   1552  HA  CYS A 308      10.787  22.303  20.996  1.0   0.00      PROT H \n",
            "ATOM   1553  HB2 CYS A 308      12.820  20.929  21.368  1.0   0.00      PROT H \n",
            "ATOM   1554  HB3 CYS A 308      13.308  22.292  20.332  1.0   0.00      PROT H \n",
            "ATOM   1555  HG  CYS A 308      15.452  21.935  22.404  1.0   0.00      PROT H \n",
            "ATOM   1556  N   CYS A 309      11.290  23.541  23.850  1.0   0.00      PROT N \n",
            "ATOM   1557  CA  CYS A 309      11.002  23.417  25.277  1.0   0.00      PROT C \n",
            "ATOM   1558  C   CYS A 309       9.850  22.465  25.590  1.0   0.00      PROT C \n",
            "ATOM   1559  O   CYS A 309      10.021  21.492  26.332  1.0   0.00      PROT O \n",
            "ATOM   1560  CB  CYS A 309      10.703  24.789  25.867  1.0   0.00      PROT C \n",
            "ATOM   1561  SG  CYS A 309      10.636  24.803  27.653  1.0   0.00      PROT S \n",
            "ATOM   1562  H   CYS A 309      11.504  24.453  23.471  1.0   0.00      PROT H \n",
            "ATOM   1563  HA  CYS A 309      11.916  22.989  25.689  1.0   0.00      PROT H \n",
            "ATOM   1564  HB2 CYS A 309      11.480  25.491  25.566  1.0   0.00      PROT H \n",
            "ATOM   1565  HB3 CYS A 309       9.737  25.138  25.503  1.0   0.00      PROT H \n",
            "ATOM   1566  HG  CYS A 309      10.381  25.946  28.305  1.0   0.00      PROT H \n",
            "ATOM   1567  N   MET A 310       8.684  22.740  25.016  1.0   0.00      PROT N \n",
            "ATOM   1568  CA  MET A 310       7.510  21.908  25.250  1.0   0.00      PROT C \n",
            "ATOM   1569  C   MET A 310       7.566  20.546  24.578  1.0   0.00      PROT C \n",
            "ATOM   1570  O   MET A 310       6.892  19.618  25.012  1.0   0.00      PROT O \n",
            "ATOM   1571  CB  MET A 310       6.246  22.628  24.793  1.0   0.00      PROT C \n",
            "ATOM   1572  CG  MET A 310       4.980  21.850  25.067  1.0   0.00      PROT C \n",
            "ATOM   1573  SD  MET A 310       4.669  21.548  26.834  1.0   0.00      PROT S \n",
            "ATOM   1574  CE  MET A 310       3.050  20.854  26.730  1.0   0.00      PROT C \n",
            "ATOM   1575  H   MET A 310       8.598  23.539  24.404  1.0   0.00      PROT H \n",
            "ATOM   1576  HA  MET A 310       7.537  21.758  26.329  1.0   0.00      PROT H \n",
            "ATOM   1577  HB3 MET A 310       6.339  22.805  23.721  1.0   0.00      PROT H \n",
            "ATOM   1578  HB2 MET A 310       6.206  23.583  25.317  1.0   0.00      PROT H \n",
            "ATOM   1579  HG2 MET A 310       4.124  22.401  24.678  1.0   0.00      PROT H \n",
            "ATOM   1580  HG3 MET A 310       5.038  20.877  24.579  1.0   0.00      PROT H \n",
            "ATOM   1581  HE1 MET A 310       2.696  20.606  27.731  1.0   0.00      PROT H \n",
            "ATOM   1582  HE2 MET A 310       2.373  21.577  26.274  1.0   0.00      PROT H \n",
            "ATOM   1583  HE3 MET A 310       3.080  19.951  26.121  1.0   0.00      PROT H \n",
            "ATOM   1584  N   GLU A 311       8.354  20.418  23.515  1.0   0.00      PROT N \n",
            "ATOM   1585  CA  GLU A 311       8.457  19.136  22.829  1.0   0.00      PROT C \n",
            "ATOM   1586  C   GLU A 311       9.258  18.183  23.710  1.0   0.00      PROT C \n",
            "ATOM   1587  O   GLU A 311       8.932  17.003  23.828  1.0   0.00      PROT O \n",
            "ATOM   1588  CB  GLU A 311       9.132  19.322  21.472  1.0   0.00      PROT C \n",
            "ATOM   1589  CG  GLU A 311       8.675  20.586  20.773  1.0   0.00      PROT C \n",
            "ATOM   1590  CD  GLU A 311       9.082  20.635  19.329  1.0   0.00      PROT C \n",
            "ATOM   1591  OE1 GLU A 311      10.268  20.378  19.036  1.0   0.00      PROT O \n",
            "ATOM   1592  OE2 GLU A 311       8.210  20.940  18.488  1.0   0.00      PROT O \n",
            "ATOM   1593  H   GLU A 311       8.883  21.210  23.181  1.0   0.00      PROT H \n",
            "ATOM   1594  HA  GLU A 311       7.447  18.757  22.674  1.0   0.00      PROT H \n",
            "ATOM   1595  HB3 GLU A 311       8.891  18.457  20.853  1.0   0.00      PROT H \n",
            "ATOM   1596  HB2 GLU A 311      10.209  19.365  21.633  1.0   0.00      PROT H \n",
            "ATOM   1597  HG3 GLU A 311       9.110  21.434  21.302  1.0   0.00      PROT H \n",
            "ATOM   1598  HG2 GLU A 311       7.588  20.631  20.846  1.0   0.00      PROT H \n",
            "ATOM   1599  N   ILE A 312      10.293  18.713  24.349  1.0   0.00      PROT N \n",
            "ATOM   1600  CA  ILE A 312      11.136  17.926  25.237  1.0   0.00      PROT C \n",
            "ATOM   1601  C   ILE A 312      10.438  17.664  26.564  1.0   0.00      PROT C \n",
            "ATOM   1602  O   ILE A 312      10.520  16.558  27.105  1.0   0.00      PROT O \n",
            "ATOM   1603  CB  ILE A 312      12.487  18.632  25.484  1.0   0.00      PROT C \n",
            "ATOM   1604  CG1 ILE A 312      13.247  18.743  24.155  1.0   0.00      PROT C \n",
            "ATOM   1605  CG2 ILE A 312      13.301  17.862  26.511  1.0   0.00      PROT C \n",
            "ATOM   1606  CD1 ILE A 312      14.484  19.599  24.206  1.0   0.00      PROT C \n",
            "ATOM   1607  H   ILE A 312      10.516  19.690  24.226  1.0   0.00      PROT H \n",
            "ATOM   1608  HA  ILE A 312      11.282  16.982  24.711  1.0   0.00      PROT H \n",
            "ATOM   1609  HB  ILE A 312      12.324  19.632  25.886  1.0   0.00      PROT H \n",
            "ATOM   1610 HG13 ILE A 312      12.549  19.156  23.427  1.0   0.00      PROT H \n",
            "ATOM   1611 HG12 ILE A 312      13.519  17.729  23.862  1.0   0.00      PROT H \n",
            "ATOM   1612 HG21 ILE A 312      12.741  17.797  27.444  1.0   0.00      PROT H \n",
            "ATOM   1613 HG22 ILE A 312      13.502  16.858  26.137  1.0   0.00      PROT H \n",
            "ATOM   1614 HG23 ILE A 312      14.244  18.379  26.689  1.0   0.00      PROT H \n",
            "ATOM   1615 HD11 ILE A 312      14.953  19.618  23.222  1.0   0.00      PROT H \n",
            "ATOM   1616 HD12 ILE A 312      14.212  20.613  24.499  1.0   0.00      PROT H \n",
            "ATOM   1617 HD13 ILE A 312      15.182  19.186  24.934  1.0   0.00      PROT H \n",
            "ATOM   1618  N   MET A 313       9.741  18.667  27.092  1.0   0.00      PROT N \n",
            "ATOM   1619  CA  MET A 313       9.031  18.482  28.353  1.0   0.00      PROT C \n",
            "ATOM   1620  C   MET A 313       7.905  17.460  28.195  1.0   0.00      PROT C \n",
            "ATOM   1621  O   MET A 313       7.683  16.637  29.078  1.0   0.00      PROT O \n",
            "ATOM   1622  CB  MET A 313       8.447  19.800  28.856  1.0   0.00      PROT C \n",
            "ATOM   1623  CG  MET A 313       9.475  20.873  29.164  1.0   0.00      PROT C \n",
            "ATOM   1624  SD  MET A 313       8.776  22.210  30.148  1.0   0.00      PROT S \n",
            "ATOM   1625  CE  MET A 313       7.745  22.994  28.934  1.0   0.00      PROT C \n",
            "ATOM   1626  H   MET A 313       9.703  19.559  26.620  1.0   0.00      PROT H \n",
            "ATOM   1627  HA  MET A 313       9.789  18.118  29.047  1.0   0.00      PROT H \n",
            "ATOM   1628  HB3 MET A 313       7.881  19.569  29.758  1.0   0.00      PROT H \n",
            "ATOM   1629  HB2 MET A 313       7.768  20.159  28.083  1.0   0.00      PROT H \n",
            "ATOM   1630  HG2 MET A 313      10.302  20.434  29.723  1.0   0.00      PROT H \n",
            "ATOM   1631  HG3 MET A 313       9.850  21.295  28.232  1.0   0.00      PROT H \n",
            "ATOM   1632  HE1 MET A 313       7.238  23.847  29.385  1.0   0.00      PROT H \n",
            "ATOM   1633  HE2 MET A 313       8.360  23.335  28.101  1.0   0.00      PROT H \n",
            "ATOM   1634  HE3 MET A 313       7.005  22.281  28.571  1.0   0.00      PROT H \n",
            "ATOM   1635  N   SER A 314       7.195  17.508  27.074  1.0   0.00      PROT N \n",
            "ATOM   1636  CA  SER A 314       6.106  16.566  26.839  1.0   0.00      PROT C \n",
            "ATOM   1637  C   SER A 314       6.663  15.153  26.713  1.0   0.00      PROT C \n",
            "ATOM   1638  O   SER A 314       6.056  14.194  27.184  1.0   0.00      PROT O \n",
            "ATOM   1639  CB  SER A 314       5.335  16.939  25.568  1.0   0.00      PROT C \n",
            "ATOM   1640  OG  SER A 314       4.739  18.222  25.678  1.0   0.00      PROT O \n",
            "ATOM   1641  H   SER A 314       7.406  18.202  26.372  1.0   0.00      PROT H \n",
            "ATOM   1642  HA  SER A 314       5.452  16.639  27.708  1.0   0.00      PROT H \n",
            "ATOM   1643  HB2 SER A 314       6.018  16.948  24.718  1.0   0.00      PROT H \n",
            "ATOM   1644  HB3 SER A 314       4.547  16.207  25.393  1.0   0.00      PROT H \n",
            "ATOM   1645  HG  SER A 314       4.210  18.588  24.926  1.0   0.00      PROT H \n",
            "ATOM   1646  N   LEU A 315       7.817  15.026  26.064  1.0   0.00      PROT N \n",
            "ATOM   1647  CA  LEU A 315       8.465  13.723  25.912  1.0   0.00      PROT C \n",
            "ATOM   1648  C   LEU A 315       8.857  13.215  27.301  1.0   0.00      PROT C \n",
            "ATOM   1649  O   LEU A 315       8.613  12.053  27.645  1.0   0.00      PROT O \n",
            "ATOM   1650  CB  LEU A 315       9.721  13.842  25.039  1.0   0.00      PROT C \n",
            "ATOM   1651  CG  LEU A 315      10.750  12.709  25.145  1.0   0.00      PROT C \n",
            "ATOM   1652  CD1 LEU A 315      10.105  11.392  24.751  1.0   0.00      PROT C \n",
            "ATOM   1653  CD2 LEU A 315      11.947  13.012  24.258  1.0   0.00      PROT C \n",
            "ATOM   1654  H   LEU A 315       8.264  15.839  25.664  1.0   0.00      PROT H \n",
            "ATOM   1655  HA  LEU A 315       7.743  13.063  25.430  1.0   0.00      PROT H \n",
            "ATOM   1656  HB3 LEU A 315      10.188  14.785  25.323  1.0   0.00      PROT H \n",
            "ATOM   1657  HB2 LEU A 315       9.360  13.910  24.013  1.0   0.00      PROT H \n",
            "ATOM   1658  HG  LEU A 315      11.103  12.621  26.172  1.0   0.00      PROT H \n",
            "ATOM   1659 HD11 LEU A 315       9.251  11.197  25.400  1.0   0.00      PROT H \n",
            "ATOM   1660 HD12 LEU A 315       9.769  11.447  23.715  1.0   0.00      PROT H \n",
            "ATOM   1661 HD13 LEU A 315      10.832  10.586  24.855  1.0   0.00      PROT H \n",
            "ATOM   1662 HD21 LEU A 315      12.392  13.960  24.560  1.0   0.00      PROT H \n",
            "ATOM   1663 HD22 LEU A 315      12.685  12.216  24.359  1.0   0.00      PROT H \n",
            "ATOM   1664 HD23 LEU A 315      11.622  13.077  23.219  1.0   0.00      PROT H \n",
            "ATOM   1665  N   ARG A 316       9.465  14.100  28.093  1.0   0.00      PROT N \n",
            "ATOM   1666  CA  ARG A 316       9.885  13.764  29.448  1.0   0.00      PROT C \n",
            "ATOM   1667  C   ARG A 316       8.708  13.337  30.312  1.0   0.00      PROT C \n",
            "ATOM   1668  O   ARG A 316       8.862  12.498  31.192  1.0   0.00      PROT O \n",
            "ATOM   1669  CB  ARG A 316      10.607  14.945  30.093  1.0   0.00      PROT C \n",
            "ATOM   1670  CG  ARG A 316      11.977  15.181  29.518  1.0   0.00      PROT C \n",
            "ATOM   1671  CD  ARG A 316      12.737  16.234  30.283  1.0   0.00      PROT C \n",
            "ATOM   1672  NE  ARG A 316      14.173  16.095  30.069  1.0   0.00      PROT N \n",
            "ATOM   1673  CZ  ARG A 316      15.100  16.781  30.729  1.0   0.00      PROT C \n",
            "ATOM   1674  NH2 ARG A 316      16.384  16.577  30.471  1.0   0.00      PROT N \n",
            "ATOM   1675  NH1 ARG A 316      14.744  17.670  31.642  1.0   0.00      PROT N \n",
            "ATOM   1676  H   ARG A 316       9.647  15.035  27.759  1.0   0.00      PROT H \n",
            "ATOM   1677  HA  ARG A 316      10.563  12.919  29.329  1.0   0.00      PROT H \n",
            "ATOM   1678  HB3 ARG A 316      10.691  14.739  31.160  1.0   0.00      PROT H \n",
            "ATOM   1679  HB2 ARG A 316       9.992  15.832  29.942  1.0   0.00      PROT H \n",
            "ATOM   1680  HG2 ARG A 316      12.550  14.255  29.553  1.0   0.00      PROT H \n",
            "ATOM   1681  HG3 ARG A 316      11.885  15.512  28.484  1.0   0.00      PROT H \n",
            "ATOM   1682  HD2 ARG A 316      12.426  17.223  29.947  1.0   0.00      PROT H \n",
            "ATOM   1683  HD3 ARG A 316      12.529  16.130  31.348  1.0   0.00      PROT H \n",
            "ATOM   1684  HE  ARG A 316      14.447  15.424  29.365  1.0   0.00      PROT H \n",
            "ATOM   1685 HH22 ARG A 316      17.096  17.105  30.976  1.0   0.00      PROT H \n",
            "ATOM   1686 HH21 ARG A 316      16.660  15.892  29.767  1.0   0.00      PROT H \n",
            "ATOM   1687  HH1 ARG A 316      15.457  18.197  32.146  1.0   0.00      PROT H \n",
            "ATOM   1688  N   ALA A 317       7.532  13.902  30.070  1.0   0.00      PROT N \n",
            "ATOM   1689  CA  ALA A 317       6.370  13.511  30.854  1.0   0.00      PROT C \n",
            "ATOM   1690  C   ALA A 317       5.822  12.178  30.339  1.0   0.00      PROT C \n",
            "ATOM   1691  O   ALA A 317       5.467  11.294  31.126  1.0   0.00      PROT O \n",
            "ATOM   1692  CB  ALA A 317       5.300  14.580  30.779  1.0   0.00      PROT C \n",
            "ATOM   1693  H   ALA A 317       7.447  14.600  29.345  1.0   0.00      PROT H \n",
            "ATOM   1694  HA  ALA A 317       6.712  13.416  31.885  1.0   0.00      PROT H \n",
            "ATOM   1695  HB1 ALA A 317       5.702  15.523  31.148  1.0   0.00      PROT H \n",
            "ATOM   1696  HB2 ALA A 317       4.980  14.702  29.744  1.0   0.00      PROT H \n",
            "ATOM   1697  HB3 ALA A 317       4.448  14.284  31.391  1.0   0.00      PROT H \n",
            "ATOM   1698  N   ALA A 318       5.773  12.046  29.013  1.0   0.00      PROT N \n",
            "ATOM   1699  CA  ALA A 318       5.276  10.848  28.336  1.0   0.00      PROT C \n",
            "ATOM   1700  C   ALA A 318       5.992   9.550  28.719  1.0   0.00      PROT C \n",
            "ATOM   1701  O   ALA A 318       5.357   8.505  28.856  1.0   0.00      PROT O \n",
            "ATOM   1702  CB  ALA A 318       5.341  11.049  26.811  1.0   0.00      PROT C \n",
            "ATOM   1703  H   ALA A 318       6.090  12.799  28.419  1.0   0.00      PROT H \n",
            "ATOM   1704  HA  ALA A 318       4.255  10.767  28.708  1.0   0.00      PROT H \n",
            "ATOM   1705  HB1 ALA A 318       4.830  11.974  26.544  1.0   0.00      PROT H \n",
            "ATOM   1706  HB2 ALA A 318       6.383  11.106  26.496  1.0   0.00      PROT H \n",
            "ATOM   1707  HB3 ALA A 318       4.856  10.209  26.313  1.0   0.00      PROT H \n",
            "ATOM   1708  N   VAL A 319       7.314   9.607  28.874  1.0   0.00      PROT N \n",
            "ATOM   1709  CA  VAL A 319       8.083   8.412  29.243  1.0   0.00      PROT C \n",
            "ATOM   1710  C   VAL A 319       7.838   8.025  30.703  1.0   0.00      PROT C \n",
            "ATOM   1711  O   VAL A 319       8.159   6.907  31.119  1.0   0.00      PROT O \n",
            "ATOM   1712  CB  VAL A 319       9.602   8.617  29.037  1.0   0.00      PROT C \n",
            "ATOM   1713  CG1 VAL A 319       9.899   8.842  27.564  1.0   0.00      PROT C \n",
            "ATOM   1714  CG2 VAL A 319      10.081   9.787  29.858  1.0   0.00      PROT C \n",
            "ATOM   1715  H   VAL A 319       7.801  10.481  28.737  1.0   0.00      PROT H \n",
            "ATOM   1716  HA  VAL A 319       7.700   7.644  28.572  1.0   0.00      PROT H \n",
            "ATOM   1717  HB  VAL A 319      10.152   7.736  29.367  1.0   0.00      PROT H \n",
            "ATOM   1718 HG11 VAL A 319       9.546   7.987  26.987  1.0   0.00      PROT H \n",
            "ATOM   1719 HG12 VAL A 319       9.390   9.744  27.224  1.0   0.00      PROT H \n",
            "ATOM   1720 HG13 VAL A 319      10.974   8.956  27.423  1.0   0.00      PROT H \n",
            "ATOM   1721 HG21 VAL A 319       9.860   9.608  30.910  1.0   0.00      PROT H \n",
            "ATOM   1722 HG22 VAL A 319      11.157   9.906  29.728  1.0   0.00      PROT H \n",
            "ATOM   1723 HG23 VAL A 319       9.573  10.694  29.529  1.0   0.00      PROT H \n",
            "ATOM   1724  N   ARG A 320       7.277   8.956  31.474  1.0   0.00      PROT N \n",
            "ATOM   1725  CA  ARG A 320       6.966   8.699  32.870  1.0   0.00      PROT C \n",
            "ATOM   1726  C   ARG A 320       5.479   8.447  33.060  1.0   0.00      PROT C \n",
            "ATOM   1727  O   ARG A 320       4.897   8.828  34.074  1.0   0.00      PROT O \n",
            "ATOM   1728  CB  ARG A 320       7.420   9.853  33.769  1.0   0.00      PROT C \n",
            "ATOM   1729  CG  ARG A 320       8.913   9.898  33.961  1.0   0.00      PROT C \n",
            "ATOM   1730  CD  ARG A 320       9.310  10.478  35.307  1.0   0.00      PROT C \n",
            "ATOM   1731  NE  ARG A 320       9.960  11.767  35.122  1.0   0.00      PROT N \n",
            "ATOM   1732  CZ  ARG A 320      11.214  12.048  35.458  1.0   0.00      PROT C \n",
            "ATOM   1733  NH1 ARG A 320      12.002  11.136  36.020  1.0   0.00      PROT N \n",
            "ATOM   1734  NH2 ARG A 320      11.674  13.260  35.205  1.0   0.00      PROT N \n",
            "ATOM   1735  H   ARG A 320       7.058   9.865  31.091  1.0   0.00      PROT H \n",
            "ATOM   1736  HA  ARG A 320       7.528   7.796  33.109  1.0   0.00      PROT H \n",
            "ATOM   1737  HB3 ARG A 320       6.933   9.730  34.736  1.0   0.00      PROT H \n",
            "ATOM   1738  HB2 ARG A 320       7.085  10.782  33.309  1.0   0.00      PROT H \n",
            "ATOM   1739  HG2 ARG A 320       9.318   8.888  33.899  1.0   0.00      PROT H \n",
            "ATOM   1740  HG3 ARG A 320       9.361  10.517  33.184  1.0   0.00      PROT H \n",
            "ATOM   1741  HD2 ARG A 320       8.420  10.610  35.922  1.0   0.00      PROT H \n",
            "ATOM   1742  HD3 ARG A 320       9.999   9.797  35.807  1.0   0.00      PROT H \n",
            "ATOM   1743  HE  ARG A 320       9.378  12.478  34.702  1.0   0.00      PROT H \n",
            "ATOM   1744 HH12 ARG A 320      12.966  11.371  36.257  1.0   0.00      PROT H \n",
            "ATOM   1745 HH11 ARG A 320      11.643  10.201  36.214  1.0   0.00      PROT H \n",
            "ATOM   1746  HH2 ARG A 320      12.637  13.499  35.441  1.0   0.00      PROT H \n",
            "ATOM   1747  N   TYR A 321       4.868   7.799  32.072  1.0   0.00      PROT N \n",
            "ATOM   1748  CA  TYR A 321       3.452   7.457  32.146  1.0   0.00      PROT C \n",
            "ATOM   1749  C   TYR A 321       3.313   6.139  32.905  1.0   0.00      PROT C \n",
            "ATOM   1750  O   TYR A 321       4.061   5.194  32.661  1.0   0.00      PROT O \n",
            "ATOM   1751  CB  TYR A 321       2.852   7.301  30.741  1.0   0.00      PROT C \n",
            "ATOM   1752  CG  TYR A 321       1.417   6.819  30.747  1.0   0.00      PROT C \n",
            "ATOM   1753  CD1 TYR A 321       0.393   7.621  31.250  1.0   0.00      PROT C \n",
            "ATOM   1754  CD2 TYR A 321       1.085   5.545  30.289  1.0   0.00      PROT C \n",
            "ATOM   1755  CE1 TYR A 321      -0.924   7.170  31.309  1.0   0.00      PROT C \n",
            "ATOM   1756  CE2 TYR A 321      -0.234   5.078  30.339  1.0   0.00      PROT C \n",
            "ATOM   1757  CZ  TYR A 321      -1.236   5.901  30.855  1.0   0.00      PROT C \n",
            "ATOM   1758  OH  TYR A 321      -2.541   5.455  30.950  1.0   0.00      PROT O \n",
            "ATOM   1759  H   TYR A 321       5.386   7.532  31.247  1.0   0.00      PROT H \n",
            "ATOM   1760  HA  TYR A 321       2.961   8.278  32.668  1.0   0.00      PROT H \n",
            "ATOM   1761  HB3 TYR A 321       3.473   6.588  30.199  1.0   0.00      PROT H \n",
            "ATOM   1762  HB2 TYR A 321       2.906   8.274  30.253  1.0   0.00      PROT H \n",
            "ATOM   1763  HD1 TYR A 321       0.643   8.622  31.602  1.0   0.00      PROT H \n",
            "ATOM   1764  HD2 TYR A 321       1.858   4.893  29.883  1.0   0.00      PROT H \n",
            "ATOM   1765  HE1 TYR A 321      -1.707   7.813  31.711  1.0   0.00      PROT H \n",
            "ATOM   1766  HE2 TYR A 321      -0.481   4.079  29.979  1.0   0.00      PROT H \n",
            "ATOM   1767  HH  TYR A 321      -3.242   6.050  31.318  1.0   0.00      PROT H \n",
            "ATOM   1768  N   ASP A 322       2.361   6.082  33.829  1.0   0.00      PROT N \n",
            "ATOM   1769  CA  ASP A 322       2.127   4.878  34.616  1.0   0.00      PROT C \n",
            "ATOM   1770  C   ASP A 322       0.753   4.306  34.267  1.0   0.00      PROT C \n",
            "ATOM   1771  O   ASP A 322      -0.269   4.872  34.642  1.0   0.00      PROT O \n",
            "ATOM   1772  CB  ASP A 322       2.201   5.228  36.094  1.0   0.00      PROT C \n",
            "ATOM   1773  CG  ASP A 322       1.837   4.076  36.984  1.0   0.00      PROT C \n",
            "ATOM   1774  OD1 ASP A 322       2.550   3.050  36.960  1.0   0.00      PROT O \n",
            "ATOM   1775  OD2 ASP A 322       0.832   4.201  37.717  1.0   0.00      PROT O \n",
            "ATOM   1776  H   ASP A 322       1.776   6.887  34.002  1.0   0.00      PROT H \n",
            "ATOM   1777  HA  ASP A 322       2.907   4.159  34.366  1.0   0.00      PROT H \n",
            "ATOM   1778  HB3 ASP A 322       1.516   6.057  36.274  1.0   0.00      PROT H \n",
            "ATOM   1779  HB2 ASP A 322       3.221   5.546  36.310  1.0   0.00      PROT H \n",
            "ATOM   1780  N   PRO A 323       0.712   3.175  33.538  1.0   0.00      PROT N \n",
            "ATOM   1781  CA  PRO A 323      -0.557   2.545  33.141  1.0   0.00      PROT C \n",
            "ATOM   1782  C   PRO A 323      -1.437   2.154  34.331  1.0   0.00      PROT C \n",
            "ATOM   1783  O   PRO A 323      -2.656   2.057  34.203  1.0   0.00      PROT O \n",
            "ATOM   1784  CB  PRO A 323      -0.108   1.317  32.343  1.0   0.00      PROT C \n",
            "ATOM   1785  CG  PRO A 323       1.265   1.694  31.859  1.0   0.00      PROT C \n",
            "ATOM   1786  CD  PRO A 323       1.856   2.383  33.057  1.0   0.00      PROT C \n",
            "ATOM   1787  HA  PRO A 323      -1.159   3.255  32.574  1.0   0.00      PROT H \n",
            "ATOM   1788  HB3 PRO A 323      -0.754   1.120  31.487  1.0   0.00      PROT H \n",
            "ATOM   1789  HB2 PRO A 323      -0.042   0.422  32.962  1.0   0.00      PROT H \n",
            "ATOM   1790  HG3 PRO A 323       1.241   2.386  31.017  1.0   0.00      PROT H \n",
            "ATOM   1791  HG2 PRO A 323       1.874   0.828  31.600  1.0   0.00      PROT H \n",
            "ATOM   1792  HD2 PRO A 323       2.195   1.636  33.775  1.0   0.00      PROT H \n",
            "ATOM   1793  HD3 PRO A 323       2.702   2.995  32.744  1.0   0.00      PROT H \n",
            "ATOM   1794  N   GLU A 324      -0.804   1.932  35.481  1.0   0.00      PROT N \n",
            "ATOM   1795  CA  GLU A 324      -1.505   1.544  36.701  1.0   0.00      PROT C \n",
            "ATOM   1796  C   GLU A 324      -2.477   2.637  37.152  1.0   0.00      PROT C \n",
            "ATOM   1797  O   GLU A 324      -3.691   2.475  37.058  1.0   0.00      PROT O \n",
            "ATOM   1798  CB  GLU A 324      -0.483   1.247  37.806  1.0   0.00      PROT C \n",
            "ATOM   1799  CG  GLU A 324      -1.055   0.594  39.063  1.0   0.00      PROT C \n",
            "ATOM   1800  CD  GLU A 324      -1.800  -0.706  38.773  1.0   0.00      PROT C \n",
            "ATOM   1801  OE1 GLU A 324      -2.987  -0.645  38.381  1.0   0.00      PROT O \n",
            "ATOM   1802  OE2 GLU A 324      -1.195  -1.790  38.927  1.0   0.00      PROT O \n",
            "ATOM   1803  H   GLU A 324       0.200   2.031  35.532  1.0   0.00      PROT H \n",
            "ATOM   1804  HA  GLU A 324      -2.073   0.649  36.447  1.0   0.00      PROT H \n",
            "ATOM   1805  HB3 GLU A 324      -0.024   2.200  38.068  1.0   0.00      PROT H \n",
            "ATOM   1806  HB2 GLU A 324       0.268   0.591  37.366  1.0   0.00      PROT H \n",
            "ATOM   1807  HG3 GLU A 324      -1.736   1.308  39.526  1.0   0.00      PROT H \n",
            "ATOM   1808  HG2 GLU A 324      -0.224   0.393  39.739  1.0   0.00      PROT H \n",
            "ATOM   1809  HE1 GLU A 324      -3.343  -1.566  38.223  1.0   0.00      PROT H \n",
            "ATOM   1810  N   SER A 325      -1.934   3.750  37.631  1.0   0.00      PROT N \n",
            "ATOM   1811  CA  SER A 325      -2.736   4.876  38.092  1.0   0.00      PROT C \n",
            "ATOM   1812  C   SER A 325      -3.220   5.728  36.925  1.0   0.00      PROT C \n",
            "ATOM   1813  O   SER A 325      -4.108   6.564  37.083  1.0   0.00      PROT O \n",
            "ATOM   1814  CB  SER A 325      -1.909   5.743  39.039  1.0   0.00      PROT C \n",
            "ATOM   1815  OG  SER A 325      -0.746   6.239  38.391  1.0   0.00      PROT O \n",
            "ATOM   1816  H   SER A 325      -0.929   3.839  37.686  1.0   0.00      PROT H \n",
            "ATOM   1817  HA  SER A 325      -3.592   4.433  38.601  1.0   0.00      PROT H \n",
            "ATOM   1818  HB2 SER A 325      -2.509   6.589  39.375  1.0   0.00      PROT H \n",
            "ATOM   1819  HB3 SER A 325      -1.603   5.150  39.901  1.0   0.00      PROT H \n",
            "ATOM   1820  HG  SER A 325      -0.113   6.818  38.885  1.0   0.00      PROT H \n",
            "ATOM   1821  N   GLU A 326      -2.628   5.508  35.757  1.0   0.00      PROT N \n",
            "ATOM   1822  CA  GLU A 326      -2.972   6.258  34.557  1.0   0.00      PROT C \n",
            "ATOM   1823  C   GLU A 326      -2.676   7.731  34.743  1.0   0.00      PROT C \n",
            "ATOM   1824  O   GLU A 326      -3.488   8.584  34.400  1.0   0.00      PROT O \n",
            "ATOM   1825  CB  GLU A 326      -4.445   6.054  34.201  1.0   0.00      PROT C \n",
            "ATOM   1826  CG  GLU A 326      -4.698   4.707  33.546  1.0   0.00      PROT C \n",
            "ATOM   1827  CD  GLU A 326      -6.164   4.408  33.341  1.0   0.00      PROT C \n",
            "ATOM   1828  OE1 GLU A 326      -6.879   5.259  32.769  1.0   0.00      PROT O \n",
            "ATOM   1829  OE2 GLU A 326      -6.600   3.312  33.746  1.0   0.00      PROT O \n",
            "ATOM   1830  H   GLU A 326      -1.912   4.800  35.679  1.0   0.00      PROT H \n",
            "ATOM   1831  HA  GLU A 326      -2.334   5.860  33.768  1.0   0.00      PROT H \n",
            "ATOM   1832  HB3 GLU A 326      -4.741   6.851  33.519  1.0   0.00      PROT H \n",
            "ATOM   1833  HB2 GLU A 326      -5.028   6.127  35.119  1.0   0.00      PROT H \n",
            "ATOM   1834  HG3 GLU A 326      -4.257   3.943  34.186  1.0   0.00      PROT H \n",
            "ATOM   1835  HG2 GLU A 326      -4.190   4.710  32.582  1.0   0.00      PROT H \n",
            "ATOM   1836  HE1 GLU A 326      -7.818   4.923  32.696  1.0   0.00      PROT H \n",
            "ATOM   1837  N   THR A 327      -1.497   8.022  35.278  1.0   0.00      PROT N \n",
            "ATOM   1838  CA  THR A 327      -1.089   9.398  35.524  1.0   0.00      PROT C \n",
            "ATOM   1839  C   THR A 327       0.278   9.674  34.923  1.0   0.00      PROT C \n",
            "ATOM   1840  O   THR A 327       1.025   8.748  34.611  1.0   0.00      PROT O \n",
            "ATOM   1841  CB  THR A 327      -1.014   9.705  37.046  1.0   0.00      PROT C \n",
            "ATOM   1842  OG1 THR A 327       0.069   8.977  37.634  1.0   0.00      PROT O \n",
            "ATOM   1843  CG2 THR A 327      -2.306   9.300  37.738  1.0   0.00      PROT C \n",
            "ATOM   1844  H   THR A 327      -0.858   7.280  35.525  1.0   0.00      PROT H \n",
            "ATOM   1845  HA  THR A 327      -1.857   9.987  35.023  1.0   0.00      PROT H \n",
            "ATOM   1846  HB  THR A 327      -0.806  10.760  37.223  1.0   0.00      PROT H \n",
            "ATOM   1847  HG1 THR A 327       0.993   9.114  37.306  1.0   0.00      PROT H \n",
            "ATOM   1848 HG21 THR A 327      -2.233   9.523  38.802  1.0   0.00      PROT H \n",
            "ATOM   1849 HG22 THR A 327      -3.139   9.855  37.306  1.0   0.00      PROT H \n",
            "ATOM   1850 HG23 THR A 327      -2.472   8.231  37.601  1.0   0.00      PROT H \n",
            "ATOM   1851  N   LEU A 328       0.588  10.952  34.731  1.0   0.00      PROT N \n",
            "ATOM   1852  CA  LEU A 328       1.890  11.353  34.214  1.0   0.00      PROT C \n",
            "ATOM   1853  C   LEU A 328       2.547  12.033  35.401  1.0   0.00      PROT C \n",
            "ATOM   1854  O   LEU A 328       1.893  12.775  36.135  1.0   0.00      PROT O \n",
            "ATOM   1855  CB  LEU A 328       1.777  12.361  33.054  1.0   0.00      PROT C \n",
            "ATOM   1856  CG  LEU A 328       1.371  11.928  31.633  1.0   0.00      PROT C \n",
            "ATOM   1857  CD1 LEU A 328       1.371  13.140  30.698  1.0   0.00      PROT C \n",
            "ATOM   1858  CD2 LEU A 328       2.338  10.872  31.113  1.0   0.00      PROT C \n",
            "ATOM   1859  H   LEU A 328      -0.086  11.673  34.945  1.0   0.00      PROT H \n",
            "ATOM   1860  HA  LEU A 328       2.410  10.471  33.841  1.0   0.00      PROT H \n",
            "ATOM   1861  HB3 LEU A 328       2.769  12.813  33.027  1.0   0.00      PROT H \n",
            "ATOM   1862  HB2 LEU A 328       1.057  13.087  33.432  1.0   0.00      PROT H \n",
            "ATOM   1863  HG  LEU A 328       0.370  11.498  31.658  1.0   0.00      PROT H \n",
            "ATOM   1864 HD11 LEU A 328       0.677  13.891  31.076  1.0   0.00      PROT H \n",
            "ATOM   1865 HD12 LEU A 328       2.374  13.563  30.652  1.0   0.00      PROT H \n",
            "ATOM   1866 HD13 LEU A 328       1.061  12.829  29.700  1.0   0.00      PROT H \n",
            "ATOM   1867 HD21 LEU A 328       2.333  10.013  31.784  1.0   0.00      PROT H \n",
            "ATOM   1868 HD22 LEU A 328       2.030  10.557  30.116  1.0   0.00      PROT H \n",
            "ATOM   1869 HD23 LEU A 328       3.343  11.291  31.067  1.0   0.00      PROT H \n",
            "ATOM   1870  N   THR A 329       3.827  11.753  35.614  1.0   0.00      PROT N \n",
            "ATOM   1871  CA  THR A 329       4.552  12.376  36.707  1.0   0.00      PROT C \n",
            "ATOM   1872  C   THR A 329       5.330  13.559  36.148  1.0   0.00      PROT C \n",
            "ATOM   1873  O   THR A 329       6.255  13.380  35.353  1.0   0.00      PROT O \n",
            "ATOM   1874  CB  THR A 329       5.514  11.389  37.391  1.0   0.00      PROT C \n",
            "ATOM   1875  OG1 THR A 329       4.760  10.466  38.184  1.0   0.00      PROT O \n",
            "ATOM   1876  CG2 THR A 329       6.483  12.129  38.290  1.0   0.00      PROT C \n",
            "ATOM   1877  H   THR A 329       4.308  11.100  35.012  1.0   0.00      PROT H \n",
            "ATOM   1878  HA  THR A 329       3.797  12.693  37.427  1.0   0.00      PROT H \n",
            "ATOM   1879  HB  THR A 329       6.060  10.801  36.654  1.0   0.00      PROT H \n",
            "ATOM   1880  HG1 THR A 329       4.074   9.893  37.758  1.0   0.00      PROT H \n",
            "ATOM   1881 HG21 THR A 329       7.155  11.415  38.765  1.0   0.00      PROT H \n",
            "ATOM   1882 HG22 THR A 329       7.063  12.835  37.696  1.0   0.00      PROT H \n",
            "ATOM   1883 HG23 THR A 329       5.927  12.670  39.056  1.0   0.00      PROT H \n",
            "ATOM   1884  N   LEU A 330       4.933  14.761  36.574  1.0   0.00      PROT N \n",
            "ATOM   1885  CA  LEU A 330       5.548  16.013  36.130  1.0   0.00      PROT C \n",
            "ATOM   1886  C   LEU A 330       6.660  16.508  37.047  1.0   0.00      PROT C \n",
            "ATOM   1887  O   LEU A 330       6.566  16.389  38.262  1.0   0.00      PROT O \n",
            "ATOM   1888  CB  LEU A 330       4.482  17.104  36.000  1.0   0.00      PROT C \n",
            "ATOM   1889  CG  LEU A 330       3.285  16.787  35.104  1.0   0.00      PROT C \n",
            "ATOM   1890  CD1 LEU A 330       2.409  18.019  35.023  1.0   0.00      PROT C \n",
            "ATOM   1891  CD2 LEU A 330       3.743  16.360  33.719  1.0   0.00      PROT C \n",
            "ATOM   1892  H   LEU A 330       4.173  14.837  37.235  1.0   0.00      PROT H \n",
            "ATOM   1893  HA  LEU A 330       5.992  15.752  35.169  1.0   0.00      PROT H \n",
            "ATOM   1894  HB3 LEU A 330       5.001  17.982  35.616  1.0   0.00      PROT H \n",
            "ATOM   1895  HB2 LEU A 330       4.134  17.302  37.014  1.0   0.00      PROT H \n",
            "ATOM   1896  HG  LEU A 330       2.711  15.961  35.524  1.0   0.00      PROT H \n",
            "ATOM   1897 HD11 LEU A 330       2.093  18.307  36.026  1.0   0.00      PROT H \n",
            "ATOM   1898 HD12 LEU A 330       2.972  18.836  34.571  1.0   0.00      PROT H \n",
            "ATOM   1899 HD13 LEU A 330       1.532  17.802  34.414  1.0   0.00      PROT H \n",
            "ATOM   1900 HD21 LEU A 330       4.371  15.473  33.801  1.0   0.00      PROT H \n",
            "ATOM   1901 HD22 LEU A 330       2.873  16.133  33.103  1.0   0.00      PROT H \n",
            "ATOM   1902 HD23 LEU A 330       4.313  17.168  33.260  1.0   0.00      PROT H \n",
            "ATOM   1903  N   ASN A 331       7.700  17.080  36.447  1.0   0.00      PROT N \n",
            "ATOM   1904  CA  ASN A 331       8.847  17.590  37.175  1.0   0.00      PROT C \n",
            "ATOM   1905  C   ASN A 331       9.403  16.475  38.061  1.0   0.00      PROT C \n",
            "ATOM   1906  O   ASN A 331      10.109  16.730  39.026  1.0   0.00      PROT O \n",
            "ATOM   1907  CB  ASN A 331       8.465  18.814  38.022  1.0   0.00      PROT C \n",
            "ATOM   1908  CG  ASN A 331       9.687  19.571  38.556  1.0   0.00      PROT C \n",
            "ATOM   1909  OD1 ASN A 331      10.531  20.033  37.799  1.0   0.00      PROT O \n",
            "ATOM   1910  ND2 ASN A 331       9.757  19.720  39.889  1.0   0.00      PROT N \n",
            "ATOM   1911  H   ASN A 331       7.714  17.174  35.442  1.0   0.00      PROT H \n",
            "ATOM   1912  HA  ASN A 331       9.576  17.890  36.422  1.0   0.00      PROT H \n",
            "ATOM   1913  HB3 ASN A 331       7.860  18.465  38.859  1.0   0.00      PROT H \n",
            "ATOM   1914  HB2 ASN A 331       7.870  19.480  37.396  1.0   0.00      PROT H \n",
            "ATOM   1915 HD21 ASN A 331       9.033  19.320  40.486  1.0   0.00      PROT H \n",
            "ATOM   1916 HD22 ASN A 331      10.534  20.234  40.305  1.0   0.00      PROT H \n",
            "ATOM   1917  N   GLY A 332       9.061  15.235  37.722  1.0   0.00      PROT N \n",
            "ATOM   1918  CA  GLY A 332       9.521  14.090  38.485  1.0   0.00      PROT C \n",
            "ATOM   1919  C   GLY A 332       8.943  13.912  39.879  1.0   0.00      PROT C \n",
            "ATOM   1920  O   GLY A 332       9.481  13.139  40.677  1.0   0.00      PROT O \n",
            "ATOM   1921  H   GLY A 332       8.469  15.076  36.919  1.0   0.00      PROT H \n",
            "ATOM   1922  HA2 GLY A 332      10.604  14.198  38.552  1.0   0.00      PROT H \n",
            "ATOM   1923  HA3 GLY A 332       9.280  13.218  37.878  1.0   0.00      PROT H \n",
            "ATOM   1924  N   GLU A 333       7.836  14.590  40.175  1.0   0.00      PROT N \n",
            "ATOM   1925  CA  GLU A 333       7.233  14.488  41.502  1.0   0.00      PROT C \n",
            "ATOM   1926  C   GLU A 333       5.730  14.707  41.624  1.0   0.00      PROT C \n",
            "ATOM   1927  O   GLU A 333       5.156  14.416  42.668  1.0   0.00      PROT O \n",
            "ATOM   1928  CB  GLU A 333       7.968  15.421  42.467  1.0   0.00      PROT C \n",
            "ATOM   1929  CG  GLU A 333       8.574  16.643  41.807  1.0   0.00      PROT C \n",
            "ATOM   1930  CD  GLU A 333       7.613  17.812  41.736  1.0   0.00      PROT C \n",
            "ATOM   1931  OE1 GLU A 333       7.110  18.225  42.801  1.0   0.00      PROT O \n",
            "ATOM   1932  OE2 GLU A 333       7.368  18.325  40.621  1.0   0.00      PROT O \n",
            "ATOM   1933  H   GLU A 333       7.405  15.182  39.479  1.0   0.00      PROT H \n",
            "ATOM   1934  HA  GLU A 333       7.373  13.426  41.705  1.0   0.00      PROT H \n",
            "ATOM   1935  HB3 GLU A 333       8.756  14.836  42.941  1.0   0.00      PROT H \n",
            "ATOM   1936  HB2 GLU A 333       7.246  15.736  43.221  1.0   0.00      PROT H \n",
            "ATOM   1937  HG3 GLU A 333       8.877  16.360  40.799  1.0   0.00      PROT H \n",
            "ATOM   1938  HG2 GLU A 333       9.453  16.930  42.384  1.0   0.00      PROT H \n",
            "ATOM   1939  HE2 GLU A 333       6.729  19.087  40.728  1.0   0.00      PROT H \n",
            "ATOM   1940  N   MET A 334       5.082  15.205  40.579  1.0   0.00      PROT N \n",
            "ATOM   1941  CA  MET A 334       3.636  15.422  40.643  1.0   0.00      PROT C \n",
            "ATOM   1942  C   MET A 334       2.900  14.524  39.648  1.0   0.00      PROT C \n",
            "ATOM   1943  O   MET A 334       2.992  14.729  38.441  1.0   0.00      PROT O \n",
            "ATOM   1944  CB  MET A 334       3.305  16.885  40.343  1.0   0.00      PROT C \n",
            "ATOM   1945  CG  MET A 334       1.870  17.302  40.663  1.0   0.00      PROT C \n",
            "ATOM   1946  SD  MET A 334       1.411  18.823  39.808  1.0   0.00      PROT S \n",
            "ATOM   1947  CE  MET A 334       2.063  20.063  40.925  1.0   0.00      PROT C \n",
            "ATOM   1948  H   MET A 334       5.587  15.436  39.735  1.0   0.00      PROT H \n",
            "ATOM   1949  HA  MET A 334       3.352  15.161  41.662  1.0   0.00      PROT H \n",
            "ATOM   1950  HB3 MET A 334       3.499  17.037  39.281  1.0   0.00      PROT H \n",
            "ATOM   1951  HB2 MET A 334       3.998  17.488  40.930  1.0   0.00      PROT H \n",
            "ATOM   1952  HG2 MET A 334       1.185  16.513  40.352  1.0   0.00      PROT H \n",
            "ATOM   1953  HG3 MET A 334       1.770  17.468  41.736  1.0   0.00      PROT H \n",
            "ATOM   1954  HE1 MET A 334       1.854  21.056  40.527  1.0   0.00      PROT H \n",
            "ATOM   1955  HE2 MET A 334       1.591  19.956  41.902  1.0   0.00      PROT H \n",
            "ATOM   1956  HE3 MET A 334       3.140  19.931  41.025  1.0   0.00      PROT H \n",
            "ATOM   1957  N   ALA A 335       2.178  13.528  40.153  1.0   0.00      PROT N \n",
            "ATOM   1958  CA  ALA A 335       1.429  12.629  39.288  1.0   0.00      PROT C \n",
            "ATOM   1959  C   ALA A 335       0.028  13.194  39.079  1.0   0.00      PROT C \n",
            "ATOM   1960  O   ALA A 335      -0.737  13.381  40.022  1.0   0.00      PROT O \n",
            "ATOM   1961  CB  ALA A 335       1.356  11.225  39.895  1.0   0.00      PROT C \n",
            "ATOM   1962  H   ALA A 335       2.143  13.386  41.152  1.0   0.00      PROT H \n",
            "ATOM   1963  HA  ALA A 335       1.971  12.584  38.343  1.0   0.00      PROT H \n",
            "ATOM   1964  HB1 ALA A 335       2.365  10.838  40.037  1.0   0.00      PROT H \n",
            "ATOM   1965  HB2 ALA A 335       0.846  11.271  40.857  1.0   0.00      PROT H \n",
            "ATOM   1966  HB3 ALA A 335       0.805  10.567  39.223  1.0   0.00      PROT H \n",
            "ATOM   1967  N   VAL A 336      -0.294  13.458  37.822  1.0   0.00      PROT N \n",
            "ATOM   1968  CA  VAL A 336      -1.572  14.024  37.447  1.0   0.00      PROT C \n",
            "ATOM   1969  C   VAL A 336      -2.328  13.100  36.485  1.0   0.00      PROT C \n",
            "ATOM   1970  O   VAL A 336      -1.726  12.279  35.792  1.0   0.00      PROT O \n",
            "ATOM   1971  CB  VAL A 336      -1.348  15.401  36.774  1.0   0.00      PROT C \n",
            "ATOM   1972  CG1 VAL A 336      -0.583  16.307  37.718  1.0   0.00      PROT C \n",
            "ATOM   1973  CG2 VAL A 336      -0.565  15.236  35.459  1.0   0.00      PROT C \n",
            "ATOM   1974  H   VAL A 336       0.364  13.264  37.081  1.0   0.00      PROT H \n",
            "ATOM   1975  HA  VAL A 336      -2.126  14.108  38.382  1.0   0.00      PROT H \n",
            "ATOM   1976  HB  VAL A 336      -2.310  15.863  36.554  1.0   0.00      PROT H \n",
            "ATOM   1977 HG11 VAL A 336      -1.142  16.420  38.647  1.0   0.00      PROT H \n",
            "ATOM   1978 HG12 VAL A 336       0.392  15.869  37.932  1.0   0.00      PROT H \n",
            "ATOM   1979 HG13 VAL A 336      -0.448  17.284  37.254  1.0   0.00      PROT H \n",
            "ATOM   1980 HG21 VAL A 336      -1.120  14.584  34.785  1.0   0.00      PROT H \n",
            "ATOM   1981 HG22 VAL A 336      -0.430  16.211  34.992  1.0   0.00      PROT H \n",
            "ATOM   1982 HG23 VAL A 336       0.410  14.796  35.669  1.0   0.00      PROT H \n",
            "ATOM   1983  N   THR A 337      -3.649  13.221  36.452  1.0   0.00      PROT N \n",
            "ATOM   1984  CA  THR A 337      -4.432  12.402  35.542  1.0   0.00      PROT C \n",
            "ATOM   1985  C   THR A 337      -4.607  13.171  34.246  1.0   0.00      PROT C \n",
            "ATOM   1986  O   THR A 337      -4.142  14.300  34.123  1.0   0.00      PROT O \n",
            "ATOM   1987  CB  THR A 337      -5.814  12.056  36.123  1.0   0.00      PROT C \n",
            "ATOM   1988  OG1 THR A 337      -6.484  13.256  36.519  1.0   0.00      PROT O \n",
            "ATOM   1989  CG2 THR A 337      -5.667  11.153  37.314  1.0   0.00      PROT C \n",
            "ATOM   1990  H   THR A 337      -4.111  13.883  37.060  1.0   0.00      PROT H \n",
            "ATOM   1991  HA  THR A 337      -3.861  11.485  35.398  1.0   0.00      PROT H \n",
            "ATOM   1992  HB  THR A 337      -6.451  11.579  35.377  1.0   0.00      PROT H \n",
            "ATOM   1993  HG1 THR A 337      -6.664  13.965  35.852  1.0   0.00      PROT H \n",
            "ATOM   1994 HG21 THR A 337      -6.653  10.917  37.715  1.0   0.00      PROT H \n",
            "ATOM   1995 HG22 THR A 337      -5.167  10.233  37.013  1.0   0.00      PROT H \n",
            "ATOM   1996 HG23 THR A 337      -5.075  11.654  38.079  1.0   0.00      PROT H \n",
            "ATOM   1997  N   ARG A 338      -5.262  12.544  33.279  1.0   0.00      PROT N \n",
            "ATOM   1998  CA  ARG A 338      -5.499  13.151  31.976  1.0   0.00      PROT C \n",
            "ATOM   1999  C   ARG A 338      -6.426  14.363  32.128  1.0   0.00      PROT C \n",
            "ATOM   2000  O   ARG A 338      -6.160  15.427  31.578  1.0   0.00      PROT O \n",
            "ATOM   2001  CB  ARG A 338      -6.113  12.098  31.041  1.0   0.00      PROT C \n",
            "ATOM   2002  CG  ARG A 338      -6.315  12.520  29.600  1.0   0.00      PROT C \n",
            "ATOM   2003  CD  ARG A 338      -7.119  11.462  28.823  1.0   0.00      PROT C \n",
            "ATOM   2004  NE  ARG A 338      -6.364  10.240  28.528  1.0   0.00      PROT N \n",
            "ATOM   2005  CZ  ARG A 338      -5.804   9.974  27.349  1.0   0.00      PROT C \n",
            "ATOM   2006  NH2 ARG A 338      -5.131   8.841  27.164  1.0   0.00      PROT N \n",
            "ATOM   2007  NH1 ARG A 338      -5.925  10.840  26.348  1.0   0.00      PROT N \n",
            "ATOM   2008  H   ARG A 338      -5.617  11.611  33.432  1.0   0.00      PROT H \n",
            "ATOM   2009  HA  ARG A 338      -4.534  13.482  31.591  1.0   0.00      PROT H \n",
            "ATOM   2010  HB3 ARG A 338      -7.074  11.830  31.479  1.0   0.00      PROT H \n",
            "ATOM   2011  HB2 ARG A 338      -5.445  11.237  31.077  1.0   0.00      PROT H \n",
            "ATOM   2012  HG2 ARG A 338      -6.859  13.464  29.571  1.0   0.00      PROT H \n",
            "ATOM   2013  HG3 ARG A 338      -5.345  12.645  29.119  1.0   0.00      PROT H \n",
            "ATOM   2014  HD2 ARG A 338      -7.992  11.169  29.406  1.0   0.00      PROT H \n",
            "ATOM   2015  HD3 ARG A 338      -7.442  11.879  27.869  1.0   0.00      PROT H \n",
            "ATOM   2016  HE  ARG A 338      -6.281   9.579  29.287  1.0   0.00      PROT H \n",
            "ATOM   2017 HH22 ARG A 338      -4.710   8.640  26.257  1.0   0.00      PROT H \n",
            "ATOM   2018 HH21 ARG A 338      -5.036   8.174  27.929  1.0   0.00      PROT H \n",
            "ATOM   2019 HH12 ARG A 338      -5.503  10.635  25.442  1.0   0.00      PROT H \n",
            "ATOM   2020 HH11 ARG A 338      -6.421  11.818  26.261  1.0   0.00      PROT H \n",
            "ATOM   2021  N   GLY A 339      -7.501  14.201  32.897  1.0   0.00      PROT N \n",
            "ATOM   2022  CA  GLY A 339      -8.442  15.288  33.100  1.0   0.00      PROT C \n",
            "ATOM   2023  C   GLY A 339      -7.904  16.481  33.875  1.0   0.00      PROT C \n",
            "ATOM   2024  O   GLY A 339      -8.315  17.622  33.618  1.0   0.00      PROT O \n",
            "ATOM   2025  H   GLY A 339      -7.670  13.313  33.348  1.0   0.00      PROT H \n",
            "ATOM   2026  HA2 GLY A 339      -9.293  14.863  33.632  1.0   0.00      PROT H \n",
            "ATOM   2027  HA3 GLY A 339      -8.758  15.612  32.108  1.0   0.00      PROT H \n",
            "ATOM   2028  N   GLN A 340      -7.006  16.225  34.831  1.0   0.00      PROT N \n",
            "ATOM   2029  CA  GLN A 340      -6.414  17.284  35.644  1.0   0.00      PROT C \n",
            "ATOM   2030  C   GLN A 340      -5.552  18.204  34.795  1.0   0.00      PROT C \n",
            "ATOM   2031  O   GLN A 340      -5.685  19.431  34.853  1.0   0.00      PROT O \n",
            "ATOM   2032  CB  GLN A 340      -5.555  16.693  36.762  1.0   0.00      PROT C \n",
            "ATOM   2033  CG  GLN A 340      -6.340  16.031  37.882  1.0   0.00      PROT C \n",
            "ATOM   2034  CD  GLN A 340      -5.442  15.411  38.936  1.0   0.00      PROT C \n",
            "ATOM   2035  OE1 GLN A 340      -4.551  14.625  38.623  1.0   0.00      PROT O \n",
            "ATOM   2036  NE2 GLN A 340      -5.680  15.753  40.195  1.0   0.00      PROT N \n",
            "ATOM   2037  H   GLN A 340      -6.716  15.274  35.010  1.0   0.00      PROT H \n",
            "ATOM   2038  HA  GLN A 340      -7.262  17.833  36.053  1.0   0.00      PROT H \n",
            "ATOM   2039  HB3 GLN A 340      -4.961  17.511  37.169  1.0   0.00      PROT H \n",
            "ATOM   2040  HB2 GLN A 340      -4.895  15.960  36.299  1.0   0.00      PROT H \n",
            "ATOM   2041  HG3 GLN A 340      -6.968  15.258  37.439  1.0   0.00      PROT H \n",
            "ATOM   2042  HG2 GLN A 340      -6.971  16.791  38.343  1.0   0.00      PROT H \n",
            "ATOM   2043 HE21 GLN A 340      -6.430  16.409  40.412  1.0   0.00      PROT H \n",
            "ATOM   2044 HE22 GLN A 340      -5.113  15.360  40.946  1.0   0.00      PROT H \n",
            "ATOM   2045  N   LEU A 341      -4.668  17.600  34.007  1.0   0.00      PROT N \n",
            "ATOM   2046  CA  LEU A 341      -3.773  18.351  33.142  1.0   0.00      PROT C \n",
            "ATOM   2047  C   LEU A 341      -4.545  19.005  31.985  1.0   0.00      PROT C \n",
            "ATOM   2048  O   LEU A 341      -4.152  20.060  31.472  1.0   0.00      PROT O \n",
            "ATOM   2049  CB  LEU A 341      -2.674  17.417  32.614  1.0   0.00      PROT C \n",
            "ATOM   2050  CG  LEU A 341      -1.574  18.051  31.769  1.0   0.00      PROT C \n",
            "ATOM   2051  CD1 LEU A 341      -0.858  19.127  32.572  1.0   0.00      PROT C \n",
            "ATOM   2052  CD2 LEU A 341      -0.603  16.980  31.325  1.0   0.00      PROT C \n",
            "ATOM   2053  H   LEU A 341      -4.605  16.592  33.998  1.0   0.00      PROT H \n",
            "ATOM   2054  HA  LEU A 341      -3.341  19.134  33.765  1.0   0.00      PROT H \n",
            "ATOM   2055  HB3 LEU A 341      -3.191  16.658  32.027  1.0   0.00      PROT H \n",
            "ATOM   2056  HB2 LEU A 341      -2.233  16.955  33.497  1.0   0.00      PROT H \n",
            "ATOM   2057  HG  LEU A 341      -2.006  18.514  30.882  1.0   0.00      PROT H \n",
            "ATOM   2058 HD11 LEU A 341      -1.575  19.887  32.881  1.0   0.00      PROT H \n",
            "ATOM   2059 HD12 LEU A 341      -0.401  18.678  33.454  1.0   0.00      PROT H \n",
            "ATOM   2060 HD13 LEU A 341      -0.085  19.586  31.956  1.0   0.00      PROT H \n",
            "ATOM   2061 HD21 LEU A 341      -1.136  16.221  30.753  1.0   0.00      PROT H \n",
            "ATOM   2062 HD22 LEU A 341       0.172  17.427  30.702  1.0   0.00      PROT H \n",
            "ATOM   2063 HD23 LEU A 341      -0.145  16.520  32.200  1.0   0.00      PROT H \n",
            "ATOM   2064  N   LYS A 342      -5.647  18.371  31.591  1.0   0.00      PROT N \n",
            "ATOM   2065  CA  LYS A 342      -6.510  18.868  30.522  1.0   0.00      PROT C \n",
            "ATOM   2066  C   LYS A 342      -7.283  20.106  30.951  1.0   0.00      PROT C \n",
            "ATOM   2067  O   LYS A 342      -7.305  21.111  30.238  1.0   0.00      PROT O \n",
            "ATOM   2068  CB  LYS A 342      -7.502  17.791  30.101  1.0   0.00      PROT C \n",
            "ATOM   2069  CG  LYS A 342      -8.483  18.252  29.051  1.0   0.00      PROT C \n",
            "ATOM   2070  CD  LYS A 342      -9.042  17.078  28.282  1.0   0.00      PROT C \n",
            "ATOM   2071  CE  LYS A 342      -9.742  17.555  27.025  1.0   0.00      PROT C \n",
            "ATOM   2072  NZ  LYS A 342     -10.173  16.415  26.182  1.0   0.00      PROT N \n",
            "ATOM   2073  H   LYS A 342      -5.919  17.506  32.036  1.0   0.00      PROT H \n",
            "ATOM   2074  HA  LYS A 342      -5.829  19.122  29.709  1.0   0.00      PROT H \n",
            "ATOM   2075  HB3 LYS A 342      -8.045  17.481  30.994  1.0   0.00      PROT H \n",
            "ATOM   2076  HB2 LYS A 342      -6.926  16.950  29.716  1.0   0.00      PROT H \n",
            "ATOM   2077  HG2 LYS A 342      -9.307  18.781  29.531  1.0   0.00      PROT H \n",
            "ATOM   2078  HG3 LYS A 342      -7.980  18.921  28.352  1.0   0.00      PROT H \n",
            "ATOM   2079  HD2 LYS A 342      -8.230  16.407  28.003  1.0   0.00      PROT H \n",
            "ATOM   2080  HD3 LYS A 342      -9.758  16.543  28.906  1.0   0.00      PROT H \n",
            "ATOM   2081  HE2 LYS A 342     -10.623  18.136  27.298  1.0   0.00      PROT H \n",
            "ATOM   2082  HE3 LYS A 342      -9.061  18.177  26.444  1.0   0.00      PROT H \n",
            "ATOM   2083  HZ1 LYS A 342     -11.176  16.441  26.068  1.0   0.00      PROT H \n",
            "ATOM   2084  HZ2 LYS A 342      -9.730  16.479  25.277  1.0   0.00      PROT H \n",
            "ATOM   2085  HZ3 LYS A 342     -10.060  15.332  26.337  1.0   0.00      PROT H \n",
            "ATOM   2086  N   ASN A 343      -7.920  20.021  32.117  1.0   0.00      PROT N \n",
            "ATOM   2087  CA  ASN A 343      -8.694  21.131  32.663  1.0   0.00      PROT C \n",
            "ATOM   2088  C   ASN A 343      -7.787  22.214  33.211  1.0   0.00      PROT C \n",
            "ATOM   2089  O   ASN A 343      -8.134  23.396  33.179  1.0   0.00      PROT O \n",
            "ATOM   2090  CB  ASN A 343      -9.600  20.653  33.791  1.0   0.00      PROT C \n",
            "ATOM   2091  CG  ASN A 343     -10.496  19.525  33.371  1.0   0.00      PROT C \n",
            "ATOM   2092  OD1 ASN A 343     -10.967  19.483  32.234  1.0   0.00      PROT O \n",
            "ATOM   2093  ND2 ASN A 343     -10.756  18.604  34.292  1.0   0.00      PROT N \n",
            "ATOM   2094  H   ASN A 343      -7.877  19.167  32.655  1.0   0.00      PROT H \n",
            "ATOM   2095  HA  ASN A 343      -9.272  21.508  31.820  1.0   0.00      PROT H \n",
            "ATOM   2096  HB3 ASN A 343     -10.207  21.498  34.114  1.0   0.00      PROT H \n",
            "ATOM   2097  HB2 ASN A 343      -8.965  20.326  34.615  1.0   0.00      PROT H \n",
            "ATOM   2098 HD21 ASN A 343     -10.342  18.681  35.221  1.0   0.00      PROT H \n",
            "ATOM   2099 HD22 ASN A 343     -11.369  17.820  34.070  1.0   0.00      PROT H \n",
            "ATOM   2100  N   GLY A 344      -6.627  21.796  33.715  1.0   0.00      PROT N \n",
            "ATOM   2101  CA  GLY A 344      -5.677  22.729  34.288  1.0   0.00      PROT C \n",
            "ATOM   2102  C   GLY A 344      -5.004  23.669  33.310  1.0   0.00      PROT C \n",
            "ATOM   2103  O   GLY A 344      -4.184  24.479  33.718  1.0   0.00      PROT O \n",
            "ATOM   2104  H   GLY A 344      -6.396  20.813  33.702  1.0   0.00      PROT H \n",
            "ATOM   2105  HA2 GLY A 344      -4.921  22.124  34.789  1.0   0.00      PROT H \n",
            "ATOM   2106  HA3 GLY A 344      -6.228  23.309  35.028  1.0   0.00      PROT H \n",
            "ATOM   2107  N   GLY A 345      -5.321  23.556  32.023  1.0   0.00      PROT N \n",
            "ATOM   2108  CA  GLY A 345      -4.720  24.452  31.055  1.0   0.00      PROT C \n",
            "ATOM   2109  C   GLY A 345      -4.381  23.914  29.674  1.0   0.00      PROT C \n",
            "ATOM   2110  O   GLY A 345      -4.511  24.629  28.681  1.0   0.00      PROT O \n",
            "ATOM   2111  H   GLY A 345      -5.977  22.849  31.725  1.0   0.00      PROT H \n",
            "ATOM   2112  HA2 GLY A 345      -3.811  24.811  31.538  1.0   0.00      PROT H \n",
            "ATOM   2113  HA3 GLY A 345      -5.430  25.273  30.961  1.0   0.00      PROT H \n",
            "ATOM   2114  N   LEU A 346      -3.955  22.663  29.582  1.0   0.00      PROT N \n",
            "ATOM   2115  CA  LEU A 346      -3.566  22.125  28.289  1.0   0.00      PROT C \n",
            "ATOM   2116  C   LEU A 346      -4.679  21.687  27.342  1.0   0.00      PROT C \n",
            "ATOM   2117  O   LEU A 346      -4.460  21.611  26.125  1.0   0.00      PROT O \n",
            "ATOM   2118  CB  LEU A 346      -2.576  20.977  28.489  1.0   0.00      PROT C \n",
            "ATOM   2119  CG  LEU A 346      -1.216  21.386  29.057  1.0   0.00      PROT C \n",
            "ATOM   2120  CD1 LEU A 346      -0.306  20.171  29.177  1.0   0.00      PROT C \n",
            "ATOM   2121  CD2 LEU A 346      -0.602  22.444  28.161  1.0   0.00      PROT C \n",
            "ATOM   2122  H   LEU A 346      -3.901  22.086  30.409  1.0   0.00      PROT H \n",
            "ATOM   2123  HA  LEU A 346      -3.133  23.012  27.827  1.0   0.00      PROT H \n",
            "ATOM   2124  HB2 LEU A 346      -2.998  20.250  29.183  1.0   0.00      PROT H \n",
            "ATOM   2125  HB3 LEU A 346      -2.381  20.494  27.531  1.0   0.00      PROT H \n",
            "ATOM   2126  HG  LEU A 346      -1.338  21.806  30.055  1.0   0.00      PROT H \n",
            "ATOM   2127 HD11 LEU A 346      -0.773  19.430  29.826  1.0   0.00      PROT H \n",
            "ATOM   2128 HD12 LEU A 346      -0.144  19.739  28.189  1.0   0.00      PROT H \n",
            "ATOM   2129 HD13 LEU A 346       0.651  20.474  29.602  1.0   0.00      PROT H \n",
            "ATOM   2130 HD21 LEU A 346      -1.275  23.299  28.095  1.0   0.00      PROT H \n",
            "ATOM   2131 HD22 LEU A 346       0.353  22.764  28.578  1.0   0.00      PROT H \n",
            "ATOM   2132 HD23 LEU A 346      -0.442  22.029  27.166  1.0   0.00      PROT H \n",
            "ATOM   2133  N   GLY A 347      -5.866  21.407  27.880  1.0   0.00      PROT N \n",
            "ATOM   2134  CA  GLY A 347      -6.948  20.969  27.024  1.0   0.00      PROT C \n",
            "ATOM   2135  C   GLY A 347      -6.558  19.718  26.242  1.0   0.00      PROT C \n",
            "ATOM   2136  O   GLY A 347      -6.017  18.771  26.820  1.0   0.00      PROT O \n",
            "ATOM   2137  H   GLY A 347      -6.004  21.501  28.876  1.0   0.00      PROT H \n",
            "ATOM   2138  HA2 GLY A 347      -7.189  21.772  26.328  1.0   0.00      PROT H \n",
            "ATOM   2139  HA3 GLY A 347      -7.818  20.753  27.644  1.0   0.00      PROT H \n",
            "ATOM   2140  N   VAL A 348      -6.818  19.709  24.934  1.0   0.00      PROT N \n",
            "ATOM   2141  CA  VAL A 348      -6.502  18.553  24.098  1.0   0.00      PROT C \n",
            "ATOM   2142  C   VAL A 348      -5.030  18.139  24.098  1.0   0.00      PROT C \n",
            "ATOM   2143  O   VAL A 348      -4.710  16.984  23.823  1.0   0.00      PROT O \n",
            "ATOM   2144  CB  VAL A 348      -6.944  18.766  22.608  1.0   0.00      PROT C \n",
            "ATOM   2145  CG1 VAL A 348      -8.449  18.633  22.482  1.0   0.00      PROT C \n",
            "ATOM   2146  CG2 VAL A 348      -6.480  20.127  22.101  1.0   0.00      PROT C \n",
            "ATOM   2147  H   VAL A 348      -7.244  20.515  24.498  1.0   0.00      PROT H \n",
            "ATOM   2148  HA  VAL A 348      -7.074  17.788  24.622  1.0   0.00      PROT H \n",
            "ATOM   2149  HB  VAL A 348      -6.496  18.007  21.967  1.0   0.00      PROT H \n",
            "ATOM   2150 HG11 VAL A 348      -8.759  17.655  22.849  1.0   0.00      PROT H \n",
            "ATOM   2151 HG12 VAL A 348      -8.933  19.412  23.071  1.0   0.00      PROT H \n",
            "ATOM   2152 HG13 VAL A 348      -8.737  18.736  21.436  1.0   0.00      PROT H \n",
            "ATOM   2153 HG21 VAL A 348      -5.397  20.202  22.200  1.0   0.00      PROT H \n",
            "ATOM   2154 HG22 VAL A 348      -6.757  20.238  21.053  1.0   0.00      PROT H \n",
            "ATOM   2155 HG23 VAL A 348      -6.954  20.914  22.688  1.0   0.00      PROT H \n",
            "ATOM   2156  N   VAL A 349      -4.138  19.079  24.394  1.0   0.00      PROT N \n",
            "ATOM   2157  CA  VAL A 349      -2.702  18.799  24.420  1.0   0.00      PROT C \n",
            "ATOM   2158  C   VAL A 349      -2.430  17.748  25.497  1.0   0.00      PROT C \n",
            "ATOM   2159  O   VAL A 349      -1.525  16.924  25.371  1.0   0.00      PROT O \n",
            "ATOM   2160  CB  VAL A 349      -1.900  20.100  24.714  1.0   0.00      PROT C \n",
            "ATOM   2161  CG1 VAL A 349      -0.406  19.852  24.594  1.0   0.00      PROT C \n",
            "ATOM   2162  CG2 VAL A 349      -2.329  21.198  23.742  1.0   0.00      PROT C \n",
            "ATOM   2163  H   VAL A 349      -4.445  20.017  24.609  1.0   0.00      PROT H \n",
            "ATOM   2164  HA  VAL A 349      -2.437  18.411  23.436  1.0   0.00      PROT H \n",
            "ATOM   2165  HB  VAL A 349      -2.094  20.432  25.734  1.0   0.00      PROT H \n",
            "ATOM   2166 HG11 VAL A 349      -0.111  19.068  25.291  1.0   0.00      PROT H \n",
            "ATOM   2167 HG12 VAL A 349      -0.170  19.541  23.576  1.0   0.00      PROT H \n",
            "ATOM   2168 HG13 VAL A 349       0.135  20.769  24.829  1.0   0.00      PROT H \n",
            "ATOM   2169 HG21 VAL A 349      -3.402  21.368  23.836  1.0   0.00      PROT H \n",
            "ATOM   2170 HG22 VAL A 349      -1.794  22.119  23.974  1.0   0.00      PROT H \n",
            "ATOM   2171 HG23 VAL A 349      -2.098  20.891  22.722  1.0   0.00      PROT H \n",
            "ATOM   2172  N   SER A 350      -3.224  17.782  26.560  1.0   0.00      PROT N \n",
            "ATOM   2173  CA  SER A 350      -3.088  16.810  27.628  1.0   0.00      PROT C \n",
            "ATOM   2174  C   SER A 350      -3.316  15.429  27.047  1.0   0.00      PROT C \n",
            "ATOM   2175  O   SER A 350      -2.549  14.510  27.314  1.0   0.00      PROT O \n",
            "ATOM   2176  CB  SER A 350      -4.115  17.062  28.728  1.0   0.00      PROT C \n",
            "ATOM   2177  OG  SER A 350      -4.076  16.024  29.691  1.0   0.00      PROT O \n",
            "ATOM   2178  H   SER A 350      -3.937  18.493  26.635  1.0   0.00      PROT H \n",
            "ATOM   2179  HA  SER A 350      -2.080  16.924  28.026  1.0   0.00      PROT H \n",
            "ATOM   2180  HB2 SER A 350      -3.895  18.010  29.220  1.0   0.00      PROT H \n",
            "ATOM   2181  HB3 SER A 350      -5.113  17.102  28.291  1.0   0.00      PROT H \n",
            "ATOM   2182  HG  SER A 350      -4.687  16.048  30.469  1.0   0.00      PROT H \n",
            "ATOM   2183  N   ASP A 351      -4.377  15.290  26.253  1.0   0.00      PROT N \n",
            "ATOM   2184  CA  ASP A 351      -4.713  14.014  25.620  1.0   0.00      PROT C \n",
            "ATOM   2185  C   ASP A 351      -3.617  13.529  24.684  1.0   0.00      PROT C \n",
            "ATOM   2186  O   ASP A 351      -3.350  12.341  24.604  1.0   0.00      PROT O \n",
            "ATOM   2187  CB  ASP A 351      -6.021  14.125  24.837  1.0   0.00      PROT C \n",
            "ATOM   2188  CG  ASP A 351      -7.240  14.134  25.733  1.0   0.00      PROT C \n",
            "ATOM   2189  OD1 ASP A 351      -7.375  13.219  26.578  1.0   0.00      PROT O \n",
            "ATOM   2190  OD2 ASP A 351      -8.071  15.054  25.583  1.0   0.00      PROT O \n",
            "ATOM   2191  H   ASP A 351      -4.979  16.080  26.072  1.0   0.00      PROT H \n",
            "ATOM   2192  HA  ASP A 351      -4.808  13.320  26.455  1.0   0.00      PROT H \n",
            "ATOM   2193  HB3 ASP A 351      -6.076  13.275  24.157  1.0   0.00      PROT H \n",
            "ATOM   2194  HB2 ASP A 351      -5.987  15.049  24.260  1.0   0.00      PROT H \n",
            "ATOM   2195  N   ALA A 352      -2.997  14.460  23.970  1.0   0.00      PROT N \n",
            "ATOM   2196  CA  ALA A 352      -1.924  14.141  23.051  1.0   0.00      PROT C \n",
            "ATOM   2197  C   ALA A 352      -0.762  13.505  23.806  1.0   0.00      PROT C \n",
            "ATOM   2198  O   ALA A 352      -0.246  12.456  23.397  1.0   0.00      PROT O \n",
            "ATOM   2199  CB  ALA A 352      -1.458  15.393  22.348  1.0   0.00      PROT C \n",
            "ATOM   2200  H   ALA A 352      -3.269  15.429  24.058  1.0   0.00      PROT H \n",
            "ATOM   2201  HA  ALA A 352      -2.339  13.442  22.325  1.0   0.00      PROT H \n",
            "ATOM   2202  HB1 ALA A 352      -2.294  15.841  21.811  1.0   0.00      PROT H \n",
            "ATOM   2203  HB2 ALA A 352      -1.077  16.102  23.083  1.0   0.00      PROT H \n",
            "ATOM   2204  HB3 ALA A 352      -0.667  15.140  21.642  1.0   0.00      PROT H \n",
            "ATOM   2205  N   ILE A 353      -0.357  14.134  24.907  1.0   0.00      PROT N \n",
            "ATOM   2206  CA  ILE A 353       0.754  13.631  25.705  1.0   0.00      PROT C \n",
            "ATOM   2207  C   ILE A 353       0.452  12.286  26.369  1.0   0.00      PROT C \n",
            "ATOM   2208  O   ILE A 353       1.334  11.428  26.464  1.0   0.00      PROT O \n",
            "ATOM   2209  CB  ILE A 353       1.171  14.645  26.797  1.0   0.00      PROT C \n",
            "ATOM   2210  CG1 ILE A 353       1.505  15.998  26.170  1.0   0.00      PROT C \n",
            "ATOM   2211  CG2 ILE A 353       2.393  14.132  27.526  1.0   0.00      PROT C \n",
            "ATOM   2212  CD1 ILE A 353       1.596  17.118  27.162  1.0   0.00      PROT C \n",
            "ATOM   2213  H   ILE A 353      -0.822  14.979  25.206  1.0   0.00      PROT H \n",
            "ATOM   2214  HA  ILE A 353       1.534  13.496  24.955  1.0   0.00      PROT H \n",
            "ATOM   2215  HB  ILE A 353       0.354  14.770  27.507  1.0   0.00      PROT H \n",
            "ATOM   2216 HG13 ILE A 353       0.724  16.219  25.443  1.0   0.00      PROT H \n",
            "ATOM   2217 HG12 ILE A 353       2.460  15.890  25.656  1.0   0.00      PROT H \n",
            "ATOM   2218 HG21 ILE A 353       2.167  13.166  27.978  1.0   0.00      PROT H \n",
            "ATOM   2219 HG22 ILE A 353       3.216  14.020  26.821  1.0   0.00      PROT H \n",
            "ATOM   2220 HG23 ILE A 353       2.676  14.841  28.304  1.0   0.00      PROT H \n",
            "ATOM   2221 HD11 ILE A 353       1.836  18.045  26.642  1.0   0.00      PROT H \n",
            "ATOM   2222 HD12 ILE A 353       0.641  17.226  27.676  1.0   0.00      PROT H \n",
            "ATOM   2223 HD13 ILE A 353       2.377  16.897  27.889  1.0   0.00      PROT H \n",
            "ATOM   2224  N   PHE A 354      -0.774  12.090  26.841  1.0   0.00      PROT N \n",
            "ATOM   2225  CA  PHE A 354      -1.105  10.813  27.460  1.0   0.00      PROT C \n",
            "ATOM   2226  C   PHE A 354      -1.132   9.707  26.408  1.0   0.00      PROT C \n",
            "ATOM   2227  O   PHE A 354      -0.659   8.597  26.652  1.0   0.00      PROT O \n",
            "ATOM   2228  CB  PHE A 354      -2.456  10.870  28.194  1.0   0.00      PROT C \n",
            "ATOM   2229  CG  PHE A 354      -2.358  11.393  29.605  1.0   0.00      PROT C \n",
            "ATOM   2230  CD1 PHE A 354      -2.337  12.766  29.862  1.0   0.00      PROT C \n",
            "ATOM   2231  CD2 PHE A 354      -2.243  10.510  30.677  1.0   0.00      PROT C \n",
            "ATOM   2232  CE1 PHE A 354      -2.198  13.247  31.156  1.0   0.00      PROT C \n",
            "ATOM   2233  CE2 PHE A 354      -2.101  10.986  31.989  1.0   0.00      PROT C \n",
            "ATOM   2234  CZ  PHE A 354      -2.079  12.354  32.226  1.0   0.00      PROT C \n",
            "ATOM   2235  H   PHE A 354      -1.466  12.822  26.769  1.0   0.00      PROT H \n",
            "ATOM   2236  HA  PHE A 354      -0.303  10.638  28.178  1.0   0.00      PROT H \n",
            "ATOM   2237  HB3 PHE A 354      -2.860   9.858  28.209  1.0   0.00      PROT H \n",
            "ATOM   2238  HB2 PHE A 354      -3.114  11.515  27.612  1.0   0.00      PROT H \n",
            "ATOM   2239  HD1 PHE A 354      -2.432  13.462  29.029  1.0   0.00      PROT H \n",
            "ATOM   2240  HD2 PHE A 354      -2.263   9.434  30.502  1.0   0.00      PROT H \n",
            "ATOM   2241  HE1 PHE A 354      -2.181  14.321  31.339  1.0   0.00      PROT H \n",
            "ATOM   2242  HE2 PHE A 354      -2.008  10.287  32.820  1.0   0.00      PROT H \n",
            "ATOM   2243  HZ  PHE A 354      -1.969  12.729  33.244  1.0   0.00      PROT H \n",
            "ATOM   2244  N   ASP A 355      -1.693  10.007  25.242  1.0   0.00      PROT N \n",
            "ATOM   2245  CA  ASP A 355      -1.747   9.035  24.157  1.0   0.00      PROT C \n",
            "ATOM   2246  C   ASP A 355      -0.316   8.625  23.781  1.0   0.00      PROT C \n",
            "ATOM   2247  O   ASP A 355      -0.038   7.452  23.531  1.0   0.00      PROT O \n",
            "ATOM   2248  CB  ASP A 355      -2.457   9.627  22.934  1.0   0.00      PROT C \n",
            "ATOM   2249  CG  ASP A 355      -3.966   9.725  23.115  1.0   0.00      PROT C \n",
            "ATOM   2250  OD1 ASP A 355      -4.505   9.112  24.063  1.0   0.00      PROT O \n",
            "ATOM   2251  OD2 ASP A 355      -4.624  10.403  22.299  1.0   0.00      PROT O \n",
            "ATOM   2252  H   ASP A 355      -2.092  10.923  25.095  1.0   0.00      PROT H \n",
            "ATOM   2253  HA  ASP A 355      -2.317   8.186  24.535  1.0   0.00      PROT H \n",
            "ATOM   2254  HB3 ASP A 355      -2.234   8.988  22.079  1.0   0.00      PROT H \n",
            "ATOM   2255  HB2 ASP A 355      -2.046  10.622  22.761  1.0   0.00      PROT H \n",
            "ATOM   2256  N   LEU A 356       0.588   9.599  23.752  1.0   0.00      PROT N \n",
            "ATOM   2257  CA  LEU A 356       1.984   9.348  23.419  1.0   0.00      PROT C \n",
            "ATOM   2258  C   LEU A 356       2.625   8.451  24.482  1.0   0.00      PROT C \n",
            "ATOM   2259  O   LEU A 356       3.370   7.511  24.159  1.0   0.00      PROT O \n",
            "ATOM   2260  CB  LEU A 356       2.748  10.681  23.335  1.0   0.00      PROT C \n",
            "ATOM   2261  CG  LEU A 356       4.229  10.630  22.937  1.0   0.00      PROT C \n",
            "ATOM   2262  CD1 LEU A 356       4.342  10.140  21.505  1.0   0.00      PROT C \n",
            "ATOM   2263  CD2 LEU A 356       4.874  12.008  23.099  1.0   0.00      PROT C \n",
            "ATOM   2264  H   LEU A 356       0.315  10.548  23.965  1.0   0.00      PROT H \n",
            "ATOM   2265  HA  LEU A 356       1.977   8.842  22.453  1.0   0.00      PROT H \n",
            "ATOM   2266  HB3 LEU A 356       2.653  11.128  24.324  1.0   0.00      PROT H \n",
            "ATOM   2267  HB2 LEU A 356       2.198  11.281  22.611  1.0   0.00      PROT H \n",
            "ATOM   2268  HG  LEU A 356       4.765   9.934  23.583  1.0   0.00      PROT H \n",
            "ATOM   2269 HD11 LEU A 356       3.875   9.159  21.419  1.0   0.00      PROT H \n",
            "ATOM   2270 HD12 LEU A 356       3.838  10.842  20.840  1.0   0.00      PROT H \n",
            "ATOM   2271 HD13 LEU A 356       5.394  10.068  21.228  1.0   0.00      PROT H \n",
            "ATOM   2272 HD21 LEU A 356       4.778  12.334  24.135  1.0   0.00      PROT H \n",
            "ATOM   2273 HD22 LEU A 356       5.929  11.949  22.833  1.0   0.00      PROT H \n",
            "ATOM   2274 HD23 LEU A 356       4.374  12.722  22.445  1.0   0.00      PROT H \n",
            "ATOM   2275  N   GLY A 357       2.337   8.763  25.746  1.0   0.00      PROT N \n",
            "ATOM   2276  CA  GLY A 357       2.876   7.998  26.854  1.0   0.00      PROT C \n",
            "ATOM   2277  C   GLY A 357       2.386   6.567  26.827  1.0   0.00      PROT C \n",
            "ATOM   2278  O   GLY A 357       3.094   5.642  27.226  1.0   0.00      PROT O \n",
            "ATOM   2279  H   GLY A 357       1.732   9.546  25.948  1.0   0.00      PROT H \n",
            "ATOM   2280  HA2 GLY A 357       2.564   8.471  27.785  1.0   0.00      PROT H \n",
            "ATOM   2281  HA3 GLY A 357       3.964   8.011  26.786  1.0   0.00      PROT H \n",
            "ATOM   2282  N   MET A 358       1.162   6.381  26.360  1.0   0.00      PROT N \n",
            "ATOM   2283  CA  MET A 358       0.601   5.048  26.264  1.0   0.00      PROT C \n",
            "ATOM   2284  C   MET A 358       1.343   4.225  25.211  1.0   0.00      PROT C \n",
            "ATOM   2285  O   MET A 358       1.660   3.060  25.445  1.0   0.00      PROT O \n",
            "ATOM   2286  CB  MET A 358      -0.888   5.117  25.906  1.0   0.00      PROT C \n",
            "ATOM   2287  CG  MET A 358      -1.762   5.672  27.014  1.0   0.00      PROT C \n",
            "ATOM   2288  SD  MET A 358      -3.508   5.732  26.596  1.0   0.00      PROT S \n",
            "ATOM   2289  CE  MET A 358      -3.839   3.994  26.378  1.0   0.00      PROT C \n",
            "ATOM   2290  H   MET A 358       0.608   7.173  26.066  1.0   0.00      PROT H \n",
            "ATOM   2291  HA  MET A 358       0.741   4.612  27.253  1.0   0.00      PROT H \n",
            "ATOM   2292  HB3 MET A 358      -1.208   4.104  25.663  1.0   0.00      PROT H \n",
            "ATOM   2293  HB2 MET A 358      -0.978   5.748  25.022  1.0   0.00      PROT H \n",
            "ATOM   2294  HG2 MET A 358      -1.663   5.048  27.902  1.0   0.00      PROT H \n",
            "ATOM   2295  HG3 MET A 358      -1.450   6.690  27.248  1.0   0.00      PROT H \n",
            "ATOM   2296  HE1 MET A 358      -4.887   3.854  26.113  1.0   0.00      PROT H \n",
            "ATOM   2297  HE2 MET A 358      -3.207   3.601  25.581  1.0   0.00      PROT H \n",
            "ATOM   2298  HE3 MET A 358      -3.625   3.464  27.306  1.0   0.00      PROT H \n",
            "ATOM   2299  N   SER A 359       1.636   4.832  24.063  1.0   0.00      PROT N \n",
            "ATOM   2300  CA  SER A 359       2.310   4.107  22.995  1.0   0.00      PROT C \n",
            "ATOM   2301  C   SER A 359       3.824   4.005  23.138  1.0   0.00      PROT C \n",
            "ATOM   2302  O   SER A 359       4.460   3.224  22.424  1.0   0.00      PROT O \n",
            "ATOM   2303  CB  SER A 359       1.956   4.703  21.629  1.0   0.00      PROT C \n",
            "ATOM   2304  OG  SER A 359       2.580   5.946  21.421  1.0   0.00      PROT O \n",
            "ATOM   2305  H   SER A 359       1.391   5.802  23.928  1.0   0.00      PROT H \n",
            "ATOM   2306  HA  SER A 359       1.909   3.101  23.123  1.0   0.00      PROT H \n",
            "ATOM   2307  HB2 SER A 359       2.279   4.023  20.841  1.0   0.00      PROT H \n",
            "ATOM   2308  HB3 SER A 359       0.878   4.848  21.564  1.0   0.00      PROT H \n",
            "ATOM   2309  HG  SER A 359       2.433   6.438  20.575  1.0   0.00      PROT H \n",
            "ATOM   2310  N   LEU A 360       4.407   4.772  24.058  1.0   0.00      PROT N \n",
            "ATOM   2311  CA  LEU A 360       5.852   4.705  24.266  1.0   0.00      PROT C \n",
            "ATOM   2312  C   LEU A 360       6.275   3.548  25.181  1.0   0.00      PROT C \n",
            "ATOM   2313  O   LEU A 360       7.427   3.106  25.134  1.0   0.00      PROT O \n",
            "ATOM   2314  CB  LEU A 360       6.376   6.026  24.834  1.0   0.00      PROT C \n",
            "ATOM   2315  CG  LEU A 360       6.530   7.199  23.859  1.0   0.00      PROT C \n",
            "ATOM   2316  CD1 LEU A 360       6.952   8.437  24.624  1.0   0.00      PROT C \n",
            "ATOM   2317  CD2 LEU A 360       7.556   6.858  22.792  1.0   0.00      PROT C \n",
            "ATOM   2318  H   LEU A 360       3.850   5.404  24.616  1.0   0.00      PROT H \n",
            "ATOM   2319  HA  LEU A 360       6.243   4.524  23.265  1.0   0.00      PROT H \n",
            "ATOM   2320  HB2 LEU A 360       5.699   6.382  25.611  1.0   0.00      PROT H \n",
            "ATOM   2321  HB3 LEU A 360       7.367   5.871  25.260  1.0   0.00      PROT H \n",
            "ATOM   2322  HG  LEU A 360       5.580   7.402  23.365  1.0   0.00      PROT H \n",
            "ATOM   2323 HD11 LEU A 360       6.205   8.666  25.384  1.0   0.00      PROT H \n",
            "ATOM   2324 HD12 LEU A 360       7.915   8.258  25.103  1.0   0.00      PROT H \n",
            "ATOM   2325 HD13 LEU A 360       7.040   9.277  23.935  1.0   0.00      PROT H \n",
            "ATOM   2326 HD21 LEU A 360       7.237   5.966  22.253  1.0   0.00      PROT H \n",
            "ATOM   2327 HD22 LEU A 360       7.646   7.691  22.095  1.0   0.00      PROT H \n",
            "ATOM   2328 HD23 LEU A 360       8.521   6.672  23.263  1.0   0.00      PROT H \n",
            "ATOM   2329  N   SER A 361       5.343   3.057  25.999  1.0   0.00      PROT N \n",
            "ATOM   2330  CA  SER A 361       5.611   1.954  26.925  1.0   0.00      PROT C \n",
            "ATOM   2331  C   SER A 361       6.280   0.730  26.298  1.0   0.00      PROT C \n",
            "ATOM   2332  O   SER A 361       7.146   0.104  26.912  1.0   0.00      PROT O \n",
            "ATOM   2333  CB  SER A 361       4.319   1.506  27.604  1.0   0.00      PROT C \n",
            "ATOM   2334  OG  SER A 361       3.873   2.467  28.545  1.0   0.00      PROT O \n",
            "ATOM   2335  H   SER A 361       4.411   3.447  25.995  1.0   0.00      PROT H \n",
            "ATOM   2336  HA  SER A 361       6.319   2.406  27.620  1.0   0.00      PROT H \n",
            "ATOM   2337  HB2 SER A 361       3.542   1.370  26.852  1.0   0.00      PROT H \n",
            "ATOM   2338  HB3 SER A 361       4.490   0.564  28.125  1.0   0.00      PROT H \n",
            "ATOM   2339  HG  SER A 361       3.044   2.306  29.061  1.0   0.00      PROT H \n",
            "ATOM   2340  N   SER A 362       5.878   0.378  25.082  1.0   0.00      PROT N \n",
            "ATOM   2341  CA  SER A 362       6.467  -0.777  24.419  1.0   0.00      PROT C \n",
            "ATOM   2342  C   SER A 362       7.821  -0.461  23.776  1.0   0.00      PROT C \n",
            "ATOM   2343  O   SER A 362       8.601  -1.372  23.478  1.0   0.00      PROT O \n",
            "ATOM   2344  CB  SER A 362       5.508  -1.320  23.359  1.0   0.00      PROT C \n",
            "ATOM   2345  OG  SER A 362       5.173  -0.311  22.423  1.0   0.00      PROT O \n",
            "ATOM   2346  H   SER A 362       5.161   0.912  24.613  1.0   0.00      PROT H \n",
            "ATOM   2347  HA  SER A 362       6.617  -1.496  25.225  1.0   0.00      PROT H \n",
            "ATOM   2348  HB2 SER A 362       5.981  -2.148  22.831  1.0   0.00      PROT H \n",
            "ATOM   2349  HB3 SER A 362       4.595  -1.671  23.840  1.0   0.00      PROT H \n",
            "ATOM   2350  HG  SER A 362       4.558  -0.515  21.675  1.0   0.00      PROT H \n",
            "ATOM   2351  N   PHE A 363       8.111   0.820  23.562  1.0   0.00      PROT N \n",
            "ATOM   2352  CA  PHE A 363       9.381   1.186  22.944  1.0   0.00      PROT C \n",
            "ATOM   2353  C   PHE A 363      10.568   0.934  23.878  1.0   0.00      PROT C \n",
            "ATOM   2354  O   PHE A 363      11.702   0.765  23.422  1.0   0.00      PROT O \n",
            "ATOM   2355  CB  PHE A 363       9.364   2.655  22.485  1.0   0.00      PROT C \n",
            "ATOM   2356  CG  PHE A 363       8.624   2.887  21.188  1.0   0.00      PROT C \n",
            "ATOM   2357  CD1 PHE A 363       7.243   2.742  21.120  1.0   0.00      PROT C \n",
            "ATOM   2358  CD2 PHE A 363       9.315   3.238  20.031  1.0   0.00      PROT C \n",
            "ATOM   2359  CE1 PHE A 363       6.555   2.940  19.922  1.0   0.00      PROT C \n",
            "ATOM   2360  CE2 PHE A 363       8.639   3.441  18.823  1.0   0.00      PROT C \n",
            "ATOM   2361  CZ  PHE A 363       7.255   3.291  18.769  1.0   0.00      PROT C \n",
            "ATOM   2362  H   PHE A 363       7.452   1.538  23.827  1.0   0.00      PROT H \n",
            "ATOM   2363  HA  PHE A 363       9.459   0.518  22.086  1.0   0.00      PROT H \n",
            "ATOM   2364  HB3 PHE A 363      10.402   2.968  22.372  1.0   0.00      PROT H \n",
            "ATOM   2365  HB2 PHE A 363       8.894   3.234  23.280  1.0   0.00      PROT H \n",
            "ATOM   2366  HD1 PHE A 363       6.679   2.469  22.012  1.0   0.00      PROT H \n",
            "ATOM   2367  HD2 PHE A 363      10.398   3.358  20.058  1.0   0.00      PROT H \n",
            "ATOM   2368  HE1 PHE A 363       5.472   2.821  19.887  1.0   0.00      PROT H \n",
            "ATOM   2369  HE2 PHE A 363       9.193   3.716  17.925  1.0   0.00      PROT H \n",
            "ATOM   2370  HZ  PHE A 363       6.726   3.448  17.829  1.0   0.00      PROT H \n",
            "ATOM   2371  N   ASN A 364      10.308   0.897  25.183  1.0   0.00      PROT N \n",
            "ATOM   2372  CA  ASN A 364      11.372   0.659  26.158  1.0   0.00      PROT C \n",
            "ATOM   2373  C   ASN A 364      12.577   1.550  25.861  1.0   0.00      PROT C \n",
            "ATOM   2374  O   ASN A 364      13.674   1.059  25.601  1.0   0.00      PROT O \n",
            "ATOM   2375  CB  ASN A 364      11.807  -0.813  26.115  1.0   0.00      PROT C \n",
            "ATOM   2376  CG  ASN A 364      12.778  -1.165  27.229  1.0   0.00      PROT C \n",
            "ATOM   2377  OD1 ASN A 364      12.487  -0.942  28.406  1.0   0.00      PROT O \n",
            "ATOM   2378  ND2 ASN A 364      13.934  -1.718  26.867  1.0   0.00      PROT N \n",
            "ATOM   2379  H   ASN A 364       9.364   1.034  25.514  1.0   0.00      PROT H \n",
            "ATOM   2380  HA  ASN A 364      10.947   0.912  27.129  1.0   0.00      PROT H \n",
            "ATOM   2381  HB3 ASN A 364      12.280  -0.995  25.150  1.0   0.00      PROT H \n",
            "ATOM   2382  HB2 ASN A 364      10.913  -1.430  26.204  1.0   0.00      PROT H \n",
            "ATOM   2383 HD21 ASN A 364      14.133  -1.884  25.881  1.0   0.00      PROT H \n",
            "ATOM   2384 HD22 ASN A 364      14.621  -1.975  27.576  1.0   0.00      PROT H \n",
            "ATOM   2385  N   LEU A 365      12.370   2.860  25.901  1.0   0.00      PROT N \n",
            "ATOM   2386  CA  LEU A 365      13.433   3.812  25.610  1.0   0.00      PROT C \n",
            "ATOM   2387  C   LEU A 365      14.349   4.059  26.802  1.0   0.00      PROT C \n",
            "ATOM   2388  O   LEU A 365      13.884   4.349  27.898  1.0   0.00      PROT O \n",
            "ATOM   2389  CB  LEU A 365      12.812   5.133  25.156  1.0   0.00      PROT C \n",
            "ATOM   2390  CG  LEU A 365      11.873   5.088  23.944  1.0   0.00      PROT C \n",
            "ATOM   2391  CD1 LEU A 365      11.291   6.483  23.712  1.0   0.00      PROT C \n",
            "ATOM   2392  CD2 LEU A 365      12.625   4.591  22.699  1.0   0.00      PROT C \n",
            "ATOM   2393  H   LEU A 365      11.457   3.220  26.138  1.0   0.00      PROT H \n",
            "ATOM   2394  HA  LEU A 365      14.021   3.341  24.822  1.0   0.00      PROT H \n",
            "ATOM   2395  HB2 LEU A 365      12.218   5.553  25.968  1.0   0.00      PROT H \n",
            "ATOM   2396  HB3 LEU A 365      13.603   5.832  24.885  1.0   0.00      PROT H \n",
            "ATOM   2397  HG  LEU A 365      11.056   4.393  24.139  1.0   0.00      PROT H \n",
            "ATOM   2398 HD11 LEU A 365      10.764   6.811  24.608  1.0   0.00      PROT H \n",
            "ATOM   2399 HD12 LEU A 365      12.098   7.181  23.489  1.0   0.00      PROT H \n",
            "ATOM   2400 HD13 LEU A 365      10.596   6.452  22.873  1.0   0.00      PROT H \n",
            "ATOM   2401 HD21 LEU A 365      13.029   3.597  22.891  1.0   0.00      PROT H \n",
            "ATOM   2402 HD22 LEU A 365      11.939   4.547  21.853  1.0   0.00      PROT H \n",
            "ATOM   2403 HD23 LEU A 365      13.441   5.276  22.470  1.0   0.00      PROT H \n",
            "ATOM   2404  N   ASP A 366      15.656   3.951  26.603  1.0   0.00      PROT N \n",
            "ATOM   2405  CA  ASP A 366      16.567   4.192  27.711  1.0   0.00      PROT C \n",
            "ATOM   2406  C   ASP A 366      16.981   5.670  27.774  1.0   0.00      PROT C \n",
            "ATOM   2407  O   ASP A 366      16.603   6.464  26.912  1.0   0.00      PROT O \n",
            "ATOM   2408  CB  ASP A 366      17.790   3.259  27.627  1.0   0.00      PROT C \n",
            "ATOM   2409  CG  ASP A 366      18.648   3.499  26.403  1.0   0.00      PROT C \n",
            "ATOM   2410  OD1 ASP A 366      18.392   4.459  25.655  1.0   0.00      PROT O \n",
            "ATOM   2411  OD2 ASP A 366      19.598   2.721  26.194  1.0   0.00      PROT O \n",
            "ATOM   2412  H   ASP A 366      16.009   3.704  25.690  1.0   0.00      PROT H \n",
            "ATOM   2413  HA  ASP A 366      15.992   3.951  28.605  1.0   0.00      PROT H \n",
            "ATOM   2414  HB3 ASP A 366      17.419   2.234  27.615  1.0   0.00      PROT H \n",
            "ATOM   2415  HB2 ASP A 366      18.386   3.420  28.526  1.0   0.00      PROT H \n",
            "ATOM   2416  N   ASP A 367      17.749   6.042  28.791  1.0   0.00      PROT N \n",
            "ATOM   2417  CA  ASP A 367      18.157   7.432  28.947  1.0   0.00      PROT C \n",
            "ATOM   2418  C   ASP A 367      18.881   7.970  27.722  1.0   0.00      PROT C \n",
            "ATOM   2419  O   ASP A 367      18.881   9.180  27.474  1.0   0.00      PROT O \n",
            "ATOM   2420  CB  ASP A 367      19.041   7.613  30.190  1.0   0.00      PROT C \n",
            "ATOM   2421  CG  ASP A 367      18.362   7.145  31.468  1.0   0.00      PROT C \n",
            "ATOM   2422  OD1 ASP A 367      17.199   7.524  31.719  1.0   0.00      PROT O \n",
            "ATOM   2423  OD2 ASP A 367      18.999   6.400  32.234  1.0   0.00      PROT O \n",
            "ATOM   2424  H   ASP A 367      18.057   5.358  29.468  1.0   0.00      PROT H \n",
            "ATOM   2425  HA  ASP A 367      17.210   7.961  29.056  1.0   0.00      PROT H \n",
            "ATOM   2426  HB3 ASP A 367      19.284   8.672  30.275  1.0   0.00      PROT H \n",
            "ATOM   2427  HB2 ASP A 367      19.956   7.041  30.034  1.0   0.00      PROT H \n",
            "ATOM   2428  HD1 ASP A 367      16.896   7.142  32.592  1.0   0.00      PROT H \n",
            "ATOM   2429  N   THR A 368      19.498   7.076  26.956  1.0   0.00      PROT N \n",
            "ATOM   2430  CA  THR A 368      20.213   7.487  25.755  1.0   0.00      PROT C \n",
            "ATOM   2431  C   THR A 368      19.228   7.787  24.631  1.0   0.00      PROT C \n",
            "ATOM   2432  O   THR A 368      19.254   8.864  24.041  1.0   0.00      PROT O \n",
            "ATOM   2433  CB  THR A 368      21.192   6.397  25.289  1.0   0.00      PROT C \n",
            "ATOM   2434  OG1 THR A 368      22.213   6.215  26.280  1.0   0.00      PROT O \n",
            "ATOM   2435  CG2 THR A 368      21.825   6.785  23.961  1.0   0.00      PROT C \n",
            "ATOM   2436  H   THR A 368      19.475   6.097  27.204  1.0   0.00      PROT H \n",
            "ATOM   2437  HA  THR A 368      20.754   8.387  26.047  1.0   0.00      PROT H \n",
            "ATOM   2438  HB  THR A 368      20.690   5.434  25.193  1.0   0.00      PROT H \n",
            "ATOM   2439  HG1 THR A 368      21.966   5.959  27.204  1.0   0.00      PROT H \n",
            "ATOM   2440 HG21 THR A 368      22.515   6.003  23.645  1.0   0.00      PROT H \n",
            "ATOM   2441 HG22 THR A 368      21.046   6.907  23.209  1.0   0.00      PROT H \n",
            "ATOM   2442 HG23 THR A 368      22.368   7.723  24.077  1.0   0.00      PROT H \n",
            "ATOM   2443  N   GLU A 369      18.348   6.836  24.344  1.0   0.00      PROT N \n",
            "ATOM   2444  CA  GLU A 369      17.364   7.020  23.288  1.0   0.00      PROT C \n",
            "ATOM   2445  C   GLU A 369      16.500   8.243  23.546  1.0   0.00      PROT C \n",
            "ATOM   2446  O   GLU A 369      16.194   8.993  22.611  1.0   0.00      PROT O \n",
            "ATOM   2447  CB  GLU A 369      16.516   5.761  23.140  1.0   0.00      PROT C \n",
            "ATOM   2448  CG  GLU A 369      17.371   4.587  22.722  1.0   0.00      PROT C \n",
            "ATOM   2449  CD  GLU A 369      16.718   3.248  22.944  1.0   0.00      PROT C \n",
            "ATOM   2450  OE1 GLU A 369      16.080   3.049  23.995  1.0   0.00      PROT O \n",
            "ATOM   2451  OE2 GLU A 369      16.873   2.382  22.068  1.0   0.00      PROT O \n",
            "ATOM   2452  H   GLU A 369      18.355   5.969  24.862  1.0   0.00      PROT H \n",
            "ATOM   2453  HA  GLU A 369      17.934   7.192  22.375  1.0   0.00      PROT H \n",
            "ATOM   2454  HB3 GLU A 369      15.750   5.939  22.386  1.0   0.00      PROT H \n",
            "ATOM   2455  HB2 GLU A 369      16.041   5.542  24.096  1.0   0.00      PROT H \n",
            "ATOM   2456  HG3 GLU A 369      18.296   4.640  23.297  1.0   0.00      PROT H \n",
            "ATOM   2457  HG2 GLU A 369      17.594   4.711  21.662  1.0   0.00      PROT H \n",
            "ATOM   2458  N   VAL A 370      16.120   8.454  24.806  1.0   0.00      PROT N \n",
            "ATOM   2459  CA  VAL A 370      15.316   9.622  25.165  1.0   0.00      PROT C \n",
            "ATOM   2460  C   VAL A 370      16.181  10.882  25.032  1.0   0.00      PROT C \n",
            "ATOM   2461  O   VAL A 370      15.717  11.908  24.530  1.0   0.00      PROT O \n",
            "ATOM   2462  CB  VAL A 370      14.765   9.517  26.610  1.0   0.00      PROT C \n",
            "ATOM   2463  CG1 VAL A 370      14.243  10.866  27.080  1.0   0.00      PROT C \n",
            "ATOM   2464  CG2 VAL A 370      13.652   8.505  26.656  1.0   0.00      PROT C \n",
            "ATOM   2465  H   VAL A 370      16.387   7.802  25.530  1.0   0.00      PROT H \n",
            "ATOM   2466  HA  VAL A 370      14.487   9.632  24.458  1.0   0.00      PROT H \n",
            "ATOM   2467  HB  VAL A 370      15.556   9.195  27.288  1.0   0.00      PROT H \n",
            "ATOM   2468 HG11 VAL A 370      15.049  11.599  27.047  1.0   0.00      PROT H \n",
            "ATOM   2469 HG12 VAL A 370      13.431  11.189  26.428  1.0   0.00      PROT H \n",
            "ATOM   2470 HG13 VAL A 370      13.875  10.777  28.102  1.0   0.00      PROT H \n",
            "ATOM   2471 HG21 VAL A 370      14.026   7.538  26.319  1.0   0.00      PROT H \n",
            "ATOM   2472 HG22 VAL A 370      13.284   8.416  27.678  1.0   0.00      PROT H \n",
            "ATOM   2473 HG23 VAL A 370      12.840   8.828  26.004  1.0   0.00      PROT H \n",
            "ATOM   2474  N   ALA A 371      17.439  10.792  25.461  1.0   0.00      PROT N \n",
            "ATOM   2475  CA  ALA A 371      18.354  11.923  25.374  1.0   0.00      PROT C \n",
            "ATOM   2476  C   ALA A 371      18.619  12.320  23.921  1.0   0.00      PROT C \n",
            "ATOM   2477  O   ALA A 371      18.718  13.504  23.609  1.0   0.00      PROT O \n",
            "ATOM   2478  CB  ALA A 371      19.664  11.597  26.072  1.0   0.00      PROT C \n",
            "ATOM   2479  H   ALA A 371      17.777   9.926  25.857  1.0   0.00      PROT H \n",
            "ATOM   2480  HA  ALA A 371      17.846  12.739  25.888  1.0   0.00      PROT H \n",
            "ATOM   2481  HB1 ALA A 371      19.465  11.316  27.106  1.0   0.00      PROT H \n",
            "ATOM   2482  HB2 ALA A 371      20.153  10.770  25.558  1.0   0.00      PROT H \n",
            "ATOM   2483  HB3 ALA A 371      20.314  12.472  26.053  1.0   0.00      PROT H \n",
            "ATOM   2484  N   LEU A 372      18.733  11.335  23.032  1.0   0.00      PROT N \n",
            "ATOM   2485  CA  LEU A 372      18.976  11.623  21.619  1.0   0.00      PROT C \n",
            "ATOM   2486  C   LEU A 372      17.737  12.218  20.976  1.0   0.00      PROT C \n",
            "ATOM   2487  O   LEU A 372      17.842  13.057  20.085  1.0   0.00      PROT O \n",
            "ATOM   2488  CB  LEU A 372      19.393  10.360  20.873  1.0   0.00      PROT C \n",
            "ATOM   2489  CG  LEU A 372      20.787   9.816  21.205  1.0   0.00      PROT C \n",
            "ATOM   2490  CD1 LEU A 372      20.977   8.493  20.491  1.0   0.00      PROT C \n",
            "ATOM   2491  CD2 LEU A 372      21.863  10.808  20.785  1.0   0.00      PROT C \n",
            "ATOM   2492  H   LEU A 372      18.652  10.374  23.330  1.0   0.00      PROT H \n",
            "ATOM   2493  HA  LEU A 372      19.787  12.351  21.601  1.0   0.00      PROT H \n",
            "ATOM   2494  HB2 LEU A 372      18.692   9.557  21.099  1.0   0.00      PROT H \n",
            "ATOM   2495  HB3 LEU A 372      19.390  10.554  19.800  1.0   0.00      PROT H \n",
            "ATOM   2496  HG  LEU A 372      20.881   9.660  22.280  1.0   0.00      PROT H \n",
            "ATOM   2497 HD11 LEU A 372      20.199   7.796  20.804  1.0   0.00      PROT H \n",
            "ATOM   2498 HD12 LEU A 372      20.914   8.649  19.414  1.0   0.00      PROT H \n",
            "ATOM   2499 HD13 LEU A 372      21.955   8.082  20.742  1.0   0.00      PROT H \n",
            "ATOM   2500 HD21 LEU A 372      21.711  11.752  21.308  1.0   0.00      PROT H \n",
            "ATOM   2501 HD22 LEU A 372      22.845  10.407  21.037  1.0   0.00      PROT H \n",
            "ATOM   2502 HD23 LEU A 372      21.804  10.974  19.709  1.0   0.00      PROT H \n",
            "ATOM   2503  N   LEU A 373      16.562  11.784  21.426  1.0   0.00      PROT N \n",
            "ATOM   2504  CA  LEU A 373      15.307  12.316  20.905  1.0   0.00      PROT C \n",
            "ATOM   2505  C   LEU A 373      15.244  13.815  21.213  1.0   0.00      PROT C \n",
            "ATOM   2506  O   LEU A 373      14.792  14.611  20.389  1.0   0.00      PROT O \n",
            "ATOM   2507  CB  LEU A 373      14.109  11.607  21.551  1.0   0.00      PROT C \n",
            "ATOM   2508  CG  LEU A 373      13.644  10.266  20.976  1.0   0.00      PROT C \n",
            "ATOM   2509  CD1 LEU A 373      12.594   9.655  21.868  1.0   0.00      PROT C \n",
            "ATOM   2510  CD2 LEU A 373      13.092  10.473  19.595  1.0   0.00      PROT C \n",
            "ATOM   2511  H   LEU A 373      16.527  11.073  22.142  1.0   0.00      PROT H \n",
            "ATOM   2512  HA  LEU A 373      15.311  12.128  19.831  1.0   0.00      PROT H \n",
            "ATOM   2513  HB3 LEU A 373      13.296  12.330  21.489  1.0   0.00      PROT H \n",
            "ATOM   2514  HB2 LEU A 373      14.398  11.475  22.593  1.0   0.00      PROT H \n",
            "ATOM   2515  HG  LEU A 373      14.484   9.574  20.912  1.0   0.00      PROT H \n",
            "ATOM   2516 HD11 LEU A 373      13.006   9.510  22.867  1.0   0.00      PROT H \n",
            "ATOM   2517 HD12 LEU A 373      11.732  10.320  21.923  1.0   0.00      PROT H \n",
            "ATOM   2518 HD13 LEU A 373      12.285   8.693  21.459  1.0   0.00      PROT H \n",
            "ATOM   2519 HD21 LEU A 373      13.859  10.915  18.959  1.0   0.00      PROT H \n",
            "ATOM   2520 HD22 LEU A 373      12.784   9.514  19.179  1.0   0.00      PROT H \n",
            "ATOM   2521 HD23 LEU A 373      12.232  11.141  19.644  1.0   0.00      PROT H \n",
            "ATOM   2522  N   GLN A 374      15.702  14.194  22.406  1.0   0.00      PROT N \n",
            "ATOM   2523  CA  GLN A 374      15.702  15.599  22.814  1.0   0.00      PROT C \n",
            "ATOM   2524  C   GLN A 374      16.635  16.450  21.944  1.0   0.00      PROT C \n",
            "ATOM   2525  O   GLN A 374      16.276  17.558  21.540  1.0   0.00      PROT O \n",
            "ATOM   2526  CB  GLN A 374      16.072  15.703  24.297  1.0   0.00      PROT C \n",
            "ATOM   2527  CG  GLN A 374      15.005  15.102  25.214  1.0   0.00      PROT C \n",
            "ATOM   2528  CD  GLN A 374      15.399  15.083  26.686  1.0   0.00      PROT C \n",
            "ATOM   2529  OE1 GLN A 374      14.601  14.695  27.541  1.0   0.00      PROT O \n",
            "ATOM   2530  NE2 GLN A 374      16.633  15.497  26.988  1.0   0.00      PROT N \n",
            "ATOM   2531  H   GLN A 374      16.060  13.505  23.052  1.0   0.00      PROT H \n",
            "ATOM   2532  HA  GLN A 374      14.683  15.951  22.654  1.0   0.00      PROT H \n",
            "ATOM   2533  HB3 GLN A 374      16.203  16.758  24.536  1.0   0.00      PROT H \n",
            "ATOM   2534  HB2 GLN A 374      17.015  15.177  24.444  1.0   0.00      PROT H \n",
            "ATOM   2535  HG3 GLN A 374      14.823  14.083  24.874  1.0   0.00      PROT H \n",
            "ATOM   2536  HG2 GLN A 374      14.099  15.695  25.088  1.0   0.00      PROT H \n",
            "ATOM   2537 HE21 GLN A 374      17.261  15.810  26.248  1.0   0.00      PROT H \n",
            "ATOM   2538 HE22 GLN A 374      16.947  15.500  27.959  1.0   0.00      PROT H \n",
            "ATOM   2539  N   ALA A 375      17.819  15.912  21.649  1.0   0.00      PROT N \n",
            "ATOM   2540  CA  ALA A 375      18.820  16.576  20.810  1.0   0.00      PROT C \n",
            "ATOM   2541  C   ALA A 375      18.272  16.813  19.396  1.0   0.00      PROT C \n",
            "ATOM   2542  O   ALA A 375      18.483  17.868  18.799  1.0   0.00      PROT O \n",
            "ATOM   2543  CB  ALA A 375      20.093  15.726  20.745  1.0   0.00      PROT C \n",
            "ATOM   2544  H   ALA A 375      18.064  15.001  22.010  1.0   0.00      PROT H \n",
            "ATOM   2545  HA  ALA A 375      19.027  17.533  21.290  1.0   0.00      PROT H \n",
            "ATOM   2546  HB1 ALA A 375      20.473  15.564  21.753  1.0   0.00      PROT H \n",
            "ATOM   2547  HB2 ALA A 375      19.865  14.765  20.284  1.0   0.00      PROT H \n",
            "ATOM   2548  HB3 ALA A 375      20.846  16.244  20.152  1.0   0.00      PROT H \n",
            "ATOM   2549  N   VAL A 376      17.568  15.820  18.866  1.0   0.00      PROT N \n",
            "ATOM   2550  CA  VAL A 376      16.972  15.932  17.543  1.0   0.00      PROT C \n",
            "ATOM   2551  C   VAL A 376      15.937  17.061  17.552  1.0   0.00      PROT C \n",
            "ATOM   2552  O   VAL A 376      15.906  17.888  16.641  1.0   0.00      PROT O \n",
            "ATOM   2553  CB  VAL A 376      16.296  14.606  17.138  1.0   0.00      PROT C \n",
            "ATOM   2554  CG1 VAL A 376      15.497  14.782  15.860  1.0   0.00      PROT C \n",
            "ATOM   2555  CG2 VAL A 376      17.346  13.543  16.944  1.0   0.00      PROT C \n",
            "ATOM   2556  H   VAL A 376      17.436  14.962  19.383  1.0   0.00      PROT H \n",
            "ATOM   2557  HA  VAL A 376      17.792  16.159  16.861  1.0   0.00      PROT H \n",
            "ATOM   2558  HB  VAL A 376      15.613  14.283  17.924  1.0   0.00      PROT H \n",
            "ATOM   2559 HG11 VAL A 376      14.740  15.552  16.008  1.0   0.00      PROT H \n",
            "ATOM   2560 HG12 VAL A 376      16.165  15.080  15.052  1.0   0.00      PROT H \n",
            "ATOM   2561 HG13 VAL A 376      15.012  13.840  15.602  1.0   0.00      PROT H \n",
            "ATOM   2562 HG21 VAL A 376      17.915  13.422  17.866  1.0   0.00      PROT H \n",
            "ATOM   2563 HG22 VAL A 376      16.865  12.599  16.688  1.0   0.00      PROT H \n",
            "ATOM   2564 HG23 VAL A 376      18.018  13.839  16.138  1.0   0.00      PROT H \n",
            "ATOM   2565  N   LEU A 377      15.098  17.094  18.589  1.0   0.00      PROT N \n",
            "ATOM   2566  CA  LEU A 377      14.083  18.129  18.726  1.0   0.00      PROT C \n",
            "ATOM   2567  C   LEU A 377      14.726  19.503  18.884  1.0   0.00      PROT C \n",
            "ATOM   2568  O   LEU A 377      14.281  20.480  18.283  1.0   0.00      PROT O \n",
            "ATOM   2569  CB  LEU A 377      13.187  17.835  19.935  1.0   0.00      PROT C \n",
            "ATOM   2570  CG  LEU A 377      12.351  16.551  19.853  1.0   0.00      PROT C \n",
            "ATOM   2571  CD1 LEU A 377      11.708  16.257  21.208  1.0   0.00      PROT C \n",
            "ATOM   2572  CD2 LEU A 377      11.297  16.692  18.740  1.0   0.00      PROT C \n",
            "ATOM   2573  H   LEU A 377      15.156  16.387  19.308  1.0   0.00      PROT H \n",
            "ATOM   2574  HA  LEU A 377      13.509  18.095  17.800  1.0   0.00      PROT H \n",
            "ATOM   2575  HB2 LEU A 377      13.803  17.740  20.829  1.0   0.00      PROT H \n",
            "ATOM   2576  HB3 LEU A 377      12.477  18.651  20.068  1.0   0.00      PROT H \n",
            "ATOM   2577  HG  LEU A 377      12.995  15.705  19.611  1.0   0.00      PROT H \n",
            "ATOM   2578 HD11 LEU A 377      12.485  16.165  21.967  1.0   0.00      PROT H \n",
            "ATOM   2579 HD12 LEU A 377      11.035  17.071  21.475  1.0   0.00      PROT H \n",
            "ATOM   2580 HD13 LEU A 377      11.146  15.325  21.149  1.0   0.00      PROT H \n",
            "ATOM   2581 HD21 LEU A 377      11.794  16.901  17.793  1.0   0.00      PROT H \n",
            "ATOM   2582 HD22 LEU A 377      10.731  15.764  18.656  1.0   0.00      PROT H \n",
            "ATOM   2583 HD23 LEU A 377      10.620  17.511  18.983  1.0   0.00      PROT H \n",
            "ATOM   2584  N   LEU A 378      15.778  19.570  19.694  1.0   0.00      PROT N \n",
            "ATOM   2585  CA  LEU A 378      16.490  20.821  19.942  1.0   0.00      PROT C \n",
            "ATOM   2586  C   LEU A 378      17.106  21.415  18.679  1.0   0.00      PROT C \n",
            "ATOM   2587  O   LEU A 378      16.917  22.602  18.390  1.0   0.00      PROT O \n",
            "ATOM   2588  CB  LEU A 378      17.603  20.606  20.978  1.0   0.00      PROT C \n",
            "ATOM   2589  CG  LEU A 378      18.508  21.809  21.290  1.0   0.00      PROT C \n",
            "ATOM   2590  CD1 LEU A 378      17.772  22.795  22.165  1.0   0.00      PROT C \n",
            "ATOM   2591  CD2 LEU A 378      19.780  21.340  21.987  1.0   0.00      PROT C \n",
            "ATOM   2592  H   LEU A 378      16.109  18.738  20.161  1.0   0.00      PROT H \n",
            "ATOM   2593  HA  LEU A 378      15.710  21.490  20.306  1.0   0.00      PROT H \n",
            "ATOM   2594  HB2 LEU A 378      17.161  20.317  21.931  1.0   0.00      PROT H \n",
            "ATOM   2595  HB3 LEU A 378      18.272  19.817  20.634  1.0   0.00      PROT H \n",
            "ATOM   2596  HG  LEU A 378      18.780  22.314  20.363  1.0   0.00      PROT H \n",
            "ATOM   2597 HD11 LEU A 378      16.865  23.124  21.658  1.0   0.00      PROT H \n",
            "ATOM   2598 HD12 LEU A 378      17.508  22.317  23.108  1.0   0.00      PROT H \n",
            "ATOM   2599 HD13 LEU A 378      18.412  23.656  22.360  1.0   0.00      PROT H \n",
            "ATOM   2600 HD21 LEU A 378      20.304  20.629  21.348  1.0   0.00      PROT H \n",
            "ATOM   2601 HD22 LEU A 378      20.425  22.197  22.182  1.0   0.00      PROT H \n",
            "ATOM   2602 HD23 LEU A 378      19.521  20.858  22.930  1.0   0.00      PROT H \n",
            "ATOM   2603  N   MET A 379      17.843  20.588  17.936  1.0   0.00      PROT N \n",
            "ATOM   2604  CA  MET A 379      18.508  21.032  16.713  1.0   0.00      PROT C \n",
            "ATOM   2605  C   MET A 379      17.628  21.029  15.464  1.0   0.00      PROT C \n",
            "ATOM   2606  O   MET A 379      17.989  20.432  14.446  1.0   0.00      PROT O \n",
            "ATOM   2607  CB  MET A 379      19.753  20.176  16.455  1.0   0.00      PROT C \n",
            "ATOM   2608  CG  MET A 379      20.800  20.263  17.552  1.0   0.00      PROT C \n",
            "ATOM   2609  SD  MET A 379      21.351  21.947  17.894  1.0   0.00      PROT S \n",
            "ATOM   2610  CE  MET A 379      22.849  21.642  18.777  1.0   0.00      PROT C \n",
            "ATOM   2611  H   MET A 379      17.954  19.623  18.214  1.0   0.00      PROT H \n",
            "ATOM   2612  HA  MET A 379      18.747  22.071  16.939  1.0   0.00      PROT H \n",
            "ATOM   2613  HB3 MET A 379      20.185  20.512  15.513  1.0   0.00      PROT H \n",
            "ATOM   2614  HB2 MET A 379      19.418  19.144  16.353  1.0   0.00      PROT H \n",
            "ATOM   2615  HG2 MET A 379      20.390  19.861  18.478  1.0   0.00      PROT H \n",
            "ATOM   2616  HG3 MET A 379      21.678  19.686  17.263  1.0   0.00      PROT H \n",
            "ATOM   2617  HE1 MET A 379      23.306  22.591  19.056  1.0   0.00      PROT H \n",
            "ATOM   2618  HE2 MET A 379      22.630  21.066  19.676  1.0   0.00      PROT H \n",
            "ATOM   2619  HE3 MET A 379      23.536  21.080  18.144  1.0   0.00      PROT H \n",
            "ATOM   2620  N   SER A 380      16.478  21.699  15.540  1.0   0.00      PROT N \n",
            "ATOM   2621  CA  SER A 380      15.570  21.784  14.401  1.0   0.00      PROT C \n",
            "ATOM   2622  C   SER A 380      15.818  23.096  13.657  1.0   0.00      PROT C \n",
            "ATOM   2623  O   SER A 380      15.492  24.159  14.168  1.0   0.00      PROT O \n",
            "ATOM   2624  CB  SER A 380      14.129  21.713  14.886  1.0   0.00      PROT C \n",
            "ATOM   2625  OG  SER A 380      13.927  20.525  15.618  1.0   0.00      PROT O \n",
            "ATOM   2626  H   SER A 380      16.219  22.162  16.400  1.0   0.00      PROT H \n",
            "ATOM   2627  HA  SER A 380      15.780  20.933  13.753  1.0   0.00      PROT H \n",
            "ATOM   2628  HB2 SER A 380      13.918  22.570  15.526  1.0   0.00      PROT H \n",
            "ATOM   2629  HB3 SER A 380      13.456  21.726  14.029  1.0   0.00      PROT H \n",
            "ATOM   2630  HG  SER A 380      13.037  20.327  16.004  1.0   0.00      PROT H \n",
            "ATOM   2631  N   SER A 381      16.387  23.018  12.455  1.0   0.00      PROT N \n",
            "ATOM   2632  CA  SER A 381      16.709  24.221  11.673  1.0   0.00      PROT C \n",
            "ATOM   2633  C   SER A 381      15.542  24.911  10.966  1.0   0.00      PROT C \n",
            "ATOM   2634  O   SER A 381      15.713  25.962  10.346  1.0   0.00      PROT O \n",
            "ATOM   2635  CB  SER A 381      17.824  23.917  10.650  1.0   0.00      PROT C \n",
            "ATOM   2636  OG  SER A 381      17.538  22.770   9.864  1.0   0.00      PROT O \n",
            "ATOM   2637  H   SER A 381      16.608  22.116  12.059  1.0   0.00      PROT H \n",
            "ATOM   2638  HA  SER A 381      17.021  24.903  12.464  1.0   0.00      PROT H \n",
            "ATOM   2639  HB2 SER A 381      17.941  24.766   9.977  1.0   0.00      PROT H \n",
            "ATOM   2640  HB3 SER A 381      18.762  23.739  11.177  1.0   0.00      PROT H \n",
            "ATOM   2641  HG  SER A 381      18.179  22.473   9.171  1.0   0.00      PROT H \n",
            "ATOM   2642  N   ASP A 382      14.356  24.334  11.076  1.0   0.00      PROT N \n",
            "ATOM   2643  CA  ASP A 382      13.176  24.904  10.456  1.0   0.00      PROT C \n",
            "ATOM   2644  C   ASP A 382      12.546  26.023  11.293  1.0   0.00      PROT C \n",
            "ATOM   2645  O   ASP A 382      11.582  26.659  10.853  1.0   0.00      PROT O \n",
            "ATOM   2646  CB  ASP A 382      12.149  23.801  10.199  1.0   0.00      PROT C \n",
            "ATOM   2647  CG  ASP A 382      11.961  22.894  11.403  1.0   0.00      PROT C \n",
            "ATOM   2648  OD1 ASP A 382      10.800  22.654  11.813  1.0   0.00      PROT O \n",
            "ATOM   2649  OD2 ASP A 382      12.978  22.412  11.947  1.0   0.00      PROT O \n",
            "ATOM   2650  H   ASP A 382      14.258  23.477  11.601  1.0   0.00      PROT H \n",
            "ATOM   2651  HA  ASP A 382      13.546  25.334   9.525  1.0   0.00      PROT H \n",
            "ATOM   2652  HB3 ASP A 382      12.494  23.211   9.350  1.0   0.00      PROT H \n",
            "ATOM   2653  HB2 ASP A 382      11.200  24.276   9.950  1.0   0.00      PROT H \n",
            "ATOM   2654  HD1 ASP A 382      10.836  22.045  12.605  1.0   0.00      PROT H \n",
            "ATOM   2655  N   ARG A 383      13.070  26.265  12.494  1.0   0.00      PROT N \n",
            "ATOM   2656  CA  ARG A 383      12.522  27.326  13.345  1.0   0.00      PROT C \n",
            "ATOM   2657  C   ARG A 383      12.719  28.684  12.648  1.0   0.00      PROT C \n",
            "ATOM   2658  O   ARG A 383      13.733  28.915  11.994  1.0   0.00      PROT O \n",
            "ATOM   2659  CB  ARG A 383      13.224  27.360  14.705  1.0   0.00      PROT C \n",
            "ATOM   2660  CG  ARG A 383      13.433  26.020  15.382  1.0   0.00      PROT C \n",
            "ATOM   2661  CD  ARG A 383      12.166  25.412  15.934  1.0   0.00      PROT C \n",
            "ATOM   2662  NE  ARG A 383      12.476  24.272  16.797  1.0   0.00      PROT N \n",
            "ATOM   2663  CZ  ARG A 383      11.582  23.388  17.230  1.0   0.00      PROT C \n",
            "ATOM   2664  NH1 ARG A 383      10.304  23.506  16.883  1.0   0.00      PROT N \n",
            "ATOM   2665  NH2 ARG A 383      11.972  22.379  17.997  1.0   0.00      PROT N \n",
            "ATOM   2666  H   ARG A 383      13.851  25.716  12.824  1.0   0.00      PROT H \n",
            "ATOM   2667  HA  ARG A 383      11.468  27.085  13.482  1.0   0.00      PROT H \n",
            "ATOM   2668  HB3 ARG A 383      12.613  27.998  15.343  1.0   0.00      PROT H \n",
            "ATOM   2669  HB2 ARG A 383      14.190  27.835  14.534  1.0   0.00      PROT H \n",
            "ATOM   2670  HG3 ARG A 383      14.145  26.175  16.193  1.0   0.00      PROT H \n",
            "ATOM   2671  HG2 ARG A 383      13.866  25.348  14.642  1.0   0.00      PROT H \n",
            "ATOM   2672  HD2 ARG A 383      11.628  26.160  16.516  1.0   0.00      PROT H \n",
            "ATOM   2673  HD3 ARG A 383      11.538  25.071  15.111  1.0   0.00      PROT H \n",
            "ATOM   2674  HE  ARG A 383      13.446  24.181  17.063  1.0   0.00      PROT H \n",
            "ATOM   2675 HH12 ARG A 383       9.622  22.820  17.207  1.0   0.00      PROT H \n",
            "ATOM   2676 HH11 ARG A 383      10.005  24.283  16.293  1.0   0.00      PROT H \n",
            "ATOM   2677 HH22 ARG A 383      11.291  21.692  18.321  1.0   0.00      PROT H \n",
            "ATOM   2678 HH21 ARG A 383      12.951  22.084  18.402  1.0   0.00      PROT H \n",
            "ATOM   2679  N   PRO A 384      11.739  29.590  12.767  1.0   0.00      PROT N \n",
            "ATOM   2680  CA  PRO A 384      11.798  30.923  12.155  1.0   0.00      PROT C \n",
            "ATOM   2681  C   PRO A 384      12.865  31.839  12.744  1.0   0.00      PROT C \n",
            "ATOM   2682  O   PRO A 384      13.049  31.878  13.963  1.0   0.00      PROT O \n",
            "ATOM   2683  CB  PRO A 384      10.401  31.485  12.416  1.0   0.00      PROT C \n",
            "ATOM   2684  CG  PRO A 384       9.545  30.263  12.441  1.0   0.00      PROT C \n",
            "ATOM   2685  CD  PRO A 384      10.377  29.309  13.257  1.0   0.00      PROT C \n",
            "ATOM   2686  HA  PRO A 384      12.077  30.819  11.106  1.0   0.00      PROT H \n",
            "ATOM   2687  HB3 PRO A 384      10.062  32.150  11.621  1.0   0.00      PROT H \n",
            "ATOM   2688  HB2 PRO A 384      10.329  31.999  13.375  1.0   0.00      PROT H \n",
            "ATOM   2689  HG2 PRO A 384       8.588  30.499  12.907  1.0   0.00      PROT H \n",
            "ATOM   2690  HG3 PRO A 384       9.377  29.916  11.421  1.0   0.00      PROT H \n",
            "ATOM   2691  HD2 PRO A 384      10.251  29.532  14.317  1.0   0.00      PROT H \n",
            "ATOM   2692  HD3 PRO A 384      10.056  28.286  13.060  1.0   0.00      PROT H \n",
            "ATOM   2693  N   GLY A 385      13.551  32.577  11.874  1.0   0.00      PROT N \n",
            "ATOM   2694  CA  GLY A 385      14.558  33.523  12.314  1.0   0.00      PROT C \n",
            "ATOM   2695  C   GLY A 385      15.951  33.018  12.624  1.0   0.00      PROT C \n",
            "ATOM   2696  O   GLY A 385      16.836  33.823  12.935  1.0   0.00      PROT O \n",
            "ATOM   2697  H   GLY A 385      13.375  32.484  10.884  1.0   0.00      PROT H \n",
            "ATOM   2698  HA2 GLY A 385      14.131  33.983  13.206  1.0   0.00      PROT H \n",
            "ATOM   2699  HA3 GLY A 385      14.609  34.261  11.514  1.0   0.00      PROT H \n",
            "ATOM   2700  N   LEU A 386      16.162  31.707  12.556  1.0   0.00      PROT N \n",
            "ATOM   2701  CA  LEU A 386      17.483  31.149  12.844  1.0   0.00      PROT C \n",
            "ATOM   2702  C   LEU A 386      18.522  31.680  11.875  1.0   0.00      PROT C \n",
            "ATOM   2703  O   LEU A 386      18.243  31.874  10.697  1.0   0.00      PROT O \n",
            "ATOM   2704  CB  LEU A 386      17.464  29.623  12.766  1.0   0.00      PROT C \n",
            "ATOM   2705  CG  LEU A 386      16.803  28.904  13.942  1.0   0.00      PROT C \n",
            "ATOM   2706  CD1 LEU A 386      16.731  27.405  13.642  1.0   0.00      PROT C \n",
            "ATOM   2707  CD2 LEU A 386      17.596  29.181  15.229  1.0   0.00      PROT C \n",
            "ATOM   2708  H   LEU A 386      15.406  31.086  12.304  1.0   0.00      PROT H \n",
            "ATOM   2709  HA  LEU A 386      17.711  31.481  13.857  1.0   0.00      PROT H \n",
            "ATOM   2710  HB2 LEU A 386      16.920  29.312  11.874  1.0   0.00      PROT H \n",
            "ATOM   2711  HB3 LEU A 386      18.487  29.249  12.717  1.0   0.00      PROT H \n",
            "ATOM   2712  HG  LEU A 386      15.787  29.274  14.081  1.0   0.00      PROT H \n",
            "ATOM   2713 HD11 LEU A 386      16.163  27.245  12.725  1.0   0.00      PROT H \n",
            "ATOM   2714 HD12 LEU A 386      17.740  27.010  13.518  1.0   0.00      PROT H \n",
            "ATOM   2715 HD13 LEU A 386      16.239  26.892  14.468  1.0   0.00      PROT H \n",
            "ATOM   2716 HD21 LEU A 386      17.628  30.255  15.412  1.0   0.00      PROT H \n",
            "ATOM   2717 HD22 LEU A 386      17.111  28.683  16.069  1.0   0.00      PROT H \n",
            "ATOM   2718 HD23 LEU A 386      18.612  28.801  15.119  1.0   0.00      PROT H \n",
            "ATOM   2719  N   ALA A 387      19.729  31.908  12.375  1.0   0.00      PROT N \n",
            "ATOM   2720  CA  ALA A 387      20.808  32.411  11.539  1.0   0.00      PROT C \n",
            "ATOM   2721  C   ALA A 387      21.681  31.278  11.000  1.0   0.00      PROT C \n",
            "ATOM   2722  O   ALA A 387      21.966  31.226   9.809  1.0   0.00      PROT O \n",
            "ATOM   2723  CB  ALA A 387      21.651  33.381  12.324  1.0   0.00      PROT C \n",
            "ATOM   2724  H   ALA A 387      19.912  31.732  13.353  1.0   0.00      PROT H \n",
            "ATOM   2725  HA  ALA A 387      20.319  32.922  10.709  1.0   0.00      PROT H \n",
            "ATOM   2726  HB1 ALA A 387      21.021  34.185  12.704  1.0   0.00      PROT H \n",
            "ATOM   2727  HB2 ALA A 387      22.120  32.861  13.159  1.0   0.00      PROT H \n",
            "ATOM   2728  HB3 ALA A 387      22.422  33.799  11.677  1.0   0.00      PROT H \n",
            "ATOM   2729  N   CYS A 388      22.082  30.363  11.879  1.0   0.00      PROT N \n",
            "ATOM   2730  CA  CYS A 388      22.942  29.251  11.496  1.0   0.00      PROT C \n",
            "ATOM   2731  C   CYS A 388      22.150  28.001  11.153  1.0   0.00      PROT C \n",
            "ATOM   2732  O   CYS A 388      22.289  26.966  11.804  1.0   0.00      PROT O \n",
            "ATOM   2733  CB  CYS A 388      23.928  28.946  12.625  1.0   0.00      PROT C \n",
            "ATOM   2734  SG  CYS A 388      24.815  30.412  13.253  1.0   0.00      PROT S \n",
            "ATOM   2735  H   CYS A 388      21.790  30.427  12.844  1.0   0.00      PROT H \n",
            "ATOM   2736  HA  CYS A 388      23.455  29.592  10.596  1.0   0.00      PROT H \n",
            "ATOM   2737  HB2 CYS A 388      23.392  28.509  13.468  1.0   0.00      PROT H \n",
            "ATOM   2738  HB3 CYS A 388      24.681  28.243  12.270  1.0   0.00      PROT H \n",
            "ATOM   2739  HG  CYS A 388      25.708  30.281  14.243  1.0   0.00      PROT H \n",
            "ATOM   2740  N   VAL A 389      21.325  28.103  10.119  1.0   0.00      PROT N \n",
            "ATOM   2741  CA  VAL A 389      20.498  26.988   9.670  1.0   0.00      PROT C \n",
            "ATOM   2742  C   VAL A 389      21.339  25.775   9.263  1.0   0.00      PROT C \n",
            "ATOM   2743  O   VAL A 389      21.021  24.637   9.610  1.0   0.00      PROT O \n",
            "ATOM   2744  CB  VAL A 389      19.610  27.407   8.464  1.0   0.00      PROT C \n",
            "ATOM   2745  CG1 VAL A 389      18.739  26.242   8.021  1.0   0.00      PROT C \n",
            "ATOM   2746  CG2 VAL A 389      18.746  28.616   8.844  1.0   0.00      PROT C \n",
            "ATOM   2747  H   VAL A 389      21.254  28.975   9.614  1.0   0.00      PROT H \n",
            "ATOM   2748  HA  VAL A 389      19.907  26.745  10.553  1.0   0.00      PROT H \n",
            "ATOM   2749  HB  VAL A 389      20.238  27.685   7.617  1.0   0.00      PROT H \n",
            "ATOM   2750 HG11 VAL A 389      19.372  25.395   7.756  1.0   0.00      PROT H \n",
            "ATOM   2751 HG12 VAL A 389      18.072  25.957   8.835  1.0   0.00      PROT H \n",
            "ATOM   2752 HG13 VAL A 389      18.148  26.539   7.155  1.0   0.00      PROT H \n",
            "ATOM   2753 HG21 VAL A 389      19.389  29.438   9.158  1.0   0.00      PROT H \n",
            "ATOM   2754 HG22 VAL A 389      18.155  28.926   7.982  1.0   0.00      PROT H \n",
            "ATOM   2755 HG23 VAL A 389      18.079  28.343   9.662  1.0   0.00      PROT H \n",
            "ATOM   2756  N   GLU A 390      22.412  26.021   8.525  1.0   0.00      PROT N \n",
            "ATOM   2757  CA  GLU A 390      23.271  24.939   8.074  1.0   0.00      PROT C \n",
            "ATOM   2758  C   GLU A 390      23.976  24.243   9.238  1.0   0.00      PROT C \n",
            "ATOM   2759  O   GLU A 390      23.925  23.021   9.365  1.0   0.00      PROT O \n",
            "ATOM   2760  CB  GLU A 390      24.306  25.481   7.098  1.0   0.00      PROT C \n",
            "ATOM   2761  CG  GLU A 390      25.194  24.409   6.500  1.0   0.00      PROT C \n",
            "ATOM   2762  CD  GLU A 390      26.428  24.994   5.842  1.0   0.00      PROT C \n",
            "ATOM   2763  OE1 GLU A 390      27.324  25.456   6.593  1.0   0.00      PROT O \n",
            "ATOM   2764  OE2 GLU A 390      26.486  25.000   4.584  1.0   0.00      PROT O \n",
            "ATOM   2765  H   GLU A 390      22.642  26.971   8.269  1.0   0.00      PROT H \n",
            "ATOM   2766  HA  GLU A 390      22.600  24.232   7.587  1.0   0.00      PROT H \n",
            "ATOM   2767  HB3 GLU A 390      24.920  26.200   7.640  1.0   0.00      PROT H \n",
            "ATOM   2768  HB2 GLU A 390      23.766  25.992   6.301  1.0   0.00      PROT H \n",
            "ATOM   2769  HG2 GLU A 390      25.519  23.726   7.285  1.0   0.00      PROT H \n",
            "ATOM   2770  HG3 GLU A 390      24.636  23.855   5.745  1.0   0.00      PROT H \n",
            "ATOM   2771  HE2 GLU A 390      27.341  25.422   4.283  1.0   0.00      PROT H \n",
            "ATOM   2772  N   ARG A 391      24.645  25.029  10.075  1.0   0.00      PROT N \n",
            "ATOM   2773  CA  ARG A 391      25.355  24.499  11.227  1.0   0.00      PROT C \n",
            "ATOM   2774  C   ARG A 391      24.434  23.691  12.167  1.0   0.00      PROT C \n",
            "ATOM   2775  O   ARG A 391      24.866  22.710  12.771  1.0   0.00      PROT O \n",
            "ATOM   2776  CB  ARG A 391      26.021  25.649  11.990  1.0   0.00      PROT C \n",
            "ATOM   2777  CG  ARG A 391      26.821  25.204  13.207  1.0   0.00      PROT C \n",
            "ATOM   2778  CD  ARG A 391      28.171  25.904  13.266  1.0   0.00      PROT C \n",
            "ATOM   2779  NE  ARG A 391      28.085  27.345  13.520  1.0   0.00      PROT N \n",
            "ATOM   2780  CZ  ARG A 391      28.066  27.900  14.732  1.0   0.00      PROT C \n",
            "ATOM   2781  NH1 ARG A 391      28.121  27.138  15.820  1.0   0.00      PROT N \n",
            "ATOM   2782  NH2 ARG A 391      28.021  29.224  14.858  1.0   0.00      PROT N \n",
            "ATOM   2783  H   ARG A 391      24.670  26.027   9.920  1.0   0.00      PROT H \n",
            "ATOM   2784  HA  ARG A 391      26.100  23.826  10.802  1.0   0.00      PROT H \n",
            "ATOM   2785  HB3 ARG A 391      25.228  26.328  12.305  1.0   0.00      PROT H \n",
            "ATOM   2786  HB2 ARG A 391      26.680  26.161  11.289  1.0   0.00      PROT H \n",
            "ATOM   2787  HG3 ARG A 391      26.973  24.126  13.147  1.0   0.00      PROT H \n",
            "ATOM   2788  HG2 ARG A 391      26.249  25.441  14.104  1.0   0.00      PROT H \n",
            "ATOM   2789  HD2 ARG A 391      28.770  25.471  14.067  1.0   0.00      PROT H \n",
            "ATOM   2790  HD3 ARG A 391      28.689  25.775  12.316  1.0   0.00      PROT H \n",
            "ATOM   2791  HE  ARG A 391      28.038  27.928  12.697  1.0   0.00      PROT H \n",
            "ATOM   2792 HH12 ARG A 391      28.129  27.569  16.745  1.0   0.00      PROT H \n",
            "ATOM   2793 HH11 ARG A 391      28.156  26.123  15.730  1.0   0.00      PROT H \n",
            "ATOM   2794  HH2 ARG A 391      28.029  29.649  15.785  1.0   0.00      PROT H \n",
            "ATOM   2795  N   ILE A 392      23.169  24.100  12.278  1.0   0.00      PROT N \n",
            "ATOM   2796  CA  ILE A 392      22.206  23.415  13.135  1.0   0.00      PROT C \n",
            "ATOM   2797  C   ILE A 392      21.747  22.103  12.497  1.0   0.00      PROT C \n",
            "ATOM   2798  O   ILE A 392      21.631  21.070  13.167  1.0   0.00      PROT O \n",
            "ATOM   2799  CB  ILE A 392      20.970  24.323  13.404  1.0   0.00      PROT C \n",
            "ATOM   2800  CG1 ILE A 392      21.382  25.491  14.296  1.0   0.00      PROT C \n",
            "ATOM   2801  CG2 ILE A 392      19.829  23.522  14.057  1.0   0.00      PROT C \n",
            "ATOM   2802  CD1 ILE A 392      20.367  26.608  14.358  1.0   0.00      PROT C \n",
            "ATOM   2803  H   ILE A 392      22.852  24.907  11.760  1.0   0.00      PROT H \n",
            "ATOM   2804  HA  ILE A 392      22.754  23.205  14.053  1.0   0.00      PROT H \n",
            "ATOM   2805  HB  ILE A 392      20.596  24.723  12.462  1.0   0.00      PROT H \n",
            "ATOM   2806 HG13 ILE A 392      22.323  25.877  13.905  1.0   0.00      PROT H \n",
            "ATOM   2807 HG12 ILE A 392      21.542  25.089  15.297  1.0   0.00      PROT H \n",
            "ATOM   2808 HG21 ILE A 392      19.548  22.693  13.407  1.0   0.00      PROT H \n",
            "ATOM   2809 HG22 ILE A 392      20.163  23.132  15.019  1.0   0.00      PROT H \n",
            "ATOM   2810 HG23 ILE A 392      18.968  24.173  14.208  1.0   0.00      PROT H \n",
            "ATOM   2811 HD11 ILE A 392      20.736  27.398  15.012  1.0   0.00      PROT H \n",
            "ATOM   2812 HD12 ILE A 392      20.207  27.010  13.357  1.0   0.00      PROT H \n",
            "ATOM   2813 HD13 ILE A 392      19.426  26.222  14.749  1.0   0.00      PROT H \n",
            "ATOM   2814  N   GLU A 393      21.488  22.161  11.196  1.0   0.00      PROT N \n",
            "ATOM   2815  CA  GLU A 393      21.045  20.998  10.445  1.0   0.00      PROT C \n",
            "ATOM   2816  C   GLU A 393      22.141  19.932  10.476  1.0   0.00      PROT C \n",
            "ATOM   2817  O   GLU A 393      21.855  18.738  10.476  1.0   0.00      PROT O \n",
            "ATOM   2818  CB  GLU A 393      20.714  21.415   9.003  1.0   0.00      PROT C \n",
            "ATOM   2819  CG  GLU A 393      19.820  20.442   8.244  1.0   0.00      PROT C \n",
            "ATOM   2820  CD  GLU A 393      19.066  21.099   7.085  1.0   0.00      PROT C \n",
            "ATOM   2821  OE1 GLU A 393      18.329  22.080   7.325  1.0   0.00      PROT O \n",
            "ATOM   2822  OE2 GLU A 393      19.202  20.626   5.933  1.0   0.00      PROT O \n",
            "ATOM   2823  H   GLU A 393      21.597  23.033  10.698  1.0   0.00      PROT H \n",
            "ATOM   2824  HA  GLU A 393      20.149  20.617  10.935  1.0   0.00      PROT H \n",
            "ATOM   2825  HB3 GLU A 393      21.665  21.518   8.480  1.0   0.00      PROT H \n",
            "ATOM   2826  HB2 GLU A 393      20.225  22.387   9.062  1.0   0.00      PROT H \n",
            "ATOM   2827  HG2 GLU A 393      20.428  19.638   7.829  1.0   0.00      PROT H \n",
            "ATOM   2828  HG3 GLU A 393      19.079  20.023   8.925  1.0   0.00      PROT H \n",
            "ATOM   2829  HE2 GLU A 393      18.647  21.145   5.283  1.0   0.00      PROT H \n",
            "ATOM   2830  N   LYS A 394      23.396  20.371  10.521  1.0   0.00      PROT N \n",
            "ATOM   2831  CA  LYS A 394      24.525  19.454  10.562  1.0   0.00      PROT C \n",
            "ATOM   2832  C   LYS A 394      24.568  18.733  11.898  1.0   0.00      PROT C \n",
            "ATOM   2833  O   LYS A 394      24.757  17.519  11.946  1.0   0.00      PROT O \n",
            "ATOM   2834  CB  LYS A 394      25.846  20.201  10.364  1.0   0.00      PROT C \n",
            "ATOM   2835  CG  LYS A 394      27.039  19.265  10.277  1.0   0.00      PROT C \n",
            "ATOM   2836  CD  LYS A 394      28.380  19.970  10.454  1.0   0.00      PROT C \n",
            "ATOM   2837  CE  LYS A 394      29.523  18.963  10.283  1.0   0.00      PROT C \n",
            "ATOM   2838  NZ  LYS A 394      30.883  19.570  10.360  1.0   0.00      PROT N \n",
            "ATOM   2839  H   LYS A 394      23.584  21.363  10.527  1.0   0.00      PROT H \n",
            "ATOM   2840  HA  LYS A 394      24.350  18.753   9.746  1.0   0.00      PROT H \n",
            "ATOM   2841  HB3 LYS A 394      25.977  20.879  11.208  1.0   0.00      PROT H \n",
            "ATOM   2842  HB2 LYS A 394      25.769  20.778   9.443  1.0   0.00      PROT H \n",
            "ATOM   2843  HG2 LYS A 394      26.964  18.507  11.057  1.0   0.00      PROT H \n",
            "ATOM   2844  HG3 LYS A 394      27.051  18.782   9.300  1.0   0.00      PROT H \n",
            "ATOM   2845  HD2 LYS A 394      28.478  20.757   9.706  1.0   0.00      PROT H \n",
            "ATOM   2846  HD3 LYS A 394      28.432  20.408  11.451  1.0   0.00      PROT H \n",
            "ATOM   2847  HE2 LYS A 394      29.463  18.208  11.067  1.0   0.00      PROT H \n",
            "ATOM   2848  HE3 LYS A 394      29.440  18.482   9.309  1.0   0.00      PROT H \n",
            "ATOM   2849  HZ1 LYS A 394      31.396  19.152  11.123  1.0   0.00      PROT H \n",
            "ATOM   2850  HZ2 LYS A 394      31.375  19.405   9.493  1.0   0.00      PROT H \n",
            "ATOM   2851  N   TYR A 395      24.406  19.485  12.985  1.0   0.00      PROT N \n",
            "ATOM   2852  CA  TYR A 395      24.426  18.897  14.323  1.0   0.00      PROT C \n",
            "ATOM   2853  C   TYR A 395      23.290  17.884  14.447  1.0   0.00      PROT C \n",
            "ATOM   2854  O   TYR A 395      23.449  16.820  15.040  1.0   0.00      PROT O \n",
            "ATOM   2855  CB  TYR A 395      24.269  19.979  15.399  1.0   0.00      PROT C \n",
            "ATOM   2856  CG  TYR A 395      25.403  20.989  15.479  1.0   0.00      PROT C \n",
            "ATOM   2857  CD1 TYR A 395      26.720  20.628  15.173  1.0   0.00      PROT C \n",
            "ATOM   2858  CD2 TYR A 395      25.170  22.293  15.936  1.0   0.00      PROT C \n",
            "ATOM   2859  CE1 TYR A 395      27.776  21.539  15.324  1.0   0.00      PROT C \n",
            "ATOM   2860  CE2 TYR A 395      26.218  23.206  16.094  1.0   0.00      PROT C \n",
            "ATOM   2861  CZ  TYR A 395      27.519  22.823  15.788  1.0   0.00      PROT C \n",
            "ATOM   2862  OH  TYR A 395      28.552  23.723  15.968  1.0   0.00      PROT O \n",
            "ATOM   2863  H   TYR A 395      24.266  20.481  12.894  1.0   0.00      PROT H \n",
            "ATOM   2864  HA  TYR A 395      25.396  18.411  14.425  1.0   0.00      PROT H \n",
            "ATOM   2865  HB3 TYR A 395      24.180  19.456  16.351  1.0   0.00      PROT H \n",
            "ATOM   2866  HB2 TYR A 395      23.335  20.497  15.181  1.0   0.00      PROT H \n",
            "ATOM   2867  HD1 TYR A 395      26.940  19.624  14.810  1.0   0.00      PROT H \n",
            "ATOM   2868  HD2 TYR A 395      24.148  22.591  16.170  1.0   0.00      PROT H \n",
            "ATOM   2869  HE1 TYR A 395      28.796  21.242  15.079  1.0   0.00      PROT H \n",
            "ATOM   2870  HE2 TYR A 395      26.018  24.215  16.456  1.0   0.00      PROT H \n",
            "ATOM   2871  HH  TYR A 395      29.482  23.459  15.753  1.0   0.00      PROT H \n",
            "ATOM   2872  N   GLN A 396      22.143  18.219  13.871  1.0   0.00      PROT N \n",
            "ATOM   2873  CA  GLN A 396      21.003  17.325  13.917  1.0   0.00      PROT C \n",
            "ATOM   2874  C   GLN A 396      21.301  16.034  13.164  1.0   0.00      PROT C \n",
            "ATOM   2875  O   GLN A 396      20.871  14.960  13.583  1.0   0.00      PROT O \n",
            "ATOM   2876  CB  GLN A 396      19.775  17.988  13.313  1.0   0.00      PROT C \n",
            "ATOM   2877  CG  GLN A 396      18.500  17.215  13.569  1.0   0.00      PROT C \n",
            "ATOM   2878  CD  GLN A 396      17.317  17.824  12.862  1.0   0.00      PROT C \n",
            "ATOM   2879  OE1 GLN A 396      16.191  17.778  13.359  1.0   0.00      PROT O \n",
            "ATOM   2880  NE2 GLN A 396      17.561  18.395  11.690  1.0   0.00      PROT N \n",
            "ATOM   2881  H   GLN A 396      22.057  19.105  13.394  1.0   0.00      PROT H \n",
            "ATOM   2882  HA  GLN A 396      20.844  17.124  14.977  1.0   0.00      PROT H \n",
            "ATOM   2883  HB3 GLN A 396      19.939  18.073  12.239  1.0   0.00      PROT H \n",
            "ATOM   2884  HB2 GLN A 396      19.689  18.984  13.747  1.0   0.00      PROT H \n",
            "ATOM   2885  HG3 GLN A 396      18.317  17.208  14.644  1.0   0.00      PROT H \n",
            "ATOM   2886  HG2 GLN A 396      18.646  16.193  13.219  1.0   0.00      PROT H \n",
            "ATOM   2887 HE21 GLN A 396      18.510  18.409  11.317  1.0   0.00      PROT H \n",
            "ATOM   2888 HE22 GLN A 396      16.799  18.820  11.161  1.0   0.00      PROT H \n",
            "ATOM   2889  N   ASP A 397      22.025  16.134  12.051  1.0   0.00      PROT N \n",
            "ATOM   2890  CA  ASP A 397      22.365  14.941  11.283  1.0   0.00      PROT C \n",
            "ATOM   2891  C   ASP A 397      23.302  14.061  12.096  1.0   0.00      PROT C \n",
            "ATOM   2892  O   ASP A 397      23.180  12.838  12.100  1.0   0.00      PROT O \n",
            "ATOM   2893  CB  ASP A 397      23.022  15.310   9.950  1.0   0.00      PROT C \n",
            "ATOM   2894  CG  ASP A 397      22.046  15.926   8.970  1.0   0.00      PROT C \n",
            "ATOM   2895  OD1 ASP A 397      20.856  15.535   8.980  1.0   0.00      PROT O \n",
            "ATOM   2896  OD2 ASP A 397      22.472  16.794   8.175  1.0   0.00      PROT O \n",
            "ATOM   2897  H   ASP A 397      22.342  17.039  11.734  1.0   0.00      PROT H \n",
            "ATOM   2898  HA  ASP A 397      21.422  14.428  11.096  1.0   0.00      PROT H \n",
            "ATOM   2899  HB3 ASP A 397      23.442  14.400   9.521  1.0   0.00      PROT H \n",
            "ATOM   2900  HB2 ASP A 397      23.824  16.018  10.157  1.0   0.00      PROT H \n",
            "ATOM   2901  HD1 ASP A 397      20.332  16.015   8.276  1.0   0.00      PROT H \n",
            "ATOM   2902  N   SER A 398      24.230  14.699  12.796  1.0   0.00      PROT N \n",
            "ATOM   2903  CA  SER A 398      25.182  13.989  13.632  1.0   0.00      PROT C \n",
            "ATOM   2904  C   SER A 398      24.450  13.190  14.710  1.0   0.00      PROT C \n",
            "ATOM   2905  O   SER A 398      24.834  12.062  15.013  1.0   0.00      PROT O \n",
            "ATOM   2906  CB  SER A 398      26.150  14.978  14.284  1.0   0.00      PROT C \n",
            "ATOM   2907  OG  SER A 398      26.755  15.816  13.315  1.0   0.00      PROT O \n",
            "ATOM   2908  H   SER A 398      24.289  15.707  12.757  1.0   0.00      PROT H \n",
            "ATOM   2909  HA  SER A 398      25.718  13.316  12.963  1.0   0.00      PROT H \n",
            "ATOM   2910  HB2 SER A 398      25.608  15.602  14.994  1.0   0.00      PROT H \n",
            "ATOM   2911  HB3 SER A 398      26.933  14.429  14.807  1.0   0.00      PROT H \n",
            "ATOM   2912  HG  SER A 398      27.406  16.507  13.595  1.0   0.00      PROT H \n",
            "ATOM   2913  N   PHE A 399      23.394  13.771  15.282  1.0   0.00      PROT N \n",
            "ATOM   2914  CA  PHE A 399      22.618  13.094  16.325  1.0   0.00      PROT C \n",
            "ATOM   2915  C   PHE A 399      21.784  11.960  15.744  1.0   0.00      PROT C \n",
            "ATOM   2916  O   PHE A 399      21.784  10.843  16.270  1.0   0.00      PROT O \n",
            "ATOM   2917  CB  PHE A 399      21.689  14.081  17.050  1.0   0.00      PROT C \n",
            "ATOM   2918  CG  PHE A 399      22.400  15.010  18.004  1.0   0.00      PROT C \n",
            "ATOM   2919  CD1 PHE A 399      22.338  16.390  17.823  1.0   0.00      PROT C \n",
            "ATOM   2920  CD2 PHE A 399      23.123  14.506  19.088  1.0   0.00      PROT C \n",
            "ATOM   2921  CE1 PHE A 399      22.985  17.259  18.706  1.0   0.00      PROT C \n",
            "ATOM   2922  CE2 PHE A 399      23.777  15.366  19.983  1.0   0.00      PROT C \n",
            "ATOM   2923  CZ  PHE A 399      23.709  16.745  19.791  1.0   0.00      PROT C \n",
            "ATOM   2924  H   PHE A 399      23.113  14.699  14.998  1.0   0.00      PROT H \n",
            "ATOM   2925  HA  PHE A 399      23.367  12.692  17.007  1.0   0.00      PROT H \n",
            "ATOM   2926  HB3 PHE A 399      20.957  13.488  17.598  1.0   0.00      PROT H \n",
            "ATOM   2927  HB2 PHE A 399      21.182  14.666  16.283  1.0   0.00      PROT H \n",
            "ATOM   2928  HD1 PHE A 399      21.779  16.805  16.984  1.0   0.00      PROT H \n",
            "ATOM   2929  HD2 PHE A 399      23.185  13.430  19.248  1.0   0.00      PROT H \n",
            "ATOM   2930  HE1 PHE A 399      22.927  18.336  18.551  1.0   0.00      PROT H \n",
            "ATOM   2931  HE2 PHE A 399      24.337  14.960  20.826  1.0   0.00      PROT H \n",
            "ATOM   2932  HZ  PHE A 399      24.217  17.417  20.482  1.0   0.00      PROT H \n",
            "ATOM   2933  N   LEU A 400      21.064  12.255  14.665  1.0   0.00      PROT N \n",
            "ATOM   2934  CA  LEU A 400      20.222  11.263  14.008  1.0   0.00      PROT C \n",
            "ATOM   2935  C   LEU A 400      21.051  10.032  13.664  1.0   0.00      PROT C \n",
            "ATOM   2936  O   LEU A 400      20.623   8.896  13.870  1.0   0.00      PROT O \n",
            "ATOM   2937  CB  LEU A 400      19.622  11.857  12.734  1.0   0.00      PROT C \n",
            "ATOM   2938  CG  LEU A 400      18.437  12.806  12.928  1.0   0.00      PROT C \n",
            "ATOM   2939  CD1 LEU A 400      18.213  13.654  11.689  1.0   0.00      PROT C \n",
            "ATOM   2940  CD2 LEU A 400      17.203  11.991  13.246  1.0   0.00      PROT C \n",
            "ATOM   2941  H   LEU A 400      21.092  13.188  14.279  1.0   0.00      PROT H \n",
            "ATOM   2942  HA  LEU A 400      19.434  11.002  14.715  1.0   0.00      PROT H \n",
            "ATOM   2943  HB2 LEU A 400      20.384  12.430  12.206  1.0   0.00      PROT H \n",
            "ATOM   2944  HB3 LEU A 400      19.263  11.053  12.092  1.0   0.00      PROT H \n",
            "ATOM   2945  HG  LEU A 400      18.640  13.486  13.756  1.0   0.00      PROT H \n",
            "ATOM   2946 HD11 LEU A 400      19.112  14.233  11.477  1.0   0.00      PROT H \n",
            "ATOM   2947 HD12 LEU A 400      17.989  13.007  10.841  1.0   0.00      PROT H \n",
            "ATOM   2948 HD13 LEU A 400      17.376  14.332  11.859  1.0   0.00      PROT H \n",
            "ATOM   2949 HD21 LEU A 400      17.384  11.390  14.137  1.0   0.00      PROT H \n",
            "ATOM   2950 HD22 LEU A 400      16.361  12.660  13.424  1.0   0.00      PROT H \n",
            "ATOM   2951 HD23 LEU A 400      16.974  11.335  12.406  1.0   0.00      PROT H \n",
            "ATOM   2952  N   LEU A 401      22.255  10.278  13.164  1.0   0.00      PROT N \n",
            "ATOM   2953  CA  LEU A 401      23.157   9.213  12.768  1.0   0.00      PROT C \n",
            "ATOM   2954  C   LEU A 401      23.584   8.363  13.953  1.0   0.00      PROT C \n",
            "ATOM   2955  O   LEU A 401      23.552   7.136  13.895  1.0   0.00      PROT O \n",
            "ATOM   2956  CB  LEU A 401      24.380   9.810  12.071  1.0   0.00      PROT C \n",
            "ATOM   2957  CG  LEU A 401      25.079   8.861  11.097  1.0   0.00      PROT C \n",
            "ATOM   2958  CD1 LEU A 401      24.120   8.454   9.997  1.0   0.00      PROT C \n",
            "ATOM   2959  CD2 LEU A 401      26.300   9.539  10.520  1.0   0.00      PROT C \n",
            "ATOM   2960  H   LEU A 401      22.569  11.231  13.049  1.0   0.00      PROT H \n",
            "ATOM   2961  HA  LEU A 401      22.581   8.587  12.087  1.0   0.00      PROT H \n",
            "ATOM   2962  HB3 LEU A 401      25.078  10.106  12.854  1.0   0.00      PROT H \n",
            "ATOM   2963  HB2 LEU A 401      24.039  10.695  11.534  1.0   0.00      PROT H \n",
            "ATOM   2964  HG  LEU A 401      25.402   7.959  11.617  1.0   0.00      PROT H \n",
            "ATOM   2965 HD11 LEU A 401      23.249   7.968  10.436  1.0   0.00      PROT H \n",
            "ATOM   2966 HD12 LEU A 401      23.803   9.339   9.446  1.0   0.00      PROT H \n",
            "ATOM   2967 HD13 LEU A 401      24.618   7.762   9.318  1.0   0.00      PROT H \n",
            "ATOM   2968 HD21 LEU A 401      26.974   9.822  11.328  1.0   0.00      PROT H \n",
            "ATOM   2969 HD22 LEU A 401      26.811   8.853   9.844  1.0   0.00      PROT H \n",
            "ATOM   2970 HD23 LEU A 401      25.996  10.430   9.972  1.0   0.00      PROT H \n",
            "ATOM   2971  N   ALA A 402      23.988   9.019  15.030  1.0   0.00      PROT N \n",
            "ATOM   2972  CA  ALA A 402      24.398   8.315  16.238  1.0   0.00      PROT C \n",
            "ATOM   2973  C   ALA A 402      23.193   7.583  16.832  1.0   0.00      PROT C \n",
            "ATOM   2974  O   ALA A 402      23.327   6.496  17.399  1.0   0.00      PROT O \n",
            "ATOM   2975  CB  ALA A 402      24.956   9.306  17.248  1.0   0.00      PROT C \n",
            "ATOM   2976  H   ALA A 402      24.017  10.029  15.025  1.0   0.00      PROT H \n",
            "ATOM   2977  HA  ALA A 402      25.175   7.608  15.947  1.0   0.00      PROT H \n",
            "ATOM   2978  HB1 ALA A 402      25.813   9.822  16.815  1.0   0.00      PROT H \n",
            "ATOM   2979  HB2 ALA A 402      24.187  10.034  17.506  1.0   0.00      PROT H \n",
            "ATOM   2980  HB3 ALA A 402      25.268   8.773  18.146  1.0   0.00      PROT H \n",
            "ATOM   2981  N   PHE A 403      22.014   8.184  16.682  1.0   0.00      PROT N \n",
            "ATOM   2982  CA  PHE A 403      20.789   7.610  17.221  1.0   0.00      PROT C \n",
            "ATOM   2983  C   PHE A 403      20.488   6.285  16.537  1.0   0.00      PROT C \n",
            "ATOM   2984  O   PHE A 403      20.299   5.265  17.198  1.0   0.00      PROT O \n",
            "ATOM   2985  CB  PHE A 403      19.625   8.585  17.020  1.0   0.00      PROT C \n",
            "ATOM   2986  CG  PHE A 403      18.458   8.363  17.957  1.0   0.00      PROT C \n",
            "ATOM   2987  CD1 PHE A 403      18.402   7.254  18.792  1.0   0.00      PROT C \n",
            "ATOM   2988  CD2 PHE A 403      17.402   9.268  17.985  1.0   0.00      PROT C \n",
            "ATOM   2989  CE1 PHE A 403      17.304   7.051  19.640  1.0   0.00      PROT C \n",
            "ATOM   2990  CE2 PHE A 403      16.307   9.073  18.827  1.0   0.00      PROT C \n",
            "ATOM   2991  CZ  PHE A 403      16.257   7.962  19.655  1.0   0.00      PROT C \n",
            "ATOM   2992  H   PHE A 403      21.954   9.061  16.184  1.0   0.00      PROT H \n",
            "ATOM   2993  HA  PHE A 403      20.968   7.443  18.283  1.0   0.00      PROT H \n",
            "ATOM   2994  HB3 PHE A 403      19.290   8.473  15.989  1.0   0.00      PROT H \n",
            "ATOM   2995  HB2 PHE A 403      20.022   9.590  17.166  1.0   0.00      PROT H \n",
            "ATOM   2996  HD1 PHE A 403      19.217   6.530  18.792  1.0   0.00      PROT H \n",
            "ATOM   2997  HD2 PHE A 403      17.421  10.148  17.342  1.0   0.00      PROT H \n",
            "ATOM   2998  HE1 PHE A 403      17.274   6.175  20.288  1.0   0.00      PROT H \n",
            "ATOM   2999  HE2 PHE A 403      15.490   9.795  18.834  1.0   0.00      PROT H \n",
            "ATOM   3000  HZ  PHE A 403      15.400   7.809  20.311  1.0   0.00      PROT H \n",
            "ATOM   3001  N   GLU A 404      20.456   6.311  15.209  1.0   0.00      PROT N \n",
            "ATOM   3002  CA  GLU A 404      20.177   5.117  14.421  1.0   0.00      PROT C \n",
            "ATOM   3003  C   GLU A 404      21.198   3.998  14.693  1.0   0.00      PROT C \n",
            "ATOM   3004  O   GLU A 404      20.827   2.830  14.853  1.0   0.00      PROT O \n",
            "ATOM   3005  CB  GLU A 404      20.170   5.479  12.936  1.0   0.00      PROT C \n",
            "ATOM   3006  CG  GLU A 404      19.729   4.352  12.033  1.0   0.00      PROT C \n",
            "ATOM   3007  CD  GLU A 404      19.781   4.729  10.565  1.0   0.00      PROT C \n",
            "ATOM   3008  OE1 GLU A 404      20.230   5.856  10.258  1.0   0.00      PROT O \n",
            "ATOM   3009  OE2 GLU A 404      19.372   3.898   9.722  1.0   0.00      PROT O \n",
            "ATOM   3010  H   GLU A 404      20.628   7.176  14.717  1.0   0.00      PROT H \n",
            "ATOM   3011  HA  GLU A 404      19.191   4.784  14.745  1.0   0.00      PROT H \n",
            "ATOM   3012  HB3 GLU A 404      21.184   5.780  12.673  1.0   0.00      PROT H \n",
            "ATOM   3013  HB2 GLU A 404      19.496   6.327  12.814  1.0   0.00      PROT H \n",
            "ATOM   3014  HG2 GLU A 404      20.381   3.491  12.181  1.0   0.00      PROT H \n",
            "ATOM   3015  HG3 GLU A 404      18.702   4.076  12.272  1.0   0.00      PROT H \n",
            "ATOM   3016  HE2 GLU A 404      19.462   4.274   8.800  1.0   0.00      PROT H \n",
            "ATOM   3017  N   HIS A 405      22.479   4.357  14.745  1.0   0.00      PROT N \n",
            "ATOM   3018  CA  HIS A 405      23.533   3.383  15.017  1.0   0.00      PROT C \n",
            "ATOM   3019  C   HIS A 405      23.348   2.776  16.410  1.0   0.00      PROT C \n",
            "ATOM   3020  O   HIS A 405      23.540   1.577  16.604  1.0   0.00      PROT O \n",
            "ATOM   3021  CB  HIS A 405      24.907   4.047  14.898  1.0   0.00      PROT C \n",
            "ATOM   3022  CG  HIS A 405      25.265   4.424  13.495  1.0   0.00      PROT C \n",
            "ATOM   3023  ND1 HIS A 405      26.366   5.195  13.187  1.0   0.00      PROT N \n",
            "ATOM   3024  CD2 HIS A 405      24.662   4.145  12.318  1.0   0.00      PROT C \n",
            "ATOM   3025  CE1 HIS A 405      26.420   5.377  11.881  1.0   0.00      PROT C \n",
            "ATOM   3026  NE2 HIS A 405      25.398   4.751  11.328  1.0   0.00      PROT N \n",
            "ATOM   3027  H   HIS A 405      22.740   5.321  14.595  1.0   0.00      PROT H \n",
            "ATOM   3028  HA  HIS A 405      23.431   2.605  14.261  1.0   0.00      PROT H \n",
            "ATOM   3029  HB3 HIS A 405      25.645   3.344  15.285  1.0   0.00      PROT H \n",
            "ATOM   3030  HB2 HIS A 405      24.894   4.940  15.522  1.0   0.00      PROT H \n",
            "ATOM   3031  HD1 HIS A 405      27.078   5.591  13.911  1.0   0.00      PROT H \n",
            "ATOM   3032  HD2 HIS A 405      23.758   3.536  12.300  1.0   0.00      PROT H \n",
            "ATOM   3033  HE1 HIS A 405      27.226   5.968  11.446  1.0   0.00      PROT H \n",
            "ATOM   3034  HE2 HIS A 405      25.198   4.737  10.247  1.0   0.00      PROT H \n",
            "ATOM   3035  N   TYR A 406      22.968   3.607  17.375  1.0   0.00      PROT N \n",
            "ATOM   3036  CA  TYR A 406      22.735   3.124  18.718  1.0   0.00      PROT C \n",
            "ATOM   3037  C   TYR A 406      21.543   2.160  18.681  1.0   0.00      PROT C \n",
            "ATOM   3038  O   TYR A 406      21.489   1.193  19.438  1.0   0.00      PROT O \n",
            "ATOM   3039  CB  TYR A 406      22.423   4.293  19.651  1.0   0.00      PROT C \n",
            "ATOM   3040  CG  TYR A 406      22.297   3.887  21.103  1.0   0.00      PROT C \n",
            "ATOM   3041  CD1 TYR A 406      23.430   3.566  21.860  1.0   0.00      PROT C \n",
            "ATOM   3042  CD2 TYR A 406      21.043   3.766  21.707  1.0   0.00      PROT C \n",
            "ATOM   3043  CE1 TYR A 406      23.312   3.137  23.180  1.0   0.00      PROT C \n",
            "ATOM   3044  CE2 TYR A 406      20.917   3.335  23.018  1.0   0.00      PROT C \n",
            "ATOM   3045  CZ  TYR A 406      22.053   3.021  23.749  1.0   0.00      PROT C \n",
            "ATOM   3046  OH  TYR A 406      21.922   2.590  25.047  1.0   0.00      PROT O \n",
            "ATOM   3047  H   TYR A 406      22.839   4.588  17.174  1.0   0.00      PROT H \n",
            "ATOM   3048  HA  TYR A 406      23.651   2.628  19.040  1.0   0.00      PROT H \n",
            "ATOM   3049  HB3 TYR A 406      21.486   4.738  19.316  1.0   0.00      PROT H \n",
            "ATOM   3050  HB2 TYR A 406      23.229   5.020  19.545  1.0   0.00      PROT H \n",
            "ATOM   3051  HD1 TYR A 406      24.415   3.656  21.402  1.0   0.00      PROT H \n",
            "ATOM   3052  HD2 TYR A 406      20.154   4.018  21.128  1.0   0.00      PROT H \n",
            "ATOM   3053  HE1 TYR A 406      24.199   2.893  23.765  1.0   0.00      PROT H \n",
            "ATOM   3054  HE2 TYR A 406      19.929   3.244  23.469  1.0   0.00      PROT H \n",
            "ATOM   3055  HH  TYR A 406      22.727   2.363  25.577  1.0   0.00      PROT H \n",
            "ATOM   3056  N   ILE A 407      20.589   2.436  17.796  1.0   0.00      PROT N \n",
            "ATOM   3057  CA  ILE A 407      19.397   1.602  17.647  1.0   0.00      PROT C \n",
            "ATOM   3058  C   ILE A 407      19.793   0.215  17.166  1.0   0.00      PROT C \n",
            "ATOM   3059  O   ILE A 407      19.213  -0.785  17.595  1.0   0.00      PROT O \n",
            "ATOM   3060  CB  ILE A 407      18.383   2.225  16.634  1.0   0.00      PROT C \n",
            "ATOM   3061  CG1 ILE A 407      17.699   3.440  17.263  1.0   0.00      PROT C \n",
            "ATOM   3062  CG2 ILE A 407      17.344   1.197  16.210  1.0   0.00      PROT C \n",
            "ATOM   3063  CD1 ILE A 407      16.994   3.133  18.576  1.0   0.00      PROT C \n",
            "ATOM   3064  H   ILE A 407      20.674   3.245  17.198  1.0   0.00      PROT H \n",
            "ATOM   3065  HA  ILE A 407      18.970   1.550  18.648  1.0   0.00      PROT H \n",
            "ATOM   3066  HB  ILE A 407      18.907   2.547  15.734  1.0   0.00      PROT H \n",
            "ATOM   3067 HG13 ILE A 407      16.974   3.815  16.540  1.0   0.00      PROT H \n",
            "ATOM   3068 HG12 ILE A 407      18.468   4.194  17.431  1.0   0.00      PROT H \n",
            "ATOM   3069 HG21 ILE A 407      17.844   0.337  15.764  1.0   0.00      PROT H \n",
            "ATOM   3070 HG22 ILE A 407      16.775   0.874  17.082  1.0   0.00      PROT H \n",
            "ATOM   3071 HG23 ILE A 407      16.668   1.643  15.480  1.0   0.00      PROT H \n",
            "ATOM   3072 HD11 ILE A 407      16.533   4.042  18.962  1.0   0.00      PROT H \n",
            "ATOM   3073 HD12 ILE A 407      16.225   2.379  18.408  1.0   0.00      PROT H \n",
            "ATOM   3074 HD13 ILE A 407      17.719   2.758  19.299  1.0   0.00      PROT H \n",
            "ATOM   3075  N   ASN A 408      20.778   0.159  16.272  1.0   0.00      PROT N \n",
            "ATOM   3076  CA  ASN A 408      21.251  -1.119  15.754  1.0   0.00      PROT C \n",
            "ATOM   3077  C   ASN A 408      21.973  -1.883  16.863  1.0   0.00      PROT C \n",
            "ATOM   3078  O   ASN A 408      21.855  -3.102  16.954  1.0   0.00      PROT O \n",
            "ATOM   3079  CB  ASN A 408      22.188  -0.905  14.560  1.0   0.00      PROT C \n",
            "ATOM   3080  CG  ASN A 408      21.527  -0.138  13.429  1.0   0.00      PROT C \n",
            "ATOM   3081  OD1 ASN A 408      20.317  -0.219  13.233  1.0   0.00      PROT O \n",
            "ATOM   3082  ND2 ASN A 408      22.324   0.596  12.667  1.0   0.00      PROT N \n",
            "ATOM   3083  H   ASN A 408      21.210   1.010  15.942  1.0   0.00      PROT H \n",
            "ATOM   3084  HA  ASN A 408      20.364  -1.663  15.430  1.0   0.00      PROT H \n",
            "ATOM   3085  HB3 ASN A 408      22.501  -1.885  14.200  1.0   0.00      PROT H \n",
            "ATOM   3086  HB2 ASN A 408      23.059  -0.353  14.913  1.0   0.00      PROT H \n",
            "ATOM   3087 HD21 ASN A 408      23.324   0.636  12.865  1.0   0.00      PROT H \n",
            "ATOM   3088 HD22 ASN A 408      21.938   1.121  11.883  1.0   0.00      PROT H \n",
            "ATOM   3089  N   TYR A 409      22.709  -1.156  17.703  1.0   0.00      PROT N \n",
            "ATOM   3090  CA  TYR A 409      23.444  -1.739  18.830  1.0   0.00      PROT C \n",
            "ATOM   3091  C   TYR A 409      22.494  -2.349  19.860  1.0   0.00      PROT C \n",
            "ATOM   3092  O   TYR A 409      22.768  -3.410  20.413  1.0   0.00      PROT O \n",
            "ATOM   3093  CB  TYR A 409      24.305  -0.662  19.503  1.0   0.00      PROT C \n",
            "ATOM   3094  CG  TYR A 409      24.729  -0.982  20.925  1.0   0.00      PROT C \n",
            "ATOM   3095  CD1 TYR A 409      25.736  -1.915  21.192  1.0   0.00      PROT C \n",
            "ATOM   3096  CD2 TYR A 409      24.120  -0.340  22.009  1.0   0.00      PROT C \n",
            "ATOM   3097  CE1 TYR A 409      26.127  -2.199  22.509  1.0   0.00      PROT C \n",
            "ATOM   3098  CE2 TYR A 409      24.500  -0.614  23.321  1.0   0.00      PROT C \n",
            "ATOM   3099  CZ  TYR A 409      25.502  -1.541  23.570  1.0   0.00      PROT C \n",
            "ATOM   3100  OH  TYR A 409      25.875  -1.791  24.880  1.0   0.00      PROT O \n",
            "ATOM   3101  H   TYR A 409      22.779  -0.156  17.578  1.0   0.00      PROT H \n",
            "ATOM   3102  HA  TYR A 409      24.062  -2.526  18.398  1.0   0.00      PROT H \n",
            "ATOM   3103  HB3 TYR A 409      23.719   0.257  19.497  1.0   0.00      PROT H \n",
            "ATOM   3104  HB2 TYR A 409      25.190  -0.533  18.880  1.0   0.00      PROT H \n",
            "ATOM   3105  HD1 TYR A 409      26.230  -2.434  20.371  1.0   0.00      PROT H \n",
            "ATOM   3106  HD2 TYR A 409      23.333   0.388  21.810  1.0   0.00      PROT H \n",
            "ATOM   3107  HE1 TYR A 409      26.913  -2.928  22.707  1.0   0.00      PROT H \n",
            "ATOM   3108  HE2 TYR A 409      24.012  -0.102  24.151  1.0   0.00      PROT H \n",
            "ATOM   3109  HH  TYR A 409      26.588  -2.449  25.075  1.0   0.00      PROT H \n",
            "ATOM   3110  N   ARG A 410      21.382  -1.665  20.110  1.0   0.00      PROT N \n",
            "ATOM   3111  CA  ARG A 410      20.372  -2.114  21.069  1.0   0.00      PROT C \n",
            "ATOM   3112  C   ARG A 410      19.644  -3.371  20.608  1.0   0.00      PROT C \n",
            "ATOM   3113  O   ARG A 410      19.272  -4.210  21.422  1.0   0.00      PROT O \n",
            "ATOM   3114  CB  ARG A 410      19.359  -0.991  21.319  1.0   0.00      PROT C \n",
            "ATOM   3115  CG  ARG A 410      19.922   0.156  22.152  1.0   0.00      PROT C \n",
            "ATOM   3116  CD  ARG A 410      19.618  -0.032  23.634  1.0   0.00      PROT C \n",
            "ATOM   3117  NE  ARG A 410      18.237   0.345  23.932  1.0   0.00      PROT N \n",
            "ATOM   3118  CZ  ARG A 410      17.591   0.047  25.055  1.0   0.00      PROT C \n",
            "ATOM   3119  NH2 ARG A 410      16.339   0.445  25.221  1.0   0.00      PROT N \n",
            "ATOM   3120  NH1 ARG A 410      18.181  -0.661  26.004  1.0   0.00      PROT N \n",
            "ATOM   3121  H   ARG A 410      21.205  -0.795  19.629  1.0   0.00      PROT H \n",
            "ATOM   3122  HA  ARG A 410      20.935  -2.357  21.970  1.0   0.00      PROT H \n",
            "ATOM   3123  HB3 ARG A 410      18.504  -1.429  21.835  1.0   0.00      PROT H \n",
            "ATOM   3124  HB2 ARG A 410      19.042  -0.613  20.347  1.0   0.00      PROT H \n",
            "ATOM   3125  HG3 ARG A 410      19.475   1.088  21.805  1.0   0.00      PROT H \n",
            "ATOM   3126  HG2 ARG A 410      21.001   0.194  22.004  1.0   0.00      PROT H \n",
            "ATOM   3127  HD2 ARG A 410      19.763  -1.078  23.905  1.0   0.00      PROT H \n",
            "ATOM   3128  HD3 ARG A 410      20.288   0.593  24.224  1.0   0.00      PROT H \n",
            "ATOM   3129  HE  ARG A 410      17.772   0.871  23.206  1.0   0.00      PROT H \n",
            "ATOM   3130 HH22 ARG A 410      15.837   0.205  26.076  1.0   0.00      PROT H \n",
            "ATOM   3131 HH21 ARG A 410      15.877   0.991  24.494  1.0   0.00      PROT H \n",
            "ATOM   3132 HH12 ARG A 410      17.672  -0.896  26.856  1.0   0.00      PROT H \n",
            "ATOM   3133 HH11 ARG A 410      19.188  -1.094  26.093  1.0   0.00      PROT H \n",
            "ATOM   3134  N   LYS A 411      19.447  -3.488  19.298  1.0   0.00      PROT N \n",
            "ATOM   3135  CA  LYS A 411      18.760  -4.629  18.703  1.0   0.00      PROT C \n",
            "ATOM   3136  C   LYS A 411      17.368  -4.892  19.284  1.0   0.00      PROT C \n",
            "ATOM   3137  O   LYS A 411      17.106  -5.961  19.835  1.0   0.00      PROT O \n",
            "ATOM   3138  CB  LYS A 411      19.616  -5.894  18.826  1.0   0.00      PROT C \n",
            "ATOM   3139  CG  LYS A 411      20.901  -5.856  18.012  1.0   0.00      PROT C \n",
            "ATOM   3140  CD  LYS A 411      21.612  -7.197  18.032  1.0   0.00      PROT C \n",
            "ATOM   3141  CE  LYS A 411      22.893  -7.159  17.213  1.0   0.00      PROT C \n",
            "ATOM   3142  NZ  LYS A 411      23.885  -6.198  17.770  1.0   0.00      PROT N \n",
            "ATOM   3143  H   LYS A 411      19.779  -2.769  18.671  1.0   0.00      PROT H \n",
            "ATOM   3144  HA  LYS A 411      18.639  -4.320  17.665  1.0   0.00      PROT H \n",
            "ATOM   3145  HB3 LYS A 411      19.001  -6.731  18.495  1.0   0.00      PROT H \n",
            "ATOM   3146  HB2 LYS A 411      19.858  -6.016  19.882  1.0   0.00      PROT H \n",
            "ATOM   3147  HG3 LYS A 411      21.554  -5.092  18.434  1.0   0.00      PROT H \n",
            "ATOM   3148  HG2 LYS A 411      20.651  -5.593  16.984  1.0   0.00      PROT H \n",
            "ATOM   3149  HD2 LYS A 411      21.866  -7.458  19.059  1.0   0.00      PROT H \n",
            "ATOM   3150  HD3 LYS A 411      20.957  -7.962  17.614  1.0   0.00      PROT H \n",
            "ATOM   3151  HE2 LYS A 411      23.348  -8.150  17.204  1.0   0.00      PROT H \n",
            "ATOM   3152  HE3 LYS A 411      22.662  -6.856  16.192  1.0   0.00      PROT H \n",
            "ATOM   3153  HZ1 LYS A 411      24.731  -6.692  18.016  1.0   0.00      PROT H \n",
            "ATOM   3154  HZ2 LYS A 411      24.096  -5.493  17.078  1.0   0.00      PROT H \n",
            "ATOM   3155  HZ3 LYS A 411      23.828  -5.540  18.650  1.0   0.00      PROT H \n",
            "ATOM   3156  N   HIS A 412      16.476  -3.916  19.149  1.0   0.00      PROT N \n",
            "ATOM   3157  CA  HIS A 412      15.104  -4.052  19.641  1.0   0.00      PROT C \n",
            "ATOM   3158  C   HIS A 412      14.364  -5.147  18.865  1.0   0.00      PROT C \n",
            "ATOM   3159  O   HIS A 412      14.631  -5.365  17.681  1.0   0.00      PROT O \n",
            "ATOM   3160  CB  HIS A 412      14.366  -2.720  19.482  1.0   0.00      PROT C \n",
            "ATOM   3161  CG  HIS A 412      14.797  -1.669  20.458  1.0   0.00      PROT C \n",
            "ATOM   3162  ND1 HIS A 412      14.311  -1.608  21.744  1.0   0.00      PROT N \n",
            "ATOM   3163  CD2 HIS A 412      15.672  -0.645  20.333  1.0   0.00      PROT C \n",
            "ATOM   3164  CE1 HIS A 412      14.866  -0.584  22.374  1.0   0.00      PROT C \n",
            "ATOM   3165  NE2 HIS A 412      15.693   0.015  21.541  1.0   0.00      PROT N \n",
            "ATOM   3166  H   HIS A 412      16.738  -3.052  18.697  1.0   0.00      PROT H \n",
            "ATOM   3167  HA  HIS A 412      15.182  -4.327  20.693  1.0   0.00      PROT H \n",
            "ATOM   3168  HB3 HIS A 412      13.303  -2.922  19.612  1.0   0.00      PROT H \n",
            "ATOM   3169  HB2 HIS A 412      14.548  -2.369  18.466  1.0   0.00      PROT H \n",
            "ATOM   3170  HD1 HIS A 412      13.583  -2.296  22.174  1.0   0.00      PROT H \n",
            "ATOM   3171  HD2 HIS A 412      16.196  -0.488  19.390  1.0   0.00      PROT H \n",
            "ATOM   3172  HE1 HIS A 412      14.601  -0.373  23.410  1.0   0.00      PROT H \n",
            "ATOM   3173  HE2 HIS A 412      16.290   0.901  21.805  1.0   0.00      PROT H \n",
            "ATOM   3174  N   HIS A 413      13.438  -5.836  19.524  1.0   0.00      PROT N \n",
            "ATOM   3175  CA  HIS A 413      12.685  -6.891  18.853  1.0   0.00      PROT C \n",
            "ATOM   3176  C   HIS A 413      11.398  -6.358  18.210  1.0   0.00      PROT C \n",
            "ATOM   3177  O   HIS A 413      10.291  -6.818  18.507  1.0   0.00      PROT O \n",
            "ATOM   3178  CB  HIS A 413      12.338  -8.018  19.833  1.0   0.00      PROT C \n",
            "ATOM   3179  CG  HIS A 413      11.773  -9.235  19.169  1.0   0.00      PROT C \n",
            "ATOM   3180  ND1 HIS A 413      12.558 -10.160  18.511  1.0   0.00      PROT N \n",
            "ATOM   3181  CD2 HIS A 413      10.494  -9.651  19.006  1.0   0.00      PROT C \n",
            "ATOM   3182  CE1 HIS A 413      11.790 -11.087  17.973  1.0   0.00      PROT C \n",
            "ATOM   3183  NE2 HIS A 413      10.530 -10.801  18.258  1.0   0.00      PROT N \n",
            "ATOM   3184  H   HIS A 413      13.252  -5.631  20.495  1.0   0.00      PROT H \n",
            "ATOM   3185  HA  HIS A 413      13.363  -7.246  18.077  1.0   0.00      PROT H \n",
            "ATOM   3186  HB3 HIS A 413      11.612  -7.621  20.542  1.0   0.00      PROT H \n",
            "ATOM   3187  HB2 HIS A 413      13.254  -8.284  20.361  1.0   0.00      PROT H \n",
            "ATOM   3188  HD1 HIS A 413      13.646 -10.125  18.449  1.0   0.00      PROT H \n",
            "ATOM   3189  HD2 HIS A 413       9.675  -9.080  19.444  1.0   0.00      PROT H \n",
            "ATOM   3190  HE1 HIS A 413      12.237 -11.905  17.409  1.0   0.00      PROT H \n",
            "ATOM   3191  N   VAL A 414      11.544  -5.380  17.326  1.0   0.00      PROT N \n",
            "ATOM   3192  CA  VAL A 414      10.391  -4.807  16.645  1.0   0.00      PROT C \n",
            "ATOM   3193  C   VAL A 414      10.839  -4.306  15.282  1.0   0.00      PROT C \n",
            "ATOM   3194  O   VAL A 414      11.790  -3.530  15.178  1.0   0.00      PROT O \n",
            "ATOM   3195  CB  VAL A 414       9.754  -3.655  17.482  1.0   0.00      PROT C \n",
            "ATOM   3196  CG1 VAL A 414      10.820  -2.703  17.946  1.0   0.00      PROT C \n",
            "ATOM   3197  CG2 VAL A 414       8.712  -2.918  16.662  1.0   0.00      PROT C \n",
            "ATOM   3198  H   VAL A 414      12.465  -5.022  17.117  1.0   0.00      PROT H \n",
            "ATOM   3199  HA  VAL A 414       9.669  -5.616  16.529  1.0   0.00      PROT H \n",
            "ATOM   3200  HB  VAL A 414       9.262  -4.054  18.369  1.0   0.00      PROT H \n",
            "ATOM   3201 HG11 VAL A 414      11.561  -3.245  18.533  1.0   0.00      PROT H \n",
            "ATOM   3202 HG12 VAL A 414      11.303  -2.249  17.081  1.0   0.00      PROT H \n",
            "ATOM   3203 HG13 VAL A 414      10.369  -1.924  18.561  1.0   0.00      PROT H \n",
            "ATOM   3204 HG21 VAL A 414       7.946  -3.620  16.332  1.0   0.00      PROT H \n",
            "ATOM   3205 HG22 VAL A 414       8.253  -2.140  17.272  1.0   0.00      PROT H \n",
            "ATOM   3206 HG23 VAL A 414       9.188  -2.465  15.792  1.0   0.00      PROT H \n",
            "ATOM   3207  N   THR A 415      10.169  -4.774  14.235  1.0   0.00      PROT N \n",
            "ATOM   3208  CA  THR A 415      10.530  -4.372  12.884  1.0   0.00      PROT C \n",
            "ATOM   3209  C   THR A 415      10.254  -2.896  12.635  1.0   0.00      PROT C \n",
            "ATOM   3210  O   THR A 415       9.223  -2.358  13.051  1.0   0.00      PROT O \n",
            "ATOM   3211  CB  THR A 415       9.810  -5.236  11.816  1.0   0.00      PROT C \n",
            "ATOM   3212  OG1 THR A 415       8.412  -5.331  12.121  1.0   0.00      PROT O \n",
            "ATOM   3213  CG2 THR A 415      10.427  -6.632  11.770  1.0   0.00      PROT C \n",
            "ATOM   3214  H   THR A 415       9.401  -5.415  14.373  1.0   0.00      PROT H \n",
            "ATOM   3215  HA  THR A 415      11.607  -4.539  12.857  1.0   0.00      PROT H \n",
            "ATOM   3216  HB  THR A 415       9.855  -4.771  10.831  1.0   0.00      PROT H \n",
            "ATOM   3217  HG1 THR A 415       7.863  -4.510  12.188  1.0   0.00      PROT H \n",
            "ATOM   3218 HG21 THR A 415       9.915  -7.231  11.017  1.0   0.00      PROT H \n",
            "ATOM   3219 HG22 THR A 415      11.484  -6.554  11.515  1.0   0.00      PROT H \n",
            "ATOM   3220 HG23 THR A 415      10.323  -7.108  12.745  1.0   0.00      PROT H \n",
            "ATOM   3221  N   HIS A 416      11.207  -2.252  11.964  1.0   0.00      PROT N \n",
            "ATOM   3222  CA  HIS A 416      11.127  -0.830  11.650  1.0   0.00      PROT C \n",
            "ATOM   3223  C   HIS A 416      10.989   0.005  12.930  1.0   0.00      PROT C \n",
            "ATOM   3224  O   HIS A 416      10.095   0.850  13.046  1.0   0.00      PROT O \n",
            "ATOM   3225  CB  HIS A 416       9.947  -0.556  10.707  1.0   0.00      PROT C \n",
            "ATOM   3226  CG  HIS A 416       9.983  -1.361   9.443  1.0   0.00      PROT C \n",
            "ATOM   3227  ND1 HIS A 416      11.135  -1.543   8.707  1.0   0.00      PROT N \n",
            "ATOM   3228  CD2 HIS A 416       9.006  -2.023   8.780  1.0   0.00      PROT C \n",
            "ATOM   3229  CE1 HIS A 416      10.868  -2.284   7.648  1.0   0.00      PROT C \n",
            "ATOM   3230  NE2 HIS A 416       9.583  -2.588   7.668  1.0   0.00      PROT N \n",
            "ATOM   3231  H   HIS A 416      12.028  -2.750  11.650  1.0   0.00      PROT H \n",
            "ATOM   3232  HA  HIS A 416      12.068  -0.592  11.155  1.0   0.00      PROT H \n",
            "ATOM   3233  HB3 HIS A 416       9.968   0.506  10.462  1.0   0.00      PROT H \n",
            "ATOM   3234  HB2 HIS A 416       9.034  -0.784  11.257  1.0   0.00      PROT H \n",
            "ATOM   3235  HD1 HIS A 416      12.113  -1.136   8.964  1.0   0.00      PROT H \n",
            "ATOM   3236  HD2 HIS A 416       7.988  -2.026   9.170  1.0   0.00      PROT H \n",
            "ATOM   3237  HE1 HIS A 416      11.662  -2.536   6.945  1.0   0.00      PROT H \n",
            "ATOM   3238  N   PHE A 417      11.879  -0.243  13.888  1.0   0.00      PROT N \n",
            "ATOM   3239  CA  PHE A 417      11.858   0.487  15.145  1.0   0.00      PROT C \n",
            "ATOM   3240  C   PHE A 417      12.245   1.937  14.905  1.0   0.00      PROT C \n",
            "ATOM   3241  O   PHE A 417      11.606   2.845  15.424  1.0   0.00      PROT O \n",
            "ATOM   3242  CB  PHE A 417      12.831  -0.134  16.142  1.0   0.00      PROT C \n",
            "ATOM   3243  CG  PHE A 417      12.802   0.509  17.499  1.0   0.00      PROT C \n",
            "ATOM   3244  CD1 PHE A 417      11.682   0.395  18.312  1.0   0.00      PROT C \n",
            "ATOM   3245  CD2 PHE A 417      13.881   1.258  17.952  1.0   0.00      PROT C \n",
            "ATOM   3246  CE1 PHE A 417      11.642   1.003  19.559  1.0   0.00      PROT C \n",
            "ATOM   3247  CE2 PHE A 417      13.849   1.871  19.202  1.0   0.00      PROT C \n",
            "ATOM   3248  CZ  PHE A 417      12.727   1.749  20.001  1.0   0.00      PROT C \n",
            "ATOM   3249  H   PHE A 417      12.587  -0.950  13.749  1.0   0.00      PROT H \n",
            "ATOM   3250  HA  PHE A 417      10.835   0.414  15.515  1.0   0.00      PROT H \n",
            "ATOM   3251  HB3 PHE A 417      13.831  -0.042  15.718  1.0   0.00      PROT H \n",
            "ATOM   3252  HB2 PHE A 417      12.571  -1.188  16.234  1.0   0.00      PROT H \n",
            "ATOM   3253  HD1 PHE A 417      10.827  -0.181  17.959  1.0   0.00      PROT H \n",
            "ATOM   3254  HD2 PHE A 417      14.767   1.371  17.327  1.0   0.00      PROT H \n",
            "ATOM   3255  HE1 PHE A 417      10.760   0.895  20.190  1.0   0.00      PROT H \n",
            "ATOM   3256  HE2 PHE A 417      14.707   2.446  19.551  1.0   0.00      PROT H \n",
            "ATOM   3257  HZ  PHE A 417      12.696   2.238  20.975  1.0   0.00      PROT H \n",
            "ATOM   3258  N   TRP A 418      13.297   2.147  14.118  1.0   0.00      PROT N \n",
            "ATOM   3259  CA  TRP A 418      13.770   3.493  13.815  1.0   0.00      PROT C \n",
            "ATOM   3260  C   TRP A 418      12.662   4.329  13.168  1.0   0.00      PROT C \n",
            "ATOM   3261  O   TRP A 418      12.374   5.444  13.611  1.0   0.00      PROT O \n",
            "ATOM   3262  CB  TRP A 418      14.989   3.421  12.894  1.0   0.00      PROT C \n",
            "ATOM   3263  CG  TRP A 418      15.556   4.747  12.527  1.0   0.00      PROT C \n",
            "ATOM   3264  CD1 TRP A 418      15.795   5.213  11.268  1.0   0.00      PROT C \n",
            "ATOM   3265  CD2 TRP A 418      15.974   5.781  13.424  1.0   0.00      PROT C \n",
            "ATOM   3266  NE1 TRP A 418      16.337   6.472  11.320  1.0   0.00      PROT N \n",
            "ATOM   3267  CE2 TRP A 418      16.457   6.848  12.633  1.0   0.00      PROT C \n",
            "ATOM   3268  CE3 TRP A 418      15.990   5.910  14.819  1.0   0.00      PROT C \n",
            "ATOM   3269  CZ2 TRP A 418      16.951   8.034  13.193  1.0   0.00      PROT C \n",
            "ATOM   3270  CZ3 TRP A 418      16.482   7.090  15.375  1.0   0.00      PROT C \n",
            "ATOM   3271  CH2 TRP A 418      16.956   8.134  14.559  1.0   0.00      PROT C \n",
            "ATOM   3272  H   TRP A 418      13.790   1.363  13.714  1.0   0.00      PROT H \n",
            "ATOM   3273  HA  TRP A 418      14.044   3.934  14.773  1.0   0.00      PROT H \n",
            "ATOM   3274  HB3 TRP A 418      14.677   2.897  11.991  1.0   0.00      PROT H \n",
            "ATOM   3275  HB2 TRP A 418      15.746   2.833  13.412  1.0   0.00      PROT H \n",
            "ATOM   3276  HD1 TRP A 418      15.554   4.595  10.403  1.0   0.00      PROT H \n",
            "ATOM   3277  HE1 TRP A 418      16.632   7.100  10.586  1.0   0.00      PROT H \n",
            "ATOM   3278  HE3 TRP A 418      15.628   5.112  15.467  1.0   0.00      PROT H \n",
            "ATOM   3279  HZ2 TRP A 418      17.322   8.859  12.585  1.0   0.00      PROT H \n",
            "ATOM   3280  HZ3 TRP A 418      16.504   7.215  16.458  1.0   0.00      PROT H \n",
            "ATOM   3281  HH2 TRP A 418      17.333   9.038  15.038  1.0   0.00      PROT H \n",
            "ATOM   3282  N   PRO A 419      12.036   3.805  12.099  1.0   0.00      PROT N \n",
            "ATOM   3283  CA  PRO A 419      10.961   4.524  11.414  1.0   0.00      PROT C \n",
            "ATOM   3284  C   PRO A 419       9.776   4.797  12.334  1.0   0.00      PROT C \n",
            "ATOM   3285  O   PRO A 419       9.080   5.795  12.176  1.0   0.00      PROT O \n",
            "ATOM   3286  CB  PRO A 419      10.603   3.584  10.265  1.0   0.00      PROT C \n",
            "ATOM   3287  CG  PRO A 419      11.919   2.976   9.937  1.0   0.00      PROT C \n",
            "ATOM   3288  CD  PRO A 419      12.442   2.630  11.307  1.0   0.00      PROT C \n",
            "ATOM   3289  HA  PRO A 419      11.294   5.510  11.090  1.0   0.00      PROT H \n",
            "ATOM   3290  HB3 PRO A 419      10.217   4.108   9.391  1.0   0.00      PROT H \n",
            "ATOM   3291  HB2 PRO A 419       9.904   2.801  10.558  1.0   0.00      PROT H \n",
            "ATOM   3292  HG2 PRO A 419      11.768   2.104   9.301  1.0   0.00      PROT H \n",
            "ATOM   3293  HG3 PRO A 419      12.536   3.706   9.412  1.0   0.00      PROT H \n",
            "ATOM   3294  HD2 PRO A 419      11.979   1.705  11.650  1.0   0.00      PROT H \n",
            "ATOM   3295  HD3 PRO A 419      13.523   2.500  11.261  1.0   0.00      PROT H \n",
            "ATOM   3296  N   LYS A 420       9.545   3.905  13.290  1.0   0.00      PROT N \n",
            "ATOM   3297  CA  LYS A 420       8.443   4.082  14.226  1.0   0.00      PROT C \n",
            "ATOM   3298  C   LYS A 420       8.804   5.141  15.254  1.0   0.00      PROT C \n",
            "ATOM   3299  O   LYS A 420       7.941   5.865  15.729  1.0   0.00      PROT O \n",
            "ATOM   3300  CB  LYS A 420       8.104   2.761  14.911  1.0   0.00      PROT C \n",
            "ATOM   3301  CG  LYS A 420       7.469   1.760  13.966  1.0   0.00      PROT C \n",
            "ATOM   3302  CD  LYS A 420       7.153   0.458  14.660  1.0   0.00      PROT C \n",
            "ATOM   3303  CE  LYS A 420       6.776  -0.621  13.660  1.0   0.00      PROT C \n",
            "ATOM   3304  NZ  LYS A 420       6.725  -1.958  14.327  1.0   0.00      PROT N \n",
            "ATOM   3305  H   LYS A 420      10.137   3.091  13.375  1.0   0.00      PROT H \n",
            "ATOM   3306  HA  LYS A 420       7.589   4.415  13.636  1.0   0.00      PROT H \n",
            "ATOM   3307  HB3 LYS A 420       7.416   2.975  15.729  1.0   0.00      PROT H \n",
            "ATOM   3308  HB2 LYS A 420       9.028   2.346  15.313  1.0   0.00      PROT H \n",
            "ATOM   3309  HG3 LYS A 420       8.163   1.575  13.146  1.0   0.00      PROT H \n",
            "ATOM   3310  HG2 LYS A 420       6.550   2.195  13.573  1.0   0.00      PROT H \n",
            "ATOM   3311  HD2 LYS A 420       8.026   0.123  15.219  1.0   0.00      PROT H \n",
            "ATOM   3312  HD3 LYS A 420       6.317   0.605  15.344  1.0   0.00      PROT H \n",
            "ATOM   3313  HE2 LYS A 420       5.796  -0.398  13.237  1.0   0.00      PROT H \n",
            "ATOM   3314  HE3 LYS A 420       7.518  -0.652  12.862  1.0   0.00      PROT H \n",
            "ATOM   3315  HZ1 LYS A 420       5.796  -2.344  14.237  1.0   0.00      PROT H \n",
            "ATOM   3316  HZ2 LYS A 420       7.391  -2.578  13.889  1.0   0.00      PROT H \n",
            "ATOM   3317  N   LEU A 421      10.085   5.235  15.586  1.0   0.00      PROT N \n",
            "ATOM   3318  CA  LEU A 421      10.557   6.237  16.533  1.0   0.00      PROT C \n",
            "ATOM   3319  C   LEU A 421      10.409   7.627  15.918  1.0   0.00      PROT C \n",
            "ATOM   3320  O   LEU A 421       9.927   8.556  16.570  1.0   0.00      PROT O \n",
            "ATOM   3321  CB  LEU A 421      12.027   6.000  16.884  1.0   0.00      PROT C \n",
            "ATOM   3322  CG  LEU A 421      12.355   5.353  18.230  1.0   0.00      PROT C \n",
            "ATOM   3323  CD1 LEU A 421      13.861   5.203  18.352  1.0   0.00      PROT C \n",
            "ATOM   3324  CD2 LEU A 421      11.812   6.207  19.369  1.0   0.00      PROT C \n",
            "ATOM   3325  H   LEU A 421      10.760   4.603  15.180  1.0   0.00      PROT H \n",
            "ATOM   3326  HA  LEU A 421       9.934   6.126  17.421  1.0   0.00      PROT H \n",
            "ATOM   3327  HB3 LEU A 421      12.490   6.985  16.832  1.0   0.00      PROT H \n",
            "ATOM   3328  HB2 LEU A 421      12.411   5.375  16.077  1.0   0.00      PROT H \n",
            "ATOM   3329  HG  LEU A 421      11.892   4.368  18.292  1.0   0.00      PROT H \n",
            "ATOM   3330 HD11 LEU A 421      14.233   4.590  17.531  1.0   0.00      PROT H \n",
            "ATOM   3331 HD12 LEU A 421      14.329   6.187  18.312  1.0   0.00      PROT H \n",
            "ATOM   3332 HD13 LEU A 421      14.103   4.724  19.301  1.0   0.00      PROT H \n",
            "ATOM   3333 HD21 LEU A 421      10.731   6.305  19.267  1.0   0.00      PROT H \n",
            "ATOM   3334 HD22 LEU A 421      12.045   5.732  20.322  1.0   0.00      PROT H \n",
            "ATOM   3335 HD23 LEU A 421      12.271   7.195  19.333  1.0   0.00      PROT H \n",
            "ATOM   3336  N   LEU A 422      10.820   7.762  14.658  1.0   0.00      PROT N \n",
            "ATOM   3337  CA  LEU A 422      10.734   9.041  13.968  1.0   0.00      PROT C \n",
            "ATOM   3338  C   LEU A 422       9.294   9.566  13.923  1.0   0.00      PROT C \n",
            "ATOM   3339  O   LEU A 422       9.066  10.779  13.923  1.0   0.00      PROT O \n",
            "ATOM   3340  CB  LEU A 422      11.310   8.918  12.557  1.0   0.00      PROT C \n",
            "ATOM   3341  CG  LEU A 422      12.800   8.549  12.447  1.0   0.00      PROT C \n",
            "ATOM   3342  CD1 LEU A 422      13.249   8.541  10.978  1.0   0.00      PROT C \n",
            "ATOM   3343  CD2 LEU A 422      13.626   9.541  13.252  1.0   0.00      PROT C \n",
            "ATOM   3344  H   LEU A 422      11.200   6.968  14.163  1.0   0.00      PROT H \n",
            "ATOM   3345  HA  LEU A 422      11.340   9.724  14.564  1.0   0.00      PROT H \n",
            "ATOM   3346  HB2 LEU A 422      10.773   8.142  12.012  1.0   0.00      PROT H \n",
            "ATOM   3347  HB3 LEU A 422      11.202   9.869  12.036  1.0   0.00      PROT H \n",
            "ATOM   3348  HG  LEU A 422      12.959   7.551  12.856  1.0   0.00      PROT H \n",
            "ATOM   3349 HD11 LEU A 422      12.647   7.826  10.418  1.0   0.00      PROT H \n",
            "ATOM   3350 HD12 LEU A 422      13.119   9.536  10.553  1.0   0.00      PROT H \n",
            "ATOM   3351 HD13 LEU A 422      14.299   8.255  10.921  1.0   0.00      PROT H \n",
            "ATOM   3352 HD21 LEU A 422      13.294   9.533  14.290  1.0   0.00      PROT H \n",
            "ATOM   3353 HD22 LEU A 422      14.678   9.260  13.205  1.0   0.00      PROT H \n",
            "ATOM   3354 HD23 LEU A 422      13.498  10.541  12.838  1.0   0.00      PROT H \n",
            "ATOM   3355  N   MET A 423       8.320   8.662  13.903  1.0   0.00      PROT N \n",
            "ATOM   3356  CA  MET A 423       6.931   9.101  13.887  1.0   0.00      PROT C \n",
            "ATOM   3357  C   MET A 423       6.582   9.782  15.209  1.0   0.00      PROT C \n",
            "ATOM   3358  O   MET A 423       5.758  10.701  15.242  1.0   0.00      PROT O \n",
            "ATOM   3359  CB  MET A 423       5.983   7.924  13.648  1.0   0.00      PROT C \n",
            "ATOM   3360  CG  MET A 423       6.011   7.374  12.241  1.0   0.00      PROT C \n",
            "ATOM   3361  SD  MET A 423       4.647   6.227  11.944  1.0   0.00      PROT S \n",
            "ATOM   3362  CE  MET A 423       5.290   4.731  12.721  1.0   0.00      PROT C \n",
            "ATOM   3363  H   MET A 423       8.545   7.677  13.899  1.0   0.00      PROT H \n",
            "ATOM   3364  HA  MET A 423       6.862   9.806  13.058  1.0   0.00      PROT H \n",
            "ATOM   3365  HB3 MET A 423       4.977   8.274  13.881  1.0   0.00      PROT H \n",
            "ATOM   3366  HB2 MET A 423       6.267   7.140  14.350  1.0   0.00      PROT H \n",
            "ATOM   3367  HG2 MET A 423       5.930   8.195  11.528  1.0   0.00      PROT H \n",
            "ATOM   3368  HG3 MET A 423       6.948   6.842  12.077  1.0   0.00      PROT H \n",
            "ATOM   3369  HE1 MET A 423       4.560   3.927  12.621  1.0   0.00      PROT H \n",
            "ATOM   3370  HE2 MET A 423       6.220   4.439  12.233  1.0   0.00      PROT H \n",
            "ATOM   3371  HE3 MET A 423       5.478   4.922  13.777  1.0   0.00      PROT H \n",
            "ATOM   3372  N   LYS A 424       7.212   9.321  16.292  1.0   0.00      PROT N \n",
            "ATOM   3373  CA  LYS A 424       6.985   9.886  17.621  1.0   0.00      PROT C \n",
            "ATOM   3374  C   LYS A 424       7.597  11.274  17.690  1.0   0.00      PROT C \n",
            "ATOM   3375  O   LYS A 424       7.101  12.143  18.413  1.0   0.00      PROT O \n",
            "ATOM   3376  CB  LYS A 424       7.604   8.999  18.709  1.0   0.00      PROT C \n",
            "ATOM   3377  CG  LYS A 424       7.073   7.584  18.698  1.0   0.00      PROT C \n",
            "ATOM   3378  CD  LYS A 424       5.571   7.583  18.833  1.0   0.00      PROT C \n",
            "ATOM   3379  CE  LYS A 424       4.969   6.290  18.332  1.0   0.00      PROT C \n",
            "ATOM   3380  NZ  LYS A 424       3.477   6.379  18.290  1.0   0.00      PROT N \n",
            "ATOM   3381  H   LYS A 424       7.869   8.559  16.208  1.0   0.00      PROT H \n",
            "ATOM   3382  HA  LYS A 424       5.904   9.933  17.749  1.0   0.00      PROT H \n",
            "ATOM   3383  HB3 LYS A 424       7.388   9.461  19.672  1.0   0.00      PROT H \n",
            "ATOM   3384  HB2 LYS A 424       8.682   8.984  18.547  1.0   0.00      PROT H \n",
            "ATOM   3385  HG3 LYS A 424       7.513   7.034  19.530  1.0   0.00      PROT H \n",
            "ATOM   3386  HG2 LYS A 424       7.354   7.108  17.759  1.0   0.00      PROT H \n",
            "ATOM   3387  HD2 LYS A 424       5.300   7.705  19.882  1.0   0.00      PROT H \n",
            "ATOM   3388  HD3 LYS A 424       5.153   8.405  18.252  1.0   0.00      PROT H \n",
            "ATOM   3389  HE2 LYS A 424       5.337   6.083  17.327  1.0   0.00      PROT H \n",
            "ATOM   3390  HE3 LYS A 424       5.253   5.475  18.998  1.0   0.00      PROT H \n",
            "ATOM   3391  HZ1 LYS A 424       3.158   6.234  17.343  1.0   0.00      PROT H \n",
            "ATOM   3392  HZ2 LYS A 424       3.080   5.671  18.891  1.0   0.00      PROT H \n",
            "ATOM   3393  HZ3 LYS A 424       2.798   7.203  18.556  1.0   0.00      PROT H \n",
            "ATOM   3394  N   VAL A 425       8.681  11.472  16.942  1.0   0.00      PROT N \n",
            "ATOM   3395  CA  VAL A 425       9.349  12.764  16.908  1.0   0.00      PROT C \n",
            "ATOM   3396  C   VAL A 425       8.399  13.752  16.236  1.0   0.00      PROT C \n",
            "ATOM   3397  O   VAL A 425       8.313  14.909  16.647  1.0   0.00      PROT O \n",
            "ATOM   3398  CB  VAL A 425      10.694  12.717  16.116  1.0   0.00      PROT C \n",
            "ATOM   3399  CG1 VAL A 425      11.308  14.116  16.062  1.0   0.00      PROT C \n",
            "ATOM   3400  CG2 VAL A 425      11.669  11.732  16.778  1.0   0.00      PROT C \n",
            "ATOM   3401  H   VAL A 425       9.054  10.717  16.384  1.0   0.00      PROT H \n",
            "ATOM   3402  HA  VAL A 425       9.548  13.019  17.949  1.0   0.00      PROT H \n",
            "ATOM   3403  HB  VAL A 425      10.516  12.381  15.095  1.0   0.00      PROT H \n",
            "ATOM   3404 HG11 VAL A 425      10.606  14.803  15.590  1.0   0.00      PROT H \n",
            "ATOM   3405 HG12 VAL A 425      11.525  14.457  17.074  1.0   0.00      PROT H \n",
            "ATOM   3406 HG13 VAL A 425      12.231  14.086  15.483  1.0   0.00      PROT H \n",
            "ATOM   3407 HG21 VAL A 425      11.218  10.740  16.809  1.0   0.00      PROT H \n",
            "ATOM   3408 HG22 VAL A 425      12.593  11.693  16.202  1.0   0.00      PROT H \n",
            "ATOM   3409 HG23 VAL A 425      11.888  12.064  17.793  1.0   0.00      PROT H \n",
            "ATOM   3410  N   THR A 426       7.678  13.283  15.216  1.0   0.00      PROT N \n",
            "ATOM   3411  CA  THR A 426       6.704  14.117  14.512  1.0   0.00      PROT C \n",
            "ATOM   3412  C   THR A 426       5.524  14.449  15.436  1.0   0.00      PROT C \n",
            "ATOM   3413  O   THR A 426       4.929  15.523  15.332  1.0   0.00      PROT O \n",
            "ATOM   3414  CB  THR A 426       6.170  13.414  13.228  1.0   0.00      PROT C \n",
            "ATOM   3415  OG1 THR A 426       7.196  13.383  12.230  1.0   0.00      PROT O \n",
            "ATOM   3416  CG2 THR A 426       4.962  14.156  12.653  1.0   0.00      PROT C \n",
            "ATOM   3417  H   THR A 426       7.797  12.328  14.910  1.0   0.00      PROT H \n",
            "ATOM   3418  HA  THR A 426       7.258  15.020  14.254  1.0   0.00      PROT H \n",
            "ATOM   3419  HB  THR A 426       5.912  12.376  13.440  1.0   0.00      PROT H \n",
            "ATOM   3420  HG1 THR A 426       8.060  12.940  12.422  1.0   0.00      PROT H \n",
            "ATOM   3421 HG21 THR A 426       4.612  13.642  11.758  1.0   0.00      PROT H \n",
            "ATOM   3422 HG22 THR A 426       4.163  14.179  13.394  1.0   0.00      PROT H \n",
            "ATOM   3423 HG23 THR A 426       5.250  15.176  12.397  1.0   0.00      PROT H \n",
            "ATOM   3424  N   ASP A 427       5.190  13.530  16.339  1.0   0.00      PROT N \n",
            "ATOM   3425  CA  ASP A 427       4.094  13.739  17.273  1.0   0.00      PROT C \n",
            "ATOM   3426  C   ASP A 427       4.418  14.881  18.223  1.0   0.00      PROT C \n",
            "ATOM   3427  O   ASP A 427       3.566  15.735  18.498  1.0   0.00      PROT O \n",
            "ATOM   3428  CB  ASP A 427       3.824  12.473  18.096  1.0   0.00      PROT C \n",
            "ATOM   3429  CG  ASP A 427       3.178  11.370  17.283  1.0   0.00      PROT C \n",
            "ATOM   3430  OD1 ASP A 427       2.527  11.696  16.271  1.0   0.00      PROT O \n",
            "ATOM   3431  OD2 ASP A 427       3.302  10.183  17.661  1.0   0.00      PROT O \n",
            "ATOM   3432  H   ASP A 427       5.702  12.661  16.388  1.0   0.00      PROT H \n",
            "ATOM   3433  HA  ASP A 427       3.237  13.987  16.647  1.0   0.00      PROT H \n",
            "ATOM   3434  HB3 ASP A 427       3.168  12.747  18.922  1.0   0.00      PROT H \n",
            "ATOM   3435  HB2 ASP A 427       4.778  12.123  18.490  1.0   0.00      PROT H \n",
            "ATOM   3436  N   LEU A 428       5.651  14.888  18.726  1.0   0.00      PROT N \n",
            "ATOM   3437  CA  LEU A 428       6.108  15.911  19.660  1.0   0.00      PROT C \n",
            "ATOM   3438  C   LEU A 428       6.163  17.296  19.027  1.0   0.00      PROT C \n",
            "ATOM   3439  O   LEU A 428       5.900  18.299  19.692  1.0   0.00      PROT O \n",
            "ATOM   3440  CB  LEU A 428       7.474  15.517  20.218  1.0   0.00      PROT C \n",
            "ATOM   3441  CG  LEU A 428       7.392  14.280  21.118  1.0   0.00      PROT C \n",
            "ATOM   3442  CD1 LEU A 428       8.749  13.591  21.242  1.0   0.00      PROT C \n",
            "ATOM   3443  CD2 LEU A 428       6.848  14.708  22.478  1.0   0.00      PROT C \n",
            "ATOM   3444  H   LEU A 428       6.308  14.167  18.463  1.0   0.00      PROT H \n",
            "ATOM   3445  HA  LEU A 428       5.355  15.936  20.448  1.0   0.00      PROT H \n",
            "ATOM   3446  HB2 LEU A 428       8.152  15.294  19.394  1.0   0.00      PROT H \n",
            "ATOM   3447  HB3 LEU A 428       7.877  16.340  20.808  1.0   0.00      PROT H \n",
            "ATOM   3448  HG  LEU A 428       6.718  13.545  20.678  1.0   0.00      PROT H \n",
            "ATOM   3449 HD11 LEU A 428       9.102  13.302  20.252  1.0   0.00      PROT H \n",
            "ATOM   3450 HD12 LEU A 428       9.464  14.276  21.697  1.0   0.00      PROT H \n",
            "ATOM   3451 HD13 LEU A 428       8.650  12.703  21.866  1.0   0.00      PROT H \n",
            "ATOM   3452 HD21 LEU A 428       5.881  15.194  22.346  1.0   0.00      PROT H \n",
            "ATOM   3453 HD22 LEU A 428       6.730  13.831  23.114  1.0   0.00      PROT H \n",
            "ATOM   3454 HD23 LEU A 428       7.544  15.405  22.945  1.0   0.00      PROT H \n",
            "ATOM   3455  N   ARG A 429       6.501  17.358  17.744  1.0   0.00      PROT N \n",
            "ATOM   3456  CA  ARG A 429       6.548  18.638  17.051  1.0   0.00      PROT C \n",
            "ATOM   3457  C   ARG A 429       5.121  19.158  16.874  1.0   0.00      PROT C \n",
            "ATOM   3458  O   ARG A 429       4.887  20.361  16.902  1.0   0.00      PROT O \n",
            "ATOM   3459  CB  ARG A 429       7.232  18.496  15.689  1.0   0.00      PROT C \n",
            "ATOM   3460  CG  ARG A 429       8.728  18.266  15.774  1.0   0.00      PROT C \n",
            "ATOM   3461  CD  ARG A 429       9.316  18.018  14.398  1.0   0.00      PROT C \n",
            "ATOM   3462  NE  ARG A 429      10.726  17.618  14.441  1.0   0.00      PROT N \n",
            "ATOM   3463  CZ  ARG A 429      11.704  18.349  14.976  1.0   0.00      PROT C \n",
            "ATOM   3464  NH1 ARG A 429      11.430  19.531  15.522  1.0   0.00      PROT N \n",
            "ATOM   3465  NH2 ARG A 429      12.955  17.900  14.962  1.0   0.00      PROT N \n",
            "ATOM   3466  H   ARG A 429       6.728  16.513  17.240  1.0   0.00      PROT H \n",
            "ATOM   3467  HA  ARG A 429       7.135  19.308  17.679  1.0   0.00      PROT H \n",
            "ATOM   3468  HB3 ARG A 429       7.036  19.413  15.133  1.0   0.00      PROT H \n",
            "ATOM   3469  HB2 ARG A 429       6.763  17.654  15.179  1.0   0.00      PROT H \n",
            "ATOM   3470  HG2 ARG A 429       9.207  19.145  16.207  1.0   0.00      PROT H \n",
            "ATOM   3471  HG3 ARG A 429       8.927  17.398  16.402  1.0   0.00      PROT H \n",
            "ATOM   3472  HD2 ARG A 429       8.761  17.221  13.904  1.0   0.00      PROT H \n",
            "ATOM   3473  HD3 ARG A 429       9.249  18.930  13.804  1.0   0.00      PROT H \n",
            "ATOM   3474  HE  ARG A 429      10.935  16.721  14.027  1.0   0.00      PROT H \n",
            "ATOM   3475 HH12 ARG A 429      12.179  20.091  15.929  1.0   0.00      PROT H \n",
            "ATOM   3476 HH11 ARG A 429      10.471  19.877  15.534  1.0   0.00      PROT H \n",
            "ATOM   3477  HH2 ARG A 429      13.703  18.461  15.369  1.0   0.00      PROT H \n",
            "ATOM   3478  N   MET A 430       4.171  18.243  16.699  1.0   0.00      PROT N \n",
            "ATOM   3479  CA  MET A 430       2.770  18.619  16.553  1.0   0.00      PROT C \n",
            "ATOM   3480  C   MET A 430       2.273  19.140  17.892  1.0   0.00      PROT C \n",
            "ATOM   3481  O   MET A 430       1.585  20.157  17.948  1.0   0.00      PROT O \n",
            "ATOM   3482  CB  MET A 430       1.931  17.417  16.109  1.0   0.00      PROT C \n",
            "ATOM   3483  CG  MET A 430       2.322  16.871  14.741  1.0   0.00      PROT C \n",
            "ATOM   3484  SD  MET A 430       2.436  18.189  13.488  1.0   0.00      PROT S \n",
            "ATOM   3485  CE  MET A 430       0.802  18.347  13.074  1.0   0.00      PROT C \n",
            "ATOM   3486  H   MET A 430       4.414  17.263  16.665  1.0   0.00      PROT H \n",
            "ATOM   3487  HA  MET A 430       2.722  19.395  15.789  1.0   0.00      PROT H \n",
            "ATOM   3488  HB3 MET A 430       0.889  17.738  16.088  1.0   0.00      PROT H \n",
            "ATOM   3489  HB2 MET A 430       2.056  16.638  16.861  1.0   0.00      PROT H \n",
            "ATOM   3490  HG2 MET A 430       3.293  16.381  14.809  1.0   0.00      PROT H \n",
            "ATOM   3491  HG3 MET A 430       1.574  16.150  14.410  1.0   0.00      PROT H \n",
            "ATOM   3492  HE1 MET A 430       0.693  19.119  12.312  1.0   0.00      PROT H \n",
            "ATOM   3493  HE2 MET A 430       0.432  17.398  12.685  1.0   0.00      PROT H \n",
            "ATOM   3494  HE3 MET A 430       0.229  18.626  13.958  1.0   0.00      PROT H \n",
            "ATOM   3495  N   ILE A 431       2.635  18.437  18.965  1.0   0.00      PROT N \n",
            "ATOM   3496  CA  ILE A 431       2.260  18.829  20.321  1.0   0.00      PROT C \n",
            "ATOM   3497  C   ILE A 431       2.845  20.203  20.688  1.0   0.00      PROT C \n",
            "ATOM   3498  O   ILE A 431       2.115  21.104  21.091  1.0   0.00      PROT O \n",
            "ATOM   3499  CB  ILE A 431       2.735  17.780  21.371  1.0   0.00      PROT C \n",
            "ATOM   3500  CG1 ILE A 431       1.954  16.476  21.211  1.0   0.00      PROT C \n",
            "ATOM   3501  CG2 ILE A 431       2.549  18.324  22.783  1.0   0.00      PROT C \n",
            "ATOM   3502  CD1 ILE A 431       2.336  15.410  22.217  1.0   0.00      PROT C \n",
            "ATOM   3503  H   ILE A 431       3.189  17.600  18.857  1.0   0.00      PROT H \n",
            "ATOM   3504  HA  ILE A 431       1.172  18.880  20.275  1.0   0.00      PROT H \n",
            "ATOM   3505  HB  ILE A 431       3.794  17.568  21.223  1.0   0.00      PROT H \n",
            "ATOM   3506 HG13 ILE A 431       2.141  16.107  20.202  1.0   0.00      PROT H \n",
            "ATOM   3507 HG12 ILE A 431       0.896  16.716  21.320  1.0   0.00      PROT H \n",
            "ATOM   3508 HG21 ILE A 431       3.108  19.253  22.891  1.0   0.00      PROT H \n",
            "ATOM   3509 HG22 ILE A 431       1.491  18.514  22.963  1.0   0.00      PROT H \n",
            "ATOM   3510 HG23 ILE A 431       2.915  17.594  23.505  1.0   0.00      PROT H \n",
            "ATOM   3511 HD11 ILE A 431       1.740  14.514  22.041  1.0   0.00      PROT H \n",
            "ATOM   3512 HD12 ILE A 431       3.394  15.170  22.108  1.0   0.00      PROT H \n",
            "ATOM   3513 HD13 ILE A 431       2.149  15.779  23.226  1.0   0.00      PROT H \n",
            "ATOM   3514  N   GLY A 432       4.157  20.364  20.554  1.0   0.00      PROT N \n",
            "ATOM   3515  CA  GLY A 432       4.764  21.640  20.876  1.0   0.00      PROT C \n",
            "ATOM   3516  C   GLY A 432       4.081  22.806  20.179  1.0   0.00      PROT C \n",
            "ATOM   3517  O   GLY A 432       3.863  23.854  20.783  1.0   0.00      PROT O \n",
            "ATOM   3518  H   GLY A 432       4.729  19.598  20.228  1.0   0.00      PROT H \n",
            "ATOM   3519  HA2 GLY A 432       5.810  21.598  20.573  1.0   0.00      PROT H \n",
            "ATOM   3520  HA3 GLY A 432       4.703  21.776  21.956  1.0   0.00      PROT H \n",
            "ATOM   3521  N   ALA A 433       3.736  22.638  18.907  1.0   0.00      PROT N \n",
            "ATOM   3522  CA  ALA A 433       3.077  23.704  18.155  1.0   0.00      PROT C \n",
            "ATOM   3523  C   ALA A 433       1.652  23.944  18.655  1.0   0.00      PROT C \n",
            "ATOM   3524  O   ALA A 433       1.158  25.074  18.670  1.0   0.00      PROT O \n",
            "ATOM   3525  CB  ALA A 433       3.046  23.359  16.679  1.0   0.00      PROT C \n",
            "ATOM   3526  H   ALA A 433       3.928  21.762  18.443  1.0   0.00      PROT H \n",
            "ATOM   3527  HA  ALA A 433       3.683  24.594  18.325  1.0   0.00      PROT H \n",
            "ATOM   3528  HB1 ALA A 433       4.062  23.188  16.324  1.0   0.00      PROT H \n",
            "ATOM   3529  HB2 ALA A 433       2.453  22.457  16.529  1.0   0.00      PROT H \n",
            "ATOM   3530  HB3 ALA A 433       2.601  24.184  16.122  1.0   0.00      PROT H \n",
            "ATOM   3531  N   CYS A 434       0.985  22.872  19.053  1.0   0.00      PROT N \n",
            "ATOM   3532  CA  CYS A 434      -0.375  22.995  19.529  1.0   0.00      PROT C \n",
            "ATOM   3533  C   CYS A 434      -0.375  23.725  20.858  1.0   0.00      PROT C \n",
            "ATOM   3534  O   CYS A 434      -1.318  24.439  21.176  1.0   0.00      PROT O \n",
            "ATOM   3535  CB  CYS A 434      -1.012  21.617  19.679  1.0   0.00      PROT C \n",
            "ATOM   3536  SG  CYS A 434      -2.785  21.714  19.962  1.0   0.00      PROT S \n",
            "ATOM   3537  H   CYS A 434       1.426  21.964  19.025  1.0   0.00      PROT H \n",
            "ATOM   3538  HA  CYS A 434      -0.916  23.572  18.779  1.0   0.00      PROT H \n",
            "ATOM   3539  HB2 CYS A 434      -0.846  21.039  18.770  1.0   0.00      PROT H \n",
            "ATOM   3540  HB3 CYS A 434      -0.562  21.099  20.526  1.0   0.00      PROT H \n",
            "ATOM   3541  HG  CYS A 434      -3.501  20.592  20.116  1.0   0.00      PROT H \n",
            "ATOM   3542  N   HIS A 435       0.700  23.544  21.622  1.0   0.00      PROT N \n",
            "ATOM   3543  CA  HIS A 435       0.863  24.182  22.926  1.0   0.00      PROT C \n",
            "ATOM   3544  C   HIS A 435       0.995  25.690  22.779  1.0   0.00      PROT C \n",
            "ATOM   3545  O   HIS A 435       0.559  26.450  23.634  1.0   0.00      PROT O \n",
            "ATOM   3546  CB  HIS A 435       2.094  23.618  23.632  1.0   0.00      PROT C \n",
            "ATOM   3547  CG  HIS A 435       2.467  24.363  24.877  1.0   0.00      PROT C \n",
            "ATOM   3548  ND1 HIS A 435       3.338  25.428  24.873  1.0   0.00      PROT N \n",
            "ATOM   3549  CD2 HIS A 435       2.026  24.237  26.153  1.0   0.00      PROT C \n",
            "ATOM   3550  CE1 HIS A 435       3.415  25.934  26.095  1.0   0.00      PROT C \n",
            "ATOM   3551  NE2 HIS A 435       2.629  25.229  26.885  1.0   0.00      PROT N \n",
            "ATOM   3552  H   HIS A 435       1.448  22.944  21.307  1.0   0.00      PROT H \n",
            "ATOM   3553  HA  HIS A 435      -0.042  23.951  23.488  1.0   0.00      PROT H \n",
            "ATOM   3554  HB3 HIS A 435       2.921  23.657  22.923  1.0   0.00      PROT H \n",
            "ATOM   3555  HB2 HIS A 435       1.878  22.579  23.881  1.0   0.00      PROT H \n",
            "ATOM   3556  HD1 HIS A 435       3.878  25.795  24.001  1.0   0.00      PROT H \n",
            "ATOM   3557  HD2 HIS A 435       1.321  23.446  26.411  1.0   0.00      PROT H \n",
            "ATOM   3558  HE1 HIS A 435       4.057  26.793  26.290  1.0   0.00      PROT H \n",
            "ATOM   3559  N   ALA A 436       1.602  26.111  21.679  1.0   0.00      PROT N \n",
            "ATOM   3560  CA  ALA A 436       1.792  27.524  21.393  1.0   0.00      PROT C \n",
            "ATOM   3561  C   ALA A 436       0.449  28.244  21.314  1.0   0.00      PROT C \n",
            "ATOM   3562  O   ALA A 436       0.255  29.289  21.937  1.0   0.00      PROT O \n",
            "ATOM   3563  CB  ALA A 436       2.543  27.678  20.088  1.0   0.00      PROT C \n",
            "ATOM   3564  H   ALA A 436       1.950  25.442  21.007  1.0   0.00      PROT H \n",
            "ATOM   3565  HA  ALA A 436       2.379  27.929  22.217  1.0   0.00      PROT H \n",
            "ATOM   3566  HB1 ALA A 436       3.500  27.160  20.156  1.0   0.00      PROT H \n",
            "ATOM   3567  HB2 ALA A 436       1.954  27.249  19.277  1.0   0.00      PROT H \n",
            "ATOM   3568  HB3 ALA A 436       2.716  28.736  19.891  1.0   0.00      PROT H \n",
            "ATOM   3569  N   SER A 437      -0.482  27.681  20.551  1.0   0.00      PROT N \n",
            "ATOM   3570  CA  SER A 437      -1.795  28.288  20.416  1.0   0.00      PROT C \n",
            "ATOM   3571  C   SER A 437      -2.542  28.184  21.739  1.0   0.00      PROT C \n",
            "ATOM   3572  O   SER A 437      -3.392  29.015  22.049  1.0   0.00      PROT O \n",
            "ATOM   3573  CB  SER A 437      -2.596  27.589  19.315  1.0   0.00      PROT C \n",
            "ATOM   3574  OG  SER A 437      -2.994  26.290  19.713  1.0   0.00      PROT O \n",
            "ATOM   3575  H   SER A 437      -0.280  26.823  20.058  1.0   0.00      PROT H \n",
            "ATOM   3576  HA  SER A 437      -1.621  29.331  20.152  1.0   0.00      PROT H \n",
            "ATOM   3577  HB2 SER A 437      -3.490  28.171  19.090  1.0   0.00      PROT H \n",
            "ATOM   3578  HB3 SER A 437      -1.983  27.504  18.418  1.0   0.00      PROT H \n",
            "ATOM   3579  HG  SER A 437      -3.521  25.723  19.096  1.0   0.00      PROT H \n",
            "ATOM   3580  N   ARG A 438      -2.207  27.163  22.518  1.0   0.00      PROT N \n",
            "ATOM   3581  CA  ARG A 438      -2.844  26.925  23.805  1.0   0.00      PROT C \n",
            "ATOM   3582  C   ARG A 438      -2.407  27.942  24.861  1.0   0.00      PROT C \n",
            "ATOM   3583  O   ARG A 438      -3.195  28.338  25.725  1.0   0.00      PROT O \n",
            "ATOM   3584  CB  ARG A 438      -2.525  25.508  24.285  1.0   0.00      PROT C \n",
            "ATOM   3585  CG  ARG A 438      -3.365  25.070  25.457  1.0   0.00      PROT C \n",
            "ATOM   3586  CD  ARG A 438      -4.828  25.078  25.085  1.0   0.00      PROT C \n",
            "ATOM   3587  NE  ARG A 438      -5.681  25.018  26.262  1.0   0.00      PROT N \n",
            "ATOM   3588  CZ  ARG A 438      -6.997  25.180  26.229  1.0   0.00      PROT C \n",
            "ATOM   3589  NH1 ARG A 438      -7.603  25.410  25.068  1.0   0.00      PROT N \n",
            "ATOM   3590  NH2 ARG A 438      -7.705  25.120  27.352  1.0   0.00      PROT N \n",
            "ATOM   3591  H   ARG A 438      -1.487  26.518  22.224  1.0   0.00      PROT H \n",
            "ATOM   3592  HA  ARG A 438      -3.910  27.049  23.614  1.0   0.00      PROT H \n",
            "ATOM   3593  HB3 ARG A 438      -1.472  25.486  24.565  1.0   0.00      PROT H \n",
            "ATOM   3594  HB2 ARG A 438      -2.692  24.833  23.446  1.0   0.00      PROT H \n",
            "ATOM   3595  HG2 ARG A 438      -3.078  24.061  25.752  1.0   0.00      PROT H \n",
            "ATOM   3596  HG3 ARG A 438      -3.207  25.752  26.293  1.0   0.00      PROT H \n",
            "ATOM   3597  HD2 ARG A 438      -5.059  25.993  24.539  1.0   0.00      PROT H \n",
            "ATOM   3598  HD3 ARG A 438      -5.048  24.215  24.457  1.0   0.00      PROT H \n",
            "ATOM   3599  HE  ARG A 438      -5.216  24.841  27.141  1.0   0.00      PROT H \n",
            "ATOM   3600 HH12 ARG A 438      -8.614  25.541  25.038  1.0   0.00      PROT H \n",
            "ATOM   3601 HH11 ARG A 438      -7.057  25.455  24.208  1.0   0.00      PROT H \n",
            "ATOM   3602  HH2 ARG A 438      -8.716  25.251  27.325  1.0   0.00      PROT H \n",
            "ATOM   3603  N   PHE A 439      -1.146  28.353  24.790  1.0   0.00      PROT N \n",
            "ATOM   3604  CA  PHE A 439      -0.601  29.337  25.718  1.0   0.00      PROT C \n",
            "ATOM   3605  C   PHE A 439      -1.411  30.621  25.604  1.0   0.00      PROT C \n",
            "ATOM   3606  O   PHE A 439      -1.826  31.196  26.605  1.0   0.00      PROT O \n",
            "ATOM   3607  CB  PHE A 439       0.863  29.630  25.371  1.0   0.00      PROT C \n",
            "ATOM   3608  CG  PHE A 439       1.509  30.643  26.271  1.0   0.00      PROT C \n",
            "ATOM   3609  CD1 PHE A 439       1.778  30.342  27.601  1.0   0.00      PROT C \n",
            "ATOM   3610  CD2 PHE A 439       1.853  31.899  25.792  1.0   0.00      PROT C \n",
            "ATOM   3611  CE1 PHE A 439       2.384  31.276  28.439  1.0   0.00      PROT C \n",
            "ATOM   3612  CE2 PHE A 439       2.458  32.841  26.623  1.0   0.00      PROT C \n",
            "ATOM   3613  CZ  PHE A 439       2.722  32.527  27.947  1.0   0.00      PROT C \n",
            "ATOM   3614  H   PHE A 439      -0.533  27.982  24.078  1.0   0.00      PROT H \n",
            "ATOM   3615  HA  PHE A 439      -0.679  28.910  26.718  1.0   0.00      PROT H \n",
            "ATOM   3616  HB3 PHE A 439       0.885  29.991  24.343  1.0   0.00      PROT H \n",
            "ATOM   3617  HB2 PHE A 439       1.406  28.687  25.437  1.0   0.00      PROT H \n",
            "ATOM   3618  HD1 PHE A 439       1.514  29.364  28.002  1.0   0.00      PROT H \n",
            "ATOM   3619  HD2 PHE A 439       1.651  32.160  24.753  1.0   0.00      PROT H \n",
            "ATOM   3620  HE1 PHE A 439       2.592  31.025  29.479  1.0   0.00      PROT H \n",
            "ATOM   3621  HE2 PHE A 439       2.723  33.823  26.233  1.0   0.00      PROT H \n",
            "ATOM   3622  HZ  PHE A 439       3.194  33.263  28.598  1.0   0.00      PROT H \n",
            "ATOM   3623  N   LEU A 440      -1.627  31.061  24.366  1.0   0.00      PROT N \n",
            "ATOM   3624  CA  LEU A 440      -2.375  32.281  24.087  1.0   0.00      PROT C \n",
            "ATOM   3625  C   LEU A 440      -3.740  32.277  24.770  1.0   0.00      PROT C \n",
            "ATOM   3626  O   LEU A 440      -4.183  33.293  25.308  1.0   0.00      PROT O \n",
            "ATOM   3627  CB  LEU A 440      -2.537  32.459  22.569  1.0   0.00      PROT C \n",
            "ATOM   3628  CG  LEU A 440      -1.210  32.628  21.805  1.0   0.00      PROT C \n",
            "ATOM   3629  CD1 LEU A 440      -1.441  32.645  20.302  1.0   0.00      PROT C \n",
            "ATOM   3630  CD2 LEU A 440      -0.523  33.910  22.260  1.0   0.00      PROT C \n",
            "ATOM   3631  H   LEU A 440      -1.267  30.544  23.576  1.0   0.00      PROT H \n",
            "ATOM   3632  HA  LEU A 440      -1.776  33.085  24.514  1.0   0.00      PROT H \n",
            "ATOM   3633  HB2 LEU A 440      -3.036  31.584  22.152  1.0   0.00      PROT H \n",
            "ATOM   3634  HB3 LEU A 440      -3.135  33.348  22.369  1.0   0.00      PROT H \n",
            "ATOM   3635  HG  LEU A 440      -0.557  31.781  22.018  1.0   0.00      PROT H \n",
            "ATOM   3636 HD11 LEU A 440      -1.932  31.720  19.999  1.0   0.00      PROT H \n",
            "ATOM   3637 HD12 LEU A 440      -2.072  33.494  20.041  1.0   0.00      PROT H \n",
            "ATOM   3638 HD13 LEU A 440      -0.484  32.733  19.788  1.0   0.00      PROT H \n",
            "ATOM   3639 HD21 LEU A 440      -0.368  33.876  23.338  1.0   0.00      PROT H \n",
            "ATOM   3640 HD22 LEU A 440       0.439  34.005  21.757  1.0   0.00      PROT H \n",
            "ATOM   3641 HD23 LEU A 440      -1.149  34.766  22.010  1.0   0.00      PROT H \n",
            "ATOM   3642  N   HIS A 441      -4.395  31.123  24.755  1.0   0.00      PROT N \n",
            "ATOM   3643  CA  HIS A 441      -5.710  30.981  25.362  1.0   0.00      PROT C \n",
            "ATOM   3644  C   HIS A 441      -5.647  31.085  26.881  1.0   0.00      PROT C \n",
            "ATOM   3645  O   HIS A 441      -6.532  31.674  27.511  1.0   0.00      PROT O \n",
            "ATOM   3646  CB  HIS A 441      -6.321  29.641  24.965  1.0   0.00      PROT C \n",
            "ATOM   3647  CG  HIS A 441      -7.750  29.492  25.374  1.0   0.00      PROT C \n",
            "ATOM   3648  ND1 HIS A 441      -8.665  30.516  25.252  1.0   0.00      PROT N \n",
            "ATOM   3649  CD2 HIS A 441      -8.437  28.429  25.853  1.0   0.00      PROT C \n",
            "ATOM   3650  CE1 HIS A 441      -9.853  30.090  25.637  1.0   0.00      PROT C \n",
            "ATOM   3651  NE2 HIS A 441      -9.743  28.826  26.007  1.0   0.00      PROT N \n",
            "ATOM   3652  H   HIS A 441      -3.982  30.314  24.314  1.0   0.00      PROT H \n",
            "ATOM   3653  HA  HIS A 441      -6.294  31.814  24.970  1.0   0.00      PROT H \n",
            "ATOM   3654  HB3 HIS A 441      -5.725  28.859  25.435  1.0   0.00      PROT H \n",
            "ATOM   3655  HB2 HIS A 441      -6.242  29.555  23.881  1.0   0.00      PROT H \n",
            "ATOM   3656  HD1 HIS A 441      -8.438  31.520  24.893  1.0   0.00      PROT H \n",
            "ATOM   3657  HD2 HIS A 441      -7.922  27.486  26.038  1.0   0.00      PROT H \n",
            "ATOM   3658  HE1 HIS A 441     -10.709  30.764  25.609  1.0   0.00      PROT H \n",
            "ATOM   3659  N   MET A 442      -4.605  30.494  27.463  1.0   0.00      PROT N \n",
            "ATOM   3660  CA  MET A 442      -4.406  30.531  28.906  1.0   0.00      PROT C \n",
            "ATOM   3661  C   MET A 442      -4.300  31.995  29.323  1.0   0.00      PROT C \n",
            "ATOM   3662  O   MET A 442      -4.810  32.395  30.366  1.0   0.00      PROT O \n",
            "ATOM   3663  CB  MET A 442      -3.118  29.797  29.289  1.0   0.00      PROT C \n",
            "ATOM   3664  CG  MET A 442      -3.126  28.280  29.105  1.0   0.00      PROT C \n",
            "ATOM   3665  SD  MET A 442      -1.521  27.564  29.594  1.0   0.00      PROT S \n",
            "ATOM   3666  CE  MET A 442      -1.321  26.238  28.385  1.0   0.00      PROT C \n",
            "ATOM   3667  H   MET A 442      -3.925  30.002  26.901  1.0   0.00      PROT H \n",
            "ATOM   3668  HA  MET A 442      -5.261  30.035  29.366  1.0   0.00      PROT H \n",
            "ATOM   3669  HB3 MET A 442      -2.937  30.035  30.337  1.0   0.00      PROT H \n",
            "ATOM   3670  HB2 MET A 442      -2.330  30.233  28.676  1.0   0.00      PROT H \n",
            "ATOM   3671  HG2 MET A 442      -3.910  27.843  29.723  1.0   0.00      PROT H \n",
            "ATOM   3672  HG3 MET A 442      -3.314  28.042  28.058  1.0   0.00      PROT H \n",
            "ATOM   3673  HE1 MET A 442      -0.381  25.718  28.568  1.0   0.00      PROT H \n",
            "ATOM   3674  HE2 MET A 442      -2.149  25.535  28.477  1.0   0.00      PROT H \n",
            "ATOM   3675  HE3 MET A 442      -1.313  26.661  27.380  1.0   0.00      PROT H \n",
            "ATOM   3676  N   LYS A 443      -3.630  32.788  28.494  1.0   0.00      PROT N \n",
            "ATOM   3677  CA  LYS A 443      -3.454  34.212  28.754  1.0   0.00      PROT C \n",
            "ATOM   3678  C   LYS A 443      -4.789  34.925  28.839  1.0   0.00      PROT C \n",
            "ATOM   3679  O   LYS A 443      -4.909  35.957  29.498  1.0   0.00      PROT O \n",
            "ATOM   3680  CB  LYS A 443      -2.634  34.859  27.646  1.0   0.00      PROT C \n",
            "ATOM   3681  CG  LYS A 443      -1.143  34.925  27.906  1.0   0.00      PROT C \n",
            "ATOM   3682  CD  LYS A 443      -0.495  35.653  26.744  1.0   0.00      PROT C \n",
            "ATOM   3683  CE  LYS A 443       0.951  35.985  26.997  1.0   0.00      PROT C \n",
            "ATOM   3684  NZ  LYS A 443       1.463  36.785  25.855  1.0   0.00      PROT N \n",
            "ATOM   3685  H   LYS A 443      -3.223  32.409  27.651  1.0   0.00      PROT H \n",
            "ATOM   3686  HA  LYS A 443      -2.938  34.260  29.713  1.0   0.00      PROT H \n",
            "ATOM   3687  HB3 LYS A 443      -3.025  35.868  27.517  1.0   0.00      PROT H \n",
            "ATOM   3688  HB2 LYS A 443      -2.820  34.279  26.742  1.0   0.00      PROT H \n",
            "ATOM   3689  HG2 LYS A 443      -0.960  35.463  28.836  1.0   0.00      PROT H \n",
            "ATOM   3690  HG3 LYS A 443      -0.743  33.914  27.986  1.0   0.00      PROT H \n",
            "ATOM   3691  HD3 LYS A 443      -1.055  36.573  26.574  1.0   0.00      PROT H \n",
            "ATOM   3692  HD2 LYS A 443      -0.572  35.013  25.865  1.0   0.00      PROT H \n",
            "ATOM   3693  HE2 LYS A 443       1.036  36.560  27.919  1.0   0.00      PROT H \n",
            "ATOM   3694  HE3 LYS A 443       1.525  35.063  27.090  1.0   0.00      PROT H \n",
            "ATOM   3695  HZ1 LYS A 443       1.780  37.684  26.189  1.0   0.00      PROT H \n",
            "ATOM   3696  HZ2 LYS A 443       2.233  36.297  25.420  1.0   0.00      PROT H \n",
            "ATOM   3697  HZ3 LYS A 443       0.958  37.127  24.940  1.0   0.00      PROT H \n",
            "ATOM   3698  N   VAL A 444      -5.785  34.373  28.153  1.0   0.00      PROT N \n",
            "ATOM   3699  CA  VAL A 444      -7.124  34.940  28.131  1.0   0.00      PROT C \n",
            "ATOM   3700  C   VAL A 444      -7.974  34.435  29.292  1.0   0.00      PROT C \n",
            "ATOM   3701  O   VAL A 444      -8.665  35.206  29.950  1.0   0.00      PROT O \n",
            "ATOM   3702  CB  VAL A 444      -7.842  34.590  26.810  1.0   0.00      PROT C \n",
            "ATOM   3703  CG1 VAL A 444      -9.319  34.984  26.892  1.0   0.00      PROT C \n",
            "ATOM   3704  CG2 VAL A 444      -7.152  35.290  25.640  1.0   0.00      PROT C \n",
            "ATOM   3705  H   VAL A 444      -5.625  33.529  27.622  1.0   0.00      PROT H \n",
            "ATOM   3706  HA  VAL A 444      -6.951  36.011  28.235  1.0   0.00      PROT H \n",
            "ATOM   3707  HB  VAL A 444      -7.791  33.515  26.637  1.0   0.00      PROT H \n",
            "ATOM   3708 HG11 VAL A 444      -9.786  34.472  27.733  1.0   0.00      PROT H \n",
            "ATOM   3709 HG12 VAL A 444      -9.400  36.062  27.033  1.0   0.00      PROT H \n",
            "ATOM   3710 HG13 VAL A 444      -9.823  34.699  25.968  1.0   0.00      PROT H \n",
            "ATOM   3711 HG21 VAL A 444      -6.104  34.992  25.605  1.0   0.00      PROT H \n",
            "ATOM   3712 HG22 VAL A 444      -7.641  35.007  24.708  1.0   0.00      PROT H \n",
            "ATOM   3713 HG23 VAL A 444      -7.219  36.370  25.772  1.0   0.00      PROT H \n",
            "ATOM   3714  N   GLU A 445      -7.912  33.135  29.547  1.0   0.00      PROT N \n",
            "ATOM   3715  CA  GLU A 445      -8.697  32.529  30.614  1.0   0.00      PROT C \n",
            "ATOM   3716  C   GLU A 445      -8.125  32.667  32.026  1.0   0.00      PROT C \n",
            "ATOM   3717  O   GLU A 445      -8.866  32.551  33.003  1.0   0.00      PROT O \n",
            "ATOM   3718  CB  GLU A 445      -8.906  31.049  30.299  1.0   0.00      PROT C \n",
            "ATOM   3719  CG  GLU A 445      -9.752  30.800  29.076  1.0   0.00      PROT C \n",
            "ATOM   3720  CD  GLU A 445     -11.206  31.136  29.316  1.0   0.00      PROT C \n",
            "ATOM   3721  OE1 GLU A 445     -11.495  32.278  29.725  1.0   0.00      PROT O \n",
            "ATOM   3722  OE2 GLU A 445     -12.066  30.260  29.097  1.0   0.00      PROT O \n",
            "ATOM   3723  H   GLU A 445      -7.310  32.540  28.995  1.0   0.00      PROT H \n",
            "ATOM   3724  HA  GLU A 445      -9.614  33.117  30.590  1.0   0.00      PROT H \n",
            "ATOM   3725  HB3 GLU A 445      -9.383  30.598  31.169  1.0   0.00      PROT H \n",
            "ATOM   3726  HB2 GLU A 445      -7.920  30.607  30.155  1.0   0.00      PROT H \n",
            "ATOM   3727  HG2 GLU A 445      -9.391  31.417  28.253  1.0   0.00      PROT H \n",
            "ATOM   3728  HG3 GLU A 445      -9.687  29.748  28.797  1.0   0.00      PROT H \n",
            "ATOM   3729  HE1 GLU A 445     -12.485  32.360  29.839  1.0   0.00      PROT H \n",
            "ATOM   3730  N   CYS A 446      -6.822  32.921  32.135  1.0   0.00      PROT N \n",
            "ATOM   3731  CA  CYS A 446      -6.173  33.024  33.444  1.0   0.00      PROT C \n",
            "ATOM   3732  C   CYS A 446      -5.544  34.381  33.764  1.0   0.00      PROT C \n",
            "ATOM   3733  O   CYS A 446      -5.044  35.079  32.876  1.0   0.00      PROT O \n",
            "ATOM   3734  CB  CYS A 446      -5.089  31.944  33.583  1.0   0.00      PROT C \n",
            "ATOM   3735  SG  CYS A 446      -5.615  30.256  33.224  1.0   0.00      PROT S \n",
            "ATOM   3736  H   CYS A 446      -6.261  33.047  31.305  1.0   0.00      PROT H \n",
            "ATOM   3737  HA  CYS A 446      -7.018  32.886  34.119  1.0   0.00      PROT H \n",
            "ATOM   3738  HB2 CYS A 446      -4.270  32.161  32.897  1.0   0.00      PROT H \n",
            "ATOM   3739  HB3 CYS A 446      -4.714  31.936  34.606  1.0   0.00      PROT H \n",
            "ATOM   3740  HG  CYS A 446      -4.739  29.247  33.318  1.0   0.00      PROT H \n",
            "ATOM   3741  N   PRO A 447      -5.544  34.757  35.056  1.0   0.00      PROT N \n",
            "ATOM   3742  CA  PRO A 447      -4.978  36.025  35.522  1.0   0.00      PROT C \n",
            "ATOM   3743  C   PRO A 447      -3.476  36.089  35.245  1.0   0.00      PROT C \n",
            "ATOM   3744  O   PRO A 447      -2.755  35.109  35.440  1.0   0.00      PROT O \n",
            "ATOM   3745  CB  PRO A 447      -5.299  36.010  37.010  1.0   0.00      PROT C \n",
            "ATOM   3746  CG  PRO A 447      -5.221  34.553  37.343  1.0   0.00      PROT C \n",
            "ATOM   3747  CD  PRO A 447      -5.975  33.928  36.197  1.0   0.00      PROT C \n",
            "ATOM   3748  HA  PRO A 447      -5.409  36.879  35.000  1.0   0.00      PROT H \n",
            "ATOM   3749  HB3 PRO A 447      -6.300  36.380  37.231  1.0   0.00      PROT H \n",
            "ATOM   3750  HB2 PRO A 447      -4.569  36.558  37.606  1.0   0.00      PROT H \n",
            "ATOM   3751  HG2 PRO A 447      -4.176  34.245  37.385  1.0   0.00      PROT H \n",
            "ATOM   3752  HG3 PRO A 447      -5.691  34.376  38.310  1.0   0.00      PROT H \n",
            "ATOM   3753  HD2 PRO A 447      -5.684  32.882  36.098  1.0   0.00      PROT H \n",
            "ATOM   3754  HD3 PRO A 447      -7.046  33.991  36.389  1.0   0.00      PROT H \n",
            "ATOM   3755  N   THR A 448      -3.010  37.248  34.795  1.0   0.00      PROT N \n",
            "ATOM   3756  CA  THR A 448      -1.599  37.427  34.462  1.0   0.00      PROT C \n",
            "ATOM   3757  C   THR A 448      -0.653  37.250  35.637  1.0   0.00      PROT C \n",
            "ATOM   3758  O   THR A 448       0.529  36.984  35.442  1.0   0.00      PROT O \n",
            "ATOM   3759  CB  THR A 448      -1.352  38.808  33.821  1.0   0.00      PROT C \n",
            "ATOM   3760  OG1 THR A 448      -2.164  38.937  32.645  1.0   0.00      PROT O \n",
            "ATOM   3761  CG2 THR A 448       0.113  38.954  33.417  1.0   0.00      PROT C \n",
            "ATOM   3762  H   THR A 448      -3.636  38.032  34.676  1.0   0.00      PROT H \n",
            "ATOM   3763  HA  THR A 448      -1.424  36.606  33.767  1.0   0.00      PROT H \n",
            "ATOM   3764  HB  THR A 448      -1.646  39.608  34.500  1.0   0.00      PROT H \n",
            "ATOM   3765  HG1 THR A 448      -3.149  38.869  32.714  1.0   0.00      PROT H \n",
            "ATOM   3766 HG21 THR A 448       0.270  39.934  32.966  1.0   0.00      PROT H \n",
            "ATOM   3767 HG22 THR A 448       0.745  38.855  34.300  1.0   0.00      PROT H \n",
            "ATOM   3768 HG23 THR A 448       0.370  38.178  32.697  1.0   0.00      PROT H \n",
            "ATOM   3769  N   GLU A 449      -1.171  37.388  36.854  1.0   0.00      PROT N \n",
            "ATOM   3770  CA  GLU A 449      -0.351  37.237  38.052  1.0   0.00      PROT C \n",
            "ATOM   3771  C   GLU A 449       0.084  35.789  38.265  1.0   0.00      PROT C \n",
            "ATOM   3772  O   GLU A 449       1.126  35.523  38.868  1.0   0.00      PROT O \n",
            "ATOM   3773  CB  GLU A 449      -1.115  37.714  39.293  1.0   0.00      PROT C \n",
            "ATOM   3774  CG  GLU A 449      -1.636  39.134  39.197  1.0   0.00      PROT C \n",
            "ATOM   3775  CD  GLU A 449      -3.044  39.207  38.632  1.0   0.00      PROT C \n",
            "ATOM   3776  OE1 GLU A 449      -3.980  38.757  39.324  1.0   0.00      PROT O \n",
            "ATOM   3777  OE2 GLU A 449      -3.219  39.710  37.501  1.0   0.00      PROT O \n",
            "ATOM   3778  H   GLU A 449      -2.152  37.602  36.964  1.0   0.00      PROT H \n",
            "ATOM   3779  HA  GLU A 449       0.522  37.860  37.857  1.0   0.00      PROT H \n",
            "ATOM   3780  HB3 GLU A 449      -0.432  37.634  40.138  1.0   0.00      PROT H \n",
            "ATOM   3781  HB2 GLU A 449      -1.951  37.029  39.436  1.0   0.00      PROT H \n",
            "ATOM   3782  HG2 GLU A 449      -1.652  39.583  40.190  1.0   0.00      PROT H \n",
            "ATOM   3783  HG3 GLU A 449      -0.985  39.717  38.545  1.0   0.00      PROT H \n",
            "ATOM   3784  HE2 GLU A 449      -4.190  39.688  37.264  1.0   0.00      PROT H \n",
            "ATOM   3785  N   LEU A 450      -0.714  34.849  37.774  1.0   0.00      PROT N \n",
            "ATOM   3786  CA  LEU A 450      -0.386  33.443  37.939  1.0   0.00      PROT C \n",
            "ATOM   3787  C   LEU A 450       0.682  32.921  36.984  1.0   0.00      PROT C \n",
            "ATOM   3788  O   LEU A 450       1.100  31.771  37.108  1.0   0.00      PROT O \n",
            "ATOM   3789  CB  LEU A 450      -1.650  32.588  37.814  1.0   0.00      PROT C \n",
            "ATOM   3790  CG  LEU A 450      -2.588  32.569  39.022  1.0   0.00      PROT C \n",
            "ATOM   3791  CD1 LEU A 450      -3.737  31.605  38.769  1.0   0.00      PROT C \n",
            "ATOM   3792  CD2 LEU A 450      -1.813  32.129  40.251  1.0   0.00      PROT C \n",
            "ATOM   3793  H   LEU A 450      -1.557  35.109  37.281  1.0   0.00      PROT H \n",
            "ATOM   3794  HA  LEU A 450       0.037  33.420  38.943  1.0   0.00      PROT H \n",
            "ATOM   3795  HB3 LEU A 450      -1.303  31.575  37.607  1.0   0.00      PROT H \n",
            "ATOM   3796  HB2 LEU A 450      -2.184  32.974  36.946  1.0   0.00      PROT H \n",
            "ATOM   3797  HG  LEU A 450      -2.994  33.567  39.189  1.0   0.00      PROT H \n",
            "ATOM   3798 HD11 LEU A 450      -4.291  31.924  37.886  1.0   0.00      PROT H \n",
            "ATOM   3799 HD12 LEU A 450      -3.341  30.602  38.607  1.0   0.00      PROT H \n",
            "ATOM   3800 HD13 LEU A 450      -4.402  31.598  39.632  1.0   0.00      PROT H \n",
            "ATOM   3801 HD21 LEU A 450      -0.991  32.822  40.429  1.0   0.00      PROT H \n",
            "ATOM   3802 HD22 LEU A 450      -2.476  32.122  41.116  1.0   0.00      PROT H \n",
            "ATOM   3803 HD23 LEU A 450      -1.415  31.127  40.090  1.0   0.00      PROT H \n",
            "ATOM   3804  N   PHE A 451       1.128  33.754  36.045  1.0   0.00      PROT N \n",
            "ATOM   3805  CA  PHE A 451       2.158  33.343  35.082  1.0   0.00      PROT C \n",
            "ATOM   3806  C   PHE A 451       3.561  33.739  35.514  1.0   0.00      PROT C \n",
            "ATOM   3807  O   PHE A 451       3.855  34.924  35.646  1.0   0.00      PROT O \n",
            "ATOM   3808  CB  PHE A 451       1.913  33.958  33.694  1.0   0.00      PROT C \n",
            "ATOM   3809  CG  PHE A 451       0.670  33.469  33.013  1.0   0.00      PROT C \n",
            "ATOM   3810  CD1 PHE A 451      -0.590  33.800  33.506  1.0   0.00      PROT C \n",
            "ATOM   3811  CD2 PHE A 451       0.758  32.683  31.872  1.0   0.00      PROT C \n",
            "ATOM   3812  CE1 PHE A 451      -1.746  33.346  32.868  1.0   0.00      PROT C \n",
            "ATOM   3813  CE2 PHE A 451      -0.386  32.226  31.229  1.0   0.00      PROT C \n",
            "ATOM   3814  CZ  PHE A 451      -1.640  32.560  31.726  1.0   0.00      PROT C \n",
            "ATOM   3815  H   PHE A 451       0.757  34.691  35.984  1.0   0.00      PROT H \n",
            "ATOM   3816  HA  PHE A 451       2.060  32.257  35.077  1.0   0.00      PROT H \n",
            "ATOM   3817  HB3 PHE A 451       2.782  33.718  33.081  1.0   0.00      PROT H \n",
            "ATOM   3818  HB2 PHE A 451       1.852  35.038  33.830  1.0   0.00      PROT H \n",
            "ATOM   3819  HD1 PHE A 451      -0.681  34.419  34.398  1.0   0.00      PROT H \n",
            "ATOM   3820  HD2 PHE A 451       1.735  32.416  31.468  1.0   0.00      PROT H \n",
            "ATOM   3821  HE1 PHE A 451      -2.729  33.605  33.262  1.0   0.00      PROT H \n",
            "ATOM   3822  HE2 PHE A 451      -0.299  31.607  30.336  1.0   0.00      PROT H \n",
            "ATOM   3823  HZ  PHE A 451      -2.539  32.206  31.221  1.0   0.00      PROT H \n",
            "ATOM   3824  N   PRO A 452       4.446  32.750  35.739  1.0   0.00      PROT N \n",
            "ATOM   3825  CA  PRO A 452       5.835  33.004  36.151  1.0   0.00      PROT C \n",
            "ATOM   3826  C   PRO A 452       6.632  33.669  35.018  1.0   0.00      PROT C \n",
            "ATOM   3827  O   PRO A 452       6.485  33.306  33.850  1.0   0.00      PROT O \n",
            "ATOM   3828  CB  PRO A 452       6.366  31.601  36.459  1.0   0.00      PROT C \n",
            "ATOM   3829  CG  PRO A 452       5.152  30.851  36.875  1.0   0.00      PROT C \n",
            "ATOM   3830  CD  PRO A 452       4.121  31.318  35.877  1.0   0.00      PROT C \n",
            "ATOM   3831  HA  PRO A 452       5.877  33.685  37.001  1.0   0.00      PROT H \n",
            "ATOM   3832  HB3 PRO A 452       7.090  31.602  37.274  1.0   0.00      PROT H \n",
            "ATOM   3833  HB2 PRO A 452       6.806  31.123  35.583  1.0   0.00      PROT H \n",
            "ATOM   3834  HG2 PRO A 452       5.347  29.780  36.812  1.0   0.00      PROT H \n",
            "ATOM   3835  HG3 PRO A 452       4.896  31.114  37.901  1.0   0.00      PROT H \n",
            "ATOM   3836  HD2 PRO A 452       4.239  30.764  34.946  1.0   0.00      PROT H \n",
            "ATOM   3837  HD3 PRO A 452       3.122  31.145  36.278  1.0   0.00      PROT H \n",
            "ATOM   3838  N   PRO A 453       7.497  34.640  35.347  1.0   0.00      PROT N \n",
            "ATOM   3839  CA  PRO A 453       8.277  35.300  34.294  1.0   0.00      PROT C \n",
            "ATOM   3840  C   PRO A 453       9.008  34.348  33.341  1.0   0.00      PROT C \n",
            "ATOM   3841  O   PRO A 453       9.176  34.659  32.159  1.0   0.00      PROT O \n",
            "ATOM   3842  CB  PRO A 453       9.239  36.195  35.083  1.0   0.00      PROT C \n",
            "ATOM   3843  CG  PRO A 453       9.352  35.509  36.399  1.0   0.00      PROT C \n",
            "ATOM   3844  CD  PRO A 453       7.928  35.102  36.676  1.0   0.00      PROT C \n",
            "ATOM   3845  HA  PRO A 453       7.596  35.834  33.631  1.0   0.00      PROT H \n",
            "ATOM   3846  HB3 PRO A 453       8.844  37.200  35.230  1.0   0.00      PROT H \n",
            "ATOM   3847  HB2 PRO A 453      10.224  36.256  34.619  1.0   0.00      PROT H \n",
            "ATOM   3848  HG2 PRO A 453      10.026  34.657  36.308  1.0   0.00      PROT H \n",
            "ATOM   3849  HG3 PRO A 453       9.745  36.206  37.139  1.0   0.00      PROT H \n",
            "ATOM   3850  HD2 PRO A 453       7.915  34.317  37.432  1.0   0.00      PROT H \n",
            "ATOM   3851  HD3 PRO A 453       7.367  35.964  37.037  1.0   0.00      PROT H \n",
            "ATOM   3852  N   LEU A 454       9.439  33.195  33.845  1.0   0.00      PROT N \n",
            "ATOM   3853  CA  LEU A 454      10.144  32.242  33.001  1.0   0.00      PROT C \n",
            "ATOM   3854  C   LEU A 454       9.190  31.634  31.976  1.0   0.00      PROT C \n",
            "ATOM   3855  O   LEU A 454       9.557  31.422  30.820  1.0   0.00      PROT O \n",
            "ATOM   3856  CB  LEU A 454      10.777  31.132  33.853  1.0   0.00      PROT C \n",
            "ATOM   3857  CG  LEU A 454      11.569  30.089  33.054  1.0   0.00      PROT C \n",
            "ATOM   3858  CD1 LEU A 454      12.543  30.775  32.089  1.0   0.00      PROT C \n",
            "ATOM   3859  CD2 LEU A 454      12.303  29.178  34.009  1.0   0.00      PROT C \n",
            "ATOM   3860  H   LEU A 454       9.277  32.977  34.818  1.0   0.00      PROT H \n",
            "ATOM   3861  HA  LEU A 454      10.918  32.819  32.495  1.0   0.00      PROT H \n",
            "ATOM   3862  HB3 LEU A 454       9.964  30.640  34.387  1.0   0.00      PROT H \n",
            "ATOM   3863  HB2 LEU A 454      11.438  31.620  34.569  1.0   0.00      PROT H \n",
            "ATOM   3864  HG  LEU A 454      10.885  29.478  32.465  1.0   0.00      PROT H \n",
            "ATOM   3865 HD11 LEU A 454      11.987  31.425  31.413  1.0   0.00      PROT H \n",
            "ATOM   3866 HD12 LEU A 454      13.259  31.369  32.657  1.0   0.00      PROT H \n",
            "ATOM   3867 HD13 LEU A 454      13.075  30.019  31.511  1.0   0.00      PROT H \n",
            "ATOM   3868 HD21 LEU A 454      11.588  28.705  34.682  1.0   0.00      PROT H \n",
            "ATOM   3869 HD22 LEU A 454      12.833  28.411  33.444  1.0   0.00      PROT H \n",
            "ATOM   3870 HD23 LEU A 454      13.018  29.761  34.590  1.0   0.00      PROT H \n",
            "ATOM   3871  N   PHE A 455       7.964  31.366  32.418  1.0   0.00      PROT N \n",
            "ATOM   3872  CA  PHE A 455       6.921  30.785  31.582  1.0   0.00      PROT C \n",
            "ATOM   3873  C   PHE A 455       6.636  31.736  30.403  1.0   0.00      PROT C \n",
            "ATOM   3874  O   PHE A 455       6.641  31.333  29.238  1.0   0.00      PROT O \n",
            "ATOM   3875  CB  PHE A 455       5.673  30.562  32.456  1.0   0.00      PROT C \n",
            "ATOM   3876  CG  PHE A 455       4.611  29.692  31.829  1.0   0.00      PROT C \n",
            "ATOM   3877  CD1 PHE A 455       4.943  28.705  30.898  1.0   0.00      PROT C \n",
            "ATOM   3878  CD2 PHE A 455       3.276  29.837  32.203  1.0   0.00      PROT C \n",
            "ATOM   3879  CE1 PHE A 455       3.966  27.874  30.347  1.0   0.00      PROT C \n",
            "ATOM   3880  CE2 PHE A 455       2.280  29.008  31.658  1.0   0.00      PROT C \n",
            "ATOM   3881  CZ  PHE A 455       2.633  28.024  30.724  1.0   0.00      PROT C \n",
            "ATOM   3882  H   PHE A 455       7.718  31.566  33.377  1.0   0.00      PROT H \n",
            "ATOM   3883  HA  PHE A 455       7.278  29.831  31.195  1.0   0.00      PROT H \n",
            "ATOM   3884  HB3 PHE A 455       5.257  31.547  32.668  1.0   0.00      PROT H \n",
            "ATOM   3885  HB2 PHE A 455       6.016  30.105  33.384  1.0   0.00      PROT H \n",
            "ATOM   3886  HD1 PHE A 455       5.986  28.590  30.603  1.0   0.00      PROT H \n",
            "ATOM   3887  HD2 PHE A 455       2.993  30.601  32.927  1.0   0.00      PROT H \n",
            "ATOM   3888  HE1 PHE A 455       4.245  27.109  29.623  1.0   0.00      PROT H \n",
            "ATOM   3889  HE2 PHE A 455       1.239  29.127  31.957  1.0   0.00      PROT H \n",
            "ATOM   3890  HZ  PHE A 455       1.869  27.377  30.293  1.0   0.00      PROT H \n",
            "ATOM   3891  N   LEU A 456       6.413  33.007  30.714  1.0   0.00      PROT N \n",
            "ATOM   3892  CA  LEU A 456       6.139  34.007  29.694  1.0   0.00      PROT C \n",
            "ATOM   3893  C   LEU A 456       7.286  34.106  28.697  1.0   0.00      PROT C \n",
            "ATOM   3894  O   LEU A 456       7.074  34.077  27.482  1.0   0.00      PROT O \n",
            "ATOM   3895  CB  LEU A 456       5.908  35.370  30.359  1.0   0.00      PROT C \n",
            "ATOM   3896  CG  LEU A 456       4.617  35.483  31.189  1.0   0.00      PROT C \n",
            "ATOM   3897  CD1 LEU A 456       4.691  36.650  32.163  1.0   0.00      PROT C \n",
            "ATOM   3898  CD2 LEU A 456       3.427  35.622  30.241  1.0   0.00      PROT C \n",
            "ATOM   3899  H   LEU A 456       6.431  33.301  31.680  1.0   0.00      PROT H \n",
            "ATOM   3900  HA  LEU A 456       5.245  33.660  29.177  1.0   0.00      PROT H \n",
            "ATOM   3901  HB2 LEU A 456       6.733  35.588  31.037  1.0   0.00      PROT H \n",
            "ATOM   3902  HB3 LEU A 456       5.853  36.143  29.593  1.0   0.00      PROT H \n",
            "ATOM   3903  HG  LEU A 456       4.485  34.586  31.794  1.0   0.00      PROT H \n",
            "ATOM   3904 HD11 LEU A 456       5.549  36.519  32.822  1.0   0.00      PROT H \n",
            "ATOM   3905 HD12 LEU A 456       4.799  37.581  31.607  1.0   0.00      PROT H \n",
            "ATOM   3906 HD13 LEU A 456       3.778  36.686  32.757  1.0   0.00      PROT H \n",
            "ATOM   3907 HD21 LEU A 456       3.404  34.772  29.559  1.0   0.00      PROT H \n",
            "ATOM   3908 HD22 LEU A 456       2.503  35.649  30.819  1.0   0.00      PROT H \n",
            "ATOM   3909 HD23 LEU A 456       3.524  36.544  29.668  1.0   0.00      PROT H \n",
            "ATOM   3910  N   GLU A 457       8.507  34.201  29.209  1.0   0.00      PROT N \n",
            "ATOM   3911  CA  GLU A 457       9.669  34.332  28.345  1.0   0.00      PROT C \n",
            "ATOM   3912  C   GLU A 457       9.803  33.227  27.307  1.0   0.00      PROT C \n",
            "ATOM   3913  O   GLU A 457       9.981  33.503  26.119  1.0   0.00      PROT O \n",
            "ATOM   3914  CB  GLU A 457      10.946  34.403  29.181  1.0   0.00      PROT C \n",
            "ATOM   3915  CG  GLU A 457      12.158  34.821  28.387  1.0   0.00      PROT C \n",
            "ATOM   3916  CD  GLU A 457      13.405  34.977  29.239  1.0   0.00      PROT C \n",
            "ATOM   3917  OE1 GLU A 457      14.416  35.494  28.713  1.0   0.00      PROT O \n",
            "ATOM   3918  OE2 GLU A 457      13.387  34.582  30.427  1.0   0.00      PROT O \n",
            "ATOM   3919  H   GLU A 457       8.638  34.183  30.210  1.0   0.00      PROT H \n",
            "ATOM   3920  HA  GLU A 457       9.470  35.264  27.816  1.0   0.00      PROT H \n",
            "ATOM   3921  HB3 GLU A 457      11.114  33.414  29.607  1.0   0.00      PROT H \n",
            "ATOM   3922  HB2 GLU A 457      10.773  35.120  29.984  1.0   0.00      PROT H \n",
            "ATOM   3923  HG2 GLU A 457      12.371  34.070  27.626  1.0   0.00      PROT H \n",
            "ATOM   3924  HG3 GLU A 457      11.965  35.780  27.907  1.0   0.00      PROT H \n",
            "ATOM   3925  HE2 GLU A 457      14.276  34.745  30.855  1.0   0.00      PROT H \n",
            "ATOM   3926  N   VAL A 458       9.710  31.978  27.747  1.0   0.00      PROT N \n",
            "ATOM   3927  CA  VAL A 458       9.865  30.848  26.836  1.0   0.00      PROT C \n",
            "ATOM   3928  C   VAL A 458       8.760  30.662  25.789  1.0   0.00      PROT C \n",
            "ATOM   3929  O   VAL A 458       9.041  30.245  24.663  1.0   0.00      PROT O \n",
            "ATOM   3930  CB  VAL A 458      10.044  29.524  27.634  1.0   0.00      PROT C \n",
            "ATOM   3931  CG1 VAL A 458      10.126  28.336  26.692  1.0   0.00      PROT C \n",
            "ATOM   3932  CG2 VAL A 458      11.312  29.591  28.461  1.0   0.00      PROT C \n",
            "ATOM   3933  H   VAL A 458       9.530  31.799  28.725  1.0   0.00      PROT H \n",
            "ATOM   3934  HA  VAL A 458      10.755  31.161  26.291  1.0   0.00      PROT H \n",
            "ATOM   3935  HB  VAL A 458       9.192  29.376  28.298  1.0   0.00      PROT H \n",
            "ATOM   3936 HG11 VAL A 458       9.216  28.283  26.094  1.0   0.00      PROT H \n",
            "ATOM   3937 HG12 VAL A 458      10.987  28.453  26.034  1.0   0.00      PROT H \n",
            "ATOM   3938 HG13 VAL A 458      10.234  27.419  27.272  1.0   0.00      PROT H \n",
            "ATOM   3939 HG21 VAL A 458      11.259  30.442  29.139  1.0   0.00      PROT H \n",
            "ATOM   3940 HG22 VAL A 458      11.418  28.673  29.038  1.0   0.00      PROT H \n",
            "ATOM   3941 HG23 VAL A 458      12.171  29.706  27.800  1.0   0.00      PROT H \n",
            "ATOM   3942  N   PHE A 459       7.516  30.987  26.135  1.0   0.00      PROT N \n",
            "ATOM   3943  CA  PHE A 459       6.407  30.793  25.196  1.0   0.00      PROT C \n",
            "ATOM   3944  C   PHE A 459       5.731  32.056  24.665  1.0   0.00      PROT C \n",
            "ATOM   3945  O   PHE A 459       4.709  31.982  23.982  1.0   0.00      PROT O \n",
            "ATOM   3946  CB  PHE A 459       5.368  29.862  25.830  1.0   0.00      PROT C \n",
            "ATOM   3947  CG  PHE A 459       5.935  28.538  26.245  1.0   0.00      PROT C \n",
            "ATOM   3948  CD1 PHE A 459       5.915  28.141  27.579  1.0   0.00      PROT C \n",
            "ATOM   3949  CD2 PHE A 459       6.537  27.702  25.308  1.0   0.00      PROT C \n",
            "ATOM   3950  CE1 PHE A 459       6.489  26.931  27.976  1.0   0.00      PROT C \n",
            "ATOM   3951  CE2 PHE A 459       7.114  26.490  25.693  1.0   0.00      PROT C \n",
            "ATOM   3952  CZ  PHE A 459       7.091  26.104  27.028  1.0   0.00      PROT C \n",
            "ATOM   3953  H   PHE A 459       7.330  31.371  27.051  1.0   0.00      PROT H \n",
            "ATOM   3954  HA  PHE A 459       6.909  30.357  24.332  1.0   0.00      PROT H \n",
            "ATOM   3955  HB3 PHE A 459       4.577  29.705  25.097  1.0   0.00      PROT H \n",
            "ATOM   3956  HB2 PHE A 459       4.959  30.372  26.702  1.0   0.00      PROT H \n",
            "ATOM   3957  HD1 PHE A 459       5.447  28.777  28.330  1.0   0.00      PROT H \n",
            "ATOM   3958  HD2 PHE A 459       6.562  27.990  24.257  1.0   0.00      PROT H \n",
            "ATOM   3959  HE1 PHE A 459       6.467  26.633  29.024  1.0   0.00      PROT H \n",
            "ATOM   3960  HE2 PHE A 459       7.582  25.847  24.948  1.0   0.00      PROT H \n",
            "ATOM   3961  HZ  PHE A 459       7.542  25.159  27.330  1.0   0.00      PROT H \n",
            "ATOM   3962  N   GLU A 460       6.319  33.210  24.966  1.0   0.00      PROT N \n",
            "ATOM   3963  CA  GLU A 460       5.779  34.493  24.518  1.0   0.00      PROT C \n",
            "ATOM   3964  C   GLU A 460       6.079  34.686  23.039  1.0   0.00      PROT C \n",
            "ATOM   3965  O   GLU A 460       5.117  34.886  22.257  1.0   0.00      PROT O \n",
            "TER    3966      GLU A 460\n",
            "ATOM   3967  CB  GLU A 460       6.415  35.640  25.298  1.0   0.00      PROT C \n",
            "ATOM   3968  CG  GLU A 460       5.542  36.868  25.381  1.0   0.00      PROT C \n",
            "ATOM   3969  CD  GLU A 460       4.963  37.063  26.772  1.0   0.00      PROT C \n",
            "ATOM   3970  OE1 GLU A 460       5.696  37.558  27.662  1.0   0.00      PROT O \n",
            "ATOM   3971  OE2 GLU A 460       3.781  36.707  26.976  1.0   0.00      PROT O \n",
            "ATOM   3972  OXT GLU A 460       7.283  34.643  22.691  1.0   0.00      PROT O \n",
            "ATOM   3973  H   GLU A 460       7.164  33.217  25.519  1.0   0.00      PROT H \n",
            "ATOM   3974  HA  GLU A 460       4.705  34.452  24.701  1.0   0.00      PROT H \n",
            "ATOM   3975  HB3 GLU A 460       7.352  35.893  24.801  1.0   0.00      PROT H \n",
            "ATOM   3976  HB2 GLU A 460       6.624  35.274  26.303  1.0   0.00      PROT H \n",
            "ATOM   3977  HG2 GLU A 460       6.131  37.750  25.131  1.0   0.00      PROT H \n",
            "ATOM   3978  HG3 GLU A 460       4.714  36.773  24.678  1.0   0.00      PROT H \n",
            "HETATM 3979  O3  IH5 A 501       8.258  16.321  33.839  1.0   0.00      PROT O \n",
            "HETATM 3980  C15 IH5 A 501       8.950  17.348  33.292  1.0   0.00      PROT C \n",
            "HETATM 3981  O4  IH5 A 501      10.093  17.690  33.630  1.0   0.00      PROT O \n",
            "HETATM 3982  C13 IH5 A 501       8.252  18.097  32.198  1.0   0.00      PROT C \n",
            "HETATM 3983  C01 IH5 A 501       6.856  18.745  32.328  1.0   0.00      PROT C \n",
            "HETATM 3984  C11 IH5 A 501       5.955  18.402  31.250  1.0   0.00      PROT C \n",
            "HETATM 3985  C09 IH5 A 501       4.638  18.923  31.254  1.0   0.00      PROT C \n",
            "HETATM 3986  C07 IH5 A 501       4.169  19.826  32.328  1.0   0.00      PROT C \n",
            "HETATM 3987  O2  IH5 A 501       2.887  20.267  32.225  1.0   0.00      PROT O \n",
            "HETATM 3988  C2  IH5 A 501       2.596  21.554  31.536  1.0   0.00      PROT C \n",
            "HETATM 3989  C12 IH5 A 501       3.316  22.000  30.314  1.0   0.00      PROT C \n",
            "HETATM 3990  C10 IH5 A 501       3.007  23.220  29.695  1.0   0.00      PROT C \n",
            "HETATM 3991  C8  IH5 A 501       1.986  24.059  30.211  1.0   0.00      PROT C \n",
            "HETATM 3992  O1  IH5 A 501       1.737  25.219  29.560  1.0   0.00      PROT O \n",
            "HETATM 3993  C6  IH5 A 501       1.217  23.699  31.426  1.0   0.00      PROT C \n",
            "HETATM 3994  C4  IH5 A 501       1.542  22.412  32.085  1.0   0.00      PROT C \n",
            "HETATM 3995  C14 IH5 A 501       0.100  24.681  31.931  1.0   0.00      PROT C \n",
            "HETATM 3996  C16 IH5 A 501       0.708  25.497  33.090  1.0   0.00      PROT C \n",
            "HETATM 3997  C18 IH5 A 501      -1.179  23.903  32.434  1.0   0.00      PROT C \n",
            "HETATM 3998  C05 IH5 A 501       5.054  20.165  33.368  1.0   0.00      PROT C \n",
            "HETATM 3999  C03 IH5 A 501       6.395  19.637  33.393  1.0   0.00      PROT C \n",
            "HETATM 4000 CL5  IH5 A 501       4.576  21.219  34.619  1.0   0.00      PROTCL \n",
            "HETATM 4001 CL6  IH5 A 501       3.595  18.466  29.959  1.0   0.00      PROTCL \n",
            "HETATM 4002  H3  IH5 A 501       8.804  15.882  34.553  1.0   0.00      PROT H \n",
            "HETATM 4003 H131 IH5 A 501       8.859  18.951  31.898  1.0   0.00      PROT H \n",
            "HETATM 4004 H132 IH5 A 501       8.105  17.437  31.343  1.0   0.00      PROT H \n",
            "HETATM 4005  H11 IH5 A 501       6.281  17.750  30.440  1.0   0.00      PROT H \n",
            "HETATM 4006  H12 IH5 A 501       4.096  21.361  29.900  1.0   0.00      PROT H \n",
            "HETATM 4007  H10 IH5 A 501       3.554  23.534  28.806  1.0   0.00      PROT H \n",
            "HETATM 4008  H1  IH5 A 501       1.026  25.713  30.009  1.0   0.00      PROT H \n",
            "HETATM 4009  H4  IH5 A 501       0.998  22.103  32.978  1.0   0.00      PROT H \n",
            "HETATM 4010  H14 IH5 A 501      -0.212  25.341  31.122  1.0   0.00      PROT H \n",
            "HETATM 4011 H161 IH5 A 501       1.590  26.031  32.735  1.0   0.00      PROT H \n",
            "HETATM 4012 H162 IH5 A 501      -0.028  26.213  33.455  1.0   0.00      PROT H \n",
            "HETATM 4013 H163 IH5 A 501       0.992  24.824  33.899  1.0   0.00      PROT H \n",
            "HETATM 4014 H181 IH5 A 501      -1.603  23.329  31.610  1.0   0.00      PROT H \n",
            "HETATM 4015 H182 IH5 A 501      -0.899  23.227  33.241  1.0   0.00      PROT H \n",
            "HETATM 4016 H183 IH5 A 501      -1.918  24.616  32.798  1.0   0.00      PROT H \n",
            "HETATM 4017  H03 IH5 A 501       7.070  19.902  34.207  1.0   0.00      PROT H \n",
            "HETATM 4018  O   HOH A 601       7.351  11.237  10.609  1.0   0.00      PROT O \n",
            "HETATM 4019  H1  HOH A 601       7.294  12.018  11.198  1.0   0.00      PROT H \n",
            "HETATM 4020  H2  HOH A 601       6.688  10.690  11.079  1.0   0.00      PROT H \n",
            "HETATM 4021  O   HOH A 602      14.679  23.765  18.704  1.0   0.00      PROT O \n",
            "HETATM 4022  H1  HOH A 602      15.541  23.315  18.584  1.0   0.00      PROT H \n",
            "HETATM 4023  H2  HOH A 602      14.174  23.977  19.517  1.0   0.00      PROT H \n",
            "HETATM 4024  O   HOH A 603       9.781  21.565  14.110  1.0   0.00      PROT O \n",
            "HETATM 4025  H1  HOH A 603       9.743  22.172  13.342  1.0   0.00      PROT H \n",
            "HETATM 4026  H2  HOH A 603       9.224  21.949  14.819  1.0   0.00      PROT H \n",
            "HETATM 4027  O   HOH A 604      -7.727  23.978  30.264  1.0   0.00      PROT O \n",
            "HETATM 4028  H1  HOH A 604      -7.861  23.787  31.216  1.0   0.00      PROT H \n",
            "HETATM 4029  H2  HOH A 604      -8.409  24.674  30.165  1.0   0.00      PROT H \n",
            "HETATM 4030  O   HOH A 605      -6.550  12.266  45.913  1.0   0.00      PROT O \n",
            "HETATM 4031  H1  HOH A 605      -5.572  12.209  45.941  1.0   0.00      PROT H \n",
            "HETATM 4032  H2  HOH A 605      -6.743  12.773  46.729  1.0   0.00      PROT H \n",
            "HETATM 4033  O   HOH A 606       8.965  13.943  34.969  1.0   0.00      PROT O \n",
            "HETATM 4034  H1  HOH A 606       8.248  14.504  34.607  1.0   0.00      PROT H \n",
            "HETATM 4035  H2  HOH A 606       8.453  13.517  35.688  1.0   0.00      PROT H \n",
            "HETATM 4036  O   HOH A 607      12.046  19.291  31.925  1.0   0.00      PROT O \n",
            "HETATM 4037  H1  HOH A 607      12.102  20.259  32.069  1.0   0.00      PROT H \n",
            "HETATM 4038  H2  HOH A 607      11.194  19.218  32.403  1.0   0.00      PROT H \n",
            "HETATM 4039  O   HOH A 608      -5.859   9.748  33.412  1.0   0.00      PROT O \n",
            "HETATM 4040  H1  HOH A 608      -5.035   9.343  33.755  1.0   0.00      PROT H \n",
            "HETATM 4041  H2  HOH A 608      -6.209  10.163  34.228  1.0   0.00      PROT H \n",
            "HETATM 4042  O   HOH A 609       5.288  25.372  22.747  1.0   0.00      PROT O \n",
            "HETATM 4043  H1  HOH A 609       4.800  24.853  22.074  1.0   0.00      PROT H \n",
            "HETATM 4044  H2  HOH A 609       6.081  25.819  23.111  1.0   0.00      PROT H \n",
            "HETATM 4045  O   HOH A 610      -8.897  11.939  34.133  1.0   0.00      PROT O \n",
            "HETATM 4046  H1  HOH A 610      -7.965  11.666  34.267  1.0   0.00      PROT H \n",
            "HETATM 4047  H2  HOH A 610      -9.079  12.449  34.950  1.0   0.00      PROT H \n",
            "HETATM 4048  O   HOH A 611       9.777  32.144  36.552  1.0   0.00      PROT O \n",
            "HETATM 4049  H1  HOH A 611      10.755  32.157  36.606  1.0   0.00      PROT H \n",
            "HETATM 4050  H2  HOH A 611       9.531  32.734  37.295  1.0   0.00      PROT H \n",
            "HETATM 4051  O   HOH A 612      30.241  15.639  22.792  1.0   0.00      PROT O \n",
            "HETATM 4052  H1  HOH A 612      30.045  15.131  23.607  1.0   0.00      PROT H \n",
            "HETATM 4053  H2  HOH A 612      29.848  16.496  23.059  1.0   0.00      PROT H \n",
            "HETATM 4054  O   HOH A 613      -3.343   3.136  30.268  1.0   0.00      PROT O \n",
            "HETATM 4055  H1  HOH A 613      -3.035   4.029  30.529  1.0   0.00      PROT H \n",
            "HETATM 4056  H2  HOH A 613      -4.121   3.008  30.850  1.0   0.00      PROT H \n",
            "HETATM 4057  O   HOH A 614      -5.531   8.325  30.538  1.0   0.00      PROT O \n",
            "HETATM 4058  H1  HOH A 614      -5.630   8.757  31.412  1.0   0.00      PROT H \n",
            "HETATM 4059  H2  HOH A 614      -6.488   8.113  30.541  1.0   0.00      PROT H \n",
            "HETATM 4060  O   HOH A 615      11.054  29.194  18.176  1.0   0.00      PROT O \n",
            "HETATM 4061  H1  HOH A 615      11.938  28.981  18.542  1.0   0.00      PROT H \n",
            "HETATM 4062  H2  HOH A 615      10.807  29.896  18.814  1.0   0.00      PROT H \n",
            "HETATM 4063  O   HOH A 616      -0.083  12.045  20.753  1.0   0.00      PROT O \n",
            "HETATM 4064  H1  HOH A 616      -0.143  12.195  21.720  1.0   0.00      PROT H \n",
            "HETATM 4065  H2  HOH A 616      -0.757  12.717  20.519  1.0   0.00      PROT H \n",
            "HETATM 4066  O   HOH A 617      -6.575  39.862  38.301  1.0   0.00      PROT O \n",
            "HETATM 4067  H1  HOH A 617      -5.727  39.501  38.634  1.0   0.00      PROT H \n",
            "HETATM 4068  H2  HOH A 617      -6.790  40.499  39.014  1.0   0.00      PROT H \n",
            "HETATM 4069  O   HOH A 618       9.354   3.801  26.905  1.0   0.00      PROT O \n",
            "HETATM 4070  H1  HOH A 618       8.657   3.550  26.263  1.0   0.00      PROT H \n",
            "HETATM 4071  H2  HOH A 618       9.421   4.590  27.482  1.0   0.00      PROT H \n",
            "HETATM 4072  O   HOH A 619       0.287  10.752  47.292  1.0   0.00      PROT O \n",
            "HETATM 4073  H1  HOH A 619       1.266  10.699  47.297  1.0   0.00      PROT H \n",
            "HETATM 4074  H2  HOH A 619       0.112  11.290  48.092  1.0   0.00      PROT H \n",
            "HETATM 4075  O   HOH A 620     -11.445  30.005  35.002  1.0   0.00      PROT O \n",
            "HETATM 4076  H1  HOH A 620     -10.996  29.137  34.930  1.0   0.00      PROT H \n",
            "HETATM 4077  H2  HOH A 620     -11.453  30.557  35.812  1.0   0.00      PROT H \n",
            "HETATM 4078  O   HOH A 621      11.453  20.792  41.638  1.0   0.00      PROT O \n",
            "HETATM 4079  H1  HOH A 621      12.431  20.807  41.690  1.0   0.00      PROT H \n",
            "HETATM 4080  H2  HOH A 621      11.207  21.391  42.373  1.0   0.00      PROT H \n",
            "HETATM 4081  O   HOH A 622     -11.275  22.149  44.466  1.0   0.00      PROT O \n",
            "HETATM 4082  H1  HOH A 622     -10.422  22.399  44.053  1.0   0.00      PROT H \n",
            "HETATM 4083  H2  HOH A 622     -11.368  22.630  45.315  1.0   0.00      PROT H \n",
            "HETATM 4084  O   HOH A 623       6.845  23.098  17.755  1.0   0.00      PROT O \n",
            "HETATM 4085  H1  HOH A 623       6.293  22.325  17.515  1.0   0.00      PROT H \n",
            "HETATM 4086  H2  HOH A 623       7.311  22.818  18.570  1.0   0.00      PROT H \n",
            "HETATM 4087  O   HOH A 624       1.282  -2.066  38.500  1.0   0.00      PROT O \n",
            "HETATM 4088  H1  HOH A 624       0.322  -1.959  38.665  1.0   0.00      PROT H \n",
            "HETATM 4089  H2  HOH A 624       1.615  -2.509  39.309  1.0   0.00      PROT H \n",
            "HETATM 4090  O   HOH A 625       3.801  37.497  36.059  1.0   0.00      PROT O \n",
            "HETATM 4091  H1  HOH A 625       3.821  36.530  35.904  1.0   0.00      PROT H \n",
            "HETATM 4092  H2  HOH A 625       3.366  37.624  36.928  1.0   0.00      PROT H \n",
            "HETATM 4093  O   HOH A 626      30.527  25.013  23.662  1.0   0.00      PROT O \n",
            "HETATM 4094  H1  HOH A 626      31.062  24.799  22.869  1.0   0.00      PROT H \n",
            "HETATM 4095  H2  HOH A 626      30.413  25.850  24.158  1.0   0.00      PROT H \n",
            "HETATM 4096  O   HOH A 627       5.207   4.030  30.474  1.0   0.00      PROT O \n",
            "HETATM 4097  H1  HOH A 627       4.742   3.487  29.804  1.0   0.00      PROT H \n",
            "HETATM 4098  H2  HOH A 627       4.707   4.750  30.911  1.0   0.00      PROT H \n",
            "HETATM 4099  O   HOH A 628     -18.469  20.709  33.782  1.0   0.00      PROT O \n",
            "HETATM 4100  H1  HOH A 628     -18.961  21.193  33.086  1.0   0.00      PROT H \n",
            "HETATM 4101  H2  HOH A 628     -18.873  20.735  34.674  1.0   0.00      PROT H \n",
            "HETATM 4102  O   HOH A 629      15.031   9.271  30.872  1.0   0.00      PROT O \n",
            "HETATM 4103  H1  HOH A 629      15.761   8.682  31.157  1.0   0.00      PROT H \n",
            "HETATM 4104  H2  HOH A 629      14.751   9.851  31.610  1.0   0.00      PROT H \n",
            "HETATM 4105  O   HOH A 630      25.543  16.813   7.649  1.0   0.00      PROT O \n",
            "HETATM 4106  H1  HOH A 630      24.577  16.807   7.814  1.0   0.00      PROT H \n",
            "HETATM 4107  H2  HOH A 630      25.683  16.508   8.570  1.0   0.00      PROT H \n",
            "HETATM 4108  O   HOH A 631      26.530  25.970  29.496  1.0   0.00      PROT O \n",
            "HETATM 4109  H1  HOH A 631      26.261  25.079  29.189  1.0   0.00      PROT H \n",
            "HETATM 4110  H2  HOH A 631      26.542  26.242  30.437  1.0   0.00      PROT H \n",
            "HETATM 4111  O   HOH A 632      22.568  35.680  15.419  1.0   0.00      PROT O \n",
            "HETATM 4112  H1  HOH A 632      22.714  35.922  16.357  1.0   0.00      PROT H \n",
            "HETATM 4113  H2  HOH A 632      21.742  36.203  15.350  1.0   0.00      PROT H \n",
            "HETATM 4114  O   HOH A 633      27.145  22.288   7.490  1.0   0.00      PROT O \n",
            "HETATM 4115  H1  HOH A 633      27.198  23.230   7.224  1.0   0.00      PROT H \n",
            "HETATM 4116  H2  HOH A 633      26.698  22.531   8.328  1.0   0.00      PROT H \n",
            "END\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "# Se n√£o tiver arquivo .log √© porque a adi√ß√£o de Hs ocorreu sem maiores problemas\n",
        "!ls -la *.log"
      ],
      "metadata": {
        "id": "ezAQiFLi0RUE",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "920aea69-8213-498b-9b4b-c9ef1e50779d"
      },
      "execution_count": 16,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "-rw-r--r-- 1 root root 445 Dec 10 13:37 1nax_add_H.log\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "# Agora vamos ver como est√£o as cargas.\n",
        "%%file 1nax_charges.mop\n",
        "GEO_DAT=\"1nax_add_H.pdb\" OUTPUT CHARGES HTML\n",
        "Seeing the charges"
      ],
      "metadata": {
        "id": "thKtGMuQ0Rah",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "d3c8b19b-5bf0-44fa-d286-15f4ab124fc1"
      },
      "execution_count": 17,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Writing 1nax_charges.mop\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!mopac 1nax_charges.mop"
      ],
      "metadata": {
        "id": "MHyKgiQp4eje",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "3f5fecad-bb79-4645-ceb9-2364d5ec5d60"
      },
      "execution_count": 18,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "          Tue Dec 10 13:40:35 2024  Job: '1nax_charges' started successfully\n",
            "\n",
            "\n",
            "          MOPAC Job: \"1nax_charges.mop\" ended normally on Dec 10, 2024, at 13:40.\n",
            "\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!cat < 1nax_charges.out"
      ],
      "metadata": {
        "id": "xQ3x4TO04ew-",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "f405af14-b02a-4a86-a395-75907d5df597"
      },
      "execution_count": 19,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            " *******************************************************************************\n",
            " **                                                                           **\n",
            " **                              MOPAC v23.0.3                                **\n",
            " **                                                                           **\n",
            " *******************************************************************************\n",
            " **          Digital Object Identifier (DOI): 10.5281/zenodo.6511958          **\n",
            " **    Visit the DOI location for information on how to cite this program.    **\n",
            " *******************************************************************************\n",
            "\n",
            "                              PM7 CALCULATION RESULTS\n",
            "\n",
            " *******************************************************************************\n",
            " *  CALCULATION DONE:                                Tue Dec 10 13:40:35 2024  *\n",
            " *  CHARGES    - IDENTIFY AND PRINT CHARGED ATOMS\n",
            " *  START_RES  - STARTING RESIDUE NUMBERS DEFINED\n",
            " *  Keyword:     START_RES=(211A 460A-501-601) \n",
            " *  GEO_DAT    - DATA SET GEOMETRY IS IN FILE \"1nax_add_H.pdb\"\n",
            " *  CHAINS     - PDB CHAIN LETTERS EXPLICITLY DEFINED\n",
            " *  Keyword:     CHAINS=(AA) \n",
            " *  T=         - A TIME OF 172800.0 SECONDS REQUESTED\n",
            " *  DUMP=N     - RESTART FILE WRITTEN EVERY 7200.0 SECONDS REQUESTED\n",
            " *  OUTPUT     - MINIMIZE OUTPUT\n",
            " *  HTML       - WRITE HTML SCRIPT TO READ PDB FILE USING JSMOL\n",
            " *******************************************************************************\n",
            " START_RES=(211A 460A-501-601)  CHAINS=(AA) GEO_DAT=\"1nax_add_H.pdb\" OUTPUT CHARGES HTML\n",
            " Seeing the charges\n",
            "\n",
            " \n",
            "\n",
            " General Reference for PM7:\n",
            " \"Optimization of Parameters for Semiempirical Methods VI: More Modifications to the \n",
            " NDDO Approximations and Re-optimization of Parameters\", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)\n",
            " https://link.springer.com/article/10.1007/s00894-012-1667-x\n",
            "\n",
            "           Empirical Formula: C1288 H2078 N330 O401 S16 Cl2  =  4115 atoms\n",
            "\n",
            " \n",
            "\n",
            "   Ion     <--------PDB Label------->   Charge     Distance        PDB label for Salt Bridge   Charge\n",
            "\n",
            "    1     (ATOM    210  NZ  LYS A 223)   +1         3.36         (ATOM    275  OE1 GLU A 227)   -1\n",
            "    2     (ATOM    458  NZ  LYS A 240)   +1         2.55         (ATOM   1774  OD1 ASP A 322)   -1\n",
            "    3     (ATOM   1524  NZ  LYS A 306)   +1         2.88         (ATOM   3972  OXT GLU A 460)   -1\n",
            "    4     (ATOM   2007  NH1 ARG A 338)   +1         2.80         (ATOM   2189  OD1 ASP A 351)   -1\n",
            "    5     (ATOM   2665  NH2 ARG A 383)   +1         2.83         (ATOM   1591  OE1 GLU A 311)   -1\n",
            "    6     (ATOM   3023  ND1 HIS A 405)   +1         2.56         (ATOM    116  OE1 GLU A 218)   -1\n",
            "    7     (ATOM   3117  NE  ARG A 410)   +1         3.55         (ATOM   2411  OD2 ASP A 366)   -1\n",
            "    8     (ATOM   3142  NZ  LYS A 411)   +1         3.23         (ATOM     45  OE2 GLU A 213)   -1\n",
            "    9     (ATOM   3165  NE2 HIS A 412)   +1         2.70         (ATOM   2451  OE2 GLU A 369)   -1\n",
            "   10     (ATOM   3380  NZ  LYS A 424)   +1         3.86         (ATOM   3431  OD2 ASP A 427)   -1\n",
            "   11     (ATOM   3684  NZ  LYS A 443)   +1         2.58         (ATOM   3971  OE2 GLU A 460)   -1\n",
            "\n",
            "                 All other ions\n",
            " \n",
            "    1     (ATOM   2072  NZ  LYS A 342)   +1\n",
            " \n",
            "    1     (ATOM   2251  OD2 ASP A 355)   -1\n",
            "\n",
            "                RESIDUE SEQUENCE IN PROTEIN Chain: A\n",
            "\n",
            "             1     2     3     4     5     6     7     8     9    10\n",
            "      210   LYS   PRO   GLU-  PRO   THR   ASP   GLU   GLU-  TRP   GLU   \n",
            "      220   LEU   ILE   LYS+  THR   VAL   THR   GLU-  ALA   HIS   VAL   \n",
            "      230   ALA   THR   ASN   ALA   GLN   GLY   SER   HIS   TRP   LYS+  \n",
            "      240   GLN   LYS   ARG   LYS   PHE   LEU   PRO   GLU   ASP   ILE   \n",
            "      250   GLY   GLN   ALA   PRO   ILE   VAL   ASN   ALA   PRO   GLU   \n",
            "      260   GLY   GLY   LYS   VAL   ASP   LEU   GLU   ALA   PHE   SER   \n",
            "      270   HIS   PHE   THR   LYS   ILE   ILE   THR   PRO   ALA   ILE   \n",
            "      280   THR   ARG   VAL   VAL   ASP   PHE   ALA   LYS   LYS   LEU   \n",
            "      290   PRO   MET   PHE   CYS   GLU   LEU   PRO   CYS   GLU   ASP   \n",
            "      300   GLN   ILE   ILE   LEU   LEU   LYS+  GLY   CYS   CYS   MET   \n",
            "      310   GLU-  ILE   MET   SER   LEU   ARG   ALA   ALA   VAL   ARG   \n",
            "      320   TYR   ASP-  PRO   GLU   SER   GLU   THR   LEU   THR   LEU   \n",
            "      330   ASN   GLY   GLU   MET   ALA   VAL   THR   ARG+  GLY   GLN   \n",
            "      340   LEU   LYS+  ASN   GLY   GLY   LEU   GLY   VAL   VAL   SER   \n",
            "      350   ASP-  ALA   ILE   PHE   ASP-  LEU   GLY   MET   SER   LEU   \n",
            "      360   SER   SER   PHE   ASN   LEU   ASP-  ASP   THR   GLU-  VAL   \n",
            "      370   ALA   LEU   LEU   GLN   ALA   VAL   LEU   LEU   MET   SER   \n",
            "      380   SER   ASP   ARG+  PRO   GLY   LEU   ALA   CYS   VAL   GLU   \n",
            "      390   ARG   ILE   GLU   LYS   TYR   GLN   ASP   SER   PHE   LEU   \n",
            "      400   LEU   ALA   PHE   GLU   HIS+  TYR   ILE   ASN   TYR   ARG+  \n",
            "      410   LYS+  HIS+  HIS   VAL   THR   HIS   PHE   TRP   PRO   LYS   \n",
            "      420   LEU   LEU   MET   LYS+  VAL   THR   ASP-  LEU   ARG   MET   \n",
            "      430   ILE   GLY   ALA   CYS   HIS   ALA   SER   ARG   PHE   LEU   \n",
            "      440   HIS   MET   LYS+  VAL   GLU   CYS   PRO   THR   GLU   LEU   \n",
            "      450   PHE   PRO   PRO   LEU   PHE   LEU   GLU   VAL   PHE   GLU   \n",
            "\n",
            "                RESIDUE SEQUENCE IN PROTEIN Chain: A\n",
            "\n",
            "      210  KPEPTDEEWE LIKTVTEAHV ATNAQGSHWK QKRKFLPEDI GQAPIVNAPE\n",
            "      260  GGKVDLEAFS HFTKIITPAI TRVVDFAKKL PMFCELPCED QIILLKGCCM\n",
            "      310  EIMSLRAAVR YDPESETLTL NGEMAVTRGQ LKNGGLGVVS DAIFDLGMSL\n",
            "      360  SSFNLDDTEV ALLQAVLLMS SDRPGLACVE RIEKYQDSFL LAFEHYINYR\n",
            "      410  KHHVTHFWPK LLMKVTDLRM IGACHASRFL HMKVECPTEL FPPLFLEVFE\n",
            "\n",
            "          JOB STOPPED HERE BECAUSE KEYWORD \"CHARGES\" WAS USED\n",
            "\n",
            " ******************************************************************************\n",
            " *                                                                            *\n",
            " *     Error and normal termination messages reported in this calculation     *\n",
            " *                                                                            *\n",
            " * JOB STOPPED HERE BECAUSE KEYWORD \"CHARGES\" WAS USED                        *\n",
            " * JOB ENDED NORMALLY                                                         *\n",
            " *                                                                            *\n",
            " ******************************************************************************\n",
            "\n",
            "\n",
            "\n",
            " TOTAL JOB TIME:             0.00 SECONDS\n",
            "\n",
            " == MOPAC DONE ==\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!head -n 50 1nax_charges.pdb"
      ],
      "metadata": {
        "id": "CgiZ6eGX4e4P",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "3de7f44c-4017-4453-a402-276446a3a73b"
      },
      "execution_count": 21,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "HEADER  data-set: 1nax_add_H\n",
            "REMARK  MOPAC, Version: 23.0.3\n",
            "REMARK  Date: Dec 10 2024 13:37\n",
            "ATOM      1  N   LYS A 211      23.762  -3.506   9.387  1.0   0.00      PROT N \n",
            "ATOM      2  CA  LYS A 211      23.345  -2.328  10.202  1.0   0.00      PROT C \n",
            "ATOM      3  C   LYS A 211      24.494  -1.837  11.070  1.0   0.00      PROT C \n",
            "ATOM      4  O   LYS A 211      24.771  -2.400  12.129  1.0   0.00      PROT O \n",
            "ATOM      5  CB  LYS A 211      22.153  -2.690  11.086  1.0   0.00      PROT C \n",
            "ATOM      6  CG  LYS A 211      20.884  -2.996  10.315  1.0   0.00      PROT C \n",
            "ATOM      7  CD  LYS A 211      19.698  -3.131  11.256  1.0   0.00      PROT C \n",
            "ATOM      8  CE  LYS A 211      18.388  -3.267  10.493  1.0   0.00      PROT C \n",
            "ATOM      9  NZ  LYS A 211      18.357  -4.488   9.634  1.0   0.00      PROT N \n",
            "ATOM     10  H1  LYS A 211      23.748  -3.452   8.369  1.0   0.00      PROT H \n",
            "ATOM     11  H2  LYS A 211      24.705  -3.884   9.482  1.0   0.00      PROT H \n",
            "ATOM     12  HA  LYS A 211      23.065  -1.542   9.500  1.0   0.00      PROT H \n",
            "ATOM     13  HB3 LYS A 211      21.978  -1.845  11.752  1.0   0.00      PROT H \n",
            "ATOM     14  HB2 LYS A 211      22.441  -3.562  11.673  1.0   0.00      PROT H \n",
            "ATOM     15  HG2 LYS A 211      20.683  -2.189   9.611  1.0   0.00      PROT H \n",
            "ATOM     16  HG3 LYS A 211      21.007  -3.932   9.769  1.0   0.00      PROT H \n",
            "ATOM     17  HD2 LYS A 211      19.827  -4.017  11.878  1.0   0.00      PROT H \n",
            "ATOM     18  HD3 LYS A 211      19.635  -2.247  11.891  1.0   0.00      PROT H \n",
            "ATOM     19  HE2 LYS A 211      17.561  -3.330  11.200  1.0   0.00      PROT H \n",
            "ATOM     20  HE3 LYS A 211      18.250  -2.398   9.850  1.0   0.00      PROT H \n",
            "ATOM     21  HZ1 LYS A 211      17.592  -5.082   9.920  1.0   0.00      PROT H \n",
            "ATOM     22  HZ2 LYS A 211      18.231  -4.218   8.669  1.0   0.00      PROT H \n",
            "ATOM     23  N   PRO A 212      25.169  -0.763  10.634  1.0   0.00      PROT N \n",
            "ATOM     24  CA  PRO A 212      26.301  -0.177  11.356  1.0   0.00      PROT C \n",
            "ATOM     25  C   PRO A 212      25.986   0.277  12.776  1.0   0.00      PROT C \n",
            "ATOM     26  O   PRO A 212      24.950   0.896  13.030  1.0   0.00      PROT O \n",
            "ATOM     27  CB  PRO A 212      26.709   0.991  10.464  1.0   0.00      PROT C \n",
            "ATOM     28  CG  PRO A 212      25.408   1.404   9.853  1.0   0.00      PROT C \n",
            "ATOM     29  CD  PRO A 212      24.792   0.081   9.486  1.0   0.00      PROT C \n",
            "ATOM     30  HA  PRO A 212      27.066  -0.940  11.499  1.0   0.00      PROT H \n",
            "ATOM     31  HB3 PRO A 212      27.404   0.699   9.677  1.0   0.00      PROT H \n",
            "ATOM     32  HB2 PRO A 212      27.128   1.826  11.026  1.0   0.00      PROT H \n",
            "ATOM     33  HG2 PRO A 212      24.821   1.957  10.587  1.0   0.00      PROT H \n",
            "ATOM     34  HG3 PRO A 212      25.598   2.039   8.987  1.0   0.00      PROT H \n",
            "ATOM     35  HD2 PRO A 212      23.714   0.199   9.379  1.0   0.00      PROT H \n",
            "ATOM     36  HD3 PRO A 212      25.214  -0.267   8.543  1.0   0.00      PROT H \n",
            "ATOM     37  N   GLU A 213      26.881  -0.043  13.704  1.0   0.00      PROT N \n",
            "ATOM     38  CA  GLU A 213      26.718   0.370  15.087  1.0   0.00      PROT C \n",
            "ATOM     39  C   GLU A 213      27.501   1.670  15.263  1.0   0.00      PROT C \n",
            "ATOM     40  O   GLU A 213      28.323   2.025  14.419  1.0   0.00      PROT O \n",
            "ATOM     41  CB  GLU A 213      27.229  -0.720  16.029  1.0   0.00      PROT C \n",
            "ATOM     42  CG  GLU A 213      26.197  -1.801  16.306  1.0   0.00      PROT C \n",
            "ATOM     43  CD  GLU A 213      26.719  -2.919  17.199  1.0   0.00      PROT C \n",
            "ATOM     44  OE1 GLU A 213      27.627  -2.664  18.021  1.0   0.00      PROT O \n",
            "ATOM     45  OE2 GLU A 213      26.203  -4.053  17.091  1.0   0.00      PROT O \n",
            "ATOM     46  H   GLU A 213      27.696  -0.585  13.454  1.0   0.00      PROT H \n",
            "ATOM     47  HA  GLU A 213      25.656   0.518  15.280  1.0   0.00      PROT H \n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "%%bash\n",
        "cp 1nax_charges.pdb 1nax_1scf.mop\n",
        "sed -i 's/HEADER  data-set: 1nax_add_H/PM7 PDB 1SCF MOZYME EPS=78.4 CUTOFF=9.0/g' 1nax_1scf.mop\n",
        "mopac 1nax_1scf"
      ],
      "metadata": {
        "id": "eVAz7Py395Lm",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "35ce29dc-5252-4f2d-a479-fad59da0f79d"
      },
      "execution_count": 22,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\n",
            "\n",
            "          MOPAC Job: \"1nax_1scf.mop\" ended normally on Dec 10, 2024, at 14:21.\n",
            "\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "          Tue Dec 10 13:42:50 2024  Job: '1nax_1scf' started successfully\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#!head -n 200 1nax_1scf.arc # O c√°lculo leva cerca de 40min nessa plataforma."
      ],
      "metadata": {
        "id": "w-EnhIWTHhE9",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "2bbf9102-8ee4-4a15-b6e8-0c89950b2b58"
      },
      "execution_count": 24,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\n",
            "\n",
            "                     SUMMARY OF  PM7 CALCULATION\n",
            "\n",
            "                                                       MOPAC v23.0.3 Linux\n",
            "                                                       Tue Dec 10 14:21:10 2024\n",
            "\n",
            "           Empirical Formula: C1288 H2078 N330 O401 S16 Cl2  =  4115 atoms\n",
            "\n",
            " START_RES=(211A 460A-501-601)  CHAINS=(AA) PM7 PDB 1SCF MOZYME EPS=78.4 CUTOFF=9.0\n",
            " REMARK  MOPAC, Version: 23.0.3\n",
            " REMARK  Date: Dec 10 2024 13:37\n",
            "\n",
            "\n",
            "     1SCF WAS SPECIFIED, SO BFGS WAS NOT USED                 \n",
            "     SCF FIELD WAS ACHIEVED                                   \n",
            "\n",
            "          HEAT OF FORMATION       =     -21501.67237 KCAL/MOL =  -89962.99718 KJ/MOL\n",
            "          DIELECTRIC ENERGY       =        -70.86032 EV\n",
            "          DIPOLE                  =         96.30983 DEBYE   \n",
            "          NO. OF FILLED LEVELS    =       5698\n",
            "          MOLECULAR WEIGHT        =      29186.4206\n",
            "          COSMO AREA              =       9882.69 SQUARE ANGSTROMS\n",
            "          COSMO VOLUME            =      35481.23 CUBIC ANGSTROMS\n",
            "\n",
            "          MOLECULAR DIMENSIONS (Angstroms)\n",
            "\n",
            "            Atom       Atom       Distance\n",
            "            H   720    O    87    67.32769\n",
            "            O  1402    H  3190    50.35404\n",
            "            H   553    H  2829    39.30835\n",
            "          SCF CALCULATIONS        =          1\n",
            "          WALL-CLOCK TIME         =         38 MINUTES AND 19.676 SECONDS\n",
            "          COMPUTATION TIME        =         37 MINUTES AND 31.869 SECONDS\n",
            "\n",
            "\n",
            "          FINAL GEOMETRY OBTAINED\n",
            " START_RES=(211A 460A-501-601)  CHAINS=(AA) PM7 PDB 1SCF MOZYME EPS=78.4 CUTOFF=9.0\n",
            " REMARK  MOPAC, Version: 23.0.3\n",
            " REMARK  Date: Dec 10 2024 13:37\n",
            "  N(ATOM      1  N   LYS A 211)  23.76200000 +1  -3.50600000 +1   9.38700000 +1\n",
            "  C(ATOM      2  CA  LYS A 211)  23.34500000 +1  -2.32800000 +1  10.20200000 +1\n",
            "  C(ATOM      3  C   LYS A 211)  24.49400000 +1  -1.83700000 +1  11.07000000 +1\n",
            "  O(ATOM      4  O   LYS A 211)  24.77100000 +1  -2.40000000 +1  12.12900000 +1\n",
            "  C(ATOM      5  CB  LYS A 211)  22.15300000 +1  -2.69000000 +1  11.08600000 +1\n",
            "  C(ATOM      6  CG  LYS A 211)  20.88400000 +1  -2.99600000 +1  10.31500000 +1\n",
            "  C(ATOM      7  CD  LYS A 211)  19.69800000 +1  -3.13100000 +1  11.25600000 +1\n",
            "  C(ATOM      8  CE  LYS A 211)  18.38800000 +1  -3.26700000 +1  10.49300000 +1\n",
            "  N(ATOM      9  NZ  LYS A 211)  18.35700000 +1  -4.48800000 +1   9.63400000 +1\n",
            "  H(ATOM     10  H1  LYS A 211)  23.74800000 +1  -3.45200000 +1   8.36900000 +1\n",
            "  H(ATOM     11  H2  LYS A 211)  24.70500000 +1  -3.88400000 +1   9.48200000 +1\n",
            "  H(ATOM     12  HA  LYS A 211)  23.06500000 +1  -1.54200000 +1   9.50000000 +1\n",
            "  H(ATOM     13  HB3 LYS A 211)  21.97800000 +1  -1.84500000 +1  11.75200000 +1\n",
            "  H(ATOM     14  HB2 LYS A 211)  22.44100000 +1  -3.56200000 +1  11.67300000 +1\n",
            "  H(ATOM     15  HG2 LYS A 211)  20.68300000 +1  -2.18900000 +1   9.61100000 +1\n",
            "  H(ATOM     16  HG3 LYS A 211)  21.00700000 +1  -3.93200000 +1   9.76900000 +1\n",
            "  H(ATOM     17  HD2 LYS A 211)  19.82700000 +1  -4.01700000 +1  11.87800000 +1\n",
            "  H(ATOM     18  HD3 LYS A 211)  19.63500000 +1  -2.24700000 +1  11.89100000 +1\n",
            "  H(ATOM     19  HE2 LYS A 211)  17.56100000 +1  -3.33000000 +1  11.20000000 +1\n",
            "  H(ATOM     20  HE3 LYS A 211)  18.25000000 +1  -2.39800000 +1   9.85000000 +1\n",
            "  H(ATOM     21  HZ1 LYS A 211)  17.59200000 +1  -5.08200000 +1   9.92000000 +1\n",
            "  H(ATOM     22  HZ2 LYS A 211)  18.23100000 +1  -4.21800000 +1   8.66900000 +1\n",
            "  N(ATOM     23  N   PRO A 212)  25.16900000 +1  -0.76300000 +1  10.63400000 +1\n",
            "  C(ATOM     24  CA  PRO A 212)  26.30100000 +1  -0.17700000 +1  11.35600000 +1\n",
            "  C(ATOM     25  C   PRO A 212)  25.98600000 +1   0.27700000 +1  12.77600000 +1\n",
            "  O(ATOM     26  O   PRO A 212)  24.95000000 +1   0.89600000 +1  13.03000000 +1\n",
            "  C(ATOM     27  CB  PRO A 212)  26.70900000 +1   0.99100000 +1  10.46400000 +1\n",
            "  C(ATOM     28  CG  PRO A 212)  25.40800000 +1   1.40400000 +1   9.85300000 +1\n",
            "  C(ATOM     29  CD  PRO A 212)  24.79200000 +1   0.08100000 +1   9.48600000 +1\n",
            "  H(ATOM     30  HA  PRO A 212)  27.06600000 +1  -0.94000000 +1  11.49900000 +1\n",
            "  H(ATOM     31  HB3 PRO A 212)  27.40400000 +1   0.69900000 +1   9.67700000 +1\n",
            "  H(ATOM     32  HB2 PRO A 212)  27.12800000 +1   1.82600000 +1  11.02600000 +1\n",
            "  H(ATOM     33  HG2 PRO A 212)  24.82100000 +1   1.95700000 +1  10.58700000 +1\n",
            "  H(ATOM     34  HG3 PRO A 212)  25.59800000 +1   2.03900000 +1   8.98700000 +1\n",
            "  H(ATOM     35  HD2 PRO A 212)  23.71400000 +1   0.19900000 +1   9.37900000 +1\n",
            "  H(ATOM     36  HD3 PRO A 212)  25.21400000 +1  -0.26700000 +1   8.54300000 +1\n",
            "  N(ATOM     37  N   GLU A 213)  26.88100000 +1  -0.04300000 +1  13.70400000 +1\n",
            "  C(ATOM     38  CA  GLU A 213)  26.71800000 +1   0.37000000 +1  15.08700000 +1\n",
            "  C(ATOM     39  C   GLU A 213)  27.50100000 +1   1.67000000 +1  15.26300000 +1\n",
            "  O(ATOM     40  O   GLU A 213)  28.32300000 +1   2.02500000 +1  14.41900000 +1\n",
            "  C(ATOM     41  CB  GLU A 213)  27.22900000 +1  -0.72000000 +1  16.02900000 +1\n",
            "  C(ATOM     42  CG  GLU A 213)  26.19700000 +1  -1.80100000 +1  16.30600000 +1\n",
            "  C(ATOM     43  CD  GLU A 213)  26.71900000 +1  -2.91900000 +1  17.19900000 +1\n",
            "  O(ATOM     44  OE1 GLU A 213)  27.62700000 +1  -2.66400000 +1  18.02100000 +1\n",
            "  O(ATOM     45  OE2 GLU A 213)  26.20300000 +1  -4.05300000 +1  17.09100000 +1\n",
            "  H(ATOM     46  H   GLU A 213)  27.69600000 +1  -0.58500000 +1  13.45400000 +1\n",
            "  H(ATOM     47  HA  GLU A 213)  25.65600000 +1   0.51800000 +1  15.28000000 +1\n",
            "  H(ATOM     48  HB3 GLU A 213)  27.51300000 +1  -0.24000000 +1  16.96600000 +1\n",
            "  H(ATOM     49  HB2 GLU A 213)  28.11000000 +1  -1.16800000 +1  15.56900000 +1\n",
            "  H(ATOM     50  HG2 GLU A 213)  25.33300000 +1  -1.36000000 +1  16.80400000 +1\n",
            "  H(ATOM     51  HG3 GLU A 213)  25.88200000 +1  -2.25300000 +1  15.36500000 +1\n",
            "  N(ATOM     52  N   PRO A 214)  27.26800000 +1   2.39900000 +1  16.36400000 +1\n",
            "  C(ATOM     53  CA  PRO A 214)  27.99000000 +1   3.66000000 +1  16.56600000 +1\n",
            "  C(ATOM     54  C   PRO A 214)  29.52300000 +1   3.60400000 +1  16.51700000 +1\n",
            "  O(ATOM     55  O   PRO A 214)  30.15100000 +1   2.67400000 +1  17.02800000 +1\n",
            "  C(ATOM     56  CB  PRO A 214)  27.45300000 +1   4.14800000 +1  17.91300000 +1\n",
            "  C(ATOM     57  CG  PRO A 214)  27.11000000 +1   2.88700000 +1  18.61800000 +1\n",
            "  C(ATOM     58  CD  PRO A 214)  26.45800000 +1   2.06400000 +1  17.54900000 +1\n",
            "  H(ATOM     59  HA  PRO A 214)  27.77700000 +1   4.30500000 +1  15.71300000 +1\n",
            "  H(ATOM     60  HB3 PRO A 214)  26.55800000 +1   4.76100000 +1  17.80800000 +1\n",
            "  H(ATOM     61  HB2 PRO A 214)  28.20100000 +1   4.69400000 +1  18.48800000 +1\n",
            "  H(ATOM     62  HG2 PRO A 214)  28.02000000 +1   2.42700000 +1  19.00200000 +1\n",
            "  H(ATOM     63  HG3 PRO A 214)  26.43600000 +1   3.10600000 +1  19.44600000 +1\n",
            "  H(ATOM     64  HD2 PRO A 214)  26.51300000 +1   1.00900000 +1  17.81800000 +1\n",
            "  H(ATOM     65  HD3 PRO A 214)  25.41400000 +1   2.36000000 +1  17.44900000 +1\n",
            "  N(ATOM     66  N   THR A 215)  30.11100000 +1   4.61500000 +1  15.88400000 +1\n",
            "  C(ATOM     67  CA  THR A 215)  31.55900000 +1   4.72800000 +1  15.76700000 +1\n",
            "  C(ATOM     68  C   THR A 215)  32.13700000 +1   5.19200000 +1  17.10700000 +1\n",
            "  O(ATOM     69  O   THR A 215)  31.42100000 +1   5.29600000 +1  18.09500000 +1\n",
            "  C(ATOM     70  CB  THR A 215)  31.94400000 +1   5.75500000 +1  14.69300000 +1\n",
            "  O(ATOM     71  OG1 THR A 215)  31.54200000 +1   7.06200000 +1  15.11900000 +1\n",
            "  C(ATOM     72  CG2 THR A 215)  31.25800000 +1   5.43100000 +1  13.37800000 +1\n",
            "  H(ATOM     73  H   THR A 215)  29.55000000 +1   5.34100000 +1  15.46200000 +1\n",
            "  H(ATOM     74  HA  THR A 215)  31.90900000 +1   3.73300000 +1  15.49100000 +1\n",
            "  H(ATOM     75  HB  THR A 215)  33.02500000 +1   5.78700000 +1  14.55700000 +1\n",
            "  H(ATOM     76  HG1 THR A 215)  31.89600000 +1   7.43800000 +1  15.96400000 +1\n",
            "  H(ATOM     77 HG21 THR A 215)  31.54200000 +1   6.16900000 +1  12.62800000 +1\n",
            "  H(ATOM     78 HG22 THR A 215)  31.56200000 +1   4.43900000 +1  13.04400000 +1\n",
            "  H(ATOM     79 HG23 THR A 215)  30.17700000 +1   5.45200000 +1  13.51700000 +1\n",
            "  N(ATOM     80  N   ASP A 216)  33.43300000 +1   5.46500000 +1  17.14500000 +1\n",
            "  C(ATOM     81  CA  ASP A 216)  34.04800000 +1   5.92300000 +1  18.38200000 +1\n",
            "  C(ATOM     82  C   ASP A 216)  33.61700000 +1   7.35300000 +1  18.67200000 +1\n",
            "  O(ATOM     83  O   ASP A 216)  33.31900000 +1   7.70500000 +1  19.81400000 +1\n",
            "  C(ATOM     84  CB  ASP A 216)  35.57500000 +1   5.87100000 +1  18.28500000 +1\n",
            "  C(ATOM     85  CG  ASP A 216)  36.10500000 +1   4.45900000 +1  18.20600000 +1\n",
            "  O(ATOM     86  OD1 ASP A 216)  35.71700000 +1   3.63300000 +1  19.06300000 +1\n",
            "  O(ATOM     87  OD2 ASP A 216)  36.91500000 +1   4.17800000 +1  17.29400000 +1\n",
            "  H(ATOM     88  H   ASP A 216)  33.99700000 +1   5.35500000 +1  16.31400000 +1\n",
            "  H(ATOM     89  HA  ASP A 216)  33.69200000 +1   5.24200000 +1  19.15500000 +1\n",
            "  H(ATOM     90  HB3 ASP A 216)  35.98300000 +1   6.36300000 +1  19.16800000 +1\n",
            "  H(ATOM     91  HB2 ASP A 216)  35.87200000 +1   6.42100000 +1  17.39200000 +1\n",
            "  H(ATOM     92  HD1 ASP A 216)  36.14700000 +1   2.74500000 +1  18.90200000 +1\n",
            "  N(ATOM     93  N   GLU A 217)  33.59300000 +1   8.17400000 +1  17.62800000 +1\n",
            "  C(ATOM     94  CA  GLU A 217)  33.21100000 +1   9.56500000 +1  17.76800000 +1\n",
            "  C(ATOM     95  C   GLU A 217)  31.74600000 +1   9.65900000 +1  18.19400000 +1\n",
            "  O(ATOM     96  O   GLU A 217)  31.38400000 +1  10.49500000 +1  19.02400000 +1\n",
            "  C(ATOM     97  CB  GLU A 217)  33.44100000 +1  10.30700000 +1  16.44800000 +1\n",
            "  C(ATOM     98  CG  GLU A 217)  34.10200000 +1  11.66800000 +1  16.62400000 +1\n",
            "  C(ATOM     99  CD  GLU A 217)  34.41600000 +1  12.35800000 +1  15.30600000 +1\n",
            "  O(ATOM    100  OE1 GLU A 217)  35.16800000 +1  11.77700000 +1  14.49400000 +1\n",
            "  O(ATOM    101  OE2 GLU A 217)  33.91600000 +1  13.48600000 +1  15.08700000 +1\n",
            "  H(ATOM    102  H   GLU A 217)  33.84500000 +1   7.83300000 +1  16.71100000 +1\n",
            "  H(ATOM    103  HA  GLU A 217)  33.85200000 +1   9.98200000 +1  18.54500000 +1\n",
            "  H(ATOM    104  HB3 GLU A 217)  32.46900000 +1  10.43400000 +1  15.97100000 +1\n",
            "  H(ATOM    105  HB2 GLU A 217)  34.07200000 +1   9.67500000 +1  15.82400000 +1\n",
            "  H(ATOM    106  HG3 GLU A 217)  35.02500000 +1  11.51500000 +1  17.18300000 +1\n",
            "  H(ATOM    107  HG2 GLU A 217)  33.42300000 +1  12.29000000 +1  17.20700000 +1\n",
            "  H(ATOM    108  HE2 GLU A 217)  34.21000000 +1  13.82300000 +1  14.19200000 +1\n",
            "  N(ATOM    109  N   GLU A 218)  30.90600000 +1   8.79200000 +1  17.63600000 +1\n",
            "  C(ATOM    110  CA  GLU A 218)  29.49000000 +1   8.78600000 +1  17.97600000 +1\n",
            "  C(ATOM    111  C   GLU A 218)  29.25100000 +1   8.33700000 +1  19.41600000 +1\n",
            "  O(ATOM    112  O   GLU A 218)  28.32300000 +1   8.81000000 +1  20.06600000 +1\n",
            "  C(ATOM    113  CB  GLU A 218)  28.72100000 +1   7.88400000 +1  17.01500000 +1\n",
            "  C(ATOM    114  CG  GLU A 218)  28.69000000 +1   8.39800000 +1  15.58700000 +1\n",
            "  C(ATOM    115  CD  GLU A 218)  27.89500000 +1   7.49400000 +1  14.66000000 +1\n",
            "  O(ATOM    116  OE1 GLU A 218)  28.17900000 +1   6.27700000 +1  14.64200000 +1\n",
            "  O(ATOM    117  OE2 GLU A 218)  26.99500000 +1   7.99700000 +1  13.94700000 +1\n",
            "  H(ATOM    118  H   GLU A 218)  31.24700000 +1   8.12100000 +1  16.96200000 +1\n",
            "  H(ATOM    119  HA  GLU A 218)  29.17500000 +1   9.82400000 +1  17.86900000 +1\n",
            "  H(ATOM    120  HB3 GLU A 218)  27.70200000 +1   7.80100000 +1  17.39300000 +1\n",
            "  H(ATOM    121  HB2 GLU A 218)  29.19800000 +1   6.90400000 +1  17.03800000 +1\n",
            "  H(ATOM    122  HG2 GLU A 218)  28.23100000 +1   9.38600000 +1  15.56700000 +1\n",
            "  H(ATOM    123  HG3 GLU A 218)  29.70800000 +1   8.46200000 +1  15.20200000 +1\n",
            "  N(ATOM    124  N   TRP A 219)  30.09100000 +1   7.43000000 +1  19.91100000 +1\n",
            "  C(ATOM    125  CA  TRP A 219)  29.97200000 +1   6.93100000 +1  21.28000000 +1\n",
            "  C(ATOM    126  C   TRP A 219)  30.27000000 +1   8.03800000 +1  22.27200000 +1\n",
            "  O(ATOM    127  O   TRP A 219)  29.67800000 +1   8.09400000 +1  23.35200000 +1\n",
            "  C(ATOM    128  CB  TRP A 219)  30.93400000 +1   5.76800000 +1  21.52900000 +1\n",
            "  C(ATOM    129  CG  TRP A 219)  30.34800000 +1   4.41100000 +1  21.26600000 +1\n",
            "  C(ATOM    130  CD1 TRP A 219)  30.89900000 +1   3.41200000 +1  20.51500000 +1\n",
            "  C(ATOM    131  CD2 TRP A 219)  29.11200000 +1   3.89400000 +1  21.77500000 +1\n",
            "  N(ATOM    132  NE1 TRP A 219)  30.08400000 +1   2.30700000 +1  20.52300000 +1\n",
            "  C(ATOM    133  CE2 TRP A 219)  28.98200000 +1   2.57400000 +1  21.29000000 +1\n",
            "  C(ATOM    134  CE3 TRP A 219)  28.10300000 +1   4.41700000 +1  22.59700000 +1\n",
            "  C(ATOM    135  CZ2 TRP A 219)  27.88000000 +1   1.76700000 +1  21.59900000 +1\n",
            "  C(ATOM    136  CZ3 TRP A 219)  27.00500000 +1   3.61000000 +1  22.90700000 +1\n",
            "  C(ATOM    137  CH2 TRP A 219)  26.90700000 +1   2.30000000 +1  22.40700000 +1\n",
            "  H(ATOM    138  H   TRP A 219)  30.83800000 +1   7.06600000 +1  19.33600000 +1\n",
            "  H(ATOM    139  HA  TRP A 219)  28.93800000 +1   6.59900000 +1  21.37600000 +1\n",
            "  H(ATOM    140  HB3 TRP A 219)  31.24800000 +1   5.83300000 +1  22.57100000 +1\n",
            "  H(ATOM    141  HB2 TRP A 219)  31.79600000 +1   5.92400000 +1  20.88000000 +1\n",
            "  H(ATOM    142  HD1 TRP A 219)  31.85600000 +1   3.56800000 +1  20.01800000 +1\n",
            "  H(ATOM    143  HE1 TRP A 219)  30.18300000 +1   1.40600000 +1  20.07800000 +1\n",
            "  H(ATOM    144  HE3 TRP A 219)  28.16300000 +1   5.43100000 +1  22.99200000 +1\n",
            "  H(ATOM    145  HZ2 TRP A 219)  27.77800000 +1   0.74900000 +1  21.22200000 +1\n",
            "  H(ATOM    146  HZ3 TRP A 219)  26.20900000 +1   3.99400000 +1  23.54500000 +1\n",
            "  H(ATOM    147  HH2 TRP A 219)  26.03400000 +1   1.70500000 +1  22.67400000 +1\n",
            "  N(ATOM    148  N   GLU A 220)  31.19700000 +1   8.91600000 +1  21.90400000 +1\n",
            "  C(ATOM    149  CA  GLU A 220)  31.56100000 +1  10.03400000 +1  22.76200000 +1\n",
            "  C(ATOM    150  C   GLU A 220)  30.42300000 +1  11.04600000 +1  22.80400000 +1\n",
            "  O(ATOM    151  O   GLU A 220)  30.09400000 +1  11.57600000 +1  23.86500000 +1\n",
            "  C(ATOM    152  CB  GLU A 220)  32.85300000 +1  10.68000000 +1  22.26000000 +1\n",
            "  C(ATOM    153  CG  GLU A 220)  34.06600000 +1   9.78900000 +1  22.46300000 +1\n",
            "  C(ATOM    154  CD  GLU A 220)  34.12100000 +1   9.21100000 +1  23.87900000 +1\n",
            "  O(ATOM    155  OE1 GLU A 220)  34.15700000 +1  10.00900000 +1  24.84500000 +1\n",
            "  O(ATOM    156  OE2 GLU A 220)  34.11900000 +1   7.96300000 +1  24.03000000 +1\n",
            "  H(ATOM    157  H   GLU A 220)  31.66400000 +1   8.81500000 +1  21.01400000 +1\n",
            "  H(ATOM    158  HA  GLU A 220)  31.72000000 +1   9.62500000 +1  23.76000000 +1\n",
            "  H(ATOM    159  HB3 GLU A 220)  32.99400000 +1  11.61500000 +1  22.80300000 +1\n",
            "  H(ATOM    160  HB2 GLU A 220)  32.73100000 +1  10.89200000 +1  21.19800000 +1\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "----------------"
      ],
      "metadata": {
        "id": "R3L20NMBw5eN"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "#@title **Instalando depend√™ncias para softwares de din√¢mica molecular**\n",
        "#@markdown It will take a few minutes, please, drink a coffee and wait. ;-)\n",
        "# install dependencies\n",
        "import subprocess\n",
        "import sys\n",
        "subprocess.run(\"rm -rf /usr/local/conda-meta/pinned\", shell=True)\n",
        "subprocess.run(\"mamba install -c conda-forge ambertools -y\", shell=True)\n",
        "import pytraj as pt\n",
        "subprocess.run(\"pip install git+https://github.com/pablo-arantes/biopandas\", shell=True)\n",
        "subprocess.run(\"mamba install openmm=7.7.0\", shell=True)\n",
        "subprocess.run(\"pip install --upgrade MDAnalysis==2.4.2\", shell=True)\n",
        "!conda install openbabel -c conda-forge\n",
        "\n",
        "#load dependencies\n",
        "import sys\n",
        "from biopandas.pdb import PandasPdb\n",
        "import openmm as mm\n",
        "from openmm import *\n",
        "from openmm.app import *\n",
        "from openmm.unit import *\n",
        "import os\n",
        "import urllib.request\n",
        "import numpy as np\n",
        "import MDAnalysis as mda\n",
        "import pytraj as pt\n",
        "import platform\n",
        "import scipy.cluster.hierarchy\n",
        "from scipy.spatial.distance import squareform\n",
        "import scipy.stats as stats\n",
        "from statistics import mean, stdev\n",
        "from pytraj import matrix\n",
        "from IPython.display import set_matplotlib_formats"
      ],
      "metadata": {
        "id": "zxQmYpzRGYZV",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "dd95dae7-9598-4c07-9f8d-a79beddc1b7a",
        "cellView": "form"
      },
      "execution_count": 25,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Channels:\n",
            " - conda-forge\n",
            "Platform: linux-64\n",
            "Collecting package metadata (repodata.json): - \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\bdone\n",
            "Solving environment: - \b\b\\ \b\b| \b\b/ \b\b- \b\b\\ \b\b| \b\b/ \b\bdone\n",
            "\n",
            "\n",
            "==> WARNING: A newer version of conda exists. <==\n",
            "    current version: 23.11.0\n",
            "    latest version: 24.11.0\n",
            "\n",
            "Please update conda by running\n",
            "\n",
            "    $ conda update -n base -c conda-forge conda\n",
            "\n",
            "\n",
            "\n",
            "## Package Plan ##\n",
            "\n",
            "  environment location: /usr/local\n",
            "\n",
            "  added / updated specs:\n",
            "    - openbabel\n",
            "\n",
            "\n",
            "The following packages will be downloaded:\n",
            "\n",
            "    package                    |            build\n",
            "    ---------------------------|-----------------\n",
            "    cairo-1.18.0               |       hbb29018_2         961 KB  conda-forge\n",
            "    font-ttf-dejavu-sans-mono-2.37|       hab24e00_0         388 KB  conda-forge\n",
            "    font-ttf-inconsolata-3.000 |       h77eed37_0          94 KB  conda-forge\n",
            "    font-ttf-source-code-pro-2.038|       h77eed37_0         684 KB  conda-forge\n",
            "    font-ttf-ubuntu-0.83       |       h77eed37_3         1.5 MB  conda-forge\n",
            "    fontconfig-2.15.0          |       h7e30c49_1         259 KB  conda-forge\n",
            "    fonts-conda-ecosystem-1    |                0           4 KB  conda-forge\n",
            "    fonts-conda-forge-1        |                0           4 KB  conda-forge\n",
            "    libexpat-2.6.4             |       h5888daf_0          72 KB  conda-forge\n",
            "    libglib-2.82.2             |       h2ff4ddf_0         3.7 MB  conda-forge\n",
            "    openbabel-3.1.1            |  py310hbff9852_9         5.0 MB  conda-forge\n",
            "    pcre2-10.44                |       hba22ea6_2         930 KB  conda-forge\n",
            "    pixman-0.44.2              |       h29eaf8c_0         372 KB  conda-forge\n",
            "    xorg-libxrender-0.9.11     |       hb9d3cd8_2          37 KB  conda-forge\n",
            "    ------------------------------------------------------------\n",
            "                                           Total:        14.0 MB\n",
            "\n",
            "The following NEW packages will be INSTALLED:\n",
            "\n",
            "  cairo              conda-forge/linux-64::cairo-1.18.0-hbb29018_2 \n",
            "  font-ttf-dejavu-s~ conda-forge/noarch::font-ttf-dejavu-sans-mono-2.37-hab24e00_0 \n",
            "  font-ttf-inconsol~ conda-forge/noarch::font-ttf-inconsolata-3.000-h77eed37_0 \n",
            "  font-ttf-source-c~ conda-forge/noarch::font-ttf-source-code-pro-2.038-h77eed37_0 \n",
            "  font-ttf-ubuntu    conda-forge/noarch::font-ttf-ubuntu-0.83-h77eed37_3 \n",
            "  fontconfig         conda-forge/linux-64::fontconfig-2.15.0-h7e30c49_1 \n",
            "  fonts-conda-ecosy~ conda-forge/noarch::fonts-conda-ecosystem-1-0 \n",
            "  fonts-conda-forge  conda-forge/noarch::fonts-conda-forge-1-0 \n",
            "  libexpat           conda-forge/linux-64::libexpat-2.6.4-h5888daf_0 \n",
            "  libglib            conda-forge/linux-64::libglib-2.82.2-h2ff4ddf_0 \n",
            "  openbabel          conda-forge/linux-64::openbabel-3.1.1-py310hbff9852_9 \n",
            "  pcre2              conda-forge/linux-64::pcre2-10.44-hba22ea6_2 \n",
            "  pixman             conda-forge/linux-64::pixman-0.44.2-h29eaf8c_0 \n",
            "  xorg-libxrender    conda-forge/linux-64::xorg-libxrender-0.9.11-hb9d3cd8_2 \n",
            "\n",
            "\n",
            "\n",
            "Downloading and Extracting Packages:\n",
            "openbabel-3.1.1      | 5.0 MB    | :   0% 0/1 [00:00<?, ?it/s]\n",
            "libglib-2.82.2       | 3.7 MB    | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\n",
            "\n",
            "font-ttf-ubuntu-0.83 | 1.5 MB    | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "cairo-1.18.0         | 961 KB    | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "pcre2-10.44          | 930 KB    | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "font-ttf-source-code | 684 KB    | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "font-ttf-dejavu-sans | 388 KB    | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pixman-0.44.2        | 372 KB    | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "fontconfig-2.15.0    | 259 KB    | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "font-ttf-inconsolata | 94 KB     | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "libexpat-2.6.4       | 72 KB     | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "xorg-libxrender-0.9. | 37 KB     | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "fonts-conda-forge-1  | 4 KB      | :   0% 0/1 [00:00<?, ?it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "openbabel-3.1.1      | 5.0 MB    | :   0% 0.0031211275225574045/1 [00:00<00:52, 52.42s/it]\n",
            "\n",
            "\n",
            "cairo-1.18.0         | 961 KB    | :   2% 0.016646617030269532/1 [00:00<00:09,  9.86s/it]\u001b[A\u001b[A\u001b[A\n",
            "libglib-2.82.2       | 3.7 MB    | :   0% 0.0041669443103559654/1 [00:00<00:42, 42.95s/it]\u001b[A\n",
            "\n",
            "font-ttf-ubuntu-0.83 | 1.5 MB    | :   1% 0.010110434778315882/1 [00:00<00:18, 18.83s/it]\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "openbabel-3.1.1      | 5.0 MB    | :  35% 0.3526874100489867/1 [00:00<00:00,  1.56it/s]   \n",
            "libglib-2.82.2       | 3.7 MB    | :  56% 0.5625374818980553/1 [00:00<00:00,  2.40it/s]   \u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "openbabel-3.1.1      | 5.0 MB    | :  94% 0.9425805118123362/1 [00:00<00:00,  3.18it/s]\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "font-ttf-dejavu-sans | 388 KB    | :   4% 0.04123109444598234/1 [00:00<00:08,  9.17s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pixman-0.44.2        | 372 KB    | :   4% 0.042994499727085696/1 [00:00<00:08,  8.87s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "font-ttf-inconsolata | 94 KB     | :  17% 0.16972961773541903/1 [00:00<00:02,  2.55s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "fontconfig-2.15.0    | 259 KB    | :   6% 0.06168697924314474/1 [00:00<00:07,  7.73s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "libexpat-2.6.4       | 72 KB     | :  22% 0.22350758485212266/1 [00:00<00:01,  2.22s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "xorg-libxrender-0.9. | 37 KB     | :  44% 0.4353047452043148/1 [00:00<00:00,  1.15s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "fonts-conda-forge-1  | 4 KB      | : 100% 1.0/1 [00:00<00:00,  1.89it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "fonts-conda-ecosyste | 4 KB      | : 100% 1.0/1 [00:00<00:00,  1.79it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "cairo-1.18.0         | 961 KB    | : 100% 1.0/1 [00:00<00:00,  1.68it/s]                 \u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "cairo-1.18.0         | 961 KB    | : 100% 1.0/1 [00:00<00:00,  1.68it/s]\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "font-ttf-ubuntu-0.83 | 1.5 MB    | : 100% 1.0/1 [00:00<00:00,  1.38it/s]                 \u001b[A\u001b[A\n",
            "\n",
            "font-ttf-ubuntu-0.83 | 1.5 MB    | : 100% 1.0/1 [00:00<00:00,  1.38it/s]\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "pcre2-10.44          | 930 KB    | : 100% 1.0/1 [00:00<00:00,  1.29it/s]                 \u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "pcre2-10.44          | 930 KB    | : 100% 1.0/1 [00:00<00:00,  1.29it/s]\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pixman-0.44.2        | 372 KB    | : 100% 1.0/1 [00:00<00:00,  1.27it/s]                 \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "pixman-0.44.2        | 372 KB    | : 100% 1.0/1 [00:00<00:00,  1.27it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "font-ttf-dejavu-sans | 388 KB    | : 100% 1.0/1 [00:01<00:00,  1.06it/s]                \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "font-ttf-dejavu-sans | 388 KB    | : 100% 1.0/1 [00:01<00:00,  1.06it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "font-ttf-inconsolata | 94 KB     | : 100% 1.0/1 [00:01<00:00,  1.04s/it]                \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "font-ttf-inconsolata | 94 KB     | : 100% 1.0/1 [00:01<00:00,  1.04s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "font-ttf-source-code | 684 KB    | : 100% 1.0/1 [00:01<00:00,  1.14s/it]                \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "font-ttf-source-code | 684 KB    | : 100% 1.0/1 [00:01<00:00,  1.14s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "fontconfig-2.15.0    | 259 KB    | : 100% 1.0/1 [00:01<00:00,  1.25s/it]                \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "fontconfig-2.15.0    | 259 KB    | : 100% 1.0/1 [00:01<00:00,  1.25s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "libexpat-2.6.4       | 72 KB     | : 100% 1.0/1 [00:01<00:00,  1.34s/it]                \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "libexpat-2.6.4       | 72 KB     | : 100% 1.0/1 [00:01<00:00,  1.34s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "xorg-libxrender-0.9. | 37 KB     | : 100% 1.0/1 [00:01<00:00,  1.48s/it]               \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "xorg-libxrender-0.9. | 37 KB     | : 100% 1.0/1 [00:01<00:00,  1.48s/it]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "fonts-conda-forge-1  | 4 KB      | : 100% 1.0/1 [00:01<00:00,  1.89it/s]\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "libglib-2.82.2       | 3.7 MB    | : 100% 1.0/1 [00:01<00:00,  1.90s/it]               \u001b[A\n",
            "libglib-2.82.2       | 3.7 MB    | : 100% 1.0/1 [00:01<00:00,  1.90s/it]\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \n",
            "                                                                        \u001b[A\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "\n",
            "                                                                        \u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\u001b[A\n",
            "Preparing transaction: \\ \b\bdone\n",
            "Verifying transaction: / \b\b- \b\b\\ \b\bdone\n",
            "Executing transaction: / \b\b- \b\bdone\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "# import pytraj as pt\n",
        "print('Vers√£o da PyTraj --> ', pt.__version__)"
      ],
      "metadata": {
        "id": "hEyCVXadkCTl",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "72704fa9-47ea-45ab-abaa-25d8dd2c337a"
      },
      "execution_count": 34,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Vers√£o da PyTraj -->  2.0.6\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "# Instalando o AMBERTools\n",
        "\n",
        "#!conda install -c conda-forge ambertools"
      ],
      "metadata": {
        "id": "1oS4aCD-yZPA"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "# Verificando se foi instalado corretamente\n",
        "!which tleap"
      ],
      "metadata": {
        "id": "6dgffIdEyZPA",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "8ddbc607-47c0-4ac7-acda-c40cb352fab4"
      },
      "execution_count": 35,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "/usr/local/bin/tleap\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!which obabel"
      ],
      "metadata": {
        "id": "0zUqAb59CCSo",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "367c37be-38f0-41a3-ec8a-3d7e47e61fde"
      },
      "execution_count": 36,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "/usr/local/bin/obabel\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!obabel -:'CC' -Oetano.sdf --gen3D"
      ],
      "metadata": {
        "id": "v7RTMTC5CW2P",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "a1cf81fe-b16e-4e6d-98f8-bab1bf1e663f"
      },
      "execution_count": 37,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "1 molecule converted\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "v = p3d.view()\n",
        "v.addModel(open('etano.sdf').read())\n",
        "v.setStyle({'stick':{'colorscheme':'greenCarbon'}})\n",
        "v.zoomTo()"
      ],
      "metadata": {
        "id": "hXJjPC1iCrmb",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 514
        },
        "outputId": "55ef35a2-ed26-467a-8df4-1a3cc5971710"
      },
      "execution_count": 38,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_1733841994586641\"  style=\"position: relative; width: 640px; height: 480px;\">\n        <p id=\"3dmolwarning_1733841994586641\" style=\"background-color:#ffcccc;color:black\">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    //this is to ignore the existence of requirejs amd\n    var savedexports, savedmodule;\n    if (typeof exports !== 'undefined') savedexports = exports;\n    else exports = {}\n    if (typeof module !== 'undefined') savedmodule = module;\n    else module = {}\n\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n        exports = savedexports;\n        module = savedmodule;\n        resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.4.2/3Dmol-min.js');\n}\n\nvar viewer_1733841994586641 = null;\nvar warn = document.getElementById(\"3dmolwarning_1733841994586641\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_1733841994586641 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_1733841994586641\"),{backgroundColor:\"white\"});\nviewer_1733841994586641.zoomTo();\n\tviewer_1733841994586641.addModel(\"\\n OpenBabel12102414463D\\n\\n  8  7  0  0  0  0  0  0  0  0999 V2000\\n    0.9646    0.0489   -0.0778 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.4766    0.0489   -0.0778 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.5803    0.5788    0.7989 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.5803   -0.9752   -0.0573 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.5803    0.5433   -0.9750 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.8609   -0.4454    0.8194 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.8609    1.0731   -0.0983 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.8609   -0.4809   -0.9545 H   0  0  0  0  0  0  0  0  0  0  0  0\\n  1  2  1  0  0  0  0\\n  1  3  1  0  0  0  0\\n  1  4  1  0  0  0  0\\n  1  5  1  0  0  0  0\\n  2  6  1  0  0  0  0\\n  2  7  1  0  0  0  0\\n  2  8  1  0  0  0  0\\nM  END\\n$$$$\\n\");\n\tviewer_1733841994586641.setStyle({\"stick\": {\"colorscheme\": \"greenCarbon\"}});\n\tviewer_1733841994586641.zoomTo();\nviewer_1733841994586641.render();\n});\n</script>",
            "text/html": [
              "<div id=\"3dmolviewer_1733841994586641\"  style=\"position: relative; width: 640px; height: 480px;\">\n",
              "        <p id=\"3dmolwarning_1733841994586641\" style=\"background-color:#ffcccc;color:black\">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br></p>\n",
              "        </div>\n",
              "<script>\n",
              "\n",
              "var loadScriptAsync = function(uri){\n",
              "  return new Promise((resolve, reject) => {\n",
              "    //this is to ignore the existence of requirejs amd\n",
              "    var savedexports, savedmodule;\n",
              "    if (typeof exports !== 'undefined') savedexports = exports;\n",
              "    else exports = {}\n",
              "    if (typeof module !== 'undefined') savedmodule = module;\n",
              "    else module = {}\n",
              "\n",
              "    var tag = document.createElement('script');\n",
              "    tag.src = uri;\n",
              "    tag.async = true;\n",
              "    tag.onload = () => {\n",
              "        exports = savedexports;\n",
              "        module = savedmodule;\n",
              "        resolve();\n",
              "    };\n",
              "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
              "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
              "});\n",
              "};\n",
              "\n",
              "if(typeof $3Dmolpromise === 'undefined') {\n",
              "$3Dmolpromise = null;\n",
              "  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.4.2/3Dmol-min.js');\n",
              "}\n",
              "\n",
              "var viewer_1733841994586641 = null;\n",
              "var warn = document.getElementById(\"3dmolwarning_1733841994586641\");\n",
              "if(warn) {\n",
              "    warn.parentNode.removeChild(warn);\n",
              "}\n",
              "$3Dmolpromise.then(function() {\n",
              "viewer_1733841994586641 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_1733841994586641\"),{backgroundColor:\"white\"});\n",
              "viewer_1733841994586641.zoomTo();\n",
              "\tviewer_1733841994586641.addModel(\"\\n OpenBabel12102414463D\\n\\n  8  7  0  0  0  0  0  0  0  0999 V2000\\n    0.9646    0.0489   -0.0778 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.4766    0.0489   -0.0778 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.5803    0.5788    0.7989 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.5803   -0.9752   -0.0573 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.5803    0.5433   -0.9750 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.8609   -0.4454    0.8194 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.8609    1.0731   -0.0983 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.8609   -0.4809   -0.9545 H   0  0  0  0  0  0  0  0  0  0  0  0\\n  1  2  1  0  0  0  0\\n  1  3  1  0  0  0  0\\n  1  4  1  0  0  0  0\\n  1  5  1  0  0  0  0\\n  2  6  1  0  0  0  0\\n  2  7  1  0  0  0  0\\n  2  8  1  0  0  0  0\\nM  END\\n$$$$\\n\");\n",
              "\tviewer_1733841994586641.setStyle({\"stick\": {\"colorscheme\": \"greenCarbon\"}});\n",
              "\tviewer_1733841994586641.zoomTo();\n",
              "viewer_1733841994586641.render();\n",
              "});\n",
              "</script>"
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "<py3Dmol.view at 0x7eb65585bb80>"
            ]
          },
          "metadata": {},
          "execution_count": 38
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!obabel etano.sdf -O etano.xyz\n",
        "!cat < etano.xyz"
      ],
      "metadata": {
        "id": "zfUqn4adC2CY",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "6fbe567d-7086-42d9-8b67-c1736389eaf8"
      },
      "execution_count": 39,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "1 molecule converted\n",
            "8\n",
            "\n",
            "C          0.96460        0.04890       -0.07780\n",
            "C          2.47660        0.04890       -0.07780\n",
            "H          0.58030        0.57880        0.79890\n",
            "H          0.58030       -0.97520       -0.05730\n",
            "H          0.58030        0.54330       -0.97500\n",
            "H          2.86090       -0.44540        0.81940\n",
            "H          2.86090        1.07310       -0.09830\n",
            "H          2.86090       -0.48090       -0.95450\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [],
      "metadata": {
        "id": "ijipj-5f2rYT"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "source": [
        "------------\n",
        "#### **3.4.2 - xTB**\n",
        "\n",
        "xTB (*xtight-binding*) √© uma ferramenta de qu√≠mica computacional desenvolvida para realizar c√°lculos de estrutura eletr√¥nica de maneira eficiente. √â especialmente √∫til para estudos de sistemas grandes, como biomol√©culas e materiais, onde m√©todos *ab initio* s√£o computacionalmente proibitivos em computadores de pequeno e m√©dio porte. Desenvolvido pelo grupo de Stefan Grimme, o xTB oferece uma variedade de m√©todos semiemp√≠ricos, incluindo GFN-xTB (Geometries, Frequencies, Noncovalent interactions eXtended Tight-Binding) e outras variantes como GFN1-xTB e GFN2-xTB.\n",
        "\n",
        "#### **Uso do xTB com Python**\n",
        "\n",
        "O xTB pode ser utilizado para otimiza√ß√£o de geometria, c√°lculos de energia, an√°lise de espectros vibracionais, e muito mais. No contexto de Python, ele pode ser facilmente integrado e automatizado usando scripts, facilitando a execu√ß√£o de c√°lculos de alto desempenho.\n",
        "\n",
        "**Links**\n",
        "\n",
        "* <https://xtb-docs.readthedocs.io/en/latest/>\n",
        "\n",
        "\n"
      ],
      "metadata": {
        "id": "XPEasIlnwSL9"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "-------------\n",
        "##### **Instalando o xTB no Colab**"
      ],
      "metadata": {
        "id": "4lGQbBLpPj7g"
      }
    },
    {
      "cell_type": "code",
      "execution_count": 40,
      "metadata": {
        "id": "RLSE-Si7YWaK",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "5d980bb2-704f-4df3-9bd1-d1e905cc9818"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "xtb-dist/\n",
            "xtb-dist/lib/\n",
            "xtb-dist/lib/libxtb.a\n",
            "xtb-dist/lib/pkgconfig/\n",
            "xtb-dist/lib/pkgconfig/xtb.pc\n",
            "xtb-dist/bin/\n",
            "xtb-dist/bin/xtb\n",
            "xtb-dist/include/\n",
            "xtb-dist/include/xtb.h\n",
            "xtb-dist/include/cpx/\n",
            "xtb-dist/include/cpx/intel-2021.5.0/\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_prog_dock.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_prog_submodules.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_prog_ir.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_prog_info.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_prog_thermo.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_prog_argparser.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_prog_topology.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_prog_main.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_mctc_blas.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_mctc_param.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_constrainpot.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_solv_ddcosmo_core.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_gfnff_neighbor.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_disp_dftd4param.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_ptb_integral_types.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_coulomb_ewald.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_ptb_integrals.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_xtb_multipole.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_iff_iffini.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_pbc_optimizer_filter_cart.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_xtb_repulsion.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_xtb_calculator.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_approxrab.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_scc_core.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_mctc_lapack_eigensolve.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_gfnff_shake.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_solv_cosmo.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_param_sqrtzr4r2.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_slater.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_grad_core.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_disp_coordinationnumber.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_io_reader_turbomole.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_xtb_dispersion.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_axis.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_david2.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_type_wavefunction.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_hlex.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_tblite_calculator.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_ptb_ncoord.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_type_environment.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_coulomb_gaussian.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_disp_dftd4.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_param_atomicmass.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_gfnff_fraghess.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_gfnff_topology.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_topology.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_mctc_blas_level2.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_gfnff_ini.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_api_environment.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_readparam.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_extern_driver.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/esp.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_dipro_version.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_anharmlib.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_xtb_hamiltonian_gpu.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_type_latticepoint.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_docking_set_module.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_dipro_fragment.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_dynamic.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_constrain_param.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_geoopt.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_gfnff_setup.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_mctc_resize.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_hessian.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_type_timer.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_type_pcem.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_solv_born.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_tblite_restart.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_pbc_tools.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_lineardep.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_param_vdwradcosmo.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_dipole.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_scan.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_gfnff_neighbourlist.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_setmod.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_mctc_timings.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_biaspath.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_solv_lebedev.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_printout.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_xtb_thirdorder.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_type_setvar.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_xtb_gfn0.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_fixparam.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_type_buffer.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/mctcpar_covalent_radii.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_io_writer.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_type_anc.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_single.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_scanparam.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_bfgs.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_iff_iffprepare.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_xtb_gfn1.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_intgrad.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_pbc_optimizer_filter_type.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_mctc_version.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_iff_ifflmo.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/mctcpar_r4r2.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_cube.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_dipro_bondorder.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_pbc_optimizer_driver.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_mctc_strings.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_gfnff_data.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_param_covalentradd3.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_xtb_data.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_solv_gbsa.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_freq_project.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_iff_data.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_mctc_symbols.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_ptb_param.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_extern_orca.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_solv_ddvolume.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_peeq.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_ptb_hamiltonian.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/mctcpar_electronegativities.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_api_utils.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_gfnff_mrec.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_features.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_relaxation_engine.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_embedding.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_mctc_lapack_wrap.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_mctc_constants.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_type_topology.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_ptb_coulomb.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_basis.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_xtb_halogen.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_coffee.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_type_param.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_solv_state.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/mctcpar_chemical_hardnesses.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_mctc_blas_wrap2.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_solv_cpx.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_mctc_lapack_gst.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_main_json.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_io_reader_genformat.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_docking_param.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_extern_turbomole.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_chargemodel.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_aoparam.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_mctc_chartools.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_api_interface.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_aespot.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_thermo.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_dipro_xtb.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_shake.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/mctcpar_pse.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_iniq.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_io_writer_gaussian.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_mctc_lapack_trs.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_gfnff_ini2.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_ptb_mmlpopanalysis.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_type_dispersionmodel.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_basic_geo.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_mctc_blas_wrap3.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_mctc_blas_wrap1.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_solv_cm5.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_restart.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_solv_kernel.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_api_results.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_gfnff_param.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_mctc_math.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_type_molecule.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_blowsy.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_propertyoutput.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_type_calculator.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_disp_dftd3param.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_type_fragments.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_detrotra.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_mctc_global.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_mctc_blas_level1.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_mctc_blas_level3.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_type_neighbourlist.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_main_setup.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_solv_input.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_type_iohandler.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_mctc_la.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_type_options.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_cqpath.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_modelhessian.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_modef.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_screening.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_type_coulomb.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_type_vendordata.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_lsrmsd.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_mctc_lapack_tri.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_metadynamic.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_pbc_optimizer_filter.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_solv_model.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_ptb_response.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_scf.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_param_vdwradd3.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_dipro_output.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_lin.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_mctc_lapack_stdeigval.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_pbc.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_api_version.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_oniom.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_solv_ddcosmo_solver.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_mctc_io.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_type_wignerseitzcell.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_mctc_boundaryconditions.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_pbc_optimizer_optimizer.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_type_wsc.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_io_writer_orca.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_api_molecule.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_docking_search_nci.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_solv_sasa.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_main_defaults.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_mctc_accuracy.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_mctc_runtypes.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_ptb_property.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_param_atomicrad.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_type_data.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_io_reader.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_coulomb_klopmanohno.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_type_solvation.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_disp_encharges.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_io_writer_turbomole.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_gfnff_generator.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_param_paulingen.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_xtb_coulomb.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_freq_utils.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_ptb_guess.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_broyden.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_onetri.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_xtb_gfn2.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_gfnff_calculator.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_type_identitymap.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_ptb_vdzp.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_type_basisset.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_paramset.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_intpack.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_readin.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_dipro.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/stm.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_mctc_lapack.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_dtrafo.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_gfn0param.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_mctc_namegen.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_disp_dftd3.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_ptb_paulixc.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_sphereparam.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_optimizer.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_mctc_lapack_geneigval.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_type_reader.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_gfnff_gdisp0.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_iff_iffenergy.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_ptb_plusu.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_xtb_hamiltonian.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_ptb_data.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_eeq.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_gfnff_eg.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_local.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_type_restart.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_io_reader_orca.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_iff_calculator.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_intmodes.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_ptb_calculator.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_xtb_eeq.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_mctc_lapack_trf.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_disp_ncoord.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_ptb_scf.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_type_atomlist.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_mctc_systools.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_gfnff_convert.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_mctc_thresholds.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_ptb_corebasis.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_kopt.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_mctc_convert.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_freq_io.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_mctc_search.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_extern_mopac.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_mctc_filetools.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_vertical.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_api_calculator.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/mctcpar_atomic_masses.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_pbc_optimizer_lbfgs.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_mctc_sort.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_mdoptim.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_mctc_filetypes.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_setparam.mod\n",
            "xtb-dist/include/cpx/intel-2021.5.0/xtb_splitparam.mod\n",
            "xtb-dist/share/\n",
            "xtb-dist/share/man/\n",
            "xtb-dist/share/man/man1/\n",
            "xtb-dist/share/man/man1/xtb.1\n",
            "xtb-dist/share/man/man7/\n",
            "xtb-dist/share/man/man7/xcontrol.7\n",
            "xtb-dist/share/licenses/\n",
            "xtb-dist/share/licenses/mctc-lib/\n",
            "xtb-dist/share/licenses/mctc-lib/jsonfortran/\n",
            "xtb-dist/share/licenses/mctc-lib/jsonfortran/LICENSE\n",
            "xtb-dist/share/xtb/\n",
            "xtb-dist/share/xtb/param_gfn0-xtb.txt\n",
            "xtb-dist/share/xtb/param_gfn2-xtb.txt\n",
            "xtb-dist/share/xtb/param_gfn1-xtb.txt\n",
            "xtb-dist/share/xtb/param_gfn1-si-xtb.txt\n",
            "xtb-dist/share/xtb/param_ipea-xtb.txt\n",
            "xtb-dist/share/xtb/.param_gfnff.xtb\n",
            "xtb-dist/share/xtb/config_env.bash\n",
            "xtb-dist/share/xtb/config_env.csh\n",
            "xtb-dist/share/modules/\n",
            "xtb-dist/share/modules/modulefiles/\n",
            "xtb-dist/share/modules/modulefiles/xtb/\n",
            "xtb-dist/share/modules/modulefiles/xtb/6.7.0\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "--2024-12-10 14:46:46--  https://github.com/grimme-lab/xtb/releases/download/v6.7.0/xtb-6.7.0-linux-x86_64.tar.xz\n",
            "Resolving github.com (github.com)... 140.82.113.4\n",
            "Connecting to github.com (github.com)|140.82.113.4|:443... connected.\n",
            "HTTP request sent, awaiting response... 302 Found\n",
            "Location: https://objects.githubusercontent.com/github-production-release-asset-2e65be/211856832/dab5f2d5-5925-4773-ac11-798eb456a3e2?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=releaseassetproduction%2F20241210%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20241210T144646Z&X-Amz-Expires=300&X-Amz-Signature=9d19da5577d7ace16fd8f9d1322ba7e56c25faa119803365f4faab2230b88395&X-Amz-SignedHeaders=host&response-content-disposition=attachment%3B%20filename%3Dxtb-6.7.0-linux-x86_64.tar.xz&response-content-type=application%2Foctet-stream [following]\n",
            "--2024-12-10 14:46:46--  https://objects.githubusercontent.com/github-production-release-asset-2e65be/211856832/dab5f2d5-5925-4773-ac11-798eb456a3e2?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=releaseassetproduction%2F20241210%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20241210T144646Z&X-Amz-Expires=300&X-Amz-Signature=9d19da5577d7ace16fd8f9d1322ba7e56c25faa119803365f4faab2230b88395&X-Amz-SignedHeaders=host&response-content-disposition=attachment%3B%20filename%3Dxtb-6.7.0-linux-x86_64.tar.xz&response-content-type=application%2Foctet-stream\n",
            "Resolving objects.githubusercontent.com (objects.githubusercontent.com)... 185.199.108.133, 185.199.109.133, 185.199.110.133, ...\n",
            "Connecting to objects.githubusercontent.com (objects.githubusercontent.com)|185.199.108.133|:443... connected.\n",
            "HTTP request sent, awaiting response... 200 OK\n",
            "Length: 21031364 (20M) [application/octet-stream]\n",
            "Saving to: ‚Äòxtb-6.7.0-linux-x86_64.tar.xz‚Äô\n",
            "\n",
            "     0K .......... .......... .......... .......... ..........  0% 2.11M 9s\n",
            "    50K .......... .......... .......... .......... ..........  0% 15.3M 5s\n",
            "   100K .......... .......... .......... .......... ..........  0% 5.75M 5s\n",
            "   150K .......... .......... .......... .......... ..........  0% 19.0M 4s\n",
            "   200K .......... .......... .......... .......... ..........  1% 20.5M 3s\n",
            "   250K .......... .......... .......... .......... ..........  1% 40.8M 3s\n",
            "   300K .......... .......... .......... .......... ..........  1% 8.79M 3s\n",
            "   350K .......... .......... .......... .......... ..........  1% 32.6M 2s\n",
            "   400K .......... .......... .......... .......... ..........  2% 32.5M 2s\n",
            "   450K .......... .......... .......... .......... ..........  2% 39.6M 2s\n",
            "   500K .......... .......... .......... .......... ..........  2% 33.5M 2s\n",
            "   550K .......... .......... .......... .......... ..........  2%  183M 2s\n",
            "   600K .......... .......... .......... .......... ..........  3% 9.68M 2s\n",
            "   650K .......... .......... .......... .......... ..........  3% 80.3M 2s\n",
            "   700K .......... .......... .......... .......... ..........  3% 75.5M 2s\n",
            "   750K .......... .......... .......... .......... ..........  3% 53.6M 1s\n",
            "   800K .......... .......... .......... .......... ..........  4% 69.2M 1s\n",
            "   850K .......... .......... .......... .......... ..........  4%  179M 1s\n",
            "   900K .......... .......... .......... .......... ..........  4% 66.9M 1s\n",
            "   950K .......... .......... .......... .......... ..........  4% 50.6M 1s\n",
            "  1000K .......... .......... .......... .......... ..........  5%  102M 1s\n",
            "  1050K .......... .......... .......... .......... ..........  5%  186M 1s\n",
            "  1100K .......... .......... .......... .......... ..........  5% 66.9M 1s\n",
            "  1150K .......... .......... .......... .......... ..........  5%  135M 1s\n",
            "  1200K .......... .......... .......... .......... ..........  6% 10.2M 1s\n",
            "  1250K .......... .......... .......... .......... ..........  6%  164M 1s\n",
            "  1300K .......... .......... .......... .......... ..........  6%  135M 1s\n",
            "  1350K .......... .......... .......... .......... ..........  6%  180M 1s\n",
            "  1400K .......... .......... .......... .......... ..........  7%  158M 1s\n",
            "  1450K .......... .......... .......... .......... ..........  7% 76.6M 1s\n",
            "  1500K .......... .......... .......... .......... ..........  7%  201M 1s\n",
            "  1550K .......... .......... .......... .......... ..........  7%  147M 1s\n",
            "  1600K .......... .......... .......... .......... ..........  8%  214M 1s\n",
            "  1650K .......... .......... .......... .......... ..........  8%  194M 1s\n",
            "  1700K .......... .......... .......... .......... ..........  8%  163M 1s\n",
            "  1750K .......... .......... .......... .......... ..........  8%  166M 1s\n",
            "  1800K .......... .......... .......... .......... ..........  9%  187M 1s\n",
            "  1850K .......... .......... .......... .......... ..........  9%  160M 1s\n",
            "  1900K .......... .......... .......... .......... ..........  9%  112M 1s\n",
            "  1950K .......... .......... .......... .......... ..........  9%  157M 1s\n",
            "  2000K .......... .......... .......... .......... ..........  9%  171M 1s\n",
            "  2050K .......... .......... .......... .......... .......... 10%  163M 1s\n",
            "  2100K .......... .......... .......... .......... .......... 10%  195M 1s\n",
            "  2150K .......... .......... .......... .......... .......... 10%  177M 1s\n",
            "  2200K .......... .......... .......... .......... .......... 10%  186M 1s\n",
            "  2250K .......... .......... .......... .......... .......... 11%  174M 1s\n",
            "  2300K .......... .......... .......... .......... .......... 11%  192M 1s\n",
            "  2350K .......... .......... .......... .......... .......... 11%  149M 1s\n",
            "  2400K .......... .......... .......... .......... .......... 11%  215M 1s\n",
            "  2450K .......... .......... .......... .......... .......... 12% 12.0M 1s\n",
            "  2500K .......... .......... .......... .......... .......... 12%  152M 1s\n",
            "  2550K .......... .......... .......... .......... .......... 12%  189M 1s\n",
            "  2600K .......... .......... .......... .......... .......... 12%  204M 1s\n",
            "  2650K .......... .......... .......... .......... .......... 13%  175M 1s\n",
            "  2700K .......... .......... .......... .......... .......... 13%  187M 1s\n",
            "  2750K .......... .......... .......... .......... .......... 13%  169M 1s\n",
            "  2800K .......... .......... .......... .......... .......... 13%  176M 1s\n",
            "  2850K .......... .......... .......... .......... .......... 14%  145M 1s\n",
            "  2900K .......... .......... .......... .......... .......... 14%  195M 0s\n",
            "  2950K .......... .......... .......... .......... .......... 14%  137M 0s\n",
            "  3000K .......... .......... .......... .......... .......... 14%  165M 0s\n",
            "  3050K .......... .......... .......... .......... .......... 15%  185M 0s\n",
            "  3100K .......... .......... .......... .......... .......... 15%  168M 0s\n",
            "  3150K .......... .......... .......... .......... .......... 15%  150M 0s\n",
            "  3200K .......... .......... .......... .......... .......... 15%  203M 0s\n",
            "  3250K .......... .......... .......... .......... .......... 16%  197M 0s\n",
            "  3300K .......... .......... .......... .......... .......... 16%  192M 0s\n",
            "  3350K .......... .......... .......... .......... .......... 16%  161M 0s\n",
            "  3400K .......... .......... .......... .......... .......... 16%  171M 0s\n",
            "  3450K .......... .......... .......... .......... .......... 17%  206M 0s\n",
            "  3500K .......... .......... .......... .......... .......... 17%  186M 0s\n",
            "  3550K .......... .......... .......... .......... .......... 17%  135M 0s\n",
            "  3600K .......... .......... .......... .......... .......... 17%  185M 0s\n",
            "  3650K .......... .......... .......... .......... .......... 18%  192M 0s\n",
            "  3700K .......... .......... .......... .......... .......... 18%  141M 0s\n",
            "  3750K .......... .......... .......... .......... .......... 18%  159M 0s\n",
            "  3800K .......... .......... .......... .......... .......... 18%  192M 0s\n",
            "  3850K .......... .......... .......... .......... .......... 18%  167M 0s\n",
            "  3900K .......... .......... .......... .......... .......... 19%  269M 0s\n",
            "  3950K .......... .......... .......... .......... .......... 19%  206M 0s\n",
            "  4000K .......... .......... .......... .......... .......... 19%  162M 0s\n",
            "  4050K .......... .......... .......... .......... .......... 19%  165M 0s\n",
            "  4100K .......... .......... .......... .......... .......... 20%  181M 0s\n",
            "  4150K .......... .......... .......... .......... .......... 20%  182M 0s\n",
            "  4200K .......... .......... .......... .......... .......... 20%  199M 0s\n",
            "  4250K .......... .......... .......... .......... .......... 20%  169M 0s\n",
            "  4300K .......... .......... .......... .......... .......... 21%  189M 0s\n",
            "  4350K .......... .......... .......... .......... .......... 21%  149M 0s\n",
            "  4400K .......... .......... .......... .......... .......... 21%  185M 0s\n",
            "  4450K .......... .......... .......... .......... .......... 21%  201M 0s\n",
            "  4500K .......... .......... .......... .......... .......... 22%  204M 0s\n",
            "  4550K .......... .......... .......... .......... .......... 22%  147M 0s\n",
            "  4600K .......... .......... .......... .......... .......... 22%  148M 0s\n",
            "  4650K .......... .......... .......... .......... .......... 22%  163M 0s\n",
            "  4700K .......... .......... .......... .......... .......... 23%  152M 0s\n",
            "  4750K .......... .......... .......... .......... .......... 23%  115M 0s\n",
            "  4800K .......... .......... .......... .......... .......... 23%  171M 0s\n",
            "  4850K .......... .......... .......... .......... .......... 23%  188M 0s\n",
            "  4900K .......... .......... .......... .......... .......... 24%  201M 0s\n",
            "  4950K .......... .......... .......... .......... .......... 24%  204M 0s\n",
            "  5000K .......... .......... .......... .......... .......... 24%  206M 0s\n",
            "  5050K .......... .......... .......... .......... .......... 24%  158M 0s\n",
            "  5100K .......... .......... .......... .......... .......... 25%  212M 0s\n",
            "  5150K .......... .......... .......... .......... .......... 25%  179M 0s\n",
            "  5200K .......... .......... .......... .......... .......... 25%  240M 0s\n",
            "  5250K .......... .......... .......... .......... .......... 25%  227M 0s\n",
            "  5300K .......... .......... .......... .......... .......... 26%  213M 0s\n",
            "  5350K .......... .......... .......... .......... .......... 26% 17.4M 0s\n",
            "  5400K .......... .......... .......... .......... .......... 26%  195M 0s\n",
            "  5450K .......... .......... .......... .......... .......... 26%  206M 0s\n",
            "  5500K .......... .......... .......... .......... .......... 27%  195M 0s\n",
            "  5550K .......... .......... .......... .......... .......... 27%  168M 0s\n",
            "  5600K .......... .......... .......... .......... .......... 27%  183M 0s\n",
            "  5650K .......... .......... .......... .......... .......... 27%  194M 0s\n",
            "  5700K .......... .......... .......... .......... .......... 27%  226M 0s\n",
            "  5750K .......... .......... .......... .......... .......... 28%  211M 0s\n",
            "  5800K .......... .......... .......... .......... .......... 28%  167M 0s\n",
            "  5850K .......... .......... .......... .......... .......... 28%  170M 0s\n",
            "  5900K .......... .......... .......... .......... .......... 28%  207M 0s\n",
            "  5950K .......... .......... .......... .......... .......... 29%  189M 0s\n",
            "  6000K .......... .......... .......... .......... .......... 29%  139M 0s\n",
            "  6050K .......... .......... .......... .......... .......... 29%  208M 0s\n",
            "  6100K .......... .......... .......... .......... .......... 29%  211M 0s\n",
            "  6150K .......... .......... .......... .......... .......... 30%  184M 0s\n",
            "  6200K .......... .......... .......... .......... .......... 30%  166M 0s\n",
            "  6250K .......... .......... .......... .......... .......... 30%  187M 0s\n",
            "  6300K .......... .......... .......... .......... .......... 30%  165M 0s\n",
            "  6350K .......... .......... .......... .......... .......... 31%  131M 0s\n",
            "  6400K .......... .......... .......... .......... .......... 31%  172M 0s\n",
            "  6450K .......... .......... .......... .......... .......... 31%  209M 0s\n",
            "  6500K .......... .......... .......... .......... .......... 31%  205M 0s\n",
            "  6550K .......... .......... .......... .......... .......... 32%  148M 0s\n",
            "  6600K .......... .......... .......... .......... .......... 32%  195M 0s\n",
            "  6650K .......... .......... .......... .......... .......... 32%  211M 0s\n",
            "  6700K .......... .......... .......... .......... .......... 32%  194M 0s\n",
            "  6750K .......... .......... .......... .......... .......... 33%  131M 0s\n",
            "  6800K .......... .......... .......... .......... .......... 33%  181M 0s\n",
            "  6850K .......... .......... .......... .......... .......... 33%  189M 0s\n",
            "  6900K .......... .......... .......... .......... .......... 33%  139M 0s\n",
            "  6950K .......... .......... .......... .......... .......... 34% 93.7M 0s\n",
            "  7000K .......... .......... .......... .......... .......... 34%  153M 0s\n",
            "  7050K .......... .......... .......... .......... .......... 34%  204M 0s\n",
            "  7100K .......... .......... .......... .......... .......... 34%  202M 0s\n",
            "  7150K .......... .......... .......... .......... .......... 35%  180M 0s\n",
            "  7200K .......... .......... .......... .......... .......... 35%  134M 0s\n",
            "  7250K .......... .......... .......... .......... .......... 35%  152M 0s\n",
            "  7300K .......... .......... .......... .......... .......... 35%  168M 0s\n",
            "  7350K .......... .......... .......... .......... .......... 36%  131M 0s\n",
            "  7400K .......... .......... .......... .......... .......... 36%  176M 0s\n",
            "  7450K .......... .......... .......... .......... .......... 36%  144M 0s\n",
            "  7500K .......... .......... .......... .......... .......... 36%  166M 0s\n",
            "  7550K .......... .......... .......... .......... .......... 37%  193M 0s\n",
            "  7600K .......... .......... .......... .......... .......... 37%  195M 0s\n",
            "  7650K .......... .......... .......... .......... .......... 37%  181M 0s\n",
            "  7700K .......... .......... .......... .......... .......... 37%  175M 0s\n",
            "  7750K .......... .......... .......... .......... .......... 37%  128M 0s\n",
            "  7800K .......... .......... .......... .......... .......... 38%  174M 0s\n",
            "  7850K .......... .......... .......... .......... .......... 38%  173M 0s\n",
            "  7900K .......... .......... .......... .......... .......... 38%  200M 0s\n",
            "  7950K .......... .......... .......... .......... .......... 38%  172M 0s\n",
            "  8000K .......... .......... .......... .......... .......... 39%  206M 0s\n",
            "  8050K .......... .......... .......... .......... .......... 39%  158M 0s\n",
            "  8100K .......... .......... .......... .......... .......... 39%  190M 0s\n",
            "  8150K .......... .......... .......... .......... .......... 39%  163M 0s\n",
            "  8200K .......... .......... .......... .......... .......... 40%  198M 0s\n",
            "  8250K .......... .......... .......... .......... .......... 40%  195M 0s\n",
            "  8300K .......... .......... .......... .......... .......... 40%  199M 0s\n",
            "  8350K .......... .......... .......... .......... .......... 40%  184M 0s\n",
            "  8400K .......... .......... .......... .......... .......... 41%  219M 0s\n",
            "  8450K .......... .......... .......... .......... .......... 41%  157M 0s\n",
            "  8500K .......... .......... .......... .......... .......... 41%  208M 0s\n",
            "  8550K .......... .......... .......... .......... .......... 41%  203M 0s\n",
            "  8600K .......... .......... .......... .......... .......... 42%  154M 0s\n",
            "  8650K .......... .......... .......... .......... .......... 42%  184M 0s\n",
            "  8700K .......... .......... .......... .......... .......... 42%  206M 0s\n",
            "  8750K .......... .......... .......... .......... .......... 42%  212M 0s\n",
            "  8800K .......... .......... .......... .......... .......... 43%  163M 0s\n",
            "  8850K .......... .......... .......... .......... .......... 43%  201M 0s\n",
            "  8900K .......... .......... .......... .......... .......... 43%  213M 0s\n",
            "  8950K .......... .......... .......... .......... .......... 43%  189M 0s\n",
            "  9000K .......... .......... .......... .......... .......... 44%  181M 0s\n",
            "  9050K .......... .......... .......... .......... .......... 44%  204M 0s\n",
            "  9100K .......... .......... .......... .......... .......... 44%  212M 0s\n",
            "  9150K .......... .......... .......... .......... .......... 44%  178M 0s\n",
            "  9200K .......... .......... .......... .......... .......... 45%  188M 0s\n",
            "  9250K .......... .......... .......... .......... .......... 45%  216M 0s\n",
            "  9300K .......... .......... .......... .......... .......... 45%  190M 0s\n",
            "  9350K .......... .......... .......... .......... .......... 45%  216M 0s\n",
            "  9400K .......... .......... .......... .......... .......... 46%  214M 0s\n",
            "  9450K .......... .......... .......... .......... .......... 46%  171M 0s\n",
            "  9500K .......... .......... .......... .......... .......... 46%  213M 0s\n",
            "  9550K .......... .......... .......... .......... .......... 46%  207M 0s\n",
            "  9600K .......... .......... .......... .......... .......... 46%  182M 0s\n",
            "  9650K .......... .......... .......... .......... .......... 47%  188M 0s\n",
            "  9700K .......... .......... .......... .......... .......... 47%  211M 0s\n",
            "  9750K .......... .......... .......... .......... .......... 47%  181M 0s\n",
            "  9800K .......... .......... .......... .......... .......... 47%  185M 0s\n",
            "  9850K .......... .......... .......... .......... .......... 48%  130M 0s\n",
            "  9900K .......... .......... .......... .......... .......... 48%  143M 0s\n",
            "  9950K .......... .......... .......... .......... .......... 48% 94.2M 0s\n",
            " 10000K .......... .......... .......... .......... .......... 48%  178M 0s\n",
            " 10050K .......... .......... .......... .......... .......... 49%  178M 0s\n",
            " 10100K .......... .......... .......... .......... .......... 49%  131M 0s\n",
            " 10150K .......... .......... .......... .......... .......... 49%  178M 0s\n",
            " 10200K .......... .......... .......... .......... .......... 49%  189M 0s\n",
            " 10250K .......... .......... .......... .......... .......... 50%  215M 0s\n",
            " 10300K .......... .......... .......... .......... .......... 50%  159M 0s\n",
            " 10350K .......... .......... .......... .......... .......... 50%  179M 0s\n",
            " 10400K .......... .......... .......... .......... .......... 50%  181M 0s\n",
            " 10450K .......... .......... .......... .......... .......... 51%  154M 0s\n",
            " 10500K .......... .......... .......... .......... .......... 51%  167M 0s\n",
            " 10550K .......... .......... .......... .......... .......... 51%  181M 0s\n",
            " 10600K .......... .......... .......... .......... .......... 51%  200M 0s\n",
            " 10650K .......... .......... .......... .......... .......... 52%  149M 0s\n",
            " 10700K .......... .......... .......... .......... .......... 52%  178M 0s\n",
            " 10750K .......... .......... .......... .......... .......... 52%  188M 0s\n",
            " 10800K .......... .......... .......... .......... .......... 52%  174M 0s\n",
            " 10850K .......... .......... .......... .......... .......... 53%  201M 0s\n",
            " 10900K .......... .......... .......... .......... .......... 53%  167M 0s\n",
            " 10950K .......... .......... .......... .......... .......... 53%  224M 0s\n",
            " 11000K .......... .......... .......... .......... .......... 53%  166M 0s\n",
            " 11050K .......... .......... .......... .......... .......... 54%  162M 0s\n",
            " 11100K .......... .......... .......... .......... .......... 54%  147M 0s\n",
            " 11150K .......... .......... .......... .......... .......... 54%  145M 0s\n",
            " 11200K .......... .......... .......... .......... .......... 54%  171M 0s\n",
            " 11250K .......... .......... .......... .......... .......... 55%  188M 0s\n",
            " 11300K .......... .......... .......... .......... .......... 55%  160M 0s\n",
            " 11350K .......... .......... .......... .......... .......... 55%  197M 0s\n",
            " 11400K .......... .......... .......... .......... .......... 55%  211M 0s\n",
            " 11450K .......... .......... .......... .......... .......... 55%  210M 0s\n",
            " 11500K .......... .......... .......... .......... .......... 56%  176M 0s\n",
            " 11550K .......... .......... .......... .......... .......... 56%  166M 0s\n",
            " 11600K .......... .......... .......... .......... .......... 56%  203M 0s\n",
            " 11650K .......... .......... .......... .......... .......... 56%  212M 0s\n",
            " 11700K .......... .......... .......... .......... .......... 57%  181M 0s\n",
            " 11750K .......... .......... .......... .......... .......... 57%  190M 0s\n",
            " 11800K .......... .......... .......... .......... .......... 57%  202M 0s\n",
            " 11850K .......... .......... .......... .......... .......... 57%  207M 0s\n",
            " 11900K .......... .......... .......... .......... .......... 58%  165M 0s\n",
            " 11950K .......... .......... .......... .......... .......... 58%  201M 0s\n",
            " 12000K .......... .......... .......... .......... .......... 58%  214M 0s\n",
            " 12050K .......... .......... .......... .......... .......... 58%  219M 0s\n",
            " 12100K .......... .......... .......... .......... .......... 59%  188M 0s\n",
            " 12150K .......... .......... .......... .......... .......... 59%  182M 0s\n",
            " 12200K .......... .......... .......... .......... .......... 59%  220M 0s\n",
            " 12250K .......... .......... .......... .......... .......... 59%  161M 0s\n",
            " 12300K .......... .......... .......... .......... .......... 60%  156M 0s\n",
            " 12350K .......... .......... .......... .......... .......... 60%  213M 0s\n",
            " 12400K .......... .......... .......... .......... .......... 60%  203M 0s\n",
            " 12450K .......... .......... .......... .......... .......... 60%  222M 0s\n",
            " 12500K .......... .......... .......... .......... .......... 61%  177M 0s\n",
            " 12550K .......... .......... .......... .......... .......... 61%  198M 0s\n",
            " 12600K .......... .......... .......... .......... .......... 61%  210M 0s\n",
            " 12650K .......... .......... .......... .......... .......... 61%  198M 0s\n",
            " 12700K .......... .......... .......... .......... .......... 62% 73.5M 0s\n",
            " 12750K .......... .......... .......... .......... .......... 62%  236M 0s\n",
            " 12800K .......... .......... .......... .......... .......... 62%  207M 0s\n",
            " 12850K .......... .......... .......... .......... .......... 62%  198M 0s\n",
            " 12900K .......... .......... .......... .......... .......... 63%  164M 0s\n",
            " 12950K .......... .......... .......... .......... .......... 63%  243M 0s\n",
            " 13000K .......... .......... .......... .......... .......... 63%  269M 0s\n",
            " 13050K .......... .......... .......... .......... .......... 63%  212M 0s\n",
            " 13100K .......... .......... .......... .......... .......... 64%  197M 0s\n",
            " 13150K .......... .......... .......... .......... .......... 64%  270M 0s\n",
            " 13200K .......... .......... .......... .......... .......... 64%  247M 0s\n",
            " 13250K .......... .......... .......... .......... .......... 64%  238M 0s\n",
            " 13300K .......... .......... .......... .......... .......... 65%  172M 0s\n",
            " 13350K .......... .......... .......... .......... .......... 65%  230M 0s\n",
            " 13400K .......... .......... .......... .......... .......... 65%  268M 0s\n",
            " 13450K .......... .......... .......... .......... .......... 65% 70.7M 0s\n",
            " 13500K .......... .......... .......... .......... .......... 65%  178M 0s\n",
            " 13550K .......... .......... .......... .......... .......... 66%  268M 0s\n",
            " 13600K .......... .......... .......... .......... .......... 66%  118M 0s\n",
            " 13650K .......... .......... .......... .......... .......... 66%  214M 0s\n",
            " 13700K .......... .......... .......... .......... .......... 66%  181M 0s\n",
            " 13750K .......... .......... .......... .......... .......... 67%  214M 0s\n",
            " 13800K .......... .......... .......... .......... .......... 67%  171M 0s\n",
            " 13850K .......... .......... .......... .......... .......... 67%  219M 0s\n",
            " 13900K .......... .......... .......... .......... .......... 67%  172M 0s\n",
            " 13950K .......... .......... .......... .......... .......... 68%  205M 0s\n",
            " 14000K .......... .......... .......... .......... .......... 68%  188M 0s\n",
            " 14050K .......... .......... .......... .......... .......... 68%  206M 0s\n",
            " 14100K .......... .......... .......... .......... .......... 68%  188M 0s\n",
            " 14150K .......... .......... .......... .......... .......... 69%  225M 0s\n",
            " 14200K .......... .......... .......... .......... .......... 69%  201M 0s\n",
            " 14250K .......... .......... .......... .......... .......... 69%  216M 0s\n",
            " 14300K .......... .......... .......... .......... .......... 69%  156M 0s\n",
            " 14350K .......... .......... .......... .......... .......... 70%  181M 0s\n",
            " 14400K .......... .......... .......... .......... .......... 70%  158M 0s\n",
            " 14450K .......... .......... .......... .......... .......... 70%  193M 0s\n",
            " 14500K .......... .......... .......... .......... .......... 70%  167M 0s\n",
            " 14550K .......... .......... .......... .......... .......... 71%  201M 0s\n",
            " 14600K .......... .......... .......... .......... .......... 71%  191M 0s\n",
            " 14650K .......... .......... .......... .......... .......... 71%  132M 0s\n",
            " 14700K .......... .......... .......... .......... .......... 71%  113M 0s\n",
            " 14750K .......... .......... .......... .......... .......... 72%  161M 0s\n",
            " 14800K .......... .......... .......... .......... .......... 72%  165M 0s\n",
            " 14850K .......... .......... .......... .......... .......... 72%  161M 0s\n",
            " 14900K .......... .......... .......... .......... .......... 72%  148M 0s\n",
            " 14950K .......... .......... .......... .......... .......... 73%  169M 0s\n",
            " 15000K .......... .......... .......... .......... .......... 73%  199M 0s\n",
            " 15050K .......... .......... .......... .......... .......... 73%  154M 0s\n",
            " 15100K .......... .......... .......... .......... .......... 73%  139M 0s\n",
            " 15150K .......... .......... .......... .......... .......... 74%  175M 0s\n",
            " 15200K .......... .......... .......... .......... .......... 74%  100M 0s\n",
            " 15250K .......... .......... .......... .......... .......... 74%  169M 0s\n",
            " 15300K .......... .......... .......... .......... .......... 74%  101M 0s\n",
            " 15350K .......... .......... .......... .......... .......... 74%  179M 0s\n",
            " 15400K .......... .......... .......... .......... .......... 75%  202M 0s\n",
            " 15450K .......... .......... .......... .......... .......... 75%  200M 0s\n",
            " 15500K .......... .......... .......... .......... .......... 75%  123M 0s\n",
            " 15550K .......... .......... .......... .......... .......... 75%  165M 0s\n",
            " 15600K .......... .......... .......... .......... .......... 76%  174M 0s\n",
            " 15650K .......... .......... .......... .......... .......... 76%  178M 0s\n",
            " 15700K .......... .......... .......... .......... .......... 76%  148M 0s\n",
            " 15750K .......... .......... .......... .......... .......... 76%  181M 0s\n",
            " 15800K .......... .......... .......... .......... .......... 77%  189M 0s\n",
            " 15850K .......... .......... .......... .......... .......... 77%  155M 0s\n",
            " 15900K .......... .......... .......... .......... .......... 77%  168M 0s\n",
            " 15950K .......... .......... .......... .......... .......... 77%  200M 0s\n",
            " 16000K .......... .......... .......... .......... .......... 78%  159M 0s\n",
            " 16050K .......... .......... .......... .......... .......... 78%  194M 0s\n",
            " 16100K .......... .......... .......... .......... .......... 78%  167M 0s\n",
            " 16150K .......... .......... .......... .......... .......... 78%  174M 0s\n",
            " 16200K .......... .......... .......... .......... .......... 79%  167M 0s\n",
            " 16250K .......... .......... .......... .......... .......... 79%  193M 0s\n",
            " 16300K .......... .......... .......... .......... .......... 79%  157M 0s\n",
            " 16350K .......... .......... .......... .......... .......... 79%  174M 0s\n",
            " 16400K .......... .......... .......... .......... .......... 80%  199M 0s\n",
            " 16450K .......... .......... .......... .......... .......... 80%  205M 0s\n",
            " 16500K .......... .......... .......... .......... .......... 80%  166M 0s\n",
            " 16550K .......... .......... .......... .......... .......... 80%  152M 0s\n",
            " 16600K .......... .......... .......... .......... .......... 81%  195M 0s\n",
            " 16650K .......... .......... .......... .......... .......... 81%  207M 0s\n",
            " 16700K .......... .......... .......... .......... .......... 81%  178M 0s\n",
            " 16750K .......... .......... .......... .......... .......... 81%  188M 0s\n",
            " 16800K .......... .......... .......... .......... .......... 82%  203M 0s\n",
            " 16850K .......... .......... .......... .......... .......... 82%  199M 0s\n",
            " 16900K .......... .......... .......... .......... .......... 82%  180M 0s\n",
            " 16950K .......... .......... .......... .......... .......... 82%  189M 0s\n",
            " 17000K .......... .......... .......... .......... .......... 83%  208M 0s\n",
            " 17050K .......... .......... .......... .......... .......... 83%  216M 0s\n",
            " 17100K .......... .......... .......... .......... .......... 83%  171M 0s\n",
            " 17150K .......... .......... .......... .......... .......... 83%  181M 0s\n",
            " 17200K .......... .......... .......... .......... .......... 83%  119M 0s\n",
            " 17250K .......... .......... .......... .......... .......... 84%  175M 0s\n",
            " 17300K .......... .......... .......... .......... .......... 84%  111M 0s\n",
            " 17350K .......... .......... .......... .......... .......... 84%  187M 0s\n",
            " 17400K .......... .......... .......... .......... .......... 84%  181M 0s\n",
            " 17450K .......... .......... .......... .......... .......... 85%  149M 0s\n",
            " 17500K .......... .......... .......... .......... .......... 85%  153M 0s\n",
            " 17550K .......... .......... .......... .......... .......... 85%  159M 0s\n",
            " 17600K .......... .......... .......... .......... .......... 85%  140M 0s\n",
            " 17650K .......... .......... .......... .......... .......... 86%  144M 0s\n",
            " 17700K .......... .......... .......... .......... .......... 86%  179M 0s\n",
            " 17750K .......... .......... .......... .......... .......... 86%  197M 0s\n",
            " 17800K .......... .......... .......... .......... .......... 86%  209M 0s\n",
            " 17850K .......... .......... .......... .......... .......... 87%  163M 0s\n",
            " 17900K .......... .......... .......... .......... .......... 87%  201M 0s\n",
            " 17950K .......... .......... .......... .......... .......... 87%  190M 0s\n",
            " 18000K .......... .......... .......... .......... .......... 87%  193M 0s\n",
            " 18050K .......... .......... .......... .......... .......... 88%  148M 0s\n",
            " 18100K .......... .......... .......... .......... .......... 88%  202M 0s\n",
            " 18150K .......... .......... .......... .......... .......... 88%  148M 0s\n",
            " 18200K .......... .......... .......... .......... .......... 88%  175M 0s\n",
            " 18250K .......... .......... .......... .......... .......... 89%  142M 0s\n",
            " 18300K .......... .......... .......... .......... .......... 89%  160M 0s\n",
            " 18350K .......... .......... .......... .......... .......... 89%  174M 0s\n",
            " 18400K .......... .......... .......... .......... .......... 89%  179M 0s\n",
            " 18450K .......... .......... .......... .......... .......... 90%  174M 0s\n",
            " 18500K .......... .......... .......... .......... .......... 90%  171M 0s\n",
            " 18550K .......... .......... .......... .......... .......... 90%  159M 0s\n",
            " 18600K .......... .......... .......... .......... .......... 90%  166M 0s\n",
            " 18650K .......... .......... .......... .......... .......... 91%  163M 0s\n",
            " 18700K .......... .......... .......... .......... .......... 91%  152M 0s\n",
            " 18750K .......... .......... .......... .......... .......... 91%  183M 0s\n",
            " 18800K .......... .......... .......... .......... .......... 91%  179M 0s\n",
            " 18850K .......... .......... .......... .......... .......... 92%  168M 0s\n",
            " 18900K .......... .......... .......... .......... .......... 92%  128M 0s\n",
            " 18950K .......... .......... .......... .......... .......... 92%  105M 0s\n",
            " 19000K .......... .......... .......... .......... .......... 92%  168M 0s\n",
            " 19050K .......... .......... .......... .......... .......... 92%  159M 0s\n",
            " 19100K .......... .......... .......... .......... .......... 93%  119M 0s\n",
            " 19150K .......... .......... .......... .......... .......... 93%  156M 0s\n",
            " 19200K .......... .......... .......... .......... .......... 93%  153M 0s\n",
            " 19250K .......... .......... .......... .......... .......... 93%  148M 0s\n",
            " 19300K .......... .......... .......... .......... .......... 94%  149M 0s\n",
            " 19350K .......... .......... .......... .......... .......... 94%  154M 0s\n",
            " 19400K .......... .......... .......... .......... .......... 94%  145M 0s\n",
            " 19450K .......... .......... .......... .......... .......... 94%  164M 0s\n",
            " 19500K .......... .......... .......... .......... .......... 95%  156M 0s\n",
            " 19550K .......... .......... .......... .......... .......... 95%  209M 0s\n",
            " 19600K .......... .......... .......... .......... .......... 95%  186M 0s\n",
            " 19650K .......... .......... .......... .......... .......... 95%  201M 0s\n",
            " 19700K .......... .......... .......... .......... .......... 96% 91.6M 0s\n",
            " 19750K .......... .......... .......... .......... .......... 96%  153M 0s\n",
            " 19800K .......... .......... .......... .......... .......... 96%  161M 0s\n",
            " 19850K .......... .......... .......... .......... .......... 96%  169M 0s\n",
            " 19900K .......... .......... .......... .......... .......... 97%  127M 0s\n",
            " 19950K .......... .......... .......... .......... .......... 97%  199M 0s\n",
            " 20000K .......... .......... .......... .......... .......... 97%  187M 0s\n",
            " 20050K .......... .......... .......... .......... .......... 97%  188M 0s\n",
            " 20100K .......... .......... .......... .......... .......... 98%  196M 0s\n",
            " 20150K .......... .......... .......... .......... .......... 98%  199M 0s\n",
            " 20200K .......... .......... .......... .......... .......... 98%  207M 0s\n",
            " 20250K .......... .......... .......... .......... .......... 98%  160M 0s\n",
            " 20300K .......... .......... .......... .......... .......... 99%  173M 0s\n",
            " 20350K .......... .......... .......... .......... .......... 99%  208M 0s\n",
            " 20400K .......... .......... .......... .......... .......... 99%  204M 0s\n",
            " 20450K .......... .......... .......... .......... .......... 99% 4.76M 0s\n",
            " 20500K .......... .......... .......... ........             100%  139M=0.2s\n",
            "\n",
            "2024-12-10 14:46:46 (103 MB/s) - ‚Äòxtb-6.7.0-linux-x86_64.tar.xz‚Äô saved [21031364/21031364]\n",
            "\n"
          ]
        }
      ],
      "source": [
        "# Baixar e instalar o xTB\n",
        "%%bash\n",
        "wget https://github.com/grimme-lab/xtb/releases/download/v6.7.0/xtb-6.7.0-linux-x86_64.tar.xz\n",
        "tar -xvJf xtb-6.7.0-linux-x86_64.tar.xz\n",
        "export PATH=$PATH:$(pwd)/xtb-dist/bin"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!/content/xtb-dist/bin/xtb --version"
      ],
      "metadata": {
        "id": "4kIzqK27Z5O-",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "342ee136-41c1-4e9f-9881-6244c6ca6d80"
      },
      "execution_count": 41,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "      -----------------------------------------------------------      \n",
            "     |                   =====================                   |     \n",
            "     |                           x T B                           |     \n",
            "     |                   =====================                   |     \n",
            "     |                         S. Grimme                         |     \n",
            "     |          Mulliken Center for Theoretical Chemistry        |     \n",
            "     |                    University of Bonn                     |     \n",
            "      -----------------------------------------------------------      \n",
            "\n",
            "   * xtb version 6.7.0 (08769fc) compiled by 'albert@albert-system' on 2024-03-04\n",
            "\n",
            "normal termination of xtb\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "import os\n",
        "os.environ['PATH'] += ':/content/xtb-dist/bin'\n",
        "!xtb --version"
      ],
      "metadata": {
        "id": "WOz2WbBDDwgO",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "52ad6004-2dc3-4b0c-fb97-eaa1c8fe1614"
      },
      "execution_count": 42,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "      -----------------------------------------------------------      \n",
            "     |                   =====================                   |     \n",
            "     |                           x T B                           |     \n",
            "     |                   =====================                   |     \n",
            "     |                         S. Grimme                         |     \n",
            "     |          Mulliken Center for Theoretical Chemistry        |     \n",
            "     |                    University of Bonn                     |     \n",
            "      -----------------------------------------------------------      \n",
            "\n",
            "   * xtb version 6.7.0 (08769fc) compiled by 'albert@albert-system' on 2024-03-04\n",
            "\n",
            "normal termination of xtb\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "---------------------\n",
        "##### ***Scan* de coordenadas com o xTB**"
      ],
      "metadata": {
        "id": "mSzaDbhmP8I4"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "%%file ethane.xyz\n",
        "8\n",
        "-7.46994680\n",
        "C          -0.01503441120750    0.04820403778911   -0.01075686629161\n",
        "H          -0.02618280545732    0.12324247717853    1.07751478989514\n",
        "H           1.02025706611374    0.10521724412205   -0.34995019287160\n",
        "H          -0.56108089374185    0.89840506905011   -0.42160774836772\n",
        "C          -0.65104926960692   -1.26213120711759   -0.46045958867132\n",
        "H          -1.67328207424848   -1.33815676349645   -0.08720971228148\n",
        "H          -0.67543548798470   -1.32088660386766   -1.54947400918083\n",
        "H          -0.08161812386697   -2.11204925365809   -0.08140067223057"
      ],
      "metadata": {
        "id": "kUmbji51Z5XC",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "0408814f-1a87-45d2-bbe6-a213755aec16"
      },
      "execution_count": 43,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Writing ethane.xyz\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "%%file scan.inp\n",
        "$constrain\n",
        " force constant=0.05\n",
        "$scan\n",
        " dihedral: 8,5,1,4,60.0; 60.0,420.0,72\n",
        "$end"
      ],
      "metadata": {
        "id": "7cVgPqtNb1zl",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "e845ea51-9315-4d41-a140-0e60f9feff99"
      },
      "execution_count": 44,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Writing scan.inp\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!xtb ethane.xyz --opt --input scan.inp"
      ],
      "metadata": {
        "id": "BMZf-gpfb18O",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "4e041b1d-d9f5-4ce4-c679-6890dec9e4b1"
      },
      "execution_count": 45,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "      -----------------------------------------------------------      \n",
            "     |                   =====================                   |     \n",
            "     |                           x T B                           |     \n",
            "     |                   =====================                   |     \n",
            "     |                         S. Grimme                         |     \n",
            "     |          Mulliken Center for Theoretical Chemistry        |     \n",
            "     |                    University of Bonn                     |     \n",
            "      -----------------------------------------------------------      \n",
            "\n",
            "   * xtb version 6.7.0 (08769fc) compiled by 'albert@albert-system' on 2024-03-04\n",
            "\n",
            "   xtb is free software: you can redistribute it and/or modify it under\n",
            "   the terms of the GNU Lesser General Public License as published by\n",
            "   the Free Software Foundation, either version 3 of the License, or\n",
            "   (at your option) any later version.\n",
            "   \n",
            "   xtb is distributed in the hope that it will be useful,\n",
            "   but WITHOUT ANY WARRANTY; without even the implied warranty of\n",
            "   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the\n",
            "   GNU Lesser General Public License for more details.\n",
            "   \n",
            "   Cite this work as:\n",
            "   * C. Bannwarth, E. Caldeweyher, S. Ehlert, A. Hansen, P. Pracht,\n",
            "     J. Seibert, S. Spicher, S. Grimme, WIREs Comput. Mol. Sci., 2020, 11,\n",
            "     e01493. DOI: 10.1002/wcms.1493\n",
            "   \n",
            "   for GFN2-xTB:\n",
            "   * C. Bannwarth, S. Ehlert and S. Grimme., J. Chem. Theory Comput., 2019,\n",
            "     15, 1652-1671. DOI: 10.1021/acs.jctc.8b01176\n",
            "   for GFN1-xTB:\n",
            "   * S. Grimme, C. Bannwarth, P. Shushkov, J. Chem. Theory Comput., 2017,\n",
            "     13, 1989-2009. DOI: 10.1021/acs.jctc.7b00118\n",
            "   for GFN0-xTB:\n",
            "   * P. Pracht, E. Caldeweyher, S. Ehlert, S. Grimme, ChemRxiv, 2019, preprint.\n",
            "     DOI: 10.26434/chemrxiv.8326202.v1\n",
            "   for GFN-FF:\n",
            "   * S. Spicher and S. Grimme, Angew. Chem. Int. Ed., 2020, 59, 15665-15673.\n",
            "     DOI: 10.1002/anie.202004239\n",
            "   \n",
            "   for ALPB and GBSA implicit solvation:\n",
            "   * S. Ehlert, M. Stahn, S. Spicher, S. Grimme, J. Chem. Theory Comput.,\n",
            "     2021, 17, 4250-4261. DOI: 10.1021/acs.jctc.1c00471\n",
            "   \n",
            "   for ddCOSMO and CPCM-X implicit solvation:\n",
            "   * M. Stahn, S. Ehlert, S. Grimme, J. Phys. Chem. A,\n",
            "     2023, XX, XXXX-XXXX. DOI: 10.1021/acs.jpca.3c04382\n",
            "   \n",
            "   for DFT-D4:\n",
            "   * E. Caldeweyher, C. Bannwarth and S. Grimme, J. Chem. Phys., 2017,\n",
            "     147, 034112. DOI: 10.1063/1.4993215\n",
            "   * E. Caldeweyher, S. Ehlert, A. Hansen, H. Neugebauer, S. Spicher,\n",
            "     C. Bannwarth and S. Grimme, J. Chem. Phys., 2019, 150, 154122.\n",
            "     DOI: 10.1063/1.5090222\n",
            "   * E. Caldeweyher, J.-M. Mewes, S. Ehlert and S. Grimme, Phys. Chem. Chem. Phys.\n",
            "     2020, 22, 8499-8512. DOI: 10.1039/D0CP00502A\n",
            "   \n",
            "   for sTDA-xTB:\n",
            "   * S. Grimme and C. Bannwarth, J. Chem. Phys., 2016, 145, 054103.\n",
            "     DOI: 10.1063/1.4959605\n",
            "   \n",
            "   in the mass-spec context:\n",
            "   * V. Asgeirsson, C. Bauer and S. Grimme, Chem. Sci., 2017, 8, 4879.\n",
            "     DOI: 10.1039/c7sc00601b\n",
            "   * J. Koopman and S. Grimme, ACS Omega 2019, 4, 12, 15120-15133.\n",
            "     DOI: 10.1021/acsomega.9b02011\n",
            "   \n",
            "   for metadynamics refer to:\n",
            "   * S. Grimme, J. Chem. Theory Comput., 2019, 155, 2847-2862\n",
            "     DOI: 10.1021/acs.jctc.9b00143\n",
            "   \n",
            "   for SPH calculations refer to:\n",
            "   * S. Spicher and S. Grimme, J. Chem. Theory Comput., 2021, 17, 1701-1714\n",
            "     DOI: 10.1021/acs.jctc.0c01306\n",
            "   \n",
            "   for ONIOM refer to:\n",
            "   * C. Plett, A. Katbashev, S. Ehlert, S. Grimme, M. Bursch,\n",
            "     Phys. Chem. Chem. Phys., 2023, 25, 17860-17868. DOI: 10.1039/D3CP02178E\n",
            "   \n",
            "   for DIPRO refer to:\n",
            "   * J. Kohn, N. Gildemeister, S. Grimme, D. Fazzi, A. Hansen,\n",
            "     J. Chem. Phys., 2023, just accepted.\n",
            "   \n",
            "   for PTB refer to:\n",
            "   * S. Grimme, M. Mueller, A. Hansen, J. Chem. Phys., 2023, 158, 124111.\n",
            "     DOI: 10.1063/5.0137838\n",
            "   \n",
            "   with help from (in alphabetical order)\n",
            "   P. Atkinson, C. Bannwarth, F. Bohle, G. Brandenburg, E. Caldeweyher\n",
            "   M. Checinski, S. Dohm, S. Ehlert, S. Ehrlich, I. Gerasimov, C. H√∂lzer\n",
            "   A. Katbashev, J. Kohn, J. Koopman, C. Lavigne, S. Lehtola, F. M√§rz, M. M√ºller,\n",
            "   F. Musil, H. Neugebauer, J. Pisarek, C. Plett, P. Pracht, F. Pultar,\n",
            "   J. Seibert, P. Shushkov, S. Spicher, M. Stahn, M. Steiner, T. Strunk,\n",
            "   J. St√ºckrath, T. Rose, and J. Unsleber\n",
            "   \n",
            " * started run on 2024/12/10 at 14:47:06.098     \n",
            "   ID    Z sym.   atoms\n",
            "    1    6 C      1, 5\n",
            "    2    1 H      2-4, 6-8\n",
            "constraining angle 8 5 1 4 to   60.0000000¬∞, actual value:  177.7877179¬∞\n",
            "\n",
            "           -------------------------------------------------\n",
            "          |                Calculation Setup                |\n",
            "           -------------------------------------------------\n",
            "\n",
            "          program call               : xtb ethane.xyz --opt --input scan.inp\n",
            "          hostname                   : ae2b5db7b212\n",
            "          coordinate file            : ethane.xyz\n",
            "          omp threads                :                     2\n",
            "\n",
            "\n",
            "           -------------------------------------------------\n",
            "          |                 G F N 2 - x T B                 |\n",
            "           -------------------------------------------------\n",
            "\n",
            "        Reference                      10.1021/acs.jctc.8b01176\n",
            "      * Hamiltonian:\n",
            "        H0-scaling (s, p, d)           1.850000    2.230000    2.230000\n",
            "        zeta-weighting                 0.500000\n",
            "      * Dispersion:\n",
            "        s8                             2.700000\n",
            "        a1                             0.520000\n",
            "        a2                             5.000000\n",
            "        s9                             5.000000\n",
            "      * Repulsion:\n",
            "        kExp                           1.500000    1.000000\n",
            "        rExp                           1.000000\n",
            "      * Coulomb:\n",
            "        alpha                          2.000000\n",
            "        third order                    shell-resolved\n",
            "        anisotropic                    true\n",
            "        a3                             3.000000\n",
            "        a5                             4.000000\n",
            "        cn-shift                       1.200000\n",
            "        cn-exp                         4.000000\n",
            "        max-rad                        5.000000\n",
            "\n",
            "\n",
            "          ...................................................\n",
            "          :                      SETUP                      :\n",
            "          :.................................................:\n",
            "          :  # basis functions                  14          :\n",
            "          :  # atomic orbitals                  14          :\n",
            "          :  # shells                           10          :\n",
            "          :  # electrons                        14          :\n",
            "          :  max. iterations                   250          :\n",
            "          :  Hamiltonian                  GFN2-xTB          :\n",
            "          :  restarted?                      false          :\n",
            "          :  GBSA solvation                  false          :\n",
            "          :  PC potential                    false          :\n",
            "          :  electronic temp.          300.0000000     K    :\n",
            "          :  accuracy                    1.0000000          :\n",
            "          :  -> integral cutoff          0.2500000E+02      :\n",
            "          :  -> integral neglect         0.1000000E-07      :\n",
            "          :  -> SCF convergence          0.1000000E-05 Eh   :\n",
            "          :  -> wf. convergence          0.1000000E-03 e    :\n",
            "          :  Broyden damping             0.4000000          :\n",
            "          :  net charge                          0          :\n",
            "          :  unpaired electrons                  0          :\n",
            "          ...................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1     -7.4265878 -0.742659E+01  0.262E+00   15.88       0.0  T\n",
            "   2     -7.4507915 -0.242037E-01  0.124E+00   15.42       1.0  T\n",
            "   3     -7.4510101 -0.218613E-03  0.670E-01   15.40       1.0  T\n",
            "   4     -7.4510797 -0.696134E-04  0.878E-02   15.40       1.0  T\n",
            "   5     -7.4510797  0.685975E-08  0.338E-03   15.40      10.4  T\n",
            "   6     -7.4510798 -0.556666E-07  0.105E-03   15.40      33.8  T\n",
            "   7     -7.4510798 -0.308322E-10  0.139E-04   15.40     254.5  T\n",
            "   8     -7.4510798 -0.926335E-10  0.354E-06   15.40    9990.9  T\n",
            "\n",
            "   *** convergence criteria satisfied after 8 iterations ***\n",
            "\n",
            "         #    Occupation            Energy/Eh            Energy/eV\n",
            "      -------------------------------------------------------------\n",
            "         1        2.0000           -0.6042203             -16.4417\n",
            "         2        2.0000           -0.5624662             -15.3055\n",
            "         3        2.0000           -0.4951704             -13.4743\n",
            "         4        2.0000           -0.4951572             -13.4739\n",
            "         5        2.0000           -0.4605069             -12.5310\n",
            "         6        2.0000           -0.4353830             -11.8474\n",
            "         7        2.0000           -0.4353601             -11.8467 (HOMO)\n",
            "         8                          0.1306756               3.5559 (LUMO)\n",
            "         9                          0.1637704               4.4564\n",
            "        10                          0.1638620               4.4589\n",
            "        11                          0.1683005               4.5797\n",
            "        12                          0.1953501               5.3157\n",
            "       ...                                ...                  ...\n",
            "        14                          0.2766231               7.5273\n",
            "      -------------------------------------------------------------\n",
            "                  HL-Gap            0.5660357 Eh           15.4026 eV\n",
            "             Fermi-level           -0.1523422 Eh           -4.1454 eV\n",
            "\n",
            " SCC (total)                   0 d,  0 h,  0 min,  0.005 sec\n",
            " SCC setup                      ...        0 min,  0.000 sec (  5.890%)\n",
            " Dispersion                     ...        0 min,  0.000 sec (  1.591%)\n",
            " classical contributions        ...        0 min,  0.000 sec (  0.601%)\n",
            " integral evaluation            ...        0 min,  0.000 sec (  8.742%)\n",
            " iterations                     ...        0 min,  0.003 sec ( 52.903%)\n",
            " molecular gradient             ...        0 min,  0.001 sec ( 24.644%)\n",
            " printout                       ...        0 min,  0.000 sec (  4.915%)\n",
            "\n",
            "         :::::::::::::::::::::::::::::::::::::::::::::::::::::\n",
            "         ::                     SUMMARY                     ::\n",
            "         :::::::::::::::::::::::::::::::::::::::::::::::::::::\n",
            "         :: total energy              -7.262997246804 Eh    ::\n",
            "         :: gradient norm              0.046053497466 Eh/a0 ::\n",
            "         :: HOMO-LUMO gap             15.402614534923 eV    ::\n",
            "         ::.................................................::\n",
            "         :: SCC energy                -7.451079622401 Eh    ::\n",
            "         :: -> isotropic ES            0.002011628863 Eh    ::\n",
            "         :: -> anisotropic ES          0.003181190577 Eh    ::\n",
            "         :: -> anisotropic XC          0.005517910051 Eh    ::\n",
            "         :: -> dispersion             -0.002034333409 Eh    ::\n",
            "         :: repulsion energy           0.114772525209 Eh    ::\n",
            "         :: add. restraining           0.073309850388 Eh    ::\n",
            "         :: total charge              -0.000000000000 e     ::\n",
            "         :::::::::::::::::::::::::::::::::::::::::::::::::::::\n",
            "\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            "\n",
            "          ...................................................\n",
            "          :                      SETUP                      :\n",
            "          :.................................................:\n",
            "          :   optimization level            normal          :\n",
            "          :   max. optcycles                   200          :\n",
            "          :   ANC micro-cycles                  20          :\n",
            "          :   degrees of freedom                18          :\n",
            "          :.................................................:\n",
            "          :   RF solver                   davidson          :\n",
            "          :   write xtbopt.log                true          :\n",
            "          :   linear?                        false          :\n",
            "          :   energy convergence         0.5000000E-05 Eh   :\n",
            "          :   grad. convergence          0.1000000E-02 Eh/Œ± :\n",
            "          :   maxmium RF displ.          1.0000000          :\n",
            "          :   Hlow (freq-cutoff)         0.1000000E-01      :\n",
            "          :   Hmax (freq-cutoff)         5.0000000          :\n",
            "          :   S6 in model hess.         20.0000000          :\n",
            "          ...................................................\n",
            "\n",
            "generating ANC from model Hessian ...\n",
            "Using Lindh-Hessian (1995)\n",
            " Shifting diagonal of input Hessian by   0.000000000000000E+000\n",
            " Lowest  eigenvalues of input Hessian\n",
            "    0.000000    0.000000    0.000000    0.000000    0.000000    0.000000\n",
            "    0.010443    0.038212    0.038222    0.057671    0.077338    0.077839\n",
            "    0.113574    0.123559    0.138285    0.139703    0.181728    0.341105\n",
            " Highest eigenvalues\n",
            "    0.655225    1.046423    1.054439    1.163058    1.165641    1.225088\n",
            " \n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE    1 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1     -7.4510798 -0.745108E+01  0.244E-06   15.40       0.0  T\n",
            "   2     -7.4510798 -0.230926E-13  0.961E-07   15.40   36779.4  T\n",
            "   3     -7.4510798  0.266454E-14  0.490E-07   15.40   72097.6  T\n",
            "     SCC iter.                  ...        0 min,  0.001 sec\n",
            "     gradient                   ...        0 min,  0.001 sec\n",
            " * total energy  :    -7.2629972 Eh     change        0.4440892E-14 Eh\n",
            "   gradient norm :     0.0460535 Eh/Œ±   predicted     0.0000000E+00 (-100.00%)\n",
            "   displ. norm   :     0.7083298 Œ±      lambda       -0.2240525E-01\n",
            "   maximum displ.:     0.6579854 Œ±      in ANC's #1, #7, #3, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE    2 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1     -7.4394998 -0.743950E+01  0.611E-01   14.47       0.0  T\n",
            "   2     -7.4395128 -0.130151E-04  0.364E-01   14.47       1.0  T\n",
            "   3     -7.4395169 -0.410006E-05  0.218E-02   14.45       1.6  T\n",
            "   4     -7.4395172 -0.288478E-06  0.254E-03   14.45      13.9  T\n",
            "   5     -7.4395172 -0.125186E-09  0.117E-03   14.45      30.1  T\n",
            "   6     -7.4395172 -0.217555E-09  0.731E-05   14.45     483.8  T\n",
            "   7     -7.4395172 -0.384759E-11  0.355E-05   14.45     995.1  T\n",
            "     SCC iter.                  ...        0 min,  0.001 sec\n",
            "     gradient                   ...        0 min,  0.001 sec\n",
            " * total energy  :    -7.2809674 Eh     change       -0.1797011E-01 Eh\n",
            "   gradient norm :     0.0363026 Eh/Œ±   predicted    -0.1682332E-01 (  -6.38%)\n",
            "   displ. norm   :     0.7305206 Œ±      lambda       -0.1373340E-01\n",
            "   maximum displ.:     0.7244711 Œ±      in ANC's #1, #12, #4, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE    3 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1     -7.4176069 -0.741761E+01  0.390E-01   12.77       0.0  T\n",
            "   2     -7.4176335 -0.266822E-04  0.229E-01   12.77       1.0  T\n",
            "   3     -7.4176353 -0.175916E-05  0.267E-02   12.76       1.3  T\n",
            "   4     -7.4176353 -0.539064E-07  0.816E-03   12.75       4.3  T\n",
            "   5     -7.4176354 -0.702742E-08  0.911E-04   12.75      38.8  T\n",
            "   6     -7.4176354 -0.252326E-09  0.790E-05   12.75     447.4  T\n",
            "     SCC iter.                  ...        0 min,  0.001 sec\n",
            "     gradient                   ...        0 min,  0.001 sec\n",
            " * total energy  :    -7.2903031 Eh     change       -0.9335759E-02 Eh\n",
            "   gradient norm :     0.0625628 Eh/Œ±   predicted    -0.1053119E-01 (  12.80%)\n",
            "   displ. norm   :     0.4524493 Œ±      lambda       -0.9819345E-02\n",
            "   maximum displ.:     0.4003638 Œ±      in ANC's #1, #12, #3, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE    4 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1     -7.4192241 -0.741922E+01  0.244E-01   12.99       0.0  T\n",
            "   2     -7.4192249 -0.794807E-06  0.146E-01   12.99       1.0  T\n",
            "   3     -7.4192252 -0.272022E-06  0.448E-03   13.00       7.9  T\n",
            "   4     -7.4192252 -0.572144E-08  0.220E-03   13.00      16.1  T\n",
            "   5     -7.4192252 -0.941036E-09  0.386E-04   13.00      91.5  T\n",
            "   6     -7.4192252 -0.369607E-10  0.221E-05   13.00    1598.6  T\n",
            "     SCC iter.                  ...        0 min,  0.001 sec\n",
            "     gradient                   ...        0 min,  0.001 sec\n",
            " * total energy  :    -7.2988549 Eh     change       -0.8551826E-02 Eh\n",
            "   gradient norm :     0.0555218 Eh/Œ±   predicted    -0.5914736E-02 ( -30.84%)\n",
            "   displ. norm   :     0.5481831 Œ±      lambda       -0.1661218E-01\n",
            "   maximum displ.:     0.3899366 Œ±      in ANC's #1, #12, #7, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE    5 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1     -7.4443788 -0.744438E+01  0.442E-01   14.37       0.0  T\n",
            "   2     -7.4444610 -0.822372E-04  0.253E-01   14.38       1.0  T\n",
            "   3     -7.4444642 -0.316626E-05  0.524E-02   14.40       1.0  T\n",
            "   4     -7.4444643 -0.855767E-07  0.110E-02   14.40       3.2  T\n",
            "   5     -7.4444643 -0.868448E-08  0.943E-04   14.40      37.5  T\n",
            "   6     -7.4444643 -0.204808E-09  0.230E-04   14.40     154.0  T\n",
            "     SCC iter.                  ...        0 min,  0.001 sec\n",
            "     gradient                   ...        0 min,  0.001 sec\n",
            " * total energy  :    -7.3091952 Eh     change       -0.1034030E-01 Eh\n",
            "   gradient norm :     0.0410178 Eh/Œ±   predicted    -0.1080212E-01 (   4.47%)\n",
            "   displ. norm   :     0.7603077 Œ±      lambda       -0.1222039E-01\n",
            "   maximum displ.:     0.7051501 Œ±      in ANC's #1, #12, #3, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE    6 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1     -7.4208519 -0.742085E+01  0.449E-01   13.42       0.0  T\n",
            "   2     -7.4209581 -0.106136E-03  0.256E-01   13.41       1.0  T\n",
            "   3     -7.4209614 -0.327367E-05  0.635E-02   13.40       1.0  T\n",
            "   4     -7.4209616 -0.222886E-06  0.114E-02   13.39       3.1  T\n",
            "   5     -7.4209616 -0.436080E-08  0.719E-04   13.40      49.2  T\n",
            "   6     -7.4209616 -0.380994E-09  0.216E-04   13.40     163.7  T\n",
            "     SCC iter.                  ...        0 min,  0.001 sec\n",
            "     gradient                   ...        0 min,  0.001 sec\n",
            " * total energy  :    -7.3171095 Eh     change       -0.7914243E-02 Eh\n",
            "   gradient norm :     0.0608630 Eh/Œ±   predicted    -0.9642289E-02 (  21.83%)\n",
            "   displ. norm   :     0.3843428 Œ±      lambda       -0.7700020E-02\n",
            "   maximum displ.:     0.2795084 Œ±      in ANC's #1, #12, #18, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE    7 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1     -7.4275798 -0.742758E+01  0.169E-01   13.64       0.0  T\n",
            "   2     -7.4275888 -0.902582E-05  0.981E-02   13.65       1.0  T\n",
            "   3     -7.4275895 -0.646147E-06  0.169E-02   13.65       2.1  T\n",
            "   4     -7.4275895 -0.675877E-08  0.479E-03   13.65       7.4  T\n",
            "   5     -7.4275895 -0.645221E-08  0.551E-04   13.65      64.1  T\n",
            "   6     -7.4275895 -0.697371E-10  0.589E-05   13.65     600.0  T\n",
            "     SCC iter.                  ...        0 min,  0.001 sec\n",
            "     gradient                   ...        0 min,  0.001 sec\n",
            " * total energy  :    -7.3229019 Eh     change       -0.5792383E-02 Eh\n",
            "   gradient norm :     0.0483005 Eh/Œ±   predicted    -0.4418733E-02 ( -23.71%)\n",
            "   displ. norm   :     0.4692280 Œ±      lambda       -0.1081189E-01\n",
            "   maximum displ.:     0.3318256 Œ±      in ANC's #12, #1, #18, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE    8 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1     -7.4481666 -0.744817E+01  0.364E-01   15.07       0.0  T\n",
            "   2     -7.4481899 -0.233132E-04  0.211E-01   15.08       1.0  T\n",
            "   3     -7.4481910 -0.102445E-05  0.246E-02   15.09       1.4  T\n",
            "   4     -7.4481910 -0.165762E-07  0.549E-03   15.09       6.4  T\n",
            "   5     -7.4481910 -0.234003E-08  0.533E-04   15.09      66.3  T\n",
            "   6     -7.4481910 -0.173941E-09  0.613E-05   15.09     576.8  T\n",
            "     SCC iter.                  ...        0 min,  0.001 sec\n",
            "     gradient                   ...        0 min,  0.001 sec\n",
            " * total energy  :    -7.3304459 Eh     change       -0.7544009E-02 Eh\n",
            "   gradient norm :     0.0162039 Eh/Œ±   predicted    -0.6596202E-02 ( -12.56%)\n",
            "   displ. norm   :     0.6749411 Œ±      lambda       -0.6770596E-02\n",
            "   maximum displ.:     0.4983480 Œ±      in ANC's #1, #12, #7, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE    9 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1     -7.4344723 -0.743447E+01  0.403E-01   14.92       0.0  T\n",
            "   2     -7.4345388 -0.664385E-04  0.231E-01   14.91       1.0  T\n",
            "   3     -7.4345416 -0.281242E-05  0.461E-02   14.90       1.0  T\n",
            "   4     -7.4345417 -0.923268E-07  0.970E-03   14.90       3.6  T\n",
            "   5     -7.4345417 -0.424103E-08  0.748E-04   14.90      47.3  T\n",
            "   6     -7.4345417 -0.330335E-09  0.156E-04   14.90     226.1  T\n",
            "     SCC iter.                  ...        0 min,  0.001 sec\n",
            "     gradient                   ...        0 min,  0.001 sec\n",
            " * total energy  :    -7.3329613 Eh     change       -0.2515400E-02 Eh\n",
            "   gradient norm :     0.0437328 Eh/Œ±   predicted    -0.4927454E-02 (  95.89%)\n",
            "   displ. norm   :     0.2334154 Œ±      lambda       -0.6474990E-02\n",
            "   maximum displ.:     0.1626937 Œ±      in ANC's #12, #10, #3, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   10 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1     -7.4573967 -0.745740E+01  0.422E-01   15.27       0.0  T\n",
            "   2     -7.4574830 -0.863064E-04  0.240E-01   15.29       1.0  T\n",
            "   3     -7.4574865 -0.351342E-05  0.474E-02   15.29       1.0  T\n",
            "   4     -7.4574867 -0.156943E-06  0.107E-02   15.29       3.3  T\n",
            "   5     -7.4574867 -0.652981E-08  0.950E-04   15.29      37.2  T\n",
            "   6     -7.4574867 -0.490379E-09  0.181E-04   15.29     194.9  T\n",
            "     SCC iter.                  ...        0 min,  0.001 sec\n",
            "     gradient                   ...        0 min,  0.001 sec\n",
            " * total energy  :    -7.3353315 Eh     change       -0.2370266E-02 Eh\n",
            "   gradient norm :     0.0201961 Eh/Œ±   predicted    -0.3413886E-02 (  44.03%)\n",
            "   displ. norm   :     0.1960570 Œ±      lambda       -0.1959305E-02\n",
            "   maximum displ.:     0.1183527 Œ±      in ANC's #1, #12, #3, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   11 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1     -7.4523000 -0.745230E+01  0.210E-01   15.40       0.0  T\n",
            "   2     -7.4523084 -0.843196E-05  0.123E-01   15.39       1.0  T\n",
            "   3     -7.4523089 -0.474119E-06  0.126E-02   15.39       2.8  T\n",
            "   4     -7.4523089 -0.852596E-08  0.385E-03   15.39       9.2  T\n",
            "   5     -7.4523089 -0.200876E-08  0.427E-04   15.39      82.9  T\n",
            "   6     -7.4523089 -0.740288E-10  0.342E-05   15.39    1034.0  T\n",
            "     SCC iter.                  ...        0 min,  0.001 sec\n",
            "     gradient                   ...        0 min,  0.001 sec\n",
            " * total energy  :    -7.3360489 Eh     change       -0.7173537E-03 Eh\n",
            "   gradient norm :     0.0186461 Eh/Œ±   predicted    -0.1017309E-02 (  41.81%)\n",
            "   displ. norm   :     0.1227758 Œ±      lambda       -0.7907648E-03\n",
            "   maximum displ.:     0.0761580 Œ±      in ANC's #3, #12, #1, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   12 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1     -7.4502287 -0.745023E+01  0.146E-01   15.42       0.0  T\n",
            "   2     -7.4502306 -0.195335E-05  0.863E-02   15.42       1.0  T\n",
            "   3     -7.4502309 -0.291025E-06  0.608E-03   15.42       5.8  T\n",
            "   4     -7.4502309 -0.103207E-07  0.268E-03   15.42      13.2  T\n",
            "   5     -7.4502309 -0.280213E-08  0.303E-04   15.42     116.8  T\n",
            "   6     -7.4502309 -0.307860E-10  0.265E-05   15.42    1332.1  T\n",
            "     SCC iter.                  ...        0 min,  0.001 sec\n",
            "     gradient                   ...        0 min,  0.001 sec\n",
            " * total energy  :    -7.3361904 Eh     change       -0.1414762E-03 Eh\n",
            "   gradient norm :     0.0100962 Eh/Œ±   predicted    -0.4013406E-03 ( 183.68%)\n",
            "   displ. norm   :     0.0517108 Œ±      lambda       -0.3142386E-03\n",
            "   maximum displ.:     0.0322051 Œ±      in ANC's #3, #1, #5, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   13 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1     -7.4535340 -0.745353E+01  0.906E-02   15.49       0.0  T\n",
            "   2     -7.4535359 -0.189887E-05  0.527E-02   15.50       1.0  T\n",
            "   3     -7.4535360 -0.936458E-07  0.590E-03   15.50       6.0  T\n",
            "   4     -7.4535360 -0.186194E-08  0.164E-03   15.50      21.6  T\n",
            "   5     -7.4535360 -0.266622E-09  0.174E-04   15.50     203.6  T\n",
            "   6     -7.4535360 -0.147464E-10  0.130E-05   15.50    2718.2  T\n",
            "     SCC iter.                  ...        0 min,  0.001 sec\n",
            "     gradient                   ...        0 min,  0.001 sec\n",
            " * total energy  :    -7.3363539 Eh     change       -0.1635584E-03 Eh\n",
            "   gradient norm :     0.0015763 Eh/Œ±   predicted    -0.1575385E-03 (  -3.68%)\n",
            "   displ. norm   :     0.0469647 Œ±      lambda       -0.2058961E-04\n",
            "   maximum displ.:     0.0356486 Œ±      in ANC's #12, #1, #5, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   14 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1     -7.4535032 -0.745350E+01  0.273E-02   15.51       0.0  T\n",
            "   2     -7.4535032 -0.172650E-07  0.162E-02   15.51       2.2  T\n",
            "   3     -7.4535032 -0.635307E-08  0.740E-04   15.51      47.8  T\n",
            "   4     -7.4535032 -0.117038E-09  0.308E-04   15.51     114.9  T\n",
            "     SCC iter.                  ...        0 min,  0.001 sec\n",
            "     gradient                   ...        0 min,  0.001 sec\n",
            " * total energy  :    -7.3363627 Eh     change       -0.8794761E-05 Eh\n",
            "   gradient norm :     0.0026249 Eh/Œ±   predicted    -0.1030352E-04 (  17.16%)\n",
            "   displ. norm   :     0.0153144 Œ±      lambda       -0.1583268E-04\n",
            "   maximum displ.:     0.0094900 Œ±      in ANC's #2, #12, #8, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   15 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1     -7.4538250 -0.745383E+01  0.187E-02   15.51       0.0  T\n",
            "   2     -7.4538250 -0.261251E-07  0.110E-02   15.51       3.2  T\n",
            "   3     -7.4538251 -0.427308E-08  0.726E-04   15.51      48.7  T\n",
            "   4     -7.4538251 -0.660156E-10  0.332E-04   15.51     106.6  T\n",
            "     SCC iter.                  ...        0 min,  0.001 sec\n",
            "     gradient                   ...        0 min,  0.001 sec\n",
            " * total energy  :    -7.3363636 Eh     change       -0.9057116E-06 Eh\n",
            "   gradient norm :     0.0024552 Eh/Œ±   predicted    -0.7916501E-05 ( 774.06%)\n",
            "   displ. norm   :     0.0164571 Œ±      lambda       -0.1423417E-04\n",
            "   maximum displ.:     0.0102674 Œ±      in ANC's #12, #1, #2, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   16 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1     -7.4541144 -0.745411E+01  0.151E-02   15.51       0.0  T\n",
            "   2     -7.4541144 -0.243519E-07  0.880E-03   15.51       4.0  T\n",
            "   3     -7.4541144 -0.271920E-08  0.657E-04   15.51      53.8  T\n",
            "   4     -7.4541144 -0.382681E-10  0.263E-04   15.51     134.6  T\n",
            "     SCC iter.                  ...        0 min,  0.001 sec\n",
            "     gradient                   ...        0 min,  0.001 sec\n",
            " * total energy  :    -7.3363692 Eh     change       -0.5527514E-05 Eh\n",
            "   gradient norm :     0.0006804 Eh/Œ±   predicted    -0.7118025E-05 (  28.77%)\n",
            "   displ. norm   :     0.0090975 Œ±      lambda       -0.2525506E-05\n",
            "   maximum displ.:     0.0062643 Œ±      in ANC's #3, #1, #7, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   17 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1     -7.4536796 -0.745368E+01  0.141E-02   15.51       0.0  T\n",
            "   2     -7.4536797 -0.348087E-07  0.810E-03   15.51       4.4  T\n",
            "   3     -7.4536797 -0.288909E-08  0.776E-04   15.51      45.6  T\n",
            "   4     -7.4536797 -0.450049E-10  0.244E-04   15.51     144.7  T\n",
            "     SCC iter.                  ...        0 min,  0.001 sec\n",
            "     gradient                   ...        0 min,  0.001 sec\n",
            " * total energy  :    -7.3363697 Eh     change       -0.5899127E-06 Eh\n",
            "   gradient norm :     0.0006870 Eh/Œ±   predicted    -0.1263628E-05 ( 114.21%)\n",
            "   displ. norm   :     0.0034384 Œ±      lambda       -0.1056564E-05\n",
            "   maximum displ.:     0.0015553 Œ±      in ANC's #3, #10, #1, ...\n",
            "\n",
            "   *** GEOMETRY OPTIMIZATION CONVERGED AFTER 17 ITERATIONS ***\n",
            "\n",
            "------------------------------------------------------------------------\n",
            " total energy gain   :        -0.0733725 Eh      -46.0419 kcal/mol\n",
            " total RMSD          :         1.6414586 a0        0.8686 √Ö\n",
            " total power (kW/mol):       -11.3317341 (step)-1877.5803 (real)\n",
            "------------------------------------------------------------------------\n",
            "\n",
            " ANCopt (total)                0 d,  0 h,  0 min,  0.103 sec\n",
            " optimizer setup                ...        0 min,  0.000 sec (  0.235%)\n",
            " model hessian                  ...        0 min,  0.001 sec (  0.913%)\n",
            " ANC generation                 ...        0 min,  0.000 sec (  0.199%)\n",
            " coordinate transformation      ...        0 min,  0.000 sec (  0.118%)\n",
            " single point calculation       ...        0 min,  0.035 sec ( 34.519%)\n",
            " optimization log               ...        0 min,  0.002 sec (  1.803%)\n",
            " hessian update                 ...        0 min,  0.000 sec (  0.161%)\n",
            " rational function              ...        0 min,  0.001 sec (  1.279%)\n",
            "\n",
            "================\n",
            " final structure:\n",
            "================\n",
            "8\n",
            " xtb: 6.7.0 (08769fc)\n",
            "C           -0.01557678914544        0.04676952851446       -0.01140071053055\n",
            "H           -0.72244760195278        0.63613433703088        0.56910045483325\n",
            "H            0.85821432826868       -0.14545593530701        0.60776190560494\n",
            "H            0.29610899794758        0.63657615178766       -0.87079497028301\n",
            "C           -0.64988992692328       -1.26128735487501       -0.46007407919487\n",
            "H           -0.96201349834668       -1.85062335079973        0.39986038880813\n",
            "H           -1.52456516250983       -1.06910210267221       -1.07796226527881\n",
            "H            0.05674365266173       -1.85116627367905       -1.03983472395905\n",
            " \n",
            " Bond Distances (Angstroems)\n",
            " ---------------------------\n",
            "C1-H2=1.0881         C1-H3=1.0880         C1-H4=1.0879         C1-C5=1.5214         H2-C1=1.0881         H3-C1=1.0880         \n",
            "H4-C1=1.0879         C5-C1=1.5214         C5-H6=1.0882         C5-H7=1.0880         C5-H8=1.0878         H6-C5=1.0882         \n",
            "H7-C5=1.0880         H8-C5=1.0878         \n",
            " C  H  Rav=1.0880 sigma=0.0001  Rmin=1.0878  Rmax=1.0882     6\n",
            " C  C  Rav=1.5214 sigma=0.0000  Rmin=1.5214  Rmax=1.5214     1\n",
            " \n",
            " selected bond angles (degree)\n",
            " --------------------\n",
            "H3-C1-H2=108.29                H4-C1-H2=108.29                H4-C1-H3=108.37                C5-C1-H2=110.62                \n",
            "C5-C1-H3=110.53                C5-C1-H4=110.65                H6-C5-C1=110.62                H7-C5-C1=110.54                \n",
            "H7-C5-H6=108.29                H8-C5-C1=110.64                H8-C5-H6=108.29                H8-C5-H7=108.38                \n",
            " \n",
            " selected dihedral angles (degree)\n",
            " ---------------------------------\n",
            "H6-C5-C1-H2=300.05            H6-C5-C1-H3= 59.98            H6-C5-C1-H4=180.03            H7-C5-C1-H2= 59.98            \n",
            "H7-C5-C1-H3=179.91            H7-C5-C1-H4=299.97            H8-C5-C1-H2=180.04            H8-C5-C1-H3=299.98            \n",
            "H8-C5-C1-H4= 60.03            \n",
            "           -------------------------------------------------\n",
            "          |                Final Singlepoint                |\n",
            "           -------------------------------------------------\n",
            "\n",
            "          ...................................................\n",
            "          :                      SETUP                      :\n",
            "          :.................................................:\n",
            "          :  # basis functions                  14          :\n",
            "          :  # atomic orbitals                  14          :\n",
            "          :  # shells                           10          :\n",
            "          :  # electrons                        14          :\n",
            "          :  max. iterations                   250          :\n",
            "          :  Hamiltonian                  GFN2-xTB          :\n",
            "          :  restarted?                      false          :\n",
            "          :  GBSA solvation                  false          :\n",
            "          :  PC potential                    false          :\n",
            "          :  electronic temp.          300.0000000     K    :\n",
            "          :  accuracy                    1.0000000          :\n",
            "          :  -> integral cutoff          0.2500000E+02      :\n",
            "          :  -> integral neglect         0.1000000E-07      :\n",
            "          :  -> SCF convergence          0.1000000E-05 Eh   :\n",
            "          :  -> wf. convergence          0.1000000E-03 e    :\n",
            "          :  Broyden damping             0.4000000          :\n",
            "          :  net charge                          0          :\n",
            "          :  unpaired electrons                  0          :\n",
            "          ...................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1     -7.4536797 -0.745368E+01  0.290E-05   15.51       0.0  T\n",
            "   2     -7.4536797 -0.323297E-12  0.157E-05   15.51    2255.9  T\n",
            "   3     -7.4536797 -0.861533E-13  0.280E-06   15.51   12641.2  T\n",
            "\n",
            "   *** convergence criteria satisfied after 3 iterations ***\n",
            "\n",
            "         #    Occupation            Energy/Eh            Energy/eV\n",
            "      -------------------------------------------------------------\n",
            "         1        2.0000           -0.6047549             -16.4562\n",
            "         2        2.0000           -0.5630672             -15.3218\n",
            "         3        2.0000           -0.4954365             -13.4815\n",
            "         4        2.0000           -0.4953711             -13.4797\n",
            "         5        2.0000           -0.4610364             -12.5454\n",
            "         6        2.0000           -0.4353357             -11.8461\n",
            "         7        2.0000           -0.4353217             -11.8457 (HOMO)\n",
            "         8                          0.1344946               3.6598 (LUMO)\n",
            "         9                          0.1670273               4.5450\n",
            "        10                          0.1673613               4.5541\n",
            "        11                          0.1742781               4.7423\n",
            "        12                          0.2014345               5.4813\n",
            "       ...                                ...                  ...\n",
            "        14                          0.2814074               7.6575\n",
            "      -------------------------------------------------------------\n",
            "                  HL-Gap            0.5698164 Eh           15.5055 eV\n",
            "             Fermi-level           -0.1504136 Eh           -4.0930 eV\n",
            "\n",
            " SCC (total)                   0 d,  0 h,  0 min,  0.069 sec\n",
            " SCC setup                      ...        0 min,  0.000 sec (  0.285%)\n",
            " Dispersion                     ...        0 min,  0.000 sec (  0.047%)\n",
            " classical contributions        ...        0 min,  0.000 sec (  0.019%)\n",
            " integral evaluation            ...        0 min,  0.000 sec (  0.404%)\n",
            " iterations                     ...        0 min,  0.001 sec (  1.120%)\n",
            " molecular gradient             ...        0 min,  0.001 sec (  1.101%)\n",
            " printout                       ...        0 min,  0.067 sec ( 96.976%)\n",
            "\n",
            "         :::::::::::::::::::::::::::::::::::::::::::::::::::::\n",
            "         ::                     SUMMARY                     ::\n",
            "         :::::::::::::::::::::::::::::::::::::::::::::::::::::\n",
            "         :: total energy              -7.336369749310 Eh    ::\n",
            "         :: gradient norm              0.000690332412 Eh/a0 ::\n",
            "         :: HOMO-LUMO gap             15.505493026795 eV    ::\n",
            "         ::.................................................::\n",
            "         :: SCC energy                -7.453679519845 Eh    ::\n",
            "         :: -> isotropic ES            0.002035232176 Eh    ::\n",
            "         :: -> anisotropic ES          0.003097849841 Eh    ::\n",
            "         :: -> anisotropic XC          0.005394675185 Eh    ::\n",
            "         :: -> dispersion             -0.002034021476 Eh    ::\n",
            "         :: repulsion energy           0.117309763097 Eh    ::\n",
            "         :: add. restraining           0.000000007438 Eh    ::\n",
            "         :: total charge              -0.000000000000 e     ::\n",
            "         :::::::::::::::::::::::::::::::::::::::::::::::::::::\n",
            "\n",
            "========================================================================\n",
            " RELAXED SCAN\n",
            "------------------------------------------------------------------------\n",
            " output written to xtbscan.log\n",
            " scaning constraint 1\n",
            "... step 1 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33636975\n",
            "    bias energy:           0.00000001\n",
            "unbiased energy:          -7.33636976\n",
            "... step 2 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33632053\n",
            "    bias energy:           0.00001273\n",
            "unbiased energy:          -7.33633326\n",
            "... step 3 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33617160\n",
            "    bias energy:           0.00004962\n",
            "unbiased energy:          -7.33622122\n",
            "... step 4 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33592739\n",
            "    bias energy:           0.00010731\n",
            "unbiased energy:          -7.33603470\n",
            "... step 5 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33559430\n",
            "    bias energy:           0.00018141\n",
            "unbiased energy:          -7.33577571\n",
            "... step 6 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33518161\n",
            "    bias energy:           0.00025949\n",
            "unbiased energy:          -7.33544110\n",
            "... step 7 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33470267\n",
            "    bias energy:           0.00033779\n",
            "unbiased energy:          -7.33504046\n",
            "... step 8 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33417661\n",
            "    bias energy:           0.00038585\n",
            "unbiased energy:          -7.33456246\n",
            "... step 9 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33362967\n",
            "    bias energy:           0.00039007\n",
            "unbiased energy:          -7.33401974\n",
            "... step 10 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33309960\n",
            "    bias energy:           0.00033627\n",
            "unbiased energy:          -7.33343588\n",
            "... step 11 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33264169\n",
            "    bias energy:           0.00021079\n",
            "unbiased energy:          -7.33285248\n",
            "... step 12 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33232871\n",
            "    bias energy:           0.00006608\n",
            "unbiased energy:          -7.33239480\n",
            "... step 13 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33224197\n",
            "    bias energy:           0.00000043\n",
            "unbiased energy:          -7.33224240\n",
            "... step 14 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33241118\n",
            "    bias energy:           0.00009017\n",
            "unbiased energy:          -7.33250135\n",
            "... step 15 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33278202\n",
            "    bias energy:           0.00024473\n",
            "unbiased energy:          -7.33302675\n",
            "... step 16 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33327159\n",
            "    bias energy:           0.00035196\n",
            "unbiased energy:          -7.33362355\n",
            "... step 17 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33381139\n",
            "    bias energy:           0.00037550\n",
            "unbiased energy:          -7.33418689\n",
            "... step 18 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33435518\n",
            "    bias energy:           0.00035579\n",
            "unbiased energy:          -7.33471098\n",
            "... step 19 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33486831\n",
            "    bias energy:           0.00029784\n",
            "unbiased energy:          -7.33516615\n",
            "... step 20 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33532728\n",
            "    bias energy:           0.00023073\n",
            "unbiased energy:          -7.33555801\n",
            "... step 21 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33571484\n",
            "    bias energy:           0.00015378\n",
            "unbiased energy:          -7.33586862\n",
            "... step 22 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33601918\n",
            "    bias energy:           0.00008563\n",
            "unbiased energy:          -7.33610481\n",
            "... step 23 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33623216\n",
            "    bias energy:           0.00003450\n",
            "unbiased energy:          -7.33626666\n",
            "... step 24 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33634837\n",
            "    bias energy:           0.00000553\n",
            "unbiased energy:          -7.33635390\n",
            "... step 25 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33636506\n",
            "    bias energy:           0.00000150\n",
            "unbiased energy:          -7.33636656\n",
            "... step 26 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33628184\n",
            "    bias energy:           0.00002281\n",
            "unbiased energy:          -7.33630465\n",
            "... step 27 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33610067\n",
            "    bias energy:           0.00006749\n",
            "unbiased energy:          -7.33616817\n",
            "... step 28 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33582598\n",
            "    bias energy:           0.00013117\n",
            "unbiased energy:          -7.33595715\n",
            "... step 29 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33546500\n",
            "    bias energy:           0.00020677\n",
            "unbiased energy:          -7.33567177\n",
            "... step 30 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33502848\n",
            "    bias energy:           0.00028318\n",
            "unbiased energy:          -7.33531166\n",
            "... step 31 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33453115\n",
            "    bias energy:           0.00035634\n",
            "unbiased energy:          -7.33488749\n",
            "... step 32 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33399452\n",
            "    bias energy:           0.00039132\n",
            "unbiased energy:          -7.33438585\n",
            "... step 33 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33344823\n",
            "    bias energy:           0.00037651\n",
            "unbiased energy:          -7.33382474\n",
            "... step 34 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33293584\n",
            "    bias energy:           0.00029432\n",
            "unbiased energy:          -7.33323016\n",
            "... step 35 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33251676\n",
            "    bias energy:           0.00016065\n",
            "unbiased energy:          -7.33267741\n",
            "... step 36 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33227145\n",
            "    bias energy:           0.00002876\n",
            "unbiased energy:          -7.33230022\n",
            "... step 37 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33227125\n",
            "    bias energy:           0.00001451\n",
            "unbiased energy:          -7.33228577\n",
            "... step 38 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33251682\n",
            "    bias energy:           0.00014286\n",
            "unbiased energy:          -7.33265968\n",
            "... step 39 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33293635\n",
            "    bias energy:           0.00028883\n",
            "unbiased energy:          -7.33322518\n",
            "... step 40 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33344837\n",
            "    bias energy:           0.00035941\n",
            "unbiased energy:          -7.33380778\n",
            "... step 41 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33399472\n",
            "    bias energy:           0.00037589\n",
            "unbiased energy:          -7.33437061\n",
            "... step 42 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33453129\n",
            "    bias energy:           0.00033752\n",
            "unbiased energy:          -7.33486881\n",
            "... step 43 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33502830\n",
            "    bias energy:           0.00027257\n",
            "unbiased energy:          -7.33530087\n",
            "... step 44 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33546480\n",
            "    bias energy:           0.00019719\n",
            "unbiased energy:          -7.33566198\n",
            "... step 45 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33582592\n",
            "    bias energy:           0.00012893\n",
            "unbiased energy:          -7.33595485\n",
            "... step 46 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33610063\n",
            "    bias energy:           0.00006604\n",
            "unbiased energy:          -7.33616666\n",
            "... step 47 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33628184\n",
            "    bias energy:           0.00002198\n",
            "unbiased energy:          -7.33630382\n",
            "... step 48 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33636500\n",
            "    bias energy:           0.00000144\n",
            "unbiased energy:          -7.33636644\n",
            "... step 49 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33634832\n",
            "    bias energy:           0.00000562\n",
            "unbiased energy:          -7.33635394\n",
            "... step 50 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33623207\n",
            "    bias energy:           0.00003474\n",
            "unbiased energy:          -7.33626681\n",
            "... step 51 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33601908\n",
            "    bias energy:           0.00008600\n",
            "unbiased energy:          -7.33610508\n",
            "... step 52 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33571484\n",
            "    bias energy:           0.00015599\n",
            "unbiased energy:          -7.33587083\n",
            "... step 53 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33532740\n",
            "    bias energy:           0.00023957\n",
            "unbiased energy:          -7.33556697\n",
            "... step 54 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33486920\n",
            "    bias energy:           0.00031536\n",
            "unbiased energy:          -7.33518456\n",
            "... step 55 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33435697\n",
            "    bias energy:           0.00037331\n",
            "unbiased energy:          -7.33473028\n",
            "... step 56 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33381417\n",
            "    bias energy:           0.00039476\n",
            "unbiased energy:          -7.33420893\n",
            "... step 57 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33327388\n",
            "    bias energy:           0.00036011\n",
            "unbiased energy:          -7.33363399\n",
            "... step 58 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33278492\n",
            "    bias energy:           0.00025643\n",
            "unbiased energy:          -7.33304136\n",
            "... step 59 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33241282\n",
            "    bias energy:           0.00011273\n",
            "unbiased energy:          -7.33252555\n",
            "... step 60 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33224207\n",
            "    bias energy:           0.00000616\n",
            "unbiased energy:          -7.33224823\n",
            "... step 61 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33232910\n",
            "    bias energy:           0.00004571\n",
            "unbiased energy:          -7.33237481\n",
            "... step 62 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33264373\n",
            "    bias energy:           0.00019616\n",
            "unbiased energy:          -7.33283988\n",
            "... step 63 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33310301\n",
            "    bias energy:           0.00032429\n",
            "unbiased energy:          -7.33342730\n",
            "... step 64 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33363140\n",
            "    bias energy:           0.00036855\n",
            "unbiased energy:          -7.33399995\n",
            "... step 65 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33417778\n",
            "    bias energy:           0.00036501\n",
            "unbiased energy:          -7.33454279\n",
            "... step 66 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33470334\n",
            "    bias energy:           0.00031667\n",
            "unbiased energy:          -7.33502000\n",
            "... step 67 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33518175\n",
            "    bias energy:           0.00024729\n",
            "unbiased energy:          -7.33542903\n",
            "... step 68 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33559426\n",
            "    bias energy:           0.00017172\n",
            "unbiased energy:          -7.33576598\n",
            "... step 69 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33592752\n",
            "    bias energy:           0.00010585\n",
            "unbiased energy:          -7.33603337\n",
            "... step 70 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33617170\n",
            "    bias energy:           0.00004879\n",
            "unbiased energy:          -7.33622049\n",
            "... step 71 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33632062\n",
            "    bias energy:           0.00001227\n",
            "unbiased energy:          -7.33633289\n",
            "... step 72 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:          -7.33637056\n",
            "    bias energy:           0.00000000\n",
            "unbiased energy:          -7.33637056\n",
            "           -------------------------------------------------\n",
            "          |                Property Printout                |\n",
            "           -------------------------------------------------\n",
            "\n",
            "    * Orbital Energies and Occupations\n",
            "\n",
            "         #    Occupation            Energy/Eh            Energy/eV\n",
            "      -------------------------------------------------------------\n",
            "         1        2.0000           -0.6047776             -16.4568\n",
            "         2        2.0000           -0.5631346             -15.3237\n",
            "         3        2.0000           -0.4954034             -13.4806\n",
            "         4        2.0000           -0.4953985             -13.4805\n",
            "         5        2.0000           -0.4610851             -12.5468\n",
            "         6        2.0000           -0.4353473             -11.8464\n",
            "         7        2.0000           -0.4353297             -11.8459 (HOMO)\n",
            "         8                          0.1344970               3.6599 (LUMO)\n",
            "         9                          0.1675207               4.5585\n",
            "        10                          0.1676080               4.5608\n",
            "        11                          0.1749665               4.7611\n",
            "        12                          0.2021071               5.4996\n",
            "        13                          0.2816966               7.6654\n",
            "        14                          0.2819165               7.6713\n",
            "      -------------------------------------------------------------\n",
            "                  HL-Gap            0.5698268 Eh           15.5058 eV\n",
            "             Fermi-level           -0.1504163 Eh           -4.0930 eV\n",
            "\n",
            "     #   Z          covCN         q      C6AA      Œ±(0)\n",
            "     1   6 C        3.750    -0.097    22.414     6.752\n",
            "     2   1 H        0.925     0.032     2.554     2.499\n",
            "     3   1 H        0.925     0.032     2.554     2.499\n",
            "     4   1 H        0.925     0.032     2.554     2.499\n",
            "     5   6 C        3.750    -0.097    22.414     6.752\n",
            "     6   1 H        0.925     0.032     2.554     2.499\n",
            "     7   1 H        0.925     0.032     2.554     2.499\n",
            "     8   1 H        0.925     0.032     2.554     2.499\n",
            "\n",
            " Mol. C6AA /au¬∑bohr‚Å∂  :        362.044649\n",
            " Mol. C8AA /au¬∑bohr‚Å∏  :       7078.448773\n",
            " Mol. Œ±(0) /au        :         28.498831\n",
            "\n",
            "\n",
            "Wiberg/Mayer (AO) data.\n",
            "largest (>0.10) Wiberg bond orders for each atom\n",
            "\n",
            " ---------------------------------------------------------------------------\n",
            "     #   Z sym  total        # sym  WBO       # sym  WBO       # sym  WBO\n",
            " ---------------------------------------------------------------------------\n",
            "     1   6 C    3.997 --     5 C    1.031     2 H    0.989     4 H    0.989\n",
            "                             3 H    0.989\n",
            "     2   1 H    0.999 --     1 C    0.989\n",
            "     3   1 H    0.999 --     1 C    0.989\n",
            "     4   1 H    0.999 --     1 C    0.989\n",
            "     5   6 C    3.997 --     1 C    1.031     6 H    0.989     8 H    0.989\n",
            "                             7 H    0.989\n",
            "     6   1 H    0.999 --     5 C    0.989\n",
            "     7   1 H    0.999 --     5 C    0.989\n",
            "     8   1 H    0.999 --     5 C    0.989\n",
            " ---------------------------------------------------------------------------\n",
            "\n",
            "Topologies differ in total number of bonds\n",
            "Writing topology from bond orders to xtbtopo.mol\n",
            "\n",
            "molecular dipole:\n",
            "                 x           y           z       tot (Debye)\n",
            " q only:       -0.000       0.000       0.000\n",
            "   full:       -0.000       0.000       0.000       0.000\n",
            "molecular quadrupole (traceless):\n",
            "                xx          xy          yy          xz          yz          zz\n",
            " q only:       -0.035       0.079       0.089       0.027       0.055      -0.054\n",
            "  q+dip:        0.147      -0.335      -0.376      -0.116      -0.237       0.229\n",
            "   full:        0.063      -0.144      -0.161      -0.050      -0.102       0.098\n",
            "\n",
            "           -------------------------------------------------\n",
            "          |                Geometry Summary                 |\n",
            "           -------------------------------------------------\n",
            "\n",
            "      molecular mass/u    :       30.0691163\n",
            "   center of mass at/√Ö    :       -0.3329144      -0.6072694      -0.2354376\n",
            "  moments of inertia/u¬∑√Ö¬≤ :        0.6267879E+01   0.2494529E+02   0.2494587E+02\n",
            "rotational constants/cm‚Åª¬π :        0.2689527E+01   0.6757842E+00   0.6757685E+00\n",
            "\n",
            " * 7 selected distances\n",
            "\n",
            "     #   Z          #   Z                                           value/√Ö\n",
            "     1   6 C        2   1 H                                       1.0878609\n",
            "     1   6 C        3   1 H                                       1.0876220\n",
            "     1   6 C        4   1 H                                       1.0876667\n",
            "     1   6 C        5   6 C                                       1.5214102 (max)\n",
            "     5   6 C        6   1 H                                       1.0878609\n",
            "     5   6 C        7   1 H                                       1.0876220 (min)\n",
            "     5   6 C        8   1 H                                       1.0876667\n",
            "\n",
            " * 2 distinct bonds (by element types)\n",
            "\n",
            "   Z      Z             #   av. dist./√Ö        max./√Ö        min./√Ö\n",
            "   1 H    6 C           6     1.0877165     1.0878609     1.0876220\n",
            "   6 C    6 C           1     1.5214102     1.5214102     1.5214102\n",
            "\n",
            "\n",
            "optimized geometry written to: xtbopt.xyz\n",
            "\n",
            "\n",
            "           -------------------------------------------------\n",
            "          | TOTAL ENERGY               -7.336369749310 Eh   |\n",
            "          | GRADIENT NORM               0.000690332412 Eh/Œ± |\n",
            "          | HOMO-LUMO GAP              15.505775115394 eV   |\n",
            "           -------------------------------------------------\n",
            "\n",
            "------------------------------------------------------------------------\n",
            " * finished run on 2024/12/10 at 14:47:09.175     \n",
            "------------------------------------------------------------------------\n",
            " total:\n",
            " * wall-time:     0 d,  0 h,  0 min,  3.077 sec\n",
            " *  cpu-time:     0 d,  0 h,  0 min,  3.431 sec\n",
            " * ratio c/w:     1.115 speedup\n",
            " SCF:\n",
            " * wall-time:     0 d,  0 h,  0 min,  0.005 sec\n",
            " *  cpu-time:     0 d,  0 h,  0 min,  0.009 sec\n",
            " * ratio c/w:     1.830 speedup\n",
            " ANC optimizer:\n",
            " * wall-time:     0 d,  0 h,  0 min,  3.047 sec\n",
            " *  cpu-time:     0 d,  0 h,  0 min,  3.389 sec\n",
            " * ratio c/w:     1.112 speedup\n",
            "\n",
            "normal termination of xtb\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!cat < xtbscan.log"
      ],
      "metadata": {
        "id": "IhBjncCmEIWV",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "20314c78-bc10-46f8-b40a-6cf1de10f6fd"
      },
      "execution_count": 46,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "8\n",
            " energy: -7.336369756748 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01557678914544        0.04676952851446       -0.01140071053055\n",
            "H           -0.72244760195278        0.63613433703088        0.56910045483325\n",
            "H            0.85821432826868       -0.14545593530701        0.60776190560494\n",
            "H            0.29610899794758        0.63657615178766       -0.87079497028301\n",
            "C           -0.64988992692328       -1.26128735487501       -0.46007407919487\n",
            "H           -0.96201349834668       -1.85062335079973        0.39986038880813\n",
            "H           -1.52456516250983       -1.06910210267221       -1.07796226527881\n",
            "H            0.05674365266173       -1.85116627367905       -1.03983472395905\n",
            "8\n",
            " energy: -7.336333263138 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01420502669525        0.04602897026964       -0.01100112538491\n",
            "H           -0.71206026681302        0.62228137727202        0.59299882590896\n",
            "H            0.87584863742636       -0.14450594829268        0.58389835603764\n",
            "H            0.26822007086304        0.65038669039304       -0.87009843537225\n",
            "C           -0.65068618525168       -1.26056894182954       -0.46119692581622\n",
            "H           -0.98801335059685       -1.83668873416564        0.39797254817177\n",
            "H           -1.50789986206766       -1.07003604570283       -1.10251605060068\n",
            "H            0.06536998313504       -1.86505236794402       -1.01340119294427\n",
            "8\n",
            " energy: -7.336221222362 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01287823396353        0.04562533422993       -0.01128083581802\n",
            "H           -0.70109539441141        0.60825241166685        0.61650952715840\n",
            "H            0.89258313469929       -0.14402479543607        0.55974765609212\n",
            "H            0.23984097287035        0.66527139848417       -0.86872798786110\n",
            "C           -0.65086595315077       -1.26016429185724       -0.46254094500444\n",
            "H           -1.01383612668103       -1.82266398880034        0.39545678326156\n",
            "H           -1.49069856441476       -1.07052223490840       -1.12633157611982\n",
            "H            0.07352416505185       -1.87992883337890       -0.98617662170867\n",
            "8\n",
            " energy: -7.336034699107 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01158099530827        0.04536019400288       -0.01159521026920\n",
            "H           -0.68900219986925        0.59387374351996        0.64027443654528\n",
            "H            0.90917023721602       -0.14273302153357        0.53448781909530\n",
            "H            0.21089393725028        0.68031708971906       -0.86618907552442\n",
            "C           -0.65100099276165       -1.25989934515802       -0.46386910379030\n",
            "H           -1.04026968329599       -1.80828296043551        0.39196142877435\n",
            "H           -1.47239990712321       -1.07181822335785       -1.15038048963373\n",
            "H            0.08076360389206       -1.89497247675695       -0.95803380519725\n",
            "8\n",
            " energy: -7.335775709481 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01053077065906        0.04537760386069       -0.01205866519827\n",
            "H           -0.67566924258730        0.57884797803700        0.66448309677166\n",
            "H            0.92552601611409       -0.14099579519476        0.50773731166080\n",
            "H            0.18100963570414        0.69583499678498       -0.86257936448219\n",
            "C           -0.65091355165413       -1.25991689105178       -0.46501367472884\n",
            "H           -1.06753462883872       -1.79325555492792        0.38744472828155\n",
            "H           -1.45261878126901       -1.07355984968312       -1.17470726161974\n",
            "H            0.08730532318998       -1.91048748782509       -0.92865017068494\n",
            "8\n",
            " energy: -7.335441101708 xtb: 6.7.0 (08769fc)\n",
            "C           -0.00937241465481        0.04541106498129       -0.01239239087613\n",
            "H           -0.66055404924262        0.56315005108852        0.68961573795018\n",
            "H            0.94221957727377       -0.13784597087253        0.47907019305826\n",
            "H            0.14986982723605        0.71146677002330       -0.85741862870658\n",
            "C           -0.65098441320486       -1.25995048458489       -0.46621705521190\n",
            "H           -1.09626399454927       -1.77755597511662        0.38155487323527\n",
            "H           -1.43114286107559       -1.07671432088318       -1.19999034195830\n",
            "H            0.09280232821730       -1.92611613463588       -0.89756638749078\n",
            "8\n",
            " energy: -7.335040458077 xtb: 6.7.0 (08769fc)\n",
            "C           -0.00851880128027        0.04565620854679       -0.01252391209690\n",
            "H           -0.64316368878029        0.54646090630537        0.71625933004867\n",
            "H            0.95919446545715       -0.13324103625777        0.44763818711320\n",
            "H            0.11715223037754        0.72705710903738       -0.85072938580621\n",
            "C           -0.65114453224770       -1.26019571303724       -0.46706574452056\n",
            "H           -1.12717535566444       -1.76086518480514        0.37402237719268\n",
            "H           -1.40715346039288       -1.08132424279983       -1.22645882272380\n",
            "H            0.09738314253088       -1.94170304698956       -0.86448602920706\n",
            "8\n",
            " energy: -7.334562462917 xtb: 6.7.0 (08769fc)\n",
            "C           -0.00781727328261        0.04610073418205       -0.01267816530394\n",
            "H           -0.62283385110980        0.52813901016042        0.74493773613601\n",
            "H            0.97663007332657       -0.12737088766018        0.41233941534389\n",
            "H            0.08142888315901        0.74371312783050       -0.84201406573853\n",
            "C           -0.65123261123394       -1.26064031300941       -0.46777856297248\n",
            "H           -1.16098376702720       -1.74254164715733        0.36439510097464\n",
            "H           -1.37967122245812       -1.08719986018805       -1.25464864561069\n",
            "H            0.10105376862608       -1.95835516415801       -0.82789681282887\n",
            "8\n",
            " energy: -7.334019744680 xtb: 6.7.0 (08769fc)\n",
            "C           -0.00749862357327        0.04679376294950       -0.01264928174356\n",
            "H           -0.59844647620457        0.50759412968275        0.77648220836331\n",
            "H            0.99459895376295       -0.11961557346494        0.37175114214356\n",
            "H            0.04193360074309        0.76120357216994       -0.83075004700311\n",
            "C           -0.65136596535909       -1.26133336530803       -0.46806934990503\n",
            "H           -1.19884517850082       -1.72199512603425        0.35189534497271\n",
            "H           -1.34737873458659       -1.09496129370053       -1.28510170529465\n",
            "H            0.10357642371828       -1.97584110629445       -0.78690231153321\n",
            "8\n",
            " energy: -7.333435876664 xtb: 6.7.0 (08769fc)\n",
            "C           -0.00769004481664        0.04773937148876       -0.01236478269665\n",
            "H           -0.56870054237425        0.48404409846989        0.81145823204552\n",
            "H            1.01287253688064       -0.10949338998687        0.32431523859777\n",
            "H           -0.00241075709382        0.77966426578728       -0.81611045550843\n",
            "C           -0.65157060722837       -1.26227892391296       -0.46779404011678\n",
            "H           -1.24172580777970       -1.69844349665647        0.33548440118605\n",
            "H           -1.30873067061508       -1.10509040773538       -1.31812102710533\n",
            "H            0.10452989302721       -1.99429651745425       -0.74021156640212\n",
            "8\n",
            " energy: -7.332852479030 xtb: 6.7.0 (08769fc)\n",
            "C           -0.00813838721312        0.04875627807554       -0.01184063040783\n",
            "H           -0.53190823909880        0.45615542562854        0.85013537623302\n",
            "H            1.03128850107034       -0.09656295905040        0.26824740139961\n",
            "H           -0.05363192769682        0.79952461659033       -0.79674077438899\n",
            "C           -0.65191572590409       -1.26329573422463       -0.46719668442538\n",
            "H           -1.29044152673525       -1.67055337206680        0.31366026627262\n",
            "H           -1.26199047374223       -1.11802877132756       -1.35414738972518\n",
            "H            0.10331177931994       -2.01415048362500       -0.68546156495784\n",
            "8\n",
            " energy: -7.332394796759 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01016430387855        0.05007232725041       -0.01045376421163\n",
            "H           -0.48400670559496        0.42210197638109        0.89468116631735\n",
            "H            1.04832294801960       -0.07926328311433        0.19842829926269\n",
            "H           -0.11364097191792        0.81990431070232       -0.77080229820286\n",
            "C           -0.65254930416966       -1.26461145966297       -0.46482451915388\n",
            "H           -1.34838842672778       -1.63649870012924        0.28331298125491\n",
            "H           -1.20182361352644       -1.13533784669937       -1.39344755690915\n",
            "H            0.09882437779570       -2.03452232472791       -0.62023830835740\n",
            "8\n",
            " energy: -7.332242402076 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01326624940186        0.05114847859722       -0.00843309324724\n",
            "H           -0.42581146062707        0.38225729201472        0.94088958490621\n",
            "H            1.06104299783815       -0.05835009520737        0.11755695779945\n",
            "H           -0.18064619341843        0.83954856140785       -0.73758845366579\n",
            "C           -0.65342237726085       -1.26568712707518       -0.46122674770444\n",
            "H           -1.41132890239290       -1.59665343747387        0.24381165323519\n",
            "H           -1.12979896219406       -1.15626135240565       -1.43235760287537\n",
            "H            0.08980514745701       -2.05415731985772       -0.54599629844798\n",
            "8\n",
            " energy: -7.332501350740 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01647163325400        0.05152565551091       -0.00636005148375\n",
            "H           -0.36469005597938        0.34147665723000        0.98101037331407\n",
            "H            1.06683192263652       -0.03674250274558        0.03658621627732\n",
            "H           -0.24638988473814        0.85586050164098       -0.70025373144132\n",
            "C           -0.65431035061461       -1.26606380517078       -0.45751408373167\n",
            "H           -1.46950274000948       -1.55587316293899        0.19952466109310\n",
            "H           -1.05537364172485       -1.17787871075329       -1.46476473287109\n",
            "H            0.07648038368392       -2.07045963277326       -0.47157265115665\n",
            "8\n",
            " energy: -7.333026747449 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01874615483048        0.05123776484562       -0.00490689266687\n",
            "H           -0.31029517545785        0.30565634189836        1.01007344879352\n",
            "H            1.06681829059529       -0.01752154314887       -0.03244637642874\n",
            "H           -0.30218421657450        0.86755805352546       -0.66434588586339\n",
            "C           -0.65492356282051       -1.26577556355917       -0.45488560759034\n",
            "H           -1.51501086930052       -1.52005392772737        0.15790071224472\n",
            "H           -0.99027422195541       -1.19710759477209       -1.48772687232201\n",
            "H            0.06118991034396       -2.08214853106194       -0.40700652616687\n",
            "8\n",
            " energy: -7.333623545214 xtb: 6.7.0 (08769fc)\n",
            "C           -0.02001278772154        0.05062699946026       -0.00417327657816\n",
            "H           -0.26461862073085        0.27572987465734        1.02997777127694\n",
            "H            1.06380714202408       -0.00093537457297       -0.08831018635732\n",
            "H           -0.34747527179645        0.87565082668485       -0.63187327866278\n",
            "C           -0.65519370610942       -1.26516461103392       -0.45344708650896\n",
            "H           -1.54898108722818       -1.49012888581382        0.12145020766763\n",
            "H           -0.93659479330733       -1.21369993257345       -1.50347953967611\n",
            "H            0.04564312486967       -2.09023389680830       -0.35348861116121\n",
            "8\n",
            " energy: -7.334186889397 xtb: 6.7.0 (08769fc)\n",
            "C           -0.02119367583224        0.04996642573935       -0.00362846569107\n",
            "H           -0.22476089051792        0.24993217294216        1.04436599314128\n",
            "H            1.05861207290786        0.01379931119936       -0.13567071435883\n",
            "H           -0.38510678924468        0.88114889114231       -0.60238116958520\n",
            "C           -0.65531434546662       -1.26450387948162       -0.45215226844027\n",
            "H           -1.57581329064334       -1.46433276942045        0.08865120924895\n",
            "H           -0.89020674117878       -1.22843963325917       -1.51434274244277\n",
            "H            0.03035765997571       -2.09572551886195       -0.30818584187207\n",
            "8\n",
            " energy: -7.334710975474 xtb: 6.7.0 (08769fc)\n",
            "C           -0.02117289405202        0.04924864668460       -0.00387234821826\n",
            "H           -0.19023822525678        0.22756885555390        1.05428551826884\n",
            "H            1.05354214099474        0.02761653238275       -0.17531429881178\n",
            "H           -0.41761370210886        0.88547148324753       -0.57455891532054\n",
            "C           -0.65509122682507       -1.26378612953863       -0.45225312347377\n",
            "H           -1.59665612230143       -1.44197111844754        0.05939608547163\n",
            "H           -0.85113720014029       -1.24226099370780       -1.52275550162376\n",
            "H            0.01494122968970       -2.10004227617481       -0.26827141629235\n",
            "8\n",
            " energy: -7.335166146751 xtb: 6.7.0 (08769fc)\n",
            "C           -0.02076754056987        0.04859438516728       -0.00445500291041\n",
            "H           -0.15917434897782        0.20758291582140        1.06140315647945\n",
            "H            1.04812689672338        0.04078096819333       -0.21001429546278\n",
            "H           -0.44616724083645        0.88873046051629       -0.54816196107294\n",
            "C           -0.65467666042907       -1.26313196820128       -0.45282937400767\n",
            "H           -1.61370577837908       -1.42198688606897        0.03247021218341\n",
            "H           -0.81661438030041       -1.25542897202379       -1.52919725877899\n",
            "H           -0.00044694723069       -2.10329590340425       -0.23255947643006\n",
            "8\n",
            " energy: -7.335558009938 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01995176491192        0.04800890282854       -0.00549047698564\n",
            "H           -0.13038187378031        0.18892612250453        1.06648117268194\n",
            "H            1.04259932998067        0.05378860370778       -0.24149184368494\n",
            "H           -0.47236077596430        0.89132769246456       -0.52222147083557\n",
            "C           -0.65397170893376       -1.26254667072883       -0.45394335481114\n",
            "H           -1.62807618637419       -1.40333184714889        0.00700749972351\n",
            "H           -0.78509287115120       -1.26843977400215       -1.53446054186733\n",
            "H           -0.01619014886499       -2.10588802962554       -0.19922498422080\n",
            "8\n",
            " energy: -7.335868616349 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01920967801147        0.04753820411001       -0.00647165227749\n",
            "H           -0.10283494639333        0.17172422152849        1.07018536725782\n",
            "H            1.03641541361278        0.06672283454529       -0.27070392904395\n",
            "H           -0.49610894890859        0.89315752111253       -0.49689462154282\n",
            "C           -0.65329344136317       -1.26207614493581       -0.45496976257083\n",
            "H           -1.64073665216611       -1.38613175886695       -0.01773778726124\n",
            "H           -0.75548918130837       -1.28137685598160       -1.53835085602392\n",
            "H           -0.03216856546175       -2.10771302151195       -0.16840075853753\n",
            "8\n",
            " energy: -7.336104810094 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01823957914619        0.04716735283317       -0.00762836822741\n",
            "H           -0.07627531732314        0.15587780269216        1.07269853616840\n",
            "H            1.02987708379741        0.08001219690987       -0.29784577884115\n",
            "H           -0.51787497939810        0.89446617297092       -0.47165713298137\n",
            "C           -0.65252554291372       -1.26170550517053       -0.45626959872524\n",
            "H           -1.65194549420610       -1.37028720861219       -0.04194834277209\n",
            "H           -0.72771974923607       -1.29466880359763       -1.54121785554763\n",
            "H           -0.04872242157411       -2.10901700802577       -0.13947545907348\n",
            "8\n",
            " energy: -7.336266660115 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01734082517331        0.04692194756198       -0.00888337447774\n",
            "H           -0.05049660489833        0.14045843051368        1.07409955555828\n",
            "H            1.02295232924798        0.09321972627620       -0.32383390804069\n",
            "H           -0.53854585673662        0.89535622231991       -0.44646447854088\n",
            "C           -0.65164122168484       -1.26146031696186       -0.45753485843610\n",
            "H           -1.66184571048548       -1.35486976881754       -0.06579263359856\n",
            "H           -0.70090962739121       -1.30787875459629       -1.54333646224621\n",
            "H           -0.06559848287820       -2.10990248629608       -0.11159784021808\n",
            "8\n",
            " energy: -7.336353895727 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01638394037344        0.04678581256189       -0.01020831313264\n",
            "H           -0.02504105748043        0.12575894769336        1.07460374265664\n",
            "H            1.01558176346277        0.10676824046484       -0.34878633078275\n",
            "H           -0.55804924631900        0.89581846169197       -0.42101775962022\n",
            "C           -0.65071031327658       -1.26132441195392       -0.45887819883828\n",
            "H           -1.67079272569037       -1.34017229758291       -0.08963060952942\n",
            "H           -0.67492779861868       -1.32142953364444       -1.54468990714154\n",
            "H           -0.08310268170429       -2.11036021923079       -0.08473662361176\n",
            "8\n",
            " energy: -7.336366557842 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01542037243824        0.04676513641146       -0.01157453261955\n",
            "H            0.00040906359187        0.11161900802834        1.07428586957659\n",
            "H            1.00763792903659        0.12072722511018       -0.37305760622520\n",
            "H           -0.57659774510609        0.89588680435542       -0.39511802010736\n",
            "C           -0.64974271177994       -1.26130397108765       -0.46024156770921\n",
            "H           -1.67896279203634       -1.32603446106078       -0.11373702401522\n",
            "H           -0.64939750240891       -1.33539065978685       -1.54527600689241\n",
            "H           -0.10135186885896       -2.11042408197014       -0.05862511200762\n",
            "8\n",
            " energy: -7.336304648329 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01447719500117        0.04685908335004       -0.01294143711148\n",
            "H            0.02618036706964        0.09793528800981        1.07319447471754\n",
            "H            0.99898344183804        0.13520043218451       -0.39694497782977\n",
            "H           -0.59434879167803        0.89556016631609       -0.36855381983072\n",
            "C           -0.64876768663221       -1.26139815130458       -0.46158592389590\n",
            "H           -1.68651038390160       -1.31235296158556       -0.13840371788451\n",
            "H           -0.62399272422865       -1.34986590536318       -1.54506413428939\n",
            "H           -0.12049302746604       -2.11009295160712       -0.03304446387575\n",
            "8\n",
            " energy: -7.336168166528 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01358083464819        0.04706702659569       -0.01426825476112\n",
            "H            0.05261015616002        0.08460910802736        1.07133745053996\n",
            "H            0.98944309591489        0.15030096348292       -0.42073063279920\n",
            "H           -0.61143487051052        0.89482607110049       -0.34108937642822\n",
            "C           -0.64781488888994       -1.26160631976924       -0.46287277655556\n",
            "H           -1.69355520259703       -1.29902913812600       -0.16394614818631\n",
            "H           -0.59838906038847       -1.36496839436119       -1.54398298641633\n",
            "H           -0.14070439504078       -2.10935431695002       -0.00779127539321\n",
            "8\n",
            " energy: -7.335957146662 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01275557656222        0.04738693359813       -0.01550996946734\n",
            "H            0.08007256432749        0.07154053271439        1.06868166332148\n",
            "H            0.97879321477709        0.16616583728880       -0.44469839420390\n",
            "H           -0.62797192345063        0.89365388862721       -0.31244551958846\n",
            "C           -0.64691868507624       -1.26192643638403       -0.46406424162032\n",
            "H           -1.70019052850552       -1.28596310010009       -0.19072815299015\n",
            "H           -0.57224443078856       -1.38083515371477       -1.54191609225612\n",
            "H           -0.16221063472142       -2.10817750202964        0.01733670680484\n",
            "8\n",
            " energy: -7.335671767187 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01202498335047        0.04781509952968       -0.01661504091908\n",
            "H            0.10900711725684        0.05861782614338        1.06514372911917\n",
            "H            0.96674018865599        0.18297014803870       -0.46915330009945\n",
            "H           -0.64406640699680        0.89199050736480       -0.28227411607135\n",
            "C           -0.64611983281183       -1.26235478822939       -0.46512094683544\n",
            "H           -1.70648401475910       -1.27304315024042       -0.21919196492640\n",
            "H           -0.54517347882084       -1.39764129303679       -1.53868809507282\n",
            "H           -0.18530458917382       -2.10650934956997        0.04255573480540\n",
            "8\n",
            " energy: -7.335311661773 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01148445793366        0.04837257904736       -0.01740708186949\n",
            "H            0.14013003407053        0.04539335128044        1.06045317341864\n",
            "H            0.95279382216566        0.20098347801067       -0.49475035886877\n",
            "H           -0.66006141579291        0.88979791170567       -0.24997638028418\n",
            "C           -0.64555291009813       -1.26291240268613       -0.46589421548544\n",
            "H           -1.71241896430269       -1.25982174338952       -0.25010304273243\n",
            "H           -0.51639540573835       -1.41565642777450       -1.53405473424075\n",
            "H           -0.21043670237047       -2.10431174619399        0.06838864006244\n",
            "8\n",
            " energy: -7.334887494143 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01118441938399        0.04897389089582       -0.01767708946461\n",
            "H            0.17427981587139        0.03235716145875        1.05463451490832\n",
            "H            0.93606878197551        0.22042677618578       -0.52155409307671\n",
            "H           -0.67535868279646        0.88690304709131       -0.21536688139688\n",
            "C           -0.64539821332717       -1.26351377000289       -0.46626694829353\n",
            "H           -1.71829762759088       -1.24678899514120       -0.28424386062960\n",
            "H           -0.48555470725749       -1.43510144469613       -1.52720260888725\n",
            "H           -0.23798094749093       -2.10141166579144        0.09433296684028\n",
            "8\n",
            " energy: -7.334385847854 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01096667497308        0.04965986337507       -0.01771768314578\n",
            "H            0.21245477567722        0.01864939172424        1.04693980528616\n",
            "H            0.91608262257850        0.24222059496048       -0.55042287323835\n",
            "H           -0.69083104721149        0.88304553663047       -0.17668730379741\n",
            "C           -0.64543246538170       -1.26419976434362       -0.46648570915547\n",
            "H           -1.72374691787052       -1.23308520961369       -0.32280485861881\n",
            "H           -0.45168145035122       -1.45689695472578       -1.51796238231176\n",
            "H           -0.26930484246772       -2.09754845800716        0.12179700498146\n",
            "8\n",
            " energy: -7.333824738817 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01096703830211        0.05036638023012       -0.01716933756201\n",
            "H            0.25600239016365        0.00405878945123        1.03670109236294\n",
            "H            0.89139112378561        0.26683985379394       -0.58195264404431\n",
            "H           -0.70627353051852        0.87788768122009       -0.13305277619360\n",
            "C           -0.64594948118281       -1.26490621844612       -0.46630277212290\n",
            "H           -1.72858545463159       -1.21849932013239       -0.36727903123517\n",
            "H           -0.41373300665697       -1.48151782734846       -1.50517048192242\n",
            "H           -0.30531100265727       -2.09238433876842        0.15088195071750\n",
            "8\n",
            " energy: -7.333230164518 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01091615065108        0.05106338608157       -0.01648956542892\n",
            "H            0.30500773290153       -0.01160393382783        1.02283645434531\n",
            "H            0.86162456201361        0.29473352161910       -0.61593521201538\n",
            "H           -0.72183964354696        0.87073769799048       -0.08325144352877\n",
            "C           -0.64660748769888       -1.26560315050540       -0.46612441871665\n",
            "H           -1.73182130498325       -1.20284216899583       -0.41810673627216\n",
            "H           -0.37178267428319       -1.50941297891928       -1.48840892076650\n",
            "H           -0.34709103375180       -2.08522737344283        0.18213584238310\n",
            "8\n",
            " energy: -7.332677405655 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01145787331507        0.05147210039053       -0.01413105068704\n",
            "H            0.36480278163367       -0.02956221155767        1.00314856415136\n",
            "H            0.82119000941587        0.32779755793481       -0.65498101489048\n",
            "H           -0.73690064763569        0.86085096660710       -0.02421814818049\n",
            "C           -0.64865124606427       -1.26601154990477       -0.46482851608451\n",
            "H           -1.73315705952178       -1.18489100900019       -0.48104744015286\n",
            "H           -0.32150743076161       -1.54247821309625       -1.46327232519865\n",
            "H           -0.39774453375114       -2.07533264137354        0.21598593104270\n",
            "8\n",
            " energy: -7.332300218703 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01228264565481        0.05135535677590       -0.01070454350309\n",
            "H            0.43469990320867       -0.05002256764442        0.97507020933162\n",
            "H            0.76876734131160        0.36619608610059       -0.69770340971512\n",
            "H           -0.75081222426332        0.84682940346311        0.04509296136176\n",
            "C           -0.65160784661848       -1.26589434607951       -0.46291028588520\n",
            "H           -1.72994299071973       -1.16443955363763       -0.55630722789987\n",
            "H           -0.26377547111077       -1.58087738737662       -1.42805428596302\n",
            "H           -0.45847206615318       -2.06130199160143        0.25217258227295\n",
            "8\n",
            " energy: -7.332285767003 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01320506133629        0.05050693077791       -0.00689158070561\n",
            "H            0.50923635736352       -0.07164223073063        0.93798152282228\n",
            "H            0.70636918401949        0.40724901484976       -0.73986946705769\n",
            "H           -0.76167254547038        0.82825259247446        0.12078629923949\n",
            "C           -0.65489630932428       -1.26504540772742       -0.46077143148187\n",
            "H           -1.71977643297589       -1.14283020280374       -0.63894100223624\n",
            "H           -0.20324804364294       -1.62193008919018       -1.38324407084236\n",
            "H           -0.52623314863324       -2.04271560765014        0.28760573026205\n",
            "8\n",
            " energy: -7.332659680644 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01391928639816        0.04921156790947       -0.00404351417867\n",
            "H            0.57617483040537       -0.09089337028538        0.89706012338432\n",
            "H            0.64414929317404        0.44473505171348       -0.77468647522028\n",
            "H           -0.76794403722000        0.80728337744050        0.19207300044740\n",
            "C           -0.65734415136448       -1.26374965966951       -0.45915011431032\n",
            "H           -1.70346694030231       -1.12358919999348       -0.71568545134144\n",
            "H           -0.14970967613416       -1.65941506830087       -1.33615626207740\n",
            "H           -0.59136603216031       -2.02173769881421        0.31724469329641\n",
            "8\n",
            " energy: -7.333225182290 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01433103365374        0.04797657223863       -0.00264615236691\n",
            "H            0.62899900866273       -0.10569123341369        0.85876435761731\n",
            "H            0.59055570067720        0.47529774693153       -0.79992050399152\n",
            "H           -0.77021968128807        0.78686174987053        0.25176026967193\n",
            "C           -0.65852468745705       -1.26251446989087       -0.45829688207235\n",
            "H           -1.68493584877871       -1.10880002416069       -0.77824594343626\n",
            "H           -0.10807854481448       -1.68997630740193       -1.29401403518074\n",
            "H           -0.64689091334788       -2.00130903417352        0.33925488975858\n",
            "8\n",
            " energy: -7.333807783948 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01472834170596        0.04671455705905       -0.00169789461955\n",
            "H            0.67121640918015       -0.11733018263467        0.82389597570846\n",
            "H            0.54406394051511        0.50026363062728       -0.81824065637859\n",
            "H           -0.76967297195665        0.76798119378944        0.30108274699109\n",
            "C           -0.65928653245625       -1.26125231383520       -0.45760674973075\n",
            "H           -1.66610655157470       -1.09716850704662       -0.82966328970511\n",
            "H           -0.07533042891678       -1.71494051769592       -1.25626809214438\n",
            "H           -0.69358152308493       -1.98242286026337        0.35515395987886\n",
            "8\n",
            " energy: -7.334370608860 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01483776505897        0.04602732171118       -0.00206499562776\n",
            "H            0.70491574024170       -0.12557764164807        0.79281428834603\n",
            "H            0.50523069256964        0.52160818925945       -0.83174385472312\n",
            "H           -0.76767802340165        0.75017840169136        0.34373697851331\n",
            "C           -0.65890388565726       -1.26056511174085       -0.45762583004888\n",
            "H           -1.64801310930718       -1.08892735410568       -0.87178748172606\n",
            "H           -0.04967341913363       -1.73628347267730       -1.22414109839131\n",
            "H           -0.73446623025266       -1.96461533249008        0.36746799365783\n",
            "8\n",
            " energy: -7.334868810470 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01488100547883        0.04561406302855       -0.00290709714506\n",
            "H            0.73290909173915       -0.13169968835498        0.76460574737350\n",
            "H            0.47135558570037        0.54024478322125       -0.84195401032299\n",
            "H           -0.76451643409568        0.73345133580647        0.38104307957524\n",
            "C           -0.65809537408304       -1.26015194667613       -0.45786539339685\n",
            "H           -1.63072995187619       -1.08281081955591       -0.90757448289934\n",
            "H           -0.02877176366408       -1.75491848684234       -1.19559377203487\n",
            "H           -0.77069614824170       -1.94788424062690        0.37690192885041\n",
            "8\n",
            " energy: -7.335300867591 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01491809649035        0.04538279204643       -0.00398234844627\n",
            "H            0.75749460793057       -0.13637170257690        0.73797590502995\n",
            "H            0.44025033281891        0.55732187060467       -0.85000604220355\n",
            "H           -0.76031795360375        0.71728321765113        0.41513203110964\n",
            "C           -0.65706906729126       -1.25992079614765       -0.45818833669657\n",
            "H           -1.61380131260832       -1.07814385982131       -0.93962232968073\n",
            "H           -0.01082698563325       -1.77199397193557       -1.16894022396610\n",
            "H           -0.80423752512256       -1.93171254982079        0.38428734485368\n",
            "8\n",
            " energy: -7.335661983757 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01496895360118        0.04531725667281       -0.00520327943818\n",
            "H            0.77957431765216       -0.13985327970186        0.71248352494250\n",
            "H            0.41113811443126        0.57328977091159       -0.85638837062455\n",
            "H           -0.75518378989139        0.70154454644607        0.44684890580821\n",
            "C           -0.65590081948281       -1.25985539623242       -0.45854676939793\n",
            "H           -1.59711132912385       -1.07466700318035       -0.96892851137522\n",
            "H            0.00488001078943       -1.78796023989581       -1.14361044274370\n",
            "H           -0.83585355077363       -1.91597065502002        0.39000094282891\n",
            "8\n",
            " energy: -7.335954846301 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01497891801514        0.04541495708445       -0.00663663197360\n",
            "H            0.79995297362151       -0.14218532569267        0.68773780448615\n",
            "H            0.38341139106151        0.58851408088727       -0.86157591074932\n",
            "H           -0.74920363886194        0.68569266265241        0.47724092207436\n",
            "C           -0.65454588629301       -1.25995326043259       -0.45901444690514\n",
            "H           -1.58055933607838       -1.07233945346664       -0.99638197717504\n",
            "H            0.01899924830952       -1.80318289320984       -1.11918956386223\n",
            "H           -0.86650183374407       -1.90011576782240        0.39447580410485\n",
            "8\n",
            " energy: -7.336166662800 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01510530123195        0.04563302200698       -0.00795461046095\n",
            "H            0.81869003683020       -0.14377011824897        0.66353409825918\n",
            "H            0.35668575739824        0.60308701425401       -0.86543215102275\n",
            "H           -0.74225867601175        0.67057226968306        0.50579124963167\n",
            "C           -0.65326100889713       -1.26017146643634       -0.45933392834294\n",
            "H           -1.56397211593877       -1.07075896157959       -1.02210695885378\n",
            "H            0.03152993935796       -1.81775409832271       -1.09526965101363\n",
            "H           -0.89573463150680       -1.88499266135643        0.39742795180323\n",
            "8\n",
            " energy: -7.336303823206 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01527195677429        0.04597397829801       -0.00926556991195\n",
            "H            0.83630642895808       -0.14453856001477        0.63960527543678\n",
            "H            0.33060868594646        0.61722981196137       -0.86826924213768\n",
            "H           -0.73440950987311        0.65564353868773        0.53331767747249\n",
            "C           -0.65196935552364       -1.26051255971330       -0.45961308333360\n",
            "H           -1.54727116125075       -1.06999469803625       -1.04668358091224\n",
            "H            0.04288375548893       -1.83189510359584       -1.07162215045080\n",
            "H           -0.92430288697169       -1.87006140758695        0.39918667383703\n",
            "8\n",
            " energy: -7.336366443594 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01562281472304        0.04655294768509       -0.01066319174580\n",
            "H            0.85304891961625       -0.14456858581688        0.61556229700597\n",
            "H            0.30479299861509        0.63140072043421       -0.86990562622509\n",
            "H           -0.72543364581184        0.64110778173836        0.56007755330641\n",
            "C           -0.65053469040450       -1.26109166090785       -0.45974735657064\n",
            "H           -1.53017174559149       -1.06996879235036       -1.07047404183968\n",
            "H            0.05301832487057       -1.84606410289150       -1.04782147670949\n",
            "H           -0.95252334657106       -1.85552330789105        0.39962784277836\n",
            "8\n",
            " energy: -7.336353939675 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01589502120179        0.04710649550386       -0.01185114532222\n",
            "H            0.86920773941076       -0.14389493517341        0.59137180976461\n",
            "H            0.27899978414649        0.64511288134530       -0.87091704404611\n",
            "H           -0.71559095721725        0.62634597557127        0.58642217890628\n",
            "C           -0.64932391399802       -1.26164532332678       -0.45988601265630\n",
            "H           -1.51273901746817       -1.07064653291279       -1.09376312904040\n",
            "H            0.06255611958001       -1.85977428447561       -1.02383402917088\n",
            "H           -0.98064073325204       -1.84075927653182        0.39911337156506\n",
            "8\n",
            " energy: -7.336266809393 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01620900467761        0.04774742758868       -0.01293383316086\n",
            "H            0.88495140247493       -0.14252126262729        0.56669024515876\n",
            "H            0.25290207131730        0.65879300120733       -0.87100904252334\n",
            "H           -0.70460536865499        0.61155213262617        0.61250302118977\n",
            "C           -0.64819850325058       -1.26228635436936       -0.45995022742749\n",
            "H           -1.49470508587387       -1.07202431797569       -1.11682410356472\n",
            "H            0.07132827266083       -1.87345229484362       -0.99925342439148\n",
            "H           -1.00888978399601       -1.82596333160622        0.39743336471939\n",
            "8\n",
            " energy: -7.336105083980 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01655963417672        0.04846305091864       -0.01388104961115\n",
            "H            0.90045183560600       -0.14045216380144        0.54120040412442\n",
            "H            0.22617055693192        0.67256735176010       -0.87013721320323\n",
            "H           -0.69233817064522        0.59661110119796        0.63855308835995\n",
            "C           -0.64718864886416       -1.26300205760411       -0.45993478420209\n",
            "H           -1.47582806273165       -1.07409760224111       -1.13992466843823\n",
            "H            0.07940250489490       -1.88722437403621       -0.97375434073245\n",
            "H           -1.03753638101507       -1.81102030619383        0.39453456370281\n",
            "8\n",
            " energy: -7.335870825005 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01673165412177        0.04909391387518       -0.01452644287555\n",
            "H            0.91597030599750       -0.13762543129367        0.51464030924320\n",
            "H            0.19839155552793        0.68633670216491       -0.86842230357956\n",
            "H           -0.67874737052128        0.58104945371876        0.66486297863435\n",
            "C           -0.64652285023264       -1.26363298062722       -0.45998731678988\n",
            "H           -1.45594787611020       -1.07692864457344       -1.16339838454072\n",
            "H            0.08703038031700       -1.90099127107875       -0.94698688056312\n",
            "H           -1.06686849085655       -1.79545674218577        0.39047404047132\n",
            "8\n",
            " energy: -7.335566972652 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01709713365699        0.04988006073338       -0.01499197124393\n",
            "H            0.93137256481723       -0.13427136365585        0.48630797066450\n",
            "H            0.16924694963837        0.70076336578723       -0.86507419877558\n",
            "H           -0.66340165406505        0.56573735331046        0.69118826091212\n",
            "C           -0.64596228119778       -1.26441915114568       -0.45979753500036\n",
            "H           -1.43435785902991       -1.08028721601292       -1.18735233858514\n",
            "H            0.09357266530164       -1.91541518293395       -0.91838800370885\n",
            "H           -1.09679925180751       -1.78014286608266        0.38476381573727\n",
            "8\n",
            " energy: -7.335184558881 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01739300225101        0.05068540252720       -0.01528686383551\n",
            "H            0.94729498915940       -0.12983174511701        0.45559467804084\n",
            "H            0.13787892220448        0.71584069509140       -0.86028840532344\n",
            "H           -0.64582859893015        0.54927380797907        0.71870615583524\n",
            "C           -0.64558578997857       -1.26522450279048       -0.45961659794477\n",
            "H           -1.41069586944999       -1.08473165399809       -1.21258916431074\n",
            "H            0.09949924004835       -1.93048941458123       -0.88721387860555\n",
            "H           -1.12859589080251       -1.76367758911086        0.37735007614395\n",
            "8\n",
            " energy: -7.334730284445 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01759359453571        0.05146586486452       -0.01528185822493\n",
            "H            0.96375436253678       -0.12433080371537        0.42172460447753\n",
            "H            0.10368085787678        0.73177900702085       -0.85352223900268\n",
            "H           -0.62564117577546        0.53168455273550        0.74739520570432\n",
            "C           -0.64552380697006       -1.26600494840652       -0.45942568750457\n",
            "H           -1.38423590234480       -1.09023782086456       -1.23938292220887\n",
            "H            0.10450025583797       -1.94642417159395       -0.85271182397550\n",
            "H           -1.16236699662550       -1.74608668004045        0.36786072073473\n",
            "8\n",
            " energy: -7.334208930714 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01761352981866        0.05215857545738       -0.01490514101658\n",
            "H            0.98103537526972       -0.11748315593977        0.38337423155854\n",
            "H            0.06548806736023        0.74893199827882       -0.84405443916131\n",
            "H           -0.60203401152920        0.51238051811761        0.77779346508641\n",
            "C           -0.64587246564767       -1.26669761331198       -0.45928142059821\n",
            "H           -1.35382468451718       -1.09709127527845       -1.26844256444615\n",
            "H            0.10828321095136       -1.96357297405934       -0.81354345164497\n",
            "H           -1.19888796206861       -1.72678107326427        0.35571532022231\n",
            "8\n",
            " energy: -7.333633985662 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01732901279488        0.05266935226590       -0.01408788221282\n",
            "H            0.99938908884578       -0.10881583524654        0.33869185507839\n",
            "H            0.02173955671704        0.76769629810544       -0.83077131353172\n",
            "H           -0.57383110256239        0.49047733023543        0.81056295519113\n",
            "C           -0.64673786639711       -1.26720832006496       -0.45927763572244\n",
            "H           -1.31779969830770       -1.10576521659954       -1.30062111468472\n",
            "H            0.11031745129859       -1.98233219407193       -0.76786603800340\n",
            "H           -1.23917441679935       -1.70487641462380        0.34002517388561\n",
            "8\n",
            " energy: -7.333041355774 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01694681801821        0.05314002955908       -0.01314735717719\n",
            "H            1.01800408679644       -0.09772544167127        0.28697047039510\n",
            "H           -0.02781391274292        0.78771608219226       -0.81321057501075\n",
            "H           -0.53993869152348        0.46510050153298        0.84614284002398\n",
            "C           -0.64775201407438       -1.26767892071223       -0.45932493435141\n",
            "H           -1.27522422194942       -1.11686306013383       -1.33538867640327\n",
            "H            0.11025003619270       -2.00234593738614       -0.71529072937391\n",
            "H           -1.28400446468074       -1.67949825338085        0.31990496189749\n",
            "8\n",
            " energy: -7.332525545051 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01560575238942        0.05280430001444       -0.01139628936166\n",
            "H            1.03780565185171       -0.08328987839661        0.22232796145885\n",
            "H           -0.08812486255365        0.81009142583205       -0.78702384101393\n",
            "H           -0.49761741123636        0.43420111775897        0.88507777046751\n",
            "C           -0.64984957930445       -1.26734309912083       -0.46000673660391\n",
            "H           -1.22085958357940       -1.13130765328402       -1.37557550507949\n",
            "H            0.10562885949919       -2.02471337607882       -0.64969992141033\n",
            "H           -1.33480332228763       -1.64859783672518        0.29295256154301\n",
            "8\n",
            " energy: -7.332248232936 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01348143912452        0.05159804133227       -0.00942278928581\n",
            "H            1.05614464963472       -0.06479139629042        0.14483427714534\n",
            "H           -0.15803952959553        0.83304421543574       -0.75099067498168\n",
            "H           -0.44616336615980        0.39681345374662        0.92598143424194\n",
            "C           -0.65241752351610       -1.26613678565634       -0.46135319230836\n",
            "H           -1.15389004792523       -1.14981652184106       -1.41866015278368\n",
            "H            0.09491918982160       -2.04765634937703       -0.57177557761922\n",
            "H           -1.39049793313514       -1.61120965734977        0.25804267559150\n",
            "8\n",
            " energy: -7.332374812909 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01109941953477        0.04969797890894       -0.00798390831456\n",
            "H            1.06903264400914       -0.04339665876944        0.06224433074316\n",
            "H           -0.22994349180060        0.85295648008947       -0.70784063835151\n",
            "H           -0.38968211406109        0.35555218443365        0.96387459143651\n",
            "C           -0.65456705227742       -1.26423675209357       -0.46312068409060\n",
            "H           -1.08030072446610       -1.17122178802880       -1.45830054237967\n",
            "H            0.07816082012914       -2.06755781977157       -0.48960689654621\n",
            "H           -1.44502666199830       -1.56994862476868        0.21738974750292\n",
            "8\n",
            " energy: -7.332839884916 xtb: 6.7.0 (08769fc)\n",
            "C           -0.00936352571414        0.04787033867706       -0.00754214286004\n",
            "H            1.07447926012723       -0.02256682172088       -0.01121968391134\n",
            "H           -0.29198085235338        0.86693586169780       -0.66576819279132\n",
            "H           -0.33686015644424        0.31697439029419        0.99338661585108\n",
            "C           -0.65556126452556       -1.26240920236499       -0.46461077009852\n",
            "H           -1.01284010761510       -1.19206050121825       -1.48788638883727\n",
            "H            0.05913502446414       -2.08152733178496       -0.41710158795981\n",
            "H           -1.49043437793895       -1.53137173357997        0.17739815060725\n",
            "8\n",
            " energy: -7.333427301329 xtb: 6.7.0 (08769fc)\n",
            "C           -0.00848625687541        0.04652598869296       -0.00775169289659\n",
            "H            1.07459251501131       -0.00426420082418       -0.07065349897014\n",
            "H           -0.34105245510122        0.87591260183827       -0.62921720900214\n",
            "H           -0.29154350686092        0.28437219311608        1.01452375207454\n",
            "C           -0.65565554938589       -1.26106494749161       -0.46550792048500\n",
            "H           -0.95683431390588       -1.21036995494035       -1.50777337454571\n",
            "H            0.04100067295007       -2.09049588902062       -0.35866111607469\n",
            "H           -1.52544710583207       -1.49877079137055        0.14169705989978\n",
            "8\n",
            " energy: -7.333999952458 xtb: 6.7.0 (08769fc)\n",
            "C           -0.00784291355509        0.04525110391915       -0.00810474447436\n",
            "H            1.07173905313602        0.01190338570296       -0.12047789805816\n",
            "H           -0.38113768236822        0.88168860617839       -0.59643850711242\n",
            "H           -0.25240077114066        0.25662694146060        1.02969265520118\n",
            "C           -0.65553665935867       -1.25979015454824       -0.46623224013937\n",
            "H           -0.90890238802059       -1.22654302980070       -1.52166459360721\n",
            "H            0.02343269691844       -2.09626487440290       -0.30995058285541\n",
            "H           -1.55277733561123       -1.47102697850925        0.10983191104579\n",
            "8\n",
            " energy: -7.334542794917 xtb: 6.7.0 (08769fc)\n",
            "C           -0.00810393866175        0.04477578744739       -0.00874282998812\n",
            "H            1.06700537621892        0.02662222750448       -0.16177163516435\n",
            "H           -0.41420655191838        0.88591995011412       -0.56809993017208\n",
            "H           -0.21693095533209        0.23263365352007        1.04145130765435\n",
            "C           -0.65484803600645       -1.25931489073012       -0.46619853681292\n",
            "H           -0.86838877780603       -1.24126622771418       -1.53094357768712\n",
            "H            0.00771643588552       -2.10049027818000       -0.26933439826855\n",
            "H           -1.57566955237973       -1.44703522196175        0.08029560043883\n",
            "8\n",
            " energy: -7.335020003605 xtb: 6.7.0 (08769fc)\n",
            "C           -0.00869517267906        0.04458971741204       -0.00935604852847\n",
            "H            1.06114904692973        0.04020498115585       -0.19764292494630\n",
            "H           -0.44263997460867        0.88895933603196       -0.54232137517113\n",
            "H           -0.18431077240003        0.21148257102162        1.05055816132673\n",
            "C           -0.65407297813044       -1.25912884329944       -0.46584514145358\n",
            "H           -0.83261467157163       -1.25485262429481       -1.53735921403699\n",
            "H           -0.00712856340641       -2.10352439525774       -0.23394158325035\n",
            "H           -1.59511291413349       -1.42588574276946        0.05256412606013\n",
            "8\n",
            " energy: -7.335429032595 xtb: 6.7.0 (08769fc)\n",
            "C           -0.00949200415392        0.04458185452571       -0.00988653149493\n",
            "H            1.05443220358377        0.05316243930073       -0.23010932334550\n",
            "H           -0.46807990134817        0.89121586840042       -0.51790901773671\n",
            "H           -0.15330648354944        0.19227974306579        1.05776390058951\n",
            "C           -0.65330751955606       -1.25912097680242       -0.46527031491977\n",
            "H           -0.79976484508873       -1.26781326360258       -1.54183016938302\n",
            "H           -0.02168157327698       -2.10577605799609       -0.20182640026915\n",
            "H           -1.61222587661048       -1.40668460689155        0.02572385655960\n",
            "8\n",
            " energy: -7.335765977031 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01044473947794        0.04472107828701       -0.01030937188905\n",
            "H            1.04700788587754        0.06570132816931       -0.26001590243518\n",
            "H           -0.49122753620123        0.89291000770091       -0.49440965792894\n",
            "H           -0.12340128417901        0.17465558997225        1.06344846083586\n",
            "C           -0.65259168663516       -1.25926017165631       -0.46451262783813\n",
            "H           -0.76909336649393       -1.28035498290308       -1.54478201313289\n",
            "H           -0.03613657678602       -2.10746561924643       -0.17217675833079\n",
            "H           -1.62753869610425       -1.38906223032365       -0.00058612928085\n",
            "8\n",
            " energy: -7.336033371123 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01160401777332        0.04500322917170       -0.01054182797584\n",
            "H            1.03892810080300        0.07802467738361       -0.28808734250263\n",
            "H           -0.51284376632744        0.89401867973043       -0.47152738647103\n",
            "H           -0.09366807530157        0.15821390029147        1.06807645125007\n",
            "C           -0.65198664715742       -1.25954225503674       -0.46349678948062\n",
            "H           -0.73993420258211       -1.29268089835303       -1.54650496980681\n",
            "H           -0.05051935155432       -2.10856990847238       -0.14417666481783\n",
            "H           -1.64179804010681       -1.37262242471505       -0.02708547019526\n",
            "8\n",
            " energy: -7.336220489565 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01275042235836        0.04536052815074       -0.01071317846678\n",
            "H            1.03026093974334        0.09020435916553       -0.31459419336675\n",
            "H           -0.53276720719915        0.89496959965149       -0.44880022196533\n",
            "H           -0.06489467267136        0.14297756993930        1.07134589419674\n",
            "C           -0.65144352001297       -1.25989948056670       -0.46247274389421\n",
            "H           -0.71205489988900       -1.30486291939043       -1.54715329201825\n",
            "H           -0.06531928822111       -2.10951660061528       -0.11782448456225\n",
            "H           -1.65445692939139       -1.35738805633465       -0.05313177992313\n",
            "8\n",
            " energy: -7.336332889035 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01394767817782        0.04579981612725       -0.01074884014815\n",
            "H            1.02099648240032        0.10241002675018       -0.33998763051367\n",
            "H           -0.55153548982104        0.89563760471522       -0.42606927186521\n",
            "H           -0.03627707135032        0.12864620209169        1.07361086487891\n",
            "C           -0.65101142591967       -1.26033867537468       -0.46135557346742\n",
            "H           -0.68502672017866       -1.31707074975467       -1.54686777703050\n",
            "H           -0.08050726102461       -2.11018047153222       -0.09256036942919\n",
            "H           -1.66611683592820       -1.34305875302277       -0.07936540242473\n",
            "8\n",
            " energy: -7.336370557561 xtb: 6.7.0 (08769fc)\n",
            "C           -0.01531752180721        0.04644713871308       -0.01080659584368\n",
            "H            1.01109997257679        0.11500870910621       -0.36465349783145\n",
            "H           -0.56926527789264        0.89644722976473       -0.40267794603379\n",
            "H           -0.00766850896699        0.11502582276095        1.07466891472725\n",
            "C           -0.65050106899959       -1.26098589332565       -0.46008298603301\n",
            "H           -0.65847428117183       -1.32967146035618       -1.54574403437608\n",
            "H           -0.09666362561468       -2.11098597110248       -0.06805571634325\n",
            "H           -1.67663568812384       -1.32944057556066       -0.10599213826596\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "def extract_energies_from_xyz(filename):\n",
        "    energies = []\n",
        "\n",
        "    with open(filename, 'r') as file:\n",
        "        lines = file.readlines()\n",
        "\n",
        "        for line in lines:\n",
        "            if line.startswith(' energy:'):\n",
        "                energy = float(line.split()[1])\n",
        "                energies.append(energy)\n",
        "\n",
        "    return energies\n",
        "\n",
        "# Exemplo de uso da fun√ß√£o\n",
        "filename = 'xtbscan.log'  # Substitua pelo nome do seu arquivo XYZ\n",
        "energies = extract_energies_from_xyz(filename)\n",
        "print(energies)\n",
        "\n",
        "# Definir os par√¢metros do scan\n",
        "start_angle = 60.0  # √¢ngulo inicial em graus\n",
        "end_angle = 420.0   # √¢ngulo final em graus\n",
        "num_steps = 72      # n√∫mero de passos\n",
        "\n",
        "# Gerar os √¢ngulos usando linspace do NumPy\n",
        "angles = np.linspace(start_angle, end_angle, num_steps)\n",
        "\n",
        "# Imprimir a lista de √¢ngulos\n",
        "print(angles)"
      ],
      "metadata": {
        "id": "Vs0r5ffQRzKb",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "e8e562ad-bb15-4115-ce28-52b51b3789ee"
      },
      "execution_count": 47,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "[-7.336369756748, -7.336333263138, -7.336221222362, -7.336034699107, -7.335775709481, -7.335441101708, -7.335040458077, -7.334562462917, -7.33401974468, -7.333435876664, -7.33285247903, -7.332394796759, -7.332242402076, -7.33250135074, -7.333026747449, -7.333623545214, -7.334186889397, -7.334710975474, -7.335166146751, -7.335558009938, -7.335868616349, -7.336104810094, -7.336266660115, -7.336353895727, -7.336366557842, -7.336304648329, -7.336168166528, -7.335957146662, -7.335671767187, -7.335311661773, -7.334887494143, -7.334385847854, -7.333824738817, -7.333230164518, -7.332677405655, -7.332300218703, -7.332285767003, -7.332659680644, -7.33322518229, -7.333807783948, -7.33437060886, -7.33486881047, -7.335300867591, -7.335661983757, -7.335954846301, -7.3361666628, -7.336303823206, -7.336366443594, -7.336353939675, -7.336266809393, -7.33610508398, -7.335870825005, -7.335566972652, -7.335184558881, -7.334730284445, -7.334208930714, -7.333633985662, -7.333041355774, -7.332525545051, -7.332248232936, -7.332374812909, -7.332839884916, -7.333427301329, -7.333999952458, -7.334542794917, -7.335020003605, -7.335429032595, -7.335765977031, -7.336033371123, -7.336220489565, -7.336332889035, -7.336370557561]\n",
            "[ 60.          65.07042254  70.14084507 ... 409.85915493 414.92957746\n",
            " 420.        ]\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "%matplotlib inline\n",
        "import matplotlib.pyplot as plt\n",
        "\n",
        "# Plotar os √¢ngulos e os valores de energia\n",
        "plt.figure()\n",
        "plt.plot(angles, energies, 'r-o')\n",
        "plt.xlabel('Dihedral angle (degrees)')\n",
        "plt.ylabel('Energy (hartrees)')\n",
        "plt.show()"
      ],
      "metadata": {
        "id": "RNh5mZi9FD3D",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 449
        },
        "outputId": "0e43c90e-4730-47f1-ff3c-e1aa11bfd749"
      },
      "execution_count": 48,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<Figure size 640x480 with 1 Axes>"
            ],
            "image/png": "iVBORw0KGgoAAAANSUhEUgAAAlQAAAGwCAYAAABvpfsgAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjguMCwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy81sbWrAAAACXBIWXMAAA9hAAAPYQGoP6dpAAB6R0lEQVR4nO3deVxU9f4/8NewCCiCGwgICq5kqblVaqamuVRGkWlqVyu92U1vWbbYvtzK7N5Ku992Datf2aKU1dVME82tUnPfF9wQtVBBkEXg8/vj05kFZmCGWc72ej4e85iZM4eZz2HOnPM+n+X9sQghBIiIiIiozoLULgARERGR3jGgIiIiIvISAyoiIiIiLzGgIiIiIvISAyoiIiIiLzGgIiIiIvISAyoiIiIiL4WoXQAzqKysxIkTJ9CwYUNYLBa1i0NERERuEELg/PnzSEhIQFBQzXVQDKgC4MSJE0hKSlK7GERERFQHx44dQ2JiYo3rMKAKgIYNGwKQX0hUVJTKpSEiIiJ3FBQUICkpyXoerwkDqgBQmvmioqIYUBEREemMO9112CmdiIiIyEsMqIiIiIi8xICKiIiIyEsMqIiIiIi8xICKiIiIyEsMqIiIiIi8xICKiIiIyEsMqIiIiIi8xICKiIiIyEvMlE7mUVEBrF4N5OYC8fFA375AcLDapSIicsRjlS4xoCJzyMwEHngAOH7ctiwxEZg9G0hPV69cRET2eKzSLTb5kfFlZgIjRjgeoAAgJ0cuz8xUp1xERPZ4rNI1ixBCqF0IoysoKEB0dDTy8/M5OXKgVVQAycnVD1AKi0Ve/WVns0qdvMemGqorHqs0yZPzN2uoyNhWr3Z9gAIAIYBjx+R6RN7IzJQnxAEDgDFj5H1yMmsVyD08VukeAyoyttxc365HVFEBrFwJzJ8v7ysq2FRD3uOxSvfYKZ2MLT7et+uRuTnrMNyiBVBSImsQqhJCNtVMnQqkpbGphlzjsUr3WENFxta3r+x34IrFAiQlyfWIalJTLVRenuu/Y1MNuUM5Vlkszl/nsUrzGFCRsQUHA2+8UfM6s2ax5oBqVlEha6a8GcPDphqqSXCwTI1Q0z7GY5WmMaAi4yspkfdVr/zq1wcWLGBuF6pdbR2G3cGmGqpNejoweLDz1+65h8cqjWNARcZWXg48/7x8/K9/AVlZwJNPyucREbJfC1FtvKldYlMNuUsIYO9e+fjll4HPPgMmT5bPv/gCOH1avbJRrZiHKgCYh0pFH34ITJgAxMQAhw4BkZHAxYvyeX4+sGYN0KeP2qUkrVu5UqZB8JRSK8qaUHLHnj3AJZcA9eoBZ84ADRrIi8IrrgA2bwbGjgUmTmSeswBiHioyN2VY+yefAE88IZc99pgMpgAgNBQYNkw+/u47VYpIOuNOh+GmTasPgEhMZDBF7vvhB3l/zTUymAKAkBDg3Xfl408/ZZ4zDWNARcZin1xx3Djg1CkgKAhISHBcb/hwec+AityhdBh2Rgmy3n8fOHzY1kTTq5fMas1gitylBFRDhzoud9V/j3nONIUBFRmHq2HtlZWyqtz+oDNsmDxJ7tolmwKJapOeLmubQqqk77OvhQoOlrUHgNyvgniIJTcVFwOrVsnH9gGVMsLUGaXHztSpcj1SFX/tZAzuDGu3P+g0bmzrJMxaKnLXtdfKPi0A8N57cpBD1Vqoyy+XgdWpU96PDCTzWLVKjkhOTAQ6drQt55Q0usGAioyhLgcdNvuRpzZtkvcpKXIYe//+1TsF168PdOokH2/YENDikY7ZN/fZ99XjlDS6wYCKjKEuBx0loFq1So74I6qNEiD17Fnzesrrv/3m3/KQcbjqP8UpaXSDARUZQ10OOu3aAR06yCYc5WBGVBMloOrRo+b1rrjCcX2immRny/xTwcHAoEGOr3FKGt1gQEXGUNeDDpv9yBOe1lBt3CgHRRDVZOlSed+7NxAd7fia/QjTqsc35TmnpNEEBlRkDO4Ma3d20FECqsWLbZ2NiZw5dUr2w7NYgO7da1730ktlJv6CAmDfvsCUj/TLVXOfQhlh2qKF43LmOdMUBlRkHOnpwJdfVr+Kq+mg07s30KQJcPYs8NZbwPz5MikohyBTVUrtVGoq0LBhzeuGhADdujn+HZEzZWXATz/Jx64CKkAevw4fBhYutC3buZPBlIYwoCJj6dZNjugLDZWZ0p0Na7cXEgJcdpl8PHUqMxCTa+429ynYMZ3csW4dUFgIxMbKlBs1CQ4GbrlFZuUHgP37/V48ch8DKjKWHTvkfceOwB13OB/Wbi8zE/j55+rLmYGYqvI0oGLHdKqJMkXW66/L59dd514iWItFNikDsoaKNIMBFRmLcoBRap1qwgzE5C4hZAdzwPMaqs2bZbMOkcJ+iixlQMySJe5fwDGg0iQGVGQsSg2VcsCpCTMQk7uOHgX++EM2EXfp4t7ftGkjM/KXlQHbt/u3fKQfrqbIOnvW/VpxBlSaxICKjEUJqNypoWIGYnKX0mzXuTMQHu7e31gs7EdFjmqaIsuTWnEGVJrEgIqMo7wc2LNHPnYnoGIGYnKXp/2nFMr67EdFgO9qxZWAKjsbKCryXfnIKwyoyDgOHJDNK/XrA61a1b4+MxCTu+oaULFjOtnzVa14TIy8AcDu3d6ViXyGARUZh33/KXdGyzADMbmjstI2KXJda6h27ZJD48ncfFkrzmY/zWFARcbhyQg/BTMQU2327ZMZzyMiZDoOT8THy32pshL4/Xf/lI/0w5e14gyoNIcBFRmHJyP87CkZiJXM1o8+WnMyUDIXpbmuWzc5ys9T7JhOirpOkeUMAyrNYUBFxuHJCL+qgoOBq66Sj4OC2MxHNnXtP6VgPyqyp9SKV50E2dNacQZUmsOAioyhtNQ2DUNdAioAaN9e3nMyW7LnbUDFGiqqKj0duO02+fjmm2ufIssZJaA6coT98zSCARUZw969MndLdDSQkFC392BARVVdvAhs2SIf1zWg6t5d3h8+DLz3HiffJkm5ALz11tqnyHKmaVOgeXP5eNcunxaN6oYBFRmDfXOfqw6ftVECqv37ZSdiMreKCmDePKCkBGjQAEhJqdv7rFhh63t1772cfJukvXvlvXLcqQullooBlSYwoCJjqMsIv6patQJCQ2Xz4bFjvikX6ZMy19o998jnRUUyoPI0CFKmGSkvd1zOybfNraAAOHlSPu7Qoe7vw35UmsKAioyhriP87IWEyPnXADb7mZmrudY8DYJ8Nc0IGY9yfGnevHrndE8woNIUBlRkDN6M8LPHflTm5ssgiJNvkyu+aO4DGFBpDAMq0r+iIjlCBvCuhgpgQGV2vgyCOPk2uaIcX7xp7gNsx7ujR4Hz5717L/IaAyrSv9275YkuJgaIjfXuvRhQmZsvgyBOvk2uKDVU3gZUjRvb9h92TFcdAyrSP1819wEMqMzOl0EQJ98mV3wVUAFs9tMQBlSkf8qBxNvmPsAWUB0+LEf7kbn4Mgji5NvkTGWl7YLN2z5UgG1+SQZUqmNARfrnyxqquDggMlIe9A4d8v79SF98OdcawMm3qboTJ4ALF+So4tatvX8/1lBpBgMq0j9fBlQWC5v9zE4JgurVc1xe1yBImXz7ySfl886dOfm2mSnNfa1by7x33mJApRkMqEjf8vNto7J80eQHMKAiGew0aiQfv/RS3eZasxccDAwfLh//+Seb+czMl/2nANtx7/hxeTwk1TCgIn1TrspatLCdAL3FgIrOnwdOn5aPJ0+u21xrVbVtK+9PnACKi717L9IvX/afAuRxT5m/lCP9VMWAivTNF1POVMWAig4elPcxMd5lsrbXpIntvdg/z7x8XUMFsNlPIxhQkb75YsqZqhhQ0f798l6pVfIFi8U2tdGBA757X9IXBlSGxYCK9M2XHdIVSkB18qScxJTMRwl4fBlQ2b+fUgNG5lJSIgcoAL5r8gOASy6R98uXAytXcn5IlTCgIn2qqJAHjo0b5XPlgOIL0dFy0lLAVlNB5uKvgEqpoWJAZU4HD8pZHaKibMcYb2VmAk8/LR/v2AEMGAAkJ7s/iTf5DAMq0p/MTHnAGDDAVoM0YoRvDyBs9jM3JaBq186378smP3Ozb+5zlTzWE5mZ8tinDKBQ5OT4/phItWJARfqiHECqTmB74oRvDyAMqMyNTX7kD77sP1VRATzwgKzxqkpZNnUqm/8CiAEV6UcgDyAMqMyrqEgG6ID/mvyOHAEuXvTte5P2KQGVL/pPrV5d/cLSnhDAsWNyPQoIBlSkH4E8gDCgMi8lpUGTJkDjxr5974QEICwMKC8Hjh717XuT9inHE1/UUOXm+nY98hoDKtKPQB5A7AMqZzViZFz+SJmgCApix3Qz82WTX3y8b9cjrzGgIv0I5AGkTRvZabSgoHqHTzI2f/WfUjCgMqc//wTOnJGPfTHYoW9fOb+kq87tFguQlCTXo4BgQEX6EcgDSFiYHEkIsNnPbAIVUHGkn7kox5GkJKB+fe/fLzgYmD1bPq56TFSez5rFeSMDiAEV6UegDyDsR2VO/g6oONLPnPyRIT09HViwQM5lai8hQS6v62TeVCcMqEhflANIs2aOyxMTfX8AYUBlTv7KQaVgk585+SOgAuQx7/BhICsLaNhQLlu0iMGUChhQkf6kpwMvvigfd+4sDyTZ2b4/gDCgMp/iYjlSFAhMHyoOeDAPX6ZMqCo4GOjfH+jYUT4/csT3n0G1YkBF+qTMh3X11fJA4o9+AgyozCc7W95HRwNNm/rnM1q1kqP9ios5pN1MfJkywZXWreW9kvqDAooBFemTcsBQDiD+oARUBw4w27BZ2KdM8MXUIM7UqyeDKoDNfmZRUWFrSmZAZVi6CKhWrlwJi8Xi9LZhwwaXfzdp0iS0adMGERERiImJQVpaGvbs2WN9PS8vD0OHDkVCQgLCwsKQlJSEKVOmoECZHw7AmjVr0KdPHzRt2hQRERFITU3FG2+84dftJTcoBwyl+cQfkpLkaL+yMiZhNAt/d0hXsB+VuRw+LI8jYWFAy5b++xwGVKrSRUDVu3dv5ObmOtwmTpyIlJQU9OjRw+Xfde/eHRkZGdi9ezeWLl0KIQQGDx6Mir9qG4KCgpCWloZvv/0W+/btw7x587B8+XLce++91vdo0KABpkyZgp9//hm7d+/GU089haeeegrvv/++37ebaqCciPxZQxUcbDvxvfcesHIla6qMLtABFVMnmIPS3NeunWzu9RcGVOoSOlRWViZiYmLECy+84NHfbd26VQAQBw4ccLnO7NmzRWJiYo3vc8stt4g77rjD7c/Nz88XAER+fr7bf0M1OHtWCNmdV4iCAv99zsKFQoSH2z4LECIxUS4nYxo0SH7PGRn+/Zx//1t+zu23+/dzSH3l5UJMniy/72uukc/95cgR+Tmhof79HBPx5Pytixqqqr799lvk5eXhrrvucvtvioqKkJGRgZSUFCQlJTld58SJE8jMzES/fv1cvs/mzZuxbt26GtcpLS1FQUGBw418SOk4HBNjGybsa5mZwIgRQEmJ4/KcHLk8M9M/n0vqYpMf+VJmpkwQ/NZb8vnPP8vn/jp+tGgBhIbKibdzcvzzGeSSLgOquXPnYsiQIUhMTKx13bfffhuRkZGIjIzEkiVLsGzZMtSrV89hndGjR6N+/fpo0aIFoqKiMGfOnGrvk5iYiLCwMPTo0QOTJ0/GxIkTXX7mjBkzEB0dbb25CuCojvzdf6qiAnjgAedD2pVlU6ey+c9oSkttfeX8lYNKwSY/41MuyqpO6O7Pi7LgYNsMD2z2CzhVA6rp06e77Gyu3Ow7kQPA8ePHsXTpUkyYMMGtzxg7diw2b96MVatWoX379hg5ciRKqtQ6vPHGG/j999+xaNEiHDx4EA899FC191m9ejU2btyId999F7NmzcL8+fNdfubjjz+O/Px86+2YkteGfMPf/adWr65+ELQnhMxVtHq1fz6f1JGdDVRWApGRQGysfz9LCajOnpU3MhY1L8rYj0o1IWp++LRp03DnnXfWuE7rKifNjIwMNG3aFDfddJNbn6HUErVr1w5XXXUVGjdujK+//hqjR4+2rhMXF4e4uDikpqaiSZMm6Nu3L55++mnE202ym5KSAgDo1KkTTp06heeee87hPeyFhYUhLCzMrfJRHfg7ZYK7uYGYQ8hY7Jv7/JUyQdGgARAXB5w8KS8QahhcQzrkyUVZ//6+/WwGVKpRNaCKiYlBTEyM2+sLIZCRkYFx48YhNDTU488TQkAIgdLSUpfrVFZWAkCt69T0OvmZvwMqu0DaJ+uRPgSq/5SiTRsZUB04wIDKaNS8KGNApRpVAypPrVixAtnZ2U77L+Xk5GDgwIH4+OOPccUVV+DQoUP44osvMHjwYMTExOD48eN45ZVXEBERgeuvvx4AsHjxYpw6dQo9e/ZEZGQkdu7ciUceeQR9+vRB8l/t0G+99RZatmyJ1NRUAMDPP/+M//znP7j//vsDtt1Uhb/7UPXtK+cGzMlxXmVvscjX+/b1z+eTOgIdULVtC6xdy47pRqTmRRkDKtXoKqCaO3cuevfubQ1u7F28eBF79+7FhQsXAADh4eFYvXo1Zs2ahbNnz6J58+a45pprsG7dOsT+1T8iIiICH3zwAR588EGUlpYiKSkJ6enpmD59uvV9Kysr8fjjjyM7OxshISFo06YNZs6ciUmTJgVmo8lRebltnip/1VAFBwOzZ8uOoxaLY1ClNAXNmuWf6W5IPWrUUAEMqIxIzYsyBlSqsQjB2Tn9raCgANHR0cjPz0dUVJTaxdG37Gx5wKhXT86F5s8keZmZsmOpfV+IpCQZTHEmd+Np21YGNytXAjWkRfGZzz4Dxo6VJ9Wff/b/51FgKaP8qp5ilYuyBQv8cxzJzwcaNZKPCwr8l1rGJDw5f+sybQKZmHLVlZLi32AKkAe7w4eB++6Tz/v3lwEdgynjuXjRNuE2a6jIF9LTZdAUHu64PDHRf8EU4Dixt5KzjwKCARXpSyDm8LMXHAxcd518XFTEZj6jOnxYDmGPiAASEgLzmUrgduIE8FdXBTKY9HTbRNiPPw5kZQXmoozNfqpgQEX64u8Rfs78lTKDBycDC2TKBEWTJrI2AeC+ZVSVlbY+nxMnylruQFyUMaBSBQMq0pdATIpclRJQ5eXJPglkPIHukA7IwI3NfsZ28qScvio4WPa/DBQGVKpgQEX6EugmPwCIimKfBKNTI6Cy/zwGVMakHK+SkuQce4HCgEoVDKhIX9Ro8rP/PAZUxqRWQMU5/YxNOV6odbxiQBVQDKhIP+znPVOa4QJF+TwGVMZTUQFs2yYfFxYGdtJrZb9as0ama+CE28aihQvAv2b/IP9jQEX6oRycmjeXc6EFEq/4jCkzE0hOtuUamzZNPs/MDMxnP/mkfLx9OzBgQOA+mwJDrYAqMREICQHKyuQoUgoIBlSkH2r0n1Kwhsp4lMSLVSexzcmRy/0Z2Cif/ccfgf9sChz7vHmBFBJiS9fAi8CAYUBF+qHW1Z79Z/LgZAwVFTILvrOJIpRlU6f6pwlOzc+mwFKrD5X9Z/KYFTAMqEg/1Ayo7GuoOFuT/q1eXb1myp4QwLFjcj0jfTYFTkmJrHEEGFCZBAMq0g81A6qWLeVUNyUlMrcM6Vturm/X08tnU+AoUxlFRtrSrgQSA6qAY0BF+qHk6lGjD1VoqC0xH/tR6V98vG/X08tnU+DYN/cFKvu+PQZUAceAivTh4kXg6FH5WI0aKvvP5QFK//r2lSOhXJ3oLBYZQPfta6zPpsBRs0bd/nN5vAoYBlSkD8eOyU664eFAXJw6ZeBIP+MIDgZmz3b+mhLozJrln3nX7D+7alDl78+mwFFrhJ9CCahOnZITu5PfMaAifbA/OAWptNvyis9Y0tOBDz+svjwxEViwQL7uz89esABo0SLwn02BoeYIPwBo1Aho3NixLORXDKhIH9TsP6VgDZXxXHqpvG/aFPjsMyArS36/gQho0tNlx+UffrAt27CBwZRRqN3kZ//ZvAgMCAZUpA88OJE/KN9laiowejTQv39gm9qCg4EhQ4CEBPn8yJHAfTb5jxA8ZpkQAyrSBy0cnJQaquPH5ZQOpH9a2K8AW82rUhNL+paXB5w/Lx8nJ6tXDgZUAcWAivRBCye+2Figfn159cmaBGNQmm/V6jis4InPWJT9KiFBDqRRC/ergGJARdonhDb6UFks7EdlNFoI1O0/nyc+Y+B+ZUoMqEj7zp4F8vPlYzWrzwEeoIxG7ZFYCuVCgfuVMaidMkGh7NfZ2UBlpbplMQEGVKR9ysEpPl42uamJNVTGUV5ua7rVyomPfaiMQSuBelKSbcqsd98FVq7kpNt+xICKtK2iAvjuO/m4aVP1DwYMqIxDSRZbr55tlJ1alBPv8eNAaam6ZSHvaaXJ77vvbMliJ08GBgyQtfyZmaoWy6gYUJF2ZWbKH/8LL8jnO3aofzBgk59xKEFxcrJ6yWIVsbFAgwYc8GAUWmjyy8wERoyofhGakyOXM6jyOQZUpE3KweD4ccflah8MWENlHFqpRQBkLQKb/YyhvFz9eUcrKoAHHpABelXKsqlT1a/xNxgGVKQ9Wj4YKAHVmTO2jvKkT1rp56Jg7acxKE3JYWGy36caVq+ufjFqTwhZztWrA1cmE2BARdqj5YNBZCQQEyMfs5ZK37TQLGOPAZUxKN+fmk3Jubm+XY/cwoCKtEfrBwOe+IxBS01+ALOlG4UW9it3a8bUqkEzKAZUpD1aPxiwH5UxaCVLuoKBujFooSm5b18gMdE2wq8qi0WmVOjbN7DlMjgGVKQ9Wj8Y8MSnf4WFwB9/yMdaqaGy36+c9R8kfdBCDVVwMDB7tnxc9TiqPJ81K7ATgZsAAyrSHvuDQVVaOBiwhkr/lO+uSRMgOlrdsiiSk+X+XVRkC/ZIf7TSNy89HViwAGjRwnF5YqJcnp6uTrkMjAEVaZNyMIiMdFyuhYMBa6j0TysnPXthYXL/BtiPSs+00OSnSE8HDh+WCT0B4N57ZfkYTPkFAyrSrvR0YPBg+Xj8eCArSxsHA+UkfPgw58fSKy00yzjDYF3fCgqAP/+Uj7USrAcHA716yccWC5v5/IgBFWmbcmK57Tagf39tHAySkmQ5Sks57FivtNYhXcGASt+U/appUyAqSt2y2OMI0oBgQEXaJYQ2axJCQoCWLeVj9qPSJy3uVwBPfHrH/crUGFCRdp05I6vQAdlhV0tYk6BvWurnYo/7lb5pdb9SAqojR+TUOOQXDKhIu5STSosWQESEumWpiiP99Mu+5pNNfuRLWt2vEhLkoIfycjnLBPkFAyrSLqV6WmtXe4Ctxuynn4CVKznJqJ6cPAmUlMhpQZSmW61Q9vWcHFlG0hetNvkFBXHy7QBgQEXapdWDU2Ym8Prr8vHq1XJIcnKyXE7ap9QqJiYC9eqpW5aqmjUDGjaUj1n7qS8VFcCOHfJxQYH2LrLYj8rvGFCRdmkxoMrMBEaMkP277OXkyOUMqrRPi/uVwmJhs58eZWbKiyqlOe3hh7V3kcUaKr/zKKDavXs3nn32WVx77bVo06YN4uPj0blzZ4wfPx6fffYZSktL/VVOMiOtnfgqKoAHHnA+LYiybOpU7V2ZkiOtdhxWMKDSF+Ui6/hxx+Vau8hiDZXfuRVQ/f777xg0aBC6du2KNWvW4Morr8TUqVPxr3/9C3fccQeEEHjyySeRkJCAmTNnMrAi31BOKMqBQG2rV1c/aNoTQl6hrl4duDKR57TacVjBE59+6Okii/uV34W4s9Ktt96KRx55BAsWLECjRo1crrd+/XrMnj0br732Gp544glflZHMqKzMVn2ulZoEd5N4Mtmntmmt5rMq1lDphycXWf37B6xYTtkHVEK4nnye6sytgGrfvn0IDQ2tdb1evXqhV69euHjxotcFI5M7ckRO61K/PhAbq3ZppPh4365H6tBqlnQFAyr90NNFVkqKDKIKC+Xk21o5rhqIW01+tQVT586d82h9olrZ1yJo5Uqqb185MsxVeSwWOS1N376BLRe5r7TUVqOg1RoqpSbh0CHnTUmkHXq6yOLk237n8Si/mTNn4osvvrA+HzlyJJo2bYoWLVpg69atPi0cmZgWm2WCg4HZs+XjqkGV8nzWLG3MN0jOHT0qgxQt1XxW1bKlzBtUXCxzZpF26e0iyz5YJ5/zOKB69913kZSUBABYtmwZli1bhiVLlmDYsGF45JFHfF5AMinlCkorHdIV6enAggUye7u9xES5PD1dnXKRe+w7pGul5rOqevXkSRjgiU/r7C+yqtLiRRY7pvuVxwHVyZMnrQHV999/j5EjR2Lw4MF49NFHsWHDBp8XkExKizVUivR04PBh4LPP5PPQUHmAYjClfVrer+yxH5V+pKcDn3xSfbkWL7KYi8qvPA6oGjdujGN/jb764YcfMGjQIACAEAIVWhgaSsag9RNfcDBw220ymLp4kU0zeqH1DukK1iToS6dO8r5hQ3mhlZUl9zUtBVMA9ys/c2uUn7309HSMGTMG7dq1Q15eHoYNGwYA2Lx5M9q2bevzApIJ2U9eq9WACgBCQmQ25P37gQMHbM00pF162K8A1lDpjRKgpKYCo0erW5aaMKDyK49rqN544w1MmTIFHTt2xLJlyxAZGQkAyM3NxX333efzApIJ5eUB58/LPgjKJMRapVxE8AClD1rPkq5gQKUvWu3zWZVSvpMngaIidctiQB7XUIWGhuLhhx+utvzBBx/0SYGIrAenFi2A8HB1y1Ib5QB14IC65SD3aD1LuoKjsfRFOWZpPVBv3Fjezp6V+5bSVEk+UafJkT/55BNcffXVSEhIwJEjRwAAs2bNwqJFi3xaODIpvTTLAKyh0pOzZwElZ57WAypl38/NBebNA1au1Mb0JeSc1qbJqgmb/fzG44DqnXfewUMPPYRhw4bh3Llz1o7ojRo1wqxZs3xdPjIjPQVUrKHSD6W5LzYWaNBA3bLUJivLNuz+rruAAQNk87dWJtolR3pp8gMYUPmRxwHVf//7X3zwwQd48sknEWyXW6NHjx7Yvn27TwtHJqWngMq+hopZrbWrogL47jv5uGlTbdf2ZGbKEaRV96ecHGDECAZVWlNeLqfKAvQVULE52ec8Dqiys7PRtWvXasvDwsJQxE5u5At6qj5XEkSePy/nxyLtycyUtTvPPSef796t3dqeigrggQecB+fKsqlTtR0Qms3RozKoCgsDEhLULk3tWEPlNx4HVCkpKdiyZUu15T/88AMuueQSX5SJzE4vHTwBzo+ldZmZslZHmb9PodXantWrq5fVnhDAsWNyPdIG+4EOQXXqlhxYTO7pNx6P8nvooYcwefJklJSUQAiB3377DfPnz8eMGTMwZ84cf5SRzEQPk9dW1batPMkdOAD06qV2aUhRW22PxSJre9LStDM1SG6ub9cj/9NT/ynAVs7Dh2XNWojHYQC54PF/cuLEiYiIiMBTTz2FCxcuYMyYMUhISMDs2bNx++23+6OMZCZHjsiTXYMGQEyM2qVxT5s2shMxr/i0xZPanv79A1asGsXH+3Y98j+9BVQtWsia9dJSuf9rfcSrjtSpfnLs2LHYv38/CgsLcfLkSRw/fhwTJkzwddnIjOw7pGt18tqqmDpBm/RY29O3r2xCdrXvWywyI3/fvoEtF7mmpy4KgGyWVIIoHrN8qk4BVXl5OZYvX45PPvkEERERAIATJ06gsLDQp4UjE9Lb1R7A1AlapcfanuBgYPZs+bhqUKU8nzVLO02UpK9BNAp2TPcLjwOqI0eOoFOnTkhLS8PkyZPxx18jm2bOnOk0gzqRR/SUMkHBGipt0mttT3o6sGCBbJqxl5gol2ttwl0zE0LfF4E8ZvmUxwHVAw88gB49euDs2bPW2ikAuOWWW/DTTz/5tHBkQnoMqJSD0x9/AAUF6paFbOxre6rSem1PerrsNHzllfL5Qw/JxKQMprTlzz9lyhRAX32RGFD5hccB1erVq/HUU0+hXr16DsuTk5ORk5Pjs4KRSekxoGrYUGbfBniA0hqltuevSdyt9FDbExwMXHGF7bEWAz+z09O8o/aY3NMvPA6oKisrrdPN2Dt+/DgaNmzok0KRSQmhz/4IAPtRaVl6OnDttfLx+PFyRKZeanvYnKxtej1e2eei4gwPPuNxQDV48GCHOfssFgsKCwvx7LPP4vrrr/dl2chs/vgDKCyUzTGtWqldGs/wxKdtyonv9ttligS91PYwUNc2PfafAhxnePjzT7VLYxgeB1T/+c9/sHbtWnTs2BElJSUYM2aMtblv5syZ/igjmYVy0ktMlHlS9IQnPu2y7zisBL56wbkitU2vAVV4uG3QAy8CfcbjxJ5JSUnYunUrvvjiC2zduhWFhYWYMGECxo4d69BJnchjeuw/pWAnT+3KzQWKi2WtlN5qPpOTZU1CURFw6hQQF6d2icieXgMqQJb5+HG5DVddpXZpDMGjgOrixYtITU3F999/j7Fjx2Ls2LH+KheZkZ4DKqUmgTVU2qN8J61aAaGh6pbFU2FhQMuWcgaBgwcZUGmNno9ZKSnAqlXAN9/I2qq+ffXTFK5RHjX5hYaGoqSkxF9lcWnlypWwWCxObxs2bHD5d5MmTUKbNm0QERGBmJgYpKWlYc+ePdbX8/LyMHToUCQkJCAsLAxJSUmYMmUKClwMfV+7di1CQkJw+eWX+3oTCdD/1R4gr/iKi9UtCzlSAio97lcAm5O1qrgYOHFCPtbbvpWZaZsYfMECYMAAWRuqtcnCdcbjPlSTJ0/GzJkzUV5e7o/yONW7d2/k5uY63CZOnIiUlBT06NHD5d91794dGRkZ2L17N5YuXQohBAYPHmwdpRgUFIS0tDR8++232LdvH+bNm4fly5fj3nvvrfZe586dw7hx4zBw4EC/baepVVQAv/8uH5eUyOd60qwZEBUlH2dnq1sWcqTX/lMKDnjQJqV2KjoaaNJE3bJ4IjMTGDGies68nBy5nEFVnXnch2rDhg346aef8OOPP6JTp05o0KCBw+uZfvgy6tWrhzi7qu6LFy9i0aJF+Oc//wlLDfO93XPPPdbHycnJePHFF9GlSxccPnwYbdq0QePGjfGPf/zDuk6rVq1w33334d///ne197r33nsxZswYBAcH45tvvvHNhpGUmQk88IBtItsXXwTmzZNJGfUwtB2Q/VzatAE2b5Ynvo4d1S4RKYxSQ8WASlvsa9T1Mu9oRYU81job4CCE3I6pU4G0NDb/1YHHNVSNGjXCrbfeiiFDhiAhIQHR0dEOt0D49ttvkZeXh7vuusvtvykqKkJGRgZSUlKQlJTkdJ0TJ04gMzMT/fr1c1iekZGBQ4cO4dlnn3Xrs0pLS1FQUOBwIxeUqyUlmFLo8WqJ/ai0ySg1VNyvtEWP/adWr65+rLUnBHDsmFyPPOZxDVVGRoY/yuGRuXPnYsiQIUhMTKx13bfffhuPPvooioqK0KFDByxbtqxalvfRo0dj0aJFKC4uxvDhwzFnzhzra/v378f06dOxevVqhIS49++aMWMGnn/+ec82yoyMdrXEmgTtEcIWiOg1oOJ+pU167POZm+vb9ciBxzVU1157Lc6dO1dteUFBAa5VshG7afr06S47mys3+07kgMzIvnTpUkyYMMGtzxg7diw2b96MVatWoX379hg5cmS1jvVvvPEGfv/9dyxatAgHDx7EQw89BACoqKjAmDFj8Pzzz6N9+/Zub9fjjz+O/Px86+3YsWNu/62pGO1qiTUJ2pOXB+Tny8d6qkmwp5Q7Lw9wcuwllegxoIqP9+165MAihGfZ4oKCgnDy5EnEKnOX/eX06dNo0aIFLl686PZ7/fHHH8jLy6txndatWzvUKP3rX//Cf//7X+Tk5CDUwyHQZWVlaNy4MebMmYPRo0c7XWfNmjXo27cvTpw4gYiICDRu3BjBdrUjlZWVEEIgODgYP/74o1tBZEFBAaKjo5Gfn48opeMyAfPnA2PG1L7eZ58BLr4vTVm5Uo6WadsW2L9f7dIQAPz6q8yx06JFzcG71sXFyTxUGzcC3burXRoCgA4dgH37gJ9+sk1tpHUVFXI0X06O85YBi0UmVs7O1kerQAB4cv52u8lv27Zt1se7du3CyZMnrc8rKirwww8/oIWSedVNMTExiImJcXt9IQQyMjIwbtw4j4Mp5e+FECgtLXW5TmVlJQDZD6p58+bYvn27w+tvv/02VqxYgQULFiBFT7OLa5HRrpaUGqrDh4HycsDNJmLyI713SFe0aSMDqgMHGFBpQUWF/J0D+qr5DA6Wg31GjJDBk31QpXSsnzWLwVQduX3Ev/zyy63NcM5qZSIiIvDf//7Xp4WrasWKFcjOzsbEiROrvZaTk4OBAwfi448/xhVXXIFDhw7hiy++wODBgxETE4Pjx4/jlVdeQUREhHXOwcWLF+PUqVPo2bMnIiMjsXPnTjzyyCPo06cPkpOTAQCXXXaZw+fExsYiPDy82nKqg7595dVQbVdLffsGvmx1kZAgEzGWlgJHj+rrQGtUeu+QrmjbFli3jv2otCInBygrk4liXQxy0qz0dJl7yn5kNSCPtbNm6WdktQa5HVBlZ2dDCIHWrVvjt99+c6hZqlevHmJjYx2axvxh7ty56N27N1JTU6u9dvHiRezduxcXLlwAAISHh2P16tWYNWsWzp49i+bNm+Oaa67BunXrrM2VERER+OCDD/Dggw+itLQUSUlJSE9Px/Tp0/26HfQX+6ulqvR4tRQUJIOo3bvliY8Blfr03iFdweSe2qIEtsnJ+jk+2UtPl4N93n8fuO8+mUeLzXxeczugatWqFS5evIjx48ejadOmaKXCnFifffaZy9eSk5Nh3x0sISEBixcvrvH9BgwYgHXr1nlUhueeew7PPfecR39DNUhPBz7/HBg1ynG5Xq+W2raVAdWBA8B116ldGtJjx2FnmNxTW4ywXwUHA7ffLgOqM2dkQuUqeSXJMx5PPfP111/7qyxkVl26yPvwcODTT4GsLHm1pLdgCuAQd61hDRX5gx5zUDnTuDHQtKl8zH3Lax6nTUhLS2OmcPItZURchw5y1F///vqtembqBO04fx44fVo+1nNNAmDbr06cAP7q1kAqMkINlULZtzgy2WseD0Nq164dXnjhBaxduxbdu3evNvXM/fff77PCkUkowUe7duqWwxdYQ6UdynfQrJmcb03PmjSR25CfL2tHOChGXUYKqNq1k+lFGFB5zeOAau7cuWjUqBE2bdqETZs2ObxmsVgYUJHnlB+yEQIq5Wpv3z6ZPyshQY5S1GuNm54ZpbkPkIM02rYFNm2SJ3MGVOoyWkAFsFbdBzwOqLKzs/1RDjIzIwVUv/8u78vKgLFj5ePERH1N9GwURjrpAXI7Nm3iiU9tZ8/aMtYbIRehctxlDZXXPO5DReRzRgmoMjPlqJmq9DjRsxEYqYYK4Eg/LaioAL78Uj5u3FgOpNE7BlQ+U6dUzsePH8e3336Lo0ePoqyszOG1119/3ScFI5NQkmAC+j7xGW2iZyMwSpZ0BUf6qSsz0zEZ5tmzMg+V3muflePuyZNyIEfDhuqWR8c8Dqh++ukn3HTTTWjdujX27NmDyy67DIcPH4YQAt26dfNHGcnIsrOBykogMhJo3lzt0tSdJxM99+8fsGKZmlGypCtYQ6WezExZy1z1gkmpfV6wQL9BVaNGcuDGn3/KYL1rV7VLpFseN/k9/vjjePjhh7F9+3aEh4dj4cKFOHbsGPr164fbbrvNH2UkI7Nv7lOyo+tRbq5v1yPvlJTYAlyjBFRKDdWRI4AHk9CTl2qrfQZk7XNFRUCL5VPsmO4THgdUu3fvxrhx4wAAISEhKC4uRmRkJF544QXMnDnT5wUkgzNK/ymjTfSsd9nZ8mTXsKG8+jaC+HggIkKeuI8cUbs05uFJ7bNesR+VT3gcUDVo0MDabyo+Ph4H7aqf//zzT9+VjMzBKAGVMtGzq1o2i0VOoqqXiZ71zr5Dup5rPu0pc0UCrEkIJDPUPjOg8gmPA6qrrroKa9asAQBcf/31mDZtGl566SXcfffduOqqq3xeQDI45Qes92YZZaJnoPoJXI8TPeud0TqkK9iPKvDMUPvMbOk+4XFA9frrr+PKK68EADz//PMYOHAgvvjiCyQnJ2Pu3Lk+LyAZnJGypKeny86pLVo4Lk9M1HenVT0yWod0BUf6BZ4Zap9ZQ+UTHo/ya203GWSDBg3w7rvv+rRAZCIlJbaUCUYIqAAZNKWlAVdfDfzyC/Dgg8C//82aqUAzWg4qBWuoAk+pfR4xovprRql9Vo6/p08DBQVAVJS65dGpOif2LCsrw/Hjx3H06FGHG5HbDh2SHTqjooCYGLVL4zvBwUCvXo7PKbCMliVdwRoqdSi1z1UDDaPUPkdFAbGx8jH3rTrzOKDat28f+vbti4iICLRq1QopKSlISUlBcnIyUoyQhp8CxygpE5xp317e79unbjnM6OJF4PBh+dioNVSHDsn8bRQ46enADTfIx6NGAVlZcjSp3oMpBZv9vOZxk99dd92FkJAQfP/994iPj4fFaCdCChyjdEh3pkMHec+AKvCOHgXKy4GwMDk5tZG0bAmEhMgZBnJyZN8dChyl9mbkSOMl6G3bFli7lgGVFzwOqLZs2YJNmzYhNTXVH+UhMzFSh/SqlBqqQ4dkjUloqLrlMRP75r4gg01XGhIipzs5cEBuJwOqwBHCdoGkXDAZCWuovObx0aZjx47MN0W+YZQcVM4kJAD168skjNnZapfGXIyaMkGhDAz69FNg5Up9Z+jWk9Ongfx82T3BiPsWs6V7za2AqqCgwHqbOXMmHn30UaxcuRJ5eXkOrxUUFPi7vGQkRg6oLBb2o1JDRYUMMgDZ5Ge0YCMzUzbLAMCcOcCAAbLGKjNT1WKZgvI7btUKCA9Xtyz+wBoqr7nV5NeoUSOHvlJCCAwcONBhHSEELBYLKox2ACP/KC6W0zUAxgyoABlQbdnCgCpQMjPlnGvKNCELFshgY/ZsY3QcNvIEvXpg5OY+wNaX9Y8/ZE1cdLS65dEhtwKqrKwsf5eDzEbp5xIdDTRtqm5Z/EWpodq7V91ymIHRg43aJui1WOQEvWlpTNPhL8rvWPldG03DhkDz5sCpU7KWqkcPtUukO24FVP369fN3Ochs7DukG3WkKJv8AsMMwYYnE/QabfSZVii/Y6MGVIA8Hp86JY/PDKg85lYfKk8Tdubk5NSpMGQiRu4/pWBAFRieBBt6ZYYJerXOLAEVwH5UdeRWQNWzZ09MmjQJGzZscLlOfn4+PvjgA1x22WVYuHChzwpIBmWGgErZthMngMJCdctiZGYINswwQa+WVVTYatWN2ocKYEDlJbea/Hbt2oWXXnoJ1113HcLDw9G9e3ckJCQgPDwcZ8+exa5du7Bz505069YNr776Kq6//np/l5v0zshJPRVNmgDNmgF//im3t2tXtUtkTGYINpQJenNynDdtWizydT1P0Ktlhw/LfHJhYcbO/aUcjxlQ1YlbNVRNmzbF66+/jtzcXPzf//0f2rVrhz///BP7//qnjx07Fps2bcL69esZTJF7zFBDBTBjeiAowYarvngWizwJ6jnYUCboBapvp1Em6NUy5ffbrp3xksXaYw2VVzzKlB4REYERI0ZghLNZt4ncdeGCvNIGjB9QtW8v8wYxoPIfJdhwdlwyUrChTNBrnxoCkMHkrFn6HsWodUZPmaBQaqjy8oCzZ4HGjdUtj84YONQmzVJSJjRubNyUCQp2TA8MJdioGjQlJuo/ZYK99HTZ/KTUVjVvbqwJerXK6CkTFJGRtqZxZkz3GAMqCjyzNPcBDKgCaehQW2b0d98FsrKMGWwEBwOjR8vHp07JiZLJv8wwwk/BZr86Y0BFgWeGDukK++SezjoTk+8o+1WTJsA998h8THpv5nOlWTO5nQCD9UAwS5MfwI7pXmBARYFnphqqNm1kP578fDmlA/nPnj3yPjXVuMliFRYLcMkl8rGy3eQfRUW2abJYQ0U18DigKioq8kc5yEzss6QbXUQE0LKlfMyaBP+yD6jMQNlOBlT+pRyvmjQxfp9PAGjdWt7/8oucaJzz87rN44CqefPmuPvuu7FmzRp/lIfMwEw1VAD7UQWKEliYoVkGYEAVKGbqP5WZCdx/v3x88CAwYICcYDwzU9Vi6YXHAdX/+3//D2fOnMG1116L9u3b45VXXsGJEyf8UTYymooKYMkSmTkcsF0JGR0DqsBgDRX5gzLCz+iBujLB+KlTjsuVCcYZVNXK44Dq5ptvxjfffIOcnBzce++9+Oyzz9CqVSvceOONyMzMRHl5uT/KSXqXmSmvdOwTv3btao4fKZN7+l9lpe3/a7aAau9euf3kH2aooaptgnFATjDO5r8a1blTekxMDB566CFs27YNr7/+OpYvX44RI0YgISEBzzzzDC5cuODLcpKeKVc+VSewNcuVD2uo/O/4cZkwNjQUSElRuzSBkZwM1KsHlJQAHk5gTx4wQ0BlhgnGA6DOAdWpU6fw6quvomPHjpg+fTpGjBiBn376Ca+99hoyMzNx8803+7CYpFu88rEdiA8cMPZ2qklp9mrbVgZVZhASYuuHyGY//xDCHE1+ZphgPAA8mnoGADIzM5GRkYGlS5eiY8eOuO+++3DHHXegUaNG1nV69+6NS5QhvWRunlz59O8fsGIFVMuWsiahtFRua3Ky2iUyHrP1n1KkpgI7d8rtHzpU7dIYz59/AufOycdGzptnhgnGA8DjGqq77roLCQkJWLt2LbZs2YIpU6Y4BFMAkJCQgCeffNJXZSQ945WPTC6pHIyVq13yLbON8FMoAeTu3eqWw6iU5r6WLWUKFKMywwTjAeBxDVVubi7q169f4zoRERF49tln61woMhBe+Ujt2wO7dskD9JAhapfGeJRA1Ww1VEzu6V9myZBuP8G4xeLYRcNIE4z7mcc1VOXl5SgoKKh2O3/+PMrKyvxRRtIzXvlI7JjuX2Zu8gMYUPmLWSZFBmwTjLdo4bi8eXNjTTDuRx4HVI0aNULjxo2r3Ro1aoSIiAi0atUKzz77LCo5jJcA25WPM2a68mFA5T8FBbbcZkavSahK2d7Tp4EzZ9QtixGZYYSfvfR04PBhObF4mzZy2axZDKbc5HFANW/ePCQkJOCJJ57AN998g2+++QZPPPEEWrRogXfeeQf33HMP3nzzTbzyyiv+KC/pkXLlU6WvHRITzXPlw4DKf5RahLi46vuY0UVGyt8RwP55/mC2gAqQF7f9+9taDbhfuc3jPlQfffQRXnvtNYwcOdK6bPjw4ejUqRPee+89/PTTT2jZsiVeeuklPPHEEz4tLOlYejqwdCnw/vtAWppMldC3r/FrphTKAfnIEZk3KDxc3fIYiVmb+xSpqXIk7Z49QK9eapfGOCoqbPP4ma3mEwA6dpT3u3apWw4d8biGat26dejatWu15V27dsX69esBAFdffTWOMtEcVaWc+G67TV4BmSWYAoDYWCA6Wnb2PHhQ7dIYi1lH+CnYj8o/jh6VqU7q1bNNcG4mSkDFEaRu8zigSkpKwty5c6stnzt3LpKSkgAAeXl5aNy4sfelI2NRrnTMmKPMYrElYZw7l7O4+5JZR/gpGFD5h9Lc17atuS7+FEpAtXcvwCnl3OJxk99//vMf3HbbbViyZAl69uwJANi4cSP27NmDBQsWAAA2bNiAUaNG+bakpG9//CGT5Fks5jzxZWbaAso33pC3xETZYd8Mfcj8iU1+8p4Ble9UVADffy8fN2kin5stqGrVSubeKi4GsrNtF4Tkksc1VDfddBP27t2L66+/HmfOnMGZM2cwbNgw7NmzBzfeeCMA4B//+Adef/11nxeWdEypNk5OBmrJY2Y4ylyGVee3NMtchv5UXg7s3y8fmz2gOngQYOoa7ykTuf/f/8nna9bI52b7nQYF2VoT2I/KLR7VUF28eBFDhw7Fu+++ixkzZvirTGREyg9SqUY2i9rmMrRYZAf9tDTzXQH7wuHDMogIDzdnPxcASEgAGjYEzp+XQZUZm9R9Rbn4qfp7VS5+zDIqWXHJJcDvv8vjd1qa2qXRPI9qqEJDQ7Ft2zZ/lYWMzKwBFWdx9y/7DulBdZ7rXd/sm9HZgbjuOJF7dRzp5xGPj0B33HGH007pRDUya4d0zmXoX0qHdLOO8FOwH5X3ePFTHQMqj3jcKb28vBwffvghli9fju7du6NBgwYOr7PvFDll1hoqzmXoX2bvkK5gQOU9XvxUpxyv9+wBKivNWwvsJo8Dqh07dqBbt24AgH1Vsj5bXM3XRuZ27pztIGS2GiplLsOcHOdNCRaLfN3ocxn6CwMqiQGV93jxU13r1jIP14ULMi9XcrLaJdI0jwOqrKwsf5SDjEzp15GYCERFqVuWQOMs7v7FgEqyD6iUwQ7kGV78VBcSIpvTt2+XrQwMqGpU5/q7AwcOYOnSpSguLgYACGc7IBFg3uY+hatZ3M00l6E//PmnvAHmmmvNmTZtZFB+/ry5mqR8iRO5O8d+VG7zOKDKy8vDwIED0b59e1x//fXI/evHO2HCBEybNs3nBSQDMGuHdHvKLO5z5sjnDRoAhw4xmPKG0iE9KUn+P80sLEw2zwBs9vOGcvFTNWgy88UPAyq3eRxQPfjggwgNDcXRo0dR3y5B46hRo/DDDz/4tHBkEGavoVIEBwN33CHvi4qAkyfVLpG+mX3KmarYj8o3Bg+2pUZ47z0gK0tmCjdjMAUwuacHPA6ofvzxR8ycOROJiYkOy9u1a4cjR474rGBkIEofKrMHVICsSVCap3bsULcsesf+U44YUPmGEjjExQH33GO+idyrsq+hYteeGnkcUBUVFTnUTCnOnDmDsLAwnxSKDKSwEFACbTM3+dm77DJ5z4DKOwyoHCmBelYWJ9/2hvK7VH6nZteuna1/Xk6O2qXRNI8Dqr59++Ljjz+2PrdYLKisrMSrr76KAQMG+LRwZADKSS82FmjaVN2yaAUDKu9UVMiAYcMG+ZyTtsopU558Uj7esQMYMMCc88/5AgMqR/Xq2X5jzMRfI48DqldffRXvv/8+hg0bhrKyMjz66KO47LLL8PPPP2PmzJn+KCPpGftPVaccqHfuVLcceqRMXDtggK0P2vjx5g4clPnnTp92XM7Jt+uGAVV17JjuFo8Dqssuuwz79u3D1VdfjbS0NBQVFSE9PR2bN29GmzZt/FFG0jMGVNXZB1SVleqWRU+UwKHq9CAnT5o3cOD8c77HgKo6BlRu8TixJwBER0fjSaV6magm7JBeXZs2snN6cbEcPcQLkdrVFjhYLDJwSEszVwdiT+af698/YMXSrbw8Wx4vHrNsGFC5pU4B1blz5/Dbb7/h9OnTqKxyhT1u3DifFIwMgjVU1QUHy//H5s3yapgBVe0YODjH+ed8S2mGT04GGjZUtSiaogwo2rmTmfhr4HFA9d1332Hs2LEoLCxEVFSUw/x9FouFARXZFBfL5JUAR/hVdemltoAqLU3t0mgfAwfnOP+cb7G5z7kOHWQQdfas7KvXvLnaJdIkj/tQTZs2DXfffTcKCwtx7tw5nD171no7c+aMP8pIerVvn+wj1Lgxf4BVcaSfZxg4OKfMP+eqxsBikZnkzTT/nDcYUDkXEWHLxM9mP5c8DqhycnJw//33O81FReTAvv8Uq4gdMaDyDAMH5+znn6v6vzHz/HN1pfweL71U3XJokdJtg6kTXPI4oBoyZAg2btzoj7KQ0bD/lGtKQLVnD1BWpm5Z9IAT17rmavLtZs3MO/9cXQjBGqqasGN6rTzuQ3XDDTfgkUcewa5du9CpUyeEhoY6vH7TTTf5rHCkcwyoXGvZEoiMlJnk9+/nFbE7lMBh4kTZl0ORmCiDKTMHDunpsi/e6tXAI48AGzcC06eb+3/iqdxcuV8FBTH7vjMMqGrlcUD197//HQDwwgsvVHvNYrGggvlOSKH88NghvTqLRV4F//KLHDnDgMo96ekyS/p//wsMHQo89phs5jNjzVRVwcFyhOMNN8iAis3JnlH+X+3aAeHh6pZFixhQ1crjJr/KykqXN38FUytXroTFYnF626BMP+HEpEmT0KZNG0RERCAmJgZpaWnYYzdxaF5eHoYOHYqEhASEhYUhKSkJU6ZMQUFBQa2ffVLJ0kzOXbwoa14A1lC5wn5UdbN9u7wfNYoT1zrTubO837pV3XLoDZv7aqbU2p06Bbz/PueLdMLjgEoNvXv3Rm5ursNt4sSJSElJQY8ePVz+Xffu3ZGRkYHdu3dj6dKlEEJg8ODB1sAvKCgIaWlp+Pbbb7Fv3z7MmzcPy5cvx7333lvtvfbu3evw+bGxsX7bXkM4cAAoL5fNWomJapdGmxhQeU4IW6DQpYu6ZdEq5f+yc6f8DZJ7GFDV7McfbRcvkyZxvkhnhJuGDRsmzp07Z30+Y8YMcfbsWevzP//8U1xyySXuvp1XysrKRExMjHjhhRc8+rutW7cKAOLAgQMu15k9e7ZITEy0Ps/KyhIAHLa1NiUlJSI/P996O3bsmAAg8vPzPSqvbpWXC/Hcc0IAQqSmyudU3fLl8n/Urp3aJdGPo0fl/ywkRIiSErVLo00VFUI0aCD/T7t2qV0a/ejZU/7PvvpK7ZJoz8KFQlgs8v9jf7NY5G3hQrVL6Df5+flun7/drqFaunQpSktLrc9ffvllh7xT5eXl2Lt3r+8ivRp8++23yMvLw1133eX23xQVFSEjIwMpKSlISkpyus6JEyeQmZmJfv36VXvt8ssvR3x8PK677jqsXbu2xs+aMWMGoqOjrTdXn2dIyuS1zz0nn+/Zw6sYV5R+UwcOyCSoVDuldio1VU7fQ9UFBQGdOsnH27apWxa9qKy09Q1iDZUjzhfpNrcDKlHln1n1eSDNnTsXQ4YMQaIbTUlvv/02IiMjERkZiSVLlmDZsmWoV6+ewzqjR49G/fr10aJFC0RFRWHOnDnW1+Lj4/Huu+9i4cKFWLhwIZKSktC/f3/8/vvvLj/z8ccfR35+vvV27Nixum+snriavJaz3jvXvDnQtKk8KDG3i3vY3Oce9qPyzJEjQFERUK8e0Lat2qXRFk+mfTI5VftQTZ8+3WVnc+Vm34kcAI4fP46lS5diwoQJbn3G2LFjsXnzZqxatQrt27fHyJEjUVJS4rDOG2+8gd9//x2LFi3CwYMH8dBDD1lf69ChAyZNmoTu3bujd+/e+PDDD9G7d2+88cYbLj8zLCwMUVFRDjfD41WM55SRfgD7UblryxZ5z4CqZsr/hzVU7lF+f5dcAoTUaYpb4+K0T25ze89RApyqy7wxbdo03HnnnTWu01pJd/+XjIwMNG3a1O18V0qzW7t27XDVVVehcePG+PrrrzF69GjrOnFxcYiLi0NqaiqaNGmCvn374umnn0a8i2ksrrjiCqxZs8atzzcNTl5bN5ddBqxaxYDKXayhco9SQ8WAyj3skO4ap31ym9sBlRACd955J8L+6rdQUlKCe++9Fw0aNAAAh/5V7oqJiUFMTIzb6wshkJGRgXHjxlVLKOru3wshaixrZWUlgJq3Z8uWLS6DLdPiVUzdsIbKfUVFsr8ZwICqNkofqmPHZLLKxo3VLY/WMaByTZn2KSfHeQuExSJfN9u0T064HVCNHz/e4fkdd9xRbZ1x48Z5X6IarFixAtnZ2Zg4cWK113JycjBw4EB8/PHHuOKKK3Do0CF88cUXGDx4MGJiYnD8+HG88soriIiIwPXXXw8AWLx4MU6dOoWePXsiMjISO3fuxCOPPII+ffogOTkZADBr1iykpKTg0ksvRUlJCebMmYMVK1bgxx9/9Ou26g6vYuqGAZX7tm+XB/S4OE62XZvoaKBVK9k3aNs2wMlAG7LDgMo1ZdqnESNk8GQfVJl92qcq3A6oMjIy/FkOt8ydOxe9e/dGqpNpAS5evIi9e/fiwoULAIDw8HCsXr0as2bNwtmzZ9G8eXNcc801WLdunTWHVEREBD744AM8+OCDKC0tRVJSEtLT0zF9+nTr+5aVlWHatGnIyclB/fr10blzZyxfvhwDBgwIzEbrBa9i6kYZ6XfsGFBQAJihv11dsbnPM507M6Byx8WLcjQywIDKFWXapwcecOzawWmfHFiEmsP1TKKgoADR0dHIz883dgd1ZZRf1V1KuYrhRK3OKYHounVAr15ql0a77rsPeOcd4NFHgZkz1S6N9j39NPDii3Luww8+ULs02rV7t5zNITISyM+XaSfIuYoKGUA9/DCQkAAcPWr4milPzt/cc8h30tOBL7+0BVCKxEQGUzVRaqnY7Fcz1lB5hh3T3aP87i69lMFUbYKDAWUg2YkTcnJ3suLeQ77VubOsoQoNBT75BMjKArKzGUzVRJnrcOFCzo/lSmWlLTBgQOUeJaDavp37VE3Yf8ozTZsCSrJq5jlzwICKfEtJeNqtG3DHHZy8tjaZmcBHH8nHS5dyfixXsrPl1XBYGNChg9ql0Ye2bYGICJmF/+BBtUujTRUV8qIPAMLDGXi6q2tXeb95s7rl0BgGVORbyg+sWzd1y6EHSp+zs2cdlzOzfHXKlfCllzLxoruCg221Lmz2q06ZJkvJ8P3WW7yYcRcDKqcYUJFv2ddQkWvMLO8Z9p+qG/ajco7TZHmHAZVTDKjId4RgQOUuzo/lGQZUdcM5/arjxYz3lIBq1y6gDkm9jYoBFfnO0aPAmTOySUYZuUbOMbO8ZziHX92whqo6Xsx4LykJaNIEKC8Hdu5UuzSawYCKfEepnbrsMtl5mFxjZnn3nTsnE1QCDKg8pQRUhw/LHEvEixlfsFiAyy+Xj9nsZ8WAinyHzX3uUzLLu5pg3GKRV4HMLG+rXUlK4px0nmrSRO5nAPOcKXgx4xvsR1UNAyryHQZU7lPmxwKqB1WcH8uR0v9HuSImz7AflSNezPgGA6pqGFCR7zBlgmeU+bFatHBcHhPDzPL22CHdO+xH5cj+YqYqXsy4Twmotm5lB/6/MKAi38jNlbegINsBnGqXni77t2RlAd27y2WPPspgyh4DKu8woKouPR345z+rL+c0We7r0EEmji0qYuLYvzCgIt9Qaqc6dAAaNFC3LHoTHCwzyqelyeesQrcpL7f1/WFAVTfK/23zZuDTTzm9UVW33gp89hmnyfJUcLAtWOcxCwADKvIV9p/ynlJDtWmTuuXQiooKGQCUlMhpQZKT1S6RPinD2ktK5HRQnN5I2rhR3t98MzB6NKfJqguO9HPAgIp8gwGV95SAau9e4Px5dcuiNmVaEGVm+5ISoHVrBgGeyswERo2qvtzsGcHLy21BQM+e6pZFz9gx3QEDKvINBlTea95cdlAXwpbI0ow4LYhvMCO4a7t3y0mjGzYE2rVTuzT6ZR9QOdvPTIYBFXkvL8+WeJFD271j9mY/BgG+w4zgrm3YIO+7d5cDaahuOnWSzaR//AGcOKF2aVTHPYm8p9SmtGkDNGqkZkn0z+wBFYMA32FGcNeU/lM9eqhbDr2LiABSU+VjM9eq/4UBFXmPzX2+Y/aAikGA7zAjuGtKQMX+U95jx3QrBlTkPSWgUtrTqe6UgGrPHqCwUN2yqIFBgO8wI7hzZWW23GasofIeO6ZbMaAi77GGynfi4oCEBPN2TGcQ4Duc3si57dtlUNW4MZCSonZp9I8BlRUDKvJOQQGwb598zBoq3zBzsx+nBfEtV9MbmTkjuH3/KVeBO7lPafLLzgbOnVOzJKpjQEXeUarOExOB2Fh1y2IUZg6oAHmS//zz6svNHAR4Q5ne6MMP5fOwMGD/fvP+H9l/yreaNAFatpSPX3vN1Jn4GVBR3VVUAF9+KR+3bGnaH5HPmT2gAoC2beV9gwbA//t/nBbEW8HBwLhxMu9Saanso2dWHOHnW5mZwOnT8vGLL5o6Ez8DKqobJZP1//2ffL5unWl/RD5n3zG9qEjdsqjl11/lfZ8+wNixnBbEF4KDgSuukI9/+UXdsqiluFj2oQIYUPmCkoS3pMRxuUmT8DKgIs8xk7V/xcfLW2WlOTumA7YT/pVXqlsOo1H+n0rAajZbt8qa9NhY2YRMdcckvNUwoCLP8EcUGGZv9lNO+FddpW45jMbsAZV9/yl2SPcOk/BWw4CKPMMfUWCYOaA6e1ZOEA3YmqjIN5SAavduID9f3bKogf2nfIdJeKthQEWe4Y8oMJSASsnxZSa//Sbv27YFmjVTtyxG07y57OsohG0+OzNRtpkBlfeYhLcaBlTkGf6IAkMJqHbtAi5cULcsgcb+U/5l1ma/wkJZMwfYfl9Ud0zCWw0DKvIMf0SBkZAgs6ZXVtpyfZkF+0/5l/J/NVtAtXmzrJlr0YIXfL7ATPzVMKAizzCTdeCYsR+VELYTPWuo/EP5v/7yi/PBJUbFhJ6+5yoTf1ycKZPwMqAiz6WnA/ffX305M1n7lhkDqgMHgDNnZDbvLl3ULo0xde0KhIYCf/whM6ibBftP+YeSiT8rC2jfXi77179MeR5gQEV1U1oq72+7DfjsM2ay9gcloFq1Cpg/3xxTOij9p7p1A+rVU7csRhUebpt/zUzNfhzh5z/BwTL57i23yOfr16taHLUwoKK6WbtW3t9+OzB6NDNZ+8OpU/I+OxsYM8YcUzqw/1Rg2Df7GV1FBfD993L+QsAWTJLv9ekj75Xzg8kwoCLPnT0L7NghHys/IPKtzExg0qTqy42ejZ4j/ALDLCP9lCmyhg+3LevRw7i/H7X17i3v9+wB/vxT3bKogAEVeW79etmZtV07mdeGfMus2eiLi20jGllD5V9KQLV5s6353mg4RVbgNW0KXHKJfLxunbplUQEDKvLcmjXynrVT/mHWbPS//w6Ul8sgvWVLtUtjbG3bAk2ayGDKiGk5zHpRogUmbvZjQEWeU34oV1+tbjmMyqzZ6O37T3GeNf+yWIzd7GfWixItYEBF5KbSUtvUIAyo/MOs2ejZfyqwjBxQmfWiRAuUgGrDBqCkRN2yBBgDKvLM77/LH0mzZracI+RbZs1Gz4SegWXkjOlmvSjRgrZtgdhYoKzMXDn0wICKPKVU4/bpw2YZfzHjlA65ucDRo3L7mMk6MK64Qt4fOAC8/76x8pyZ9aJECywW0zb7MaAizygd0tnc51+upnQwYjb6igpg7lz5uFUroH59dctjFllZQEiIfDxpkrHynHGKLHUxoCKqhRAMqAJJmdJh8WLbSWDlSmMFU0qeoKefls8PHzbOSV3LlJQC5eWOy42UUiA9HXj99erLjXhRojXK+WHtWlPNF8mAity3dy+QlyenrujWTe3SmENwMDBsmK1fkZFGJTFPkDrMlFJAqYHr1o1TZAVS167yPJGXJ88bJsGAitynVN9ecQXnWQu0AQPk/cqVqhbDZ8x0UtcaM6UUWLFC3o8YwSmyAqlePVsfPRM1+zGgIvexuU89/fvL+6wsVYvhM2Y6qWuNWVIKVFTYLkCuvVbVopiScp5QzhsmwICK3McM6erp3Vs2Xxw5IvsZ6Z1ZTupaZJaUAlu3ynlHGzYEundXuzTmY8KO6QyoyD2nTsnh1RYL0KuX2qUxn8hIWxW6EZr9zHJS1yKzpBRQanOvucbWl4oCRzlP7N8PnD6tblkChAEVuUe5yrjsMqBxY3XLYlZGavYzy0ldi8yS50zpP8XmPnU0bgxceql8bJKJkhlQkXvYf0p9SkC1cqX+hyIrJ3Vn22Gkk7pWucpzFhNjjJQCFy8CP/8sHzOgUk/v3vL+gw+MlTjWBQZUVDOlY+e338rnbO5TT+/eQGiozChuhH5U6enALbdUX848QYGh5DnLyrL9rv/+d2P83zduBAoLgSZNgM6d1S6NOWVmAgsXyseLFxsrcawLDKjINSXp4oABwMGDctljjxn6B6FpDRrY+lEZodkPkJ3sAeCJJ5gnSA3BwbLmc+JE+fynn1Qtjs8ov4/+/YEgnuYCTskxd+aM43KD55jjnkbOuUq6ePKkoX8Qmmff7Kd3f/whJ9sGgH/+k3mC1DRokLz/7TcgP1/dsvgC+0+px8Q55hhQUXUm/kFonn2CT733o1q+XN536QLExalbFrNr2RJo3x6orNR/sF5SYhtEw4Aq8EycY44BFVVn4h+E5vXqJftRHTsGHDqkdmm88+OP8n7wYHXLQZJSS6UEunr1yy8yqIqLA1JT1S6N+Zg4xxwDKqrOxD8Izatf3zavn55rEoRgQKU1RgmolP5TAwa4TstB/mPiHHMMqKg6E/8gdMEI/ah27QJOnJATqDIVhzYMGCA7cO/ZU3MNtdax/5S6TJxjjgEVVWfiH4QuKP2ofvhBjozTY34XpXaqXz8ZVJH6GjUCevaUj/VYS1VRASxZYksi2a+fuuUxq5oSxyoMmmOOARVVZ/+DqIpJF9V36pS8//NPYOxYfeZ3YXOfNum12U9J8XL99bJjPSBrqPT0mzASV4ljLRbgyy8NmxaFARU5l54OPP989eVMuqiuzEwZRFWlp/wuJSXAqlXyMQMqbbEPqPQyitRVihc9/SaMyD5x7CefyDx6QgAJCWqXzG8YUJFrSj6a665j0kUtMEo6i7VrgeJi2QdPmeuLtKFXLyAiQtaC7typdmlqZ5TfhFEpiWPvuAO48Ua57H//U7VI/sSAilz77jt5//e/M+miFhglnYV9cx9HYWlLWBhwzTXy8bJl6pbFHUb5TZjBDTfI+8WL1S2HHzGgIuf27ZO30FBgyBC1S0OAcdJZsP+Utl13nbzXQz8qo/wmzGDoUHkBtWWLbI41IAZU5JxSO9WvHxAVpW5ZSDJCOotTp+QBFbD11yFtUb6XVauAsjJ1y1IbI/wmzCImxpZDz6C1VAyoyDkloBo+XN1ykI2e01lUVMj0Di+8IJ9ffjkQG6tmiciVTp2AZs2AoiLgpZe0nZZDz78JM7r+enlv0H5UDKiourNngTVr5GMGVNqh1/wuypD2AQOAt9+Wyw4e5OgrrfrmGxlMATIA1nJaDuU34axTOlO8aI/Sj2r5cqC0VN2y+AEDKqpuyRJ5RXrppUBKitqlIXuu8rs0bKjNdBauhrSfP88h7VqkfF/FxY7LtZyCID3d1pRkjyletKdrV9n8WlQE/Pyz2qXxOQZUVN2338p71k5pk31+l3/+Uy6LjQVuuUXVYlVT05B2BYe0a4deUxAUFgJbt8rHH3zAFC9aZrEAw4bJxwZs9mNARY4uXpRTmgAMqLRMye/y8sty6paDB20nFa3gkHZ90ev39f33Mlls27bAhAlM8aJ1SrMfAyp1rFy5EhaLxeltw4YNLv9u0qRJaNOmDSIiIhATE4O0tDTs2bPH+npeXh6GDh2KhIQEhIWFISkpCVOmTEFBQYHD+5SWluLJJ59Eq1atEBYWhuTkZHz44Yd+215VrV4tE3raj8gg7YqMtF3xLVigblmq4pB2fdHr9/XVV/L+ttuY10wPBg2S6XgOHAD271e7ND6li4Cqd+/eyM3NdbhNnDgRKSkp6NGjh8u/6969OzIyMrB7924sXboUQggMHjwYFX9VWQcFBSEtLQ3ffvst9u3bh3nz5mH58uW49957Hd5n5MiR+OmnnzB37lzs3bsX8+fPR4cOHfy6zapRRvfdcAOv8PTittvk/VdfaWu6EA5p1xc9fl+FhbYh+CNHqlsWck9UlG3U5ezZwPz52h5J6gmhQ2VlZSImJka88MILHv3d1q1bBQBx4MABl+vMnj1bJCYmWp8vWbJEREdHi7y8vDqXNz8/XwAQ+fn5dX4PvysvF2LFCiFiY4UAhPjyS7VLRO4qKBAiLEx+b9u2qV0am/JyIRIThbBYZNmq3iwWIZKS5HqkPj1+X/Pny7K1bStEZaXapSF3jR9fff9KTBRi4UK1S1aNJ+dvXdRQVfXtt98iLy8Pd911l9t/U1RUhIyMDKSkpCApKcnpOidOnEBmZib69evn8Fk9evTAq6++ihYtWqB9+/Z4+OGHUVx1FIyd0tJSFBQUONw0TRnWfu21wOnTctmDD2pzRA9V17ChzEIM2Jo/tMA+zUNVHNKuPXpMy8HmPv3JzAQ+/rj6ci2PJHVXAAI8nxs2bJgYNmyYW+u+9dZbokGDBgKA6NChg9Paqdtvv11EREQIAGL48OGiuLjY+tqQIUNEWFiYuOGGG8Svv/4q/ve//4lWrVqJO++80+VnPvvsswJAtZsma6gWLnR+RWqxyJsGrxjIiU8+kd9baqr2rtSfe676/pWUxH1LqxYulLUFVb+zd95Ru2SOzp8XIjxclm3zZrVLQ+5QakGd1YBqtBbUkxoqVQOqxx57zGngYX/bvXu3w98cO3ZMBAUFiQULFrj1GefOnRP79u0Tq1atEsOHDxfdunVzCJiEECI3N1fs3r1bLFq0SHTs2FH84x//sL523XXXifDwcHHu3DnrsoULFwqLxSIuXLjg9DNLSkpEfn6+9Xbs2DFtBlQ63LnJhXPnhKhXT35vO3aoXRpHSvX+jTcK8dlnQmRlcZ/SuvJy+T199pkQ3brJ78/DLhZ+9/nnbO7Tm6ws1+cb+1tWltoltfIkoAoJdI2YvWnTpuHOO++scZ3WrVs7PM/IyEDTpk1x0003ufUZ0dHRiI6ORrt27XDVVVehcePG+PrrrzF69GjrOnFxcYiLi0NqaiqaNGmCvn374umnn0Z8fDzi4+PRokULREdHW9e/5JJLIITA8ePH0a5du2qfGRYWhrCwMLfKpypPhkn37x+wYlEdREfLSay/+042g1x6qdolkgoLbaMPp08H+vRRtzzkHiUtByBTqYwfL5tpnnpKO01rX34p79ncpx96HUnqJlUDqpiYGMTExLi9vhACGRkZGDduHEJDQz3+PCFr5FBaQ8r7yspKALCu06dPH3z11VcoLCxEZGQkAGDfvn0ICgpCYmKix2XQFIPv3KYzYoQMqD76COjQQY7G6ttX3T4vX38tsyK3bQv07q1eOaju0tOB++6Tw9x/+QXo1Uu9slRUyAu87GzbiGRllCtpnx5HknpAV53SV6xYgezsbEycOLHaazk5OUhNTcVvv/0GADh06BBmzJiBTZs24ejRo1i3bh1uu+02RERE4Pq/JmhcvHgxMjIysGPHDhw+fBj/+9//cO+996JPnz5ITk4GAIwZMwZNmzbFXXfdhV27duHnn3/GI488grvvvhsREREB23a/MPjObTrKVfrhw8CYMdqYg+2jj+T9uHGsRdCryEjg1lvlY+X7VIP9nJB33y1rzoKDgUOH1CsTecbok1n7ufnRp0aPHi169+7t9LXs7GwBQGT91faak5Mjhg0bJmJjY0VoaKhITEwUY8aMEXv27LH+zYoVK0SvXr1EdHS0CA8PF+3atROPPfaYOHv2rMN77969WwwaNEhERESIxMRE8dBDD7nsP+WMZtMmsA+VcWhxcMHRo7YyZWcH/vPJd5Yvl99jo0ZCVOmDGhCu9m9lH+cAB/1QvktXxyuNfZeenL8tQmgpE6AxFRQUIDo6Gvn5+YiKilK7OI4yM21Xn/aUKwhOLqp9FRXyyt1VfziLRV4VZmcHtvlvxgzgiSeAfv1k4j7Sr8pKuY8dOyb76I0YEbjP1ur+TXWXmSnnjbT/Ths0kP30NHa+8eT8rasmP/KDKp3+rThTu35ocQ42IWzNQ+PHB+5zyT+CgoA77pCPA93sp8X9m7xjP8H7s8/KZRUVsjlXxxhQmd1bb8n7226TOzdnatcfLQ0uqKiQtVEvvgjs3Ssnbg5kbQb5z9/+Ju8XL5aDDQI1ZYiW9m/yHWUk6bPPAp07ywmuMzLULpVXGFCZ2dmzwKefysf33y93bs7Urj9aGVxg32n4mWfksqAgYNky/34uBcYllwBt2sjmv/T0wA180Mr+Tf5hsQCTJ8vHb78t9y+dYkBlZvPmAcXF8uqA+YH0SwsjZzIzZU1U1aaZCxf0P50ESZmZwMGD1Zf7e8oQLezf5F9jx8pcegcP6voCjAGVWVVW2pr7Jk/mkHY9U3sOtooK2cG0pvEtU6caYzZ5s1K+Y2eU791f37GyfzvbvzgnpDE0aAAoSb6V85IOMaAyqx9/lFcD0dHy6oD0LT1dDiJo0aL6a7fd5t/+cOw0bHxqf8fp6UDPntWXc/CMcfzjH/L+++9lh3UdUjVTOqlAyTT8xBPy+bhx8uqA9C89HUhLk99vbq7sFP7887IKvaAA8FfKDnYaNj61v+Pt24ENG+Tjjz4CQkO1MRMA+U6HDsCgQcDy5fL8NHy47r5jBlRm4iz3x1dfyU7ovMIzBvs52CoqgM8/l4HVO+8Ajz3mn89kp2HjU/s7fvlleT9ypLwIJGPq1k0GVPPnyxsgayFnz9bFOYqJPQNAE4k9lU7DVb9uJvA0to8+kn0TYmNlNbo/pkti4kXjU77jnBzXfZn89R3v2wekpsrP3bIF6NLFt+9P2qDRcxQTe5KjmjoN+7tDKalrzBh5Ijx9Gnj/fZk3yNf5g4KDgf/8x/lr7DRsDIEe+KDkM5s/33bsGj6cwZRRGeQcxYDKDNTuUErqCQ21NfVNmybzBvkjf5CyfwVVOaSw07Bx1DTw4dFHffcd2+czGzMG+OEHubx3b9+8P2mPQc5RDKjMQO0OpaSuRo3kfdWrO1/lD/rjD+CFF+Tj999nxn0js58y5LPPZMADAAsXAmVl3r+/q3xmgOyozHxmxmSQcxQ7pZuB2h1KST0VFcAjjzh/TQjZfDN1qhwdWNfmmqeflqMIu3YF7rqrei0VGYv9wIcbbwR++gk4cAB4802gRw950qvL6Cx385l5s6+SNhnkHMUjnxn07Qs0bOj6dWYaNi5/VaUrfVxmzpS1UoDsY8NgylwaNrSNwHv0Ue+alA3S7EN1UFs2fEAX5yge/cxg/Xrg/Hnnr7HTsLH5oyrdvo/L9OnyRBcRIZv+yHyUkU9Va5Y8bVI2SLMP1YE7gx6efFLz5ygGVEZkP0Lmf/8D7rhDLh8wQF4F2GOnYWPzdVW6qz4uJSWcs8+MKiqABx90/pqno7MM0uxDdeRq0EO9evL+88+Bixf9M1LZR5iHKgACmofKWfJOAIiJkf0cGjSwZdLWWRZaqgNf5g9ivimqauVKeaFWm6wsW78rVyoq5P5z8qTz17l/mYMym4dyjoqPB7p3B4qK5FRp+fm2dQOQ9NOT8zc7pRuJq8RogGyOWb5c7ni1HdjIOJSq9BEj5Amp6r4hBPDKK+4F2Z70ceE+Zg7eNtPZnzzr1QPKy52vx64J5mE/6EFxxx3Ae+85BlOArVlZI60sDKj0zP5gFBtb8wgZX4zmIn1SqtKr1lwqAdbEiUBxsW25ctVnPy9gfLw8eLmDfVzMw93mt1OnZDONfcDuqja9USOgfn3gxAnbssREGUxp4KRJAVZRIbuuOKOMVH7gAVl7dfq0qi0vbPILAL80+bk6GNXGnap3MqaqVelZWbb8UfaUQKtpUyAvz7a8QQNZ7V4b7mPmUVuTMiBHflZW2p4nJgKjR8vs+q6aob/4QnZTYNcEcrdZ2Z4PmwI9OX8zoAoAnwdUNTXt1eazz+TBjMyttv5QdcE+LuakHI+Auh2TquJ+RPbmz7clkHWXD+f/41x+RuZO8ruacIQMAbX3h6pN1aHN7ONiXq5GZ9V1P2C+KbJXl3OWSvP/MaDSm7qeCJm8k+x528+pWTPH50y/YW5Vp6R54w3vT2Tsi0eAe0k/nVEhMGendL2py0GGtQdUlbc1lW+8IWsk2MeFFPajs+bP9/79WJtOQO0jlWsTwMCcNVR6U5eDDGsPqKq6XvUpWrSQJ8/Ro+U9gymy500wxNp0qspVs7I7AhiYs1N6APi0U7o7iRpbtADmzVN9CClpXF06E7PDMLnDndF/QPUaBx92JiYDqpoq6M47fZO0uAbslG5kNc15pDyfPRsYOJC1B1QzV1d9TZvKe3Y8p7qq7ThlsQCPPFJ932NtOtVEaVYePVqe42o7Fwb4WMWASo9cnQh5MCJPVe1MnJUlkzAuXMj9i7xT23Hq1Ver73vZ2dy/yH0aOxeyyS8A/DaXX9VEjWzaI1/i/kW+wP2I/M2P+xgTe2pMQCdHJiIiIp9gHyoiIiKiAGJARUREROQlBlREREREXmJARUREROQlBlREREREXmJARUREROQlBlREREREXmJARUREROQlBlREREREXgpRuwBmoCSjLygoULkkRERE5C7lvO3OpDIMqALg/PnzAICkpCSVS0JERESeOn/+PKKjo2tch3P5BUBlZSVOnDiBhg0bwmKxuFyvoKAASUlJOHbsmOnm/OO2m2/bzbrdALfdjNtu1u0G9L3tQgicP38eCQkJCAqquZcUa6gCICgoCImJiW6vHxUVpbudzle47ebbdrNuN8BtN+O2m3W7Af1ue201Uwp2SiciIiLyEgMqIiIiIi8xoNKQsLAwPPvsswgLC1O7KAHHbTfftpt1uwFuuxm33azbDZhn29kpnYiIiMhLrKEiIiIi8hIDKiIiIiIvMaAiIiIi8hIDKiIiIiIvMaBSwXPPPQeLxeJwS01Ntb5eUlKCyZMno2nTpoiMjMStt96KU6dOqVjiuvn5558xfPhwJCQkwGKx4JtvvnF4XQiBZ555BvHx8YiIiMCgQYOwf/9+h3XOnDmDsWPHIioqCo0aNcKECRNQWFgYwK2om9q2/c4776y2DwwdOtRhHT1u+4wZM9CzZ080bNgQsbGxuPnmm7F3716HddzZv48ePYobbrgB9evXR2xsLB555BGUl5cHclM85s629+/fv9r3fu+99zqso7dtf+edd9C5c2dr0sZevXphyZIl1teN+n0DtW+7Eb9vZ1555RVYLBZMnTrVuszI37srDKhUcumllyI3N9d6W7NmjfW1Bx98EN999x2++uorrFq1CidOnEB6erqKpa2boqIidOnSBW+99ZbT11999VW8+eabePfdd/Hrr7+iQYMGGDJkCEpKSqzrjB07Fjt37sSyZcvw/fff4+eff8Y999wTqE2os9q2HQCGDh3qsA/Mnz/f4XU9bvuqVaswefJk/PLLL1i2bBkuXryIwYMHo6ioyLpObft3RUUFbrjhBpSVlWHdunX46KOPMG/ePDzzzDNqbJLb3Nl2APj73//u8L2/+uqr1tf0uO2JiYl45ZVXsGnTJmzcuBHXXnst0tLSsHPnTgDG/b6B2rcdMN73XdWGDRvw3nvvoXPnzg7Ljfy9uyQo4J599lnRpUsXp6+dO3dOhIaGiq+++sq6bPfu3QKAWL9+fYBK6HsAxNdff219XllZKeLi4sS///1v67Jz586JsLAwMX/+fCGEELt27RIAxIYNG6zrLFmyRFgsFpGTkxOwsnur6rYLIcT48eNFWlqay78xyrafPn1aABCrVq0SQri3fy9evFgEBQWJkydPWtd55513RFRUlCgtLQ3sBnih6rYLIUS/fv3EAw884PJvjLLtjRs3FnPmzDHV961Qtl0I43/f58+fF+3atRPLli1z2FYzfu9CCMEaKpXs378fCQkJaN26NcaOHYujR48CADZt2oSLFy9i0KBB1nVTU1PRsmVLrF+/Xq3i+lx2djZOnjzpsJ3R0dG48sorrdu5fv16NGrUCD169LCuM2jQIAQFBeHXX38NeJl9beXKlYiNjUWHDh3wj3/8A3l5edbXjLLt+fn5AIAmTZoAcG//Xr9+PTp16oTmzZtb1xkyZAgKCgocrvy1ruq2Kz799FM0a9YMl112GR5//HFcuHDB+pret72iogKff/45ioqK0KtXL1N931W3XWHk73vy5Mm44YYbHL5fwFy/c3ucHFkFV155JebNm4cOHTogNzcXzz//PPr27YsdO3bg5MmTqFevHho1auTwN82bN8fJkyfVKbAfKNti/2NSniuvnTx5ErGxsQ6vh4SEoEmTJrr/XwwdOhTp6elISUnBwYMH8cQTT2DYsGFYv349goODDbHtlZWVmDp1Kvr06YPLLrsMANzav0+ePOl0v1Be0wNn2w4AY8aMQatWrZCQkIBt27bhsccew969e5GZmQlAv9u+fft29OrVCyUlJYiMjMTXX3+Njh07YsuWLYb/vl1tO2Dc7xsAPv/8c/z+++/YsGFDtdfM8juvigGVCoYNG2Z93LlzZ1x55ZVo1aoVvvzyS0RERKhYMgqU22+/3fq4U6dO6Ny5M9q0aYOVK1di4MCBKpbMdyZPnowdO3Y49A80C1fbbt8HrlOnToiPj8fAgQNx8OBBtGnTJtDF9JkOHTpgy5YtyM/Px4IFCzB+/HisWrVK7WIFhKtt79ixo2G/72PHjuGBBx7AsmXLEB4ernZxNINNfhrQqFEjtG/fHgcOHEBcXBzKyspw7tw5h3VOnTqFuLg4dQroB8q2VB31Yb+dcXFxOH36tMPr5eXlOHPmjKH+FwDQunVrNGvWDAcOHACg/22fMmUKvv/+e2RlZSExMdG63J39Oy4uzul+obymda623Zkrr7wSABy+dz1ue7169dC2bVt0794dM2bMQJcuXTB79mxTfN+utt0Zo3zfmzZtwunTp9GtWzeEhIQgJCQEq1atwptvvomQkBA0b97c8N+7MwyoNKCwsBAHDx5EfHw8unfvjtDQUPz000/W1/fu3YujR486tMvrXUpKCuLi4hy2s6CgAL/++qt1O3v16oVz585h06ZN1nVWrFiByspK64HJKI4fP468vDzEx8cD0O+2CyEwZcoUfP3111ixYgVSUlIcXndn/+7Vqxe2b9/uEFAuW7YMUVFR1qYULapt253ZsmULADh873rc9qoqKytRWlpq6O/bFWXbnTHK9z1w4EBs374dW7Zssd569OiBsWPHWh+b7XsHwFF+apg2bZpYuXKlyM7OFmvXrhWDBg0SzZo1E6dPnxZCCHHvvfeKli1bihUrVoiNGzeKXr16iV69eqlcas+dP39ebN68WWzevFkAEK+//rrYvHmzOHLkiBBCiFdeeUU0atRILFq0SGzbtk2kpaWJlJQUUVxcbH2PoUOHiq5du4pff/1VrFmzRrRr106MHj1arU1yW03bfv78efHwww+L9evXi+zsbLF8+XLRrVs30a5dO1FSUmJ9Dz1u+z/+8Q8RHR0tVq5cKXJzc623CxcuWNepbf8uLy8Xl112mRg8eLDYsmWL+OGHH0RMTIx4/PHH1dgkt9W27QcOHBAvvPCC2Lhxo8jOzhaLFi0SrVu3Ftdcc431PfS47dOnTxerVq0S2dnZYtu2bWL69OnCYrGIH3/8UQhh3O9biJq33ajftytVRzQa+Xt3hQGVCkaNGiXi4+NFvXr1RIsWLcSoUaPEgQMHrK8XFxeL++67TzRu3FjUr19f3HLLLSI3N1fFEtdNVlaWAFDtNn78eCGETJ3w9NNPi+bNm4uwsDAxcOBAsXfvXof3yMvLE6NHjxaRkZEiKipK3HXXXeL8+fMqbI1natr2CxcuiMGDB4uYmBgRGhoqWrVqJf7+9787DB8WQp/b7mybAYiMjAzrOu7s34cPHxbDhg0TERERolmzZmLatGni4sWLAd4az9S27UePHhXXXHONaNKkiQgLCxNt27YVjzzyiMjPz3d4H71t+9133y1atWol6tWrJ2JiYsTAgQOtwZQQxv2+hah52436fbtSNaAy8vfuikUIIQJXH0ZERERkPOxDRUREROQlBlREREREXmJARUREROQlBlREREREXmJARUREROQlBlREREREXmJARUREROQlBlREREREXmJARUQuWSwWfPPNNwCAw4cPw2KxWOcj86Xk5GTMmjXL6/e58847cfPNN3v9Pt7y5f/qb3/7G15++eUa1/HV/09tZWVlSE5OxsaNG9UuCpHHGFARmcydd94Ji8UCi8WC0NBQNG/eHNdddx0+/PBDVFZWOqybm5uLYcOGqVRS2rp1KxYvXoz7779f7aIERL169fDwww/jscceU7soRB5jQEVkQkOHDkVubi4OHz6MJUuWYMCAAXjggQdw4403ory83LpeXFwcwsLCVCypdPHiRbWLoIr//ve/uO222xAZGal2UVBWVhaQzxk7dizWrFmDnTt3BuTziHyFARWRCYWFhSEuLg4tWrRAt27d8MQTT2DRokVYsmQJ5s2bZ13PvslPcejQIQwYMAD169dHly5dsH79eofX16xZg759+yIiIgJJSUm4//77UVRUZH399OnTGD58OCIiIpCSkoJPP/20WvksFgveeecd3HTTTWjQoAFeeuklVFRUYMKECUhJSUFERAQ6dOiA2bNne7TdeXl5GD16NFq0aIH69eujU6dOmD9/vsM6/fv3x/33349HH30UTZo0QVxcHJ577jmHdfbs2YOrr74a4eHh6NixI5YvX+70f2Vvx44dGDZsGCIjI9G8eXP87W9/w59//uly/YqKCixYsADDhw93WO7O/+/cuXOYOHEiYmJiEBUVhWuvvRZbt251WOfFF19EbGwsGjZsiIkTJ2L69Om4/PLLra8rzacvvfQSEhIS0KFDBwDAsWPHMHLkSDRq1AhNmjRBWloaDh8+7PDec+bMwSWXXILw8HCkpqbi7bfftr5WVlaGKVOmID4+HuHh4WjVqhVmzJhhfb1x48bo06cPPv/8c5f/GyItYkBFRACAa6+9Fl26dEFmZmaN6z355JN4+OGHsWXLFrRv3x6jR4+21modPHgQQ4cOxa233opt27bhiy++wJo1azBlyhTr39955504duwYsrKysGDBArz99ts4ffp0tc957rnncMstt2D79u24++67UVlZicTERHz11VfYtWsXnnnmGTzxxBP48ssv3d7GkpISdO/eHf/73/+wY8cO3HPPPfjb3/6G3377zWG9jz76CA0aNMCvv/6KV199FS+88AKWLVsGQAY6N998M+rXr49ff/0V77//Pp588skaP/fcuXO49tpr0bVrV2zcuBE//PADTp06hZEjR7r8m23btiE/Px89evRwWO7O/++2227D6dOnsWTJEmzatAndunXDwIEDcebMGQDAp59+ipdeegkzZ87Epk2b0LJlS7zzzjvVyvDTTz9h7969WLZsGb7//ntcvHgRQ4YMQcOGDbF69WqsXbsWkZGRGDp0qLUG69NPP8UzzzyDl156Cbt378bLL7+Mp59+Gh999BEA4M0338S3336LL7/8Env37sWnn36K5ORkh8+94oorsHr16hr/p0SaI4jIVMaPHy/S0tKcvjZq1ChxySWXWJ8DEF9//bUQQojs7GwBQMyZM8f6+s6dOwUAsXv3biGEEBMmTBD33HOPw3uuXr1aBAUFieLiYrF3714BQPz222/W13fv3i0AiDfeeMPhc6dOnVrrtkyePFnceuutbm2bKzfccIOYNm2a9Xm/fv3E1Vdf7bBOz549xWOPPSaEEGLJkiUiJCRE5ObmWl9ftmyZ0//V5s2bhRBC/Otf/xKDBw92eM9jx44JAGLv3r1Oy/X111+L4OBgUVlZaV3mzv9v9erVIioqSpSUlDi8X5s2bcR7770nhBDiyiuvFJMnT3Z4vU+fPqJLly7W5+PHjxfNmzcXpaWl1mWffPKJ6NChg0OZSktLRUREhFi6dKn1cz777DOH9/7Xv/4levXqJYQQ4p///Ke49tprHd6jqtmzZ4vk5GSXrxNpUYhKcRwRaZAQAhaLpcZ1OnfubH0cHx8PQDZDpaamYuvWrdi2bZtDM5QQApWVlcjOzsa+ffsQEhKC7t27W19PTU1Fo0aNqn1O1ZoZAHjrrbfw4Ycf4ujRoyguLkZZWZlDM1VtKioq8PLLL+PLL79ETk4OysrKUFpaivr167vcRmU7lVqgvXv3IikpCXFxcdbXr7jiiho/d+vWrcjKynLaF+rgwYNo3759teXFxcUICwtz+D52795d6/9v69atKCwsRNOmTau938GDB63bcN999zm8fsUVV2DFihUOyzp16oR69eo5vPeBAwfQsGFDh/VKSkpw8OBBFBUV4eDBg5gwYQL+/ve/W18vLy9HdHQ0AFnDdt1116FDhw4YOnQobrzxRgwePNjh/SIiInDhwoVq/xMiLWNARURWu3fvRkpKSo3rhIaGWh8rJ3tldGBhYSEmTZrkdFRay5YtsW/fPrfL0qBBA4fnn3/+OR5++GG89tpr6NWrFxo2bIh///vf+PXXX91+z3//+9+YPXs2Zs2ahU6dOqFBgwaYOnVqtQ7X9tsIyO2sOgLSE4WFhRg+fDhmzpxZ7TUlKK2qWbNmuHDhAsrKyhyCGnc+Kz4+HitXrqz2mrPAtSZVv4PCwkJ0797dab+tmJgYFBYWAgA++OADXHnllQ6vBwcHAwC6deuG7OxsLFmyBMuXL8fIkSMxaNAgLFiwwLrumTNnEBMT41FZidTGgIqIAAArVqzA9u3b8eCDD9b5Pbp164Zdu3ahbdu2Tl9PTU1FeXk5Nm3ahJ49ewKQtSXnzp2r9b3Xrl2L3r17O9SsKDUu7lq7di3S0tJwxx13AJCB4L59+9CxY0e336NDhw44duwYTp06hebNmwMANmzYUOPfdOvWDQsXLkRycjJCQtw77Co1b7t27bI+duf/161bN5w8eRIhISHV+ibZb8OGDRswbtw467LatkF57y+++AKxsbGIioqq9np0dDQSEhJw6NAhjB071uX7REVFYdSoURg1ahRGjBiBoUOH4syZM2jSpAkA2YG/a9eutZaHSEvYKZ3IhEpLS3Hy5Enk5OTg999/x8svv4y0tDTceOONDidZTz322GNYt24dpkyZgi1btmD//v1YtGiRtVO60swzadIk/Prrr9i0aRMmTpyIiIiIWt+7Xbt22LhxI5YuXYp9+/bh6aefdisIqPoey5Ytw7p167B7925MmjQJp06d8ug9rrvuOrRp0wbjx4/Htm3bsHbtWjz11FMA4LK5dPLkyThz5gxGjx6NDRs24ODBg1i6dCnuuusuVFRUOP2bmJgYdOvWDWvWrLEuc+f/N2jQIPTq1Qs333wzfvzxRxw+fBjr1q3Dk08+aU2Y+c9//hNz587FRx99hP379+PFF1/Etm3bam3uHTt2LJo1a4a0tDSsXr0a2dnZWLlyJe6//34cP34cAPD8889jxowZePPNN7Fv3z5s374dGRkZeP311wEAr7/+OubPn489e/Zg3759+OqrrxAXF+dQe7Z69epqzYBEWseAisiEfvjhB8THxyM5ORlDhw5FVlYW3nzzTSxatMjaNFMXnTt3xqpVq7Bv3z707dsXXbt2xTPPPIOEhATrOhkZGUhISEC/fv2Qnp6Oe+65B7GxsbW+96RJk5Ceno5Ro0bhyiuvRF5eXrV+QLV56qmn0K1bNwwZMgT9+/dHXFycx5nVg4OD8c0336CwsBA9e/bExIkTraP8wsPDnf5NQkIC1q5di4qKCgwePBidOnXC1KlT0ahRIwQFuT4MT5w4sVrzWm3/P4vFgsWLF+Oaa67BXXfdhfbt2+P222/HkSNHrDVqY8eOxeOPP46HH37Y2gR35513uiy/on79+vj555/RsmVLpKen45JLLsGECRNQUlJirbGaOHEi5syZg4yMDHTq1An9+vXDvHnzrE3JDRs2xKuvvooePXqgZ8+eOHz4MBYvXmz9P6xfvx75+fkYMWJEjWUh0hqLEEKoXQgiIj1bu3Ytrr76ahw4cABt2rTx2fsWFxejQ4cO+OKLL9CrVy+fva8z1113HeLi4vDJJ5/49XNqM2rUKHTp0gVPPPGEquUg8hT7UBEReejrr79GZGQk2rVrhwMHDuCBBx5Anz59fBpMAXK028cff1xjAtC6uHDhAt59910MGTIEwcHBmD9/PpYvX27NtaWWsrIydOrUyat+fERqYQ0VEZGHPv74Y7z44os4evQomjVrhkGDBuG1116rlqpAq4qLizF8+HBs3rwZJSUl6NChA5566imkp6erXTQi3WJARUREROQldkonIiIi8hIDKiIiIiIvMaAiIiIi8hIDKiIiIiIvMaAiIiIi8hIDKiIiIiIvMaAiIiIi8hIDKiIiIiIv/X8cqd5hnLPdYQAAAABJRU5ErkJggg==\n"
          },
          "metadata": {}
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!pip install py3Dmol\n",
        "\n",
        "import py3Dmol"
      ],
      "metadata": {
        "id": "sgqaTpD2FD5e",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "5c947f91-6e1a-4157-b44b-13d88cded0ed"
      },
      "execution_count": 49,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Requirement already satisfied: py3Dmol in /usr/local/lib/python3.10/site-packages (2.4.2)\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "# Ler o arquivo XYZ\n",
        "with open('xtbscan.log', 'r') as file:\n",
        "    xyz_data = file.read()"
      ],
      "metadata": {
        "id": "4q3XDdq7FD8O"
      },
      "execution_count": 50,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "# Criar a visualiza√ß√£o\n",
        "view = py3Dmol.view(width=800, height=600)\n",
        "\n",
        "# Adicionar a trajet√≥ria do XYZ\n",
        "view.addModelsAsFrames(xyz_data, 'xyz')\n",
        "\n",
        "# Ajustar o estilo da visualiza√ß√£o\n",
        "view.setStyle({'stick': {'colorscheme':'greenCarbon'}})\n",
        "\n",
        "# Ajustar o zoom para que todas as mol√©culas estejam vis√≠veis\n",
        "view.zoomTo()\n",
        "\n",
        "# Anima√ß√£o\n",
        "view.animate({'loop': 'backAndForth'})  # Anima√ß√£o de ida e volta\n",
        "\n",
        "# Mostrar a visualiza√ß√£o\n",
        "view.show()"
      ],
      "metadata": {
        "id": "OkFp2-o6FD_V",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 617
        },
        "outputId": "b4ca8f16-80e5-44f0-8080-bbea2a7f756a"
      },
      "execution_count": 51,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_17338420579899318\"  style=\"position: relative; width: 800px; height: 600px;\">\n        <p id=\"3dmolwarning_17338420579899318\" style=\"background-color:#ffcccc;color:black\">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    //this is to ignore the existence of requirejs amd\n    var savedexports, savedmodule;\n    if (typeof exports !== 'undefined') savedexports = exports;\n    else exports = {}\n    if (typeof module !== 'undefined') savedmodule = module;\n    else module = {}\n\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n        exports = savedexports;\n        module = savedmodule;\n        resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.4.2/3Dmol-min.js');\n}\n\nvar viewer_17338420579899318 = null;\nvar warn = document.getElementById(\"3dmolwarning_17338420579899318\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_17338420579899318 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_17338420579899318\"),{backgroundColor:\"white\"});\nviewer_17338420579899318.zoomTo();\n\tviewer_17338420579899318.addModelsAsFrames(\"8\\n energy: -7.336369756748 xtb: 6.7.0 (08769fc)\\nC           -0.01557678914544        0.04676952851446       -0.01140071053055\\nH           -0.72244760195278        0.63613433703088        0.56910045483325\\nH            0.85821432826868       -0.14545593530701        0.60776190560494\\nH            0.29610899794758        0.63657615178766       -0.87079497028301\\nC           -0.64988992692328       -1.26128735487501       -0.46007407919487\\nH           -0.96201349834668       -1.85062335079973        0.39986038880813\\nH           -1.52456516250983       -1.06910210267221       -1.07796226527881\\nH            0.05674365266173       -1.85116627367905       -1.03983472395905\\n8\\n energy: -7.336333263138 xtb: 6.7.0 (08769fc)\\nC           -0.01420502669525        0.04602897026964       -0.01100112538491\\nH           -0.71206026681302        0.62228137727202        0.59299882590896\\nH            0.87584863742636       -0.14450594829268        0.58389835603764\\nH            0.26822007086304        0.65038669039304       -0.87009843537225\\nC           -0.65068618525168       -1.26056894182954       -0.46119692581622\\nH           -0.98801335059685       -1.83668873416564        0.39797254817177\\nH           -1.50789986206766       -1.07003604570283       -1.10251605060068\\nH            0.06536998313504       -1.86505236794402       -1.01340119294427\\n8\\n energy: -7.336221222362 xtb: 6.7.0 (08769fc)\\nC           -0.01287823396353        0.04562533422993       -0.01128083581802\\nH           -0.70109539441141        0.60825241166685        0.61650952715840\\nH            0.89258313469929       -0.14402479543607        0.55974765609212\\nH            0.23984097287035        0.66527139848417       -0.86872798786110\\nC           -0.65086595315077       -1.26016429185724       -0.46254094500444\\nH           -1.01383612668103       -1.82266398880034        0.39545678326156\\nH           -1.49069856441476       -1.07052223490840       -1.12633157611982\\nH            0.07352416505185       -1.87992883337890       -0.98617662170867\\n8\\n energy: -7.336034699107 xtb: 6.7.0 (08769fc)\\nC           -0.01158099530827        0.04536019400288       -0.01159521026920\\nH           -0.68900219986925        0.59387374351996        0.64027443654528\\nH            0.90917023721602       -0.14273302153357        0.53448781909530\\nH            0.21089393725028        0.68031708971906       -0.86618907552442\\nC           -0.65100099276165       -1.25989934515802       -0.46386910379030\\nH           -1.04026968329599       -1.80828296043551        0.39196142877435\\nH           -1.47239990712321       -1.07181822335785       -1.15038048963373\\nH            0.08076360389206       -1.89497247675695       -0.95803380519725\\n8\\n energy: -7.335775709481 xtb: 6.7.0 (08769fc)\\nC           -0.01053077065906        0.04537760386069       -0.01205866519827\\nH           -0.67566924258730        0.57884797803700        0.66448309677166\\nH            0.92552601611409       -0.14099579519476        0.50773731166080\\nH            0.18100963570414        0.69583499678498       -0.86257936448219\\nC           -0.65091355165413       -1.25991689105178       -0.46501367472884\\nH           -1.06753462883872       -1.79325555492792        0.38744472828155\\nH           -1.45261878126901       -1.07355984968312       -1.17470726161974\\nH            0.08730532318998       -1.91048748782509       -0.92865017068494\\n8\\n energy: -7.335441101708 xtb: 6.7.0 (08769fc)\\nC           -0.00937241465481        0.04541106498129       -0.01239239087613\\nH           -0.66055404924262        0.56315005108852        0.68961573795018\\nH            0.94221957727377       -0.13784597087253        0.47907019305826\\nH            0.14986982723605        0.71146677002330       -0.85741862870658\\nC           -0.65098441320486       -1.25995048458489       -0.46621705521190\\nH           -1.09626399454927       -1.77755597511662        0.38155487323527\\nH           -1.43114286107559       -1.07671432088318       -1.19999034195830\\nH            0.09280232821730       -1.92611613463588       -0.89756638749078\\n8\\n energy: -7.335040458077 xtb: 6.7.0 (08769fc)\\nC           -0.00851880128027        0.04565620854679       -0.01252391209690\\nH           -0.64316368878029        0.54646090630537        0.71625933004867\\nH            0.95919446545715       -0.13324103625777        0.44763818711320\\nH            0.11715223037754        0.72705710903738       -0.85072938580621\\nC           -0.65114453224770       -1.26019571303724       -0.46706574452056\\nH           -1.12717535566444       -1.76086518480514        0.37402237719268\\nH           -1.40715346039288       -1.08132424279983       -1.22645882272380\\nH            0.09738314253088       -1.94170304698956       -0.86448602920706\\n8\\n energy: -7.334562462917 xtb: 6.7.0 (08769fc)\\nC           -0.00781727328261        0.04610073418205       -0.01267816530394\\nH           -0.62283385110980        0.52813901016042        0.74493773613601\\nH            0.97663007332657       -0.12737088766018        0.41233941534389\\nH            0.08142888315901        0.74371312783050       -0.84201406573853\\nC           -0.65123261123394       -1.26064031300941       -0.46777856297248\\nH           -1.16098376702720       -1.74254164715733        0.36439510097464\\nH           -1.37967122245812       -1.08719986018805       -1.25464864561069\\nH            0.10105376862608       -1.95835516415801       -0.82789681282887\\n8\\n energy: -7.334019744680 xtb: 6.7.0 (08769fc)\\nC           -0.00749862357327        0.04679376294950       -0.01264928174356\\nH           -0.59844647620457        0.50759412968275        0.77648220836331\\nH            0.99459895376295       -0.11961557346494        0.37175114214356\\nH            0.04193360074309        0.76120357216994       -0.83075004700311\\nC           -0.65136596535909       -1.26133336530803       -0.46806934990503\\nH           -1.19884517850082       -1.72199512603425        0.35189534497271\\nH           -1.34737873458659       -1.09496129370053       -1.28510170529465\\nH            0.10357642371828       -1.97584110629445       -0.78690231153321\\n8\\n energy: -7.333435876664 xtb: 6.7.0 (08769fc)\\nC           -0.00769004481664        0.04773937148876       -0.01236478269665\\nH           -0.56870054237425        0.48404409846989        0.81145823204552\\nH            1.01287253688064       -0.10949338998687        0.32431523859777\\nH           -0.00241075709382        0.77966426578728       -0.81611045550843\\nC           -0.65157060722837       -1.26227892391296       -0.46779404011678\\nH           -1.24172580777970       -1.69844349665647        0.33548440118605\\nH           -1.30873067061508       -1.10509040773538       -1.31812102710533\\nH            0.10452989302721       -1.99429651745425       -0.74021156640212\\n8\\n energy: -7.332852479030 xtb: 6.7.0 (08769fc)\\nC           -0.00813838721312        0.04875627807554       -0.01184063040783\\nH           -0.53190823909880        0.45615542562854        0.85013537623302\\nH            1.03128850107034       -0.09656295905040        0.26824740139961\\nH           -0.05363192769682        0.79952461659033       -0.79674077438899\\nC           -0.65191572590409       -1.26329573422463       -0.46719668442538\\nH           -1.29044152673525       -1.67055337206680        0.31366026627262\\nH           -1.26199047374223       -1.11802877132756       -1.35414738972518\\nH            0.10331177931994       -2.01415048362500       -0.68546156495784\\n8\\n energy: -7.332394796759 xtb: 6.7.0 (08769fc)\\nC           -0.01016430387855        0.05007232725041       -0.01045376421163\\nH           -0.48400670559496        0.42210197638109        0.89468116631735\\nH            1.04832294801960       -0.07926328311433        0.19842829926269\\nH           -0.11364097191792        0.81990431070232       -0.77080229820286\\nC           -0.65254930416966       -1.26461145966297       -0.46482451915388\\nH           -1.34838842672778       -1.63649870012924        0.28331298125491\\nH           -1.20182361352644       -1.13533784669937       -1.39344755690915\\nH            0.09882437779570       -2.03452232472791       -0.62023830835740\\n8\\n energy: -7.332242402076 xtb: 6.7.0 (08769fc)\\nC           -0.01326624940186        0.05114847859722       -0.00843309324724\\nH           -0.42581146062707        0.38225729201472        0.94088958490621\\nH            1.06104299783815       -0.05835009520737        0.11755695779945\\nH           -0.18064619341843        0.83954856140785       -0.73758845366579\\nC           -0.65342237726085       -1.26568712707518       -0.46122674770444\\nH           -1.41132890239290       -1.59665343747387        0.24381165323519\\nH           -1.12979896219406       -1.15626135240565       -1.43235760287537\\nH            0.08980514745701       -2.05415731985772       -0.54599629844798\\n8\\n energy: -7.332501350740 xtb: 6.7.0 (08769fc)\\nC           -0.01647163325400        0.05152565551091       -0.00636005148375\\nH           -0.36469005597938        0.34147665723000        0.98101037331407\\nH            1.06683192263652       -0.03674250274558        0.03658621627732\\nH           -0.24638988473814        0.85586050164098       -0.70025373144132\\nC           -0.65431035061461       -1.26606380517078       -0.45751408373167\\nH           -1.46950274000948       -1.55587316293899        0.19952466109310\\nH           -1.05537364172485       -1.17787871075329       -1.46476473287109\\nH            0.07648038368392       -2.07045963277326       -0.47157265115665\\n8\\n energy: -7.333026747449 xtb: 6.7.0 (08769fc)\\nC           -0.01874615483048        0.05123776484562       -0.00490689266687\\nH           -0.31029517545785        0.30565634189836        1.01007344879352\\nH            1.06681829059529       -0.01752154314887       -0.03244637642874\\nH           -0.30218421657450        0.86755805352546       -0.66434588586339\\nC           -0.65492356282051       -1.26577556355917       -0.45488560759034\\nH           -1.51501086930052       -1.52005392772737        0.15790071224472\\nH           -0.99027422195541       -1.19710759477209       -1.48772687232201\\nH            0.06118991034396       -2.08214853106194       -0.40700652616687\\n8\\n energy: -7.333623545214 xtb: 6.7.0 (08769fc)\\nC           -0.02001278772154        0.05062699946026       -0.00417327657816\\nH           -0.26461862073085        0.27572987465734        1.02997777127694\\nH            1.06380714202408       -0.00093537457297       -0.08831018635732\\nH           -0.34747527179645        0.87565082668485       -0.63187327866278\\nC           -0.65519370610942       -1.26516461103392       -0.45344708650896\\nH           -1.54898108722818       -1.49012888581382        0.12145020766763\\nH           -0.93659479330733       -1.21369993257345       -1.50347953967611\\nH            0.04564312486967       -2.09023389680830       -0.35348861116121\\n8\\n energy: -7.334186889397 xtb: 6.7.0 (08769fc)\\nC           -0.02119367583224        0.04996642573935       -0.00362846569107\\nH           -0.22476089051792        0.24993217294216        1.04436599314128\\nH            1.05861207290786        0.01379931119936       -0.13567071435883\\nH           -0.38510678924468        0.88114889114231       -0.60238116958520\\nC           -0.65531434546662       -1.26450387948162       -0.45215226844027\\nH           -1.57581329064334       -1.46433276942045        0.08865120924895\\nH           -0.89020674117878       -1.22843963325917       -1.51434274244277\\nH            0.03035765997571       -2.09572551886195       -0.30818584187207\\n8\\n energy: -7.334710975474 xtb: 6.7.0 (08769fc)\\nC           -0.02117289405202        0.04924864668460       -0.00387234821826\\nH           -0.19023822525678        0.22756885555390        1.05428551826884\\nH            1.05354214099474        0.02761653238275       -0.17531429881178\\nH           -0.41761370210886        0.88547148324753       -0.57455891532054\\nC           -0.65509122682507       -1.26378612953863       -0.45225312347377\\nH           -1.59665612230143       -1.44197111844754        0.05939608547163\\nH           -0.85113720014029       -1.24226099370780       -1.52275550162376\\nH            0.01494122968970       -2.10004227617481       -0.26827141629235\\n8\\n energy: -7.335166146751 xtb: 6.7.0 (08769fc)\\nC           -0.02076754056987        0.04859438516728       -0.00445500291041\\nH           -0.15917434897782        0.20758291582140        1.06140315647945\\nH            1.04812689672338        0.04078096819333       -0.21001429546278\\nH           -0.44616724083645        0.88873046051629       -0.54816196107294\\nC           -0.65467666042907       -1.26313196820128       -0.45282937400767\\nH           -1.61370577837908       -1.42198688606897        0.03247021218341\\nH           -0.81661438030041       -1.25542897202379       -1.52919725877899\\nH           -0.00044694723069       -2.10329590340425       -0.23255947643006\\n8\\n energy: -7.335558009938 xtb: 6.7.0 (08769fc)\\nC           -0.01995176491192        0.04800890282854       -0.00549047698564\\nH           -0.13038187378031        0.18892612250453        1.06648117268194\\nH            1.04259932998067        0.05378860370778       -0.24149184368494\\nH           -0.47236077596430        0.89132769246456       -0.52222147083557\\nC           -0.65397170893376       -1.26254667072883       -0.45394335481114\\nH           -1.62807618637419       -1.40333184714889        0.00700749972351\\nH           -0.78509287115120       -1.26843977400215       -1.53446054186733\\nH           -0.01619014886499       -2.10588802962554       -0.19922498422080\\n8\\n energy: -7.335868616349 xtb: 6.7.0 (08769fc)\\nC           -0.01920967801147        0.04753820411001       -0.00647165227749\\nH           -0.10283494639333        0.17172422152849        1.07018536725782\\nH            1.03641541361278        0.06672283454529       -0.27070392904395\\nH           -0.49610894890859        0.89315752111253       -0.49689462154282\\nC           -0.65329344136317       -1.26207614493581       -0.45496976257083\\nH           -1.64073665216611       -1.38613175886695       -0.01773778726124\\nH           -0.75548918130837       -1.28137685598160       -1.53835085602392\\nH           -0.03216856546175       -2.10771302151195       -0.16840075853753\\n8\\n energy: -7.336104810094 xtb: 6.7.0 (08769fc)\\nC           -0.01823957914619        0.04716735283317       -0.00762836822741\\nH           -0.07627531732314        0.15587780269216        1.07269853616840\\nH            1.02987708379741        0.08001219690987       -0.29784577884115\\nH           -0.51787497939810        0.89446617297092       -0.47165713298137\\nC           -0.65252554291372       -1.26170550517053       -0.45626959872524\\nH           -1.65194549420610       -1.37028720861219       -0.04194834277209\\nH           -0.72771974923607       -1.29466880359763       -1.54121785554763\\nH           -0.04872242157411       -2.10901700802577       -0.13947545907348\\n8\\n energy: -7.336266660115 xtb: 6.7.0 (08769fc)\\nC           -0.01734082517331        0.04692194756198       -0.00888337447774\\nH           -0.05049660489833        0.14045843051368        1.07409955555828\\nH            1.02295232924798        0.09321972627620       -0.32383390804069\\nH           -0.53854585673662        0.89535622231991       -0.44646447854088\\nC           -0.65164122168484       -1.26146031696186       -0.45753485843610\\nH           -1.66184571048548       -1.35486976881754       -0.06579263359856\\nH           -0.70090962739121       -1.30787875459629       -1.54333646224621\\nH           -0.06559848287820       -2.10990248629608       -0.11159784021808\\n8\\n energy: -7.336353895727 xtb: 6.7.0 (08769fc)\\nC           -0.01638394037344        0.04678581256189       -0.01020831313264\\nH           -0.02504105748043        0.12575894769336        1.07460374265664\\nH            1.01558176346277        0.10676824046484       -0.34878633078275\\nH           -0.55804924631900        0.89581846169197       -0.42101775962022\\nC           -0.65071031327658       -1.26132441195392       -0.45887819883828\\nH           -1.67079272569037       -1.34017229758291       -0.08963060952942\\nH           -0.67492779861868       -1.32142953364444       -1.54468990714154\\nH           -0.08310268170429       -2.11036021923079       -0.08473662361176\\n8\\n energy: -7.336366557842 xtb: 6.7.0 (08769fc)\\nC           -0.01542037243824        0.04676513641146       -0.01157453261955\\nH            0.00040906359187        0.11161900802834        1.07428586957659\\nH            1.00763792903659        0.12072722511018       -0.37305760622520\\nH           -0.57659774510609        0.89588680435542       -0.39511802010736\\nC           -0.64974271177994       -1.26130397108765       -0.46024156770921\\nH           -1.67896279203634       -1.32603446106078       -0.11373702401522\\nH           -0.64939750240891       -1.33539065978685       -1.54527600689241\\nH           -0.10135186885896       -2.11042408197014       -0.05862511200762\\n8\\n energy: -7.336304648329 xtb: 6.7.0 (08769fc)\\nC           -0.01447719500117        0.04685908335004       -0.01294143711148\\nH            0.02618036706964        0.09793528800981        1.07319447471754\\nH            0.99898344183804        0.13520043218451       -0.39694497782977\\nH           -0.59434879167803        0.89556016631609       -0.36855381983072\\nC           -0.64876768663221       -1.26139815130458       -0.46158592389590\\nH           -1.68651038390160       -1.31235296158556       -0.13840371788451\\nH           -0.62399272422865       -1.34986590536318       -1.54506413428939\\nH           -0.12049302746604       -2.11009295160712       -0.03304446387575\\n8\\n energy: -7.336168166528 xtb: 6.7.0 (08769fc)\\nC           -0.01358083464819        0.04706702659569       -0.01426825476112\\nH            0.05261015616002        0.08460910802736        1.07133745053996\\nH            0.98944309591489        0.15030096348292       -0.42073063279920\\nH           -0.61143487051052        0.89482607110049       -0.34108937642822\\nC           -0.64781488888994       -1.26160631976924       -0.46287277655556\\nH           -1.69355520259703       -1.29902913812600       -0.16394614818631\\nH           -0.59838906038847       -1.36496839436119       -1.54398298641633\\nH           -0.14070439504078       -2.10935431695002       -0.00779127539321\\n8\\n energy: -7.335957146662 xtb: 6.7.0 (08769fc)\\nC           -0.01275557656222        0.04738693359813       -0.01550996946734\\nH            0.08007256432749        0.07154053271439        1.06868166332148\\nH            0.97879321477709        0.16616583728880       -0.44469839420390\\nH           -0.62797192345063        0.89365388862721       -0.31244551958846\\nC           -0.64691868507624       -1.26192643638403       -0.46406424162032\\nH           -1.70019052850552       -1.28596310010009       -0.19072815299015\\nH           -0.57224443078856       -1.38083515371477       -1.54191609225612\\nH           -0.16221063472142       -2.10817750202964        0.01733670680484\\n8\\n energy: -7.335671767187 xtb: 6.7.0 (08769fc)\\nC           -0.01202498335047        0.04781509952968       -0.01661504091908\\nH            0.10900711725684        0.05861782614338        1.06514372911917\\nH            0.96674018865599        0.18297014803870       -0.46915330009945\\nH           -0.64406640699680        0.89199050736480       -0.28227411607135\\nC           -0.64611983281183       -1.26235478822939       -0.46512094683544\\nH           -1.70648401475910       -1.27304315024042       -0.21919196492640\\nH           -0.54517347882084       -1.39764129303679       -1.53868809507282\\nH           -0.18530458917382       -2.10650934956997        0.04255573480540\\n8\\n energy: -7.335311661773 xtb: 6.7.0 (08769fc)\\nC           -0.01148445793366        0.04837257904736       -0.01740708186949\\nH            0.14013003407053        0.04539335128044        1.06045317341864\\nH            0.95279382216566        0.20098347801067       -0.49475035886877\\nH           -0.66006141579291        0.88979791170567       -0.24997638028418\\nC           -0.64555291009813       -1.26291240268613       -0.46589421548544\\nH           -1.71241896430269       -1.25982174338952       -0.25010304273243\\nH           -0.51639540573835       -1.41565642777450       -1.53405473424075\\nH           -0.21043670237047       -2.10431174619399        0.06838864006244\\n8\\n energy: -7.334887494143 xtb: 6.7.0 (08769fc)\\nC           -0.01118441938399        0.04897389089582       -0.01767708946461\\nH            0.17427981587139        0.03235716145875        1.05463451490832\\nH            0.93606878197551        0.22042677618578       -0.52155409307671\\nH           -0.67535868279646        0.88690304709131       -0.21536688139688\\nC           -0.64539821332717       -1.26351377000289       -0.46626694829353\\nH           -1.71829762759088       -1.24678899514120       -0.28424386062960\\nH           -0.48555470725749       -1.43510144469613       -1.52720260888725\\nH           -0.23798094749093       -2.10141166579144        0.09433296684028\\n8\\n energy: -7.334385847854 xtb: 6.7.0 (08769fc)\\nC           -0.01096667497308        0.04965986337507       -0.01771768314578\\nH            0.21245477567722        0.01864939172424        1.04693980528616\\nH            0.91608262257850        0.24222059496048       -0.55042287323835\\nH           -0.69083104721149        0.88304553663047       -0.17668730379741\\nC           -0.64543246538170       -1.26419976434362       -0.46648570915547\\nH           -1.72374691787052       -1.23308520961369       -0.32280485861881\\nH           -0.45168145035122       -1.45689695472578       -1.51796238231176\\nH           -0.26930484246772       -2.09754845800716        0.12179700498146\\n8\\n energy: -7.333824738817 xtb: 6.7.0 (08769fc)\\nC           -0.01096703830211        0.05036638023012       -0.01716933756201\\nH            0.25600239016365        0.00405878945123        1.03670109236294\\nH            0.89139112378561        0.26683985379394       -0.58195264404431\\nH           -0.70627353051852        0.87788768122009       -0.13305277619360\\nC           -0.64594948118281       -1.26490621844612       -0.46630277212290\\nH           -1.72858545463159       -1.21849932013239       -0.36727903123517\\nH           -0.41373300665697       -1.48151782734846       -1.50517048192242\\nH           -0.30531100265727       -2.09238433876842        0.15088195071750\\n8\\n energy: -7.333230164518 xtb: 6.7.0 (08769fc)\\nC           -0.01091615065108        0.05106338608157       -0.01648956542892\\nH            0.30500773290153       -0.01160393382783        1.02283645434531\\nH            0.86162456201361        0.29473352161910       -0.61593521201538\\nH           -0.72183964354696        0.87073769799048       -0.08325144352877\\nC           -0.64660748769888       -1.26560315050540       -0.46612441871665\\nH           -1.73182130498325       -1.20284216899583       -0.41810673627216\\nH           -0.37178267428319       -1.50941297891928       -1.48840892076650\\nH           -0.34709103375180       -2.08522737344283        0.18213584238310\\n8\\n energy: -7.332677405655 xtb: 6.7.0 (08769fc)\\nC           -0.01145787331507        0.05147210039053       -0.01413105068704\\nH            0.36480278163367       -0.02956221155767        1.00314856415136\\nH            0.82119000941587        0.32779755793481       -0.65498101489048\\nH           -0.73690064763569        0.86085096660710       -0.02421814818049\\nC           -0.64865124606427       -1.26601154990477       -0.46482851608451\\nH           -1.73315705952178       -1.18489100900019       -0.48104744015286\\nH           -0.32150743076161       -1.54247821309625       -1.46327232519865\\nH           -0.39774453375114       -2.07533264137354        0.21598593104270\\n8\\n energy: -7.332300218703 xtb: 6.7.0 (08769fc)\\nC           -0.01228264565481        0.05135535677590       -0.01070454350309\\nH            0.43469990320867       -0.05002256764442        0.97507020933162\\nH            0.76876734131160        0.36619608610059       -0.69770340971512\\nH           -0.75081222426332        0.84682940346311        0.04509296136176\\nC           -0.65160784661848       -1.26589434607951       -0.46291028588520\\nH           -1.72994299071973       -1.16443955363763       -0.55630722789987\\nH           -0.26377547111077       -1.58087738737662       -1.42805428596302\\nH           -0.45847206615318       -2.06130199160143        0.25217258227295\\n8\\n energy: -7.332285767003 xtb: 6.7.0 (08769fc)\\nC           -0.01320506133629        0.05050693077791       -0.00689158070561\\nH            0.50923635736352       -0.07164223073063        0.93798152282228\\nH            0.70636918401949        0.40724901484976       -0.73986946705769\\nH           -0.76167254547038        0.82825259247446        0.12078629923949\\nC           -0.65489630932428       -1.26504540772742       -0.46077143148187\\nH           -1.71977643297589       -1.14283020280374       -0.63894100223624\\nH           -0.20324804364294       -1.62193008919018       -1.38324407084236\\nH           -0.52623314863324       -2.04271560765014        0.28760573026205\\n8\\n energy: -7.332659680644 xtb: 6.7.0 (08769fc)\\nC           -0.01391928639816        0.04921156790947       -0.00404351417867\\nH            0.57617483040537       -0.09089337028538        0.89706012338432\\nH            0.64414929317404        0.44473505171348       -0.77468647522028\\nH           -0.76794403722000        0.80728337744050        0.19207300044740\\nC           -0.65734415136448       -1.26374965966951       -0.45915011431032\\nH           -1.70346694030231       -1.12358919999348       -0.71568545134144\\nH           -0.14970967613416       -1.65941506830087       -1.33615626207740\\nH           -0.59136603216031       -2.02173769881421        0.31724469329641\\n8\\n energy: -7.333225182290 xtb: 6.7.0 (08769fc)\\nC           -0.01433103365374        0.04797657223863       -0.00264615236691\\nH            0.62899900866273       -0.10569123341369        0.85876435761731\\nH            0.59055570067720        0.47529774693153       -0.79992050399152\\nH           -0.77021968128807        0.78686174987053        0.25176026967193\\nC           -0.65852468745705       -1.26251446989087       -0.45829688207235\\nH           -1.68493584877871       -1.10880002416069       -0.77824594343626\\nH           -0.10807854481448       -1.68997630740193       -1.29401403518074\\nH           -0.64689091334788       -2.00130903417352        0.33925488975858\\n8\\n energy: -7.333807783948 xtb: 6.7.0 (08769fc)\\nC           -0.01472834170596        0.04671455705905       -0.00169789461955\\nH            0.67121640918015       -0.11733018263467        0.82389597570846\\nH            0.54406394051511        0.50026363062728       -0.81824065637859\\nH           -0.76967297195665        0.76798119378944        0.30108274699109\\nC           -0.65928653245625       -1.26125231383520       -0.45760674973075\\nH           -1.66610655157470       -1.09716850704662       -0.82966328970511\\nH           -0.07533042891678       -1.71494051769592       -1.25626809214438\\nH           -0.69358152308493       -1.98242286026337        0.35515395987886\\n8\\n energy: -7.334370608860 xtb: 6.7.0 (08769fc)\\nC           -0.01483776505897        0.04602732171118       -0.00206499562776\\nH            0.70491574024170       -0.12557764164807        0.79281428834603\\nH            0.50523069256964        0.52160818925945       -0.83174385472312\\nH           -0.76767802340165        0.75017840169136        0.34373697851331\\nC           -0.65890388565726       -1.26056511174085       -0.45762583004888\\nH           -1.64801310930718       -1.08892735410568       -0.87178748172606\\nH           -0.04967341913363       -1.73628347267730       -1.22414109839131\\nH           -0.73446623025266       -1.96461533249008        0.36746799365783\\n8\\n energy: -7.334868810470 xtb: 6.7.0 (08769fc)\\nC           -0.01488100547883        0.04561406302855       -0.00290709714506\\nH            0.73290909173915       -0.13169968835498        0.76460574737350\\nH            0.47135558570037        0.54024478322125       -0.84195401032299\\nH           -0.76451643409568        0.73345133580647        0.38104307957524\\nC           -0.65809537408304       -1.26015194667613       -0.45786539339685\\nH           -1.63072995187619       -1.08281081955591       -0.90757448289934\\nH           -0.02877176366408       -1.75491848684234       -1.19559377203487\\nH           -0.77069614824170       -1.94788424062690        0.37690192885041\\n8\\n energy: -7.335300867591 xtb: 6.7.0 (08769fc)\\nC           -0.01491809649035        0.04538279204643       -0.00398234844627\\nH            0.75749460793057       -0.13637170257690        0.73797590502995\\nH            0.44025033281891        0.55732187060467       -0.85000604220355\\nH           -0.76031795360375        0.71728321765113        0.41513203110964\\nC           -0.65706906729126       -1.25992079614765       -0.45818833669657\\nH           -1.61380131260832       -1.07814385982131       -0.93962232968073\\nH           -0.01082698563325       -1.77199397193557       -1.16894022396610\\nH           -0.80423752512256       -1.93171254982079        0.38428734485368\\n8\\n energy: -7.335661983757 xtb: 6.7.0 (08769fc)\\nC           -0.01496895360118        0.04531725667281       -0.00520327943818\\nH            0.77957431765216       -0.13985327970186        0.71248352494250\\nH            0.41113811443126        0.57328977091159       -0.85638837062455\\nH           -0.75518378989139        0.70154454644607        0.44684890580821\\nC           -0.65590081948281       -1.25985539623242       -0.45854676939793\\nH           -1.59711132912385       -1.07466700318035       -0.96892851137522\\nH            0.00488001078943       -1.78796023989581       -1.14361044274370\\nH           -0.83585355077363       -1.91597065502002        0.39000094282891\\n8\\n energy: -7.335954846301 xtb: 6.7.0 (08769fc)\\nC           -0.01497891801514        0.04541495708445       -0.00663663197360\\nH            0.79995297362151       -0.14218532569267        0.68773780448615\\nH            0.38341139106151        0.58851408088727       -0.86157591074932\\nH           -0.74920363886194        0.68569266265241        0.47724092207436\\nC           -0.65454588629301       -1.25995326043259       -0.45901444690514\\nH           -1.58055933607838       -1.07233945346664       -0.99638197717504\\nH            0.01899924830952       -1.80318289320984       -1.11918956386223\\nH           -0.86650183374407       -1.90011576782240        0.39447580410485\\n8\\n energy: -7.336166662800 xtb: 6.7.0 (08769fc)\\nC           -0.01510530123195        0.04563302200698       -0.00795461046095\\nH            0.81869003683020       -0.14377011824897        0.66353409825918\\nH            0.35668575739824        0.60308701425401       -0.86543215102275\\nH           -0.74225867601175        0.67057226968306        0.50579124963167\\nC           -0.65326100889713       -1.26017146643634       -0.45933392834294\\nH           -1.56397211593877       -1.07075896157959       -1.02210695885378\\nH            0.03152993935796       -1.81775409832271       -1.09526965101363\\nH           -0.89573463150680       -1.88499266135643        0.39742795180323\\n8\\n energy: -7.336303823206 xtb: 6.7.0 (08769fc)\\nC           -0.01527195677429        0.04597397829801       -0.00926556991195\\nH            0.83630642895808       -0.14453856001477        0.63960527543678\\nH            0.33060868594646        0.61722981196137       -0.86826924213768\\nH           -0.73440950987311        0.65564353868773        0.53331767747249\\nC           -0.65196935552364       -1.26051255971330       -0.45961308333360\\nH           -1.54727116125075       -1.06999469803625       -1.04668358091224\\nH            0.04288375548893       -1.83189510359584       -1.07162215045080\\nH           -0.92430288697169       -1.87006140758695        0.39918667383703\\n8\\n energy: -7.336366443594 xtb: 6.7.0 (08769fc)\\nC           -0.01562281472304        0.04655294768509       -0.01066319174580\\nH            0.85304891961625       -0.14456858581688        0.61556229700597\\nH            0.30479299861509        0.63140072043421       -0.86990562622509\\nH           -0.72543364581184        0.64110778173836        0.56007755330641\\nC           -0.65053469040450       -1.26109166090785       -0.45974735657064\\nH           -1.53017174559149       -1.06996879235036       -1.07047404183968\\nH            0.05301832487057       -1.84606410289150       -1.04782147670949\\nH           -0.95252334657106       -1.85552330789105        0.39962784277836\\n8\\n energy: -7.336353939675 xtb: 6.7.0 (08769fc)\\nC           -0.01589502120179        0.04710649550386       -0.01185114532222\\nH            0.86920773941076       -0.14389493517341        0.59137180976461\\nH            0.27899978414649        0.64511288134530       -0.87091704404611\\nH           -0.71559095721725        0.62634597557127        0.58642217890628\\nC           -0.64932391399802       -1.26164532332678       -0.45988601265630\\nH           -1.51273901746817       -1.07064653291279       -1.09376312904040\\nH            0.06255611958001       -1.85977428447561       -1.02383402917088\\nH           -0.98064073325204       -1.84075927653182        0.39911337156506\\n8\\n energy: -7.336266809393 xtb: 6.7.0 (08769fc)\\nC           -0.01620900467761        0.04774742758868       -0.01293383316086\\nH            0.88495140247493       -0.14252126262729        0.56669024515876\\nH            0.25290207131730        0.65879300120733       -0.87100904252334\\nH           -0.70460536865499        0.61155213262617        0.61250302118977\\nC           -0.64819850325058       -1.26228635436936       -0.45995022742749\\nH           -1.49470508587387       -1.07202431797569       -1.11682410356472\\nH            0.07132827266083       -1.87345229484362       -0.99925342439148\\nH           -1.00888978399601       -1.82596333160622        0.39743336471939\\n8\\n energy: -7.336105083980 xtb: 6.7.0 (08769fc)\\nC           -0.01655963417672        0.04846305091864       -0.01388104961115\\nH            0.90045183560600       -0.14045216380144        0.54120040412442\\nH            0.22617055693192        0.67256735176010       -0.87013721320323\\nH           -0.69233817064522        0.59661110119796        0.63855308835995\\nC           -0.64718864886416       -1.26300205760411       -0.45993478420209\\nH           -1.47582806273165       -1.07409760224111       -1.13992466843823\\nH            0.07940250489490       -1.88722437403621       -0.97375434073245\\nH           -1.03753638101507       -1.81102030619383        0.39453456370281\\n8\\n energy: -7.335870825005 xtb: 6.7.0 (08769fc)\\nC           -0.01673165412177        0.04909391387518       -0.01452644287555\\nH            0.91597030599750       -0.13762543129367        0.51464030924320\\nH            0.19839155552793        0.68633670216491       -0.86842230357956\\nH           -0.67874737052128        0.58104945371876        0.66486297863435\\nC           -0.64652285023264       -1.26363298062722       -0.45998731678988\\nH           -1.45594787611020       -1.07692864457344       -1.16339838454072\\nH            0.08703038031700       -1.90099127107875       -0.94698688056312\\nH           -1.06686849085655       -1.79545674218577        0.39047404047132\\n8\\n energy: -7.335566972652 xtb: 6.7.0 (08769fc)\\nC           -0.01709713365699        0.04988006073338       -0.01499197124393\\nH            0.93137256481723       -0.13427136365585        0.48630797066450\\nH            0.16924694963837        0.70076336578723       -0.86507419877558\\nH           -0.66340165406505        0.56573735331046        0.69118826091212\\nC           -0.64596228119778       -1.26441915114568       -0.45979753500036\\nH           -1.43435785902991       -1.08028721601292       -1.18735233858514\\nH            0.09357266530164       -1.91541518293395       -0.91838800370885\\nH           -1.09679925180751       -1.78014286608266        0.38476381573727\\n8\\n energy: -7.335184558881 xtb: 6.7.0 (08769fc)\\nC           -0.01739300225101        0.05068540252720       -0.01528686383551\\nH            0.94729498915940       -0.12983174511701        0.45559467804084\\nH            0.13787892220448        0.71584069509140       -0.86028840532344\\nH           -0.64582859893015        0.54927380797907        0.71870615583524\\nC           -0.64558578997857       -1.26522450279048       -0.45961659794477\\nH           -1.41069586944999       -1.08473165399809       -1.21258916431074\\nH            0.09949924004835       -1.93048941458123       -0.88721387860555\\nH           -1.12859589080251       -1.76367758911086        0.37735007614395\\n8\\n energy: -7.334730284445 xtb: 6.7.0 (08769fc)\\nC           -0.01759359453571        0.05146586486452       -0.01528185822493\\nH            0.96375436253678       -0.12433080371537        0.42172460447753\\nH            0.10368085787678        0.73177900702085       -0.85352223900268\\nH           -0.62564117577546        0.53168455273550        0.74739520570432\\nC           -0.64552380697006       -1.26600494840652       -0.45942568750457\\nH           -1.38423590234480       -1.09023782086456       -1.23938292220887\\nH            0.10450025583797       -1.94642417159395       -0.85271182397550\\nH           -1.16236699662550       -1.74608668004045        0.36786072073473\\n8\\n energy: -7.334208930714 xtb: 6.7.0 (08769fc)\\nC           -0.01761352981866        0.05215857545738       -0.01490514101658\\nH            0.98103537526972       -0.11748315593977        0.38337423155854\\nH            0.06548806736023        0.74893199827882       -0.84405443916131\\nH           -0.60203401152920        0.51238051811761        0.77779346508641\\nC           -0.64587246564767       -1.26669761331198       -0.45928142059821\\nH           -1.35382468451718       -1.09709127527845       -1.26844256444615\\nH            0.10828321095136       -1.96357297405934       -0.81354345164497\\nH           -1.19888796206861       -1.72678107326427        0.35571532022231\\n8\\n energy: -7.333633985662 xtb: 6.7.0 (08769fc)\\nC           -0.01732901279488        0.05266935226590       -0.01408788221282\\nH            0.99938908884578       -0.10881583524654        0.33869185507839\\nH            0.02173955671704        0.76769629810544       -0.83077131353172\\nH           -0.57383110256239        0.49047733023543        0.81056295519113\\nC           -0.64673786639711       -1.26720832006496       -0.45927763572244\\nH           -1.31779969830770       -1.10576521659954       -1.30062111468472\\nH            0.11031745129859       -1.98233219407193       -0.76786603800340\\nH           -1.23917441679935       -1.70487641462380        0.34002517388561\\n8\\n energy: -7.333041355774 xtb: 6.7.0 (08769fc)\\nC           -0.01694681801821        0.05314002955908       -0.01314735717719\\nH            1.01800408679644       -0.09772544167127        0.28697047039510\\nH           -0.02781391274292        0.78771608219226       -0.81321057501075\\nH           -0.53993869152348        0.46510050153298        0.84614284002398\\nC           -0.64775201407438       -1.26767892071223       -0.45932493435141\\nH           -1.27522422194942       -1.11686306013383       -1.33538867640327\\nH            0.11025003619270       -2.00234593738614       -0.71529072937391\\nH           -1.28400446468074       -1.67949825338085        0.31990496189749\\n8\\n energy: -7.332525545051 xtb: 6.7.0 (08769fc)\\nC           -0.01560575238942        0.05280430001444       -0.01139628936166\\nH            1.03780565185171       -0.08328987839661        0.22232796145885\\nH           -0.08812486255365        0.81009142583205       -0.78702384101393\\nH           -0.49761741123636        0.43420111775897        0.88507777046751\\nC           -0.64984957930445       -1.26734309912083       -0.46000673660391\\nH           -1.22085958357940       -1.13130765328402       -1.37557550507949\\nH            0.10562885949919       -2.02471337607882       -0.64969992141033\\nH           -1.33480332228763       -1.64859783672518        0.29295256154301\\n8\\n energy: -7.332248232936 xtb: 6.7.0 (08769fc)\\nC           -0.01348143912452        0.05159804133227       -0.00942278928581\\nH            1.05614464963472       -0.06479139629042        0.14483427714534\\nH           -0.15803952959553        0.83304421543574       -0.75099067498168\\nH           -0.44616336615980        0.39681345374662        0.92598143424194\\nC           -0.65241752351610       -1.26613678565634       -0.46135319230836\\nH           -1.15389004792523       -1.14981652184106       -1.41866015278368\\nH            0.09491918982160       -2.04765634937703       -0.57177557761922\\nH           -1.39049793313514       -1.61120965734977        0.25804267559150\\n8\\n energy: -7.332374812909 xtb: 6.7.0 (08769fc)\\nC           -0.01109941953477        0.04969797890894       -0.00798390831456\\nH            1.06903264400914       -0.04339665876944        0.06224433074316\\nH           -0.22994349180060        0.85295648008947       -0.70784063835151\\nH           -0.38968211406109        0.35555218443365        0.96387459143651\\nC           -0.65456705227742       -1.26423675209357       -0.46312068409060\\nH           -1.08030072446610       -1.17122178802880       -1.45830054237967\\nH            0.07816082012914       -2.06755781977157       -0.48960689654621\\nH           -1.44502666199830       -1.56994862476868        0.21738974750292\\n8\\n energy: -7.332839884916 xtb: 6.7.0 (08769fc)\\nC           -0.00936352571414        0.04787033867706       -0.00754214286004\\nH            1.07447926012723       -0.02256682172088       -0.01121968391134\\nH           -0.29198085235338        0.86693586169780       -0.66576819279132\\nH           -0.33686015644424        0.31697439029419        0.99338661585108\\nC           -0.65556126452556       -1.26240920236499       -0.46461077009852\\nH           -1.01284010761510       -1.19206050121825       -1.48788638883727\\nH            0.05913502446414       -2.08152733178496       -0.41710158795981\\nH           -1.49043437793895       -1.53137173357997        0.17739815060725\\n8\\n energy: -7.333427301329 xtb: 6.7.0 (08769fc)\\nC           -0.00848625687541        0.04652598869296       -0.00775169289659\\nH            1.07459251501131       -0.00426420082418       -0.07065349897014\\nH           -0.34105245510122        0.87591260183827       -0.62921720900214\\nH           -0.29154350686092        0.28437219311608        1.01452375207454\\nC           -0.65565554938589       -1.26106494749161       -0.46550792048500\\nH           -0.95683431390588       -1.21036995494035       -1.50777337454571\\nH            0.04100067295007       -2.09049588902062       -0.35866111607469\\nH           -1.52544710583207       -1.49877079137055        0.14169705989978\\n8\\n energy: -7.333999952458 xtb: 6.7.0 (08769fc)\\nC           -0.00784291355509        0.04525110391915       -0.00810474447436\\nH            1.07173905313602        0.01190338570296       -0.12047789805816\\nH           -0.38113768236822        0.88168860617839       -0.59643850711242\\nH           -0.25240077114066        0.25662694146060        1.02969265520118\\nC           -0.65553665935867       -1.25979015454824       -0.46623224013937\\nH           -0.90890238802059       -1.22654302980070       -1.52166459360721\\nH            0.02343269691844       -2.09626487440290       -0.30995058285541\\nH           -1.55277733561123       -1.47102697850925        0.10983191104579\\n8\\n energy: -7.334542794917 xtb: 6.7.0 (08769fc)\\nC           -0.00810393866175        0.04477578744739       -0.00874282998812\\nH            1.06700537621892        0.02662222750448       -0.16177163516435\\nH           -0.41420655191838        0.88591995011412       -0.56809993017208\\nH           -0.21693095533209        0.23263365352007        1.04145130765435\\nC           -0.65484803600645       -1.25931489073012       -0.46619853681292\\nH           -0.86838877780603       -1.24126622771418       -1.53094357768712\\nH            0.00771643588552       -2.10049027818000       -0.26933439826855\\nH           -1.57566955237973       -1.44703522196175        0.08029560043883\\n8\\n energy: -7.335020003605 xtb: 6.7.0 (08769fc)\\nC           -0.00869517267906        0.04458971741204       -0.00935604852847\\nH            1.06114904692973        0.04020498115585       -0.19764292494630\\nH           -0.44263997460867        0.88895933603196       -0.54232137517113\\nH           -0.18431077240003        0.21148257102162        1.05055816132673\\nC           -0.65407297813044       -1.25912884329944       -0.46584514145358\\nH           -0.83261467157163       -1.25485262429481       -1.53735921403699\\nH           -0.00712856340641       -2.10352439525774       -0.23394158325035\\nH           -1.59511291413349       -1.42588574276946        0.05256412606013\\n8\\n energy: -7.335429032595 xtb: 6.7.0 (08769fc)\\nC           -0.00949200415392        0.04458185452571       -0.00988653149493\\nH            1.05443220358377        0.05316243930073       -0.23010932334550\\nH           -0.46807990134817        0.89121586840042       -0.51790901773671\\nH           -0.15330648354944        0.19227974306579        1.05776390058951\\nC           -0.65330751955606       -1.25912097680242       -0.46527031491977\\nH           -0.79976484508873       -1.26781326360258       -1.54183016938302\\nH           -0.02168157327698       -2.10577605799609       -0.20182640026915\\nH           -1.61222587661048       -1.40668460689155        0.02572385655960\\n8\\n energy: -7.335765977031 xtb: 6.7.0 (08769fc)\\nC           -0.01044473947794        0.04472107828701       -0.01030937188905\\nH            1.04700788587754        0.06570132816931       -0.26001590243518\\nH           -0.49122753620123        0.89291000770091       -0.49440965792894\\nH           -0.12340128417901        0.17465558997225        1.06344846083586\\nC           -0.65259168663516       -1.25926017165631       -0.46451262783813\\nH           -0.76909336649393       -1.28035498290308       -1.54478201313289\\nH           -0.03613657678602       -2.10746561924643       -0.17217675833079\\nH           -1.62753869610425       -1.38906223032365       -0.00058612928085\\n8\\n energy: -7.336033371123 xtb: 6.7.0 (08769fc)\\nC           -0.01160401777332        0.04500322917170       -0.01054182797584\\nH            1.03892810080300        0.07802467738361       -0.28808734250263\\nH           -0.51284376632744        0.89401867973043       -0.47152738647103\\nH           -0.09366807530157        0.15821390029147        1.06807645125007\\nC           -0.65198664715742       -1.25954225503674       -0.46349678948062\\nH           -0.73993420258211       -1.29268089835303       -1.54650496980681\\nH           -0.05051935155432       -2.10856990847238       -0.14417666481783\\nH           -1.64179804010681       -1.37262242471505       -0.02708547019526\\n8\\n energy: -7.336220489565 xtb: 6.7.0 (08769fc)\\nC           -0.01275042235836        0.04536052815074       -0.01071317846678\\nH            1.03026093974334        0.09020435916553       -0.31459419336675\\nH           -0.53276720719915        0.89496959965149       -0.44880022196533\\nH           -0.06489467267136        0.14297756993930        1.07134589419674\\nC           -0.65144352001297       -1.25989948056670       -0.46247274389421\\nH           -0.71205489988900       -1.30486291939043       -1.54715329201825\\nH           -0.06531928822111       -2.10951660061528       -0.11782448456225\\nH           -1.65445692939139       -1.35738805633465       -0.05313177992313\\n8\\n energy: -7.336332889035 xtb: 6.7.0 (08769fc)\\nC           -0.01394767817782        0.04579981612725       -0.01074884014815\\nH            1.02099648240032        0.10241002675018       -0.33998763051367\\nH           -0.55153548982104        0.89563760471522       -0.42606927186521\\nH           -0.03627707135032        0.12864620209169        1.07361086487891\\nC           -0.65101142591967       -1.26033867537468       -0.46135557346742\\nH           -0.68502672017866       -1.31707074975467       -1.54686777703050\\nH           -0.08050726102461       -2.11018047153222       -0.09256036942919\\nH           -1.66611683592820       -1.34305875302277       -0.07936540242473\\n8\\n energy: -7.336370557561 xtb: 6.7.0 (08769fc)\\nC           -0.01531752180721        0.04644713871308       -0.01080659584368\\nH            1.01109997257679        0.11500870910621       -0.36465349783145\\nH           -0.56926527789264        0.89644722976473       -0.40267794603379\\nH           -0.00766850896699        0.11502582276095        1.07466891472725\\nC           -0.65050106899959       -1.26098589332565       -0.46008298603301\\nH           -0.65847428117183       -1.32967146035618       -1.54574403437608\\nH           -0.09666362561468       -2.11098597110248       -0.06805571634325\\nH           -1.67663568812384       -1.32944057556066       -0.10599213826596\\n\",\"xyz\");\n\tviewer_17338420579899318.setStyle({\"stick\": {\"colorscheme\": \"greenCarbon\"}});\n\tviewer_17338420579899318.zoomTo();\n\tviewer_17338420579899318.animate({\"loop\": \"backAndForth\"});\nviewer_17338420579899318.render();\n});\n</script>",
            "text/html": [
              "<div id=\"3dmolviewer_17338420579899318\"  style=\"position: relative; width: 800px; height: 600px;\">\n",
              "        <p id=\"3dmolwarning_17338420579899318\" style=\"background-color:#ffcccc;color:black\">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br></p>\n",
              "        </div>\n",
              "<script>\n",
              "\n",
              "var loadScriptAsync = function(uri){\n",
              "  return new Promise((resolve, reject) => {\n",
              "    //this is to ignore the existence of requirejs amd\n",
              "    var savedexports, savedmodule;\n",
              "    if (typeof exports !== 'undefined') savedexports = exports;\n",
              "    else exports = {}\n",
              "    if (typeof module !== 'undefined') savedmodule = module;\n",
              "    else module = {}\n",
              "\n",
              "    var tag = document.createElement('script');\n",
              "    tag.src = uri;\n",
              "    tag.async = true;\n",
              "    tag.onload = () => {\n",
              "        exports = savedexports;\n",
              "        module = savedmodule;\n",
              "        resolve();\n",
              "    };\n",
              "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
              "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
              "});\n",
              "};\n",
              "\n",
              "if(typeof $3Dmolpromise === 'undefined') {\n",
              "$3Dmolpromise = null;\n",
              "  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.4.2/3Dmol-min.js');\n",
              "}\n",
              "\n",
              "var viewer_17338420579899318 = null;\n",
              "var warn = document.getElementById(\"3dmolwarning_17338420579899318\");\n",
              "if(warn) {\n",
              "    warn.parentNode.removeChild(warn);\n",
              "}\n",
              "$3Dmolpromise.then(function() {\n",
              "viewer_17338420579899318 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_17338420579899318\"),{backgroundColor:\"white\"});\n",
              "viewer_17338420579899318.zoomTo();\n",
              "\tviewer_17338420579899318.addModelsAsFrames(\"8\\n energy: -7.336369756748 xtb: 6.7.0 (08769fc)\\nC           -0.01557678914544        0.04676952851446       -0.01140071053055\\nH           -0.72244760195278        0.63613433703088        0.56910045483325\\nH            0.85821432826868       -0.14545593530701        0.60776190560494\\nH            0.29610899794758        0.63657615178766       -0.87079497028301\\nC           -0.64988992692328       -1.26128735487501       -0.46007407919487\\nH           -0.96201349834668       -1.85062335079973        0.39986038880813\\nH           -1.52456516250983       -1.06910210267221       -1.07796226527881\\nH            0.05674365266173       -1.85116627367905       -1.03983472395905\\n8\\n energy: -7.336333263138 xtb: 6.7.0 (08769fc)\\nC           -0.01420502669525        0.04602897026964       -0.01100112538491\\nH           -0.71206026681302        0.62228137727202        0.59299882590896\\nH            0.87584863742636       -0.14450594829268        0.58389835603764\\nH            0.26822007086304        0.65038669039304       -0.87009843537225\\nC           -0.65068618525168       -1.26056894182954       -0.46119692581622\\nH           -0.98801335059685       -1.83668873416564        0.39797254817177\\nH           -1.50789986206766       -1.07003604570283       -1.10251605060068\\nH            0.06536998313504       -1.86505236794402       -1.01340119294427\\n8\\n energy: -7.336221222362 xtb: 6.7.0 (08769fc)\\nC           -0.01287823396353        0.04562533422993       -0.01128083581802\\nH           -0.70109539441141        0.60825241166685        0.61650952715840\\nH            0.89258313469929       -0.14402479543607        0.55974765609212\\nH            0.23984097287035        0.66527139848417       -0.86872798786110\\nC           -0.65086595315077       -1.26016429185724       -0.46254094500444\\nH           -1.01383612668103       -1.82266398880034        0.39545678326156\\nH           -1.49069856441476       -1.07052223490840       -1.12633157611982\\nH            0.07352416505185       -1.87992883337890       -0.98617662170867\\n8\\n energy: -7.336034699107 xtb: 6.7.0 (08769fc)\\nC           -0.01158099530827        0.04536019400288       -0.01159521026920\\nH           -0.68900219986925        0.59387374351996        0.64027443654528\\nH            0.90917023721602       -0.14273302153357        0.53448781909530\\nH            0.21089393725028        0.68031708971906       -0.86618907552442\\nC           -0.65100099276165       -1.25989934515802       -0.46386910379030\\nH           -1.04026968329599       -1.80828296043551        0.39196142877435\\nH           -1.47239990712321       -1.07181822335785       -1.15038048963373\\nH            0.08076360389206       -1.89497247675695       -0.95803380519725\\n8\\n energy: -7.335775709481 xtb: 6.7.0 (08769fc)\\nC           -0.01053077065906        0.04537760386069       -0.01205866519827\\nH           -0.67566924258730        0.57884797803700        0.66448309677166\\nH            0.92552601611409       -0.14099579519476        0.50773731166080\\nH            0.18100963570414        0.69583499678498       -0.86257936448219\\nC           -0.65091355165413       -1.25991689105178       -0.46501367472884\\nH           -1.06753462883872       -1.79325555492792        0.38744472828155\\nH           -1.45261878126901       -1.07355984968312       -1.17470726161974\\nH            0.08730532318998       -1.91048748782509       -0.92865017068494\\n8\\n energy: -7.335441101708 xtb: 6.7.0 (08769fc)\\nC           -0.00937241465481        0.04541106498129       -0.01239239087613\\nH           -0.66055404924262        0.56315005108852        0.68961573795018\\nH            0.94221957727377       -0.13784597087253        0.47907019305826\\nH            0.14986982723605        0.71146677002330       -0.85741862870658\\nC           -0.65098441320486       -1.25995048458489       -0.46621705521190\\nH           -1.09626399454927       -1.77755597511662        0.38155487323527\\nH           -1.43114286107559       -1.07671432088318       -1.19999034195830\\nH            0.09280232821730       -1.92611613463588       -0.89756638749078\\n8\\n energy: -7.335040458077 xtb: 6.7.0 (08769fc)\\nC           -0.00851880128027        0.04565620854679       -0.01252391209690\\nH           -0.64316368878029        0.54646090630537        0.71625933004867\\nH            0.95919446545715       -0.13324103625777        0.44763818711320\\nH            0.11715223037754        0.72705710903738       -0.85072938580621\\nC           -0.65114453224770       -1.26019571303724       -0.46706574452056\\nH           -1.12717535566444       -1.76086518480514        0.37402237719268\\nH           -1.40715346039288       -1.08132424279983       -1.22645882272380\\nH            0.09738314253088       -1.94170304698956       -0.86448602920706\\n8\\n energy: -7.334562462917 xtb: 6.7.0 (08769fc)\\nC           -0.00781727328261        0.04610073418205       -0.01267816530394\\nH           -0.62283385110980        0.52813901016042        0.74493773613601\\nH            0.97663007332657       -0.12737088766018        0.41233941534389\\nH            0.08142888315901        0.74371312783050       -0.84201406573853\\nC           -0.65123261123394       -1.26064031300941       -0.46777856297248\\nH           -1.16098376702720       -1.74254164715733        0.36439510097464\\nH           -1.37967122245812       -1.08719986018805       -1.25464864561069\\nH            0.10105376862608       -1.95835516415801       -0.82789681282887\\n8\\n energy: -7.334019744680 xtb: 6.7.0 (08769fc)\\nC           -0.00749862357327        0.04679376294950       -0.01264928174356\\nH           -0.59844647620457        0.50759412968275        0.77648220836331\\nH            0.99459895376295       -0.11961557346494        0.37175114214356\\nH            0.04193360074309        0.76120357216994       -0.83075004700311\\nC           -0.65136596535909       -1.26133336530803       -0.46806934990503\\nH           -1.19884517850082       -1.72199512603425        0.35189534497271\\nH           -1.34737873458659       -1.09496129370053       -1.28510170529465\\nH            0.10357642371828       -1.97584110629445       -0.78690231153321\\n8\\n energy: -7.333435876664 xtb: 6.7.0 (08769fc)\\nC           -0.00769004481664        0.04773937148876       -0.01236478269665\\nH           -0.56870054237425        0.48404409846989        0.81145823204552\\nH            1.01287253688064       -0.10949338998687        0.32431523859777\\nH           -0.00241075709382        0.77966426578728       -0.81611045550843\\nC           -0.65157060722837       -1.26227892391296       -0.46779404011678\\nH           -1.24172580777970       -1.69844349665647        0.33548440118605\\nH           -1.30873067061508       -1.10509040773538       -1.31812102710533\\nH            0.10452989302721       -1.99429651745425       -0.74021156640212\\n8\\n energy: -7.332852479030 xtb: 6.7.0 (08769fc)\\nC           -0.00813838721312        0.04875627807554       -0.01184063040783\\nH           -0.53190823909880        0.45615542562854        0.85013537623302\\nH            1.03128850107034       -0.09656295905040        0.26824740139961\\nH           -0.05363192769682        0.79952461659033       -0.79674077438899\\nC           -0.65191572590409       -1.26329573422463       -0.46719668442538\\nH           -1.29044152673525       -1.67055337206680        0.31366026627262\\nH           -1.26199047374223       -1.11802877132756       -1.35414738972518\\nH            0.10331177931994       -2.01415048362500       -0.68546156495784\\n8\\n energy: -7.332394796759 xtb: 6.7.0 (08769fc)\\nC           -0.01016430387855        0.05007232725041       -0.01045376421163\\nH           -0.48400670559496        0.42210197638109        0.89468116631735\\nH            1.04832294801960       -0.07926328311433        0.19842829926269\\nH           -0.11364097191792        0.81990431070232       -0.77080229820286\\nC           -0.65254930416966       -1.26461145966297       -0.46482451915388\\nH           -1.34838842672778       -1.63649870012924        0.28331298125491\\nH           -1.20182361352644       -1.13533784669937       -1.39344755690915\\nH            0.09882437779570       -2.03452232472791       -0.62023830835740\\n8\\n energy: -7.332242402076 xtb: 6.7.0 (08769fc)\\nC           -0.01326624940186        0.05114847859722       -0.00843309324724\\nH           -0.42581146062707        0.38225729201472        0.94088958490621\\nH            1.06104299783815       -0.05835009520737        0.11755695779945\\nH           -0.18064619341843        0.83954856140785       -0.73758845366579\\nC           -0.65342237726085       -1.26568712707518       -0.46122674770444\\nH           -1.41132890239290       -1.59665343747387        0.24381165323519\\nH           -1.12979896219406       -1.15626135240565       -1.43235760287537\\nH            0.08980514745701       -2.05415731985772       -0.54599629844798\\n8\\n energy: -7.332501350740 xtb: 6.7.0 (08769fc)\\nC           -0.01647163325400        0.05152565551091       -0.00636005148375\\nH           -0.36469005597938        0.34147665723000        0.98101037331407\\nH            1.06683192263652       -0.03674250274558        0.03658621627732\\nH           -0.24638988473814        0.85586050164098       -0.70025373144132\\nC           -0.65431035061461       -1.26606380517078       -0.45751408373167\\nH           -1.46950274000948       -1.55587316293899        0.19952466109310\\nH           -1.05537364172485       -1.17787871075329       -1.46476473287109\\nH            0.07648038368392       -2.07045963277326       -0.47157265115665\\n8\\n energy: -7.333026747449 xtb: 6.7.0 (08769fc)\\nC           -0.01874615483048        0.05123776484562       -0.00490689266687\\nH           -0.31029517545785        0.30565634189836        1.01007344879352\\nH            1.06681829059529       -0.01752154314887       -0.03244637642874\\nH           -0.30218421657450        0.86755805352546       -0.66434588586339\\nC           -0.65492356282051       -1.26577556355917       -0.45488560759034\\nH           -1.51501086930052       -1.52005392772737        0.15790071224472\\nH           -0.99027422195541       -1.19710759477209       -1.48772687232201\\nH            0.06118991034396       -2.08214853106194       -0.40700652616687\\n8\\n energy: -7.333623545214 xtb: 6.7.0 (08769fc)\\nC           -0.02001278772154        0.05062699946026       -0.00417327657816\\nH           -0.26461862073085        0.27572987465734        1.02997777127694\\nH            1.06380714202408       -0.00093537457297       -0.08831018635732\\nH           -0.34747527179645        0.87565082668485       -0.63187327866278\\nC           -0.65519370610942       -1.26516461103392       -0.45344708650896\\nH           -1.54898108722818       -1.49012888581382        0.12145020766763\\nH           -0.93659479330733       -1.21369993257345       -1.50347953967611\\nH            0.04564312486967       -2.09023389680830       -0.35348861116121\\n8\\n energy: -7.334186889397 xtb: 6.7.0 (08769fc)\\nC           -0.02119367583224        0.04996642573935       -0.00362846569107\\nH           -0.22476089051792        0.24993217294216        1.04436599314128\\nH            1.05861207290786        0.01379931119936       -0.13567071435883\\nH           -0.38510678924468        0.88114889114231       -0.60238116958520\\nC           -0.65531434546662       -1.26450387948162       -0.45215226844027\\nH           -1.57581329064334       -1.46433276942045        0.08865120924895\\nH           -0.89020674117878       -1.22843963325917       -1.51434274244277\\nH            0.03035765997571       -2.09572551886195       -0.30818584187207\\n8\\n energy: -7.334710975474 xtb: 6.7.0 (08769fc)\\nC           -0.02117289405202        0.04924864668460       -0.00387234821826\\nH           -0.19023822525678        0.22756885555390        1.05428551826884\\nH            1.05354214099474        0.02761653238275       -0.17531429881178\\nH           -0.41761370210886        0.88547148324753       -0.57455891532054\\nC           -0.65509122682507       -1.26378612953863       -0.45225312347377\\nH           -1.59665612230143       -1.44197111844754        0.05939608547163\\nH           -0.85113720014029       -1.24226099370780       -1.52275550162376\\nH            0.01494122968970       -2.10004227617481       -0.26827141629235\\n8\\n energy: -7.335166146751 xtb: 6.7.0 (08769fc)\\nC           -0.02076754056987        0.04859438516728       -0.00445500291041\\nH           -0.15917434897782        0.20758291582140        1.06140315647945\\nH            1.04812689672338        0.04078096819333       -0.21001429546278\\nH           -0.44616724083645        0.88873046051629       -0.54816196107294\\nC           -0.65467666042907       -1.26313196820128       -0.45282937400767\\nH           -1.61370577837908       -1.42198688606897        0.03247021218341\\nH           -0.81661438030041       -1.25542897202379       -1.52919725877899\\nH           -0.00044694723069       -2.10329590340425       -0.23255947643006\\n8\\n energy: -7.335558009938 xtb: 6.7.0 (08769fc)\\nC           -0.01995176491192        0.04800890282854       -0.00549047698564\\nH           -0.13038187378031        0.18892612250453        1.06648117268194\\nH            1.04259932998067        0.05378860370778       -0.24149184368494\\nH           -0.47236077596430        0.89132769246456       -0.52222147083557\\nC           -0.65397170893376       -1.26254667072883       -0.45394335481114\\nH           -1.62807618637419       -1.40333184714889        0.00700749972351\\nH           -0.78509287115120       -1.26843977400215       -1.53446054186733\\nH           -0.01619014886499       -2.10588802962554       -0.19922498422080\\n8\\n energy: -7.335868616349 xtb: 6.7.0 (08769fc)\\nC           -0.01920967801147        0.04753820411001       -0.00647165227749\\nH           -0.10283494639333        0.17172422152849        1.07018536725782\\nH            1.03641541361278        0.06672283454529       -0.27070392904395\\nH           -0.49610894890859        0.89315752111253       -0.49689462154282\\nC           -0.65329344136317       -1.26207614493581       -0.45496976257083\\nH           -1.64073665216611       -1.38613175886695       -0.01773778726124\\nH           -0.75548918130837       -1.28137685598160       -1.53835085602392\\nH           -0.03216856546175       -2.10771302151195       -0.16840075853753\\n8\\n energy: -7.336104810094 xtb: 6.7.0 (08769fc)\\nC           -0.01823957914619        0.04716735283317       -0.00762836822741\\nH           -0.07627531732314        0.15587780269216        1.07269853616840\\nH            1.02987708379741        0.08001219690987       -0.29784577884115\\nH           -0.51787497939810        0.89446617297092       -0.47165713298137\\nC           -0.65252554291372       -1.26170550517053       -0.45626959872524\\nH           -1.65194549420610       -1.37028720861219       -0.04194834277209\\nH           -0.72771974923607       -1.29466880359763       -1.54121785554763\\nH           -0.04872242157411       -2.10901700802577       -0.13947545907348\\n8\\n energy: -7.336266660115 xtb: 6.7.0 (08769fc)\\nC           -0.01734082517331        0.04692194756198       -0.00888337447774\\nH           -0.05049660489833        0.14045843051368        1.07409955555828\\nH            1.02295232924798        0.09321972627620       -0.32383390804069\\nH           -0.53854585673662        0.89535622231991       -0.44646447854088\\nC           -0.65164122168484       -1.26146031696186       -0.45753485843610\\nH           -1.66184571048548       -1.35486976881754       -0.06579263359856\\nH           -0.70090962739121       -1.30787875459629       -1.54333646224621\\nH           -0.06559848287820       -2.10990248629608       -0.11159784021808\\n8\\n energy: -7.336353895727 xtb: 6.7.0 (08769fc)\\nC           -0.01638394037344        0.04678581256189       -0.01020831313264\\nH           -0.02504105748043        0.12575894769336        1.07460374265664\\nH            1.01558176346277        0.10676824046484       -0.34878633078275\\nH           -0.55804924631900        0.89581846169197       -0.42101775962022\\nC           -0.65071031327658       -1.26132441195392       -0.45887819883828\\nH           -1.67079272569037       -1.34017229758291       -0.08963060952942\\nH           -0.67492779861868       -1.32142953364444       -1.54468990714154\\nH           -0.08310268170429       -2.11036021923079       -0.08473662361176\\n8\\n energy: -7.336366557842 xtb: 6.7.0 (08769fc)\\nC           -0.01542037243824        0.04676513641146       -0.01157453261955\\nH            0.00040906359187        0.11161900802834        1.07428586957659\\nH            1.00763792903659        0.12072722511018       -0.37305760622520\\nH           -0.57659774510609        0.89588680435542       -0.39511802010736\\nC           -0.64974271177994       -1.26130397108765       -0.46024156770921\\nH           -1.67896279203634       -1.32603446106078       -0.11373702401522\\nH           -0.64939750240891       -1.33539065978685       -1.54527600689241\\nH           -0.10135186885896       -2.11042408197014       -0.05862511200762\\n8\\n energy: -7.336304648329 xtb: 6.7.0 (08769fc)\\nC           -0.01447719500117        0.04685908335004       -0.01294143711148\\nH            0.02618036706964        0.09793528800981        1.07319447471754\\nH            0.99898344183804        0.13520043218451       -0.39694497782977\\nH           -0.59434879167803        0.89556016631609       -0.36855381983072\\nC           -0.64876768663221       -1.26139815130458       -0.46158592389590\\nH           -1.68651038390160       -1.31235296158556       -0.13840371788451\\nH           -0.62399272422865       -1.34986590536318       -1.54506413428939\\nH           -0.12049302746604       -2.11009295160712       -0.03304446387575\\n8\\n energy: -7.336168166528 xtb: 6.7.0 (08769fc)\\nC           -0.01358083464819        0.04706702659569       -0.01426825476112\\nH            0.05261015616002        0.08460910802736        1.07133745053996\\nH            0.98944309591489        0.15030096348292       -0.42073063279920\\nH           -0.61143487051052        0.89482607110049       -0.34108937642822\\nC           -0.64781488888994       -1.26160631976924       -0.46287277655556\\nH           -1.69355520259703       -1.29902913812600       -0.16394614818631\\nH           -0.59838906038847       -1.36496839436119       -1.54398298641633\\nH           -0.14070439504078       -2.10935431695002       -0.00779127539321\\n8\\n energy: -7.335957146662 xtb: 6.7.0 (08769fc)\\nC           -0.01275557656222        0.04738693359813       -0.01550996946734\\nH            0.08007256432749        0.07154053271439        1.06868166332148\\nH            0.97879321477709        0.16616583728880       -0.44469839420390\\nH           -0.62797192345063        0.89365388862721       -0.31244551958846\\nC           -0.64691868507624       -1.26192643638403       -0.46406424162032\\nH           -1.70019052850552       -1.28596310010009       -0.19072815299015\\nH           -0.57224443078856       -1.38083515371477       -1.54191609225612\\nH           -0.16221063472142       -2.10817750202964        0.01733670680484\\n8\\n energy: -7.335671767187 xtb: 6.7.0 (08769fc)\\nC           -0.01202498335047        0.04781509952968       -0.01661504091908\\nH            0.10900711725684        0.05861782614338        1.06514372911917\\nH            0.96674018865599        0.18297014803870       -0.46915330009945\\nH           -0.64406640699680        0.89199050736480       -0.28227411607135\\nC           -0.64611983281183       -1.26235478822939       -0.46512094683544\\nH           -1.70648401475910       -1.27304315024042       -0.21919196492640\\nH           -0.54517347882084       -1.39764129303679       -1.53868809507282\\nH           -0.18530458917382       -2.10650934956997        0.04255573480540\\n8\\n energy: -7.335311661773 xtb: 6.7.0 (08769fc)\\nC           -0.01148445793366        0.04837257904736       -0.01740708186949\\nH            0.14013003407053        0.04539335128044        1.06045317341864\\nH            0.95279382216566        0.20098347801067       -0.49475035886877\\nH           -0.66006141579291        0.88979791170567       -0.24997638028418\\nC           -0.64555291009813       -1.26291240268613       -0.46589421548544\\nH           -1.71241896430269       -1.25982174338952       -0.25010304273243\\nH           -0.51639540573835       -1.41565642777450       -1.53405473424075\\nH           -0.21043670237047       -2.10431174619399        0.06838864006244\\n8\\n energy: -7.334887494143 xtb: 6.7.0 (08769fc)\\nC           -0.01118441938399        0.04897389089582       -0.01767708946461\\nH            0.17427981587139        0.03235716145875        1.05463451490832\\nH            0.93606878197551        0.22042677618578       -0.52155409307671\\nH           -0.67535868279646        0.88690304709131       -0.21536688139688\\nC           -0.64539821332717       -1.26351377000289       -0.46626694829353\\nH           -1.71829762759088       -1.24678899514120       -0.28424386062960\\nH           -0.48555470725749       -1.43510144469613       -1.52720260888725\\nH           -0.23798094749093       -2.10141166579144        0.09433296684028\\n8\\n energy: -7.334385847854 xtb: 6.7.0 (08769fc)\\nC           -0.01096667497308        0.04965986337507       -0.01771768314578\\nH            0.21245477567722        0.01864939172424        1.04693980528616\\nH            0.91608262257850        0.24222059496048       -0.55042287323835\\nH           -0.69083104721149        0.88304553663047       -0.17668730379741\\nC           -0.64543246538170       -1.26419976434362       -0.46648570915547\\nH           -1.72374691787052       -1.23308520961369       -0.32280485861881\\nH           -0.45168145035122       -1.45689695472578       -1.51796238231176\\nH           -0.26930484246772       -2.09754845800716        0.12179700498146\\n8\\n energy: -7.333824738817 xtb: 6.7.0 (08769fc)\\nC           -0.01096703830211        0.05036638023012       -0.01716933756201\\nH            0.25600239016365        0.00405878945123        1.03670109236294\\nH            0.89139112378561        0.26683985379394       -0.58195264404431\\nH           -0.70627353051852        0.87788768122009       -0.13305277619360\\nC           -0.64594948118281       -1.26490621844612       -0.46630277212290\\nH           -1.72858545463159       -1.21849932013239       -0.36727903123517\\nH           -0.41373300665697       -1.48151782734846       -1.50517048192242\\nH           -0.30531100265727       -2.09238433876842        0.15088195071750\\n8\\n energy: -7.333230164518 xtb: 6.7.0 (08769fc)\\nC           -0.01091615065108        0.05106338608157       -0.01648956542892\\nH            0.30500773290153       -0.01160393382783        1.02283645434531\\nH            0.86162456201361        0.29473352161910       -0.61593521201538\\nH           -0.72183964354696        0.87073769799048       -0.08325144352877\\nC           -0.64660748769888       -1.26560315050540       -0.46612441871665\\nH           -1.73182130498325       -1.20284216899583       -0.41810673627216\\nH           -0.37178267428319       -1.50941297891928       -1.48840892076650\\nH           -0.34709103375180       -2.08522737344283        0.18213584238310\\n8\\n energy: -7.332677405655 xtb: 6.7.0 (08769fc)\\nC           -0.01145787331507        0.05147210039053       -0.01413105068704\\nH            0.36480278163367       -0.02956221155767        1.00314856415136\\nH            0.82119000941587        0.32779755793481       -0.65498101489048\\nH           -0.73690064763569        0.86085096660710       -0.02421814818049\\nC           -0.64865124606427       -1.26601154990477       -0.46482851608451\\nH           -1.73315705952178       -1.18489100900019       -0.48104744015286\\nH           -0.32150743076161       -1.54247821309625       -1.46327232519865\\nH           -0.39774453375114       -2.07533264137354        0.21598593104270\\n8\\n energy: -7.332300218703 xtb: 6.7.0 (08769fc)\\nC           -0.01228264565481        0.05135535677590       -0.01070454350309\\nH            0.43469990320867       -0.05002256764442        0.97507020933162\\nH            0.76876734131160        0.36619608610059       -0.69770340971512\\nH           -0.75081222426332        0.84682940346311        0.04509296136176\\nC           -0.65160784661848       -1.26589434607951       -0.46291028588520\\nH           -1.72994299071973       -1.16443955363763       -0.55630722789987\\nH           -0.26377547111077       -1.58087738737662       -1.42805428596302\\nH           -0.45847206615318       -2.06130199160143        0.25217258227295\\n8\\n energy: -7.332285767003 xtb: 6.7.0 (08769fc)\\nC           -0.01320506133629        0.05050693077791       -0.00689158070561\\nH            0.50923635736352       -0.07164223073063        0.93798152282228\\nH            0.70636918401949        0.40724901484976       -0.73986946705769\\nH           -0.76167254547038        0.82825259247446        0.12078629923949\\nC           -0.65489630932428       -1.26504540772742       -0.46077143148187\\nH           -1.71977643297589       -1.14283020280374       -0.63894100223624\\nH           -0.20324804364294       -1.62193008919018       -1.38324407084236\\nH           -0.52623314863324       -2.04271560765014        0.28760573026205\\n8\\n energy: -7.332659680644 xtb: 6.7.0 (08769fc)\\nC           -0.01391928639816        0.04921156790947       -0.00404351417867\\nH            0.57617483040537       -0.09089337028538        0.89706012338432\\nH            0.64414929317404        0.44473505171348       -0.77468647522028\\nH           -0.76794403722000        0.80728337744050        0.19207300044740\\nC           -0.65734415136448       -1.26374965966951       -0.45915011431032\\nH           -1.70346694030231       -1.12358919999348       -0.71568545134144\\nH           -0.14970967613416       -1.65941506830087       -1.33615626207740\\nH           -0.59136603216031       -2.02173769881421        0.31724469329641\\n8\\n energy: -7.333225182290 xtb: 6.7.0 (08769fc)\\nC           -0.01433103365374        0.04797657223863       -0.00264615236691\\nH            0.62899900866273       -0.10569123341369        0.85876435761731\\nH            0.59055570067720        0.47529774693153       -0.79992050399152\\nH           -0.77021968128807        0.78686174987053        0.25176026967193\\nC           -0.65852468745705       -1.26251446989087       -0.45829688207235\\nH           -1.68493584877871       -1.10880002416069       -0.77824594343626\\nH           -0.10807854481448       -1.68997630740193       -1.29401403518074\\nH           -0.64689091334788       -2.00130903417352        0.33925488975858\\n8\\n energy: -7.333807783948 xtb: 6.7.0 (08769fc)\\nC           -0.01472834170596        0.04671455705905       -0.00169789461955\\nH            0.67121640918015       -0.11733018263467        0.82389597570846\\nH            0.54406394051511        0.50026363062728       -0.81824065637859\\nH           -0.76967297195665        0.76798119378944        0.30108274699109\\nC           -0.65928653245625       -1.26125231383520       -0.45760674973075\\nH           -1.66610655157470       -1.09716850704662       -0.82966328970511\\nH           -0.07533042891678       -1.71494051769592       -1.25626809214438\\nH           -0.69358152308493       -1.98242286026337        0.35515395987886\\n8\\n energy: -7.334370608860 xtb: 6.7.0 (08769fc)\\nC           -0.01483776505897        0.04602732171118       -0.00206499562776\\nH            0.70491574024170       -0.12557764164807        0.79281428834603\\nH            0.50523069256964        0.52160818925945       -0.83174385472312\\nH           -0.76767802340165        0.75017840169136        0.34373697851331\\nC           -0.65890388565726       -1.26056511174085       -0.45762583004888\\nH           -1.64801310930718       -1.08892735410568       -0.87178748172606\\nH           -0.04967341913363       -1.73628347267730       -1.22414109839131\\nH           -0.73446623025266       -1.96461533249008        0.36746799365783\\n8\\n energy: -7.334868810470 xtb: 6.7.0 (08769fc)\\nC           -0.01488100547883        0.04561406302855       -0.00290709714506\\nH            0.73290909173915       -0.13169968835498        0.76460574737350\\nH            0.47135558570037        0.54024478322125       -0.84195401032299\\nH           -0.76451643409568        0.73345133580647        0.38104307957524\\nC           -0.65809537408304       -1.26015194667613       -0.45786539339685\\nH           -1.63072995187619       -1.08281081955591       -0.90757448289934\\nH           -0.02877176366408       -1.75491848684234       -1.19559377203487\\nH           -0.77069614824170       -1.94788424062690        0.37690192885041\\n8\\n energy: -7.335300867591 xtb: 6.7.0 (08769fc)\\nC           -0.01491809649035        0.04538279204643       -0.00398234844627\\nH            0.75749460793057       -0.13637170257690        0.73797590502995\\nH            0.44025033281891        0.55732187060467       -0.85000604220355\\nH           -0.76031795360375        0.71728321765113        0.41513203110964\\nC           -0.65706906729126       -1.25992079614765       -0.45818833669657\\nH           -1.61380131260832       -1.07814385982131       -0.93962232968073\\nH           -0.01082698563325       -1.77199397193557       -1.16894022396610\\nH           -0.80423752512256       -1.93171254982079        0.38428734485368\\n8\\n energy: -7.335661983757 xtb: 6.7.0 (08769fc)\\nC           -0.01496895360118        0.04531725667281       -0.00520327943818\\nH            0.77957431765216       -0.13985327970186        0.71248352494250\\nH            0.41113811443126        0.57328977091159       -0.85638837062455\\nH           -0.75518378989139        0.70154454644607        0.44684890580821\\nC           -0.65590081948281       -1.25985539623242       -0.45854676939793\\nH           -1.59711132912385       -1.07466700318035       -0.96892851137522\\nH            0.00488001078943       -1.78796023989581       -1.14361044274370\\nH           -0.83585355077363       -1.91597065502002        0.39000094282891\\n8\\n energy: -7.335954846301 xtb: 6.7.0 (08769fc)\\nC           -0.01497891801514        0.04541495708445       -0.00663663197360\\nH            0.79995297362151       -0.14218532569267        0.68773780448615\\nH            0.38341139106151        0.58851408088727       -0.86157591074932\\nH           -0.74920363886194        0.68569266265241        0.47724092207436\\nC           -0.65454588629301       -1.25995326043259       -0.45901444690514\\nH           -1.58055933607838       -1.07233945346664       -0.99638197717504\\nH            0.01899924830952       -1.80318289320984       -1.11918956386223\\nH           -0.86650183374407       -1.90011576782240        0.39447580410485\\n8\\n energy: -7.336166662800 xtb: 6.7.0 (08769fc)\\nC           -0.01510530123195        0.04563302200698       -0.00795461046095\\nH            0.81869003683020       -0.14377011824897        0.66353409825918\\nH            0.35668575739824        0.60308701425401       -0.86543215102275\\nH           -0.74225867601175        0.67057226968306        0.50579124963167\\nC           -0.65326100889713       -1.26017146643634       -0.45933392834294\\nH           -1.56397211593877       -1.07075896157959       -1.02210695885378\\nH            0.03152993935796       -1.81775409832271       -1.09526965101363\\nH           -0.89573463150680       -1.88499266135643        0.39742795180323\\n8\\n energy: -7.336303823206 xtb: 6.7.0 (08769fc)\\nC           -0.01527195677429        0.04597397829801       -0.00926556991195\\nH            0.83630642895808       -0.14453856001477        0.63960527543678\\nH            0.33060868594646        0.61722981196137       -0.86826924213768\\nH           -0.73440950987311        0.65564353868773        0.53331767747249\\nC           -0.65196935552364       -1.26051255971330       -0.45961308333360\\nH           -1.54727116125075       -1.06999469803625       -1.04668358091224\\nH            0.04288375548893       -1.83189510359584       -1.07162215045080\\nH           -0.92430288697169       -1.87006140758695        0.39918667383703\\n8\\n energy: -7.336366443594 xtb: 6.7.0 (08769fc)\\nC           -0.01562281472304        0.04655294768509       -0.01066319174580\\nH            0.85304891961625       -0.14456858581688        0.61556229700597\\nH            0.30479299861509        0.63140072043421       -0.86990562622509\\nH           -0.72543364581184        0.64110778173836        0.56007755330641\\nC           -0.65053469040450       -1.26109166090785       -0.45974735657064\\nH           -1.53017174559149       -1.06996879235036       -1.07047404183968\\nH            0.05301832487057       -1.84606410289150       -1.04782147670949\\nH           -0.95252334657106       -1.85552330789105        0.39962784277836\\n8\\n energy: -7.336353939675 xtb: 6.7.0 (08769fc)\\nC           -0.01589502120179        0.04710649550386       -0.01185114532222\\nH            0.86920773941076       -0.14389493517341        0.59137180976461\\nH            0.27899978414649        0.64511288134530       -0.87091704404611\\nH           -0.71559095721725        0.62634597557127        0.58642217890628\\nC           -0.64932391399802       -1.26164532332678       -0.45988601265630\\nH           -1.51273901746817       -1.07064653291279       -1.09376312904040\\nH            0.06255611958001       -1.85977428447561       -1.02383402917088\\nH           -0.98064073325204       -1.84075927653182        0.39911337156506\\n8\\n energy: -7.336266809393 xtb: 6.7.0 (08769fc)\\nC           -0.01620900467761        0.04774742758868       -0.01293383316086\\nH            0.88495140247493       -0.14252126262729        0.56669024515876\\nH            0.25290207131730        0.65879300120733       -0.87100904252334\\nH           -0.70460536865499        0.61155213262617        0.61250302118977\\nC           -0.64819850325058       -1.26228635436936       -0.45995022742749\\nH           -1.49470508587387       -1.07202431797569       -1.11682410356472\\nH            0.07132827266083       -1.87345229484362       -0.99925342439148\\nH           -1.00888978399601       -1.82596333160622        0.39743336471939\\n8\\n energy: -7.336105083980 xtb: 6.7.0 (08769fc)\\nC           -0.01655963417672        0.04846305091864       -0.01388104961115\\nH            0.90045183560600       -0.14045216380144        0.54120040412442\\nH            0.22617055693192        0.67256735176010       -0.87013721320323\\nH           -0.69233817064522        0.59661110119796        0.63855308835995\\nC           -0.64718864886416       -1.26300205760411       -0.45993478420209\\nH           -1.47582806273165       -1.07409760224111       -1.13992466843823\\nH            0.07940250489490       -1.88722437403621       -0.97375434073245\\nH           -1.03753638101507       -1.81102030619383        0.39453456370281\\n8\\n energy: -7.335870825005 xtb: 6.7.0 (08769fc)\\nC           -0.01673165412177        0.04909391387518       -0.01452644287555\\nH            0.91597030599750       -0.13762543129367        0.51464030924320\\nH            0.19839155552793        0.68633670216491       -0.86842230357956\\nH           -0.67874737052128        0.58104945371876        0.66486297863435\\nC           -0.64652285023264       -1.26363298062722       -0.45998731678988\\nH           -1.45594787611020       -1.07692864457344       -1.16339838454072\\nH            0.08703038031700       -1.90099127107875       -0.94698688056312\\nH           -1.06686849085655       -1.79545674218577        0.39047404047132\\n8\\n energy: -7.335566972652 xtb: 6.7.0 (08769fc)\\nC           -0.01709713365699        0.04988006073338       -0.01499197124393\\nH            0.93137256481723       -0.13427136365585        0.48630797066450\\nH            0.16924694963837        0.70076336578723       -0.86507419877558\\nH           -0.66340165406505        0.56573735331046        0.69118826091212\\nC           -0.64596228119778       -1.26441915114568       -0.45979753500036\\nH           -1.43435785902991       -1.08028721601292       -1.18735233858514\\nH            0.09357266530164       -1.91541518293395       -0.91838800370885\\nH           -1.09679925180751       -1.78014286608266        0.38476381573727\\n8\\n energy: -7.335184558881 xtb: 6.7.0 (08769fc)\\nC           -0.01739300225101        0.05068540252720       -0.01528686383551\\nH            0.94729498915940       -0.12983174511701        0.45559467804084\\nH            0.13787892220448        0.71584069509140       -0.86028840532344\\nH           -0.64582859893015        0.54927380797907        0.71870615583524\\nC           -0.64558578997857       -1.26522450279048       -0.45961659794477\\nH           -1.41069586944999       -1.08473165399809       -1.21258916431074\\nH            0.09949924004835       -1.93048941458123       -0.88721387860555\\nH           -1.12859589080251       -1.76367758911086        0.37735007614395\\n8\\n energy: -7.334730284445 xtb: 6.7.0 (08769fc)\\nC           -0.01759359453571        0.05146586486452       -0.01528185822493\\nH            0.96375436253678       -0.12433080371537        0.42172460447753\\nH            0.10368085787678        0.73177900702085       -0.85352223900268\\nH           -0.62564117577546        0.53168455273550        0.74739520570432\\nC           -0.64552380697006       -1.26600494840652       -0.45942568750457\\nH           -1.38423590234480       -1.09023782086456       -1.23938292220887\\nH            0.10450025583797       -1.94642417159395       -0.85271182397550\\nH           -1.16236699662550       -1.74608668004045        0.36786072073473\\n8\\n energy: -7.334208930714 xtb: 6.7.0 (08769fc)\\nC           -0.01761352981866        0.05215857545738       -0.01490514101658\\nH            0.98103537526972       -0.11748315593977        0.38337423155854\\nH            0.06548806736023        0.74893199827882       -0.84405443916131\\nH           -0.60203401152920        0.51238051811761        0.77779346508641\\nC           -0.64587246564767       -1.26669761331198       -0.45928142059821\\nH           -1.35382468451718       -1.09709127527845       -1.26844256444615\\nH            0.10828321095136       -1.96357297405934       -0.81354345164497\\nH           -1.19888796206861       -1.72678107326427        0.35571532022231\\n8\\n energy: -7.333633985662 xtb: 6.7.0 (08769fc)\\nC           -0.01732901279488        0.05266935226590       -0.01408788221282\\nH            0.99938908884578       -0.10881583524654        0.33869185507839\\nH            0.02173955671704        0.76769629810544       -0.83077131353172\\nH           -0.57383110256239        0.49047733023543        0.81056295519113\\nC           -0.64673786639711       -1.26720832006496       -0.45927763572244\\nH           -1.31779969830770       -1.10576521659954       -1.30062111468472\\nH            0.11031745129859       -1.98233219407193       -0.76786603800340\\nH           -1.23917441679935       -1.70487641462380        0.34002517388561\\n8\\n energy: -7.333041355774 xtb: 6.7.0 (08769fc)\\nC           -0.01694681801821        0.05314002955908       -0.01314735717719\\nH            1.01800408679644       -0.09772544167127        0.28697047039510\\nH           -0.02781391274292        0.78771608219226       -0.81321057501075\\nH           -0.53993869152348        0.46510050153298        0.84614284002398\\nC           -0.64775201407438       -1.26767892071223       -0.45932493435141\\nH           -1.27522422194942       -1.11686306013383       -1.33538867640327\\nH            0.11025003619270       -2.00234593738614       -0.71529072937391\\nH           -1.28400446468074       -1.67949825338085        0.31990496189749\\n8\\n energy: -7.332525545051 xtb: 6.7.0 (08769fc)\\nC           -0.01560575238942        0.05280430001444       -0.01139628936166\\nH            1.03780565185171       -0.08328987839661        0.22232796145885\\nH           -0.08812486255365        0.81009142583205       -0.78702384101393\\nH           -0.49761741123636        0.43420111775897        0.88507777046751\\nC           -0.64984957930445       -1.26734309912083       -0.46000673660391\\nH           -1.22085958357940       -1.13130765328402       -1.37557550507949\\nH            0.10562885949919       -2.02471337607882       -0.64969992141033\\nH           -1.33480332228763       -1.64859783672518        0.29295256154301\\n8\\n energy: -7.332248232936 xtb: 6.7.0 (08769fc)\\nC           -0.01348143912452        0.05159804133227       -0.00942278928581\\nH            1.05614464963472       -0.06479139629042        0.14483427714534\\nH           -0.15803952959553        0.83304421543574       -0.75099067498168\\nH           -0.44616336615980        0.39681345374662        0.92598143424194\\nC           -0.65241752351610       -1.26613678565634       -0.46135319230836\\nH           -1.15389004792523       -1.14981652184106       -1.41866015278368\\nH            0.09491918982160       -2.04765634937703       -0.57177557761922\\nH           -1.39049793313514       -1.61120965734977        0.25804267559150\\n8\\n energy: -7.332374812909 xtb: 6.7.0 (08769fc)\\nC           -0.01109941953477        0.04969797890894       -0.00798390831456\\nH            1.06903264400914       -0.04339665876944        0.06224433074316\\nH           -0.22994349180060        0.85295648008947       -0.70784063835151\\nH           -0.38968211406109        0.35555218443365        0.96387459143651\\nC           -0.65456705227742       -1.26423675209357       -0.46312068409060\\nH           -1.08030072446610       -1.17122178802880       -1.45830054237967\\nH            0.07816082012914       -2.06755781977157       -0.48960689654621\\nH           -1.44502666199830       -1.56994862476868        0.21738974750292\\n8\\n energy: -7.332839884916 xtb: 6.7.0 (08769fc)\\nC           -0.00936352571414        0.04787033867706       -0.00754214286004\\nH            1.07447926012723       -0.02256682172088       -0.01121968391134\\nH           -0.29198085235338        0.86693586169780       -0.66576819279132\\nH           -0.33686015644424        0.31697439029419        0.99338661585108\\nC           -0.65556126452556       -1.26240920236499       -0.46461077009852\\nH           -1.01284010761510       -1.19206050121825       -1.48788638883727\\nH            0.05913502446414       -2.08152733178496       -0.41710158795981\\nH           -1.49043437793895       -1.53137173357997        0.17739815060725\\n8\\n energy: -7.333427301329 xtb: 6.7.0 (08769fc)\\nC           -0.00848625687541        0.04652598869296       -0.00775169289659\\nH            1.07459251501131       -0.00426420082418       -0.07065349897014\\nH           -0.34105245510122        0.87591260183827       -0.62921720900214\\nH           -0.29154350686092        0.28437219311608        1.01452375207454\\nC           -0.65565554938589       -1.26106494749161       -0.46550792048500\\nH           -0.95683431390588       -1.21036995494035       -1.50777337454571\\nH            0.04100067295007       -2.09049588902062       -0.35866111607469\\nH           -1.52544710583207       -1.49877079137055        0.14169705989978\\n8\\n energy: -7.333999952458 xtb: 6.7.0 (08769fc)\\nC           -0.00784291355509        0.04525110391915       -0.00810474447436\\nH            1.07173905313602        0.01190338570296       -0.12047789805816\\nH           -0.38113768236822        0.88168860617839       -0.59643850711242\\nH           -0.25240077114066        0.25662694146060        1.02969265520118\\nC           -0.65553665935867       -1.25979015454824       -0.46623224013937\\nH           -0.90890238802059       -1.22654302980070       -1.52166459360721\\nH            0.02343269691844       -2.09626487440290       -0.30995058285541\\nH           -1.55277733561123       -1.47102697850925        0.10983191104579\\n8\\n energy: -7.334542794917 xtb: 6.7.0 (08769fc)\\nC           -0.00810393866175        0.04477578744739       -0.00874282998812\\nH            1.06700537621892        0.02662222750448       -0.16177163516435\\nH           -0.41420655191838        0.88591995011412       -0.56809993017208\\nH           -0.21693095533209        0.23263365352007        1.04145130765435\\nC           -0.65484803600645       -1.25931489073012       -0.46619853681292\\nH           -0.86838877780603       -1.24126622771418       -1.53094357768712\\nH            0.00771643588552       -2.10049027818000       -0.26933439826855\\nH           -1.57566955237973       -1.44703522196175        0.08029560043883\\n8\\n energy: -7.335020003605 xtb: 6.7.0 (08769fc)\\nC           -0.00869517267906        0.04458971741204       -0.00935604852847\\nH            1.06114904692973        0.04020498115585       -0.19764292494630\\nH           -0.44263997460867        0.88895933603196       -0.54232137517113\\nH           -0.18431077240003        0.21148257102162        1.05055816132673\\nC           -0.65407297813044       -1.25912884329944       -0.46584514145358\\nH           -0.83261467157163       -1.25485262429481       -1.53735921403699\\nH           -0.00712856340641       -2.10352439525774       -0.23394158325035\\nH           -1.59511291413349       -1.42588574276946        0.05256412606013\\n8\\n energy: -7.335429032595 xtb: 6.7.0 (08769fc)\\nC           -0.00949200415392        0.04458185452571       -0.00988653149493\\nH            1.05443220358377        0.05316243930073       -0.23010932334550\\nH           -0.46807990134817        0.89121586840042       -0.51790901773671\\nH           -0.15330648354944        0.19227974306579        1.05776390058951\\nC           -0.65330751955606       -1.25912097680242       -0.46527031491977\\nH           -0.79976484508873       -1.26781326360258       -1.54183016938302\\nH           -0.02168157327698       -2.10577605799609       -0.20182640026915\\nH           -1.61222587661048       -1.40668460689155        0.02572385655960\\n8\\n energy: -7.335765977031 xtb: 6.7.0 (08769fc)\\nC           -0.01044473947794        0.04472107828701       -0.01030937188905\\nH            1.04700788587754        0.06570132816931       -0.26001590243518\\nH           -0.49122753620123        0.89291000770091       -0.49440965792894\\nH           -0.12340128417901        0.17465558997225        1.06344846083586\\nC           -0.65259168663516       -1.25926017165631       -0.46451262783813\\nH           -0.76909336649393       -1.28035498290308       -1.54478201313289\\nH           -0.03613657678602       -2.10746561924643       -0.17217675833079\\nH           -1.62753869610425       -1.38906223032365       -0.00058612928085\\n8\\n energy: -7.336033371123 xtb: 6.7.0 (08769fc)\\nC           -0.01160401777332        0.04500322917170       -0.01054182797584\\nH            1.03892810080300        0.07802467738361       -0.28808734250263\\nH           -0.51284376632744        0.89401867973043       -0.47152738647103\\nH           -0.09366807530157        0.15821390029147        1.06807645125007\\nC           -0.65198664715742       -1.25954225503674       -0.46349678948062\\nH           -0.73993420258211       -1.29268089835303       -1.54650496980681\\nH           -0.05051935155432       -2.10856990847238       -0.14417666481783\\nH           -1.64179804010681       -1.37262242471505       -0.02708547019526\\n8\\n energy: -7.336220489565 xtb: 6.7.0 (08769fc)\\nC           -0.01275042235836        0.04536052815074       -0.01071317846678\\nH            1.03026093974334        0.09020435916553       -0.31459419336675\\nH           -0.53276720719915        0.89496959965149       -0.44880022196533\\nH           -0.06489467267136        0.14297756993930        1.07134589419674\\nC           -0.65144352001297       -1.25989948056670       -0.46247274389421\\nH           -0.71205489988900       -1.30486291939043       -1.54715329201825\\nH           -0.06531928822111       -2.10951660061528       -0.11782448456225\\nH           -1.65445692939139       -1.35738805633465       -0.05313177992313\\n8\\n energy: -7.336332889035 xtb: 6.7.0 (08769fc)\\nC           -0.01394767817782        0.04579981612725       -0.01074884014815\\nH            1.02099648240032        0.10241002675018       -0.33998763051367\\nH           -0.55153548982104        0.89563760471522       -0.42606927186521\\nH           -0.03627707135032        0.12864620209169        1.07361086487891\\nC           -0.65101142591967       -1.26033867537468       -0.46135557346742\\nH           -0.68502672017866       -1.31707074975467       -1.54686777703050\\nH           -0.08050726102461       -2.11018047153222       -0.09256036942919\\nH           -1.66611683592820       -1.34305875302277       -0.07936540242473\\n8\\n energy: -7.336370557561 xtb: 6.7.0 (08769fc)\\nC           -0.01531752180721        0.04644713871308       -0.01080659584368\\nH            1.01109997257679        0.11500870910621       -0.36465349783145\\nH           -0.56926527789264        0.89644722976473       -0.40267794603379\\nH           -0.00766850896699        0.11502582276095        1.07466891472725\\nC           -0.65050106899959       -1.26098589332565       -0.46008298603301\\nH           -0.65847428117183       -1.32967146035618       -1.54574403437608\\nH           -0.09666362561468       -2.11098597110248       -0.06805571634325\\nH           -1.67663568812384       -1.32944057556066       -0.10599213826596\\n\",\"xyz\");\n",
              "\tviewer_17338420579899318.setStyle({\"stick\": {\"colorscheme\": \"greenCarbon\"}});\n",
              "\tviewer_17338420579899318.zoomTo();\n",
              "\tviewer_17338420579899318.animate({\"loop\": \"backAndForth\"});\n",
              "viewer_17338420579899318.render();\n",
              "});\n",
              "</script>"
            ]
          },
          "metadata": {}
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "---------------"
      ],
      "metadata": {
        "id": "1qIwNANWSjz8"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "def visualizar_molecula(nome_arquivo):\n",
        "    import py3Dmol\n",
        "    import os\n",
        "    # Verifica se o arquivo existe\n",
        "    if not os.path.exists(nome_arquivo):\n",
        "        print(f\"O arquivo '{nome_arquivo}' n√£o existe.\")\n",
        "        return\n",
        "    # L√™ o conte√∫do do arquivo XYZ\n",
        "    with open(nome_arquivo, 'r') as file:\n",
        "        xyz_content = file.read()\n",
        "\n",
        "    # Cria a visualiza√ß√£o\n",
        "    view = py3Dmol.view(width=800, height=600)\n",
        "    view.addModel(xyz_content, 'xyz')\n",
        "\n",
        "    # Adiciona a visualiza√ß√£o\n",
        "    view.setStyle({'stick': {}})\n",
        "\n",
        "    # Adiciona r√≥tulos com a numera√ß√£o dos √°tomos\n",
        "    for i, line in enumerate(xyz_content.splitlines()[2:]):\n",
        "        atom_num = i + 1\n",
        "        atom = line.split()[0]\n",
        "        x, y, z = map(float, line.split()[1:])\n",
        "        label_text = f'{atom_num} ({atom})'\n",
        "        view.addLabel(label_text, {'position': {'x': x, 'y': y, 'z': z},\n",
        "                                   'backgroundColor': 'white',\n",
        "                                   'backgroundOpacity': 0.8,\n",
        "                                   'fontColor': 'black'})\n",
        "\n",
        "    # Renderiza a visualiza√ß√£o\n",
        "    view.zoomTo()\n",
        "    view.show()\n",
        "\n",
        "# Exemplo de uso:\n",
        "# visualizar_molecula('caminho/para/seu_arquivo.xyz')"
      ],
      "metadata": {
        "id": "_mwWaOrJ2-B_"
      },
      "execution_count": 52,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "%%file water.xyz\n",
        "3\n",
        "Water molecule\n",
        "O       0.000000       0.000000      -0.065000\n",
        "H       0.750000       0.750000       0.520000\n",
        "H      -0.750000       0.750000       0.520000"
      ],
      "metadata": {
        "id": "o69bZpiS2_FC",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "91b6fae4-8a48-4b15-804d-ad767704d0d4"
      },
      "execution_count": 53,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Writing water.xyz\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "visualizar_molecula('water.xyz')"
      ],
      "metadata": {
        "id": "IbXehMKf3bIM",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 617
        },
        "outputId": "12a05991-1cbe-4221-9e10-094faa8b31d4"
      },
      "execution_count": 54,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_17338420758517187\"  style=\"position: relative; width: 800px; height: 600px;\">\n        <p id=\"3dmolwarning_17338420758517187\" style=\"background-color:#ffcccc;color:black\">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    //this is to ignore the existence of requirejs amd\n    var savedexports, savedmodule;\n    if (typeof exports !== 'undefined') savedexports = exports;\n    else exports = {}\n    if (typeof module !== 'undefined') savedmodule = module;\n    else module = {}\n\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n        exports = savedexports;\n        module = savedmodule;\n        resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.4.2/3Dmol-min.js');\n}\n\nvar viewer_17338420758517187 = null;\nvar warn = document.getElementById(\"3dmolwarning_17338420758517187\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_17338420758517187 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_17338420758517187\"),{backgroundColor:\"white\"});\nviewer_17338420758517187.zoomTo();\n\tviewer_17338420758517187.addModel(\"3\\nWater molecule\\nO       0.000000       0.000000      -0.065000\\nH       0.750000       0.750000       0.520000\\nH      -0.750000       0.750000       0.520000\\n\",\"xyz\");\n\tviewer_17338420758517187.setStyle({\"stick\": {}});\n\tviewer_17338420758517187.addLabel(\"1 (O)\",{\"position\": {\"x\": 0.0, \"y\": 0.0, \"z\": -0.065}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17338420758517187.addLabel(\"2 (H)\",{\"position\": {\"x\": 0.75, \"y\": 0.75, \"z\": 0.52}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17338420758517187.addLabel(\"3 (H)\",{\"position\": {\"x\": -0.75, \"y\": 0.75, \"z\": 0.52}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17338420758517187.zoomTo();\nviewer_17338420758517187.render();\n});\n</script>",
            "text/html": [
              "<div id=\"3dmolviewer_17338420758517187\"  style=\"position: relative; width: 800px; height: 600px;\">\n",
              "        <p id=\"3dmolwarning_17338420758517187\" style=\"background-color:#ffcccc;color:black\">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br></p>\n",
              "        </div>\n",
              "<script>\n",
              "\n",
              "var loadScriptAsync = function(uri){\n",
              "  return new Promise((resolve, reject) => {\n",
              "    //this is to ignore the existence of requirejs amd\n",
              "    var savedexports, savedmodule;\n",
              "    if (typeof exports !== 'undefined') savedexports = exports;\n",
              "    else exports = {}\n",
              "    if (typeof module !== 'undefined') savedmodule = module;\n",
              "    else module = {}\n",
              "\n",
              "    var tag = document.createElement('script');\n",
              "    tag.src = uri;\n",
              "    tag.async = true;\n",
              "    tag.onload = () => {\n",
              "        exports = savedexports;\n",
              "        module = savedmodule;\n",
              "        resolve();\n",
              "    };\n",
              "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
              "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
              "});\n",
              "};\n",
              "\n",
              "if(typeof $3Dmolpromise === 'undefined') {\n",
              "$3Dmolpromise = null;\n",
              "  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.4.2/3Dmol-min.js');\n",
              "}\n",
              "\n",
              "var viewer_17338420758517187 = null;\n",
              "var warn = document.getElementById(\"3dmolwarning_17338420758517187\");\n",
              "if(warn) {\n",
              "    warn.parentNode.removeChild(warn);\n",
              "}\n",
              "$3Dmolpromise.then(function() {\n",
              "viewer_17338420758517187 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_17338420758517187\"),{backgroundColor:\"white\"});\n",
              "viewer_17338420758517187.zoomTo();\n",
              "\tviewer_17338420758517187.addModel(\"3\\nWater molecule\\nO       0.000000       0.000000      -0.065000\\nH       0.750000       0.750000       0.520000\\nH      -0.750000       0.750000       0.520000\\n\",\"xyz\");\n",
              "\tviewer_17338420758517187.setStyle({\"stick\": {}});\n",
              "\tviewer_17338420758517187.addLabel(\"1 (O)\",{\"position\": {\"x\": 0.0, \"y\": 0.0, \"z\": -0.065}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17338420758517187.addLabel(\"2 (H)\",{\"position\": {\"x\": 0.75, \"y\": 0.75, \"z\": 0.52}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17338420758517187.addLabel(\"3 (H)\",{\"position\": {\"x\": -0.75, \"y\": 0.75, \"z\": 0.52}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17338420758517187.zoomTo();\n",
              "viewer_17338420758517187.render();\n",
              "});\n",
              "</script>"
            ]
          },
          "metadata": {}
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "# Exemplo de uma mol√©cula flex√≠vel: a mol√©cula de omeprazole\n",
        "# Sugest√£o: Ir no site do pubchem (https://pubchem.ncbi.nlm.nih.gov/)\n",
        "#           e consultar o c√≥digo smiles da mol√©cula da sua prefer√™ncia\n",
        "smiles = 'CC1=CN=C(C(=C1OC)C)CS(=O)C2=NC3=C(N2)C=C(C=C3)OC'\n",
        "mol = Chem.MolFromSmiles(smiles)\n",
        "mol"
      ],
      "metadata": {
        "id": "qke7S35X5YQv",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 193
        },
        "outputId": "56ba961c-02fb-413d-905b-955f9616517e"
      },
      "execution_count": 55,
      "outputs": [
        {
          "output_type": "error",
          "ename": "NameError",
          "evalue": "name 'Chem' is not defined",
          "traceback": [
            "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
            "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
            "\u001b[0;32m<ipython-input-55-53b96ed5af5a>\u001b[0m in \u001b[0;36m<cell line: 5>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      3\u001b[0m \u001b[0;31m#           e consultar o c√≥digo smiles da mol√©cula da sua prefer√™ncia\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      4\u001b[0m \u001b[0msmiles\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m'CC1=CN=C(C(=C1OC)C)CS(=O)C2=NC3=C(N2)C=C(C=C3)OC'\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 5\u001b[0;31m \u001b[0mmol\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mChem\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mMolFromSmiles\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0msmiles\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      6\u001b[0m \u001b[0mmol\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;31mNameError\u001b[0m: name 'Chem' is not defined"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "# Adicionar hidrog√™nios expl√≠citos √† mol√©cula\n",
        "mol_H = Chem.AddHs(mol)\n",
        "AllChem.EmbedMolecule(mol_H)\n",
        "AllChem.MMFFOptimizeMolecule(mol_H) # Otimizando a geometria com o campo de for√ßa MMFF\n",
        "print(Chem.MolToXYZBlock(mol_H))"
      ],
      "metadata": {
        "id": "F12PTCLC5fOs",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "f9b24b87-cdf7-43bd-dd89-26a05236afe7"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "43\n",
            "\n",
            "C     -6.484587   -0.988147    0.796649\n",
            "C     -5.018221   -0.866479    0.527034\n",
            "C     -4.100462   -1.538540    1.314277\n",
            "N     -2.763308   -1.474875    1.142300\n",
            "C     -2.277219   -0.711780    0.127621\n",
            "C     -3.117850   -0.004356   -0.744569\n",
            "C     -4.497709   -0.054003   -0.482347\n",
            "O     -5.367185    0.651034   -1.280454\n",
            "C     -5.583426    1.957923   -0.742702\n",
            "C     -2.588054    0.812281   -1.889826\n",
            "C     -0.773475   -0.698977    0.008297\n",
            "S     -0.056215    0.889798    0.529058\n",
            "O     -0.033413    0.969710    2.029893\n",
            "C      1.638664    0.502603    0.062647\n",
            "N      2.027199    0.127055   -1.135568\n",
            "C      3.385271   -0.054625   -1.025210\n",
            "C      3.824989    0.216120    0.267096\n",
            "N      2.685666    0.572245    0.939742\n",
            "C      5.162440    0.109710    0.667212\n",
            "C      6.083589   -0.291628   -0.309455\n",
            "C      5.661783   -0.567245   -1.616144\n",
            "C      4.315896   -0.454305   -1.991938\n",
            "O      7.430289   -0.451835   -0.110829\n",
            "C      7.924617   -0.197015    1.197822\n",
            "H     -7.056852   -1.001274   -0.136600\n",
            "H     -6.826133   -0.152944    1.415470\n",
            "H     -6.711824   -1.922140    1.322016\n",
            "H     -4.418534   -2.172890    2.137676\n",
            "H     -6.282835    2.478414   -1.403665\n",
            "H     -4.653922    2.536049   -0.711426\n",
            "H     -6.032055    1.911287    0.255055\n",
            "H     -3.332233    0.920245   -2.685796\n",
            "H     -2.305994    1.810843   -1.543017\n",
            "H     -1.721488    0.336557   -2.357582\n",
            "H     -0.340596   -1.481735    0.642252\n",
            "H     -0.481181   -0.920990   -1.022513\n",
            "H      2.598931    0.829216    1.913668\n",
            "H      5.446244    0.332403    1.687590\n",
            "H      6.393975   -0.877315   -2.358952\n",
            "H      3.997815   -0.670612   -3.005486\n",
            "H      9.004564   -0.373158    1.187352\n",
            "H      7.760383    0.846709    1.485344\n",
            "H      7.482459   -0.883332    1.927551\n",
            "\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "%%file ome.xyz\n",
        "43\n",
        "\n",
        "C     -6.474593   -1.386555   -0.085378\n",
        "C     -5.010672   -1.114409    0.059177\n",
        "C     -4.153592   -2.102349    0.510946\n",
        "N     -2.824337   -1.936193    0.674526\n",
        "C     -2.279859   -0.735437    0.347187\n",
        "C     -3.055068    0.338660   -0.115204\n",
        "C     -4.442281    0.134371   -0.202095\n",
        "O     -5.257125    1.157798   -0.623738\n",
        "C     -5.712181    1.922362    0.495209\n",
        "C     -2.446673    1.671177   -0.452735\n",
        "C     -0.784509   -0.639865    0.520217\n",
        "S      0.093970   -0.967575   -1.039405\n",
        "O      0.310485   -2.447112   -1.197267\n",
        "C      1.692332   -0.331621   -0.506293\n",
        "N      1.906874    0.869064   -0.015498\n",
        "C      3.256240    0.908021    0.244012\n",
        "C      3.868305   -0.294656   -0.095925\n",
        "N      2.843371   -1.067076   -0.575541\n",
        "C      5.237506   -0.538459    0.065366\n",
        "C      6.005992    0.504755    0.598781\n",
        "C      5.409604    1.723367    0.946098\n",
        "C      4.035473    1.942725    0.775645\n",
        "O      7.356184    0.456793    0.829870\n",
        "C      8.024829   -0.754065    0.500177\n",
        "H     -7.010977   -1.056879    0.809473\n",
        "H     -6.663638   -2.456287   -0.226535\n",
        "H     -6.886116   -0.873089   -0.960372\n",
        "H     -4.517237   -3.095233    0.763116\n",
        "H     -6.303973    1.310527    1.183965\n",
        "H     -6.355958    2.720429    0.113650\n",
        "H     -4.876013    2.387804    1.027737\n",
        "H     -1.531667    1.553640   -1.039730\n",
        "H     -2.216215    2.219709    0.465915\n",
        "H     -3.115661    2.287728   -1.061147\n",
        "H     -0.519142    0.350129    0.903819\n",
        "H     -0.442738   -1.373880    1.258977\n",
        "H      2.892933   -2.018989   -0.912600\n",
        "H      5.658359   -1.494970   -0.216269\n",
        "H      6.025270    2.520225    1.358373\n",
        "H      3.582018    2.889108    1.047457\n",
        "H      7.952448   -0.965680   -0.571736\n",
        "H      7.643832   -1.590198    1.095783\n",
        "H      9.084201   -0.627816    0.743519"
      ],
      "metadata": {
        "id": "GCylf42U6k6T",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "6d22ea00-1c33-4f48-efde-456ee47c3673"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Writing ome.xyz\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "visualizar_molecula('ome.xyz')"
      ],
      "metadata": {
        "id": "RB_BjAG-6snc",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 637
        },
        "outputId": "3757e858-c595-4fb6-d454-fb0247a87060"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_17200163316965322\"  style=\"position: relative; width: 800px; height: 600px;\">\n        <p id=\"3dmolwarning_17200163316965322\" style=\"background-color:#ffcccc;color:black\">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    //this is to ignore the existence of requirejs amd\n    var savedexports, savedmodule;\n    if (typeof exports !== 'undefined') savedexports = exports;\n    else exports = {}\n    if (typeof module !== 'undefined') savedmodule = module;\n    else module = {}\n\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n        exports = savedexports;\n        module = savedmodule;\n        resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.1.0/3Dmol-min.js');\n}\n\nvar viewer_17200163316965322 = null;\nvar warn = document.getElementById(\"3dmolwarning_17200163316965322\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_17200163316965322 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_17200163316965322\"),{backgroundColor:\"white\"});\nviewer_17200163316965322.zoomTo();\n\tviewer_17200163316965322.addModel(\"43\\n\\nC     -6.474593   -1.386555   -0.085378\\nC     -5.010672   -1.114409    0.059177\\nC     -4.153592   -2.102349    0.510946\\nN     -2.824337   -1.936193    0.674526\\nC     -2.279859   -0.735437    0.347187\\nC     -3.055068    0.338660   -0.115204\\nC     -4.442281    0.134371   -0.202095\\nO     -5.257125    1.157798   -0.623738\\nC     -5.712181    1.922362    0.495209\\nC     -2.446673    1.671177   -0.452735\\nC     -0.784509   -0.639865    0.520217\\nS      0.093970   -0.967575   -1.039405\\nO      0.310485   -2.447112   -1.197267\\nC      1.692332   -0.331621   -0.506293\\nN      1.906874    0.869064   -0.015498\\nC      3.256240    0.908021    0.244012\\nC      3.868305   -0.294656   -0.095925\\nN      2.843371   -1.067076   -0.575541\\nC      5.237506   -0.538459    0.065366\\nC      6.005992    0.504755    0.598781\\nC      5.409604    1.723367    0.946098\\nC      4.035473    1.942725    0.775645\\nO      7.356184    0.456793    0.829870\\nC      8.024829   -0.754065    0.500177\\nH     -7.010977   -1.056879    0.809473\\nH     -6.663638   -2.456287   -0.226535\\nH     -6.886116   -0.873089   -0.960372\\nH     -4.517237   -3.095233    0.763116\\nH     -6.303973    1.310527    1.183965\\nH     -6.355958    2.720429    0.113650\\nH     -4.876013    2.387804    1.027737\\nH     -1.531667    1.553640   -1.039730\\nH     -2.216215    2.219709    0.465915\\nH     -3.115661    2.287728   -1.061147\\nH     -0.519142    0.350129    0.903819\\nH     -0.442738   -1.373880    1.258977\\nH      2.892933   -2.018989   -0.912600\\nH      5.658359   -1.494970   -0.216269\\nH      6.025270    2.520225    1.358373\\nH      3.582018    2.889108    1.047457\\nH      7.952448   -0.965680   -0.571736\\nH      7.643832   -1.590198    1.095783\\nH      9.084201   -0.627816    0.743519\\n\",\"xyz\");\n\tviewer_17200163316965322.setStyle({\"stick\": {}});\n\tviewer_17200163316965322.addLabel(\"1 (C)\",{\"position\": {\"x\": -6.474593, \"y\": -1.386555, \"z\": -0.085378}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17200163316965322.addLabel(\"2 (C)\",{\"position\": {\"x\": -5.010672, \"y\": -1.114409, \"z\": 0.059177}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17200163316965322.addLabel(\"3 (C)\",{\"position\": {\"x\": -4.153592, \"y\": -2.102349, \"z\": 0.510946}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17200163316965322.addLabel(\"4 (N)\",{\"position\": {\"x\": -2.824337, \"y\": -1.936193, \"z\": 0.674526}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17200163316965322.addLabel(\"5 (C)\",{\"position\": {\"x\": -2.279859, \"y\": -0.735437, \"z\": 0.347187}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17200163316965322.addLabel(\"6 (C)\",{\"position\": {\"x\": -3.055068, \"y\": 0.33866, \"z\": -0.115204}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17200163316965322.addLabel(\"7 (C)\",{\"position\": {\"x\": -4.442281, \"y\": 0.134371, \"z\": -0.202095}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17200163316965322.addLabel(\"8 (O)\",{\"position\": {\"x\": -5.257125, \"y\": 1.157798, \"z\": -0.623738}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17200163316965322.addLabel(\"9 (C)\",{\"position\": {\"x\": -5.712181, \"y\": 1.922362, \"z\": 0.495209}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17200163316965322.addLabel(\"10 (C)\",{\"position\": {\"x\": -2.446673, \"y\": 1.671177, \"z\": -0.452735}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17200163316965322.addLabel(\"11 (C)\",{\"position\": {\"x\": -0.784509, \"y\": -0.639865, \"z\": 0.520217}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17200163316965322.addLabel(\"12 (S)\",{\"position\": {\"x\": 0.09397, \"y\": -0.967575, \"z\": -1.039405}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17200163316965322.addLabel(\"13 (O)\",{\"position\": {\"x\": 0.310485, \"y\": -2.447112, \"z\": -1.197267}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17200163316965322.addLabel(\"14 (C)\",{\"position\": {\"x\": 1.692332, \"y\": -0.331621, \"z\": -0.506293}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17200163316965322.addLabel(\"15 (N)\",{\"position\": {\"x\": 1.906874, \"y\": 0.869064, \"z\": -0.015498}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17200163316965322.addLabel(\"16 (C)\",{\"position\": {\"x\": 3.25624, \"y\": 0.908021, \"z\": 0.244012}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17200163316965322.addLabel(\"17 (C)\",{\"position\": {\"x\": 3.868305, \"y\": -0.294656, \"z\": -0.095925}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17200163316965322.addLabel(\"18 (N)\",{\"position\": {\"x\": 2.843371, \"y\": -1.067076, \"z\": -0.575541}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17200163316965322.addLabel(\"19 (C)\",{\"position\": {\"x\": 5.237506, \"y\": -0.538459, \"z\": 0.065366}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17200163316965322.addLabel(\"20 (C)\",{\"position\": {\"x\": 6.005992, \"y\": 0.504755, \"z\": 0.598781}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17200163316965322.addLabel(\"21 (C)\",{\"position\": {\"x\": 5.409604, \"y\": 1.723367, \"z\": 0.946098}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17200163316965322.addLabel(\"22 (C)\",{\"position\": {\"x\": 4.035473, \"y\": 1.942725, \"z\": 0.775645}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17200163316965322.addLabel(\"23 (O)\",{\"position\": {\"x\": 7.356184, \"y\": 0.456793, \"z\": 0.82987}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17200163316965322.addLabel(\"24 (C)\",{\"position\": {\"x\": 8.024829, \"y\": -0.754065, \"z\": 0.500177}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17200163316965322.addLabel(\"25 (H)\",{\"position\": {\"x\": -7.010977, \"y\": -1.056879, \"z\": 0.809473}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17200163316965322.addLabel(\"26 (H)\",{\"position\": {\"x\": -6.663638, \"y\": -2.456287, \"z\": -0.226535}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17200163316965322.addLabel(\"27 (H)\",{\"position\": {\"x\": -6.886116, \"y\": -0.873089, \"z\": -0.960372}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17200163316965322.addLabel(\"28 (H)\",{\"position\": {\"x\": -4.517237, \"y\": -3.095233, \"z\": 0.763116}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17200163316965322.addLabel(\"29 (H)\",{\"position\": {\"x\": -6.303973, \"y\": 1.310527, \"z\": 1.183965}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17200163316965322.addLabel(\"30 (H)\",{\"position\": {\"x\": -6.355958, \"y\": 2.720429, \"z\": 0.11365}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17200163316965322.addLabel(\"31 (H)\",{\"position\": {\"x\": -4.876013, \"y\": 2.387804, \"z\": 1.027737}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17200163316965322.addLabel(\"32 (H)\",{\"position\": {\"x\": -1.531667, \"y\": 1.55364, \"z\": -1.03973}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17200163316965322.addLabel(\"33 (H)\",{\"position\": {\"x\": -2.216215, \"y\": 2.219709, \"z\": 0.465915}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17200163316965322.addLabel(\"34 (H)\",{\"position\": {\"x\": -3.115661, \"y\": 2.287728, \"z\": -1.061147}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17200163316965322.addLabel(\"35 (H)\",{\"position\": {\"x\": -0.519142, \"y\": 0.350129, \"z\": 0.903819}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17200163316965322.addLabel(\"36 (H)\",{\"position\": {\"x\": -0.442738, \"y\": -1.37388, \"z\": 1.258977}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17200163316965322.addLabel(\"37 (H)\",{\"position\": {\"x\": 2.892933, \"y\": -2.018989, \"z\": -0.9126}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17200163316965322.addLabel(\"38 (H)\",{\"position\": {\"x\": 5.658359, \"y\": -1.49497, \"z\": -0.216269}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17200163316965322.addLabel(\"39 (H)\",{\"position\": {\"x\": 6.02527, \"y\": 2.520225, \"z\": 1.358373}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17200163316965322.addLabel(\"40 (H)\",{\"position\": {\"x\": 3.582018, \"y\": 2.889108, \"z\": 1.047457}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17200163316965322.addLabel(\"41 (H)\",{\"position\": {\"x\": 7.952448, \"y\": -0.96568, \"z\": -0.571736}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17200163316965322.addLabel(\"42 (H)\",{\"position\": {\"x\": 7.643832, \"y\": -1.590198, \"z\": 1.095783}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17200163316965322.addLabel(\"43 (H)\",{\"position\": {\"x\": 9.084201, \"y\": -0.627816, \"z\": 0.743519}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n\tviewer_17200163316965322.zoomTo();\nviewer_17200163316965322.render();\n});\n</script>",
            "text/html": [
              "<div id=\"3dmolviewer_17200163316965322\"  style=\"position: relative; width: 800px; height: 600px;\">\n",
              "        <p id=\"3dmolwarning_17200163316965322\" style=\"background-color:#ffcccc;color:black\">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br></p>\n",
              "        </div>\n",
              "<script>\n",
              "\n",
              "var loadScriptAsync = function(uri){\n",
              "  return new Promise((resolve, reject) => {\n",
              "    //this is to ignore the existence of requirejs amd\n",
              "    var savedexports, savedmodule;\n",
              "    if (typeof exports !== 'undefined') savedexports = exports;\n",
              "    else exports = {}\n",
              "    if (typeof module !== 'undefined') savedmodule = module;\n",
              "    else module = {}\n",
              "\n",
              "    var tag = document.createElement('script');\n",
              "    tag.src = uri;\n",
              "    tag.async = true;\n",
              "    tag.onload = () => {\n",
              "        exports = savedexports;\n",
              "        module = savedmodule;\n",
              "        resolve();\n",
              "    };\n",
              "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
              "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
              "});\n",
              "};\n",
              "\n",
              "if(typeof $3Dmolpromise === 'undefined') {\n",
              "$3Dmolpromise = null;\n",
              "  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.1.0/3Dmol-min.js');\n",
              "}\n",
              "\n",
              "var viewer_17200163316965322 = null;\n",
              "var warn = document.getElementById(\"3dmolwarning_17200163316965322\");\n",
              "if(warn) {\n",
              "    warn.parentNode.removeChild(warn);\n",
              "}\n",
              "$3Dmolpromise.then(function() {\n",
              "viewer_17200163316965322 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_17200163316965322\"),{backgroundColor:\"white\"});\n",
              "viewer_17200163316965322.zoomTo();\n",
              "\tviewer_17200163316965322.addModel(\"43\\n\\nC     -6.474593   -1.386555   -0.085378\\nC     -5.010672   -1.114409    0.059177\\nC     -4.153592   -2.102349    0.510946\\nN     -2.824337   -1.936193    0.674526\\nC     -2.279859   -0.735437    0.347187\\nC     -3.055068    0.338660   -0.115204\\nC     -4.442281    0.134371   -0.202095\\nO     -5.257125    1.157798   -0.623738\\nC     -5.712181    1.922362    0.495209\\nC     -2.446673    1.671177   -0.452735\\nC     -0.784509   -0.639865    0.520217\\nS      0.093970   -0.967575   -1.039405\\nO      0.310485   -2.447112   -1.197267\\nC      1.692332   -0.331621   -0.506293\\nN      1.906874    0.869064   -0.015498\\nC      3.256240    0.908021    0.244012\\nC      3.868305   -0.294656   -0.095925\\nN      2.843371   -1.067076   -0.575541\\nC      5.237506   -0.538459    0.065366\\nC      6.005992    0.504755    0.598781\\nC      5.409604    1.723367    0.946098\\nC      4.035473    1.942725    0.775645\\nO      7.356184    0.456793    0.829870\\nC      8.024829   -0.754065    0.500177\\nH     -7.010977   -1.056879    0.809473\\nH     -6.663638   -2.456287   -0.226535\\nH     -6.886116   -0.873089   -0.960372\\nH     -4.517237   -3.095233    0.763116\\nH     -6.303973    1.310527    1.183965\\nH     -6.355958    2.720429    0.113650\\nH     -4.876013    2.387804    1.027737\\nH     -1.531667    1.553640   -1.039730\\nH     -2.216215    2.219709    0.465915\\nH     -3.115661    2.287728   -1.061147\\nH     -0.519142    0.350129    0.903819\\nH     -0.442738   -1.373880    1.258977\\nH      2.892933   -2.018989   -0.912600\\nH      5.658359   -1.494970   -0.216269\\nH      6.025270    2.520225    1.358373\\nH      3.582018    2.889108    1.047457\\nH      7.952448   -0.965680   -0.571736\\nH      7.643832   -1.590198    1.095783\\nH      9.084201   -0.627816    0.743519\\n\",\"xyz\");\n",
              "\tviewer_17200163316965322.setStyle({\"stick\": {}});\n",
              "\tviewer_17200163316965322.addLabel(\"1 (C)\",{\"position\": {\"x\": -6.474593, \"y\": -1.386555, \"z\": -0.085378}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17200163316965322.addLabel(\"2 (C)\",{\"position\": {\"x\": -5.010672, \"y\": -1.114409, \"z\": 0.059177}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17200163316965322.addLabel(\"3 (C)\",{\"position\": {\"x\": -4.153592, \"y\": -2.102349, \"z\": 0.510946}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17200163316965322.addLabel(\"4 (N)\",{\"position\": {\"x\": -2.824337, \"y\": -1.936193, \"z\": 0.674526}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17200163316965322.addLabel(\"5 (C)\",{\"position\": {\"x\": -2.279859, \"y\": -0.735437, \"z\": 0.347187}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17200163316965322.addLabel(\"6 (C)\",{\"position\": {\"x\": -3.055068, \"y\": 0.33866, \"z\": -0.115204}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17200163316965322.addLabel(\"7 (C)\",{\"position\": {\"x\": -4.442281, \"y\": 0.134371, \"z\": -0.202095}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17200163316965322.addLabel(\"8 (O)\",{\"position\": {\"x\": -5.257125, \"y\": 1.157798, \"z\": -0.623738}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17200163316965322.addLabel(\"9 (C)\",{\"position\": {\"x\": -5.712181, \"y\": 1.922362, \"z\": 0.495209}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17200163316965322.addLabel(\"10 (C)\",{\"position\": {\"x\": -2.446673, \"y\": 1.671177, \"z\": -0.452735}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17200163316965322.addLabel(\"11 (C)\",{\"position\": {\"x\": -0.784509, \"y\": -0.639865, \"z\": 0.520217}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17200163316965322.addLabel(\"12 (S)\",{\"position\": {\"x\": 0.09397, \"y\": -0.967575, \"z\": -1.039405}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17200163316965322.addLabel(\"13 (O)\",{\"position\": {\"x\": 0.310485, \"y\": -2.447112, \"z\": -1.197267}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17200163316965322.addLabel(\"14 (C)\",{\"position\": {\"x\": 1.692332, \"y\": -0.331621, \"z\": -0.506293}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17200163316965322.addLabel(\"15 (N)\",{\"position\": {\"x\": 1.906874, \"y\": 0.869064, \"z\": -0.015498}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17200163316965322.addLabel(\"16 (C)\",{\"position\": {\"x\": 3.25624, \"y\": 0.908021, \"z\": 0.244012}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17200163316965322.addLabel(\"17 (C)\",{\"position\": {\"x\": 3.868305, \"y\": -0.294656, \"z\": -0.095925}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17200163316965322.addLabel(\"18 (N)\",{\"position\": {\"x\": 2.843371, \"y\": -1.067076, \"z\": -0.575541}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17200163316965322.addLabel(\"19 (C)\",{\"position\": {\"x\": 5.237506, \"y\": -0.538459, \"z\": 0.065366}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17200163316965322.addLabel(\"20 (C)\",{\"position\": {\"x\": 6.005992, \"y\": 0.504755, \"z\": 0.598781}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17200163316965322.addLabel(\"21 (C)\",{\"position\": {\"x\": 5.409604, \"y\": 1.723367, \"z\": 0.946098}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17200163316965322.addLabel(\"22 (C)\",{\"position\": {\"x\": 4.035473, \"y\": 1.942725, \"z\": 0.775645}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17200163316965322.addLabel(\"23 (O)\",{\"position\": {\"x\": 7.356184, \"y\": 0.456793, \"z\": 0.82987}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17200163316965322.addLabel(\"24 (C)\",{\"position\": {\"x\": 8.024829, \"y\": -0.754065, \"z\": 0.500177}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17200163316965322.addLabel(\"25 (H)\",{\"position\": {\"x\": -7.010977, \"y\": -1.056879, \"z\": 0.809473}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17200163316965322.addLabel(\"26 (H)\",{\"position\": {\"x\": -6.663638, \"y\": -2.456287, \"z\": -0.226535}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17200163316965322.addLabel(\"27 (H)\",{\"position\": {\"x\": -6.886116, \"y\": -0.873089, \"z\": -0.960372}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17200163316965322.addLabel(\"28 (H)\",{\"position\": {\"x\": -4.517237, \"y\": -3.095233, \"z\": 0.763116}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17200163316965322.addLabel(\"29 (H)\",{\"position\": {\"x\": -6.303973, \"y\": 1.310527, \"z\": 1.183965}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17200163316965322.addLabel(\"30 (H)\",{\"position\": {\"x\": -6.355958, \"y\": 2.720429, \"z\": 0.11365}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17200163316965322.addLabel(\"31 (H)\",{\"position\": {\"x\": -4.876013, \"y\": 2.387804, \"z\": 1.027737}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17200163316965322.addLabel(\"32 (H)\",{\"position\": {\"x\": -1.531667, \"y\": 1.55364, \"z\": -1.03973}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17200163316965322.addLabel(\"33 (H)\",{\"position\": {\"x\": -2.216215, \"y\": 2.219709, \"z\": 0.465915}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17200163316965322.addLabel(\"34 (H)\",{\"position\": {\"x\": -3.115661, \"y\": 2.287728, \"z\": -1.061147}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17200163316965322.addLabel(\"35 (H)\",{\"position\": {\"x\": -0.519142, \"y\": 0.350129, \"z\": 0.903819}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17200163316965322.addLabel(\"36 (H)\",{\"position\": {\"x\": -0.442738, \"y\": -1.37388, \"z\": 1.258977}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17200163316965322.addLabel(\"37 (H)\",{\"position\": {\"x\": 2.892933, \"y\": -2.018989, \"z\": -0.9126}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17200163316965322.addLabel(\"38 (H)\",{\"position\": {\"x\": 5.658359, \"y\": -1.49497, \"z\": -0.216269}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17200163316965322.addLabel(\"39 (H)\",{\"position\": {\"x\": 6.02527, \"y\": 2.520225, \"z\": 1.358373}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17200163316965322.addLabel(\"40 (H)\",{\"position\": {\"x\": 3.582018, \"y\": 2.889108, \"z\": 1.047457}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17200163316965322.addLabel(\"41 (H)\",{\"position\": {\"x\": 7.952448, \"y\": -0.96568, \"z\": -0.571736}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17200163316965322.addLabel(\"42 (H)\",{\"position\": {\"x\": 7.643832, \"y\": -1.590198, \"z\": 1.095783}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17200163316965322.addLabel(\"43 (H)\",{\"position\": {\"x\": 9.084201, \"y\": -0.627816, \"z\": 0.743519}, \"backgroundColor\": \"white\", \"backgroundOpacity\": 0.8, \"fontColor\": \"black\"});\n",
              "\tviewer_17200163316965322.zoomTo();\n",
              "viewer_17200163316965322.render();\n",
              "});\n",
              "</script>"
            ]
          },
          "metadata": {}
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "%%file scan.inp\n",
        "$constrain\n",
        " force constant=0.05\n",
        "$scan\n",
        " dihedral: 4,5,11,12,60.0; 60.0,420.0,36\n",
        "$end"
      ],
      "metadata": {
        "id": "ky6Pnr6I7DUq",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "f2f5a8d3-65fe-4821-dbd4-b304f8504cf5"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Overwriting scan.inp\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!xtb ome.xyz --opt --input scan.inp"
      ],
      "metadata": {
        "id": "eFN4Eh4c7mYd",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "b0df58a5-8c00-46a9-9b2c-a596d27c0b24"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "      -----------------------------------------------------------      \n",
            "     |                   =====================                   |     \n",
            "     |                           x T B                           |     \n",
            "     |                   =====================                   |     \n",
            "     |                         S. Grimme                         |     \n",
            "     |          Mulliken Center for Theoretical Chemistry        |     \n",
            "     |                    University of Bonn                     |     \n",
            "      -----------------------------------------------------------      \n",
            "\n",
            "   * xtb version 6.7.0 (08769fc) compiled by 'albert@albert-system' on 2024-03-04\n",
            "\n",
            "   xtb is free software: you can redistribute it and/or modify it under\n",
            "   the terms of the GNU Lesser General Public License as published by\n",
            "   the Free Software Foundation, either version 3 of the License, or\n",
            "   (at your option) any later version.\n",
            "   \n",
            "   xtb is distributed in the hope that it will be useful,\n",
            "   but WITHOUT ANY WARRANTY; without even the implied warranty of\n",
            "   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the\n",
            "   GNU Lesser General Public License for more details.\n",
            "   \n",
            "   Cite this work as:\n",
            "   * C. Bannwarth, E. Caldeweyher, S. Ehlert, A. Hansen, P. Pracht,\n",
            "     J. Seibert, S. Spicher, S. Grimme, WIREs Comput. Mol. Sci., 2020, 11,\n",
            "     e01493. DOI: 10.1002/wcms.1493\n",
            "   \n",
            "   for GFN2-xTB:\n",
            "   * C. Bannwarth, S. Ehlert and S. Grimme., J. Chem. Theory Comput., 2019,\n",
            "     15, 1652-1671. DOI: 10.1021/acs.jctc.8b01176\n",
            "   for GFN1-xTB:\n",
            "   * S. Grimme, C. Bannwarth, P. Shushkov, J. Chem. Theory Comput., 2017,\n",
            "     13, 1989-2009. DOI: 10.1021/acs.jctc.7b00118\n",
            "   for GFN0-xTB:\n",
            "   * P. Pracht, E. Caldeweyher, S. Ehlert, S. Grimme, ChemRxiv, 2019, preprint.\n",
            "     DOI: 10.26434/chemrxiv.8326202.v1\n",
            "   for GFN-FF:\n",
            "   * S. Spicher and S. Grimme, Angew. Chem. Int. Ed., 2020, 59, 15665-15673.\n",
            "     DOI: 10.1002/anie.202004239\n",
            "   \n",
            "   for ALPB and GBSA implicit solvation:\n",
            "   * S. Ehlert, M. Stahn, S. Spicher, S. Grimme, J. Chem. Theory Comput.,\n",
            "     2021, 17, 4250-4261. DOI: 10.1021/acs.jctc.1c00471\n",
            "   \n",
            "   for ddCOSMO and CPCM-X implicit solvation:\n",
            "   * M. Stahn, S. Ehlert, S. Grimme, J. Phys. Chem. A,\n",
            "     2023, XX, XXXX-XXXX. DOI: 10.1021/acs.jpca.3c04382\n",
            "   \n",
            "   for DFT-D4:\n",
            "   * E. Caldeweyher, C. Bannwarth and S. Grimme, J. Chem. Phys., 2017,\n",
            "     147, 034112. DOI: 10.1063/1.4993215\n",
            "   * E. Caldeweyher, S. Ehlert, A. Hansen, H. Neugebauer, S. Spicher,\n",
            "     C. Bannwarth and S. Grimme, J. Chem. Phys., 2019, 150, 154122.\n",
            "     DOI: 10.1063/1.5090222\n",
            "   * E. Caldeweyher, J.-M. Mewes, S. Ehlert and S. Grimme, Phys. Chem. Chem. Phys.\n",
            "     2020, 22, 8499-8512. DOI: 10.1039/D0CP00502A\n",
            "   \n",
            "   for sTDA-xTB:\n",
            "   * S. Grimme and C. Bannwarth, J. Chem. Phys., 2016, 145, 054103.\n",
            "     DOI: 10.1063/1.4959605\n",
            "   \n",
            "   in the mass-spec context:\n",
            "   * V. Asgeirsson, C. Bauer and S. Grimme, Chem. Sci., 2017, 8, 4879.\n",
            "     DOI: 10.1039/c7sc00601b\n",
            "   * J. Koopman and S. Grimme, ACS Omega 2019, 4, 12, 15120-15133.\n",
            "     DOI: 10.1021/acsomega.9b02011\n",
            "   \n",
            "   for metadynamics refer to:\n",
            "   * S. Grimme, J. Chem. Theory Comput., 2019, 155, 2847-2862\n",
            "     DOI: 10.1021/acs.jctc.9b00143\n",
            "   \n",
            "   for SPH calculations refer to:\n",
            "   * S. Spicher and S. Grimme, J. Chem. Theory Comput., 2021, 17, 1701-1714\n",
            "     DOI: 10.1021/acs.jctc.0c01306\n",
            "   \n",
            "   for ONIOM refer to:\n",
            "   * C. Plett, A. Katbashev, S. Ehlert, S. Grimme, M. Bursch,\n",
            "     Phys. Chem. Chem. Phys., 2023, 25, 17860-17868. DOI: 10.1039/D3CP02178E\n",
            "   \n",
            "   for DIPRO refer to:\n",
            "   * J. Kohn, N. Gildemeister, S. Grimme, D. Fazzi, A. Hansen,\n",
            "     J. Chem. Phys., 2023, just accepted.\n",
            "   \n",
            "   for PTB refer to:\n",
            "   * S. Grimme, M. Mueller, A. Hansen, J. Chem. Phys., 2023, 158, 124111.\n",
            "     DOI: 10.1063/5.0137838\n",
            "   \n",
            "   with help from (in alphabetical order)\n",
            "   P. Atkinson, C. Bannwarth, F. Bohle, G. Brandenburg, E. Caldeweyher\n",
            "   M. Checinski, S. Dohm, S. Ehlert, S. Ehrlich, I. Gerasimov, C. H√∂lzer\n",
            "   A. Katbashev, J. Kohn, J. Koopman, C. Lavigne, S. Lehtola, F. M√§rz, M. M√ºller,\n",
            "   F. Musil, H. Neugebauer, J. Pisarek, C. Plett, P. Pracht, F. Pultar,\n",
            "   J. Seibert, P. Shushkov, S. Spicher, M. Stahn, M. Steiner, T. Strunk,\n",
            "   J. St√ºckrath, T. Rose, and J. Unsleber\n",
            "   \n",
            " * started run on 2024/07/03 at 14:20:17.465     \n",
            "   ID    Z sym.   atoms\n",
            "    1    6 C      1-3, 5-7, 9-11, 14, 16, 17, 19-22, 24\n",
            "    2    7 N      4, 15, 18\n",
            "    3    8 O      8, 13, 23\n",
            "    4   16 S      12\n",
            "    5    1 H      25-43\n",
            "constraining angle 4 5 11 12 to   60.0000000¬∞, actual value:   95.7181748¬∞\n",
            "\n",
            "           -------------------------------------------------\n",
            "          |                Calculation Setup                |\n",
            "           -------------------------------------------------\n",
            "\n",
            "          program call               : xtb ome.xyz --opt --input scan.inp\n",
            "          hostname                   : ff56854ec18c\n",
            "          coordinate file            : ome.xyz\n",
            "          omp threads                :                     2\n",
            "\n",
            "\n",
            "           -------------------------------------------------\n",
            "          |                 G F N 2 - x T B                 |\n",
            "           -------------------------------------------------\n",
            "\n",
            "        Reference                      10.1021/acs.jctc.8b01176\n",
            "      * Hamiltonian:\n",
            "        H0-scaling (s, p, d)           1.850000    2.230000    2.230000\n",
            "        zeta-weighting                 0.500000\n",
            "      * Dispersion:\n",
            "        s8                             2.700000\n",
            "        a1                             0.520000\n",
            "        a2                             5.000000\n",
            "        s9                             5.000000\n",
            "      * Repulsion:\n",
            "        kExp                           1.500000    1.000000\n",
            "        rExp                           1.000000\n",
            "      * Coulomb:\n",
            "        alpha                          2.000000\n",
            "        third order                    shell-resolved\n",
            "        anisotropic                    true\n",
            "        a3                             3.000000\n",
            "        a5                             4.000000\n",
            "        cn-shift                       1.200000\n",
            "        cn-exp                         4.000000\n",
            "        max-rad                        5.000000\n",
            "\n",
            "\n",
            "          ...................................................\n",
            "          :                      SETUP                      :\n",
            "          :.................................................:\n",
            "          :  # basis functions                 121          :\n",
            "          :  # atomic orbitals                 120          :\n",
            "          :  # shells                           68          :\n",
            "          :  # electrons                       126          :\n",
            "          :  max. iterations                   250          :\n",
            "          :  Hamiltonian                  GFN2-xTB          :\n",
            "          :  restarted?                      false          :\n",
            "          :  GBSA solvation                  false          :\n",
            "          :  PC potential                    false          :\n",
            "          :  electronic temp.          300.0000000     K    :\n",
            "          :  accuracy                    1.0000000          :\n",
            "          :  -> integral cutoff          0.2500000E+02      :\n",
            "          :  -> integral neglect         0.1000000E-07      :\n",
            "          :  -> SCF convergence          0.1000000E-05 Eh   :\n",
            "          :  -> wf. convergence          0.1000000E-03 e    :\n",
            "          :  Broyden damping             0.4000000          :\n",
            "          :  net charge                          0          :\n",
            "          :  unpaired electrons                  0          :\n",
            "          ...................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.0619691 -0.710620E+02  0.451E+00    3.22       0.0  T\n",
            "   2    -71.1690300 -0.107061E+00  0.259E+00    3.15       1.0  T\n",
            "   3    -71.1688859  0.144034E-03  0.115E+00    3.08       1.0  T\n",
            "   4    -71.1777634 -0.887752E-02  0.408E-01    3.03       1.0  T\n",
            "   5    -71.1813461 -0.358271E-02  0.152E-01    3.00       1.0  T\n",
            "   6    -71.1813748 -0.286515E-04  0.743E-02    3.00       1.0  T\n",
            "   7    -71.1814857 -0.110953E-03  0.333E-02    3.00       1.0  T\n",
            "   8    -71.1814979 -0.121139E-04  0.183E-02    3.01       1.0  T\n",
            "   9    -71.1814983 -0.469535E-06  0.644E-03    3.01       2.4  T\n",
            "  10    -71.1814990 -0.644297E-06  0.287E-03    3.01       5.3  T\n",
            "  11    -71.1814990 -0.456637E-07  0.158E-03    3.01       9.7  T\n",
            "  12    -71.1814990 -0.197451E-07  0.540E-04    3.01      28.2  T\n",
            "  13    -71.1814990 -0.313673E-08  0.239E-04    3.01      63.8  T\n",
            "\n",
            "   *** convergence criteria satisfied after 13 iterations ***\n",
            "\n",
            "         #    Occupation            Energy/Eh            Energy/eV\n",
            "      -------------------------------------------------------------\n",
            "         1        2.0000           -0.7559706             -20.5710\n",
            "       ...           ...                  ...                  ...\n",
            "        57        2.0000           -0.4097323             -11.1494\n",
            "        58        2.0000           -0.4026654             -10.9571\n",
            "        59        2.0000           -0.3963424             -10.7850\n",
            "        60        2.0000           -0.3944271             -10.7329\n",
            "        61        2.0000           -0.3884136             -10.5693\n",
            "        62        2.0000           -0.3774842             -10.2719\n",
            "        63        2.0000           -0.3722788             -10.1302 (HOMO)\n",
            "        64                         -0.2617604              -7.1229 (LUMO)\n",
            "        65                         -0.2510785              -6.8322\n",
            "        66                         -0.2367235              -6.4416\n",
            "        67                         -0.2158888              -5.8746\n",
            "        68                         -0.2009158              -5.4672\n",
            "       ...                                ...                  ...\n",
            "       120                          0.8046747              21.8963\n",
            "      -------------------------------------------------------------\n",
            "                  HL-Gap            0.1105185 Eh            3.0074 eV\n",
            "             Fermi-level           -0.3170196 Eh           -8.6265 eV\n",
            "\n",
            " SCC (total)                   0 d,  0 h,  0 min,  0.106 sec\n",
            " SCC setup                      ...        0 min,  0.001 sec (  0.500%)\n",
            " Dispersion                     ...        0 min,  0.001 sec (  0.808%)\n",
            " classical contributions        ...        0 min,  0.000 sec (  0.092%)\n",
            " integral evaluation            ...        0 min,  0.011 sec ( 10.184%)\n",
            " iterations                     ...        0 min,  0.056 sec ( 52.929%)\n",
            " molecular gradient             ...        0 min,  0.037 sec ( 34.870%)\n",
            " printout                       ...        0 min,  0.001 sec (  0.573%)\n",
            "\n",
            "         :::::::::::::::::::::::::::::::::::::::::::::::::::::\n",
            "         ::                     SUMMARY                     ::\n",
            "         :::::::::::::::::::::::::::::::::::::::::::::::::::::\n",
            "         :: total energy             -70.289863370567 Eh    ::\n",
            "         :: gradient norm              0.107399670667 Eh/a0 ::\n",
            "         :: HOMO-LUMO gap              3.007360418860 eV    ::\n",
            "         ::.................................................::\n",
            "         :: SCC energy               -71.181753152876 Eh    ::\n",
            "         :: -> isotropic ES            0.095103394998 Eh    ::\n",
            "         :: -> anisotropic ES          0.008288695309 Eh    ::\n",
            "         :: -> anisotropic XC          0.031204587908 Eh    ::\n",
            "         :: -> dispersion             -0.046692856309 Eh    ::\n",
            "         :: repulsion energy           0.882484701340 Eh    ::\n",
            "         :: add. restraining           0.009405080969 Eh    ::\n",
            "         :: total charge              -0.000000000000 e     ::\n",
            "         :::::::::::::::::::::::::::::::::::::::::::::::::::::\n",
            "\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            "\n",
            "          ...................................................\n",
            "          :                      SETUP                      :\n",
            "          :.................................................:\n",
            "          :   optimization level            normal          :\n",
            "          :   max. optcycles                   200          :\n",
            "          :   ANC micro-cycles                  20          :\n",
            "          :   degrees of freedom               123          :\n",
            "          :.................................................:\n",
            "          :   RF solver                   davidson          :\n",
            "          :   write xtbopt.log                true          :\n",
            "          :   linear?                        false          :\n",
            "          :   energy convergence         0.5000000E-05 Eh   :\n",
            "          :   grad. convergence          0.1000000E-02 Eh/Œ± :\n",
            "          :   maxmium RF displ.          1.0000000          :\n",
            "          :   Hlow (freq-cutoff)         0.1000000E-01      :\n",
            "          :   Hmax (freq-cutoff)         5.0000000          :\n",
            "          :   S6 in model hess.         20.0000000          :\n",
            "          ...................................................\n",
            "\n",
            "generating ANC from model Hessian ...\n",
            "Using Lindh-Hessian (1995)\n",
            " Shifting diagonal of input Hessian by   9.808222731644804E-003\n",
            " Lowest  eigenvalues of input Hessian\n",
            "    0.000000    0.000000    0.000000    0.000000    0.000000    0.000000\n",
            "    0.010000    0.010041    0.010286    0.010778    0.011762    0.011895\n",
            "    0.012382    0.013049    0.013908    0.014863    0.015618    0.016067\n",
            " Highest eigenvalues\n",
            "    1.987632    2.009943    2.038803    2.093282    2.228500    2.270086\n",
            " \n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE    1 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.1814990 -0.711815E+02  0.104E-04    3.01       0.0  T\n",
            "   2    -71.1814990 -0.808456E-10  0.662E-05    3.01     230.3  T\n",
            "   3    -71.1814990 -0.732996E-10  0.425E-05    3.01     358.7  T\n",
            "     SCC iter.                  ...        0 min,  0.013 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.2898634 Eh     change       -0.1092587E-08 Eh\n",
            "   gradient norm :     0.1074001 Eh/Œ±   predicted     0.0000000E+00 (-100.00%)\n",
            "   displ. norm   :     0.3142380 Œ±      lambda       -0.1515289E-01\n",
            "   maximum displ.:     0.0979399 Œ±      in ANC's #30, #19, #11, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE    2 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2138024 -0.712138E+02  0.206E-01    2.98       0.0  T\n",
            "   2    -71.2139328 -0.130333E-03  0.131E-01    3.02       1.0  T\n",
            "   3    -71.2139119  0.208294E-04  0.570E-02    2.99       1.0  T\n",
            "   4    -71.2139760 -0.640170E-04  0.158E-02    3.01       1.0  T\n",
            "   5    -71.2139815 -0.550415E-05  0.661E-03    3.00       2.3  T\n",
            "   6    -71.2139815 -0.767858E-07  0.242E-03    3.00       6.3  T\n",
            "   7    -71.2139816 -0.671987E-07  0.108E-03    3.00      14.1  T\n",
            "   8    -71.2139816 -0.864290E-08  0.626E-04    3.00      24.3  T\n",
            "   9    -71.2139816 -0.251333E-08  0.200E-04    3.00      76.2  T\n",
            "     SCC iter.                  ...        0 min,  0.034 sec\n",
            "     gradient                   ...        0 min,  0.035 sec\n",
            " * total energy  :   -70.3003701 Eh     change       -0.1050671E-01 Eh\n",
            "   gradient norm :     0.0534872 Eh/Œ±   predicted    -0.8336904E-02 ( -20.65%)\n",
            "   displ. norm   :     0.3347602 Œ±      lambda       -0.5582561E-02\n",
            "   maximum displ.:     0.1667718 Œ±      in ANC's #11, #19, #10, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE    3 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2336007 -0.712336E+02  0.157E-01    2.99       0.0  T\n",
            "   2    -71.2337096 -0.108946E-03  0.105E-01    3.01       1.0  T\n",
            "   3    -71.2337248 -0.151882E-04  0.513E-02    3.00       1.0  T\n",
            "   4    -71.2337370 -0.121428E-04  0.149E-02    3.00       1.0  T\n",
            "   5    -71.2337437 -0.675027E-05  0.497E-03    3.00       3.1  T\n",
            "   6    -71.2337437 -0.319423E-07  0.266E-03    3.00       5.7  T\n",
            "   7    -71.2337438 -0.442407E-07  0.760E-04    3.00      20.1  T\n",
            "   8    -71.2337438 -0.707016E-08  0.469E-04    3.00      32.5  T\n",
            "     SCC iter.                  ...        0 min,  0.038 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3038623 Eh     change       -0.3492270E-02 Eh\n",
            "   gradient norm :     0.0179838 Eh/Œ±   predicted    -0.3107290E-02 ( -11.02%)\n",
            "   displ. norm   :     0.3023516 Œ±      lambda       -0.1508269E-02\n",
            "   maximum displ.:     0.1783047 Œ±      in ANC's #11, #10, #19, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE    4 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2356177 -0.712356E+02  0.103E-01    3.00       0.0  T\n",
            "   2    -71.2356349 -0.172027E-04  0.623E-02    2.99       1.0  T\n",
            "   3    -71.2356334  0.149309E-05  0.214E-02    2.98       1.0  T\n",
            "   4    -71.2356378 -0.437199E-05  0.485E-03    2.99       3.1  T\n",
            "   5    -71.2356381 -0.324712E-06  0.277E-03    2.99       5.5  T\n",
            "   6    -71.2356382 -0.846725E-07  0.995E-04    2.99      15.3  T\n",
            "   7    -71.2356382 -0.200942E-07  0.305E-04    2.99      50.0  T\n",
            "     SCC iter.                  ...        0 min,  0.029 sec\n",
            "     gradient                   ...        0 min,  0.052 sec\n",
            " * total energy  :   -70.3049972 Eh     change       -0.1134836E-02 Eh\n",
            "   gradient norm :     0.0114933 Eh/Œ±   predicted    -0.8232458E-03 ( -27.46%)\n",
            "   displ. norm   :     0.3969774 Œ±      lambda       -0.1209779E-02\n",
            "   maximum displ.:     0.2238007 Œ±      in ANC's #11, #10, #6, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE    5 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2344073 -0.712344E+02  0.120E-01    2.99       0.0  T\n",
            "   2    -71.2344146 -0.725160E-05  0.693E-02    2.98       1.0  T\n",
            "   3    -71.2344132  0.137898E-05  0.124E-02    2.97       1.2  T\n",
            "   4    -71.2344160 -0.285451E-05  0.526E-03    2.97       2.9  T\n",
            "   5    -71.2344162 -0.205068E-06  0.221E-03    2.97       6.9  T\n",
            "   6    -71.2344163 -0.824363E-07  0.658E-04    2.97      23.2  T\n",
            "   7    -71.2344163 -0.714934E-08  0.311E-04    2.97      49.1  T\n",
            "     SCC iter.                  ...        0 min,  0.039 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3058365 Eh     change       -0.8393600E-03 Eh\n",
            "   gradient norm :     0.0093930 Eh/Œ±   predicted    -0.7002154E-03 ( -16.58%)\n",
            "   displ. norm   :     0.3096821 Œ±      lambda       -0.5584196E-03\n",
            "   maximum displ.:     0.1479156 Œ±      in ANC's #11, #10, #6, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE    6 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2340345 -0.712340E+02  0.815E-02    2.98       0.0  T\n",
            "   2    -71.2340381 -0.359961E-05  0.456E-02    2.97       1.0  T\n",
            "   3    -71.2340368  0.122554E-05  0.109E-02    2.97       1.4  T\n",
            "   4    -71.2340386 -0.177589E-05  0.496E-03    2.97       3.1  T\n",
            "   5    -71.2340391 -0.456882E-06  0.176E-03    2.97       8.7  T\n",
            "   6    -71.2340391 -0.354300E-07  0.560E-04    2.97      27.2  T\n",
            "   7    -71.2340391 -0.212316E-08  0.320E-04    2.97      47.7  T\n",
            "     SCC iter.                  ...        0 min,  0.057 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3062359 Eh     change       -0.3993333E-03 Eh\n",
            "   gradient norm :     0.0082532 Eh/Œ±   predicted    -0.3059982E-03 ( -23.37%)\n",
            "   displ. norm   :     0.2728186 Œ±      lambda       -0.3715428E-03\n",
            "   maximum displ.:     0.1249697 Œ±      in ANC's #3, #6, #10, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE    7 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2346494 -0.712346E+02  0.688E-02    2.98       0.0  T\n",
            "   2    -71.2346526 -0.320109E-05  0.377E-02    2.97       1.0  T\n",
            "   3    -71.2346520  0.604161E-06  0.106E-02    2.98       1.4  T\n",
            "   4    -71.2346529 -0.945481E-06  0.481E-03    2.98       3.2  T\n",
            "   5    -71.2346534 -0.460847E-06  0.177E-03    2.98       8.6  T\n",
            "   6    -71.2346534 -0.329968E-07  0.658E-04    2.98      23.2  T\n",
            "   7    -71.2346534 -0.359087E-08  0.369E-04    2.98      41.4  T\n",
            "     SCC iter.                  ...        0 min,  0.027 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3065074 Eh     change       -0.2715276E-03 Eh\n",
            "   gradient norm :     0.0067749 Eh/Œ±   predicted    -0.1995987E-03 ( -26.49%)\n",
            "   displ. norm   :     0.2582732 Œ±      lambda       -0.2703852E-03\n",
            "   maximum displ.:     0.1423175 Œ±      in ANC's #3, #6, #10, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE    8 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2350885 -0.712351E+02  0.643E-02    2.99       0.0  T\n",
            "   2    -71.2350915 -0.300952E-05  0.345E-02    2.98       1.0  T\n",
            "   3    -71.2350912  0.260243E-06  0.102E-02    2.99       1.5  T\n",
            "   4    -71.2350917 -0.512179E-06  0.476E-03    2.99       3.2  T\n",
            "   5    -71.2350922 -0.436150E-06  0.176E-03    2.99       8.7  T\n",
            "   6    -71.2350922 -0.299842E-07  0.679E-04    2.99      22.5  T\n",
            "   7    -71.2350922 -0.376720E-08  0.388E-04    2.99      39.3  T\n",
            "     SCC iter.                  ...        0 min,  0.029 sec\n",
            "     gradient                   ...        0 min,  0.037 sec\n",
            " * total energy  :   -70.3067050 Eh     change       -0.1976038E-03 Eh\n",
            "   gradient norm :     0.0057518 Eh/Œ±   predicted    -0.1442111E-03 ( -27.02%)\n",
            "   displ. norm   :     0.2424056 Œ±      lambda       -0.1902489E-03\n",
            "   maximum displ.:     0.1511212 Œ±      in ANC's #3, #6, #8, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE    9 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2351887 -0.712352E+02  0.593E-02    2.99       0.0  T\n",
            "   2    -71.2351910 -0.234104E-05  0.318E-02    3.00       1.0  T\n",
            "   3    -71.2351910  0.254782E-07  0.879E-03    3.00       1.7  T\n",
            "   4    -71.2351912 -0.255590E-06  0.417E-03    3.00       3.7  T\n",
            "   5    -71.2351915 -0.257244E-06  0.147E-03    3.00      10.4  T\n",
            "   6    -71.2351915 -0.321299E-07  0.618E-04    3.00      24.7  T\n",
            "   7    -71.2351915 -0.374774E-08  0.349E-04    3.00      43.7  T\n",
            "     SCC iter.                  ...        0 min,  0.042 sec\n",
            "     gradient                   ...        0 min,  0.041 sec\n",
            " * total energy  :   -70.3068478 Eh     change       -0.1428322E-03 Eh\n",
            "   gradient norm :     0.0050591 Eh/Œ±   predicted    -0.1007159E-03 ( -29.49%)\n",
            "   displ. norm   :     0.2323657 Œ±      lambda       -0.1539973E-03\n",
            "   maximum displ.:     0.1559804 Œ±      in ANC's #3, #6, #4, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   10 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2353693 -0.712354E+02  0.557E-02    3.00       0.0  T\n",
            "   2    -71.2353706 -0.130935E-05  0.304E-02    3.00       1.0  T\n",
            "   3    -71.2353705  0.251362E-07  0.643E-03    3.01       2.4  T\n",
            "   4    -71.2353707 -0.188005E-06  0.341E-03    3.01       4.5  T\n",
            "   5    -71.2353709 -0.198249E-06  0.103E-03    3.01      14.9  T\n",
            "   6    -71.2353709 -0.195239E-07  0.494E-04    3.01      30.9  T\n",
            "   7    -71.2353709 -0.168292E-08  0.280E-04    3.01      54.4  T\n",
            "     SCC iter.                  ...        0 min,  0.035 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3069673 Eh     change       -0.1195091E-03 Eh\n",
            "   gradient norm :     0.0044840 Eh/Œ±   predicted    -0.8115880E-04 ( -32.09%)\n",
            "   displ. norm   :     0.2434626 Œ±      lambda       -0.1405520E-03\n",
            "   maximum displ.:     0.1674878 Œ±      in ANC's #3, #4, #6, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   11 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2351006 -0.712351E+02  0.584E-02    3.01       0.0  T\n",
            "   2    -71.2351019 -0.127649E-05  0.320E-02    3.01       1.0  T\n",
            "   3    -71.2351017  0.223012E-06  0.665E-03    3.01       2.3  T\n",
            "   4    -71.2351020 -0.360030E-06  0.364E-03    3.01       4.2  T\n",
            "   5    -71.2351023 -0.245158E-06  0.966E-04    3.01      15.8  T\n",
            "   6    -71.2351023 -0.219399E-07  0.416E-04    3.01      36.6  T\n",
            "     SCC iter.                  ...        0 min,  0.030 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3070784 Eh     change       -0.1110455E-03 Eh\n",
            "   gradient norm :     0.0041882 Eh/Œ±   predicted    -0.7444338E-04 ( -32.96%)\n",
            "   displ. norm   :     0.2345120 Œ±      lambda       -0.1229412E-03\n",
            "   maximum displ.:     0.1640232 Œ±      in ANC's #3, #4, #6, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   12 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2351173 -0.712351E+02  0.558E-02    3.01       0.0  T\n",
            "   2    -71.2351186 -0.128413E-05  0.315E-02    3.01       1.0  T\n",
            "   3    -71.2351179  0.700726E-06  0.735E-03    3.01       2.1  T\n",
            "   4    -71.2351188 -0.986926E-06  0.320E-03    3.01       4.8  T\n",
            "   5    -71.2351189 -0.100366E-06  0.113E-03    3.01      13.5  T\n",
            "   6    -71.2351190 -0.301518E-07  0.410E-04    3.01      37.2  T\n",
            "   7    -71.2351190 -0.202300E-08  0.197E-04    3.01      77.4  T\n",
            "     SCC iter.                  ...        0 min,  0.028 sec\n",
            "     gradient                   ...        0 min,  0.036 sec\n",
            " * total energy  :   -70.3071770 Eh     change       -0.9857211E-04 Eh\n",
            "   gradient norm :     0.0036418 Eh/Œ±   predicted    -0.6485251E-04 ( -34.21%)\n",
            "   displ. norm   :     0.2435291 Œ±      lambda       -0.1110358E-03\n",
            "   maximum displ.:     0.1701328 Œ±      in ANC's #3, #4, #6, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   13 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2342752 -0.712343E+02  0.586E-02    3.00       0.0  T\n",
            "   2    -71.2342762 -0.103146E-05  0.331E-02    3.00       1.0  T\n",
            "   3    -71.2342755  0.697061E-06  0.678E-03    3.00       2.2  T\n",
            "   4    -71.2342764 -0.897518E-06  0.334E-03    3.00       4.6  T\n",
            "   5    -71.2342766 -0.188502E-06  0.821E-04    3.00      18.6  T\n",
            "   6    -71.2342766 -0.131688E-07  0.314E-04    3.00      48.6  T\n",
            "     SCC iter.                  ...        0 min,  0.023 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3072663 Eh     change       -0.8934485E-04 Eh\n",
            "   gradient norm :     0.0034527 Eh/Œ±   predicted    -0.5881098E-04 ( -34.18%)\n",
            "   displ. norm   :     0.2469241 Œ±      lambda       -0.1070170E-03\n",
            "   maximum displ.:     0.1733190 Œ±      in ANC's #3, #4, #7, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   14 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2347074 -0.712347E+02  0.583E-02    3.00       0.0  T\n",
            "   2    -71.2347091 -0.169239E-05  0.344E-02    3.00       1.0  T\n",
            "   3    -71.2347078  0.126161E-05  0.900E-03    3.00       1.7  T\n",
            "   4    -71.2347095 -0.167897E-05  0.279E-03    3.00       5.5  T\n",
            "   5    -71.2347096 -0.834749E-07  0.142E-03    3.00      10.8  T\n",
            "   6    -71.2347096 -0.320636E-07  0.493E-04    3.00      31.0  T\n",
            "   7    -71.2347096 -0.218563E-08  0.187E-04    3.00      81.6  T\n",
            "     SCC iter.                  ...        0 min,  0.035 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3073483 Eh     change       -0.8194445E-04 Eh\n",
            "   gradient norm :     0.0040510 Eh/Œ±   predicted    -0.5677328E-04 ( -30.72%)\n",
            "   displ. norm   :     0.2726410 Œ±      lambda       -0.1014258E-03\n",
            "   maximum displ.:     0.1915462 Œ±      in ANC's #3, #4, #6, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   15 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2331793 -0.712332E+02  0.678E-02    3.00       0.0  T\n",
            "   2    -71.2331809 -0.161221E-05  0.385E-02    2.99       1.0  T\n",
            "   3    -71.2331800  0.874896E-06  0.860E-03    2.99       1.8  T\n",
            "   4    -71.2331811 -0.106732E-05  0.372E-03    2.99       4.1  T\n",
            "   5    -71.2331814 -0.337305E-06  0.120E-03    2.99      12.7  T\n",
            "   6    -71.2331814 -0.159372E-07  0.397E-04    2.99      38.4  T\n",
            "   7    -71.2331814 -0.112857E-08  0.205E-04    2.99      74.5  T\n",
            "     SCC iter.                  ...        0 min,  0.028 sec\n",
            "     gradient                   ...        0 min,  0.035 sec\n",
            " * total energy  :   -70.3074237 Eh     change       -0.7545167E-04 Eh\n",
            "   gradient norm :     0.0037786 Eh/Œ±   predicted    -0.5448432E-04 ( -27.79%)\n",
            "   displ. norm   :     0.2531414 Œ±      lambda       -0.9231453E-04\n",
            "   maximum displ.:     0.1813081 Œ±      in ANC's #3, #4, #8, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   16 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2337056 -0.712337E+02  0.622E-02    2.99       0.0  T\n",
            "   2    -71.2337065 -0.828667E-06  0.365E-02    2.99       1.0  T\n",
            "   3    -71.2337061  0.392236E-06  0.520E-03    2.99       2.9  T\n",
            "   4    -71.2337068 -0.678674E-06  0.232E-03    2.99       6.6  T\n",
            "   5    -71.2337068 -0.706790E-07  0.724E-04    2.99      21.1  T\n",
            "   6    -71.2337068 -0.783291E-08  0.355E-04    2.99      42.9  T\n",
            "     SCC iter.                  ...        0 min,  0.024 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3074963 Eh     change       -0.7263287E-04 Eh\n",
            "   gradient norm :     0.0039542 Eh/Œ±   predicted    -0.4911806E-04 ( -32.37%)\n",
            "   displ. norm   :     0.2797426 Œ±      lambda       -0.9519888E-04\n",
            "   maximum displ.:     0.2055669 Œ±      in ANC's #3, #4, #8, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   17 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2335446 -0.712335E+02  0.697E-02    3.00       0.0  T\n",
            "   2    -71.2335460 -0.143370E-05  0.404E-02    2.99       1.0  T\n",
            "   3    -71.2335454  0.604883E-06  0.670E-03    3.00       2.3  T\n",
            "   4    -71.2335462 -0.783817E-06  0.328E-03    2.99       4.7  T\n",
            "   5    -71.2335465 -0.315806E-06  0.105E-03    2.99      14.5  T\n",
            "   6    -71.2335465 -0.738426E-08  0.402E-04    2.99      37.9  T\n",
            "   7    -71.2335465 -0.154850E-08  0.207E-04    2.99      73.5  T\n",
            "     SCC iter.                  ...        0 min,  0.031 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3075722 Eh     change       -0.7581905E-04 Eh\n",
            "   gradient norm :     0.0040452 Eh/Œ±   predicted    -0.5132782E-04 ( -32.30%)\n",
            "   displ. norm   :     0.2751124 Œ±      lambda       -0.9943550E-04\n",
            "   maximum displ.:     0.2088595 Œ±      in ANC's #3, #4, #8, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   18 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2346538 -0.712347E+02  0.701E-02    3.00       0.0  T\n",
            "   2    -71.2346562 -0.241079E-05  0.405E-02    3.00       1.0  T\n",
            "   3    -71.2346552  0.948648E-06  0.917E-03    3.00       1.7  T\n",
            "   4    -71.2346565 -0.120926E-05  0.371E-03    3.00       4.1  T\n",
            "   5    -71.2346569 -0.397169E-06  0.129E-03    3.00      11.8  T\n",
            "   6    -71.2346569 -0.146048E-07  0.494E-04    3.00      30.8  T\n",
            "   7    -71.2346569 -0.271433E-08  0.217E-04    3.00      70.2  T\n",
            "     SCC iter.                  ...        0 min,  0.036 sec\n",
            "     gradient                   ...        0 min,  0.038 sec\n",
            " * total energy  :   -70.3076457 Eh     change       -0.7353771E-04 Eh\n",
            "   gradient norm :     0.0032339 Eh/Œ±   predicted    -0.5348145E-04 ( -27.27%)\n",
            "   displ. norm   :     0.3123564 Œ±      lambda       -0.8512157E-04\n",
            "   maximum displ.:     0.2429399 Œ±      in ANC's #3, #4, #8, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   19 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2335081 -0.712335E+02  0.772E-02    3.01       0.0  T\n",
            "   2    -71.2335102 -0.210033E-05  0.450E-02    3.01       1.0  T\n",
            "   3    -71.2335096  0.630630E-06  0.728E-03    3.01       2.1  T\n",
            "   4    -71.2335106 -0.996109E-06  0.332E-03    3.01       4.6  T\n",
            "   5    -71.2335109 -0.287104E-06  0.123E-03    3.01      12.4  T\n",
            "   6    -71.2335109 -0.161366E-07  0.467E-04    3.01      32.7  T\n",
            "   7    -71.2335109 -0.214261E-08  0.228E-04    3.01      67.0  T\n",
            "     SCC iter.                  ...        0 min,  0.028 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3077008 Eh     change       -0.5505650E-04 Eh\n",
            "   gradient norm :     0.0038590 Eh/Œ±   predicted    -0.4671381E-04 ( -15.15%)\n",
            "   displ. norm   :     0.1563607 Œ±      lambda       -0.5849015E-04\n",
            "   maximum displ.:     0.1279617 Œ±      in ANC's #3, #8, #5, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   20 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2350458 -0.712350E+02  0.377E-02    3.02       0.0  T\n",
            "   2    -71.2350466 -0.795053E-06  0.221E-02    3.02       1.0  T\n",
            "   3    -71.2350463  0.319181E-06  0.473E-03    3.02       3.2  T\n",
            "   4    -71.2350468 -0.482486E-06  0.164E-03    3.02       9.3  T\n",
            "   5    -71.2350468 -0.531600E-07  0.725E-04    3.02      21.0  T\n",
            "   6    -71.2350468 -0.667868E-08  0.270E-04    3.02      56.4  T\n",
            "     SCC iter.                  ...        0 min,  0.023 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3077434 Eh     change       -0.4268163E-04 Eh\n",
            "   gradient norm :     0.0026140 Eh/Œ±   predicted    -0.2996380E-04 ( -29.80%)\n",
            "   displ. norm   :     0.2629727 Œ±      lambda       -0.6241255E-04\n",
            "   maximum displ.:     0.2238302 Œ±      in ANC's #3, #8, #5, ...\n",
            " * RMSD in coord.:     0.6366457 Œ±      energy gain  -0.1788006E-01 Eh\n",
            "\n",
            "generating ANC from model Hessian ...\n",
            "Using Lindh-Hessian (1995)\n",
            " Shifting diagonal of input Hessian by   9.820694463817141E-003\n",
            " Lowest  eigenvalues of input Hessian\n",
            "    0.000000    0.000000    0.000000    0.000000    0.000000    0.000000\n",
            "    0.010000    0.010077    0.010319    0.010791    0.011735    0.012204\n",
            "    0.012543    0.013147    0.014053    0.014687    0.015694    0.015956\n",
            " Highest eigenvalues\n",
            "    2.046081    2.052884    2.141369    2.156507    2.256505    2.503022\n",
            " \n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   21 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2347605 -0.712348E+02  0.618E-02    3.02       0.0  T\n",
            "   2    -71.2347617 -0.121110E-05  0.362E-02    3.02       1.0  T\n",
            "   3    -71.2347614  0.340313E-06  0.566E-03    3.02       2.7  T\n",
            "   4    -71.2347621 -0.664319E-06  0.221E-03    3.02       6.9  T\n",
            "   5    -71.2347621 -0.763181E-07  0.983E-04    3.02      15.5  T\n",
            "   6    -71.2347622 -0.172674E-07  0.337E-04    3.02      45.2  T\n",
            "     SCC iter.                  ...        0 min,  0.025 sec\n",
            "     gradient                   ...        0 min,  0.043 sec\n",
            " * total energy  :   -70.3077915 Eh     change       -0.4803160E-04 Eh\n",
            "   gradient norm :     0.0026658 Eh/Œ±   predicted     0.0000000E+00 (-100.00%)\n",
            "   displ. norm   :     0.0396779 Œ±      lambda       -0.3094355E-04\n",
            "   maximum displ.:     0.0219867 Œ±      in ANC's #3, #4, #10, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   22 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2350886 -0.712351E+02  0.995E-03    3.02       0.0  T\n",
            "   2    -71.2350886 -0.527860E-07  0.576E-03    3.02       2.6  T\n",
            "   3    -71.2350886  0.487319E-07  0.154E-03    3.02       9.9  T\n",
            "   4    -71.2350887 -0.653791E-07  0.600E-04    3.02      25.4  T\n",
            "   5    -71.2350887 -0.421161E-08  0.185E-04    3.02      82.6  T\n",
            "     SCC iter.                  ...        0 min,  0.019 sec\n",
            "     gradient                   ...        0 min,  0.054 sec\n",
            " * total energy  :   -70.3078184 Eh     change       -0.2697573E-04 Eh\n",
            "   gradient norm :     0.0015568 Eh/Œ±   predicted    -0.1551413E-04 ( -42.49%)\n",
            "   displ. norm   :     0.2202502 Œ±      lambda       -0.8070734E-04\n",
            "   maximum displ.:     0.1199575 Œ±      in ANC's #3, #4, #10, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   23 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2353718 -0.712354E+02  0.535E-02    3.01       0.0  T\n",
            "   2    -71.2353731 -0.129759E-05  0.305E-02    3.00       1.0  T\n",
            "   3    -71.2353724  0.621603E-06  0.676E-03    3.00       2.3  T\n",
            "   4    -71.2353733 -0.806532E-06  0.350E-03    3.00       4.4  T\n",
            "   5    -71.2353735 -0.227988E-06  0.986E-04    3.00      15.5  T\n",
            "   6    -71.2353735 -0.988587E-08  0.436E-04    3.00      35.0  T\n",
            "     SCC iter.                  ...        0 min,  0.023 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3078762 Eh     change       -0.5779657E-04 Eh\n",
            "   gradient norm :     0.0031286 Eh/Œ±   predicted    -0.4122547E-04 ( -28.67%)\n",
            "   displ. norm   :     0.0381323 Œ±      lambda       -0.1875679E-04\n",
            "   maximum displ.:     0.0245654 Œ±      in ANC's #3, #4, #7, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   24 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2351901 -0.712352E+02  0.878E-03    3.00       0.0  T\n",
            "   2    -71.2351902 -0.465941E-07  0.477E-03    3.00       3.2  T\n",
            "   3    -71.2351901  0.318459E-07  0.155E-03    3.00       9.8  T\n",
            "   4    -71.2351902 -0.446659E-07  0.624E-04    3.00      24.4  T\n",
            "   5    -71.2351902 -0.553412E-08  0.222E-04    3.00      68.8  T\n",
            "     SCC iter.                  ...        0 min,  0.020 sec\n",
            "     gradient                   ...        0 min,  0.035 sec\n",
            " * total energy  :   -70.3078932 Eh     change       -0.1696513E-04 Eh\n",
            "   gradient norm :     0.0023548 Eh/Œ±   predicted    -0.9395960E-05 ( -44.62%)\n",
            "   displ. norm   :     0.1230486 Œ±      lambda       -0.5214795E-04\n",
            "   maximum displ.:     0.0783986 Œ±      in ANC's #3, #4, #7, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   25 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2347858 -0.712348E+02  0.277E-02    3.00       0.0  T\n",
            "   2    -71.2347862 -0.417659E-06  0.150E-02    3.01       1.0  T\n",
            "   3    -71.2347860  0.219164E-06  0.416E-03    3.00       3.7  T\n",
            "   4    -71.2347863 -0.314636E-06  0.192E-03    3.01       7.9  T\n",
            "   5    -71.2347863 -0.513263E-07  0.568E-04    3.01      26.8  T\n",
            "   6    -71.2347863 -0.526299E-08  0.186E-04    3.01      82.2  T\n",
            "     SCC iter.                  ...        0 min,  0.029 sec\n",
            "     gradient                   ...        0 min,  0.044 sec\n",
            " * total energy  :   -70.3079298 Eh     change       -0.3661454E-04 Eh\n",
            "   gradient norm :     0.0010855 Eh/Œ±   predicted    -0.2648409E-04 ( -27.67%)\n",
            "   displ. norm   :     0.1734929 Œ±      lambda       -0.2318724E-04\n",
            "   maximum displ.:     0.1132869 Œ±      in ANC's #3, #4, #7, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   26 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2344571 -0.712345E+02  0.363E-02    3.01       0.0  T\n",
            "   2    -71.2344577 -0.535659E-06  0.198E-02    3.01       1.0  T\n",
            "   3    -71.2344574  0.227115E-06  0.447E-03    3.01       3.4  T\n",
            "   4    -71.2344577 -0.282130E-06  0.257E-03    3.01       5.9  T\n",
            "   5    -71.2344578 -0.130419E-06  0.549E-04    3.01      27.8  T\n",
            "   6    -71.2344578 -0.465829E-08  0.230E-04    3.01      66.2  T\n",
            "     SCC iter.                  ...        0 min,  0.024 sec\n",
            "     gradient                   ...        0 min,  0.038 sec\n",
            " * total energy  :   -70.3079532 Eh     change       -0.2339297E-04 Eh\n",
            "   gradient norm :     0.0021349 Eh/Œ±   predicted    -0.1175032E-04 ( -49.77%)\n",
            "   displ. norm   :     0.0701013 Œ±      lambda       -0.1437839E-04\n",
            "   maximum displ.:     0.0478180 Œ±      in ANC's #3, #4, #2, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   27 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2347028 -0.712347E+02  0.143E-02    3.01       0.0  T\n",
            "   2    -71.2347029 -0.775900E-07  0.792E-03    3.01       1.9  T\n",
            "   3    -71.2347028  0.847630E-07  0.207E-03    3.01       7.4  T\n",
            "   4    -71.2347029 -0.105480E-06  0.101E-03    3.01      15.1  T\n",
            "   5    -71.2347029 -0.156599E-07  0.175E-04    3.01      87.0  T\n",
            "   6    -71.2347029 -0.301412E-09  0.975E-05    3.01     156.4  T\n",
            "     SCC iter.                  ...        0 min,  0.026 sec\n",
            "     gradient                   ...        0 min,  0.035 sec\n",
            " * total energy  :   -70.3079662 Eh     change       -0.1301605E-04 Eh\n",
            "   gradient norm :     0.0016261 Eh/Œ±   predicted    -0.7228282E-05 ( -44.47%)\n",
            "   displ. norm   :     0.1594474 Œ±      lambda       -0.2404167E-04\n",
            "   maximum displ.:     0.1103978 Œ±      in ANC's #3, #4, #2, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   28 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2354787 -0.712355E+02  0.323E-02    3.01       0.0  T\n",
            "   2    -71.2354792 -0.429953E-06  0.181E-02    3.01       1.0  T\n",
            "   3    -71.2354786  0.557508E-06  0.504E-03    3.01       3.0  T\n",
            "   4    -71.2354793 -0.697378E-06  0.220E-03    3.01       6.9  T\n",
            "   5    -71.2354794 -0.695820E-07  0.435E-04    3.01      35.1  T\n",
            "   6    -71.2354794 -0.221131E-08  0.236E-04    3.01      64.5  T\n",
            "     SCC iter.                  ...        0 min,  0.023 sec\n",
            "     gradient                   ...        0 min,  0.035 sec\n",
            " * total energy  :   -70.3079921 Eh     change       -0.2587147E-04 Eh\n",
            "   gradient norm :     0.0007876 Eh/Œ±   predicted    -0.1215793E-04 ( -53.01%)\n",
            "   displ. norm   :     0.1182883 Œ±      lambda       -0.1131575E-04\n",
            "   maximum displ.:     0.0856858 Œ±      in ANC's #3, #2, #4, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   29 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2356352 -0.712356E+02  0.230E-02    3.01       0.0  T\n",
            "   2    -71.2356354 -0.205316E-06  0.129E-02    3.01       1.2  T\n",
            "   3    -71.2356353  0.106226E-06  0.278E-03    3.01       5.5  T\n",
            "   4    -71.2356354 -0.121849E-06  0.156E-03    3.01       9.8  T\n",
            "   5    -71.2356355 -0.568907E-07  0.324E-04    3.01      47.1  T\n",
            "   6    -71.2356355 -0.623075E-09  0.166E-04    3.01      92.1  T\n",
            "     SCC iter.                  ...        0 min,  0.024 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3080064 Eh     change       -0.1427420E-04 Eh\n",
            "   gradient norm :     0.0010219 Eh/Œ±   predicted    -0.5694489E-05 ( -60.11%)\n",
            "   displ. norm   :     0.0837744 Œ±      lambda       -0.8978624E-05\n",
            "   maximum displ.:     0.0623343 Œ±      in ANC's #3, #2, #8, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   30 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2357037 -0.712357E+02  0.158E-02    3.01       0.0  T\n",
            "   2    -71.2357038 -0.121925E-06  0.887E-03    3.01       1.7  T\n",
            "   3    -71.2357038  0.110868E-07  0.189E-03    3.01       8.1  T\n",
            "   4    -71.2357038 -0.216070E-07  0.895E-04    3.01      17.0  T\n",
            "   5    -71.2357038 -0.186208E-07  0.294E-04    3.01      52.0  T\n",
            "     SCC iter.                  ...        0 min,  0.019 sec\n",
            "     gradient                   ...        0 min,  0.035 sec\n",
            " * total energy  :   -70.3080184 Eh     change       -0.1197875E-04 Eh\n",
            "   gradient norm :     0.0007458 Eh/Œ±   predicted    -0.4504473E-05 ( -62.40%)\n",
            "   displ. norm   :     0.0862937 Œ±      lambda       -0.9754005E-05\n",
            "   maximum displ.:     0.0634013 Œ±      in ANC's #3, #2, #8, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   31 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2356000 -0.712356E+02  0.160E-02    3.01       0.0  T\n",
            "   2    -71.2356001 -0.172059E-06  0.886E-03    3.01       1.7  T\n",
            "   3    -71.2356001  0.237211E-07  0.245E-03    3.01       6.2  T\n",
            "   4    -71.2356001 -0.386650E-07  0.951E-04    3.01      16.0  T\n",
            "   5    -71.2356002 -0.227835E-07  0.367E-04    3.01      41.6  T\n",
            "     SCC iter.                  ...        0 min,  0.019 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3080312 Eh     change       -0.1281605E-04 Eh\n",
            "   gradient norm :     0.0004431 Eh/Œ±   predicted    -0.4893464E-05 ( -61.82%)\n",
            "   displ. norm   :     0.0991508 Œ±      lambda       -0.1019225E-04\n",
            "   maximum displ.:     0.0714819 Œ±      in ANC's #3, #2, #7, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   32 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2354539 -0.712355E+02  0.179E-02    3.01       0.0  T\n",
            "   2    -71.2354541 -0.229533E-06  0.990E-03    3.01       1.5  T\n",
            "   3    -71.2354541  0.487756E-07  0.304E-03    3.01       5.0  T\n",
            "   4    -71.2354541 -0.683088E-07  0.116E-03    3.01      13.2  T\n",
            "   5    -71.2354542 -0.350903E-07  0.428E-04    3.01      35.6  T\n",
            "   6    -71.2354542 -0.188138E-08  0.173E-04    3.01      88.4  T\n",
            "     SCC iter.                  ...        0 min,  0.031 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3080444 Eh     change       -0.1324297E-04 Eh\n",
            "   gradient norm :     0.0005252 Eh/Œ±   predicted    -0.5120839E-05 ( -61.33%)\n",
            "   displ. norm   :     0.1079443 Œ±      lambda       -0.1002882E-04\n",
            "   maximum displ.:     0.0765407 Œ±      in ANC's #3, #2, #7, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   33 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2353290 -0.712353E+02  0.189E-02    3.01       0.0  T\n",
            "   2    -71.2353292 -0.244326E-06  0.105E-02    3.01       1.5  T\n",
            "   3    -71.2353291  0.799112E-07  0.319E-03    3.01       4.8  T\n",
            "   4    -71.2353292 -0.100267E-06  0.129E-03    3.01      11.8  T\n",
            "   5    -71.2353293 -0.465211E-07  0.469E-04    3.01      32.5  T\n",
            "   6    -71.2353293 -0.144388E-08  0.180E-04    3.01      84.6  T\n",
            "     SCC iter.                  ...        0 min,  0.023 sec\n",
            "     gradient                   ...        0 min,  0.033 sec\n",
            " * total energy  :   -70.3080578 Eh     change       -0.1336498E-04 Eh\n",
            "   gradient norm :     0.0007050 Eh/Œ±   predicted    -0.5041153E-05 ( -62.28%)\n",
            "   displ. norm   :     0.1237223 Œ±      lambda       -0.1123459E-04\n",
            "   maximum displ.:     0.0865793 Œ±      in ANC's #3, #2, #7, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   34 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2351665 -0.712352E+02  0.212E-02    3.01       0.0  T\n",
            "   2    -71.2351667 -0.236629E-06  0.120E-02    3.01       1.3  T\n",
            "   3    -71.2351666  0.115917E-06  0.314E-03    3.01       4.8  T\n",
            "   4    -71.2351668 -0.144077E-06  0.133E-03    3.01      11.4  T\n",
            "   5    -71.2351668 -0.480757E-07  0.490E-04    3.01      31.1  T\n",
            "   6    -71.2351668 -0.140518E-08  0.185E-04    3.01      82.5  T\n",
            "     SCC iter.                  ...        0 min,  0.029 sec\n",
            "     gradient                   ...        0 min,  0.035 sec\n",
            " * total energy  :   -70.3080733 Eh     change       -0.1557196E-04 Eh\n",
            "   gradient norm :     0.0007825 Eh/Œ±   predicted    -0.5656621E-05 ( -63.67%)\n",
            "   displ. norm   :     0.1707697 Œ±      lambda       -0.1578550E-04\n",
            "   maximum displ.:     0.1180573 Œ±      in ANC's #3, #2, #4, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   35 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2349885 -0.712350E+02  0.293E-02    3.01       0.0  T\n",
            "   2    -71.2349888 -0.371580E-06  0.167E-02    3.01       1.0  T\n",
            "   3    -71.2349886  0.222673E-06  0.380E-03    3.01       4.0  T\n",
            "   4    -71.2349889 -0.279881E-06  0.171E-03    3.01       8.9  T\n",
            "   5    -71.2349890 -0.759596E-07  0.628E-04    3.01      24.3  T\n",
            "   6    -71.2349890 -0.212614E-08  0.236E-04    3.01      64.5  T\n",
            "     SCC iter.                  ...        0 min,  0.027 sec\n",
            "     gradient                   ...        0 min,  0.035 sec\n",
            " * total energy  :   -70.3080953 Eh     change       -0.2198459E-04 Eh\n",
            "   gradient norm :     0.0007782 Eh/Œ±   predicted    -0.7997196E-05 ( -63.62%)\n",
            "   displ. norm   :     0.2667064 Œ±      lambda       -0.2400234E-04\n",
            "   maximum displ.:     0.1837006 Œ±      in ANC's #3, #2, #4, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   36 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2345687 -0.712346E+02  0.462E-02    3.01       0.0  T\n",
            "   2    -71.2345695 -0.809188E-06  0.265E-02    3.01       1.0  T\n",
            "   3    -71.2345691  0.463886E-06  0.532E-03    3.01       2.9  T\n",
            "   4    -71.2345697 -0.608162E-06  0.247E-03    3.01       6.2  T\n",
            "   5    -71.2345698 -0.154099E-06  0.908E-04    3.01      16.8  T\n",
            "   6    -71.2345698 -0.457376E-08  0.347E-04    3.01      44.0  T\n",
            "     SCC iter.                  ...        0 min,  0.025 sec\n",
            "     gradient                   ...        0 min,  0.049 sec\n",
            " * total energy  :   -70.3081279 Eh     change       -0.3254037E-04 Eh\n",
            "   gradient norm :     0.0008454 Eh/Œ±   predicted    -0.1238113E-04 ( -61.95%)\n",
            "   displ. norm   :     0.4116017 Œ±      lambda       -0.3335405E-04\n",
            "   maximum displ.:     0.2846880 Œ±      in ANC's #3, #2, #4, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   37 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2337460 -0.712337E+02  0.720E-02    3.02       0.0  T\n",
            "   2    -71.2337480 -0.193681E-05  0.413E-02    3.01       1.0  T\n",
            "   3    -71.2337470  0.942435E-06  0.789E-03    3.02       1.9  T\n",
            "   4    -71.2337483 -0.128144E-05  0.371E-03    3.01       4.1  T\n",
            "   5    -71.2337487 -0.353153E-06  0.138E-03    3.01      11.1  T\n",
            "   6    -71.2337487 -0.106243E-07  0.525E-04    3.01      29.0  T\n",
            "   7    -71.2337487 -0.132478E-08  0.276E-04    3.01      55.2  T\n",
            "     SCC iter.                  ...        0 min,  0.028 sec\n",
            "     gradient                   ...        0 min,  0.033 sec\n",
            " * total energy  :   -70.3081702 Eh     change       -0.4235956E-04 Eh\n",
            "   gradient norm :     0.0012855 Eh/Œ±   predicted    -0.1793340E-04 ( -57.66%)\n",
            "   displ. norm   :     0.5784012 Œ±      lambda       -0.3893413E-04\n",
            "   maximum displ.:     0.4028663 Œ±      in ANC's #3, #2, #4, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   38 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2322373 -0.712322E+02  0.101E-01    3.02       0.0  T\n",
            "   2    -71.2322412 -0.394780E-05  0.580E-02    3.02       1.0  T\n",
            "   3    -71.2322395  0.170303E-05  0.110E-02    3.02       1.4  T\n",
            "   4    -71.2322419 -0.237196E-05  0.511E-03    3.02       3.0  T\n",
            "   5    -71.2322426 -0.683760E-06  0.193E-03    3.02       7.9  T\n",
            "   6    -71.2322426 -0.214428E-07  0.736E-04    3.02      20.7  T\n",
            "   7    -71.2322426 -0.281577E-08  0.380E-04    3.02      40.1  T\n",
            "     SCC iter.                  ...        0 min,  0.028 sec\n",
            "     gradient                   ...        0 min,  0.037 sec\n",
            " * total energy  :   -70.3082124 Eh     change       -0.4218148E-04 Eh\n",
            "   gradient norm :     0.0023336 Eh/Œ±   predicted    -0.2236231E-04 ( -46.99%)\n",
            "   displ. norm   :     0.2742384 Œ±      lambda       -0.2871162E-04\n",
            "   maximum displ.:     0.1907814 Œ±      in ANC's #3, #2, #4, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   39 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2328396 -0.712328E+02  0.475E-02    3.02       0.0  T\n",
            "   2    -71.2328406 -0.921080E-06  0.271E-02    3.02       1.0  T\n",
            "   3    -71.2328403  0.294851E-06  0.502E-03    3.02       3.0  T\n",
            "   4    -71.2328407 -0.450119E-06  0.237E-03    3.02       6.4  T\n",
            "   5    -71.2328409 -0.145991E-06  0.891E-04    3.02      17.1  T\n",
            "   6    -71.2328409 -0.493480E-08  0.347E-04    3.02      44.0  T\n",
            "     SCC iter.                  ...        0 min,  0.023 sec\n",
            "     gradient                   ...        0 min,  0.035 sec\n",
            " * total energy  :   -70.3082369 Eh     change       -0.2449798E-04 Eh\n",
            "   gradient norm :     0.0018894 Eh/Œ±   predicted    -0.1543663E-04 ( -36.99%)\n",
            "   displ. norm   :     0.6442750 Œ±      lambda       -0.4728108E-04\n",
            "   maximum displ.:     0.4468557 Œ±      in ANC's #3, #2, #6, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   40 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2337297 -0.712337E+02  0.111E-01    3.03       0.0  T\n",
            "   2    -71.2337350 -0.531313E-05  0.636E-02    3.02       1.0  T\n",
            "   3    -71.2337334  0.163673E-05  0.121E-02    3.03       1.3  T\n",
            "   4    -71.2337359 -0.251827E-05  0.555E-03    3.02       2.7  T\n",
            "   5    -71.2337367 -0.793482E-06  0.210E-03    3.02       7.3  T\n",
            "   6    -71.2337367 -0.271467E-07  0.816E-04    3.02      18.7  T\n",
            "   7    -71.2337367 -0.286916E-08  0.405E-04    3.02      37.7  T\n",
            "     SCC iter.                  ...        0 min,  0.027 sec\n",
            "     gradient                   ...        0 min,  0.035 sec\n",
            " * total energy  :   -70.3082893 Eh     change       -0.5236540E-04 Eh\n",
            "   gradient norm :     0.0012961 Eh/Œ±   predicted    -0.2800602E-04 ( -46.52%)\n",
            "   displ. norm   :     0.5322463 Œ±      lambda       -0.3304389E-04\n",
            "   maximum displ.:     0.3675681 Œ±      in ANC's #2, #3, #7, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   41 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2337635 -0.712338E+02  0.903E-02    3.03       0.0  T\n",
            "   2    -71.2337672 -0.371813E-05  0.512E-02    3.03       1.0  T\n",
            "   3    -71.2337661  0.104843E-05  0.103E-02    3.03       1.5  T\n",
            "   4    -71.2337678 -0.163228E-05  0.467E-03    3.03       3.3  T\n",
            "   5    -71.2337683 -0.551174E-06  0.175E-03    3.03       8.7  T\n",
            "   6    -71.2337683 -0.192222E-07  0.700E-04    3.03      21.8  T\n",
            "   7    -71.2337684 -0.170070E-08  0.333E-04    3.03      45.8  T\n",
            "     SCC iter.                  ...        0 min,  0.027 sec\n",
            "     gradient                   ...        0 min,  0.037 sec\n",
            " * total energy  :   -70.3083295 Eh     change       -0.4020886E-04 Eh\n",
            "   gradient norm :     0.0011734 Eh/Œ±   predicted    -0.1860533E-04 ( -53.73%)\n",
            "   displ. norm   :     0.5201520 Œ±      lambda       -0.3886535E-04\n",
            "   maximum displ.:     0.3592995 Œ±      in ANC's #2, #3, #7, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   42 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2368246 -0.712368E+02  0.865E-02    3.04       0.0  T\n",
            "   2    -71.2368283 -0.377856E-05  0.486E-02    3.03       1.0  T\n",
            "   3    -71.2368266  0.176273E-05  0.116E-02    3.04       1.3  T\n",
            "   4    -71.2368290 -0.241848E-05  0.515E-03    3.03       3.0  T\n",
            "   5    -71.2368296 -0.602515E-06  0.169E-03    3.03       9.0  T\n",
            "   6    -71.2368296 -0.160460E-07  0.731E-04    3.03      20.9  T\n",
            "   7    -71.2368296 -0.195796E-08  0.377E-04    3.03      40.4  T\n",
            "     SCC iter.                  ...        0 min,  0.028 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3083648 Eh     change       -0.3536187E-04 Eh\n",
            "   gradient norm :     0.0020393 Eh/Œ±   predicted    -0.2177488E-04 ( -38.42%)\n",
            "   displ. norm   :     0.7616158 Œ±      lambda       -0.5967200E-04\n",
            "   maximum displ.:     0.5314606 Œ±      in ANC's #2, #3, #8, ...\n",
            " * RMSD in coord.:     0.7870869 Œ±      energy gain  -0.6214131E-03 Eh\n",
            "\n",
            "generating ANC from model Hessian ...\n",
            "Using Lindh-Hessian (1995)\n",
            " Shifting diagonal of input Hessian by   9.830451729244660E-003\n",
            " Lowest  eigenvalues of input Hessian\n",
            "    0.000000    0.000000    0.000000    0.000000    0.000000    0.000000\n",
            "    0.010000    0.010079    0.010324    0.010751    0.011645    0.012466\n",
            "    0.012517    0.013120    0.014031    0.014371    0.015654    0.015881\n",
            " Highest eigenvalues\n",
            "    2.041577    2.047915    2.137021    2.149026    2.251932    2.483913\n",
            " \n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   43 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2300994 -0.712301E+02  0.129E-01    3.04       0.0  T\n",
            "   2    -71.2301082 -0.879590E-05  0.729E-02    3.04       1.0  T\n",
            "   3    -71.2301045  0.372315E-05  0.174E-02    3.04       1.0  T\n",
            "   4    -71.2301096 -0.508782E-05  0.681E-03    3.04       2.2  T\n",
            "   5    -71.2301108 -0.121246E-05  0.279E-03    3.04       5.5  T\n",
            "   6    -71.2301108 -0.654313E-07  0.110E-03    3.04      13.8  T\n",
            "   7    -71.2301108 -0.547620E-08  0.532E-04    3.04      28.7  T\n",
            "   8    -71.2301108 -0.179577E-08  0.253E-04    3.04      60.2  T\n",
            "     SCC iter.                  ...        0 min,  0.032 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3083766 Eh     change       -0.1179774E-04 Eh\n",
            "   gradient norm :     0.0039889 Eh/Œ±   predicted     0.0000000E+00 (-100.00%)\n",
            "   displ. norm   :     0.0272680 Œ±      lambda       -0.4899130E-04\n",
            "   maximum displ.:     0.0111984 Œ±      in ANC's #2, #7, #6, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   44 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2339724 -0.712340E+02  0.123E-02    3.04       0.0  T\n",
            "   2    -71.2339726 -0.205377E-06  0.724E-03    3.04       2.1  T\n",
            "   3    -71.2339724  0.199388E-06  0.328E-03    3.04       4.7  T\n",
            "   4    -71.2339727 -0.281714E-06  0.643E-04    3.04      23.7  T\n",
            "   5    -71.2339727 -0.427393E-08  0.315E-04    3.04      48.5  T\n",
            "     SCC iter.                  ...        0 min,  0.019 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3084085 Eh     change       -0.3185092E-04 Eh\n",
            "   gradient norm :     0.0010489 Eh/Œ±   predicted    -0.2465203E-04 ( -22.60%)\n",
            "   displ. norm   :     0.0474971 Œ±      lambda       -0.1234734E-04\n",
            "   maximum displ.:     0.0257888 Œ±      in ANC's #2, #7, #6, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   45 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2362679 -0.712363E+02  0.114E-02    3.04       0.0  T\n",
            "   2    -71.2362680 -0.137763E-06  0.663E-03    3.04       2.3  T\n",
            "   3    -71.2362679  0.741687E-07  0.239E-03    3.04       6.4  T\n",
            "   4    -71.2362680 -0.112707E-06  0.581E-04    3.04      26.3  T\n",
            "   5    -71.2362681 -0.554344E-08  0.281E-04    3.04      54.4  T\n",
            "     SCC iter.                  ...        0 min,  0.022 sec\n",
            "     gradient                   ...        0 min,  0.042 sec\n",
            " * total energy  :   -70.3084191 Eh     change       -0.1060619E-04 Eh\n",
            "   gradient norm :     0.0011436 Eh/Œ±   predicted    -0.6180389E-05 ( -41.73%)\n",
            "   displ. norm   :     0.0506455 Œ±      lambda       -0.7496774E-05\n",
            "   maximum displ.:     0.0294176 Œ±      in ANC's #2, #7, #3, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   46 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2360751 -0.712361E+02  0.101E-02    3.04       0.0  T\n",
            "   2    -71.2360752 -0.624537E-07  0.584E-03    3.04       2.6  T\n",
            "   3    -71.2360751  0.370067E-07  0.149E-03    3.04      10.3  T\n",
            "   4    -71.2360752 -0.480997E-07  0.581E-04    3.04      26.2  T\n",
            "   5    -71.2360752 -0.908921E-08  0.195E-04    3.04      78.4  T\n",
            "     SCC iter.                  ...        0 min,  0.019 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3084290 Eh     change       -0.9914890E-05 Eh\n",
            "   gradient norm :     0.0010341 Eh/Œ±   predicted    -0.3762601E-05 ( -62.05%)\n",
            "   displ. norm   :     0.1273103 Œ±      lambda       -0.1643038E-04\n",
            "   maximum displ.:     0.0757783 Œ±      in ANC's #2, #7, #3, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   47 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2348065 -0.712348E+02  0.247E-02    3.03       0.0  T\n",
            "   2    -71.2348068 -0.293015E-06  0.144E-02    3.03       1.1  T\n",
            "   3    -71.2348066  0.241049E-06  0.319E-03    3.03       4.8  T\n",
            "   4    -71.2348069 -0.327414E-06  0.115E-03    3.03      13.3  T\n",
            "   5    -71.2348069 -0.256387E-07  0.386E-04    3.03      39.5  T\n",
            "   6    -71.2348069 -0.888392E-09  0.160E-04    3.03      95.0  T\n",
            "     SCC iter.                  ...        0 min,  0.023 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3084463 Eh     change       -0.1725981E-04 Eh\n",
            "   gradient norm :     0.0005675 Eh/Œ±   predicted    -0.8304721E-05 ( -51.88%)\n",
            "   displ. norm   :     0.0932275 Œ±      lambda       -0.7128708E-05\n",
            "   maximum displ.:     0.0614397 Œ±      in ANC's #2, #3, #1, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   48 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2344049 -0.712344E+02  0.173E-02    3.03       0.0  T\n",
            "   2    -71.2344050 -0.102763E-06  0.100E-02    3.03       1.5  T\n",
            "   3    -71.2344049  0.428642E-07  0.157E-03    3.03       9.7  T\n",
            "   4    -71.2344050 -0.781791E-07  0.651E-04    3.03      23.4  T\n",
            "   5    -71.2344050 -0.520480E-08  0.179E-04    3.03      85.2  T\n",
            "     SCC iter.                  ...        0 min,  0.022 sec\n",
            "     gradient                   ...        0 min,  0.035 sec\n",
            " * total energy  :   -70.3084547 Eh     change       -0.8384832E-05 Eh\n",
            "   gradient norm :     0.0006270 Eh/Œ±   predicted    -0.3578253E-05 ( -57.32%)\n",
            "   displ. norm   :     0.0584137 Œ±      lambda       -0.5285872E-05\n",
            "   maximum displ.:     0.0410931 Œ±      in ANC's #2, #3, #1, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   49 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2344404 -0.712344E+02  0.956E-03    3.03       0.0  T\n",
            "   2    -71.2344404 -0.303688E-07  0.559E-03    3.03       2.7  T\n",
            "   3    -71.2344404  0.469353E-08  0.824E-04    3.03      18.5  T\n",
            "   4    -71.2344404 -0.136149E-07  0.291E-04    3.03      52.4  T\n",
            "     SCC iter.                  ...        0 min,  0.015 sec\n",
            "     gradient                   ...        0 min,  0.044 sec\n",
            " * total energy  :   -70.3084624 Eh     change       -0.7760125E-05 Eh\n",
            "   gradient norm :     0.0005085 Eh/Œ±   predicted    -0.2650050E-05 ( -65.85%)\n",
            "   displ. norm   :     0.1150764 Œ±      lambda       -0.1231240E-04\n",
            "   maximum displ.:     0.0788116 Œ±      in ANC's #2, #3, #1, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   50 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2348005 -0.712348E+02  0.181E-02    3.03       0.0  T\n",
            "   2    -71.2348006 -0.129293E-06  0.105E-02    3.03       1.4  T\n",
            "   3    -71.2348005  0.667829E-07  0.210E-03    3.03       7.3  T\n",
            "   4    -71.2348006 -0.919279E-07  0.777E-04    3.03      19.6  T\n",
            "   5    -71.2348007 -0.152905E-07  0.300E-04    3.03      50.8  T\n",
            "     SCC iter.                  ...        0 min,  0.030 sec\n",
            "     gradient                   ...        0 min,  0.052 sec\n",
            " * total energy  :   -70.3084786 Eh     change       -0.1613607E-04 Eh\n",
            "   gradient norm :     0.0005222 Eh/Œ±   predicted    -0.6194083E-05 ( -61.61%)\n",
            "   displ. norm   :     0.1668795 Œ±      lambda       -0.1475150E-04\n",
            "   maximum displ.:     0.1143420 Œ±      in ANC's #2, #3, #1, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   51 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2349739 -0.712350E+02  0.264E-02    3.03       0.0  T\n",
            "   2    -71.2349741 -0.214053E-06  0.153E-02    3.03       1.0  T\n",
            "   3    -71.2349740  0.797664E-07  0.247E-03    3.03       6.2  T\n",
            "   4    -71.2349741 -0.131907E-06  0.100E-03    3.03      15.3  T\n",
            "   5    -71.2349741 -0.199962E-07  0.423E-04    3.03      36.0  T\n",
            "     SCC iter.                  ...        0 min,  0.020 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3084977 Eh     change       -0.1911562E-04 Eh\n",
            "   gradient norm :     0.0007316 Eh/Œ±   predicted    -0.7470723E-05 ( -60.92%)\n",
            "   displ. norm   :     0.1826586 Œ±      lambda       -0.1382407E-04\n",
            "   maximum displ.:     0.1258546 Œ±      in ANC's #2, #3, #1, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   52 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2350978 -0.712351E+02  0.295E-02    3.04       0.0  T\n",
            "   2    -71.2350980 -0.227942E-06  0.172E-02    3.03       1.0  T\n",
            "   3    -71.2350980  0.303286E-07  0.219E-03    3.03       7.0  T\n",
            "   4    -71.2350981 -0.931042E-07  0.101E-03    3.03      15.1  T\n",
            "   5    -71.2350981 -0.168119E-07  0.439E-04    3.03      34.8  T\n",
            "   6    -71.2350981 -0.209195E-08  0.162E-04    3.03      94.4  T\n",
            "     SCC iter.                  ...        0 min,  0.023 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3085165 Eh     change       -0.1878248E-04 Eh\n",
            "   gradient norm :     0.0007800 Eh/Œ±   predicted    -0.7015854E-05 ( -62.65%)\n",
            "   displ. norm   :     0.1903436 Œ±      lambda       -0.1432669E-04\n",
            "   maximum displ.:     0.1317244 Œ±      in ANC's #2, #3, #1, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   53 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2350879 -0.712351E+02  0.317E-02    3.04       0.0  T\n",
            "   2    -71.2350881 -0.254444E-06  0.185E-02    3.03       1.0  T\n",
            "   3    -71.2350881  0.180635E-07  0.218E-03    3.03       7.0  T\n",
            "   4    -71.2350882 -0.936957E-07  0.106E-03    3.03      14.4  T\n",
            "   5    -71.2350882 -0.173746E-07  0.439E-04    3.03      34.7  T\n",
            "   6    -71.2350882 -0.193710E-08  0.156E-04    3.03      98.0  T\n",
            "     SCC iter.                  ...        0 min,  0.023 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3085366 Eh     change       -0.2016086E-04 Eh\n",
            "   gradient norm :     0.0007024 Eh/Œ±   predicted    -0.7279912E-05 ( -63.89%)\n",
            "   displ. norm   :     0.2060678 Œ±      lambda       -0.1641295E-04\n",
            "   maximum displ.:     0.1434741 Œ±      in ANC's #2, #3, #1, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   54 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2350535 -0.712351E+02  0.350E-02    3.03       0.0  T\n",
            "   2    -71.2350538 -0.292451E-06  0.205E-02    3.03       1.0  T\n",
            "   3    -71.2350537  0.633075E-07  0.250E-03    3.03       6.1  T\n",
            "   4    -71.2350539 -0.161345E-06  0.115E-03    3.03      13.2  T\n",
            "   5    -71.2350539 -0.196483E-07  0.447E-04    3.03      34.1  T\n",
            "   6    -71.2350539 -0.156001E-08  0.160E-04    3.03      95.0  T\n",
            "     SCC iter.                  ...        0 min,  0.028 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3085596 Eh     change       -0.2299588E-04 Eh\n",
            "   gradient norm :     0.0005635 Eh/Œ±   predicted    -0.8361938E-05 ( -63.64%)\n",
            "   displ. norm   :     0.2216602 Œ±      lambda       -0.1778873E-04\n",
            "   maximum displ.:     0.1558182 Œ±      in ANC's #2, #3, #1, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   55 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2348918 -0.712349E+02  0.382E-02    3.03       0.0  T\n",
            "   2    -71.2348922 -0.364469E-06  0.224E-02    3.03       1.0  T\n",
            "   3    -71.2348921  0.563487E-07  0.263E-03    3.03       5.8  T\n",
            "   4    -71.2348923 -0.180725E-06  0.125E-03    3.03      12.2  T\n",
            "   5    -71.2348923 -0.219035E-07  0.442E-04    3.03      34.5  T\n",
            "   6    -71.2348923 -0.149373E-08  0.164E-04    3.03      92.8  T\n",
            "     SCC iter.                  ...        0 min,  0.023 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3085840 Eh     change       -0.2438643E-04 Eh\n",
            "   gradient norm :     0.0005652 Eh/Œ±   predicted    -0.9093374E-05 ( -62.71%)\n",
            "   displ. norm   :     0.2143350 Œ±      lambda       -0.1652415E-04\n",
            "   maximum displ.:     0.1528196 Œ±      in ANC's #2, #1, #3, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   56 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2348214 -0.712348E+02  0.365E-02    3.03       0.0  T\n",
            "   2    -71.2348217 -0.305813E-06  0.214E-02    3.03       1.0  T\n",
            "   3    -71.2348217  0.445354E-07  0.242E-03    3.03       6.3  T\n",
            "   4    -71.2348218 -0.151688E-06  0.119E-03    3.03      12.8  T\n",
            "   5    -71.2348218 -0.190445E-07  0.392E-04    3.03      38.9  T\n",
            "   6    -71.2348218 -0.126353E-08  0.157E-04    3.03      97.4  T\n",
            "     SCC iter.                  ...        0 min,  0.026 sec\n",
            "     gradient                   ...        0 min,  0.033 sec\n",
            " * total energy  :   -70.3086066 Eh     change       -0.2256308E-04 Eh\n",
            "   gradient norm :     0.0005759 Eh/Œ±   predicted    -0.8431455E-05 ( -62.63%)\n",
            "   displ. norm   :     0.1783139 Œ±      lambda       -0.1345361E-04\n",
            "   maximum displ.:     0.1296240 Œ±      in ANC's #2, #1, #3, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   57 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2348754 -0.712349E+02  0.297E-02    3.03       0.0  T\n",
            "   2    -71.2348756 -0.216847E-06  0.174E-02    3.03       1.0  T\n",
            "   3    -71.2348756 -0.530491E-10  0.182E-03    3.03       8.4  T\n",
            "   4    -71.2348757 -0.673796E-07  0.101E-03    3.03      15.0  T\n",
            "   5    -71.2348757 -0.175263E-07  0.299E-04    3.03      51.0  T\n",
            "   6    -71.2348757 -0.704659E-09  0.125E-04    3.03     122.1  T\n",
            "     SCC iter.                  ...        0 min,  0.023 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3086252 Eh     change       -0.1863173E-04 Eh\n",
            "   gradient norm :     0.0005266 Eh/Œ±   predicted    -0.6823809E-05 ( -63.38%)\n",
            "   displ. norm   :     0.1483782 Œ±      lambda       -0.1131536E-04\n",
            "   maximum displ.:     0.1097330 Œ±      in ANC's #2, #1, #3, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   58 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2349531 -0.712350E+02  0.240E-02    3.03       0.0  T\n",
            "   2    -71.2349532 -0.138739E-06  0.140E-02    3.03       1.1  T\n",
            "   3    -71.2349532 -0.106743E-08  0.150E-03    3.03      10.2  T\n",
            "   4    -71.2349533 -0.385121E-07  0.860E-04    3.03      17.7  T\n",
            "   5    -71.2349533 -0.130263E-07  0.244E-04    3.03      62.4  T\n",
            "     SCC iter.                  ...        0 min,  0.018 sec\n",
            "     gradient                   ...        0 min,  0.042 sec\n",
            " * total energy  :   -70.3086412 Eh     change       -0.1599839E-04 Eh\n",
            "   gradient norm :     0.0004322 Eh/Œ±   predicted    -0.5715031E-05 ( -64.28%)\n",
            "   displ. norm   :     0.1294767 Œ±      lambda       -0.1020448E-04\n",
            "   maximum displ.:     0.0973489 Œ±      in ANC's #2, #1, #3, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   59 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2350904 -0.712351E+02  0.198E-02    3.03       0.0  T\n",
            "   2    -71.2350905 -0.967574E-07  0.115E-02    3.03       1.3  T\n",
            "   3    -71.2350905  0.605226E-09  0.129E-03    3.03      11.8  T\n",
            "   4    -71.2350905 -0.254111E-07  0.752E-04    3.03      20.3  T\n",
            "   5    -71.2350905 -0.102874E-07  0.224E-04    3.03      68.0  T\n",
            "     SCC iter.                  ...        0 min,  0.025 sec\n",
            "     gradient                   ...        0 min,  0.035 sec\n",
            " * total energy  :   -70.3086557 Eh     change       -0.1451549E-04 Eh\n",
            "   gradient norm :     0.0003523 Eh/Œ±   predicted    -0.5141705E-05 ( -64.58%)\n",
            "   displ. norm   :     0.1198532 Œ±      lambda       -0.9562183E-05\n",
            "   maximum displ.:     0.0914264 Œ±      in ANC's #2, #1, #3, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   60 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2351107 -0.712351E+02  0.180E-02    3.03       0.0  T\n",
            "   2    -71.2351108 -0.995916E-07  0.103E-02    3.03       1.5  T\n",
            "   3    -71.2351108  0.299895E-07  0.159E-03    3.03       9.6  T\n",
            "   4    -71.2351108 -0.522890E-07  0.803E-04    3.03      19.0  T\n",
            "   5    -71.2351108 -0.118910E-07  0.246E-04    3.03      61.9  T\n",
            "     SCC iter.                  ...        0 min,  0.020 sec\n",
            "     gradient                   ...        0 min,  0.035 sec\n",
            " * total energy  :   -70.3086693 Eh     change       -0.1357940E-04 Eh\n",
            "   gradient norm :     0.0002990 Eh/Œ±   predicted    -0.4812342E-05 ( -64.56%)\n",
            "   displ. norm   :     0.1091175 Œ±      lambda       -0.8648512E-05\n",
            "   maximum displ.:     0.0846387 Œ±      in ANC's #2, #1, #3, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   61 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2351961 -0.712352E+02  0.156E-02    3.03       0.0  T\n",
            "   2    -71.2351962 -0.827379E-07  0.894E-03    3.03       1.7  T\n",
            "   3    -71.2351961  0.348161E-07  0.158E-03    3.03       9.6  T\n",
            "   4    -71.2351962 -0.501817E-07  0.774E-04    3.03      19.7  T\n",
            "   5    -71.2351962 -0.121844E-07  0.264E-04    3.03      57.8  T\n",
            "     SCC iter.                  ...        0 min,  0.019 sec\n",
            "     gradient                   ...        0 min,  0.035 sec\n",
            " * total energy  :   -70.3086816 Eh     change       -0.1232109E-04 Eh\n",
            "   gradient norm :     0.0002906 Eh/Œ±   predicted    -0.4349572E-05 ( -64.70%)\n",
            "   displ. norm   :     0.0991115 Œ±      lambda       -0.7844956E-05\n",
            "   maximum displ.:     0.0783906 Œ±      in ANC's #2, #1, #3, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   62 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2352486 -0.712352E+02  0.141E-02    3.03       0.0  T\n",
            "   2    -71.2352486 -0.864471E-07  0.804E-03    3.03       1.9  T\n",
            "   3    -71.2352486  0.444663E-07  0.176E-03    3.03       8.7  T\n",
            "   4    -71.2352487 -0.590445E-07  0.797E-04    3.03      19.1  T\n",
            "   5    -71.2352487 -0.137035E-07  0.277E-04    3.03      55.2  T\n",
            "     SCC iter.                  ...        0 min,  0.019 sec\n",
            "     gradient                   ...        0 min,  0.036 sec\n",
            " * total energy  :   -70.3086929 Eh     change       -0.1126504E-04 Eh\n",
            "   gradient norm :     0.0002991 Eh/Œ±   predicted    -0.3940399E-05 ( -65.02%)\n",
            "   displ. norm   :     0.0889227 Œ±      lambda       -0.7210803E-05\n",
            "   maximum displ.:     0.0716645 Œ±      in ANC's #2, #1, #3, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   63 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2352804 -0.712353E+02  0.127E-02    3.03       0.0  T\n",
            "   2    -71.2352804 -0.864540E-07  0.716E-03    3.03       2.1  T\n",
            "   3    -71.2352804  0.475222E-07  0.188E-03    3.03       8.1  T\n",
            "   4    -71.2352804 -0.607872E-07  0.799E-04    3.03      19.1  T\n",
            "   5    -71.2352805 -0.147327E-07  0.287E-04    3.03      53.0  T\n",
            "     SCC iter.                  ...        0 min,  0.019 sec\n",
            "     gradient                   ...        0 min,  0.033 sec\n",
            " * total energy  :   -70.3087033 Eh     change       -0.1045574E-04 Eh\n",
            "   gradient norm :     0.0003260 Eh/Œ±   predicted    -0.3618695E-05 ( -65.39%)\n",
            "   displ. norm   :     0.0820735 Œ±      lambda       -0.6974929E-05\n",
            "   maximum displ.:     0.0673787 Œ±      in ANC's #2, #1, #8, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   64 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2353169 -0.712353E+02  0.118E-02    3.03       0.0  T\n",
            "   2    -71.2353170 -0.827970E-07  0.663E-03    3.03       2.3  T\n",
            "   3    -71.2353169  0.436719E-07  0.189E-03    3.03       8.1  T\n",
            "   4    -71.2353170 -0.560121E-07  0.786E-04    3.03      19.4  T\n",
            "   5    -71.2353170 -0.146607E-07  0.291E-04    3.03      52.4  T\n",
            "     SCC iter.                  ...        0 min,  0.027 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3087135 Eh     change       -0.1021448E-04 Eh\n",
            "   gradient norm :     0.0003151 Eh/Œ±   predicted    -0.3498518E-05 ( -65.75%)\n",
            "   displ. norm   :     0.0801672 Œ±      lambda       -0.7182303E-05\n",
            "   maximum displ.:     0.0666047 Œ±      in ANC's #2, #1, #8, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   65 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2352975 -0.712353E+02  0.118E-02    3.03       0.0  T\n",
            "   2    -71.2352976 -0.913109E-07  0.664E-03    3.03       2.3  T\n",
            "   3    -71.2352975  0.461006E-07  0.201E-03    3.03       7.6  T\n",
            "   4    -71.2352976 -0.593523E-07  0.815E-04    3.03      18.7  T\n",
            "   5    -71.2352976 -0.163588E-07  0.302E-04    3.03      50.4  T\n",
            "     SCC iter.                  ...        0 min,  0.029 sec\n",
            "     gradient                   ...        0 min,  0.040 sec\n",
            " * total energy  :   -70.3087241 Eh     change       -0.1056792E-04 Eh\n",
            "   gradient norm :     0.0002986 Eh/Œ±   predicted    -0.3601795E-05 ( -65.92%)\n",
            "   displ. norm   :     0.0807788 Œ±      lambda       -0.7587709E-05\n",
            "   maximum displ.:     0.0675814 Œ±      in ANC's #2, #1, #8, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   66 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2353054 -0.712353E+02  0.120E-02    3.03       0.0  T\n",
            "   2    -71.2353055 -0.937150E-07  0.680E-03    3.03       2.2  T\n",
            "   3    -71.2353055  0.486149E-07  0.205E-03    3.03       7.4  T\n",
            "   4    -71.2353055 -0.628889E-07  0.820E-04    3.03      18.6  T\n",
            "   5    -71.2353055 -0.167154E-07  0.311E-04    3.03      49.0  T\n",
            "     SCC iter.                  ...        0 min,  0.022 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3087353 Eh     change       -0.1118051E-04 Eh\n",
            "   gradient norm :     0.0002752 Eh/Œ±   predicted    -0.3805325E-05 ( -65.96%)\n",
            "   displ. norm   :     0.0848795 Œ±      lambda       -0.8234945E-05\n",
            "   maximum displ.:     0.0709127 Œ±      in ANC's #2, #1, #3, ...\n",
            " * RMSD in coord.:     0.4084668 Œ±      energy gain  -0.3704447E-03 Eh\n",
            "\n",
            "generating ANC from model Hessian ...\n",
            "Using Lindh-Hessian (1995)\n",
            " Shifting diagonal of input Hessian by   9.819755416398372E-003\n",
            " Lowest  eigenvalues of input Hessian\n",
            "    0.000000    0.000000    0.000000    0.000000    0.000000    0.000000\n",
            "    0.010000    0.010067    0.010306    0.010753    0.011628    0.012453\n",
            "    0.012590    0.013130    0.014006    0.014298    0.015696    0.015901\n",
            " Highest eigenvalues\n",
            "    2.047028    2.052808    2.141699    2.153694    2.257326    2.484697\n",
            " \n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   67 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2352567 -0.712353E+02  0.129E-02    3.03       0.0  T\n",
            "   2    -71.2352568 -0.991833E-07  0.733E-03    3.03       2.1  T\n",
            "   3    -71.2352567  0.515067E-07  0.207E-03    3.03       7.4  T\n",
            "   4    -71.2352568 -0.667373E-07  0.841E-04    3.03      18.1  T\n",
            "   5    -71.2352568 -0.177909E-07  0.321E-04    3.03      47.5  T\n",
            "     SCC iter.                  ...        0 min,  0.019 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3087474 Eh     change       -0.1211851E-04 Eh\n",
            "   gradient norm :     0.0002447 Eh/Œ±   predicted     0.0000000E+00 (-100.00%)\n",
            "   displ. norm   :     0.0272598 Œ±      lambda       -0.9748686E-07\n",
            "   maximum displ.:     0.0244762 Œ±      in ANC's #2, #3, #5, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   68 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2352216 -0.712352E+02  0.339E-03    3.03       0.0  T\n",
            "   2    -71.2352216 -0.495774E-08  0.200E-03    3.03       7.6  T\n",
            "   3    -71.2352216  0.851290E-08  0.527E-04    3.03      28.9  T\n",
            "   4    -71.2352216 -0.108869E-07  0.162E-04    3.03      94.2  T\n",
            "     SCC iter.                  ...        0 min,  0.015 sec\n",
            "     gradient                   ...        0 min,  0.035 sec\n",
            " * total energy  :   -70.3087525 Eh     change       -0.5130896E-05 Eh\n",
            "   gradient norm :     0.0002051 Eh/Œ±   predicted    -0.1744149E-05 ( -66.01%)\n",
            "   displ. norm   :     0.0473773 Œ±      lambda       -0.5786738E-05\n",
            "   maximum displ.:     0.0374286 Œ±      in ANC's #2, #3, #4, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   69 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2351782 -0.712352E+02  0.686E-03    3.03       0.0  T\n",
            "   2    -71.2351782 -0.121414E-07  0.397E-03    3.03       3.8  T\n",
            "   3    -71.2351782  0.123689E-07  0.740E-04    3.03      20.6  T\n",
            "   4    -71.2351782 -0.160676E-07  0.339E-04    3.03      45.0  T\n",
            "     SCC iter.                  ...        0 min,  0.015 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3087609 Eh     change       -0.8396697E-05 Eh\n",
            "   gradient norm :     0.0002199 Eh/Œ±   predicted    -0.2896516E-05 ( -65.50%)\n",
            "   displ. norm   :     0.0796802 Œ±      lambda       -0.9055037E-05\n",
            "   maximum displ.:     0.0600093 Œ±      in ANC's #2, #3, #4, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   70 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2351250 -0.712351E+02  0.120E-02    3.03       0.0  T\n",
            "   2    -71.2351250 -0.358038E-07  0.691E-03    3.03       2.2  T\n",
            "   3    -71.2351250  0.168689E-07  0.112E-03    3.03      13.6  T\n",
            "   4    -71.2351250 -0.228223E-07  0.610E-04    3.03      25.0  T\n",
            "   5    -71.2351250 -0.927986E-08  0.173E-04    3.03      87.9  T\n",
            "     SCC iter.                  ...        0 min,  0.020 sec\n",
            "     gradient                   ...        0 min,  0.033 sec\n",
            " * total energy  :   -70.3087739 Eh     change       -0.1293135E-04 Eh\n",
            "   gradient norm :     0.0002325 Eh/Œ±   predicted    -0.4542731E-05 ( -64.87%)\n",
            "   displ. norm   :     0.1156487 Œ±      lambda       -0.1213356E-04\n",
            "   maximum displ.:     0.0859857 Œ±      in ANC's #2, #3, #4, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   71 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2350307 -0.712350E+02  0.176E-02    3.03       0.0  T\n",
            "   2    -71.2350307 -0.829780E-07  0.101E-02    3.03       1.5  T\n",
            "   3    -71.2350307  0.355718E-07  0.165E-03    3.03       9.2  T\n",
            "   4    -71.2350308 -0.476945E-07  0.895E-04    3.03      17.0  T\n",
            "   5    -71.2350308 -0.205076E-07  0.277E-04    3.03      55.2  T\n",
            "     SCC iter.                  ...        0 min,  0.020 sec\n",
            "     gradient                   ...        0 min,  0.041 sec\n",
            " * total energy  :   -70.3087910 Eh     change       -0.1715035E-04 Eh\n",
            "   gradient norm :     0.0002907 Eh/Œ±   predicted    -0.6110596E-05 ( -64.37%)\n",
            "   displ. norm   :     0.1420560 Œ±      lambda       -0.1386559E-04\n",
            "   maximum displ.:     0.1052034 Œ±      in ANC's #2, #3, #4, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   72 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2349538 -0.712350E+02  0.221E-02    3.03       0.0  T\n",
            "   2    -71.2349539 -0.152479E-06  0.126E-02    3.03       1.2  T\n",
            "   3    -71.2349539  0.523042E-07  0.216E-03    3.03       7.1  T\n",
            "   4    -71.2349540 -0.731329E-07  0.116E-03    3.03      13.2  T\n",
            "   5    -71.2349540 -0.338978E-07  0.375E-04    3.03      40.7  T\n",
            "   6    -71.2349540 -0.118293E-08  0.167E-04    3.03      91.1  T\n",
            "     SCC iter.                  ...        0 min,  0.022 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3088107 Eh     change       -0.1966407E-04 Eh\n",
            "   gradient norm :     0.0003421 Eh/Œ±   predicted    -0.7004021E-05 ( -64.38%)\n",
            "   displ. norm   :     0.1557880 Œ±      lambda       -0.1471004E-04\n",
            "   maximum displ.:     0.1157181 Œ±      in ANC's #2, #3, #4, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   73 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2349032 -0.712349E+02  0.242E-02    3.03       0.0  T\n",
            "   2    -71.2349034 -0.202879E-06  0.139E-02    3.03       1.1  T\n",
            "   3    -71.2349034  0.606346E-07  0.245E-03    3.03       6.2  T\n",
            "   4    -71.2349035 -0.889428E-07  0.125E-03    3.03      12.2  T\n",
            "   5    -71.2349035 -0.386817E-07  0.439E-04    3.03      34.7  T\n",
            "   6    -71.2349035 -0.184652E-08  0.188E-04    3.03      81.3  T\n",
            "     SCC iter.                  ...        0 min,  0.028 sec\n",
            "     gradient                   ...        0 min,  0.036 sec\n",
            " * total energy  :   -70.3088318 Eh     change       -0.2111855E-04 Eh\n",
            "   gradient norm :     0.0003975 Eh/Œ±   predicted    -0.7438254E-05 ( -64.78%)\n",
            "   displ. norm   :     0.1637285 Œ±      lambda       -0.1562026E-04\n",
            "   maximum displ.:     0.1217864 Œ±      in ANC's #2, #3, #8, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   74 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2349142 -0.712349E+02  0.260E-02    3.03       0.0  T\n",
            "   2    -71.2349145 -0.271057E-06  0.148E-02    3.03       1.0  T\n",
            "   3    -71.2349144  0.866511E-07  0.289E-03    3.03       5.3  T\n",
            "   4    -71.2349145 -0.123832E-06  0.141E-03    3.03      10.9  T\n",
            "   5    -71.2349146 -0.485990E-07  0.502E-04    3.03      30.4  T\n",
            "   6    -71.2349146 -0.218026E-08  0.208E-04    3.03      73.4  T\n",
            "     SCC iter.                  ...        0 min,  0.024 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3088545 Eh     change       -0.2271130E-04 Eh\n",
            "   gradient norm :     0.0004076 Eh/Œ±   predicted    -0.7905722E-05 ( -65.19%)\n",
            "   displ. norm   :     0.1780917 Œ±      lambda       -0.1768426E-04\n",
            "   maximum displ.:     0.1328811 Œ±      in ANC's #2, #3, #8, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   75 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2349320 -0.712349E+02  0.284E-02    3.03       0.0  T\n",
            "   2    -71.2349324 -0.374562E-06  0.161E-02    3.03       1.0  T\n",
            "   3    -71.2349323  0.109055E-06  0.342E-03    3.03       4.5  T\n",
            "   4    -71.2349324 -0.161059E-06  0.156E-03    3.03       9.8  T\n",
            "   5    -71.2349325 -0.578026E-07  0.583E-04    3.03      26.2  T\n",
            "   6    -71.2349325 -0.315654E-08  0.237E-04    3.03      64.3  T\n",
            "     SCC iter.                  ...        0 min,  0.022 sec\n",
            "     gradient                   ...        0 min,  0.053 sec\n",
            " * total energy  :   -70.3088804 Eh     change       -0.2588772E-04 Eh\n",
            "   gradient norm :     0.0003998 Eh/Œ±   predicted    -0.8968971E-05 ( -65.35%)\n",
            "   displ. norm   :     0.2010954 Œ±      lambda       -0.2089668E-04\n",
            "   maximum displ.:     0.1505757 Œ±      in ANC's #2, #3, #8, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   76 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2349607 -0.712350E+02  0.325E-02    3.03       0.0  T\n",
            "   2    -71.2349612 -0.543028E-06  0.184E-02    3.03       1.0  T\n",
            "   3    -71.2349611  0.172089E-06  0.424E-03    3.03       3.6  T\n",
            "   4    -71.2349613 -0.244899E-06  0.185E-03    3.03       8.2  T\n",
            "   5    -71.2349614 -0.846706E-07  0.701E-04    3.03      21.8  T\n",
            "   6    -71.2349614 -0.410955E-08  0.281E-04    3.03      54.2  T\n",
            "     SCC iter.                  ...        0 min,  0.023 sec\n",
            "     gradient                   ...        0 min,  0.036 sec\n",
            " * total energy  :   -70.3089110 Eh     change       -0.3061478E-04 Eh\n",
            "   gradient norm :     0.0003950 Eh/Œ±   predicted    -0.1063907E-04 ( -65.25%)\n",
            "   displ. norm   :     0.2330675 Œ±      lambda       -0.2507589E-04\n",
            "   maximum displ.:     0.1755132 Œ±      in ANC's #2, #3, #8, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   77 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2349389 -0.712349E+02  0.378E-02    3.03       0.0  T\n",
            "   2    -71.2349397 -0.809153E-06  0.213E-02    3.03       1.0  T\n",
            "   3    -71.2349395  0.254547E-06  0.526E-03    3.03       2.9  T\n",
            "   4    -71.2349398 -0.361287E-06  0.221E-03    3.03       6.9  T\n",
            "   5    -71.2349400 -0.120732E-06  0.843E-04    3.03      18.1  T\n",
            "   6    -71.2349400 -0.595362E-08  0.334E-04    3.03      45.7  T\n",
            "     SCC iter.                  ...        0 min,  0.024 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3089477 Eh     change       -0.3668460E-04 Eh\n",
            "   gradient norm :     0.0004256 Eh/Œ±   predicted    -0.1284481E-04 ( -64.99%)\n",
            "   displ. norm   :     0.2596525 Œ±      lambda       -0.2866217E-04\n",
            "   maximum displ.:     0.1970914 Œ±      in ANC's #2, #3, #8, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   78 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2349419 -0.712349E+02  0.421E-02    3.04       0.0  T\n",
            "   2    -71.2349429 -0.104290E-05  0.237E-02    3.03       1.0  T\n",
            "   3    -71.2349425  0.355858E-06  0.607E-03    3.04       2.5  T\n",
            "   4    -71.2349430 -0.492072E-06  0.253E-03    3.03       6.0  T\n",
            "   5    -71.2349432 -0.162650E-06  0.973E-04    3.03      15.7  T\n",
            "   6    -71.2349432 -0.721639E-08  0.381E-04    3.03      40.0  T\n",
            "     SCC iter.                  ...        0 min,  0.024 sec\n",
            "     gradient                   ...        0 min,  0.039 sec\n",
            " * total energy  :   -70.3089896 Eh     change       -0.4191219E-04 Eh\n",
            "   gradient norm :     0.0004912 Eh/Œ±   predicted    -0.1476370E-04 ( -64.77%)\n",
            "   displ. norm   :     0.2743536 Œ±      lambda       -0.3092106E-04\n",
            "   maximum displ.:     0.2104976 Œ±      in ANC's #2, #3, #4, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   79 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2349151 -0.712349E+02  0.444E-02    3.04       0.0  T\n",
            "   2    -71.2349163 -0.119982E-05  0.249E-02    3.04       1.0  T\n",
            "   3    -71.2349159  0.422927E-06  0.658E-03    3.04       2.3  T\n",
            "   4    -71.2349165 -0.579032E-06  0.272E-03    3.04       5.6  T\n",
            "   5    -71.2349167 -0.187918E-06  0.104E-03    3.04      14.7  T\n",
            "   6    -71.2349167 -0.808573E-08  0.399E-04    3.04      38.2  T\n",
            "   7    -71.2349167 -0.608694E-09  0.189E-04    3.04      80.8  T\n",
            "     SCC iter.                  ...        0 min,  0.026 sec\n",
            "     gradient                   ...        0 min,  0.056 sec\n",
            " * total energy  :   -70.3090350 Eh     change       -0.4536071E-04 Eh\n",
            "   gradient norm :     0.0005866 Eh/Œ±   predicted    -0.1598150E-04 ( -64.77%)\n",
            "   displ. norm   :     0.2642159 Œ±      lambda       -0.3075299E-04\n",
            "   maximum displ.:     0.2054322 Œ±      in ANC's #2, #3, #4, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   80 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2350090 -0.712350E+02  0.424E-02    3.04       0.0  T\n",
            "   2    -71.2350100 -0.107496E-05  0.238E-02    3.04       1.0  T\n",
            "   3    -71.2350096  0.429349E-06  0.627E-03    3.04       2.4  T\n",
            "   4    -71.2350102 -0.574456E-06  0.263E-03    3.04       5.8  T\n",
            "   5    -71.2350104 -0.178741E-06  0.101E-03    3.04      15.1  T\n",
            "   6    -71.2350104 -0.708158E-08  0.384E-04    3.04      39.8  T\n",
            "   7    -71.2350104 -0.602057E-09  0.187E-04    3.04      81.6  T\n",
            "     SCC iter.                  ...        0 min,  0.027 sec\n",
            "     gradient                   ...        0 min,  0.036 sec\n",
            " * total energy  :   -70.3090802 Eh     change       -0.4521471E-04 Eh\n",
            "   gradient norm :     0.0006003 Eh/Œ±   predicted    -0.1586147E-04 ( -64.92%)\n",
            "   displ. norm   :     0.2415909 Œ±      lambda       -0.2925039E-04\n",
            "   maximum displ.:     0.1909320 Œ±      in ANC's #2, #3, #4, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   81 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2350938 -0.712351E+02  0.385E-02    3.04       0.0  T\n",
            "   2    -71.2350947 -0.873113E-06  0.216E-02    3.04       1.0  T\n",
            "   3    -71.2350943  0.365453E-06  0.568E-03    3.04       2.7  T\n",
            "   4    -71.2350948 -0.486069E-06  0.241E-03    3.04       6.3  T\n",
            "   5    -71.2350949 -0.146953E-06  0.907E-04    3.04      16.8  T\n",
            "   6    -71.2350949 -0.582129E-08  0.338E-04    3.04      45.1  T\n",
            "     SCC iter.                  ...        0 min,  0.023 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3091233 Eh     change       -0.4312819E-04 Eh\n",
            "   gradient norm :     0.0005597 Eh/Œ±   predicted    -0.1501571E-04 ( -65.18%)\n",
            "   displ. norm   :     0.2043667 Œ±      lambda       -0.2600807E-04\n",
            "   maximum displ.:     0.1646883 Œ±      in ANC's #2, #3, #4, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   82 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2352276 -0.712352E+02  0.323E-02    3.04       0.0  T\n",
            "   2    -71.2352282 -0.600929E-06  0.181E-02    3.04       1.0  T\n",
            "   3    -71.2352279  0.277075E-06  0.476E-03    3.04       3.2  T\n",
            "   4    -71.2352283 -0.361627E-06  0.208E-03    3.04       7.3  T\n",
            "   5    -71.2352284 -0.108295E-06  0.789E-04    3.04      19.3  T\n",
            "   6    -71.2352284 -0.414259E-08  0.291E-04    3.04      52.5  T\n",
            "     SCC iter.                  ...        0 min,  0.023 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3091617 Eh     change       -0.3838159E-04 Eh\n",
            "   gradient norm :     0.0004937 Eh/Œ±   predicted    -0.1324585E-04 ( -65.49%)\n",
            "   displ. norm   :     0.1694157 Œ±      lambda       -0.2266016E-04\n",
            "   maximum displ.:     0.1397723 Œ±      in ANC's #2, #3, #4, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   83 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2353424 -0.712353E+02  0.261E-02    3.04       0.0  T\n",
            "   2    -71.2353427 -0.377829E-06  0.146E-02    3.04       1.0  T\n",
            "   3    -71.2353426  0.177145E-06  0.378E-03    3.04       4.0  T\n",
            "   4    -71.2353428 -0.232014E-06  0.169E-03    3.04       9.0  T\n",
            "   5    -71.2353429 -0.687129E-07  0.632E-04    3.04      24.1  T\n",
            "   6    -71.2353429 -0.278641E-08  0.232E-04    3.04      65.8  T\n",
            "     SCC iter.                  ...        0 min,  0.023 sec\n",
            "     gradient                   ...        0 min,  0.038 sec\n",
            " * total energy  :   -70.3091952 Eh     change       -0.3351321E-04 Eh\n",
            "   gradient norm :     0.0004248 Eh/Œ±   predicted    -0.1147923E-04 ( -65.75%)\n",
            "   displ. norm   :     0.1341323 Œ±      lambda       -0.1892351E-04\n",
            "   maximum displ.:     0.1137551 Œ±      in ANC's #2, #3, #4, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   84 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2353791 -0.712354E+02  0.204E-02    3.04       0.0  T\n",
            "   2    -71.2353793 -0.239820E-06  0.114E-02    3.04       1.3  T\n",
            "   3    -71.2353792  0.111513E-06  0.303E-03    3.04       5.0  T\n",
            "   4    -71.2353793 -0.144726E-06  0.136E-03    3.04      11.2  T\n",
            "   5    -71.2353794 -0.440322E-07  0.498E-04    3.04      30.6  T\n",
            "   6    -71.2353794 -0.179732E-08  0.181E-04    3.04      84.2  T\n",
            "     SCC iter.                  ...        0 min,  0.023 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3092232 Eh     change       -0.2795155E-04 Eh\n",
            "   gradient norm :     0.0003804 Eh/Œ±   predicted    -0.9538065E-05 ( -65.88%)\n",
            "   displ. norm   :     0.1059618 Œ±      lambda       -0.1576747E-04\n",
            "   maximum displ.:     0.0927916 Œ±      in ANC's #2, #3, #4, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   85 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2354439 -0.712354E+02  0.153E-02    3.04       0.0  T\n",
            "   2    -71.2354440 -0.132286E-06  0.854E-03    3.04       1.8  T\n",
            "   3    -71.2354439  0.675658E-07  0.229E-03    3.04       6.6  T\n",
            "   4    -71.2354440 -0.869191E-07  0.105E-03    3.04      14.6  T\n",
            "   5    -71.2354440 -0.255254E-07  0.389E-04    3.04      39.2  T\n",
            "   6    -71.2354440 -0.102332E-08  0.142E-04    3.04     107.2  T\n",
            "     SCC iter.                  ...        0 min,  0.026 sec\n",
            "     gradient                   ...        0 min,  0.033 sec\n",
            " * total energy  :   -70.3092465 Eh     change       -0.2330406E-04 Eh\n",
            "   gradient norm :     0.0003589 Eh/Œ±   predicted    -0.7923488E-05 ( -66.00%)\n",
            "   displ. norm   :     0.0848016 Œ±      lambda       -0.1335538E-04\n",
            "   maximum displ.:     0.0769422 Œ±      in ANC's #2, #3, #4, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   86 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2354838 -0.712355E+02  0.115E-02    3.04       0.0  T\n",
            "   2    -71.2354839 -0.748878E-07  0.643E-03    3.04       2.4  T\n",
            "   3    -71.2354839  0.353988E-07  0.170E-03    3.04       9.0  T\n",
            "   4    -71.2354839 -0.462758E-07  0.779E-04    3.04      19.6  T\n",
            "   5    -71.2354839 -0.135410E-07  0.287E-04    3.04      53.2  T\n",
            "     SCC iter.                  ...        0 min,  0.019 sec\n",
            "     gradient                   ...        0 min,  0.036 sec\n",
            " * total energy  :   -70.3092662 Eh     change       -0.1977143E-04 Eh\n",
            "   gradient norm :     0.0003580 Eh/Œ±   predicted    -0.6699313E-05 ( -66.12%)\n",
            "   displ. norm   :     0.0702660 Œ±      lambda       -0.1168535E-04\n",
            "   maximum displ.:     0.0656044 Œ±      in ANC's #2, #4, #3, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   87 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2354830 -0.712355E+02  0.888E-03    3.04       0.0  T\n",
            "   2    -71.2354830 -0.428100E-07  0.497E-03    3.04       3.1  T\n",
            "   3    -71.2354830  0.209999E-07  0.132E-03    3.04      11.5  T\n",
            "   4    -71.2354830 -0.276641E-07  0.605E-04    3.04      25.2  T\n",
            "   5    -71.2354830 -0.776717E-08  0.231E-04    3.04      66.0  T\n",
            "     SCC iter.                  ...        0 min,  0.019 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3092836 Eh     change       -0.1732979E-04 Eh\n",
            "   gradient norm :     0.0003551 Eh/Œ±   predicted    -0.5859438E-05 ( -66.19%)\n",
            "   displ. norm   :     0.0622977 Œ±      lambda       -0.1084706E-04\n",
            "   maximum displ.:     0.0589099 Œ±      in ANC's #2, #16, #1, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   88 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2355155 -0.712355E+02  0.735E-03    3.04       0.0  T\n",
            "   2    -71.2355155 -0.249997E-07  0.419E-03    3.04       3.6  T\n",
            "   3    -71.2355155  0.133276E-07  0.997E-04    3.04      15.3  T\n",
            "   4    -71.2355155 -0.178688E-07  0.453E-04    3.04      33.7  T\n",
            "     SCC iter.                  ...        0 min,  0.016 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3092997 Eh     change       -0.1608315E-04 Eh\n",
            "   gradient norm :     0.0003638 Eh/Œ±   predicted    -0.5433987E-05 ( -66.21%)\n",
            "   displ. norm   :     0.0606041 Œ±      lambda       -0.1079598E-04\n",
            "   maximum displ.:     0.0568350 Œ±      in ANC's #2, #3, #1, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   89 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2354995 -0.712355E+02  0.709E-03    3.04       0.0  T\n",
            "   2    -71.2354995 -0.174831E-07  0.412E-03    3.04       3.7  T\n",
            "   3    -71.2354995  0.142657E-07  0.883E-04    3.04      17.3  T\n",
            "   4    -71.2354995 -0.189005E-07  0.381E-04    3.04      40.0  T\n",
            "     SCC iter.                  ...        0 min,  0.029 sec\n",
            "     gradient                   ...        0 min,  0.035 sec\n",
            " * total energy  :   -70.3093157 Eh     change       -0.1609685E-04 Eh\n",
            "   gradient norm :     0.0003667 Eh/Œ±   predicted    -0.5408704E-05 ( -66.40%)\n",
            "   displ. norm   :     0.0639337 Œ±      lambda       -0.1145928E-04\n",
            "   maximum displ.:     0.0582370 Œ±      in ANC's #2, #3, #1, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   90 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2355040 -0.712355E+02  0.785E-03    3.04       0.0  T\n",
            "   2    -71.2355040 -0.166797E-07  0.461E-03    3.04       3.3  T\n",
            "   3    -71.2355040  0.121008E-07  0.807E-04    3.04      18.9  T\n",
            "   4    -71.2355040 -0.175410E-07  0.341E-04    3.04      44.7  T\n",
            "     SCC iter.                  ...        0 min,  0.016 sec\n",
            "     gradient                   ...        0 min,  0.038 sec\n",
            " * total energy  :   -70.3093328 Eh     change       -0.1709933E-04 Eh\n",
            "   gradient norm :     0.0003655 Eh/Œ±   predicted    -0.5742105E-05 ( -66.42%)\n",
            "   displ. norm   :     0.0730760 Œ±      lambda       -0.1299394E-04\n",
            "   maximum displ.:     0.0640020 Œ±      in ANC's #2, #3, #1, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   91 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2355051 -0.712355E+02  0.962E-03    3.04       0.0  T\n",
            "   2    -71.2355051 -0.231513E-07  0.566E-03    3.04       2.7  T\n",
            "   3    -71.2355051  0.112083E-07  0.854E-04    3.04      17.9  T\n",
            "   4    -71.2355051 -0.194352E-07  0.367E-04    3.04      41.5  T\n",
            "     SCC iter.                  ...        0 min,  0.015 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3093522 Eh     change       -0.1938129E-04 Eh\n",
            "   gradient norm :     0.0003724 Eh/Œ±   predicted    -0.6514459E-05 ( -66.39%)\n",
            "   displ. norm   :     0.0880021 Œ±      lambda       -0.1541484E-04\n",
            "   maximum displ.:     0.0741617 Œ±      in ANC's #2, #3, #6, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   92 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2354810 -0.712355E+02  0.123E-02    3.04       0.0  T\n",
            "   2    -71.2354810 -0.360404E-07  0.725E-03    3.04       2.1  T\n",
            "   3    -71.2354810  0.140578E-07  0.101E-03    3.04      15.0  T\n",
            "   4    -71.2354810 -0.273018E-07  0.438E-04    3.04      34.8  T\n",
            "   5    -71.2354810 -0.226947E-08  0.164E-04    3.04      93.1  T\n",
            "     SCC iter.                  ...        0 min,  0.019 sec\n",
            "     gradient                   ...        0 min,  0.038 sec\n",
            " * total energy  :   -70.3093752 Eh     change       -0.2297155E-04 Eh\n",
            "   gradient norm :     0.0003677 Eh/Œ±   predicted    -0.7736404E-05 ( -66.32%)\n",
            "   displ. norm   :     0.1093872 Œ±      lambda       -0.1880364E-04\n",
            "   maximum displ.:     0.0891248 Œ±      in ANC's #2, #3, #4, ...\n",
            " * RMSD in coord.:     0.5050036 Œ±      energy gain  -0.6399084E-03 Eh\n",
            "\n",
            "generating ANC from model Hessian ...\n",
            "Using Lindh-Hessian (1995)\n",
            " Shifting diagonal of input Hessian by   9.814027059977890E-003\n",
            " Lowest  eigenvalues of input Hessian\n",
            "    0.000000    0.000000    0.000000    0.000000    0.000000    0.000000\n",
            "    0.010000    0.010064    0.010304    0.010767    0.011580    0.012502\n",
            "    0.012716    0.013138    0.013897    0.014284    0.015692    0.015905\n",
            " Highest eigenvalues\n",
            "    2.046614    2.053529    2.142466    2.152757    2.257759    2.474014\n",
            " \n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   93 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2354960 -0.712355E+02  0.161E-02    3.04       0.0  T\n",
            "   2    -71.2354961 -0.653231E-07  0.944E-03    3.04       1.6  T\n",
            "   3    -71.2354961  0.987798E-08  0.120E-03    3.04      12.7  T\n",
            "   4    -71.2354961 -0.309912E-07  0.575E-04    3.04      26.5  T\n",
            "   5    -71.2354961 -0.489659E-08  0.198E-04    3.04      77.1  T\n",
            "     SCC iter.                  ...        0 min,  0.019 sec\n",
            "     gradient                   ...        0 min,  0.035 sec\n",
            " * total energy  :   -70.3094032 Eh     change       -0.2795181E-04 Eh\n",
            "   gradient norm :     0.0003699 Eh/Œ±   predicted     0.0000000E+00 (-100.00%)\n",
            "   displ. norm   :     0.0429995 Œ±      lambda       -0.2187286E-06\n",
            "   maximum displ.:     0.0405921 Œ±      in ANC's #2, #3, #8, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   94 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2353721 -0.712354E+02  0.454E-03    3.04       0.0  T\n",
            "   2    -71.2353721 -0.124591E-07  0.263E-03    3.04       5.8  T\n",
            "   3    -71.2353721  0.935825E-08  0.663E-04    3.04      23.0  T\n",
            "   4    -71.2353721 -0.129828E-07  0.243E-04    3.04      62.8  T\n",
            "     SCC iter.                  ...        0 min,  0.031 sec\n",
            "     gradient                   ...        0 min,  0.040 sec\n",
            " * total energy  :   -70.3094158 Eh     change       -0.1265773E-04 Eh\n",
            "   gradient norm :     0.0003200 Eh/Œ±   predicted    -0.4268656E-05 ( -66.28%)\n",
            "   displ. norm   :     0.0613860 Œ±      lambda       -0.1181315E-04\n",
            "   maximum displ.:     0.0523220 Œ±      in ANC's #2, #3, #8, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   95 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2353308 -0.712353E+02  0.794E-03    3.04       0.0  T\n",
            "   2    -71.2353309 -0.279072E-07  0.453E-03    3.04       3.4  T\n",
            "   3    -71.2353309  0.475436E-08  0.795E-04    3.04      19.2  T\n",
            "   4    -71.2353309 -0.104318E-07  0.393E-04    3.04      38.9  T\n",
            "     SCC iter.                  ...        0 min,  0.015 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3094333 Eh     change       -0.1751941E-04 Eh\n",
            "   gradient norm :     0.0003696 Eh/Œ±   predicted    -0.5916879E-05 ( -66.23%)\n",
            "   displ. norm   :     0.0937816 Œ±      lambda       -0.1733767E-04\n",
            "   maximum displ.:     0.0751502 Œ±      in ANC's #2, #3, #8, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   96 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2353704 -0.712354E+02  0.133E-02    3.04       0.0  T\n",
            "   2    -71.2353704 -0.719903E-07  0.761E-03    3.04       2.0  T\n",
            "   3    -71.2353704  0.158039E-07  0.129E-03    3.04      11.8  T\n",
            "   4    -71.2353705 -0.336887E-07  0.610E-04    3.04      25.0  T\n",
            "   5    -71.2353705 -0.687231E-08  0.196E-04    3.04      78.0  T\n",
            "     SCC iter.                  ...        0 min,  0.019 sec\n",
            "     gradient                   ...        0 min,  0.043 sec\n",
            " * total energy  :   -70.3094588 Eh     change       -0.2551558E-04 Eh\n",
            "   gradient norm :     0.0003598 Eh/Œ±   predicted    -0.8704209E-05 ( -65.89%)\n",
            "   displ. norm   :     0.1368953 Œ±      lambda       -0.2416475E-04\n",
            "   maximum displ.:     0.1068702 Œ±      in ANC's #2, #3, #7, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   97 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2354068 -0.712354E+02  0.200E-02    3.04       0.0  T\n",
            "   2    -71.2354070 -0.154274E-06  0.115E-02    3.04       1.3  T\n",
            "   3    -71.2354069  0.233589E-07  0.182E-03    3.04       8.4  T\n",
            "   4    -71.2354070 -0.636408E-07  0.860E-04    3.04      17.7  T\n",
            "   5    -71.2354070 -0.122382E-07  0.303E-04    3.04      50.4  T\n",
            "     SCC iter.                  ...        0 min,  0.020 sec\n",
            "     gradient                   ...        0 min,  0.040 sec\n",
            " * total energy  :   -70.3094940 Eh     change       -0.3520145E-04 Eh\n",
            "   gradient norm :     0.0003530 Eh/Œ±   predicted    -0.1218647E-04 ( -65.38%)\n",
            "   displ. norm   :     0.1780025 Œ±      lambda       -0.2982900E-04\n",
            "   maximum displ.:     0.1379904 Œ±      in ANC's #2, #3, #7, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   98 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2353855 -0.712354E+02  0.264E-02    3.04       0.0  T\n",
            "   2    -71.2353857 -0.254619E-06  0.152E-02    3.04       1.0  T\n",
            "   3    -71.2353857  0.192300E-07  0.221E-03    3.04       6.9  T\n",
            "   4    -71.2353858 -0.869861E-07  0.107E-03    3.03      14.3  T\n",
            "   5    -71.2353858 -0.180332E-07  0.381E-04    3.04      40.0  T\n",
            "   6    -71.2353858 -0.167570E-08  0.167E-04    3.04      91.3  T\n",
            "     SCC iter.                  ...        0 min,  0.024 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3095373 Eh     change       -0.4324959E-04 Eh\n",
            "   gradient norm :     0.0003879 Eh/Œ±   predicted    -0.1512478E-04 ( -65.03%)\n",
            "   displ. norm   :     0.1990416 Œ±      lambda       -0.3190365E-04\n",
            "   maximum displ.:     0.1543432 Œ±      in ANC's #2, #3, #7, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE   99 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2354067 -0.712354E+02  0.294E-02    3.04       0.0  T\n",
            "   2    -71.2354070 -0.285813E-06  0.170E-02    3.03       1.0  T\n",
            "   3    -71.2354070  0.169302E-07  0.235E-03    3.03       6.5  T\n",
            "   4    -71.2354071 -0.957390E-07  0.115E-03    3.03      13.2  T\n",
            "   5    -71.2354071 -0.206064E-07  0.423E-04    3.03      36.0  T\n",
            "   6    -71.2354071 -0.188660E-08  0.186E-04    3.03      81.8  T\n",
            "     SCC iter.                  ...        0 min,  0.024 sec\n",
            "     gradient                   ...        0 min,  0.035 sec\n",
            " * total energy  :   -70.3095835 Eh     change       -0.4617028E-04 Eh\n",
            "   gradient norm :     0.0004357 Eh/Œ±   predicted    -0.1623645E-04 ( -64.83%)\n",
            "   displ. norm   :     0.1946258 Œ±      lambda       -0.3037620E-04\n",
            "   maximum displ.:     0.1517728 Œ±      in ANC's #2, #3, #7, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  100 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2354161 -0.712354E+02  0.288E-02    3.03       0.0  T\n",
            "   2    -71.2354164 -0.263501E-06  0.167E-02    3.03       1.0  T\n",
            "   3    -71.2354164  0.174821E-07  0.224E-03    3.03       6.8  T\n",
            "   4    -71.2354165 -0.934878E-07  0.110E-03    3.03      13.9  T\n",
            "   5    -71.2354165 -0.181399E-07  0.396E-04    3.03      38.5  T\n",
            "   6    -71.2354165 -0.149755E-08  0.172E-04    3.03      88.7  T\n",
            "     SCC iter.                  ...        0 min,  0.023 sec\n",
            "     gradient                   ...        0 min,  0.039 sec\n",
            " * total energy  :   -70.3096276 Eh     change       -0.4416432E-04 Eh\n",
            "   gradient norm :     0.0004772 Eh/Œ±   predicted    -0.1544562E-04 ( -65.03%)\n",
            "   displ. norm   :     0.1733251 Œ±      lambda       -0.2712953E-04\n",
            "   maximum displ.:     0.1365186 Œ±      in ANC's #2, #3, #7, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  101 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2355229 -0.712355E+02  0.253E-02    3.03       0.0  T\n",
            "   2    -71.2355231 -0.178713E-06  0.148E-02    3.03       1.0  T\n",
            "   3    -71.2355231  0.408130E-08  0.179E-03    3.03       8.5  T\n",
            "   4    -71.2355232 -0.568239E-07  0.925E-04    3.03      16.5  T\n",
            "   5    -71.2355232 -0.130197E-07  0.337E-04    3.03      45.3  T\n",
            "     SCC iter.                  ...        0 min,  0.021 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3096673 Eh     change       -0.3967890E-04 Eh\n",
            "   gradient norm :     0.0004688 Eh/Œ±   predicted    -0.1374796E-04 ( -65.35%)\n",
            "   displ. norm   :     0.1517309 Œ±      lambda       -0.2434044E-04\n",
            "   maximum displ.:     0.1208607 Œ±      in ANC's #2, #3, #1, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  102 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2356379 -0.712356E+02  0.221E-02    3.03       0.0  T\n",
            "   2    -71.2356380 -0.129173E-06  0.129E-02    3.03       1.2  T\n",
            "   3    -71.2356380  0.225577E-08  0.155E-03    3.03       9.8  T\n",
            "   4    -71.2356381 -0.425537E-07  0.813E-04    3.03      18.8  T\n",
            "   5    -71.2356381 -0.946559E-08  0.299E-04    3.03      51.1  T\n",
            "     SCC iter.                  ...        0 min,  0.019 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3097030 Eh     change       -0.3570438E-04 Eh\n",
            "   gradient norm :     0.0004348 Eh/Œ±   predicted    -0.1229682E-04 ( -65.56%)\n",
            "   displ. norm   :     0.1371616 Œ±      lambda       -0.2269368E-04\n",
            "   maximum displ.:     0.1105864 Œ±      in ANC's #2, #3, #1, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  103 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2357031 -0.712357E+02  0.198E-02    3.03       0.0  T\n",
            "   2    -71.2357032 -0.986207E-07  0.116E-02    3.03       1.3  T\n",
            "   3    -71.2357032  0.687537E-08  0.140E-03    3.03      10.9  T\n",
            "   4    -71.2357032 -0.387530E-07  0.711E-04    3.03      21.5  T\n",
            "   5    -71.2357032 -0.642623E-08  0.269E-04    3.03      56.8  T\n",
            "     SCC iter.                  ...        0 min,  0.019 sec\n",
            "     gradient                   ...        0 min,  0.044 sec\n",
            " * total energy  :   -70.3097363 Eh     change       -0.3326494E-04 Eh\n",
            "   gradient norm :     0.0004050 Eh/Œ±   predicted    -0.1144303E-04 ( -65.60%)\n",
            "   displ. norm   :     0.1297230 Œ±      lambda       -0.2191417E-04\n",
            "   maximum displ.:     0.1056867 Œ±      in ANC's #2, #3, #1, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  104 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2358123 -0.712358E+02  0.185E-02    3.03       0.0  T\n",
            "   2    -71.2358124 -0.844681E-07  0.109E-02    3.03       1.4  T\n",
            "   3    -71.2358124  0.754729E-08  0.134E-03    3.03      11.3  T\n",
            "   4    -71.2358125 -0.343280E-07  0.705E-04    3.03      21.6  T\n",
            "   5    -71.2358125 -0.649793E-08  0.276E-04    3.03      55.2  T\n",
            "     SCC iter.                  ...        0 min,  0.020 sec\n",
            "     gradient                   ...        0 min,  0.039 sec\n",
            " * total energy  :   -70.3097683 Eh     change       -0.3204405E-04 Eh\n",
            "   gradient norm :     0.0003665 Eh/Œ±   predicted    -0.1103991E-04 ( -65.55%)\n",
            "   displ. norm   :     0.1264346 Œ±      lambda       -0.2147465E-04\n",
            "   maximum displ.:     0.1041269 Œ±      in ANC's #2, #3, #1, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  105 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2358436 -0.712358E+02  0.180E-02    3.03       0.0  T\n",
            "   2    -71.2358437 -0.869623E-07  0.105E-02    3.03       1.4  T\n",
            "   3    -71.2358436  0.219642E-07  0.151E-03    3.03      10.1  T\n",
            "   4    -71.2358437 -0.483081E-07  0.726E-04    3.03      21.0  T\n",
            "   5    -71.2358437 -0.686673E-08  0.286E-04    3.03      53.3  T\n",
            "     SCC iter.                  ...        0 min,  0.020 sec\n",
            "     gradient                   ...        0 min,  0.045 sec\n",
            " * total energy  :   -70.3097997 Eh     change       -0.3138846E-04 Eh\n",
            "   gradient norm :     0.0003360 Eh/Œ±   predicted    -0.1081410E-04 ( -65.55%)\n",
            "   displ. norm   :     0.1238423 Œ±      lambda       -0.2091806E-04\n",
            "   maximum displ.:     0.1028787 Œ±      in ANC's #2, #3, #1, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  106 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2359140 -0.712359E+02  0.172E-02    3.03       0.0  T\n",
            "   2    -71.2359141 -0.832509E-07  0.101E-02    3.02       1.5  T\n",
            "   3    -71.2359141  0.237293E-07  0.151E-03    3.02      10.1  T\n",
            "   4    -71.2359141 -0.463302E-07  0.744E-04    3.02      20.5  T\n",
            "   5    -71.2359141 -0.787811E-08  0.300E-04    3.02      50.8  T\n",
            "     SCC iter.                  ...        0 min,  0.020 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3098303 Eh     change       -0.3060796E-04 Eh\n",
            "   gradient norm :     0.0003216 Eh/Œ±   predicted    -0.1053084E-04 ( -65.59%)\n",
            "   displ. norm   :     0.1210065 Œ±      lambda       -0.2029453E-04\n",
            "   maximum displ.:     0.1012621 Œ±      in ANC's #2, #3, #1, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  107 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2359641 -0.712360E+02  0.167E-02    3.02       0.0  T\n",
            "   2    -71.2359642 -0.927161E-07  0.964E-03    3.02       1.6  T\n",
            "   3    -71.2359642  0.330176E-07  0.171E-03    3.02       8.9  T\n",
            "   4    -71.2359643 -0.547828E-07  0.808E-04    3.02      18.9  T\n",
            "   5    -71.2359643 -0.987603E-08  0.321E-04    3.02      47.5  T\n",
            "     SCC iter.                  ...        0 min,  0.020 sec\n",
            "     gradient                   ...        0 min,  0.041 sec\n",
            " * total energy  :   -70.3098601 Eh     change       -0.2976039E-04 Eh\n",
            "   gradient norm :     0.0003261 Eh/Œ±   predicted    -0.1021520E-04 ( -65.68%)\n",
            "   displ. norm   :     0.1185508 Œ±      lambda       -0.1977555E-04\n",
            "   maximum displ.:     0.0995271 Œ±      in ANC's #2, #1, #3, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  108 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2360042 -0.712360E+02  0.160E-02    3.02       0.0  T\n",
            "   2    -71.2360043 -0.946965E-07  0.920E-03    3.02       1.7  T\n",
            "   3    -71.2360043  0.361815E-07  0.176E-03    3.02       8.7  T\n",
            "   4    -71.2360044 -0.554262E-07  0.820E-04    3.02      18.6  T\n",
            "   5    -71.2360044 -0.112197E-07  0.324E-04    3.02      47.0  T\n",
            "     SCC iter.                  ...        0 min,  0.023 sec\n",
            "     gradient                   ...        0 min,  0.040 sec\n",
            " * total energy  :   -70.3098892 Eh     change       -0.2908008E-04 Eh\n",
            "   gradient norm :     0.0003512 Eh/Œ±   predicted    -0.9952220E-05 ( -65.78%)\n",
            "   displ. norm   :     0.1170730 Œ±      lambda       -0.1951321E-04\n",
            "   maximum displ.:     0.0982972 Œ±      in ANC's #2, #1, #3, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  109 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2360627 -0.712361E+02  0.155E-02    3.02       0.0  T\n",
            "   2    -71.2360628 -0.106885E-06  0.883E-03    3.02       1.7  T\n",
            "   3    -71.2360628  0.400393E-07  0.193E-03    3.02       7.9  T\n",
            "   4    -71.2360628 -0.590739E-07  0.873E-04    3.02      17.5  T\n",
            "   5    -71.2360628 -0.131793E-07  0.347E-04    3.02      43.9  T\n",
            "     SCC iter.                  ...        0 min,  0.020 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3099179 Eh     change       -0.2876202E-04 Eh\n",
            "   gradient norm :     0.0003702 Eh/Œ±   predicted    -0.9819210E-05 ( -65.86%)\n",
            "   displ. norm   :     0.1176905 Œ±      lambda       -0.1963157E-04\n",
            "   maximum displ.:     0.0984299 Œ±      in ANC's #2, #1, #4, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  110 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2360779 -0.712361E+02  0.155E-02    3.02       0.0  T\n",
            "   2    -71.2360780 -0.115870E-06  0.876E-03    3.02       1.7  T\n",
            "   3    -71.2360780  0.441792E-07  0.201E-03    3.02       7.6  T\n",
            "   4    -71.2360780 -0.633544E-07  0.895E-04    3.02      17.0  T\n",
            "   5    -71.2360781 -0.150651E-07  0.353E-04    3.02      43.2  T\n",
            "     SCC iter.                  ...        0 min,  0.020 sec\n",
            "     gradient                   ...        0 min,  0.035 sec\n",
            " * total energy  :   -70.3099469 Eh     change       -0.2897057E-04 Eh\n",
            "   gradient norm :     0.0003917 Eh/Œ±   predicted    -0.9879248E-05 ( -65.90%)\n",
            "   displ. norm   :     0.1199560 Œ±      lambda       -0.2004988E-04\n",
            "   maximum displ.:     0.0997467 Œ±      in ANC's #2, #1, #4, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  111 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2361263 -0.712361E+02  0.155E-02    3.02       0.0  T\n",
            "   2    -71.2361264 -0.133775E-06  0.870E-03    3.02       1.8  T\n",
            "   3    -71.2361264  0.462629E-07  0.216E-03    3.02       7.1  T\n",
            "   4    -71.2361265 -0.667515E-07  0.939E-04    3.02      16.2  T\n",
            "   5    -71.2361265 -0.170208E-07  0.375E-04    3.02      40.7  T\n",
            "     SCC iter.                  ...        0 min,  0.021 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3099765 Eh     change       -0.2959832E-04 Eh\n",
            "   gradient norm :     0.0003893 Eh/Œ±   predicted    -0.1009208E-04 ( -65.90%)\n",
            "   displ. norm   :     0.1245898 Œ±      lambda       -0.2080910E-04\n",
            "   maximum displ.:     0.1027700 Œ±      in ANC's #2, #1, #4, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  112 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2361291 -0.712361E+02  0.163E-02    3.02       0.0  T\n",
            "   2    -71.2361293 -0.150082E-06  0.912E-03    3.02       1.7  T\n",
            "   3    -71.2361292  0.567424E-07  0.230E-03    3.02       6.6  T\n",
            "   4    -71.2361293 -0.804921E-07  0.989E-04    3.02      15.4  T\n",
            "   5    -71.2361293 -0.201224E-07  0.396E-04    3.02      38.5  T\n",
            "     SCC iter.                  ...        0 min,  0.030 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3100072 Eh     change       -0.3070037E-04 Eh\n",
            "   gradient norm :     0.0003864 Eh/Œ±   predicted    -0.1047907E-04 ( -65.87%)\n",
            "   displ. norm   :     0.1301816 Œ±      lambda       -0.2168983E-04\n",
            "   maximum displ.:     0.1064746 Œ±      in ANC's #2, #1, #4, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  113 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2361320 -0.712361E+02  0.171E-02    3.02       0.0  T\n",
            "   2    -71.2361321 -0.173169E-06  0.952E-03    3.02       1.6  T\n",
            "   3    -71.2361321  0.587803E-07  0.242E-03    3.02       6.3  T\n",
            "   4    -71.2361322 -0.846746E-07  0.103E-03    3.02      14.8  T\n",
            "   5    -71.2361322 -0.223758E-07  0.409E-04    3.02      37.3  T\n",
            "   6    -71.2361322 -0.145619E-08  0.140E-04    3.02     109.1  T\n",
            "     SCC iter.                  ...        0 min,  0.025 sec\n",
            "     gradient                   ...        0 min,  0.041 sec\n",
            " * total energy  :   -70.3100392 Eh     change       -0.3199460E-04 Eh\n",
            "   gradient norm :     0.0003724 Eh/Œ±   predicted    -0.1092909E-04 ( -65.84%)\n",
            "   displ. norm   :     0.1369578 Œ±      lambda       -0.2273267E-04\n",
            "   maximum displ.:     0.1110132 Œ±      in ANC's #2, #1, #4, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  114 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2361610 -0.712362E+02  0.182E-02    3.02       0.0  T\n",
            "   2    -71.2361612 -0.195690E-06  0.102E-02    3.02       1.5  T\n",
            "   3    -71.2361611  0.800243E-07  0.262E-03    3.02       5.8  T\n",
            "   4    -71.2361613 -0.111246E-06  0.111E-03    3.02      13.8  T\n",
            "   5    -71.2361613 -0.270824E-07  0.444E-04    3.02      34.3  T\n",
            "   6    -71.2361613 -0.148484E-08  0.152E-04    3.02     100.2  T\n",
            "     SCC iter.                  ...        0 min,  0.023 sec\n",
            "     gradient                   ...        0 min,  0.035 sec\n",
            " * total energy  :   -70.3100727 Eh     change       -0.3355407E-04 Eh\n",
            "   gradient norm :     0.0003663 Eh/Œ±   predicted    -0.1146309E-04 ( -65.84%)\n",
            "   displ. norm   :     0.1442238 Œ±      lambda       -0.2383290E-04\n",
            "   maximum displ.:     0.1158624 Œ±      in ANC's #2, #1, #3, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  115 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2361198 -0.712361E+02  0.197E-02    3.02       0.0  T\n",
            "   2    -71.2361201 -0.216009E-06  0.110E-02    3.02       1.4  T\n",
            "   3    -71.2361200  0.912855E-07  0.273E-03    3.02       5.6  T\n",
            "   4    -71.2361201 -0.125682E-06  0.116E-03    3.02      13.2  T\n",
            "   5    -71.2361201 -0.307357E-07  0.457E-04    3.02      33.4  T\n",
            "   6    -71.2361201 -0.158002E-08  0.157E-04    3.02      96.8  T\n",
            "     SCC iter.                  ...        0 min,  0.024 sec\n",
            "     gradient                   ...        0 min,  0.034 sec\n",
            " * total energy  :   -70.3101080 Eh     change       -0.3526109E-04 Eh\n",
            "   gradient norm :     0.0003613 Eh/Œ±   predicted    -0.1202834E-04 ( -65.89%)\n",
            "   displ. norm   :     0.1518894 Œ±      lambda       -0.2508304E-04\n",
            "   maximum displ.:     0.1208344 Œ±      in ANC's #2, #1, #3, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  116 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2361563 -0.712362E+02  0.210E-02    3.02       0.0  T\n",
            "   2    -71.2361566 -0.245784E-06  0.118E-02    3.02       1.3  T\n",
            "   3    -71.2361565  0.107779E-06  0.290E-03    3.02       5.3  T\n",
            "   4    -71.2361566 -0.147776E-06  0.122E-03    3.02      12.5  T\n",
            "   5    -71.2361567 -0.346522E-07  0.489E-04    3.02      31.2  T\n",
            "   6    -71.2361567 -0.186895E-08  0.170E-04    3.02      89.9  T\n",
            "     SCC iter.                  ...        0 min,  0.053 sec\n",
            "     gradient                   ...        0 min,  0.051 sec\n",
            " * total energy  :   -70.3101452 Eh     change       -0.3715241E-04 Eh\n",
            "   gradient norm :     0.0003947 Eh/Œ±   predicted    -0.1267164E-04 ( -65.89%)\n",
            "   displ. norm   :     0.1631976 Œ±      lambda       -0.2698778E-04\n",
            "   maximum displ.:     0.1285267 Œ±      in ANC's #2, #1, #3, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  117 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2361171 -0.712361E+02  0.233E-02    3.02       0.0  T\n",
            "   2    -71.2361174 -0.276266E-06  0.132E-02    3.02       1.2  T\n",
            "   3    -71.2361172  0.138077E-06  0.312E-03    3.02       4.9  T\n",
            "   4    -71.2361174 -0.185748E-06  0.131E-03    3.02      11.6  T\n",
            "   5    -71.2361174 -0.406799E-07  0.525E-04    3.02      29.1  T\n",
            "   6    -71.2361174 -0.207423E-08  0.183E-04    3.02      83.5  T\n",
            "     SCC iter.                  ...        0 min,  0.061 sec\n",
            "     gradient                   ...        0 min,  0.066 sec\n",
            " * total energy  :   -70.3101851 Eh     change       -0.3999499E-04 Eh\n",
            "   gradient norm :     0.0004068 Eh/Œ±   predicted    -0.1365503E-04 ( -65.86%)\n",
            "   displ. norm   :     0.1781268 Œ±      lambda       -0.2951051E-04\n",
            "   maximum displ.:     0.1388105 Œ±      in ANC's #2, #1, #3, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  118 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2361247 -0.712361E+02  0.260E-02    3.02       0.0  T\n",
            "   2    -71.2361251 -0.344870E-06  0.148E-02    3.02       1.0  T\n",
            "   3    -71.2361249  0.172878E-06  0.347E-03    3.02       4.4  T\n",
            "   4    -71.2361251 -0.231154E-06  0.145E-03    3.02      10.5  T\n",
            "   5    -71.2361252 -0.505958E-07  0.579E-04    3.02      26.3  T\n",
            "   6    -71.2361252 -0.269694E-08  0.203E-04    3.02      75.2  T\n",
            "     SCC iter.                  ...        0 min,  0.028 sec\n",
            "     gradient                   ...        0 min,  0.069 sec\n",
            " * total energy  :   -70.3102289 Eh     change       -0.4371000E-04 Eh\n",
            "   gradient norm :     0.0004499 Eh/Œ±   predicted    -0.1496529E-04 ( -65.76%)\n",
            "   displ. norm   :     0.1987644 Œ±      lambda       -0.3293416E-04\n",
            "   maximum displ.:     0.1534658 Œ±      in ANC's #2, #1, #3, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  119 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2360851 -0.712361E+02  0.296E-02    3.02       0.0  T\n",
            "   2    -71.2360855 -0.420008E-06  0.169E-02    3.02       1.0  T\n",
            "   3    -71.2360853  0.236324E-06  0.389E-03    3.02       3.9  T\n",
            "   4    -71.2360856 -0.310850E-06  0.163E-03    3.02       9.4  T\n",
            "   5    -71.2360857 -0.637428E-07  0.648E-04    3.02      23.5  T\n",
            "   6    -71.2360857 -0.333876E-08  0.227E-04    3.02      67.1  T\n",
            "     SCC iter.                  ...        0 min,  0.062 sec\n",
            "     gradient                   ...        0 min,  0.055 sec\n",
            " * total energy  :   -70.3102775 Eh     change       -0.4868624E-04 Eh\n",
            "   gradient norm :     0.0004704 Eh/Œ±   predicted    -0.1675846E-04 ( -65.58%)\n",
            "   displ. norm   :     0.2250903 Œ±      lambda       -0.3710880E-04\n",
            "   maximum displ.:     0.1724552 Œ±      in ANC's #2, #1, #3, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  120 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2360540 -0.712361E+02  0.343E-02    3.02       0.0  T\n",
            "   2    -71.2360546 -0.563922E-06  0.197E-02    3.02       1.0  T\n",
            "   3    -71.2360542  0.325534E-06  0.452E-03    3.02       3.4  T\n",
            "   4    -71.2360547 -0.423424E-06  0.188E-03    3.02       8.1  T\n",
            "   5    -71.2360548 -0.866847E-07  0.745E-04    3.02      20.5  T\n",
            "   6    -71.2360548 -0.457361E-08  0.262E-04    3.02      58.2  T\n",
            "     SCC iter.                  ...        0 min,  0.060 sec\n",
            "     gradient                   ...        0 min,  0.058 sec\n",
            " * total energy  :   -70.3103322 Eh     change       -0.5467336E-04 Eh\n",
            "   gradient norm :     0.0005140 Eh/Œ±   predicted    -0.1897529E-04 ( -65.29%)\n",
            "   displ. norm   :     0.2553820 Œ±      lambda       -0.4159864E-04\n",
            "   maximum displ.:     0.1948404 Œ±      in ANC's #2, #1, #3, ...\n",
            " * RMSD in coord.:     0.5418697 Œ±      energy gain  -0.9570174E-03 Eh\n",
            "\n",
            "generating ANC from model Hessian ...\n",
            "Using Lindh-Hessian (1995)\n",
            " Shifting diagonal of input Hessian by   9.803634125876955E-003\n",
            " Lowest  eigenvalues of input Hessian\n",
            "    0.000000    0.000000    0.000000    0.000000    0.000000    0.000000\n",
            "    0.010000    0.010019    0.010269    0.010767    0.011538    0.012495\n",
            "    0.012872    0.013097    0.013788    0.014230    0.015697    0.015940\n",
            " Highest eigenvalues\n",
            "    2.046341    2.052923    2.141324    2.151402    2.257095    2.465420\n",
            " \n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  121 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2359986 -0.712360E+02  0.394E-02    3.02       0.0  T\n",
            "   2    -71.2359993 -0.717065E-06  0.227E-02    3.02       1.0  T\n",
            "   3    -71.2359989  0.447744E-06  0.516E-03    3.02       3.0  T\n",
            "   4    -71.2359995 -0.576054E-06  0.213E-03    3.02       7.1  T\n",
            "   5    -71.2359996 -0.112536E-06  0.847E-04    3.02      18.0  T\n",
            "   6    -71.2359996 -0.596069E-08  0.298E-04    3.02      51.2  T\n",
            "     SCC iter.                  ...        0 min,  0.064 sec\n",
            "     gradient                   ...        0 min,  0.070 sec\n",
            " * total energy  :   -70.3103932 Eh     change       -0.6098740E-04 Eh\n",
            "   gradient norm :     0.0005468 Eh/Œ±   predicted     0.0000000E+00 (-100.00%)\n",
            "   displ. norm   :     0.0450875 Œ±      lambda       -0.3558111E-06\n",
            "   maximum displ.:     0.0410184 Œ±      in ANC's #2, #1, #3, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  122 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2364568 -0.712365E+02  0.553E-03    3.02       0.0  T\n",
            "   2    -71.2364568 -0.158377E-07  0.320E-03    3.02       4.8  T\n",
            "   3    -71.2364568  0.167235E-07  0.882E-04    3.02      17.3  T\n",
            "   4    -71.2364568 -0.223963E-07  0.349E-04    3.02      43.7  T\n",
            "     SCC iter.                  ...        0 min,  0.040 sec\n",
            "     gradient                   ...        0 min,  0.071 sec\n",
            " * total energy  :   -70.3104073 Eh     change       -0.1409181E-04 Eh\n",
            "   gradient norm :     0.0003440 Eh/Œ±   predicted    -0.4868497E-05 ( -65.45%)\n",
            "   displ. norm   :     0.0746917 Œ±      lambda       -0.1494274E-04\n",
            "   maximum displ.:     0.0595919 Œ±      in ANC's #2, #1, #3, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  123 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2366247 -0.712366E+02  0.102E-02    3.02       0.0  T\n",
            "   2    -71.2366248 -0.329099E-07  0.593E-03    3.02       2.6  T\n",
            "   3    -71.2366247  0.363167E-07  0.127E-03    3.02      12.0  T\n",
            "   4    -71.2366248 -0.464744E-07  0.590E-04    3.02      25.9  T\n",
            "   5    -71.2366248 -0.610471E-08  0.198E-04    3.02      76.9  T\n",
            "     SCC iter.                  ...        0 min,  0.060 sec\n",
            "     gradient                   ...        0 min,  0.059 sec\n",
            " * total energy  :   -70.3104291 Eh     change       -0.2175167E-04 Eh\n",
            "   gradient norm :     0.0004928 Eh/Œ±   predicted    -0.7490269E-05 ( -65.56%)\n",
            "   displ. norm   :     0.1288475 Œ±      lambda       -0.2409684E-04\n",
            "   maximum displ.:     0.0958804 Œ±      in ANC's #2, #1, #3, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  124 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2365040 -0.712365E+02  0.183E-02    3.02       0.0  T\n",
            "   2    -71.2365041 -0.127585E-06  0.107E-02    3.02       1.4  T\n",
            "   3    -71.2365040  0.124395E-06  0.242E-03    3.02       6.3  T\n",
            "   4    -71.2365042 -0.154735E-06  0.103E-03    3.02      14.8  T\n",
            "   5    -71.2365042 -0.231719E-07  0.397E-04    3.02      38.4  T\n",
            "   6    -71.2365042 -0.142538E-08  0.151E-04    3.02     101.2  T\n",
            "     SCC iter.                  ...        0 min,  0.062 sec\n",
            "     gradient                   ...        0 min,  0.060 sec\n",
            " * total energy  :   -70.3104636 Eh     change       -0.3457076E-04 Eh\n",
            "   gradient norm :     0.0005211 Eh/Œ±   predicted    -0.1214156E-04 ( -64.88%)\n",
            "   displ. norm   :     0.2008267 Œ±      lambda       -0.3458110E-04\n",
            "   maximum displ.:     0.1454745 Œ±      in ANC's #2, #1, #3, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  125 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2362745 -0.712363E+02  0.290E-02    3.02       0.0  T\n",
            "   2    -71.2362749 -0.380478E-06  0.169E-02    3.02       1.0  T\n",
            "   3    -71.2362745  0.343764E-06  0.420E-03    3.02       3.6  T\n",
            "   4    -71.2362749 -0.420967E-06  0.168E-03    3.02       9.1  T\n",
            "   5    -71.2362750 -0.675769E-07  0.664E-04    3.02      23.0  T\n",
            "   6    -71.2362750 -0.421254E-08  0.243E-04    3.02      62.8  T\n",
            "     SCC iter.                  ...        0 min,  0.066 sec\n",
            "     gradient                   ...        0 min,  0.067 sec\n",
            " * total energy  :   -70.3105123 Eh     change       -0.4868932E-04 Eh\n",
            "   gradient norm :     0.0004884 Eh/Œ±   predicted    -0.1760077E-04 ( -63.85%)\n",
            "   displ. norm   :     0.2635626 Œ±      lambda       -0.4131564E-04\n",
            "   maximum displ.:     0.1889381 Œ±      in ANC's #2, #1, #3, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  126 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2358982 -0.712359E+02  0.383E-02    3.02       0.0  T\n",
            "   2    -71.2358990 -0.729715E-06  0.222E-02    3.02       1.0  T\n",
            "   3    -71.2358984  0.588485E-06  0.570E-03    3.02       2.7  T\n",
            "   4    -71.2358991 -0.723741E-06  0.227E-03    3.02       6.7  T\n",
            "   5    -71.2358992 -0.128166E-06  0.882E-04    3.02      17.3  T\n",
            "   6    -71.2358992 -0.721514E-08  0.322E-04    3.02      47.4  T\n",
            "     SCC iter.                  ...        0 min,  0.049 sec\n",
            "     gradient                   ...        0 min,  0.048 sec\n",
            " * total energy  :   -70.3105697 Eh     change       -0.5740980E-04 Eh\n",
            "   gradient norm :     0.0006280 Eh/Œ±   predicted    -0.2129583E-04 ( -62.91%)\n",
            "   displ. norm   :     0.2987238 Œ±      lambda       -0.4342698E-04\n",
            "   maximum displ.:     0.2133459 Œ±      in ANC's #2, #1, #3, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  127 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2356747 -0.712357E+02  0.432E-02    3.02       0.0  T\n",
            "   2    -71.2356757 -0.101239E-05  0.250E-02    3.02       1.0  T\n",
            "   3    -71.2356749  0.775656E-06  0.669E-03    3.02       2.3  T\n",
            "   4    -71.2356759 -0.950484E-06  0.264E-03    3.02       5.8  T\n",
            "   5    -71.2356761 -0.177952E-06  0.103E-03    3.02      14.8  T\n",
            "   6    -71.2356761 -0.990262E-08  0.375E-04    3.02      40.7  T\n",
            "   7    -71.2356761 -0.557549E-09  0.187E-04    3.02      81.6  T\n",
            "     SCC iter.                  ...        0 min,  0.044 sec\n",
            "     gradient                   ...        0 min,  0.066 sec\n",
            " * total energy  :   -70.3106298 Eh     change       -0.6010355E-04 Eh\n",
            "   gradient norm :     0.0008130 Eh/Œ±   predicted    -0.2258531E-04 ( -62.42%)\n",
            "   displ. norm   :     0.3237837 Œ±      lambda       -0.4511196E-04\n",
            "   maximum displ.:     0.2304052 Œ±      in ANC's #2, #1, #3, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  128 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2355598 -0.712356E+02  0.465E-02    3.02       0.0  T\n",
            "   2    -71.2355610 -0.124335E-05  0.268E-02    3.02       1.0  T\n",
            "   3    -71.2355601  0.893314E-06  0.717E-03    3.02       2.1  T\n",
            "   4    -71.2355613 -0.111337E-05  0.284E-03    3.02       5.4  T\n",
            "   5    -71.2355615 -0.206051E-06  0.110E-03    3.02      13.8  T\n",
            "   6    -71.2355615 -0.105144E-07  0.401E-04    3.02      38.0  T\n",
            "   7    -71.2355615 -0.631942E-09  0.199E-04    3.02      76.5  T\n",
            "     SCC iter.                  ...        0 min,  0.072 sec\n",
            "     gradient                   ...        0 min,  0.066 sec\n",
            " * total energy  :   -70.3106924 Eh     change       -0.6255523E-04 Eh\n",
            "   gradient norm :     0.0009246 Eh/Œ±   predicted    -0.2360722E-04 ( -62.26%)\n",
            "   displ. norm   :     0.3615214 Œ±      lambda       -0.4932384E-04\n",
            "   maximum displ.:     0.2562515 Œ±      in ANC's #2, #1, #3, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  129 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2355835 -0.712356E+02  0.511E-02    3.02       0.0  T\n",
            "   2    -71.2355851 -0.157798E-05  0.293E-02    3.02       1.0  T\n",
            "   3    -71.2355840  0.104531E-05  0.792E-03    3.02       1.9  T\n",
            "   4    -71.2355853 -0.130635E-05  0.315E-03    3.02       4.8  T\n",
            "   5    -71.2355856 -0.258206E-06  0.123E-03    3.02      12.4  T\n",
            "   6    -71.2355856 -0.132866E-07  0.449E-04    3.02      33.9  T\n",
            "   7    -71.2355856 -0.835612E-09  0.224E-04    3.02      68.1  T\n",
            "     SCC iter.                  ...        0 min,  0.071 sec\n",
            "     gradient                   ...        0 min,  0.075 sec\n",
            " * total energy  :   -70.3107608 Eh     change       -0.6838499E-04 Eh\n",
            "   gradient norm :     0.0009943 Eh/Œ±   predicted    -0.2609559E-04 ( -61.84%)\n",
            "   displ. norm   :     0.4371361 Œ±      lambda       -0.5718087E-04\n",
            "   maximum displ.:     0.3075119 Œ±      in ANC's #2, #1, #3, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  130 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2355135 -0.712355E+02  0.609E-02    3.02       0.0  T\n",
            "   2    -71.2355158 -0.229022E-05  0.348E-02    3.02       1.0  T\n",
            "   3    -71.2355143  0.147534E-05  0.936E-03    3.02       1.6  T\n",
            "   4    -71.2355162 -0.187769E-05  0.372E-03    3.02       4.1  T\n",
            "   5    -71.2355165 -0.350909E-06  0.148E-03    3.02      10.3  T\n",
            "   6    -71.2355165 -0.175980E-07  0.539E-04    3.02      28.3  T\n",
            "   7    -71.2355165 -0.112462E-08  0.271E-04    3.02      56.2  T\n",
            "     SCC iter.                  ...        0 min,  0.063 sec\n",
            "     gradient                   ...        0 min,  0.053 sec\n",
            " * total energy  :   -70.3108390 Eh     change       -0.7826280E-04 Eh\n",
            "   gradient norm :     0.0011522 Eh/Œ±   predicted    -0.3102113E-04 ( -60.36%)\n",
            "   displ. norm   :     0.5359588 Œ±      lambda       -0.6573014E-04\n",
            "   maximum displ.:     0.3757043 Œ±      in ANC's #2, #1, #3, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  131 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2353116 -0.712353E+02  0.740E-02    3.02       0.0  T\n",
            "   2    -71.2353151 -0.343105E-05  0.421E-02    3.02       1.0  T\n",
            "   3    -71.2353131  0.197023E-05  0.111E-02    3.02       1.4  T\n",
            "   4    -71.2353156 -0.253038E-05  0.443E-03    3.01       3.4  T\n",
            "   5    -71.2353161 -0.508334E-06  0.175E-03    3.02       8.7  T\n",
            "   6    -71.2353162 -0.260763E-07  0.645E-04    3.02      23.7  T\n",
            "   7    -71.2353162 -0.167006E-08  0.324E-04    3.02      47.0  T\n",
            "     SCC iter.                  ...        0 min,  0.071 sec\n",
            "     gradient                   ...        0 min,  0.071 sec\n",
            " * total energy  :   -70.3109260 Eh     change       -0.8697340E-04 Eh\n",
            "   gradient norm :     0.0016470 Eh/Œ±   predicted    -0.3706421E-04 ( -57.38%)\n",
            "   displ. norm   :     0.6649869 Œ±      lambda       -0.7200908E-04\n",
            "   maximum displ.:     0.4644403 Œ±      in ANC's #2, #1, #3, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  132 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2345991 -0.712346E+02  0.909E-02    3.02       0.0  T\n",
            "   2    -71.2346041 -0.504195E-05  0.518E-02    3.01       1.0  T\n",
            "   3    -71.2346009  0.319294E-05  0.136E-02    3.02       1.1  T\n",
            "   4    -71.2346050 -0.410287E-05  0.534E-03    3.01       2.9  T\n",
            "   5    -71.2346058 -0.717677E-06  0.218E-03    3.01       7.0  T\n",
            "   6    -71.2346058 -0.367698E-07  0.803E-04    3.01      19.0  T\n",
            "   7    -71.2346058 -0.256131E-08  0.412E-04    3.01      37.0  T\n",
            "     SCC iter.                  ...        0 min,  0.069 sec\n",
            "     gradient                   ...        0 min,  0.061 sec\n",
            " * total energy  :   -70.3110155 Eh     change       -0.8946018E-04 Eh\n",
            "   gradient norm :     0.0022740 Eh/Œ±   predicted    -0.4308420E-04 ( -51.84%)\n",
            "   displ. norm   :     0.5203326 Œ±      lambda       -0.7490142E-04\n",
            "   maximum displ.:     0.3646359 Œ±      in ANC's #2, #1, #3, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  133 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2347804 -0.712348E+02  0.711E-02    3.02       0.0  T\n",
            "   2    -71.2347834 -0.298439E-05  0.404E-02    3.01       1.0  T\n",
            "   3    -71.2347818  0.162978E-05  0.100E-02    3.02       1.5  T\n",
            "   4    -71.2347839 -0.212028E-05  0.402E-03    3.01       3.8  T\n",
            "   5    -71.2347843 -0.412202E-06  0.162E-03    3.01       9.4  T\n",
            "   6    -71.2347843 -0.217222E-07  0.598E-04    3.01      25.5  T\n",
            "   7    -71.2347843 -0.143187E-08  0.309E-04    3.01      49.4  T\n",
            "     SCC iter.                  ...        0 min,  0.048 sec\n",
            "     gradient                   ...        0 min,  0.062 sec\n",
            " * total energy  :   -70.3110778 Eh     change       -0.6237449E-04 Eh\n",
            "   gradient norm :     0.0027654 Eh/Œ±   predicted    -0.4759013E-04 ( -23.70%)\n",
            "   displ. norm   :     0.6022534 Œ±      lambda       -0.7908303E-04\n",
            "   maximum displ.:     0.4209884 Œ±      in ANC's #2, #1, #3, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  134 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2349128 -0.712349E+02  0.825E-02    3.02       0.0  T\n",
            "   2    -71.2349164 -0.364155E-05  0.470E-02    3.01       1.0  T\n",
            "   3    -71.2349142  0.216275E-05  0.111E-02    3.02       1.4  T\n",
            "   4    -71.2349170 -0.279857E-05  0.452E-03    3.01       3.4  T\n",
            "   5    -71.2349175 -0.506462E-06  0.186E-03    3.01       8.2  T\n",
            "   6    -71.2349176 -0.269431E-07  0.678E-04    3.01      22.5  T\n",
            "   7    -71.2349176 -0.180766E-08  0.360E-04    3.01      42.3  T\n",
            "     SCC iter.                  ...        0 min,  0.078 sec\n",
            "     gradient                   ...        0 min,  0.057 sec\n",
            " * total energy  :   -70.3111422 Eh     change       -0.6434539E-04 Eh\n",
            "   gradient norm :     0.0027998 Eh/Œ±   predicted    -0.5388726E-04 ( -16.25%)\n",
            "   displ. norm   :     0.5294644 Œ±      lambda       -0.7654751E-04\n",
            "   maximum displ.:     0.3674830 Œ±      in ANC's #2, #1, #3, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  135 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2368765 -0.712369E+02  0.745E-02    3.02       0.0  T\n",
            "   2    -71.2368788 -0.237896E-05  0.426E-02    3.02       1.0  T\n",
            "   3    -71.2368775  0.128425E-05  0.854E-03    3.02       1.8  T\n",
            "   4    -71.2368793 -0.173387E-05  0.381E-03    3.02       4.0  T\n",
            "   5    -71.2368796 -0.314847E-06  0.157E-03    3.02       9.7  T\n",
            "   6    -71.2368796 -0.195691E-07  0.563E-04    3.02      27.1  T\n",
            "   7    -71.2368796 -0.105304E-08  0.313E-04    3.02      48.8  T\n",
            "     SCC iter.                  ...        0 min,  0.065 sec\n",
            "     gradient                   ...        0 min,  0.059 sec\n",
            " * total energy  :   -70.3112031 Eh     change       -0.6088432E-04 Eh\n",
            "   gradient norm :     0.0020725 Eh/Œ±   predicted    -0.4900537E-04 ( -19.51%)\n",
            "   displ. norm   :     0.6182746 Œ±      lambda       -0.6693898E-04\n",
            "   maximum displ.:     0.4255592 Œ±      in ANC's #2, #1, #3, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  136 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2357027 -0.712357E+02  0.882E-02    3.02       0.0  T\n",
            "   2    -71.2357065 -0.375859E-05  0.504E-02    3.02       1.0  T\n",
            "   3    -71.2357044  0.209395E-05  0.109E-02    3.02       1.4  T\n",
            "   4    -71.2357071 -0.274983E-05  0.452E-03    3.02       3.4  T\n",
            "   5    -71.2357076 -0.482751E-06  0.189E-03    3.02       8.1  T\n",
            "   6    -71.2357077 -0.286698E-07  0.684E-04    3.02      22.3  T\n",
            "   7    -71.2357077 -0.210703E-08  0.371E-04    3.02      41.1  T\n",
            "     SCC iter.                  ...        0 min,  0.080 sec\n",
            "     gradient                   ...        0 min,  0.060 sec\n",
            " * total energy  :   -70.3112557 Eh     change       -0.5266199E-04 Eh\n",
            "   gradient norm :     0.0018488 Eh/Œ±   predicted    -0.4626619E-04 ( -12.15%)\n",
            "   displ. norm   :     0.7189904 Œ±      lambda       -0.6991759E-04\n",
            "   maximum displ.:     0.4741829 Œ±      in ANC's #2, #1, #3, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  137 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2396084 -0.712396E+02  0.112E-01    3.02       0.0  T\n",
            "   2    -71.2396129 -0.449552E-05  0.647E-02    3.02       1.0  T\n",
            "   3    -71.2396110  0.185518E-05  0.111E-02    3.03       1.4  T\n",
            "   4    -71.2396138 -0.280702E-05  0.513E-03    3.02       3.0  T\n",
            "   5    -71.2396142 -0.437299E-06  0.210E-03    3.02       7.3  T\n",
            "   6    -71.2396143 -0.404910E-07  0.779E-04    3.02      19.6  T\n",
            "   7    -71.2396143 -0.250724E-08  0.437E-04    3.02      34.9  T\n",
            "     SCC iter.                  ...        0 min,  0.064 sec\n",
            "     gradient                   ...        0 min,  0.057 sec\n",
            " * total energy  :   -70.3113044 Eh     change       -0.4865540E-04 Eh\n",
            "   gradient norm :     0.0024444 Eh/Œ±   predicted    -0.4299397E-04 ( -11.64%)\n",
            "   displ. norm   :     0.5777734 Œ±      lambda       -0.4911746E-04\n",
            "   maximum displ.:     0.3971068 Œ±      in ANC's #2, #1, #3, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  138 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2340142 -0.712340E+02  0.838E-02    3.02       0.0  T\n",
            "   2    -71.2340190 -0.478438E-05  0.480E-02    3.02       1.0  T\n",
            "   3    -71.2340155  0.345294E-05  0.137E-02    3.02       1.1  T\n",
            "   4    -71.2340199 -0.439651E-05  0.444E-03    3.02       3.4  T\n",
            "   5    -71.2340205 -0.537394E-06  0.188E-03    3.02       8.1  T\n",
            "   6    -71.2340205 -0.285163E-07  0.746E-04    3.02      20.5  T\n",
            "   7    -71.2340205 -0.342094E-08  0.352E-04    3.02      43.3  T\n",
            "     SCC iter.                  ...        0 min,  0.065 sec\n",
            "     gradient                   ...        0 min,  0.046 sec\n",
            " * total energy  :   -70.3113172 Eh     change       -0.1283491E-04 Eh\n",
            "   gradient norm :     0.0029150 Eh/Œ±   predicted    -0.3276143E-04 ( 155.25%)\n",
            "   displ. norm   :     0.1711353 Œ±      lambda       -0.2462084E-04\n",
            "   maximum displ.:     0.1177130 Œ±      in ANC's #2, #1, #3, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  139 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2367010 -0.712367E+02  0.253E-02    3.02       0.0  T\n",
            "   2    -71.2367014 -0.461423E-06  0.145E-02    3.02       1.0  T\n",
            "   3    -71.2367009  0.544724E-06  0.468E-03    3.02       3.3  T\n",
            "   4    -71.2367016 -0.686097E-06  0.125E-03    3.02      12.2  T\n",
            "   5    -71.2367016 -0.348813E-07  0.577E-04    3.02      26.4  T\n",
            "   6    -71.2367016 -0.251451E-08  0.224E-04    3.02      68.1  T\n",
            "     SCC iter.                  ...        0 min,  0.043 sec\n",
            "     gradient                   ...        0 min,  0.057 sec\n",
            " * total energy  :   -70.3113315 Eh     change       -0.1425118E-04 Eh\n",
            "   gradient norm :     0.0010711 Eh/Œ±   predicted    -0.1269164E-04 ( -10.94%)\n",
            "   displ. norm   :     0.0955338 Œ±      lambda       -0.9420228E-05\n",
            "   maximum displ.:     0.0574853 Œ±      in ANC's #2, #1, #3, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  140 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2376696 -0.712377E+02  0.177E-02    3.03       0.0  T\n",
            "   2    -71.2376698 -0.172651E-06  0.102E-02    3.03       1.5  T\n",
            "   3    -71.2376696  0.177964E-06  0.262E-03    3.03       5.8  T\n",
            "   4    -71.2376698 -0.207350E-06  0.103E-03    3.03      14.8  T\n",
            "   5    -71.2376699 -0.320388E-07  0.349E-04    3.03      43.7  T\n",
            "   6    -71.2376699 -0.976016E-09  0.127E-04    3.03     119.7  T\n",
            "     SCC iter.                  ...        0 min,  0.062 sec\n",
            "     gradient                   ...        0 min,  0.055 sec\n",
            " * total energy  :   -70.3113422 Eh     change       -0.1073651E-04 Eh\n",
            "   gradient norm :     0.0009819 Eh/Œ±   predicted    -0.4736577E-05 ( -55.88%)\n",
            "   displ. norm   :     0.3361580 Œ±      lambda       -0.1559298E-04\n",
            "   maximum displ.:     0.2199337 Œ±      in ANC's #2, #1, #3, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  141 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2372508 -0.712373E+02  0.552E-02    3.03       0.0  T\n",
            "   2    -71.2372525 -0.165352E-05  0.317E-02    3.03       1.0  T\n",
            "   3    -71.2372513  0.118528E-05  0.765E-03    3.03       2.0  T\n",
            "   4    -71.2372527 -0.145218E-05  0.302E-03    3.03       5.0  T\n",
            "   5    -71.2372530 -0.252015E-06  0.119E-03    3.03      12.8  T\n",
            "   6    -71.2372530 -0.118548E-07  0.435E-04    3.03      35.1  T\n",
            "   7    -71.2372530 -0.956973E-09  0.221E-04    3.03      69.0  T\n",
            "     SCC iter.                  ...        0 min,  0.069 sec\n",
            "     gradient                   ...        0 min,  0.061 sec\n",
            " * total energy  :   -70.3113581 Eh     change       -0.1588626E-04 Eh\n",
            "   gradient norm :     0.0013515 Eh/Œ±   predicted    -0.8188159E-05 ( -48.46%)\n",
            "   displ. norm   :     0.2270542 Œ±      lambda       -0.9629043E-05\n",
            "   maximum displ.:     0.1451360 Œ±      in ANC's #2, #1, #3, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  142 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2369057 -0.712369E+02  0.397E-02    3.03       0.0  T\n",
            "   2    -71.2369065 -0.844810E-06  0.229E-02    3.03       1.0  T\n",
            "   3    -71.2369057  0.861203E-06  0.614E-03    3.03       2.5  T\n",
            "   4    -71.2369067 -0.105660E-05  0.223E-03    3.03       6.8  T\n",
            "   5    -71.2369068 -0.124360E-06  0.892E-04    3.03      17.1  T\n",
            "   6    -71.2369068 -0.513221E-08  0.321E-04    3.03      47.6  T\n",
            "     SCC iter.                  ...        0 min,  0.063 sec\n",
            "     gradient                   ...        0 min,  0.056 sec\n",
            " * total energy  :   -70.3113670 Eh     change       -0.8955203E-05 Eh\n",
            "   gradient norm :     0.0015197 Eh/Œ±   predicted    -0.4928445E-05 ( -44.97%)\n",
            "   displ. norm   :     0.1712203 Œ±      lambda       -0.9046065E-05\n",
            "   maximum displ.:     0.1054324 Œ±      in ANC's #2, #3, #1, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  143 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2372176 -0.712372E+02  0.323E-02    3.03       0.0  T\n",
            "   2    -71.2372182 -0.575627E-06  0.187E-02    3.03       1.0  T\n",
            "   3    -71.2372177  0.490606E-06  0.483E-03    3.03       3.2  T\n",
            "   4    -71.2372183 -0.601468E-06  0.181E-03    3.03       8.4  T\n",
            "   5    -71.2372184 -0.858000E-07  0.697E-04    3.03      21.9  T\n",
            "   6    -71.2372184 -0.440743E-08  0.243E-04    3.03      62.7  T\n",
            "     SCC iter.                  ...        0 min,  0.061 sec\n",
            "     gradient                   ...        0 min,  0.073 sec\n",
            " * total energy  :   -70.3113751 Eh     change       -0.8076541E-05 Eh\n",
            "   gradient norm :     0.0010149 Eh/Œ±   predicted    -0.4583556E-05 ( -43.25%)\n",
            "   displ. norm   :     0.1308688 Œ±      lambda       -0.7125531E-05\n",
            "   maximum displ.:     0.0805927 Œ±      in ANC's #2, #3, #1, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  144 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2370479 -0.712370E+02  0.249E-02    3.03       0.0  T\n",
            "   2    -71.2370482 -0.314043E-06  0.147E-02    3.03       1.0  T\n",
            "   3    -71.2370478  0.376459E-06  0.386E-03    3.03       4.0  T\n",
            "   4    -71.2370483 -0.459694E-06  0.131E-03    3.03      11.6  T\n",
            "   5    -71.2370483 -0.449032E-07  0.540E-04    3.03      28.2  T\n",
            "   6    -71.2370483 -0.207591E-08  0.177E-04    3.03      86.0  T\n",
            "     SCC iter.                  ...        0 min,  0.066 sec\n",
            "     gradient                   ...        0 min,  0.063 sec\n",
            " * total energy  :   -70.3113828 Eh     change       -0.7665616E-05 Eh\n",
            "   gradient norm :     0.0009719 Eh/Œ±   predicted    -0.3592000E-05 ( -53.14%)\n",
            "   displ. norm   :     0.1802183 Œ±      lambda       -0.1030838E-04\n",
            "   maximum displ.:     0.1087680 Œ±      in ANC's #2, #3, #1, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  145 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2370577 -0.712371E+02  0.339E-02    3.03       0.0  T\n",
            "   2    -71.2370582 -0.503783E-06  0.201E-02    3.03       1.0  T\n",
            "   3    -71.2370577  0.591434E-06  0.483E-03    3.03       3.2  T\n",
            "   4    -71.2370584 -0.745345E-06  0.165E-03    3.03       9.2  T\n",
            "   5    -71.2370585 -0.576511E-07  0.716E-04    3.03      21.3  T\n",
            "   6    -71.2370585 -0.342655E-08  0.240E-04    3.03      63.5  T\n",
            "     SCC iter.                  ...        0 min,  0.062 sec\n",
            "     gradient                   ...        0 min,  0.064 sec\n",
            " * total energy  :   -70.3113918 Eh     change       -0.8986065E-05 Eh\n",
            "   gradient norm :     0.0014108 Eh/Œ±   predicted    -0.5231225E-05 ( -41.79%)\n",
            "   displ. norm   :     0.1414657 Œ±      lambda       -0.7230408E-05\n",
            "   maximum displ.:     0.0867529 Œ±      in ANC's #2, #3, #1, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  146 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2369381 -0.712369E+02  0.246E-02    3.03       0.0  T\n",
            "   2    -71.2369383 -0.241007E-06  0.145E-02    3.03       1.1  T\n",
            "   3    -71.2369380  0.263278E-06  0.306E-03    3.03       5.0  T\n",
            "   4    -71.2369384 -0.331448E-06  0.113E-03    3.03      13.5  T\n",
            "   5    -71.2369384 -0.280655E-07  0.456E-04    3.03      33.4  T\n",
            "   6    -71.2369384 -0.254572E-08  0.160E-04    3.03      95.2  T\n",
            "     SCC iter.                  ...        0 min,  0.069 sec\n",
            "     gradient                   ...        0 min,  0.068 sec\n",
            " * total energy  :   -70.3113993 Eh     change       -0.7515674E-05 Eh\n",
            "   gradient norm :     0.0012843 Eh/Œ±   predicted    -0.3651551E-05 ( -51.41%)\n",
            "   displ. norm   :     0.2506985 Œ±      lambda       -0.9846060E-05\n",
            "   maximum displ.:     0.1599563 Œ±      in ANC's #2, #3, #1, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  147 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2366172 -0.712366E+02  0.410E-02    3.03       0.0  T\n",
            "   2    -71.2366180 -0.738455E-06  0.239E-02    3.03       1.0  T\n",
            "   3    -71.2366173  0.624634E-06  0.504E-03    3.03       3.0  T\n",
            "   4    -71.2366181 -0.775668E-06  0.194E-03    3.03       7.9  T\n",
            "   5    -71.2366182 -0.102303E-06  0.804E-04    3.03      19.0  T\n",
            "   6    -71.2366182 -0.619832E-08  0.297E-04    3.03      51.4  T\n",
            "     SCC iter.                  ...        0 min,  0.064 sec\n",
            "     gradient                   ...        0 min,  0.062 sec\n",
            " * total energy  :   -70.3114098 Eh     change       -0.1047155E-04 Eh\n",
            "   gradient norm :     0.0013192 Eh/Œ±   predicted    -0.5063666E-05 ( -51.64%)\n",
            "   displ. norm   :     0.3215984 Œ±      lambda       -0.1461184E-04\n",
            "   maximum displ.:     0.1943841 Œ±      in ANC's #2, #3, #1, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  148 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2375260 -0.712375E+02  0.569E-02    3.04       0.0  T\n",
            "   2    -71.2375271 -0.111759E-05  0.333E-02    3.04       1.0  T\n",
            "   3    -71.2375263  0.836115E-06  0.592E-03    3.04       2.6  T\n",
            "   4    -71.2375274 -0.109302E-05  0.250E-03    3.04       6.1  T\n",
            "   5    -71.2375275 -0.143369E-06  0.989E-04    3.04      15.4  T\n",
            "   6    -71.2375275 -0.876663E-08  0.355E-04    3.04      42.9  T\n",
            "     SCC iter.                  ...        0 min,  0.057 sec\n",
            "     gradient                   ...        0 min,  0.059 sec\n",
            " * total energy  :   -70.3114222 Eh     change       -0.1248097E-04 Eh\n",
            "   gradient norm :     0.0016607 Eh/Œ±   predicted    -0.7642951E-05 ( -38.76%)\n",
            "   displ. norm   :     0.5364986 Œ±      lambda       -0.1324646E-04\n",
            "   maximum displ.:     0.3500110 Œ±      in ANC's #2, #3, #1, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  149 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2343980 -0.712344E+02  0.884E-02    3.04       0.0  T\n",
            "   2    -71.2344022 -0.419683E-05  0.512E-02    3.04       1.0  T\n",
            "   3    -71.2343987  0.348402E-05  0.125E-02    3.04       1.2  T\n",
            "   4    -71.2344030 -0.427146E-05  0.431E-03    3.04       3.5  T\n",
            "   5    -71.2344035 -0.542949E-06  0.184E-03    3.04       8.3  T\n",
            "   6    -71.2344036 -0.262214E-07  0.706E-04    3.04      21.6  T\n",
            "   7    -71.2344036 -0.305680E-08  0.325E-04    3.04      46.9  T\n",
            "     SCC iter.                  ...        0 min,  0.074 sec\n",
            "     gradient                   ...        0 min,  0.050 sec\n",
            " * total energy  :   -70.3114275 Eh     change       -0.5307463E-05 Eh\n",
            "   gradient norm :     0.0029610 Eh/Œ±   predicted    -0.7473105E-05 (  40.80%)\n",
            "   displ. norm   :     0.0914633 Œ±      lambda       -0.3014684E-04\n",
            "   maximum displ.:     0.0617909 Œ±      in ANC's #3, #2, #4, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  150 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2372388 -0.712372E+02  0.214E-02    3.05       0.0  T\n",
            "   2    -71.2372390 -0.190249E-06  0.125E-02    3.05       1.2  T\n",
            "   3    -71.2372387  0.273451E-06  0.315E-03    3.05       4.8  T\n",
            "   4    -71.2372390 -0.353628E-06  0.847E-04    3.05      18.0  T\n",
            "   5    -71.2372391 -0.779475E-08  0.337E-04    3.05      45.2  T\n",
            "     SCC iter.                  ...        0 min,  0.039 sec\n",
            "     gradient                   ...        0 min,  0.056 sec\n",
            " * total energy  :   -70.3114402 Eh     change       -0.1266511E-04 Eh\n",
            "   gradient norm :     0.0022566 Eh/Œ±   predicted    -0.1520218E-04 (  20.03%)\n",
            "   displ. norm   :     0.0854283 Œ±      lambda       -0.1395857E-04\n",
            "   maximum displ.:     0.0491177 Œ±      in ANC's #3, #2, #1, ...\n",
            " * RMSD in coord.:     1.3656936 Œ±      energy gain  -0.1107996E-02 Eh\n",
            "\n",
            "generating ANC from model Hessian ...\n",
            "Using Lindh-Hessian (1995)\n",
            " Shifting diagonal of input Hessian by   9.844407030872684E-003\n",
            " Lowest  eigenvalues of input Hessian\n",
            "    0.000000    0.000000    0.000000    0.000000    0.000000    0.000000\n",
            "    0.010000    0.010046    0.010211    0.010882    0.011410    0.012557\n",
            "    0.012892    0.013216    0.013726    0.014482    0.015715    0.015951\n",
            " Highest eigenvalues\n",
            "    2.048718    2.055379    2.140729    2.150044    2.256699    2.451431\n",
            " \n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  151 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2376205 -0.712376E+02  0.183E-02    3.05       0.0  T\n",
            "   2    -71.2376207 -0.128641E-06  0.109E-02    3.05       1.4  T\n",
            "   3    -71.2376205  0.122410E-06  0.211E-03    3.05       7.2  T\n",
            "   4    -71.2376207 -0.160142E-06  0.724E-04    3.05      21.1  T\n",
            "   5    -71.2376207 -0.987090E-08  0.317E-04    3.05      48.2  T\n",
            "     SCC iter.                  ...        0 min,  0.046 sec\n",
            "     gradient                   ...        0 min,  0.065 sec\n",
            " * total energy  :   -70.3114498 Eh     change       -0.9589792E-05 Eh\n",
            "   gradient norm :     0.0013386 Eh/Œ±   predicted     0.0000000E+00 (-100.00%)\n",
            "   displ. norm   :     0.0215791 Œ±      lambda       -0.1704485E-05\n",
            "   maximum displ.:     0.0079531 Œ±      in ANC's #15, #13, #8, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  152 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2379368 -0.712379E+02  0.669E-03    3.05       0.0  T\n",
            "   2    -71.2379368 -0.276318E-07  0.387E-03    3.05       3.9  T\n",
            "   3    -71.2379368  0.179592E-07  0.985E-04    3.05      15.5  T\n",
            "   4    -71.2379368 -0.235217E-07  0.389E-04    3.05      39.2  T\n",
            "     SCC iter.                  ...        0 min,  0.043 sec\n",
            "     gradient                   ...        0 min,  0.079 sec\n",
            " * total energy  :   -70.3114557 Eh     change       -0.5884658E-05 Eh\n",
            "   gradient norm :     0.0004493 Eh/Œ±   predicted    -0.3005676E-05 ( -48.92%)\n",
            "   displ. norm   :     0.0373630 Œ±      lambda       -0.4647698E-05\n",
            "   maximum displ.:     0.0145297 Œ±      in ANC's #8, #15, #4, ...\n",
            "\n",
            "........................................................................\n",
            ".............................. CYCLE  153 ..............................\n",
            "........................................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2380716 -0.712381E+02  0.876E-03    3.05       0.0  T\n",
            "   2    -71.2380716 -0.366774E-07  0.496E-03    3.05       3.1  T\n",
            "   3    -71.2380716  0.236712E-07  0.122E-03    3.05      12.5  T\n",
            "   4    -71.2380716 -0.281119E-07  0.612E-04    3.05      24.9  T\n",
            "   5    -71.2380716 -0.762964E-08  0.186E-04    3.05      82.0  T\n",
            "     SCC iter.                  ...        0 min,  0.053 sec\n",
            "     gradient                   ...        0 min,  0.069 sec\n",
            " * total energy  :   -70.3114606 Eh     change       -0.4903719E-05 Eh\n",
            "   gradient norm :     0.0007234 Eh/Œ±   predicted    -0.2335216E-05 ( -52.38%)\n",
            "   displ. norm   :     0.0530073 Œ±      lambda       -0.4177233E-05\n",
            "   maximum displ.:     0.0200395 Œ±      in ANC's #8, #4, #15, ...\n",
            "\n",
            "   *** GEOMETRY OPTIMIZATION CONVERGED AFTER 153 ITERATIONS ***\n",
            "\n",
            "------------------------------------------------------------------------\n",
            " total energy gain   :        -0.0215972 Eh      -13.5525 kcal/mol\n",
            " total RMSD          :         3.4820253 a0        1.8426 √Ö\n",
            " total power (kW/mol):        -0.3706111 (step)   -4.0484 (real)\n",
            "------------------------------------------------------------------------\n",
            "\n",
            " ANCopt (total)                0 d,  0 h,  0 min, 14.006 sec\n",
            " optimizer setup                ...        0 min,  0.000 sec (  0.002%)\n",
            " model hessian                  ...        0 min,  0.148 sec (  1.055%)\n",
            " ANC generation                 ...        0 min,  0.022 sec (  0.155%)\n",
            " coordinate transformation      ...        0 min,  0.007 sec (  0.047%)\n",
            " single point calculation       ...        0 min, 13.721 sec ( 97.962%)\n",
            " optimization log               ...        0 min,  0.052 sec (  0.369%)\n",
            " hessian update                 ...        0 min,  0.019 sec (  0.134%)\n",
            " rational function              ...        0 min,  0.024 sec (  0.174%)\n",
            "\n",
            "================\n",
            " final structure:\n",
            "================\n",
            "43\n",
            " xtb: 6.7.0 (08769fc)\n",
            "C           -5.38940635858081       -0.09281392942543       -1.62157543131579\n",
            "C           -4.23943344221169       -0.49267316071980       -0.75148796161917\n",
            "C           -3.88587706124477       -1.82525993340304       -0.58335295116622\n",
            "N           -2.88863105207705       -2.22874358683721        0.17540199093572\n",
            "C           -2.14358453923092       -1.32802376317536        0.80479018533147\n",
            "C           -2.36934543736326        0.04679680397360        0.69717841326816\n",
            "C           -3.45803698898169        0.45218497188428       -0.08106788124485\n",
            "O           -3.73721482003351        1.80135030680773       -0.21722423061408\n",
            "C           -4.71800844313505        2.27377940313145        0.68133015047824\n",
            "C           -1.48511903781236        1.05954487643970        1.35404555483722\n",
            "C           -1.04661320303918       -1.92036312466057        1.61977086802876\n",
            "S            0.04347547479708       -2.89239201185228        0.45363764983982\n",
            "O            0.84730353406379       -3.80237799296632        1.28407342981580\n",
            "C            1.21787865776442       -1.55054516607273        0.17254059209574\n",
            "N            0.99205971766160       -0.33521598608057       -0.22836249395691\n",
            "C            2.22875675857793        0.25927183742353       -0.29230304529166\n",
            "C            3.22232490937665       -0.67368455099243        0.08857224226019\n",
            "N            2.53855262295791       -1.82382099442775        0.37168991090722\n",
            "C            4.57256021750363       -0.35569639046986        0.12720993768171\n",
            "C            4.91624953469089        0.93831129236282       -0.22914864605678\n",
            "C            3.93057416289689        1.87341279946789       -0.60563874973414\n",
            "C            2.59575661800247        1.55655740043045       -0.64219166018973\n",
            "O            6.19019036497287        1.41944478358739       -0.24933071249286\n",
            "C            7.24154609871348        0.55385933118568        0.10060147344521\n",
            "H           -6.32325494414460       -0.11985370144267       -1.05843586878840\n",
            "H           -5.48385298530286       -0.77926408089639       -2.45943289815050\n",
            "H           -5.24159519167427        0.91654911048690       -1.99848650131287\n",
            "H           -4.43993799909669       -2.60936497685819       -1.08283980519977\n",
            "H           -5.67028349282925        1.74916352957019        0.55414797539846\n",
            "H           -4.85342980050054        3.32988810164878        0.45427494173523\n",
            "H           -4.39164589179887        2.16262272233856        1.72156600719087\n",
            "H           -0.46865873914399        0.93523505499834        0.98335265463404\n",
            "H           -1.48670447957558        0.94204752104446        2.43731791587357\n",
            "H           -1.82251044204056        2.06166399109598        1.10400831324591\n",
            "H           -0.45095784668333       -1.19417634635622        2.16141679405167\n",
            "H           -1.43666573200526       -2.68209456385283        2.29425254115607\n",
            "H            2.87753038946151       -2.70807761132856        0.71725238458564\n",
            "H            5.30015930092154       -1.09430036245959        0.42167878884873\n",
            "H            4.27117645577958        2.86187828695743       -0.86823207089534\n",
            "H            1.85089813211726        2.27832889717705       -0.93343456447739\n",
            "H            7.29540174771452       -0.30647818646246       -0.57428743483670\n",
            "H            7.14572861261376        0.20103168524934        1.13266685186258\n",
            "H            8.15264561791576        1.14229671345447        0.00558233981767\n",
            " \n",
            " Bond Distances (Angstroems)\n",
            " ---------------------------\n",
            "C1-C2=1.4965         C1-H25=1.0908        C1-H26=1.0873        C1-H27=1.0875        C2-C1=1.4965         C2-C3=1.3889         \n",
            "C2-C7=1.3974         C3-C2=1.3889         C3-N4=1.3164         C3-H28=1.0823        N4-C3=1.3164         N4-C5=1.3276         \n",
            "C5-N4=1.3276         C5-C6=1.3974         C5-C11=1.4894        C6-C5=1.3974         C6-C7=1.3983         C6-C10=1.4963        \n",
            "C7-C2=1.3974         C7-C6=1.3983         C7-O8=1.3845         O8-C7=1.3845         O8-C9=1.4116         C9-O8=1.4116         \n",
            "C9-H29=1.0946        C9-H30=1.0887        C9-H31=1.0959        C10-C6=1.4963        C10-H32=1.0891       C10-H33=1.0896       \n",
            "C10-H34=1.0866       C11-C5=1.4894        C11-S12=1.8690       C11-H35=1.0842       C11-H36=1.0896       S12-C11=1.8690       \n",
            "S12-O13=1.4710       S12-C14=1.8052       O13-S12=1.4710       C14-S12=1.8052       C14-N15=1.2995       C14-N18=1.3633       \n",
            "N15-C14=1.2995       N15-C16=1.3737       C16-N15=1.3737       C16-C17=1.4152       C16-C22=1.3929       C17-C16=1.4152       \n",
            "C17-N18=1.3677       C17-C19=1.3877       N18-C14=1.3633       N18-C17=1.3677       N18-H37=1.0081       C19-C17=1.3877       \n",
            "C19-C20=1.3855       C19-H38=1.0778       C20-C19=1.3855       C20-C21=1.4099       C20-O23=1.3619       C21-C20=1.4099       \n",
            "C21-C22=1.3724       C21-H39=1.0780       C22-C16=1.3929       C22-C21=1.3724       C22-H40=1.0773       O23-C20=1.3619       \n",
            "O23-C24=1.4061       C24-O23=1.4061       C24-H41=1.0948       C24-H42=1.0949       C24-H43=1.0888       H25-C1=1.0908        \n",
            "H26-C1=1.0873        H27-C1=1.0875        H28-C3=1.0823        H29-C9=1.0946        H30-C9=1.0887        H31-C9=1.0959        \n",
            "H32-C10=1.0891       H33-C10=1.0896       H34-C10=1.0866       H35-C11=1.0842       H36-C11=1.0896       H37-N18=1.0081       \n",
            "H38-C19=1.0778       H39-C21=1.0780       H40-C22=1.0773       H41-C24=1.0948       H42-C24=1.0949       H43-C24=1.0888       \n",
            " C  H  Rav=1.0877 sigma=0.0057  Rmin=1.0773  Rmax=1.0959    18\n",
            " C  C  Rav=1.4175 sigma=0.0432  Rmin=1.3724  Rmax=1.4965    13\n",
            " N  H  Rav=1.0081 sigma=0.0000  Rmin=1.0081  Rmax=1.0081     1\n",
            " N  C  Rav=1.3414 sigma=0.0282  Rmin=1.2995  Rmax=1.3737     6\n",
            " O  C  Rav=1.3910 sigma=0.0196  Rmin=1.3619  Rmax=1.4116     4\n",
            " S  C  Rav=1.8371 sigma=0.0319  Rmin=1.8052  Rmax=1.8690     2\n",
            " S  O  Rav=1.4710 sigma=0.0000  Rmin=1.4710  Rmax=1.4710     1\n",
            " \n",
            " selected bond angles (degree)\n",
            " --------------------\n",
            "H25-C1-C2=110.55               H26-C1-C2=110.25               H26-C1-H25=107.93              H27-C1-C2=110.19               \n",
            "H27-C1-H25=108.56              H27-C1-H26=109.31              C3-C2-C1=121.49                C7-C2-C1=121.87                \n",
            "C7-C2-C3=116.64                N4-C3-C2=123.83                H28-C3-C2=120.59               H28-C3-N4=115.58               \n",
            "C5-N4-C3=119.37                C6-C5-N4=122.71                C11-C5-N4=113.76               C11-C5-C6=123.53               \n",
            "C7-C6-C5=116.99                C10-C6-C5=122.45               C10-C6-C7=120.54               C6-C7-C2=120.42                \n",
            "O8-C7-C2=119.93                O8-C7-C6=119.62                C9-O8-C7=113.81                H29-C9-O8=111.72               \n",
            "H30-C9-O8=106.16               H30-C9-H29=109.42              H31-C9-O8=111.31               H31-C9-H29=108.71              \n",
            "H31-C9-H30=109.48              H32-C10-C6=108.96              H33-C10-C6=111.26              H33-C10-H32=109.11             \n",
            "H34-C10-C6=109.86              H34-C10-H32=108.47             H34-C10-H33=109.13             \n",
            " \n",
            " selected dihedral angles (degree)\n",
            " ---------------------------------\n",
            "C3-C2-C1-H25= 86.28           C3-C2-C1-H26=327.03           C3-C2-C1-H27=206.28           C7-C2-C1-H25=265.50           \n",
            "C7-C2-C1-H26=146.24           C7-C2-C1-H27= 25.50           N4-C3-C2-C1=180.37            N4-C3-C2-C7=  1.11            \n",
            "H28-C3-C2-C1=  0.00           H28-C3-C2-C7=180.75           C5-N4-C3-C2=358.48            C5-N4-C3-H28=178.82           \n",
            "C6-C5-N4-C3=359.90            C11-C5-N4-C3=180.39           C7-C6-C5-N4=  1.98            C7-C6-C5-C11=181.43           \n",
            "C10-C6-C5-N4=183.12           C10-C6-C5-C11=  2.57          C6-C7-C2-C1=181.64            C6-C7-C2-C3=  0.89            \n",
            "O8-C7-C2-C1=359.76            O8-C7-C2-C3=179.01            C2-C7-C6-C5=357.68            C2-C7-C6-C10=176.57           \n",
            "O8-C7-C6-C5=179.56            O8-C7-C6-C10=358.44           C9-O8-C7-C2= 85.79            C9-O8-C7-C6=263.92            \n",
            "H29-C9-O8-C7=300.36           H30-C9-O8-C7=181.15           H31-C9-O8-C7= 62.09           H32-C10-C6-C5= 58.99          \n",
            "H32-C10-C6-C7=240.16          H33-C10-C6-C5=298.64          H33-C10-C6-C7=119.82          H34-C10-C6-C5=177.68          \n",
            "H34-C10-C6-C7=358.86          \n",
            "           -------------------------------------------------\n",
            "          |                Final Singlepoint                |\n",
            "           -------------------------------------------------\n",
            "\n",
            "          ...................................................\n",
            "          :                      SETUP                      :\n",
            "          :.................................................:\n",
            "          :  # basis functions                 121          :\n",
            "          :  # atomic orbitals                 120          :\n",
            "          :  # shells                           68          :\n",
            "          :  # electrons                       126          :\n",
            "          :  max. iterations                   250          :\n",
            "          :  Hamiltonian                  GFN2-xTB          :\n",
            "          :  restarted?                      false          :\n",
            "          :  GBSA solvation                  false          :\n",
            "          :  PC potential                    false          :\n",
            "          :  electronic temp.          300.0000000     K    :\n",
            "          :  accuracy                    1.0000000          :\n",
            "          :  -> integral cutoff          0.2500000E+02      :\n",
            "          :  -> integral neglect         0.1000000E-07      :\n",
            "          :  -> SCF convergence          0.1000000E-05 Eh   :\n",
            "          :  -> wf. convergence          0.1000000E-03 e    :\n",
            "          :  Broyden damping             0.4000000          :\n",
            "          :  net charge                          0          :\n",
            "          :  unpaired electrons                  0          :\n",
            "          ...................................................\n",
            "\n",
            " iter      E             dE          RMSdq      gap      omega  full diag\n",
            "   1    -71.2380716 -0.712381E+02  0.667E-05    3.05       0.0  T\n",
            "   2    -71.2380716 -0.219273E-10  0.441E-05    3.05     345.8  T\n",
            "   3    -71.2380716 -0.858336E-11  0.291E-05    3.05     524.5  T\n",
            "\n",
            "   *** convergence criteria satisfied after 3 iterations ***\n",
            "\n",
            "         #    Occupation            Energy/Eh            Energy/eV\n",
            "      -------------------------------------------------------------\n",
            "         1        2.0000           -0.7560196             -20.5723\n",
            "       ...           ...                  ...                  ...\n",
            "        57        2.0000           -0.4052236             -11.0267\n",
            "        58        2.0000           -0.3993095             -10.8658\n",
            "        59        2.0000           -0.3963691             -10.7858\n",
            "        60        2.0000           -0.3922602             -10.6739\n",
            "        61        2.0000           -0.3872524             -10.5377\n",
            "        62        2.0000           -0.3749864             -10.2039\n",
            "        63        2.0000           -0.3691740             -10.0457 (HOMO)\n",
            "        64                         -0.2572669              -7.0006 (LUMO)\n",
            "        65                         -0.2425594              -6.6004\n",
            "        66                         -0.2316648              -6.3039\n",
            "        67                         -0.2116978              -5.7606\n",
            "        68                         -0.2005333              -5.4568\n",
            "       ...                                ...                  ...\n",
            "       120                          0.8949406              24.3526\n",
            "      -------------------------------------------------------------\n",
            "                  HL-Gap            0.1119071 Eh            3.0451 eV\n",
            "             Fermi-level           -0.3132204 Eh           -8.5232 eV\n",
            "\n",
            " SCC (total)                   0 d,  0 h,  0 min,  0.131 sec\n",
            " SCC setup                      ...        0 min,  0.001 sec (  0.535%)\n",
            " Dispersion                     ...        0 min,  0.001 sec (  0.785%)\n",
            " classical contributions        ...        0 min,  0.000 sec (  0.251%)\n",
            " integral evaluation            ...        0 min,  0.021 sec ( 16.331%)\n",
            " iterations                     ...        0 min,  0.030 sec ( 22.844%)\n",
            " molecular gradient             ...        0 min,  0.077 sec ( 58.825%)\n",
            " printout                       ...        0 min,  0.001 sec (  0.391%)\n",
            "\n",
            "         :::::::::::::::::::::::::::::::::::::::::::::::::::::\n",
            "         ::                     SUMMARY                     ::\n",
            "         :::::::::::::::::::::::::::::::::::::::::::::::::::::\n",
            "         :: total energy             -70.311460593377 Eh    ::\n",
            "         :: gradient norm              0.000723136159 Eh/a0 ::\n",
            "         :: HOMO-LUMO gap              3.045146908210 eV    ::\n",
            "         ::.................................................::\n",
            "         :: SCC energy               -71.237891185459 Eh    ::\n",
            "         :: -> isotropic ES            0.091106763933 Eh    ::\n",
            "         :: -> anisotropic ES          0.006955573123 Eh    ::\n",
            "         :: -> anisotropic XC          0.029144495574 Eh    ::\n",
            "         :: -> dispersion             -0.048534951652 Eh    ::\n",
            "         :: repulsion energy           0.926430213997 Eh    ::\n",
            "         :: add. restraining           0.000000378085 Eh    ::\n",
            "         :: total charge              -0.000000000000 e     ::\n",
            "         :::::::::::::::::::::::::::::::::::::::::::::::::::::\n",
            "\n",
            "========================================================================\n",
            " RELAXED SCAN\n",
            "------------------------------------------------------------------------\n",
            " output written to xtbscan.log\n",
            " scaning constraint 1\n",
            "... step 1 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:         -70.31146059\n",
            "    bias energy:           0.00000038\n",
            "unbiased energy:         -70.31146097\n",
            "... step 2 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:         -70.31116672\n",
            "    bias energy:           0.00010336\n",
            "unbiased energy:         -70.31127008\n",
            "... step 3 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:         -70.31132706\n",
            "    bias energy:           0.00002500\n",
            "unbiased energy:         -70.31135206\n",
            "... step 4 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:         -70.31170540\n",
            "    bias energy:           0.00000583\n",
            "unbiased energy:         -70.31171123\n",
            "... step 5 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:         -70.31199324\n",
            "    bias energy:           0.00000701\n",
            "unbiased energy:         -70.31200025\n",
            "... step 6 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:         -70.31199447\n",
            "    bias energy:           0.00001835\n",
            "unbiased energy:         -70.31201282\n",
            "... step 7 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:         -70.31200524\n",
            "    bias energy:           0.00000887\n",
            "unbiased energy:         -70.31201412\n",
            "... step 8 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:         -70.31152918\n",
            "    bias energy:           0.00019702\n",
            "unbiased energy:         -70.31172620\n",
            "... step 9 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:         -70.31049241\n",
            "    bias energy:           0.00054821\n",
            "unbiased energy:         -70.31104062\n",
            "... step 10 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:         -70.30896570\n",
            "    bias energy:           0.00096458\n",
            "unbiased energy:         -70.30993028\n",
            "... step 11 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:         -70.30740829\n",
            "    bias energy:           0.00065348\n",
            "unbiased energy:         -70.30806177\n",
            "... step 12 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:         -70.30759047\n",
            "    bias energy:           0.00018015\n",
            "unbiased energy:         -70.30777062\n",
            "... step 13 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:         -70.31009773\n",
            "    bias energy:           0.00178767\n",
            "unbiased energy:         -70.31188540\n",
            "... step 14 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:         -70.31252724\n",
            "    bias energy:           0.00147310\n",
            "unbiased energy:         -70.31400034\n",
            "... step 15 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:         -70.31461316\n",
            "    bias energy:           0.00090807\n",
            "unbiased energy:         -70.31552123\n",
            "... step 16 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:         -70.31650483\n",
            "    bias energy:           0.00055712\n",
            "unbiased energy:         -70.31706196\n",
            "... step 17 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:         -70.31801468\n",
            "    bias energy:           0.00027345\n",
            "unbiased energy:         -70.31828813\n",
            "... step 18 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:         -70.31924909\n",
            "    bias energy:           0.00011637\n",
            "unbiased energy:         -70.31936546\n",
            "... step 19 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:         -70.32035631\n",
            "    bias energy:           0.00005574\n",
            "unbiased energy:         -70.32041206\n",
            "... step 20 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:         -70.32102260\n",
            "    bias energy:           0.00000320\n",
            "unbiased energy:         -70.32102579\n",
            "... step 21 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:         -70.32100305\n",
            "    bias energy:           0.00003934\n",
            "unbiased energy:         -70.32104239\n",
            "... step 22 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:         -70.32054526\n",
            "    bias energy:           0.00017938\n",
            "unbiased energy:         -70.32072464\n",
            "... step 23 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:         -70.31977244\n",
            "    bias energy:           0.00034642\n",
            "unbiased energy:         -70.32011886\n",
            "... step 24 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:         -70.31882229\n",
            "    bias energy:           0.00049557\n",
            "unbiased energy:         -70.31931785\n",
            "... step 25 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:         -70.31776872\n",
            "    bias energy:           0.00058816\n",
            "unbiased energy:         -70.31835688\n",
            "... step 26 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:         -70.31652102\n",
            "    bias energy:           0.00071444\n",
            "unbiased energy:         -70.31723546\n",
            "... step 27 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:         -70.31499780\n",
            "    bias energy:           0.00088415\n",
            "unbiased energy:         -70.31588195\n",
            "... step 28 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:         -70.31323503\n",
            "    bias energy:           0.00112910\n",
            "unbiased energy:         -70.31436413\n",
            "... step 29 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:         -70.31145366\n",
            "    bias energy:           0.00077156\n",
            "unbiased energy:         -70.31222522\n",
            "... step 30 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:         -70.31186522\n",
            "    bias energy:           0.00001903\n",
            "unbiased energy:         -70.31188424\n",
            "... step 31 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:         -70.31202357\n",
            "    bias energy:           0.00000279\n",
            "unbiased energy:         -70.31202636\n",
            "... step 32 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:         -70.31230278\n",
            "    bias energy:           0.00002525\n",
            "unbiased energy:         -70.31232802\n",
            "... step 33 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:         -70.31265478\n",
            "    bias energy:           0.00005935\n",
            "unbiased energy:         -70.31271413\n",
            "... step 34 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:         -70.31313076\n",
            "    bias energy:           0.00010796\n",
            "unbiased energy:         -70.31323872\n",
            "... step 35 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:         -70.31378386\n",
            "    bias energy:           0.00020094\n",
            "unbiased energy:         -70.31398480\n",
            "... step 36 ...\n",
            "      ----------------------------------------------------------- \n",
            "     |                   =====================                   |\n",
            "     |                        A N C O P T                        |\n",
            "     |                   =====================                   |\n",
            "     |               Approximate Normal Coordinate               |\n",
            "     |                Rational Function Optimizer                |\n",
            "      ----------------------------------------------------------- \n",
            " current energy:         -70.31473723\n",
            "    bias energy:           0.00037885\n",
            "unbiased energy:         -70.31511608\n",
            "           -------------------------------------------------\n",
            "          |                Property Printout                |\n",
            "           -------------------------------------------------\n",
            "\n",
            "    * Orbital Energies and Occupations\n",
            "\n",
            "         #    Occupation            Energy/Eh            Energy/eV\n",
            "      -------------------------------------------------------------\n",
            "         1        2.0000           -0.7559912             -20.5716\n",
            "       ...           ...                  ...                  ...\n",
            "        51        2.0000           -0.4515209             -12.2865\n",
            "        52        2.0000           -0.4472982             -12.1716\n",
            "        53        2.0000           -0.4347168             -11.8292\n",
            "        54        2.0000           -0.4314637             -11.7407\n",
            "        55        2.0000           -0.4203071             -11.4371\n",
            "        56        2.0000           -0.4181319             -11.3779\n",
            "        57        2.0000           -0.4104321             -11.1684\n",
            "        58        2.0000           -0.4031492             -10.9702\n",
            "        59        2.0000           -0.3897495             -10.6056\n",
            "        60        2.0000           -0.3880503             -10.5594\n",
            "        61        2.0000           -0.3828904             -10.4190\n",
            "        62        2.0000           -0.3767472             -10.2518\n",
            "        63        2.0000           -0.3666172              -9.9762 (HOMO)\n",
            "        64                         -0.2580448              -7.0218 (LUMO)\n",
            "        65                         -0.2425049              -6.5989\n",
            "        66                         -0.2358065              -6.4166\n",
            "        67                         -0.2054411              -5.5903\n",
            "        68                         -0.1943584              -5.2888\n",
            "        69                         -0.1738019              -4.7294\n",
            "        70                         -0.1643136              -4.4712\n",
            "        71                         -0.1169240              -3.1817\n",
            "        72                         -0.0774823              -2.1084\n",
            "        73                         -0.0534992              -1.4558\n",
            "        74                          0.0148105               0.4030\n",
            "       ...                                ...                  ...\n",
            "       120                          0.9047775              24.6202\n",
            "      -------------------------------------------------------------\n",
            "                  HL-Gap            0.1085724 Eh            2.9544 eV\n",
            "             Fermi-level           -0.3123310 Eh           -8.4990 eV\n",
            "\n",
            "     #   Z          covCN         q      C6AA      Œ±(0)\n",
            "     1   6 C        3.756    -0.103    22.522     6.766\n",
            "     2   6 C        2.958    -0.015    28.396     8.740\n",
            "     3   6 C        2.908     0.044    27.006     8.521\n",
            "     4   7 N        1.844    -0.249    27.340     7.749\n",
            "     5   6 C        2.995     0.091    25.963     8.358\n",
            "     6   6 C        2.960    -0.015    28.375     8.737\n",
            "     7   6 C        2.896     0.137    25.007     8.199\n",
            "     8   8 O        1.703    -0.307    19.108     5.902\n",
            "     9   6 C        3.656     0.046    20.080     6.432\n",
            "    10   6 C        3.756    -0.106    22.577     6.775\n",
            "    11   6 C        3.801    -0.091    22.228     6.712\n",
            "    12  16 S        2.873     0.412   125.384    18.846\n",
            "    13   8 O        0.859    -0.494    23.661     6.522\n",
            "    14   6 C        3.217     0.117    24.785     8.097\n",
            "    15   7 N        1.897    -0.281    28.147     7.865\n",
            "    16   6 C        3.232     0.060    25.840     8.252\n",
            "    17   6 C        3.106     0.045    26.853     8.491\n",
            "    18   7 N        2.803    -0.103    23.259     7.163\n",
            "    19   6 C        2.906    -0.091    30.282     9.023\n",
            "    20   6 C        2.873     0.139    24.960     8.190\n",
            "    21   6 C        2.924    -0.061    29.522     8.910\n",
            "    22   6 C        2.909    -0.039    28.970     8.826\n",
            "    23   8 O        1.706    -0.267    18.432     5.796\n",
            "    24   6 C        3.644     0.038    20.272     6.472\n",
            "    25   1 H        0.924     0.047     2.360     2.402\n",
            "    26   1 H        0.925     0.045     2.380     2.412\n",
            "    27   1 H        0.925     0.060     2.199     2.319\n",
            "    28   1 H        0.925     0.039     2.465     2.455\n",
            "    29   1 H        0.924     0.033     2.548     2.496\n",
            "    30   1 H        0.925     0.058     2.217     2.328\n",
            "    31   1 H        0.924     0.032     2.565     2.505\n",
            "    32   1 H        0.925     0.059     2.209     2.324\n",
            "    33   1 H        0.924     0.042     2.418     2.432\n",
            "    34   1 H        0.925     0.062     2.175     2.306\n",
            "    35   1 H        0.925     0.065     2.145     2.290\n",
            "    36   1 H        0.925     0.081     1.966     2.192\n",
            "    37   1 H        0.860     0.186     1.183     1.704\n",
            "    38   1 H        0.926     0.034     2.527     2.486\n",
            "    39   1 H        0.926     0.054     2.275     2.359\n",
            "    40   1 H        0.926     0.055     2.260     2.351\n",
            "    41   1 H        0.924     0.040     2.454     2.450\n",
            "    42   1 H        0.924     0.038     2.469     2.457\n",
            "    43   1 H        0.925     0.062     2.176     2.307\n",
            "\n",
            " Mol. C6AA /au¬∑bohr‚Å∂  :      23699.007947\n",
            " Mol. C8AA /au¬∑bohr‚Å∏  :     592244.407664\n",
            " Mol. Œ±(0) /au        :        239.921469\n",
            "\n",
            "\n",
            "Wiberg/Mayer (AO) data.\n",
            "largest (>0.10) Wiberg bond orders for each atom\n",
            "\n",
            " ---------------------------------------------------------------------------\n",
            "     #   Z sym  total        # sym  WBO       # sym  WBO       # sym  WBO\n",
            " ---------------------------------------------------------------------------\n",
            "     1   6 C    3.994 --     2 C    1.027    27 H    0.980    26 H    0.979\n",
            "                            25 H    0.966\n",
            "     2   6 C    3.987 --     3 C    1.395     7 C    1.365     1 C    1.027\n",
            "     3   6 C    3.983 --     4 N    1.443     2 C    1.395    28 H    0.962\n",
            "                             6 C    0.102\n",
            "     4   7 N    3.068 --     3 C    1.443     5 C    1.397\n",
            "     5   6 C    3.982 --     4 N    1.397     6 C    1.356    11 C    1.034\n",
            "     6   6 C    3.987 --     7 C    1.378     5 C    1.356    10 C    1.028\n",
            "                             3 C    0.102\n",
            "     7   6 C    3.944 --     6 C    1.378     2 C    1.365     8 O    1.001\n",
            "     8   8 O    2.106 --     7 C    1.001     9 C    0.971\n",
            "     9   6 C    3.948 --    30 H    0.980    31 H    0.979    29 H    0.979\n",
            "                             8 O    0.971\n",
            "    10   6 C    3.994 --     6 C    1.028    34 H    0.980    33 H    0.969\n",
            "                            32 H    0.969\n",
            "    11   6 C    3.982 --     5 C    1.034    35 H    0.967    36 H    0.960\n",
            "                            12 S    0.867\n",
            "    12  16 S    3.774 --    13 O    1.811    14 C    0.935    11 C    0.867\n",
            "    13   8 O    1.985 --    12 S    1.811\n",
            "    14   6 C    3.943 --    15 N    1.555    18 N    1.214    12 S    0.935\n",
            "    15   7 N    3.018 --    14 C    1.555    16 C    1.206    18 N    0.120\n",
            "    16   6 C    3.975 --    22 C    1.315    17 C    1.249    15 N    1.206\n",
            "    17   6 C    3.954 --    19 C    1.334    16 C    1.249    18 N    1.169\n",
            "    18   7 N    3.568 --    14 C    1.214    17 C    1.169    37 H    0.932\n",
            "                            15 N    0.120\n",
            "    19   6 C    3.977 --    20 C    1.421    17 C    1.334    38 H    0.964\n",
            "                            22 C    0.103\n",
            "    20   6 C    3.930 --    19 C    1.421    21 C    1.296    23 O    1.059\n",
            "    21   6 C    3.982 --    22 C    1.527    20 C    1.296    39 H    0.968\n",
            "    22   6 C    3.987 --    21 C    1.527    16 C    1.315    40 H    0.967\n",
            "                            19 C    0.103\n",
            "    23   8 O    2.223 --    20 C    1.059    24 C    0.988\n",
            "    24   6 C    3.948 --    23 O    0.988    43 H    0.980    42 H    0.976\n",
            "                            41 H    0.976\n",
            "    25   1 H    0.997 --     1 C    0.966\n",
            "    26   1 H    0.997 --     1 C    0.979\n",
            "    27   1 H    0.996 --     1 C    0.980\n",
            "    28   1 H    0.998 --     3 C    0.962\n",
            "    29   1 H    0.999 --     9 C    0.979\n",
            "    30   1 H    0.996 --     9 C    0.980\n",
            "    31   1 H    0.999 --     9 C    0.979\n",
            "    32   1 H    0.996 --    10 C    0.969\n",
            "    33   1 H    0.998 --    10 C    0.969\n",
            "    34   1 H    0.996 --    10 C    0.980\n",
            "    35   1 H    0.995 --    11 C    0.967\n",
            "    36   1 H    0.993 --    11 C    0.960\n",
            "    37   1 H    0.965 --    18 N    0.932\n",
            "    38   1 H    0.999 --    19 C    0.964\n",
            "    39   1 H    0.997 --    21 C    0.968\n",
            "    40   1 H    0.996 --    22 C    0.967\n",
            "    41   1 H    0.998 --    24 C    0.976\n",
            "    42   1 H    0.998 --    24 C    0.976\n",
            "    43   1 H    0.996 --    24 C    0.980\n",
            " ---------------------------------------------------------------------------\n",
            "\n",
            "Topologies differ in total number of bonds\n",
            "Writing topology from bond orders to xtbtopo.mol\n",
            "\n",
            "molecular dipole:\n",
            "                 x           y           z       tot (Debye)\n",
            " q only:       -0.539       0.692       0.063\n",
            "   full:       -1.117       0.718       0.868       4.033\n",
            "molecular quadrupole (traceless):\n",
            "                xx          xy          yy          xz          yz          zz\n",
            " q only:        2.620     -10.490     -15.197       9.718       2.703      12.576\n",
            "  q+dip:        4.344     -11.693     -15.981      14.616       0.176      11.637\n",
            "   full:        4.011     -14.821     -14.023      14.958      -0.251      10.012\n",
            "\n",
            "           -------------------------------------------------\n",
            "          |                Geometry Summary                 |\n",
            "           -------------------------------------------------\n",
            "\n",
            "      molecular mass/u    :      345.4164963\n",
            "   center of mass at/√Ö    :        0.3585731      -0.2770974      -0.2101423\n",
            "  moments of inertia/u¬∑√Ö¬≤ :        0.1360940E+04   0.3800479E+04   0.4679703E+04\n",
            "rotational constants/cm‚Åª¬π :        0.1238676E-01   0.4435660E-02   0.3602287E-02\n",
            "\n",
            " * 42 selected distances\n",
            "\n",
            "     #   Z          #   Z                                           value/√Ö\n",
            "     2   6 C        3   6 C                                       1.3886457\n",
            "     3   6 C        4   7 N                                       1.3161685\n",
            "     4   7 N        5   6 C                                       1.3264504\n",
            "     5   6 C        6   6 C                                       1.3991502\n",
            "     2   6 C        7   6 C                                       1.3979202\n",
            "     6   6 C        7   6 C                                       1.3973175\n",
            "     7   6 C        8   8 O                                       1.3840427\n",
            "     8   8 O        9   6 C                                       1.4116173\n",
            "     5   6 C       11   6 C                                       1.4858469\n",
            "    12  16 S       13   8 O                                       1.4685311\n",
            "    12  16 S       14   6 C                                       1.7904826 (max)\n",
            "    14   6 C       15   7 N                                       1.2986506\n",
            "    15   7 N       16   6 C                                       1.3723754\n",
            "    16   6 C       17   6 C                                       1.4160193\n",
            "    14   6 C       18   7 N                                       1.3609094\n",
            "    17   6 C       18   7 N                                       1.3662332\n",
            "    17   6 C       19   6 C                                       1.3877661\n",
            "    19   6 C       20   6 C                                       1.3852756\n",
            "    20   6 C       21   6 C                                       1.4090575\n",
            "    16   6 C       22   6 C                                       1.3925460\n",
            "    21   6 C       22   6 C                                       1.3727938\n",
            "    20   6 C       23   8 O                                       1.3641521\n",
            "    23   8 O       24   6 C                                       1.4053149\n",
            "     1   6 C       25   1 H                                       1.0909764\n",
            "     1   6 C       26   1 H                                       1.0880241\n",
            "     1   6 C       27   1 H                                       1.0878236\n",
            "     3   6 C       28   1 H                                       1.0827369\n",
            "     9   6 C       29   1 H                                       1.0949070\n",
            "     9   6 C       30   1 H                                       1.0885828\n",
            "     9   6 C       31   1 H                                       1.0958339\n",
            "    10   6 C       32   1 H                                       1.0887183\n",
            "    10   6 C       33   1 H                                       1.0903352\n",
            "    10   6 C       34   1 H                                       1.0868460\n",
            "    11   6 C       35   1 H                                       1.0850306\n",
            "    11   6 C       36   1 H                                       1.0875302\n",
            "    18   7 N       37   1 H                                       1.0075313 (min)\n",
            "    19   6 C       38   1 H                                       1.0781448\n",
            "    21   6 C       39   1 H                                       1.0784741\n",
            "    22   6 C       40   1 H                                       1.0774719\n",
            "    24   6 C       41   1 H                                       1.0950539\n",
            "    24   6 C       42   1 H                                       1.0951939\n",
            "    24   6 C       43   1 H                                       1.0888748\n",
            "\n",
            " * 7 distinct bonds (by element types)\n",
            "\n",
            "   Z      Z             #   av. dist./√Ö        max./√Ö        min./√Ö\n",
            "   1 H    6 C          18     1.0878088     1.0958339     1.0774719\n",
            "   6 C    6 C          11     1.4029399     1.4858469     1.3727938\n",
            "   1 H    7 N           1     1.0075313     1.0075313     1.0075313\n",
            "   6 C    7 N           6     1.3401312     1.3723754     1.2986506\n",
            "   6 C    8 O           4     1.3912818     1.4116173     1.3641521\n",
            "   6 C   16 S           1     1.7904826     1.7904826     1.7904826\n",
            "   8 O   16 S           1     1.4685311     1.4685311     1.4685311\n",
            "\n",
            "\n",
            "optimized geometry written to: xtbopt.xyz\n",
            "\n",
            "\n",
            "           -------------------------------------------------\n",
            "          | TOTAL ENERGY              -70.311460593377 Eh   |\n",
            "          | GRADIENT NORM               0.000723136159 Eh/Œ± |\n",
            "          | HOMO-LUMO GAP               2.954404058372 eV   |\n",
            "           -------------------------------------------------\n",
            "########################################################################\n",
            "[WARNING] Runtime exception occurred\n",
            "-2- restart_readRestart: Dimension missmatch in restart file.\n",
            "-1- restart_readRestart: Number of electron missmatch in restart file.\n",
            "########################################################################\n",
            "\n",
            "------------------------------------------------------------------------\n",
            " * finished run on 2024/07/03 at 14:22:03.930     \n",
            "------------------------------------------------------------------------\n",
            " total:\n",
            " * wall-time:     0 d,  0 h,  1 min, 46.464 sec\n",
            " *  cpu-time:     0 d,  0 h,  2 min, 53.101 sec\n",
            " * ratio c/w:     1.626 speedup\n",
            " SCF:\n",
            " * wall-time:     0 d,  0 h,  0 min,  0.107 sec\n",
            " *  cpu-time:     0 d,  0 h,  0 min,  0.201 sec\n",
            " * ratio c/w:     1.883 speedup\n",
            " ANC optimizer:\n",
            " * wall-time:     0 d,  0 h,  1 min, 46.328 sec\n",
            " *  cpu-time:     0 d,  0 h,  2 min, 52.856 sec\n",
            " * ratio c/w:     1.626 speedup\n",
            "\n",
            "normal termination of xtb\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "'''\n",
        "def extract_energies_from_xyz(filename):\n",
        "    energies = []\n",
        "\n",
        "    with open(filename, 'r') as file:\n",
        "        lines = file.readlines()\n",
        "\n",
        "        for line in lines:\n",
        "            if line.startswith(' energy:'):\n",
        "                energy = float(line.split()[1])\n",
        "                energies.append(energy)\n",
        "\n",
        "    return energies\n",
        "'''\n",
        "\n",
        "filename = 'xtbscan.log'  # Substitua pelo nome do seu arquivo XYZ\n",
        "energies = extract_energies_from_xyz(filename)\n",
        "#print(energies)\n",
        "\n",
        "# Definir os par√¢metros do scan\n",
        "start_angle = 60.0  # √¢ngulo inicial em graus\n",
        "end_angle = 420.0   # √¢ngulo final em graus\n",
        "num_steps = 36      # n√∫mero de passos\n",
        "\n",
        "# Gerar os √¢ngulos usando linspace do NumPy\n",
        "angles = np.linspace(start_angle, end_angle, num_steps)\n",
        "\n",
        "# Imprimir a lista de √¢ngulos\n",
        "#print(angles)"
      ],
      "metadata": {
        "id": "gCbTtBny8b5W"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "%matplotlib inline\n",
        "import matplotlib.pyplot as plt\n",
        "\n",
        "# Plotar os √¢ngulos e os valores de energia\n",
        "plt.figure()\n",
        "plt.plot(angles, energies, 'r-o')\n",
        "plt.xlabel('Dihedral angle (degrees)')\n",
        "plt.ylabel('Energy (hartrees)')\n",
        "plt.show()"
      ],
      "metadata": {
        "id": "g2MqzNdf86rt",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 449
        },
        "outputId": "10cbcb5c-c3f5-4280-e3d7-6cf0bff53182"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<Figure size 640x480 with 1 Axes>"
            ],
            "image/png": "iVBORw0KGgoAAAANSUhEUgAAAl0AAAGwCAYAAACTsNDqAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjcuMSwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy/bCgiHAAAACXBIWXMAAA9hAAAPYQGoP6dpAABxSklEQVR4nO3deVhUZfsH8O+AbC4gKrILagpumUv5ouKSJlqZRZaBuYWppaWpuWS2WKZpqbTY4oK+/XIpxTSXDBfKhdcd96UQEhW0UAdxAYTn98fTjAzMwAwMc2aG7+e65pqZc86cuc+AzO2z3I9KCCFARERERJXKQekAiIiIiKoCJl1EREREFsCki4iIiMgCmHQRERERWQCTLiIiIiILYNJFREREZAFMuoiIiIgsoJrSAZBUWFiIy5cvo1atWlCpVEqHQ0REREYQQuDmzZvw8/ODg0PpbVlMuqzE5cuXERgYqHQYREREVA7p6ekICAgo9RgmXVaiVq1aAOQPzd3dXeFoiIiIyBjZ2dkIDAzUfo+XhkmXldB0Kbq7uzPpIiIisjHGDA3iQHoiIiIiC2DSRURERGQBTLqIiIiILIBJFxEREZEFMOkiIiIisgAmXUREREQWwKSLiIiIyAKYdBERERFZAJMuIiIiIgtgRXoiKqmgANi1C8jIAHx9gfBwwNFR6aiIiGwaky4i0hUfD4wdC1y8eH9bQAAQGwtERioXFxGRjWP3IhHdFx8P9O+vm3ABwKVLcnt8vDJxERHZASZdRCQVFMgWLiFK7tNsGzdOHkdERCZj0kVE0q5dJVu4ihICSE+XxxERkcmYdBGRlJJi3HEZGZUbBxGRnWLSRVTVFRQA334LTJhg3PG+vpUbDxGRnbKJpCsxMREqlUrv7cCBA9rjjh07hvDwcLi6uiIwMBBz5swp9bxZWVno3bs3/Pz84OLigsDAQIwZMwbZ2dkl3r9t27ZwcXHBAw88gGXLlunsLygowPTp09GwYUO4ubmhcePG+OCDDyD0jY0hsrSCAiAxEVi5Ut4XHZO1cyfQti0wciSgVgPVSpnQrFIBgYGyfAQREZlO2IDc3FyRkZGhcxs+fLho2LChKCwsFEIIoVarhbe3txg4cKA4ceKEWLlypXBzcxPffPONwfNeu3ZNLFy4UBw4cECkpaWJbdu2iZCQEBEVFaU95vz586J69epi/Pjx4tSpU+Lzzz8Xjo6O4pdfftEeM3PmTFG3bl2xceNGkZqaKn788UdRs2ZNERsba/Q1qtVqAUCo1epyfEJEBqxdK0RAgBByRJa8BQQI8eWXQjzzzP1ttWsLsWCBEKtXC6FSyVvR12i2rV2r9BUREVkVU76/bSLpKi4vL094eXmJGTNmaLctXLhQeHp6itzcXO22yZMni5CQEJPOHRsbKwICArTPJ02aJFq0aKFzzIABA0RERIT2+RNPPCFeeuklnWMiIyPFwIEDjX5fJl1kdmvXlkyeit8cHYUYPVqIf/7RfV3xRC0wkAkXEZEepnx/20T3YnEbNmxAVlYWhg0bpt2WlJSELl26wNnZWbstIiICZ8+exfXr14067+XLlxEfH4+uXbvqnLdnz546x0VERCApKUn7vGPHjti+fTvOnTsHADh69Ch2796NPn36GHyv3NxcZGdn69yIzKa08g8aLi7A4cPAF18Adeve3x4ZCaSlAcHB8vmcOUBqKgujEhFVkE0mXUuWLEFERAQCAgK02zIzM+Ht7a1znOZ5ZmZmqeeLiopC9erV4e/vD3d3dyxevLjM82ZnZ+POnTsAgClTpuCFF15AaGgonJyc0KZNG4wbNw4DBw40+J6zZs2Ch4eH9hYYGGjcxRMZo6zyDwCQmwtcu6Z/n6Mj0KKFfOzhwSWAiIjMQNGka8qUKQYHyGtuZ86c0XnNxYsXsXXrVsTExJgtjvnz5+Pw4cNYv349UlJSMH78eJNe/8MPP+D777/HihUrcPjwYSxfvhyffPIJli9fbvA1U6dOhVqt1t7S09MrehlE9xlb1qG04zQtXWlpFY2GiIig8NqLEyZMwNChQ0s9plGjRjrP4+LiULduXTz11FM62318fHDlyhWdbZrnPj4+pb6Hj48PfHx8EBoaijp16iA8PBzTp0+Hr6+vwfO6u7vDzc0NAPDmm29qW7sAoFWrVvjrr78wa9YsDBkyRO97uri4wMXFpdS4iMrN2LIOpR2nSbr++qvC4RARkcJJl5eXF7y8vIw+XgiBuLg4DB48GE5OTjr7wsLCMG3aNOTn52v3JSQkICQkBJ6enka/R2FhIQA55kpz3s2bN+sck5CQgLCwMO3z27dvw8FBt9HQ0dFRey4iiwsPl4tUX7qkf1yXSiX3l1b+IShI3rOli4jILGxqTNeOHTuQmpqK4cOHl9gXHR0NZ2dnxMTE4OTJk1i9ejViY2N1ugrXrVuH0NBQ7fPNmzcjLi4OJ06cQFpaGjZt2oRRo0ahU6dOCP73f/mjRo3C+fPnMWnSJJw5cwYLFy7EDz/8gDfeeEN7nr59+2LmzJnYtGkT0tLSsG7dOsybNw/PPPNM5X0YRKVxdARiYw0nXACwYEHpY7XYvUhEZF6VP5nSfKKiokTHjh0N7j969Kjo3LmzcHFxEf7+/mL27Nk6++Pi4kTRS96xY4cICwsTHh4ewtXVVTRp0kRMnjxZXL9+Xed1O3fuFA899JBwdnYWjRo1EnFxcTr7s7OzxdixY0WDBg2Eq6uraNSokZg2bZpO+YqysGQEVYo+fUqWiTC2/MPVq/drdN29W/mxEhHZIFO+v1VCsGy6NcjOzoaHhwfUajXc3d2VDofsRUgIcO4cMGMG8MADcgxXeLhxsxGFAGrUAO7cAf74Q76eiIh0mPL9reiYLiKqRH/9JRMuR0fgtdeA2rVNe71KJbsYT5+WXYxMuoiIKsSmxnQRkQkSEuR9hw6mJ1wanMFIRGQ2TLqI7NWvv8r7xx4r/zk4g5GIyGyYdBHZo4ICYNs2+bhXr/KfhzMYiYjMhkkXkT06fBi4fh1wdwceeaT852H3IhGR2TDpIrJHmq7FHj2AahWYL8PuRSIis2HSRWSPzDGeC7jf0nXpEpCfX7FzERFVcUy6iOzNzZtAUpJ8XJHxXADg7Q24ugKFhcDFixWPjYioCmPSRWRvfvtNtko1agQ0blyxc6lUQIMG8jG7GImIKoRJF5G9MVfXogYH0xMRmQWTLiJ7o0m6Ktq1qMGyEUREZsGki8ieXLgAnD0LODgAjz5qnnNyBiMRkVkw6SKyJ+ZY+qc4di8SEZkFky4ie2Lu8VwAuxeJiMyESReRvTDX0j/FaboX09OBe/fMd14ioiqGSReRvThyBLh2reJL/xTn6ws4Ocmk7vJl852XiKiKYdJFZC80XYvdu8skyVwcHDiYnojIDJh0EdkLc5eKKIpJFxFRhTHpIrIHOTnA3r3ycWUkXZzBSERUYUy6iOyBZumfhg0rvvSPPmzpIiKqMCZdRPagaKkIlcr852fZCCKiCmPSRWQPNEVRK6NrEWD3IhGRGTDpIrJ16enA6dPmXfqnOE334oULsnQEERGZjEkXka3TtHI98gjg6Vk57+HnB1SrJseNZWRUznsQEdk5Jl1Etq4ylv4prlo1IDBQPmYXIxFRuTDpIrJlhYWVs/SPPpzBSERUIUy6iGzZkSNAVhZQqxbQoUPlvhdnMBIRVQiTLiJbVllL/+jDGYxERBXCpIvIllV2qYii2L1IRFQhTLqIbNWtW8Du3fKxJZIudi8SEVUIky4iW6VZ+ic4GHjggcp/P03SdeGCHMBPREQmYdJFZKs0XYuVtfRPcf7+sgBrbi5w5Urlvx8RkZ1h0kVkqzSD6C3RtQjIgfoBAfIxuxiJiEzGpIvIFl28CJw6VblL/+jDGYxEROXGpIvIFmm6Ftu3B+rUsdz7cgYjEVG5MekiskWWLBVRFGcwEhGVG5MuIltTWKh80sXuRSIik1VTOgAiMkFBAbBkCfDPP4CrK/Dww5Z9f3YvEhGVG1u6iGxFfLxsaRo5Uj6/exdo0kRut5SiLV1CWO59iYjsAJMuIlsQHw/07y9nLRZ16ZLcbqnEKzBQ1gS7cwf4+2/LvCcRkZ2wiaQrMTERKpVK7+3AgQPa444dO4bw8HC4uroiMDAQc+bMKfW8WVlZ6N27N/z8/ODi4oLAwECMGTMG2dnZ2mMyMjIQHR2Npk2bwsHBAePGjdN7rh9//BGhoaFwdXVFq1atsHnzZrNcOxEKCoCxY/W3LGm2jRsnj6tszs6An598zC5GIiKT2ETS1bFjR2RkZOjchg8fjoYNG6J9+/YAgOzsbPTq1QtBQUE4dOgQ5s6di/feew/ffvutwfM6ODigX79+2LBhA86dO4dly5Zh27ZtGDVqlPaY3NxceHl54e2330br1q31nmfv3r2IiopCTEwMjhw5gqeffhpPP/00Tpw4Yd4PgqqmXbtKtnAVJQSQni6PswTOYCQiKhebGEjv7OwMHx8f7fP8/HysX78er732GlT/Ln/y/fffIy8vD0uXLoWzszNatGiB5ORkzJs3DyNGjNB7Xk9PT7zyyiva50FBQXj11Vcxd+5c7bbg4GDExsYCAJYuXar3PLGxsejduzfefPNNAMAHH3yAhIQEfPHFF/j666/1viY3Nxe5ubna50Vb14h0ZGSY97iKCg4G9uzhDEYiIhPZREtXcRs2bEBWVhaGDRum3ZaUlIQuXbrA2dlZuy0iIgJnz57F9evXjTrv5cuXER8fj65du5oUT1JSEnr27KmzLSIiAklJSQZfM2vWLHh4eGhvgYGBJr0nVSG+vuY9rqI4g5GIqFxsMulasmQJIiIiEKBZBw5AZmYmvL29dY7TPM/MzCz1fFFRUahevTr8/f3h7u6OxYsXmxSPofcu7X2nTp0KtVqtvaWnp5v0nlSFhIffX/NQH5VKDnAPD7dMPKzVRURULoomXVOmTDE4QF5zO3PmjM5rLl68iK1btyImJsZsccyfPx+HDx/G+vXrkZKSgvHjx5vt3Ia4uLjA3d1d50akl6MjMGGC/n3/dq9jwQJ5nCVwTBcRUbkoOqZrwoQJGDp0aKnHNGrUSOd5XFwc6tati6eeekpnu4+PD65cuaKzTfO86HgwfXx8fODj44PQ0FDUqVMH4eHhmD59OnyN7K4x9N5lvS+R0ZKT5b2bmyzXoBEQIBOuyEjLxVK0e1GI+4kfERGVStGky8vLC15eXkYfL4RAXFwcBg8eDCcnJ519YWFhmDZtGvLz87X7EhISEBISAk9PT6Pfo7CwEAB0BrmXJSwsDNu3b9cpJ5GQkICwsDCjz0Fk0KVLwIoV8vG2bUBenhw07+sruxQt1cKl0aCBvL91C7h2Dahb17LvT0Rko2xi9qLGjh07kJqaiuHDh5fYFx0djffffx8xMTGYPHkyTpw4gdjYWMyfP197zLp16zB16lRtl+XmzZtx5coVPPzww6hZsyZOnjyJN998E506dUKwpgsFQPK/rQw5OTn4+++/kZycDGdnZzRv3hwAMHbsWHTt2hWffvopnnjiCaxatQoHDx4stVwFkdFiY4H8fKBLF6BjR6WjkcsP+frKxC8tjUkXEZGxhA2JiooSHTt2NLj/6NGjonPnzsLFxUX4+/uL2bNn6+yPi4sTRS95x44dIiwsTHh4eAhXV1fRpEkTMXnyZHH9+nWd1wEocQsKCtI55ocffhBNmzYVzs7OokWLFmLTpk0mXZtarRYAhFqtNul1ZOdu3BCiVi0hACF+/lnpaO77z39kTGvWKB0JEZGiTPn+VgnBBdSsQXZ2Njw8PKBWqzmonu6bOxeYNAlo3hw4fhxwsJIJx1FRwKpVwKefAhaYeEJEZK1M+f62kr/gRFRCXp4cJA8AEydaT8IFcAYjEVE5WNFfcSLSsWIFcPmyHD8VHa10NLpYIJWIyGRMuoiskRDAJ5/Ix+PGAS4uioZTAgukEhGZjEkXkTXasgU4eRKoVQsYOVLpaEoq2r3IYaFEREZh0kVkjTSLro8YAXh4KBuLPppaXdnZwI0bioZCRGQrmHQRWZsDB4DERKBaNdm1aI2qVwfq15eP2cVIRGQUJl1E1kbTyhUdXfpC10rjDEYiIpMw6SKyJikpwNq18vHEicrGUhbOYCQiMgmTLiJrMm8eUFgI9OkDtGqldDSl4wxGIiKTMOkishb//APExcnHb76pbCzGYPciEZFJmHQRWYsvvwTu3AHatQO6dVM6mrKxe5GIyCRMuoiswe3bwOefy8eTJgEqlbLxGIPdi0REJmHSRWQNli0DsrKAhg2ByEilozGOpqXr+nVArVY2FiIiG8Cki0hpBQXAp5/Kx+PHy/pctqBmTaBuXfmYrV1ERGVi0kWklIICWQR1/Hjg/HmgTh1g2DClozINuxiJiIzGpItICfHxMmHp3h347DO57d49YOtWRcMyGWcwEhEZjUkXkaXFxwP9+wMXL+puv3lTbo+PVyau8uAMRiIiozHpIrKkggJg7FhAiJL7NNvGjZPH2QJ2LxIRGY1JF5El7dpVsoWrKCGA9HR5nC1g9yIRkdGYdBFZUkaGeY9TGrsXiYiMZiNz08miCgpkS0tGBuDrC4SHA46OSkdlH3x9zXuc0jRJV1YWkJMjy0gQEZFebOkiXUVn1UVHy/vgYNsa3G3NwsMBd3fD+1UqIDBQHmcLPDwAT0/5mOO6iIhKxaSL7jM0q+7SpbJn1WlqTq1cKe9tZSC4pe3dK2cp6qNZ+mfBAttqWWQXIxGRUZh0kVSRWXVsHTPOP/8AUVHy8+zWDQgI0N0fEACsWWM7ywBpcAYjEZFROKbL3hk7PmvLFuNm1T31FNCjB9CsmbwdPAg8/3zJZE3TOmaLSURlKCwEhg6Vn0toKPDzz4Cbm32MneMMRiIiozDpsmfx8bL1qmgyFRAAxMbKRKigANixQy62/OOPxp1z82Z501CpDLeOqVSydaxfP9tMJsxp3jxg0ybA1RVYvfr+gPNu3RQNyyw03Yts6SIiKhWTLnulGZ9lqAXqmWeA/ftLb93SZ/Bg4PZt4PRp4MyZ0sduFa05ZQ/JRXn973/A1KnycWws8OCDysZjbmzpIiIyCsd02aOyxmcJIZOyixflzLNXX5WJQUDA/cHcxWlm1S1dKlvFTpyQLWTGsJWaU5Xh+nXghRfkuooDBgAvv6x0RObHpIuIyChMuuxRWVXPNd59F7h8GfjyS6BDB9kKA5RMvAzNqis+ENwQW6k5ZW5CADExstutcWPg228NJ7W2TNO9ePUqcOeOsrEQEVkxJl32yNiWpZAQOcZIIzJSDnz399c9ztCsuvDw0lvHNBISgPx842KyJ19+CaxbBzg5yXFcpdXnsmW1a9+/No7rIiIyiEmXPapI1fPISNlNtHMnsGKFvE9N1T8D0dGx7NYxAPjoI6BTJ+DcOePisgeHDwMTJsjHn3wCtGunbDyVSaVirS4iIiMw6bJHZbVAlVX13NFRDnyPipL3pc08LK11bO1a2cJTuzZw4ADQpg2waJHsdrO3YqpFr2fTJllGIy9Pztx87TWlo6t8rNVFRFQmzl60R5oWqP79S5Z0qIyq55GRMrkwVHOqY0dgyBBZnmLECJl4Xbokx5NpFC1lYWv0leYAgHr15MQDexzHVRwH0xMRlYktXfbK1PFZFVVa61hAgBzXNXeu3H7ggG7CBRi31JA1MrR0EiAr0CcmWjwkRbB7kYioTCoh9NUVIEvLzs6Gh4cH1Go13M054NrYivSWUFAgk8ArV/TvV6lkgpaaahvFVAsKZAuPoZmitnY9FbF2rUw+w8Lk+pJERFWEKd/f7F60d5oWKGuwa5fhhAuwvWKqZZXmsLXrqYjAQHl/+rRs3bPVJY2IrJ01/UeaTMbuRbIcY0tZ2EoxVXu7nvKKj5crHADAjRtc8JyossTHy39b3bsD0dH8t2aDmHSR5RhbyuLbb61/bNDff8uZisaw5+KwmjFt9jJGj8haGRo/yn9rNoVjuqxEpY3psiaaMVCXLulfoqgoFxdg/Hi5ZmGtWvdfb2qzurlfk5sLfPYZ8OGHQHZ26eex9zFdHNNGZBn8t2bVTPn+tomWrsTERKhUKr23AwcOaI87duwYwsPD4erqisDAQMyZM6fU82ZlZaF3797w8/ODi4sLAgMDMWbMGGQX+TLNyMhAdHQ0mjZtCgcHB4wbN67EeRYtWoTw8HB4enrC09MTPXv2xP79+812/XajrGKqKpUsJNq9u0xuZs0CmjQBliyRMy5NbVYvT1O8odesXStvzZsDkybJhKtNG+D99+/HXvx6APOW5rA2poxpI6Lyq+i/NXuri2jLhA3Izc0VGRkZOrfhw4eLhg0bisLCQiGEEGq1Wnh7e4uBAweKEydOiJUrVwo3NzfxzTffGDzvtWvXxMKFC8WBAwdEWlqa2LZtmwgJCRFRUVHaY1JTU8Xrr78uli9fLh566CExduzYEueJjo4WX375pThy5Ig4ffq0GDp0qPDw8BAXL140+hrVarUAINRqtfEfjK1au1aIgADN0tvyFhgotwshRGGhED/9JMQDD+geU/ymUsmb5nXF30OlMs9rit98fYWIixPi3j3jrsderVhR9mcFyOOIqPyM/bfWvbsQa9YIcf36/dfq+/sUEGD/f58syJTvb5tIuorLy8sTXl5eYsaMGdptCxcuFJ6eniI3N1e7bfLkySIkJMSkc8fGxoqAgAC9+7p27ao36Sru3r17olatWmL58uVGv2+VSrqEkAnLzp3yj8nOnfcTmKJyc4WYO7f0REilkglO0dffu1fyj0xFX6O5TZsmxM2b5bsee7Nzp3FfBDt3Kh0pkW3bssW4f2uam6OjEJ06CfHCC6b/h5VMZsr3t02WjNiwYQOysrIwbNgw7bakpCR06dIFzs7O2m0RERH4+OOPcf36dXh6epZ53suXLyM+Ph5du3atUHy3b99Gfn4+6tSpY/CY3Nxc5Obmap9nlzU+yN4YU8rC2Rlo37708V+aZvW6dYGaNeVYsPx845riO3WSr1OpgKys0l+j0bOnfJ/yXI+90Sw3ZWiMnmaciaHlpoiobHv3lr2UmEol/5ZFRQG//gqcPQvs2WP4eCHka8aNk6uJ2OsQCCtkE2O6iluyZAkiIiIQEBCg3ZaZmQlvb2+d4zTPMzMzSz1fVFQUqlevDn9/f7i7u2Px4sUVim/y5Mnw8/NDz549DR4za9YseHh4aG+BmjpHpMvYcgtqtfzyP39eJlTG2LcP2LxZrpX4v/+ZN56qoLQxehr2PKaNqDLduQNMnAh07gz8+SegaTgwNH70m2/kJJ8zZ+Ts7/HjSz8/x1wqQtGka8qUKQYHyGtuZ86c0XnNxYsXsXXrVsTExJgtjvnz5+Pw4cNYv349UlJSML6sX9ZSzJ49G6tWrcK6devg6upq8LipU6dCrVZrb+nGJgpVjbHlFpYuBQ4dApKSgPnzjXvN5MnydUuXAm++ad54qgpDy03VqlU5y00R2Rt9g9z/9z85UefTT2VyNHSo/A/l2rXGLe0WFCR7CYzB/0halgW6Ow26evWqOH36dKm3omO0hBBixowZwsvLS+Tl5elsHzRokOjXr5/Oth07dggA4tq1a0bHtGvXLgFAXL58ucS+ssZ0zZ07V3h4eIgDBw4Y/X4aVW5Ml7E0Y60MjesqbXxWZb+G7tOMaXv1Vfl5tW+vdERE1k/fIPdate7/HfL1FWLjRt3XGDt+1NgxlzExQty9W8kXat8qbUzX6dOnsWrVKuzatQt//fUXbt++DS8vL7Rp0wYRERF49tln4eLiYvT5vLy84OXlZUqCiLi4OAwePBhOTk46+8LCwjBt2jTk5+dr9yUkJCAkJMSo8VwahYWFAKAz3soYc+bMwcyZM7F161a0N/Z/GFQ2TRdW//6yGb3o2CFDZRks9Rq6TzOmrXFjYOFC4MgRICdH//g3Irpf7LT4eMibN+V9167AunX3uxU1jB0/WtaYS40lS4DffpN//x5//P52LjdUOYzJ4g4dOiR69OghXFxcxKOPPiqmTJkiPvvsM7Fo0SLx8ccfi0GDBong4GBRp04dMXv2bHG3krLmbdu2CQDi9OnTJfbduHFDeHt7i0GDBokTJ06IVatWierVq+uUjIiPj9eZzbhp0yaxdOlScfz4cZGamio2btwomjVrJjp16qRz7iNHjogjR46Idu3aiejoaHHkyBFx8uRJ7f7Zs2cLZ2dnsWbNGp2yFjf1zXIzgC1dZShPWQZLvYZ0BQXJz+3XX5WOhMg6GTNb2hwt65oyOMVb8DXbxo6VrWma7X37CvHnnywzYSKzl4wIDg4WX375pbhetPaHHnv37hUDBgwQM2fONCpQU0VFRYmOHTsa3H/06FHRuXNn4eLiIvz9/cXs2bN19sfFxYmieeaOHTtEWFiY8PDwEK6urqJJkyZi8uTJJa4TQIlbUFCQdn9QUJDeY959912jr41JlxHKU5bBUq+h+wYNkn+k335b6UiIrJMly62U9R/J7Gwh3nxTiGrV5D7NPctMGM2U72+jlgEq2mVnDFOPpyqyDBBVDYsXAy+/LLtHEhOVjobI+qxcKVe8KMuKFbIMREUZ01V45gwwZgywfbvh83C5Ib3MvgxQWQnUjRs3TDqeiOyYpi7X//4nl3MiIl3GzoI212xpzTiwqCh5ry9hCg0Fpk0r/TwsM1FhJpeM+Pjjj7F69Wrt8+effx5169aFv78/jh49atbgiMgGNW0K1K8vE64ia6MS0b80g9wN1bdTqYDAQMsXFi6jpqUWy0yUm8lJ19dff60t5JmQkICEhARs2bIFffr0wZvG1joiIvulUt3/suD/iIlK0syWNrSSA6DMbGlLt8BVQSYnXZmZmdqka+PGjXj++efRq1cvTJo0CQf4v1oiAoAuXeT9778rGweRtYqMBMLCSm7XV+zUUspqgQPkUmtNm1ouJjtjctLl6emprZ7+yy+/aJe6EUKgoKDAvNERkW3SJF179shBvESkSwg5IB2QrV4rVgA7d8ptSq3kYMzSXrm5sto9/0NVLiYnXZGRkYiOjsZjjz2GrKws9OnTBwBw5MgRPPDAA2YPkIhsUKtWgLu7LPTIsZ5EJaWkyDFUzs7AiBGlD3K3JENLewUGyi7P5s3lmK5HHwU+/hj4t6A4GcfkpGv+/PkYM2YMmjdvjoSEBNT8t+J0RkYGXn31VbMHSEQ2yNFRLtQL8H/ERPpoxjs+/DBQyjq9ioiMlItm79yp2wI3diywfz8waJBswZ4yBXjqKSArS75O3zqSpMOkZYAAWQ5i4sSJJba/8cYbZgmIiOxEeDiwebP8chk3TuloiKyLJumy9AxFYxlabqhGDWD5cjmEYMwYYNMmoG1b4JVXgC+/BC5evH9sQIDsruTC91omt3QBwHfffYfOnTvDz88Pf/31FwBgwYIFWL9+vVmDIyIbphnXtWtX6Wu/EVVF1p50lUalAoYPl7X4HngAuHABmDpVN+EC5LqP/fvLdSYJQDmSrq+++grjx49Hnz59cOPGDe3g+dq1a2PBggXmjo+IbFX79rLb5O+/gbNnlY6GyHpkZgJ//imTl44dlY6m/B56SHY3urnp36/5z9a4cexq/JfJSdfnn3+ORYsWYdq0aXAsMuCvffv2OH78uFmDIyIb5uwM/Oc/8jHHdRHdp2nlevBBoHZtRUOpsKNHgTt3DO9nFXsdJiddqampaNOmTYntLi4uuHXrllmCIiI7wXpdRCXZctdiccZWp2cVewDlSLoaNmyI5OTkEtt/+eUXNGvWzBwxEZG9YGV6opJ275b39pB0sYq9SUyevTh+/HiMHj0ad+/ehRAC+/fvx8qVKzFr1iwsXry4MmIkIlsVFgZUqyYH2v71FxAUpHRERMrKzr5fu84eki5NFftLlwxPmPH3t49rNQOTk67hw4fDzc0Nb7/9Nm7fvo3o6Gj4+fkhNjYWL7zwQmXESES2qkYNoF07YN8+2drFpIuqur17ZUHRxo3to/VHU8W+f385MUBf4uXiIgsl2/r4NTMoV8mIgQMH4o8//kBOTg4yMzNx8eJFxMTEmDs2IrIHmv/hclwXkX2N59IwVMXexweoVQs4fx7o00cmXlVcuZKue/fuYdu2bfjuu+/g9u9U0cuXLyMnJ8eswRGRHeBgeqL7NEmXZsUGe6Gviv3Fi/J6PT1lTa8nngCq+IQ7lRCmVS3866+/0Lt3b1y4cAG5ubk4d+4cGjVqhLFjxyI3Nxdff/11ZcVq17Kzs+Hh4QG1Wg13d3elwyEyn2vXgLp15eMrV4D69ZWNh0gpubmAh4e8P3sWaNpU6Ygs4+BBoEcPOZ6tRw/g558N1/ayQaZ8f5vc0jV27Fi0b98e169f17ZyAcAzzzyD7du3mx4tEdm3OnXkAtgAZzFS1XbggEy46tcHmjRROhrLad8e+OUXoGZNYPt24Nln5edQBZmcdO3atQtvv/02nJ2ddbYHBwfj0qVLZguMiOxI0SWBiKqqouO5VCplY7G0sDC5TqObG7BlCzBgAJCfX+UWyTY56SosLNQu/VPUxYsXUatWLbMERUR2hoPpiexzEL0punQBNmyQsxnXrwe6dgWCg4Hu3YHoaHkfHGzXazWanHT16tVLZ41FlUqFnJwcvPvuu3j88cfNGRsR2QvNl0xyMqBWKxoKkSIKCoA9e+Tjqpp0AUDPnjKpcnQEkpKq3CLZJg+kT09PR+/evSGEwB9//IH27dvjjz/+QL169fD777+jPgfJlgsH0pPde+ABICUF2LxZTh8nqkqSk4E2bWQJhevXZdJRVRUUAN7eQFaW/v0qlSy4mppqE5+TKd/fJhdHDQwMxNGjR7F69WocPXoUOTk5iImJwcCBA3UG1hMR6ejSRSZdv//OpIuqHk3XYseONpFIVKpduwwnXIDuItndulksLEswKenKz89HaGgoNm7ciIEDB2LgwIGVFRcR2ZsuXYC4OA6mp6rJntZbrKgqvEi2SWO6nJyccPfu3cqKhYjsmebLZv9+4M4dZWMhsiQhOIi+qCq8SLbJA+lHjx6Njz/+GPfu3auMeIjIXjVqBPj5yWni+/YpHQ2R5Zw/L1ttnJ2BRx5ROhrlaRbJNlQ2Q6UCAgPtMkE1eUzXgQMHsH37dvz6669o1aoVatSoobM/3k5nHBBRBalU8o/o6tV2OVaDyCBNK1f79oCrq7KxWIOyFskWAliwwC7HvpmcdNWuXRvPPvtsZcRCRPauSxeZdLFeF1Ul7FosSbNI9tixJctGuLnJBNUOmVwygioHS0ZQlXDihFwSqEYNOW3eyUnpiIgqX9OmwB9/ABs3ykWf6b6CApmUZmTI5ZGmTZPDD3r2BLZuBRxMHgVlcZW69uKjjz6KGzdu6H3TRx991NTTEVFV0rw54OkJ3LoFHDmidDRElS8zUyZcKhXQqZPS0VgfR0c51CAqSi6G/d//ypaubduAr75SOjqzMznpSkxMRF5eXontd+/exS5OBSei0jg4cEkgqlo0pSJatQJq11Y0FJvQtCnw8cfy8aRJMmG1I0aP6Tp27Jj28alTp5CZmal9XlBQgF9++QX+/v7mjY6I7E94uFx/bdcuYOJEpaMhqlwcz2W60aOBdeuAnTuBoUPlf9DsZFC90UnXQw89BJVKBZVKpbcb0c3NDZ9//rlZgyMiO9Sli7zftQsoLLSJMRtE5caky3QODrKQcqtWwN69wCefAJMnKx2VWRg9kP6vv/6CEAKNGjXC/v374eXlpd3n7OyM+vXrw9FOMlElcCA9VRn5+ffHdR0/DrRsqXRERJUjO1v+rhcWyoWc/fyUjsi2xMUBL70k65sdPCiTMCtUKQPpg4KC4O/vjyFDhqBu3boICgrS3nx9fZlwEZFxnJyAsDD5mOO6yJ7t3SsTLk1hYDLN0KHAk08CeXnA4MHy3saZvAzQunXrKisWIqoqNF2MTLrInnG9xYpRqYBFi4A6dYDkZODDD5WOqMJMHkzRr18//PTTT5UQChFVGZovoV27SlajJrIXHM9VcT4+90tHfPQRcOCAsvFUkMkV6Zs0aYIZM2Zgz549aNeuXYllgF5//XWzBUdEdqpDB9nNePmyXJeucWOlIyIyr9zc+2uMdu6sbCy27vnn5WzGVatkN+OBA3KMV0aGXBQ7PNxmZjeaXJG+YcOGhk+mUuH8+fMVDqoq4kB6qnI6dZJjXkaOBF54wab+cBKVac8emWzVry8LpBpa3JmMc+0a0KKF/Cxr1gRycu7vCwiQazlGRioSWqVWpE9NTTV4q6yEKzExUVuuovjtQJGmxmPHjiE8PByurq4IDAzEnDlzSj1vVlYWevfuDT8/P7i4uCAwMBBjxoxBdna29piMjAxER0ejadOmcHBwwLhx40o956pVq6BSqfD0009X5JKJ7Ft8vJy5CADffAN07w4EB8vtRPZA07XYuTMTLnOoU0fOZAR0Ey5Azgzt398m/n7YRIGcjh07IiMjQ+c2fPhwNGzYEO3/XRQzOzsbvXr1QlBQEA4dOoS5c+fivffew7fffmvwvA4ODujXrx82bNiAc+fOYdmyZdi2bRtGjRqlPSY3NxdeXl54++230bp161LjTEtLw8SJExHO/nsiw+Lj5R/Imzd1t9vQH06iMnE8l3kVFMglgvTRdNiNGyePs2LlWvD64sWL2LBhAy5cuFBiSaB58+aZLThD8vPz4e/vj9deew3Tp08HAHz11VeYNm0aMjMz4ezsDACYMmUKfvrpJ5w5c8boc3/22WeYO3cu0tPTS+zr1q0bHnroISxYsKDEvoKCAnTp0gUvvfQSdu3ahRs3bpg04YDdi1QlFBTIFq2LF/XvV6lkV0FqKrsayXYVFAB16wJqtRx71K6d0hHZvsRE2SJelp075VqOFmTK97fJA+m3b9+Op556Co0aNcKZM2fQsmVLpKWlQQiBtm3bljtoU2zYsAFZWVkYNmyYdltSUhK6dOmiTbgAICIiAh9//DGuX78OT0/PMs97+fJlxMfHo2vXribHNGPGDNSvXx8xMTFGrUGZm5uL3Nxc7fOiXZpEdmvXLsMJFyD/x5qeLo+z8B9OIrM5cUImXDVrAmX0kJCRMjLMe5xCTO5enDp1KiZOnIjjx4/D1dUVa9euRXp6Orp27YrnnnuuMmIsYcmSJYiIiEBAQIB2W2ZmJry9vXWO0zwvuk6kPlFRUahevTr8/f3h7u6OxYsXmxTP7t27sWTJEixatMjo18yaNQseHh7aW2BgoEnvSWST7OQPJ1GpNP/x7tgRqGZy2wbp4+tr3uMUYnLSdfr0aQwePBgAUK1aNdy5cwc1a9bEjBkz8LFmZXAjTZkyxeAAec2teNfgxYsXsXXrVsTExJgaukHz58/H4cOHsX79eqSkpGD8+PFGv/bmzZsYNGgQFi1ahHr16hn9uqlTp0KtVmtv+rozieyOnfzhJCoVx3OZX3i4HHpgaFKCSgUEBlr9Z25yCl6jRg3tOC5fX1+kpKSgRYsWAIB//vnHpHNNmDABQ4cOLfWYRo0a6TyPi4tD3bp18dRTT+ls9/HxwZUrV3S2aZ77+PiU+h4+Pj7w8fFBaGgo6tSpg/DwcEyfPh2+RvzhT0lJQVpaGvr27avdVlhYCEAmpWfPnkVjPTWIXFxc4OLiUub5ieyK5g/npUv6i6JqxnRZ+R9OIoOEYNJVGRwdZVmI/v3l3wl9fz8WLLD6saAmJ13/+c9/sHv3bjRr1gyPP/44JkyYgOPHjyM+Ph7/+c9/TDqXl5eXzsLZZRFCIC4uDoMHD4aTk5POvrCwMEybNg35+fnafQkJCQgJCTFqPJeGJmEqOt6qNKGhoTiumfr+r7fffhs3b95EbGwsuw2JirKTP5xEBp0/L7vHnZyARx5ROhr7EhkJrFkDjB2rOzbU0RFYsUKxOl2mMLl7cd68eejQoQMA4P3330ePHj2wevVqBAcHY8mSJWYPsKgdO3YgNTUVw4cPL7EvOjoazs7OiImJwcmTJ7F69WrExsbqdBWuW7cOoaGh2uebN29GXFwcTpw4gbS0NGzatAmjRo1Cp06dEBwcrD0uOTkZycnJyMnJwd9//43k5GScOnUKAODq6oqWLVvq3GrXro1atWqhZcuWOgP7iQj3/3D6+5fct3y5TfzhJNKroECuFQgATZoA/PtvfpGRQFqanKW4fDng6Sk/9/x8pSMzjrAhUVFRomPHjgb3Hz16VHTu3Fm4uLgIf39/MXv2bJ39cXFxougl79ixQ4SFhQkPDw/h6uoqmjRpIiZPniyuX7+u8zoAJW5BQUEG4xgyZIjo16+fSdemVqsFAKFWq016HZHNundPiJ07hfj+eyEaNBACEOK775SOiqh81q4VIiBA/h5rbgEBcjtVng8+kJ91mzZCFBYqEoIp39/lqtMFAHl5ebh69aq2O06jQYMGFUoCqyrW6aIq7d13gRkzgL59gQ0blI6GyDSagr/Fv041g77XrGELbmXJypID6O/cAXbsMK6Wl5lV6jJA586dQ3h4ONzc3BAUFISGDRuiYcOGCA4OLnVdRiIig55/Xt5v3SrrGxHZioICOcZIX/uFDVVKt1l16wKamp2ffqpsLEYwOekaNmwYHBwcsHHjRhw6dAiHDx/G4cOHceTIERw+fLgyYiQie9eiBdC8OZCXx5Yusi2mFPylyvHGG7JVcdMm4PRppaMplcmzF5OTk3Ho0CGdAelERBX23HPA++8DP/wADBqkdDRExmHBX+U98ADQrx/w00/AvHn3JzNYIZNbupo3b25yPS4iojJpVrTYuhW4cUPRUIiMxoK/1mHCBHn/3XdAsZqd1sSopCs7O1t7+/jjjzFp0iQkJiYiKytLZx/XDySicmvRQt7y89nFSLZDU/DXEBuplG7zOnWSddFyc4GFC5WOxiCjkq7atWvD09MTnp6eeOyxx/C///0PPXr0QP369bXbNccQEZWbprXrhx+UjYPIWJqCv/poZi+y4G/lU6mAiRPl44UL5WxGK2RUyYjffvvN6BN27dq1QgFVVSwZQQTg1CnZ2uXkBFy9CtSurXRERGW7fVsW6fx3iTytwECZcLFchGXcuyeL0qalAV9/DYwcaZG3NeX726iB9EykiMgimjcHWrYETpwA1q8HhgxROiKism3eLBOuoCAgLg7IzJRjuMLD2cJlSdWqyfIc48bJAfUvvww4mDx0vVIZFc2FCxdMOumlS5fKFQwREbsYyeZoflcHDJDFOaOigG7dmHAp4aWXAA8P4Nw5WULCyhiVdD388MMYOXIkDhw4YPAYtVqNRYsWoWXLlli7dq3ZAiSiKkaTdCUkANevKxsLUVlu3QI2bpSPNUV+STm1at3vVvzkE2Vj0cOo7sVTp05h5syZeOyxx+Dq6op27drBz88Prq6uuH79Ok6dOoWTJ0+ibdu2mDNnDh5//PHKjpuI7FWzZkCrVsDx47KLcehQpSMiMmzTJjlou1EjoG1bpaMhAHjtNdm9+PvvwMGDQPv2SkekZVRLV926dTFv3jxkZGTgiy++QJMmTfDPP//gjz/+AAAMHDgQhw4dQlJSEhMuIqo4djGSrdD8jj733P3ZiqSsgADZxQtY3dJA5V7wmsyLsxeJijhzRrZ4VasmCx3WqaN0REQl5eQA9evLlq5Dh9jSZU2OHgUeekiOqzt/HmjQoNLeqlIXvCYiqnShocCDD8op4OvXKx0NkX6arsXGjYE2bZSOhopq3Rro0UMuNG6ojpoCmHQRkXViFyNZO83v5vPPs2vRGmmWBlq0CFCrlY3lX0y6iMg6aZKubduAa9eUjYWouJs3ZX0ugLMWrVXv3rL2382bwOLFSkcDgEkXEVmrkBDZRXDvHvDTT0pHQ6Rr40bg7l1ZAb11a6WjIX1UqvutXfPny//ArVwJJCbKbkcFmJx03bp1qzLiICIqiV2MZK04a9E2DBwoi6VeugQ89hgQHS0L2AYHA/HxFg/H5KTL29sbL730Enbv3l0Z8RAR3Ve0izErS9lYiDRu3gS2bJGP2bVo3TZt0j+e69IloH9/iydeJidd//d//4dr167h0UcfRdOmTTF79mxcvny5MmIjoqquaVM57buggF2MZD1+/hnIzZW/nw8+qHQ0ZEhBATB2rP59mmpZ48ZZtKvR5KTr6aefxk8//YRLly5h1KhRWLFiBYKCgvDkk08iPj4e9+7dq4w4iaiqYhcjWRvOWrQNu3YBFy8a3i8EkJ4uj7OQcg+k9/Lywvjx43Hs2DHMmzcP27ZtQ//+/eHn54d33nkHt2/fNmecRFRVaZKu7duBf/5RNhai7Gx2LdqKjAzzHmcG5U66rly5gjlz5qB58+aYMmUK+vfvj+3bt+PTTz9FfHw8nn76aTOGSURVVpMmsvAkuxjJGmzYAOTlyQK+LVsqHQ2VxtfXvMeZgVELXhcVHx+PuLg4bN26Fc2bN8err76KF198EbVr19Ye07FjRzRr1syccRJRVfbcc8CRI7JbZ/hwpaOhqoyzFm1HeLhch/HSpftjuIpSqeT+8HCLhWRyS9ewYcPg5+eHPXv2IDk5GWPGjNFJuADAz88P06ZNM1eMRFTVaboYd+xgFyMpR60Gtm6Vj9m1aP0cHe8vAVQ8QdY8X7BAHmchJiddGRkZ+Oabb/Dwww8bPMbNzQ3vvvtuhQIjItJ64AG5mHBBAbBundLRUFWl6Vps1gxo0ULpaMgYkZHAmjWAv7/u9oAAuT0y0qLhmNy9eO/ePWRnZ5fYrlKp4OLiAmdnZ7MERkSk47nngMOHga+/BmrWlOMwwsMt+r9UquI4a9E2RUYC/frJWYoZGYr+7TA56apduzZUpfyyBQQEYOjQoXj33Xfh4MBVhojITNzd5f3hw7KqNCD/txoba/H/rVIVdOPG/a5FTXc32Q5HR6BbN6WjMD3pWrZsGaZNm4ahQ4fikUceAQDs378fy5cvx9tvv42///4bn3zyCVxcXPDWW2+ZPWAiqoLi44ExY0pu11SVVqCbgKqY9euB/HzZrciuRSonk5Ou5cuX49NPP8XzRQYR9u3bF61atcI333yD7du3o0GDBpg5cyaTLiKqOE1VaX2zj4SQ3TzjxsnuA3Y1UmUpOmuRqJxM7v/bu3cv2rRpU2J7mzZtkJSUBADo3LkzLly4UPHoiIissKo0VTHXrwMJCfIxky6qAJOTrsDAQCxZsqTE9iVLliAwMBAAkJWVBU9Pz4pHR0RkhVWlqYrRdC22bAk0b650NGTDTO5e/OSTT/Dcc89hy5Yt2rIRBw8exJkzZ7BmzRoAwIEDBzBgwADzRkpEVZMVVpWmKqborEWiClAJoW+gROnS0tLwzTff4OzZswCAkJAQjBw5EsHBweaOr8rIzs6Gh4cH1Go13DWztIhIjukKDi67qnRqKsd0kflduwZ4ewP37gGnT8vlf4iKMOX726SWrvz8fPTu3Rtff/01Zs2aVaEgiYiMoqkq3b+/TLD0JV4WripNVchPP8mE68EHmXBRhZk0psvJyQnHjh2rrFiIiPQzVFXaxYXlIqhyFBQAiYn3l5Hp31/RcMg+mDyQ/sUXX9Q7kJ6IqFJFRgJpacDOncC8eXJbfj7QtauiYZEdio+XXdrduwOahoaFC+V2ogoo1zJAS5cuxbZt29CuXTvUqFFDZ/88zR9DIiJz01SV7tYNWL4cOHpUzix76SWlIyN7ER8vW7WKd2NfucJCvFRhJiddJ06cQNu2bQEA586d09lX2vJARERm1b+/TLrWrGHSRebBQrxUyco1e5HMj7MXiUx09qwc2OzkJFshWBuQKioxUXYplmXnTqtYx4+sgynf3+VekfrPP//E1q1bcefOHQBAZeZuiYmJUKlUem8HDhzQHnfs2DGEh4fD1dUVgYGBmDNnTqnnzcrKQu/eveHn5wcXFxcEBgZizJgxyM7O1h6TkZGB6OhoNG3aFA4ODhg3bpzec924cQOjR4+Gr68vXFxc0LRpU2zevNks109EeoSEAK1ayXFd69crHQ3ZAxbipUpmctKVlZWFHj16oGnTpnj88ceR8e8vX0xMDCZMmGD2AAGgY8eOyMjI0LkNHz4cDRs2RPv27QHITLNXr14ICgrCoUOHMHfuXLz33nv49ttvDZ7XwcEB/fr1w4YNG3Du3DksW7YM27Ztw6hRo7TH5ObmwsvLC2+//TZat26t9zx5eXl47LHHkJaWhjVr1uDs2bNYtGgR/IvPtCIi89LMKPu3MDNRhbAQL1U2YaJBgwaJiIgIkZ6eLmrWrClSUlKEEEL88ssvonnz5qaerlzy8vKEl5eXmDFjhnbbwoULhaenp8jNzdVumzx5sggJCTHp3LGxsSIgIEDvvq5du4qxY8eW2P7VV1+JRo0aiby8PKPf5+7du0KtVmtv6enpAoBQq9UmxUtUpZ06JQQghJOTENevKx0N2bp794QICBBCpZK/V8VvKpUQgYHyOKJ/qdVqo7+/TW7p+vXXX/Hxxx8jICBAZ3uTJk3w119/mSkVLN2GDRuQlZWFYcOGabclJSWhS5cucHZ21m6LiIjA2bNncf36daPOe/nyZcTHx6OriVPQN2zYgLCwMIwePRre3t5o2bIlPvroIxQUFBh8zaxZs+Dh4aG9adatJCITNGsm18LLzwd+/lnpaMjWaQrxGlr5AGAhXqoQk5OuW7duoXr16iW2X7t2DS4uLmYJqixLlixBRESETuKXmZkJb29vneM0zzMzM0s9X1RUFKpXrw5/f3+4u7tj8eLFJsVz/vx5rFmzBgUFBdi8eTOmT5+OTz/9FB9++KHB10ydOhVqtVp7S09PN+k9iehfzz0n73/8Udk4yD707Qt4eJTcHhDAchFUYSYnXeHh4fjvf/+rfa5SqVBYWIg5c+aguzGzPoqYMmWKwQHymtuZM2d0XnPx4kVs3boVMTExpoZu0Pz583H48GGsX78eKSkpGD9+vEmvLywsRP369fHtt9+iXbt2GDBgAKZNm4avv/7a4GtcXFzg7u6ucyOictCM69q6FSgyCYaoXLZuBdRqoF49ICEBWLFCzlZMTWXCRRVmcp2uOXPmoEePHjh48CDy8vIwadIknDx5EteuXcOePXtMOteECRMwdOjQUo9p1KiRzvO4uDjUrVsXTz31lM52Hx8fXLlyRWeb5rmPj0+p7+Hj4wMfHx+EhoaiTp06CA8Px/Tp0+Fr5GBJX19fODk5wbFIk3OzZs2QmZmJvLw8nS5PIjKzFi1k6YgzZ2QX48CBSkdEtmzZMnk/aBDQs6eioZD9MTnpatmyJc6dO4cvvvgCtWrVQk5ODiIjI7XlEkzh5eUFLy8vo48XQiAuLg6DBw+Gk5OTzr6wsDBMmzYN+fn52n0JCQkICQmBpwn1ewoLCwHIWYvG6tSpE1asWIHCwkI4OMjGw3PnzsHX15cJF1FlU6lka9eHH8ruHyZdVF7//ANs2CAfl9EgQFQulT+u33y2bdsmAIjTp0+X2Hfjxg3h7e0tBg0aJE6cOCFWrVolqlevLr755hvtMfHx8TqzGTdt2iSWLl0qjh8/LlJTU8XGjRtFs2bNRKdOnXTOfeTIEXHkyBHRrl07ER0dLY4cOSJOnjyp3X/hwgVRq1YtMWbMGHH27FmxceNGUb9+ffHhhx8afW2mzH4gomKOHpWzy1xchMjOVjoaslWffSZ/j9q0UToSsiGmfH+b3NIFyEKg+/fvx9WrV7UtQxqDBw+ueCZowJIlS9CxY0eEhoaW2Ofh4YFff/0Vo0ePRrt27VCvXj288847GDFihPYYtVqNs2fPap+7ublh0aJFeOONN5Cbm4vAwEBERkZiypQpOudu06aN9vGhQ4ewYsUKBAUFIS0tDQAQGBiIrVu34o033sCDDz4If39/jB07FpMnTzbzJ0BEerVqBTRtCpw7B2zcCERFKR0R2SJN12KRmfFE5mTyMkA///wzBg4ciJycHLi7u+ust6hSqXDt2jWzB1kVcBkgogqaNg346CM52HntWqWjIVtz7BjQurVcVuryZTmQnsgIlboM0IQJE/DSSy8hJycHN27cwPXr17U3JlxEpBhN6YjNm4GcHGVjIduzfLm879uXCRdVGpOTrkuXLuH111/XW6uLiEgxrVsDjRsDd+/KxIvIWPn5wP/9n3zMAfRUiUxOuiIiInDw4MHKiIWIqPxUKhZKpfLZsgW4ehXw9gZ691Y6GrJjJg+kf+KJJ/Dmm2/i1KlTaNWqVYnSDcXrZxERWUz//sDs2bKl69YtoEYNpSMiW6AZQP/ii3JMF1ElMXkgvaYOld6TqVSlrjdIhnEgPZEZCCG7GFNTZWuXplo9kSF//w34+QH37snB9K1aKR0R2ZhKHUhfWFho8MaEi4gUpSmUCshCqURlWbFCJlzt2jHhokpnctJFRGTVNOO6Nm4E7txRNhayfqzNRRZkdNL1+OOPQ61Wa5/Pnj0bN27c0D7PyspC8+bNzRocEZHJ2rcHgoLkmK4tW5SOhqxZcrK8OTsDL7ygdDRUBRiddG3dulVnPcKPPvpIpy7XvXv3dKq9ExEpgl2MZCxNba6nngLq1lU2FqoSjE66io+3N3H8PRGR5Wi6GH/+mV2MpF9eHmtzkcVxTBcR2Z9HHgECA2Vl+l9/VToaskabNwP//AP4+AAREUpHQ1WE0UmXSqXSWWdRs42IyOoU7WJkoVTSRzOAftAgoJrJJSuJysXo3zQhBIYOHQoXFxcAwN27dzFq1CjU+Lf4YNHxXkREiuvfH5g/H9iwAcjNBf7920WEq1eBTZvk4yFDlI2FqhSjk64hxX4xX3zxxRLHDB48uOIRERGZw3/+A/j7A5cuyS7Gvn2VjoishaY218MPAy1aKB0NVSFGJ11xcXGVGQcRkXk5OADPPgt89hnwxRdyfJevLxAeDjg6Kh0dKUUIQPN9xtpcZGEcSE9E9qt+fXn/669AdDTQvTsQHAzExysaFikoOVku98PaXKQAJl1EZJ/i44Hp00tuv3RJjvdi4lU1aQbQP/004OmpZCRUBTHpIiL7U1AAjB0ru5KK02wbN04eR1VHXh7w/ffyMWtzkQKYdBGR/dm1C7h40fB+IYD0dHkcVR0bNwJZWYCfH9Crl9LRUBXE4iREZH8yMsx7HNm2ggKZYM+cKZ8PHMjJFKQItnQRkf3x9TXvcWS74uPl5Inu3YHDh+W2//6XY/pIEUy6iMj+hIcDAQGyMr0+KpVcJig83LJxkWXFx8tJE8W7mq9e5WQKUgSTLiKyP46OQGysfGwo8VqwgF1M9oyTKcgKMekiIvsUGQmsWSOr0helUsnupchIZeIiy+BkCrJCTLqIyH5FRgJpacDOnbJUQIMG8stWrVY6MqpsnExBVohJFxHZN0dHoFs3WZH+zTfltoUL9Xc7kf3gZAqyQky6iKjqGDQIqFEDOHUK+P13paOhyqSZTGEIJ1OQAph0EVHV4eEBvPiifPzll8rGQpXL0RGYMUP/Ps3kCk6mIAtj0kVEVcsrr8j7deuAy5eVjYUq14UL8t7JSXd7QICcZMHJFGRhrEhPRFVL69ZAp07Anj3A4sXAO+8oHRFVhjt3gC++kI+XL5djtzIy5H14OFu4SBFMuoio6hk9WiZd33wDTJ1asiWEbN9//wv88w8QFAQ89xxQjV93pDx2LxJR1RMZCdSvL7sXN2xQOhoyt8JC4NNP5eM33mDCRVaDSRcRVT0uLsDw4fLxwoXKxkLm9/PPwB9/ALVrAzExSkdDpMWki4iqppEjAQcHYMcO4PRppaMhc5o7V96/8gpQs6aysRAVwaSLiKqmBg2Avn3l46+/VjYWMp+kJDlez8kJeO01paMh0sGki4iqrldflffLlgE5OYqGQmaiGcv14ousNk9Wh0kXEVVdPXsCDzwAZGcDK1YoHQ1VVEoKEB8vH0+YoGwsRHow6SKiqsvB4X6x1C+/5HqMtm7+fPkz7NMHaNFC6WiISmDSRURV27BhgJsbcOyYHA9EtikrC1i6VD6eOFHZWIgMsImkKzExESqVSu/twIED2uOOHTuG8PBwuLq6IjAwEHPmzCn1vFlZWejduzf8/Pzg4uKCwMBAjBkzBtnZ2dpjMjIyEB0djaZNm8LBwQHjxo3Te64FCxYgJCQEbm5uCAwMxBtvvIG7d++a5fqJqBJ5egJRUfIx12O0XQsXyir0bdsC3bsrHQ2RXjaRdHXs2BEZGRk6t+HDh6Nhw4Zo3749ACA7Oxu9evVCUFAQDh06hLlz5+K9997Dt99+a/C8Dg4O6NevHzZs2IBz585h2bJl2LZtG0aNGqU9Jjc3F15eXnj77bfRunVrvedZsWIFpkyZgnfffRenT5/GkiVLsHr1arz11lvm/SCIqHJoBtT/+CNw9aqysZDp7t4FPv9cPp448f6C1kRWxibK9Do7O8PHx0f7PD8/H+vXr8drr70G1b//uL7//nvk5eVh6dKlcHZ2RosWLZCcnIx58+ZhxIgRes/r6emJVzTjOQAEBQXh1VdfxVxNjRcAwcHBiI2NBQAs1TRdF7N371506tQJ0dHR2tdERUVh3759FbtwIrKMdu2ARx4B9u8HliyRSwOR7fjuO+Dvv2UZkP79lY6GyCCbaOkqbsOGDcjKysKwYcO025KSktClSxc4Oztrt0VERODs2bO4fv26Uee9fPky4uPj0bVrV5Pi6dixIw4dOoT9+/cDAM6fP4/Nmzfj8ccfN/ia3NxcZGdn69yISEGa1q6vvwYKCpSNhYxXdMmfceO4jiZZNZtMupYsWYKIiAgEBARot2VmZsLb21vnOM3zzMzMUs8XFRWF6tWrw9/fH+7u7li8eLFJ8URHR2PGjBno3LkznJyc0LhxY3Tr1q3U7sVZs2bBw8NDewsMDDTpPYnIzAYMAOrUAS5cADZtUjoaMtamTcDZs4CHx/2lnYislKJJ15QpUwwOkNfczpw5o/OaixcvYuvWrYgx43pa8+fPx+HDh7F+/XqkpKRg/PjxJr0+MTERH330ERYuXIjDhw8jPj4emzZtwgcffGDwNVOnToVardbe0tPTK3oZRFQRrq731+njeoy2QzMcZORIoFYtZWMhKoOiY7omTJiAoUOHlnpMo0aNdJ7HxcWhbt26eOqpp3S2+/j44MqVKzrbNM+LjgfTx8fHBz4+PggNDUWdOnUQHh6O6dOnw9fIasbTp0/HoEGDMPzf/2W1atUKt27dwogRIzBt2jQ4OJTMbV1cXODi4mLU+YnIQkaOBD75BNi6Ffj+e1nHy9cXCA8HHB2Vjo6K27cP2LVLdim+/rrS0RCVSdGky8vLC15eXkYfL4RAXFwcBg8eDKdi/fZhYWGYNm0a8vPztfsSEhIQEhICT09Po9+jsLAQgBxzZazbt2+XSKwc//0DLVhskch2NG4MPPQQcOSIXEZGIyAAiI0FIiMVC4300Izlio4G/P2VjYXICDY1pmvHjh1ITU3VtigVFR0dDWdnZ8TExODkyZNYvXo1YmNjdboK161bh9DQUO3zzZs3Iy4uDidOnEBaWho2bdqEUaNGoVOnTggODtYel5ycjOTkZOTk5ODvv/9GcnIyTp06pd3ft29ffPXVV1i1ahVSU1ORkJCA6dOno2/fvtrki4hsQHy8TLiKu3RJzorTLDFDyikoABITgQULgDVr5DYu+UO2QtiQqKgo0bFjR4P7jx49Kjp37ixcXFyEv7+/mD17ts7+uLg4UfSSd+zYIcLCwoSHh4dwdXUVTZo0EZMnTxbXr1/XeR2AEregoCDt/vz8fPHee++Jxo0bC1dXVxEYGCheffXVEucpjVqtFgCEWq02+jVEZEb37gkRECCEXEim5E2lEiIwUB5Hyli7tuTPyMVFbidSiCnf3yoh2P9lDbKzs+Hh4QG1Wg13d3elwyGqehITjatkvnMn0K1bZUdDxcXHy9ZGfV9ZKpVs9WL3LynAlO9vm+peJCKqNBkZ5j2OzKegABg7tvQFyceNY301snpMuoiIADlL0ZzHkfns2gVcvGh4vxBAero8jsiKMekiIgJkWYiAAMPr9qlUQGCgPI4si62QZCeYdBERAbIO17/rrBpMvBYsYL0uJbAVkuwEky4iIo3ISDkgW1/NJ9bpUo6mFdIQtkKSjWDSRURUVGQkkJYmZymuWAGEhcntyclKRlW1OToC772nf5+mVZKtkGQDmHQRERXn6CjLQkRF3a96/t13pQ/mpsp1+LC8d3bW3R4QwHIRZDMUXQaIiMjqhYUBXbsCv/0mE7D585WOqOpJSQG+/VY+3rJFromZkcF1McnmMOkiIirL1Kky6fr2W2DaNKBePaUjqlrefRe4dw+IiAAefVTpaIjKjd2LRERl6dULaNMGuH0b+PxzpaOpWo4dk2PrAOCjj5SNhaiCmHQREZVFpZKtXYBMum7eVDaeqmTaNFn89PnngbZtlY6GqEKYdBERGSMyEmjaFLh+/f74Iqpce/YAGzfKMVsffKB0NEQVxqSLiMgYjo7ApEny8bx5QG6usvHYOyGAKVPk45dekgkvkY1j0kVEZKwXXwT8/IDLl2UJCao8W7YAu3cDrq7AO+8oHQ2RWTDpIiIylosLMGGCfDxnDlBQoGw89qqwEHjrLfn4tddKr0ZPZEOYdBERmWLECKBOHeCPP4C1a5WOxj6tXg0cPQq4uwOTJysdDZHZMOkiIjJFzZqy9QUAZs2SY4/IfPLzgenT5eNJk4C6dZWNh8iMmHQREZnqtdeAGjXkeoxbtyodjX1ZskRWoK9fHxg7VuloiMyKSRcRkanq1pXdjIBs7SLzuH0bmDFDPp4+XbYqEtkRJl1EROUxfjzg5AT8/juwd6/S0diHzz+XayoGB99PaonsCJMuIqLyCAgABg+Wj9naVXHXrwOzZ8vHM2YAzs7KxkNUCZh0ERGV16RJcomgjRuB48eVjsb2FBQAiYnAypXAmDHAjRtAy5ZAdLTSkRFVimpKB0BEZLOaNgX69wd+/FEuxjxypOwe8/UFwsNlFXvSLz5eDpS/eFF3+5NP8nMju6USgvOdrUF2djY8PDygVqvh7u6udDhEZKwjR/QvxBwQAMTGyjUbSVd8vExW9X39qFTAmjX83MhmmPL9ze5FIqKKSE3Vv/3SJZlYxMdbNh5rV1AgW7hK+//+uHGs9k92iUkXEVF5aRIIfTRJBRMIXbt2lexSLEoIID1dHkdkZ5h0ERGVFxMI02VkmPc4IhvCpIuIqLyYQJjO19e8xxHZECZdRETlxQTCdOHhcpKBISoVEBgojyOyM0y6iIjKS5NAqFT69zOBKMnREXjjDf37NJ/jggUsG0F2iUkXEVF5OTrKshCA4cSLCYSuggJg9Wr52M1Nd19AAMtFkF1jcVQiooqIjJSJQvFCnw4OwHffMYEo7ssvgf37AXd34MQJICWFBWWpymDSRURUUZGRQL9+92czTpokE4krV5SOzLpcuAC89ZZ8/PHHsus1MFDZmIgsiEkXEZE5ODoC3brJx3fvAi+/DMydC7zyCuDqqmhoVkEIYPRo4NYtoFMnYMQIpSMisjiO6SIiMrfBg2ULTkYGsHSp0tFYhx9/lAuDOzkB334ru1+Jqhj+1hMRmZuzMzB5snw8ezaQl6dsPEq7fh14/XX5+K23gObNlY2HSCFMuoiIKsNLLwE+PrIi/XffKR2NsiZNkuPbQkOBqVOVjoZIMUy6iIgqg5sb8Oab8vFHHwH37ikbj1J++w1YvFg+XrQIcHFRNh4iBTHpIiKqLCNHAvXqAefPA6tWKR2N5d29e3/A/MiRQOfOysZDpDAmXURElaVGDWD8ePl45kygsFDZeCxt5kzg3DlZg2v2bKWjIVKcTSRdiYmJUKlUem8HDhzQHnfs2DGEh4fD1dUVgYGBmDNnTqnnzcrKQu/eveHn5wcXFxcEBgZizJgxyM7O1h4THx+Pxx57DF5eXnB3d0dYWBi2bt1a4lxffvklgoOD4erqig4dOmD//v3m+wCIyHaNHg3Urg2cOQOsXat0NJZz4sT9ROvzz+VnQFTF2UTS1bFjR2RkZOjchg8fjoYNG6J9+/YAgOzsbPTq1QtBQUE4dOgQ5s6di/feew/ffvutwfM6ODigX79+2LBhA86dO4dly5Zh27ZtGDVqlPaY33//HY899hg2b96MQ4cOoXv37ujbty+OHDmiPWb16tUYP3483n33XRw+fBitW7dGREQErl69WnkfChHZBnd3Wa0eAD78UNarsneFhbJb8d49WTSWVfmJJGGD8vLyhJeXl5gxY4Z228KFC4Wnp6fIzc3Vbps8ebIICQkx6dyxsbEiICCg1GOaN28u3n//fe3zRx55RIwePVr7vKCgQPj5+YlZs2YZ/b5qtVoAEGq12qR4icgGZGUJUbOmEIAQ69crHU3luHdPiJ07hVixQojXX5fXWquWEOnpSkdGVKlM+f62iZau4jZs2ICsrCwMGzZMuy0pKQldunSBs7OzdltERATOnj2L69evG3Xey5cvIz4+Hl27djV4TGFhIW7evIk6deoAAPLy8nDo0CH07NlTe4yDgwN69uyJpKQkg+fJzc1Fdna2zo2I7FSdOsCYMfKxPbZ2xccDwcFA9+5AdDTw2Wdy+4ABchFrIgJgI92LxS1ZsgQREREIKPKPOTMzE97e3jrHaZ5nZmaWer6oqChUr14d/v7+cHd3x2LN9GY9PvnkE+Tk5OD5558HAPzzzz8oKCjQ+96lve+sWbPg4eGhvQVy/TEi+/bGG7KMxIEDwK+/Kh2N+cTHA/376y72rbFkidxPRAAUTrqmTJlicIC85nbmzBmd11y8eBFbt25FTEyM2eKYP38+Dh8+jPXr1yMlJQXjNbONilmxYgXef/99/PDDD6hfv36F3nPq1KlQq9XaW3p6eoXOR0RWrn59WTYBAD74wD5auwoK5Hi10q5l3Dh5HBEpu+D1hAkTMHTo0FKPadSokc7zuLg41K1bF0899ZTOdh8fH1y5ckVnm+a5j49Pqe/h4+MDHx8fhIaGok6dOggPD8f06dPh6+urPWbVqlUYPnw4fvzxR52uxHr16sHR0VHve5f2vi4uLnBhkUCiquXNN4GFC4E9e2TRUM0C2bZq1y79LVwaQsiK/Lt22f61EpmBoi1dXl5eCA0NLfVWdIyWEAJxcXEYPHgwnJycdM4VFhaG33//Hfn5+dptCQkJCAkJgaenp9ExFf5bRyc3N1e7beXKlRg2bBhWrlyJJ554Qud4Z2dntGvXDtu3b9c5x/bt2xEWFmb0+xJRFeDnB2ha6T/8UNlYzCEjw7zHEdk5mxrTtWPHDqSmpmL48OEl9kVHR8PZ2RkxMTE4efIkVq9ejdjYWJ2uwnXr1iE0NFT7fPPmzYiLi8OJEyeQlpaGTZs2YdSoUejUqROCg4MByC7FwYMH49NPP0WHDh2QmZmJzMxMqNVq7XnGjx+PRYsWYfny5Th9+jReeeUV3Lp1S2egPxERALkQdrVqwPbtQCmTbWxCkd4AsxxHZO8qfzKl+URFRYmOHTsa3H/06FHRuXNn4eLiIvz9/cXs2bN19sfFxYmil7xjxw4RFhYmPDw8hKurq2jSpImYPHmyuH79uvaYrl27CgAlbkOGDNE59+effy4aNGggnJ2dxSOPPCL+97//mXRtLBlBVIW89JIsqdCnz/0yCzt3yrILtiQ/X4gaNeS16LupVEIEBtredRGZwJTvb5UQ9jCa0/ZlZ2fDw8MDarUa7u7uSodDRJXpzz+Bpk1LDkAPCABiY22nmOiCBXJWpj4qlbxfs8Z2roeoHEz5/rap7kUiIrtw7Jj+GX+XLsnyC7ZQZmHzZmDCBPl4yJCS9bgCAphwERXDli4rwZYuoiqioEAWEjU060+lkglLairg6GjR0Ix24gTQsSNw86acGLBokVz6Z9cuOWje1xcID7fe+InMyJTvb0VLRhARVTm2Xmbh6lXgySdlwtW1qyyBoVLJBMsa4yWyIuxeJCKyJFsus3D3LvDMM8BffwEPPACsXQsUKetDRKVj0kVEZEm2WmZBCODll4G9e4HatYGNG4G6dZWOisimMOkiIrKk8HA5Zkszu0+fwEB5nDWZNQv4v/+T3Yg//giEhCgdEZHN4ZguIiJLcnSUZSH695eJl765TNOnKzsIvaBAd1D81avAtGly3xdfAEWWQiMi47Gli4jI0iIjZTkFf3/d7Zrlzb75Brh92/JxAbJcRXAw0L07EB0t7wcMkPtefx0YNUqZuIjsAJMuIiIlREYCaWnAzp3AihXy/tQpOU7q0CFg2DD9rWCVKT5etsAZml3ZubNl4yGyM6zTZSVYp4uIAAC//y677/LzgfffB955xzLvaw/1w4gUwIr0RES2qksX4Kuv5ON335XdkJZgSv0wIioXJl1ERNYmJgYYN04+HjwYOHKk8t/TluuHEdkIJl1ERNZo7lwgIgK4cwd46ikgM7Ny3y8lxbjjrK1+GJENYdJFRGSNqlUDVq2S9bAuXgSeflpWhDe3CxeAfv1kmYrSqFTWWT+MyIYw6SIisla1awM//wx4egL79smK8PfuAYmJwMqV8r6goPRzFBToP/7ePWD+fKB5c2DDBpnkRUbK5Kp44VbN8wULOIieqAI4e9FKcPYiERm0fbvsaiwoADw8ALX6/r6AAFlsNTKy5Ovi44GxY3UHyAcEAK+9JlvRNGPFOnWStcFatND/msBAmXDpew+iKs6U728mXVaCSRcRlWrECGDRopLbNa1Qa9boJkWamlul/YmvXRuYM0cO3Hco0vFRvCJ9eDhbuIgMMOX7m8sAERFZu4ICYMsW/fs0SdXLL8sq9jVqyMr2o0aVnnBVrw6cPAn4+ZXc5+gIdOtW4bCJSBeTLiIia1dWDS0AuHYNGDTI+HPevg2cO6c/6SKiSsGki4jI2hlbG6tFCznm6/JlucSQuc5LRGbB2YtERNbO2NpYX3wB7NkDxMWZ97xEZBZMuoiIrF14uJx1WLyUg0bxGlqmHk9EFsGki4jI2jk6yrIQgHE1tEw9nogsgkkXEZEtiIyUZSH8/XW3BwSULBdRnuOJqNKxTpeVYJ0uIjKKqTW0WHOLqFKxThcRkb0ytYYWa24RWQ12LxIRERFZAJMuIiIiIgtg0kVERERkAUy6iIiIiCyASRcRERGRBTDpIiIiIrIAJl1EREREFsCki4iIiMgCmHQRERERWQAr0lsJzWpM2dnZCkdCRERExtJ8bxuzqiKTLitx8+ZNAEBgYKDCkRAREZGpbt68CQ8Pj1KP4YLXVqKwsBCXL19GrVq1oFKpDB6XnZ2NwMBApKenV7mFsXntVe/aq+p1A1X32qvqdQO8dlu9diEEbt68CT8/Pzg4lD5qiy1dVsLBwQEBAQFGH+/u7m5zv5jmwmuvetdeVa8bqLrXXlWvG+C12+K1l9XCpcGB9EREREQWwKSLiIiIyAKYdNkYFxcXvPvuu3BxcVE6FIvjtVe9a6+q1w1U3WuvqtcN8NqrwrVzID0RERGRBbCli4iIiMgCmHQRERERWQCTLiIiIiILYNJFREREZAFMuqzUe++9B5VKpXMLDQ3V7r979y5Gjx6NunXrombNmnj22Wdx5coVBSMun99//x19+/aFn58fVCoVfvrpJ539Qgi888478PX1hZubG3r27Ik//vhD55hr165h4MCBcHd3R+3atRETE4OcnBwLXkX5lHXtQ4cOLfE70Lt3b51jbPHaZ82ahYcffhi1atVC/fr18fTTT+Ps2bM6xxjz+33hwgU88cQTqF69OurXr48333wT9+7ds+SlmMyYa+/WrVuJn/uoUaN0jrG1a//qq6/w4IMPagtfhoWFYcuWLdr99vrzBsq+dnv8eesze/ZsqFQqjBs3TrvNnn/uhjDpsmItWrRARkaG9rZ7927tvjfeeAM///wzfvzxR/z222+4fPkyIiMjFYy2fG7duoXWrVvjyy+/1Lt/zpw5+Oyzz/D1119j3759qFGjBiIiInD37l3tMQMHDsTJkyeRkJCAjRs34vfff8eIESMsdQnlVta1A0Dv3r11fgdWrlyps98Wr/23337D6NGj8b///Q8JCQnIz89Hr169cOvWLe0xZf1+FxQU4IknnkBeXh727t2L5cuXY9myZXjnnXeUuCSjGXPtAPDyyy/r/NznzJmj3WeL1x4QEIDZs2fj0KFDOHjwIB599FH069cPJ0+eBGC/P2+g7GsH7O/nXdyBAwfwzTff4MEHH9TZbs8/d4MEWaV3331XtG7dWu++GzduCCcnJ/Hjjz9qt50+fVoAEElJSRaK0PwAiHXr1mmfFxYWCh8fHzF37lztths3bggXFxexcuVKIYQQp06dEgDEgQMHtMds2bJFqFQqcenSJYvFXlHFr10IIYYMGSL69etn8DX2cu1Xr14VAMRvv/0mhDDu93vz5s3CwcFBZGZmao/56quvhLu7u8jNzbXsBVRA8WsXQoiuXbuKsWPHGnyNvVy7p6enWLx4cZX6eWtorl0I+/9537x5UzRp0kQkJCToXGtV/LkLIQRbuqzYH3/8AT8/PzRq1AgDBw7EhQsXAACHDh1Cfn4+evbsqT02NDQUDRo0QFJSklLhml1qaioyMzN1rtPDwwMdOnTQXmdSUhJq166N9u3ba4/p2bMnHBwcsG/fPovHbG6JiYmoX78+QkJC8MorryArK0u7z16uXa1WAwDq1KkDwLjf76SkJLRq1Qre3t7aYyIiIpCdna3TgmDtil+7xvfff4969eqhZcuWmDp1Km7fvq3dZ+vXXlBQgFWrVuHWrVsICwurUj/v4teuYc8/79GjR+OJJ57Q+fkCVevfeVFc8NpKdejQAcuWLUNISAgyMjLw/vvvIzw8HCdOnEBmZiacnZ1Ru3Ztndd4e3sjMzNTmYArgeZaiv6D0zzX7MvMzET9+vV19lerVg116tSx+c+id+/eiIyMRMOGDZGSkoK33noLffr0QVJSEhwdHe3i2gsLCzFu3Dh06tQJLVu2BACjfr8zMzP1/l5o9tkCfdcOANHR0QgKCoKfnx+OHTuGyZMn4+zZs4iPjwdgu9d+/PhxhIWF4e7du6hZsybWrVuH5s2bIzk52e5/3oauHbDfnzcArFq1CocPH8aBAwdK7Ksq/86LY9Jlpfr06aN9/OCDD6JDhw4ICgrCDz/8ADc3NwUjI0t54YUXtI9btWqFBx98EI0bN0ZiYiJ69OihYGTmM3r0aJw4cUJnvGJVYejai47Ja9WqFXx9fdGjRw+kpKSgcePGlg7TbEJCQpCcnAy1Wo01a9ZgyJAh+O2335QOyyIMXXvz5s3t9uednp6OsWPHIiEhAa6urkqHYzXYvWgjateujaZNm+LPP/+Ej48P8vLycOPGDZ1jrly5Ah8fH2UCrASaayk+m6Xodfr4+ODq1as6++/du4dr167Z1WcBAI0aNUK9evXw559/ArD9ax8zZgw2btyInTt3IiAgQLvdmN9vHx8fvb8Xmn3WztC169OhQwcA0Pm52+K1Ozs744EHHkC7du0wa9YstG7dGrGxsVXi523o2vWxl5/3oUOHcPXqVbRt2xbVqlVDtWrV8Ntvv+Gzzz5DtWrV4O3tbfc/d32YdNmInJwcpKSkwNfXF+3atYOTkxO2b9+u3X/27FlcuHBBZ5yArWvYsCF8fHx0rjM7Oxv79u3TXmdYWBhu3LiBQ4cOaY/ZsWMHCgsLtX+87MXFixeRlZUFX19fALZ77UIIjBkzBuvWrcOOHTvQsGFDnf3G/H6HhYXh+PHjOklnQkIC3N3dtd021qisa9cnOTkZAHR+7rZ47cUVFhYiNzfXrn/ehmiuXR97+Xn36NEDx48fR3JysvbWvn17DBw4UPu4qv3cAXD2orWaMGGCSExMFKmpqWLPnj2iZ8+eol69euLq1atCCCFGjRolGjRoIHbs2CEOHjwowsLCRFhYmMJRm+7mzZviyJEj4siRIwKAmDdvnjhy5Ij466+/hBBCzJ49W9SuXVusX79eHDt2TPTr1080bNhQ3LlzR3uO3r17izZt2oh9+/aJ3bt3iyZNmoioqCilLslopV37zZs3xcSJE0VSUpJITU0V27ZtE23bthVNmjQRd+/e1Z7DFq/9lVdeER4eHiIxMVFkZGRob7dv39YeU9bv971790TLli1Fr169RHJysvjll1+El5eXmDp1qhKXZLSyrv3PP/8UM2bMEAcPHhSpqali/fr1olGjRqJLly7ac9jitU+ZMkX89ttvIjU1VRw7dkxMmTJFqFQq8euvvwoh7PfnLUTp126vP29Dis/UtOefuyFMuqzUgAEDhK+vr3B2dhb+/v5iwIAB4s8//9Tuv3Pnjnj11VeFp6enqF69unjmmWdERkaGghGXz86dOwWAErchQ4YIIWTZiOnTpwtvb2/h4uIievToIc6ePatzjqysLBEVFSVq1qwp3N3dxbBhw8TNmzcVuBrTlHbtt2/fFr169RJeXl7CyclJBAUFiZdfflln6rQQtnnt+q4ZgIiLi9MeY8zvd1pamujTp49wc3MT9erVExMmTBD5+fkWvhrTlHXtFy5cEF26dBF16tQRLi4u4oEHHhBvvvmmUKvVOuextWt/6aWXRFBQkHB2dhZeXl6iR48e2oRLCPv9eQtR+rXb68/bkOJJlz3/3A1RCSGE5drViIiIiKomjukiIiIisgAmXUREREQWwKSLiIiIyAKYdBERERFZAJMuIiIiIgtg0kVERERkAUy6iIiIiCyASRcRERGRBTDpIqIKUalU+OmnnwAAaWlpUKlU2vXjzCk4OBgLFiyo8HmGDh2Kp59+usLnqShzflaDBg3CRx99VOox5vr8lJaXl4fg4GAcPHhQ6VCITMaki4hKGDp0KFQqFVQqFZycnODt7Y3HHnsMS5cuRWFhoc6xGRkZ6NOnj0KR0tGjR7F582a8/vrrSodiEc7Ozpg4cSImT56sdChEJmPSRUR69e7dGxkZGUhLS8OWLVvQvXt3jB07Fk8++STu3bunPc7HxwcuLi4KRirl5+crHYIiPv/8czz33HOoWbOm0qEgLy/PIu8zcOBA7N69GydPnrTI+xGZC5MuItLLxcUFPj4+8Pf3R9u2bfHWW29h/fr12LJlC5YtW6Y9rmj3osb58+fRvXt3VK9eHa1bt0ZSUpLO/t27dyM8PBxubm4IDAzE66+/jlu3bmn3X716FX379oWbmxsaNmyI77//vkR8KpUKX331FZ566inUqFEDM2fOREFBAWJiYtCwYUO4ubkhJCQEsbGxJl13VlYWoqKi4O/vj+rVq6NVq1ZYuXKlzjHdunXD66+/jkmTJqFOnTrw8fHBe++9p3PMmTNn0LlzZ7i6uqJ58+bYtm2b3s+qqBMnTqBPnz6oWbMmvL29MWjQIPzzzz8Gjy8oKMCaNWvQt29fne3GfH43btzA8OHD4eXlBXd3dzz66KM4evSozjEffvgh6tevj1q1amH48OGYMmUKHnroIe1+TVftzJkz4efnh5CQEABAeno6nn/+edSuXRt16tRBv379kJaWpnPuxYsXo1mzZnB1dUVoaCgWLlyo3ZeXl4cxY8bA19cXrq6uCAoKwqxZs7T7PT090alTJ6xatcrgZ0NkjZh0EZHRHn30UbRu3Rrx8fGlHjdt2jRMnDgRycnJaNq0KaKiorStYykpKejduzeeffZZHDt2DKtXr8bu3bsxZswY7euHDh2K9PR07Ny5E2vWrMHChQtx9erVEu/z3nvv4ZlnnsHx48fx0ksvobCwEAEBAfjxxx9x6tQpvPPOO3jrrbfwww8/GH2Nd+/eRbt27bBp0yacOHECI0aMwKBBg7B//36d45YvX44aNWpg3759mDNnDmbMmIGEhAQAMhl6+umnUb16dezbtw/ffvstpk2bVur73rhxA48++ijatGmDgwcP4pdffsGVK1fw/PPPG3zNsWPHoFar0b59e53txnx+zz33HK5evYotW7bg0KFDaNu2LXr06IFr164BAL7//nvMnDkTH3/8MQ4dOoQGDRrgq6++KhHD9u3bcfbsWSQkJGDjxo3Iz89HREQEatWqhV27dmHPnj2oWbMmevfurW0J+/777/HOO+9g5syZOH36ND766CNMnz4dy5cvBwB89tln2LBhA3744QecPXsW33//PYKDg3Xe95FHHsGuXbtK/UyJrI4gIipmyJAhol+/fnr3DRgwQDRr1kz7HIBYt26dEEKI1NRUAUAsXrxYu//kyZMCgDh9+rQQQoiYmBgxYsQInXPu2rVLODg4iDt37oizZ88KAGL//v3a/adPnxYAxPz583Xed9y4cWVey+jRo8Wzzz5r1LUZ8sQTT4gJEyZon3ft2lV07txZ55iHH35YTJ48WQghxJYtW0S1atVERkaGdn9CQoLez+rIkSNCCCE++OAD0atXL51zpqenCwDi7NmzeuNat26dcHR0FIWFhdptxnx+u3btEu7u7uLu3bs652vcuLH45ptvhBBCdOjQQYwePVpnf6dOnUTr1q21z4cMGSK8vb1Fbm6udtt3330nQkJCdGLKzc0Vbm5uYuvWrdr3WbFihc65P/jgAxEWFiaEEOK1114Tjz76qM45iouNjRXBwcEG9xNZo2oK5XpEZKOEEFCpVKUe8+CDD2of+/r6ApBdXqGhoTh69CiOHTum0+UlhEBhYSFSU1Nx7tw5VKtWDe3atdPuDw0NRe3atUu8T/EWHgD48ssvsXTpUly4cAF37txBXl6eTpdYWQoKCvDRRx/hhx9+wKVLl5CXl4fc3FxUr17d4DVqrlPTmnT27FkEBgbCx8dHu/+RRx4p9X2PHj2KnTt36h2blZKSgqZNm5bYfufOHbi4uOj8PE6fPl3m53f06FHk5OSgbt26Jc6XkpKivYZXX31VZ/8jjzyCHTt26Gxr1aoVnJ2ddc79559/olatWjrH3b17FykpKbh16xZSUlIQExODl19+Wbv/3r178PDwACBb6h577DGEhISgd+/eePLJJ9GrVy+d87m5ueH27dslPhMia8aki4hMcvr0aTRs2LDUY5ycnLSPNQmBZtZjTk4ORo4cqXe2XYMGDXDu3DmjY6lRo4bO81WrVmHixIn49NNPERYWhlq1amHu3LnYt2+f0eecO3cuYmNjsWDBArRq1Qo1atTAuHHjSgwSL3qNgLzO4jM7TZGTk4O+ffvi448/LrFPk7gWV69ePdy+fRt5eXk6iY8x7+Xr64vExMQS+/Qlt6Up/jPIyclBu3bt9I4j8/LyQk5ODgBg0aJF6NChg85+R0dHAEDbtm2RmpqKLVu2YNu2bXj++efRs2dPrFmzRnvstWvX4OXlZVKsREpj0kVERtuxYweOHz+ON954o9znaNu2LU6dOoUHHnhA7/7Q0FDcu3cPhw4dwsMPPwxAtrrcuHGjzHPv2bMHHTt21Gmh0bTcGGvPnj3o168fXnzxRQAyWTx37hyaN29u9DlCQkKQnp6OK1euwNvbGwBw4MCBUl/Ttm1brF27FsHBwahWzbg/zZoWvFOnTmkfG/P5tW3bFpmZmahWrVqJsVJFr+HAgQMYPHiwdltZ16A59+rVq1G/fn24u7uX2O/h4QE/Pz+cP38eAwcONHged3d3DBgwAAMGDED//v3Ru3dvXLt2DXXq1AEgJx20adOmzHiIrAkH0hORXrm5ucjMzMSlS5dw+PBhfPTRR+jXrx+efPJJnS9iU02ePBl79+7FmDFjkJycjD/++APr16/XDqTXdCmNHDkS+/btw6FDhzB8+HC4ubmVee4mTZrg4MGD2Lp1K86dO4fp06cblSgUP0dCQgL27t2L06dPY+TIkbhy5YpJ53jsscfQuHFjDBkyBMeOHcOePXvw9ttvA4DBrtnRo0fj2rVriIqKwoEDB5CSkoKtW7di2LBhKCgo0PsaLy8vtG3bFrt379ZuM+bz69mzJ8LCwvD000/j119/RVpaGvbu3Ytp06Zpi46+9tprWLJkCZYvX44//vgDH374IY4dO1Zm1/LAgQNRr1499OvXD7t27UJqaioSExPx+uuv4+LFiwCA999/H7NmzcJnn32Gc+fO4fjx44iLi8O8efMAAPPmzcPKlStx5swZnDt3Dj/++CN8fHx0WuF27dpVosuRyNox6SIivX755Rf4+voiODgYvXv3xs6dO/HZZ59h/fr12m6g8njwwQfx22+/4dy5cwgPD0ebNm3wzjvvwM/PT3tMXFwc/Pz80LVrV0RGRmLEiBGoX79+meceOXIkIiMjMWDAAHTo0AFZWVklxiWV5e2330bbtm0RERGBbt26wcfHx+QK9o6Ojvjpp5+Qk5ODhx9+GMOHD9fOXnR1ddX7Gj8/P+zZswcFBQXo1asXWrVqhXHjxqF27dpwcDD8p3r48OEluvLK+vxUKhU2b96MLl26YNiwYWjatCleeOEF/PXXX9qWuYEDB2Lq1KmYOHGitrtv6NChBuPXqF69On7//Xc0aNAAkZGRaNasGWJiYnD37l1ty9fw4cOxePFixMXFoVWrVujatSuWLVum7bauVasW5syZg/bt2+Phhx9GWloaNm/erP0ckpKSoFar0b9//1JjIbI2KiGEUDoIIiJ7t2fPHnTu3Bl//vknGjdubLbz3rlzByEhIVi9ejXCwsLMdl59HnvsMfj4+OC7776r1Pcpy4ABA9C6dWu89dZbisZBZCqO6SIiqgTr1q1DzZo10aRJE/z5558YO3YsOnXqZNaEC5Cz+P773/+WWkS1PG7fvo2vv/4aERERcHR0xMqVK7Ft2zZtLTKl5OXloVWrVhUaV0ikFLZ0ERFVgv/+97/48MMPceHCBdSrVw89e/bEp59+WqJMg7W6c+cO+vbtiyNHjuDu3bsICQnB22+/jcjISKVDI7JZTLqIiIiILIAD6YmIiIgsgEkXERERkQUw6SIiIiKyACZdRERERBbApIuIiIjIAph0EREREVkAky4iIiIiC2DSRURERGQB/w8FJhXQFOWjlAAAAABJRU5ErkJggg==\n"
          },
          "metadata": {}
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "# Ler o arquivo XYZ\n",
        "with open('xtbscan.log', 'r') as file:\n",
        "    xyz_data = file.read()"
      ],
      "metadata": {
        "id": "7S26tkWv9KBi"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "# Criar a visualiza√ß√£o\n",
        "view = py3Dmol.view(width=800, height=600)\n",
        "\n",
        "# Adicionar a trajet√≥ria do XYZ\n",
        "view.addModelsAsFrames(xyz_data, 'xyz')\n",
        "\n",
        "# Ajustar o estilo da visualiza√ß√£o\n",
        "view.setStyle({'stick': {'colorscheme':'greenCarbon'}})\n",
        "\n",
        "# Ajustar o zoom para que todas as mol√©culas estejam vis√≠veis\n",
        "view.zoomTo()\n",
        "\n",
        "# Anima√ß√£o\n",
        "view.animate({'loop': 'backAndForth'})  # Anima√ß√£o de ida e volta\n",
        "\n",
        "# Mostrar a visualiza√ß√£o\n",
        "view.show()"
      ],
      "metadata": {
        "id": "Yb5fmqZh9RFF",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 637
        },
        "outputId": "04f73ace-c3b9-4d0e-813e-ba80dbce2855"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_17200165783103456\"  style=\"position: relative; width: 800px; height: 600px;\">\n        <p id=\"3dmolwarning_17200165783103456\" style=\"background-color:#ffcccc;color:black\">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    //this is to ignore the existence of requirejs amd\n    var savedexports, savedmodule;\n    if (typeof exports !== 'undefined') savedexports = exports;\n    else exports = {}\n    if (typeof module !== 'undefined') savedmodule = module;\n    else module = {}\n\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n        exports = savedexports;\n        module = savedmodule;\n        resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.1.0/3Dmol-min.js');\n}\n\nvar viewer_17200165783103456 = null;\nvar warn = document.getElementById(\"3dmolwarning_17200165783103456\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_17200165783103456 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_17200165783103456\"),{backgroundColor:\"white\"});\nviewer_17200165783103456.zoomTo();\n\tviewer_17200165783103456.addModelsAsFrames(\"43\\n energy: -70.311460971468 xtb: 6.7.0 (08769fc)\\nC           -5.38940635858081       -0.09281392942543       -1.62157543131579\\nC           -4.23943344221169       -0.49267316071980       -0.75148796161917\\nC           -3.88587706124477       -1.82525993340304       -0.58335295116622\\nN           -2.88863105207705       -2.22874358683721        0.17540199093572\\nC           -2.14358453923092       -1.32802376317536        0.80479018533147\\nC           -2.36934543736326        0.04679680397360        0.69717841326816\\nC           -3.45803698898169        0.45218497188428       -0.08106788124485\\nO           -3.73721482003351        1.80135030680773       -0.21722423061408\\nC           -4.71800844313505        2.27377940313145        0.68133015047824\\nC           -1.48511903781236        1.05954487643970        1.35404555483722\\nC           -1.04661320303918       -1.92036312466057        1.61977086802876\\nS            0.04347547479708       -2.89239201185228        0.45363764983982\\nO            0.84730353406379       -3.80237799296632        1.28407342981580\\nC            1.21787865776442       -1.55054516607273        0.17254059209574\\nN            0.99205971766160       -0.33521598608057       -0.22836249395691\\nC            2.22875675857793        0.25927183742353       -0.29230304529166\\nC            3.22232490937665       -0.67368455099243        0.08857224226019\\nN            2.53855262295791       -1.82382099442775        0.37168991090722\\nC            4.57256021750363       -0.35569639046986        0.12720993768171\\nC            4.91624953469089        0.93831129236282       -0.22914864605678\\nC            3.93057416289689        1.87341279946789       -0.60563874973414\\nC            2.59575661800247        1.55655740043045       -0.64219166018973\\nO            6.19019036497287        1.41944478358739       -0.24933071249286\\nC            7.24154609871348        0.55385933118568        0.10060147344521\\nH           -6.32325494414460       -0.11985370144267       -1.05843586878840\\nH           -5.48385298530286       -0.77926408089639       -2.45943289815050\\nH           -5.24159519167427        0.91654911048690       -1.99848650131287\\nH           -4.43993799909669       -2.60936497685819       -1.08283980519977\\nH           -5.67028349282925        1.74916352957019        0.55414797539846\\nH           -4.85342980050054        3.32988810164878        0.45427494173523\\nH           -4.39164589179887        2.16262272233856        1.72156600719087\\nH           -0.46865873914399        0.93523505499834        0.98335265463404\\nH           -1.48670447957558        0.94204752104446        2.43731791587357\\nH           -1.82251044204056        2.06166399109598        1.10400831324591\\nH           -0.45095784668333       -1.19417634635622        2.16141679405167\\nH           -1.43666573200526       -2.68209456385283        2.29425254115607\\nH            2.87753038946151       -2.70807761132856        0.71725238458564\\nH            5.30015930092154       -1.09430036245959        0.42167878884873\\nH            4.27117645577958        2.86187828695743       -0.86823207089534\\nH            1.85089813211726        2.27832889717705       -0.93343456447739\\nH            7.29540174771452       -0.30647818646246       -0.57428743483670\\nH            7.14572861261376        0.20103168524934        1.13266685186258\\nH            8.15264561791576        1.14229671345447        0.00558233981767\\n43\\n energy: -70.311270077368 xtb: 6.7.0 (08769fc)\\nC           -5.43410873190899       -0.08702358141601       -1.58437703476231\\nC           -4.28860730883252       -0.50277500769231       -0.71516308794683\\nC           -3.98887945727715       -1.84309476962500       -0.50730120474140\\nN           -2.98718432113743       -2.26340264356177        0.23556792175190\\nC           -2.18102965865802       -1.37499128948421        0.80553553780693\\nC           -2.37013495364893        0.00417260889102        0.68086768838767\\nC           -3.46165602526407        0.42951423612903       -0.08337399931579\\nO           -3.69556694270788        1.78433176984142       -0.24298454302527\\nC           -4.66951010694749        2.30658459162371        0.63524249842599\\nC           -1.45490349833665        1.00239158679690        1.31664997801181\\nC           -1.07445446673207       -1.98734184971476        1.59212243476333\\nS            0.06014162762183       -2.87147842827108        0.39296530259801\\nO            0.86467966385549       -3.80614764213589        1.19475590345517\\nC            1.22664013249240       -1.51285692687922        0.17436907667667\\nN            1.00929760240617       -0.30208119610581       -0.24527986706808\\nC            2.24673355360427        0.29136652486570       -0.29301168302703\\nC            3.23315725153130       -0.63820379151108        0.11363672440469\\nN            2.54365973356392       -1.78381129184138        0.40022826450129\\nC            4.58383653384790       -0.32303882026558        0.16178359813709\\nC            4.93481871086773        0.96393642132380       -0.21250960810705\\nC            3.95545775941049        1.89680330293105       -0.61079939057061\\nC            2.62027965276598        1.58327451138992       -0.65560627875951\\nO            6.21058552765457        1.43951154904701       -0.23362017839685\\nC            7.25823962977840        0.57097045455960        0.12066298513141\\nH           -6.37165279578263       -0.12477377096466       -1.02801518814889\\nH           -5.52299936093212       -0.76038928001698       -2.43396871509531\\nH           -5.28308331411360        0.92700002367417       -1.94689036270703\\nH           -4.59136853762707       -2.61876818302279       -0.96178611460359\\nH           -5.63946345143592        1.81624216736042        0.50506053382924\\nH           -4.76182232552671        3.36325662178844        0.39004890676412\\nH           -4.35961932741729        2.20036947132236        1.68117923747195\\nH           -0.44994938736526        0.86871980731376        0.91919169024107\\nH           -1.43076328480480        0.87785456863937        2.39900110689201\\nH           -1.78582927506820        2.00990409591121        1.08200626765967\\nH           -0.49937567769389       -1.27927967959478        2.17893069584918\\nH           -1.45081924184138       -2.79691479796925        2.21603626639230\\nH            2.87569628941370       -2.66913738569708        0.75043582819805\\nH            5.30658592815905       -1.05900957691242        0.47414636890242\\nH            4.30197343704051        2.88021842928352       -0.88564622136256\\nH            1.88000613301374        2.30253945130442       -0.96475804063315\\nH            7.30225850332527       -0.29755424624423       -0.54465308835922\\nH            7.16621075121883        0.23082181624179        1.15739448336776\\nH            8.17252402948441        1.15228914866306        0.01345230699318\\n43\\n energy: -70.311352057441 xtb: 6.7.0 (08769fc)\\nC           -6.04450091456565       -0.39751844546097       -0.97625391837603\\nC           -4.69851798016457       -0.68815923167710       -0.39041470591720\\nC           -4.31276976551304       -1.98201925912584       -0.06324993412507\\nN           -3.14060535478154       -2.28987243877335        0.45000555752490\\nC           -2.24623870906246       -1.32988489830605        0.65404737114400\\nC           -2.50236981832489        0.01438102595351        0.36431484476797\\nC           -3.76525183047146        0.32347261725013       -0.15080665575803\\nO           -4.07434788066474        1.63616890597985       -0.46449908784403\\nC           -4.75358280773763        2.32622753005721        0.56247176534496\\nC           -1.49068720100326        1.09249922821301        0.58952813180274\\nC           -0.94869295013076       -1.80689899590362        1.20659540989743\\nS            0.07708953103620       -2.55260966950084       -0.18741292840776\\nO            0.80437068980302       -3.68133956828398        0.41015830088417\\nC            1.36095540432256       -1.28314301623971       -0.18800286441112\\nN            1.26552634357326       -0.02662197977722       -0.50887794724289\\nC            2.53671725601862        0.47620566652505       -0.38337665831602\\nC            3.41423150198981       -0.55637183770841        0.02431197601928\\nN            2.62690557941233       -1.66941359755898        0.13255672222681\\nC            4.76825355035023       -0.34441785884124        0.24271504957484\\nC            5.23564095218888        0.94400225793148        0.03971000530007\\nC            4.36615824484068        1.97591955631130       -0.36954849302154\\nC            3.02738097036884        1.76430184694225       -0.58281028870918\\nO            6.52993616947657        1.33005401395936        0.20984779956325\\nC            7.46710937432642        0.37280461208060        0.63773225511699\\nH           -6.79019271330409       -0.29207054883622       -0.18706556075037\\nH           -6.35703119732321       -1.21242265224560       -1.62504513313382\\nH           -6.01266067094641        0.52625492988049       -1.54925568185869\\nH           -4.98761461013939       -2.81366025732713       -0.21890198972545\\nH           -5.71206942856201        1.85412250508826        0.80176041136484\\nH           -4.92931732043151        3.33235518031644        0.18561364109354\\nH           -4.14927592706945        2.37856055259082        1.47499989492315\\nH           -0.55848217328756        0.85338844674364        0.07762695563062\\nH           -1.28095793297322        1.20611141861989        1.65379481987256\\nH           -1.86627775376647        2.03583495813240        0.20337060176265\\nH           -0.37407477969259       -1.02704514027969        1.69487056627815\\nH           -1.11179593647405       -2.65541457172762        1.86859776136917\\nH            2.86438888808993       -2.61076586953110        0.40611274941756\\nH            5.40610126408335       -1.15597480159415        0.55262487597528\\nH            4.80071945861292        2.95279869869596       -0.50941574867065\\nH            2.37127295448936        2.56000266488797       -0.89452230938350\\nH            7.57076167702222       -0.44109918093763       -0.08728725821961\\nH            7.19958173610345       -0.04216183013982        1.61505654012079\\nH            8.41421511029475        0.90341803359569        0.71985015688573\\n43\\n energy: -70.311711233735 xtb: 6.7.0 (08769fc)\\nC           -6.32779721179640       -0.76282528451670       -0.57727203609260\\nC           -4.90074070387492       -0.87613413383645       -0.14090381100891\\nC           -4.38948412105595       -2.06350319714119        0.36800491469219\\nN           -3.14087938663101       -2.21886584878554        0.75622672038458\\nC           -2.29903309566097       -1.19888028047297        0.65271000779714\\nC           -2.68010203142603        0.05118804951632        0.15273966262265\\nC           -4.01572263147050        0.20170663263701       -0.23056660412375\\nO           -4.45131194942161        1.41134612927045       -0.74483125678933\\nC           -5.00729355276862        2.27598387646908        0.22256345031742\\nC           -1.72036969766501        1.19340223289483        0.03946944858312\\nC           -0.89451265880269       -1.48608366513162        1.05051515996719\\nS            0.04985903273979       -2.03093030136679       -0.48844412736863\\nO            0.64590040945431       -3.33496051008564       -0.17500227213649\\nC            1.45258618433086       -0.92499537559928       -0.23334893679520\\nN            1.46065442877340        0.37565152971120       -0.24556970098128\\nC            2.77644454144517        0.72800861648678       -0.07153267258223\\nC            3.56754784506159       -0.44029276467298        0.03964895060429\\nN            2.68576822361056       -1.47968508322104       -0.07486467882461\\nC            4.94215644431072       -0.39961939312611        0.22527802523851\\nC            5.52237075318810        0.85659495266223        0.29511379717588\\nC            4.73981491974716        2.02424171265305        0.18377610214582\\nC            3.38022409752688        1.98093174015962        0.00356645239363\\nO            6.85329513763032        1.08216163582111        0.47075666571353\\nC            7.71296895320068       -0.02414583277132        0.59500131077068\\nH           -6.97632701891489       -0.58389001219920        0.28136545890404\\nH           -6.65046256696939       -1.68593910447168       -1.05357008032480\\nH           -6.44180753726067        0.06094543751220       -1.27798864722014\\nH           -5.02159839666368       -2.93621647278882        0.46826779867106\\nH           -5.88287256623295        1.82968640068018        0.70587066200602\\nH           -5.30753713350492        3.17530195697392       -0.31237525326904\\nH           -4.27373364045698        2.53717509637772        0.99307872090875\\nH           -0.77460531267638        0.88434053815252       -0.40102382498506\\nH           -1.50517854551933        1.60360756873993        1.02740235513318\\nH           -2.15617401185212        1.97773683623975       -0.57387359102285\\nH           -0.39172309328816       -0.62274077186319        1.47386371709359\\nH           -0.85602884988518       -2.34783359840789        1.71308417911788\\nH            2.84647547312515       -2.47437932833222       -0.03521090235873\\nH            5.51056445875601       -1.31184492485416        0.30564840064809\\nH            5.25956673170500        2.96696226304241        0.24592451633612\\nH            2.79266531648636        2.87990391092023       -0.08137421057608\\nH            7.69598601238293       -0.64985098563610       -0.30356142548249\\nH            7.45939794111360       -0.63427641999695        1.46830759819217\\nH            8.71104980922648        0.39101517233835        0.72465695651682\\n43\\n energy: -70.312000250681 xtb: 6.7.0 (08769fc)\\nC           -6.43035247263649       -0.85804661156428       -0.23286178004975\\nC           -4.96172799235149       -0.93351307671452        0.04386427960676\\nC           -4.38468542452034       -2.06347214766773        0.60915524616232\\nN           -3.09936323859283       -2.17752916756505        0.87644974477636\\nC           -2.28833815762676       -1.17249620483444        0.57905043956575\\nC           -2.73400570532928        0.01634322385762       -0.00929040293265\\nC           -4.10386994338466        0.12877348601264       -0.25617375741271\\nO           -4.60819109089808        1.27887871065045       -0.83943000193471\\nC           -5.00813269319181        2.26066544438624        0.09276756725226\\nC           -1.79651629487381        1.13052628921417       -0.35135197274727\\nC           -0.84535196348631       -1.38647327414372        0.86894310940110\\nS            0.03932858598992       -1.78546364076418       -0.74414541981934\\nO            0.58131802272895       -3.14127994023355       -0.60530143452433\\nC            1.48109417922254       -0.76830655292933       -0.36947319889955\\nN            1.52003330621818        0.52332266416133       -0.22419232933913\\nC            2.83962278241936        0.81628011089501        0.01638137561678\\nC            3.59954850419505       -0.37774204693758        0.00229472410584\\nN            2.69588395083000       -1.37222252548619       -0.25551301024826\\nC            4.96904818278654       -0.39650851480623        0.22497159004372\\nC            5.57586256492785        0.82563201105666        0.46517569434904\\nC            4.82459031534446        2.01861666026258        0.47867953842110\\nC            3.47019569296504        2.03398440078963        0.25922709417010\\nO            6.90569674788100        0.99215327926633        0.70393276689241\\nC            7.73576394679521       -0.14302100152149        0.70970081793766\\nH           -6.96999213054914       -0.54541424633157        0.66208699611500\\nH           -6.80996891243125       -1.83265912745949       -0.53012818995208\\nH           -6.62577373385476       -0.13823296156730       -1.02409158069169\\nH           -4.99221833632436       -2.92196085958510        0.86496411284228\\nH           -5.82080964654474        1.89919009145442        0.73214670175994\\nH           -5.35759609102858        3.10949636379649       -0.49243748030758\\nH           -4.17248805253743        2.57410569143442        0.72777703615095\\nH           -0.90184759416886        0.76338050397919       -0.84765859569801\\nH           -1.47518398801125        1.64855720907816        0.55278845137390\\nH           -2.29554994274961        1.84302543284783       -1.00295192501853\\nH           -0.37582879474177       -0.50157232450254        1.28862819292228\\nH           -0.70946637527002       -2.26137215150688        1.50034000926042\\nH            2.83360367831769       -2.36807641996216       -0.33183106598227\\nH            5.51367404180976       -1.32639489448605        0.20746161834215\\nH            5.36388365866264        2.93209979004315        0.67187227192506\\nH            2.90576189976153        2.95167441598373        0.27197128257133\\nH            7.73031714497882       -0.64994366185544       -0.26069455042226\\nH            7.44315461881859       -0.85107211605044        1.49190818748125\\nH            8.73887775046621        0.22606668929193        0.91651484692455\\n43\\n energy: -70.312012823581 xtb: 6.7.0 (08769fc)\\nC           -6.45874831473585       -0.90181130553859       -0.09159261727195\\nC           -4.97999335345085       -0.93722803067520        0.13675038214171\\nC           -4.37193364763085       -2.01195760407368        0.77226259319175\\nN           -3.07540092233972       -2.10042861959533        0.99207765875452\\nC           -2.28443203622331       -1.12634852374977        0.56724779554359\\nC           -2.76401311214913        0.01291983117754       -0.08911401695311\\nC           -4.14379575846854        0.10196230348585       -0.28384888185043\\nO           -4.67477430962819        1.20455200219013       -0.93075723396352\\nC           -5.15274028650843        2.20461621537957       -0.05578748017695\\nC           -1.84622664308909        1.10095533431105       -0.54918866567113\\nC           -0.82919792542484       -1.30057018407844        0.82239922548556\\nS            0.04934067669706       -1.61740158323881       -0.80577817627541\\nO            0.55659672376104       -2.99276503337560       -0.75909092556731\\nC            1.51014408624958       -0.66095371244617       -0.35806294107004\\nN            1.57099086683668        0.60940336500829       -0.08753449382915\\nC            2.89929900692956        0.85916296826343        0.15540259792749\\nC            3.64050370731370       -0.33788444142241        0.00922862860483\\nN            2.71729284731749       -1.28982848210342       -0.32768980933536\\nC            5.01329337352421       -0.39796346697258        0.20156192451120\\nC            5.64423052637268        0.78611554380714        0.54732570774672\\nC            4.91236986133061        1.98246394390521        0.69207690718280\\nC            3.55430873448945        2.03816106246687        0.50318210890011\\nO            6.98101360422251        0.90886296774076        0.77417157186363\\nC            7.78939657768345       -0.23663585818627        0.66310040479319\\nH           -6.97541297008909       -0.54864951449172        0.80193111539892\\nH           -6.82802263181662       -1.89939705016868       -0.31861833215102\\nH           -6.69587094761470       -0.23560755967853       -0.91785097776969\\nH           -4.96285541648560       -2.84641115575825        1.12677079191238\\nH           -5.94459003039102        1.82567143413434        0.59816653448466\\nH           -5.55250247764761        2.99463692129436       -0.68916433239705\\nH           -4.34520609012836        2.60764421761822        0.56555750552069\\nH           -1.05523842419455        0.70699680658189       -1.18246182100115\\nH           -1.36744981455826        1.58547196409099        0.30155933195274\\nH           -2.40669999135565        1.84276853932957       -1.11204685969634\\nH           -0.38389329319705       -0.40472349545950        1.24799392040316\\nH           -0.65267754187851       -2.17777050122298        1.44034776696244\\nH            2.83888348744148       -2.27575482049515       -0.49904382039910\\nH            5.54197456486038       -1.32949381923307        0.08192415704821\\nH            5.46951155984805        2.86545270458692        0.96172075779730\\nH            3.00566711836291        2.95847141909001        0.61665159993267\\nH            7.76527397100195       -0.64903559471745       -0.35098892639478\\nH            7.49027167302093       -1.01032089968768        1.37786524167320\\nH            8.80131397175838        0.09265071189432        0.89287108202988\\n43\\n energy: -70.312014116532 xtb: 6.7.0 (08769fc)\\nC           -6.46690265533601       -0.79710120089345        0.08387842171005\\nC           -4.98090479126676       -0.87318998833521        0.25051931291600\\nC           -4.39033698989652       -1.92898319127562        0.93244927014624\\nN           -3.08988142318216       -2.06760559470710        1.09318148215337\\nC           -2.27600486449340       -1.17015915461401        0.55621603318335\\nC           -2.73494233367526       -0.06173744481345       -0.16293988385366\\nC           -4.11921033506461        0.08678085575362       -0.28982874691807\\nO           -4.61520254403389        1.16475052006853       -0.99980995419905\\nC           -5.13231780942120        2.20571470842539       -0.19754324109457\\nC           -1.80530608839981        0.92362838162653       -0.79989427822512\\nC           -0.82083160075541       -1.40203364102244        0.77773192904249\\nS            0.05875335460461       -1.57910309860636       -0.86419499130456\\nO            0.59171419852981       -2.94558522616470       -0.92176156544205\\nC            1.50375273529844       -0.63534631208577       -0.34514594381202\\nN            1.54267769007542        0.60319688267356        0.05042166825549\\nC            2.86988176780949        0.85762057433388        0.29788203380882\\nC            3.63260820112754       -0.30207445069410        0.02202990328169\\nN            2.72377783088666       -1.23571308329265       -0.39562257686925\\nC            5.00937767524207       -0.35089903434021        0.18674845567338\\nC            5.62209176559080        0.80645453506431        0.63904388102665\\nC            4.86846232836918        1.96594215224576        0.91349139198049\\nC            3.50637132754841        2.01026843309081        0.75196465426071\\nO            6.95988607325374        0.93546395887423        0.85584820605933\\nC            7.78983136341596       -0.17445279120358        0.61597937817030\\nH           -6.93022375178634       -0.33408635778290        0.95631891980208\\nH           -6.88139503334695       -1.79735514188672       -0.02130704460729\\nH           -6.71920963932381       -0.21299239352586       -0.79827645628882\\nH           -4.99992613555431       -2.70705321142084        1.37319851414444\\nH           -5.95349737832457        1.86526268629833        0.43972198679190\\nH           -5.49956881661765        2.96316195077828       -0.88792522854166\\nH           -4.35241823836306        2.63998178527783        0.43865764437365\\nH           -1.52911375924154        0.57565251707598       -1.79419184026110\\nH           -0.89343378656500        1.06073822638619       -0.22428716919906\\nH           -2.30439153477011        1.88152894927174       -0.92046625841301\\nH           -0.35989738349491       -0.56385709068455        1.29608465637502\\nH           -0.66867514887964       -2.33338553367946        1.31885238666687\\nH            2.86186010464310       -2.19869502741919       -0.66011816619503\\nH            5.55458548233769       -1.25426316483345       -0.03289753682610\\nH            5.41208406461439        2.82975943067678        1.26143669285275\\nH            2.94126584582364        2.90238939844076        0.96557530722088\\nH            7.75304346252146       -0.48474085635861       -0.43343919102328\\nH            7.52238964189465       -1.02184305645674        1.25574767536763\\nH            8.79917812822340        0.15395909972104        0.85819726780099\\n43\\n energy: -70.311726202235 xtb: 6.7.0 (08769fc)\\nC           -6.46341053358615       -0.74861245187340        0.14877646993134\\nC           -4.97615499862734       -0.84610599745070        0.29271621789465\\nC           -4.39480026348035       -1.89003976830016        1.00039277494990\\nN           -3.09481808287159       -2.05935641243005        1.13080744423269\\nC           -2.27129014745880       -1.20856720220704        0.53497313015743\\nC           -2.72015394035195       -0.10032094238110       -0.18922653355855\\nC           -4.10392883658065        0.07846555399862       -0.29046294291876\\nO           -4.58172598236867        1.15065705055694       -1.01940274834801\\nC           -5.11285061710709        2.20662959751793       -0.24634913967161\\nC           -1.78771098959137        0.86873578090270       -0.84691272077511\\nC           -0.81788085753798       -1.47978017668016        0.73288766178318\\nS            0.06417372104862       -1.56776996207284       -0.91325105406228\\nO            0.61596555753398       -2.92355088310273       -1.02829086619336\\nC            1.49867486070834       -0.62914184602714       -0.35855629593545\\nN            1.52840927318954        0.60027697706283        0.06537479247337\\nC            2.85319125397581        0.85748906512428        0.32251645913985\\nC            3.62424894174623       -0.29050927491563        0.02181642932125\\nN            2.72270762631358       -1.22003176976960       -0.41993171206332\\nC            5.00096064695815       -0.33413775743211        0.18868498737497\\nC            5.60511830529600        0.81675071613419        0.66816556612276\\nC            4.84328657688215        1.96489389022783        0.96666786451431\\nC            3.48124946314287        2.00383591804505        0.80350759017140\\nO            6.94156977427412        0.94949740602319        0.89112921580434\\nC            7.78050370241366       -0.14594566671593        0.61899937746194\\nH           -6.90356095766843       -0.23946752501382        1.00748590328670\\nH           -6.89719012423007       -1.74481929111041        0.09529143507646\\nH           -6.72164769043744       -0.20059046440649       -0.75464653239296\\nH           -5.01268702732536       -2.63297612212697        1.48794284692076\\nH           -5.95364458681951        1.88419591990041        0.37371626254414\\nH           -5.45465340502297        2.95586821530567       -0.95842569354305\\nH           -4.34662254224234        2.64486843647536        0.40347763475084\\nH           -1.57466897913045        0.54467525560082       -1.86437728713115\\nH           -0.84657290344233        0.96117984654460       -0.31094243199385\\nH           -2.25923086072491        1.84611400895423       -0.91439599807884\\nH           -0.34697782387556       -0.68144547891638        1.30422266931868\\nH           -0.68043873541981       -2.44235070226233        1.22049358272515\\nH            2.86608098154075       -2.17552871415413       -0.70789368118700\\nH            5.55244798347325       -1.22882972896672       -0.04995048518220\\nH            5.38050380876486        2.82449415422357        1.33447042130602\\nH            2.90991675917080        2.88744518419662        1.03518789208829\\nH            7.73990716435346       -0.43083164560721       -0.43744727792616\\nH            7.52547316387027       -1.01150172453648        1.23936401433233\\nH            8.78823232126222        0.18613753165100        0.86292175727020\\n43\\n energy: -70.311040616114 xtb: 6.7.0 (08769fc)\\nC           -6.45490315560205       -0.67146785030554        0.27311551932401\\nC           -4.96619209100919       -0.80792028187937        0.35807320301782\\nC           -4.38457669505566       -1.83814396239756        1.08496796857961\\nN           -3.08598495199800       -2.04644565142330        1.15932037164236\\nC           -2.26456611330878       -1.25416645906386        0.48423604682806\\nC           -2.71139084290367       -0.15139363412065       -0.24905760934757\\nC           -4.09302425570211        0.06607862252075       -0.29717669329406\\nO           -4.56925640059328        1.12660220177873       -1.04361971642415\\nC           -5.04736062562734        2.22205103753237       -0.29077258282660\\nC           -1.78180132876616        0.78283371225036       -0.95946284181599\\nC           -0.81419216917363       -1.57165455567451        0.64520739171585\\nS            0.07898507052199       -1.58508018912025       -0.99471194167697\\nO            0.65534852273468       -2.92707873300701       -1.14849604567995\\nC            1.49371719992068       -0.64172203726379       -0.39870836707596\\nN            1.49882169378273        0.57001116971196        0.07410572194853\\nC            2.81692993931422        0.83922559436799        0.35240717438238\\nC            3.61012225702157       -0.28206267243536        0.01139149344896\\nN            2.72810283216640       -1.20848684304858       -0.47434273865194\\nC            4.98619830388246       -0.30862397424871        0.18695595383139\\nC            5.56632003805262        0.83141640051772        0.71909845559783\\nC            4.78195369673987        1.95216315329801        1.06034093335186\\nC            3.42081428084703        1.97464305571323        0.88710406558747\\nO            6.89857833225351        0.97846880035629        0.95734740211868\\nC            7.75962372583381       -0.08820502850423        0.64333495651809\\nH           -6.84187330141281       -0.10291104454131        1.11996249982988\\nH           -6.91914569136816       -1.65497390854953        0.29374293565135\\nH           -6.73682608307795       -0.16542256547112       -0.64754231649104\\nH           -5.00057141275544       -2.53707499903944        1.63571980522528\\nH           -5.88108838218002        1.94604440214286        0.36028015095409\\nH           -5.38492082461803        2.95892378792645       -1.01768148800393\\nH           -4.25102832876605        2.65602022702321        0.32470495355519\\nH           -1.58767896875722        0.42244308394299       -1.96832819665315\\nH           -0.83348927527585        0.88639764121261       -0.43737405511405\\nH           -2.24675249310373        1.76135619389799       -1.05317422079842\\nH           -0.32809546474452       -0.81941356432946        1.26548855377171\\nH           -0.69732781268543       -2.56189593770311        1.08000901624107\\nH            2.89062370470354       -2.14891322528653       -0.79930451918304\\nH            5.55525220952287       -1.18278236309939       -0.08432082922507\\nH            5.30097966066621        2.80411065073218        1.46965612896486\\nH            2.83214112596542        2.83729578795258        1.15194205868767\\nH            7.73230626691606       -0.32563143145542       -0.42520538613212\\nH            7.51609460378737       -0.98541308249308        1.22182077933467\\nH            8.75913420386975        0.25079747156944        0.91047300827687\\n43\\n energy: -70.309930276272 xtb: 6.7.0 (08769fc)\\nC           -6.42864376141671       -0.54816753864467        0.39875491290379\\nC           -4.94509064449752       -0.75019975884568        0.41961967584897\\nC           -4.37504660254951       -1.78105097110741        1.15451243688280\\nN           -3.08581894046128       -2.04916136940916        1.16916956682113\\nC           -2.26297708586645       -1.32511697256438        0.42159337292537\\nC           -2.69341365638497       -0.21810701011611       -0.31438321792247\\nC           -4.06531055915884        0.06191195502551       -0.30232649840069\\nO           -4.52972679457845        1.12417616781036       -1.05382136197454\\nC           -4.92093625047509        2.25925220368312       -0.30921513316640\\nC           -1.76025530294415        0.66672394406666       -1.08102763482788\\nC           -0.82583047637749       -1.72148265771584        0.54256802103673\\nS            0.09999742838388       -1.65595255998441       -1.07512023082881\\nO            0.71764332187216       -2.97786077119437       -1.24908533105988\\nC            1.47807929924834       -0.69093306805240       -0.42969918490401\\nN            1.44395896242816        0.50362940250957        0.08363744027176\\nC            2.75117082132620        0.80106539451919        0.38362499657744\\nC            3.57972593247639       -0.28398507284370        0.01113512838340\\nN            2.72908396400494       -1.21871095388285       -0.51316076903247\\nC            4.95503933642947       -0.27476009098991        0.19456716452715\\nC            5.49713340515476        0.86356495957067        0.76880455903079\\nC            4.67677563624038        1.94659666598658        1.14528364854154\\nC            3.31668130182463        1.93408511929378        0.96331522384031\\nO            6.82326403782833        1.04438071969496        1.01755609044374\\nC            7.72018500478229        0.02161252567010        0.66109437261944\\nH           -6.74997956565676        0.06392407176890        1.24263852258841\\nH           -6.93422684021351       -1.50833526285874        0.47373581568006\\nH           -6.73101490986214       -0.05909235999756       -0.52459496415825\\nH           -4.99251447109351       -2.42925472037958        1.76269577937038\\nH           -5.72741203419133        2.03634755394401        0.39423065325602\\nH           -5.26880837012902        2.98910520049142       -1.03837268860918\\nH           -4.07469742854370        2.67636745930329        0.24848646749944\\nH           -1.58321715840849        0.25925112261889       -2.07470601812524\\nH           -0.80653358036521        0.78036245743011       -0.57029014888710\\nH           -2.21279252205174        1.64660818696268       -1.21248337750601\\nH           -0.31486748579718       -1.04799510518277        1.23068014373769\\nH           -0.75720244331204       -2.74646510938169        0.90101518093347\\nH            2.92190590544119       -2.14257867032478       -0.86765932818769\\nH            5.55213922262076       -1.12126003296502       -0.10286093272214\\nH            5.16712000540783        2.79857565521137        1.58845091549138\\nH            2.70024587131130        2.76822748644032        1.25485316591359\\nH            7.69859125839095       -0.17426788459377       -0.41597351618376\\nH            7.50971890530423       -0.90568250453440        1.20388191623163\\nH            8.70785826387566        0.38465119355268        0.94040216513095\\n43\\n energy: -70.308061767975 xtb: 6.7.0 (08769fc)\\nC           -6.26522015195225       -0.27526230566943        0.74905461463487\\nC           -4.82609886416011       -0.64648918267220        0.56938247278922\\nC           -4.27671414517387       -1.73814813742320        1.22815185039139\\nN           -3.02965976334160       -2.13154707362399        1.07996328495463\\nC           -2.23386949726945       -1.48861894670228        0.23264798706537\\nC           -2.64581197680961       -0.35110294573622       -0.46450009894150\\nC           -3.96940953511406        0.06750407272831       -0.27151781978809\\nO           -4.42791941550618        1.17855210595107       -0.95563273371783\\nC           -4.49522616901044        2.35542265465833       -0.17632172646107\\nC           -1.75226390714753        0.42759067481354       -1.37807246492055\\nC           -0.84612722679340       -2.05772400314878        0.21885357409320\\nS            0.14377274984732       -1.84057223919492       -1.34211981748468\\nO            0.87630426993070       -3.10555704947522       -1.50346394721726\\nC            1.41111852218217       -0.78971709413623       -0.60717058538818\\nN            1.28977109286267        0.41530667224640       -0.13303632552231\\nC            2.55603296972547        0.77374013762529        0.26143623166787\\nC            3.45348660085496       -0.28529134309972       -0.01367131730047\\nN            2.68423028507838       -1.26773823954233       -0.57415352069963\\nC            4.80926856855829       -0.21302586225062        0.27271348308054\\nC            5.25979658778876        0.96273409711617        0.85054386442031\\nC            4.36987368856862        2.01955450826720        1.13222640290075\\nC            3.02930043537679        1.94461258494106        0.84877269735492\\nO            6.55512928843277        1.20727785480245        1.19009165107874\\nC            7.51818845803035        0.21675561599493        0.92618873876783\\nH           -6.39949372737188        0.32698129482587        1.64856483818884\\nH           -6.87036293326347       -1.17290698927507        0.85616805145060\\nH           -6.61893138353805        0.29209552146396       -0.10874417358935\\nH           -4.87354003587815       -2.32693985524821        1.91265319989812\\nH           -5.16661499137026        2.24226327568498        0.67980047054167\\nH           -4.87818299784851        3.13144839556961       -0.83701414581241\\nH           -3.50382340259896        2.64407446724712        0.19042888835427\\nH           -1.61743993467602       -0.10183980791205       -2.31903192462144\\nH           -0.77932460197855        0.60096581937138       -0.92211724243303\\nH           -2.21314605188064        1.38467730422134       -1.60806840478611\\nH           -0.27233368271851       -1.61588038402533        1.03453286618671\\nH           -0.90281782674638       -3.13467697183965        0.37281588858387\\nH            2.93945414829857       -2.18863808414936       -0.89656604348277\\nH            5.46077755602849       -1.04121586650068        0.04655410971071\\nH            4.79039207791730        2.90327068204850        1.58500746114688\\nH            2.35941359374153        2.75952492233240        1.06782114936558\\nH            7.57896838845607       -0.00950083753157       -0.14341792719266\\nH            7.31140489348772       -0.70385919739940        1.48177811952154\\nH            8.46764904699737        0.63189875463470        1.25999532319837\\n43\\n energy: -70.307770617422 xtb: 6.7.0 (08769fc)\\nC           -5.55442968045721        0.14340103351451        1.54458427639914\\nC           -4.32839041751177       -0.47615201941542        0.95129355248337\\nC           -3.79213476541048       -1.65530244262209        1.45237435005177\\nN           -2.70988841858817       -2.22604464967002        0.97057817130689\\nC           -2.08369702647487       -1.68595973533461       -0.07090703584490\\nC           -2.54236847914210       -0.52969548754935       -0.70094369198011\\nC           -3.66995491270111        0.08865105490510       -0.14069962263610\\nO           -4.15919944994646        1.25452987393556       -0.70336439208721\\nC           -3.60849159146904        2.42534359673621       -0.13387944351147\\nC           -1.92006905392810        0.06145951194761       -1.92587490510850\\nC           -0.88433609421526       -2.50477169142374       -0.45564922588758\\nS            0.16133776540385       -2.03406017225170       -1.90910395204993\\nO            1.05604395582326       -3.19058157994943       -2.06564569183618\\nC            1.21982127137053       -0.88789364531774       -1.00426858561077\\nN            0.96772532060433        0.32638596664759       -0.61359083900242\\nC            2.12593201317831        0.75673732252463       -0.01253552188765\\nC            3.09952810059042       -0.26829520833043       -0.07543544655951\\nN            2.48254374384542       -1.30588915845516       -0.71783159393006\\nC            4.37770005006574       -0.12285458171198        0.44449331738250\\nC            4.67077826416208        1.09334295916234        1.03962585477638\\nC            3.70443119639912        2.11769814110804        1.10903133228282\\nC            2.44055530520423        1.96985102010324        0.59507923577562\\nO            5.87299187916845        1.40974408003073        1.59414050964679\\nC            6.90212674432169        0.45148893147515        1.56003617382081\\nH           -5.29520086615164        0.72991998236501        2.42655141935234\\nH           -6.25660368864828       -0.62897443516161        1.85007030805613\\nH           -6.03268610804809        0.79732959381670        0.81958860451153\\nH           -4.25549941881233       -2.16308745901322        2.28832062652930\\nH           -3.86624994693573        2.51164811305155        0.92688281789626\\nH           -4.04193467708293        3.26045255562757       -0.68154135239522\\nH           -2.51863485217994        2.44323973248224       -0.23588122540733\\nH           -1.87730895110220       -0.67329731215746       -2.72589197819975\\nH           -0.91402136067131        0.41670888801230       -1.70805796747726\\nH           -2.52579281155638        0.89682077090841       -2.26826999339597\\nH           -0.22062631962227       -2.59681576229950        0.40473987238503\\nH           -1.22050462622632       -3.51313976096008       -0.71115744371413\\nH            2.83737486168082       -2.21357649856075       -0.97860506921887\\nH            5.09110257434888       -0.92768479407082        0.37495637896229\\nH            4.00212946975057        3.03687796174790        1.58786007992899\\nH            1.71190176243947        2.76139346290959        0.65380127400874\\nH            7.17177882041839        0.18808741531836        0.53197189979880\\nH            6.62563120317269       -0.45545443384165        2.10765887520994\\nH            7.75659021495095        0.91841785974802        2.04702304716527\\n43\\n energy: -70.311885395698 xtb: 6.7.0 (08769fc)\\nC           -4.42020773408942        0.51772709844608        2.16979325522412\\nC           -3.57017017624043       -0.31314619239206        1.26111398490543\\nC           -3.13373141619458       -1.58126400141704        1.62335988726189\\nN           -2.32355479261213       -2.31104202346635        0.88653323105618\\nC           -1.92218356868060       -1.85555722101736       -0.29571088727420\\nC           -2.40254886921543       -0.66645853188347       -0.84115241805212\\nC           -3.18250188756945        0.13908515107213        0.00011853827529\\nO           -3.60452216852742        1.37843651499040       -0.43844776813211\\nC           -2.59561650681345        2.36871756510572       -0.34459627725739\\nC           -2.18648683145768       -0.24396855199024       -2.26090861220866\\nC           -0.99181082713537       -2.80477126823058       -0.98728143390041\\nS            0.06064456235953       -2.12670052291952       -2.34948314791934\\nO            1.08120176993049       -3.16402908288102       -2.55541136665049\\nC            0.92753733328629       -0.95701002353533       -1.28430543349655\\nN            0.52792976287791        0.20653207975374       -0.86244807858919\\nC            1.56709194919833        0.68868518888946       -0.10459751631112\\nC            2.62780520988992       -0.24781238727980       -0.10640550574842\\nN            2.18082628770231       -1.28936990601673       -0.87210009017752\\nC            3.82116791862195       -0.03038009340245        0.56733720544825\\nC            3.93530339831500        1.16556467575076        1.25741268657334\\nC            2.87970858262767        2.10012769594404        1.26822319576248\\nC            1.70061700283534        1.88211290417472        0.60066604974777\\nO            5.03406701381193        1.54508291080734        1.96555640964513\\nC            6.14386272496496        0.68181959207271        2.00051268780606\\nH           -3.80315304150089        0.98549759941741        2.93754749129989\\nH           -5.16343914381658       -0.10209509755606        2.66649630863411\\nH           -4.92149193576679        1.29770824539011        1.60269816797809\\nH           -3.43469568122408       -2.02314504187881        2.56437311018123\\nH           -2.42544749907669        2.64620015334765        0.70175119828867\\nH           -2.96391788418406        3.23366252852939       -0.89378117863698\\nH           -1.65234949000911        2.02400682070310       -0.77597542503183\\nH           -2.28774745780472       -1.09322613548986       -2.93207401718557\\nH           -1.20302340230375        0.20108521598595       -2.39184457278393\\nH           -2.93844129588364        0.49800021458565       -2.52306700686882\\nH           -0.32514234958140       -3.26466621210910       -0.25876113807543\\nH           -1.56207738687968       -3.60579926047937       -1.46686251389941\\nH            2.64486601575755       -2.14401430767285       -1.13961565347480\\nH            4.60734078951481       -0.76715611795454        0.54004462833317\\nH            3.03804099926559        3.00790714851531        1.82821125023732\\nH            0.90207505723344        2.60579993121457        0.61528837000117\\nH            6.54878222960110        0.50731959171705        0.99829158449862\\nH            5.89273388727290       -0.27788277431770        2.46387991179338\\nH            6.89265985147339        1.18841492744164        2.60714788872702\\n43\\n energy: -70.314000336654 xtb: 6.7.0 (08769fc)\\nC           -4.10176583677526        0.59681565272052        2.26108796306799\\nC           -3.34727059207608       -0.27157289118363        1.30465840140146\\nC           -2.89492018125374       -1.53514769440528        1.66144518705674\\nN           -2.16977408543240       -2.30062544442867        0.87255687751127\\nC           -1.87921993306555       -1.88373500921406       -0.35432523414531\\nC           -2.37945970462873       -0.69431588359417       -0.88204333422652\\nC           -3.06898966996655        0.14400214082383        0.00223181999223\\nO           -3.50814655873254        1.37964988575342       -0.43023168412889\\nC           -2.48926793746703        2.36354329737561       -0.39398484352484\\nC           -2.25881712102307       -0.30470887888999       -2.32215071760931\\nC           -1.04338147411922       -2.85723080636128       -1.12371937445243\\nS            0.05145467500036       -2.13447513693404       -2.43316326447294\\nO            1.09322089651030       -3.15179803592758       -2.63194733715404\\nC            0.86790101550409       -0.97349201284145       -1.32174784140222\\nN            0.44530913318129        0.18497821126668       -0.90793721234792\\nC            1.45474854912898        0.67137284800923       -0.11388154409225\\nC            2.52326756170181       -0.25579457346895       -0.08693993927013\\nN            2.11017331426915       -1.29708848080880       -0.87144004786346\\nC            3.69442721694281       -0.02978865189270        0.62191797637866\\nC            3.77725280842378        1.16448310649097        1.31928431561928\\nC            2.71224566996737        2.08819342252041        1.30437312204596\\nC            1.55556706673109        1.86218630676990        0.60126455240106\\nO            4.85287344953062        1.55265569984463        2.05753374081001\\nC            5.97478895991515        0.70590705643824        2.10780539419030\\nH           -3.40837240743189        1.13041998572371        2.91189524045288\\nH           -4.75623626022483       -0.00552988165192        2.88695016679201\\nH           -4.69407010354648        1.32634707214651        1.71447116701658\\nH           -3.10948266981635       -1.94464904228985        2.64002693803554\\nH           -2.26572978464322        2.64836195194784        0.64055506478676\\nH           -2.87791816692190        3.22661420649956       -0.93179577508468\\nH           -1.57130619110938        2.00596443332493       -0.86809297602347\\nH           -2.37802704934996       -1.17143131919921       -2.96758820780290\\nH           -1.29471570319245        0.15781793557874       -2.52005048637028\\nH           -3.03893537680335        0.41751439821062       -2.55562839048315\\nH           -0.40864809697037       -3.42266348012385       -0.44312456873484\\nH           -1.67797179370090       -3.56515348770706       -1.66411209434258\\nH            2.59095297281569       -2.14553762684135       -1.12873499537886\\nH            4.48805882398794       -0.75889738348102        0.61402015516167\\nH            2.84499539951529        2.99474220786256        1.87299735473694\\nH            0.75046061929273        2.57862913170698        0.59542873942590\\nH            6.40042128275212        0.54422773121475        1.11201353304562\\nH            5.72978079590774       -0.26049068519534        2.56036641778453\\nH            6.70452748714418        1.21969872417438        2.73128274119827\\n43\\n energy: -70.315521227417 xtb: 6.7.0 (08769fc)\\nC           -3.94932784101150        0.62920196177893        2.29975387159745\\nC           -3.22954745565438       -0.24610708033623        1.32292769725666\\nC           -2.72389337995519       -1.48714480949892        1.68679092155092\\nN           -2.04319668702679       -2.26761068723208        0.87268734477393\\nC           -1.85119866446620       -1.88566635195308       -0.38407760159598\\nC           -2.38452241641512       -0.70725235890496       -0.90444750647370\\nC           -3.03672848139564        0.14156290107395       -0.00371239373298\\nO           -3.52124603123876        1.35852259983080       -0.44136725389358\\nC           -2.52841709631881        2.36901395490262       -0.45088868133837\\nC           -2.31548535534559       -0.33906446905712       -2.35320473856039\\nC           -1.07345377491220       -2.86805977620082       -1.19807988224391\\nS            0.06007719143991       -2.12403358609681       -2.46657903124525\\nO            1.10680855778250       -3.13744708764701       -2.65858452645209\\nC            0.85090077257678       -0.97691128766661       -1.32516898254537\\nN            0.42242730749492        0.18088471972515       -0.91533970511415\\nC            1.42000746039903        0.66374653063352       -0.10387888285499\\nC            2.48778928185208       -0.26401446090075       -0.06408658640224\\nN            2.08604018644427       -1.30266666464890       -0.85735760401587\\nC            3.65027346568867       -0.04023175845624        0.65955891826630\\nC            3.72584652898327        1.15300543288976        1.35929517693022\\nC            2.66106866330053        2.07677758831260        1.33336162364223\\nC            1.51288430413661        1.85265861017230        0.61566473724566\\nO            4.79373007515796        1.53993625438955        2.10946443149257\\nC            5.91700619073945        0.69525340458046        2.16603884686352\\nH           -3.23381425855908        1.17325875515603        2.91695965476483\\nH           -4.57467206975414        0.02985919542717        2.95768664549011\\nH           -4.56868973330512        1.34851235655593        1.76980024317588\\nH           -2.85980688600404       -1.86682981821727        2.69114756442953\\nH           -2.28753127576634        2.68306798926415        0.57114093116253\\nH           -2.95206941252248        3.20935306659153       -0.99827533688810\\nH           -1.61312717719083        2.02409943089943       -0.93936963532051\\nH           -2.43317864472362       -1.21703388688342       -2.98356122735715\\nH           -1.36480823783802        0.13756905312007       -2.58208098429831\\nH           -3.11421769525735        0.36568342497857       -2.57638934676042\\nH           -0.46656611362107       -3.49551536091398       -0.54743656282433\\nH           -1.74165372270469       -3.51146844129131       -1.77621864225825\\nH            2.57035239682542       -2.15049790442710       -1.10994652255483\\nH            4.44330795602045       -0.77004959202663        0.65997687115597\\nH            2.78747161331562        2.98247588502909        1.90482317424062\\nH            0.70881577600976        2.57012887511729        0.60101004562721\\nH            6.35011777248771        0.53659255319571        1.17296268730859\\nH            5.67090867587160       -0.27267081444323        2.61472937135523\\nH            6.64131923443184        1.20911065314290        2.79579787640174\\n43\\n energy: -70.317061955081 xtb: 6.7.0 (08769fc)\\nC           -3.64146430686236        0.67818267167462        2.37721552027730\\nC           -3.00954437127143       -0.21694273027245        1.35844431843412\\nC           -2.45125234523322       -1.43959390669578        1.70631965118863\\nN           -1.85071764746477       -2.24181721517889        0.85052676217277\\nC           -1.79646476485385       -1.89725419373506       -0.42969010453631\\nC           -2.37605590926641       -0.72948974224700       -0.92515268417208\\nC           -2.95165876680175        0.13636910907896        0.00912628050935\\nO           -3.49011976441360        1.33579738481871       -0.41384359367209\\nC           -2.52357555761941        2.36776258737913       -0.49897055215389\\nC           -2.40883853351650       -0.38848086575637       -2.38151964782983\\nC           -1.09663903064761       -2.88498186080667       -1.30181188352938\\nS            0.05476472596896       -2.11933948600068       -2.54817034221483\\nO            1.11212446008124       -3.12206160875298       -2.73531614959340\\nC            0.81643384569839       -0.97435273642213       -1.38644994545810\\nN            0.38643429893101        0.19420050629609       -1.01028952774279\\nC            1.35317249502907        0.67320570524880       -0.16027265314911\\nC            2.40450989122619       -0.26877345619583       -0.06116766989312\\nN            2.02438560676749       -1.31144531794442       -0.85978045026072\\nC            3.53602001742968       -0.05246739487863        0.71216532397294\\nC            3.59715897121376        1.14799150516068        1.40085280776455\\nC            2.54857954036933        2.08652867833943        1.31506978588432\\nC            1.43112274009859        1.86971868631349        0.54834945677909\\nO            4.63502252946770        1.52901589481479        2.19486455894772\\nC            5.74109305061954        0.66837859945110        2.31492903793880\\nH           -2.87673068621722        1.25553780361977        2.89768793022326\\nH           -4.18141094228715        0.09066726204536        3.11640630365142\\nH           -4.32835953849477        1.36795889429014        1.89304705897975\\nH           -2.47578254618022       -1.78647904624823        2.73125161992537\\nH           -2.22611244581904        2.70301729471234        0.50110212124385\\nH           -2.99985284193730        3.18994139981229       -1.03050889973847\\nH           -1.63349474578032        2.03466411354137       -1.03955313409569\\nH           -2.54730031800108       -1.27917035289619       -2.98938941257321\\nH           -1.48159623413253        0.09902762857326       -2.67574313972801\\nH           -3.23126925003382        0.29948887980484       -2.56704472661336\\nH           -0.50661726837273       -3.56999852493585       -0.69551917956681\\nH           -1.80560425307835       -3.46125091133668       -1.90032288157464\\nH            2.50995052864855       -2.16697076741868       -1.08166444090271\\nH            4.31756263681770       -0.79320919020449        0.75725086025967\\nH            2.66242438236966        2.99767684695977        1.88051099794306\\nH            0.63988888047586        2.59889369334667        0.48775691821115\\nH            6.21832397447998        0.49063426826438        1.34560224114087\\nH            5.45886916116869       -0.28992675508665        2.76297548122559\\nH            6.44262133139578        1.17934564943254        2.97225298232626\\n43\\n energy: -70.318288130761 xtb: 6.7.0 (08769fc)\\nC           -3.32342176450343        0.73132387257583        2.43280031681871\\nC           -2.78177571151157       -0.18361135111416        1.38021776839209\\nC           -2.16501307270516       -1.38417050486095        1.70467958034159\\nN           -1.64620953611644       -2.20753652740290        0.81556703307693\\nC           -1.73739017120931       -1.90455765455551       -0.47265585931532\\nC           -2.36930245219247       -0.75141419629755       -0.93894693256602\\nC           -2.86655189294730        0.13154938296304        0.02245185078670\\nO           -3.46351384190638        1.31042784693550       -0.37938059943795\\nC           -2.53367086324858        2.36634948302832       -0.54164067403878\\nC           -2.51195441274310       -0.43971369604908       -2.39501806498553\\nC           -1.11100955148129       -2.89516418031835       -1.39250293177779\\nS            0.05191210053988       -2.11592682834699       -2.62728264286589\\nO            1.11802843781121       -3.10988950234311       -2.80815725230222\\nC            0.78839344262671       -0.96959936376729       -1.45178196537679\\nN            0.35761777859090        0.20788249770987       -1.10588721607124\\nC            1.29049974274643        0.68103968911538       -0.21577696237210\\nC            2.32263006224471       -0.27440480652250       -0.06018269443616\\nN            1.96582061451356       -1.31858952941932       -0.86743069930320\\nC            3.41954352099587       -0.06742253307635        0.76383836479761\\nC            3.46479636720544        1.13706662405489        1.44663567902404\\nC            2.43452334202011        2.08896334727754        1.30501400002902\\nC            1.35190916638370        1.88171370515601        0.48735720931079\\nO            4.46924596489234        1.50996578963238        2.28620146696776\\nC            5.55784230174292        0.63665103407586        2.46185769231866\\nH           -2.52180447059042        1.34118330298553        2.85072398393439\\nH           -3.76746679918890        0.15739628672862        3.24308008272265\\nH           -4.07467594940908        1.39020711177191        2.00420814087838\\nH           -2.07123540718405       -1.69504486641587        2.73714951216400\\nH           -2.17876652964156        2.72436293594227        0.43128856167742\\nH           -3.06693453613115        3.16764372261983       -1.05053997297222\\nH           -1.67354821670929        2.04836069551044       -1.13716307021372\\nH           -2.68436155263622       -1.34214322230852       -2.97627258146574\\nH           -1.60997592304619        0.04678344819482       -2.76168969014556\\nH           -3.34980761427797        0.23992094232709       -2.53555635026111\\nH           -0.53248624620114       -3.62292200495403       -0.82645516596815\\nH           -1.85448330793124       -3.41579127119236       -1.99832485816936\\nH            2.45031821747460       -2.18197293287768       -1.05843070815774\\nH            4.18769696992922       -0.81833775664800        0.85090824338856\\nH            2.53414984747871        3.00272677228304        1.86896792196176\\nH            0.57544271196666        2.62173671450290        0.38355072105879\\nH            6.07636603126082        0.44664286855369        1.51637108346811\\nH            5.24354427455369       -0.31510376219118        2.90241933100180\\nH            6.23507992850354        1.14341741668079        3.14731534808425\\n43\\n energy: -70.319365459719 xtb: 6.7.0 (08769fc)\\nC           -2.92113163702166        0.80598659371270        2.47194653517273\\nC           -2.49539724526415       -0.13622612027437        1.39072906041499\\nC           -1.80756756112434       -1.30768629684462        1.67534913729512\\nN           -1.38722902240266       -2.15573021756343        0.75750794497271\\nC           -1.65298701385399       -1.90530890140350       -0.51768855839474\\nC           -2.35805076406014       -0.77736829566080       -0.94050603197258\\nC           -2.75969185113012        0.12672185746250        0.04458851741879\\nO           -3.43463126582553        1.27485529919027       -0.32075063847196\\nC           -2.56700743042046        2.36002526780335       -0.59508126955753\\nC           -2.65132207078467       -0.50880568327127       -2.38237780133649\\nC           -1.10603207294573       -2.89656842703976       -1.48312075258347\\nS            0.04856503050275       -2.10302178216712       -2.72564538225224\\nO            1.12467695903318       -3.08473387000733       -2.90936062742203\\nC            0.76423105957471       -0.95243343017241       -1.54193302704000\\nN            0.33432956163317        0.23353799085885       -1.22594651334312\\nC            1.22198159467272        0.69410149370004       -0.28438599979688\\nC            2.22570266885609       -0.27885743092154       -0.06523609785612\\nN            1.89915698460141       -1.31964502402061       -0.88930143169648\\nC            3.27428910514576       -0.08941620461320        0.82325314403532\\nC            3.29836924128047        1.11450212113939        1.50801379898142\\nC            2.29486204651856        2.08345642152994        1.30416705880820\\nC            1.26147463685232        1.89428625560701        0.42099270229845\\nO            4.25606442659893        1.47080432342079        2.40741885912521\\nC            5.32049327619682        0.58198488212439        2.64340744836870\\nH           -2.09090907114112        1.45474162067808        2.75342278173420\\nH           -3.23226670511416        0.25418268623642        3.35613748483957\\nH           -3.74528918674434        1.42487590698492        2.12540828849924\\nH           -1.56953354192860       -1.57277413288480        2.69735300242733\\nH           -2.14273163419507        2.76337315239585        0.33095388364850\\nH           -3.17576694629800        3.12321319500973       -1.07733768274051\\nH           -1.75080070369977        2.05599208061643       -1.25613677831514\\nH           -2.88662095195786       -1.42650071466483       -2.91583372831383\\nH           -1.78765302829252       -0.04308283443526       -2.85420493643588\\nH           -3.49535723245550        0.17302570104994       -2.45827710127399\\nH           -0.52653094188127       -3.65556120876865       -0.96099946912457\\nH           -1.88486240260066       -3.37219448048029       -2.07934582576729\\nH            2.37865575414404       -2.19265473546635       -1.04473254039879\\nH            4.02247293147115       -0.85345667507655        0.95716429294008\\nH            2.37519527289118        2.99577785205727        1.87360235204330\\nH            0.50663068366729        2.64811478943222        0.26883691204385\\nH            5.88913225677356        0.38542772105860        1.72858264887412\\nH            4.96833773073638       -0.36558155395619        3.06386758976539\\nH            5.96475005996601        1.07861980758615        3.36702575038747\\n43\\n energy: -70.320412055041 xtb: 6.7.0 (08769fc)\\nC           -2.39921620713478        0.89890293474081        2.47766127286355\\nC           -2.12801558860077       -0.07939173918974        1.37923300154205\\nC           -1.36821925855708       -1.22043613196203        1.59509253868996\\nN           -1.07069295866042       -2.09604013942182        0.65530733758723\\nC           -1.53775409405676       -1.90322420716173       -0.57167763931005\\nC           -2.33275405497790       -0.80927125183468       -0.91982979632069\\nC           -2.61059655981146        0.12046876187982        0.08332723263746\\nO           -3.37487749096126        1.23122714208975       -0.21424819551730\\nC           -2.60227408427961        2.34553310468080       -0.62301328247999\\nC           -2.82998614589906       -0.60024481003552       -2.31478083013828\\nC           -1.08532500659562       -2.89095158017192       -1.58537933331545\\nS            0.05050168177091       -2.08631530617963       -2.84973402943074\\nO            1.13521350093007       -3.05626653529492       -3.03818847367325\\nC            0.74822595082408       -0.92551781669376       -1.66494365066380\\nN            0.32721688220402        0.27419649708649       -1.39267294511335\\nC            1.15178036518308        0.71995962387529       -0.38854921559137\\nC            2.10625180944115       -0.27859385736185       -0.08261149188993\\nN            1.81895307649319       -1.31662865133504       -0.92409519343836\\nC            3.08091961216426       -0.11334197968132        0.89061061505680\\nC            3.07868721698809        1.09154179366271        1.57391558579679\\nC            2.12410573067057        2.08700490437870        1.28233869528579\\nC            1.16629843413879        1.92277443167054        0.31309290438034\\nO            3.96346145276823        1.42396063449761        2.55377533786154\\nC            4.98074390022106        0.50848595581351        2.87917126886194\\nH           -1.55608727851474        1.58244261480563        2.58552412250778\\nH           -2.53564539934441        0.38017212500113        3.42386738791637\\nH           -3.29211337061572        1.47726575026617        2.25257898309699\\nH           -0.96247914124149       -1.43445765697580        2.57540181590792\\nH           -2.09199108103333        2.80208896028586        0.23176121291751\\nH           -3.30253896419421        3.06130671458316       -1.05102164716708\\nH           -1.85681717450411        2.05648045834225       -1.36903205944415\\nH           -3.18489501890921       -1.53166706755670       -2.75016579148201\\nH           -2.02723400261618       -0.20989304634298       -2.93876125886407\\nH           -3.64593202029571        0.11825020930810       -2.30615764425334\\nH           -0.50150995769602       -3.68084383500255       -1.11702602471323\\nH           -1.90482177021493       -3.31920904749495       -2.16012133881266\\nH            2.28154905101242       -2.20618978930083       -1.02518153433955\\nH            3.79329172574539       -0.89711203607530        1.08957604288269\\nH            2.18063650335351        2.99937533421140        1.85472390505082\\nH            0.45062432821371        2.69766477248769        0.09292945500520\\nH            5.62045997636232        0.29667416725849        2.01619231096602\\nH            4.56985134136382       -0.42943232750426        3.26664757323979\\nH            5.57300508883149        0.98925092161268        3.65598977590674\\n43\\n energy: -70.321025794389 xtb: 6.7.0 (08769fc)\\nC           -2.01841057698502        0.94069087449689        2.45704213301070\\nC           -1.86459861886405       -0.05520408836447        1.35210668322594\\nC           -1.06552286403610       -1.18023778669001        1.49890192131470\\nN           -0.86010253808822       -2.06783287941053        0.54625352029715\\nC           -1.46663222095923       -1.90540896789398       -0.62370711232338\\nC           -2.31587177327348       -0.83034346755025       -0.89619379922430\\nC           -2.49720501143892        0.11190377638203        0.11731539083481\\nO           -3.31231744891639        1.20276355017617       -0.11020580661498\\nC           -2.61218746074480        2.33429967771388       -0.59492784878230\\nC           -2.96543450075164       -0.65704798793257       -2.23227476993484\\nC           -1.08973579855391       -2.88294182399122       -1.67509350296193\\nS            0.07858437832752       -2.09340243256220       -2.93041362071513\\nO            1.15938878211055       -3.07198344570639       -3.08881830943851\\nC            0.75708177881313       -0.92033505940754       -1.74698051593370\\nN            0.35052297285661        0.29430958025732       -1.52357575788955\\nC            1.12654789456391        0.74128348871394       -0.48189352269933\\nC            2.03303549428250       -0.27494938151193       -0.09807364283806\\nN            1.77392089337807       -1.32075940086475       -0.93837122785793\\nC            2.94158409837665       -0.11903903327698        0.93843648907468\\nC            2.92280322344455        1.09549506660316        1.60402803514333\\nC            2.02144404574633        2.11224513558376        1.22887404520643\\nC            1.12953560101684        1.95706252620555        0.19711271190605\\nO            3.74067773448632        1.41789104938897        2.64374941177678\\nC            4.70002086204134        0.47700183224039        3.06003124859519\\nH           -1.17746166102594        1.63553440525476        2.45058941179754\\nH           -2.03320728278529        0.43874950736839        3.42185675354561\\nH           -2.94027755113306        1.50332413809754        2.33053677968357\\nH           -0.54547896032480       -1.36763060212087        2.42948262123292\\nH           -2.02846731482667        2.80578290422884        0.20234460875954\\nH           -3.37000831013671        3.03133670274649       -0.94968571215574\\nH           -1.94030556870176        2.06132861508459       -1.41361701805527\\nH           -3.43252763767403       -1.58310402782881       -2.56198515849193\\nH           -2.22196582526868       -0.36257679846314       -2.97203035891282\\nH           -3.72774585242228        0.11509874548180       -2.17101361791044\\nH           -0.53299335956859       -3.71450406464298       -1.24795778423308\\nH           -1.93725316767467       -3.24673379519373       -2.25139446961040\\nH            2.21148744151468       -2.22715405219819       -0.98308754298270\\nH            3.61624643733083       -0.91806880901735        1.19842397944156\\nH            2.06440089145873        3.03313993269936        1.78885431135931\\nH            0.45523191669801        2.74814264269841       -0.08692410635785\\nH            5.39868267560537        0.23283696573062        2.25319267268817\\nH            4.23043900953837       -0.44209197790087        3.42538461504072\\nH            5.24407617253326        0.95112776533290        3.87523486199054\\n43\\n energy: -70.321042392420 xtb: 6.7.0 (08769fc)\\nC           -1.88675328460444        0.92108791153370        2.44275954100117\\nC           -1.77947341634499       -0.06596100988043        1.32447411067671\\nC           -0.97264837673579       -1.19033303972797        1.42927674008943\\nN           -0.80924571206856       -2.07295492585658        0.46467060654973\\nC           -1.47166786674265       -1.90985448517403       -0.67529680783358\\nC           -2.32889717484354       -0.83087900351850       -0.90573661023965\\nC           -2.46546702200852        0.10670346737113        0.11939009491773\\nO           -3.29031439918521        1.19816008541146       -0.06806036407177\\nC           -2.60731737023206        2.34442801223412       -0.54154854324320\\nC           -3.03648998047230       -0.65114198934224       -2.21133651899943\\nC           -1.13408731824862       -2.87707488006593       -1.74784927518433\\nS            0.14601864107032       -2.14814935677656       -2.93632853437186\\nO            1.20833778853077       -3.15727504460760       -2.99596850097583\\nC            0.78623739258460       -0.95615873649402       -1.75167683161539\\nN            0.38217010641297        0.26561646506068       -1.56527553156872\\nC            1.13459074397751        0.72944669286314       -0.51369412693851\\nC            2.02335909375091       -0.28469568598941       -0.08458489071499\\nN            1.78074632750611       -1.34502668928968       -0.91074221002756\\nC            2.90502543234637       -0.11278167504557        0.97247963217795\\nC            2.87771029773711        1.11576615266931        1.61140623329869\\nC            1.99511168934252        2.13090125712595        1.19018434982509\\nC            1.12996824765622        1.95996456187281        0.13821583413426\\nO            3.66976202517540        1.45457451201342        2.66601389015462\\nC            4.61471793015271        0.51841050384597        3.12374720721608\\nH           -1.05229837276166        1.62261881376758        2.40120451311268\\nH           -1.85058729776789        0.41172854810171        3.40311027814887\\nH           -2.81822670847779        1.47698388586296        2.36615093811021\\nH           -0.41013924489322       -1.38035349989922        2.33439110737457\\nH           -1.98688040702518        2.78680685191985        0.24422700247220\\nH           -3.37770726481798        3.05466955807330       -0.83821078821185\\nH           -1.97375115437669        2.09882737640045       -1.39888950066009\\nH           -3.57162667584946       -1.55715205978209       -2.49070502787130\\nH           -2.31786565206396       -0.41769974974762       -2.99615723230378\\nH           -3.75216713352967        0.16247527272786       -2.13156775018786\\nH           -0.65160307294312       -3.76080507590543       -1.33540384465785\\nH           -1.98391273181075       -3.15428229952983       -2.36566223967811\\nH            2.20181923052413       -2.26037930154503       -0.91867632144142\\nH            3.56635122466691       -0.91082037955379        1.26741127597303\\nH            2.03094851220959        3.06369914658461        1.73071503233201\\nH            0.47098268889692        2.75024704277967       -0.18181175916113\\nH            5.33031401086215        0.25425580321018        2.33833604437758\\nH            4.13259801741629       -0.39079322501915        3.49746374371676\\nH            5.14235923695447        1.00719919127984        3.94108203429972\\n43\\n energy: -70.320724642113 xtb: 6.7.0 (08769fc)\\nC           -1.77070897177798        0.88165840880039        2.43967301583044\\nC           -1.70858962248717       -0.08967861682110        1.30441161882806\\nC           -0.90302953337356       -1.21842827799234        1.36416449474520\\nN           -0.77925666988077       -2.08815966875846        0.38294768689260\\nC           -1.48427387952818       -1.91087592726702       -0.72979752053763\\nC           -2.34199383173784       -0.82320384492341       -0.91491165524158\\nC           -2.43713945266283        0.10087972004266        0.12765352564003\\nO           -3.26634783043488        1.19636741015973       -0.01340659386702\\nC           -2.59967727217878        2.35214998131901       -0.48565334638968\\nC           -3.09918695523371       -0.61974759216856       -2.18923812300844\\nC           -1.18278634835393       -2.86342492449767       -1.82529466122833\\nS            0.22295180347997       -2.22662394473213       -2.92699719012075\\nO            1.24928159953532       -3.27193387762863       -2.86271358839532\\nC            0.82406005569795       -1.00610405419487       -1.75231344453326\\nN            0.42286232500769        0.22269640420026       -1.61024510157885\\nC            1.15204982776541        0.71074567240332       -0.55321382653324\\nC            2.02308892967206       -0.29669819148237       -0.07391287692303\\nN            1.79641966062802       -1.37657770721907       -0.87823409096671\\nC            2.87724676664876       -0.10153761945450        1.00164587047043\\nC            2.84117979655630        1.14384306894340        1.60653390378630\\nC            1.97806291056130        2.15305096656691        1.13381709046014\\nC            1.13992447695774        1.95898816683594        0.06400579654422\\nO            3.60606698034710        1.50561948050627        2.67371629168767\\nC            4.53190361849336        0.57618811228319        3.18147789795529\\nH           -0.93616885486529        1.58157651595659        2.37655463450709\\nH           -1.69972535636934        0.35843018347703        3.39059923751276\\nH           -2.70278226188390        1.44084561786646        2.40653098885554\\nH           -0.30789536713854       -1.42213342704756        2.24524175999535\\nH           -1.93527732722662        2.76630378292214        0.27912030702489\\nH           -3.37916129646493        3.07563875501184       -0.71999474289654\\nH           -2.01398836271242        2.12980529736546       -1.38266064535948\\nH           -3.66020016355218       -1.51434541437820       -2.45466346390101\\nH           -2.40949352125327       -0.38911032156486       -3.00058100911824\\nH           -3.79873152667243        0.20338167669665       -2.07252621526702\\nH           -0.79674565415690       -3.80083862609030       -1.42976190307694\\nH           -2.02156493006416       -3.03957824295762       -2.49182465359309\\nH            2.20102412670334       -2.29904218266697       -0.84632048027800\\nH            3.52507767535457       -0.89536855996038        1.33532413720701\\nH            2.00667427705873        3.10029755204596        1.64914966375647\\nH            0.49594185477777        2.74473540439028       -0.29534116147431\\nH            5.26611950128992        0.28620180924109        2.42284441684680\\nH            4.03359994206150       -0.31948415898030        3.56654316471692\\nH            5.04118986138270        1.08349019371214        3.99917779102703\\n43\\n energy: -70.320118863999 xtb: 6.7.0 (08769fc)\\nC           -1.65931919347164        0.80679229078010        2.45482932614324\\nC           -1.64230523451896       -0.13679649194534        1.29489612540313\\nC           -0.85152919989501       -1.27751565707838        1.30405950696044\\nN           -0.76692750155183       -2.12292060119769        0.29844560896772\\nC           -1.49977617888600       -1.91103967057334       -0.79035949225384\\nC           -2.34742471285035       -0.80750383952844       -0.92376585699315\\nC           -2.40133202740334        0.09080339065693        0.14457076488472\\nO           -3.22373584860789        1.19743975151880        0.05773094418217\\nC           -2.56282479278700        2.35986986230371       -0.40468237877951\\nC           -3.14509804108340       -0.56034707997455       -2.16583704841157\\nC           -1.23638327044020       -2.84027186974379       -1.91564588484130\\nS            0.30665826542997       -2.33993683769839       -2.90036860875658\\nO            1.27056981029469       -3.42204268715038       -2.67811967196497\\nC            0.86822602715225       -1.07462599587301       -1.75477571851560\\nN            0.47450084139014        0.16222844431236       -1.67432425755791\\nC            1.17532192905811        0.68516775567963       -0.61467511055599\\nC            2.02089570281817       -0.31034856703099       -0.06862826394632\\nN            1.81106417177199       -1.41782485632275       -0.83853275659307\\nC            2.84100524236058       -0.08079536651177        1.02660033287513\\nC            2.79732800642985        1.18730573728164        1.58160913692583\\nC            1.96107931856162        2.18510854249570        1.04121007801713\\nC            1.15664904731494        1.95692645684597       -0.04771063512554\\nO            3.52904822242388        1.58309489327127        2.66017883429193\\nC            4.42860806502204        0.66607945762712        3.23342253703817\\nH           -0.81835363683510        1.49839651846802        2.38440934941367\\nH           -1.56749144169926        0.25932151203951        3.39022726316503\\nH           -2.58485982953310        1.37762715674161        2.46266303314594\\nH           -0.23582459235011       -1.51174927665667        2.16320148046129\\nH           -1.84858182769679        2.73178401450993        0.33648726965167\\nH           -3.34079452875280        3.10460306008498       -0.56619929916757\\nH           -2.03282988539956        2.16493270191723       -1.34202990269161\\nH           -3.72414093644212       -1.44134528940855       -2.43772959669564\\nH           -2.48125587576147       -0.31065127118642       -2.99302894650103\\nH           -3.83262150729519        0.26475804937600       -2.00096869914870\\nH           -0.98243363144765       -3.83350380800183       -1.55016507588609\\nH           -2.04387679191924       -2.88645422127774       -2.63878277735998\\nH            2.19273610638405       -2.34699739329896       -0.75326459528482\\nH            3.46975256163512       -0.86665428010955        1.41171202348200\\nH            1.98294835201198        3.15179810195643        1.51950177350537\\nH            0.53396212267694        2.73426668042153       -0.45909640695338\\nH            5.18438193809044        0.34085734447682        2.51117408524487\\nH            3.90924220163274       -0.20988676190025        3.63566460514224\\nH            4.91574355413630        1.20004909966636        4.04762390508126\\n43\\n energy: -70.319317854648 xtb: 6.7.0 (08769fc)\\nC           -1.53559542620822        0.70481843960488        2.48398070024103\\nC           -1.57116571733606       -0.20351374566347        1.29665935513072\\nC           -0.80780762485229       -1.36181988359970        1.25066502254198\\nN           -0.76371341375720       -2.17317806272872        0.21536915579724\\nC           -1.51252903155847       -1.90954561817235       -0.85083514687953\\nC           -2.34131157625416       -0.78592577288230       -0.92628069418018\\nC           -2.35354831694210        0.07538985076319        0.17383040405838\\nO           -3.16083750832641        1.19663812042772        0.14931022967058\\nC           -2.50199669533240        2.36319659883528       -0.30414104527003\\nC           -3.16872965713019       -0.47976144690931       -2.13592500644310\\nC           -1.28630744912393       -2.80226872328595       -2.01455705457932\\nS            0.38135702518004       -2.47820640555206       -2.85774922482727\\nO            1.25266814628614       -3.58849564329843       -2.46266486251176\\nC            0.91454537919378       -1.15796586760601       -1.76162542362050\\nN            0.53374522332904        0.08545577720357       -1.75643271773277\\nC            1.20095043947694        0.65167622933504       -0.69719912178406\\nC            2.01201865880946       -0.32561092062349       -0.07116681279716\\nN            1.81984069066462       -1.46583064433938       -0.79641619104799\\nC            2.79014539023909       -0.05242771780486        1.04463986846112\\nC            2.73894225446520        1.24081504705987        1.53751917851309\\nC            1.93803529127292        2.22057696639276        0.91657668544388\\nC            1.17574645322628        1.94929876482504       -0.19272304077834\\nO            3.42963426874826        1.67972928606284        2.62670394746371\\nC            4.29149872981978        0.78136193439162        3.28149109436621\\nH           -0.67906301960169        1.37707374457134        2.41189094365201\\nH           -1.43494681802773        0.12760149784836        3.40040964711391\\nH           -2.44529961575398        1.29854472878726        2.53201422432813\\nH           -0.17973769004243       -1.63930763626830        2.08776264245048\\nH           -1.73368883683621        2.68710066863875        0.40483030302586\\nH           -3.27071588161480        3.13029674922149       -0.38513726490808\\nH           -2.03652630798777        2.19860025862173       -1.28086338158986\\nH           -3.76871116815713       -1.34095894693824       -2.42504628273812\\nH           -2.52403633897823       -0.20606129930635       -2.97074520054510\\nH           -3.83924425074785        0.34769736513328       -1.92018524445517\\nH           -1.19513752865146       -3.84095119644872       -1.70135146242006\\nH           -2.03369183470768       -2.69517249977911       -2.79284675404361\\nH            2.17141528101732       -2.39863232313686       -0.64436274710795\\nH            3.39417329928924       -0.82508097602259        1.49119918863943\\nH            1.95255179674829        3.20876472990949        1.34904957507304\\nH            0.58013475437570        2.71279083622990       -0.66546785416778\\nH            5.07734761467026        0.41857181158463        2.61106011428233\\nH            3.74473811982012       -0.07183766314563        3.69614594855586\\nH            4.74485389126933        1.34655258803393        4.09414130561809\\n43\\n energy: -70.318356883790 xtb: 6.7.0 (08769fc)\\nC           -1.43273022521919        0.58590632915122        2.52725435492729\\nC           -1.51047283413997       -0.27956642623450        1.31028777020703\\nC           -0.77792473608082       -1.45389893524048        1.20956652002038\\nN           -0.76326549755968       -2.22128545334081        0.14058002685167\\nC           -1.51614994344076       -1.89904274549672       -0.90538072552033\\nC           -2.32730064049468       -0.75981395886606       -0.92121417885508\\nC           -2.30402780681179        0.05786708899350        0.21167735956140\\nO           -3.08821406886845        1.19554440580904        0.24767051344758\\nC           -2.42179375655013        2.36136935319910       -0.19536504311034\\nC           -3.17844792988147       -0.39336656273077       -2.09764843907071\\nC           -1.31680597785780       -2.74223796081505       -2.11308162780304\\nS            0.44348803662126       -2.63117832471656       -2.79671787518137\\nO            1.19346891561100       -3.75118632032442       -2.22389449370490\\nC            0.96765983426621       -1.24969540037458       -1.77286856134948\\nN            0.60545967336792       -0.00333560839705       -1.84977951641981\\nC            1.23593827860687        0.61011388786817       -0.79405049773087\\nC            2.00466096159969       -0.34286680593456       -0.08270977576534\\nN            1.82954940856141       -1.51667877223476       -0.75703243056473\\nC            2.73413106604767       -0.02148186982169        1.05293501326853\\nC            2.67595756354563        1.29539341842649        1.47790338309310\\nC            1.91705413671412        2.25072372218937        0.77214958199499\\nC            1.20372648466026        1.93165474903458       -0.35699990024501\\nO            3.31888944433918        1.78096641738805        2.57689558069383\\nC            4.13900112244006        0.90818813668504        3.31458822200638\\nH           -0.56304113468055        1.24182176339589        2.46136229401491\\nH           -1.32866984207460       -0.02439345802348        3.42158843073265\\nH           -2.32732750482084        1.19809269966956        2.61216936557107\\nH           -0.14827041859925       -1.78121121386954        2.02727714823423\\nH           -1.59728733249507        2.62749825543060        0.47350849159229\\nH           -3.16821355694082        3.15420875011908       -0.19001517928509\\nH           -2.02736631342385        2.23005209554461       -1.20783775646334\\nH           -3.78129676393870       -1.24044967310793       -2.41998490277589\\nH           -2.55141366215648       -0.07225974623786       -2.92921587407218\\nH           -3.84678614747732        0.41919768885358       -1.82521982066153\\nH           -1.41281025381206       -3.80158762399914       -1.87868120018666\\nH           -1.96570221107917       -2.46832350896015       -2.93717027328972\\nH            2.14170448235082       -2.44898315685013       -0.53182553591158\\nH            3.30675486691471       -0.77677648606764        1.56567743005073\\nH            1.92316557080810        3.25910628493779        1.15539861639321\\nH            0.63949106732644        2.67622070930562       -0.89408779087452\\nH            4.95114508523069        0.50579269111295        2.70049916592689\\nH            3.56387113550789        0.08133380974377        3.74406320175412\\nH            4.56020242385706        1.50856675475604        4.11925592849738\\n43\\n energy: -70.317235461154 xtb: 6.7.0 (08769fc)\\nC           -1.38016588892747        0.49207525117178        2.56779680534042\\nC           -1.47728958572163       -0.33597286280904        1.32640893835198\\nC           -0.75998510895296       -1.51465571540283        1.18190239567780\\nN           -0.75491282230722       -2.24046869211577        0.08392748195353\\nC           -1.51109254948465       -1.87775710984216       -0.94445104769164\\nC           -2.32010083504035       -0.73701523919100       -0.91145125506971\\nC           -2.27553579814438        0.04496684538776        0.24542958550891\\nO           -3.03985973707831        1.19433984511140        0.32471570893701\\nC           -2.36236135741837        2.35500597670756       -0.11467581122968\\nC           -3.19280867356081       -0.33351042763717       -2.05980510335089\\nC           -1.31902020301915       -2.66694183392311       -2.19275760810119\\nS            0.48466111787720       -2.76031913570910       -2.73114095881492\\nO            1.11253417265635       -3.87393145515576       -2.01889090329894\\nC            1.02269028307533       -1.33228847296848       -1.77859718134920\\nN            0.67583515122808       -0.08753050278008       -1.92071863998710\\nC            1.27669855518175        0.56526555357402       -0.87105355575913\\nC            2.01168940837699       -0.36302262666395       -0.09444356260768\\nN            1.85060571372856       -1.56257102660844       -0.72606706364427\\nC            2.70026966434068       -0.00085658598502        1.05422150117708\\nC            2.63331935318276        1.33172739879599        1.42579186705814\\nC            1.90628083227369        2.26204654057285        0.65630549271698\\nC            1.23446966887439        1.90250765033407       -0.48627849260609\\nO            3.23576484862586        1.85575690934760        2.53046343199082\\nC            4.02526050369982        1.00843588355190        3.32869629001364\\nH           -0.50927611739540        1.14719362241970        2.51051553407823\\nH           -1.26764411298332       -0.14492299592800        3.44222202516261\\nH           -2.27079737301701        1.10544843658050        2.68131990543533\\nH           -0.13069306976250       -1.87854961456571        1.98427348485902\\nH           -1.49064596600279        2.56903069884380        0.51182860199029\\nH           -3.07998139287602        3.17003007855648       -0.03587384583068\\nH           -2.03257561234417        2.25109230332894       -1.15324867213727\\nH           -3.77869105781775       -1.17856677282283       -2.41632846456215\\nH           -2.58471967275767        0.04316218613156       -2.88235020308484\\nH           -3.87576445873459        0.44946858120507       -1.74024619943724\\nH           -1.58792703855170       -3.71380872888940       -2.05311199945113\\nH           -1.84697530318562       -2.24570578925796       -3.04098487989469\\nH            2.12835952148418       -2.48982904499134       -0.44217176599089\\nH            3.24736530263748       -0.73856800027006        1.61788433248636\\nH            1.90161000692089        3.28404089230314        1.00164961899530\\nH            0.69251990915699        2.62753574154523       -1.07104000251664\\nH            4.85380857923766        0.57683755739985        2.75792592956066\\nH            3.43044854604678        0.20287576040782        3.77179291028539\\nH            4.42463359645089        1.63794792020938        4.12214237483285\\n43\\n energy: -70.315881950946 xtb: 6.7.0 (08769fc)\\nC           -1.34978342170285        0.45564077671974        2.58728649826641\\nC           -1.45770302639252       -0.35017923146383        1.33214532430686\\nC           -0.72890986802821       -1.51683085042344        1.15422604509332\\nN           -0.72538440935464       -2.21559063780563        0.03880390966279\\nC           -1.49695567605365       -1.84169355618548       -0.97223739047792\\nC           -2.32494799387224       -0.71656054693663       -0.90235124022109\\nC           -2.27470288898276        0.04235474139715        0.26951057292778\\nO           -3.04787806469025        1.18340026762054        0.38146393818633\\nC           -2.38954058757866        2.35515230027692       -0.05708527143712\\nC           -3.21958243476232       -0.30477569617935       -2.03067623824006\\nC           -1.30027582198728       -2.58606896792050       -2.25033347511482\\nS            0.50891433548947       -2.86280014616220       -2.67473295464859\\nO            1.02486647089863       -3.97135211826513       -1.87212504741200\\nC            1.08438054720517       -1.41247787263164       -1.77674088524125\\nN            0.74498159102567       -0.17141342599309       -1.96154514454954\\nC            1.32764008997006        0.51026818149611       -0.92027412175728\\nC            2.04604860617534       -0.39669715344787       -0.10407881250627\\nN            1.89616160646946       -1.61377879799831       -0.70566315885496\\nC            2.70602149407448       -0.00373936713245        1.05118701742136\\nC            2.62440949345939        1.33709621047110        1.38913487941221\\nC            1.91275403545388        2.24568538242925        0.58064480201061\\nC            1.26990900523740        1.85580069006846       -0.56871235009107\\nO            3.19427587162440        1.88941380122422        2.49774191887659\\nC            3.95684321969710        1.06246299531734        3.34205134176664\\nH           -0.47724071565578        1.10897814030708        2.53519536935507\\nH           -1.23282517271108       -0.19717237227633        3.44944145970116\\nH           -2.23763507889879        1.06938970865382        2.71826865005903\\nH           -0.08601251421241       -1.89098410716740        1.94083942437417\\nH           -1.49486292327355        2.55773420892671        0.54036784826417\\nH           -3.10509495783845        3.16634654181746        0.06659049502531\\nH           -2.09883673169663        2.27607507523875       -1.10945276066725\\nH           -3.77544195730507       -1.15795481838363       -2.41420735623838\\nH           -2.63149289218277        0.12191860521391       -2.84363911099193\\nH           -3.92740697810208        0.44239689843108       -1.68057337782177\\nH           -1.71244179233614       -3.59445204582928       -2.20168403074309\\nH           -1.70540543216545       -2.05208229356827       -3.10356063748433\\nH            2.15311465504869       -2.53282749832938       -0.37930297550130\\nH            3.24068011065357       -0.72539079981937        1.64676409188998\\nH            1.89395224582240        3.27521477430026        0.90229909082878\\nH            0.73471697355546        2.56292950941521       -1.18108223154038\\nH            4.80753079105412        0.62262698762208        2.81151425314655\\nH            3.34895243730609        0.26374754431498        3.77979929570413\\nH            4.32420875953523        1.71018796262567        4.13630934525752\\n43\\n energy: -70.314364132069 xtb: 6.7.0 (08769fc)\\nC           -1.35095456264166        0.43585544074789        2.60258371539695\\nC           -1.45788354031721       -0.35431826370464        1.33738533427271\\nC           -0.70922206973312       -1.50402557036545        1.13527820155222\\nN           -0.69465421188560       -2.17842032944499        0.00497876264099\\nC           -1.48396391712076       -1.80800747784911       -0.99152199131042\\nC           -2.33982946088383       -0.70592107326555       -0.89335680107725\\nC           -2.28986384246413        0.03967426553320        0.28676204845280\\nO           -3.07101530294985        1.17370360707391        0.41512303909923\\nC           -2.42038701085253        2.35307799490424       -0.01471709142031\\nC           -3.25560630504180       -0.30280124713386       -2.00722217348923\\nC           -1.27167002595695       -2.50362988053442       -2.29710314144937\\nS            0.52059620358350       -2.93709416283150       -2.62860799806274\\nO            0.93681739693089       -4.04063530892886       -1.76482469939825\\nC            1.13814258198637       -1.47797175565846       -1.77040264841412\\nN            0.80665524765094       -0.23991455862057       -1.98712592240862\\nC            1.37795989283026        0.46311270325068       -0.95407569208890\\nC            2.08416665242080       -0.42743999501056       -0.10990227002208\\nN            1.94061687789835       -1.65737769665218       -0.68811348092347\\nC            2.72389301236634       -0.01114311099787        1.04853650141769\\nC            2.62994409554295        1.33495917636268        1.36210455674508\\nC            1.92755798601155        2.22636237278527        0.52709483928947\\nC            1.30710734026387        1.81393094893738       -0.62668388298529\\nO            3.17603014937156        1.90871341662768        2.47213117368504\\nC            3.92253727348841        1.09936809530586        3.34704153913036\\nH           -0.48113909238077        1.09330683665798        2.55716416609444\\nH           -1.22928968904499       -0.22748990913419        3.45606654717680\\nH           -2.24110300844595        1.04400402153883        2.74353047284613\\nH           -0.05461228607724       -1.87998744780535        1.91127416022325\\nH           -1.51551825454629        2.54666831150834        0.57040289067123\\nH           -3.13396709225528        3.16173878471901        0.13401053515263\\nH           -2.14708549969167        2.29197899049780       -1.07309787731085\\nH           -3.78509597053620       -1.16770886045837       -2.40160972388626\\nH           -2.68869169730597        0.15506292653392       -2.81853758247778\\nH           -3.98337036983420        0.41644849944888       -1.63979102159495\\nH           -1.79866238280949       -3.45821416308530       -2.34397458313497\\nH           -1.55637269106793       -1.87677165287144       -3.13705592081427\\nH            2.18171669131107       -2.56834628393388       -0.32954956583606\\nH            3.25029296133034       -0.71965567679119        1.66691455070835\\nH            1.89606990493551        3.26044426196526        0.83276838685383\\nH            0.77578521391182        2.50671749912983       -1.25856223788139\\nH            4.78701693266176        0.65469765376008        2.84349897831864\\nH            3.30817878336851        0.30519399141892        3.78412591825859\\nH            4.26887408595044        1.76185362633799        4.13858698799713\\n43\\n energy: -70.312225218563 xtb: 6.7.0 (08769fc)\\nC           -1.66841628788010        0.40509142943454        2.63941116945824\\nC           -1.65055067031361       -0.33270412270705        1.33818078113474\\nC           -0.77710347705634       -1.38713218876007        1.11762099390429\\nN           -0.67405796627457       -2.02237449365934       -0.03060360483486\\nC           -1.47725467281959       -1.70453380304141       -1.03185905506450\\nC           -2.43644708975059       -0.69309264522560       -0.92673401124201\\nC           -2.49039862889157        0.01397990769672        0.27690337110390\\nO           -3.38155067639986        1.06319470385436        0.41611605762567\\nC           -2.82219212182919        2.31561337143590        0.07401944984616\\nC           -3.34923320625571       -0.34540529859846       -2.06074704683516\\nC           -1.19796552924153       -2.36198327686122       -2.34643976252808\\nS            0.55344192670552       -2.98110803508855       -2.52368363967850\\nO            0.80481457457001       -4.08200391140622       -1.59588761521653\\nC            1.23839302027590       -1.53876396188303       -1.68765594073463\\nN            0.98856963253200       -0.29358991422818       -1.96422023481237\\nC            1.60093876415458        0.41972026142246       -0.96222222156334\\nC            2.24904893417231       -0.47296394768035       -0.07503499252209\\nN            2.03017821783348       -1.71797725457988       -0.59669253275010\\nC            2.91530752459866       -0.04255444661599        1.06309339522820\\nC            2.91084925182100        1.32109536261267        1.30948147172904\\nC            2.26756402745581        2.21436546321541        0.43072572700559\\nC            1.61737188177650        1.78650534335970       -0.70096585077458\\nO            3.49579631571587        1.91209511869811        2.39080845562921\\nC            4.18005086253888        1.09864108720373        3.31116584539737\\nH           -0.85830993383514        1.13557553559963        2.66692700413553\\nH           -1.52638706831329       -0.28594589811712        3.46752852577715\\nH           -2.61451088000038        0.92599312651302        2.76526565584013\\nH           -0.10105863634582       -1.71637769395846        1.89630110641487\\nH           -1.95440327400414        2.55186386878315        0.69868857925235\\nH           -3.60540529236244        3.05304730565691        0.24224003948501\\nH           -2.51380118317057        2.33866406472967       -0.97674522068057\\nH           -3.79471196757870       -1.24372075391635       -2.48344115783489\\nH           -2.79638467162303        0.16491987833941       -2.85028839431058\\nH           -4.14236223131232        0.30898020351163       -1.70789193169472\\nH           -1.83648131767617       -3.23191542768062       -2.51069430209440\\nH           -1.31385826577125       -1.65920927268251       -3.16833992863072\\nH            2.21107949383377       -2.62239895037198       -0.19074509159712\\nH            3.39535804733448       -0.75252278514597        1.71679580917084\\nH            2.30440652674109        3.26217790626852        0.68441683870046\\nH            1.12799886240788        2.47987795895739       -1.36535232841297\\nH            5.01446965056979        0.57056866871771        2.83815533127055\\nH            3.51000425524261        0.37151868434702        3.78215151669035\\nH            4.56720427839861        1.77478783181777        4.07177473901108\\n43\\n energy: -70.311884242715 xtb: 6.7.0 (08769fc)\\nC           -2.42280316329622        0.37798306420134        2.65142467860771\\nC           -2.12729182185433       -0.23143406084497        1.31703199720275\\nC           -0.99941162752207       -1.01282791589187        1.11470814258418\\nN           -0.69249957132288       -1.57008497591366       -0.03873943935939\\nC           -1.49828528066807       -1.41563110122274       -1.07313325644185\\nC           -2.67485514675623       -0.66347189981568       -1.00320393836971\\nC           -2.97059022506440       -0.05053824040381        0.21634434387933\\nO           -4.11878158079316        0.71156003581599        0.33559310079382\\nC           -3.90212420008421        2.09806213101868        0.16869194272544\\nC           -3.57607971818674       -0.50604019329979       -2.18679756210282\\nC           -1.07451750398262       -2.04375446070024       -2.36184262863173\\nS            0.58972162719695       -2.86673564601300       -2.30260152776377\\nO            0.58234790352612       -3.93248978673673       -1.29980270773757\\nC            1.39261923863127       -1.48307150958844       -1.48256929139416\\nN            1.40732026364561       -0.24772081683906       -1.88542640838383\\nC            2.15501713471099        0.41998655896409       -0.94463309367973\\nC            2.60723749646659       -0.49433482282740        0.03677689479873\\nN            2.13019294626442       -1.71215263969476       -0.36243936126714\\nC            3.37290676669070       -0.10424249037400        1.12582537006568\\nC            3.68134712305558        1.24346895234439        1.21987628971300\\nC            3.23663113427329        2.15797340561797        0.24596646544543\\nC            2.48090752406617        1.76815155477636       -0.83294550109142\\nO            4.41448617195949        1.79926467089420        2.22791604261746\\nC            4.90565978768215        0.95863128878644        3.24158719630954\\nH           -1.84935643844793        1.29596171233874        2.78636470228092\\nH           -2.14579753107647       -0.30945107758307        3.44766284245384\\nH           -3.48182387620118        0.61074329177965        2.73342299545845\\nH           -0.30106065362111       -1.19341972106450        1.92165065419658\\nH           -3.19540141502650        2.48907027399948        0.90711498094148\\nH           -4.87173280049059        2.57346244336916        0.30584082871456\\nH           -3.52259945993625        2.32170672183361       -0.83439629600713\\nH           -3.84939217113473       -1.47983346882294       -2.58970202557414\\nH           -3.07445591307817        0.05481096126699       -2.97560620388359\\nH           -4.48274491407427        0.01816883027419       -1.89686737658416\\nH           -1.78429514494171       -2.80726240731919       -2.68453084456001\\nH           -0.96995314655366       -1.29194523295028       -3.14394585609884\\nH            2.13791657664195       -2.59471768748678        0.12289407342378\\nH            3.69808915839735       -0.82990722038131        1.85368973785708\\nH            3.51523278952562        3.19100273523131        0.38143735647824\\nH            2.14674759791630        2.47773077347493       -1.57200661494801\\nH            5.58416568809992        0.19832204570733        2.84087902082349\\nH            4.09250650556114        0.46817178688725        3.78729838561057\\nH            5.45480086978299        1.60683313717614        3.92271889089905\\n43\\n energy: -70.312026358464 xtb: 6.7.0 (08769fc)\\nC           -2.58165584341317        0.44186587728637        2.63378422667329\\nC           -2.23343095068412       -0.13118769437784        1.29591285025085\\nC           -1.01380209507236       -0.75263353445805        1.07286344971377\\nN           -0.66042408648288       -1.27782120883543       -0.08235474246710\\nC           -1.50009007468111       -1.24275588732455       -1.10031923450837\\nC           -2.76894391514313       -0.66102244253412       -1.00721672248419\\nC           -3.11948341044021       -0.08241966283852        0.21464609261916\\nO           -4.36342197181779        0.50509482021878        0.35692612778693\\nC           -4.35651281146029        1.91008041720097        0.20672645170466\\nC           -3.71420108682877       -0.64870203024225       -2.16687958413278\\nC           -1.02573578949136       -1.83486666364694       -2.38769324508867\\nS            0.57781537840152       -2.76213589567743       -2.25419253994378\\nO            0.46205442155483       -3.78178909795966       -1.20953137847628\\nC            1.47052790804897       -1.41834696866718       -1.46385999449461\\nN            1.63505286650691       -0.21401824000538       -1.92298878386618\\nC            2.42171476539513        0.41420100198664       -0.98608310919658\\nC            2.73843689468395       -0.49436808200232        0.05226442697465\\nN            2.14355679007166       -1.67089413117693       -0.30862635569984\\nC            3.50536622586945       -0.13742395211063        1.15159393240821\\nC            3.95922687396526        1.17103241589645        1.19609499936008\\nC            3.65497831985748        2.07815535672708        0.16317967535411\\nC            2.89562728909776        1.72114309319309       -0.92475929138528\\nO            4.71319420303130        1.69218955943872        2.20768401695586\\nC            5.05142432668402        0.85904585421068        3.28779228270444\\nH           -2.16326455540688        1.44357600615539        2.74084900703895\\nH           -2.16967657467007       -0.17859837155505        3.42650456324105\\nH           -3.66142274604734        0.49923059053492        2.74999392773086\\nH           -0.27974657269563       -0.82867182795605        1.86466151775112\\nH           -3.69987379785779        2.39353852345603        0.93624190362785\\nH           -5.38248827635447        2.23531695593741        0.36978111262816\\nH           -4.03649422039787        2.19882894962350       -0.80067423994248\\nH           -3.91231767432532       -1.66495597004275       -2.50505880497644\\nH           -3.28902099660773       -0.09320314646402       -3.00243627298317\\nH           -4.65675259846076       -0.19415977657215       -1.87422869872141\\nH           -1.75451478604945       -2.53526859791137       -2.79689274104272\\nH           -0.81815695687514       -1.05565433362509       -3.12171916288043\\nH            2.04293686446227       -2.52715919154128        0.21215947648654\\nH            3.72243370510727       -0.85787109886663        1.92365214441015\\nH            4.04456219726836        3.07907932356806        0.26074608108341\\nH            2.66928263490916        2.42573609366929       -1.70814033907528\\nH            5.65672712578420        0.00549354354918        2.96488065088729\\nH            4.15993661059923        0.49480464495050        3.80965917434026\\nH            5.63657738994940        1.47751377877976        3.96658414962181\\n43\\n energy: -70.312328024143 xtb: 6.7.0 (08769fc)\\nC           -2.77601513317033        0.40976329989191        2.64109389655603\\nC           -2.36262875483341       -0.09022206501461        1.29244827565401\\nC           -1.08212560919278       -0.57090878304036        1.06206773854335\\nN           -0.67420086267114       -1.03968348033523       -0.09895981720027\\nC           -1.51280608509740       -1.08185449749702       -1.11890886650523\\nC           -2.83438431118969       -0.63308897091134       -1.02217804840493\\nC           -3.24562489412293       -0.11469992739109        0.20798652214659\\nO           -4.54516696365813        0.33503959334295        0.35561334471104\\nC           -4.68695213406793        1.73601571903788        0.24186164854617\\nC           -3.77661988144690       -0.69411840273326       -2.18353488713144\\nC           -0.97437230275242       -1.63175337473840       -2.39829862146976\\nS            0.53502041458174       -2.68982068567471       -2.15930648186464\\nO            0.29099629449823       -3.63485112005412       -1.06741707667679\\nC            1.51544299401210       -1.38624070638027       -1.40922312665491\\nN            1.80640313017550       -0.23493108941784       -1.93634731492518\\nC            2.62682701193764        0.37525422579365       -1.01714312491359\\nC            2.82904155383588       -0.48998052613913        0.08482406671844\\nN            2.12989902995750       -1.62467257489491       -0.21907452542359\\nC            3.60568244681912       -0.13929136941016        1.17941785352159\\nC            4.18843275960729        1.11758284782286        1.15257677052230\\nC            3.99678531188192        1.98118217724980        0.05716078156603\\nC            3.22708529897002        1.63070341145249       -1.02547962695771\\nO            4.97365382116509        1.62564052100834        2.14693752972348\\nC            5.21430460787448        0.83193691610763        3.28131569316932\\nH           -2.46647809979923        1.44688710760021        2.77675214096184\\nH           -2.30527043925503       -0.18400891362780        3.42149350886586\\nH           -3.85634311141584        0.34969844528721        2.75026917713384\\nH           -0.34495463790743       -0.58003657404819        1.85463276505368\\nH           -4.08776437674815        2.26750067621154        0.98727482695600\\nH           -5.74214469193512        1.94616330867129        0.40721066364773\\nH           -4.39659144948446        2.08355688597990       -0.75604697135625\\nH           -3.90184415700753       -1.72358832437958       -2.51684138809958\\nH           -3.39328823905897       -0.11128825564456       -3.02041732996158\\nH           -4.75043679798961       -0.31020535090644       -1.89241073026582\\nH           -1.71068244431606       -2.24578272632094       -2.91512794884126\\nH           -0.62871653863354       -0.83568038024104       -3.05812537585973\\nH            1.93986926982665       -2.43418643651878        0.34903403787169\\nH            3.73491637908934       -0.82572267141616        2.00054884547967\\nH            4.48272111232487        2.94284090299792        0.10173867718920\\nH            3.08787623748738        2.30130626302497       -1.85732410884177\\nH            5.73527770560796       -0.09550692751667        3.02137360970940\\nH            4.28499881189601        0.59053354186487        3.80828565729918\\nH            5.85017872416996        1.43051729090435        3.93177433979190\\n43\\n energy: -70.312714130899 xtb: 6.7.0 (08769fc)\\nC           -2.82750163866460        0.50608259405366        2.61734142176580\\nC           -2.40125009256833        0.02195447938857        1.26694975397809\\nC           -1.08102753894729       -0.31283969535490        1.00244600155823\\nN           -0.65859872315572       -0.75578683796452       -0.16273021297888\\nC           -1.51764792844188       -0.91541776424953       -1.15551610733436\\nC           -2.88026977862352       -0.62913078357388       -1.01943841736674\\nC           -3.30921288920398       -0.13457731512092        0.21518262284686\\nO           -4.64798268774473        0.15867109563050        0.40243646647855\\nC           -4.96486670430089        1.52658640238983        0.25080725770606\\nC           -3.85396229438252       -0.83366728858993       -2.13854337101335\\nC           -0.94457812059183       -1.43226865280293       -2.43192923371371\\nS            0.49344242313019       -2.57940173088813       -2.14119390961123\\nO            0.16663751362642       -3.47173730976022       -1.02558607319615\\nC            1.54438235631560       -1.32599181689772       -1.40372425861583\\nN            1.93783690986636       -0.21802752723132       -1.95643301021840\\nC            2.77650554671736        0.35750697207248       -1.03173339058608\\nC            2.88205282040001       -0.48508773591845        0.10092854725702\\nN            2.10445857033609       -1.57132181610202       -0.18841332158010\\nC            3.65751053344230       -0.16235255928641        1.20498708859887\\nC            4.33877415922788        1.04320261864982        1.15643378075988\\nC            4.23998252658630        1.88579573773055        0.03263239468929\\nC            3.47191223990849        1.56272702174244       -1.05956601544422\\nO            5.14060996143466        1.51713663815327        2.15429715931935\\nC            5.30195843912823        0.73596040966582        3.31120968787342\\nH           -2.63451509141313        1.57498742330531        2.71890125761840\\nH           -2.26812527336553       -0.00962153217998        3.39476752836570\\nH           -3.89010388787651        0.32662569415311        2.76384274745532\\nH           -0.32285960787629       -0.22028078171129        1.76952413106402\\nH           -4.41168586764573        2.15282326095969        0.95728862582929\\nH           -6.03187972813161        1.61228382869255        0.44835123319029\\nH           -4.75243765465141        1.87229102806227       -0.76717939097339\\nH           -3.85863427780361       -1.87709013473890       -2.45097585442679\\nH           -3.58834297366748       -0.22034669873036       -2.99889315149488\\nH           -4.85617118094804       -0.57485065524337       -1.80830698410661\\nH           -1.67593347678482       -1.98380007921468       -3.01848413531186\\nH           -0.51130653202912       -0.63081516670327       -3.03009902910219\\nH            1.84157884904770       -2.34818086302487        0.39613330290880\\nH            3.71213361864257       -0.83089262011579        2.04905238000462\\nH            4.79842879628710        2.80795440300690        0.06244903110905\\nH            3.40559890470569        2.21629776710082       -1.91380740539274\\nH            5.75214228550932       -0.23564319991165        3.08181915826846\\nH            4.34930929459903        0.58113564179143        3.82901000416848\\nH            5.97363919985238        1.29910654875583        3.95728868963282\\n43\\n energy: -70.313238721527 xtb: 6.7.0 (08769fc)\\nC           -2.86601457935967        0.60522761689388        2.59651975151356\\nC           -2.42895094035397        0.14354819015017        1.24180096914379\\nC           -1.08730981064931       -0.03620204031647        0.93545565123853\\nN           -0.65428431009519       -0.44860941775795       -0.23645968398327\\nC           -1.52199979753578       -0.73164271591999       -1.19585056149025\\nC           -2.90392091929353       -0.61674735342148       -1.01227901002660\\nC           -3.34614945236225       -0.14804208784210        0.22809934167290\\nO           -4.70343593008851       -0.01688198672063        0.46253480112777\\nC           -5.19422717151929        1.29353681980918        0.27340808766360\\nC           -3.88922253367597       -0.97673496049927       -2.08162086432526\\nC           -0.92322789731725       -1.21753886333191       -2.47045346466034\\nS            0.43597658898400       -2.45403969144096       -2.14038117371761\\nO            0.03144081474712       -3.29599226295049       -1.01017674889020\\nC            1.55568783648107       -1.26082948384408       -1.40731510555452\\nN            2.05694747846870       -0.21046695939350       -1.98446155964919\\nC            2.91018752558576        0.32944744452864       -1.05262355985291\\nC            2.90939461053384       -0.47489659489413        0.11252127271120\\nN            2.05150700135111       -1.50229640064525       -0.16366290648566\\nC            3.67863471535865       -0.17779835092358        1.22826726698564\\nC            4.46374128384150        0.96164214928559        1.15754422220203\\nC            4.46851417469008        1.76749746053956        0.00249204713668\\nC            3.70639807272576        1.47021952655715       -1.10081516586199\\nO            5.27909291270700        1.40060935760399        2.16015451063651\\nC            5.35237578358428        0.64526627681416        3.34295088289476\\nH           -2.80031270680471        1.69168872477569        2.66957670294964\\nH           -2.22387063198214        0.17933372637587        3.36431477233558\\nH           -3.89404517481438        0.30362366554550        2.78327556901448\\nH           -0.31963848191446        0.16281370984993        1.67284291898316\\nH           -4.70113680471841        2.01103425261264        0.93657267335432\\nH           -6.25671392152358        1.25449570036075        0.50700026400391\\nH           -5.05955587253780        1.62236333306013       -0.76332387438446\\nH           -3.75861871476575       -2.01530204872453       -2.38200153005836\\nH           -3.75835873130655       -0.34471028271917       -2.95951215894789\\nH           -4.90115319935400       -0.85868907716551       -1.70391348933865\\nH           -1.64774761349941       -1.70460495379559       -3.11699397055122\\nH           -0.41011962312673       -0.42001005551961       -3.00622958315296\\nH            1.71377802506901       -2.23750905953281        0.43622229935482\\nH            3.65210276504182       -0.81614955904504        2.09673264223793\\nH            5.10327469817618        2.63918636260450        0.01756353463861\\nH            3.71973828235914        2.09444623807248       -1.97898128871360\\nH            5.72084195223414       -0.36810892361104        3.15139735081101\\nH            4.38125747826329        0.58907998463974        3.84634943012152\\nH            6.05912381834797        1.16874158992195        3.98498573689257\\n43\\n energy: -70.313984803447 xtb: 6.7.0 (08769fc)\\nC           -2.82458690163836        0.74664675574345        2.56944543140153\\nC           -2.39941762312272        0.29810572461804        1.20656840161750\\nC           -1.06313544969161        0.28568465163446        0.83007416894524\\nN           -0.64059092136372       -0.10345247598539       -0.35349494754509\\nC           -1.51411528009518       -0.52780292915190       -1.25580409049432\\nC           -2.88630021337848       -0.59941304554369       -0.99392110989694\\nC           -3.32108179750041       -0.14986836660538        0.25616059365897\\nO           -4.66874807750815       -0.19451251481919        0.56760955837788\\nC           -5.34804729068750        1.02244827694044        0.34195636345262\\nC           -3.86858207346935       -1.14170209203231       -1.98666318550689\\nC           -0.91736141695357       -0.99095843670578       -2.53753304890618\\nS            0.36493093568766       -2.31268769066262       -2.20196264929622\\nO           -0.10672461386526       -3.12192006983362       -1.07258017105802\\nC            1.54720187631718       -1.19320721889769       -1.45401845168382\\nN            2.16494217054867       -0.21326136537538       -2.04177931528130\\nC            3.01786477590199        0.28712823931983       -1.08906081333510\\nC            2.89211452323388       -0.46706382863532        0.10273358831293\\nN            1.95856703993482       -1.42463790282830       -0.17792740213415\\nC            3.63478355305919       -0.19516336759909        1.24290735655865\\nC            4.52329839022494        0.86524231565501        1.16846187936951\\nC            4.65272567848856        1.62087739671504       -0.01340348101308\\nC            3.91611604953210        1.34964335578763       -1.14006366082738\\nO            5.33127854194055        1.26874607433412        2.19129703743210\\nC            5.28572070283924        0.55708355028554        3.40232648776666\\nH           -2.90364269633961        1.83415909211409        2.60573004204338\\nH           -2.09220004990918        0.43973726831386        3.31303443056258\\nH           -3.79164307422525        0.31746887226023        2.82160197521245\\nH           -0.29069214638220        0.60466595198227        1.51863112288891\\nH           -4.93096167390057        1.83628506526275        0.94369753685054\\nH           -6.38266872841637        0.84950620207722        0.63306233279527\\nH           -5.30884510784977        1.31039069802594       -0.71481774422399\\nH           -3.59343927638945       -2.15606519294735       -2.27138100978174\\nH           -3.89799565905293       -0.52440271632518       -2.88460239736293\\nH           -4.86021105067135       -1.17132041126048       -1.54289720370880\\nH           -1.64593517822497       -1.41720339016162       -3.21984563992115\\nH           -0.34411160086636       -0.20514086948412       -3.02582968761681\\nH            1.53929217081165       -2.11243194510604        0.42699060580073\\nH            3.51172195861683       -0.79312481663040        2.13162166352331\\nH            5.36396411249464        2.43145891863043        0.00077317274751\\nH            4.02542203433906        1.93394803165696       -2.03878912991878\\nH            5.56678122138315       -0.49224784879611        3.26349158614079\\nH            4.29362487753718        0.60905800431587        3.86348494894568\\nH            6.01068828856508        1.03930304969650        4.05624185508910\\n43\\n energy: -70.315116075923 xtb: 6.7.0 (08769fc)\\nC           -2.64802044201848        0.93170096056360        2.52962189835666\\nC           -2.27205764301831        0.48549029165145        1.15163266498887\\nC           -0.98316427461906        0.64622503967923        0.66045569317645\\nN           -0.60666003929409        0.27139473779624       -0.54372283878312\\nC           -1.48393799178493       -0.30784670080953       -1.35263134233202\\nC           -2.80169549541912       -0.57373337401973       -0.96476504325054\\nC           -3.19317449926499       -0.13533724556809        0.30292928469507\\nO           -4.48698290018787       -0.36283398345797        0.73868018440001\\nC           -5.36664574760196        0.71472419954235        0.49842060459545\\nC           -3.76068565027466       -1.31896226004599       -1.84104479569765\\nC           -0.93290678540048       -0.75921258746651       -2.65661519975306\\nS            0.27464130096943       -2.16317695037444       -2.35769045628454\\nO           -0.26156316451936       -2.96718354912282       -1.25195930969402\\nC            1.50744908989183       -1.13515540762400       -1.56447918930014\\nN            2.25370398554198       -0.24037181245244       -2.13802859135162\\nC            3.08001760847912        0.21940215090205       -1.14342934523270\\nC            2.79922258980407       -0.46831915554050        0.06210121079571\\nN            1.80061744566279       -1.34609399814501       -0.25236969990655\\nC            3.47977176253879       -0.21507205037151        1.24473099699681\\nC            4.46808160699242        0.75449219911398        1.19807407611978\\nC            4.75511545123848        1.44060301622954        0.00128415964642\\nC            4.07836434867464        1.18994582115800       -1.16650927335919\\nO            5.23375715059687        1.12823797573165        2.26342173295577\\nC            5.02173406315536        0.49143508486568        3.49810307914753\\nH           -2.90014599841547        1.99314381517720        2.53062765456509\\nH           -1.81615828037052        0.78473738299985        3.21533552447621\\nH           -3.50891944329789        0.36894465511865        2.88390736132270\\nH           -0.20983402511828        1.09729864801540        1.26939894809275\\nH           -5.04239376778926        1.62324551724239        1.01636919772963\\nH           -6.33680706685236        0.40490383979626        0.88288198721619\\nH           -5.44997962556549        0.93072487687624       -0.57267734869191\\nH           -3.33659636647791       -2.28295443759118       -2.11704252139499\\nH           -3.97811170553079       -0.75929332496558       -2.75116959224314\\nH           -4.68840282376726       -1.49592779417764       -1.30321763856778\\nH           -1.68896895778337       -1.12879840768630       -3.34150015758983\\nH           -0.32122427631517        0.00822224474016       -3.12524080742151\\nH            1.29065469335614       -1.98190071334062        0.33991988426347\\nH            3.23607482271700       -0.75864497210242        2.14336116610998\\nH            5.53876989789793        2.18058076907163        0.03903190150979\\nH            4.30704749717229        1.72077088592023       -2.07583556099444\\nH            5.20714552989856       -0.58583765714319        3.43291909784414\\nH            4.00619317322484        0.66297396007840        3.87052818982573\\nH            5.73667595282819        0.93745730971734        4.18771921300251\\n\",\"xyz\");\n\tviewer_17200165783103456.setStyle({\"stick\": {\"colorscheme\": \"greenCarbon\"}});\n\tviewer_17200165783103456.zoomTo();\n\tviewer_17200165783103456.animate({\"loop\": \"backAndForth\"});\nviewer_17200165783103456.render();\n});\n</script>",
            "text/html": [
              "<div id=\"3dmolviewer_17200165783103456\"  style=\"position: relative; width: 800px; height: 600px;\">\n",
              "        <p id=\"3dmolwarning_17200165783103456\" style=\"background-color:#ffcccc;color:black\">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br></p>\n",
              "        </div>\n",
              "<script>\n",
              "\n",
              "var loadScriptAsync = function(uri){\n",
              "  return new Promise((resolve, reject) => {\n",
              "    //this is to ignore the existence of requirejs amd\n",
              "    var savedexports, savedmodule;\n",
              "    if (typeof exports !== 'undefined') savedexports = exports;\n",
              "    else exports = {}\n",
              "    if (typeof module !== 'undefined') savedmodule = module;\n",
              "    else module = {}\n",
              "\n",
              "    var tag = document.createElement('script');\n",
              "    tag.src = uri;\n",
              "    tag.async = true;\n",
              "    tag.onload = () => {\n",
              "        exports = savedexports;\n",
              "        module = savedmodule;\n",
              "        resolve();\n",
              "    };\n",
              "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
              "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
              "});\n",
              "};\n",
              "\n",
              "if(typeof $3Dmolpromise === 'undefined') {\n",
              "$3Dmolpromise = null;\n",
              "  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.1.0/3Dmol-min.js');\n",
              "}\n",
              "\n",
              "var viewer_17200165783103456 = null;\n",
              "var warn = document.getElementById(\"3dmolwarning_17200165783103456\");\n",
              "if(warn) {\n",
              "    warn.parentNode.removeChild(warn);\n",
              "}\n",
              "$3Dmolpromise.then(function() {\n",
              "viewer_17200165783103456 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_17200165783103456\"),{backgroundColor:\"white\"});\n",
              "viewer_17200165783103456.zoomTo();\n",
              "\tviewer_17200165783103456.addModelsAsFrames(\"43\\n energy: -70.311460971468 xtb: 6.7.0 (08769fc)\\nC           -5.38940635858081       -0.09281392942543       -1.62157543131579\\nC           -4.23943344221169       -0.49267316071980       -0.75148796161917\\nC           -3.88587706124477       -1.82525993340304       -0.58335295116622\\nN           -2.88863105207705       -2.22874358683721        0.17540199093572\\nC           -2.14358453923092       -1.32802376317536        0.80479018533147\\nC           -2.36934543736326        0.04679680397360        0.69717841326816\\nC           -3.45803698898169        0.45218497188428       -0.08106788124485\\nO           -3.73721482003351        1.80135030680773       -0.21722423061408\\nC           -4.71800844313505        2.27377940313145        0.68133015047824\\nC           -1.48511903781236        1.05954487643970        1.35404555483722\\nC           -1.04661320303918       -1.92036312466057        1.61977086802876\\nS            0.04347547479708       -2.89239201185228        0.45363764983982\\nO            0.84730353406379       -3.80237799296632        1.28407342981580\\nC            1.21787865776442       -1.55054516607273        0.17254059209574\\nN            0.99205971766160       -0.33521598608057       -0.22836249395691\\nC            2.22875675857793        0.25927183742353       -0.29230304529166\\nC            3.22232490937665       -0.67368455099243        0.08857224226019\\nN            2.53855262295791       -1.82382099442775        0.37168991090722\\nC            4.57256021750363       -0.35569639046986        0.12720993768171\\nC            4.91624953469089        0.93831129236282       -0.22914864605678\\nC            3.93057416289689        1.87341279946789       -0.60563874973414\\nC            2.59575661800247        1.55655740043045       -0.64219166018973\\nO            6.19019036497287        1.41944478358739       -0.24933071249286\\nC            7.24154609871348        0.55385933118568        0.10060147344521\\nH           -6.32325494414460       -0.11985370144267       -1.05843586878840\\nH           -5.48385298530286       -0.77926408089639       -2.45943289815050\\nH           -5.24159519167427        0.91654911048690       -1.99848650131287\\nH           -4.43993799909669       -2.60936497685819       -1.08283980519977\\nH           -5.67028349282925        1.74916352957019        0.55414797539846\\nH           -4.85342980050054        3.32988810164878        0.45427494173523\\nH           -4.39164589179887        2.16262272233856        1.72156600719087\\nH           -0.46865873914399        0.93523505499834        0.98335265463404\\nH           -1.48670447957558        0.94204752104446        2.43731791587357\\nH           -1.82251044204056        2.06166399109598        1.10400831324591\\nH           -0.45095784668333       -1.19417634635622        2.16141679405167\\nH           -1.43666573200526       -2.68209456385283        2.29425254115607\\nH            2.87753038946151       -2.70807761132856        0.71725238458564\\nH            5.30015930092154       -1.09430036245959        0.42167878884873\\nH            4.27117645577958        2.86187828695743       -0.86823207089534\\nH            1.85089813211726        2.27832889717705       -0.93343456447739\\nH            7.29540174771452       -0.30647818646246       -0.57428743483670\\nH            7.14572861261376        0.20103168524934        1.13266685186258\\nH            8.15264561791576        1.14229671345447        0.00558233981767\\n43\\n energy: -70.311270077368 xtb: 6.7.0 (08769fc)\\nC           -5.43410873190899       -0.08702358141601       -1.58437703476231\\nC           -4.28860730883252       -0.50277500769231       -0.71516308794683\\nC           -3.98887945727715       -1.84309476962500       -0.50730120474140\\nN           -2.98718432113743       -2.26340264356177        0.23556792175190\\nC           -2.18102965865802       -1.37499128948421        0.80553553780693\\nC           -2.37013495364893        0.00417260889102        0.68086768838767\\nC           -3.46165602526407        0.42951423612903       -0.08337399931579\\nO           -3.69556694270788        1.78433176984142       -0.24298454302527\\nC           -4.66951010694749        2.30658459162371        0.63524249842599\\nC           -1.45490349833665        1.00239158679690        1.31664997801181\\nC           -1.07445446673207       -1.98734184971476        1.59212243476333\\nS            0.06014162762183       -2.87147842827108        0.39296530259801\\nO            0.86467966385549       -3.80614764213589        1.19475590345517\\nC            1.22664013249240       -1.51285692687922        0.17436907667667\\nN            1.00929760240617       -0.30208119610581       -0.24527986706808\\nC            2.24673355360427        0.29136652486570       -0.29301168302703\\nC            3.23315725153130       -0.63820379151108        0.11363672440469\\nN            2.54365973356392       -1.78381129184138        0.40022826450129\\nC            4.58383653384790       -0.32303882026558        0.16178359813709\\nC            4.93481871086773        0.96393642132380       -0.21250960810705\\nC            3.95545775941049        1.89680330293105       -0.61079939057061\\nC            2.62027965276598        1.58327451138992       -0.65560627875951\\nO            6.21058552765457        1.43951154904701       -0.23362017839685\\nC            7.25823962977840        0.57097045455960        0.12066298513141\\nH           -6.37165279578263       -0.12477377096466       -1.02801518814889\\nH           -5.52299936093212       -0.76038928001698       -2.43396871509531\\nH           -5.28308331411360        0.92700002367417       -1.94689036270703\\nH           -4.59136853762707       -2.61876818302279       -0.96178611460359\\nH           -5.63946345143592        1.81624216736042        0.50506053382924\\nH           -4.76182232552671        3.36325662178844        0.39004890676412\\nH           -4.35961932741729        2.20036947132236        1.68117923747195\\nH           -0.44994938736526        0.86871980731376        0.91919169024107\\nH           -1.43076328480480        0.87785456863937        2.39900110689201\\nH           -1.78582927506820        2.00990409591121        1.08200626765967\\nH           -0.49937567769389       -1.27927967959478        2.17893069584918\\nH           -1.45081924184138       -2.79691479796925        2.21603626639230\\nH            2.87569628941370       -2.66913738569708        0.75043582819805\\nH            5.30658592815905       -1.05900957691242        0.47414636890242\\nH            4.30197343704051        2.88021842928352       -0.88564622136256\\nH            1.88000613301374        2.30253945130442       -0.96475804063315\\nH            7.30225850332527       -0.29755424624423       -0.54465308835922\\nH            7.16621075121883        0.23082181624179        1.15739448336776\\nH            8.17252402948441        1.15228914866306        0.01345230699318\\n43\\n energy: -70.311352057441 xtb: 6.7.0 (08769fc)\\nC           -6.04450091456565       -0.39751844546097       -0.97625391837603\\nC           -4.69851798016457       -0.68815923167710       -0.39041470591720\\nC           -4.31276976551304       -1.98201925912584       -0.06324993412507\\nN           -3.14060535478154       -2.28987243877335        0.45000555752490\\nC           -2.24623870906246       -1.32988489830605        0.65404737114400\\nC           -2.50236981832489        0.01438102595351        0.36431484476797\\nC           -3.76525183047146        0.32347261725013       -0.15080665575803\\nO           -4.07434788066474        1.63616890597985       -0.46449908784403\\nC           -4.75358280773763        2.32622753005721        0.56247176534496\\nC           -1.49068720100326        1.09249922821301        0.58952813180274\\nC           -0.94869295013076       -1.80689899590362        1.20659540989743\\nS            0.07708953103620       -2.55260966950084       -0.18741292840776\\nO            0.80437068980302       -3.68133956828398        0.41015830088417\\nC            1.36095540432256       -1.28314301623971       -0.18800286441112\\nN            1.26552634357326       -0.02662197977722       -0.50887794724289\\nC            2.53671725601862        0.47620566652505       -0.38337665831602\\nC            3.41423150198981       -0.55637183770841        0.02431197601928\\nN            2.62690557941233       -1.66941359755898        0.13255672222681\\nC            4.76825355035023       -0.34441785884124        0.24271504957484\\nC            5.23564095218888        0.94400225793148        0.03971000530007\\nC            4.36615824484068        1.97591955631130       -0.36954849302154\\nC            3.02738097036884        1.76430184694225       -0.58281028870918\\nO            6.52993616947657        1.33005401395936        0.20984779956325\\nC            7.46710937432642        0.37280461208060        0.63773225511699\\nH           -6.79019271330409       -0.29207054883622       -0.18706556075037\\nH           -6.35703119732321       -1.21242265224560       -1.62504513313382\\nH           -6.01266067094641        0.52625492988049       -1.54925568185869\\nH           -4.98761461013939       -2.81366025732713       -0.21890198972545\\nH           -5.71206942856201        1.85412250508826        0.80176041136484\\nH           -4.92931732043151        3.33235518031644        0.18561364109354\\nH           -4.14927592706945        2.37856055259082        1.47499989492315\\nH           -0.55848217328756        0.85338844674364        0.07762695563062\\nH           -1.28095793297322        1.20611141861989        1.65379481987256\\nH           -1.86627775376647        2.03583495813240        0.20337060176265\\nH           -0.37407477969259       -1.02704514027969        1.69487056627815\\nH           -1.11179593647405       -2.65541457172762        1.86859776136917\\nH            2.86438888808993       -2.61076586953110        0.40611274941756\\nH            5.40610126408335       -1.15597480159415        0.55262487597528\\nH            4.80071945861292        2.95279869869596       -0.50941574867065\\nH            2.37127295448936        2.56000266488797       -0.89452230938350\\nH            7.57076167702222       -0.44109918093763       -0.08728725821961\\nH            7.19958173610345       -0.04216183013982        1.61505654012079\\nH            8.41421511029475        0.90341803359569        0.71985015688573\\n43\\n energy: -70.311711233735 xtb: 6.7.0 (08769fc)\\nC           -6.32779721179640       -0.76282528451670       -0.57727203609260\\nC           -4.90074070387492       -0.87613413383645       -0.14090381100891\\nC           -4.38948412105595       -2.06350319714119        0.36800491469219\\nN           -3.14087938663101       -2.21886584878554        0.75622672038458\\nC           -2.29903309566097       -1.19888028047297        0.65271000779714\\nC           -2.68010203142603        0.05118804951632        0.15273966262265\\nC           -4.01572263147050        0.20170663263701       -0.23056660412375\\nO           -4.45131194942161        1.41134612927045       -0.74483125678933\\nC           -5.00729355276862        2.27598387646908        0.22256345031742\\nC           -1.72036969766501        1.19340223289483        0.03946944858312\\nC           -0.89451265880269       -1.48608366513162        1.05051515996719\\nS            0.04985903273979       -2.03093030136679       -0.48844412736863\\nO            0.64590040945431       -3.33496051008564       -0.17500227213649\\nC            1.45258618433086       -0.92499537559928       -0.23334893679520\\nN            1.46065442877340        0.37565152971120       -0.24556970098128\\nC            2.77644454144517        0.72800861648678       -0.07153267258223\\nC            3.56754784506159       -0.44029276467298        0.03964895060429\\nN            2.68576822361056       -1.47968508322104       -0.07486467882461\\nC            4.94215644431072       -0.39961939312611        0.22527802523851\\nC            5.52237075318810        0.85659495266223        0.29511379717588\\nC            4.73981491974716        2.02424171265305        0.18377610214582\\nC            3.38022409752688        1.98093174015962        0.00356645239363\\nO            6.85329513763032        1.08216163582111        0.47075666571353\\nC            7.71296895320068       -0.02414583277132        0.59500131077068\\nH           -6.97632701891489       -0.58389001219920        0.28136545890404\\nH           -6.65046256696939       -1.68593910447168       -1.05357008032480\\nH           -6.44180753726067        0.06094543751220       -1.27798864722014\\nH           -5.02159839666368       -2.93621647278882        0.46826779867106\\nH           -5.88287256623295        1.82968640068018        0.70587066200602\\nH           -5.30753713350492        3.17530195697392       -0.31237525326904\\nH           -4.27373364045698        2.53717509637772        0.99307872090875\\nH           -0.77460531267638        0.88434053815252       -0.40102382498506\\nH           -1.50517854551933        1.60360756873993        1.02740235513318\\nH           -2.15617401185212        1.97773683623975       -0.57387359102285\\nH           -0.39172309328816       -0.62274077186319        1.47386371709359\\nH           -0.85602884988518       -2.34783359840789        1.71308417911788\\nH            2.84647547312515       -2.47437932833222       -0.03521090235873\\nH            5.51056445875601       -1.31184492485416        0.30564840064809\\nH            5.25956673170500        2.96696226304241        0.24592451633612\\nH            2.79266531648636        2.87990391092023       -0.08137421057608\\nH            7.69598601238293       -0.64985098563610       -0.30356142548249\\nH            7.45939794111360       -0.63427641999695        1.46830759819217\\nH            8.71104980922648        0.39101517233835        0.72465695651682\\n43\\n energy: -70.312000250681 xtb: 6.7.0 (08769fc)\\nC           -6.43035247263649       -0.85804661156428       -0.23286178004975\\nC           -4.96172799235149       -0.93351307671452        0.04386427960676\\nC           -4.38468542452034       -2.06347214766773        0.60915524616232\\nN           -3.09936323859283       -2.17752916756505        0.87644974477636\\nC           -2.28833815762676       -1.17249620483444        0.57905043956575\\nC           -2.73400570532928        0.01634322385762       -0.00929040293265\\nC           -4.10386994338466        0.12877348601264       -0.25617375741271\\nO           -4.60819109089808        1.27887871065045       -0.83943000193471\\nC           -5.00813269319181        2.26066544438624        0.09276756725226\\nC           -1.79651629487381        1.13052628921417       -0.35135197274727\\nC           -0.84535196348631       -1.38647327414372        0.86894310940110\\nS            0.03932858598992       -1.78546364076418       -0.74414541981934\\nO            0.58131802272895       -3.14127994023355       -0.60530143452433\\nC            1.48109417922254       -0.76830655292933       -0.36947319889955\\nN            1.52003330621818        0.52332266416133       -0.22419232933913\\nC            2.83962278241936        0.81628011089501        0.01638137561678\\nC            3.59954850419505       -0.37774204693758        0.00229472410584\\nN            2.69588395083000       -1.37222252548619       -0.25551301024826\\nC            4.96904818278654       -0.39650851480623        0.22497159004372\\nC            5.57586256492785        0.82563201105666        0.46517569434904\\nC            4.82459031534446        2.01861666026258        0.47867953842110\\nC            3.47019569296504        2.03398440078963        0.25922709417010\\nO            6.90569674788100        0.99215327926633        0.70393276689241\\nC            7.73576394679521       -0.14302100152149        0.70970081793766\\nH           -6.96999213054914       -0.54541424633157        0.66208699611500\\nH           -6.80996891243125       -1.83265912745949       -0.53012818995208\\nH           -6.62577373385476       -0.13823296156730       -1.02409158069169\\nH           -4.99221833632436       -2.92196085958510        0.86496411284228\\nH           -5.82080964654474        1.89919009145442        0.73214670175994\\nH           -5.35759609102858        3.10949636379649       -0.49243748030758\\nH           -4.17248805253743        2.57410569143442        0.72777703615095\\nH           -0.90184759416886        0.76338050397919       -0.84765859569801\\nH           -1.47518398801125        1.64855720907816        0.55278845137390\\nH           -2.29554994274961        1.84302543284783       -1.00295192501853\\nH           -0.37582879474177       -0.50157232450254        1.28862819292228\\nH           -0.70946637527002       -2.26137215150688        1.50034000926042\\nH            2.83360367831769       -2.36807641996216       -0.33183106598227\\nH            5.51367404180976       -1.32639489448605        0.20746161834215\\nH            5.36388365866264        2.93209979004315        0.67187227192506\\nH            2.90576189976153        2.95167441598373        0.27197128257133\\nH            7.73031714497882       -0.64994366185544       -0.26069455042226\\nH            7.44315461881859       -0.85107211605044        1.49190818748125\\nH            8.73887775046621        0.22606668929193        0.91651484692455\\n43\\n energy: -70.312012823581 xtb: 6.7.0 (08769fc)\\nC           -6.45874831473585       -0.90181130553859       -0.09159261727195\\nC           -4.97999335345085       -0.93722803067520        0.13675038214171\\nC           -4.37193364763085       -2.01195760407368        0.77226259319175\\nN           -3.07540092233972       -2.10042861959533        0.99207765875452\\nC           -2.28443203622331       -1.12634852374977        0.56724779554359\\nC           -2.76401311214913        0.01291983117754       -0.08911401695311\\nC           -4.14379575846854        0.10196230348585       -0.28384888185043\\nO           -4.67477430962819        1.20455200219013       -0.93075723396352\\nC           -5.15274028650843        2.20461621537957       -0.05578748017695\\nC           -1.84622664308909        1.10095533431105       -0.54918866567113\\nC           -0.82919792542484       -1.30057018407844        0.82239922548556\\nS            0.04934067669706       -1.61740158323881       -0.80577817627541\\nO            0.55659672376104       -2.99276503337560       -0.75909092556731\\nC            1.51014408624958       -0.66095371244617       -0.35806294107004\\nN            1.57099086683668        0.60940336500829       -0.08753449382915\\nC            2.89929900692956        0.85916296826343        0.15540259792749\\nC            3.64050370731370       -0.33788444142241        0.00922862860483\\nN            2.71729284731749       -1.28982848210342       -0.32768980933536\\nC            5.01329337352421       -0.39796346697258        0.20156192451120\\nC            5.64423052637268        0.78611554380714        0.54732570774672\\nC            4.91236986133061        1.98246394390521        0.69207690718280\\nC            3.55430873448945        2.03816106246687        0.50318210890011\\nO            6.98101360422251        0.90886296774076        0.77417157186363\\nC            7.78939657768345       -0.23663585818627        0.66310040479319\\nH           -6.97541297008909       -0.54864951449172        0.80193111539892\\nH           -6.82802263181662       -1.89939705016868       -0.31861833215102\\nH           -6.69587094761470       -0.23560755967853       -0.91785097776969\\nH           -4.96285541648560       -2.84641115575825        1.12677079191238\\nH           -5.94459003039102        1.82567143413434        0.59816653448466\\nH           -5.55250247764761        2.99463692129436       -0.68916433239705\\nH           -4.34520609012836        2.60764421761822        0.56555750552069\\nH           -1.05523842419455        0.70699680658189       -1.18246182100115\\nH           -1.36744981455826        1.58547196409099        0.30155933195274\\nH           -2.40669999135565        1.84276853932957       -1.11204685969634\\nH           -0.38389329319705       -0.40472349545950        1.24799392040316\\nH           -0.65267754187851       -2.17777050122298        1.44034776696244\\nH            2.83888348744148       -2.27575482049515       -0.49904382039910\\nH            5.54197456486038       -1.32949381923307        0.08192415704821\\nH            5.46951155984805        2.86545270458692        0.96172075779730\\nH            3.00566711836291        2.95847141909001        0.61665159993267\\nH            7.76527397100195       -0.64903559471745       -0.35098892639478\\nH            7.49027167302093       -1.01032089968768        1.37786524167320\\nH            8.80131397175838        0.09265071189432        0.89287108202988\\n43\\n energy: -70.312014116532 xtb: 6.7.0 (08769fc)\\nC           -6.46690265533601       -0.79710120089345        0.08387842171005\\nC           -4.98090479126676       -0.87318998833521        0.25051931291600\\nC           -4.39033698989652       -1.92898319127562        0.93244927014624\\nN           -3.08988142318216       -2.06760559470710        1.09318148215337\\nC           -2.27600486449340       -1.17015915461401        0.55621603318335\\nC           -2.73494233367526       -0.06173744481345       -0.16293988385366\\nC           -4.11921033506461        0.08678085575362       -0.28982874691807\\nO           -4.61520254403389        1.16475052006853       -0.99980995419905\\nC           -5.13231780942120        2.20571470842539       -0.19754324109457\\nC           -1.80530608839981        0.92362838162653       -0.79989427822512\\nC           -0.82083160075541       -1.40203364102244        0.77773192904249\\nS            0.05875335460461       -1.57910309860636       -0.86419499130456\\nO            0.59171419852981       -2.94558522616470       -0.92176156544205\\nC            1.50375273529844       -0.63534631208577       -0.34514594381202\\nN            1.54267769007542        0.60319688267356        0.05042166825549\\nC            2.86988176780949        0.85762057433388        0.29788203380882\\nC            3.63260820112754       -0.30207445069410        0.02202990328169\\nN            2.72377783088666       -1.23571308329265       -0.39562257686925\\nC            5.00937767524207       -0.35089903434021        0.18674845567338\\nC            5.62209176559080        0.80645453506431        0.63904388102665\\nC            4.86846232836918        1.96594215224576        0.91349139198049\\nC            3.50637132754841        2.01026843309081        0.75196465426071\\nO            6.95988607325374        0.93546395887423        0.85584820605933\\nC            7.78983136341596       -0.17445279120358        0.61597937817030\\nH           -6.93022375178634       -0.33408635778290        0.95631891980208\\nH           -6.88139503334695       -1.79735514188672       -0.02130704460729\\nH           -6.71920963932381       -0.21299239352586       -0.79827645628882\\nH           -4.99992613555431       -2.70705321142084        1.37319851414444\\nH           -5.95349737832457        1.86526268629833        0.43972198679190\\nH           -5.49956881661765        2.96316195077828       -0.88792522854166\\nH           -4.35241823836306        2.63998178527783        0.43865764437365\\nH           -1.52911375924154        0.57565251707598       -1.79419184026110\\nH           -0.89343378656500        1.06073822638619       -0.22428716919906\\nH           -2.30439153477011        1.88152894927174       -0.92046625841301\\nH           -0.35989738349491       -0.56385709068455        1.29608465637502\\nH           -0.66867514887964       -2.33338553367946        1.31885238666687\\nH            2.86186010464310       -2.19869502741919       -0.66011816619503\\nH            5.55458548233769       -1.25426316483345       -0.03289753682610\\nH            5.41208406461439        2.82975943067678        1.26143669285275\\nH            2.94126584582364        2.90238939844076        0.96557530722088\\nH            7.75304346252146       -0.48474085635861       -0.43343919102328\\nH            7.52238964189465       -1.02184305645674        1.25574767536763\\nH            8.79917812822340        0.15395909972104        0.85819726780099\\n43\\n energy: -70.311726202235 xtb: 6.7.0 (08769fc)\\nC           -6.46341053358615       -0.74861245187340        0.14877646993134\\nC           -4.97615499862734       -0.84610599745070        0.29271621789465\\nC           -4.39480026348035       -1.89003976830016        1.00039277494990\\nN           -3.09481808287159       -2.05935641243005        1.13080744423269\\nC           -2.27129014745880       -1.20856720220704        0.53497313015743\\nC           -2.72015394035195       -0.10032094238110       -0.18922653355855\\nC           -4.10392883658065        0.07846555399862       -0.29046294291876\\nO           -4.58172598236867        1.15065705055694       -1.01940274834801\\nC           -5.11285061710709        2.20662959751793       -0.24634913967161\\nC           -1.78771098959137        0.86873578090270       -0.84691272077511\\nC           -0.81788085753798       -1.47978017668016        0.73288766178318\\nS            0.06417372104862       -1.56776996207284       -0.91325105406228\\nO            0.61596555753398       -2.92355088310273       -1.02829086619336\\nC            1.49867486070834       -0.62914184602714       -0.35855629593545\\nN            1.52840927318954        0.60027697706283        0.06537479247337\\nC            2.85319125397581        0.85748906512428        0.32251645913985\\nC            3.62424894174623       -0.29050927491563        0.02181642932125\\nN            2.72270762631358       -1.22003176976960       -0.41993171206332\\nC            5.00096064695815       -0.33413775743211        0.18868498737497\\nC            5.60511830529600        0.81675071613419        0.66816556612276\\nC            4.84328657688215        1.96489389022783        0.96666786451431\\nC            3.48124946314287        2.00383591804505        0.80350759017140\\nO            6.94156977427412        0.94949740602319        0.89112921580434\\nC            7.78050370241366       -0.14594566671593        0.61899937746194\\nH           -6.90356095766843       -0.23946752501382        1.00748590328670\\nH           -6.89719012423007       -1.74481929111041        0.09529143507646\\nH           -6.72164769043744       -0.20059046440649       -0.75464653239296\\nH           -5.01268702732536       -2.63297612212697        1.48794284692076\\nH           -5.95364458681951        1.88419591990041        0.37371626254414\\nH           -5.45465340502297        2.95586821530567       -0.95842569354305\\nH           -4.34662254224234        2.64486843647536        0.40347763475084\\nH           -1.57466897913045        0.54467525560082       -1.86437728713115\\nH           -0.84657290344233        0.96117984654460       -0.31094243199385\\nH           -2.25923086072491        1.84611400895423       -0.91439599807884\\nH           -0.34697782387556       -0.68144547891638        1.30422266931868\\nH           -0.68043873541981       -2.44235070226233        1.22049358272515\\nH            2.86608098154075       -2.17552871415413       -0.70789368118700\\nH            5.55244798347325       -1.22882972896672       -0.04995048518220\\nH            5.38050380876486        2.82449415422357        1.33447042130602\\nH            2.90991675917080        2.88744518419662        1.03518789208829\\nH            7.73990716435346       -0.43083164560721       -0.43744727792616\\nH            7.52547316387027       -1.01150172453648        1.23936401433233\\nH            8.78823232126222        0.18613753165100        0.86292175727020\\n43\\n energy: -70.311040616114 xtb: 6.7.0 (08769fc)\\nC           -6.45490315560205       -0.67146785030554        0.27311551932401\\nC           -4.96619209100919       -0.80792028187937        0.35807320301782\\nC           -4.38457669505566       -1.83814396239756        1.08496796857961\\nN           -3.08598495199800       -2.04644565142330        1.15932037164236\\nC           -2.26456611330878       -1.25416645906386        0.48423604682806\\nC           -2.71139084290367       -0.15139363412065       -0.24905760934757\\nC           -4.09302425570211        0.06607862252075       -0.29717669329406\\nO           -4.56925640059328        1.12660220177873       -1.04361971642415\\nC           -5.04736062562734        2.22205103753237       -0.29077258282660\\nC           -1.78180132876616        0.78283371225036       -0.95946284181599\\nC           -0.81419216917363       -1.57165455567451        0.64520739171585\\nS            0.07898507052199       -1.58508018912025       -0.99471194167697\\nO            0.65534852273468       -2.92707873300701       -1.14849604567995\\nC            1.49371719992068       -0.64172203726379       -0.39870836707596\\nN            1.49882169378273        0.57001116971196        0.07410572194853\\nC            2.81692993931422        0.83922559436799        0.35240717438238\\nC            3.61012225702157       -0.28206267243536        0.01139149344896\\nN            2.72810283216640       -1.20848684304858       -0.47434273865194\\nC            4.98619830388246       -0.30862397424871        0.18695595383139\\nC            5.56632003805262        0.83141640051772        0.71909845559783\\nC            4.78195369673987        1.95216315329801        1.06034093335186\\nC            3.42081428084703        1.97464305571323        0.88710406558747\\nO            6.89857833225351        0.97846880035629        0.95734740211868\\nC            7.75962372583381       -0.08820502850423        0.64333495651809\\nH           -6.84187330141281       -0.10291104454131        1.11996249982988\\nH           -6.91914569136816       -1.65497390854953        0.29374293565135\\nH           -6.73682608307795       -0.16542256547112       -0.64754231649104\\nH           -5.00057141275544       -2.53707499903944        1.63571980522528\\nH           -5.88108838218002        1.94604440214286        0.36028015095409\\nH           -5.38492082461803        2.95892378792645       -1.01768148800393\\nH           -4.25102832876605        2.65602022702321        0.32470495355519\\nH           -1.58767896875722        0.42244308394299       -1.96832819665315\\nH           -0.83348927527585        0.88639764121261       -0.43737405511405\\nH           -2.24675249310373        1.76135619389799       -1.05317422079842\\nH           -0.32809546474452       -0.81941356432946        1.26548855377171\\nH           -0.69732781268543       -2.56189593770311        1.08000901624107\\nH            2.89062370470354       -2.14891322528653       -0.79930451918304\\nH            5.55525220952287       -1.18278236309939       -0.08432082922507\\nH            5.30097966066621        2.80411065073218        1.46965612896486\\nH            2.83214112596542        2.83729578795258        1.15194205868767\\nH            7.73230626691606       -0.32563143145542       -0.42520538613212\\nH            7.51609460378737       -0.98541308249308        1.22182077933467\\nH            8.75913420386975        0.25079747156944        0.91047300827687\\n43\\n energy: -70.309930276272 xtb: 6.7.0 (08769fc)\\nC           -6.42864376141671       -0.54816753864467        0.39875491290379\\nC           -4.94509064449752       -0.75019975884568        0.41961967584897\\nC           -4.37504660254951       -1.78105097110741        1.15451243688280\\nN           -3.08581894046128       -2.04916136940916        1.16916956682113\\nC           -2.26297708586645       -1.32511697256438        0.42159337292537\\nC           -2.69341365638497       -0.21810701011611       -0.31438321792247\\nC           -4.06531055915884        0.06191195502551       -0.30232649840069\\nO           -4.52972679457845        1.12417616781036       -1.05382136197454\\nC           -4.92093625047509        2.25925220368312       -0.30921513316640\\nC           -1.76025530294415        0.66672394406666       -1.08102763482788\\nC           -0.82583047637749       -1.72148265771584        0.54256802103673\\nS            0.09999742838388       -1.65595255998441       -1.07512023082881\\nO            0.71764332187216       -2.97786077119437       -1.24908533105988\\nC            1.47807929924834       -0.69093306805240       -0.42969918490401\\nN            1.44395896242816        0.50362940250957        0.08363744027176\\nC            2.75117082132620        0.80106539451919        0.38362499657744\\nC            3.57972593247639       -0.28398507284370        0.01113512838340\\nN            2.72908396400494       -1.21871095388285       -0.51316076903247\\nC            4.95503933642947       -0.27476009098991        0.19456716452715\\nC            5.49713340515476        0.86356495957067        0.76880455903079\\nC            4.67677563624038        1.94659666598658        1.14528364854154\\nC            3.31668130182463        1.93408511929378        0.96331522384031\\nO            6.82326403782833        1.04438071969496        1.01755609044374\\nC            7.72018500478229        0.02161252567010        0.66109437261944\\nH           -6.74997956565676        0.06392407176890        1.24263852258841\\nH           -6.93422684021351       -1.50833526285874        0.47373581568006\\nH           -6.73101490986214       -0.05909235999756       -0.52459496415825\\nH           -4.99251447109351       -2.42925472037958        1.76269577937038\\nH           -5.72741203419133        2.03634755394401        0.39423065325602\\nH           -5.26880837012902        2.98910520049142       -1.03837268860918\\nH           -4.07469742854370        2.67636745930329        0.24848646749944\\nH           -1.58321715840849        0.25925112261889       -2.07470601812524\\nH           -0.80653358036521        0.78036245743011       -0.57029014888710\\nH           -2.21279252205174        1.64660818696268       -1.21248337750601\\nH           -0.31486748579718       -1.04799510518277        1.23068014373769\\nH           -0.75720244331204       -2.74646510938169        0.90101518093347\\nH            2.92190590544119       -2.14257867032478       -0.86765932818769\\nH            5.55213922262076       -1.12126003296502       -0.10286093272214\\nH            5.16712000540783        2.79857565521137        1.58845091549138\\nH            2.70024587131130        2.76822748644032        1.25485316591359\\nH            7.69859125839095       -0.17426788459377       -0.41597351618376\\nH            7.50971890530423       -0.90568250453440        1.20388191623163\\nH            8.70785826387566        0.38465119355268        0.94040216513095\\n43\\n energy: -70.308061767975 xtb: 6.7.0 (08769fc)\\nC           -6.26522015195225       -0.27526230566943        0.74905461463487\\nC           -4.82609886416011       -0.64648918267220        0.56938247278922\\nC           -4.27671414517387       -1.73814813742320        1.22815185039139\\nN           -3.02965976334160       -2.13154707362399        1.07996328495463\\nC           -2.23386949726945       -1.48861894670228        0.23264798706537\\nC           -2.64581197680961       -0.35110294573622       -0.46450009894150\\nC           -3.96940953511406        0.06750407272831       -0.27151781978809\\nO           -4.42791941550618        1.17855210595107       -0.95563273371783\\nC           -4.49522616901044        2.35542265465833       -0.17632172646107\\nC           -1.75226390714753        0.42759067481354       -1.37807246492055\\nC           -0.84612722679340       -2.05772400314878        0.21885357409320\\nS            0.14377274984732       -1.84057223919492       -1.34211981748468\\nO            0.87630426993070       -3.10555704947522       -1.50346394721726\\nC            1.41111852218217       -0.78971709413623       -0.60717058538818\\nN            1.28977109286267        0.41530667224640       -0.13303632552231\\nC            2.55603296972547        0.77374013762529        0.26143623166787\\nC            3.45348660085496       -0.28529134309972       -0.01367131730047\\nN            2.68423028507838       -1.26773823954233       -0.57415352069963\\nC            4.80926856855829       -0.21302586225062        0.27271348308054\\nC            5.25979658778876        0.96273409711617        0.85054386442031\\nC            4.36987368856862        2.01955450826720        1.13222640290075\\nC            3.02930043537679        1.94461258494106        0.84877269735492\\nO            6.55512928843277        1.20727785480245        1.19009165107874\\nC            7.51818845803035        0.21675561599493        0.92618873876783\\nH           -6.39949372737188        0.32698129482587        1.64856483818884\\nH           -6.87036293326347       -1.17290698927507        0.85616805145060\\nH           -6.61893138353805        0.29209552146396       -0.10874417358935\\nH           -4.87354003587815       -2.32693985524821        1.91265319989812\\nH           -5.16661499137026        2.24226327568498        0.67980047054167\\nH           -4.87818299784851        3.13144839556961       -0.83701414581241\\nH           -3.50382340259896        2.64407446724712        0.19042888835427\\nH           -1.61743993467602       -0.10183980791205       -2.31903192462144\\nH           -0.77932460197855        0.60096581937138       -0.92211724243303\\nH           -2.21314605188064        1.38467730422134       -1.60806840478611\\nH           -0.27233368271851       -1.61588038402533        1.03453286618671\\nH           -0.90281782674638       -3.13467697183965        0.37281588858387\\nH            2.93945414829857       -2.18863808414936       -0.89656604348277\\nH            5.46077755602849       -1.04121586650068        0.04655410971071\\nH            4.79039207791730        2.90327068204850        1.58500746114688\\nH            2.35941359374153        2.75952492233240        1.06782114936558\\nH            7.57896838845607       -0.00950083753157       -0.14341792719266\\nH            7.31140489348772       -0.70385919739940        1.48177811952154\\nH            8.46764904699737        0.63189875463470        1.25999532319837\\n43\\n energy: -70.307770617422 xtb: 6.7.0 (08769fc)\\nC           -5.55442968045721        0.14340103351451        1.54458427639914\\nC           -4.32839041751177       -0.47615201941542        0.95129355248337\\nC           -3.79213476541048       -1.65530244262209        1.45237435005177\\nN           -2.70988841858817       -2.22604464967002        0.97057817130689\\nC           -2.08369702647487       -1.68595973533461       -0.07090703584490\\nC           -2.54236847914210       -0.52969548754935       -0.70094369198011\\nC           -3.66995491270111        0.08865105490510       -0.14069962263610\\nO           -4.15919944994646        1.25452987393556       -0.70336439208721\\nC           -3.60849159146904        2.42534359673621       -0.13387944351147\\nC           -1.92006905392810        0.06145951194761       -1.92587490510850\\nC           -0.88433609421526       -2.50477169142374       -0.45564922588758\\nS            0.16133776540385       -2.03406017225170       -1.90910395204993\\nO            1.05604395582326       -3.19058157994943       -2.06564569183618\\nC            1.21982127137053       -0.88789364531774       -1.00426858561077\\nN            0.96772532060433        0.32638596664759       -0.61359083900242\\nC            2.12593201317831        0.75673732252463       -0.01253552188765\\nC            3.09952810059042       -0.26829520833043       -0.07543544655951\\nN            2.48254374384542       -1.30588915845516       -0.71783159393006\\nC            4.37770005006574       -0.12285458171198        0.44449331738250\\nC            4.67077826416208        1.09334295916234        1.03962585477638\\nC            3.70443119639912        2.11769814110804        1.10903133228282\\nC            2.44055530520423        1.96985102010324        0.59507923577562\\nO            5.87299187916845        1.40974408003073        1.59414050964679\\nC            6.90212674432169        0.45148893147515        1.56003617382081\\nH           -5.29520086615164        0.72991998236501        2.42655141935234\\nH           -6.25660368864828       -0.62897443516161        1.85007030805613\\nH           -6.03268610804809        0.79732959381670        0.81958860451153\\nH           -4.25549941881233       -2.16308745901322        2.28832062652930\\nH           -3.86624994693573        2.51164811305155        0.92688281789626\\nH           -4.04193467708293        3.26045255562757       -0.68154135239522\\nH           -2.51863485217994        2.44323973248224       -0.23588122540733\\nH           -1.87730895110220       -0.67329731215746       -2.72589197819975\\nH           -0.91402136067131        0.41670888801230       -1.70805796747726\\nH           -2.52579281155638        0.89682077090841       -2.26826999339597\\nH           -0.22062631962227       -2.59681576229950        0.40473987238503\\nH           -1.22050462622632       -3.51313976096008       -0.71115744371413\\nH            2.83737486168082       -2.21357649856075       -0.97860506921887\\nH            5.09110257434888       -0.92768479407082        0.37495637896229\\nH            4.00212946975057        3.03687796174790        1.58786007992899\\nH            1.71190176243947        2.76139346290959        0.65380127400874\\nH            7.17177882041839        0.18808741531836        0.53197189979880\\nH            6.62563120317269       -0.45545443384165        2.10765887520994\\nH            7.75659021495095        0.91841785974802        2.04702304716527\\n43\\n energy: -70.311885395698 xtb: 6.7.0 (08769fc)\\nC           -4.42020773408942        0.51772709844608        2.16979325522412\\nC           -3.57017017624043       -0.31314619239206        1.26111398490543\\nC           -3.13373141619458       -1.58126400141704        1.62335988726189\\nN           -2.32355479261213       -2.31104202346635        0.88653323105618\\nC           -1.92218356868060       -1.85555722101736       -0.29571088727420\\nC           -2.40254886921543       -0.66645853188347       -0.84115241805212\\nC           -3.18250188756945        0.13908515107213        0.00011853827529\\nO           -3.60452216852742        1.37843651499040       -0.43844776813211\\nC           -2.59561650681345        2.36871756510572       -0.34459627725739\\nC           -2.18648683145768       -0.24396855199024       -2.26090861220866\\nC           -0.99181082713537       -2.80477126823058       -0.98728143390041\\nS            0.06064456235953       -2.12670052291952       -2.34948314791934\\nO            1.08120176993049       -3.16402908288102       -2.55541136665049\\nC            0.92753733328629       -0.95701002353533       -1.28430543349655\\nN            0.52792976287791        0.20653207975374       -0.86244807858919\\nC            1.56709194919833        0.68868518888946       -0.10459751631112\\nC            2.62780520988992       -0.24781238727980       -0.10640550574842\\nN            2.18082628770231       -1.28936990601673       -0.87210009017752\\nC            3.82116791862195       -0.03038009340245        0.56733720544825\\nC            3.93530339831500        1.16556467575076        1.25741268657334\\nC            2.87970858262767        2.10012769594404        1.26822319576248\\nC            1.70061700283534        1.88211290417472        0.60066604974777\\nO            5.03406701381193        1.54508291080734        1.96555640964513\\nC            6.14386272496496        0.68181959207271        2.00051268780606\\nH           -3.80315304150089        0.98549759941741        2.93754749129989\\nH           -5.16343914381658       -0.10209509755606        2.66649630863411\\nH           -4.92149193576679        1.29770824539011        1.60269816797809\\nH           -3.43469568122408       -2.02314504187881        2.56437311018123\\nH           -2.42544749907669        2.64620015334765        0.70175119828867\\nH           -2.96391788418406        3.23366252852939       -0.89378117863698\\nH           -1.65234949000911        2.02400682070310       -0.77597542503183\\nH           -2.28774745780472       -1.09322613548986       -2.93207401718557\\nH           -1.20302340230375        0.20108521598595       -2.39184457278393\\nH           -2.93844129588364        0.49800021458565       -2.52306700686882\\nH           -0.32514234958140       -3.26466621210910       -0.25876113807543\\nH           -1.56207738687968       -3.60579926047937       -1.46686251389941\\nH            2.64486601575755       -2.14401430767285       -1.13961565347480\\nH            4.60734078951481       -0.76715611795454        0.54004462833317\\nH            3.03804099926559        3.00790714851531        1.82821125023732\\nH            0.90207505723344        2.60579993121457        0.61528837000117\\nH            6.54878222960110        0.50731959171705        0.99829158449862\\nH            5.89273388727290       -0.27788277431770        2.46387991179338\\nH            6.89265985147339        1.18841492744164        2.60714788872702\\n43\\n energy: -70.314000336654 xtb: 6.7.0 (08769fc)\\nC           -4.10176583677526        0.59681565272052        2.26108796306799\\nC           -3.34727059207608       -0.27157289118363        1.30465840140146\\nC           -2.89492018125374       -1.53514769440528        1.66144518705674\\nN           -2.16977408543240       -2.30062544442867        0.87255687751127\\nC           -1.87921993306555       -1.88373500921406       -0.35432523414531\\nC           -2.37945970462873       -0.69431588359417       -0.88204333422652\\nC           -3.06898966996655        0.14400214082383        0.00223181999223\\nO           -3.50814655873254        1.37964988575342       -0.43023168412889\\nC           -2.48926793746703        2.36354329737561       -0.39398484352484\\nC           -2.25881712102307       -0.30470887888999       -2.32215071760931\\nC           -1.04338147411922       -2.85723080636128       -1.12371937445243\\nS            0.05145467500036       -2.13447513693404       -2.43316326447294\\nO            1.09322089651030       -3.15179803592758       -2.63194733715404\\nC            0.86790101550409       -0.97349201284145       -1.32174784140222\\nN            0.44530913318129        0.18497821126668       -0.90793721234792\\nC            1.45474854912898        0.67137284800923       -0.11388154409225\\nC            2.52326756170181       -0.25579457346895       -0.08693993927013\\nN            2.11017331426915       -1.29708848080880       -0.87144004786346\\nC            3.69442721694281       -0.02978865189270        0.62191797637866\\nC            3.77725280842378        1.16448310649097        1.31928431561928\\nC            2.71224566996737        2.08819342252041        1.30437312204596\\nC            1.55556706673109        1.86218630676990        0.60126455240106\\nO            4.85287344953062        1.55265569984463        2.05753374081001\\nC            5.97478895991515        0.70590705643824        2.10780539419030\\nH           -3.40837240743189        1.13041998572371        2.91189524045288\\nH           -4.75623626022483       -0.00552988165192        2.88695016679201\\nH           -4.69407010354648        1.32634707214651        1.71447116701658\\nH           -3.10948266981635       -1.94464904228985        2.64002693803554\\nH           -2.26572978464322        2.64836195194784        0.64055506478676\\nH           -2.87791816692190        3.22661420649956       -0.93179577508468\\nH           -1.57130619110938        2.00596443332493       -0.86809297602347\\nH           -2.37802704934996       -1.17143131919921       -2.96758820780290\\nH           -1.29471570319245        0.15781793557874       -2.52005048637028\\nH           -3.03893537680335        0.41751439821062       -2.55562839048315\\nH           -0.40864809697037       -3.42266348012385       -0.44312456873484\\nH           -1.67797179370090       -3.56515348770706       -1.66411209434258\\nH            2.59095297281569       -2.14553762684135       -1.12873499537886\\nH            4.48805882398794       -0.75889738348102        0.61402015516167\\nH            2.84499539951529        2.99474220786256        1.87299735473694\\nH            0.75046061929273        2.57862913170698        0.59542873942590\\nH            6.40042128275212        0.54422773121475        1.11201353304562\\nH            5.72978079590774       -0.26049068519534        2.56036641778453\\nH            6.70452748714418        1.21969872417438        2.73128274119827\\n43\\n energy: -70.315521227417 xtb: 6.7.0 (08769fc)\\nC           -3.94932784101150        0.62920196177893        2.29975387159745\\nC           -3.22954745565438       -0.24610708033623        1.32292769725666\\nC           -2.72389337995519       -1.48714480949892        1.68679092155092\\nN           -2.04319668702679       -2.26761068723208        0.87268734477393\\nC           -1.85119866446620       -1.88566635195308       -0.38407760159598\\nC           -2.38452241641512       -0.70725235890496       -0.90444750647370\\nC           -3.03672848139564        0.14156290107395       -0.00371239373298\\nO           -3.52124603123876        1.35852259983080       -0.44136725389358\\nC           -2.52841709631881        2.36901395490262       -0.45088868133837\\nC           -2.31548535534559       -0.33906446905712       -2.35320473856039\\nC           -1.07345377491220       -2.86805977620082       -1.19807988224391\\nS            0.06007719143991       -2.12403358609681       -2.46657903124525\\nO            1.10680855778250       -3.13744708764701       -2.65858452645209\\nC            0.85090077257678       -0.97691128766661       -1.32516898254537\\nN            0.42242730749492        0.18088471972515       -0.91533970511415\\nC            1.42000746039903        0.66374653063352       -0.10387888285499\\nC            2.48778928185208       -0.26401446090075       -0.06408658640224\\nN            2.08604018644427       -1.30266666464890       -0.85735760401587\\nC            3.65027346568867       -0.04023175845624        0.65955891826630\\nC            3.72584652898327        1.15300543288976        1.35929517693022\\nC            2.66106866330053        2.07677758831260        1.33336162364223\\nC            1.51288430413661        1.85265861017230        0.61566473724566\\nO            4.79373007515796        1.53993625438955        2.10946443149257\\nC            5.91700619073945        0.69525340458046        2.16603884686352\\nH           -3.23381425855908        1.17325875515603        2.91695965476483\\nH           -4.57467206975414        0.02985919542717        2.95768664549011\\nH           -4.56868973330512        1.34851235655593        1.76980024317588\\nH           -2.85980688600404       -1.86682981821727        2.69114756442953\\nH           -2.28753127576634        2.68306798926415        0.57114093116253\\nH           -2.95206941252248        3.20935306659153       -0.99827533688810\\nH           -1.61312717719083        2.02409943089943       -0.93936963532051\\nH           -2.43317864472362       -1.21703388688342       -2.98356122735715\\nH           -1.36480823783802        0.13756905312007       -2.58208098429831\\nH           -3.11421769525735        0.36568342497857       -2.57638934676042\\nH           -0.46656611362107       -3.49551536091398       -0.54743656282433\\nH           -1.74165372270469       -3.51146844129131       -1.77621864225825\\nH            2.57035239682542       -2.15049790442710       -1.10994652255483\\nH            4.44330795602045       -0.77004959202663        0.65997687115597\\nH            2.78747161331562        2.98247588502909        1.90482317424062\\nH            0.70881577600976        2.57012887511729        0.60101004562721\\nH            6.35011777248771        0.53659255319571        1.17296268730859\\nH            5.67090867587160       -0.27267081444323        2.61472937135523\\nH            6.64131923443184        1.20911065314290        2.79579787640174\\n43\\n energy: -70.317061955081 xtb: 6.7.0 (08769fc)\\nC           -3.64146430686236        0.67818267167462        2.37721552027730\\nC           -3.00954437127143       -0.21694273027245        1.35844431843412\\nC           -2.45125234523322       -1.43959390669578        1.70631965118863\\nN           -1.85071764746477       -2.24181721517889        0.85052676217277\\nC           -1.79646476485385       -1.89725419373506       -0.42969010453631\\nC           -2.37605590926641       -0.72948974224700       -0.92515268417208\\nC           -2.95165876680175        0.13636910907896        0.00912628050935\\nO           -3.49011976441360        1.33579738481871       -0.41384359367209\\nC           -2.52357555761941        2.36776258737913       -0.49897055215389\\nC           -2.40883853351650       -0.38848086575637       -2.38151964782983\\nC           -1.09663903064761       -2.88498186080667       -1.30181188352938\\nS            0.05476472596896       -2.11933948600068       -2.54817034221483\\nO            1.11212446008124       -3.12206160875298       -2.73531614959340\\nC            0.81643384569839       -0.97435273642213       -1.38644994545810\\nN            0.38643429893101        0.19420050629609       -1.01028952774279\\nC            1.35317249502907        0.67320570524880       -0.16027265314911\\nC            2.40450989122619       -0.26877345619583       -0.06116766989312\\nN            2.02438560676749       -1.31144531794442       -0.85978045026072\\nC            3.53602001742968       -0.05246739487863        0.71216532397294\\nC            3.59715897121376        1.14799150516068        1.40085280776455\\nC            2.54857954036933        2.08652867833943        1.31506978588432\\nC            1.43112274009859        1.86971868631349        0.54834945677909\\nO            4.63502252946770        1.52901589481479        2.19486455894772\\nC            5.74109305061954        0.66837859945110        2.31492903793880\\nH           -2.87673068621722        1.25553780361977        2.89768793022326\\nH           -4.18141094228715        0.09066726204536        3.11640630365142\\nH           -4.32835953849477        1.36795889429014        1.89304705897975\\nH           -2.47578254618022       -1.78647904624823        2.73125161992537\\nH           -2.22611244581904        2.70301729471234        0.50110212124385\\nH           -2.99985284193730        3.18994139981229       -1.03050889973847\\nH           -1.63349474578032        2.03466411354137       -1.03955313409569\\nH           -2.54730031800108       -1.27917035289619       -2.98938941257321\\nH           -1.48159623413253        0.09902762857326       -2.67574313972801\\nH           -3.23126925003382        0.29948887980484       -2.56704472661336\\nH           -0.50661726837273       -3.56999852493585       -0.69551917956681\\nH           -1.80560425307835       -3.46125091133668       -1.90032288157464\\nH            2.50995052864855       -2.16697076741868       -1.08166444090271\\nH            4.31756263681770       -0.79320919020449        0.75725086025967\\nH            2.66242438236966        2.99767684695977        1.88051099794306\\nH            0.63988888047586        2.59889369334667        0.48775691821115\\nH            6.21832397447998        0.49063426826438        1.34560224114087\\nH            5.45886916116869       -0.28992675508665        2.76297548122559\\nH            6.44262133139578        1.17934564943254        2.97225298232626\\n43\\n energy: -70.318288130761 xtb: 6.7.0 (08769fc)\\nC           -3.32342176450343        0.73132387257583        2.43280031681871\\nC           -2.78177571151157       -0.18361135111416        1.38021776839209\\nC           -2.16501307270516       -1.38417050486095        1.70467958034159\\nN           -1.64620953611644       -2.20753652740290        0.81556703307693\\nC           -1.73739017120931       -1.90455765455551       -0.47265585931532\\nC           -2.36930245219247       -0.75141419629755       -0.93894693256602\\nC           -2.86655189294730        0.13154938296304        0.02245185078670\\nO           -3.46351384190638        1.31042784693550       -0.37938059943795\\nC           -2.53367086324858        2.36634948302832       -0.54164067403878\\nC           -2.51195441274310       -0.43971369604908       -2.39501806498553\\nC           -1.11100955148129       -2.89516418031835       -1.39250293177779\\nS            0.05191210053988       -2.11592682834699       -2.62728264286589\\nO            1.11802843781121       -3.10988950234311       -2.80815725230222\\nC            0.78839344262671       -0.96959936376729       -1.45178196537679\\nN            0.35761777859090        0.20788249770987       -1.10588721607124\\nC            1.29049974274643        0.68103968911538       -0.21577696237210\\nC            2.32263006224471       -0.27440480652250       -0.06018269443616\\nN            1.96582061451356       -1.31858952941932       -0.86743069930320\\nC            3.41954352099587       -0.06742253307635        0.76383836479761\\nC            3.46479636720544        1.13706662405489        1.44663567902404\\nC            2.43452334202011        2.08896334727754        1.30501400002902\\nC            1.35190916638370        1.88171370515601        0.48735720931079\\nO            4.46924596489234        1.50996578963238        2.28620146696776\\nC            5.55784230174292        0.63665103407586        2.46185769231866\\nH           -2.52180447059042        1.34118330298553        2.85072398393439\\nH           -3.76746679918890        0.15739628672862        3.24308008272265\\nH           -4.07467594940908        1.39020711177191        2.00420814087838\\nH           -2.07123540718405       -1.69504486641587        2.73714951216400\\nH           -2.17876652964156        2.72436293594227        0.43128856167742\\nH           -3.06693453613115        3.16764372261983       -1.05053997297222\\nH           -1.67354821670929        2.04836069551044       -1.13716307021372\\nH           -2.68436155263622       -1.34214322230852       -2.97627258146574\\nH           -1.60997592304619        0.04678344819482       -2.76168969014556\\nH           -3.34980761427797        0.23992094232709       -2.53555635026111\\nH           -0.53248624620114       -3.62292200495403       -0.82645516596815\\nH           -1.85448330793124       -3.41579127119236       -1.99832485816936\\nH            2.45031821747460       -2.18197293287768       -1.05843070815774\\nH            4.18769696992922       -0.81833775664800        0.85090824338856\\nH            2.53414984747871        3.00272677228304        1.86896792196176\\nH            0.57544271196666        2.62173671450290        0.38355072105879\\nH            6.07636603126082        0.44664286855369        1.51637108346811\\nH            5.24354427455369       -0.31510376219118        2.90241933100180\\nH            6.23507992850354        1.14341741668079        3.14731534808425\\n43\\n energy: -70.319365459719 xtb: 6.7.0 (08769fc)\\nC           -2.92113163702166        0.80598659371270        2.47194653517273\\nC           -2.49539724526415       -0.13622612027437        1.39072906041499\\nC           -1.80756756112434       -1.30768629684462        1.67534913729512\\nN           -1.38722902240266       -2.15573021756343        0.75750794497271\\nC           -1.65298701385399       -1.90530890140350       -0.51768855839474\\nC           -2.35805076406014       -0.77736829566080       -0.94050603197258\\nC           -2.75969185113012        0.12672185746250        0.04458851741879\\nO           -3.43463126582553        1.27485529919027       -0.32075063847196\\nC           -2.56700743042046        2.36002526780335       -0.59508126955753\\nC           -2.65132207078467       -0.50880568327127       -2.38237780133649\\nC           -1.10603207294573       -2.89656842703976       -1.48312075258347\\nS            0.04856503050275       -2.10302178216712       -2.72564538225224\\nO            1.12467695903318       -3.08473387000733       -2.90936062742203\\nC            0.76423105957471       -0.95243343017241       -1.54193302704000\\nN            0.33432956163317        0.23353799085885       -1.22594651334312\\nC            1.22198159467272        0.69410149370004       -0.28438599979688\\nC            2.22570266885609       -0.27885743092154       -0.06523609785612\\nN            1.89915698460141       -1.31964502402061       -0.88930143169648\\nC            3.27428910514576       -0.08941620461320        0.82325314403532\\nC            3.29836924128047        1.11450212113939        1.50801379898142\\nC            2.29486204651856        2.08345642152994        1.30416705880820\\nC            1.26147463685232        1.89428625560701        0.42099270229845\\nO            4.25606442659893        1.47080432342079        2.40741885912521\\nC            5.32049327619682        0.58198488212439        2.64340744836870\\nH           -2.09090907114112        1.45474162067808        2.75342278173420\\nH           -3.23226670511416        0.25418268623642        3.35613748483957\\nH           -3.74528918674434        1.42487590698492        2.12540828849924\\nH           -1.56953354192860       -1.57277413288480        2.69735300242733\\nH           -2.14273163419507        2.76337315239585        0.33095388364850\\nH           -3.17576694629800        3.12321319500973       -1.07733768274051\\nH           -1.75080070369977        2.05599208061643       -1.25613677831514\\nH           -2.88662095195786       -1.42650071466483       -2.91583372831383\\nH           -1.78765302829252       -0.04308283443526       -2.85420493643588\\nH           -3.49535723245550        0.17302570104994       -2.45827710127399\\nH           -0.52653094188127       -3.65556120876865       -0.96099946912457\\nH           -1.88486240260066       -3.37219448048029       -2.07934582576729\\nH            2.37865575414404       -2.19265473546635       -1.04473254039879\\nH            4.02247293147115       -0.85345667507655        0.95716429294008\\nH            2.37519527289118        2.99577785205727        1.87360235204330\\nH            0.50663068366729        2.64811478943222        0.26883691204385\\nH            5.88913225677356        0.38542772105860        1.72858264887412\\nH            4.96833773073638       -0.36558155395619        3.06386758976539\\nH            5.96475005996601        1.07861980758615        3.36702575038747\\n43\\n energy: -70.320412055041 xtb: 6.7.0 (08769fc)\\nC           -2.39921620713478        0.89890293474081        2.47766127286355\\nC           -2.12801558860077       -0.07939173918974        1.37923300154205\\nC           -1.36821925855708       -1.22043613196203        1.59509253868996\\nN           -1.07069295866042       -2.09604013942182        0.65530733758723\\nC           -1.53775409405676       -1.90322420716173       -0.57167763931005\\nC           -2.33275405497790       -0.80927125183468       -0.91982979632069\\nC           -2.61059655981146        0.12046876187982        0.08332723263746\\nO           -3.37487749096126        1.23122714208975       -0.21424819551730\\nC           -2.60227408427961        2.34553310468080       -0.62301328247999\\nC           -2.82998614589906       -0.60024481003552       -2.31478083013828\\nC           -1.08532500659562       -2.89095158017192       -1.58537933331545\\nS            0.05050168177091       -2.08631530617963       -2.84973402943074\\nO            1.13521350093007       -3.05626653529492       -3.03818847367325\\nC            0.74822595082408       -0.92551781669376       -1.66494365066380\\nN            0.32721688220402        0.27419649708649       -1.39267294511335\\nC            1.15178036518308        0.71995962387529       -0.38854921559137\\nC            2.10625180944115       -0.27859385736185       -0.08261149188993\\nN            1.81895307649319       -1.31662865133504       -0.92409519343836\\nC            3.08091961216426       -0.11334197968132        0.89061061505680\\nC            3.07868721698809        1.09154179366271        1.57391558579679\\nC            2.12410573067057        2.08700490437870        1.28233869528579\\nC            1.16629843413879        1.92277443167054        0.31309290438034\\nO            3.96346145276823        1.42396063449761        2.55377533786154\\nC            4.98074390022106        0.50848595581351        2.87917126886194\\nH           -1.55608727851474        1.58244261480563        2.58552412250778\\nH           -2.53564539934441        0.38017212500113        3.42386738791637\\nH           -3.29211337061572        1.47726575026617        2.25257898309699\\nH           -0.96247914124149       -1.43445765697580        2.57540181590792\\nH           -2.09199108103333        2.80208896028586        0.23176121291751\\nH           -3.30253896419421        3.06130671458316       -1.05102164716708\\nH           -1.85681717450411        2.05648045834225       -1.36903205944415\\nH           -3.18489501890921       -1.53166706755670       -2.75016579148201\\nH           -2.02723400261618       -0.20989304634298       -2.93876125886407\\nH           -3.64593202029571        0.11825020930810       -2.30615764425334\\nH           -0.50150995769602       -3.68084383500255       -1.11702602471323\\nH           -1.90482177021493       -3.31920904749495       -2.16012133881266\\nH            2.28154905101242       -2.20618978930083       -1.02518153433955\\nH            3.79329172574539       -0.89711203607530        1.08957604288269\\nH            2.18063650335351        2.99937533421140        1.85472390505082\\nH            0.45062432821371        2.69766477248769        0.09292945500520\\nH            5.62045997636232        0.29667416725849        2.01619231096602\\nH            4.56985134136382       -0.42943232750426        3.26664757323979\\nH            5.57300508883149        0.98925092161268        3.65598977590674\\n43\\n energy: -70.321025794389 xtb: 6.7.0 (08769fc)\\nC           -2.01841057698502        0.94069087449689        2.45704213301070\\nC           -1.86459861886405       -0.05520408836447        1.35210668322594\\nC           -1.06552286403610       -1.18023778669001        1.49890192131470\\nN           -0.86010253808822       -2.06783287941053        0.54625352029715\\nC           -1.46663222095923       -1.90540896789398       -0.62370711232338\\nC           -2.31587177327348       -0.83034346755025       -0.89619379922430\\nC           -2.49720501143892        0.11190377638203        0.11731539083481\\nO           -3.31231744891639        1.20276355017617       -0.11020580661498\\nC           -2.61218746074480        2.33429967771388       -0.59492784878230\\nC           -2.96543450075164       -0.65704798793257       -2.23227476993484\\nC           -1.08973579855391       -2.88294182399122       -1.67509350296193\\nS            0.07858437832752       -2.09340243256220       -2.93041362071513\\nO            1.15938878211055       -3.07198344570639       -3.08881830943851\\nC            0.75708177881313       -0.92033505940754       -1.74698051593370\\nN            0.35052297285661        0.29430958025732       -1.52357575788955\\nC            1.12654789456391        0.74128348871394       -0.48189352269933\\nC            2.03303549428250       -0.27494938151193       -0.09807364283806\\nN            1.77392089337807       -1.32075940086475       -0.93837122785793\\nC            2.94158409837665       -0.11903903327698        0.93843648907468\\nC            2.92280322344455        1.09549506660316        1.60402803514333\\nC            2.02144404574633        2.11224513558376        1.22887404520643\\nC            1.12953560101684        1.95706252620555        0.19711271190605\\nO            3.74067773448632        1.41789104938897        2.64374941177678\\nC            4.70002086204134        0.47700183224039        3.06003124859519\\nH           -1.17746166102594        1.63553440525476        2.45058941179754\\nH           -2.03320728278529        0.43874950736839        3.42185675354561\\nH           -2.94027755113306        1.50332413809754        2.33053677968357\\nH           -0.54547896032480       -1.36763060212087        2.42948262123292\\nH           -2.02846731482667        2.80578290422884        0.20234460875954\\nH           -3.37000831013671        3.03133670274649       -0.94968571215574\\nH           -1.94030556870176        2.06132861508459       -1.41361701805527\\nH           -3.43252763767403       -1.58310402782881       -2.56198515849193\\nH           -2.22196582526868       -0.36257679846314       -2.97203035891282\\nH           -3.72774585242228        0.11509874548180       -2.17101361791044\\nH           -0.53299335956859       -3.71450406464298       -1.24795778423308\\nH           -1.93725316767467       -3.24673379519373       -2.25139446961040\\nH            2.21148744151468       -2.22715405219819       -0.98308754298270\\nH            3.61624643733083       -0.91806880901735        1.19842397944156\\nH            2.06440089145873        3.03313993269936        1.78885431135931\\nH            0.45523191669801        2.74814264269841       -0.08692410635785\\nH            5.39868267560537        0.23283696573062        2.25319267268817\\nH            4.23043900953837       -0.44209197790087        3.42538461504072\\nH            5.24407617253326        0.95112776533290        3.87523486199054\\n43\\n energy: -70.321042392420 xtb: 6.7.0 (08769fc)\\nC           -1.88675328460444        0.92108791153370        2.44275954100117\\nC           -1.77947341634499       -0.06596100988043        1.32447411067671\\nC           -0.97264837673579       -1.19033303972797        1.42927674008943\\nN           -0.80924571206856       -2.07295492585658        0.46467060654973\\nC           -1.47166786674265       -1.90985448517403       -0.67529680783358\\nC           -2.32889717484354       -0.83087900351850       -0.90573661023965\\nC           -2.46546702200852        0.10670346737113        0.11939009491773\\nO           -3.29031439918521        1.19816008541146       -0.06806036407177\\nC           -2.60731737023206        2.34442801223412       -0.54154854324320\\nC           -3.03648998047230       -0.65114198934224       -2.21133651899943\\nC           -1.13408731824862       -2.87707488006593       -1.74784927518433\\nS            0.14601864107032       -2.14814935677656       -2.93632853437186\\nO            1.20833778853077       -3.15727504460760       -2.99596850097583\\nC            0.78623739258460       -0.95615873649402       -1.75167683161539\\nN            0.38217010641297        0.26561646506068       -1.56527553156872\\nC            1.13459074397751        0.72944669286314       -0.51369412693851\\nC            2.02335909375091       -0.28469568598941       -0.08458489071499\\nN            1.78074632750611       -1.34502668928968       -0.91074221002756\\nC            2.90502543234637       -0.11278167504557        0.97247963217795\\nC            2.87771029773711        1.11576615266931        1.61140623329869\\nC            1.99511168934252        2.13090125712595        1.19018434982509\\nC            1.12996824765622        1.95996456187281        0.13821583413426\\nO            3.66976202517540        1.45457451201342        2.66601389015462\\nC            4.61471793015271        0.51841050384597        3.12374720721608\\nH           -1.05229837276166        1.62261881376758        2.40120451311268\\nH           -1.85058729776789        0.41172854810171        3.40311027814887\\nH           -2.81822670847779        1.47698388586296        2.36615093811021\\nH           -0.41013924489322       -1.38035349989922        2.33439110737457\\nH           -1.98688040702518        2.78680685191985        0.24422700247220\\nH           -3.37770726481798        3.05466955807330       -0.83821078821185\\nH           -1.97375115437669        2.09882737640045       -1.39888950066009\\nH           -3.57162667584946       -1.55715205978209       -2.49070502787130\\nH           -2.31786565206396       -0.41769974974762       -2.99615723230378\\nH           -3.75216713352967        0.16247527272786       -2.13156775018786\\nH           -0.65160307294312       -3.76080507590543       -1.33540384465785\\nH           -1.98391273181075       -3.15428229952983       -2.36566223967811\\nH            2.20181923052413       -2.26037930154503       -0.91867632144142\\nH            3.56635122466691       -0.91082037955379        1.26741127597303\\nH            2.03094851220959        3.06369914658461        1.73071503233201\\nH            0.47098268889692        2.75024704277967       -0.18181175916113\\nH            5.33031401086215        0.25425580321018        2.33833604437758\\nH            4.13259801741629       -0.39079322501915        3.49746374371676\\nH            5.14235923695447        1.00719919127984        3.94108203429972\\n43\\n energy: -70.320724642113 xtb: 6.7.0 (08769fc)\\nC           -1.77070897177798        0.88165840880039        2.43967301583044\\nC           -1.70858962248717       -0.08967861682110        1.30441161882806\\nC           -0.90302953337356       -1.21842827799234        1.36416449474520\\nN           -0.77925666988077       -2.08815966875846        0.38294768689260\\nC           -1.48427387952818       -1.91087592726702       -0.72979752053763\\nC           -2.34199383173784       -0.82320384492341       -0.91491165524158\\nC           -2.43713945266283        0.10087972004266        0.12765352564003\\nO           -3.26634783043488        1.19636741015973       -0.01340659386702\\nC           -2.59967727217878        2.35214998131901       -0.48565334638968\\nC           -3.09918695523371       -0.61974759216856       -2.18923812300844\\nC           -1.18278634835393       -2.86342492449767       -1.82529466122833\\nS            0.22295180347997       -2.22662394473213       -2.92699719012075\\nO            1.24928159953532       -3.27193387762863       -2.86271358839532\\nC            0.82406005569795       -1.00610405419487       -1.75231344453326\\nN            0.42286232500769        0.22269640420026       -1.61024510157885\\nC            1.15204982776541        0.71074567240332       -0.55321382653324\\nC            2.02308892967206       -0.29669819148237       -0.07391287692303\\nN            1.79641966062802       -1.37657770721907       -0.87823409096671\\nC            2.87724676664876       -0.10153761945450        1.00164587047043\\nC            2.84117979655630        1.14384306894340        1.60653390378630\\nC            1.97806291056130        2.15305096656691        1.13381709046014\\nC            1.13992447695774        1.95898816683594        0.06400579654422\\nO            3.60606698034710        1.50561948050627        2.67371629168767\\nC            4.53190361849336        0.57618811228319        3.18147789795529\\nH           -0.93616885486529        1.58157651595659        2.37655463450709\\nH           -1.69972535636934        0.35843018347703        3.39059923751276\\nH           -2.70278226188390        1.44084561786646        2.40653098885554\\nH           -0.30789536713854       -1.42213342704756        2.24524175999535\\nH           -1.93527732722662        2.76630378292214        0.27912030702489\\nH           -3.37916129646493        3.07563875501184       -0.71999474289654\\nH           -2.01398836271242        2.12980529736546       -1.38266064535948\\nH           -3.66020016355218       -1.51434541437820       -2.45466346390101\\nH           -2.40949352125327       -0.38911032156486       -3.00058100911824\\nH           -3.79873152667243        0.20338167669665       -2.07252621526702\\nH           -0.79674565415690       -3.80083862609030       -1.42976190307694\\nH           -2.02156493006416       -3.03957824295762       -2.49182465359309\\nH            2.20102412670334       -2.29904218266697       -0.84632048027800\\nH            3.52507767535457       -0.89536855996038        1.33532413720701\\nH            2.00667427705873        3.10029755204596        1.64914966375647\\nH            0.49594185477777        2.74473540439028       -0.29534116147431\\nH            5.26611950128992        0.28620180924109        2.42284441684680\\nH            4.03359994206150       -0.31948415898030        3.56654316471692\\nH            5.04118986138270        1.08349019371214        3.99917779102703\\n43\\n energy: -70.320118863999 xtb: 6.7.0 (08769fc)\\nC           -1.65931919347164        0.80679229078010        2.45482932614324\\nC           -1.64230523451896       -0.13679649194534        1.29489612540313\\nC           -0.85152919989501       -1.27751565707838        1.30405950696044\\nN           -0.76692750155183       -2.12292060119769        0.29844560896772\\nC           -1.49977617888600       -1.91103967057334       -0.79035949225384\\nC           -2.34742471285035       -0.80750383952844       -0.92376585699315\\nC           -2.40133202740334        0.09080339065693        0.14457076488472\\nO           -3.22373584860789        1.19743975151880        0.05773094418217\\nC           -2.56282479278700        2.35986986230371       -0.40468237877951\\nC           -3.14509804108340       -0.56034707997455       -2.16583704841157\\nC           -1.23638327044020       -2.84027186974379       -1.91564588484130\\nS            0.30665826542997       -2.33993683769839       -2.90036860875658\\nO            1.27056981029469       -3.42204268715038       -2.67811967196497\\nC            0.86822602715225       -1.07462599587301       -1.75477571851560\\nN            0.47450084139014        0.16222844431236       -1.67432425755791\\nC            1.17532192905811        0.68516775567963       -0.61467511055599\\nC            2.02089570281817       -0.31034856703099       -0.06862826394632\\nN            1.81106417177199       -1.41782485632275       -0.83853275659307\\nC            2.84100524236058       -0.08079536651177        1.02660033287513\\nC            2.79732800642985        1.18730573728164        1.58160913692583\\nC            1.96107931856162        2.18510854249570        1.04121007801713\\nC            1.15664904731494        1.95692645684597       -0.04771063512554\\nO            3.52904822242388        1.58309489327127        2.66017883429193\\nC            4.42860806502204        0.66607945762712        3.23342253703817\\nH           -0.81835363683510        1.49839651846802        2.38440934941367\\nH           -1.56749144169926        0.25932151203951        3.39022726316503\\nH           -2.58485982953310        1.37762715674161        2.46266303314594\\nH           -0.23582459235011       -1.51174927665667        2.16320148046129\\nH           -1.84858182769679        2.73178401450993        0.33648726965167\\nH           -3.34079452875280        3.10460306008498       -0.56619929916757\\nH           -2.03282988539956        2.16493270191723       -1.34202990269161\\nH           -3.72414093644212       -1.44134528940855       -2.43772959669564\\nH           -2.48125587576147       -0.31065127118642       -2.99302894650103\\nH           -3.83262150729519        0.26475804937600       -2.00096869914870\\nH           -0.98243363144765       -3.83350380800183       -1.55016507588609\\nH           -2.04387679191924       -2.88645422127774       -2.63878277735998\\nH            2.19273610638405       -2.34699739329896       -0.75326459528482\\nH            3.46975256163512       -0.86665428010955        1.41171202348200\\nH            1.98294835201198        3.15179810195643        1.51950177350537\\nH            0.53396212267694        2.73426668042153       -0.45909640695338\\nH            5.18438193809044        0.34085734447682        2.51117408524487\\nH            3.90924220163274       -0.20988676190025        3.63566460514224\\nH            4.91574355413630        1.20004909966636        4.04762390508126\\n43\\n energy: -70.319317854648 xtb: 6.7.0 (08769fc)\\nC           -1.53559542620822        0.70481843960488        2.48398070024103\\nC           -1.57116571733606       -0.20351374566347        1.29665935513072\\nC           -0.80780762485229       -1.36181988359970        1.25066502254198\\nN           -0.76371341375720       -2.17317806272872        0.21536915579724\\nC           -1.51252903155847       -1.90954561817235       -0.85083514687953\\nC           -2.34131157625416       -0.78592577288230       -0.92628069418018\\nC           -2.35354831694210        0.07538985076319        0.17383040405838\\nO           -3.16083750832641        1.19663812042772        0.14931022967058\\nC           -2.50199669533240        2.36319659883528       -0.30414104527003\\nC           -3.16872965713019       -0.47976144690931       -2.13592500644310\\nC           -1.28630744912393       -2.80226872328595       -2.01455705457932\\nS            0.38135702518004       -2.47820640555206       -2.85774922482727\\nO            1.25266814628614       -3.58849564329843       -2.46266486251176\\nC            0.91454537919378       -1.15796586760601       -1.76162542362050\\nN            0.53374522332904        0.08545577720357       -1.75643271773277\\nC            1.20095043947694        0.65167622933504       -0.69719912178406\\nC            2.01201865880946       -0.32561092062349       -0.07116681279716\\nN            1.81984069066462       -1.46583064433938       -0.79641619104799\\nC            2.79014539023909       -0.05242771780486        1.04463986846112\\nC            2.73894225446520        1.24081504705987        1.53751917851309\\nC            1.93803529127292        2.22057696639276        0.91657668544388\\nC            1.17574645322628        1.94929876482504       -0.19272304077834\\nO            3.42963426874826        1.67972928606284        2.62670394746371\\nC            4.29149872981978        0.78136193439162        3.28149109436621\\nH           -0.67906301960169        1.37707374457134        2.41189094365201\\nH           -1.43494681802773        0.12760149784836        3.40040964711391\\nH           -2.44529961575398        1.29854472878726        2.53201422432813\\nH           -0.17973769004243       -1.63930763626830        2.08776264245048\\nH           -1.73368883683621        2.68710066863875        0.40483030302586\\nH           -3.27071588161480        3.13029674922149       -0.38513726490808\\nH           -2.03652630798777        2.19860025862173       -1.28086338158986\\nH           -3.76871116815713       -1.34095894693824       -2.42504628273812\\nH           -2.52403633897823       -0.20606129930635       -2.97074520054510\\nH           -3.83924425074785        0.34769736513328       -1.92018524445517\\nH           -1.19513752865146       -3.84095119644872       -1.70135146242006\\nH           -2.03369183470768       -2.69517249977911       -2.79284675404361\\nH            2.17141528101732       -2.39863232313686       -0.64436274710795\\nH            3.39417329928924       -0.82508097602259        1.49119918863943\\nH            1.95255179674829        3.20876472990949        1.34904957507304\\nH            0.58013475437570        2.71279083622990       -0.66546785416778\\nH            5.07734761467026        0.41857181158463        2.61106011428233\\nH            3.74473811982012       -0.07183766314563        3.69614594855586\\nH            4.74485389126933        1.34655258803393        4.09414130561809\\n43\\n energy: -70.318356883790 xtb: 6.7.0 (08769fc)\\nC           -1.43273022521919        0.58590632915122        2.52725435492729\\nC           -1.51047283413997       -0.27956642623450        1.31028777020703\\nC           -0.77792473608082       -1.45389893524048        1.20956652002038\\nN           -0.76326549755968       -2.22128545334081        0.14058002685167\\nC           -1.51614994344076       -1.89904274549672       -0.90538072552033\\nC           -2.32730064049468       -0.75981395886606       -0.92121417885508\\nC           -2.30402780681179        0.05786708899350        0.21167735956140\\nO           -3.08821406886845        1.19554440580904        0.24767051344758\\nC           -2.42179375655013        2.36136935319910       -0.19536504311034\\nC           -3.17844792988147       -0.39336656273077       -2.09764843907071\\nC           -1.31680597785780       -2.74223796081505       -2.11308162780304\\nS            0.44348803662126       -2.63117832471656       -2.79671787518137\\nO            1.19346891561100       -3.75118632032442       -2.22389449370490\\nC            0.96765983426621       -1.24969540037458       -1.77286856134948\\nN            0.60545967336792       -0.00333560839705       -1.84977951641981\\nC            1.23593827860687        0.61011388786817       -0.79405049773087\\nC            2.00466096159969       -0.34286680593456       -0.08270977576534\\nN            1.82954940856141       -1.51667877223476       -0.75703243056473\\nC            2.73413106604767       -0.02148186982169        1.05293501326853\\nC            2.67595756354563        1.29539341842649        1.47790338309310\\nC            1.91705413671412        2.25072372218937        0.77214958199499\\nC            1.20372648466026        1.93165474903458       -0.35699990024501\\nO            3.31888944433918        1.78096641738805        2.57689558069383\\nC            4.13900112244006        0.90818813668504        3.31458822200638\\nH           -0.56304113468055        1.24182176339589        2.46136229401491\\nH           -1.32866984207460       -0.02439345802348        3.42158843073265\\nH           -2.32732750482084        1.19809269966956        2.61216936557107\\nH           -0.14827041859925       -1.78121121386954        2.02727714823423\\nH           -1.59728733249507        2.62749825543060        0.47350849159229\\nH           -3.16821355694082        3.15420875011908       -0.19001517928509\\nH           -2.02736631342385        2.23005209554461       -1.20783775646334\\nH           -3.78129676393870       -1.24044967310793       -2.41998490277589\\nH           -2.55141366215648       -0.07225974623786       -2.92921587407218\\nH           -3.84678614747732        0.41919768885358       -1.82521982066153\\nH           -1.41281025381206       -3.80158762399914       -1.87868120018666\\nH           -1.96570221107917       -2.46832350896015       -2.93717027328972\\nH            2.14170448235082       -2.44898315685013       -0.53182553591158\\nH            3.30675486691471       -0.77677648606764        1.56567743005073\\nH            1.92316557080810        3.25910628493779        1.15539861639321\\nH            0.63949106732644        2.67622070930562       -0.89408779087452\\nH            4.95114508523069        0.50579269111295        2.70049916592689\\nH            3.56387113550789        0.08133380974377        3.74406320175412\\nH            4.56020242385706        1.50856675475604        4.11925592849738\\n43\\n energy: -70.317235461154 xtb: 6.7.0 (08769fc)\\nC           -1.38016588892747        0.49207525117178        2.56779680534042\\nC           -1.47728958572163       -0.33597286280904        1.32640893835198\\nC           -0.75998510895296       -1.51465571540283        1.18190239567780\\nN           -0.75491282230722       -2.24046869211577        0.08392748195353\\nC           -1.51109254948465       -1.87775710984216       -0.94445104769164\\nC           -2.32010083504035       -0.73701523919100       -0.91145125506971\\nC           -2.27553579814438        0.04496684538776        0.24542958550891\\nO           -3.03985973707831        1.19433984511140        0.32471570893701\\nC           -2.36236135741837        2.35500597670756       -0.11467581122968\\nC           -3.19280867356081       -0.33351042763717       -2.05980510335089\\nC           -1.31902020301915       -2.66694183392311       -2.19275760810119\\nS            0.48466111787720       -2.76031913570910       -2.73114095881492\\nO            1.11253417265635       -3.87393145515576       -2.01889090329894\\nC            1.02269028307533       -1.33228847296848       -1.77859718134920\\nN            0.67583515122808       -0.08753050278008       -1.92071863998710\\nC            1.27669855518175        0.56526555357402       -0.87105355575913\\nC            2.01168940837699       -0.36302262666395       -0.09444356260768\\nN            1.85060571372856       -1.56257102660844       -0.72606706364427\\nC            2.70026966434068       -0.00085658598502        1.05422150117708\\nC            2.63331935318276        1.33172739879599        1.42579186705814\\nC            1.90628083227369        2.26204654057285        0.65630549271698\\nC            1.23446966887439        1.90250765033407       -0.48627849260609\\nO            3.23576484862586        1.85575690934760        2.53046343199082\\nC            4.02526050369982        1.00843588355190        3.32869629001364\\nH           -0.50927611739540        1.14719362241970        2.51051553407823\\nH           -1.26764411298332       -0.14492299592800        3.44222202516261\\nH           -2.27079737301701        1.10544843658050        2.68131990543533\\nH           -0.13069306976250       -1.87854961456571        1.98427348485902\\nH           -1.49064596600279        2.56903069884380        0.51182860199029\\nH           -3.07998139287602        3.17003007855648       -0.03587384583068\\nH           -2.03257561234417        2.25109230332894       -1.15324867213727\\nH           -3.77869105781775       -1.17856677282283       -2.41632846456215\\nH           -2.58471967275767        0.04316218613156       -2.88235020308484\\nH           -3.87576445873459        0.44946858120507       -1.74024619943724\\nH           -1.58792703855170       -3.71380872888940       -2.05311199945113\\nH           -1.84697530318562       -2.24570578925796       -3.04098487989469\\nH            2.12835952148418       -2.48982904499134       -0.44217176599089\\nH            3.24736530263748       -0.73856800027006        1.61788433248636\\nH            1.90161000692089        3.28404089230314        1.00164961899530\\nH            0.69251990915699        2.62753574154523       -1.07104000251664\\nH            4.85380857923766        0.57683755739985        2.75792592956066\\nH            3.43044854604678        0.20287576040782        3.77179291028539\\nH            4.42463359645089        1.63794792020938        4.12214237483285\\n43\\n energy: -70.315881950946 xtb: 6.7.0 (08769fc)\\nC           -1.34978342170285        0.45564077671974        2.58728649826641\\nC           -1.45770302639252       -0.35017923146383        1.33214532430686\\nC           -0.72890986802821       -1.51683085042344        1.15422604509332\\nN           -0.72538440935464       -2.21559063780563        0.03880390966279\\nC           -1.49695567605365       -1.84169355618548       -0.97223739047792\\nC           -2.32494799387224       -0.71656054693663       -0.90235124022109\\nC           -2.27470288898276        0.04235474139715        0.26951057292778\\nO           -3.04787806469025        1.18340026762054        0.38146393818633\\nC           -2.38954058757866        2.35515230027692       -0.05708527143712\\nC           -3.21958243476232       -0.30477569617935       -2.03067623824006\\nC           -1.30027582198728       -2.58606896792050       -2.25033347511482\\nS            0.50891433548947       -2.86280014616220       -2.67473295464859\\nO            1.02486647089863       -3.97135211826513       -1.87212504741200\\nC            1.08438054720517       -1.41247787263164       -1.77674088524125\\nN            0.74498159102567       -0.17141342599309       -1.96154514454954\\nC            1.32764008997006        0.51026818149611       -0.92027412175728\\nC            2.04604860617534       -0.39669715344787       -0.10407881250627\\nN            1.89616160646946       -1.61377879799831       -0.70566315885496\\nC            2.70602149407448       -0.00373936713245        1.05118701742136\\nC            2.62440949345939        1.33709621047110        1.38913487941221\\nC            1.91275403545388        2.24568538242925        0.58064480201061\\nC            1.26990900523740        1.85580069006846       -0.56871235009107\\nO            3.19427587162440        1.88941380122422        2.49774191887659\\nC            3.95684321969710        1.06246299531734        3.34205134176664\\nH           -0.47724071565578        1.10897814030708        2.53519536935507\\nH           -1.23282517271108       -0.19717237227633        3.44944145970116\\nH           -2.23763507889879        1.06938970865382        2.71826865005903\\nH           -0.08601251421241       -1.89098410716740        1.94083942437417\\nH           -1.49486292327355        2.55773420892671        0.54036784826417\\nH           -3.10509495783845        3.16634654181746        0.06659049502531\\nH           -2.09883673169663        2.27607507523875       -1.10945276066725\\nH           -3.77544195730507       -1.15795481838363       -2.41420735623838\\nH           -2.63149289218277        0.12191860521391       -2.84363911099193\\nH           -3.92740697810208        0.44239689843108       -1.68057337782177\\nH           -1.71244179233614       -3.59445204582928       -2.20168403074309\\nH           -1.70540543216545       -2.05208229356827       -3.10356063748433\\nH            2.15311465504869       -2.53282749832938       -0.37930297550130\\nH            3.24068011065357       -0.72539079981937        1.64676409188998\\nH            1.89395224582240        3.27521477430026        0.90229909082878\\nH            0.73471697355546        2.56292950941521       -1.18108223154038\\nH            4.80753079105412        0.62262698762208        2.81151425314655\\nH            3.34895243730609        0.26374754431498        3.77979929570413\\nH            4.32420875953523        1.71018796262567        4.13630934525752\\n43\\n energy: -70.314364132069 xtb: 6.7.0 (08769fc)\\nC           -1.35095456264166        0.43585544074789        2.60258371539695\\nC           -1.45788354031721       -0.35431826370464        1.33738533427271\\nC           -0.70922206973312       -1.50402557036545        1.13527820155222\\nN           -0.69465421188560       -2.17842032944499        0.00497876264099\\nC           -1.48396391712076       -1.80800747784911       -0.99152199131042\\nC           -2.33982946088383       -0.70592107326555       -0.89335680107725\\nC           -2.28986384246413        0.03967426553320        0.28676204845280\\nO           -3.07101530294985        1.17370360707391        0.41512303909923\\nC           -2.42038701085253        2.35307799490424       -0.01471709142031\\nC           -3.25560630504180       -0.30280124713386       -2.00722217348923\\nC           -1.27167002595695       -2.50362988053442       -2.29710314144937\\nS            0.52059620358350       -2.93709416283150       -2.62860799806274\\nO            0.93681739693089       -4.04063530892886       -1.76482469939825\\nC            1.13814258198637       -1.47797175565846       -1.77040264841412\\nN            0.80665524765094       -0.23991455862057       -1.98712592240862\\nC            1.37795989283026        0.46311270325068       -0.95407569208890\\nC            2.08416665242080       -0.42743999501056       -0.10990227002208\\nN            1.94061687789835       -1.65737769665218       -0.68811348092347\\nC            2.72389301236634       -0.01114311099787        1.04853650141769\\nC            2.62994409554295        1.33495917636268        1.36210455674508\\nC            1.92755798601155        2.22636237278527        0.52709483928947\\nC            1.30710734026387        1.81393094893738       -0.62668388298529\\nO            3.17603014937156        1.90871341662768        2.47213117368504\\nC            3.92253727348841        1.09936809530586        3.34704153913036\\nH           -0.48113909238077        1.09330683665798        2.55716416609444\\nH           -1.22928968904499       -0.22748990913419        3.45606654717680\\nH           -2.24110300844595        1.04400402153883        2.74353047284613\\nH           -0.05461228607724       -1.87998744780535        1.91127416022325\\nH           -1.51551825454629        2.54666831150834        0.57040289067123\\nH           -3.13396709225528        3.16173878471901        0.13401053515263\\nH           -2.14708549969167        2.29197899049780       -1.07309787731085\\nH           -3.78509597053620       -1.16770886045837       -2.40160972388626\\nH           -2.68869169730597        0.15506292653392       -2.81853758247778\\nH           -3.98337036983420        0.41644849944888       -1.63979102159495\\nH           -1.79866238280949       -3.45821416308530       -2.34397458313497\\nH           -1.55637269106793       -1.87677165287144       -3.13705592081427\\nH            2.18171669131107       -2.56834628393388       -0.32954956583606\\nH            3.25029296133034       -0.71965567679119        1.66691455070835\\nH            1.89606990493551        3.26044426196526        0.83276838685383\\nH            0.77578521391182        2.50671749912983       -1.25856223788139\\nH            4.78701693266176        0.65469765376008        2.84349897831864\\nH            3.30817878336851        0.30519399141892        3.78412591825859\\nH            4.26887408595044        1.76185362633799        4.13858698799713\\n43\\n energy: -70.312225218563 xtb: 6.7.0 (08769fc)\\nC           -1.66841628788010        0.40509142943454        2.63941116945824\\nC           -1.65055067031361       -0.33270412270705        1.33818078113474\\nC           -0.77710347705634       -1.38713218876007        1.11762099390429\\nN           -0.67405796627457       -2.02237449365934       -0.03060360483486\\nC           -1.47725467281959       -1.70453380304141       -1.03185905506450\\nC           -2.43644708975059       -0.69309264522560       -0.92673401124201\\nC           -2.49039862889157        0.01397990769672        0.27690337110390\\nO           -3.38155067639986        1.06319470385436        0.41611605762567\\nC           -2.82219212182919        2.31561337143590        0.07401944984616\\nC           -3.34923320625571       -0.34540529859846       -2.06074704683516\\nC           -1.19796552924153       -2.36198327686122       -2.34643976252808\\nS            0.55344192670552       -2.98110803508855       -2.52368363967850\\nO            0.80481457457001       -4.08200391140622       -1.59588761521653\\nC            1.23839302027590       -1.53876396188303       -1.68765594073463\\nN            0.98856963253200       -0.29358991422818       -1.96422023481237\\nC            1.60093876415458        0.41972026142246       -0.96222222156334\\nC            2.24904893417231       -0.47296394768035       -0.07503499252209\\nN            2.03017821783348       -1.71797725457988       -0.59669253275010\\nC            2.91530752459866       -0.04255444661599        1.06309339522820\\nC            2.91084925182100        1.32109536261267        1.30948147172904\\nC            2.26756402745581        2.21436546321541        0.43072572700559\\nC            1.61737188177650        1.78650534335970       -0.70096585077458\\nO            3.49579631571587        1.91209511869811        2.39080845562921\\nC            4.18005086253888        1.09864108720373        3.31116584539737\\nH           -0.85830993383514        1.13557553559963        2.66692700413553\\nH           -1.52638706831329       -0.28594589811712        3.46752852577715\\nH           -2.61451088000038        0.92599312651302        2.76526565584013\\nH           -0.10105863634582       -1.71637769395846        1.89630110641487\\nH           -1.95440327400414        2.55186386878315        0.69868857925235\\nH           -3.60540529236244        3.05304730565691        0.24224003948501\\nH           -2.51380118317057        2.33866406472967       -0.97674522068057\\nH           -3.79471196757870       -1.24372075391635       -2.48344115783489\\nH           -2.79638467162303        0.16491987833941       -2.85028839431058\\nH           -4.14236223131232        0.30898020351163       -1.70789193169472\\nH           -1.83648131767617       -3.23191542768062       -2.51069430209440\\nH           -1.31385826577125       -1.65920927268251       -3.16833992863072\\nH            2.21107949383377       -2.62239895037198       -0.19074509159712\\nH            3.39535804733448       -0.75252278514597        1.71679580917084\\nH            2.30440652674109        3.26217790626852        0.68441683870046\\nH            1.12799886240788        2.47987795895739       -1.36535232841297\\nH            5.01446965056979        0.57056866871771        2.83815533127055\\nH            3.51000425524261        0.37151868434702        3.78215151669035\\nH            4.56720427839861        1.77478783181777        4.07177473901108\\n43\\n energy: -70.311884242715 xtb: 6.7.0 (08769fc)\\nC           -2.42280316329622        0.37798306420134        2.65142467860771\\nC           -2.12729182185433       -0.23143406084497        1.31703199720275\\nC           -0.99941162752207       -1.01282791589187        1.11470814258418\\nN           -0.69249957132288       -1.57008497591366       -0.03873943935939\\nC           -1.49828528066807       -1.41563110122274       -1.07313325644185\\nC           -2.67485514675623       -0.66347189981568       -1.00320393836971\\nC           -2.97059022506440       -0.05053824040381        0.21634434387933\\nO           -4.11878158079316        0.71156003581599        0.33559310079382\\nC           -3.90212420008421        2.09806213101868        0.16869194272544\\nC           -3.57607971818674       -0.50604019329979       -2.18679756210282\\nC           -1.07451750398262       -2.04375446070024       -2.36184262863173\\nS            0.58972162719695       -2.86673564601300       -2.30260152776377\\nO            0.58234790352612       -3.93248978673673       -1.29980270773757\\nC            1.39261923863127       -1.48307150958844       -1.48256929139416\\nN            1.40732026364561       -0.24772081683906       -1.88542640838383\\nC            2.15501713471099        0.41998655896409       -0.94463309367973\\nC            2.60723749646659       -0.49433482282740        0.03677689479873\\nN            2.13019294626442       -1.71215263969476       -0.36243936126714\\nC            3.37290676669070       -0.10424249037400        1.12582537006568\\nC            3.68134712305558        1.24346895234439        1.21987628971300\\nC            3.23663113427329        2.15797340561797        0.24596646544543\\nC            2.48090752406617        1.76815155477636       -0.83294550109142\\nO            4.41448617195949        1.79926467089420        2.22791604261746\\nC            4.90565978768215        0.95863128878644        3.24158719630954\\nH           -1.84935643844793        1.29596171233874        2.78636470228092\\nH           -2.14579753107647       -0.30945107758307        3.44766284245384\\nH           -3.48182387620118        0.61074329177965        2.73342299545845\\nH           -0.30106065362111       -1.19341972106450        1.92165065419658\\nH           -3.19540141502650        2.48907027399948        0.90711498094148\\nH           -4.87173280049059        2.57346244336916        0.30584082871456\\nH           -3.52259945993625        2.32170672183361       -0.83439629600713\\nH           -3.84939217113473       -1.47983346882294       -2.58970202557414\\nH           -3.07445591307817        0.05481096126699       -2.97560620388359\\nH           -4.48274491407427        0.01816883027419       -1.89686737658416\\nH           -1.78429514494171       -2.80726240731919       -2.68453084456001\\nH           -0.96995314655366       -1.29194523295028       -3.14394585609884\\nH            2.13791657664195       -2.59471768748678        0.12289407342378\\nH            3.69808915839735       -0.82990722038131        1.85368973785708\\nH            3.51523278952562        3.19100273523131        0.38143735647824\\nH            2.14674759791630        2.47773077347493       -1.57200661494801\\nH            5.58416568809992        0.19832204570733        2.84087902082349\\nH            4.09250650556114        0.46817178688725        3.78729838561057\\nH            5.45480086978299        1.60683313717614        3.92271889089905\\n43\\n energy: -70.312026358464 xtb: 6.7.0 (08769fc)\\nC           -2.58165584341317        0.44186587728637        2.63378422667329\\nC           -2.23343095068412       -0.13118769437784        1.29591285025085\\nC           -1.01380209507236       -0.75263353445805        1.07286344971377\\nN           -0.66042408648288       -1.27782120883543       -0.08235474246710\\nC           -1.50009007468111       -1.24275588732455       -1.10031923450837\\nC           -2.76894391514313       -0.66102244253412       -1.00721672248419\\nC           -3.11948341044021       -0.08241966283852        0.21464609261916\\nO           -4.36342197181779        0.50509482021878        0.35692612778693\\nC           -4.35651281146029        1.91008041720097        0.20672645170466\\nC           -3.71420108682877       -0.64870203024225       -2.16687958413278\\nC           -1.02573578949136       -1.83486666364694       -2.38769324508867\\nS            0.57781537840152       -2.76213589567743       -2.25419253994378\\nO            0.46205442155483       -3.78178909795966       -1.20953137847628\\nC            1.47052790804897       -1.41834696866718       -1.46385999449461\\nN            1.63505286650691       -0.21401824000538       -1.92298878386618\\nC            2.42171476539513        0.41420100198664       -0.98608310919658\\nC            2.73843689468395       -0.49436808200232        0.05226442697465\\nN            2.14355679007166       -1.67089413117693       -0.30862635569984\\nC            3.50536622586945       -0.13742395211063        1.15159393240821\\nC            3.95922687396526        1.17103241589645        1.19609499936008\\nC            3.65497831985748        2.07815535672708        0.16317967535411\\nC            2.89562728909776        1.72114309319309       -0.92475929138528\\nO            4.71319420303130        1.69218955943872        2.20768401695586\\nC            5.05142432668402        0.85904585421068        3.28779228270444\\nH           -2.16326455540688        1.44357600615539        2.74084900703895\\nH           -2.16967657467007       -0.17859837155505        3.42650456324105\\nH           -3.66142274604734        0.49923059053492        2.74999392773086\\nH           -0.27974657269563       -0.82867182795605        1.86466151775112\\nH           -3.69987379785779        2.39353852345603        0.93624190362785\\nH           -5.38248827635447        2.23531695593741        0.36978111262816\\nH           -4.03649422039787        2.19882894962350       -0.80067423994248\\nH           -3.91231767432532       -1.66495597004275       -2.50505880497644\\nH           -3.28902099660773       -0.09320314646402       -3.00243627298317\\nH           -4.65675259846076       -0.19415977657215       -1.87422869872141\\nH           -1.75451478604945       -2.53526859791137       -2.79689274104272\\nH           -0.81815695687514       -1.05565433362509       -3.12171916288043\\nH            2.04293686446227       -2.52715919154128        0.21215947648654\\nH            3.72243370510727       -0.85787109886663        1.92365214441015\\nH            4.04456219726836        3.07907932356806        0.26074608108341\\nH            2.66928263490916        2.42573609366929       -1.70814033907528\\nH            5.65672712578420        0.00549354354918        2.96488065088729\\nH            4.15993661059923        0.49480464495050        3.80965917434026\\nH            5.63657738994940        1.47751377877976        3.96658414962181\\n43\\n energy: -70.312328024143 xtb: 6.7.0 (08769fc)\\nC           -2.77601513317033        0.40976329989191        2.64109389655603\\nC           -2.36262875483341       -0.09022206501461        1.29244827565401\\nC           -1.08212560919278       -0.57090878304036        1.06206773854335\\nN           -0.67420086267114       -1.03968348033523       -0.09895981720027\\nC           -1.51280608509740       -1.08185449749702       -1.11890886650523\\nC           -2.83438431118969       -0.63308897091134       -1.02217804840493\\nC           -3.24562489412293       -0.11469992739109        0.20798652214659\\nO           -4.54516696365813        0.33503959334295        0.35561334471104\\nC           -4.68695213406793        1.73601571903788        0.24186164854617\\nC           -3.77661988144690       -0.69411840273326       -2.18353488713144\\nC           -0.97437230275242       -1.63175337473840       -2.39829862146976\\nS            0.53502041458174       -2.68982068567471       -2.15930648186464\\nO            0.29099629449823       -3.63485112005412       -1.06741707667679\\nC            1.51544299401210       -1.38624070638027       -1.40922312665491\\nN            1.80640313017550       -0.23493108941784       -1.93634731492518\\nC            2.62682701193764        0.37525422579365       -1.01714312491359\\nC            2.82904155383588       -0.48998052613913        0.08482406671844\\nN            2.12989902995750       -1.62467257489491       -0.21907452542359\\nC            3.60568244681912       -0.13929136941016        1.17941785352159\\nC            4.18843275960729        1.11758284782286        1.15257677052230\\nC            3.99678531188192        1.98118217724980        0.05716078156603\\nC            3.22708529897002        1.63070341145249       -1.02547962695771\\nO            4.97365382116509        1.62564052100834        2.14693752972348\\nC            5.21430460787448        0.83193691610763        3.28131569316932\\nH           -2.46647809979923        1.44688710760021        2.77675214096184\\nH           -2.30527043925503       -0.18400891362780        3.42149350886586\\nH           -3.85634311141584        0.34969844528721        2.75026917713384\\nH           -0.34495463790743       -0.58003657404819        1.85463276505368\\nH           -4.08776437674815        2.26750067621154        0.98727482695600\\nH           -5.74214469193512        1.94616330867129        0.40721066364773\\nH           -4.39659144948446        2.08355688597990       -0.75604697135625\\nH           -3.90184415700753       -1.72358832437958       -2.51684138809958\\nH           -3.39328823905897       -0.11128825564456       -3.02041732996158\\nH           -4.75043679798961       -0.31020535090644       -1.89241073026582\\nH           -1.71068244431606       -2.24578272632094       -2.91512794884126\\nH           -0.62871653863354       -0.83568038024104       -3.05812537585973\\nH            1.93986926982665       -2.43418643651878        0.34903403787169\\nH            3.73491637908934       -0.82572267141616        2.00054884547967\\nH            4.48272111232487        2.94284090299792        0.10173867718920\\nH            3.08787623748738        2.30130626302497       -1.85732410884177\\nH            5.73527770560796       -0.09550692751667        3.02137360970940\\nH            4.28499881189601        0.59053354186487        3.80828565729918\\nH            5.85017872416996        1.43051729090435        3.93177433979190\\n43\\n energy: -70.312714130899 xtb: 6.7.0 (08769fc)\\nC           -2.82750163866460        0.50608259405366        2.61734142176580\\nC           -2.40125009256833        0.02195447938857        1.26694975397809\\nC           -1.08102753894729       -0.31283969535490        1.00244600155823\\nN           -0.65859872315572       -0.75578683796452       -0.16273021297888\\nC           -1.51764792844188       -0.91541776424953       -1.15551610733436\\nC           -2.88026977862352       -0.62913078357388       -1.01943841736674\\nC           -3.30921288920398       -0.13457731512092        0.21518262284686\\nO           -4.64798268774473        0.15867109563050        0.40243646647855\\nC           -4.96486670430089        1.52658640238983        0.25080725770606\\nC           -3.85396229438252       -0.83366728858993       -2.13854337101335\\nC           -0.94457812059183       -1.43226865280293       -2.43192923371371\\nS            0.49344242313019       -2.57940173088813       -2.14119390961123\\nO            0.16663751362642       -3.47173730976022       -1.02558607319615\\nC            1.54438235631560       -1.32599181689772       -1.40372425861583\\nN            1.93783690986636       -0.21802752723132       -1.95643301021840\\nC            2.77650554671736        0.35750697207248       -1.03173339058608\\nC            2.88205282040001       -0.48508773591845        0.10092854725702\\nN            2.10445857033609       -1.57132181610202       -0.18841332158010\\nC            3.65751053344230       -0.16235255928641        1.20498708859887\\nC            4.33877415922788        1.04320261864982        1.15643378075988\\nC            4.23998252658630        1.88579573773055        0.03263239468929\\nC            3.47191223990849        1.56272702174244       -1.05956601544422\\nO            5.14060996143466        1.51713663815327        2.15429715931935\\nC            5.30195843912823        0.73596040966582        3.31120968787342\\nH           -2.63451509141313        1.57498742330531        2.71890125761840\\nH           -2.26812527336553       -0.00962153217998        3.39476752836570\\nH           -3.89010388787651        0.32662569415311        2.76384274745532\\nH           -0.32285960787629       -0.22028078171129        1.76952413106402\\nH           -4.41168586764573        2.15282326095969        0.95728862582929\\nH           -6.03187972813161        1.61228382869255        0.44835123319029\\nH           -4.75243765465141        1.87229102806227       -0.76717939097339\\nH           -3.85863427780361       -1.87709013473890       -2.45097585442679\\nH           -3.58834297366748       -0.22034669873036       -2.99889315149488\\nH           -4.85617118094804       -0.57485065524337       -1.80830698410661\\nH           -1.67593347678482       -1.98380007921468       -3.01848413531186\\nH           -0.51130653202912       -0.63081516670327       -3.03009902910219\\nH            1.84157884904770       -2.34818086302487        0.39613330290880\\nH            3.71213361864257       -0.83089262011579        2.04905238000462\\nH            4.79842879628710        2.80795440300690        0.06244903110905\\nH            3.40559890470569        2.21629776710082       -1.91380740539274\\nH            5.75214228550932       -0.23564319991165        3.08181915826846\\nH            4.34930929459903        0.58113564179143        3.82901000416848\\nH            5.97363919985238        1.29910654875583        3.95728868963282\\n43\\n energy: -70.313238721527 xtb: 6.7.0 (08769fc)\\nC           -2.86601457935967        0.60522761689388        2.59651975151356\\nC           -2.42895094035397        0.14354819015017        1.24180096914379\\nC           -1.08730981064931       -0.03620204031647        0.93545565123853\\nN           -0.65428431009519       -0.44860941775795       -0.23645968398327\\nC           -1.52199979753578       -0.73164271591999       -1.19585056149025\\nC           -2.90392091929353       -0.61674735342148       -1.01227901002660\\nC           -3.34614945236225       -0.14804208784210        0.22809934167290\\nO           -4.70343593008851       -0.01688198672063        0.46253480112777\\nC           -5.19422717151929        1.29353681980918        0.27340808766360\\nC           -3.88922253367597       -0.97673496049927       -2.08162086432526\\nC           -0.92322789731725       -1.21753886333191       -2.47045346466034\\nS            0.43597658898400       -2.45403969144096       -2.14038117371761\\nO            0.03144081474712       -3.29599226295049       -1.01017674889020\\nC            1.55568783648107       -1.26082948384408       -1.40731510555452\\nN            2.05694747846870       -0.21046695939350       -1.98446155964919\\nC            2.91018752558576        0.32944744452864       -1.05262355985291\\nC            2.90939461053384       -0.47489659489413        0.11252127271120\\nN            2.05150700135111       -1.50229640064525       -0.16366290648566\\nC            3.67863471535865       -0.17779835092358        1.22826726698564\\nC            4.46374128384150        0.96164214928559        1.15754422220203\\nC            4.46851417469008        1.76749746053956        0.00249204713668\\nC            3.70639807272576        1.47021952655715       -1.10081516586199\\nO            5.27909291270700        1.40060935760399        2.16015451063651\\nC            5.35237578358428        0.64526627681416        3.34295088289476\\nH           -2.80031270680471        1.69168872477569        2.66957670294964\\nH           -2.22387063198214        0.17933372637587        3.36431477233558\\nH           -3.89404517481438        0.30362366554550        2.78327556901448\\nH           -0.31963848191446        0.16281370984993        1.67284291898316\\nH           -4.70113680471841        2.01103425261264        0.93657267335432\\nH           -6.25671392152358        1.25449570036075        0.50700026400391\\nH           -5.05955587253780        1.62236333306013       -0.76332387438446\\nH           -3.75861871476575       -2.01530204872453       -2.38200153005836\\nH           -3.75835873130655       -0.34471028271917       -2.95951215894789\\nH           -4.90115319935400       -0.85868907716551       -1.70391348933865\\nH           -1.64774761349941       -1.70460495379559       -3.11699397055122\\nH           -0.41011962312673       -0.42001005551961       -3.00622958315296\\nH            1.71377802506901       -2.23750905953281        0.43622229935482\\nH            3.65210276504182       -0.81614955904504        2.09673264223793\\nH            5.10327469817618        2.63918636260450        0.01756353463861\\nH            3.71973828235914        2.09444623807248       -1.97898128871360\\nH            5.72084195223414       -0.36810892361104        3.15139735081101\\nH            4.38125747826329        0.58907998463974        3.84634943012152\\nH            6.05912381834797        1.16874158992195        3.98498573689257\\n43\\n energy: -70.313984803447 xtb: 6.7.0 (08769fc)\\nC           -2.82458690163836        0.74664675574345        2.56944543140153\\nC           -2.39941762312272        0.29810572461804        1.20656840161750\\nC           -1.06313544969161        0.28568465163446        0.83007416894524\\nN           -0.64059092136372       -0.10345247598539       -0.35349494754509\\nC           -1.51411528009518       -0.52780292915190       -1.25580409049432\\nC           -2.88630021337848       -0.59941304554369       -0.99392110989694\\nC           -3.32108179750041       -0.14986836660538        0.25616059365897\\nO           -4.66874807750815       -0.19451251481919        0.56760955837788\\nC           -5.34804729068750        1.02244827694044        0.34195636345262\\nC           -3.86858207346935       -1.14170209203231       -1.98666318550689\\nC           -0.91736141695357       -0.99095843670578       -2.53753304890618\\nS            0.36493093568766       -2.31268769066262       -2.20196264929622\\nO           -0.10672461386526       -3.12192006983362       -1.07258017105802\\nC            1.54720187631718       -1.19320721889769       -1.45401845168382\\nN            2.16494217054867       -0.21326136537538       -2.04177931528130\\nC            3.01786477590199        0.28712823931983       -1.08906081333510\\nC            2.89211452323388       -0.46706382863532        0.10273358831293\\nN            1.95856703993482       -1.42463790282830       -0.17792740213415\\nC            3.63478355305919       -0.19516336759909        1.24290735655865\\nC            4.52329839022494        0.86524231565501        1.16846187936951\\nC            4.65272567848856        1.62087739671504       -0.01340348101308\\nC            3.91611604953210        1.34964335578763       -1.14006366082738\\nO            5.33127854194055        1.26874607433412        2.19129703743210\\nC            5.28572070283924        0.55708355028554        3.40232648776666\\nH           -2.90364269633961        1.83415909211409        2.60573004204338\\nH           -2.09220004990918        0.43973726831386        3.31303443056258\\nH           -3.79164307422525        0.31746887226023        2.82160197521245\\nH           -0.29069214638220        0.60466595198227        1.51863112288891\\nH           -4.93096167390057        1.83628506526275        0.94369753685054\\nH           -6.38266872841637        0.84950620207722        0.63306233279527\\nH           -5.30884510784977        1.31039069802594       -0.71481774422399\\nH           -3.59343927638945       -2.15606519294735       -2.27138100978174\\nH           -3.89799565905293       -0.52440271632518       -2.88460239736293\\nH           -4.86021105067135       -1.17132041126048       -1.54289720370880\\nH           -1.64593517822497       -1.41720339016162       -3.21984563992115\\nH           -0.34411160086636       -0.20514086948412       -3.02582968761681\\nH            1.53929217081165       -2.11243194510604        0.42699060580073\\nH            3.51172195861683       -0.79312481663040        2.13162166352331\\nH            5.36396411249464        2.43145891863043        0.00077317274751\\nH            4.02542203433906        1.93394803165696       -2.03878912991878\\nH            5.56678122138315       -0.49224784879611        3.26349158614079\\nH            4.29362487753718        0.60905800431587        3.86348494894568\\nH            6.01068828856508        1.03930304969650        4.05624185508910\\n43\\n energy: -70.315116075923 xtb: 6.7.0 (08769fc)\\nC           -2.64802044201848        0.93170096056360        2.52962189835666\\nC           -2.27205764301831        0.48549029165145        1.15163266498887\\nC           -0.98316427461906        0.64622503967923        0.66045569317645\\nN           -0.60666003929409        0.27139473779624       -0.54372283878312\\nC           -1.48393799178493       -0.30784670080953       -1.35263134233202\\nC           -2.80169549541912       -0.57373337401973       -0.96476504325054\\nC           -3.19317449926499       -0.13533724556809        0.30292928469507\\nO           -4.48698290018787       -0.36283398345797        0.73868018440001\\nC           -5.36664574760196        0.71472419954235        0.49842060459545\\nC           -3.76068565027466       -1.31896226004599       -1.84104479569765\\nC           -0.93290678540048       -0.75921258746651       -2.65661519975306\\nS            0.27464130096943       -2.16317695037444       -2.35769045628454\\nO           -0.26156316451936       -2.96718354912282       -1.25195930969402\\nC            1.50744908989183       -1.13515540762400       -1.56447918930014\\nN            2.25370398554198       -0.24037181245244       -2.13802859135162\\nC            3.08001760847912        0.21940215090205       -1.14342934523270\\nC            2.79922258980407       -0.46831915554050        0.06210121079571\\nN            1.80061744566279       -1.34609399814501       -0.25236969990655\\nC            3.47977176253879       -0.21507205037151        1.24473099699681\\nC            4.46808160699242        0.75449219911398        1.19807407611978\\nC            4.75511545123848        1.44060301622954        0.00128415964642\\nC            4.07836434867464        1.18994582115800       -1.16650927335919\\nO            5.23375715059687        1.12823797573165        2.26342173295577\\nC            5.02173406315536        0.49143508486568        3.49810307914753\\nH           -2.90014599841547        1.99314381517720        2.53062765456509\\nH           -1.81615828037052        0.78473738299985        3.21533552447621\\nH           -3.50891944329789        0.36894465511865        2.88390736132270\\nH           -0.20983402511828        1.09729864801540        1.26939894809275\\nH           -5.04239376778926        1.62324551724239        1.01636919772963\\nH           -6.33680706685236        0.40490383979626        0.88288198721619\\nH           -5.44997962556549        0.93072487687624       -0.57267734869191\\nH           -3.33659636647791       -2.28295443759118       -2.11704252139499\\nH           -3.97811170553079       -0.75929332496558       -2.75116959224314\\nH           -4.68840282376726       -1.49592779417764       -1.30321763856778\\nH           -1.68896895778337       -1.12879840768630       -3.34150015758983\\nH           -0.32122427631517        0.00822224474016       -3.12524080742151\\nH            1.29065469335614       -1.98190071334062        0.33991988426347\\nH            3.23607482271700       -0.75864497210242        2.14336116610998\\nH            5.53876989789793        2.18058076907163        0.03903190150979\\nH            4.30704749717229        1.72077088592023       -2.07583556099444\\nH            5.20714552989856       -0.58583765714319        3.43291909784414\\nH            4.00619317322484        0.66297396007840        3.87052818982573\\nH            5.73667595282819        0.93745730971734        4.18771921300251\\n\",\"xyz\");\n",
              "\tviewer_17200165783103456.setStyle({\"stick\": {\"colorscheme\": \"greenCarbon\"}});\n",
              "\tviewer_17200165783103456.zoomTo();\n",
              "\tviewer_17200165783103456.animate({\"loop\": \"backAndForth\"});\n",
              "viewer_17200165783103456.render();\n",
              "});\n",
              "</script>"
            ]
          },
          "metadata": {}
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [],
      "metadata": {
        "id": "xws-Y49xiJpd"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [],
      "metadata": {
        "id": "-kfjYvLM2rb4"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [],
      "metadata": {
        "id": "agenifTM2rfC"
      },
      "execution_count": null,
      "outputs": []
    }
  ]
}